# Comparing `tmp/bioregistry-0.9.90.tar.gz` & `tmp/bioregistry-0.9.91.tar.gz`

## filetype from file(1)

```diff
@@ -1 +1 @@
-gzip compressed data, was "bioregistry-0.9.90.tar", last modified: Sun Jul 30 01:29:58 2023, max compression
+gzip compressed data, was "bioregistry-0.9.91.tar", last modified: Tue Aug  1 01:35:54 2023, max compression
```

## Comparing `bioregistry-0.9.90.tar` & `bioregistry-0.9.91.tar`

### file list

```diff
@@ -1,345 +1,345 @@
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.681258 bioregistry-0.9.90/
--rw-r--r--   0 runner    (1001) docker     (123)     1081 2023-07-30 00:50:39.000000 bioregistry-0.9.90/LICENSE
--rw-r--r--   0 runner    (1001) docker     (123)      630 2023-07-30 00:50:39.000000 bioregistry-0.9.90/MANIFEST.in
--rw-r--r--   0 runner    (1001) docker     (123)    31461 2023-07-30 01:29:58.681258 bioregistry-0.9.90/PKG-INFO
--rw-r--r--   0 runner    (1001) docker     (123)    30194 2023-07-30 00:50:39.000000 bioregistry-0.9.90/README.md
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.597257 bioregistry-0.9.90/docs/
--rw-r--r--   0 runner    (1001) docker     (123)     2968 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/bulk_prefix_request_template.tsv
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.601257 bioregistry-0.9.90/docs/img/
--rw-r--r--   0 runner    (1001) docker     (123)    47424 2023-07-30 01:16:15.000000 bioregistry-0.9.90/docs/img/bibliography_years.svg
--rw-r--r--   0 runner    (1001) docker     (123)   157399 2023-07-30 01:16:34.000000 bioregistry-0.9.90/docs/img/bioregistry_coverage.svg
--rw-r--r--   0 runner    (1001) docker     (123)   107131 2023-07-30 01:16:33.000000 bioregistry-0.9.90/docs/img/bioregistry_coverage_bar.svg
--rw-r--r--   0 runner    (1001) docker     (123)    84716 2023-07-30 01:16:34.000000 bioregistry-0.9.90/docs/img/bioregistry_coverage_bar_short.svg
--rw-r--r--   0 runner    (1001) docker     (123)    77372 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/img/datamodel_umls.svg
--rw-r--r--   0 runner    (1001) docker     (123)  1380510 2023-07-30 01:20:20.000000 bioregistry-0.9.90/docs/img/external_overlap.svg
--rw-r--r--   0 runner    (1001) docker     (123)    71970 2023-07-30 01:17:00.000000 bioregistry-0.9.90/docs/img/has_attribute.svg
--rw-r--r--   0 runner    (1001) docker     (123)    28400 2023-07-30 01:20:21.000000 bioregistry-0.9.90/docs/img/providers.svg
--rw-r--r--   0 runner    (1001) docker     (123)    22344 2023-07-30 01:20:22.000000 bioregistry-0.9.90/docs/img/regex_report.svg
--rw-r--r--   0 runner    (1001) docker     (123)    51349 2023-07-30 01:16:32.000000 bioregistry-0.9.90/docs/img/xrefs.svg
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.605257 bioregistry-0.9.90/docs/source/
--rw-r--r--   0 runner    (1001) docker     (123)      121 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/source/alignment.rst
--rw-r--r--   0 runner    (1001) docker     (123)      236 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/source/cli.rst
--rw-r--r--   0 runner    (1001) docker     (123)     7034 2023-07-30 01:29:49.000000 bioregistry-0.9.90/docs/source/conf.py
--rw-r--r--   0 runner    (1001) docker     (123)     8480 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/source/deployment.rst
--rw-r--r--   0 runner    (1001) docker     (123)     2212 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/source/index.rst
--rw-r--r--   0 runner    (1001) docker     (123)   214468 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/source/logo.png
--rw-r--r--   0 runner    (1001) docker     (123)       97 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/source/pandas.rst
--rw-r--r--   0 runner    (1001) docker     (123)      225 2023-07-30 00:50:39.000000 bioregistry-0.9.90/docs/source/reference.rst
--rw-r--r--   0 runner    (1001) docker     (123)      467 2023-07-30 00:50:40.000000 bioregistry-0.9.90/pyproject.toml
--rw-r--r--   0 runner    (1001) docker     (123)     2847 2023-07-30 01:29:58.685258 bioregistry-0.9.90/setup.cfg
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.593257 bioregistry-0.9.90/src/
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.605257 bioregistry-0.9.90/src/bioregistry/
--rw-r--r--   0 runner    (1001) docker     (123)     3125 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)      137 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/__main__.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.609257 bioregistry-0.9.90/src/bioregistry/align/
--rw-r--r--   0 runner    (1001) docker     (123)     1726 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)      428 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/aberowl.py
--rw-r--r--   0 runner    (1001) docker     (123)      172 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/agroportal.py
--rw-r--r--   0 runner    (1001) docker     (123)      454 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/bartoc.py
--rw-r--r--   0 runner    (1001) docker     (123)     1384 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/biocontext.py
--rw-r--r--   0 runner    (1001) docker     (123)     1717 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/biolink.py
--rw-r--r--   0 runner    (1001) docker     (123)      932 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/bioportal.py
--rw-r--r--   0 runner    (1001) docker     (123)      762 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/cellosaurus.py
--rw-r--r--   0 runner    (1001) docker     (123)      804 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/cheminf.py
--rw-r--r--   0 runner    (1001) docker     (123)      424 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/cropoct.py
--rw-r--r--   0 runner    (1001) docker     (123)      169 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/ecoportal.py
--rw-r--r--   0 runner    (1001) docker     (123)      785 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/edam.py
--rw-r--r--   0 runner    (1001) docker     (123)      530 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/fairsharing.py
--rw-r--r--   0 runner    (1001) docker     (123)     2544 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/go.py
--rw-r--r--   0 runner    (1001) docker     (123)     1030 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/hl7.py
--rw-r--r--   0 runner    (1001) docker     (123)      453 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/miriam.py
--rw-r--r--   0 runner    (1001) docker     (123)      521 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/n2t.py
--rw-r--r--   0 runner    (1001) docker     (123)      824 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/ncbi.py
--rw-r--r--   0 runner    (1001) docker     (123)     1219 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/obofoundry.py
--rw-r--r--   0 runner    (1001) docker     (123)      545 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/ols.py
--rw-r--r--   0 runner    (1001) docker     (123)      743 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/ontobee.py
--rw-r--r--   0 runner    (1001) docker     (123)     3017 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/prefixcommons.py
--rw-r--r--   0 runner    (1001) docker     (123)      533 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/re3data.py
--rw-r--r--   0 runner    (1001) docker     (123)      529 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/uniprot.py
--rw-r--r--   0 runner    (1001) docker     (123)    10718 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/utils.py
--rw-r--r--   0 runner    (1001) docker     (123)      769 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/align/wikidata.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.609257 bioregistry-0.9.90/src/bioregistry/analysis/
--rw-r--r--   0 runner    (1001) docker     (123)       66 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/analysis/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)     5007 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/analysis/title_tfidf.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.613257 bioregistry-0.9.90/src/bioregistry/app/
--rw-r--r--   0 runner    (1001) docker     (123)       68 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)    17127 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/api.py
--rw-r--r--   0 runner    (1001) docker     (123)     1946 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/cli.py
--rw-r--r--   0 runner    (1001) docker     (123)     1921 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/constants.py
--rw-r--r--   0 runner    (1001) docker     (123)    10179 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/impl.py
--rw-r--r--   0 runner    (1001) docker     (123)      248 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/proxies.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.613257 bioregistry-0.9.90/src/bioregistry/app/static/
--rw-r--r--   0 runner    (1001) docker     (123)   801449 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/static/logo.svg
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.613257 bioregistry-0.9.90/src/bioregistry/app/templates/
--rw-r--r--   0 runner    (1001) docker     (123)     9907 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/base.html
--rw-r--r--   0 runner    (1001) docker     (123)     3162 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/collection.html
--rw-r--r--   0 runner    (1001) docker     (123)     3318 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/collections.html
--rw-r--r--   0 runner    (1001) docker     (123)     6758 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/context.html
--rw-r--r--   0 runner    (1001) docker     (123)     2387 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/contexts.html
--rw-r--r--   0 runner    (1001) docker     (123)     7134 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/contributor.html
--rw-r--r--   0 runner    (1001) docker     (123)     4670 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/contributors.html
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.617257 bioregistry-0.9.90/src/bioregistry/app/templates/highlights/
--rw-r--r--   0 runner    (1001) docker     (123)      825 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/highlights/keywords.html
--rw-r--r--   0 runner    (1001) docker     (123)     1083 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/highlights/owners.html
--rw-r--r--   0 runner    (1001) docker     (123)     1258 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/highlights/relations.html
--rw-r--r--   0 runner    (1001) docker     (123)      869 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/highlights/twitter.html
--rw-r--r--   0 runner    (1001) docker     (123)    15622 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/home.html
--rw-r--r--   0 runner    (1001) docker     (123)     7351 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/macros.html
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.617257 bioregistry-0.9.90/src/bioregistry/app/templates/meta/
--rw-r--r--   0 runner    (1001) docker     (123)     5577 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/meta/access.html
--rw-r--r--   0 runner    (1001) docker     (123)    19835 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/meta/acknowledgements.html
--rw-r--r--   0 runner    (1001) docker     (123)    10695 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/meta/download.html
--rw-r--r--   0 runner    (1001) docker     (123)    16544 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/meta/related.html
--rw-r--r--   0 runner    (1001) docker     (123)     1618 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/meta/schema.html
--rw-r--r--   0 runner    (1001) docker     (123)    19312 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/meta/summary.html
--rw-r--r--   0 runner    (1001) docker     (123)     8813 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/meta/sustainability.html
--rw-r--r--   0 runner    (1001) docker     (123)     8474 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/metaresource.html
--rw-r--r--   0 runner    (1001) docker     (123)     2303 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/metaresources.html
--rw-r--r--   0 runner    (1001) docker     (123)     1100 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/prose.html
--rw-r--r--   0 runner    (1001) docker     (123)     2437 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/reference.html
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.617257 bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/
--rw-r--r--   0 runner    (1001) docker     (123)      520 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/disallowed_identifier.html
--rw-r--r--   0 runner    (1001) docker     (123)     2040 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/invalid_identifier.html
--rw-r--r--   0 runner    (1001) docker     (123)     1516 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/missing_prefix.html
--rw-r--r--   0 runner    (1001) docker     (123)     1307 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/missing_providers.html
--rw-r--r--   0 runner    (1001) docker     (123)    33327 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/resource.html
--rw-r--r--   0 runner    (1001) docker     (123)     3765 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/templates/resources.html
--rw-r--r--   0 runner    (1001) docker     (123)     2042 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/test.py
--rw-r--r--   0 runner    (1001) docker     (123)    19495 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/ui.py
--rw-r--r--   0 runner    (1001) docker     (123)     8073 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/utils.py
--rw-r--r--   0 runner    (1001) docker     (123)      223 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/app/wsgi.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.617257 bioregistry-0.9.90/src/bioregistry/benchmarks/
--rw-r--r--   0 runner    (1001) docker     (123)       38 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/benchmarks/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)      117 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/benchmarks/__main__.py
--rw-r--r--   0 runner    (1001) docker     (123)      615 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/benchmarks/cli.py
--rw-r--r--   0 runner    (1001) docker     (123)     3197 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/benchmarks/curie_parsing.py
--rw-r--r--   0 runner    (1001) docker     (123)     4436 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/benchmarks/curie_validation.py
--rw-r--r--   0 runner    (1001) docker     (123)     2487 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/benchmarks/uri_parsing.py
--rw-r--r--   0 runner    (1001) docker     (123)     2141 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/bibliometrics.py
--rw-r--r--   0 runner    (1001) docker     (123)     2796 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/cli.py
--rw-r--r--   0 runner    (1001) docker     (123)      539 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/collection_api.py
--rw-r--r--   0 runner    (1001) docker     (123)    23802 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/compare.py
--rw-r--r--   0 runner    (1001) docker     (123)     5293 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/constants.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.621257 bioregistry-0.9.90/src/bioregistry/curation/
--rw-r--r--   0 runner    (1001) docker     (123)       47 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)     1158 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/add_co_providers.py
--rw-r--r--   0 runner    (1001) docker     (123)      940 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/add_descriptions_from_gs.py
--rw-r--r--   0 runner    (1001) docker     (123)     1299 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/add_examples.py
--rw-r--r--   0 runner    (1001) docker     (123)     1106 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/add_examples_from_javert.py
--rw-r--r--   0 runner    (1001) docker     (123)     2278 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/add_examples_from_ols.py
--rw-r--r--   0 runner    (1001) docker     (123)     1606 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/add_ontology_regexes.py
--rw-r--r--   0 runner    (1001) docker     (123)     2667 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/bulk_import.py
--rw-r--r--   0 runner    (1001) docker     (123)      619 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/clean_licenses.py
--rw-r--r--   0 runner    (1001) docker     (123)      679 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/clean_name_suffixes.py
--rw-r--r--   0 runner    (1001) docker     (123)      704 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/clean_publications.py
--rw-r--r--   0 runner    (1001) docker     (123)     2026 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/cleanup_authors.py
--rw-r--r--   0 runner    (1001) docker     (123)     1572 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/deprecation_diff.py
--rw-r--r--   0 runner    (1001) docker     (123)     5230 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/enrich_publications.py
--rw-r--r--   0 runner    (1001) docker     (123)     3389 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/import_pc_semiautomatic.py
--rw-r--r--   0 runner    (1001) docker     (123)     2774 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/make_description_curation_sheet.py
--rw-r--r--   0 runner    (1001) docker     (123)      860 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/map_bartoc_via_wikidata.py
--rw-r--r--   0 runner    (1001) docker     (123)      789 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/map_re3data_via_fairsharing.py
--rw-r--r--   0 runner    (1001) docker     (123)      868 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/rename_metaprefix.py
--rw-r--r--   0 runner    (1001) docker     (123)     2165 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/review_pc.py
--rw-r--r--   0 runner    (1001) docker     (123)     1017 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/suggest_author_curation.py
--rw-r--r--   0 runner    (1001) docker     (123)      827 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/curation/suggest_uniprot_providers.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.625257 bioregistry-0.9.90/src/bioregistry/data/
--rw-r--r--   0 runner    (1001) docker     (123)     2760 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/README.md
--rw-r--r--   0 runner    (1001) docker     (123)       56 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)  4581985 2023-07-30 01:14:08.000000 bioregistry-0.9.90/src/bioregistry/data/bioregistry.json
--rw-r--r--   0 runner    (1001) docker     (123)    12623 2023-07-30 01:14:08.000000 bioregistry-0.9.90/src/bioregistry/data/collections.json
--rw-r--r--   0 runner    (1001) docker     (123)     1300 2023-07-30 01:14:08.000000 bioregistry-0.9.90/src/bioregistry/data/contexts.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.597257 bioregistry-0.9.90/src/bioregistry/data/external/
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.629257 bioregistry-0.9.90/src/bioregistry/data/external/aberowl/
--rw-r--r--   0 runner    (1001) docker     (123)   307459 2023-07-30 01:14:02.000000 bioregistry-0.9.90/src/bioregistry/data/external/aberowl/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)   634433 2023-07-30 01:14:02.000000 bioregistry-0.9.90/src/bioregistry/data/external/aberowl/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)  1305869 2023-07-30 01:13:55.000000 bioregistry-0.9.90/src/bioregistry/data/external/aberowl/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.629257 bioregistry-0.9.90/src/bioregistry/data/external/agroportal/
--rw-r--r--   0 runner    (1001) docker     (123)    43126 2023-07-30 01:01:30.000000 bioregistry-0.9.90/src/bioregistry/data/external/agroportal/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)   127378 2023-07-30 01:01:30.000000 bioregistry-0.9.90/src/bioregistry/data/external/agroportal/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   677787 2023-07-30 01:01:30.000000 bioregistry-0.9.90/src/bioregistry/data/external/agroportal/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.633257 bioregistry-0.9.90/src/bioregistry/data/external/bartoc/
--rw-r--r--   0 runner    (1001) docker     (123)  1832614 2023-07-30 00:53:53.000000 bioregistry-0.9.90/src/bioregistry/data/external/bartoc/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)  2217991 2023-07-30 00:53:53.000000 bioregistry-0.9.90/src/bioregistry/data/external/bartoc/processed.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.637257 bioregistry-0.9.90/src/bioregistry/data/external/biocontext/
--rw-r--r--   0 runner    (1001) docker     (123)     4200 2023-07-30 00:51:33.000000 bioregistry-0.9.90/src/bioregistry/data/external/biocontext/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    74913 2023-07-30 00:51:33.000000 bioregistry-0.9.90/src/bioregistry/data/external/biocontext/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)    55878 2023-07-30 00:51:33.000000 bioregistry-0.9.90/src/bioregistry/data/external/biocontext/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.637257 bioregistry-0.9.90/src/bioregistry/data/external/biolink/
--rw-r--r--   0 runner    (1001) docker     (123)     1169 2023-07-30 01:06:51.000000 bioregistry-0.9.90/src/bioregistry/data/external/biolink/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    10406 2023-07-30 01:06:50.000000 bioregistry-0.9.90/src/bioregistry/data/external/biolink/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   356466 2023-07-30 01:06:49.000000 bioregistry-0.9.90/src/bioregistry/data/external/biolink/raw.yaml
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.637257 bioregistry-0.9.90/src/bioregistry/data/external/bioportal/
--rw-r--r--   0 runner    (1001) docker     (123)   258394 2023-07-30 01:06:47.000000 bioregistry-0.9.90/src/bioregistry/data/external/bioportal/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)   618236 2023-07-30 01:06:47.000000 bioregistry-0.9.90/src/bioregistry/data/external/bioportal/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)  3607480 2023-07-30 01:06:47.000000 bioregistry-0.9.90/src/bioregistry/data/external/bioportal/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.641257 bioregistry-0.9.90/src/bioregistry/data/external/cellosaurus/
--rw-r--r--   0 runner    (1001) docker     (123)     2722 2023-07-30 01:13:48.000000 bioregistry-0.9.90/src/bioregistry/data/external/cellosaurus/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    29294 2023-07-30 01:13:48.000000 bioregistry-0.9.90/src/bioregistry/data/external/cellosaurus/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)    33204 2023-07-30 01:13:48.000000 bioregistry-0.9.90/src/bioregistry/data/external/cellosaurus/raw.txt
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.641257 bioregistry-0.9.90/src/bioregistry/data/external/cheminf/
--rw-r--r--   0 runner    (1001) docker     (123)     2885 2023-07-30 01:00:16.000000 bioregistry-0.9.90/src/bioregistry/data/external/cheminf/processed.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.645257 bioregistry-0.9.90/src/bioregistry/data/external/cropoct/
--rw-r--r--   0 runner    (1001) docker     (123)      539 2023-07-30 01:14:05.000000 bioregistry-0.9.90/src/bioregistry/data/external/cropoct/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    11713 2023-07-30 01:14:05.000000 bioregistry-0.9.90/src/bioregistry/data/external/cropoct/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)    26054 2023-07-30 01:14:05.000000 bioregistry-0.9.90/src/bioregistry/data/external/cropoct/raw.yaml
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.645257 bioregistry-0.9.90/src/bioregistry/data/external/ecoportal/
--rw-r--r--   0 runner    (1001) docker     (123)     9252 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/external/ecoportal/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    18869 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/external/ecoportal/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)    98246 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/external/ecoportal/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.645257 bioregistry-0.9.90/src/bioregistry/data/external/edam/
--rw-r--r--   0 runner    (1001) docker     (123)    14599 2023-07-30 00:59:58.000000 bioregistry-0.9.90/src/bioregistry/data/external/edam/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    45616 2023-07-30 00:59:57.000000 bioregistry-0.9.90/src/bioregistry/data/external/edam/processed.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.645257 bioregistry-0.9.90/src/bioregistry/data/external/fairsharing/
--rw-r--r--   0 runner    (1001) docker     (123)      235 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/external/fairsharing/README.md
--rw-r--r--   0 runner    (1001) docker     (123)   246096 2023-07-30 00:59:44.000000 bioregistry-0.9.90/src/bioregistry/data/external/fairsharing/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)   738834 2023-07-30 00:59:44.000000 bioregistry-0.9.90/src/bioregistry/data/external/fairsharing/processed.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.649258 bioregistry-0.9.90/src/bioregistry/data/external/go/
--rw-r--r--   0 runner    (1001) docker     (123)     7556 2023-07-30 00:53:56.000000 bioregistry-0.9.90/src/bioregistry/data/external/go/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)   143532 2023-07-30 00:53:56.000000 bioregistry-0.9.90/src/bioregistry/data/external/go/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   115433 2023-07-30 00:53:55.000000 bioregistry-0.9.90/src/bioregistry/data/external/go/raw.yml
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.649258 bioregistry-0.9.90/src/bioregistry/data/external/hl7/
--rw-r--r--   0 runner    (1001) docker     (123)   157660 2023-07-30 01:14:07.000000 bioregistry-0.9.90/src/bioregistry/data/external/hl7/curation.tsv
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.649258 bioregistry-0.9.90/src/bioregistry/data/external/miriam/
--rw-r--r--   0 runner    (1001) docker     (123)   579119 2023-07-30 01:00:09.000000 bioregistry-0.9.90/src/bioregistry/data/external/miriam/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)  2211982 2023-07-30 01:00:09.000000 bioregistry-0.9.90/src/bioregistry/data/external/miriam/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.653257 bioregistry-0.9.90/src/bioregistry/data/external/n2t/
--rw-r--r--   0 runner    (1001) docker     (123)   425371 2023-07-30 01:00:05.000000 bioregistry-0.9.90/src/bioregistry/data/external/n2t/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)  1290185 2023-07-30 01:00:00.000000 bioregistry-0.9.90/src/bioregistry/data/external/n2t/raw.yml
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.653257 bioregistry-0.9.90/src/bioregistry/data/external/ncbi/
--rw-r--r--   0 runner    (1001) docker     (123)     3748 2023-07-30 01:07:58.000000 bioregistry-0.9.90/src/bioregistry/data/external/ncbi/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    18072 2023-07-30 01:07:58.000000 bioregistry-0.9.90/src/bioregistry/data/external/ncbi/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   103465 2023-07-30 01:07:58.000000 bioregistry-0.9.90/src/bioregistry/data/external/ncbi/raw.html
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.653257 bioregistry-0.9.90/src/bioregistry/data/external/obofoundry/
--rw-r--r--   0 runner    (1001) docker     (123)   216819 2023-07-30 01:00:13.000000 bioregistry-0.9.90/src/bioregistry/data/external/obofoundry/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   359237 2023-07-30 01:00:12.000000 bioregistry-0.9.90/src/bioregistry/data/external/obofoundry/raw.yaml
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.657258 bioregistry-0.9.90/src/bioregistry/data/external/ols/
--rw-r--r--   0 runner    (1001) docker     (123)   126801 2023-07-30 01:07:55.000000 bioregistry-0.9.90/src/bioregistry/data/external/ols/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   559805 2023-07-30 01:07:55.000000 bioregistry-0.9.90/src/bioregistry/data/external/ols/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.657258 bioregistry-0.9.90/src/bioregistry/data/external/ontobee/
--rw-r--r--   0 runner    (1001) docker     (123)     1826 2023-07-30 00:51:31.000000 bioregistry-0.9.90/src/bioregistry/data/external/ontobee/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    23768 2023-07-30 00:51:31.000000 bioregistry-0.9.90/src/bioregistry/data/external/ontobee/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   192862 2023-07-30 00:51:30.000000 bioregistry-0.9.90/src/bioregistry/data/external/ontobee/raw.html
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.657258 bioregistry-0.9.90/src/bioregistry/data/external/prefixcommons/
--rw-r--r--   0 runner    (1001) docker     (123)   111567 2023-07-30 00:51:36.000000 bioregistry-0.9.90/src/bioregistry/data/external/prefixcommons/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)   504428 2023-07-30 00:51:36.000000 bioregistry-0.9.90/src/bioregistry/data/external/prefixcommons/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)   883990 2023-07-30 00:51:36.000000 bioregistry-0.9.90/src/bioregistry/data/external/prefixcommons/raw.tsv
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.661258 bioregistry-0.9.90/src/bioregistry/data/external/re3data/
--rw-r--r--   0 runner    (1001) docker     (123)  1757259 2023-07-30 01:13:46.000000 bioregistry-0.9.90/src/bioregistry/data/external/re3data/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)  2460067 2023-07-30 01:13:45.000000 bioregistry-0.9.90/src/bioregistry/data/external/re3data/processed.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.665258 bioregistry-0.9.90/src/bioregistry/data/external/uniprot/
--rw-r--r--   0 runner    (1001) docker     (123)    10752 2023-07-30 00:59:54.000000 bioregistry-0.9.90/src/bioregistry/data/external/uniprot/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)    67204 2023-07-30 00:59:53.000000 bioregistry-0.9.90/src/bioregistry/data/external/uniprot/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)    89578 2023-07-30 00:59:53.000000 bioregistry-0.9.90/src/bioregistry/data/external/uniprot/raw.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.665258 bioregistry-0.9.90/src/bioregistry/data/external/wikidata/
--rw-r--r--   0 runner    (1001) docker     (123)   141159 2023-07-30 00:59:51.000000 bioregistry-0.9.90/src/bioregistry/data/external/wikidata/curation.tsv
--rw-r--r--   0 runner    (1001) docker     (123)   329972 2023-07-30 00:59:50.000000 bioregistry-0.9.90/src/bioregistry/data/external/wikidata/processed.json
--rw-r--r--   0 runner    (1001) docker     (123)    47294 2023-07-30 01:14:08.000000 bioregistry-0.9.90/src/bioregistry/data/metaregistry.json
--rw-r--r--   0 runner    (1001) docker     (123)     2047 2023-07-30 01:14:08.000000 bioregistry-0.9.90/src/bioregistry/data/mismatch.json
--rw-r--r--   0 runner    (1001) docker     (123)      508 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/processing_biolink.json
--rw-r--r--   0 runner    (1001) docker     (123)      137 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/processing_go.json
--rw-r--r--   0 runner    (1001) docker     (123)    32627 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/processing_ols.json
--rw-r--r--   0 runner    (1001) docker     (123)       18 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/data/processing_wikidata.json
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.669258 bioregistry-0.9.90/src/bioregistry/export/
--rw-r--r--   0 runner    (1001) docker     (123)       60 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)      121 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/__main__.py
--rw-r--r--   0 runner    (1001) docker     (123)      727 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/cli.py
--rw-r--r--   0 runner    (1001) docker     (123)     4489 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/prefix_maps.py
--rw-r--r--   0 runner    (1001) docker     (123)     8861 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/rdf_export.py
--rw-r--r--   0 runner    (1001) docker     (123)      481 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/schema_export.py
--rw-r--r--   0 runner    (1001) docker     (123)     3434 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/sssom_export.py
--rw-r--r--   0 runner    (1001) docker     (123)     8994 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/tables_export.py
--rw-r--r--   0 runner    (1001) docker     (123)     4041 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/tsv_export.py
--rw-r--r--   0 runner    (1001) docker     (123)     4617 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/warnings_export.py
--rw-r--r--   0 runner    (1001) docker     (123)     1325 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/export/yaml_export.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.673258 bioregistry-0.9.90/src/bioregistry/external/
--rw-r--r--   0 runner    (1001) docker     (123)     2361 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)     2202 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/aberowl.py
--rw-r--r--   0 runner    (1001) docker     (123)     2504 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/bartoc.py
--rw-r--r--   0 runner    (1001) docker     (123)     1451 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/biocontext.py
--rw-r--r--   0 runner    (1001) docker     (123)     1126 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/biolink.py
--rw-r--r--   0 runner    (1001) docker     (123)     6997 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/bioportal.py
--rw-r--r--   0 runner    (1001) docker     (123)     2261 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/cellosaurus.py
--rw-r--r--   0 runner    (1001) docker     (123)     1257 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/cheminf.py
--rw-r--r--   0 runner    (1001) docker     (123)     1943 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/cropoct.py
--rw-r--r--   0 runner    (1001) docker     (123)     1077 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/edam.py
--rw-r--r--   0 runner    (1001) docker     (123)     3954 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/fairsharing.py
--rw-r--r--   0 runner    (1001) docker     (123)     1510 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/go.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.673258 bioregistry-0.9.90/src/bioregistry/external/hl7/
--rw-r--r--   0 runner    (1001) docker     (123)   319976 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/hl7/OID_Report.csv
--rw-r--r--   0 runner    (1001) docker     (123)     1223 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/hl7/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)     3751 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/miriam.py
--rw-r--r--   0 runner    (1001) docker     (123)     2175 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/n2t.py
--rw-r--r--   0 runner    (1001) docker     (123)     4839 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/ncbi.py
--rw-r--r--   0 runner    (1001) docker     (123)     4187 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/obofoundry.py
--rw-r--r--   0 runner    (1001) docker     (123)     8358 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/ols.py
--rw-r--r--   0 runner    (1001) docker     (123)     1309 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/ontobee.py
--rw-r--r--   0 runner    (1001) docker     (123)     4379 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/prefixcommons.py
--rw-r--r--   0 runner    (1001) docker     (123)     6191 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/re3data.py
--rw-r--r--   0 runner    (1001) docker     (123)     3026 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/uniprot.py
--rw-r--r--   0 runner    (1001) docker     (123)    10755 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/external/wikidata.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.673258 bioregistry-0.9.90/src/bioregistry/gh/
--rw-r--r--   0 runner    (1001) docker     (123)       49 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/gh/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)     8350 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/gh/github_client.py
--rw-r--r--   0 runner    (1001) docker     (123)     9789 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/gh/new_prefix.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.677258 bioregistry-0.9.90/src/bioregistry/health/
--rw-r--r--   0 runner    (1001) docker     (123)      110 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/health/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)      138 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/health/__main__.py
--rw-r--r--   0 runner    (1001) docker     (123)     2523 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/health/check_homepages.py
--rw-r--r--   0 runner    (1001) docker     (123)     7053 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/health/check_providers.py
--rw-r--r--   0 runner    (1001) docker     (123)      797 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/health/cli.py
--rw-r--r--   0 runner    (1001) docker     (123)     8571 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/license_standardizer.py
--rw-r--r--   0 runner    (1001) docker     (123)     1025 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/lint.py
--rw-r--r--   0 runner    (1001) docker     (123)     2708 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/metaresource_api.py
--rw-r--r--   0 runner    (1001) docker     (123)    18936 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/pandas.py
--rw-r--r--   0 runner    (1001) docker     (123)     2768 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/parse_iri.py
--rw-r--r--   0 runner    (1001) docker     (123)     2469 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/parse_version_iri.py
--rw-r--r--   0 runner    (1001) docker     (123)        0 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/py.typed
--rw-r--r--   0 runner    (1001) docker     (123)    13359 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/record_accumulator.py
--rw-r--r--   0 runner    (1001) docker     (123)    33164 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/resolve.py
--rw-r--r--   0 runner    (1001) docker     (123)    17532 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/resolve_identifier.py
--rw-r--r--   0 runner    (1001) docker     (123)    65811 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/resource_manager.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.677258 bioregistry-0.9.90/src/bioregistry/schema/
--rw-r--r--   0 runner    (1001) docker     (123)      276 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/schema/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)    13445 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/schema/constants.py
--rw-r--r--   0 runner    (1001) docker     (123)    43203 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/schema/schema.json
--rw-r--r--   0 runner    (1001) docker     (123)   109675 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/schema/struct.py
--rw-r--r--   0 runner    (1001) docker     (123)      911 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/schema/utils.py
--rw-r--r--   0 runner    (1001) docker     (123)     8188 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/schema_utils.py
--rw-r--r--   0 runner    (1001) docker     (123)    10183 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/summary.py
--rw-r--r--   0 runner    (1001) docker     (123)     5517 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/upload_ndex.py
--rw-r--r--   0 runner    (1001) docker     (123)     7175 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/uri_format.py
--rw-r--r--   0 runner    (1001) docker     (123)     8208 2023-07-30 00:50:40.000000 bioregistry-0.9.90/src/bioregistry/utils.py
--rw-r--r--   0 runner    (1001) docker     (123)     1063 2023-07-30 01:29:49.000000 bioregistry-0.9.90/src/bioregistry/version.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.609257 bioregistry-0.9.90/src/bioregistry.egg-info/
--rw-r--r--   0 runner    (1001) docker     (123)    31461 2023-07-30 01:29:58.000000 bioregistry-0.9.90/src/bioregistry.egg-info/PKG-INFO
--rw-r--r--   0 runner    (1001) docker     (123)    11384 2023-07-30 01:29:58.000000 bioregistry-0.9.90/src/bioregistry.egg-info/SOURCES.txt
--rw-r--r--   0 runner    (1001) docker     (123)        1 2023-07-30 01:29:58.000000 bioregistry-0.9.90/src/bioregistry.egg-info/dependency_links.txt
--rw-r--r--   0 runner    (1001) docker     (123)       53 2023-07-30 01:29:58.000000 bioregistry-0.9.90/src/bioregistry.egg-info/entry_points.txt
--rw-r--r--   0 runner    (1001) docker     (123)        1 2023-07-30 00:50:53.000000 bioregistry-0.9.90/src/bioregistry.egg-info/not-zip-safe
--rw-r--r--   0 runner    (1001) docker     (123)      658 2023-07-30 01:29:58.000000 bioregistry-0.9.90/src/bioregistry.egg-info/requires.txt
--rw-r--r--   0 runner    (1001) docker     (123)       12 2023-07-30 01:29:58.000000 bioregistry-0.9.90/src/bioregistry.egg-info/top_level.txt
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.681258 bioregistry-0.9.90/tests/
--rw-r--r--   0 runner    (1001) docker     (123)       58 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)      414 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_acquisition.py
--rw-r--r--   0 runner    (1001) docker     (123)     1294 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_api.py
--rw-r--r--   0 runner    (1001) docker     (123)     3046 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_collections.py
--rw-r--r--   0 runner    (1001) docker     (123)     5199 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_contexts.py
--rw-r--r--   0 runner    (1001) docker     (123)    43662 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_data.py
--rw-r--r--   0 runner    (1001) docker     (123)     3327 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_data_slow.py
--rw-r--r--   0 runner    (1001) docker     (123)      987 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_duplicates.py
--rw-r--r--   0 runner    (1001) docker     (123)     6904 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_identifiers_org.py
--rw-r--r--   0 runner    (1001) docker     (123)     1737 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_indra.py
--rw-r--r--   0 runner    (1001) docker     (123)    10353 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_manager.py
--rw-r--r--   0 runner    (1001) docker     (123)     7039 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_metaregistry.py
--rw-r--r--   0 runner    (1001) docker     (123)      497 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_obofoundry.py
--rw-r--r--   0 runner    (1001) docker     (123)      584 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_ols.py
--rw-r--r--   0 runner    (1001) docker     (123)     3345 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_pandas.py
--rw-r--r--   0 runner    (1001) docker     (123)     6658 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_resolve.py
--rw-r--r--   0 runner    (1001) docker     (123)     3351 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_sparql.py
--rw-r--r--   0 runner    (1001) docker     (123)     1319 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_usages.py
--rw-r--r--   0 runner    (1001) docker     (123)     1943 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_utils.py
-drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-07-30 01:29:58.681258 bioregistry-0.9.90/tests/test_web/
--rw-r--r--   0 runner    (1001) docker     (123)       73 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_web/__init__.py
--rw-r--r--   0 runner    (1001) docker     (123)     8762 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_web/test_api.py
--rw-r--r--   0 runner    (1001) docker     (123)     7824 2023-07-30 00:50:40.000000 bioregistry-0.9.90/tests/test_web/test_ui.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.255963 bioregistry-0.9.91/
+-rw-r--r--   0 runner    (1001) docker     (123)     1081 2023-08-01 00:53:40.000000 bioregistry-0.9.91/LICENSE
+-rw-r--r--   0 runner    (1001) docker     (123)      630 2023-08-01 00:53:40.000000 bioregistry-0.9.91/MANIFEST.in
+-rw-r--r--   0 runner    (1001) docker     (123)    31461 2023-08-01 01:35:54.255963 bioregistry-0.9.91/PKG-INFO
+-rw-r--r--   0 runner    (1001) docker     (123)    30194 2023-08-01 00:53:40.000000 bioregistry-0.9.91/README.md
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.143962 bioregistry-0.9.91/docs/
+-rw-r--r--   0 runner    (1001) docker     (123)     2968 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/bulk_prefix_request_template.tsv
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.147962 bioregistry-0.9.91/docs/img/
+-rw-r--r--   0 runner    (1001) docker     (123)    47424 2023-08-01 01:19:49.000000 bioregistry-0.9.91/docs/img/bibliography_years.svg
+-rw-r--r--   0 runner    (1001) docker     (123)   157399 2023-08-01 01:20:13.000000 bioregistry-0.9.91/docs/img/bioregistry_coverage.svg
+-rw-r--r--   0 runner    (1001) docker     (123)   107131 2023-08-01 01:20:11.000000 bioregistry-0.9.91/docs/img/bioregistry_coverage_bar.svg
+-rw-r--r--   0 runner    (1001) docker     (123)    84716 2023-08-01 01:20:13.000000 bioregistry-0.9.91/docs/img/bioregistry_coverage_bar_short.svg
+-rw-r--r--   0 runner    (1001) docker     (123)    77372 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/img/datamodel_umls.svg
+-rw-r--r--   0 runner    (1001) docker     (123)  1380510 2023-08-01 01:24:54.000000 bioregistry-0.9.91/docs/img/external_overlap.svg
+-rw-r--r--   0 runner    (1001) docker     (123)    71970 2023-08-01 01:20:43.000000 bioregistry-0.9.91/docs/img/has_attribute.svg
+-rw-r--r--   0 runner    (1001) docker     (123)    28400 2023-08-01 01:24:56.000000 bioregistry-0.9.91/docs/img/providers.svg
+-rw-r--r--   0 runner    (1001) docker     (123)    22344 2023-08-01 01:24:57.000000 bioregistry-0.9.91/docs/img/regex_report.svg
+-rw-r--r--   0 runner    (1001) docker     (123)    51349 2023-08-01 01:20:10.000000 bioregistry-0.9.91/docs/img/xrefs.svg
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.151962 bioregistry-0.9.91/docs/source/
+-rw-r--r--   0 runner    (1001) docker     (123)      121 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/source/alignment.rst
+-rw-r--r--   0 runner    (1001) docker     (123)      236 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/source/cli.rst
+-rw-r--r--   0 runner    (1001) docker     (123)     7034 2023-08-01 01:35:43.000000 bioregistry-0.9.91/docs/source/conf.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8480 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/source/deployment.rst
+-rw-r--r--   0 runner    (1001) docker     (123)     2212 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/source/index.rst
+-rw-r--r--   0 runner    (1001) docker     (123)   214468 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/source/logo.png
+-rw-r--r--   0 runner    (1001) docker     (123)       97 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/source/pandas.rst
+-rw-r--r--   0 runner    (1001) docker     (123)      225 2023-08-01 00:53:40.000000 bioregistry-0.9.91/docs/source/reference.rst
+-rw-r--r--   0 runner    (1001) docker     (123)      467 2023-08-01 00:53:41.000000 bioregistry-0.9.91/pyproject.toml
+-rw-r--r--   0 runner    (1001) docker     (123)     2847 2023-08-01 01:35:54.255963 bioregistry-0.9.91/setup.cfg
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.131962 bioregistry-0.9.91/src/
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.155962 bioregistry-0.9.91/src/bioregistry/
+-rw-r--r--   0 runner    (1001) docker     (123)     3125 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      137 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/__main__.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.163962 bioregistry-0.9.91/src/bioregistry/align/
+-rw-r--r--   0 runner    (1001) docker     (123)     1726 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      428 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/aberowl.py
+-rw-r--r--   0 runner    (1001) docker     (123)      172 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/agroportal.py
+-rw-r--r--   0 runner    (1001) docker     (123)      454 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/bartoc.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1384 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/biocontext.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1717 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/biolink.py
+-rw-r--r--   0 runner    (1001) docker     (123)      932 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/bioportal.py
+-rw-r--r--   0 runner    (1001) docker     (123)      762 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/cellosaurus.py
+-rw-r--r--   0 runner    (1001) docker     (123)      804 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/cheminf.py
+-rw-r--r--   0 runner    (1001) docker     (123)      424 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/cropoct.py
+-rw-r--r--   0 runner    (1001) docker     (123)      169 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/ecoportal.py
+-rw-r--r--   0 runner    (1001) docker     (123)      785 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/edam.py
+-rw-r--r--   0 runner    (1001) docker     (123)      530 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/fairsharing.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2544 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/go.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1030 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/hl7.py
+-rw-r--r--   0 runner    (1001) docker     (123)      453 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/miriam.py
+-rw-r--r--   0 runner    (1001) docker     (123)      521 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/n2t.py
+-rw-r--r--   0 runner    (1001) docker     (123)      824 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/ncbi.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1219 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/obofoundry.py
+-rw-r--r--   0 runner    (1001) docker     (123)      545 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/ols.py
+-rw-r--r--   0 runner    (1001) docker     (123)      743 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/ontobee.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3017 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/prefixcommons.py
+-rw-r--r--   0 runner    (1001) docker     (123)      533 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/re3data.py
+-rw-r--r--   0 runner    (1001) docker     (123)      529 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/uniprot.py
+-rw-r--r--   0 runner    (1001) docker     (123)    10718 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/utils.py
+-rw-r--r--   0 runner    (1001) docker     (123)      769 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/align/wikidata.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.163962 bioregistry-0.9.91/src/bioregistry/analysis/
+-rw-r--r--   0 runner    (1001) docker     (123)       66 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/analysis/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)     5007 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/analysis/title_tfidf.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.167962 bioregistry-0.9.91/src/bioregistry/app/
+-rw-r--r--   0 runner    (1001) docker     (123)       68 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)    17127 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/api.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1946 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/cli.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1921 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/constants.py
+-rw-r--r--   0 runner    (1001) docker     (123)    10179 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/impl.py
+-rw-r--r--   0 runner    (1001) docker     (123)      248 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/proxies.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.167962 bioregistry-0.9.91/src/bioregistry/app/static/
+-rw-r--r--   0 runner    (1001) docker     (123)   801449 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/static/logo.svg
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.171962 bioregistry-0.9.91/src/bioregistry/app/templates/
+-rw-r--r--   0 runner    (1001) docker     (123)     9907 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/base.html
+-rw-r--r--   0 runner    (1001) docker     (123)     3162 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/collection.html
+-rw-r--r--   0 runner    (1001) docker     (123)     3318 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/collections.html
+-rw-r--r--   0 runner    (1001) docker     (123)     6758 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/context.html
+-rw-r--r--   0 runner    (1001) docker     (123)     2387 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/contexts.html
+-rw-r--r--   0 runner    (1001) docker     (123)     7134 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/contributor.html
+-rw-r--r--   0 runner    (1001) docker     (123)     4670 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/contributors.html
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.171962 bioregistry-0.9.91/src/bioregistry/app/templates/highlights/
+-rw-r--r--   0 runner    (1001) docker     (123)      825 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/highlights/keywords.html
+-rw-r--r--   0 runner    (1001) docker     (123)     1083 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/highlights/owners.html
+-rw-r--r--   0 runner    (1001) docker     (123)     1258 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/highlights/relations.html
+-rw-r--r--   0 runner    (1001) docker     (123)      869 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/highlights/twitter.html
+-rw-r--r--   0 runner    (1001) docker     (123)    15622 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/home.html
+-rw-r--r--   0 runner    (1001) docker     (123)     7351 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/macros.html
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.175962 bioregistry-0.9.91/src/bioregistry/app/templates/meta/
+-rw-r--r--   0 runner    (1001) docker     (123)     5577 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/meta/access.html
+-rw-r--r--   0 runner    (1001) docker     (123)    19835 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/meta/acknowledgements.html
+-rw-r--r--   0 runner    (1001) docker     (123)    10695 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/meta/download.html
+-rw-r--r--   0 runner    (1001) docker     (123)    16544 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/meta/related.html
+-rw-r--r--   0 runner    (1001) docker     (123)     1618 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/meta/schema.html
+-rw-r--r--   0 runner    (1001) docker     (123)    19312 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/meta/summary.html
+-rw-r--r--   0 runner    (1001) docker     (123)     8813 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/meta/sustainability.html
+-rw-r--r--   0 runner    (1001) docker     (123)     8474 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/metaresource.html
+-rw-r--r--   0 runner    (1001) docker     (123)     2303 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/metaresources.html
+-rw-r--r--   0 runner    (1001) docker     (123)     1100 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/prose.html
+-rw-r--r--   0 runner    (1001) docker     (123)     2437 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/reference.html
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.175962 bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/
+-rw-r--r--   0 runner    (1001) docker     (123)      520 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/disallowed_identifier.html
+-rw-r--r--   0 runner    (1001) docker     (123)     2040 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/invalid_identifier.html
+-rw-r--r--   0 runner    (1001) docker     (123)     1516 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/missing_prefix.html
+-rw-r--r--   0 runner    (1001) docker     (123)     1307 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/missing_providers.html
+-rw-r--r--   0 runner    (1001) docker     (123)    33327 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/resource.html
+-rw-r--r--   0 runner    (1001) docker     (123)     3765 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/templates/resources.html
+-rw-r--r--   0 runner    (1001) docker     (123)     2042 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/test.py
+-rw-r--r--   0 runner    (1001) docker     (123)    19495 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/ui.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8073 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/utils.py
+-rw-r--r--   0 runner    (1001) docker     (123)      223 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/app/wsgi.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.175962 bioregistry-0.9.91/src/bioregistry/benchmarks/
+-rw-r--r--   0 runner    (1001) docker     (123)       38 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/benchmarks/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      117 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/benchmarks/__main__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      615 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/benchmarks/cli.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3197 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/benchmarks/curie_parsing.py
+-rw-r--r--   0 runner    (1001) docker     (123)     4436 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/benchmarks/curie_validation.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2487 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/benchmarks/uri_parsing.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2141 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/bibliometrics.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2796 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/cli.py
+-rw-r--r--   0 runner    (1001) docker     (123)      539 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/collection_api.py
+-rw-r--r--   0 runner    (1001) docker     (123)    23802 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/compare.py
+-rw-r--r--   0 runner    (1001) docker     (123)     5293 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/constants.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.183962 bioregistry-0.9.91/src/bioregistry/curation/
+-rw-r--r--   0 runner    (1001) docker     (123)       47 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1158 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/add_co_providers.py
+-rw-r--r--   0 runner    (1001) docker     (123)      940 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/add_descriptions_from_gs.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1299 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/add_examples.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1106 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/add_examples_from_javert.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2278 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/add_examples_from_ols.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1606 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/add_ontology_regexes.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2667 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/bulk_import.py
+-rw-r--r--   0 runner    (1001) docker     (123)      619 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/clean_licenses.py
+-rw-r--r--   0 runner    (1001) docker     (123)      679 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/clean_name_suffixes.py
+-rw-r--r--   0 runner    (1001) docker     (123)      704 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/clean_publications.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2026 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/cleanup_authors.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1572 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/deprecation_diff.py
+-rw-r--r--   0 runner    (1001) docker     (123)     5230 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/enrich_publications.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3389 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/import_pc_semiautomatic.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2774 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/make_description_curation_sheet.py
+-rw-r--r--   0 runner    (1001) docker     (123)      860 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/map_bartoc_via_wikidata.py
+-rw-r--r--   0 runner    (1001) docker     (123)      789 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/map_re3data_via_fairsharing.py
+-rw-r--r--   0 runner    (1001) docker     (123)      868 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/rename_metaprefix.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2165 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/review_pc.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1017 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/suggest_author_curation.py
+-rw-r--r--   0 runner    (1001) docker     (123)      827 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/curation/suggest_uniprot_providers.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.191962 bioregistry-0.9.91/src/bioregistry/data/
+-rw-r--r--   0 runner    (1001) docker     (123)     2760 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/data/README.md
+-rw-r--r--   0 runner    (1001) docker     (123)       56 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/data/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)  4581985 2023-08-01 01:17:24.000000 bioregistry-0.9.91/src/bioregistry/data/bioregistry.json
+-rw-r--r--   0 runner    (1001) docker     (123)    12623 2023-08-01 01:17:24.000000 bioregistry-0.9.91/src/bioregistry/data/collections.json
+-rw-r--r--   0 runner    (1001) docker     (123)     1300 2023-08-01 01:17:24.000000 bioregistry-0.9.91/src/bioregistry/data/contexts.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.135962 bioregistry-0.9.91/src/bioregistry/data/external/
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.191962 bioregistry-0.9.91/src/bioregistry/data/external/aberowl/
+-rw-r--r--   0 runner    (1001) docker     (123)   307459 2023-08-01 01:17:18.000000 bioregistry-0.9.91/src/bioregistry/data/external/aberowl/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)   634433 2023-08-01 01:17:17.000000 bioregistry-0.9.91/src/bioregistry/data/external/aberowl/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)  1305869 2023-08-01 01:17:10.000000 bioregistry-0.9.91/src/bioregistry/data/external/aberowl/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.195962 bioregistry-0.9.91/src/bioregistry/data/external/agroportal/
+-rw-r--r--   0 runner    (1001) docker     (123)    43126 2023-08-01 01:04:34.000000 bioregistry-0.9.91/src/bioregistry/data/external/agroportal/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)   127378 2023-08-01 01:04:34.000000 bioregistry-0.9.91/src/bioregistry/data/external/agroportal/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   677787 2023-08-01 01:04:34.000000 bioregistry-0.9.91/src/bioregistry/data/external/agroportal/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.199962 bioregistry-0.9.91/src/bioregistry/data/external/bartoc/
+-rw-r--r--   0 runner    (1001) docker     (123)  1832614 2023-08-01 00:55:09.000000 bioregistry-0.9.91/src/bioregistry/data/external/bartoc/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)  2217991 2023-08-01 00:55:08.000000 bioregistry-0.9.91/src/bioregistry/data/external/bartoc/processed.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.203962 bioregistry-0.9.91/src/bioregistry/data/external/biocontext/
+-rw-r--r--   0 runner    (1001) docker     (123)     4200 2023-08-01 00:54:46.000000 bioregistry-0.9.91/src/bioregistry/data/external/biocontext/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    74913 2023-08-01 00:54:45.000000 bioregistry-0.9.91/src/bioregistry/data/external/biocontext/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)    55878 2023-08-01 00:54:45.000000 bioregistry-0.9.91/src/bioregistry/data/external/biocontext/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.203962 bioregistry-0.9.91/src/bioregistry/data/external/biolink/
+-rw-r--r--   0 runner    (1001) docker     (123)     1169 2023-08-01 01:08:49.000000 bioregistry-0.9.91/src/bioregistry/data/external/biolink/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    10406 2023-08-01 01:08:49.000000 bioregistry-0.9.91/src/bioregistry/data/external/biolink/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   356466 2023-08-01 01:08:47.000000 bioregistry-0.9.91/src/bioregistry/data/external/biolink/raw.yaml
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.207962 bioregistry-0.9.91/src/bioregistry/data/external/bioportal/
+-rw-r--r--   0 runner    (1001) docker     (123)   258394 2023-08-01 01:08:45.000000 bioregistry-0.9.91/src/bioregistry/data/external/bioportal/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)   618236 2023-08-01 01:08:44.000000 bioregistry-0.9.91/src/bioregistry/data/external/bioportal/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)  3607480 2023-08-01 01:08:44.000000 bioregistry-0.9.91/src/bioregistry/data/external/bioportal/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.211962 bioregistry-0.9.91/src/bioregistry/data/external/cellosaurus/
+-rw-r--r--   0 runner    (1001) docker     (123)     2722 2023-08-01 01:17:04.000000 bioregistry-0.9.91/src/bioregistry/data/external/cellosaurus/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    29294 2023-08-01 01:17:03.000000 bioregistry-0.9.91/src/bioregistry/data/external/cellosaurus/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)    33204 2023-08-01 01:17:03.000000 bioregistry-0.9.91/src/bioregistry/data/external/cellosaurus/raw.txt
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.211962 bioregistry-0.9.91/src/bioregistry/data/external/cheminf/
+-rw-r--r--   0 runner    (1001) docker     (123)     2885 2023-08-01 01:02:25.000000 bioregistry-0.9.91/src/bioregistry/data/external/cheminf/processed.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.211962 bioregistry-0.9.91/src/bioregistry/data/external/cropoct/
+-rw-r--r--   0 runner    (1001) docker     (123)      539 2023-08-01 01:17:21.000000 bioregistry-0.9.91/src/bioregistry/data/external/cropoct/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    11713 2023-08-01 01:17:20.000000 bioregistry-0.9.91/src/bioregistry/data/external/cropoct/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)    26054 2023-08-01 01:17:20.000000 bioregistry-0.9.91/src/bioregistry/data/external/cropoct/raw.yaml
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.211962 bioregistry-0.9.91/src/bioregistry/data/external/ecoportal/
+-rw-r--r--   0 runner    (1001) docker     (123)     9252 2023-08-01 00:55:03.000000 bioregistry-0.9.91/src/bioregistry/data/external/ecoportal/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    18869 2023-08-01 00:55:02.000000 bioregistry-0.9.91/src/bioregistry/data/external/ecoportal/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)    98246 2023-08-01 00:55:02.000000 bioregistry-0.9.91/src/bioregistry/data/external/ecoportal/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.211962 bioregistry-0.9.91/src/bioregistry/data/external/edam/
+-rw-r--r--   0 runner    (1001) docker     (123)    14599 2023-08-01 01:02:03.000000 bioregistry-0.9.91/src/bioregistry/data/external/edam/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    45616 2023-08-01 01:02:03.000000 bioregistry-0.9.91/src/bioregistry/data/external/edam/processed.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.215962 bioregistry-0.9.91/src/bioregistry/data/external/fairsharing/
+-rw-r--r--   0 runner    (1001) docker     (123)      235 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/data/external/fairsharing/README.md
+-rw-r--r--   0 runner    (1001) docker     (123)   246096 2023-08-01 01:01:47.000000 bioregistry-0.9.91/src/bioregistry/data/external/fairsharing/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)   738834 2023-08-01 01:01:46.000000 bioregistry-0.9.91/src/bioregistry/data/external/fairsharing/processed.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.215962 bioregistry-0.9.91/src/bioregistry/data/external/go/
+-rw-r--r--   0 runner    (1001) docker     (123)     7556 2023-08-01 00:55:12.000000 bioregistry-0.9.91/src/bioregistry/data/external/go/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)   143532 2023-08-01 00:55:12.000000 bioregistry-0.9.91/src/bioregistry/data/external/go/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   115433 2023-08-01 00:55:11.000000 bioregistry-0.9.91/src/bioregistry/data/external/go/raw.yml
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.215962 bioregistry-0.9.91/src/bioregistry/data/external/hl7/
+-rw-r--r--   0 runner    (1001) docker     (123)   157660 2023-08-01 01:17:23.000000 bioregistry-0.9.91/src/bioregistry/data/external/hl7/curation.tsv
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.219962 bioregistry-0.9.91/src/bioregistry/data/external/miriam/
+-rw-r--r--   0 runner    (1001) docker     (123)   579119 2023-08-01 01:02:17.000000 bioregistry-0.9.91/src/bioregistry/data/external/miriam/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)  2211982 2023-08-01 01:02:16.000000 bioregistry-0.9.91/src/bioregistry/data/external/miriam/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.223962 bioregistry-0.9.91/src/bioregistry/data/external/n2t/
+-rw-r--r--   0 runner    (1001) docker     (123)   425371 2023-08-01 01:02:12.000000 bioregistry-0.9.91/src/bioregistry/data/external/n2t/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)  1290185 2023-08-01 01:02:06.000000 bioregistry-0.9.91/src/bioregistry/data/external/n2t/raw.yml
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.223962 bioregistry-0.9.91/src/bioregistry/data/external/ncbi/
+-rw-r--r--   0 runner    (1001) docker     (123)     3748 2023-08-01 01:08:58.000000 bioregistry-0.9.91/src/bioregistry/data/external/ncbi/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    18072 2023-08-01 01:08:58.000000 bioregistry-0.9.91/src/bioregistry/data/external/ncbi/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   103465 2023-08-01 01:08:58.000000 bioregistry-0.9.91/src/bioregistry/data/external/ncbi/raw.html
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.227962 bioregistry-0.9.91/src/bioregistry/data/external/obofoundry/
+-rw-r--r--   0 runner    (1001) docker     (123)   216827 2023-08-01 01:02:21.000000 bioregistry-0.9.91/src/bioregistry/data/external/obofoundry/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   359255 2023-08-01 01:02:19.000000 bioregistry-0.9.91/src/bioregistry/data/external/obofoundry/raw.yaml
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.227962 bioregistry-0.9.91/src/bioregistry/data/external/ols/
+-rw-r--r--   0 runner    (1001) docker     (123)   126801 2023-08-01 01:08:54.000000 bioregistry-0.9.91/src/bioregistry/data/external/ols/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   559805 2023-08-01 01:08:54.000000 bioregistry-0.9.91/src/bioregistry/data/external/ols/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.227962 bioregistry-0.9.91/src/bioregistry/data/external/ontobee/
+-rw-r--r--   0 runner    (1001) docker     (123)     1826 2023-08-01 00:54:43.000000 bioregistry-0.9.91/src/bioregistry/data/external/ontobee/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    23768 2023-08-01 00:54:43.000000 bioregistry-0.9.91/src/bioregistry/data/external/ontobee/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   192862 2023-08-01 00:54:42.000000 bioregistry-0.9.91/src/bioregistry/data/external/ontobee/raw.html
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.231963 bioregistry-0.9.91/src/bioregistry/data/external/prefixcommons/
+-rw-r--r--   0 runner    (1001) docker     (123)   111567 2023-08-01 00:54:50.000000 bioregistry-0.9.91/src/bioregistry/data/external/prefixcommons/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)   504428 2023-08-01 00:54:50.000000 bioregistry-0.9.91/src/bioregistry/data/external/prefixcommons/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)   883990 2023-08-01 00:54:50.000000 bioregistry-0.9.91/src/bioregistry/data/external/prefixcommons/raw.tsv
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.235962 bioregistry-0.9.91/src/bioregistry/data/external/re3data/
+-rw-r--r--   0 runner    (1001) docker     (123)  1757259 2023-08-01 01:17:00.000000 bioregistry-0.9.91/src/bioregistry/data/external/re3data/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)  2460067 2023-08-01 01:16:59.000000 bioregistry-0.9.91/src/bioregistry/data/external/re3data/processed.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.239963 bioregistry-0.9.91/src/bioregistry/data/external/uniprot/
+-rw-r--r--   0 runner    (1001) docker     (123)    10752 2023-08-01 01:01:57.000000 bioregistry-0.9.91/src/bioregistry/data/external/uniprot/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)    67204 2023-08-01 01:01:56.000000 bioregistry-0.9.91/src/bioregistry/data/external/uniprot/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)    89578 2023-08-01 01:01:56.000000 bioregistry-0.9.91/src/bioregistry/data/external/uniprot/raw.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.239963 bioregistry-0.9.91/src/bioregistry/data/external/wikidata/
+-rw-r--r--   0 runner    (1001) docker     (123)   141153 2023-08-01 01:01:53.000000 bioregistry-0.9.91/src/bioregistry/data/external/wikidata/curation.tsv
+-rw-r--r--   0 runner    (1001) docker     (123)   329966 2023-08-01 01:01:52.000000 bioregistry-0.9.91/src/bioregistry/data/external/wikidata/processed.json
+-rw-r--r--   0 runner    (1001) docker     (123)    47294 2023-08-01 01:17:24.000000 bioregistry-0.9.91/src/bioregistry/data/metaregistry.json
+-rw-r--r--   0 runner    (1001) docker     (123)     2047 2023-08-01 01:17:24.000000 bioregistry-0.9.91/src/bioregistry/data/mismatch.json
+-rw-r--r--   0 runner    (1001) docker     (123)      508 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/data/processing_biolink.json
+-rw-r--r--   0 runner    (1001) docker     (123)      137 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/data/processing_go.json
+-rw-r--r--   0 runner    (1001) docker     (123)    32627 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/data/processing_ols.json
+-rw-r--r--   0 runner    (1001) docker     (123)       18 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/data/processing_wikidata.json
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.243963 bioregistry-0.9.91/src/bioregistry/export/
+-rw-r--r--   0 runner    (1001) docker     (123)       60 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      121 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/__main__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      727 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/cli.py
+-rw-r--r--   0 runner    (1001) docker     (123)     4489 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/prefix_maps.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8861 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/rdf_export.py
+-rw-r--r--   0 runner    (1001) docker     (123)      481 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/schema_export.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3434 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/sssom_export.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8994 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/tables_export.py
+-rw-r--r--   0 runner    (1001) docker     (123)     4041 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/tsv_export.py
+-rw-r--r--   0 runner    (1001) docker     (123)     4617 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/warnings_export.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1325 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/export/yaml_export.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.247963 bioregistry-0.9.91/src/bioregistry/external/
+-rw-r--r--   0 runner    (1001) docker     (123)     2361 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2202 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/aberowl.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2504 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/bartoc.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1451 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/biocontext.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1126 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/biolink.py
+-rw-r--r--   0 runner    (1001) docker     (123)     6997 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/bioportal.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2261 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/cellosaurus.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1257 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/cheminf.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1943 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/cropoct.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1077 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/edam.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3954 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/fairsharing.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1510 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/go.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.247963 bioregistry-0.9.91/src/bioregistry/external/hl7/
+-rw-r--r--   0 runner    (1001) docker     (123)   319976 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/hl7/OID_Report.csv
+-rw-r--r--   0 runner    (1001) docker     (123)     1223 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/hl7/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3751 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/miriam.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2175 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/n2t.py
+-rw-r--r--   0 runner    (1001) docker     (123)     4839 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/ncbi.py
+-rw-r--r--   0 runner    (1001) docker     (123)     4187 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/obofoundry.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8358 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/ols.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1309 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/ontobee.py
+-rw-r--r--   0 runner    (1001) docker     (123)     4379 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/prefixcommons.py
+-rw-r--r--   0 runner    (1001) docker     (123)     6191 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/re3data.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3026 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/uniprot.py
+-rw-r--r--   0 runner    (1001) docker     (123)    10755 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/external/wikidata.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.247963 bioregistry-0.9.91/src/bioregistry/gh/
+-rw-r--r--   0 runner    (1001) docker     (123)       49 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/gh/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8350 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/gh/github_client.py
+-rw-r--r--   0 runner    (1001) docker     (123)     9789 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/gh/new_prefix.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.247963 bioregistry-0.9.91/src/bioregistry/health/
+-rw-r--r--   0 runner    (1001) docker     (123)      110 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/health/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      138 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/health/__main__.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2523 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/health/check_homepages.py
+-rw-r--r--   0 runner    (1001) docker     (123)     7053 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/health/check_providers.py
+-rw-r--r--   0 runner    (1001) docker     (123)      797 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/health/cli.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8571 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/license_standardizer.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1025 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/lint.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2708 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/metaresource_api.py
+-rw-r--r--   0 runner    (1001) docker     (123)    18936 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/pandas.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2768 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/parse_iri.py
+-rw-r--r--   0 runner    (1001) docker     (123)     2469 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/parse_version_iri.py
+-rw-r--r--   0 runner    (1001) docker     (123)        0 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/py.typed
+-rw-r--r--   0 runner    (1001) docker     (123)    13359 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/record_accumulator.py
+-rw-r--r--   0 runner    (1001) docker     (123)    33164 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/resolve.py
+-rw-r--r--   0 runner    (1001) docker     (123)    17532 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/resolve_identifier.py
+-rw-r--r--   0 runner    (1001) docker     (123)    65811 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/resource_manager.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.251963 bioregistry-0.9.91/src/bioregistry/schema/
+-rw-r--r--   0 runner    (1001) docker     (123)      276 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/schema/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)    13445 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/schema/constants.py
+-rw-r--r--   0 runner    (1001) docker     (123)    43203 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/schema/schema.json
+-rw-r--r--   0 runner    (1001) docker     (123)   109675 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/schema/struct.py
+-rw-r--r--   0 runner    (1001) docker     (123)      911 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/schema/utils.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8188 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/schema_utils.py
+-rw-r--r--   0 runner    (1001) docker     (123)    10183 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/summary.py
+-rw-r--r--   0 runner    (1001) docker     (123)     5517 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/upload_ndex.py
+-rw-r--r--   0 runner    (1001) docker     (123)     7175 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/uri_format.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8208 2023-08-01 00:53:41.000000 bioregistry-0.9.91/src/bioregistry/utils.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1063 2023-08-01 01:35:43.000000 bioregistry-0.9.91/src/bioregistry/version.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.159962 bioregistry-0.9.91/src/bioregistry.egg-info/
+-rw-r--r--   0 runner    (1001) docker     (123)    31461 2023-08-01 01:35:54.000000 bioregistry-0.9.91/src/bioregistry.egg-info/PKG-INFO
+-rw-r--r--   0 runner    (1001) docker     (123)    11384 2023-08-01 01:35:54.000000 bioregistry-0.9.91/src/bioregistry.egg-info/SOURCES.txt
+-rw-r--r--   0 runner    (1001) docker     (123)        1 2023-08-01 01:35:54.000000 bioregistry-0.9.91/src/bioregistry.egg-info/dependency_links.txt
+-rw-r--r--   0 runner    (1001) docker     (123)       53 2023-08-01 01:35:54.000000 bioregistry-0.9.91/src/bioregistry.egg-info/entry_points.txt
+-rw-r--r--   0 runner    (1001) docker     (123)        1 2023-08-01 00:53:54.000000 bioregistry-0.9.91/src/bioregistry.egg-info/not-zip-safe
+-rw-r--r--   0 runner    (1001) docker     (123)      658 2023-08-01 01:35:54.000000 bioregistry-0.9.91/src/bioregistry.egg-info/requires.txt
+-rw-r--r--   0 runner    (1001) docker     (123)       12 2023-08-01 01:35:54.000000 bioregistry-0.9.91/src/bioregistry.egg-info/top_level.txt
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.255963 bioregistry-0.9.91/tests/
+-rw-r--r--   0 runner    (1001) docker     (123)       58 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)      414 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_acquisition.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1294 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_api.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3046 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_collections.py
+-rw-r--r--   0 runner    (1001) docker     (123)     5199 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_contexts.py
+-rw-r--r--   0 runner    (1001) docker     (123)    43662 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_data.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3327 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_data_slow.py
+-rw-r--r--   0 runner    (1001) docker     (123)      987 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_duplicates.py
+-rw-r--r--   0 runner    (1001) docker     (123)     6904 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_identifiers_org.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1737 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_indra.py
+-rw-r--r--   0 runner    (1001) docker     (123)    10353 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_manager.py
+-rw-r--r--   0 runner    (1001) docker     (123)     7039 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_metaregistry.py
+-rw-r--r--   0 runner    (1001) docker     (123)      497 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_obofoundry.py
+-rw-r--r--   0 runner    (1001) docker     (123)      584 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_ols.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3345 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_pandas.py
+-rw-r--r--   0 runner    (1001) docker     (123)     6658 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_resolve.py
+-rw-r--r--   0 runner    (1001) docker     (123)     3351 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_sparql.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1319 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_usages.py
+-rw-r--r--   0 runner    (1001) docker     (123)     1943 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_utils.py
+drwxr-xr-x   0 runner    (1001) docker     (123)        0 2023-08-01 01:35:54.255963 bioregistry-0.9.91/tests/test_web/
+-rw-r--r--   0 runner    (1001) docker     (123)       73 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_web/__init__.py
+-rw-r--r--   0 runner    (1001) docker     (123)     8762 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_web/test_api.py
+-rw-r--r--   0 runner    (1001) docker     (123)     7824 2023-08-01 00:53:41.000000 bioregistry-0.9.91/tests/test_web/test_ui.py
```

### Comparing `bioregistry-0.9.90/LICENSE` & `bioregistry-0.9.91/LICENSE`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/MANIFEST.in` & `bioregistry-0.9.91/MANIFEST.in`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/PKG-INFO` & `bioregistry-0.9.91/PKG-INFO`

 * *Files 0% similar despite different names*

```diff
@@ -1,10 +1,10 @@
 Metadata-Version: 2.1
 Name: bioregistry
-Version: 0.9.90
+Version: 0.9.91
 Summary: Integrated registry of biological databases and nomenclatures
 Home-page: https://github.com/biopragmatics/bioregistry
 Download-URL: https://github.com/biopragmatics/bioregistry/releases
 Author: Charles Tapley Hoyt
 Author-email: cthoyt@gmail.com
 Maintainer: Charles Tapley Hoyt
 Maintainer-email: cthoyt@gmail.com
```

#### html2text {}

```diff
@@ -1,8 +1,8 @@
-Metadata-Version: 2.1 Name: bioregistry Version: 0.9.90 Summary: Integrated
+Metadata-Version: 2.1 Name: bioregistry Version: 0.9.91 Summary: Integrated
 registry of biological databases and nomenclatures Home-page: https://
 github.com/biopragmatics/bioregistry Download-URL: https://github.com/
 biopragmatics/bioregistry/releases Author: Charles Tapley Hoyt Author-email:
 cthoyt@gmail.com Maintainer: Charles Tapley Hoyt Maintainer-email:
 cthoyt@gmail.com License: MIT Project-URL: Bug Tracker, https://github.com/
 biopragmatics/bioregistry/issues Keywords: databases,biological
 databases,biomedical databases Classifier: Development Status :: 4 - Beta
```

### Comparing `bioregistry-0.9.90/README.md` & `bioregistry-0.9.91/README.md`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/docs/bulk_prefix_request_template.tsv` & `bioregistry-0.9.91/docs/bulk_prefix_request_template.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/docs/img/bibliography_years.svg` & `bioregistry-0.9.91/docs/img/bibliography_years.svg`

 * *Files 3% similar despite different names*

```diff
@@ -29,16 +29,16 @@
 000001c0: 3032 2f32 322d 7264 662d 7379 6e74 6178  02/22-rdf-syntax
 000001d0: 2d6e 7323 223e 0a20 2020 3c63 633a 576f  -ns#">.   <cc:Wo
 000001e0: 726b 3e0a 2020 2020 3c64 633a 7479 7065  rk>.    <dc:type
 000001f0: 2072 6466 3a72 6573 6f75 7263 653d 2268   rdf:resource="h
 00000200: 7474 703a 2f2f 7075 726c 2e6f 7267 2f64  ttp://purl.org/d
 00000210: 632f 6463 6d69 7479 7065 2f53 7469 6c6c  c/dcmitype/Still
 00000220: 496d 6167 6522 2f3e 0a20 2020 203c 6463  Image"/>.    <dc
-00000230: 3a64 6174 653e 3230 3233 2d30 372d 3330  :date>2023-07-30
-00000240: 5430 313a 3136 3a31 352e 3536 3936 3633  T01:16:15.569663
+00000230: 3a64 6174 653e 3230 3233 2d30 382d 3031  :date>2023-08-01
+00000240: 5430 313a 3139 3a34 392e 3631 3233 3735  T01:19:49.612375
 00000250: 3c2f 6463 3a64 6174 653e 0a20 2020 203c  </dc:date>.    <
 00000260: 6463 3a66 6f72 6d61 743e 696d 6167 652f  dc:format>image/
 00000270: 7376 672b 786d 6c3c 2f64 633a 666f 726d  svg+xml</dc:form
 00000280: 6174 3e0a 2020 2020 3c64 633a 6372 6561  at>.    <dc:crea
 00000290: 746f 723e 0a20 2020 2020 3c63 633a 4167  tor>.     <cc:Ag
 000002a0: 656e 743e 0a20 2020 2020 203c 6463 3a74  ent>.      <dc:t
 000002b0: 6974 6c65 3e4d 6174 706c 6f74 6c69 6220  itle>Matplotlib 
@@ -78,380 +78,380 @@
 000004d0: 3c70 6174 6820 643d 224d 2034 362e 3334  <path d="M 46.34
 000004e0: 3838 3034 2032 3038 2e39 3938 3731 3420  8804 208.998714 
 000004f0: 0a4c 2036 302e 3537 3932 3337 2032 3038  .L 60.579237 208
 00000500: 2e39 3938 3731 3420 0a4c 2036 302e 3537  .998714 .L 60.57
 00000510: 3932 3337 2032 3038 2e39 3938 3731 3420  9237 208.998714 
 00000520: 0a4c 2034 362e 3334 3838 3034 2032 3038  .L 46.348804 208
 00000530: 2e39 3938 3731 3420 0a7a 0a22 2063 6c69  .998714 .z." cli
-00000540: 702d 7061 7468 3d22 7572 6c28 2370 3235  p-path="url(#p25
-00000550: 3537 6331 3263 6136 2922 2073 7479 6c65  57c12ca6)" style
+00000540: 702d 7061 7468 3d22 7572 6c28 2370 6364  p-path="url(#pcd
+00000550: 3037 3930 6366 3438 2922 2073 7479 6c65  0790cf48)" style
 00000560: 3d22 6669 6c6c 3a20 2365 6139 3661 3322  ="fill: #ea96a3"
 00000570: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 00000580: 2069 643d 2270 6174 6368 5f34 223e 0a20   id="patch_4">. 
 00000590: 2020 203c 7061 7468 2064 3d22 4d20 3634     <path d="M 64
 000005a0: 2e31 3336 3834 3520 3230 382e 3939 3837  .136845 208.9987
 000005b0: 3134 200a 4c20 3738 2e33 3637 3237 3820  14 .L 78.367278 
 000005c0: 3230 382e 3939 3837 3134 200a 4c20 3738  208.998714 .L 78
 000005d0: 2e33 3637 3237 3820 3230 332e 3839 3733  .367278 203.8973
 000005e0: 3520 0a4c 2036 342e 3133 3638 3435 2032  5 .L 64.136845 2
 000005f0: 3033 2e38 3937 3335 200a 7a0a 2220 636c  03.89735 .z." cl
-00000600: 6970 2d70 6174 683d 2275 726c 2823 7032  ip-path="url(#p2
-00000610: 3535 3763 3132 6361 3629 2220 7374 796c  557c12ca6)" styl
+00000600: 6970 2d70 6174 683d 2275 726c 2823 7063  ip-path="url(#pc
+00000610: 6430 3739 3063 6634 3829 2220 7374 796c  d0790cf48)" styl
 00000620: 653d 2266 696c 6c3a 2023 6539 3937 3930  e="fill: #e99790
 00000630: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 00000640: 6720 6964 3d22 7061 7463 685f 3522 3e0a  g id="patch_5">.
 00000650: 2020 2020 3c70 6174 6820 643d 224d 2038      <path d="M 8
 00000660: 312e 3932 3438 3836 2032 3038 2e39 3938  1.924886 208.998
 00000670: 3731 3420 0a4c 2039 362e 3135 3533 3139  714 .L 96.155319
 00000680: 2032 3038 2e39 3938 3731 3420 0a4c 2039   208.998714 .L 9
 00000690: 362e 3135 3533 3139 2031 3833 2e34 3931  6.155319 183.491
 000006a0: 3839 3220 0a4c 2038 312e 3932 3438 3836  892 .L 81.924886
 000006b0: 2031 3833 2e34 3931 3839 3220 0a7a 0a22   183.491892 .z."
 000006c0: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-000006d0: 2370 3235 3537 6331 3263 6136 2922 2073  #p2557c12ca6)" s
+000006d0: 2370 6364 3037 3930 6366 3438 2922 2073  #pcd0790cf48)" s
 000006e0: 7479 6c65 3d22 6669 6c6c 3a20 2365 3439  tyle="fill: #e49
 000006f0: 3436 6422 2f3e 0a20 2020 3c2f 673e 0a20  46d"/>.   </g>. 
 00000700: 2020 3c67 2069 643d 2270 6174 6368 5f36    <g id="patch_6
 00000710: 223e 0a20 2020 203c 7061 7468 2064 3d22  ">.    <path d="
 00000720: 4d20 3939 2e37 3132 3932 3720 3230 382e  M 99.712927 208.
 00000730: 3939 3837 3134 200a 4c20 3131 332e 3934  998714 .L 113.94
 00000740: 3333 3620 3230 382e 3939 3837 3134 200a  336 208.998714 .
 00000750: 4c20 3131 332e 3934 3333 3620 3138 382e  L 113.94336 188.
 00000760: 3539 3332 3536 200a 4c20 3939 2e37 3132  593256 .L 99.712
 00000770: 3932 3720 3138 382e 3539 3332 3536 200a  927 188.593256 .
 00000780: 7a0a 2220 636c 6970 2d70 6174 683d 2275  z." clip-path="u
-00000790: 726c 2823 7032 3535 3763 3132 6361 3629  rl(#p2557c12ca6)
+00000790: 726c 2823 7063 6430 3739 3063 6634 3829  rl(#pcd0790cf48)
 000007a0: 2220 7374 796c 653d 2266 696c 6c3a 2023  " style="fill: #
 000007b0: 6435 3935 3464 222f 3e0a 2020 203c 2f67  d5954d"/>.   </g
 000007c0: 3e0a 2020 203c 6720 6964 3d22 7061 7463  >.   <g id="patc
 000007d0: 685f 3722 3e0a 2020 2020 3c70 6174 6820  h_7">.    <path 
 000007e0: 643d 224d 2031 3137 2e35 3030 3936 3820  d="M 117.500968 
 000007f0: 3230 382e 3939 3837 3134 200a 4c20 3133  208.998714 .L 13
 00000800: 312e 3733 3134 3031 2032 3038 2e39 3938  1.731401 208.998
 00000810: 3731 3420 0a4c 2031 3331 2e37 3331 3430  714 .L 131.73140
 00000820: 3120 3137 382e 3339 3035 3237 200a 4c20  1 178.390527 .L 
 00000830: 3131 372e 3530 3039 3638 2031 3738 2e33  117.500968 178.3
 00000840: 3930 3532 3720 0a7a 0a22 2063 6c69 702d  90527 .z." clip-
-00000850: 7061 7468 3d22 7572 6c28 2370 3235 3537  path="url(#p2557
-00000860: 6331 3263 6136 2922 2073 7479 6c65 3d22  c12ca6)" style="
+00000850: 7061 7468 3d22 7572 6c28 2370 6364 3037  path="url(#pcd07
+00000860: 3930 6366 3438 2922 2073 7479 6c65 3d22  90cf48)" style="
 00000870: 6669 6c6c 3a20 2363 3439 3934 6222 2f3e  fill: #c4994b"/>
 00000880: 0a20 2020 3c2f 673e 0a20 2020 3c67 2069  .   </g>.   <g i
 00000890: 643d 2270 6174 6368 5f38 223e 0a20 2020  d="patch_8">.   
 000008a0: 203c 7061 7468 2064 3d22 4d20 3133 352e   <path d="M 135.
 000008b0: 3238 3930 3039 2032 3038 2e39 3938 3731  289009 208.99871
 000008c0: 3420 0a4c 2031 3439 2e35 3139 3434 3220  4 .L 149.519442 
 000008d0: 3230 382e 3939 3837 3134 200a 4c20 3134  208.998714 .L 14
 000008e0: 392e 3531 3934 3432 2031 3633 2e30 3836  9.519442 163.086
 000008f0: 3433 3320 0a4c 2031 3335 2e32 3839 3030  433 .L 135.28900
 00000900: 3920 3136 332e 3038 3634 3333 200a 7a0a  9 163.086433 .z.
 00000910: 2220 636c 6970 2d70 6174 683d 2275 726c  " clip-path="url
-00000920: 2823 7032 3535 3763 3132 6361 3629 2220  (#p2557c12ca6)" 
+00000920: 2823 7063 6430 3739 3063 6634 3829 2220  (#pcd0790cf48)" 
 00000930: 7374 796c 653d 2266 696c 6c3a 2023 6236  style="fill: #b6
 00000940: 3963 3439 222f 3e0a 2020 203c 2f67 3e0a  9c49"/>.   </g>.
 00000950: 2020 203c 6720 6964 3d22 7061 7463 685f     <g id="patch_
 00000960: 3922 3e0a 2020 2020 3c70 6174 6820 643d  9">.    <path d=
 00000970: 224d 2031 3533 2e30 3737 3035 2032 3038  "M 153.07705 208
 00000980: 2e39 3938 3731 3420 0a4c 2031 3637 2e33  .998714 .L 167.3
 00000990: 3037 3438 3320 3230 382e 3939 3837 3134  07483 208.998714
 000009a0: 200a 4c20 3136 372e 3330 3734 3833 2031   .L 167.307483 1
 000009b0: 3833 2e34 3931 3839 3220 0a4c 2031 3533  83.491892 .L 153
 000009c0: 2e30 3737 3035 2031 3833 2e34 3931 3839  .07705 183.49189
 000009d0: 3220 0a7a 0a22 2063 6c69 702d 7061 7468  2 .z." clip-path
-000009e0: 3d22 7572 6c28 2370 3235 3537 6331 3263  ="url(#p2557c12c
-000009f0: 6136 2922 2073 7479 6c65 3d22 6669 6c6c  a6)" style="fill
+000009e0: 3d22 7572 6c28 2370 6364 3037 3930 6366  ="url(#pcd0790cf
+000009f0: 3438 2922 2073 7479 6c65 3d22 6669 6c6c  48)" style="fill
 00000a00: 3a20 2361 3939 6534 3722 2f3e 0a20 2020  : #a99e47"/>.   
 00000a10: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 00000a20: 6174 6368 5f31 3022 3e0a 2020 2020 3c70  atch_10">.    <p
 00000a30: 6174 6820 643d 224d 2031 3730 2e38 3635  ath d="M 170.865
 00000a40: 3039 3120 3230 382e 3939 3837 3134 200a  091 208.998714 .
 00000a50: 4c20 3138 352e 3039 3535 3234 2032 3038  L 185.095524 208
 00000a60: 2e39 3938 3731 3420 0a4c 2031 3835 2e30  .998714 .L 185.0
 00000a70: 3935 3532 3420 3135 322e 3838 3337 3034  95524 152.883704
 00000a80: 200a 4c20 3137 302e 3836 3530 3931 2031   .L 170.865091 1
 00000a90: 3532 2e38 3833 3730 3420 0a7a 0a22 2063  52.883704 .z." c
 00000aa0: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-00000ab0: 3235 3537 6331 3263 6136 2922 2073 7479  2557c12ca6)" sty
+00000ab0: 6364 3037 3930 6366 3438 2922 2073 7479  cd0790cf48)" sty
 00000ac0: 6c65 3d22 6669 6c6c 3a20 2339 6261 3134  le="fill: #9ba14
 00000ad0: 3622 2f3e 0a20 2020 3c2f 673e 0a20 2020  6"/>.   </g>.   
 00000ae0: 3c67 2069 643d 2270 6174 6368 5f31 3122  <g id="patch_11"
 00000af0: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 00000b00: 2031 3838 2e36 3533 3133 3220 3230 382e   188.653132 208.
 00000b10: 3939 3837 3134 200a 4c20 3230 322e 3838  998714 .L 202.88
 00000b20: 3335 3635 2032 3038 2e39 3938 3731 3420  3565 208.998714 
 00000b30: 0a4c 2032 3032 2e38 3833 3536 3520 3133  .L 202.883565 13
 00000b40: 322e 3437 3832 3436 200a 4c20 3138 382e  2.478246 .L 188.
 00000b50: 3635 3331 3332 2031 3332 2e34 3738 3234  653132 132.47824
 00000b60: 3620 0a7a 0a22 2063 6c69 702d 7061 7468  6 .z." clip-path
-00000b70: 3d22 7572 6c28 2370 3235 3537 6331 3263  ="url(#p2557c12c
-00000b80: 6136 2922 2073 7479 6c65 3d22 6669 6c6c  a6)" style="fill
+00000b70: 3d22 7572 6c28 2370 6364 3037 3930 6366  ="url(#pcd0790cf
+00000b80: 3438 2922 2073 7479 6c65 3d22 6669 6c6c  48)" style="fill
 00000b90: 3a20 2338 6461 3534 3622 2f3e 0a20 2020  : #8da546"/>.   
 00000ba0: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 00000bb0: 6174 6368 5f31 3222 3e0a 2020 2020 3c70  atch_12">.    <p
 00000bc0: 6174 6820 643d 224d 2032 3036 2e34 3431  ath d="M 206.441
 00000bd0: 3137 3320 3230 382e 3939 3837 3134 200a  173 208.998714 .
 00000be0: 4c20 3232 302e 3637 3136 3036 2032 3038  L 220.671606 208
 00000bf0: 2e39 3938 3731 3420 0a4c 2032 3230 2e36  .998714 .L 220.6
 00000c00: 3731 3630 3620 3335 2e35 3532 3332 200a  71606 35.55232 .
 00000c10: 4c20 3230 362e 3434 3131 3733 2033 352e  L 206.441173 35.
 00000c20: 3535 3233 3220 0a7a 0a22 2063 6c69 702d  55232 .z." clip-
-00000c30: 7061 7468 3d22 7572 6c28 2370 3235 3537  path="url(#p2557
-00000c40: 6331 3263 6136 2922 2073 7479 6c65 3d22  c12ca6)" style="
+00000c30: 7061 7468 3d22 7572 6c28 2370 6364 3037  path="url(#pcd07
+00000c40: 3930 6366 3438 2922 2073 7479 6c65 3d22  90cf48)" style="
 00000c50: 6669 6c6c 3a20 2337 3861 6134 3722 2f3e  fill: #78aa47"/>
 00000c60: 0a20 2020 3c2f 673e 0a20 2020 3c67 2069  .   </g>.   <g i
 00000c70: 643d 2270 6174 6368 5f31 3322 3e0a 2020  d="patch_13">.  
 00000c80: 2020 3c70 6174 6820 643d 224d 2032 3234    <path d="M 224
 00000c90: 2e32 3239 3231 3420 3230 382e 3939 3837  .229214 208.9987
 00000ca0: 3134 200a 4c20 3233 382e 3435 3936 3437  14 .L 238.459647
 00000cb0: 2032 3038 2e39 3938 3731 3420 0a4c 2032   208.998714 .L 2
 00000cc0: 3338 2e34 3539 3634 3720 3931 2e36 3637  38.459647 91.667
 00000cd0: 3333 200a 4c20 3232 342e 3232 3932 3134  33 .L 224.229214
 00000ce0: 2039 312e 3636 3733 3320 0a7a 0a22 2063   91.66733 .z." c
 00000cf0: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-00000d00: 3235 3537 6331 3263 6136 2922 2073 7479  2557c12ca6)" sty
+00000d00: 6364 3037 3930 6366 3438 2922 2073 7479  cd0790cf48)" sty
 00000d10: 6c65 3d22 6669 6c6c 3a20 2334 6162 3134  le="fill: #4ab14
 00000d20: 3722 2f3e 0a20 2020 3c2f 673e 0a20 2020  7"/>.   </g>.   
 00000d30: 3c67 2069 643d 2270 6174 6368 5f31 3422  <g id="patch_14"
 00000d40: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 00000d50: 2032 3432 2e30 3137 3235 3520 3230 382e   242.017255 208.
 00000d60: 3939 3837 3134 200a 4c20 3235 362e 3234  998714 .L 256.24
 00000d70: 3736 3838 2032 3038 2e39 3938 3731 3420  7688 208.998714 
 00000d80: 0a4c 2032 3536 2e32 3437 3638 3820 3535  .L 256.247688 55
 00000d90: 2e39 3537 3737 3820 0a4c 2032 3432 2e30  .957778 .L 242.0
 00000da0: 3137 3235 3520 3535 2e39 3537 3737 3820  17255 55.957778 
 00000db0: 0a7a 0a22 2063 6c69 702d 7061 7468 3d22  .z." clip-path="
-00000dc0: 7572 6c28 2370 3235 3537 6331 3263 6136  url(#p2557c12ca6
+00000dc0: 7572 6c28 2370 6364 3037 3930 6366 3438  url(#pcd0790cf48
 00000dd0: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00000de0: 2334 3961 6637 3622 2f3e 0a20 2020 3c2f  #49af76"/>.   </
 00000df0: 673e 0a20 2020 3c67 2069 643d 2270 6174  g>.   <g id="pat
 00000e00: 6368 5f31 3522 3e0a 2020 2020 3c70 6174  ch_15">.    <pat
 00000e10: 6820 643d 224d 2032 3539 2e38 3035 3239  h d="M 259.80529
 00000e20: 3620 3230 382e 3939 3837 3134 200a 4c20  6 208.998714 .L 
 00000e30: 3237 342e 3033 3537 3239 2032 3038 2e39  274.035729 208.9
 00000e40: 3938 3731 3420 0a4c 2032 3734 2e30 3335  98714 .L 274.035
 00000e50: 3732 3920 3731 2e32 3631 3837 3220 0a4c  729 71.261872 .L
 00000e60: 2032 3539 2e38 3035 3239 3620 3731 2e32   259.805296 71.2
 00000e70: 3631 3837 3220 0a7a 0a22 2063 6c69 702d  61872 .z." clip-
-00000e80: 7061 7468 3d22 7572 6c28 2370 3235 3537  path="url(#p2557
-00000e90: 6331 3263 6136 2922 2073 7479 6c65 3d22  c12ca6)" style="
+00000e80: 7061 7468 3d22 7572 6c28 2370 6364 3037  path="url(#pcd07
+00000e90: 3930 6366 3438 2922 2073 7479 6c65 3d22  90cf48)" style="
 00000ea0: 6669 6c6c 3a20 2334 6161 6538 3922 2f3e  fill: #4aae89"/>
 00000eb0: 0a20 2020 3c2f 673e 0a20 2020 3c67 2069  .   </g>.   <g i
 00000ec0: 643d 2270 6174 6368 5f31 3622 3e0a 2020  d="patch_16">.  
 00000ed0: 2020 3c70 6174 6820 643d 224d 2032 3737    <path d="M 277
 00000ee0: 2e35 3933 3333 3720 3230 382e 3939 3837  .593337 208.9987
 00000ef0: 3134 200a 4c20 3239 312e 3832 3337 3639  14 .L 291.823769
 00000f00: 2032 3038 2e39 3938 3731 3420 0a4c 2032   208.998714 .L 2
 00000f10: 3931 2e38 3233 3736 3920 3435 2e37 3535  91.823769 45.755
 00000f20: 3034 3920 0a4c 2032 3737 2e35 3933 3333  049 .L 277.59333
 00000f30: 3720 3435 2e37 3535 3034 3920 0a7a 0a22  7 45.755049 .z."
 00000f40: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00000f50: 2370 3235 3537 6331 3263 6136 2922 2073  #p2557c12ca6)" s
+00000f50: 2370 6364 3037 3930 6366 3438 2922 2073  #pcd0790cf48)" s
 00000f60: 7479 6c65 3d22 6669 6c6c 3a20 2334 6161  tyle="fill: #4aa
 00000f70: 6439 3622 2f3e 0a20 2020 3c2f 673e 0a20  d96"/>.   </g>. 
 00000f80: 2020 3c67 2069 643d 2270 6174 6368 5f31    <g id="patch_1
 00000f90: 3722 3e0a 2020 2020 3c70 6174 6820 643d  7">.    <path d=
 00000fa0: 224d 2032 3935 2e33 3831 3337 3820 3230  "M 295.381378 20
 00000fb0: 382e 3939 3837 3134 200a 4c20 3330 392e  8.998714 .L 309.
 00000fc0: 3631 3138 3120 3230 382e 3939 3837 3134  61181 208.998714
 00000fd0: 200a 4c20 3330 392e 3631 3138 3120 3731   .L 309.61181 71
 00000fe0: 2e32 3631 3837 3220 0a4c 2032 3935 2e33  .261872 .L 295.3
 00000ff0: 3831 3337 3820 3731 2e32 3631 3837 3220  81378 71.261872 
 00001000: 0a7a 0a22 2063 6c69 702d 7061 7468 3d22  .z." clip-path="
-00001010: 7572 6c28 2370 3235 3537 6331 3263 6136  url(#p2557c12ca6
+00001010: 7572 6c28 2370 6364 3037 3930 6366 3438  url(#pcd0790cf48
 00001020: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00001030: 2334 6261 6361 3022 2f3e 0a20 2020 3c2f  #4baca0"/>.   </
 00001040: 673e 0a20 2020 3c67 2069 643d 2270 6174  g>.   <g id="pat
 00001050: 6368 5f31 3822 3e0a 2020 2020 3c70 6174  ch_18">.    <pat
 00001060: 6820 643d 224d 2033 3133 2e31 3639 3431  h d="M 313.16941
 00001070: 3920 3230 382e 3939 3837 3134 200a 4c20  9 208.998714 .L 
 00001080: 3332 372e 3339 3938 3531 2032 3038 2e39  327.399851 208.9
 00001090: 3938 3731 3420 0a4c 2033 3237 2e33 3939  98714 .L 327.399
 000010a0: 3835 3120 3736 2e33 3633 3233 3620 0a4c  851 76.363236 .L
 000010b0: 2033 3133 2e31 3639 3431 3920 3736 2e33   313.169419 76.3
 000010c0: 3633 3233 3620 0a7a 0a22 2063 6c69 702d  63236 .z." clip-
-000010d0: 7061 7468 3d22 7572 6c28 2370 3235 3537  path="url(#p2557
-000010e0: 6331 3263 6136 2922 2073 7479 6c65 3d22  c12ca6)" style="
+000010d0: 7061 7468 3d22 7572 6c28 2370 6364 3037  path="url(#pcd07
+000010e0: 3930 6366 3438 2922 2073 7479 6c65 3d22  90cf48)" style="
 000010f0: 6669 6c6c 3a20 2334 6261 6261 3822 2f3e  fill: #4baba8"/>
 00001100: 0a20 2020 3c2f 673e 0a20 2020 3c67 2069  .   </g>.   <g i
 00001110: 643d 2270 6174 6368 5f31 3922 3e0a 2020  d="patch_19">.  
 00001120: 2020 3c70 6174 6820 643d 224d 2033 3330    <path d="M 330
 00001130: 2e39 3537 3436 2032 3038 2e39 3938 3731  .95746 208.99871
 00001140: 3420 0a4c 2033 3435 2e31 3837 3839 3220  4 .L 345.187892 
 00001150: 3230 382e 3939 3837 3134 200a 4c20 3334  208.998714 .L 34
 00001160: 352e 3138 3738 3932 2034 352e 3735 3530  5.187892 45.7550
 00001170: 3439 200a 4c20 3333 302e 3935 3734 3620  49 .L 330.95746 
 00001180: 3435 2e37 3535 3034 3920 0a7a 0a22 2063  45.755049 .z." c
 00001190: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-000011a0: 3235 3537 6331 3263 6136 2922 2073 7479  2557c12ca6)" sty
+000011a0: 6364 3037 3930 6366 3438 2922 2073 7479  cd0790cf48)" sty
 000011b0: 6c65 3d22 6669 6c6c 3a20 2334 6461 6262  le="fill: #4dabb
 000011c0: 3222 2f3e 0a20 2020 3c2f 673e 0a20 2020  2"/>.   </g>.   
 000011d0: 3c67 2069 643d 2270 6174 6368 5f32 3022  <g id="patch_20"
 000011e0: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 000011f0: 2033 3438 2e37 3435 3530 3120 3230 382e   348.745501 208.
 00001200: 3939 3837 3134 200a 4c20 3336 322e 3937  998714 .L 362.97
 00001210: 3539 3333 2032 3038 2e39 3938 3731 3420  5933 208.998714 
 00001220: 0a4c 2033 3632 2e39 3735 3933 3320 3731  .L 362.975933 71
 00001230: 2e32 3631 3837 3220 0a4c 2033 3438 2e37  .261872 .L 348.7
 00001240: 3435 3530 3120 3731 2e32 3631 3837 3220  45501 71.261872 
 00001250: 0a7a 0a22 2063 6c69 702d 7061 7468 3d22  .z." clip-path="
-00001260: 7572 6c28 2370 3235 3537 6331 3263 6136  url(#p2557c12ca6
+00001260: 7572 6c28 2370 6364 3037 3930 6366 3438  url(#pcd0790cf48
 00001270: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00001280: 2334 6661 6362 6422 2f3e 0a20 2020 3c2f  #4facbd"/>.   </
 00001290: 673e 0a20 2020 3c67 2069 643d 2270 6174  g>.   <g id="pat
 000012a0: 6368 5f32 3122 3e0a 2020 2020 3c70 6174  ch_21">.    <pat
 000012b0: 6820 643d 224d 2033 3636 2e35 3333 3534  h d="M 366.53354
 000012c0: 3220 3230 382e 3939 3837 3134 200a 4c20  2 208.998714 .L 
 000012d0: 3338 302e 3736 3339 3734 2032 3038 2e39  380.763974 208.9
 000012e0: 3938 3731 3420 0a4c 2033 3830 2e37 3633  98714 .L 380.763
 000012f0: 3937 3420 3530 2e38 3536 3431 3320 0a4c  974 50.856413 .L
 00001300: 2033 3636 2e35 3333 3534 3220 3530 2e38   366.533542 50.8
 00001310: 3536 3431 3320 0a7a 0a22 2063 6c69 702d  56413 .z." clip-
-00001320: 7061 7468 3d22 7572 6c28 2370 3235 3537  path="url(#p2557
-00001330: 6331 3263 6136 2922 2073 7479 6c65 3d22  c12ca6)" style="
+00001320: 7061 7468 3d22 7572 6c28 2370 6364 3037  path="url(#pcd07
+00001330: 3930 6366 3438 2922 2073 7479 6c65 3d22  90cf48)" style="
 00001340: 6669 6c6c 3a20 2335 3261 6363 6222 2f3e  fill: #52accb"/>
 00001350: 0a20 2020 3c2f 673e 0a20 2020 3c67 2069  .   </g>.   <g i
 00001360: 643d 2270 6174 6368 5f32 3222 3e0a 2020  d="patch_22">.  
 00001370: 2020 3c70 6174 6820 643d 224d 2033 3834    <path d="M 384
 00001380: 2e33 3231 3538 3320 3230 382e 3939 3837  .321583 208.9987
 00001390: 3134 200a 4c20 3339 382e 3535 3230 3135  14 .L 398.552015
 000013a0: 2032 3038 2e39 3938 3731 3420 0a4c 2033   208.998714 .L 3
 000013b0: 3938 2e35 3532 3031 3520 3132 322e 3237  98.552015 122.27
 000013c0: 3535 3137 200a 4c20 3338 342e 3332 3135  5517 .L 384.3215
 000013d0: 3833 2031 3232 2e32 3735 3531 3720 0a7a  83 122.275517 .z
 000013e0: 0a22 2063 6c69 702d 7061 7468 3d22 7572  ." clip-path="ur
-000013f0: 6c28 2370 3235 3537 6331 3263 6136 2922  l(#p2557c12ca6)"
+000013f0: 6c28 2370 6364 3037 3930 6366 3438 2922  l(#pcd0790cf48)"
 00001400: 2073 7479 6c65 3d22 6669 6c6c 3a20 2335   style="fill: #5
 00001410: 3761 6464 6622 2f3e 0a20 2020 3c2f 673e  7addf"/>.   </g>
 00001420: 0a20 2020 3c67 2069 643d 2270 6174 6368  .   <g id="patch
 00001430: 5f32 3322 3e0a 2020 2020 3c70 6174 6820  _23">.    <path 
 00001440: 643d 224d 2034 3032 2e31 3039 3632 3320  d="M 402.109623 
 00001450: 3230 382e 3939 3837 3134 200a 4c20 3431  208.998714 .L 41
 00001460: 362e 3334 3030 3536 2032 3038 2e39 3938  6.340056 208.998
 00001470: 3731 3420 0a4c 2034 3136 2e33 3430 3035  714 .L 416.34005
 00001480: 3620 3133 372e 3537 3936 3131 200a 4c20  6 137.579611 .L 
 00001490: 3430 322e 3130 3936 3233 2031 3337 2e35  402.109623 137.5
 000014a0: 3739 3631 3120 0a7a 0a22 2063 6c69 702d  79611 .z." clip-
-000014b0: 7061 7468 3d22 7572 6c28 2370 3235 3537  path="url(#p2557
-000014c0: 6331 3263 6136 2922 2073 7479 6c65 3d22  c12ca6)" style="
+000014b0: 7061 7468 3d22 7572 6c28 2370 6364 3037  path="url(#pcd07
+000014c0: 3930 6366 3438 2922 2073 7479 6c65 3d22  90cf48)" style="
 000014d0: 6669 6c6c 3a20 2338 6161 6565 3722 2f3e  fill: #8aaee7"/>
 000014e0: 0a20 2020 3c2f 673e 0a20 2020 3c67 2069  .   </g>.   <g i
 000014f0: 643d 2270 6174 6368 5f32 3422 3e0a 2020  d="patch_24">.  
 00001500: 2020 3c70 6174 6820 643d 224d 2034 3139    <path d="M 419
 00001510: 2e38 3937 3636 3420 3230 382e 3939 3837  .897664 208.9987
 00001520: 3134 200a 4c20 3433 342e 3132 3830 3937  14 .L 434.128097
 00001530: 2032 3038 2e39 3938 3731 3420 0a4c 2034   208.998714 .L 4
 00001540: 3334 2e31 3238 3039 3720 3831 2e34 3634  34.128097 81.464
 00001550: 3630 3120 0a4c 2034 3139 2e38 3937 3636  601 .L 419.89766
 00001560: 3420 3831 2e34 3634 3630 3120 0a7a 0a22  4 81.464601 .z."
 00001570: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00001580: 2370 3235 3537 6331 3263 6136 2922 2073  #p2557c12ca6)" s
+00001580: 2370 6364 3037 3930 6366 3438 2922 2073  #pcd0790cf48)" s
 00001590: 7479 6c65 3d22 6669 6c6c 3a20 2361 3761  tyle="fill: #a7a
 000015a0: 6365 6222 2f3e 0a20 2020 3c2f 673e 0a20  ceb"/>.   </g>. 
 000015b0: 2020 3c67 2069 643d 2270 6174 6368 5f32    <g id="patch_2
 000015c0: 3522 3e0a 2020 2020 3c70 6174 6820 643d  5">.    <path d=
 000015d0: 224d 2034 3337 2e36 3835 3730 3520 3230  "M 437.685705 20
 000015e0: 382e 3939 3837 3134 200a 4c20 3435 312e  8.998714 .L 451.
 000015f0: 3931 3631 3338 2032 3038 2e39 3938 3731  916138 208.99871
 00001600: 3420 0a4c 2034 3531 2e39 3136 3133 3820  4 .L 451.916138 
 00001610: 3137 332e 3238 3931 3632 200a 4c20 3433  173.289162 .L 43
 00001620: 372e 3638 3537 3035 2031 3733 2e32 3839  7.685705 173.289
 00001630: 3136 3220 0a7a 0a22 2063 6c69 702d 7061  162 .z." clip-pa
-00001640: 7468 3d22 7572 6c28 2370 3235 3537 6331  th="url(#p2557c1
-00001650: 3263 6136 2922 2073 7479 6c65 3d22 6669  2ca6)" style="fi
+00001640: 7468 3d22 7572 6c28 2370 6364 3037 3930  th="url(#pcd0790
+00001650: 6366 3438 2922 2073 7479 6c65 3d22 6669  cf48)" style="fi
 00001660: 6c6c 3a20 2362 6161 3665 6122 2f3e 0a20  ll: #baa6ea"/>. 
 00001670: 2020 3c2f 673e 0a20 2020 3c67 2069 643d    </g>.   <g id=
 00001680: 2270 6174 6368 5f32 3622 3e0a 2020 2020  "patch_26">.    
 00001690: 3c70 6174 6820 643d 224d 2034 3535 2e34  <path d="M 455.4
 000016a0: 3733 3734 3620 3230 382e 3939 3837 3134  73746 208.998714
 000016b0: 200a 4c20 3436 392e 3730 3431 3739 2032   .L 469.704179 2
 000016c0: 3038 2e39 3938 3731 3420 0a4c 2034 3639  08.998714 .L 469
 000016d0: 2e37 3034 3137 3920 3135 322e 3838 3337  .704179 152.8837
 000016e0: 3034 200a 4c20 3435 352e 3437 3337 3436  04 .L 455.473746
 000016f0: 2031 3532 2e38 3833 3730 3420 0a7a 0a22   152.883704 .z."
 00001700: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00001710: 2370 3235 3537 6331 3263 6136 2922 2073  #p2557c12ca6)" s
+00001710: 2370 6364 3037 3930 6366 3438 2922 2073  #pcd0790cf48)" s
 00001720: 7479 6c65 3d22 6669 6c6c 3a20 2363 6239  tyle="fill: #cb9
 00001730: 6465 3922 2f3e 0a20 2020 3c2f 673e 0a20  de9"/>.   </g>. 
 00001740: 2020 3c67 2069 643d 2270 6174 6368 5f32    <g id="patch_2
 00001750: 3722 3e0a 2020 2020 3c70 6174 6820 643d  7">.    <path d=
 00001760: 224d 2034 3733 2e32 3631 3738 3720 3230  "M 473.261787 20
 00001770: 382e 3939 3837 3134 200a 4c20 3438 372e  8.998714 .L 487.
 00001780: 3439 3232 3220 3230 382e 3939 3837 3134  49222 208.998714
 00001790: 200a 4c20 3438 372e 3439 3232 3220 3134   .L 487.49222 14
 000017a0: 322e 3638 3039 3735 200a 4c20 3437 332e  2.680975 .L 473.
 000017b0: 3236 3137 3837 2031 3432 2e36 3830 3937  261787 142.68097
 000017c0: 3520 0a7a 0a22 2063 6c69 702d 7061 7468  5 .z." clip-path
-000017d0: 3d22 7572 6c28 2370 3235 3537 6331 3263  ="url(#p2557c12c
-000017e0: 6136 2922 2073 7479 6c65 3d22 6669 6c6c  a6)" style="fill
+000017d0: 3d22 7572 6c28 2370 6364 3037 3930 6366  ="url(#pcd0790cf
+000017e0: 3438 2922 2073 7479 6c65 3d22 6669 6c6c  48)" style="fill
 000017f0: 3a20 2364 6339 3265 3822 2f3e 0a20 2020  : #dc92e8"/>.   
 00001800: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 00001810: 6174 6368 5f32 3822 3e0a 2020 2020 3c70  atch_28">.    <p
 00001820: 6174 6820 643d 224d 2034 3931 2e30 3439  ath d="M 491.049
 00001830: 3832 3820 3230 382e 3939 3837 3134 200a  828 208.998714 .
 00001840: 4c20 3530 352e 3238 3032 3631 2032 3038  L 505.280261 208
 00001850: 2e39 3938 3731 3420 0a4c 2035 3035 2e32  .998714 .L 505.2
 00001860: 3830 3236 3120 3134 322e 3638 3039 3735  80261 142.680975
 00001870: 200a 4c20 3439 312e 3034 3938 3238 2031   .L 491.049828 1
 00001880: 3432 2e36 3830 3937 3520 0a7a 0a22 2063  42.680975 .z." c
 00001890: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-000018a0: 3235 3537 6331 3263 6136 2922 2073 7479  2557c12ca6)" sty
+000018a0: 6364 3037 3930 6366 3438 2922 2073 7479  cd0790cf48)" sty
 000018b0: 6c65 3d22 6669 6c6c 3a20 2365 3738 6264  le="fill: #e78bd
 000018c0: 6522 2f3e 0a20 2020 3c2f 673e 0a20 2020  e"/>.   </g>.   
 000018d0: 3c67 2069 643d 2270 6174 6368 5f32 3922  <g id="patch_29"
 000018e0: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 000018f0: 2035 3038 2e38 3337 3836 3920 3230 382e   508.837869 208.
 00001900: 3939 3837 3134 200a 4c20 3532 332e 3036  998714 .L 523.06
 00001910: 3833 3032 2032 3038 2e39 3938 3731 3420  8302 208.998714 
 00001920: 0a4c 2035 3233 2e30 3638 3330 3220 3134  .L 523.068302 14
 00001930: 322e 3638 3039 3735 200a 4c20 3530 382e  2.680975 .L 508.
 00001940: 3833 3738 3639 2031 3432 2e36 3830 3937  837869 142.68097
 00001950: 3520 0a7a 0a22 2063 6c69 702d 7061 7468  5 .z." clip-path
-00001960: 3d22 7572 6c28 2370 3235 3537 6331 3263  ="url(#p2557c12c
-00001970: 6136 2922 2073 7479 6c65 3d22 6669 6c6c  a6)" style="fill
+00001960: 3d22 7572 6c28 2370 6364 3037 3930 6366  ="url(#pcd0790cf
+00001970: 3438 2922 2073 7479 6c65 3d22 6669 6c6c  48)" style="fill
 00001980: 3a20 2365 3838 6563 6522 2f3e 0a20 2020  : #e88ece"/>.   
 00001990: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 000019a0: 6174 6368 5f33 3022 3e0a 2020 2020 3c70  atch_30">.    <p
 000019b0: 6174 6820 643d 224d 2035 3236 2e36 3235  ath d="M 526.625
 000019c0: 3931 2032 3038 2e39 3938 3731 3420 0a4c  91 208.998714 .L
 000019d0: 2035 3430 2e38 3536 3334 3320 3230 382e   540.856343 208.
 000019e0: 3939 3837 3134 200a 4c20 3534 302e 3835  998714 .L 540.85
 000019f0: 3633 3433 2031 3638 2e31 3837 3739 3820  6343 168.187798 
 00001a00: 0a4c 2035 3236 2e36 3235 3931 2031 3638  .L 526.62591 168
 00001a10: 2e31 3837 3739 3820 0a7a 0a22 2063 6c69  .187798 .z." cli
-00001a20: 702d 7061 7468 3d22 7572 6c28 2370 3235  p-path="url(#p25
-00001a30: 3537 6331 3263 6136 2922 2073 7479 6c65  57c12ca6)" style
+00001a20: 702d 7061 7468 3d22 7572 6c28 2370 6364  p-path="url(#pcd
+00001a30: 3037 3930 6366 3438 2922 2073 7479 6c65  0790cf48)" style
 00001a40: 3d22 6669 6c6c 3a20 2365 3839 3163 3022  ="fill: #e891c0"
 00001a50: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 00001a60: 2069 643d 2270 6174 6368 5f33 3122 3e0a   id="patch_31">.
 00001a70: 2020 2020 3c70 6174 6820 643d 224d 2035      <path d="M 5
 00001a80: 3434 2e34 3133 3935 3120 3230 382e 3939  44.413951 208.99
 00001a90: 3837 3134 200a 4c20 3535 382e 3634 3433  8714 .L 558.6443
 00001aa0: 3834 2032 3038 2e39 3938 3731 3420 0a4c  84 208.998714 .L
 00001ab0: 2035 3538 2e36 3434 3338 3420 3139 332e   558.644384 193.
 00001ac0: 3639 3436 3231 200a 4c20 3534 342e 3431  694621 .L 544.41
 00001ad0: 3339 3531 2031 3933 2e36 3934 3632 3120  3951 193.694621 
 00001ae0: 0a7a 0a22 2063 6c69 702d 7061 7468 3d22  .z." clip-path="
-00001af0: 7572 6c28 2370 3235 3537 6331 3263 6136  url(#p2557c12ca6
+00001af0: 7572 6c28 2370 6364 3037 3930 6366 3438  url(#pcd0790cf48
 00001b00: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00001b10: 2365 3939 3462 3322 2f3e 0a20 2020 3c2f  #e994b3"/>.   </
 00001b20: 673e 0a20 2020 3c67 2069 643d 226d 6174  g>.   <g id="mat
 00001b30: 706c 6f74 6c69 622e 6178 6973 5f31 223e  plotlib.axis_1">
 00001b40: 0a20 2020 203c 6720 6964 3d22 7874 6963  .    <g id="xtic
 00001b50: 6b5f 3122 3e0a 2020 2020 203c 6720 6964  k_1">.     <g id
 00001b60: 3d22 6c69 6e65 3264 5f31 223e 0a20 2020  ="line2d_1">.   
 00001b70: 2020 203c 6465 6673 3e0a 2020 2020 2020     <defs>.      
-00001b80: 203c 7061 7468 2069 643d 226d 3734 3137   <path id="m7417
-00001b90: 3866 3636 3434 2220 643d 224d 2030 2030  8f6644" d="M 0 0
+00001b80: 203c 7061 7468 2069 643d 226d 6332 3738   <path id="mc278
+00001b90: 3465 6337 3366 2220 643d 224d 2030 2030  4ec73f" d="M 0 0
 00001ba0: 200a 4c20 3020 332e 3520 0a22 2073 7479   .L 0 3.5 ." sty
 00001bb0: 6c65 3d22 7374 726f 6b65 3a20 2330 3030  le="stroke: #000
 00001bc0: 3030 303b 2073 7472 6f6b 652d 7769 6474  000; stroke-widt
 00001bd0: 683a 2030 2e38 222f 3e0a 2020 2020 2020  h: 0.8"/>.      
 00001be0: 3c2f 6465 6673 3e0a 2020 2020 2020 3c67  </defs>.      <g
 00001bf0: 3e0a 2020 2020 2020 203c 7573 6520 786c  >.       <use xl
-00001c00: 696e 6b3a 6872 6566 3d22 236d 3734 3137  ink:href="#m7417
-00001c10: 3866 3636 3434 2220 783d 2235 332e 3436  8f6644" x="53.46
+00001c00: 696e 6b3a 6872 6566 3d22 236d 6332 3738  ink:href="#mc278
+00001c10: 3465 6337 3366 2220 783d 2235 332e 3436  4ec73f" x="53.46
 00001c20: 3430 3222 2079 3d22 3230 382e 3939 3837  402" y="208.9987
 00001c30: 3134 2220 7374 796c 653d 2273 7472 6f6b  14" style="strok
 00001c40: 653a 2023 3030 3030 3030 3b20 7374 726f  e: #000000; stro
 00001c50: 6b65 2d77 6964 7468 3a20 302e 3822 2f3e  ke-width: 0.8"/>
 00001c60: 0a20 2020 2020 203c 2f67 3e0a 2020 2020  .      </g>.    
 00001c70: 203c 2f67 3e0a 2020 2020 203c 6720 6964   </g>.     <g id
 00001c80: 3d22 7465 7874 5f31 223e 0a20 2020 2020  ="text_1">.     
@@ -563,15 +563,15 @@
 00002320: 3431 222f 3e0a 2020 2020 2020 3c2f 673e  41"/>.      </g>
 00002330: 0a20 2020 2020 3c2f 673e 0a20 2020 203c  .     </g>.    <
 00002340: 2f67 3e0a 2020 2020 3c67 2069 643d 2278  /g>.    <g id="x
 00002350: 7469 636b 5f32 223e 0a20 2020 2020 3c67  tick_2">.     <g
 00002360: 2069 643d 226c 696e 6532 645f 3222 3e0a   id="line2d_2">.
 00002370: 2020 2020 2020 3c67 3e0a 2020 2020 2020        <g>.      
 00002380: 203c 7573 6520 786c 696e 6b3a 6872 6566   <use xlink:href
-00002390: 3d22 236d 3734 3137 3866 3636 3434 2220  ="#m74178f6644" 
+00002390: 3d22 236d 6332 3738 3465 6337 3366 2220  ="#mc2784ec73f" 
 000023a0: 783d 2237 312e 3235 3230 3631 2220 793d  x="71.252061" y=
 000023b0: 2232 3038 2e39 3938 3731 3422 2073 7479  "208.998714" sty
 000023c0: 6c65 3d22 7374 726f 6b65 3a20 2330 3030  le="stroke: #000
 000023d0: 3030 303b 2073 7472 6f6b 652d 7769 6474  000; stroke-widt
 000023e0: 683a 2030 2e38 222f 3e0a 2020 2020 2020  h: 0.8"/>.      
 000023f0: 3c2f 673e 0a20 2020 2020 3c2f 673e 0a20  </g>.     </g>. 
 00002400: 2020 2020 3c67 2069 643d 2274 6578 745f      <g id="text_
@@ -639,16 +639,16 @@
 000027e0: 3139 302e 3836 3931 3431 222f 3e0a 2020  190.869141"/>.  
 000027f0: 2020 2020 3c2f 673e 0a20 2020 2020 3c2f      </g>.     </
 00002800: 673e 0a20 2020 203c 2f67 3e0a 2020 2020  g>.    </g>.    
 00002810: 3c67 2069 643d 2278 7469 636b 5f33 223e  <g id="xtick_3">
 00002820: 0a20 2020 2020 3c67 2069 643d 226c 696e  .     <g id="lin
 00002830: 6532 645f 3322 3e0a 2020 2020 2020 3c67  e2d_3">.      <g
 00002840: 3e0a 2020 2020 2020 203c 7573 6520 786c  >.       <use xl
-00002850: 696e 6b3a 6872 6566 3d22 236d 3734 3137  ink:href="#m7417
-00002860: 3866 3636 3434 2220 783d 2238 392e 3034  8f6644" x="89.04
+00002850: 696e 6b3a 6872 6566 3d22 236d 6332 3738  ink:href="#mc278
+00002860: 3465 6337 3366 2220 783d 2238 392e 3034  4ec73f" x="89.04
 00002870: 3031 3032 2220 793d 2232 3038 2e39 3938  0102" y="208.998
 00002880: 3731 3422 2073 7479 6c65 3d22 7374 726f  714" style="stro
 00002890: 6b65 3a20 2330 3030 3030 303b 2073 7472  ke: #000000; str
 000028a0: 6f6b 652d 7769 6474 683a 2030 2e38 222f  oke-width: 0.8"/
 000028b0: 3e0a 2020 2020 2020 3c2f 673e 0a20 2020  >.      </g>.   
 000028c0: 2020 3c2f 673e 0a20 2020 2020 3c67 2069    </g>.     <g i
 000028d0: 643d 2274 6578 745f 3322 3e0a 2020 2020  d="text_3">.    
@@ -686,15 +686,15 @@
 00002ad0: 3931 3431 222f 3e0a 2020 2020 2020 3c2f  9141"/>.      </
 00002ae0: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 00002af0: 203c 2f67 3e0a 2020 2020 3c67 2069 643d   </g>.    <g id=
 00002b00: 2278 7469 636b 5f34 223e 0a20 2020 2020  "xtick_4">.     
 00002b10: 3c67 2069 643d 226c 696e 6532 645f 3422  <g id="line2d_4"
 00002b20: 3e0a 2020 2020 2020 3c67 3e0a 2020 2020  >.      <g>.    
 00002b30: 2020 203c 7573 6520 786c 696e 6b3a 6872     <use xlink:hr
-00002b40: 6566 3d22 236d 3734 3137 3866 3636 3434  ef="#m74178f6644
+00002b40: 6566 3d22 236d 6332 3738 3465 6337 3366  ef="#mc2784ec73f
 00002b50: 2220 783d 2231 3036 2e38 3238 3134 3322  " x="106.828143"
 00002b60: 2079 3d22 3230 382e 3939 3837 3134 2220   y="208.998714" 
 00002b70: 7374 796c 653d 2273 7472 6f6b 653a 2023  style="stroke: #
 00002b80: 3030 3030 3030 3b20 7374 726f 6b65 2d77  000000; stroke-w
 00002b90: 6964 7468 3a20 302e 3822 2f3e 0a20 2020  idth: 0.8"/>.   
 00002ba0: 2020 203c 2f67 3e0a 2020 2020 203c 2f67     </g>.     </g
 00002bb0: 3e0a 2020 2020 203c 6720 6964 3d22 7465  >.     <g id="te
@@ -774,15 +774,15 @@
 00003050: 3122 2f3e 0a20 2020 2020 203c 2f67 3e0a  1"/>.      </g>.
 00003060: 2020 2020 203c 2f67 3e0a 2020 2020 3c2f       </g>.    </
 00003070: 673e 0a20 2020 203c 6720 6964 3d22 7874  g>.    <g id="xt
 00003080: 6963 6b5f 3522 3e0a 2020 2020 203c 6720  ick_5">.     <g 
 00003090: 6964 3d22 6c69 6e65 3264 5f35 223e 0a20  id="line2d_5">. 
 000030a0: 2020 2020 203c 673e 0a20 2020 2020 2020       <g>.       
 000030b0: 3c75 7365 2078 6c69 6e6b 3a68 7265 663d  <use xlink:href=
-000030c0: 2223 6d37 3431 3738 6636 3634 3422 2078  "#m74178f6644" x
+000030c0: 2223 6d63 3237 3834 6563 3733 6622 2078  "#mc2784ec73f" x
 000030d0: 3d22 3132 342e 3631 3631 3834 2220 793d  ="124.616184" y=
 000030e0: 2232 3038 2e39 3938 3731 3422 2073 7479  "208.998714" sty
 000030f0: 6c65 3d22 7374 726f 6b65 3a20 2330 3030  le="stroke: #000
 00003100: 3030 303b 2073 7472 6f6b 652d 7769 6474  000; stroke-widt
 00003110: 683a 2030 2e38 222f 3e0a 2020 2020 2020  h: 0.8"/>.      
 00003120: 3c2f 673e 0a20 2020 2020 3c2f 673e 0a20  </g>.     </g>. 
 00003130: 2020 2020 3c67 2069 643d 2274 6578 745f      <g id="text_
@@ -808,16 +808,16 @@
 00003270: 2231 3930 2e38 3639 3134 3122 2f3e 0a20  "190.869141"/>. 
 00003280: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00003290: 2f67 3e0a 2020 2020 3c2f 673e 0a20 2020  /g>.    </g>.   
 000032a0: 203c 6720 6964 3d22 7874 6963 6b5f 3622   <g id="xtick_6"
 000032b0: 3e0a 2020 2020 203c 6720 6964 3d22 6c69  >.     <g id="li
 000032c0: 6e65 3264 5f36 223e 0a20 2020 2020 203c  ne2d_6">.      <
 000032d0: 673e 0a20 2020 2020 2020 3c75 7365 2078  g>.       <use x
-000032e0: 6c69 6e6b 3a68 7265 663d 2223 6d37 3431  link:href="#m741
-000032f0: 3738 6636 3634 3422 2078 3d22 3134 322e  78f6644" x="142.
+000032e0: 6c69 6e6b 3a68 7265 663d 2223 6d63 3237  link:href="#mc27
+000032f0: 3834 6563 3733 6622 2078 3d22 3134 322e  84ec73f" x="142.
 00003300: 3430 3432 3235 2220 793d 2232 3038 2e39  404225" y="208.9
 00003310: 3938 3731 3422 2073 7479 6c65 3d22 7374  98714" style="st
 00003320: 726f 6b65 3a20 2330 3030 3030 303b 2073  roke: #000000; s
 00003330: 7472 6f6b 652d 7769 6474 683a 2030 2e38  troke-width: 0.8
 00003340: 222f 3e0a 2020 2020 2020 3c2f 673e 0a20  "/>.      </g>. 
 00003350: 2020 2020 3c2f 673e 0a20 2020 2020 3c67      </g>.     <g
 00003360: 2069 643d 2274 6578 745f 3622 3e0a 2020   id="text_6">.  
@@ -904,16 +904,16 @@
 00003870: 2e38 3639 3134 3122 2f3e 0a20 2020 2020  .869141"/>.     
 00003880: 203c 2f67 3e0a 2020 2020 203c 2f67 3e0a   </g>.     </g>.
 00003890: 2020 2020 3c2f 673e 0a20 2020 203c 6720      </g>.    <g 
 000038a0: 6964 3d22 7874 6963 6b5f 3722 3e0a 2020  id="xtick_7">.  
 000038b0: 2020 203c 6720 6964 3d22 6c69 6e65 3264     <g id="line2d
 000038c0: 5f37 223e 0a20 2020 2020 203c 673e 0a20  _7">.      <g>. 
 000038d0: 2020 2020 2020 3c75 7365 2078 6c69 6e6b        <use xlink
-000038e0: 3a68 7265 663d 2223 6d37 3431 3738 6636  :href="#m74178f6
-000038f0: 3634 3422 2078 3d22 3136 302e 3139 3232  644" x="160.1922
+000038e0: 3a68 7265 663d 2223 6d63 3237 3834 6563  :href="#mc2784ec
+000038f0: 3733 6622 2078 3d22 3136 302e 3139 3232  73f" x="160.1922
 00003900: 3636 2220 793d 2232 3038 2e39 3938 3731  66" y="208.99871
 00003910: 3422 2073 7479 6c65 3d22 7374 726f 6b65  4" style="stroke
 00003920: 3a20 2330 3030 3030 303b 2073 7472 6f6b  : #000000; strok
 00003930: 652d 7769 6474 683a 2030 2e38 222f 3e0a  e-width: 0.8"/>.
 00003940: 2020 2020 2020 3c2f 673e 0a20 2020 2020        </g>.     
 00003950: 3c2f 673e 0a20 2020 2020 3c67 2069 643d  </g>.     <g id=
 00003960: 2274 6578 745f 3722 3e0a 2020 2020 2020  "text_7">.      
@@ -939,15 +939,15 @@
 00003aa0: 3122 2f3e 0a20 2020 2020 203c 2f67 3e0a  1"/>.      </g>.
 00003ab0: 2020 2020 203c 2f67 3e0a 2020 2020 3c2f       </g>.    </
 00003ac0: 673e 0a20 2020 203c 6720 6964 3d22 7874  g>.    <g id="xt
 00003ad0: 6963 6b5f 3822 3e0a 2020 2020 203c 6720  ick_8">.     <g 
 00003ae0: 6964 3d22 6c69 6e65 3264 5f38 223e 0a20  id="line2d_8">. 
 00003af0: 2020 2020 203c 673e 0a20 2020 2020 2020       <g>.       
 00003b00: 3c75 7365 2078 6c69 6e6b 3a68 7265 663d  <use xlink:href=
-00003b10: 2223 6d37 3431 3738 6636 3634 3422 2078  "#m74178f6644" x
+00003b10: 2223 6d63 3237 3834 6563 3733 6622 2078  "#mc2784ec73f" x
 00003b20: 3d22 3137 372e 3938 3033 3037 2220 793d  ="177.980307" y=
 00003b30: 2232 3038 2e39 3938 3731 3422 2073 7479  "208.998714" sty
 00003b40: 6c65 3d22 7374 726f 6b65 3a20 2330 3030  le="stroke: #000
 00003b50: 3030 303b 2073 7472 6f6b 652d 7769 6474  000; stroke-widt
 00003b60: 683a 2030 2e38 222f 3e0a 2020 2020 2020  h: 0.8"/>.      
 00003b70: 3c2f 673e 0a20 2020 2020 3c2f 673e 0a20  </g>.     </g>. 
 00003b80: 2020 2020 3c67 2069 643d 2274 6578 745f      <g id="text_
@@ -973,16 +973,16 @@
 00003cc0: 2231 3930 2e38 3639 3134 3122 2f3e 0a20  "190.869141"/>. 
 00003cd0: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00003ce0: 2f67 3e0a 2020 2020 3c2f 673e 0a20 2020  /g>.    </g>.   
 00003cf0: 203c 6720 6964 3d22 7874 6963 6b5f 3922   <g id="xtick_9"
 00003d00: 3e0a 2020 2020 203c 6720 6964 3d22 6c69  >.     <g id="li
 00003d10: 6e65 3264 5f39 223e 0a20 2020 2020 203c  ne2d_9">.      <
 00003d20: 673e 0a20 2020 2020 2020 3c75 7365 2078  g>.       <use x
-00003d30: 6c69 6e6b 3a68 7265 663d 2223 6d37 3431  link:href="#m741
-00003d40: 3738 6636 3634 3422 2078 3d22 3139 352e  78f6644" x="195.
+00003d30: 6c69 6e6b 3a68 7265 663d 2223 6d63 3237  link:href="#mc27
+00003d40: 3834 6563 3733 6622 2078 3d22 3139 352e  84ec73f" x="195.
 00003d50: 3736 3833 3438 2220 793d 2232 3038 2e39  768348" y="208.9
 00003d60: 3938 3731 3422 2073 7479 6c65 3d22 7374  98714" style="st
 00003d70: 726f 6b65 3a20 2330 3030 3030 303b 2073  roke: #000000; s
 00003d80: 7472 6f6b 652d 7769 6474 683a 2030 2e38  troke-width: 0.8
 00003d90: 222f 3e0a 2020 2020 2020 3c2f 673e 0a20  "/>.      </g>. 
 00003da0: 2020 2020 3c2f 673e 0a20 2020 2020 3c67      </g>.     <g
 00003db0: 2069 643d 2274 6578 745f 3922 3e0a 2020   id="text_9">.  
@@ -1052,16 +1052,16 @@
 000041b0: 3639 3134 3122 2f3e 0a20 2020 2020 203c  69141"/>.      <
 000041c0: 2f67 3e0a 2020 2020 203c 2f67 3e0a 2020  /g>.     </g>.  
 000041d0: 2020 3c2f 673e 0a20 2020 203c 6720 6964    </g>.    <g id
 000041e0: 3d22 7874 6963 6b5f 3130 223e 0a20 2020  ="xtick_10">.   
 000041f0: 2020 3c67 2069 643d 226c 696e 6532 645f    <g id="line2d_
 00004200: 3130 223e 0a20 2020 2020 203c 673e 0a20  10">.      <g>. 
 00004210: 2020 2020 2020 3c75 7365 2078 6c69 6e6b        <use xlink
-00004220: 3a68 7265 663d 2223 6d37 3431 3738 6636  :href="#m74178f6
-00004230: 3634 3422 2078 3d22 3231 332e 3535 3633  644" x="213.5563
+00004220: 3a68 7265 663d 2223 6d63 3237 3834 6563  :href="#mc2784ec
+00004230: 3733 6622 2078 3d22 3231 332e 3535 3633  73f" x="213.5563
 00004240: 3839 2220 793d 2232 3038 2e39 3938 3731  89" y="208.99871
 00004250: 3422 2073 7479 6c65 3d22 7374 726f 6b65  4" style="stroke
 00004260: 3a20 2330 3030 3030 303b 2073 7472 6f6b  : #000000; strok
 00004270: 652d 7769 6474 683a 2030 2e38 222f 3e0a  e-width: 0.8"/>.
 00004280: 2020 2020 2020 3c2f 673e 0a20 2020 2020        </g>.     
 00004290: 3c2f 673e 0a20 2020 2020 3c67 2069 643d  </g>.     <g id=
 000042a0: 2274 6578 745f 3130 223e 0a20 2020 2020  "text_10">.     
@@ -1106,15 +1106,15 @@
 00004510: 3e0a 2020 2020 2020 3c2f 673e 0a20 2020  >.      </g>.   
 00004520: 2020 3c2f 673e 0a20 2020 203c 2f67 3e0a    </g>.    </g>.
 00004530: 2020 2020 3c67 2069 643d 2278 7469 636b      <g id="xtick
 00004540: 5f31 3122 3e0a 2020 2020 203c 6720 6964  _11">.     <g id
 00004550: 3d22 6c69 6e65 3264 5f31 3122 3e0a 2020  ="line2d_11">.  
 00004560: 2020 2020 3c67 3e0a 2020 2020 2020 203c      <g>.       <
 00004570: 7573 6520 786c 696e 6b3a 6872 6566 3d22  use xlink:href="
-00004580: 236d 3734 3137 3866 3636 3434 2220 783d  #m74178f6644" x=
+00004580: 236d 6332 3738 3465 6337 3366 2220 783d  #mc2784ec73f" x=
 00004590: 2232 3331 2e33 3434 3433 2220 793d 2232  "231.34443" y="2
 000045a0: 3038 2e39 3938 3731 3422 2073 7479 6c65  08.998714" style
 000045b0: 3d22 7374 726f 6b65 3a20 2330 3030 3030  ="stroke: #00000
 000045c0: 303b 2073 7472 6f6b 652d 7769 6474 683a  0; stroke-width:
 000045d0: 2030 2e38 222f 3e0a 2020 2020 2020 3c2f   0.8"/>.      </
 000045e0: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 000045f0: 2020 3c67 2069 643d 2274 6578 745f 3131    <g id="text_11
@@ -1140,16 +1140,16 @@
 00004730: 3139 302e 3836 3931 3431 222f 3e0a 2020  190.869141"/>.  
 00004740: 2020 2020 3c2f 673e 0a20 2020 2020 3c2f      </g>.     </
 00004750: 673e 0a20 2020 203c 2f67 3e0a 2020 2020  g>.    </g>.    
 00004760: 3c67 2069 643d 2278 7469 636b 5f31 3222  <g id="xtick_12"
 00004770: 3e0a 2020 2020 203c 6720 6964 3d22 6c69  >.     <g id="li
 00004780: 6e65 3264 5f31 3222 3e0a 2020 2020 2020  ne2d_12">.      
 00004790: 3c67 3e0a 2020 2020 2020 203c 7573 6520  <g>.       <use 
-000047a0: 786c 696e 6b3a 6872 6566 3d22 236d 3734  xlink:href="#m74
-000047b0: 3137 3866 3636 3434 2220 783d 2232 3439  178f6644" x="249
+000047a0: 786c 696e 6b3a 6872 6566 3d22 236d 6332  xlink:href="#mc2
+000047b0: 3738 3465 6337 3366 2220 783d 2232 3439  784ec73f" x="249
 000047c0: 2e31 3332 3437 3122 2079 3d22 3230 382e  .132471" y="208.
 000047d0: 3939 3837 3134 2220 7374 796c 653d 2273  998714" style="s
 000047e0: 7472 6f6b 653a 2023 3030 3030 3030 3b20  troke: #000000; 
 000047f0: 7374 726f 6b65 2d77 6964 7468 3a20 302e  stroke-width: 0.
 00004800: 3822 2f3e 0a20 2020 2020 203c 2f67 3e0a  8"/>.      </g>.
 00004810: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00004820: 6720 6964 3d22 7465 7874 5f31 3222 3e0a  g id="text_12">.
@@ -1175,16 +1175,16 @@
 00004960: 3836 3931 3431 222f 3e0a 2020 2020 2020  869141"/>.      
 00004970: 3c2f 673e 0a20 2020 2020 3c2f 673e 0a20  </g>.     </g>. 
 00004980: 2020 203c 2f67 3e0a 2020 2020 3c67 2069     </g>.    <g i
 00004990: 643d 2278 7469 636b 5f31 3322 3e0a 2020  d="xtick_13">.  
 000049a0: 2020 203c 6720 6964 3d22 6c69 6e65 3264     <g id="line2d
 000049b0: 5f31 3322 3e0a 2020 2020 2020 3c67 3e0a  _13">.      <g>.
 000049c0: 2020 2020 2020 203c 7573 6520 786c 696e         <use xlin
-000049d0: 6b3a 6872 6566 3d22 236d 3734 3137 3866  k:href="#m74178f
-000049e0: 3636 3434 2220 783d 2232 3636 2e39 3230  6644" x="266.920
+000049d0: 6b3a 6872 6566 3d22 236d 6332 3738 3465  k:href="#mc2784e
+000049e0: 6337 3366 2220 783d 2232 3636 2e39 3230  c73f" x="266.920
 000049f0: 3531 3222 2079 3d22 3230 382e 3939 3837  512" y="208.9987
 00004a00: 3134 2220 7374 796c 653d 2273 7472 6f6b  14" style="strok
 00004a10: 653a 2023 3030 3030 3030 3b20 7374 726f  e: #000000; stro
 00004a20: 6b65 2d77 6964 7468 3a20 302e 3822 2f3e  ke-width: 0.8"/>
 00004a30: 0a20 2020 2020 203c 2f67 3e0a 2020 2020  .      </g>.    
 00004a40: 203c 2f67 3e0a 2020 2020 203c 6720 6964   </g>.     <g id
 00004a50: 3d22 7465 7874 5f31 3322 3e0a 2020 2020  ="text_13">.    
@@ -1210,16 +1210,16 @@
 00004b90: 3134 3122 2f3e 0a20 2020 2020 203c 2f67  141"/>.      </g
 00004ba0: 3e0a 2020 2020 203c 2f67 3e0a 2020 2020  >.     </g>.    
 00004bb0: 3c2f 673e 0a20 2020 203c 6720 6964 3d22  </g>.    <g id="
 00004bc0: 7874 6963 6b5f 3134 223e 0a20 2020 2020  xtick_14">.     
 00004bd0: 3c67 2069 643d 226c 696e 6532 645f 3134  <g id="line2d_14
 00004be0: 223e 0a20 2020 2020 203c 673e 0a20 2020  ">.      <g>.   
 00004bf0: 2020 2020 3c75 7365 2078 6c69 6e6b 3a68      <use xlink:h
-00004c00: 7265 663d 2223 6d37 3431 3738 6636 3634  ref="#m74178f664
-00004c10: 3422 2078 3d22 3238 342e 3730 3835 3533  4" x="284.708553
+00004c00: 7265 663d 2223 6d63 3237 3834 6563 3733  ref="#mc2784ec73
+00004c10: 6622 2078 3d22 3238 342e 3730 3835 3533  f" x="284.708553
 00004c20: 2220 793d 2232 3038 2e39 3938 3731 3422  " y="208.998714"
 00004c30: 2073 7479 6c65 3d22 7374 726f 6b65 3a20   style="stroke: 
 00004c40: 2330 3030 3030 303b 2073 7472 6f6b 652d  #000000; stroke-
 00004c50: 7769 6474 683a 2030 2e38 222f 3e0a 2020  width: 0.8"/>.  
 00004c60: 2020 2020 3c2f 673e 0a20 2020 2020 3c2f      </g>.     </
 00004c70: 673e 0a20 2020 2020 3c67 2069 643d 2274  g>.     <g id="t
 00004c80: 6578 745f 3134 223e 0a20 2020 2020 203c  ext_14">.      <
@@ -1245,15 +1245,15 @@
 00004dc0: 222f 3e0a 2020 2020 2020 3c2f 673e 0a20  "/>.      </g>. 
 00004dd0: 2020 2020 3c2f 673e 0a20 2020 203c 2f67      </g>.    </g
 00004de0: 3e0a 2020 2020 3c67 2069 643d 2278 7469  >.    <g id="xti
 00004df0: 636b 5f31 3522 3e0a 2020 2020 203c 6720  ck_15">.     <g 
 00004e00: 6964 3d22 6c69 6e65 3264 5f31 3522 3e0a  id="line2d_15">.
 00004e10: 2020 2020 2020 3c67 3e0a 2020 2020 2020        <g>.      
 00004e20: 203c 7573 6520 786c 696e 6b3a 6872 6566   <use xlink:href
-00004e30: 3d22 236d 3734 3137 3866 3636 3434 2220  ="#m74178f6644" 
+00004e30: 3d22 236d 6332 3738 3465 6337 3366 2220  ="#mc2784ec73f" 
 00004e40: 783d 2233 3032 2e34 3936 3539 3422 2079  x="302.496594" y
 00004e50: 3d22 3230 382e 3939 3837 3134 2220 7374  ="208.998714" st
 00004e60: 796c 653d 2273 7472 6f6b 653a 2023 3030  yle="stroke: #00
 00004e70: 3030 3030 3b20 7374 726f 6b65 2d77 6964  0000; stroke-wid
 00004e80: 7468 3a20 302e 3822 2f3e 0a20 2020 2020  th: 0.8"/>.     
 00004e90: 203c 2f67 3e0a 2020 2020 203c 2f67 3e0a   </g>.     </g>.
 00004ea0: 2020 2020 203c 6720 6964 3d22 7465 7874       <g id="text
@@ -1280,15 +1280,15 @@
 00004ff0: 0a20 2020 2020 203c 2f67 3e0a 2020 2020  .      </g>.    
 00005000: 203c 2f67 3e0a 2020 2020 3c2f 673e 0a20   </g>.    </g>. 
 00005010: 2020 203c 6720 6964 3d22 7874 6963 6b5f     <g id="xtick_
 00005020: 3136 223e 0a20 2020 2020 3c67 2069 643d  16">.     <g id=
 00005030: 226c 696e 6532 645f 3136 223e 0a20 2020  "line2d_16">.   
 00005040: 2020 203c 673e 0a20 2020 2020 2020 3c75     <g>.       <u
 00005050: 7365 2078 6c69 6e6b 3a68 7265 663d 2223  se xlink:href="#
-00005060: 6d37 3431 3738 6636 3634 3422 2078 3d22  m74178f6644" x="
+00005060: 6d63 3237 3834 6563 3733 6622 2078 3d22  mc2784ec73f" x="
 00005070: 3332 302e 3238 3436 3335 2220 793d 2232  320.284635" y="2
 00005080: 3038 2e39 3938 3731 3422 2073 7479 6c65  08.998714" style
 00005090: 3d22 7374 726f 6b65 3a20 2330 3030 3030  ="stroke: #00000
 000050a0: 303b 2073 7472 6f6b 652d 7769 6474 683a  0; stroke-width:
 000050b0: 2030 2e38 222f 3e0a 2020 2020 2020 3c2f   0.8"/>.      </
 000050c0: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 000050d0: 2020 3c67 2069 643d 2274 6578 745f 3136    <g id="text_16
@@ -1314,16 +1314,16 @@
 00005210: 3139 302e 3836 3931 3431 222f 3e0a 2020  190.869141"/>.  
 00005220: 2020 2020 3c2f 673e 0a20 2020 2020 3c2f      </g>.     </
 00005230: 673e 0a20 2020 203c 2f67 3e0a 2020 2020  g>.    </g>.    
 00005240: 3c67 2069 643d 2278 7469 636b 5f31 3722  <g id="xtick_17"
 00005250: 3e0a 2020 2020 203c 6720 6964 3d22 6c69  >.     <g id="li
 00005260: 6e65 3264 5f31 3722 3e0a 2020 2020 2020  ne2d_17">.      
 00005270: 3c67 3e0a 2020 2020 2020 203c 7573 6520  <g>.       <use 
-00005280: 786c 696e 6b3a 6872 6566 3d22 236d 3734  xlink:href="#m74
-00005290: 3137 3866 3636 3434 2220 783d 2233 3338  178f6644" x="338
+00005280: 786c 696e 6b3a 6872 6566 3d22 236d 6332  xlink:href="#mc2
+00005290: 3738 3465 6337 3366 2220 783d 2233 3338  784ec73f" x="338
 000052a0: 2e30 3732 3637 3622 2079 3d22 3230 382e  .072676" y="208.
 000052b0: 3939 3837 3134 2220 7374 796c 653d 2273  998714" style="s
 000052c0: 7472 6f6b 653a 2023 3030 3030 3030 3b20  troke: #000000; 
 000052d0: 7374 726f 6b65 2d77 6964 7468 3a20 302e  stroke-width: 0.
 000052e0: 3822 2f3e 0a20 2020 2020 203c 2f67 3e0a  8"/>.      </g>.
 000052f0: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00005300: 6720 6964 3d22 7465 7874 5f31 3722 3e0a  g id="text_17">.
@@ -1349,16 +1349,16 @@
 00005440: 2e38 3639 3134 3122 2f3e 0a20 2020 2020  .869141"/>.     
 00005450: 203c 2f67 3e0a 2020 2020 203c 2f67 3e0a   </g>.     </g>.
 00005460: 2020 2020 3c2f 673e 0a20 2020 203c 6720      </g>.    <g 
 00005470: 6964 3d22 7874 6963 6b5f 3138 223e 0a20  id="xtick_18">. 
 00005480: 2020 2020 3c67 2069 643d 226c 696e 6532      <g id="line2
 00005490: 645f 3138 223e 0a20 2020 2020 203c 673e  d_18">.      <g>
 000054a0: 0a20 2020 2020 2020 3c75 7365 2078 6c69  .       <use xli
-000054b0: 6e6b 3a68 7265 663d 2223 6d37 3431 3738  nk:href="#m74178
-000054c0: 6636 3634 3422 2078 3d22 3335 352e 3836  f6644" x="355.86
+000054b0: 6e6b 3a68 7265 663d 2223 6d63 3237 3834  nk:href="#mc2784
+000054c0: 6563 3733 6622 2078 3d22 3335 352e 3836  ec73f" x="355.86
 000054d0: 3037 3137 2220 793d 2232 3038 2e39 3938  0717" y="208.998
 000054e0: 3731 3422 2073 7479 6c65 3d22 7374 726f  714" style="stro
 000054f0: 6b65 3a20 2330 3030 3030 303b 2073 7472  ke: #000000; str
 00005500: 6f6b 652d 7769 6474 683a 2030 2e38 222f  oke-width: 0.8"/
 00005510: 3e0a 2020 2020 2020 3c2f 673e 0a20 2020  >.      </g>.   
 00005520: 2020 3c2f 673e 0a20 2020 2020 3c67 2069    </g>.     <g i
 00005530: 643d 2274 6578 745f 3138 223e 0a20 2020  d="text_18">.   
@@ -1384,16 +1384,16 @@
 00005670: 3931 3431 222f 3e0a 2020 2020 2020 3c2f  9141"/>.      </
 00005680: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 00005690: 203c 2f67 3e0a 2020 2020 3c67 2069 643d   </g>.    <g id=
 000056a0: 2278 7469 636b 5f31 3922 3e0a 2020 2020  "xtick_19">.    
 000056b0: 203c 6720 6964 3d22 6c69 6e65 3264 5f31   <g id="line2d_1
 000056c0: 3922 3e0a 2020 2020 2020 3c67 3e0a 2020  9">.      <g>.  
 000056d0: 2020 2020 203c 7573 6520 786c 696e 6b3a       <use xlink:
-000056e0: 6872 6566 3d22 236d 3734 3137 3866 3636  href="#m74178f66
-000056f0: 3434 2220 783d 2233 3733 2e36 3438 3735  44" x="373.64875
+000056e0: 6872 6566 3d22 236d 6332 3738 3465 6337  href="#mc2784ec7
+000056f0: 3366 2220 783d 2233 3733 2e36 3438 3735  3f" x="373.64875
 00005700: 3822 2079 3d22 3230 382e 3939 3837 3134  8" y="208.998714
 00005710: 2220 7374 796c 653d 2273 7472 6f6b 653a  " style="stroke:
 00005720: 2023 3030 3030 3030 3b20 7374 726f 6b65   #000000; stroke
 00005730: 2d77 6964 7468 3a20 302e 3822 2f3e 0a20  -width: 0.8"/>. 
 00005740: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00005750: 2f67 3e0a 2020 2020 203c 6720 6964 3d22  /g>.     <g id="
 00005760: 7465 7874 5f31 3922 3e0a 2020 2020 2020  text_19">.      
@@ -1419,15 +1419,15 @@
 000058a0: 3122 2f3e 0a20 2020 2020 203c 2f67 3e0a  1"/>.      </g>.
 000058b0: 2020 2020 203c 2f67 3e0a 2020 2020 3c2f       </g>.    </
 000058c0: 673e 0a20 2020 203c 6720 6964 3d22 7874  g>.    <g id="xt
 000058d0: 6963 6b5f 3230 223e 0a20 2020 2020 3c67  ick_20">.     <g
 000058e0: 2069 643d 226c 696e 6532 645f 3230 223e   id="line2d_20">
 000058f0: 0a20 2020 2020 203c 673e 0a20 2020 2020  .      <g>.     
 00005900: 2020 3c75 7365 2078 6c69 6e6b 3a68 7265    <use xlink:hre
-00005910: 663d 2223 6d37 3431 3738 6636 3634 3422  f="#m74178f6644"
+00005910: 663d 2223 6d63 3237 3834 6563 3733 6622  f="#mc2784ec73f"
 00005920: 2078 3d22 3339 312e 3433 3637 3939 2220   x="391.436799" 
 00005930: 793d 2232 3038 2e39 3938 3731 3422 2073  y="208.998714" s
 00005940: 7479 6c65 3d22 7374 726f 6b65 3a20 2330  tyle="stroke: #0
 00005950: 3030 3030 303b 2073 7472 6f6b 652d 7769  00000; stroke-wi
 00005960: 6474 683a 2030 2e38 222f 3e0a 2020 2020  dth: 0.8"/>.    
 00005970: 2020 3c2f 673e 0a20 2020 2020 3c2f 673e    </g>.     </g>
 00005980: 0a20 2020 2020 3c67 2069 643d 2274 6578  .     <g id="tex
@@ -1454,15 +1454,15 @@
 00005ad0: 3e0a 2020 2020 2020 3c2f 673e 0a20 2020  >.      </g>.   
 00005ae0: 2020 3c2f 673e 0a20 2020 203c 2f67 3e0a    </g>.    </g>.
 00005af0: 2020 2020 3c67 2069 643d 2278 7469 636b      <g id="xtick
 00005b00: 5f32 3122 3e0a 2020 2020 203c 6720 6964  _21">.     <g id
 00005b10: 3d22 6c69 6e65 3264 5f32 3122 3e0a 2020  ="line2d_21">.  
 00005b20: 2020 2020 3c67 3e0a 2020 2020 2020 203c      <g>.       <
 00005b30: 7573 6520 786c 696e 6b3a 6872 6566 3d22  use xlink:href="
-00005b40: 236d 3734 3137 3866 3636 3434 2220 783d  #m74178f6644" x=
+00005b40: 236d 6332 3738 3465 6337 3366 2220 783d  #mc2784ec73f" x=
 00005b50: 2234 3039 2e32 3234 3834 2220 793d 2232  "409.22484" y="2
 00005b60: 3038 2e39 3938 3731 3422 2073 7479 6c65  08.998714" style
 00005b70: 3d22 7374 726f 6b65 3a20 2330 3030 3030  ="stroke: #00000
 00005b80: 303b 2073 7472 6f6b 652d 7769 6474 683a  0; stroke-width:
 00005b90: 2030 2e38 222f 3e0a 2020 2020 2020 3c2f   0.8"/>.      </
 00005ba0: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 00005bb0: 2020 3c67 2069 643d 2274 6578 745f 3231    <g id="text_21
@@ -1488,16 +1488,16 @@
 00005cf0: 3139 302e 3836 3931 3431 222f 3e0a 2020  190.869141"/>.  
 00005d00: 2020 2020 3c2f 673e 0a20 2020 2020 3c2f      </g>.     </
 00005d10: 673e 0a20 2020 203c 2f67 3e0a 2020 2020  g>.    </g>.    
 00005d20: 3c67 2069 643d 2278 7469 636b 5f32 3222  <g id="xtick_22"
 00005d30: 3e0a 2020 2020 203c 6720 6964 3d22 6c69  >.     <g id="li
 00005d40: 6e65 3264 5f32 3222 3e0a 2020 2020 2020  ne2d_22">.      
 00005d50: 3c67 3e0a 2020 2020 2020 203c 7573 6520  <g>.       <use 
-00005d60: 786c 696e 6b3a 6872 6566 3d22 236d 3734  xlink:href="#m74
-00005d70: 3137 3866 3636 3434 2220 783d 2234 3237  178f6644" x="427
+00005d60: 786c 696e 6b3a 6872 6566 3d22 236d 6332  xlink:href="#mc2
+00005d70: 3738 3465 6337 3366 2220 783d 2234 3237  784ec73f" x="427
 00005d80: 2e30 3132 3838 3122 2079 3d22 3230 382e  .012881" y="208.
 00005d90: 3939 3837 3134 2220 7374 796c 653d 2273  998714" style="s
 00005da0: 7472 6f6b 653a 2023 3030 3030 3030 3b20  troke: #000000; 
 00005db0: 7374 726f 6b65 2d77 6964 7468 3a20 302e  stroke-width: 0.
 00005dc0: 3822 2f3e 0a20 2020 2020 203c 2f67 3e0a  8"/>.      </g>.
 00005dd0: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00005de0: 6720 6964 3d22 7465 7874 5f32 3222 3e0a  g id="text_22">.
@@ -1523,16 +1523,16 @@
 00005f20: 3836 3931 3431 222f 3e0a 2020 2020 2020  869141"/>.      
 00005f30: 3c2f 673e 0a20 2020 2020 3c2f 673e 0a20  </g>.     </g>. 
 00005f40: 2020 203c 2f67 3e0a 2020 2020 3c67 2069     </g>.    <g i
 00005f50: 643d 2278 7469 636b 5f32 3322 3e0a 2020  d="xtick_23">.  
 00005f60: 2020 203c 6720 6964 3d22 6c69 6e65 3264     <g id="line2d
 00005f70: 5f32 3322 3e0a 2020 2020 2020 3c67 3e0a  _23">.      <g>.
 00005f80: 2020 2020 2020 203c 7573 6520 786c 696e         <use xlin
-00005f90: 6b3a 6872 6566 3d22 236d 3734 3137 3866  k:href="#m74178f
-00005fa0: 3636 3434 2220 783d 2234 3434 2e38 3030  6644" x="444.800
+00005f90: 6b3a 6872 6566 3d22 236d 6332 3738 3465  k:href="#mc2784e
+00005fa0: 6337 3366 2220 783d 2234 3434 2e38 3030  c73f" x="444.800
 00005fb0: 3932 3222 2079 3d22 3230 382e 3939 3837  922" y="208.9987
 00005fc0: 3134 2220 7374 796c 653d 2273 7472 6f6b  14" style="strok
 00005fd0: 653a 2023 3030 3030 3030 3b20 7374 726f  e: #000000; stro
 00005fe0: 6b65 2d77 6964 7468 3a20 302e 3822 2f3e  ke-width: 0.8"/>
 00005ff0: 0a20 2020 2020 203c 2f67 3e0a 2020 2020  .      </g>.    
 00006000: 203c 2f67 3e0a 2020 2020 203c 6720 6964   </g>.     <g id
 00006010: 3d22 7465 7874 5f32 3322 3e0a 2020 2020  ="text_23">.    
@@ -1558,16 +1558,16 @@
 00006150: 3134 3122 2f3e 0a20 2020 2020 203c 2f67  141"/>.      </g
 00006160: 3e0a 2020 2020 203c 2f67 3e0a 2020 2020  >.     </g>.    
 00006170: 3c2f 673e 0a20 2020 203c 6720 6964 3d22  </g>.    <g id="
 00006180: 7874 6963 6b5f 3234 223e 0a20 2020 2020  xtick_24">.     
 00006190: 3c67 2069 643d 226c 696e 6532 645f 3234  <g id="line2d_24
 000061a0: 223e 0a20 2020 2020 203c 673e 0a20 2020  ">.      <g>.   
 000061b0: 2020 2020 3c75 7365 2078 6c69 6e6b 3a68      <use xlink:h
-000061c0: 7265 663d 2223 6d37 3431 3738 6636 3634  ref="#m74178f664
-000061d0: 3422 2078 3d22 3436 322e 3538 3839 3633  4" x="462.588963
+000061c0: 7265 663d 2223 6d63 3237 3834 6563 3733  ref="#mc2784ec73
+000061d0: 6622 2078 3d22 3436 322e 3538 3839 3633  f" x="462.588963
 000061e0: 2220 793d 2232 3038 2e39 3938 3731 3422  " y="208.998714"
 000061f0: 2073 7479 6c65 3d22 7374 726f 6b65 3a20   style="stroke: 
 00006200: 2330 3030 3030 303b 2073 7472 6f6b 652d  #000000; stroke-
 00006210: 7769 6474 683a 2030 2e38 222f 3e0a 2020  width: 0.8"/>.  
 00006220: 2020 2020 3c2f 673e 0a20 2020 2020 3c2f      </g>.     </
 00006230: 673e 0a20 2020 2020 3c67 2069 643d 2274  g>.     <g id="t
 00006240: 6578 745f 3234 223e 0a20 2020 2020 203c  ext_24">.      <
@@ -1593,15 +1593,15 @@
 00006380: 222f 3e0a 2020 2020 2020 3c2f 673e 0a20  "/>.      </g>. 
 00006390: 2020 2020 3c2f 673e 0a20 2020 203c 2f67      </g>.    </g
 000063a0: 3e0a 2020 2020 3c67 2069 643d 2278 7469  >.    <g id="xti
 000063b0: 636b 5f32 3522 3e0a 2020 2020 203c 6720  ck_25">.     <g 
 000063c0: 6964 3d22 6c69 6e65 3264 5f32 3522 3e0a  id="line2d_25">.
 000063d0: 2020 2020 2020 3c67 3e0a 2020 2020 2020        <g>.      
 000063e0: 203c 7573 6520 786c 696e 6b3a 6872 6566   <use xlink:href
-000063f0: 3d22 236d 3734 3137 3866 3636 3434 2220  ="#m74178f6644" 
+000063f0: 3d22 236d 6332 3738 3465 6337 3366 2220  ="#mc2784ec73f" 
 00006400: 783d 2234 3830 2e33 3737 3030 3422 2079  x="480.377004" y
 00006410: 3d22 3230 382e 3939 3837 3134 2220 7374  ="208.998714" st
 00006420: 796c 653d 2273 7472 6f6b 653a 2023 3030  yle="stroke: #00
 00006430: 3030 3030 3b20 7374 726f 6b65 2d77 6964  0000; stroke-wid
 00006440: 7468 3a20 302e 3822 2f3e 0a20 2020 2020  th: 0.8"/>.     
 00006450: 203c 2f67 3e0a 2020 2020 203c 2f67 3e0a   </g>.     </g>.
 00006460: 2020 2020 203c 6720 6964 3d22 7465 7874       <g id="text
@@ -1628,15 +1628,15 @@
 000065b0: 0a20 2020 2020 203c 2f67 3e0a 2020 2020  .      </g>.    
 000065c0: 203c 2f67 3e0a 2020 2020 3c2f 673e 0a20   </g>.    </g>. 
 000065d0: 2020 203c 6720 6964 3d22 7874 6963 6b5f     <g id="xtick_
 000065e0: 3236 223e 0a20 2020 2020 3c67 2069 643d  26">.     <g id=
 000065f0: 226c 696e 6532 645f 3236 223e 0a20 2020  "line2d_26">.   
 00006600: 2020 203c 673e 0a20 2020 2020 2020 3c75     <g>.       <u
 00006610: 7365 2078 6c69 6e6b 3a68 7265 663d 2223  se xlink:href="#
-00006620: 6d37 3431 3738 6636 3634 3422 2078 3d22  m74178f6644" x="
+00006620: 6d63 3237 3834 6563 3733 6622 2078 3d22  mc2784ec73f" x="
 00006630: 3439 382e 3136 3530 3435 2220 793d 2232  498.165045" y="2
 00006640: 3038 2e39 3938 3731 3422 2073 7479 6c65  08.998714" style
 00006650: 3d22 7374 726f 6b65 3a20 2330 3030 3030  ="stroke: #00000
 00006660: 303b 2073 7472 6f6b 652d 7769 6474 683a  0; stroke-width:
 00006670: 2030 2e38 222f 3e0a 2020 2020 2020 3c2f   0.8"/>.      </
 00006680: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 00006690: 2020 3c67 2069 643d 2274 6578 745f 3236    <g id="text_26
@@ -1662,16 +1662,16 @@
 000067d0: 3139 302e 3836 3931 3431 222f 3e0a 2020  190.869141"/>.  
 000067e0: 2020 2020 3c2f 673e 0a20 2020 2020 3c2f      </g>.     </
 000067f0: 673e 0a20 2020 203c 2f67 3e0a 2020 2020  g>.    </g>.    
 00006800: 3c67 2069 643d 2278 7469 636b 5f32 3722  <g id="xtick_27"
 00006810: 3e0a 2020 2020 203c 6720 6964 3d22 6c69  >.     <g id="li
 00006820: 6e65 3264 5f32 3722 3e0a 2020 2020 2020  ne2d_27">.      
 00006830: 3c67 3e0a 2020 2020 2020 203c 7573 6520  <g>.       <use 
-00006840: 786c 696e 6b3a 6872 6566 3d22 236d 3734  xlink:href="#m74
-00006850: 3137 3866 3636 3434 2220 783d 2235 3135  178f6644" x="515
+00006840: 786c 696e 6b3a 6872 6566 3d22 236d 6332  xlink:href="#mc2
+00006850: 3738 3465 6337 3366 2220 783d 2235 3135  784ec73f" x="515
 00006860: 2e39 3533 3038 3622 2079 3d22 3230 382e  .953086" y="208.
 00006870: 3939 3837 3134 2220 7374 796c 653d 2273  998714" style="s
 00006880: 7472 6f6b 653a 2023 3030 3030 3030 3b20  troke: #000000; 
 00006890: 7374 726f 6b65 2d77 6964 7468 3a20 302e  stroke-width: 0.
 000068a0: 3822 2f3e 0a20 2020 2020 203c 2f67 3e0a  8"/>.      </g>.
 000068b0: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 000068c0: 6720 6964 3d22 7465 7874 5f32 3722 3e0a  g id="text_27">.
@@ -1697,16 +1697,16 @@
 00006a00: 2e38 3639 3134 3122 2f3e 0a20 2020 2020  .869141"/>.     
 00006a10: 203c 2f67 3e0a 2020 2020 203c 2f67 3e0a   </g>.     </g>.
 00006a20: 2020 2020 3c2f 673e 0a20 2020 203c 6720      </g>.    <g 
 00006a30: 6964 3d22 7874 6963 6b5f 3238 223e 0a20  id="xtick_28">. 
 00006a40: 2020 2020 3c67 2069 643d 226c 696e 6532      <g id="line2
 00006a50: 645f 3238 223e 0a20 2020 2020 203c 673e  d_28">.      <g>
 00006a60: 0a20 2020 2020 2020 3c75 7365 2078 6c69  .       <use xli
-00006a70: 6e6b 3a68 7265 663d 2223 6d37 3431 3738  nk:href="#m74178
-00006a80: 6636 3634 3422 2078 3d22 3533 332e 3734  f6644" x="533.74
+00006a70: 6e6b 3a68 7265 663d 2223 6d63 3237 3834  nk:href="#mc2784
+00006a80: 6563 3733 6622 2078 3d22 3533 332e 3734  ec73f" x="533.74
 00006a90: 3131 3237 2220 793d 2232 3038 2e39 3938  1127" y="208.998
 00006aa0: 3731 3422 2073 7479 6c65 3d22 7374 726f  714" style="stro
 00006ab0: 6b65 3a20 2330 3030 3030 303b 2073 7472  ke: #000000; str
 00006ac0: 6f6b 652d 7769 6474 683a 2030 2e38 222f  oke-width: 0.8"/
 00006ad0: 3e0a 2020 2020 2020 3c2f 673e 0a20 2020  >.      </g>.   
 00006ae0: 2020 3c2f 673e 0a20 2020 2020 3c67 2069    </g>.     <g i
 00006af0: 643d 2274 6578 745f 3238 223e 0a20 2020  d="text_28">.   
@@ -1732,16 +1732,16 @@
 00006c30: 3931 3431 222f 3e0a 2020 2020 2020 3c2f  9141"/>.      </
 00006c40: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 00006c50: 203c 2f67 3e0a 2020 2020 3c67 2069 643d   </g>.    <g id=
 00006c60: 2278 7469 636b 5f32 3922 3e0a 2020 2020  "xtick_29">.    
 00006c70: 203c 6720 6964 3d22 6c69 6e65 3264 5f32   <g id="line2d_2
 00006c80: 3922 3e0a 2020 2020 2020 3c67 3e0a 2020  9">.      <g>.  
 00006c90: 2020 2020 203c 7573 6520 786c 696e 6b3a       <use xlink:
-00006ca0: 6872 6566 3d22 236d 3734 3137 3866 3636  href="#m74178f66
-00006cb0: 3434 2220 783d 2235 3531 2e35 3239 3136  44" x="551.52916
+00006ca0: 6872 6566 3d22 236d 6332 3738 3465 6337  href="#mc2784ec7
+00006cb0: 3366 2220 783d 2235 3531 2e35 3239 3136  3f" x="551.52916
 00006cc0: 3822 2079 3d22 3230 382e 3939 3837 3134  8" y="208.998714
 00006cd0: 2220 7374 796c 653d 2273 7472 6f6b 653a  " style="stroke:
 00006ce0: 2023 3030 3030 3030 3b20 7374 726f 6b65   #000000; stroke
 00006cf0: 2d77 6964 7468 3a20 302e 3822 2f3e 0a20  -width: 0.8"/>. 
 00006d00: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00006d10: 2f67 3e0a 2020 2020 203c 6720 6964 3d22  /g>.     <g id="
 00006d20: 7465 7874 5f32 3922 3e0a 2020 2020 2020  text_29">.      
@@ -1769,23 +1769,23 @@
 00006e80: 673e 0a20 2020 3c2f 673e 0a20 2020 3c67  g>.   </g>.   <g
 00006e90: 2069 643d 226d 6174 706c 6f74 6c69 622e   id="matplotlib.
 00006ea0: 6178 6973 5f32 223e 0a20 2020 203c 6720  axis_2">.    <g 
 00006eb0: 6964 3d22 7974 6963 6b5f 3122 3e0a 2020  id="ytick_1">.  
 00006ec0: 2020 203c 6720 6964 3d22 6c69 6e65 3264     <g id="line2d
 00006ed0: 5f33 3022 3e0a 2020 2020 2020 3c64 6566  _30">.      <def
 00006ee0: 733e 0a20 2020 2020 2020 3c70 6174 6820  s>.       <path 
-00006ef0: 6964 3d22 6d64 6433 6337 3666 6437 3622  id="mdd3c76fd76"
+00006ef0: 6964 3d22 6d31 6532 3435 6333 3537 6522  id="m1e245c357e"
 00006f00: 2064 3d22 4d20 3020 3020 0a4c 202d 332e   d="M 0 0 .L -3.
 00006f10: 3520 3020 0a22 2073 7479 6c65 3d22 7374  5 0 ." style="st
 00006f20: 726f 6b65 3a20 2330 3030 3030 303b 2073  roke: #000000; s
 00006f30: 7472 6f6b 652d 7769 6474 683a 2030 2e38  troke-width: 0.8
 00006f40: 222f 3e0a 2020 2020 2020 3c2f 6465 6673  "/>.      </defs
 00006f50: 3e0a 2020 2020 2020 3c67 3e0a 2020 2020  >.      <g>.    
 00006f60: 2020 203c 7573 6520 786c 696e 6b3a 6872     <use xlink:hr
-00006f70: 6566 3d22 236d 6464 3363 3736 6664 3736  ef="#mdd3c76fd76
+00006f70: 6566 3d22 236d 3165 3234 3563 3335 3765  ef="#m1e245c357e
 00006f80: 2220 783d 2234 342e 3537 2220 793d 2232  " x="44.57" y="2
 00006f90: 3038 2e39 3938 3731 3422 2073 7479 6c65  08.998714" style
 00006fa0: 3d22 7374 726f 6b65 3a20 2330 3030 3030  ="stroke: #00000
 00006fb0: 303b 2073 7472 6f6b 652d 7769 6474 683a  0; stroke-width:
 00006fc0: 2030 2e38 222f 3e0a 2020 2020 2020 3c2f   0.8"/>.      </
 00006fd0: 673e 0a20 2020 2020 3c2f 673e 0a20 2020  g>.     </g>.   
 00006fe0: 2020 3c67 2069 643d 2274 6578 745f 3330    <g id="text_30
@@ -1800,15 +1800,15 @@
 00007070: 3e0a 2020 2020 2020 3c2f 673e 0a20 2020  >.      </g>.   
 00007080: 2020 3c2f 673e 0a20 2020 203c 2f67 3e0a    </g>.    </g>.
 00007090: 2020 2020 3c67 2069 643d 2279 7469 636b      <g id="ytick
 000070a0: 5f32 223e 0a20 2020 2020 3c67 2069 643d  _2">.     <g id=
 000070b0: 226c 696e 6532 645f 3331 223e 0a20 2020  "line2d_31">.   
 000070c0: 2020 203c 673e 0a20 2020 2020 2020 3c75     <g>.       <u
 000070d0: 7365 2078 6c69 6e6b 3a68 7265 663d 2223  se xlink:href="#
-000070e0: 6d64 6433 6337 3666 6437 3622 2078 3d22  mdd3c76fd76" x="
+000070e0: 6d31 6532 3435 6333 3537 6522 2078 3d22  m1e245c357e" x="
 000070f0: 3434 2e35 3722 2079 3d22 3138 332e 3439  44.57" y="183.49
 00007100: 3138 3932 2220 7374 796c 653d 2273 7472  1892" style="str
 00007110: 6f6b 653a 2023 3030 3030 3030 3b20 7374  oke: #000000; st
 00007120: 726f 6b65 2d77 6964 7468 3a20 302e 3822  roke-width: 0.8"
 00007130: 2f3e 0a20 2020 2020 203c 2f67 3e0a 2020  />.      </g>.  
 00007140: 2020 203c 2f67 3e0a 2020 2020 203c 6720     </g>.     <g 
 00007150: 6964 3d22 7465 7874 5f33 3122 3e0a 2020  id="text_31">.  
@@ -1822,16 +1822,16 @@
 000071d0: 7553 616e 732d 3335 222f 3e0a 2020 2020  uSans-35"/>.    
 000071e0: 2020 3c2f 673e 0a20 2020 2020 3c2f 673e    </g>.     </g>
 000071f0: 0a20 2020 203c 2f67 3e0a 2020 2020 3c67  .    </g>.    <g
 00007200: 2069 643d 2279 7469 636b 5f33 223e 0a20   id="ytick_3">. 
 00007210: 2020 2020 3c67 2069 643d 226c 696e 6532      <g id="line2
 00007220: 645f 3332 223e 0a20 2020 2020 203c 673e  d_32">.      <g>
 00007230: 0a20 2020 2020 2020 3c75 7365 2078 6c69  .       <use xli
-00007240: 6e6b 3a68 7265 663d 2223 6d64 6433 6337  nk:href="#mdd3c7
-00007250: 3666 6437 3622 2078 3d22 3434 2e35 3722  6fd76" x="44.57"
+00007240: 6e6b 3a68 7265 663d 2223 6d31 6532 3435  nk:href="#m1e245
+00007250: 6333 3537 6522 2078 3d22 3434 2e35 3722  c357e" x="44.57"
 00007260: 2079 3d22 3135 372e 3938 3530 3639 2220   y="157.985069" 
 00007270: 7374 796c 653d 2273 7472 6f6b 653a 2023  style="stroke: #
 00007280: 3030 3030 3030 3b20 7374 726f 6b65 2d77  000000; stroke-w
 00007290: 6964 7468 3a20 302e 3822 2f3e 0a20 2020  idth: 0.8"/>.   
 000072a0: 2020 203c 2f67 3e0a 2020 2020 203c 2f67     </g>.     </g
 000072b0: 3e0a 2020 2020 203c 6720 6964 3d22 7465  >.     <g id="te
 000072c0: 7874 5f33 3222 3e0a 2020 2020 2020 3c21  xt_32">.      <!
@@ -1848,16 +1848,16 @@
 00007370: 3d22 3633 2e36 3233 3034 3722 2f3e 0a20  ="63.623047"/>. 
 00007380: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 00007390: 2f67 3e0a 2020 2020 3c2f 673e 0a20 2020  /g>.    </g>.   
 000073a0: 203c 6720 6964 3d22 7974 6963 6b5f 3422   <g id="ytick_4"
 000073b0: 3e0a 2020 2020 203c 6720 6964 3d22 6c69  >.     <g id="li
 000073c0: 6e65 3264 5f33 3322 3e0a 2020 2020 2020  ne2d_33">.      
 000073d0: 3c67 3e0a 2020 2020 2020 203c 7573 6520  <g>.       <use 
-000073e0: 786c 696e 6b3a 6872 6566 3d22 236d 6464  xlink:href="#mdd
-000073f0: 3363 3736 6664 3736 2220 783d 2234 342e  3c76fd76" x="44.
+000073e0: 786c 696e 6b3a 6872 6566 3d22 236d 3165  xlink:href="#m1e
+000073f0: 3234 3563 3335 3765 2220 783d 2234 342e  245c357e" x="44.
 00007400: 3537 2220 793d 2231 3332 2e34 3738 3234  57" y="132.47824
 00007410: 3622 2073 7479 6c65 3d22 7374 726f 6b65  6" style="stroke
 00007420: 3a20 2330 3030 3030 303b 2073 7472 6f6b  : #000000; strok
 00007430: 652d 7769 6474 683a 2030 2e38 222f 3e0a  e-width: 0.8"/>.
 00007440: 2020 2020 2020 3c2f 673e 0a20 2020 2020        </g>.     
 00007450: 3c2f 673e 0a20 2020 2020 3c67 2069 643d  </g>.     <g id=
 00007460: 2274 6578 745f 3333 223e 0a20 2020 2020  "text_33">.     
@@ -1875,15 +1875,15 @@
 00007520: 3e0a 2020 2020 2020 3c2f 673e 0a20 2020  >.      </g>.   
 00007530: 2020 3c2f 673e 0a20 2020 203c 2f67 3e0a    </g>.    </g>.
 00007540: 2020 2020 3c67 2069 643d 2279 7469 636b      <g id="ytick
 00007550: 5f35 223e 0a20 2020 2020 3c67 2069 643d  _5">.     <g id=
 00007560: 226c 696e 6532 645f 3334 223e 0a20 2020  "line2d_34">.   
 00007570: 2020 203c 673e 0a20 2020 2020 2020 3c75     <g>.       <u
 00007580: 7365 2078 6c69 6e6b 3a68 7265 663d 2223  se xlink:href="#
-00007590: 6d64 6433 6337 3666 6437 3622 2078 3d22  mdd3c76fd76" x="
+00007590: 6d31 6532 3435 6333 3537 6522 2078 3d22  m1e245c357e" x="
 000075a0: 3434 2e35 3722 2079 3d22 3130 362e 3937  44.57" y="106.97
 000075b0: 3134 3233 2220 7374 796c 653d 2273 7472  1423" style="str
 000075c0: 6f6b 653a 2023 3030 3030 3030 3b20 7374  oke: #000000; st
 000075d0: 726f 6b65 2d77 6964 7468 3a20 302e 3822  roke-width: 0.8"
 000075e0: 2f3e 0a20 2020 2020 203c 2f67 3e0a 2020  />.      </g>.  
 000075f0: 2020 203c 2f67 3e0a 2020 2020 203c 6720     </g>.     <g 
 00007600: 6964 3d22 7465 7874 5f33 3422 3e0a 2020  id="text_34">.  
@@ -1901,15 +1901,15 @@
 000076c0: 3722 2f3e 0a20 2020 2020 203c 2f67 3e0a  7"/>.      </g>.
 000076d0: 2020 2020 203c 2f67 3e0a 2020 2020 3c2f       </g>.    </
 000076e0: 673e 0a20 2020 203c 6720 6964 3d22 7974  g>.    <g id="yt
 000076f0: 6963 6b5f 3622 3e0a 2020 2020 203c 6720  ick_6">.     <g 
 00007700: 6964 3d22 6c69 6e65 3264 5f33 3522 3e0a  id="line2d_35">.
 00007710: 2020 2020 2020 3c67 3e0a 2020 2020 2020        <g>.      
 00007720: 203c 7573 6520 786c 696e 6b3a 6872 6566   <use xlink:href
-00007730: 3d22 236d 6464 3363 3736 6664 3736 2220  ="#mdd3c76fd76" 
+00007730: 3d22 236d 3165 3234 3563 3335 3765 2220  ="#m1e245c357e" 
 00007740: 783d 2234 342e 3537 2220 793d 2238 312e  x="44.57" y="81.
 00007750: 3436 3436 3031 2220 7374 796c 653d 2273  464601" style="s
 00007760: 7472 6f6b 653a 2023 3030 3030 3030 3b20  troke: #000000; 
 00007770: 7374 726f 6b65 2d77 6964 7468 3a20 302e  stroke-width: 0.
 00007780: 3822 2f3e 0a20 2020 2020 203c 2f67 3e0a  8"/>.      </g>.
 00007790: 2020 2020 203c 2f67 3e0a 2020 2020 203c       </g>.     <
 000077a0: 6720 6964 3d22 7465 7874 5f33 3522 3e0a  g id="text_35">.
@@ -1927,15 +1927,15 @@
 00007860: 3437 222f 3e0a 2020 2020 2020 3c2f 673e  47"/>.      </g>
 00007870: 0a20 2020 2020 3c2f 673e 0a20 2020 203c  .     </g>.    <
 00007880: 2f67 3e0a 2020 2020 3c67 2069 643d 2279  /g>.    <g id="y
 00007890: 7469 636b 5f37 223e 0a20 2020 2020 3c67  tick_7">.     <g
 000078a0: 2069 643d 226c 696e 6532 645f 3336 223e   id="line2d_36">
 000078b0: 0a20 2020 2020 203c 673e 0a20 2020 2020  .      <g>.     
 000078c0: 2020 3c75 7365 2078 6c69 6e6b 3a68 7265    <use xlink:hre
-000078d0: 663d 2223 6d64 6433 6337 3666 6437 3622  f="#mdd3c76fd76"
+000078d0: 663d 2223 6d31 6532 3435 6333 3537 6522  f="#m1e245c357e"
 000078e0: 2078 3d22 3434 2e35 3722 2079 3d22 3535   x="44.57" y="55
 000078f0: 2e39 3537 3737 3822 2073 7479 6c65 3d22  .957778" style="
 00007900: 7374 726f 6b65 3a20 2330 3030 3030 303b  stroke: #000000;
 00007910: 2073 7472 6f6b 652d 7769 6474 683a 2030   stroke-width: 0
 00007920: 2e38 222f 3e0a 2020 2020 2020 3c2f 673e  .8"/>.      </g>
 00007930: 0a20 2020 2020 3c2f 673e 0a20 2020 2020  .     </g>.     
 00007940: 3c67 2069 643d 2274 6578 745f 3336 223e  <g id="text_36">
@@ -1953,15 +1953,15 @@
 00007a00: 3034 3722 2f3e 0a20 2020 2020 203c 2f67  047"/>.      </g
 00007a10: 3e0a 2020 2020 203c 2f67 3e0a 2020 2020  >.     </g>.    
 00007a20: 3c2f 673e 0a20 2020 203c 6720 6964 3d22  </g>.    <g id="
 00007a30: 7974 6963 6b5f 3822 3e0a 2020 2020 203c  ytick_8">.     <
 00007a40: 6720 6964 3d22 6c69 6e65 3264 5f33 3722  g id="line2d_37"
 00007a50: 3e0a 2020 2020 2020 3c67 3e0a 2020 2020  >.      <g>.    
 00007a60: 2020 203c 7573 6520 786c 696e 6b3a 6872     <use xlink:hr
-00007a70: 6566 3d22 236d 6464 3363 3736 6664 3736  ef="#mdd3c76fd76
+00007a70: 6566 3d22 236d 3165 3234 3563 3335 3765  ef="#m1e245c357e
 00007a80: 2220 783d 2234 342e 3537 2220 793d 2233  " x="44.57" y="3
 00007a90: 302e 3435 3039 3535 2220 7374 796c 653d  0.450955" style=
 00007aa0: 2273 7472 6f6b 653a 2023 3030 3030 3030  "stroke: #000000
 00007ab0: 3b20 7374 726f 6b65 2d77 6964 7468 3a20  ; stroke-width: 
 00007ac0: 302e 3822 2f3e 0a20 2020 2020 203c 2f67  0.8"/>.      </g
 00007ad0: 3e0a 2020 2020 203c 2f67 3e0a 2020 2020  >.     </g>.    
 00007ae0: 203c 6720 6964 3d22 7465 7874 5f33 3722   <g id="text_37"
@@ -2307,279 +2307,279 @@
 00009020: 6566 3d22 2344 656a 6156 7553 616e 732d  ef="#DejaVuSans-
 00009030: 3733 2220 783d 2235 3438 2e37 3837 3130  73" x="548.78710
 00009040: 3922 2f3e 0a20 2020 2020 3c2f 673e 0a20  9"/>.     </g>. 
 00009050: 2020 203c 2f67 3e0a 2020 203c 2f67 3e0a     </g>.   </g>.
 00009060: 2020 203c 6720 6964 3d22 6c69 6e65 3264     <g id="line2d
 00009070: 5f33 3822 3e0a 2020 2020 3c70 6174 6820  _38">.    <path 
 00009080: 636c 6970 2d70 6174 683d 2275 726c 2823  clip-path="url(#
-00009090: 7032 3535 3763 3132 6361 3629 2220 7374  p2557c12ca6)" st
+00009090: 7063 6430 3739 3063 6634 3829 2220 7374  pcd0790cf48)" st
 000090a0: 796c 653d 2266 696c 6c3a 206e 6f6e 653b  yle="fill: none;
 000090b0: 2073 7472 6f6b 653a 2023 3432 3432 3432   stroke: #424242
 000090c0: 3b20 7374 726f 6b65 2d77 6964 7468 3a20  ; stroke-width: 
 000090d0: 322e 373b 2073 7472 6f6b 652d 6c69 6e65  2.7; stroke-line
 000090e0: 6361 703a 2073 7175 6172 6522 2f3e 0a20  cap: square"/>. 
 000090f0: 2020 3c2f 673e 0a20 2020 3c67 2069 643d    </g>.   <g id=
 00009100: 226c 696e 6532 645f 3339 223e 0a20 2020  "line2d_39">.   
 00009110: 203c 7061 7468 2063 6c69 702d 7061 7468   <path clip-path
-00009120: 3d22 7572 6c28 2370 3235 3537 6331 3263  ="url(#p2557c12c
-00009130: 6136 2922 2073 7479 6c65 3d22 6669 6c6c  a6)" style="fill
+00009120: 3d22 7572 6c28 2370 6364 3037 3930 6366  ="url(#pcd0790cf
+00009130: 3438 2922 2073 7479 6c65 3d22 6669 6c6c  48)" style="fill
 00009140: 3a20 6e6f 6e65 3b20 7374 726f 6b65 3a20  : none; stroke: 
 00009150: 2334 3234 3234 323b 2073 7472 6f6b 652d  #424242; stroke-
 00009160: 7769 6474 683a 2032 2e37 3b20 7374 726f  width: 2.7; stro
 00009170: 6b65 2d6c 696e 6563 6170 3a20 7371 7561  ke-linecap: squa
 00009180: 7265 222f 3e0a 2020 203c 2f67 3e0a 2020  re"/>.   </g>.  
 00009190: 203c 6720 6964 3d22 6c69 6e65 3264 5f34   <g id="line2d_4
 000091a0: 3022 3e0a 2020 2020 3c70 6174 6820 636c  0">.    <path cl
-000091b0: 6970 2d70 6174 683d 2275 726c 2823 7032  ip-path="url(#p2
-000091c0: 3535 3763 3132 6361 3629 2220 7374 796c  557c12ca6)" styl
+000091b0: 6970 2d70 6174 683d 2275 726c 2823 7063  ip-path="url(#pc
+000091c0: 6430 3739 3063 6634 3829 2220 7374 796c  d0790cf48)" styl
 000091d0: 653d 2266 696c 6c3a 206e 6f6e 653b 2073  e="fill: none; s
 000091e0: 7472 6f6b 653a 2023 3432 3432 3432 3b20  troke: #424242; 
 000091f0: 7374 726f 6b65 2d77 6964 7468 3a20 322e  stroke-width: 2.
 00009200: 373b 2073 7472 6f6b 652d 6c69 6e65 6361  7; stroke-lineca
 00009210: 703a 2073 7175 6172 6522 2f3e 0a20 2020  p: square"/>.   
 00009220: 3c2f 673e 0a20 2020 3c67 2069 643d 226c  </g>.   <g id="l
 00009230: 696e 6532 645f 3431 223e 0a20 2020 203c  ine2d_41">.    <
 00009240: 7061 7468 2063 6c69 702d 7061 7468 3d22  path clip-path="
-00009250: 7572 6c28 2370 3235 3537 6331 3263 6136  url(#p2557c12ca6
+00009250: 7572 6c28 2370 6364 3037 3930 6366 3438  url(#pcd0790cf48
 00009260: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00009270: 6e6f 6e65 3b20 7374 726f 6b65 3a20 2334  none; stroke: #4
 00009280: 3234 3234 323b 2073 7472 6f6b 652d 7769  24242; stroke-wi
 00009290: 6474 683a 2032 2e37 3b20 7374 726f 6b65  dth: 2.7; stroke
 000092a0: 2d6c 696e 6563 6170 3a20 7371 7561 7265  -linecap: square
 000092b0: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 000092c0: 6720 6964 3d22 6c69 6e65 3264 5f34 3222  g id="line2d_42"
 000092d0: 3e0a 2020 2020 3c70 6174 6820 636c 6970  >.    <path clip
-000092e0: 2d70 6174 683d 2275 726c 2823 7032 3535  -path="url(#p255
-000092f0: 3763 3132 6361 3629 2220 7374 796c 653d  7c12ca6)" style=
+000092e0: 2d70 6174 683d 2275 726c 2823 7063 6430  -path="url(#pcd0
+000092f0: 3739 3063 6634 3829 2220 7374 796c 653d  790cf48)" style=
 00009300: 2266 696c 6c3a 206e 6f6e 653b 2073 7472  "fill: none; str
 00009310: 6f6b 653a 2023 3432 3432 3432 3b20 7374  oke: #424242; st
 00009320: 726f 6b65 2d77 6964 7468 3a20 322e 373b  roke-width: 2.7;
 00009330: 2073 7472 6f6b 652d 6c69 6e65 6361 703a   stroke-linecap:
 00009340: 2073 7175 6172 6522 2f3e 0a20 2020 3c2f   square"/>.   </
 00009350: 673e 0a20 2020 3c67 2069 643d 226c 696e  g>.   <g id="lin
 00009360: 6532 645f 3433 223e 0a20 2020 203c 7061  e2d_43">.    <pa
 00009370: 7468 2063 6c69 702d 7061 7468 3d22 7572  th clip-path="ur
-00009380: 6c28 2370 3235 3537 6331 3263 6136 2922  l(#p2557c12ca6)"
+00009380: 6c28 2370 6364 3037 3930 6366 3438 2922  l(#pcd0790cf48)"
 00009390: 2073 7479 6c65 3d22 6669 6c6c 3a20 6e6f   style="fill: no
 000093a0: 6e65 3b20 7374 726f 6b65 3a20 2334 3234  ne; stroke: #424
 000093b0: 3234 323b 2073 7472 6f6b 652d 7769 6474  242; stroke-widt
 000093c0: 683a 2032 2e37 3b20 7374 726f 6b65 2d6c  h: 2.7; stroke-l
 000093d0: 696e 6563 6170 3a20 7371 7561 7265 222f  inecap: square"/
 000093e0: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 000093f0: 6964 3d22 6c69 6e65 3264 5f34 3422 3e0a  id="line2d_44">.
 00009400: 2020 2020 3c70 6174 6820 636c 6970 2d70      <path clip-p
-00009410: 6174 683d 2275 726c 2823 7032 3535 3763  ath="url(#p2557c
-00009420: 3132 6361 3629 2220 7374 796c 653d 2266  12ca6)" style="f
+00009410: 6174 683d 2275 726c 2823 7063 6430 3739  ath="url(#pcd079
+00009420: 3063 6634 3829 2220 7374 796c 653d 2266  0cf48)" style="f
 00009430: 696c 6c3a 206e 6f6e 653b 2073 7472 6f6b  ill: none; strok
 00009440: 653a 2023 3432 3432 3432 3b20 7374 726f  e: #424242; stro
 00009450: 6b65 2d77 6964 7468 3a20 322e 373b 2073  ke-width: 2.7; s
 00009460: 7472 6f6b 652d 6c69 6e65 6361 703a 2073  troke-linecap: s
 00009470: 7175 6172 6522 2f3e 0a20 2020 3c2f 673e  quare"/>.   </g>
 00009480: 0a20 2020 3c67 2069 643d 226c 696e 6532  .   <g id="line2
 00009490: 645f 3435 223e 0a20 2020 203c 7061 7468  d_45">.    <path
 000094a0: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-000094b0: 2370 3235 3537 6331 3263 6136 2922 2073  #p2557c12ca6)" s
+000094b0: 2370 6364 3037 3930 6366 3438 2922 2073  #pcd0790cf48)" s
 000094c0: 7479 6c65 3d22 6669 6c6c 3a20 6e6f 6e65  tyle="fill: none
 000094d0: 3b20 7374 726f 6b65 3a20 2334 3234 3234  ; stroke: #42424
 000094e0: 323b 2073 7472 6f6b 652d 7769 6474 683a  2; stroke-width:
 000094f0: 2032 2e37 3b20 7374 726f 6b65 2d6c 696e   2.7; stroke-lin
 00009500: 6563 6170 3a20 7371 7561 7265 222f 3e0a  ecap: square"/>.
 00009510: 2020 203c 2f67 3e0a 2020 203c 6720 6964     </g>.   <g id
 00009520: 3d22 6c69 6e65 3264 5f34 3622 3e0a 2020  ="line2d_46">.  
 00009530: 2020 3c70 6174 6820 636c 6970 2d70 6174    <path clip-pat
-00009540: 683d 2275 726c 2823 7032 3535 3763 3132  h="url(#p2557c12
-00009550: 6361 3629 2220 7374 796c 653d 2266 696c  ca6)" style="fil
+00009540: 683d 2275 726c 2823 7063 6430 3739 3063  h="url(#pcd0790c
+00009550: 6634 3829 2220 7374 796c 653d 2266 696c  f48)" style="fil
 00009560: 6c3a 206e 6f6e 653b 2073 7472 6f6b 653a  l: none; stroke:
 00009570: 2023 3432 3432 3432 3b20 7374 726f 6b65   #424242; stroke
 00009580: 2d77 6964 7468 3a20 322e 373b 2073 7472  -width: 2.7; str
 00009590: 6f6b 652d 6c69 6e65 6361 703a 2073 7175  oke-linecap: squ
 000095a0: 6172 6522 2f3e 0a20 2020 3c2f 673e 0a20  are"/>.   </g>. 
 000095b0: 2020 3c67 2069 643d 226c 696e 6532 645f    <g id="line2d_
 000095c0: 3437 223e 0a20 2020 203c 7061 7468 2063  47">.    <path c
 000095d0: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-000095e0: 3235 3537 6331 3263 6136 2922 2073 7479  2557c12ca6)" sty
+000095e0: 6364 3037 3930 6366 3438 2922 2073 7479  cd0790cf48)" sty
 000095f0: 6c65 3d22 6669 6c6c 3a20 6e6f 6e65 3b20  le="fill: none; 
 00009600: 7374 726f 6b65 3a20 2334 3234 3234 323b  stroke: #424242;
 00009610: 2073 7472 6f6b 652d 7769 6474 683a 2032   stroke-width: 2
 00009620: 2e37 3b20 7374 726f 6b65 2d6c 696e 6563  .7; stroke-linec
 00009630: 6170 3a20 7371 7561 7265 222f 3e0a 2020  ap: square"/>.  
 00009640: 203c 2f67 3e0a 2020 203c 6720 6964 3d22   </g>.   <g id="
 00009650: 6c69 6e65 3264 5f34 3822 3e0a 2020 2020  line2d_48">.    
 00009660: 3c70 6174 6820 636c 6970 2d70 6174 683d  <path clip-path=
-00009670: 2275 726c 2823 7032 3535 3763 3132 6361  "url(#p2557c12ca
-00009680: 3629 2220 7374 796c 653d 2266 696c 6c3a  6)" style="fill:
+00009670: 2275 726c 2823 7063 6430 3739 3063 6634  "url(#pcd0790cf4
+00009680: 3829 2220 7374 796c 653d 2266 696c 6c3a  8)" style="fill:
 00009690: 206e 6f6e 653b 2073 7472 6f6b 653a 2023   none; stroke: #
 000096a0: 3432 3432 3432 3b20 7374 726f 6b65 2d77  424242; stroke-w
 000096b0: 6964 7468 3a20 322e 373b 2073 7472 6f6b  idth: 2.7; strok
 000096c0: 652d 6c69 6e65 6361 703a 2073 7175 6172  e-linecap: squar
 000096d0: 6522 2f3e 0a20 2020 3c2f 673e 0a20 2020  e"/>.   </g>.   
 000096e0: 3c67 2069 643d 226c 696e 6532 645f 3439  <g id="line2d_49
 000096f0: 223e 0a20 2020 203c 7061 7468 2063 6c69  ">.    <path cli
-00009700: 702d 7061 7468 3d22 7572 6c28 2370 3235  p-path="url(#p25
-00009710: 3537 6331 3263 6136 2922 2073 7479 6c65  57c12ca6)" style
+00009700: 702d 7061 7468 3d22 7572 6c28 2370 6364  p-path="url(#pcd
+00009710: 3037 3930 6366 3438 2922 2073 7479 6c65  0790cf48)" style
 00009720: 3d22 6669 6c6c 3a20 6e6f 6e65 3b20 7374  ="fill: none; st
 00009730: 726f 6b65 3a20 2334 3234 3234 323b 2073  roke: #424242; s
 00009740: 7472 6f6b 652d 7769 6474 683a 2032 2e37  troke-width: 2.7
 00009750: 3b20 7374 726f 6b65 2d6c 696e 6563 6170  ; stroke-linecap
 00009760: 3a20 7371 7561 7265 222f 3e0a 2020 203c  : square"/>.   <
 00009770: 2f67 3e0a 2020 203c 6720 6964 3d22 6c69  /g>.   <g id="li
 00009780: 6e65 3264 5f35 3022 3e0a 2020 2020 3c70  ne2d_50">.    <p
 00009790: 6174 6820 636c 6970 2d70 6174 683d 2275  ath clip-path="u
-000097a0: 726c 2823 7032 3535 3763 3132 6361 3629  rl(#p2557c12ca6)
+000097a0: 726c 2823 7063 6430 3739 3063 6634 3829  rl(#pcd0790cf48)
 000097b0: 2220 7374 796c 653d 2266 696c 6c3a 206e  " style="fill: n
 000097c0: 6f6e 653b 2073 7472 6f6b 653a 2023 3432  one; stroke: #42
 000097d0: 3432 3432 3b20 7374 726f 6b65 2d77 6964  4242; stroke-wid
 000097e0: 7468 3a20 322e 373b 2073 7472 6f6b 652d  th: 2.7; stroke-
 000097f0: 6c69 6e65 6361 703a 2073 7175 6172 6522  linecap: square"
 00009800: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 00009810: 2069 643d 226c 696e 6532 645f 3531 223e   id="line2d_51">
 00009820: 0a20 2020 203c 7061 7468 2063 6c69 702d  .    <path clip-
-00009830: 7061 7468 3d22 7572 6c28 2370 3235 3537  path="url(#p2557
-00009840: 6331 3263 6136 2922 2073 7479 6c65 3d22  c12ca6)" style="
+00009830: 7061 7468 3d22 7572 6c28 2370 6364 3037  path="url(#pcd07
+00009840: 3930 6366 3438 2922 2073 7479 6c65 3d22  90cf48)" style="
 00009850: 6669 6c6c 3a20 6e6f 6e65 3b20 7374 726f  fill: none; stro
 00009860: 6b65 3a20 2334 3234 3234 323b 2073 7472  ke: #424242; str
 00009870: 6f6b 652d 7769 6474 683a 2032 2e37 3b20  oke-width: 2.7; 
 00009880: 7374 726f 6b65 2d6c 696e 6563 6170 3a20  stroke-linecap: 
 00009890: 7371 7561 7265 222f 3e0a 2020 203c 2f67  square"/>.   </g
 000098a0: 3e0a 2020 203c 6720 6964 3d22 6c69 6e65  >.   <g id="line
 000098b0: 3264 5f35 3222 3e0a 2020 2020 3c70 6174  2d_52">.    <pat
 000098c0: 6820 636c 6970 2d70 6174 683d 2275 726c  h clip-path="url
-000098d0: 2823 7032 3535 3763 3132 6361 3629 2220  (#p2557c12ca6)" 
+000098d0: 2823 7063 6430 3739 3063 6634 3829 2220  (#pcd0790cf48)" 
 000098e0: 7374 796c 653d 2266 696c 6c3a 206e 6f6e  style="fill: non
 000098f0: 653b 2073 7472 6f6b 653a 2023 3432 3432  e; stroke: #4242
 00009900: 3432 3b20 7374 726f 6b65 2d77 6964 7468  42; stroke-width
 00009910: 3a20 322e 373b 2073 7472 6f6b 652d 6c69  : 2.7; stroke-li
 00009920: 6e65 6361 703a 2073 7175 6172 6522 2f3e  necap: square"/>
 00009930: 0a20 2020 3c2f 673e 0a20 2020 3c67 2069  .   </g>.   <g i
 00009940: 643d 226c 696e 6532 645f 3533 223e 0a20  d="line2d_53">. 
 00009950: 2020 203c 7061 7468 2063 6c69 702d 7061     <path clip-pa
-00009960: 7468 3d22 7572 6c28 2370 3235 3537 6331  th="url(#p2557c1
-00009970: 3263 6136 2922 2073 7479 6c65 3d22 6669  2ca6)" style="fi
+00009960: 7468 3d22 7572 6c28 2370 6364 3037 3930  th="url(#pcd0790
+00009970: 6366 3438 2922 2073 7479 6c65 3d22 6669  cf48)" style="fi
 00009980: 6c6c 3a20 6e6f 6e65 3b20 7374 726f 6b65  ll: none; stroke
 00009990: 3a20 2334 3234 3234 323b 2073 7472 6f6b  : #424242; strok
 000099a0: 652d 7769 6474 683a 2032 2e37 3b20 7374  e-width: 2.7; st
 000099b0: 726f 6b65 2d6c 696e 6563 6170 3a20 7371  roke-linecap: sq
 000099c0: 7561 7265 222f 3e0a 2020 203c 2f67 3e0a  uare"/>.   </g>.
 000099d0: 2020 203c 6720 6964 3d22 6c69 6e65 3264     <g id="line2d
 000099e0: 5f35 3422 3e0a 2020 2020 3c70 6174 6820  _54">.    <path 
 000099f0: 636c 6970 2d70 6174 683d 2275 726c 2823  clip-path="url(#
-00009a00: 7032 3535 3763 3132 6361 3629 2220 7374  p2557c12ca6)" st
+00009a00: 7063 6430 3739 3063 6634 3829 2220 7374  pcd0790cf48)" st
 00009a10: 796c 653d 2266 696c 6c3a 206e 6f6e 653b  yle="fill: none;
 00009a20: 2073 7472 6f6b 653a 2023 3432 3432 3432   stroke: #424242
 00009a30: 3b20 7374 726f 6b65 2d77 6964 7468 3a20  ; stroke-width: 
 00009a40: 322e 373b 2073 7472 6f6b 652d 6c69 6e65  2.7; stroke-line
 00009a50: 6361 703a 2073 7175 6172 6522 2f3e 0a20  cap: square"/>. 
 00009a60: 2020 3c2f 673e 0a20 2020 3c67 2069 643d    </g>.   <g id=
 00009a70: 226c 696e 6532 645f 3535 223e 0a20 2020  "line2d_55">.   
 00009a80: 203c 7061 7468 2063 6c69 702d 7061 7468   <path clip-path
-00009a90: 3d22 7572 6c28 2370 3235 3537 6331 3263  ="url(#p2557c12c
-00009aa0: 6136 2922 2073 7479 6c65 3d22 6669 6c6c  a6)" style="fill
+00009a90: 3d22 7572 6c28 2370 6364 3037 3930 6366  ="url(#pcd0790cf
+00009aa0: 3438 2922 2073 7479 6c65 3d22 6669 6c6c  48)" style="fill
 00009ab0: 3a20 6e6f 6e65 3b20 7374 726f 6b65 3a20  : none; stroke: 
 00009ac0: 2334 3234 3234 323b 2073 7472 6f6b 652d  #424242; stroke-
 00009ad0: 7769 6474 683a 2032 2e37 3b20 7374 726f  width: 2.7; stro
 00009ae0: 6b65 2d6c 696e 6563 6170 3a20 7371 7561  ke-linecap: squa
 00009af0: 7265 222f 3e0a 2020 203c 2f67 3e0a 2020  re"/>.   </g>.  
 00009b00: 203c 6720 6964 3d22 6c69 6e65 3264 5f35   <g id="line2d_5
 00009b10: 3622 3e0a 2020 2020 3c70 6174 6820 636c  6">.    <path cl
-00009b20: 6970 2d70 6174 683d 2275 726c 2823 7032  ip-path="url(#p2
-00009b30: 3535 3763 3132 6361 3629 2220 7374 796c  557c12ca6)" styl
+00009b20: 6970 2d70 6174 683d 2275 726c 2823 7063  ip-path="url(#pc
+00009b30: 6430 3739 3063 6634 3829 2220 7374 796c  d0790cf48)" styl
 00009b40: 653d 2266 696c 6c3a 206e 6f6e 653b 2073  e="fill: none; s
 00009b50: 7472 6f6b 653a 2023 3432 3432 3432 3b20  troke: #424242; 
 00009b60: 7374 726f 6b65 2d77 6964 7468 3a20 322e  stroke-width: 2.
 00009b70: 373b 2073 7472 6f6b 652d 6c69 6e65 6361  7; stroke-lineca
 00009b80: 703a 2073 7175 6172 6522 2f3e 0a20 2020  p: square"/>.   
 00009b90: 3c2f 673e 0a20 2020 3c67 2069 643d 226c  </g>.   <g id="l
 00009ba0: 696e 6532 645f 3537 223e 0a20 2020 203c  ine2d_57">.    <
 00009bb0: 7061 7468 2063 6c69 702d 7061 7468 3d22  path clip-path="
-00009bc0: 7572 6c28 2370 3235 3537 6331 3263 6136  url(#p2557c12ca6
+00009bc0: 7572 6c28 2370 6364 3037 3930 6366 3438  url(#pcd0790cf48
 00009bd0: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00009be0: 6e6f 6e65 3b20 7374 726f 6b65 3a20 2334  none; stroke: #4
 00009bf0: 3234 3234 323b 2073 7472 6f6b 652d 7769  24242; stroke-wi
 00009c00: 6474 683a 2032 2e37 3b20 7374 726f 6b65  dth: 2.7; stroke
 00009c10: 2d6c 696e 6563 6170 3a20 7371 7561 7265  -linecap: square
 00009c20: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 00009c30: 6720 6964 3d22 6c69 6e65 3264 5f35 3822  g id="line2d_58"
 00009c40: 3e0a 2020 2020 3c70 6174 6820 636c 6970  >.    <path clip
-00009c50: 2d70 6174 683d 2275 726c 2823 7032 3535  -path="url(#p255
-00009c60: 3763 3132 6361 3629 2220 7374 796c 653d  7c12ca6)" style=
+00009c50: 2d70 6174 683d 2275 726c 2823 7063 6430  -path="url(#pcd0
+00009c60: 3739 3063 6634 3829 2220 7374 796c 653d  790cf48)" style=
 00009c70: 2266 696c 6c3a 206e 6f6e 653b 2073 7472  "fill: none; str
 00009c80: 6f6b 653a 2023 3432 3432 3432 3b20 7374  oke: #424242; st
 00009c90: 726f 6b65 2d77 6964 7468 3a20 322e 373b  roke-width: 2.7;
 00009ca0: 2073 7472 6f6b 652d 6c69 6e65 6361 703a   stroke-linecap:
 00009cb0: 2073 7175 6172 6522 2f3e 0a20 2020 3c2f   square"/>.   </
 00009cc0: 673e 0a20 2020 3c67 2069 643d 226c 696e  g>.   <g id="lin
 00009cd0: 6532 645f 3539 223e 0a20 2020 203c 7061  e2d_59">.    <pa
 00009ce0: 7468 2063 6c69 702d 7061 7468 3d22 7572  th clip-path="ur
-00009cf0: 6c28 2370 3235 3537 6331 3263 6136 2922  l(#p2557c12ca6)"
+00009cf0: 6c28 2370 6364 3037 3930 6366 3438 2922  l(#pcd0790cf48)"
 00009d00: 2073 7479 6c65 3d22 6669 6c6c 3a20 6e6f   style="fill: no
 00009d10: 6e65 3b20 7374 726f 6b65 3a20 2334 3234  ne; stroke: #424
 00009d20: 3234 323b 2073 7472 6f6b 652d 7769 6474  242; stroke-widt
 00009d30: 683a 2032 2e37 3b20 7374 726f 6b65 2d6c  h: 2.7; stroke-l
 00009d40: 696e 6563 6170 3a20 7371 7561 7265 222f  inecap: square"/
 00009d50: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 00009d60: 6964 3d22 6c69 6e65 3264 5f36 3022 3e0a  id="line2d_60">.
 00009d70: 2020 2020 3c70 6174 6820 636c 6970 2d70      <path clip-p
-00009d80: 6174 683d 2275 726c 2823 7032 3535 3763  ath="url(#p2557c
-00009d90: 3132 6361 3629 2220 7374 796c 653d 2266  12ca6)" style="f
+00009d80: 6174 683d 2275 726c 2823 7063 6430 3739  ath="url(#pcd079
+00009d90: 3063 6634 3829 2220 7374 796c 653d 2266  0cf48)" style="f
 00009da0: 696c 6c3a 206e 6f6e 653b 2073 7472 6f6b  ill: none; strok
 00009db0: 653a 2023 3432 3432 3432 3b20 7374 726f  e: #424242; stro
 00009dc0: 6b65 2d77 6964 7468 3a20 322e 373b 2073  ke-width: 2.7; s
 00009dd0: 7472 6f6b 652d 6c69 6e65 6361 703a 2073  troke-linecap: s
 00009de0: 7175 6172 6522 2f3e 0a20 2020 3c2f 673e  quare"/>.   </g>
 00009df0: 0a20 2020 3c67 2069 643d 226c 696e 6532  .   <g id="line2
 00009e00: 645f 3631 223e 0a20 2020 203c 7061 7468  d_61">.    <path
 00009e10: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00009e20: 2370 3235 3537 6331 3263 6136 2922 2073  #p2557c12ca6)" s
+00009e20: 2370 6364 3037 3930 6366 3438 2922 2073  #pcd0790cf48)" s
 00009e30: 7479 6c65 3d22 6669 6c6c 3a20 6e6f 6e65  tyle="fill: none
 00009e40: 3b20 7374 726f 6b65 3a20 2334 3234 3234  ; stroke: #42424
 00009e50: 323b 2073 7472 6f6b 652d 7769 6474 683a  2; stroke-width:
 00009e60: 2032 2e37 3b20 7374 726f 6b65 2d6c 696e   2.7; stroke-lin
 00009e70: 6563 6170 3a20 7371 7561 7265 222f 3e0a  ecap: square"/>.
 00009e80: 2020 203c 2f67 3e0a 2020 203c 6720 6964     </g>.   <g id
 00009e90: 3d22 6c69 6e65 3264 5f36 3222 3e0a 2020  ="line2d_62">.  
 00009ea0: 2020 3c70 6174 6820 636c 6970 2d70 6174    <path clip-pat
-00009eb0: 683d 2275 726c 2823 7032 3535 3763 3132  h="url(#p2557c12
-00009ec0: 6361 3629 2220 7374 796c 653d 2266 696c  ca6)" style="fil
+00009eb0: 683d 2275 726c 2823 7063 6430 3739 3063  h="url(#pcd0790c
+00009ec0: 6634 3829 2220 7374 796c 653d 2266 696c  f48)" style="fil
 00009ed0: 6c3a 206e 6f6e 653b 2073 7472 6f6b 653a  l: none; stroke:
 00009ee0: 2023 3432 3432 3432 3b20 7374 726f 6b65   #424242; stroke
 00009ef0: 2d77 6964 7468 3a20 322e 373b 2073 7472  -width: 2.7; str
 00009f00: 6f6b 652d 6c69 6e65 6361 703a 2073 7175  oke-linecap: squ
 00009f10: 6172 6522 2f3e 0a20 2020 3c2f 673e 0a20  are"/>.   </g>. 
 00009f20: 2020 3c67 2069 643d 226c 696e 6532 645f    <g id="line2d_
 00009f30: 3633 223e 0a20 2020 203c 7061 7468 2063  63">.    <path c
 00009f40: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-00009f50: 3235 3537 6331 3263 6136 2922 2073 7479  2557c12ca6)" sty
+00009f50: 6364 3037 3930 6366 3438 2922 2073 7479  cd0790cf48)" sty
 00009f60: 6c65 3d22 6669 6c6c 3a20 6e6f 6e65 3b20  le="fill: none; 
 00009f70: 7374 726f 6b65 3a20 2334 3234 3234 323b  stroke: #424242;
 00009f80: 2073 7472 6f6b 652d 7769 6474 683a 2032   stroke-width: 2
 00009f90: 2e37 3b20 7374 726f 6b65 2d6c 696e 6563  .7; stroke-linec
 00009fa0: 6170 3a20 7371 7561 7265 222f 3e0a 2020  ap: square"/>.  
 00009fb0: 203c 2f67 3e0a 2020 203c 6720 6964 3d22   </g>.   <g id="
 00009fc0: 6c69 6e65 3264 5f36 3422 3e0a 2020 2020  line2d_64">.    
 00009fd0: 3c70 6174 6820 636c 6970 2d70 6174 683d  <path clip-path=
-00009fe0: 2275 726c 2823 7032 3535 3763 3132 6361  "url(#p2557c12ca
-00009ff0: 3629 2220 7374 796c 653d 2266 696c 6c3a  6)" style="fill:
+00009fe0: 2275 726c 2823 7063 6430 3739 3063 6634  "url(#pcd0790cf4
+00009ff0: 3829 2220 7374 796c 653d 2266 696c 6c3a  8)" style="fill:
 0000a000: 206e 6f6e 653b 2073 7472 6f6b 653a 2023   none; stroke: #
 0000a010: 3432 3432 3432 3b20 7374 726f 6b65 2d77  424242; stroke-w
 0000a020: 6964 7468 3a20 322e 373b 2073 7472 6f6b  idth: 2.7; strok
 0000a030: 652d 6c69 6e65 6361 703a 2073 7175 6172  e-linecap: squar
 0000a040: 6522 2f3e 0a20 2020 3c2f 673e 0a20 2020  e"/>.   </g>.   
 0000a050: 3c67 2069 643d 226c 696e 6532 645f 3635  <g id="line2d_65
 0000a060: 223e 0a20 2020 203c 7061 7468 2063 6c69  ">.    <path cli
-0000a070: 702d 7061 7468 3d22 7572 6c28 2370 3235  p-path="url(#p25
-0000a080: 3537 6331 3263 6136 2922 2073 7479 6c65  57c12ca6)" style
+0000a070: 702d 7061 7468 3d22 7572 6c28 2370 6364  p-path="url(#pcd
+0000a080: 3037 3930 6366 3438 2922 2073 7479 6c65  0790cf48)" style
 0000a090: 3d22 6669 6c6c 3a20 6e6f 6e65 3b20 7374  ="fill: none; st
 0000a0a0: 726f 6b65 3a20 2334 3234 3234 323b 2073  roke: #424242; s
 0000a0b0: 7472 6f6b 652d 7769 6474 683a 2032 2e37  troke-width: 2.7
 0000a0c0: 3b20 7374 726f 6b65 2d6c 696e 6563 6170  ; stroke-linecap
 0000a0d0: 3a20 7371 7561 7265 222f 3e0a 2020 203c  : square"/>.   <
 0000a0e0: 2f67 3e0a 2020 203c 6720 6964 3d22 6c69  /g>.   <g id="li
 0000a0f0: 6e65 3264 5f36 3622 3e0a 2020 2020 3c70  ne2d_66">.    <p
 0000a100: 6174 6820 636c 6970 2d70 6174 683d 2275  ath clip-path="u
-0000a110: 726c 2823 7032 3535 3763 3132 6361 3629  rl(#p2557c12ca6)
+0000a110: 726c 2823 7063 6430 3739 3063 6634 3829  rl(#pcd0790cf48)
 0000a120: 2220 7374 796c 653d 2266 696c 6c3a 206e  " style="fill: n
 0000a130: 6f6e 653b 2073 7472 6f6b 653a 2023 3432  one; stroke: #42
 0000a140: 3432 3432 3b20 7374 726f 6b65 2d77 6964  4242; stroke-wid
 0000a150: 7468 3a20 322e 373b 2073 7472 6f6b 652d  th: 2.7; stroke-
 0000a160: 6c69 6e65 6361 703a 2073 7175 6172 6522  linecap: square"
 0000a170: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 0000a180: 2069 643d 2270 6174 6368 5f33 3222 3e0a   id="patch_32">.
@@ -2950,15 +2950,15 @@
 0000b850: 3d22 3231 3239 2e39 3638 3735 222f 3e0a  ="2129.96875"/>.
 0000b860: 2020 2020 203c 7573 6520 786c 696e 6b3a       <use xlink:
 0000b870: 6872 6566 3d22 2344 656a 6156 7553 616e  href="#DejaVuSan
 0000b880: 732d 3733 2220 783d 2232 3139 312e 3439  s-73" x="2191.49
 0000b890: 3231 3838 222f 3e0a 2020 2020 3c2f 673e  2188"/>.    </g>
 0000b8a0: 0a20 2020 3c2f 673e 0a20 203c 2f67 3e0a  .   </g>.  </g>.
 0000b8b0: 203c 2f67 3e0a 203c 6465 6673 3e0a 2020   </g>. <defs>.  
-0000b8c0: 3c63 6c69 7050 6174 6820 6964 3d22 7032  <clipPath id="p2
-0000b8d0: 3535 3763 3132 6361 3622 3e0a 2020 203c  557c12ca6">.   <
+0000b8c0: 3c63 6c69 7050 6174 6820 6964 3d22 7063  <clipPath id="pc
+0000b8d0: 6430 3739 3063 6634 3822 3e0a 2020 203c  d0790cf48">.   <
 0000b8e0: 7265 6374 2078 3d22 3434 2e35 3722 2079  rect x="44.57" y
 0000b8f0: 3d22 3236 2e38 3822 2077 6964 7468 3d22  ="26.88" width="
 0000b900: 3531 352e 3835 3331 3838 2220 6865 6967  515.853188" heig
 0000b910: 6874 3d22 3138 322e 3131 3837 3134 222f  ht="182.118714"/
 0000b920: 3e0a 2020 3c2f 636c 6970 5061 7468 3e0a  >.  </clipPath>.
 0000b930: 203c 2f64 6566 733e 0a3c 2f73 7667 3e0a   </defs>.</svg>.
```

### Comparing `bioregistry-0.9.90/docs/img/bioregistry_coverage.svg` & `bioregistry-0.9.91/docs/img/bioregistry_coverage.svg`

 * *Files 0% similar despite different names*

```diff
@@ -29,17 +29,17 @@
 000001c0: 392f 3032 2f32 322d 7264 662d 7379 6e74  9/02/22-rdf-synt
 000001d0: 6178 2d6e 7323 223e 0a20 2020 3c63 633a  ax-ns#">.   <cc:
 000001e0: 576f 726b 3e0a 2020 2020 3c64 633a 7479  Work>.    <dc:ty
 000001f0: 7065 2072 6466 3a72 6573 6f75 7263 653d  pe rdf:resource=
 00000200: 2268 7474 703a 2f2f 7075 726c 2e6f 7267  "http://purl.org
 00000210: 2f64 632f 6463 6d69 7479 7065 2f53 7469  /dc/dcmitype/Sti
 00000220: 6c6c 496d 6167 6522 2f3e 0a20 2020 203c  llImage"/>.    <
-00000230: 6463 3a64 6174 653e 3230 3233 2d30 372d  dc:date>2023-07-
-00000240: 3330 5430 313a 3136 3a33 342e 3232 3335  30T01:16:34.2235
-00000250: 3634 3c2f 6463 3a64 6174 653e 0a20 2020  64</dc:date>.   
+00000230: 6463 3a64 6174 653e 3230 3233 2d30 382d  dc:date>2023-08-
+00000240: 3031 5430 313a 3230 3a31 332e 3031 3730  01T01:20:13.0170
+00000250: 3831 3c2f 6463 3a64 6174 653e 0a20 2020  81</dc:date>.   
 00000260: 203c 6463 3a66 6f72 6d61 743e 696d 6167   <dc:format>imag
 00000270: 652f 7376 672b 786d 6c3c 2f64 633a 666f  e/svg+xml</dc:fo
 00000280: 726d 6174 3e0a 2020 2020 3c64 633a 6372  rmat>.    <dc:cr
 00000290: 6561 746f 723e 0a20 2020 2020 3c63 633a  eator>.     <cc:
 000002a0: 4167 656e 743e 0a20 2020 2020 203c 6463  Agent>.      <dc
 000002b0: 3a74 6974 6c65 3e4d 6174 706c 6f74 6c69  :title>Matplotli
 000002c0: 6220 7633 2e37 2e32 2c20 6874 7470 733a  b v3.7.2, https:
@@ -938,16 +938,16 @@
 00003a90: 3234 2039 392e 3030 3138 3236 2034 3030  24 99.001826 400
 00003aa0: 2e35 3531 3436 3220 3932 2e37 3339 3935  .551462 92.73995
 00003ab0: 3920 0a43 2034 3033 2e32 3339 3239 3920  9 .C 403.239299 
 00003ac0: 3836 2e34 3738 3039 3120 3430 342e 3637  86.478091 404.67
 00003ad0: 3335 3534 2037 392e 3734 3935 3731 2034  3554 79.749571 4
 00003ae0: 3034 2e37 3732 3839 3620 3732 2e39 3335  04.772896 72.935
 00003af0: 3933 3820 0a22 2063 6c69 702d 7061 7468  938 ." clip-path
-00003b00: 3d22 7572 6c28 2370 3639 3232 3162 3461  ="url(#p69221b4a
-00003b10: 3832 2922 2073 7479 6c65 3d22 6669 6c6c  82)" style="fill
+00003b00: 3d22 7572 6c28 2370 6463 3831 6131 3033  ="url(#pdc81a103
+00003b10: 3535 2922 2073 7479 6c65 3d22 6669 6c6c  55)" style="fill
 00003b20: 3a20 2363 3063 3063 303b 206f 7061 6369  : #c0c0c0; opaci
 00003b30: 7479 3a20 302e 3422 2f3e 0a20 2020 3c2f  ty: 0.4"/>.   </
 00003b40: 673e 0a20 2020 3c67 2069 643d 2270 6174  g>.   <g id="pat
 00003b50: 6368 5f36 223e 0a20 2020 203c 7061 7468  ch_6">.    <path
 00003b60: 2064 3d22 4d20 3430 342e 3737 3238 3936   d="M 404.772896
 00003b70: 2037 312e 3431 3538 3439 200a 4320 3430   71.415849 .C 40
 00003b80: 342e 3737 3635 3839 2037 312e 3636 3931  4.776589 71.6691
@@ -963,15 +963,15 @@
 00003c20: 3732 2e34 3239 3331 3420 3430 342e 3738  72.429314 404.78
 00003c30: 3736 3737 2037 322e 3137 3538 3934 200a  7677 72.175894 .
 00003c40: 4320 3430 342e 3738 3736 3737 2037 312e  C 404.787677 71.
 00003c50: 3932 3234 3734 2034 3034 2e37 3832 3734  922474 404.78274
 00003c60: 3920 3731 2e36 3639 3037 3720 3430 342e  9 71.669077 404.
 00003c70: 3737 3238 3936 2037 312e 3431 3538 3439  772896 71.415849
 00003c80: 200a 2220 636c 6970 2d70 6174 683d 2275   ." clip-path="u
-00003c90: 726c 2823 7036 3932 3231 6234 6138 3229  rl(#p69221b4a82)
+00003c90: 726c 2823 7064 6338 3161 3130 3335 3529  rl(#pdc81a10355)
 00003ca0: 2220 7374 796c 653d 2266 696c 6c3a 2023  " style="fill: #
 00003cb0: 3166 3738 6234 3b20 6f70 6163 6974 793a  1f78b4; opacity:
 00003cc0: 2030 2e34 222f 3e0a 2020 203c 2f67 3e0a   0.4"/>.   </g>.
 00003cd0: 2020 203c 6720 6964 3d22 7061 7463 685f     <g id="patch_
 00003ce0: 3722 3e0a 2020 2020 3c70 6174 6820 643d  7">.    <path d=
 00003cf0: 224d 2034 3034 2e37 3732 3839 3620 3732  "M 404.772896 72
 00003d00: 2e39 3335 3933 3820 0a43 2034 3034 2e37  .935938 .C 404.7
@@ -1045,15 +1045,15 @@
 00004140: 3832 2e35 3734 3439 3120 3430 332e 3033  82.574491 403.03
 00004150: 3436 3934 2038 302e 3236 3637 3632 200a  4694 80.266762 .
 00004160: 4320 3430 342e 3038 3339 3432 2037 372e  C 404.083942 77.
 00004170: 3935 3930 3333 2034 3034 2e36 3734 3333  959033 404.67433
 00004180: 3120 3735 2e34 3639 3038 3220 3430 342e  1 75.469082 404.
 00004190: 3737 3238 3936 2037 322e 3933 3539 3338  772896 72.935938
 000041a0: 200a 2220 636c 6970 2d70 6174 683d 2275   ." clip-path="u
-000041b0: 726c 2823 7036 3932 3231 6234 6138 3229  rl(#p69221b4a82)
+000041b0: 726c 2823 7064 6338 3161 3130 3335 3529  rl(#pdc81a10355)
 000041c0: 2220 7374 796c 653d 2266 696c 6c3a 2023  " style="fill: #
 000041d0: 3963 6461 6666 3b20 6f70 6163 6974 793a  9cdaff; opacity:
 000041e0: 2030 2e34 222f 3e0a 2020 203c 2f67 3e0a   0.4"/>.   </g>.
 000041f0: 2020 203c 6720 6964 3d22 7465 7874 5f36     <g id="text_6
 00004200: 223e 0a20 2020 203c 212d 2d20 3134 3631  ">.    <!-- 1461
 00004210: 202d 2d3e 0a20 2020 203c 6720 7374 796c   -->.    <g styl
 00004220: 653d 2266 696c 6c3a 2023 3236 3236 3236  e="fill: #262626
@@ -2440,15 +2440,15 @@
 00009870: 3937 3539 2034 3030 2e33 3639 3137 3420  9759 400.369174 
 00009880: 3233 352e 3934 3138 3438 200a 4320 3430  235.941848 .C 40
 00009890: 332e 3130 3330 3033 2032 3239 2e37 3233  3.103003 229.723
 000098a0: 3933 3820 3430 342e 3539 3535 3239 2032  938 404.595529 2
 000098b0: 3233 2e30 3331 3237 3620 3430 342e 3736  23.031276 404.76
 000098c0: 3236 3533 2032 3136 2e32 3430 3936 3720  2653 216.240967 
 000098d0: 0a22 2063 6c69 702d 7061 7468 3d22 7572  ." clip-path="ur
-000098e0: 6c28 2370 6261 3561 6564 3136 3862 2922  l(#pba5aed168b)"
+000098e0: 6c28 2370 3337 3233 3636 6263 3930 2922  l(#p372366bc90)"
 000098f0: 2073 7479 6c65 3d22 6669 6c6c 3a20 2363   style="fill: #c
 00009900: 3063 3063 303b 206f 7061 6369 7479 3a20  0c0c0; opacity: 
 00009910: 302e 3422 2f3e 0a20 2020 3c2f 673e 0a20  0.4"/>.   </g>. 
 00009920: 2020 3c67 2069 643d 2270 6174 6368 5f31    <g id="patch_1
 00009930: 3522 3e0a 2020 2020 3c70 6174 6820 643d  5">.    <path d=
 00009940: 224d 2034 3034 2e37 3632 3635 3320 3231  "M 404.762653 21
 00009950: 332e 3637 3534 3235 200a 4320 3430 342e  3.675425 .C 404.
@@ -2466,15 +2466,15 @@
 00009a10: 2034 3034 2e37 3837 3637 3720 3231 342e   404.787677 214.
 00009a20: 3935 3831 3936 200a 4320 3430 342e 3738  958196 .C 404.78
 00009a30: 3736 3737 2032 3134 2e35 3330 3438 3420  7677 214.530484 
 00009a40: 3430 342e 3737 3933 3334 2032 3134 2e31  404.779334 214.1
 00009a50: 3032 3831 3220 3430 342e 3736 3236 3533  02812 404.762653
 00009a60: 2032 3133 2e36 3735 3432 3520 0a22 2063   213.675425 ." c
 00009a70: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-00009a80: 6261 3561 6564 3136 3862 2922 2073 7479  ba5aed168b)" sty
+00009a80: 3337 3233 3636 6263 3930 2922 2073 7479  372366bc90)" sty
 00009a90: 6c65 3d22 6669 6c6c 3a20 2366 6239 6139  le="fill: #fb9a9
 00009aa0: 393b 206f 7061 6369 7479 3a20 302e 3422  9; opacity: 0.4"
 00009ab0: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 00009ac0: 2069 643d 2270 6174 6368 5f31 3622 3e0a   id="patch_16">.
 00009ad0: 2020 2020 3c70 6174 6820 643d 224d 2034      <path d="M 4
 00009ae0: 3034 2e37 3632 3635 3320 3231 362e 3234  04.762653 216.24
 00009af0: 3039 3637 200a 4320 3430 342e 3737 3331  0967 .C 404.7731
@@ -2551,15 +2551,15 @@
 00009f60: 3136 3220 3430 312e 3833 3637 3632 2032  162 401.836762 2
 00009f70: 3238 2e35 3734 3633 3320 0a43 2034 3033  28.574633 .C 403
 00009f80: 2e36 3032 3531 3420 3232 342e 3639 3231  .602514 224.6921
 00009f90: 3034 2034 3034 2e35 3936 3330 3620 3232  04 404.596306 22
 00009fa0: 302e 3530 3239 3139 2034 3034 2e37 3632  0.502919 404.762
 00009fb0: 3635 3320 3231 362e 3234 3039 3637 200a  653 216.240967 .
 00009fc0: 2220 636c 6970 2d70 6174 683d 2275 726c  " clip-path="url
-00009fd0: 2823 7062 6135 6165 6431 3638 6229 2220  (#pba5aed168b)" 
+00009fd0: 2823 7033 3732 3336 3662 6339 3029 2220  (#p372366bc90)" 
 00009fe0: 7374 796c 653d 2266 696c 6c3a 2023 6666  style="fill: #ff
 00009ff0: 6632 6632 3b20 6f70 6163 6974 793a 2030  f2f2; opacity: 0
 0000a000: 2e34 222f 3e0a 2020 203c 2f67 3e0a 2020  .4"/>.   </g>.  
 0000a010: 203c 6720 6964 3d22 7465 7874 5f32 3122   <g id="text_21"
 0000a020: 3e0a 2020 2020 3c21 2d2d 2031 3032 3420  >.    <!-- 1024 
 0000a030: 2d2d 3e0a 2020 2020 3c67 2073 7479 6c65  -->.    <g style
 0000a040: 3d22 6669 6c6c 3a20 2332 3632 3632 3622  ="fill: #262626"
@@ -3478,15 +3478,15 @@
 0000d950: 3337 3120 3336 332e 3236 3433 3339 2034  371 363.264339 4
 0000d960: 3032 2e34 3032 3337 3520 0a43 2033 3637  02.402375 .C 367
 0000d970: 2e37 3037 3335 3820 3339 392e 3839 3130  .707358 399.8910
 0000d980: 3339 2033 3731 2e37 3530 3836 3320 3339  39 371.750863 39
 0000d990: 362e 3733 3037 3038 2033 3735 2e32 3631  6.730708 375.261
 0000d9a0: 3120 3339 332e 3032 3539 3139 200a 2220  1 393.025919 ." 
 0000d9b0: 636c 6970 2d70 6174 683d 2275 726c 2823  clip-path="url(#
-0000d9c0: 7061 6166 6465 6461 6438 3729 2220 7374  paafdedad87)" st
+0000d9c0: 7062 6465 3464 3334 3738 3529 2220 7374  pbde4d34785)" st
 0000d9d0: 796c 653d 2266 696c 6c3a 2023 6330 6330  yle="fill: #c0c0
 0000d9e0: 6330 3b20 6f70 6163 6974 793a 2030 2e34  c0; opacity: 0.4
 0000d9f0: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 0000da00: 6720 6964 3d22 7061 7463 685f 3234 223e  g id="patch_24">
 0000da10: 0a20 2020 203c 7061 7468 2064 3d22 4d20  .    <path d="M 
 0000da20: 3337 352e 3236 3131 2033 3232 2e34 3535  375.2611 322.455
 0000da30: 3037 3720 0a43 2033 3834 2e32 3837 3138  077 .C 384.28718
@@ -3528,16 +3528,16 @@
 0000dc70: 3330 3734 3537 2033 3234 2e30 3134 3133  307457 324.01413
 0000dc80: 3220 3339 312e 3131 3131 3735 2033 3232  2 391.111175 322
 0000dc90: 2e37 3432 3736 3120 0a43 2033 3835 2e39  .742761 .C 385.9
 0000dca0: 3134 3839 3320 3332 312e 3437 3133 3931  14893 321.471391
 0000dcb0: 2033 3830 2e35 3030 3039 3620 3332 312e   380.500096 321.
 0000dcc0: 3337 3331 3131 2033 3735 2e32 3631 3120  373111 375.2611 
 0000dcd0: 3332 322e 3435 3530 3737 200a 2220 636c  322.455077 ." cl
-0000dce0: 6970 2d70 6174 683d 2275 726c 2823 7061  ip-path="url(#pa
-0000dcf0: 6166 6465 6461 6438 3729 2220 7374 796c  afdedad87)" styl
+0000dce0: 6970 2d70 6174 683d 2275 726c 2823 7062  ip-path="url(#pb
+0000dcf0: 6465 3464 3334 3738 3529 2220 7374 796c  de4d34785)" styl
 0000dd00: 653d 2266 696c 6c3a 2023 6666 3766 3030  e="fill: #ff7f00
 0000dd10: 3b20 6f70 6163 6974 793a 2030 2e34 222f  ; opacity: 0.4"/
 0000dd20: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 0000dd30: 6964 3d22 7061 7463 685f 3235 223e 0a20  id="patch_25">. 
 0000dd40: 2020 203c 7061 7468 2064 3d22 4d20 3337     <path d="M 37
 0000dd50: 352e 3236 3131 2033 3933 2e30 3235 3931  5.2611 393.02591
 0000dd60: 3920 0a43 2033 3834 2e32 3837 3138 3520  9 .C 384.287185 
@@ -3563,15 +3563,15 @@
 0000dea0: 3337 2033 3534 2e36 3133 3335 3720 3338  37 354.613357 38
 0000deb0: 302e 3439 3537 3833 200a 4320 3335 392e  0.495783 .C 359.
 0000dec0: 3834 3832 3139 2033 3836 2e39 3232 3232  848219 386.92222
 0000ded0: 3820 3336 372e 3134 3336 3735 2033 3931  8 367.143675 391
 0000dee0: 2e33 3439 3439 3520 3337 352e 3236 3131  .349495 375.2611
 0000def0: 2033 3933 2e30 3235 3931 3920 0a22 2063   393.025919 ." c
 0000df00: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-0000df10: 6161 6664 6564 6164 3837 2922 2073 7479  aafdedad87)" sty
+0000df10: 6264 6534 6433 3437 3835 2922 2073 7479  bde4d34785)" sty
 0000df20: 6c65 3d22 6669 6c6c 3a20 2366 6664 6638  le="fill: #ffdf8
 0000df30: 363b 206f 7061 6369 7479 3a20 302e 3422  6; opacity: 0.4"
 0000df40: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 0000df50: 2069 643d 2274 6578 745f 3336 223e 0a20   id="text_36">. 
 0000df60: 2020 203c 212d 2d20 3132 3437 202d 2d3e     <!-- 1247 -->
 0000df70: 0a20 2020 203c 6720 7374 796c 653d 2266  .    <g style="f
 0000df80: 696c 6c3a 2023 3236 3236 3236 2220 7472  ill: #262626" tr
@@ -4638,15 +4638,15 @@
 000121d0: 3238 3838 2033 3839 2e32 3834 3831 3520  2888 389.284815 
 000121e0: 3533 312e 3234 3034 3131 200a 4320 3339  531.240411 .C 39
 000121f0: 322e 3936 3631 3438 2035 3236 2e32 3037  2.966148 526.207
 00012200: 3933 3320 3339 352e 3731 3330 3037 2035  933 395.713007 5
 00012210: 3230 2e35 3534 3431 3520 3339 372e 3339  20.554415 397.39
 00012220: 3430 3938 2035 3134 2e35 3530 3038 3520  4098 514.550085 
 00012230: 0a22 2063 6c69 702d 7061 7468 3d22 7572  ." clip-path="ur
-00012240: 6c28 2370 3864 6437 6164 3836 6465 2922  l(#p8dd7ad86de)"
+00012240: 6c28 2370 6162 6230 3366 3164 3361 2922  l(#pabb03f1d3a)"
 00012250: 2073 7479 6c65 3d22 6669 6c6c 3a20 2363   style="fill: #c
 00012260: 3063 3063 303b 206f 7061 6369 7479 3a20  0c0c0; opacity: 
 00012270: 302e 3422 2f3e 0a20 2020 3c2f 673e 0a20  0.4"/>.   </g>. 
 00012280: 2020 3c67 2069 643d 2270 6174 6368 5f33    <g id="patch_3
 00012290: 3322 3e0a 2020 2020 3c70 6174 6820 643d  3">.    <path d=
 000122a0: 224d 2033 3937 2e33 3934 3039 3820 3438  "M 397.394098 48
 000122b0: 362e 3439 3535 3136 200a 4320 3339 382e  6.495516 .C 398.
@@ -4672,16 +4672,16 @@
 000123f0: 3439 332e 3637 3236 3320 3430 362e 3430  493.67263 406.40
 00012400: 3535 3038 2034 3931 2e30 3832 3937 3520  5508 491.082975 
 00012410: 0a43 2034 3034 2e30 3732 3634 3620 3438  .C 404.072646 48
 00012420: 382e 3439 3333 3139 2034 3030 2e38 3630  8.493319 400.860
 00012430: 3635 3720 3438 362e 3835 3831 3834 2033  657 486.858184 3
 00012440: 3937 2e33 3934 3039 3820 3438 362e 3439  97.394098 486.49
 00012450: 3535 3136 200a 2220 636c 6970 2d70 6174  5516 ." clip-pat
-00012460: 683d 2275 726c 2823 7038 6464 3761 6438  h="url(#p8dd7ad8
-00012470: 3664 6529 2220 7374 796c 653d 2266 696c  6de)" style="fil
+00012460: 683d 2275 726c 2823 7061 6262 3033 6631  h="url(#pabb03f1
+00012470: 6433 6129 2220 7374 796c 653d 2266 696c  d3a)" style="fil
 00012480: 6c3a 2023 6666 6666 3939 3b20 6f70 6163  l: #ffff99; opac
 00012490: 6974 793a 2030 2e34 222f 3e0a 2020 203c  ity: 0.4"/>.   <
 000124a0: 2f67 3e0a 2020 203c 6720 6964 3d22 7061  /g>.   <g id="pa
 000124b0: 7463 685f 3334 223e 0a20 2020 203c 7061  tch_34">.    <pa
 000124c0: 7468 2064 3d22 4d20 3339 372e 3339 3430  th d="M 397.3940
 000124d0: 3938 2035 3134 2e35 3530 3038 3520 0a43  98 514.550085 .C
 000124e0: 2033 3938 2e36 3732 3531 3320 3530 392e   398.672513 509.
@@ -4723,16 +4723,16 @@
 00012720: 3834 3438 2035 3133 2e33 3233 3532 3920  8448 513.323529 
 00012730: 3339 312e 3536 3334 3232 2035 3133 2e39  391.563422 513.9
 00012740: 3334 3738 3220 0a43 2033 3933 2e34 3432  34782 .C 393.442
 00012750: 3336 3520 3531 342e 3534 3630 3335 2033  365 514.546035 3
 00012760: 3935 2e34 3238 3935 3620 3531 342e 3735  95.428956 514.75
 00012770: 3536 3737 2033 3937 2e33 3934 3039 3820  5677 397.394098 
 00012780: 3531 342e 3535 3030 3835 200a 2220 636c  514.550085 ." cl
-00012790: 6970 2d70 6174 683d 2275 726c 2823 7038  ip-path="url(#p8
-000127a0: 6464 3761 6438 3664 6529 2220 7374 796c  dd7ad86de)" styl
+00012790: 6970 2d70 6174 683d 2275 726c 2823 7061  ip-path="url(#pa
+000127a0: 6262 3033 6631 6433 6129 2220 7374 796c  bb03f1d3a)" styl
 000127b0: 653d 2266 696c 6c3a 2023 6666 6666 6632  e="fill: #fffff2
 000127c0: 3b20 6f70 6163 6974 793a 2030 2e34 222f  ; opacity: 0.4"/
 000127d0: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 000127e0: 6964 3d22 7465 7874 5f35 3122 3e0a 2020  id="text_51">.  
 000127f0: 2020 3c21 2d2d 2031 3632 3320 2d2d 3e0a    <!-- 1623 -->.
 00012800: 2020 2020 3c67 2073 7479 6c65 3d22 6669      <g style="fi
 00012810: 6c6c 3a20 2332 3632 3632 3622 2074 7261  ll: #262626" tra
@@ -5289,15 +5289,15 @@
 00014a80: 3031 3635 2031 3632 2e32 3535 3333 3420  0165 162.255334 
 00014a90: 3637 332e 3334 3935 3737 200a 4320 3136  673.349577 .C 16
 00014aa0: 352e 3837 3938 3237 2036 3638 2e32 3138  5.879827 668.218
 00014ab0: 3938 3920 3136 382e 3534 3631 3736 2036  989 168.546176 6
 00014ac0: 3632 2e34 3734 3631 3320 3137 302e 3132  62.474613 170.12
 00014ad0: 3532 3633 2036 3536 2e33 3934 3631 3620  5263 656.394616 
 00014ae0: 0a22 2063 6c69 702d 7061 7468 3d22 7572  ." clip-path="ur
-00014af0: 6c28 2370 3366 3065 3432 3130 3432 2922  l(#p3f0e421042)"
+00014af0: 6c28 2370 3431 3934 3538 6233 3464 2922  l(#p419458b34d)"
 00014b00: 2073 7479 6c65 3d22 6669 6c6c 3a20 2363   style="fill: #c
 00014b10: 3063 3063 303b 206f 7061 6369 7479 3a20  0c0c0; opacity: 
 00014b20: 302e 3422 2f3e 0a20 2020 3c2f 673e 0a20  0.4"/>.   </g>. 
 00014b30: 2020 3c67 2069 643d 2270 6174 6368 5f33    <g id="patch_3
 00014b40: 3922 3e0a 2020 2020 3c70 6174 6820 643d  9">.    <path d=
 00014b50: 224d 2031 3730 2e31 3235 3236 3320 3633  "M 170.125263 63
 00014b60: 302e 3231 3535 3839 200a 4320 3137 312e  0.215589 .C 171.
@@ -5323,16 +5323,16 @@
 00014ca0: 2036 3337 2e37 3137 3431 3320 3137 362e   637.717413 176.
 00014cb0: 3630 3932 3431 2036 3335 2e33 3832 3730  609241 635.38270
 00014cc0: 3820 0a43 2031 3735 2e30 3136 3637 3520  8 .C 175.016675 
 00014cd0: 3633 332e 3034 3830 3033 2031 3732 2e37  633.048003 172.7
 00014ce0: 3536 3532 3920 3633 312e 3234 3638 3820  56529 631.24688 
 00014cf0: 3137 302e 3132 3532 3633 2036 3330 2e32  170.125263 630.2
 00014d00: 3135 3538 3920 0a22 2063 6c69 702d 7061  15589 ." clip-pa
-00014d10: 7468 3d22 7572 6c28 2370 3366 3065 3432  th="url(#p3f0e42
-00014d20: 3130 3432 2922 2073 7479 6c65 3d22 6669  1042)" style="fi
+00014d10: 7468 3d22 7572 6c28 2370 3431 3934 3538  th="url(#p419458
+00014d20: 6233 3464 2922 2073 7479 6c65 3d22 6669  b34d)" style="fi
 00014d30: 6c6c 3a20 2361 3663 6565 333b 206f 7061  ll: #a6cee3; opa
 00014d40: 6369 7479 3a20 302e 3422 2f3e 0a20 2020  city: 0.4"/>.   
 00014d50: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 00014d60: 6174 6368 5f34 3022 3e0a 2020 2020 3c70  atch_40">.    <p
 00014d70: 6174 6820 643d 224d 2031 3730 2e31 3235  ath d="M 170.125
 00014d80: 3236 3320 3635 362e 3339 3436 3136 200a  263 656.394616 .
 00014d90: 4320 3137 312e 3233 3534 3533 2036 3532  C 171.235453 652
@@ -5375,15 +5375,15 @@
 00014fe0: 3035 2031 3633 2e34 3136 3034 3220 3635  05 163.416042 65
 00014ff0: 372e 3237 3531 3338 200a 4320 3136 352e  7.275138 .C 165.
 00015000: 3639 3036 3538 2036 3537 2e35 3332 3232  690658 657.53222
 00015010: 3520 3136 372e 3939 3430 3135 2036 3537  5 167.994015 657
 00015020: 2e32 3239 3933 3120 3137 302e 3132 3532  .229931 170.1252
 00015030: 3633 2036 3536 2e33 3934 3631 3620 0a22  63 656.394616 ."
 00015040: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00015050: 2370 3366 3065 3432 3130 3432 2922 2073  #p3f0e421042)" s
+00015050: 2370 3431 3934 3538 6233 3464 2922 2073  #p419458b34d)" s
 00015060: 7479 6c65 3d22 6669 6c6c 3a20 2366 6266  tyle="fill: #fbf
 00015070: 6666 663b 206f 7061 6369 7479 3a20 302e  fff; opacity: 0.
 00015080: 3422 2f3e 0a20 2020 3c2f 673e 0a20 2020  4"/>.   </g>.   
 00015090: 3c67 2069 643d 2274 6578 745f 3631 223e  <g id="text_61">
 000150a0: 0a20 2020 203c 212d 2d20 3136 3035 202d  .    <!-- 1605 -
 000150b0: 2d3e 0a20 2020 203c 6720 7374 796c 653d  ->.    <g style=
 000150c0: 2266 696c 6c3a 2023 3236 3236 3236 2220  "fill: #262626" 
@@ -5656,16 +5656,16 @@
 00016170: 3837 3336 2036 3831 2e37 3136 3136 3520  8736 681.716165 
 00016180: 3338 382e 3139 3832 3634 2036 3737 2e31  388.198264 677.1
 00016190: 3139 3839 3620 0a43 2033 3932 2e31 3237  19896 .C 392.127
 000161a0: 3739 3220 3637 322e 3532 3336 3237 2033  792 672.523627 3
 000161b0: 3935 2e32 3236 3233 3420 3636 372e 3237  95.226234 667.27
 000161c0: 3733 3739 2033 3937 2e33 3534 3232 3520  7379 397.354225 
 000161d0: 3636 312e 3631 3731 3139 200a 2220 636c  661.617119 ." cl
-000161e0: 6970 2d70 6174 683d 2275 726c 2823 7062  ip-path="url(#pb
-000161f0: 3637 6338 6631 6639 3129 2220 7374 796c  67c8f1f91)" styl
+000161e0: 6970 2d70 6174 683d 2275 726c 2823 7033  ip-path="url(#p3
+000161f0: 3832 6166 3639 3332 3729 2220 7374 796c  82af69327)" styl
 00016200: 653d 2266 696c 6c3a 2023 6330 6330 6330  e="fill: #c0c0c0
 00016210: 3b20 6f70 6163 6974 793a 2030 2e34 222f  ; opacity: 0.4"/
 00016220: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 00016230: 6964 3d22 7061 7463 685f 3432 223e 0a20  id="patch_42">. 
 00016240: 2020 203c 7061 7468 2064 3d22 4d20 3339     <path d="M 39
 00016250: 372e 3335 3432 3235 2036 3234 2e39 3933  7.354225 624.993
 00016260: 3038 3620 0a43 2033 3939 2e35 3535 3135  086 .C 399.55515
@@ -5691,15 +5691,15 @@
 000163a0: 3637 2034 3035 2e36 3234 3333 3920 3633  67 405.624339 63
 000163b0: 322e 3533 3736 3532 200a 4320 3430 332e  2.537652 .C 403.
 000163c0: 3632 3237 3239 2036 3239 2e33 3033 3233  622729 629.30323
 000163d0: 3720 3430 302e 3735 3833 3238 2036 3236  7 400.758328 626
 000163e0: 2e36 3930 3133 3320 3339 372e 3335 3432  .690133 397.3542
 000163f0: 3235 2036 3234 2e39 3933 3038 3620 0a22  25 624.993086 ."
 00016400: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00016410: 2370 6236 3763 3866 3166 3931 2922 2073  #pb67c8f1f91)" s
+00016410: 2370 3338 3261 6636 3933 3237 2922 2073  #p382af69327)" s
 00016420: 7479 6c65 3d22 6669 6c6c 3a20 2331 6637  tyle="fill: #1f7
 00016430: 3862 343b 206f 7061 6369 7479 3a20 302e  8b4; opacity: 0.
 00016440: 3422 2f3e 0a20 2020 3c2f 673e 0a20 2020  4"/>.   </g>.   
 00016450: 3c67 2069 643d 2270 6174 6368 5f34 3322  <g id="patch_43"
 00016460: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 00016470: 2033 3937 2e33 3534 3232 3520 3636 312e   397.354225 661.
 00016480: 3631 3731 3139 200a 4320 3339 392e 3535  617119 .C 399.55
@@ -5742,16 +5742,16 @@
 000166d0: 3633 2e35 3835 3239 3420 3338 372e 3238  63.585294 387.28
 000166e0: 3733 3932 2036 3633 2e37 3435 3033 3520  7392 663.745035 
 000166f0: 0a43 2033 3930 2e37 3639 3234 3720 3636  .C 390.769247 66
 00016700: 332e 3930 3437 3736 2033 3934 2e32 3334  3.904776 394.234
 00016710: 3835 3220 3636 332e 3137 3232 3220 3339  852 663.17222 39
 00016720: 372e 3335 3432 3235 2036 3631 2e36 3137  7.354225 661.617
 00016730: 3131 3920 0a22 2063 6c69 702d 7061 7468  119 ." clip-path
-00016740: 3d22 7572 6c28 2370 6236 3763 3866 3166  ="url(#pb67c8f1f
-00016750: 3931 2922 2073 7479 6c65 3d22 6669 6c6c  91)" style="fill
+00016740: 3d22 7572 6c28 2370 3338 3261 6636 3933  ="url(#p382af693
+00016750: 3237 2922 2073 7479 6c65 3d22 6669 6c6c  27)" style="fill
 00016760: 3a20 2339 6364 6166 663b 206f 7061 6369  : #9cdaff; opaci
 00016770: 7479 3a20 302e 3422 2f3e 0a20 2020 3c2f  ty: 0.4"/>.   </
 00016780: 673e 0a20 2020 3c67 2069 643d 2274 6578  g>.   <g id="tex
 00016790: 745f 3636 223e 0a20 2020 203c 212d 2d20  t_66">.    <!-- 
 000167a0: 3134 3831 202d 2d3e 0a20 2020 203c 6720  1481 -->.    <g 
 000167b0: 7374 796c 653d 2266 696c 6c3a 2023 3236  style="fill: #26
 000167c0: 3236 3236 2220 7472 616e 7366 6f72 6d3d  2626" transform=
@@ -5989,16 +5989,16 @@
 00017640: 3520 3637 322e 3934 3338 3033 2036 3237  5 672.943803 627
 00017650: 2e32 3333 3534 3520 3636 372e 3032 3630  .233545 667.0260
 00017660: 3634 200a 4320 3633 302e 3235 3736 3535  64 .C 630.257655
 00017670: 2036 3631 2e31 3038 3332 3520 3633 322e   661.108325 632.
 00017680: 3132 3234 3033 2036 3534 2e36 3636 3531  122403 654.66651
 00017690: 3120 3633 322e 3732 3731 3133 2036 3438  1 632.727113 648
 000176a0: 2e30 3438 3431 3520 0a22 2063 6c69 702d  .048415 ." clip-
-000176b0: 7061 7468 3d22 7572 6c28 2370 6361 6331  path="url(#pcac1
-000176c0: 3864 6661 6637 2922 2073 7479 6c65 3d22  8dfaf7)" style="
+000176b0: 7061 7468 3d22 7572 6c28 2370 6262 6339  path="url(#pbbc9
+000176c0: 3363 3834 3137 2922 2073 7479 6c65 3d22  3c8417)" style="
 000176d0: 6669 6c6c 3a20 2363 3063 3063 303b 206f  fill: #c0c0c0; o
 000176e0: 7061 6369 7479 3a20 302e 3422 2f3e 0a20  pacity: 0.4"/>. 
 000176f0: 2020 3c2f 673e 0a20 2020 3c67 2069 643d    </g>.   <g id=
 00017700: 2270 6174 6368 5f34 3522 3e0a 2020 2020  "patch_45">.    
 00017710: 3c70 6174 6820 643d 224d 2036 3332 2e37  <path d="M 632.7
 00017720: 3237 3131 3320 3633 382e 3536 3137 3931  27113 638.561791
 00017730: 200a 4320 3633 322e 3837 3132 3037 2036   .C 632.871207 6
@@ -6023,16 +6023,16 @@
 00017860: 3620 3634 312e 3435 3732 3337 2036 3334  6 641.457237 634
 00017870: 2e35 3431 3236 3320 3634 302e 3633 3136  .541263 640.6316
 00017880: 200a 4320 3633 342e 3131 3136 3539 2036   .C 634.111659 6
 00017890: 3339 2e38 3035 3936 3420 3633 332e 3438  39.805964 633.48
 000178a0: 3933 3132 2036 3339 2e30 3935 3931 3320  9312 639.095913 
 000178b0: 3633 322e 3732 3731 3133 2036 3338 2e35  632.727113 638.5
 000178c0: 3631 3739 3120 0a22 2063 6c69 702d 7061  61791 ." clip-pa
-000178d0: 7468 3d22 7572 6c28 2370 6361 6331 3864  th="url(#pcac18d
-000178e0: 6661 6637 2922 2073 7479 6c65 3d22 6669  faf7)" style="fi
+000178d0: 7468 3d22 7572 6c28 2370 6262 6339 3363  th="url(#pbbc93c
+000178e0: 3834 3137 2922 2073 7479 6c65 3d22 6669  8417)" style="fi
 000178f0: 6c6c 3a20 2362 3264 6638 613b 206f 7061  ll: #b2df8a; opa
 00017900: 6369 7479 3a20 302e 3422 2f3e 0a20 2020  city: 0.4"/>.   
 00017910: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 00017920: 6174 6368 5f34 3622 3e0a 2020 2020 3c70  atch_46">.    <p
 00017930: 6174 6820 643d 224d 2036 3332 2e37 3237  ath d="M 632.727
 00017940: 3131 3320 3634 382e 3034 3834 3135 200a  113 648.048415 .
 00017950: 4320 3633 322e 3837 3132 3037 2036 3436  C 632.871207 646
@@ -6075,15 +6075,15 @@
 00017ba0: 3534 3232 3720 3632 392e 3738 3336 3036  54227 629.783606
 00017bb0: 2036 3439 2e30 3834 3632 3420 0a43 2036   649.084624 .C 6
 00017bc0: 3330 2e38 3431 3030 3420 3634 392e 3031  30.841004 649.01
 00017bd0: 3530 3231 2036 3331 2e38 3539 3239 3620  5021 631.859296 
 00017be0: 3634 382e 3635 3635 3439 2036 3332 2e37  648.656549 632.7
 00017bf0: 3237 3131 3320 3634 382e 3034 3834 3135  27113 648.048415
 00017c00: 200a 2220 636c 6970 2d70 6174 683d 2275   ." clip-path="u
-00017c10: 726c 2823 7063 6163 3138 6466 6166 3729  rl(#pcac18dfaf7)
+00017c10: 726c 2823 7062 6263 3933 6338 3431 3729  rl(#pbbc93c8417)
 00017c20: 2220 7374 796c 653d 2266 696c 6c3a 2023  " style="fill: #
 00017c30: 6666 6666 6537 3b20 6f70 6163 6974 793a  ffffe7; opacity:
 00017c40: 2030 2e34 222f 3e0a 2020 203c 2f67 3e0a   0.4"/>.   </g>.
 00017c50: 2020 203c 6720 6964 3d22 7465 7874 5f37     <g id="text_7
 00017c60: 3122 3e0a 2020 2020 3c21 2d2d 2031 3638  1">.    <!-- 168
 00017c70: 3320 2d2d 3e0a 2020 2020 3c67 2073 7479  3 -->.    <g sty
 00017c80: 6c65 3d22 6669 6c6c 3a20 2332 3632 3632  le="fill: #26262
@@ -6694,15 +6694,15 @@
 0001a250: 3635 3338 3133 2033 3835 2e39 3738 3535  653813 385.97855
 0001a260: 3220 3831 372e 3736 3737 3037 200a 4320  2 817.767707 .C 
 0001a270: 3338 392e 3733 3833 3236 2038 3132 2e38  389.738326 812.8
 0001a280: 3831 3630 3120 3339 322e 3539 3834 3339  81601 392.598439
 0001a290: 2038 3037 2e33 3634 3937 3620 3339 342e   807.364976 394.
 0001a2a0: 3432 3437 3933 2038 3031 2e34 3736 3438  424793 801.47648
 0001a2b0: 3220 0a22 2063 6c69 702d 7061 7468 3d22  2 ." clip-path="
-0001a2c0: 7572 6c28 2370 3532 6536 3864 3337 3764  url(#p52e68d377d
+0001a2c0: 7572 6c28 2370 3064 3832 3932 3437 3230  url(#p0d82924720
 0001a2d0: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 0001a2e0: 2363 3063 3063 303b 206f 7061 6369 7479  #c0c0c0; opacity
 0001a2f0: 3a20 302e 3422 2f3e 0a20 2020 3c2f 673e  : 0.4"/>.   </g>
 0001a300: 0a20 2020 3c67 2069 643d 2270 6174 6368  .   <g id="patch
 0001a310: 5f35 3122 3e0a 2020 2020 3c70 6174 6820  _51">.    <path 
 0001a320: 643d 224d 2033 3934 2e34 3234 3739 3320  d="M 394.424793 
 0001a330: 3737 302e 3639 3833 3239 200a 4320 3339  770.698329 .C 39
@@ -6745,16 +6745,16 @@
 0001a580: 3636 2037 3731 2e37 3035 3035 3920 3430  66 771.705059 40
 0001a590: 312e 3038 3937 3334 2037 3731 2e30 3933  1.089734 771.093
 0001a5a0: 3333 3720 0a43 2033 3938 2e39 3232 3830  337 .C 398.92280
 0001a5b0: 3720 3737 302e 3438 3136 3134 2033 3936  7 770.481614 396
 0001a5c0: 2e36 3438 3830 3620 3737 302e 3334 3638  .648806 770.3468
 0001a5d0: 3431 2033 3934 2e34 3234 3739 3320 3737  41 394.424793 77
 0001a5e0: 302e 3639 3833 3239 200a 2220 636c 6970  0.698329 ." clip
-0001a5f0: 2d70 6174 683d 2275 726c 2823 7035 3265  -path="url(#p52e
-0001a600: 3638 6433 3737 6429 2220 7374 796c 653d  68d377d)" style=
+0001a5f0: 2d70 6174 683d 2275 726c 2823 7030 6438  -path="url(#p0d8
+0001a600: 3239 3234 3732 3029 2220 7374 796c 653d  2924720)" style=
 0001a610: 2266 696c 6c3a 2023 6662 3961 3939 3b20  "fill: #fb9a99; 
 0001a620: 6f70 6163 6974 793a 2030 2e34 222f 3e0a  opacity: 0.4"/>.
 0001a630: 2020 203c 2f67 3e0a 2020 203c 6720 6964     </g>.   <g id
 0001a640: 3d22 7061 7463 685f 3532 223e 0a20 2020  ="patch_52">.   
 0001a650: 203c 7061 7468 2064 3d22 4d20 3339 342e   <path d="M 394.
 0001a660: 3432 3437 3933 2038 3031 2e34 3736 3438  424793 801.47648
 0001a670: 3220 0a43 2033 3935 2e39 3730 3434 3120  2 .C 395.970441 
@@ -6780,15 +6780,15 @@
 0001a7b0: 3431 3320 3338 352e 3031 3134 3137 2037  413 385.011417 7
 0001a7c0: 3936 2e32 3037 3833 3720 0a43 2033 3837  96.207837 .C 387
 0001a7d0: 2e34 3231 3130 3220 3739 392e 3032 3832  .421102 799.0282
 0001a7e0: 3631 2033 3930 2e37 3630 3633 3920 3830  61 390.760639 80
 0001a7f0: 302e 3839 3733 3932 2033 3934 2e34 3234  0.897392 394.424
 0001a800: 3739 3320 3830 312e 3437 3634 3832 200a  793 801.476482 .
 0001a810: 2220 636c 6970 2d70 6174 683d 2275 726c  " clip-path="url
-0001a820: 2823 7035 3265 3638 6433 3737 6429 2220  (#p52e68d377d)" 
+0001a820: 2823 7030 6438 3239 3234 3732 3029 2220  (#p0d82924720)" 
 0001a830: 7374 796c 653d 2266 696c 6c3a 2023 6666  style="fill: #ff
 0001a840: 6632 6632 3b20 6f70 6163 6974 793a 2030  f2f2; opacity: 0
 0001a850: 2e34 222f 3e0a 2020 203c 2f67 3e0a 2020  .4"/>.   </g>.  
 0001a860: 203c 6720 6964 3d22 7465 7874 5f38 3122   <g id="text_81"
 0001a870: 3e0a 2020 2020 3c21 2d2d 2031 3633 3020  >.    <!-- 1630 
 0001a880: 2d2d 3e0a 2020 2020 3c67 2073 7479 6c65  -->.    <g style
 0001a890: 3d22 6669 6c6c 3a20 2332 3632 3632 3622  ="fill: #262626"
@@ -7016,16 +7016,16 @@
 0001b670: 3330 3438 3920 3831 352e 3934 3439 3339  30489 815.944939
 0001b680: 2036 3233 2e32 3830 3235 3520 3831 302e   623.280255 810.
 0001b690: 3036 3239 3531 200a 4320 3632 362e 3333  062951 .C 626.33
 0001b6a0: 3030 3220 3830 342e 3138 3039 3633 2036  002 804.180963 6
 0001b6b0: 3238 2e32 3239 3635 3320 3739 372e 3737  28.229653 797.77
 0001b6c0: 3037 3233 2036 3238 2e38 3737 3034 3120  0723 628.877041 
 0001b6d0: 3739 312e 3137 3638 3037 200a 2220 636c  791.176807 ." cl
-0001b6e0: 6970 2d70 6174 683d 2275 726c 2823 7034  ip-path="url(#p4
-0001b6f0: 3935 3364 3930 3435 3429 2220 7374 796c  953d90454)" styl
+0001b6e0: 6970 2d70 6174 683d 2275 726c 2823 7064  ip-path="url(#pd
+0001b6f0: 6466 6132 3630 3035 6229 2220 7374 796c  dfa26005b)" styl
 0001b700: 653d 2266 696c 6c3a 2023 6330 6330 6330  e="fill: #c0c0c0
 0001b710: 3b20 6f70 6163 6974 793a 2030 2e34 222f  ; opacity: 0.4"/
 0001b720: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 0001b730: 6964 3d22 7061 7463 685f 3534 223e 0a20  id="patch_54">. 
 0001b740: 2020 203c 7061 7468 2064 3d22 4d20 3632     <path d="M 62
 0001b750: 382e 3837 3730 3431 2037 3830 2e39 3938  8.877041 780.998
 0001b760: 3030 3420 0a43 2036 3239 2e30 3433 3130  004 .C 629.04310
@@ -7068,15 +7068,15 @@
 0001b9b0: 3639 3831 3434 2036 3332 2e30 3935 3139  698144 632.09519
 0001b9c0: 2037 3739 2e37 3934 3232 3720 0a43 2036   779.794227 .C 6
 0001b9d0: 3330 2e39 3331 3933 3520 3737 392e 3839  30.931935 779.89
 0001b9e0: 3033 3039 2036 3239 2e38 3137 3738 3620  0309 629.817786 
 0001b9f0: 3738 302e 3330 3730 3637 2036 3238 2e38  780.307067 628.8
 0001ba00: 3737 3034 3120 3738 302e 3939 3830 3034  77041 780.998004
 0001ba10: 200a 2220 636c 6970 2d70 6174 683d 2275   ." clip-path="u
-0001ba20: 726c 2823 7034 3935 3364 3930 3435 3429  rl(#p4953d90454)
+0001ba20: 726c 2823 7064 6466 6132 3630 3035 6229  rl(#pddfa26005b)
 0001ba30: 2220 7374 796c 653d 2266 696c 6c3a 2023  " style="fill: #
 0001ba40: 6533 3161 3163 3b20 6f70 6163 6974 793a  e31a1c; opacity:
 0001ba50: 2030 2e34 222f 3e0a 2020 203c 2f67 3e0a   0.4"/>.   </g>.
 0001ba60: 2020 203c 6720 6964 3d22 7061 7463 685f     <g id="patch_
 0001ba70: 3535 223e 0a20 2020 203c 7061 7468 2064  55">.    <path d
 0001ba80: 3d22 4d20 3632 382e 3837 3730 3431 2037  ="M 628.877041 7
 0001ba90: 3931 2e31 3736 3830 3720 0a43 2036 3239  91.176807 .C 629
@@ -7102,16 +7102,16 @@
 0001bbd0: 382e 3035 3535 3432 2036 3236 2e39 3831  8.055542 626.981
 0001bbe0: 3235 3620 3738 382e 3933 3936 3439 200a  256 788.939649 .
 0001bbf0: 4320 3632 372e 3432 3838 3620 3738 392e  C 627.42886 789.
 0001bc00: 3832 3337 3535 2036 3238 2e30 3738 3335  823755 628.07835
 0001bc10: 3820 3739 302e 3539 3032 3037 2036 3238  8 790.590207 628
 0001bc20: 2e38 3737 3034 3120 3739 312e 3137 3638  .877041 791.1768
 0001bc30: 3037 200a 2220 636c 6970 2d70 6174 683d  07 ." clip-path=
-0001bc40: 2275 726c 2823 7034 3935 3364 3930 3435  "url(#p4953d9045
-0001bc50: 3429 2220 7374 796c 653d 2266 696c 6c3a  4)" style="fill:
+0001bc40: 2275 726c 2823 7064 6466 6132 3630 3035  "url(#pddfa26005
+0001bc50: 6229 2220 7374 796c 653d 2266 696c 6c3a  b)" style="fill:
 0001bc60: 2023 6666 3939 3961 3b20 6f70 6163 6974   #ff999a; opacit
 0001bc70: 793a 2030 2e34 222f 3e0a 2020 203c 2f67  y: 0.4"/>.   </g
 0001bc80: 3e0a 2020 203c 6720 6964 3d22 7465 7874  >.   <g id="text
 0001bc90: 5f38 3622 3e0a 2020 2020 3c21 2d2d 2031  _86">.    <!-- 1
 0001bca0: 3639 3720 2d2d 3e0a 2020 2020 3c67 2073  697 -->.    <g s
 0001bcb0: 7479 6c65 3d22 6669 6c6c 3a20 2332 3632  tyle="fill: #262
 0001bcc0: 3632 3622 2074 7261 6e73 666f 726d 3d22  626" transform="
@@ -7749,16 +7749,16 @@
 0001e440: 3434 3635 3220 3935 382e 3537 3039 3339  44652 958.570939
 0001e450: 2033 3938 2e32 3735 3939 3720 3935 322e   398.275997 952.
 0001e460: 3636 3037 3632 200a 4320 3430 312e 3330  660762 .C 401.30
 0001e470: 3733 3432 2039 3436 2e37 3530 3538 3420  7342 946.750584 
 0001e480: 3430 332e 3138 3133 3720 3934 302e 3331  403.18137 940.31
 0001e490: 3531 3432 2034 3033 2e37 3937 3031 3520  5142 403.797015 
 0001e4a0: 3933 332e 3730 3135 3031 200a 2220 636c  933.701501 ." cl
-0001e4b0: 6970 2d70 6174 683d 2275 726c 2823 7036  ip-path="url(#p6
-0001e4c0: 3934 6338 6362 3637 3729 2220 7374 796c  94c8cb677)" styl
+0001e4b0: 6970 2d70 6174 683d 2275 726c 2823 7062  ip-path="url(#pb
+0001e4c0: 3536 6339 3232 6662 3329 2220 7374 796c  56c922fb3)" styl
 0001e4d0: 653d 2266 696c 6c3a 2023 6330 6330 6330  e="fill: #c0c0c0
 0001e4e0: 3b20 6f70 6163 6974 793a 2030 2e34 222f  ; opacity: 0.4"/
 0001e4f0: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 0001e500: 6964 3d22 7061 7463 685f 3630 223e 0a20  id="patch_60">. 
 0001e510: 2020 203c 7061 7468 2064 3d22 4d20 3430     <path d="M 40
 0001e520: 332e 3739 3730 3135 2039 3234 2e30 3337  3.797015 924.037
 0001e530: 3931 3420 0a43 2034 3033 2e39 3436 3533  914 .C 403.94653
@@ -7775,16 +7775,16 @@
 0001e5e0: 2039 3330 2e36 3333 3035 3520 3430 352e   930.633055 405.
 0001e5f0: 3533 3534 3639 2039 3238 2e38 3639 3730  535469 928.86970
 0001e600: 3720 0a43 2034 3035 2e35 3335 3436 3920  7 .C 405.535469 
 0001e610: 3932 372e 3130 3633 3620 3430 342e 3932  927.10636 404.92
 0001e620: 3034 3638 2039 3235 2e33 3937 3034 3920  0468 925.397049 
 0001e630: 3430 332e 3739 3730 3135 2039 3234 2e30  403.797015 924.0
 0001e640: 3337 3931 3420 0a22 2063 6c69 702d 7061  37914 ." clip-pa
-0001e650: 7468 3d22 7572 6c28 2370 3639 3463 3863  th="url(#p694c8c
-0001e660: 6236 3737 2922 2073 7479 6c65 3d22 6669  b677)" style="fi
+0001e650: 7468 3d22 7572 6c28 2370 6235 3663 3932  th="url(#pb56c92
+0001e660: 3266 6233 2922 2073 7479 6c65 3d22 6669  2fb3)" style="fi
 0001e670: 6c6c 3a20 2366 6637 6630 303b 206f 7061  ll: #ff7f00; opa
 0001e680: 6369 7479 3a20 302e 3422 2f3e 0a20 2020  city: 0.4"/>.   
 0001e690: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 0001e6a0: 6174 6368 5f36 3122 3e0a 2020 2020 3c70  atch_61">.    <p
 0001e6b0: 6174 6820 643d 224d 2034 3033 2e37 3937  ath d="M 403.797
 0001e6c0: 3031 3520 3933 332e 3730 3135 3031 200a  015 933.701501 .
 0001e6d0: 4320 3430 332e 3934 3635 3337 2039 3332  C 403.946537 932
@@ -7861,15 +7861,15 @@
 0001eb40: 3320 3430 322e 3036 3736 3435 2039 3335  3 402.067645 935
 0001eb50: 2e32 3339 3432 3720 0a43 2034 3032 2e37  .239427 .C 402.7
 0001eb60: 3139 3338 2039 3334 2e38 3138 3238 3120  1938 934.818281 
 0001eb70: 3430 332e 3330 3236 3336 2039 3334 2e32  403.302636 934.2
 0001eb80: 3939 3539 3320 3430 332e 3739 3730 3135  99593 403.797015
 0001eb90: 2039 3333 2e37 3031 3530 3120 0a22 2063   933.701501 ." c
 0001eba0: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-0001ebb0: 3639 3463 3863 6236 3737 2922 2073 7479  694c8cb677)" sty
+0001ebb0: 6235 3663 3932 3266 6233 2922 2073 7479  b56c922fb3)" sty
 0001ebc0: 6c65 3d22 6669 6c6c 3a20 2366 6664 6638  le="fill: #ffdf8
 0001ebd0: 363b 206f 7061 6369 7479 3a20 302e 3422  6; opacity: 0.4"
 0001ebe0: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 0001ebf0: 2069 643d 2274 6578 745f 3936 223e 0a20   id="text_96">. 
 0001ec00: 2020 203c 212d 2d20 3136 3637 202d 2d3e     <!-- 1667 -->
 0001ec10: 0a20 2020 203c 6720 7374 796c 653d 2266  .    <g style="f
 0001ec20: 696c 6c3a 2023 3236 3236 3236 2220 7472  ill: #262626" tr
@@ -8788,15 +8788,15 @@
 00022530: 3337 302e 3336 3033 3534 2031 3130 342e  370.360354 1104.
 00022540: 3434 3737 3433 200a 4320 3337 342e 3139  447743 .C 374.19
 00022550: 3232 2031 3039 392e 3834 3432 3434 2033  22 1099.844244 3
 00022560: 3737 2e31 3837 3735 3720 3130 3934 2e36  77.187757 1094.6
 00022570: 3034 3539 3720 3337 392e 3231 3038 3337  04597 379.210837
 00022580: 2031 3038 382e 3936 3730 3120 0a22 2063   1088.96701 ." c
 00022590: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-000225a0: 6362 6633 3966 6637 3431 2922 2073 7479  cbf39ff741)" sty
+000225a0: 3562 6538 3637 3938 3566 2922 2073 7479  5be867985f)" sty
 000225b0: 6c65 3d22 6669 6c6c 3a20 2363 3063 3063  le="fill: #c0c0c
 000225c0: 303b 206f 7061 6369 7479 3a20 302e 3422  0; opacity: 0.4"
 000225d0: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 000225e0: 2069 643d 2270 6174 6368 5f36 3922 3e0a   id="patch_69">.
 000225f0: 2020 2020 3c70 6174 6820 643d 224d 2033      <path d="M 3
 00022600: 3739 2e32 3130 3833 3720 3130 3534 2e33  79.210837 1054.3
 00022610: 3337 3030 3920 0a43 2033 3831 2e32 3034  37009 .C 381.204
@@ -8876,15 +8876,15 @@
 00022ab0: 3738 2033 3835 2e32 3935 3435 2031 3034  78 385.29545 104
 00022ac0: 392e 3136 3339 3136 200a 4320 3338 332e  9.163916 .C 383.
 00022ad0: 3031 3638 3939 2031 3035 302e 3537 3030  016899 1050.5700
 00022ae0: 3534 2033 3830 2e39 3635 3233 3720 3130  54 380.965237 10
 00022af0: 3532 2e33 3134 3336 3320 3337 392e 3231  52.314363 379.21
 00022b00: 3038 3337 2031 3035 342e 3333 3730 3039  0837 1054.337009
 00022b10: 200a 2220 636c 6970 2d70 6174 683d 2275   ." clip-path="u
-00022b20: 726c 2823 7063 6266 3339 6666 3734 3129  rl(#pcbf39ff741)
+00022b20: 726c 2823 7035 6265 3836 3739 3835 6629  rl(#p5be867985f)
 00022b30: 2220 7374 796c 653d 2266 696c 6c3a 2023  " style="fill: #
 00022b40: 6666 6666 3939 3b20 6f70 6163 6974 793a  ffff99; opacity:
 00022b50: 2030 2e34 222f 3e0a 2020 203c 2f67 3e0a   0.4"/>.   </g>.
 00022b60: 2020 203c 6720 6964 3d22 7061 7463 685f     <g id="patch_
 00022b70: 3730 223e 0a20 2020 203c 7061 7468 2064  70">.    <path d
 00022b80: 3d22 4d20 3337 392e 3231 3038 3337 2031  ="M 379.210837 1
 00022b90: 3038 382e 3936 3730 3120 0a43 2033 3831  088.96701 .C 381
@@ -8902,16 +8902,16 @@
 00022c50: 3635 2e32 3932 3231 3520 3337 322e 3734  65.292215 372.74
 00022c60: 3737 3538 2031 3037 312e 3635 3230 3120  7758 1071.65201 
 00022c70: 0a43 2033 3732 2e37 3437 3735 3820 3130  .C 372.747758 10
 00022c80: 3738 2e30 3131 3830 3420 3337 352e 3034  78.011804 375.04
 00022c90: 3336 3631 2031 3038 342e 3136 3236 3736  3661 1084.162676
 00022ca0: 2033 3739 2e32 3130 3833 3720 3130 3838   379.210837 1088
 00022cb0: 2e39 3637 3031 200a 2220 636c 6970 2d70  .96701 ." clip-p
-00022cc0: 6174 683d 2275 726c 2823 7063 6266 3339  ath="url(#pcbf39
-00022cd0: 6666 3734 3129 2220 7374 796c 653d 2266  ff741)" style="f
+00022cc0: 6174 683d 2275 726c 2823 7035 6265 3836  ath="url(#p5be86
+00022cd0: 3739 3835 6629 2220 7374 796c 653d 2266  7985f)" style="f
 00022ce0: 696c 6c3a 2023 6666 6666 6632 3b20 6f70  ill: #fffff2; op
 00022cf0: 6163 6974 793a 2030 2e34 222f 3e0a 2020  acity: 0.4"/>.  
 00022d00: 203c 2f67 3e0a 2020 203c 6720 6964 3d22   </g>.   <g id="
 00022d10: 7465 7874 5f31 3131 223e 0a20 2020 203c  text_111">.    <
 00022d20: 212d 2d20 3136 3535 202d 2d3e 0a20 2020  !-- 1655 -->.   
 00022d30: 203c 6720 7374 796c 653d 2266 696c 6c3a   <g style="fill:
 00022d40: 2023 3236 3236 3236 2220 7472 616e 7366   #262626" transf
@@ -9372,15 +9372,15 @@
 000249b0: 3033 3930 3632 222f 3e0a 2020 2020 3c2f  039062"/>.    </
 000249c0: 673e 0a20 2020 3c2f 673e 0a20 203c 2f67  g>.   </g>.  </g
 000249d0: 3e0a 2020 3c67 2069 643d 2274 6578 745f  >.  <g id="text_
 000249e0: 3132 3122 3e0a 2020 203c 212d 2d20 6874  121">.   <!-- ht
 000249f0: 7470 733a 2f2f 6769 7468 7562 2e63 6f6d  tps://github.com
 00024a00: 2f62 696f 7072 6167 6d61 7469 6373 2f62  /biopragmatics/b
 00024a10: 696f 7265 6769 7374 7279 2028 3230 3233  ioregistry (2023
-00024a20: 2d30 372d 3330 2920 2d2d 3e0a 2020 203c  -07-30) -->.   <
+00024a20: 2d30 382d 3031 2920 2d2d 3e0a 2020 203c  -08-01) -->.   <
 00024a30: 6720 7374 796c 653d 2266 696c 6c3a 2023  g style="fill: #
 00024a40: 3830 3830 3830 3b20 6f70 6163 6974 793a  808080; opacity:
 00024a50: 2030 2e35 2220 7472 616e 7366 6f72 6d3d   0.5" transform=
 00024a60: 2274 7261 6e73 6c61 7465 2832 3333 2e36  "translate(233.6
 00024a70: 3736 3235 2031 3135 302e 3333 3632 3529  7625 1150.33625)
 00024a80: 2073 6361 6c65 2830 2e30 3820 2d30 2e30   scale(0.08 -0.0
 00024a90: 3829 223e 0a20 2020 203c 6465 6673 3e0a  8)">.    <defs>.
@@ -9629,41 +9629,41 @@
 000259c0: 616e 732d 3264 2220 783d 2232 3536 372e  ans-2d" x="2567.
 000259d0: 3432 3737 3334 222f 3e0a 2020 2020 3c75  427734"/>.    <u
 000259e0: 7365 2078 6c69 6e6b 3a68 7265 663d 2223  se xlink:href="#
 000259f0: 4465 6a61 5675 5361 6e73 2d33 3022 2078  DejaVuSans-30" x
 00025a00: 3d22 3236 3033 2e35 3131 3731 3922 2f3e  ="2603.511719"/>
 00025a10: 0a20 2020 203c 7573 6520 786c 696e 6b3a  .    <use xlink:
 00025a20: 6872 6566 3d22 2344 656a 6156 7553 616e  href="#DejaVuSan
-00025a30: 732d 3337 2220 783d 2232 3636 372e 3133  s-37" x="2667.13
+00025a30: 732d 3338 2220 783d 2232 3636 372e 3133  s-38" x="2667.13
 00025a40: 3437 3636 222f 3e0a 2020 2020 3c75 7365  4766"/>.    <use
 00025a50: 2078 6c69 6e6b 3a68 7265 663d 2223 4465   xlink:href="#De
 00025a60: 6a61 5675 5361 6e73 2d32 6422 2078 3d22  jaVuSans-2d" x="
 00025a70: 3237 3330 2e37 3537 3831 3222 2f3e 0a20  2730.757812"/>. 
 00025a80: 2020 203c 7573 6520 786c 696e 6b3a 6872     <use xlink:hr
 00025a90: 6566 3d22 2344 656a 6156 7553 616e 732d  ef="#DejaVuSans-
-00025aa0: 3333 2220 783d 2232 3736 362e 3834 3137  33" x="2766.8417
+00025aa0: 3330 2220 783d 2232 3736 362e 3834 3137  30" x="2766.8417
 00025ab0: 3937 222f 3e0a 2020 2020 3c75 7365 2078  97"/>.    <use x
 00025ac0: 6c69 6e6b 3a68 7265 663d 2223 4465 6a61  link:href="#Deja
-00025ad0: 5675 5361 6e73 2d33 3022 2078 3d22 3238  VuSans-30" x="28
+00025ad0: 5675 5361 6e73 2d33 3122 2078 3d22 3238  VuSans-31" x="28
 00025ae0: 3330 2e34 3634 3834 3422 2f3e 0a20 2020  30.464844"/>.   
 00025af0: 203c 7573 6520 786c 696e 6b3a 6872 6566   <use xlink:href
 00025b00: 3d22 2344 656a 6156 7553 616e 732d 3239  ="#DejaVuSans-29
 00025b10: 2220 783d 2232 3839 342e 3038 3738 3931  " x="2894.087891
 00025b20: 222f 3e0a 2020 203c 2f67 3e0a 2020 3c2f  "/>.   </g>.  </
 00025b30: 673e 0a20 3c2f 673e 0a20 3c64 6566 733e  g>. </g>. <defs>
 00025b40: 0a20 203c 636c 6970 5061 7468 2069 643d  .  <clipPath id=
 00025b50: 2270 3336 3235 3832 6533 3039 223e 0a20  "p362582e309">. 
 00025b60: 2020 3c72 6563 7420 783d 2236 312e 3637    <rect x="61.67
 00025b70: 3731 3737 2220 793d 2231 302e 3822 2077  7177" y="10.8" w
 00025b80: 6964 7468 3d22 3132 332e 3335 3531 3231  idth="123.355121
 00025b90: 2220 6865 6967 6874 3d22 3132 322e 3735  " height="122.75
 00025ba0: 3137 3837 222f 3e0a 2020 3c2f 636c 6970  1787"/>.  </clip
 00025bb0: 5061 7468 3e0a 2020 3c63 6c69 7050 6174  Path>.  <clipPat
-00025bc0: 6820 6964 3d22 7036 3932 3231 6234 6138  h id="p69221b4a8
-00025bd0: 3222 3e0a 2020 203c 7265 6374 2078 3d22  2">.   <rect x="
+00025bc0: 6820 6964 3d22 7064 6338 3161 3130 3335  h id="pdc81a1035
+00025bd0: 3522 3e0a 2020 203c 7265 6374 2078 3d22  5">.   <rect x="
 00025be0: 3239 312e 3236 3733 3638 2220 793d 2231  291.267368" y="1
 00025bf0: 302e 3822 2077 6964 7468 3d22 3132 322e  0.8" width="122.
 00025c00: 3736 3130 3238 2220 6865 6967 6874 3d22  761028" height="
 00025c10: 3132 322e 3735 3137 3837 222f 3e0a 2020  122.751787"/>.  
 00025c20: 3c2f 636c 6970 5061 7468 3e0a 2020 3c63  </clipPath>.  <c
 00025c30: 6c69 7050 6174 6820 6964 3d22 7035 3537  lipPath id="p557
 00025c40: 6331 6330 3739 3222 3e0a 2020 203c 7265  c1c0792">.   <re
@@ -9676,16 +9676,16 @@
 00025cb0: 3d22 7065 6634 3661 3238 6166 3222 3e0a  ="pef46a28af2">.
 00025cc0: 2020 203c 7265 6374 2078 3d22 3338 2e38     <rect x="38.8
 00025cd0: 3738 3538 2220 793d 2231 3533 2e35 3832  7858" y="153.582
 00025ce0: 3330 3222 2077 6964 7468 3d22 3136 382e  302" width="168.
 00025cf0: 3935 3233 3135 2220 6865 6967 6874 3d22  952315" height="
 00025d00: 3132 322e 3735 3137 3837 222f 3e0a 2020  122.751787"/>.  
 00025d10: 3c2f 636c 6970 5061 7468 3e0a 2020 3c63  </clipPath>.  <c
-00025d20: 6c69 7050 6174 6820 6964 3d22 7062 6135  lipPath id="pba5
-00025d30: 6165 6431 3638 6222 3e0a 2020 203c 7265  aed168b">.   <re
+00025d20: 6c69 7050 6174 6820 6964 3d22 7033 3732  lipPath id="p372
+00025d30: 3336 3662 6339 3022 3e0a 2020 203c 7265  366bc90">.   <re
 00025d40: 6374 2078 3d22 3239 312e 3236 3733 3638  ct x="291.267368
 00025d50: 2220 793d 2231 3533 2e35 3832 3330 3222  " y="153.582302"
 00025d60: 2077 6964 7468 3d22 3132 322e 3736 3130   width="122.7610
 00025d70: 3238 2220 6865 6967 6874 3d22 3132 322e  28" height="122.
 00025d80: 3735 3137 3837 222f 3e0a 2020 3c2f 636c  751787"/>.  </cl
 00025d90: 6970 5061 7468 3e0a 2020 3c63 6c69 7050  ipPath>.  <clipP
 00025da0: 6174 6820 6964 3d22 7061 6661 3263 3839  ath id="pafa2c89
@@ -9700,15 +9700,15 @@
 00025e30: 3e0a 2020 203c 7265 6374 2078 3d22 3338  >.   <rect x="38
 00025e40: 2e33 3235 3938 3222 2079 3d22 3239 362e  .325982" y="296.
 00025e50: 3336 3436 3035 2220 7769 6474 683d 2231  364605" width="1
 00025e60: 3730 2e30 3537 3531 3122 2068 6569 6768  70.057511" heigh
 00025e70: 743d 2231 3232 2e37 3531 3738 3722 2f3e  t="122.751787"/>
 00025e80: 0a20 203c 2f63 6c69 7050 6174 683e 0a20  .  </clipPath>. 
 00025e90: 203c 636c 6970 5061 7468 2069 643d 2270   <clipPath id="p
-00025ea0: 6161 6664 6564 6164 3837 223e 0a20 2020  aafdedad87">.   
+00025ea0: 6264 6534 6433 3437 3835 223e 0a20 2020  bde4d34785">.   
 00025eb0: 3c72 6563 7420 783d 2232 3736 2e36 3434  <rect x="276.644
 00025ec0: 3133 3422 2079 3d22 3239 362e 3336 3436  134" y="296.3646
 00025ed0: 3035 2220 7769 6474 683d 2231 3532 2e30  05" width="152.0
 00025ee0: 3037 3439 3622 2068 6569 6768 743d 2231  07496" height="1
 00025ef0: 3232 2e37 3531 3738 3722 2f3e 0a20 203c  22.751787"/>.  <
 00025f00: 2f63 6c69 7050 6174 683e 0a20 203c 636c  /clipPath>.  <cl
 00025f10: 6970 5061 7468 2069 643d 2270 6432 6664  ipPath id="pd2fd
@@ -9723,85 +9723,85 @@
 00025fa0: 3139 223e 0a20 2020 3c72 6563 7420 783d  19">.   <rect x=
 00025fb0: 2235 382e 3836 3938 3837 2220 793d 2234  "58.869887" y="4
 00025fc0: 3339 2e31 3436 3930 3722 2077 6964 7468  39.146907" width
 00025fd0: 3d22 3132 382e 3936 3937 3031 2220 6865  ="128.969701" he
 00025fe0: 6967 6874 3d22 3132 322e 3735 3137 3837  ight="122.751787
 00025ff0: 222f 3e0a 2020 3c2f 636c 6970 5061 7468  "/>.  </clipPath
 00026000: 3e0a 2020 3c63 6c69 7050 6174 6820 6964  >.  <clipPath id
-00026010: 3d22 7038 6464 3761 6438 3664 6522 3e0a  ="p8dd7ad86de">.
+00026010: 3d22 7061 6262 3033 6631 6433 6122 3e0a  ="pabb03f1d3a">.
 00026020: 2020 203c 7265 6374 2078 3d22 3238 352e     <rect x="285.
 00026030: 3931 3731 3437 2220 793d 2234 3339 2e31  917147" y="439.1
 00026040: 3436 3930 3722 2077 6964 7468 3d22 3133  46907" width="13
 00026050: 332e 3436 3134 3731 2220 6865 6967 6874  3.461471" height
 00026060: 3d22 3132 322e 3735 3137 3837 222f 3e0a  ="122.751787"/>.
 00026070: 2020 3c2f 636c 6970 5061 7468 3e0a 2020    </clipPath>.  
 00026080: 3c63 6c69 7050 6174 6820 6964 3d22 7034  <clipPath id="p4
 00026090: 3139 3639 3535 3930 6322 3e0a 2020 203c  19695590c">.   <
 000260a0: 7265 6374 2078 3d22 3531 332e 3330 3339  rect x="513.3039
 000260b0: 3731 2220 793d 2234 3339 2e31 3436 3930  71" y="439.14690
 000260c0: 3722 2077 6964 7468 3d22 3133 372e 3237  7" width="137.27
 000260d0: 3431 3131 2220 6865 6967 6874 3d22 3132  4111" height="12
 000260e0: 322e 3735 3137 3837 222f 3e0a 2020 3c2f  2.751787"/>.  </
 000260f0: 636c 6970 5061 7468 3e0a 2020 3c63 6c69  clipPath>.  <cli
-00026100: 7050 6174 6820 6964 3d22 7033 6630 6534  pPath id="p3f0e4
-00026110: 3231 3034 3222 3e0a 2020 203c 7265 6374  21042">.   <rect
+00026100: 7050 6174 6820 6964 3d22 7034 3139 3435  pPath id="p41945
+00026110: 3862 3334 6422 3e0a 2020 203c 7265 6374  8b34d">.   <rect
 00026120: 2078 3d22 3538 2e33 3530 3531 3422 2079   x="58.350514" y
 00026130: 3d22 3538 312e 3932 3932 3039 2220 7769  ="581.929209" wi
 00026140: 6474 683d 2231 3330 2e30 3038 3434 3722  dth="130.008447"
 00026150: 2068 6569 6768 743d 2231 3232 2e37 3531   height="122.751
 00026160: 3738 3722 2f3e 0a20 203c 2f63 6c69 7050  787"/>.  </clipP
 00026170: 6174 683e 0a20 203c 636c 6970 5061 7468  ath>.  <clipPath
-00026180: 2069 643d 2270 6236 3763 3866 3166 3931   id="pb67c8f1f91
+00026180: 2069 643d 2270 3338 3261 6636 3933 3237   id="p382af69327
 00026190: 223e 0a20 2020 3c72 6563 7420 783d 2232  ">.   <rect x="2
 000261a0: 3837 2e32 3730 3036 3722 2079 3d22 3538  87.270067" y="58
 000261b0: 312e 3932 3932 3039 2220 7769 6474 683d  1.929209" width=
 000261c0: 2231 3330 2e37 3535 3633 3122 2068 6569  "130.755631" hei
 000261d0: 6768 743d 2231 3232 2e37 3531 3738 3722  ght="122.751787"
 000261e0: 2f3e 0a20 203c 2f63 6c69 7050 6174 683e  />.  </clipPath>
 000261f0: 0a20 203c 636c 6970 5061 7468 2069 643d  .  <clipPath id=
-00026200: 2270 6361 6331 3864 6661 6637 223e 0a20  "pcac18dfaf7">. 
+00026200: 2270 6262 6339 3363 3834 3137 223e 0a20  "pbbc93c8417">. 
 00026210: 2020 3c72 6563 7420 783d 2235 3139 2e34    <rect x="519.4
 00026220: 3339 3231 3622 2079 3d22 3538 312e 3932  39216" y="581.92
 00026230: 3932 3039 2220 7769 6474 683d 2231 3235  9209" width="125
 00026240: 2e30 3033 3632 3122 2068 6569 6768 743d  .003621" height=
 00026250: 2231 3232 2e37 3531 3738 3722 2f3e 0a20  "122.751787"/>. 
 00026260: 203c 2f63 6c69 7050 6174 683e 0a20 203c   </clipPath>.  <
 00026270: 636c 6970 5061 7468 2069 643d 2270 6531  clipPath id="pe1
 00026280: 3536 3833 6136 6638 223e 0a20 2020 3c72  5683a6f8">.   <r
 00026290: 6563 7420 783d 2234 312e 3035 3138 3739  ect x="41.051879
 000262a0: 2220 793d 2237 3234 2e37 3131 3531 3222  " y="724.711512"
 000262b0: 2077 6964 7468 3d22 3136 342e 3630 3537   width="164.6057
 000262c0: 3137 2220 6865 6967 6874 3d22 3132 322e  17" height="122.
 000262d0: 3735 3137 3837 222f 3e0a 2020 3c2f 636c  751787"/>.  </cl
 000262e0: 6970 5061 7468 3e0a 2020 3c63 6c69 7050  ipPath>.  <clipP
-000262f0: 6174 6820 6964 3d22 7035 3265 3638 6433  ath id="p52e68d3
-00026300: 3737 6422 3e0a 2020 203c 7265 6374 2078  77d">.   <rect x
+000262f0: 6174 6820 6964 3d22 7030 6438 3239 3234  ath id="p0d82924
+00026300: 3732 3022 3e0a 2020 203c 7265 6374 2078  720">.   <rect x
 00026310: 3d22 3238 332e 3433 3137 3522 2079 3d22  ="283.43175" y="
 00026320: 3732 342e 3731 3135 3132 2220 7769 6474  724.711512" widt
 00026330: 683d 2231 3338 2e34 3332 3236 3322 2068  h="138.432263" h
 00026340: 6569 6768 743d 2231 3232 2e37 3531 3738  eight="122.75178
 00026350: 3722 2f3e 0a20 203c 2f63 6c69 7050 6174  7"/>.  </clipPat
 00026360: 683e 0a20 203c 636c 6970 5061 7468 2069  h>.  <clipPath i
-00026370: 643d 2270 3439 3533 6439 3034 3534 223e  d="p4953d90454">
+00026370: 643d 2270 6464 6661 3236 3030 3562 223e  d="pddfa26005b">
 00026380: 0a20 2020 3c72 6563 7420 783d 2235 3135  .   <rect x="515
 00026390: 2e36 3633 3437 3222 2079 3d22 3732 342e  .663472" y="724.
 000263a0: 3731 3135 3132 2220 7769 6474 683d 2231  711512" width="1
 000263b0: 3332 2e35 3535 3131 2220 6865 6967 6874  32.55511" height
 000263c0: 3d22 3132 322e 3735 3137 3837 222f 3e0a  ="122.751787"/>.
 000263d0: 2020 3c2f 636c 6970 5061 7468 3e0a 2020    </clipPath>.  
 000263e0: 3c63 6c69 7050 6174 6820 6964 3d22 7032  <clipPath id="p2
 000263f0: 3666 3134 3766 3966 3022 3e0a 2020 203c  6f147f9f0">.   <
 00026400: 7265 6374 2078 3d22 3331 2e37 3438 3937  rect x="31.74897
 00026410: 3222 2079 3d22 3836 372e 3439 3338 3134  2" y="867.493814
 00026420: 2220 7769 6474 683d 2231 3833 2e32 3131  " width="183.211
 00026430: 3533 3122 2068 6569 6768 743d 2231 3232  531" height="122
 00026440: 2e37 3531 3738 3722 2f3e 0a20 203c 2f63  .751787"/>.  </c
 00026450: 6c69 7050 6174 683e 0a20 203c 636c 6970  lipPath>.  <clip
-00026460: 5061 7468 2069 643d 2270 3639 3463 3863  Path id="p694c8c
-00026470: 6236 3737 223e 0a20 2020 3c72 6563 7420  b677">.   <rect 
+00026460: 5061 7468 2069 643d 2270 6235 3663 3932  Path id="pb56c92
+00026470: 3266 6233 223e 0a20 2020 3c72 6563 7420  2fb3">.   <rect 
 00026480: 783d 2232 3930 2e35 3134 3936 3422 2079  x="290.514964" y
 00026490: 3d22 3836 372e 3439 3338 3134 2220 7769  ="867.493814" wi
 000264a0: 6474 683d 2231 3234 2e32 3635 3833 3722  dth="124.265837"
 000264b0: 2068 6569 6768 743d 2231 3232 2e37 3531   height="122.751
 000264c0: 3738 3722 2f3e 0a20 203c 2f63 6c69 7050  787"/>.  </clipP
 000264d0: 6174 683e 0a20 203c 636c 6970 5061 7468  ath>.  <clipPath
 000264e0: 2069 643d 2270 6132 3364 3337 3361 3231   id="pa23d373a21
@@ -9815,16 +9815,16 @@
 00026560: 2270 3965 3031 6337 3632 6235 223e 0a20  "p9e01c762b5">. 
 00026570: 2020 3c72 6563 7420 783d 2233 322e 3135    <rect x="32.15
 00026580: 3034 3834 2220 793d 2231 3031 302e 3237  0484" y="1010.27
 00026590: 3631 3136 2220 7769 6474 683d 2231 3832  6116" width="182
 000265a0: 2e34 3038 3530 3822 2068 6569 6768 743d  .408508" height=
 000265b0: 2231 3232 2e37 3531 3738 3722 2f3e 0a20  "122.751787"/>. 
 000265c0: 203c 2f63 6c69 7050 6174 683e 0a20 203c   </clipPath>.  <
-000265d0: 636c 6970 5061 7468 2069 643d 2270 6362  clipPath id="pcb
-000265e0: 6633 3966 6637 3431 223e 0a20 2020 3c72  f39ff741">.   <r
+000265d0: 636c 6970 5061 7468 2069 643d 2270 3562  clipPath id="p5b
+000265e0: 6538 3637 3938 3566 223e 0a20 2020 3c72  e867985f">.   <r
 000265f0: 6563 7420 783d 2232 3639 2e35 3834 3336  ect x="269.58436
 00026600: 3322 2079 3d22 3130 3130 2e32 3736 3131  3" y="1010.27611
 00026610: 3622 2077 6964 7468 3d22 3136 362e 3132  6" width="166.12
 00026620: 3730 3339 2220 6865 6967 6874 3d22 3132  7039" height="12
 00026630: 322e 3735 3137 3837 222f 3e0a 2020 3c2f  2.751787"/>.  </
 00026640: 636c 6970 5061 7468 3e0a 2020 3c63 6c69  clipPath>.  <cli
 00026650: 7050 6174 6820 6964 3d22 7065 6134 3433  pPath id="pea443
```

### Comparing `bioregistry-0.9.90/docs/img/bioregistry_coverage_bar.svg` & `bioregistry-0.9.91/docs/img/bioregistry_coverage_bar.svg`

 * *Files 0% similar despite different names*

```diff
@@ -29,16 +29,16 @@
 000001c0: 3032 2f32 322d 7264 662d 7379 6e74 6178  02/22-rdf-syntax
 000001d0: 2d6e 7323 223e 0a20 2020 3c63 633a 576f  -ns#">.   <cc:Wo
 000001e0: 726b 3e0a 2020 2020 3c64 633a 7479 7065  rk>.    <dc:type
 000001f0: 2072 6466 3a72 6573 6f75 7263 653d 2268   rdf:resource="h
 00000200: 7474 703a 2f2f 7075 726c 2e6f 7267 2f64  ttp://purl.org/d
 00000210: 632f 6463 6d69 7479 7065 2f53 7469 6c6c  c/dcmitype/Still
 00000220: 496d 6167 6522 2f3e 0a20 2020 203c 6463  Image"/>.    <dc
-00000230: 3a64 6174 653e 3230 3233 2d30 372d 3330  :date>2023-07-30
-00000240: 5430 313a 3136 3a33 332e 3136 3236 3831  T01:16:33.162681
+00000230: 3a64 6174 653e 3230 3233 2d30 382d 3031  :date>2023-08-01
+00000240: 5430 313a 3230 3a31 312e 3335 3835 3837  T01:20:11.358587
 00000250: 3c2f 6463 3a64 6174 653e 0a20 2020 203c  </dc:date>.    <
 00000260: 6463 3a66 6f72 6d61 743e 696d 6167 652f  dc:format>image/
 00000270: 7376 672b 786d 6c3c 2f64 633a 666f 726d  svg+xml</dc:form
 00000280: 6174 3e0a 2020 2020 3c64 633a 6372 6561  at>.    <dc:crea
 00000290: 746f 723e 0a20 2020 2020 3c63 633a 4167  tor>.     <cc:Ag
 000002a0: 656e 743e 0a20 2020 2020 203c 6463 3a74  ent>.      <dc:t
 000002b0: 6974 6c65 3e4d 6174 706c 6f74 6c69 6220  itle>Matplotlib
```

### Comparing `bioregistry-0.9.90/docs/img/bioregistry_coverage_bar_short.svg` & `bioregistry-0.9.91/docs/img/bioregistry_coverage_bar_short.svg`

 * *Files 0% similar despite different names*

```diff
@@ -29,16 +29,16 @@
 000001c0: 3032 2f32 322d 7264 662d 7379 6e74 6178  02/22-rdf-syntax
 000001d0: 2d6e 7323 223e 0a20 2020 3c63 633a 576f  -ns#">.   <cc:Wo
 000001e0: 726b 3e0a 2020 2020 3c64 633a 7479 7065  rk>.    <dc:type
 000001f0: 2072 6466 3a72 6573 6f75 7263 653d 2268   rdf:resource="h
 00000200: 7474 703a 2f2f 7075 726c 2e6f 7267 2f64  ttp://purl.org/d
 00000210: 632f 6463 6d69 7479 7065 2f53 7469 6c6c  c/dcmitype/Still
 00000220: 496d 6167 6522 2f3e 0a20 2020 203c 6463  Image"/>.    <dc
-00000230: 3a64 6174 653e 3230 3233 2d30 372d 3330  :date>2023-07-30
-00000240: 5430 313a 3136 3a33 342e 3632 3530 3439  T01:16:34.625049
+00000230: 3a64 6174 653e 3230 3233 2d30 382d 3031  :date>2023-08-01
+00000240: 5430 313a 3230 3a31 332e 3634 3738 3637  T01:20:13.647867
 00000250: 3c2f 6463 3a64 6174 653e 0a20 2020 203c  </dc:date>.    <
 00000260: 6463 3a66 6f72 6d61 743e 696d 6167 652f  dc:format>image/
 00000270: 7376 672b 786d 6c3c 2f64 633a 666f 726d  svg+xml</dc:form
 00000280: 6174 3e0a 2020 2020 3c64 633a 6372 6561  at>.    <dc:crea
 00000290: 746f 723e 0a20 2020 2020 3c63 633a 4167  tor>.     <cc:Ag
 000002a0: 656e 743e 0a20 2020 2020 203c 6463 3a74  ent>.      <dc:t
 000002b0: 6974 6c65 3e4d 6174 706c 6f74 6c69 6220  itle>Matplotlib
```

### Comparing `bioregistry-0.9.90/docs/img/datamodel_umls.svg` & `bioregistry-0.9.91/docs/img/datamodel_umls.svg`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/docs/img/external_overlap.svg` & `bioregistry-0.9.91/docs/img/external_overlap.svg`

 * *Files 0% similar despite different names*

```diff
@@ -30,16 +30,16 @@
 000001d0: 6e74 6178 2d6e 7323 223e 0a20 2020 3c63  ntax-ns#">.   <c
 000001e0: 633a 576f 726b 3e0a 2020 2020 3c64 633a  c:Work>.    <dc:
 000001f0: 7479 7065 2072 6466 3a72 6573 6f75 7263  type rdf:resourc
 00000200: 653d 2268 7474 703a 2f2f 7075 726c 2e6f  e="http://purl.o
 00000210: 7267 2f64 632f 6463 6d69 7479 7065 2f53  rg/dc/dcmitype/S
 00000220: 7469 6c6c 496d 6167 6522 2f3e 0a20 2020  tillImage"/>.   
 00000230: 203c 6463 3a64 6174 653e 3230 3233 2d30   <dc:date>2023-0
-00000240: 372d 3330 5430 313a 3230 3a31 382e 3539  7-30T01:20:18.59
-00000250: 3030 3134 3c2f 6463 3a64 6174 653e 0a20  0014</dc:date>. 
+00000240: 382d 3031 5430 313a 3234 3a35 322e 3237  8-01T01:24:52.27
+00000250: 3135 3632 3c2f 6463 3a64 6174 653e 0a20  1562</dc:date>. 
 00000260: 2020 203c 6463 3a66 6f72 6d61 743e 696d     <dc:format>im
 00000270: 6167 652f 7376 672b 786d 6c3c 2f64 633a  age/svg+xml</dc:
 00000280: 666f 726d 6174 3e0a 2020 2020 3c64 633a  format>.    <dc:
 00000290: 6372 6561 746f 723e 0a20 2020 2020 3c63  creator>.     <c
 000002a0: 633a 4167 656e 743e 0a20 2020 2020 203c  c:Agent>.      <
 000002b0: 6463 3a74 6974 6c65 3e4d 6174 706c 6f74  dc:title>Matplot
 000002c0: 6c69 6220 7633 2e37 2e32 2c20 6874 7470  lib v3.7.2, http
@@ -39985,16 +39985,16 @@
 0009c300: 3836 382e 3538 3337 3820 3536 332e 3831  868.58378 563.81
 0009c310: 3630 3337 2035 3836 342e 3130 3437 3534  6037 5864.104754
 0009c320: 200a 4320 3536 362e 3034 3636 3831 2035   .C 566.046681 5
 0009c330: 3835 392e 3632 3537 3237 2035 3637 2e33  859.625727 567.3
 0009c340: 3936 3835 3320 3538 3534 2e37 3539 3833  96853 5854.75983
 0009c350: 3220 3536 372e 3739 3330 3936 2035 3834  2 567.793096 584
 0009c360: 392e 3737 3138 3032 200a 2220 636c 6970  9.771802 ." clip
-0009c370: 2d70 6174 683d 2275 726c 2823 7031 3835  -path="url(#p185
-0009c380: 6433 3662 6636 6129 2220 7374 796c 653d  d36bf6a)" style=
+0009c370: 2d70 6174 683d 2275 726c 2823 7065 3533  -path="url(#pe53
+0009c380: 6532 6337 3934 3329 2220 7374 796c 653d  e2c7943)" style=
 0009c390: 2266 696c 6c3a 2023 6664 6266 3666 3b20  "fill: #fdbf6f; 
 0009c3a0: 6f70 6163 6974 793a 2030 2e34 222f 3e0a  opacity: 0.4"/>.
 0009c3b0: 2020 203c 2f67 3e0a 2020 203c 6720 6964     </g>.   <g id
 0009c3c0: 3d22 7061 7463 685f 3338 3422 3e0a 2020  ="patch_384">.  
 0009c3d0: 2020 3c70 6174 6820 643d 224d 2035 3637    <path d="M 567
 0009c3e0: 2e37 3933 3039 3620 3538 3433 2e35 3739  .793096 5843.579
 0009c3f0: 3738 3220 0a43 2035 3637 2e38 3734 3931  782 .C 567.87491
@@ -40074,16 +40074,16 @@
 0009c890: 312e 3935 3932 3634 2035 3638 2e39 3838  1.959264 568.988
 0009c8a0: 3537 3420 3538 3432 2e33 3134 3737 3520  574 5842.314775 
 0009c8b0: 0a43 2035 3638 2e35 3235 3434 3320 3538  .C 568.525443 58
 0009c8c0: 3432 2e36 3730 3238 3620 3536 382e 3132  42.670286 568.12
 0009c8d0: 3138 3836 2035 3834 332e 3039 3733 3133  1886 5843.097313
 0009c8e0: 2035 3637 2e37 3933 3039 3620 3538 3433   567.793096 5843
 0009c8f0: 2e35 3739 3738 3220 0a22 2063 6c69 702d  .579782 ." clip-
-0009c900: 7061 7468 3d22 7572 6c28 2370 3138 3564  path="url(#p185d
-0009c910: 3336 6266 3661 2922 2073 7479 6c65 3d22  36bf6a)" style="
+0009c900: 7061 7468 3d22 7572 6c28 2370 6535 3365  path="url(#pe53e
+0009c910: 3263 3739 3433 2922 2073 7479 6c65 3d22  2c7943)" style="
 0009c920: 6669 6c6c 3a20 2362 3264 6638 613b 206f  fill: #b2df8a; o
 0009c930: 7061 6369 7479 3a20 302e 3422 2f3e 0a20  pacity: 0.4"/>. 
 0009c940: 2020 3c2f 673e 0a20 2020 3c67 2069 643d    </g>.   <g id=
 0009c950: 2270 6174 6368 5f33 3835 223e 0a20 2020  "patch_385">.   
 0009c960: 203c 7061 7468 2064 3d22 4d20 3536 372e   <path d="M 567.
 0009c970: 3739 3330 3936 2035 3834 392e 3737 3138  793096 5849.7718
 0009c980: 3032 200a 4320 3536 372e 3837 3439 3136  02 .C 567.874916
@@ -40101,16 +40101,16 @@
 0009ca40: 3436 3820 3536 362e 3833 3834 3636 2035  468 566.838466 5
 0009ca50: 3834 362e 3637 3537 3932 200a 4320 3536  846.675792 .C 56
 0009ca60: 362e 3833 3834 3636 2035 3834 372e 3738  6.838466 5847.78
 0009ca70: 3031 3137 2035 3637 2e31 3731 3230 3320  0117 567.171203 
 0009ca80: 3538 3438 2e38 3539 3233 3420 3536 372e  5848.859234 567.
 0009ca90: 3739 3330 3936 2035 3834 392e 3737 3138  793096 5849.7718
 0009caa0: 3032 200a 2220 636c 6970 2d70 6174 683d  02 ." clip-path=
-0009cab0: 2275 726c 2823 7031 3835 6433 3662 6636  "url(#p185d36bf6
-0009cac0: 6129 2220 7374 796c 653d 2266 696c 6c3a  a)" style="fill:
+0009cab0: 2275 726c 2823 7065 3533 6532 6337 3934  "url(#pe53e2c794
+0009cac0: 3329 2220 7374 796c 653d 2266 696c 6c3a  3)" style="fill:
 0009cad0: 2023 6666 6666 6165 3b20 6f70 6163 6974   #ffffae; opacit
 0009cae0: 793a 2030 2e34 222f 3e0a 2020 203c 2f67  y: 0.4"/>.   </g
 0009caf0: 3e0a 2020 203c 6720 6964 3d22 7465 7874  >.   <g id="text
 0009cb00: 5f36 3432 223e 0a20 2020 203c 212d 2d20  _642">.    <!-- 
 0009cb10: 3130 3631 202d 2d3e 0a20 2020 203c 6720  1061 -->.    <g 
 0009cb20: 7374 796c 653d 2266 696c 6c3a 2023 3236  style="fill: #26
 0009cb30: 3236 3236 2220 7472 616e 7366 6f72 6d3d  2626" transform=
@@ -60566,15 +60566,15 @@
 000ec950: 3520 3238 392e 3630 3530 3934 2038 3930  5 289.605094 890
 000ec960: 392e 3331 3230 3234 200a 4320 3239 302e  9.312024 .C 290.
 000ec970: 3136 3031 3933 2038 3930 382e 3731 3135  160193 8908.7115
 000ec980: 3433 2032 3930 2e36 3038 3230 3820 3839  43 290.608208 89
 000ec990: 3038 2e30 3230 3237 3720 3239 302e 3932  08.020277 290.92
 000ec9a0: 3936 3220 3839 3037 2e32 3638 3334 3120  962 8907.268341 
 000ec9b0: 0a22 2063 6c69 702d 7061 7468 3d22 7572  ." clip-path="ur
-000ec9c0: 6c28 2370 3236 3839 3631 6135 3562 2922  l(#p268961a55b)"
+000ec9c0: 6c28 2370 3133 3136 3739 6264 3634 2922  l(#p131679bd64)"
 000ec9d0: 2073 7479 6c65 3d22 6669 6c6c 3a20 2366   style="fill: #f
 000ec9e0: 6666 6639 393b 206f 7061 6369 7479 3a20  fff99; opacity: 
 000ec9f0: 302e 3422 2f3e 0a20 2020 3c2f 673e 0a20  0.4"/>.   </g>. 
 000eca00: 2020 3c67 2069 643d 2270 6174 6368 5f35    <g id="patch_5
 000eca10: 3737 223e 0a20 2020 203c 7061 7468 2064  77">.    <path d
 000eca20: 3d22 4d20 3239 302e 3932 3936 3220 3839  ="M 290.92962 89
 000eca30: 3031 2e36 3436 3632 3220 0a43 2032 3931  01.646622 .C 291
@@ -60654,16 +60654,16 @@
 000eced0: 3838 332e 3536 3539 3120 3239 342e 3639  883.56591 294.69
 000ecee0: 3330 3135 2038 3838 372e 3837 3333 3239  3015 8887.873329
 000ecef0: 200a 4320 3239 322e 3536 3735 3634 2038   .C 292.567564 8
 000ecf00: 3839 322e 3138 3037 3439 2032 3931 2e32  892.180749 291.2
 000ecf10: 3839 3836 3420 3838 3936 2e38 3536 3838  89864 8896.85688
 000ecf20: 2032 3930 2e39 3239 3632 2038 3930 312e   290.92962 8901.
 000ecf30: 3634 3636 3232 200a 2220 636c 6970 2d70  646622 ." clip-p
-000ecf40: 6174 683d 2275 726c 2823 7032 3638 3936  ath="url(#p26896
-000ecf50: 3161 3535 6229 2220 7374 796c 653d 2266  1a55b)" style="f
+000ecf40: 6174 683d 2275 726c 2823 7031 3331 3637  ath="url(#p13167
+000ecf50: 3962 6436 3429 2220 7374 796c 653d 2266  9bd64)" style="f
 000ecf60: 696c 6c3a 2023 6231 3539 3238 3b20 6f70  ill: #b15928; op
 000ecf70: 6163 6974 793a 2030 2e34 222f 3e0a 2020  acity: 0.4"/>.  
 000ecf80: 203c 2f67 3e0a 2020 203c 6720 6964 3d22   </g>.   <g id="
 000ecf90: 7061 7463 685f 3537 3822 3e0a 2020 2020  patch_578">.    
 000ecfa0: 3c70 6174 6820 643d 224d 2032 3930 2e39  <path d="M 290.9
 000ecfb0: 3239 3632 2038 3930 372e 3236 3833 3431  2962 8907.268341
 000ecfc0: 200a 4320 3239 312e 3330 3933 3735 2038   .C 291.309375 8
@@ -60681,15 +60681,15 @@
 000ed080: 3239 302e 3832 3430 3634 2038 3930 342e  290.824064 8904.
 000ed090: 3435 3734 3832 200a 4320 3239 302e 3832  457482 .C 290.82
 000ed0a0: 3430 3634 2038 3930 352e 3339 3534 3236  4064 8905.395426
 000ed0b0: 2032 3930 2e38 3539 3237 3420 3839 3036   290.859274 8906
 000ed0c0: 2e33 3333 3033 3920 3239 302e 3932 3936  .333039 290.9296
 000ed0d0: 3220 3839 3037 2e32 3638 3334 3120 0a22  2 8907.268341 ."
 000ed0e0: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-000ed0f0: 2370 3236 3839 3631 6135 3562 2922 2073  #p268961a55b)" s
+000ed0f0: 2370 3133 3136 3739 6264 3634 2922 2073  #p131679bd64)" s
 000ed100: 7479 6c65 3d22 6669 6c6c 3a20 2366 6666  tyle="fill: #fff
 000ed110: 3138 373b 206f 7061 6369 7479 3a20 302e  187; opacity: 0.
 000ed120: 3422 2f3e 0a20 2020 3c2f 673e 0a20 2020  4"/>.   </g>.   
 000ed130: 3c67 2069 643d 2274 6578 745f 3936 3922  <g id="text_969"
 000ed140: 3e0a 2020 2020 3c21 2d2d 2031 3233 202d  >.    <!-- 123 -
 000ed150: 2d3e 0a20 2020 203c 6720 7374 796c 653d  ->.    <g style=
 000ed160: 2266 696c 6c3a 2023 3236 3236 3236 2220  "fill: #262626" 
@@ -64101,16 +64101,16 @@
 000fa640: 3235 3720 3239 312e 3531 3632 3431 2039  257 291.516241 9
 000fa650: 3434 392e 3733 3533 3737 200a 4320 3239  449.735377 .C 29
 000fa660: 322e 3136 3339 3639 2039 3434 392e 3033  2.163969 9449.03
 000fa670: 3734 3937 2032 3932 2e36 3837 3333 3220  7497 292.687332 
 000fa680: 3934 3438 2e32 3333 3734 3520 3239 332e  9448.233745 293.
 000fa690: 3036 3335 3637 2039 3434 372e 3335 3930  063567 9447.3590
 000fa6a0: 3832 200a 2220 636c 6970 2d70 6174 683d  82 ." clip-path=
-000fa6b0: 2275 726c 2823 7030 3837 3530 3163 3533  "url(#p087501c53
-000fa6c0: 3829 2220 7374 796c 653d 2266 696c 6c3a  8)" style="fill:
+000fa6b0: 2275 726c 2823 7030 6639 6161 6365 6136  "url(#p0f9aacea6
+000fa6c0: 6129 2220 7374 796c 653d 2266 696c 6c3a  a)" style="fill:
 000fa6d0: 2023 6231 3539 3238 3b20 6f70 6163 6974   #b15928; opacit
 000fa6e0: 793a 2030 2e34 222f 3e0a 2020 203c 2f67  y: 0.4"/>.   </g
 000fa6f0: 3e0a 2020 203c 6720 6964 3d22 7061 7463  >.   <g id="patc
 000fa700: 685f 3631 3122 3e0a 2020 2020 3c70 6174  h_611">.    <pat
 000fa710: 6820 643d 224d 2032 3933 2e30 3633 3536  h d="M 293.06356
 000fa720: 3720 3934 3430 2e37 3732 3934 3920 0a43  7 9440.772949 .C
 000fa730: 2032 3933 2e35 3131 3030 3120 3934 3431   293.511001 9441
@@ -64190,15 +64190,15 @@
 000fabd0: 3936 2e39 3031 3231 3320 3934 3237 2e34  96.901213 9427.4
 000fabe0: 3739 3930 3320 0a43 2032 3934 2e37 3836  79903 .C 294.786
 000fabf0: 3136 3720 3934 3331 2e36 3235 3630 3920  167 9431.625609 
 000fac00: 3239 332e 3438 3334 3936 2039 3433 362e  293.483496 9436.
 000fac10: 3133 3738 3720 3239 332e 3036 3335 3637  13787 293.063567
 000fac20: 2039 3434 302e 3737 3239 3439 200a 2220   9440.772949 ." 
 000fac30: 636c 6970 2d70 6174 683d 2275 726c 2823  clip-path="url(#
-000fac40: 7030 3837 3530 3163 3533 3829 2220 7374  p087501c538)" st
+000fac40: 7030 6639 6161 6365 6136 6129 2220 7374  p0f9aacea6a)" st
 000fac50: 796c 653d 2266 696c 6c3a 2023 6231 3539  yle="fill: #b159
 000fac60: 3238 3b20 6f70 6163 6974 793a 2030 2e34  28; opacity: 0.4
 000fac70: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 000fac80: 6720 6964 3d22 7061 7463 685f 3631 3222  g id="patch_612"
 000fac90: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 000faca0: 2032 3933 2e30 3633 3536 3720 3934 3437   293.063567 9447
 000facb0: 2e33 3539 3038 3220 0a43 2032 3933 2e35  .359082 .C 293.5
@@ -64216,16 +64216,16 @@
 000fad70: 3936 3636 3435 2032 3932 2e39 3134 3639  966645 292.91469
 000fad80: 3920 3934 3434 2e30 3636 3031 3520 0a43  9 9444.066015 .C
 000fad90: 2032 3932 2e39 3134 3639 3920 3934 3435   292.914699 9445
 000fada0: 2e31 3635 3338 3620 3239 322e 3936 3433  .165386 292.9643
 000fadb0: 3733 2039 3434 362e 3236 3431 3935 2032  73 9446.264195 2
 000fadc0: 3933 2e30 3633 3536 3720 3934 3437 2e33  93.063567 9447.3
 000fadd0: 3539 3038 3220 0a22 2063 6c69 702d 7061  59082 ." clip-pa
-000fade0: 7468 3d22 7572 6c28 2370 3038 3735 3031  th="url(#p087501
-000fadf0: 6335 3338 2922 2073 7479 6c65 3d22 6669  c538)" style="fi
+000fade0: 7468 3d22 7572 6c28 2370 3066 3961 6163  th="url(#p0f9aac
+000fadf0: 6561 3661 2922 2073 7479 6c65 3d22 6669  ea6a)" style="fi
 000fae00: 6c6c 3a20 2366 3837 6433 383b 206f 7061  ll: #f87d38; opa
 000fae10: 6369 7479 3a20 302e 3422 2f3e 0a20 2020  city: 0.4"/>.   
 000fae20: 3c2f 673e 0a20 2020 3c67 2069 643d 2274  </g>.   <g id="t
 000fae30: 6578 745f 3130 3237 223e 0a20 2020 203c  ext_1027">.    <
 000fae40: 212d 2d20 3137 3620 2d2d 3e0a 2020 2020  !-- 176 -->.    
 000fae50: 3c67 2073 7479 6c65 3d22 6669 6c6c 3a20  <g style="fill: 
 000fae60: 2332 3632 3632 3622 2074 7261 6e73 666f  #262626" transfo
@@ -72397,16 +72397,16 @@
 0011acc0: 3839 3539 3820 3238 302e 3330 3239 3935  89598 280.302995
 0011acd0: 2031 3037 3036 2e34 3238 3432 3720 0a43   10706.428427 .C
 0011ace0: 2032 3830 2e37 3731 3639 3220 3130 3730   280.771692 1070
 0011acf0: 362e 3536 3732 3536 2032 3831 2e32 3634  6.567256 281.264
 0011ad00: 3737 3120 3130 3730 362e 3630 3338 3932  771 10706.603892
 0011ad10: 2032 3831 2e37 3438 3833 3820 3130 3730   281.748838 1070
 0011ad20: 362e 3533 3538 3534 200a 2220 636c 6970  6.535854 ." clip
-0011ad30: 2d70 6174 683d 2275 726c 2823 7035 3865  -path="url(#p58e
-0011ad40: 3839 6333 3862 3129 2220 7374 796c 653d  89c38b1)" style=
+0011ad30: 2d70 6174 683d 2275 726c 2823 7063 6239  -path="url(#pcb9
+0011ad40: 6161 3337 3931 6229 2220 7374 796c 653d  aa3791b)" style=
 0011ad50: 2266 696c 6c3a 2023 6232 6466 3861 3b20  "fill: #b2df8a; 
 0011ad60: 6f70 6163 6974 793a 2030 2e34 222f 3e0a  opacity: 0.4"/>.
 0011ad70: 2020 203c 2f67 3e0a 2020 203c 6720 6964     </g>.   <g id
 0011ad80: 3d22 7061 7463 685f 3638 3922 3e0a 2020  ="patch_689">.  
 0011ad90: 2020 3c70 6174 6820 643d 224d 2032 3831    <path d="M 281
 0011ada0: 2e37 3438 3833 3820 3130 3639 392e 3736  .748838 10699.76
 0011adb0: 3933 3332 200a 4320 3238 322e 3536 3433  9332 .C 282.5643
@@ -72499,16 +72499,16 @@
 0011b320: 3030 3139 3320 3238 362e 3032 3034 3139  00193 286.020419
 0011b330: 2031 3036 3834 2e34 3737 3332 3720 0a43   10684.477327 .C
 0011b340: 2032 3833 2e36 3331 3731 3120 3130 3638   283.631711 1068
 0011b350: 392e 3235 3434 3631 2032 3832 2e31 3831  9.254461 282.181
 0011b360: 3538 3320 3130 3639 342e 3434 3538 3331  583 10694.445831
 0011b370: 2032 3831 2e37 3438 3833 3820 3130 3639   281.748838 1069
 0011b380: 392e 3736 3933 3332 200a 2220 636c 6970  9.769332 ." clip
-0011b390: 2d70 6174 683d 2275 726c 2823 7035 3865  -path="url(#p58e
-0011b3a0: 3839 6333 3862 3129 2220 7374 796c 653d  89c38b1)" style=
+0011b390: 2d70 6174 683d 2275 726c 2823 7063 6239  -path="url(#pcb9
+0011b3a0: 6161 3337 3931 6229 2220 7374 796c 653d  aa3791b)" style=
 0011b3b0: 2266 696c 6c3a 2023 3661 3364 3961 3b20  "fill: #6a3d9a; 
 0011b3c0: 6f70 6163 6974 793a 2030 2e34 222f 3e0a  opacity: 0.4"/>.
 0011b3d0: 2020 203c 2f67 3e0a 2020 203c 6720 6964     </g>.   <g id
 0011b3e0: 3d22 7061 7463 685f 3639 3022 3e0a 2020  ="patch_690">.  
 0011b3f0: 2020 3c70 6174 6820 643d 224d 2032 3831    <path d="M 281
 0011b400: 2e37 3438 3833 3820 3130 3730 362e 3533  .748838 10706.53
 0011b410: 3538 3534 200a 4320 3238 322e 3536 3433  5854 .C 282.5643
@@ -72536,16 +72536,16 @@
 0011b570: 3720 3238 312e 3631 3135 3533 2031 3037  7 281.611553 107
 0011b580: 3033 2e31 3532 3539 3320 0a43 2032 3831  03.152593 .C 281
 0011b590: 2e36 3131 3535 3320 3130 3730 342e 3238  .611553 10704.28
 0011b5a0: 3137 3339 2032 3831 2e36 3537 3335 3320  1739 281.657353 
 0011b5b0: 3130 3730 352e 3431 3034 3231 2032 3831  10705.410421 281
 0011b5c0: 2e37 3438 3833 3820 3130 3730 362e 3533  .748838 10706.53
 0011b5d0: 3538 3534 200a 2220 636c 6970 2d70 6174  5854 ." clip-pat
-0011b5e0: 683d 2275 726c 2823 7035 3865 3839 6333  h="url(#p58e89c3
-0011b5f0: 3862 3129 2220 7374 796c 653d 2266 696c  8b1)" style="fil
+0011b5e0: 683d 2275 726c 2823 7063 6239 6161 3337  h="url(#pcb9aa37
+0011b5f0: 3931 6229 2220 7374 796c 653d 2266 696c  91b)" style="fil
 0011b600: 6c3a 2023 6337 6337 6363 3b20 6f70 6163  l: #c7c7cc; opac
 0011b610: 6974 793a 2030 2e34 222f 3e0a 2020 203c  ity: 0.4"/>.   <
 0011b620: 2f67 3e0a 2020 203c 6720 6964 3d22 7465  /g>.   <g id="te
 0011b630: 7874 5f31 3136 3122 3e0a 2020 2020 3c21  xt_1161">.    <!
 0011b640: 2d2d 2031 3220 2d2d 3e0a 2020 2020 3c67  -- 12 -->.    <g
 0011b650: 2073 7479 6c65 3d22 6669 6c6c 3a20 2332   style="fill: #2
 0011b660: 3632 3632 3622 2074 7261 6e73 666f 726d  62626" transform
@@ -73055,15 +73055,15 @@
 0011d5e0: 3034 3433 3831 2031 3037 3035 2e39 3737  044381 10705.977
 0011d5f0: 3633 3320 0a43 2037 3037 2e33 3232 3735  633 .C 707.32275
 0011d600: 3220 3130 3730 352e 3832 3734 3532 2037  2 10705.827452 7
 0011d610: 3037 2e35 3737 3430 3420 3130 3730 352e  07.577404 10705.
 0011d620: 3633 3639 3437 2037 3037 2e38 3030 3036  636947 707.80006
 0011d630: 3920 3130 3730 352e 3431 3233 3032 200a  9 10705.412302 .
 0011d640: 2220 636c 6970 2d70 6174 683d 2275 726c  " clip-path="url
-0011d650: 2823 7030 3736 3165 3063 6233 3629 2220  (#p0761e0cb36)" 
+0011d650: 2823 7061 6134 6332 6534 6137 3729 2220  (#paa4c2e4a77)" 
 0011d660: 7374 796c 653d 2266 696c 6c3a 2023 6232  style="fill: #b2
 0011d670: 6466 3861 3b20 6f70 6163 6974 793a 2030  df8a; opacity: 0
 0011d680: 2e34 222f 3e0a 2020 203c 2f67 3e0a 2020  .4"/>.   </g>.  
 0011d690: 203c 6720 6964 3d22 7061 7463 685f 3639   <g id="patch_69
 0011d6a0: 3522 3e0a 2020 2020 3c70 6174 6820 643d  5">.    <path d=
 0011d6b0: 224d 2037 3037 2e38 3030 3036 3920 3130  "M 707.800069 10
 0011d6c0: 3730 302e 3839 3238 3835 200a 4320 3730  700.892885 .C 70
@@ -73147,16 +73147,16 @@
 0011dba0: 3736 2037 3131 2e36 3632 3731 3320 3130  76 711.662713 10
 0011dbb0: 3638 352e 3633 3739 3131 200a 4320 3730  685.637911 .C 70
 0011dbc0: 392e 3430 3332 3835 2031 3036 3930 2e34  9.403285 10690.4
 0011dbd0: 3236 3034 3720 3730 382e 3039 3135 3736  26047 708.091576
 0011dbe0: 2031 3036 3935 2e36 3036 3436 3120 3730   10695.606461 70
 0011dbf0: 372e 3830 3030 3639 2031 3037 3030 2e38  7.800069 10700.8
 0011dc00: 3932 3838 3520 0a22 2063 6c69 702d 7061  92885 ." clip-pa
-0011dc10: 7468 3d22 7572 6c28 2370 3037 3631 6530  th="url(#p0761e0
-0011dc20: 6362 3336 2922 2073 7479 6c65 3d22 6669  cb36)" style="fi
+0011dc10: 7468 3d22 7572 6c28 2370 6161 3463 3265  th="url(#paa4c2e
+0011dc20: 3461 3737 2922 2073 7479 6c65 3d22 6669  4a77)" style="fi
 0011dc30: 6c6c 3a20 2362 3135 3932 383b 206f 7061  ll: #b15928; opa
 0011dc40: 6369 7479 3a20 302e 3422 2f3e 0a20 2020  city: 0.4"/>.   
 0011dc50: 3c2f 673e 0a20 2020 3c67 2069 643d 2270  </g>.   <g id="p
 0011dc60: 6174 6368 5f36 3936 223e 0a20 2020 203c  atch_696">.    <
 0011dc70: 7061 7468 2064 3d22 4d20 3730 372e 3830  path d="M 707.80
 0011dc80: 3030 3639 2031 3037 3035 2e34 3132 3330  0069 10705.41230
 0011dc90: 3220 0a43 2037 3038 2e33 3935 3639 3320  2 .C 708.395693 
@@ -73175,15 +73175,15 @@
 0011dd60: 3337 3831 3320 3130 3730 332e 3135 3235  37813 10703.1525
 0011dd70: 3933 200a 4320 3730 372e 3733 3738 3133  93 .C 707.737813
 0011dd80: 2031 3037 3033 2e39 3036 3235 3820 3730   10703.906258 70
 0011dd90: 372e 3735 3835 3733 2031 3037 3034 2e36  7.758573 10704.6
 0011dda0: 3539 3738 2037 3037 2e38 3030 3036 3920  5978 707.800069 
 0011ddb0: 3130 3730 352e 3431 3233 3032 200a 2220  10705.412302 ." 
 0011ddc0: 636c 6970 2d70 6174 683d 2275 726c 2823  clip-path="url(#
-0011ddd0: 7030 3736 3165 3063 6233 3629 2220 7374  p0761e0cb36)" st
+0011ddd0: 7061 6134 6332 6534 6137 3729 2220 7374  paa4c2e4a77)" st
 0011dde0: 796c 653d 2266 696c 6c3a 2023 6638 6461  yle="fill: #f8da
 0011ddf0: 3764 3b20 6f70 6163 6974 793a 2030 2e34  7d; opacity: 0.4
 0011de00: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 0011de10: 6720 6964 3d22 7465 7874 5f31 3137 3122  g id="text_1171"
 0011de20: 3e0a 2020 2020 3c21 2d2d 2031 3920 2d2d  >.    <!-- 19 --
 0011de30: 3e0a 2020 2020 3c67 2073 7479 6c65 3d22  >.    <g style="
 0011de40: 6669 6c6c 3a20 2332 3632 3632 3622 2074  fill: #262626" t
@@ -78044,15 +78044,15 @@
 00130db0: 3032 2e36 3533 3836 3920 3131 3432 382e  02.653869 11428.
 00130dc0: 3238 3839 3639 200a 4320 3530 332e 3238  288969 .C 503.28
 00130dd0: 3338 3434 2031 3134 3237 2e36 3837 3935  3844 11427.68795
 00130de0: 3720 3530 332e 3830 3930 3438 2031 3134  7 503.809048 114
 00130df0: 3236 2e39 3835 3933 3520 3530 342e 3230  26.985935 504.20
 00130e00: 3737 3733 2031 3134 3236 2e32 3131 3931  7773 11426.21191
 00130e10: 3720 0a22 2063 6c69 702d 7061 7468 3d22  7 ." clip-path="
-00130e20: 7572 6c28 2370 3236 6662 6265 3039 6565  url(#p26fbbe09ee
+00130e20: 7572 6c28 2370 3636 3030 3730 6233 3266  url(#p660070b32f
 00130e30: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00130e40: 2366 6239 6139 393b 206f 7061 6369 7479  #fb9a99; opacity
 00130e50: 3a20 302e 3422 2f3e 0a20 2020 3c2f 673e  : 0.4"/>.   </g>
 00130e60: 0a20 2020 3c67 2069 643d 2270 6174 6368  .   <g id="patch
 00130e70: 5f37 3430 223e 0a20 2020 203c 7061 7468  _740">.    <path
 00130e80: 2064 3d22 4d20 3530 342e 3230 3737 3733   d="M 504.207773
 00130e90: 2031 3134 3139 2e30 3439 3335 3920 0a43   11419.049359 .C
@@ -78137,15 +78137,15 @@
 00131380: 3634 3234 2031 3134 3035 2e36 3039 3331  6424 11405.60931
 00131390: 3620 0a43 2035 3036 2e30 3130 3531 3520  6 .C 506.010515 
 001313a0: 3131 3430 392e 3739 3539 3334 2035 3034  11409.795934 504
 001313b0: 2e36 3633 3437 3920 3131 3431 342e 3335  .663479 11414.35
 001313c0: 3737 3432 2035 3034 2e32 3037 3737 3320  7742 504.207773 
 001313d0: 3131 3431 392e 3034 3933 3539 200a 2220  11419.049359 ." 
 001313e0: 636c 6970 2d70 6174 683d 2275 726c 2823  clip-path="url(#
-001313f0: 7032 3666 6262 6530 3965 6529 2220 7374  p26fbbe09ee)" st
+001313f0: 7036 3630 3037 3062 3332 6629 2220 7374  p660070b32f)" st
 00131400: 796c 653d 2266 696c 6c3a 2023 6231 3539  yle="fill: #b159
 00131410: 3238 3b20 6f70 6163 6974 793a 2030 2e34  28; opacity: 0.4
 00131420: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 00131430: 6720 6964 3d22 7061 7463 685f 3734 3122  g id="patch_741"
 00131440: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 00131450: 2035 3034 2e32 3037 3737 3320 3131 3432   504.207773 1142
 00131460: 362e 3231 3139 3137 200a 4320 3530 342e  6.211917 .C 504.
@@ -78164,16 +78164,16 @@
 00131530: 3720 3530 342e 3033 3432 3533 2031 3134  7 504.034253 114
 00131540: 3232 2e36 3330 3633 3820 0a43 2035 3034  22.630638 .C 504
 00131550: 2e30 3334 3235 3320 3131 3432 332e 3832  .034253 11423.82
 00131560: 3634 3939 2035 3034 2e30 3932 3136 3120  6499 504.092161 
 00131570: 3131 3432 352e 3032 3136 3538 2035 3034  11425.021658 504
 00131580: 2e32 3037 3737 3320 3131 3432 362e 3231  .207773 11426.21
 00131590: 3139 3137 200a 2220 636c 6970 2d70 6174  1917 ." clip-pat
-001315a0: 683d 2275 726c 2823 7032 3666 6262 6530  h="url(#p26fbbe0
-001315b0: 3965 6529 2220 7374 796c 653d 2266 696c  9ee)" style="fil
+001315a0: 683d 2275 726c 2823 7036 3630 3037 3062  h="url(#p660070b
+001315b0: 3332 6629 2220 7374 796c 653d 2266 696c  32f)" style="fil
 001315c0: 6c3a 2023 6666 6161 3837 3b20 6f70 6163  l: #ffaa87; opac
 001315d0: 6974 793a 2030 2e34 222f 3e0a 2020 203c  ity: 0.4"/>.   <
 001315e0: 2f67 3e0a 2020 203c 6720 6964 3d22 7465  /g>.   <g id="te
 001315f0: 7874 5f31 3234 3622 3e0a 2020 2020 3c21  xt_1246">.    <!
 00131600: 2d2d 2031 3439 202d 2d3e 0a20 2020 203c  -- 149 -->.    <
 00131610: 6720 7374 796c 653d 2266 696c 6c3a 2023  g style="fill: #
 00131620: 3236 3236 3236 2220 7472 616e 7366 6f72  262626" transfor
@@ -82020,15 +82020,15 @@
 00140630: 3031 2e34 3737 3735 3120 3132 3135 332e  01.477751 12153.
 00140640: 3038 3132 3933 200a 4320 3530 332e 3532  081293 .C 503.52
 00140650: 3630 3632 2031 3231 3532 2e38 3738 3730  6062 12152.87870
 00140660: 3520 3530 352e 3437 3732 3936 2031 3231  5 505.477296 121
 00140670: 3532 2e31 3036 3535 3620 3530 372e 3130  52.106556 507.10
 00140680: 3935 3735 2031 3231 3530 2e38 3532 3634  9575 12150.85264
 00140690: 3820 0a22 2063 6c69 702d 7061 7468 3d22  8 ." clip-path="
-001406a0: 7572 6c28 2370 3065 3736 3965 3364 3537  url(#p0e769e3d57
+001406a0: 7572 6c28 2370 6539 6535 3632 3930 3439  url(#pe9e5629049
 001406b0: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 001406c0: 2363 6162 3264 363b 206f 7061 6369 7479  #cab2d6; opacity
 001406d0: 3a20 302e 3422 2f3e 0a20 2020 3c2f 673e  : 0.4"/>.   </g>
 001406e0: 0a20 2020 3c67 2069 643d 2270 6174 6368  .   <g id="patch
 001406f0: 5f37 3739 223e 0a20 2020 203c 7061 7468  _779">.    <path
 00140700: 2064 3d22 4d20 3530 372e 3130 3935 3735   d="M 507.109575
 00140710: 2031 3231 3333 2e33 3634 3731 3720 0a43   12133.364717 .C
@@ -82122,15 +82122,15 @@
 00140c90: 3736 3736 3920 3132 3132 312e 3533 3233  76769 12121.5323
 00140ca0: 3938 200a 4320 3530 392e 3938 3634 3531  98 .C 509.986451
 00140cb0: 2031 3231 3235 2e31 3139 3134 3820 3530   12125.119148 50
 00140cc0: 382e 3136 3738 3434 2031 3231 3239 2e31  8.167844 12129.1
 00140cd0: 3238 3338 3320 3530 372e 3130 3935 3735  28383 507.109575
 00140ce0: 2031 3231 3333 2e33 3634 3731 3720 0a22   12133.364717 ."
 00140cf0: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00140d00: 2370 3065 3736 3965 3364 3537 2922 2073  #p0e769e3d57)" s
+00140d00: 2370 6539 6535 3632 3930 3439 2922 2073  #pe9e5629049)" s
 00140d10: 7479 6c65 3d22 6669 6c6c 3a20 2336 6133  tyle="fill: #6a3
 00140d20: 6439 613b 206f 7061 6369 7479 3a20 302e  d9a; opacity: 0.
 00140d30: 3422 2f3e 0a20 2020 3c2f 673e 0a20 2020  4"/>.   </g>.   
 00140d40: 3c67 2069 643d 2270 6174 6368 5f37 3830  <g id="patch_780
 00140d50: 223e 0a20 2020 203c 7061 7468 2064 3d22  ">.    <path d="
 00140d60: 4d20 3530 372e 3130 3935 3735 2031 3231  M 507.109575 121
 00140d70: 3530 2e38 3532 3634 3820 0a43 2035 3038  50.852648 .C 508
@@ -82159,15 +82159,15 @@
 00140ee0: 3934 3620 3132 3134 322e 3130 3836 3833  946 12142.108683
 00140ef0: 200a 4320 3530 362e 3033 3339 3436 2031   .C 506.033946 1
 00140f00: 3231 3435 2e30 3536 3332 3420 3530 362e  2145.056324 506.
 00140f10: 3339 3531 3834 2031 3231 3437 2e39 3932  395184 12147.992
 00140f20: 3838 3720 3530 372e 3130 3935 3735 2031  887 507.109575 1
 00140f30: 3231 3530 2e38 3532 3634 3820 0a22 2063  2150.852648 ." c
 00140f40: 6c69 702d 7061 7468 3d22 7572 6c28 2370  lip-path="url(#p
-00140f50: 3065 3736 3965 3364 3537 2922 2073 7479  0e769e3d57)" sty
+00140f50: 6539 6535 3632 3930 3439 2922 2073 7479  e9e5629049)" sty
 00140f60: 6c65 3d22 6669 6c6c 3a20 2364 3861 3766  le="fill: #d8a7f
 00140f70: 663b 206f 7061 6369 7479 3a20 302e 3422  f; opacity: 0.4"
 00140f80: 2f3e 0a20 2020 3c2f 673e 0a20 2020 3c67  />.   </g>.   <g
 00140f90: 2069 643d 2274 6578 745f 3133 3137 223e   id="text_1317">
 00140fa0: 0a20 2020 203c 212d 2d20 3234 3320 2d2d  .    <!-- 243 --
 00140fb0: 3e0a 2020 2020 3c67 2073 7479 6c65 3d22  >.    <g style="
 00140fc0: 6669 6c6c 3a20 2332 3632 3632 3622 2074  fill: #262626" t
@@ -82363,15 +82363,15 @@
 00141ba0: 3931 3236 3820 3132 3135 372e 3638 3834  91268 12157.6884
 00141bb0: 3731 200a 4320 3733 382e 3131 3135 3032  71 .C 738.111502
 00141bc0: 2031 3231 3536 2e31 3039 3334 3620 3733   12156.109346 73
 00141bd0: 392e 3535 3833 3832 2031 3231 3534 2e32  9.558382 12154.2
 00141be0: 3536 3136 3120 3734 302e 3637 3330 3820  56161 740.67308 
 00141bf0: 3132 3135 322e 3230 3432 3633 200a 2220  12152.204263 ." 
 00141c00: 636c 6970 2d70 6174 683d 2275 726c 2823  clip-path="url(#
-00141c10: 7032 3536 3763 6537 3561 6129 2220 7374  p2567ce75aa)" st
+00141c10: 7065 3237 6366 3931 3063 6329 2220 7374  pe27cf910cc)" st
 00141c20: 796c 653d 2266 696c 6c3a 2023 6361 6232  yle="fill: #cab2
 00141c30: 6436 3b20 6f70 6163 6974 793a 2030 2e34  d6; opacity: 0.4
 00141c40: 222f 3e0a 2020 203c 2f67 3e0a 2020 203c  "/>.   </g>.   <
 00141c50: 6720 6964 3d22 7061 7463 685f 3738 3222  g id="patch_782"
 00141c60: 3e0a 2020 2020 3c70 6174 6820 643d 224d  >.    <path d="M
 00141c70: 2037 3430 2e36 3733 3038 2031 3231 3332   740.67308 12132
 00141c80: 2e30 3133 3130 3320 0a43 2037 3432 2e33  .013103 .C 742.3
@@ -82455,15 +82455,15 @@
 00142160: 352e 3438 3737 3839 2031 3231 3234 2e34  5.487789 12124.4
 00142170: 3337 3938 3120 0a43 2037 3433 2e34 3630  37981 .C 743.460
 00142180: 3139 3720 3132 3132 362e 3636 3830 3539  197 12126.668059
 00142190: 2037 3431 2e38 3331 3437 3820 3132 3132   741.831478 1212
 001421a0: 392e 3233 3035 3639 2037 3430 2e36 3733  9.230569 740.673
 001421b0: 3038 2031 3231 3332 2e30 3133 3130 3320  08 12132.013103 
 001421c0: 0a22 2063 6c69 702d 7061 7468 3d22 7572  ." clip-path="ur
-001421d0: 6c28 2370 3235 3637 6365 3735 6161 2922  l(#p2567ce75aa)"
+001421d0: 6c28 2370 6532 3763 6639 3130 6363 2922  l(#pe27cf910cc)"
 001421e0: 2073 7479 6c65 3d22 6669 6c6c 3a20 2366   style="fill: #f
 001421f0: 6666 6639 393b 206f 7061 6369 7479 3a20  fff99; opacity: 
 00142200: 302e 3422 2f3e 0a20 2020 3c2f 673e 0a20  0.4"/>.   </g>. 
 00142210: 2020 3c67 2069 643d 2270 6174 6368 5f37    <g id="patch_7
 00142220: 3833 223e 0a20 2020 203c 7061 7468 2064  83">.    <path d
 00142230: 3d22 4d20 3734 302e 3637 3330 3820 3132  ="M 740.67308 12
 00142240: 3135 322e 3230 3432 3633 200a 4320 3734  152.204263 .C 74
@@ -82482,16 +82482,16 @@
 00142310: 3336 2037 3338 2e36 3535 3535 3520 3132  36 738.655555 12
 00142320: 3134 322e 3130 3836 3833 200a 4320 3733  142.108683 .C 73
 00142330: 382e 3635 3535 3535 2031 3231 3435 2e35  8.655555 12145.5
 00142340: 3733 3932 3920 3733 392e 3334 3132 3633  73929 739.341263
 00142350: 2031 3231 3439 2e30 3035 3137 2037 3430   12149.00517 740
 00142360: 2e36 3733 3038 2031 3231 3532 2e32 3034  .67308 12152.204
 00142370: 3236 3320 0a22 2063 6c69 702d 7061 7468  263 ." clip-path
-00142380: 3d22 7572 6c28 2370 3235 3637 6365 3735  ="url(#p2567ce75
-00142390: 6161 2922 2073 7479 6c65 3d22 6669 6c6c  aa)" style="fill
+00142380: 3d22 7572 6c28 2370 6532 3763 6639 3130  ="url(#pe27cf910
+00142390: 6363 2922 2073 7479 6c65 3d22 6669 6c6c  cc)" style="fill
 001423a0: 3a20 2366 6666 6666 663b 206f 7061 6369  : #ffffff; opaci
 001423b0: 7479 3a20 302e 3422 2f3e 0a20 2020 3c2f  ty: 0.4"/>.   </
 001423c0: 673e 0a20 2020 3c67 2069 643d 2274 6578  g>.   <g id="tex
 001423d0: 745f 3133 3232 223e 0a20 2020 203c 212d  t_1322">.    <!-
 001423e0: 2d20 3238 3020 2d2d 3e0a 2020 2020 3c67  - 280 -->.    <g
 001423f0: 2073 7479 6c65 3d22 6669 6c6c 3a20 2332   style="fill: #2
 00142400: 3632 3632 3622 2074 7261 6e73 666f 726d  62626" transform
@@ -82967,16 +82967,16 @@
 00144160: 312e 3634 3739 3638 2033 3338 2e32 3535  1.647968 338.255
 00144170: 3037 3120 3132 3333 382e 3034 3137 3031  071 12338.041701
 00144180: 200a 4320 3334 302e 3237 3337 3233 2031   .C 340.273723 1
 00144190: 3233 3334 2e34 3335 3433 3320 3334 312e  2334.435433 341.
 001441a0: 3539 3436 3231 2031 3233 3330 2e34 3830  594621 12330.480
 001441b0: 3735 3520 3334 322e 3134 3834 3338 2031  755 342.148438 1
 001441c0: 3233 3236 2e33 3835 3232 200a 2220 636c  2326.38522 ." cl
-001441d0: 6970 2d70 6174 683d 2275 726c 2823 7035  ip-path="url(#p5
-001441e0: 6439 3934 3762 3837 6229 2220 7374 796c  d9947b87b)" styl
+001441d0: 6970 2d70 6174 683d 2275 726c 2823 7062  ip-path="url(#pb
+001441e0: 6430 3038 3732 6464 3329 2220 7374 796c  d00872dd3)" styl
 001441f0: 653d 2266 696c 6c3a 2023 3661 3364 3961  e="fill: #6a3d9a
 00144200: 3b20 6f70 6163 6974 793a 2030 2e34 222f  ; opacity: 0.4"/
 00144210: 3e0a 2020 203c 2f67 3e0a 2020 203c 6720  >.   </g>.   <g 
 00144220: 6964 3d22 7061 7463 685f 3738 3822 3e0a  id="patch_788">.
 00144230: 2020 2020 3c70 6174 6820 643d 224d 2033      <path d="M 3
 00144240: 3432 2e31 3438 3433 3820 3132 3331 372e  42.148438 12317.
 00144250: 3537 3131 3638 200a 4320 3334 322e 3334  571168 .C 342.34
@@ -83060,15 +83060,15 @@
 00144730: 3334 3831 3734 2031 3233 3133 2e38 3536  348174 12313.856
 00144740: 3134 3820 0a43 2033 3433 2e34 3034 3734  148 .C 343.40474
 00144750: 3120 3132 3331 342e 3935 3733 3134 2033  1 12314.957314 3
 00144760: 3432 2e36 3630 3334 3520 3132 3331 362e  42.660345 12316.
 00144770: 3231 3434 3837 2033 3432 2e31 3438 3433  214487 342.14843
 00144780: 3820 3132 3331 372e 3537 3131 3638 200a  8 12317.571168 .
 00144790: 2220 636c 6970 2d70 6174 683d 2275 726c  " clip-path="url
-001447a0: 2823 7035 6439 3934 3762 3837 6229 2220  (#p5d9947b87b)" 
+001447a0: 2823 7062 6430 3038 3732 6464 3329 2220  (#pbd00872dd3)" 
 001447b0: 7374 796c 653d 2266 696c 6c3a 2023 6666  style="fill: #ff
 001447c0: 6666 3939 3b20 6f70 6163 6974 793a 2030  ff99; opacity: 0
 001447d0: 2e34 222f 3e0a 2020 203c 2f67 3e0a 2020  .4"/>.   </g>.  
 001447e0: 203c 6720 6964 3d22 7061 7463 685f 3738   <g id="patch_78
 001447f0: 3922 3e0a 2020 2020 3c70 6174 6820 643d  9">.    <path d=
 00144800: 224d 2033 3432 2e31 3438 3433 3820 3132  "M 342.148438 12
 00144810: 3332 362e 3338 3532 3220 0a43 2033 3432  326.38522 .C 342
@@ -83087,15 +83087,15 @@
 001448e0: 3120 3334 312e 3334 3436 3620 3132 3332  1 341.34466 1232
 001448f0: 312e 3937 3831 3934 200a 4320 3334 312e  1.978194 .C 341.
 00144900: 3334 3436 3620 3132 3332 332e 3438 3336  34466 12323.4836
 00144910: 3236 2033 3431 2e36 3136 3937 3820 3132  26 341.616978 12
 00144920: 3332 342e 3937 3637 3138 2033 3432 2e31  324.976718 342.1
 00144930: 3438 3433 3820 3132 3332 362e 3338 3532  48438 12326.3852
 00144940: 3220 0a22 2063 6c69 702d 7061 7468 3d22  2 ." clip-path="
-00144950: 7572 6c28 2370 3564 3939 3437 6238 3762  url(#p5d9947b87b
+00144950: 7572 6c28 2370 6264 3030 3837 3264 6433  url(#pbd00872dd3
 00144960: 2922 2073 7479 6c65 3d22 6669 6c6c 3a20  )" style="fill: 
 00144970: 2366 6464 6464 373b 206f 7061 6369 7479  #fdddd7; opacity
 00144980: 3a20 302e 3422 2f3e 0a20 2020 3c2f 673e  : 0.4"/>.   </g>
 00144990: 0a20 2020 3c67 2069 643d 2274 6578 745f  .   <g id="text_
 001449a0: 3133 3332 223e 0a20 2020 203c 212d 2d20  1332">.    <!-- 
 001449b0: 3331 3331 202d 2d3e 0a20 2020 203c 6720  3131 -->.    <g 
 001449c0: 7374 796c 653d 2266 696c 6c3a 2023 3236  style="fill: #26
@@ -83285,15 +83285,15 @@
 00145540: 392e 3839 3530 3439 2031 3233 3333 2e34  9.895049 12333.4
 00145550: 3137 3936 3920 0a43 2035 3331 2e33 3137  17969 .C 531.317
 00145560: 3130 3820 3132 3333 312e 3033 3830 3334  108 12331.038034
 00145570: 2035 3332 2e32 3833 3031 2031 3233 3238   532.28301 12328
 00145580: 2e34 3133 3637 2035 3332 2e37 3433 3133  .41367 532.74313
 00145590: 3820 3132 3332 352e 3637 3936 3934 200a  8 12325.679694 .
 001455a0: 2220 636c 6970 2d70 6174 683d 2275 726c  " clip-path="url
-001455b0: 2823 7030 3738 6363 3337 3063 3129 2220  (#p078cc370c1)" 
+001455b0: 2823 7035 6531 3261 3033 3362 6529 2220  (#p5e12a033be)" 
 001455c0: 7374 796c 653d 2266 696c 6c3a 2023 3661  style="fill: #6a
 001455d0: 3364 3961 3b20 6f70 6163 6974 793a 2030  3d9a; opacity: 0
 001455e0: 2e34 222f 3e0a 2020 203c 2f67 3e0a 2020  .4"/>.   </g>.  
 001455f0: 203c 6720 6964 3d22 7061 7463 685f 3739   <g id="patch_79
 00145600: 3122 3e0a 2020 2020 3c70 6174 6820 643d  1">.    <path d=
 00145610: 224d 2035 3332 2e37 3433 3133 3820 3132  "M 532.743138 12
 00145620: 3331 382e 3237 3636 3933 200a 4320 3533  318.276693 .C 53
@@ -83377,16 +83377,16 @@
 00145b00: 2035 3335 2e36 3733 3432 3720 3132 3331   535.673427 1231
 00145b10: 302e 3134 3130 3536 200a 4320 3533 342e  0.141056 .C 534.
 00145b20: 3139 3832 3834 2031 3233 3132 2e36 3436  198284 12312.646
 00145b30: 3630 3720 3533 332e 3230 3434 3138 2031  607 533.204418 1
 00145b40: 3233 3135 2e34 3035 3937 2035 3332 2e37  2315.40597 532.7
 00145b50: 3433 3133 3820 3132 3331 382e 3237 3636  43138 12318.2766
 00145b60: 3933 200a 2220 636c 6970 2d70 6174 683d  93 ." clip-path=
-00145b70: 2275 726c 2823 7030 3738 6363 3337 3063  "url(#p078cc370c
-00145b80: 3129 2220 7374 796c 653d 2266 696c 6c3a  1)" style="fill:
+00145b70: 2275 726c 2823 7035 6531 3261 3033 3362  "url(#p5e12a033b
+00145b80: 6529 2220 7374 796c 653d 2266 696c 6c3a  e)" style="fill:
 00145b90: 2023 6231 3539 3238 3b20 6f70 6163 6974   #b15928; opacit
 00145ba0: 793a 2030 2e34 222f 3e0a 2020 203c 2f67  y: 0.4"/>.   </g
 00145bb0: 3e0a 2020 203c 6720 6964 3d22 7061 7463  >.   <g id="patc
 00145bc0: 685f 3739 3222 3e0a 2020 2020 3c70 6174  h_792">.    <pat
 00145bd0: 6820 643d 224d 2035 3332 2e37 3433 3133  h d="M 532.74313
 00145be0: 3820 3132 3332 352e 3637 3936 3934 200a  8 12325.679694 .
 00145bf0: 4320 3533 322e 3934 3839 3833 2031 3233  C 532.948983 123
@@ -83404,16 +83404,16 @@
 00145cb0: 3733 3834 3720 3533 322e 3434 3736 3436  73847 532.447646
 00145cc0: 2031 3233 3231 2e39 3738 3139 3420 0a43   12321.978194 .C
 00145cd0: 2035 3332 2e34 3437 3634 3620 3132 3332   532.447646 1232
 00145ce0: 332e 3231 3739 3138 2035 3332 2e35 3436  3.217918 532.546
 00145cf0: 3435 3620 3132 3332 342e 3435 3536 3732  456 12324.455672
 00145d00: 2035 3332 2e37 3433 3133 3820 3132 3332   532.743138 1232
 00145d10: 352e 3637 3936 3934 200a 2220 636c 6970  5.679694 ." clip
-00145d20: 2d70 6174 683d 2275 726c 2823 7030 3738  -path="url(#p078
-00145d30: 6363 3337 3063 3129 2220 7374 796c 653d  cc370c1)" style=
+00145d20: 2d70 6174 683d 2275 726c 2823 7035 6531  -path="url(#p5e1
+00145d30: 3261 3033 3362 6529 2220 7374 796c 653d  2a033be)" style=
 00145d40: 2266 696c 6c3a 2023 6336 3639 3838 3b20  "fill: #c66988; 
 00145d50: 6f70 6163 6974 793a 2030 2e34 222f 3e0a  opacity: 0.4"/>.
 00145d60: 2020 203c 2f67 3e0a 2020 203c 6720 6964     </g>.   <g id
 00145d70: 3d22 7465 7874 5f31 3333 3722 3e0a 2020  ="text_1337">.  
 00145d80: 2020 3c21 2d2d 2033 3133 3120 2d2d 3e0a    <!-- 3131 -->.
 00145d90: 2020 2020 3c67 2073 7479 6c65 3d22 6669      <g style="fi
 00145da0: 6c6c 3a20 2332 3632 3632 3622 2074 7261  ll: #262626" tra
@@ -83617,15 +83617,15 @@
 00146a00: 3838 3331 3536 2031 3233 3330 2e37 3438  883156 12330.748
 00146a10: 3832 3720 0a43 2037 3232 2e38 3630 3733  827 .C 722.86073
 00146a20: 3920 3132 3332 392e 3831 3133 3034 2037  9 12329.811304 7
 00146a30: 3233 2e36 3734 3733 3320 3132 3332 382e  23.674733 12328.
 00146a40: 3731 3639 3031 2037 3234 2e32 3931 3432  716901 724.29142
 00146a50: 2031 3233 3237 2e35 3130 3935 3120 0a22   12327.510951 ."
 00146a60: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00146a70: 2370 6332 3061 6166 3661 3163 2922 2073  #pc20aaf6a1c)" s
+00146a70: 2370 3630 3335 3035 3937 3736 2922 2073  #p6035059776)" s
 00146a80: 7479 6c65 3d22 6669 6c6c 3a20 2366 6666  tyle="fill: #fff
 00146a90: 6639 393b 206f 7061 6369 7479 3a20 302e  f99; opacity: 0.
 00146aa0: 3422 2f3e 0a20 2020 3c2f 673e 0a20 2020  4"/>.   </g>.   
 00146ab0: 3c67 2069 643d 2270 6174 6368 5f37 3934  <g id="patch_794
 00146ac0: 223e 0a20 2020 203c 7061 7468 2064 3d22  ">.    <path d="
 00146ad0: 4d20 3732 342e 3239 3134 3220 3132 3331  M 724.29142 1231
 00146ae0: 362e 3434 3534 3336 200a 4320 3732 352e  6.445436 .C 725.
@@ -83709,15 +83709,15 @@
 00146fc0: 382e 3536 3038 3420 3132 3330 342e 3831  8.56084 12304.81
 00146fd0: 3934 3820 0a43 2037 3236 2e34 3237 3331  948 .C 726.42731
 00146fe0: 3420 3132 3330 382e 3339 3535 3932 2037  4 12308.395592 7
 00146ff0: 3234 2e39 3739 3336 3720 3132 3331 322e  24.979367 12312.
 00147000: 3333 3834 3632 2037 3234 2e32 3931 3432  338462 724.29142
 00147010: 2031 3233 3136 2e34 3435 3433 3620 0a22   12316.445436 ."
 00147020: 2063 6c69 702d 7061 7468 3d22 7572 6c28   clip-path="url(
-00147030: 2370 6332 3061 6166 3661 3163 2922 2073  #pc20aaf6a1c)" s
+00147030: 2370 3630 3335 3035 3937 3736 2922 2073  #p6035059776)" s
 00147040: 7479 6c65 3d22 6669 6c6c 3a20 2362 3135  tyle="fill: #b15
 00147050: 3932 383b 206f 7061 6369 7479 3a20 302e  928; opacity: 0.
 00147060: 3422 2f3e 0a20 2020 3c2f 673e 0a20 2020  4"/>.   </g>.   
 00147070: 3c67 2069 643d 2270 6174 6368 5f37 3935  <g id="patch_795
 00147080: 223e 0a20 2020 203c 7061 7468 2064 3d22  ">.    <path d="
 00147090: 4d20 3732 342e 3239 3134 3220 3132 3332  M 724.29142 1232
 001470a0: 372e 3531 3039 3531 200a 4320 3732 352e  7.510951 .C 725.
@@ -83736,15 +83736,15 @@
 00147170: 332e 3833 3132 3338 2031 3233 3231 2e39  3.831238 12321.9
 00147180: 3738 3139 3420 0a43 2037 3233 2e38 3331  78194 .C 723.831
 00147190: 3233 3820 3132 3332 332e 3833 3230 3033  238 12323.832003
 001471a0: 2037 3233 2e39 3835 3136 3120 3132 3332   723.985161 1232
 001471b0: 352e 3638 3236 3135 2037 3234 2e32 3931  5.682615 724.291
 001471c0: 3432 2031 3233 3237 2e35 3130 3935 3120  42 12327.510951 
 001471d0: 0a22 2063 6c69 702d 7061 7468 3d22 7572  ." clip-path="ur
-001471e0: 6c28 2370 6332 3061 6166 3661 3163 2922  l(#pc20aaf6a1c)"
+001471e0: 6c28 2370 3630 3335 3035 3937 3736 2922  l(#p6035059776)"
 001471f0: 2073 7479 6c65 3d22 6669 6c6c 3a20 2366   style="fill: #f
 00147200: 6666 3138 373b 206f 7061 6369 7479 3a20  ff187; opacity: 
 00147210: 302e 3422 2f3e 0a20 2020 3c2f 673e 0a20  0.4"/>.   </g>. 
 00147220: 2020 3c67 2069 643d 2274 6578 745f 3133    <g id="text_13
 00147230: 3432 223e 0a20 2020 203c 212d 2d20 3433  42">.    <!-- 43
 00147240: 3920 2d2d 3e0a 2020 2020 3c67 2073 7479  9 -->.    <g sty
 00147250: 6c65 3d22 6669 6c6c 3a20 2332 3632 3632  le="fill: #26262
@@ -83849,15 +83849,15 @@
 00147880: 3033 3930 3632 222f 3e0a 2020 2020 3c2f  039062"/>.    </
 00147890: 673e 0a20 2020 3c2f 673e 0a20 203c 2f67  g>.   </g>.  </g
 001478a0: 3e0a 2020 3c67 2069 643d 2274 6578 745f  >.  <g id="text_
 001478b0: 3133 3437 223e 0a20 2020 3c21 2d2d 2068  1347">.   <!-- h
 001478c0: 7474 7073 3a2f 2f67 6974 6875 622e 636f  ttps://github.co
 001478d0: 6d2f 6269 6f70 7261 676d 6174 6963 732f  m/biopragmatics/
 001478e0: 6269 6f72 6567 6973 7472 7920 2832 3032  bioregistry (202
-001478f0: 332d 3037 2d33 3029 202d 2d3e 0a20 2020  3-07-30) -->.   
+001478f0: 332d 3038 2d30 3129 202d 2d3e 0a20 2020  3-08-01) -->.   
 00147900: 3c67 2073 7479 6c65 3d22 6669 6c6c 3a20  <g style="fill: 
 00147910: 2338 3038 3038 303b 206f 7061 6369 7479  #808080; opacity
 00147920: 3a20 302e 3522 2074 7261 6e73 666f 726d  : 0.5" transform
 00147930: 3d22 7472 616e 736c 6174 6528 3232 362e  ="translate(226.
 00147940: 3638 3334 3337 2031 3234 3137 2e30 3838  683437 12417.088
 00147950: 3433 3829 2073 6361 6c65 2830 2e31 3420  438) scale(0.14 
 00147960: 2d30 2e31 3429 223e 0a20 2020 203c 6465  -0.14)">.    <de
@@ -84125,25 +84125,25 @@
 001489c0: 2d32 6422 2078 3d22 3235 3637 2e34 3237  -2d" x="2567.427
 001489d0: 3733 3422 2f3e 0a20 2020 203c 7573 6520  734"/>.    <use 
 001489e0: 786c 696e 6b3a 6872 6566 3d22 2344 656a  xlink:href="#Dej
 001489f0: 6156 7553 616e 732d 3330 2220 783d 2232  aVuSans-30" x="2
 00148a00: 3630 332e 3531 3137 3139 222f 3e0a 2020  603.511719"/>.  
 00148a10: 2020 3c75 7365 2078 6c69 6e6b 3a68 7265    <use xlink:hre
 00148a20: 663d 2223 4465 6a61 5675 5361 6e73 2d33  f="#DejaVuSans-3
-00148a30: 3722 2078 3d22 3236 3637 2e31 3334 3736  7" x="2667.13476
+00148a30: 3822 2078 3d22 3236 3637 2e31 3334 3736  8" x="2667.13476
 00148a40: 3622 2f3e 0a20 2020 203c 7573 6520 786c  6"/>.    <use xl
 00148a50: 696e 6b3a 6872 6566 3d22 2344 656a 6156  ink:href="#DejaV
 00148a60: 7553 616e 732d 3264 2220 783d 2232 3733  uSans-2d" x="273
 00148a70: 302e 3735 3738 3132 222f 3e0a 2020 2020  0.757812"/>.    
 00148a80: 3c75 7365 2078 6c69 6e6b 3a68 7265 663d  <use xlink:href=
-00148a90: 2223 4465 6a61 5675 5361 6e73 2d33 3322  "#DejaVuSans-33"
+00148a90: 2223 4465 6a61 5675 5361 6e73 2d33 3022  "#DejaVuSans-30"
 00148aa0: 2078 3d22 3237 3636 2e38 3431 3739 3722   x="2766.841797"
 00148ab0: 2f3e 0a20 2020 203c 7573 6520 786c 696e  />.    <use xlin
 00148ac0: 6b3a 6872 6566 3d22 2344 656a 6156 7553  k:href="#DejaVuS
-00148ad0: 616e 732d 3330 2220 783d 2232 3833 302e  ans-30" x="2830.
+00148ad0: 616e 732d 3331 2220 783d 2232 3833 302e  ans-31" x="2830.
 00148ae0: 3436 3438 3434 222f 3e0a 2020 2020 3c75  464844"/>.    <u
 00148af0: 7365 2078 6c69 6e6b 3a68 7265 663d 2223  se xlink:href="#
 00148b00: 4465 6a61 5675 5361 6e73 2d32 3922 2078  DejaVuSans-29" x
 00148b10: 3d22 3238 3934 2e30 3837 3839 3122 2f3e  ="2894.087891"/>
 00148b20: 0a20 2020 3c2f 673e 0a20 203c 2f67 3e0a  .   </g>.  </g>.
 00148b30: 203c 2f67 3e0a 203c 6465 6673 3e0a 2020   </g>. <defs>.  
 00148b40: 3c63 6c69 7050 6174 6820 6964 3d22 7033  <clipPath id="p3
@@ -85145,15 +85145,15 @@
 0014c980: 3838 6231 223e 0a20 2020 3c72 6563 7420  88b1">.   <rect 
 0014c990: 783d 2232 3730 2e34 3431 3337 3622 2079  x="270.441376" y
 0014c9a0: 3d22 3537 3939 2e33 3131 3036 3122 2077  ="5799.311061" w
 0014c9b0: 6964 7468 3d22 3130 322e 3039 3939 3532  idth="102.099952
 0014c9c0: 2220 6865 6967 6874 3d22 3934 2e37 3239  " height="94.729
 0014c9d0: 3436 3222 2f3e 0a20 203c 2f63 6c69 7050  462"/>.  </clipP
 0014c9e0: 6174 683e 0a20 203c 636c 6970 5061 7468  ath>.  <clipPath
-0014c9f0: 2069 643d 2270 3138 3564 3336 6266 3661   id="p185d36bf6a
+0014c9f0: 2069 643d 2270 6535 3365 3263 3739 3433   id="pe53e2c7943
 0014ca00: 223e 0a20 2020 3c72 6563 7420 783d 2234  ">.   <rect x="4
 0014ca10: 3832 2e37 3238 3537 3522 2079 3d22 3538  82.728575" y="58
 0014ca20: 3030 2e35 3834 3834 3522 2077 6964 7468  00.584845" width
 0014ca30: 3d22 3130 322e 3039 3939 3532 2220 6865  ="102.099952" he
 0014ca40: 6967 6874 3d22 3932 2e31 3831 3839 3422  ight="92.181894"
 0014ca50: 2f3e 0a20 203c 2f63 6c69 7050 6174 683e  />.  </clipPath>
 0014ca60: 0a20 203c 636c 6970 5061 7468 2069 643d  .  <clipPath id=
@@ -85663,15 +85663,15 @@
 0014e9e0: 3822 3e0a 2020 203c 7265 6374 2078 3d22  8">.   <rect x="
 0014e9f0: 3538 2e31 3534 3137 3822 2079 3d22 3838  58.154178" y="88
 0014ea00: 3637 2e36 3235 3039 3922 2077 6964 7468  67.625099" width
 0014ea10: 3d22 3130 322e 3039 3939 3532 2220 6865  ="102.099952" he
 0014ea20: 6967 6874 3d22 3733 2e36 3634 3736 3522  ight="73.664765"
 0014ea30: 2f3e 0a20 203c 2f63 6c69 7050 6174 683e  />.  </clipPath>
 0014ea40: 0a20 203c 636c 6970 5061 7468 2069 643d  .  <clipPath id=
-0014ea50: 2270 3236 3839 3631 6135 3562 223e 0a20  "p268961a55b">. 
+0014ea50: 2270 3133 3136 3739 6264 3634 223e 0a20  "p131679bd64">. 
 0014ea60: 2020 3c72 6563 7420 783d 2232 3730 2e34    <rect x="270.4
 0014ea70: 3431 3337 3622 2079 3d22 3838 3630 2e32  41376" y="8860.2
 0014ea80: 3138 3435 2220 7769 6474 683d 2231 3032  1845" width="102
 0014ea90: 2e30 3939 3935 3222 2068 6569 6768 743d  .099952" height=
 0014eaa0: 2238 382e 3437 3830 3633 222f 3e0a 2020  "88.478063"/>.  
 0014eab0: 3c2f 636c 6970 5061 7468 3e0a 2020 3c63  </clipPath>.  <c
 0014eac0: 6c69 7050 6174 6820 6964 3d22 7064 6163  lipPath id="pdac
@@ -85755,16 +85755,16 @@
 0014efa0: 7038 3930 3864 6135 6536 6522 3e0a 2020  p8908da5e6e">.  
 0014efb0: 203c 7265 6374 2078 3d22 3538 2e31 3534   <rect x="58.154
 0014efc0: 3137 3822 2079 3d22 3934 3039 2e34 3730  178" y="9409.470
 0014efd0: 3431 3922 2077 6964 7468 3d22 3130 322e  419" width="102.
 0014efe0: 3039 3939 3532 2220 6865 6967 6874 3d22  099952" height="
 0014eff0: 3639 2e31 3931 3139 3222 2f3e 0a20 203c  69.191192"/>.  <
 0014f000: 2f63 6c69 7050 6174 683e 0a20 203c 636c  /clipPath>.  <cl
-0014f010: 6970 5061 7468 2069 643d 2270 3038 3735  ipPath id="p0875
-0014f020: 3031 6335 3338 223e 0a20 2020 3c72 6563  01c538">.   <rec
+0014f010: 6970 5061 7468 2069 643d 2270 3066 3961  ipPath id="p0f9a
+0014f020: 6163 6561 3661 223e 0a20 2020 3c72 6563  acea6a">.   <rec
 0014f030: 7420 783d 2232 3730 2e34 3431 3337 3622  t x="270.441376"
 0014f040: 2079 3d22 3934 3030 2e39 3336 3336 3622   y="9400.936366"
 0014f050: 2077 6964 7468 3d22 3130 322e 3039 3939   width="102.0999
 0014f060: 3532 2220 6865 6967 6874 3d22 3836 2e32  52" height="86.2
 0014f070: 3539 3239 3922 2f3e 0a20 203c 2f63 6c69  59299"/>.  </cli
 0014f080: 7050 6174 683e 0a20 203c 636c 6970 5061  pPath>.  <clipPa
 0014f090: 7468 2069 643d 2270 3765 3762 3861 3763  th id="p7e7b8a7c
@@ -85972,31 +85972,31 @@
 0014fd30: 3634 3536 6464 223e 0a20 2020 3c72 6563  6456dd">.   <rec
 0014fd40: 7420 783d 2235 382e 3135 3431 3738 2220  t x="58.154178" 
 0014fd50: 793d 2231 3036 3537 2e32 3637 3133 3822  y="10657.267138"
 0014fd60: 2077 6964 7468 3d22 3130 322e 3039 3939   width="102.0999
 0014fd70: 3532 2220 6865 6967 6874 3d22 3931 2e37  52" height="91.7
 0014fd80: 3730 3931 222f 3e0a 2020 3c2f 636c 6970  7091"/>.  </clip
 0014fd90: 5061 7468 3e0a 2020 3c63 6c69 7050 6174  Path>.  <clipPat
-0014fda0: 6820 6964 3d22 7035 3865 3839 6333 3862  h id="p58e89c38b
-0014fdb0: 3122 3e0a 2020 203c 7265 6374 2078 3d22  1">.   <rect x="
+0014fda0: 6820 6964 3d22 7063 6239 6161 3337 3931  h id="pcb9aa3791
+0014fdb0: 6222 3e0a 2020 203c 7265 6374 2078 3d22  b">.   <rect x="
 0014fdc0: 3237 302e 3434 3133 3736 2220 793d 2231  270.441376" y="1
 0014fdd0: 3036 3533 2e39 3822 2077 6964 7468 3d22  0653.98" width="
 0014fde0: 3130 322e 3039 3939 3532 2220 6865 6967  102.099952" heig
 0014fdf0: 6874 3d22 3938 2e33 3435 3138 3622 2f3e  ht="98.345186"/>
 0014fe00: 0a20 203c 2f63 6c69 7050 6174 683e 0a20  .  </clipPath>. 
 0014fe10: 203c 636c 6970 5061 7468 2069 643d 2270   <clipPath id="p
 0014fe20: 3133 6562 6239 6138 3437 223e 0a20 2020  13ebb9a847">.   
 0014fe30: 3c72 6563 7420 783d 2234 3832 2e37 3238  <rect x="482.728
 0014fe40: 3537 3522 2079 3d22 3130 3635 392e 3330  575" y="10659.30
 0014fe50: 3438 3031 2220 7769 6474 683d 2231 3032  4801" width="102
 0014fe60: 2e30 3939 3935 3222 2068 6569 6768 743d  .099952" height=
 0014fe70: 2238 372e 3639 3535 3834 222f 3e0a 2020  "87.695584"/>.  
 0014fe80: 3c2f 636c 6970 5061 7468 3e0a 2020 3c63  </clipPath>.  <c
-0014fe90: 6c69 7050 6174 6820 6964 3d22 7030 3736  lipPath id="p076
-0014fea0: 3165 3063 6233 3622 3e0a 2020 203c 7265  1e0cb36">.   <re
+0014fe90: 6c69 7050 6174 6820 6964 3d22 7061 6134  lipPath id="paa4
+0014fea0: 6332 6534 6137 3722 3e0a 2020 203c 7265  c2e4a77">.   <re
 0014feb0: 6374 2078 3d22 3639 352e 3031 3537 3733  ct x="695.015773
 0014fec0: 2220 793d 2231 3036 3534 2e38 3136 3336  " y="10654.81636
 0014fed0: 3222 2077 6964 7468 3d22 3130 322e 3039  2" width="102.09
 0014fee0: 3939 3532 2220 6865 6967 6874 3d22 3936  9952" height="96
 0014fef0: 2e36 3732 3436 3322 2f3e 0a20 203c 2f63  .672463"/>.  </c
 0014ff00: 6c69 7050 6174 683e 0a20 203c 636c 6970  lipPath>.  <clip
 0014ff10: 5061 7468 2069 643d 2270 3532 3933 3738  Path id="p529378
@@ -86104,16 +86104,16 @@
 00150570: 3963 6436 3361 6665 223e 0a20 2020 3c72  9cd63afe">.   <r
 00150580: 6563 7420 783d 2232 3730 2e34 3431 3337  ect x="270.44137
 00150590: 3622 2079 3d22 3131 3338 372e 3638 3633  6" y="11387.6863
 001505a0: 3738 2220 7769 6474 683d 2231 3032 2e30  78" width="102.0
 001505b0: 3939 3935 3222 2068 6569 6768 743d 2236  99952" height="6
 001505c0: 392e 3838 3835 3231 222f 3e0a 2020 3c2f  9.888521"/>.  </
 001505d0: 636c 6970 5061 7468 3e0a 2020 3c63 6c69  clipPath>.  <cli
-001505e0: 7050 6174 6820 6964 3d22 7032 3666 6262  pPath id="p26fbb
-001505f0: 6530 3965 6522 3e0a 2020 203c 7265 6374  e09ee">.   <rect
+001505e0: 7050 6174 6820 6964 3d22 7036 3630 3037  pPath id="p66007
+001505f0: 3062 3332 6622 3e0a 2020 203c 7265 6374  0b32f">.   <rect
 00150600: 2078 3d22 3438 322e 3732 3835 3735 2220   x="482.728575" 
 00150610: 793d 2231 3133 3738 2e38 3830 3039 3922  y="11378.880099"
 00150620: 2077 6964 7468 3d22 3130 322e 3039 3939   width="102.0999
 00150630: 3532 2220 6865 6967 6874 3d22 3837 2e35  52" height="87.5
 00150640: 3031 3037 3822 2f3e 0a20 203c 2f63 6c69  01078"/>.  </cli
 00150650: 7050 6174 683e 0a20 203c 636c 6970 5061  pPath>.  <clipPa
 00150660: 7468 2069 643d 2270 6637 3139 3930 3738  th id="pf7199078
@@ -86229,54 +86229,54 @@
 00150d40: 223e 0a20 2020 3c72 6563 7420 783d 2232  ">.   <rect x="2
 00150d50: 3730 2e34 3431 3337 3622 2079 3d22 3132  70.441376" y="12
 00150d60: 3130 342e 3133 3136 3138 2220 7769 6474  104.131618" widt
 00150d70: 683d 2231 3032 2e30 3939 3935 3222 2068  h="102.099952" h
 00150d80: 6569 6768 743d 2237 352e 3935 3431 3239  eight="75.954129
 00150d90: 222f 3e0a 2020 3c2f 636c 6970 5061 7468  "/>.  </clipPath
 00150da0: 3e0a 2020 3c63 6c69 7050 6174 6820 6964  >.  <clipPath id
-00150db0: 3d22 7030 6537 3639 6533 6435 3722 3e0a  ="p0e769e3d57">.
+00150db0: 3d22 7065 3965 3536 3239 3034 3922 3e0a  ="pe9e5629049">.
 00150dc0: 2020 203c 7265 6374 2078 3d22 3438 322e     <rect x="482.
 00150dd0: 3732 3835 3735 2220 793d 2231 3230 3939  728575" y="12099
 00150de0: 2e33 3932 3439 3922 2077 6964 7468 3d22  .392499" width="
 00150df0: 3130 322e 3039 3939 3532 2220 6865 6967  102.099952" heig
 00150e00: 6874 3d22 3835 2e34 3332 3336 3722 2f3e  ht="85.432367"/>
 00150e10: 0a20 203c 2f63 6c69 7050 6174 683e 0a20  .  </clipPath>. 
 00150e20: 203c 636c 6970 5061 7468 2069 643d 2270   <clipPath id="p
-00150e30: 3235 3637 6365 3735 6161 223e 0a20 2020  2567ce75aa">.   
+00150e30: 6532 3763 6639 3130 6363 223e 0a20 2020  e27cf910cc">.   
 00150e40: 3c72 6563 7420 783d 2236 3935 2e30 3135  <rect x="695.015
 00150e50: 3737 3322 2079 3d22 3132 3130 392e 3931  773" y="12109.91
 00150e60: 3631 3422 2077 6964 7468 3d22 3130 322e  614" width="102.
 00150e70: 3039 3939 3532 2220 6865 6967 6874 3d22  099952" height="
 00150e80: 3634 2e33 3835 3038 3422 2f3e 0a20 203c  64.385084"/>.  <
 00150e90: 2f63 6c69 7050 6174 683e 0a20 203c 636c  /clipPath>.  <cl
 00150ea0: 6970 5061 7468 2069 643d 2270 3438 3335  ipPath id="p4835
 00150eb0: 3365 3566 3333 223e 0a20 2020 3c72 6563  3e5f33">.   <rec
 00150ec0: 7420 783d 2235 382e 3135 3431 3738 2220  t x="58.154178" 
 00150ed0: 793d 2231 3232 3738 2e33 3831 3232 3622  y="12278.381226"
 00150ee0: 2077 6964 7468 3d22 3130 322e 3039 3939   width="102.0999
 00150ef0: 3532 2220 6865 6967 6874 3d22 3837 2e31  52" height="87.1
 00150f00: 3933 3933 3522 2f3e 0a20 203c 2f63 6c69  93935"/>.  </cli
 00150f10: 7050 6174 683e 0a20 203c 636c 6970 5061  pPath>.  <clipPa
-00150f20: 7468 2069 643d 2270 3564 3939 3437 6238  th id="p5d9947b8
-00150f30: 3762 223e 0a20 2020 3c72 6563 7420 783d  7b">.   <rect x=
+00150f20: 7468 2069 643d 2270 6264 3030 3837 3264  th id="pbd00872d
+00150f30: 6433 223e 0a20 2020 3c72 6563 7420 783d  d3">.   <rect x=
 00150f40: 2232 3730 2e34 3431 3337 3622 2079 3d22  "270.441376" y="
 00150f50: 3132 3238 322e 3836 3135 3136 2220 7769  12282.861516" wi
 00150f60: 6474 683d 2231 3032 2e30 3939 3935 3222  dth="102.099952"
 00150f70: 2068 6569 6768 743d 2237 382e 3233 3333   height="78.2333
 00150f80: 3536 222f 3e0a 2020 3c2f 636c 6970 5061  56"/>.  </clipPa
 00150f90: 7468 3e0a 2020 3c63 6c69 7050 6174 6820  th>.  <clipPath 
-00150fa0: 6964 3d22 7030 3738 6363 3337 3063 3122  id="p078cc370c1"
+00150fa0: 6964 3d22 7035 6531 3261 3033 3362 6522  id="p5e12a033be"
 00150fb0: 3e0a 2020 203c 7265 6374 2078 3d22 3438  >.   <rect x="48
 00150fc0: 322e 3732 3835 3735 2220 793d 2231 3232  2.728575" y="122
 00150fd0: 3932 2e39 3238 3633 3822 2077 6964 7468  92.928638" width
 00150fe0: 3d22 3130 322e 3039 3939 3532 2220 6865  ="102.099952" he
 00150ff0: 6967 6874 3d22 3538 2e30 3939 3131 3222  ight="58.099112"
 00151000: 2f3e 0a20 203c 2f63 6c69 7050 6174 683e  />.  </clipPath>
 00151010: 0a20 203c 636c 6970 5061 7468 2069 643d  .  <clipPath id=
-00151020: 2270 6332 3061 6166 3661 3163 223e 0a20  "pc20aaf6a1c">. 
+00151020: 2270 3630 3335 3035 3937 3736 223e 0a20  "p6035059776">. 
 00151030: 2020 3c72 6563 7420 783d 2236 3935 2e30    <rect x="695.0
 00151040: 3135 3737 3322 2079 3d22 3132 3238 322e  15773" y="12282.
 00151050: 3138 3338 3938 2220 7769 6474 683d 2231  183898" width="1
 00151060: 3032 2e30 3939 3935 3222 2068 6569 6768  02.099952" heigh
 00151070: 743d 2237 392e 3538 3835 3931 222f 3e0a  t="79.588591"/>.
 00151080: 2020 3c2f 636c 6970 5061 7468 3e0a 203c    </clipPath>. <
 00151090: 2f64 6566 733e 0a3c 2f73 7667 3e0a       /defs>.</svg>.
```

### Comparing `bioregistry-0.9.90/docs/img/has_attribute.svg` & `bioregistry-0.9.91/docs/img/has_attribute.svg`

 * *Files 0% similar despite different names*

```diff
@@ -29,16 +29,16 @@
 000001c0: 3032 2f32 322d 7264 662d 7379 6e74 6178  02/22-rdf-syntax
 000001d0: 2d6e 7323 223e 0a20 2020 3c63 633a 576f  -ns#">.   <cc:Wo
 000001e0: 726b 3e0a 2020 2020 3c64 633a 7479 7065  rk>.    <dc:type
 000001f0: 2072 6466 3a72 6573 6f75 7263 653d 2268   rdf:resource="h
 00000200: 7474 703a 2f2f 7075 726c 2e6f 7267 2f64  ttp://purl.org/d
 00000210: 632f 6463 6d69 7479 7065 2f53 7469 6c6c  c/dcmitype/Still
 00000220: 496d 6167 6522 2f3e 0a20 2020 203c 6463  Image"/>.    <dc
-00000230: 3a64 6174 653e 3230 3233 2d30 372d 3330  :date>2023-07-30
-00000240: 5430 313a 3137 3a30 302e 3039 3530 3037  T01:17:00.095007
+00000230: 3a64 6174 653e 3230 3233 2d30 382d 3031  :date>2023-08-01
+00000240: 5430 313a 3230 3a34 322e 3939 3230 3135  T01:20:42.992015
 00000250: 3c2f 6463 3a64 6174 653e 0a20 2020 203c  </dc:date>.    <
 00000260: 6463 3a66 6f72 6d61 743e 696d 6167 652f  dc:format>image/
 00000270: 7376 672b 786d 6c3c 2f64 633a 666f 726d  svg+xml</dc:form
 00000280: 6174 3e0a 2020 2020 3c64 633a 6372 6561  at>.    <dc:crea
 00000290: 746f 723e 0a20 2020 2020 3c63 633a 4167  tor>.     <cc:Ag
 000002a0: 656e 743e 0a20 2020 2020 203c 6463 3a74  ent>.      <dc:t
 000002b0: 6974 6c65 3e4d 6174 706c 6f74 6c69 6220  itle>Matplotlib 
@@ -4210,15 +4210,15 @@
 00010710: 3c2f 673e 0a20 203c 6720 6964 3d22 6178  </g>.  <g id="ax
 00010720: 6573 5f31 3522 2f3e 0a20 203c 6720 6964  es_15"/>.  <g id
 00010730: 3d22 6178 6573 5f31 3622 2f3e 0a20 203c  ="axes_16"/>.  <
 00010740: 6720 6964 3d22 7465 7874 5f37 3522 3e0a  g id="text_75">.
 00010750: 2020 203c 212d 2d20 6874 7470 733a 2f2f     <!-- https://
 00010760: 6769 7468 7562 2e63 6f6d 2f62 696f 7072  github.com/biopr
 00010770: 6167 6d61 7469 6373 2f62 696f 7265 6769  agmatics/bioregi
-00010780: 7374 7279 2028 3230 3233 2d30 372d 3330  stry (2023-07-30
+00010780: 7374 7279 2028 3230 3233 2d30 382d 3031  stry (2023-08-01
 00010790: 2920 2d2d 3e0a 2020 203c 6720 7374 796c  ) -->.   <g styl
 000107a0: 653d 2266 696c 6c3a 2023 3830 3830 3830  e="fill: #808080
 000107b0: 3b20 6f70 6163 6974 793a 2030 2e35 2220  ; opacity: 0.5" 
 000107c0: 7472 616e 7366 6f72 6d3d 2274 7261 6e73  transform="trans
 000107d0: 6c61 7465 2832 3738 2e36 3736 3235 2035  late(278.67625 5
 000107e0: 3734 2e33 3336 3235 2920 7363 616c 6528  74.33625) scale(
 000107f0: 302e 3038 202d 302e 3038 2922 3e0a 2020  0.08 -0.08)">.  
@@ -4475,25 +4475,25 @@
 000117a0: 7553 616e 732d 3264 2220 783d 2232 3536  uSans-2d" x="256
 000117b0: 372e 3432 3737 3334 222f 3e0a 2020 2020  7.427734"/>.    
 000117c0: 3c75 7365 2078 6c69 6e6b 3a68 7265 663d  <use xlink:href=
 000117d0: 2223 4465 6a61 5675 5361 6e73 2d33 3022  "#DejaVuSans-30"
 000117e0: 2078 3d22 3236 3033 2e35 3131 3731 3922   x="2603.511719"
 000117f0: 2f3e 0a20 2020 203c 7573 6520 786c 696e  />.    <use xlin
 00011800: 6b3a 6872 6566 3d22 2344 656a 6156 7553  k:href="#DejaVuS
-00011810: 616e 732d 3337 2220 783d 2232 3636 372e  ans-37" x="2667.
+00011810: 616e 732d 3338 2220 783d 2232 3636 372e  ans-38" x="2667.
 00011820: 3133 3437 3636 222f 3e0a 2020 2020 3c75  134766"/>.    <u
 00011830: 7365 2078 6c69 6e6b 3a68 7265 663d 2223  se xlink:href="#
 00011840: 4465 6a61 5675 5361 6e73 2d32 6422 2078  DejaVuSans-2d" x
 00011850: 3d22 3237 3330 2e37 3537 3831 3222 2f3e  ="2730.757812"/>
 00011860: 0a20 2020 203c 7573 6520 786c 696e 6b3a  .    <use xlink:
 00011870: 6872 6566 3d22 2344 656a 6156 7553 616e  href="#DejaVuSan
-00011880: 732d 3333 2220 783d 2232 3736 362e 3834  s-33" x="2766.84
+00011880: 732d 3330 2220 783d 2232 3736 362e 3834  s-30" x="2766.84
 00011890: 3137 3937 222f 3e0a 2020 2020 3c75 7365  1797"/>.    <use
 000118a0: 2078 6c69 6e6b 3a68 7265 663d 2223 4465   xlink:href="#De
-000118b0: 6a61 5675 5361 6e73 2d33 3022 2078 3d22  jaVuSans-30" x="
+000118b0: 6a61 5675 5361 6e73 2d33 3122 2078 3d22  jaVuSans-31" x="
 000118c0: 3238 3330 2e34 3634 3834 3422 2f3e 0a20  2830.464844"/>. 
 000118d0: 2020 203c 7573 6520 786c 696e 6b3a 6872     <use xlink:hr
 000118e0: 6566 3d22 2344 656a 6156 7553 616e 732d  ef="#DejaVuSans-
 000118f0: 3239 2220 783d 2232 3839 342e 3038 3738  29" x="2894.0878
 00011900: 3931 222f 3e0a 2020 203c 2f67 3e0a 2020  91"/>.   </g>.  
 00011910: 3c2f 673e 0a20 3c2f 673e 0a3c 2f73 7667  </g>. </g>.</svg
 00011920: 3e0a                                     >.
```

### Comparing `bioregistry-0.9.90/docs/img/providers.svg` & `bioregistry-0.9.91/docs/img/providers.svg`

 * *Files 0% similar despite different names*

```diff
@@ -29,16 +29,16 @@
 000001c0: 3032 2f32 322d 7264 662d 7379 6e74 6178  02/22-rdf-syntax
 000001d0: 2d6e 7323 223e 0a20 2020 3c63 633a 576f  -ns#">.   <cc:Wo
 000001e0: 726b 3e0a 2020 2020 3c64 633a 7479 7065  rk>.    <dc:type
 000001f0: 2072 6466 3a72 6573 6f75 7263 653d 2268   rdf:resource="h
 00000200: 7474 703a 2f2f 7075 726c 2e6f 7267 2f64  ttp://purl.org/d
 00000210: 632f 6463 6d69 7479 7065 2f53 7469 6c6c  c/dcmitype/Still
 00000220: 496d 6167 6522 2f3e 0a20 2020 203c 6463  Image"/>.    <dc
-00000230: 3a64 6174 653e 3230 3233 2d30 372d 3330  :date>2023-07-30
-00000240: 5430 313a 3230 3a32 312e 3536 3439 3332  T01:20:21.564932
+00000230: 3a64 6174 653e 3230 3233 2d30 382d 3031  :date>2023-08-01
+00000240: 5430 313a 3234 3a35 362e 3631 3232 3132  T01:24:56.612212
 00000250: 3c2f 6463 3a64 6174 653e 0a20 2020 203c  </dc:date>.    <
 00000260: 6463 3a66 6f72 6d61 743e 696d 6167 652f  dc:format>image/
 00000270: 7376 672b 786d 6c3c 2f64 633a 666f 726d  svg+xml</dc:form
 00000280: 6174 3e0a 2020 2020 3c64 633a 6372 6561  at>.    <dc:crea
 00000290: 746f 723e 0a20 2020 2020 3c63 633a 4167  tor>.     <cc:Ag
 000002a0: 656e 743e 0a20 2020 2020 203c 6463 3a74  ent>.      <dc:t
 000002b0: 6974 6c65 3e4d 6174 706c 6f74 6c69 6220  itle>Matplotlib 
@@ -1303,15 +1303,15 @@
 00005160: 7472 6f6b 652d 6c69 6e65 6361 703a 2073  troke-linecap: s
 00005170: 7175 6172 6522 2f3e 0a20 2020 3c2f 673e  quare"/>.   </g>
 00005180: 0a20 203c 2f67 3e0a 2020 3c67 2069 643d  .  </g>.  <g id=
 00005190: 2274 6578 745f 3135 223e 0a20 2020 3c21  "text_15">.   <!
 000051a0: 2d2d 2068 7474 7073 3a2f 2f67 6974 6875  -- https://githu
 000051b0: 622e 636f 6d2f 6269 6f70 7261 676d 6174  b.com/biopragmat
 000051c0: 6963 732f 6269 6f72 6567 6973 7472 7920  ics/bioregistry 
-000051d0: 2832 3032 332d 3037 2d33 3029 202d 2d3e  (2023-07-30) -->
+000051d0: 2832 3032 332d 3038 2d30 3129 202d 2d3e  (2023-08-01) -->
 000051e0: 0a20 2020 3c67 2073 7479 6c65 3d22 6669  .   <g style="fi
 000051f0: 6c6c 3a20 2338 3038 3038 303b 206f 7061  ll: #808080; opa
 00005200: 6369 7479 3a20 302e 3522 2074 7261 6e73  city: 0.5" trans
 00005210: 666f 726d 3d22 7472 616e 736c 6174 6528  form="translate(
 00005220: 3539 322e 3333 3632 3520 3234 332e 3332  592.33625 243.32
 00005230: 3337 3529 2072 6f74 6174 6528 2d39 3029  375) rotate(-90)
 00005240: 2073 6361 6c65 2830 2e30 3820 2d30 2e30   scale(0.08 -0.0
@@ -1744,25 +1744,25 @@
 00006cf0: 732d 3264 2220 783d 2232 3536 372e 3432  s-2d" x="2567.42
 00006d00: 3737 3334 222f 3e0a 2020 2020 3c75 7365  7734"/>.    <use
 00006d10: 2078 6c69 6e6b 3a68 7265 663d 2223 4465   xlink:href="#De
 00006d20: 6a61 5675 5361 6e73 2d33 3022 2078 3d22  jaVuSans-30" x="
 00006d30: 3236 3033 2e35 3131 3731 3922 2f3e 0a20  2603.511719"/>. 
 00006d40: 2020 203c 7573 6520 786c 696e 6b3a 6872     <use xlink:hr
 00006d50: 6566 3d22 2344 656a 6156 7553 616e 732d  ef="#DejaVuSans-
-00006d60: 3337 2220 783d 2232 3636 372e 3133 3437  37" x="2667.1347
+00006d60: 3338 2220 783d 2232 3636 372e 3133 3437  38" x="2667.1347
 00006d70: 3636 222f 3e0a 2020 2020 3c75 7365 2078  66"/>.    <use x
 00006d80: 6c69 6e6b 3a68 7265 663d 2223 4465 6a61  link:href="#Deja
 00006d90: 5675 5361 6e73 2d32 6422 2078 3d22 3237  VuSans-2d" x="27
 00006da0: 3330 2e37 3537 3831 3222 2f3e 0a20 2020  30.757812"/>.   
 00006db0: 203c 7573 6520 786c 696e 6b3a 6872 6566   <use xlink:href
-00006dc0: 3d22 2344 656a 6156 7553 616e 732d 3333  ="#DejaVuSans-33
+00006dc0: 3d22 2344 656a 6156 7553 616e 732d 3330  ="#DejaVuSans-30
 00006dd0: 2220 783d 2232 3736 362e 3834 3137 3937  " x="2766.841797
 00006de0: 222f 3e0a 2020 2020 3c75 7365 2078 6c69  "/>.    <use xli
 00006df0: 6e6b 3a68 7265 663d 2223 4465 6a61 5675  nk:href="#DejaVu
-00006e00: 5361 6e73 2d33 3022 2078 3d22 3238 3330  Sans-30" x="2830
+00006e00: 5361 6e73 2d33 3122 2078 3d22 3238 3330  Sans-31" x="2830
 00006e10: 2e34 3634 3834 3422 2f3e 0a20 2020 203c  .464844"/>.    <
 00006e20: 7573 6520 786c 696e 6b3a 6872 6566 3d22  use xlink:href="
 00006e30: 2344 656a 6156 7553 616e 732d 3239 2220  #DejaVuSans-29" 
 00006e40: 783d 2232 3839 342e 3038 3738 3931 222f  x="2894.087891"/
 00006e50: 3e0a 2020 203c 2f67 3e0a 2020 3c2f 673e  >.   </g>.  </g>
 00006e60: 0a20 3c2f 673e 0a20 3c64 6566 733e 0a20  . </g>. <defs>. 
 00006e70: 203c 636c 6970 5061 7468 2069 643d 2270   <clipPath id="p
```

### Comparing `bioregistry-0.9.90/docs/img/regex_report.svg` & `bioregistry-0.9.91/docs/img/regex_report.svg`

 * *Files 0% similar despite different names*

```diff
@@ -29,16 +29,16 @@
 000001c0: 3032 2f32 322d 7264 662d 7379 6e74 6178  02/22-rdf-syntax
 000001d0: 2d6e 7323 223e 0a20 2020 3c63 633a 576f  -ns#">.   <cc:Wo
 000001e0: 726b 3e0a 2020 2020 3c64 633a 7479 7065  rk>.    <dc:type
 000001f0: 2072 6466 3a72 6573 6f75 7263 653d 2268   rdf:resource="h
 00000200: 7474 703a 2f2f 7075 726c 2e6f 7267 2f64  ttp://purl.org/d
 00000210: 632f 6463 6d69 7479 7065 2f53 7469 6c6c  c/dcmitype/Still
 00000220: 496d 6167 6522 2f3e 0a20 2020 203c 6463  Image"/>.    <dc
-00000230: 3a64 6174 653e 3230 3233 2d30 372d 3330  :date>2023-07-30
-00000240: 5430 313a 3230 3a32 322e 3437 3533 3738  T01:20:22.475378
+00000230: 3a64 6174 653e 3230 3233 2d30 382d 3031  :date>2023-08-01
+00000240: 5430 313a 3234 3a35 372e 3836 3330 3831  T01:24:57.863081
 00000250: 3c2f 6463 3a64 6174 653e 0a20 2020 203c  </dc:date>.    <
 00000260: 6463 3a66 6f72 6d61 743e 696d 6167 652f  dc:format>image/
 00000270: 7376 672b 786d 6c3c 2f64 633a 666f 726d  svg+xml</dc:form
 00000280: 6174 3e0a 2020 2020 3c64 633a 6372 6561  at>.    <dc:crea
 00000290: 746f 723e 0a20 2020 2020 3c63 633a 4167  tor>.     <cc:Ag
 000002a0: 656e 743e 0a20 2020 2020 203c 6463 3a74  ent>.      <dc:t
 000002b0: 6974 6c65 3e4d 6174 706c 6f74 6c69 6220  itle>Matplotlib
```

### Comparing `bioregistry-0.9.90/docs/img/xrefs.svg` & `bioregistry-0.9.91/docs/img/xrefs.svg`

 * *Files 0% similar despite different names*

```diff
@@ -29,16 +29,16 @@
 000001c0: 3032 2f32 322d 7264 662d 7379 6e74 6178  02/22-rdf-syntax
 000001d0: 2d6e 7323 223e 0a20 2020 3c63 633a 576f  -ns#">.   <cc:Wo
 000001e0: 726b 3e0a 2020 2020 3c64 633a 7479 7065  rk>.    <dc:type
 000001f0: 2072 6466 3a72 6573 6f75 7263 653d 2268   rdf:resource="h
 00000200: 7474 703a 2f2f 7075 726c 2e6f 7267 2f64  ttp://purl.org/d
 00000210: 632f 6463 6d69 7479 7065 2f53 7469 6c6c  c/dcmitype/Still
 00000220: 496d 6167 6522 2f3e 0a20 2020 203c 6463  Image"/>.    <dc
-00000230: 3a64 6174 653e 3230 3233 2d30 372d 3330  :date>2023-07-30
-00000240: 5430 313a 3136 3a33 322e 3735 3034 3539  T01:16:32.750459
+00000230: 3a64 6174 653e 3230 3233 2d30 382d 3031  :date>2023-08-01
+00000240: 5430 313a 3230 3a31 302e 3731 3131 3230  T01:20:10.711120
 00000250: 3c2f 6463 3a64 6174 653e 0a20 2020 203c  </dc:date>.    <
 00000260: 6463 3a66 6f72 6d61 743e 696d 6167 652f  dc:format>image/
 00000270: 7376 672b 786d 6c3c 2f64 633a 666f 726d  svg+xml</dc:form
 00000280: 6174 3e0a 2020 2020 3c64 633a 6372 6561  at>.    <dc:crea
 00000290: 746f 723e 0a20 2020 2020 3c63 633a 4167  tor>.     <cc:Ag
 000002a0: 656e 743e 0a20 2020 2020 203c 6463 3a74  ent>.      <dc:t
 000002b0: 6974 6c65 3e4d 6174 706c 6f74 6c69 6220  itle>Matplotlib 
@@ -2909,15 +2909,15 @@
 0000b5c0: 3835 3534 3722 2f3e 0a20 2020 203c 2f67  85547"/>.    </g
 0000b5d0: 3e0a 2020 203c 2f67 3e0a 2020 3c2f 673e  >.   </g>.  </g>
 0000b5e0: 0a20 203c 6720 6964 3d22 7465 7874 5f34  .  <g id="text_4
 0000b5f0: 3722 3e0a 2020 203c 212d 2d20 6874 7470  7">.   <!-- http
 0000b600: 733a 2f2f 6769 7468 7562 2e63 6f6d 2f62  s://github.com/b
 0000b610: 696f 7072 6167 6d61 7469 6373 2f62 696f  iopragmatics/bio
 0000b620: 7265 6769 7374 7279 2028 3230 3233 2d30  registry (2023-0
-0000b630: 372d 3330 2920 2d2d 3e0a 2020 203c 6720  7-30) -->.   <g 
+0000b630: 382d 3031 2920 2d2d 3e0a 2020 203c 6720  8-01) -->.   <g 
 0000b640: 7374 796c 653d 2266 696c 6c3a 2023 3830  style="fill: #80
 0000b650: 3830 3830 3b20 6f70 6163 6974 793a 2030  8080; opacity: 0
 0000b660: 2e35 2220 7472 616e 7366 6f72 6d3d 2274  .5" transform="t
 0000b670: 7261 6e73 6c61 7465 2835 3932 2e33 3336  ranslate(592.336
 0000b680: 3235 2032 3433 2e33 3233 3735 2920 726f  25 243.32375) ro
 0000b690: 7461 7465 282d 3930 2920 7363 616c 6528  tate(-90) scale(
 0000b6a0: 302e 3038 202d 302e 3038 2922 3e0a 2020  0.08 -0.08)">.  
@@ -3177,26 +3177,26 @@
 0000c680: 3d22 2344 656a 6156 7553 616e 732d 3264  ="#DejaVuSans-2d
 0000c690: 2220 783d 2232 3536 372e 3432 3737 3334  " x="2567.427734
 0000c6a0: 222f 3e0a 2020 2020 3c75 7365 2078 6c69  "/>.    <use xli
 0000c6b0: 6e6b 3a68 7265 663d 2223 4465 6a61 5675  nk:href="#DejaVu
 0000c6c0: 5361 6e73 2d33 3022 2078 3d22 3236 3033  Sans-30" x="2603
 0000c6d0: 2e35 3131 3731 3922 2f3e 0a20 2020 203c  .511719"/>.    <
 0000c6e0: 7573 6520 786c 696e 6b3a 6872 6566 3d22  use xlink:href="
-0000c6f0: 2344 656a 6156 7553 616e 732d 3337 2220  #DejaVuSans-37" 
+0000c6f0: 2344 656a 6156 7553 616e 732d 3338 2220  #DejaVuSans-38" 
 0000c700: 783d 2232 3636 372e 3133 3437 3636 222f  x="2667.134766"/
 0000c710: 3e0a 2020 2020 3c75 7365 2078 6c69 6e6b  >.    <use xlink
 0000c720: 3a68 7265 663d 2223 4465 6a61 5675 5361  :href="#DejaVuSa
 0000c730: 6e73 2d32 6422 2078 3d22 3237 3330 2e37  ns-2d" x="2730.7
 0000c740: 3537 3831 3222 2f3e 0a20 2020 203c 7573  57812"/>.    <us
 0000c750: 6520 786c 696e 6b3a 6872 6566 3d22 2344  e xlink:href="#D
-0000c760: 656a 6156 7553 616e 732d 3333 2220 783d  ejaVuSans-33" x=
+0000c760: 656a 6156 7553 616e 732d 3330 2220 783d  ejaVuSans-30" x=
 0000c770: 2232 3736 362e 3834 3137 3937 222f 3e0a  "2766.841797"/>.
 0000c780: 2020 2020 3c75 7365 2078 6c69 6e6b 3a68      <use xlink:h
 0000c790: 7265 663d 2223 4465 6a61 5675 5361 6e73  ref="#DejaVuSans
-0000c7a0: 2d33 3022 2078 3d22 3238 3330 2e34 3634  -30" x="2830.464
+0000c7a0: 2d33 3122 2078 3d22 3238 3330 2e34 3634  -31" x="2830.464
 0000c7b0: 3834 3422 2f3e 0a20 2020 203c 7573 6520  844"/>.    <use 
 0000c7c0: 786c 696e 6b3a 6872 6566 3d22 2344 656a  xlink:href="#Dej
 0000c7d0: 6156 7553 616e 732d 3239 2220 783d 2232  aVuSans-29" x="2
 0000c7e0: 3839 342e 3038 3738 3931 222f 3e0a 2020  894.087891"/>.  
 0000c7f0: 203c 2f67 3e0a 2020 3c2f 673e 0a20 3c2f   </g>.  </g>. </
 0000c800: 673e 0a20 3c64 6566 733e 0a20 203c 636c  g>. <defs>.  <cl
 0000c810: 6970 5061 7468 2069 643d 2270 6235 6638  ipPath id="pb5f8
```

### Comparing `bioregistry-0.9.90/docs/source/conf.py` & `bioregistry-0.9.91/docs/source/conf.py`

 * *Files 1% similar despite different names*

```diff
@@ -23,15 +23,15 @@
 # -- Project information -----------------------------------------------------
 
 project = "bioregistry"
 copyright = f"{date.today().year}, Charles Tapley Hoyt"
 author = "Charles Tapley Hoyt"
 
 # The full version, including alpha/beta/rc tags.
-release = "0.9.90"
+release = "0.9.91"
 
 # The short X.Y version.
 parsed_version = re.match(
     "(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?:-(?P<release>[0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+(?P<build>[0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?",
     release,
 )
 version = parsed_version.expand("\g<major>.\g<minor>.\g<patch>")
```

### Comparing `bioregistry-0.9.90/docs/source/deployment.rst` & `bioregistry-0.9.91/docs/source/deployment.rst`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/docs/source/index.rst` & `bioregistry-0.9.91/docs/source/index.rst`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/docs/source/logo.png` & `bioregistry-0.9.91/docs/source/logo.png`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/setup.cfg` & `bioregistry-0.9.91/setup.cfg`

 * *Files 2% similar despite different names*

```diff
@@ -1,10 +1,10 @@
 [metadata]
 name = bioregistry
-version = 0.9.90
+version = 0.9.91
 description = Integrated registry of biological databases and nomenclatures
 long_description = file: README.md
 long_description_content_type = text/markdown
 url = https://github.com/biopragmatics/bioregistry
 download_url = https://github.com/biopragmatics/bioregistry/releases
 project_urls = 
 	Bug Tracker = https://github.com/biopragmatics/bioregistry/issues
```

### Comparing `bioregistry-0.9.90/src/bioregistry/__init__.py` & `bioregistry-0.9.91/src/bioregistry/__init__.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/__init__.py` & `bioregistry-0.9.91/src/bioregistry/align/__init__.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/biocontext.py` & `bioregistry-0.9.91/src/bioregistry/align/biocontext.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/biolink.py` & `bioregistry-0.9.91/src/bioregistry/align/biolink.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/bioportal.py` & `bioregistry-0.9.91/src/bioregistry/align/bioportal.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/cellosaurus.py` & `bioregistry-0.9.91/src/bioregistry/align/cellosaurus.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/cheminf.py` & `bioregistry-0.9.91/src/bioregistry/align/cheminf.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/edam.py` & `bioregistry-0.9.91/src/bioregistry/align/edam.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/fairsharing.py` & `bioregistry-0.9.91/src/bioregistry/align/fairsharing.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/go.py` & `bioregistry-0.9.91/src/bioregistry/align/go.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/hl7.py` & `bioregistry-0.9.91/src/bioregistry/align/hl7.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/n2t.py` & `bioregistry-0.9.91/src/bioregistry/align/n2t.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/ncbi.py` & `bioregistry-0.9.91/src/bioregistry/align/ncbi.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/obofoundry.py` & `bioregistry-0.9.91/src/bioregistry/align/obofoundry.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/ols.py` & `bioregistry-0.9.91/src/bioregistry/align/ols.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/ontobee.py` & `bioregistry-0.9.91/src/bioregistry/align/ontobee.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/prefixcommons.py` & `bioregistry-0.9.91/src/bioregistry/align/prefixcommons.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/re3data.py` & `bioregistry-0.9.91/src/bioregistry/align/re3data.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/uniprot.py` & `bioregistry-0.9.91/src/bioregistry/align/uniprot.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/utils.py` & `bioregistry-0.9.91/src/bioregistry/align/utils.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/align/wikidata.py` & `bioregistry-0.9.91/src/bioregistry/align/wikidata.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/analysis/title_tfidf.py` & `bioregistry-0.9.91/src/bioregistry/analysis/title_tfidf.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/api.py` & `bioregistry-0.9.91/src/bioregistry/app/api.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/cli.py` & `bioregistry-0.9.91/src/bioregistry/app/cli.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/constants.py` & `bioregistry-0.9.91/src/bioregistry/app/constants.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/impl.py` & `bioregistry-0.9.91/src/bioregistry/app/impl.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/static/logo.svg` & `bioregistry-0.9.91/src/bioregistry/app/static/logo.svg`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/base.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/base.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/collection.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/collection.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/collections.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/collections.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/context.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/context.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/contexts.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/contexts.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/contributor.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/contributor.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/contributors.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/contributors.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/highlights/keywords.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/highlights/keywords.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/highlights/owners.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/highlights/owners.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/highlights/relations.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/highlights/relations.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/highlights/twitter.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/highlights/twitter.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/home.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/home.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/macros.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/macros.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/meta/access.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/meta/access.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/meta/acknowledgements.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/meta/acknowledgements.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/meta/download.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/meta/download.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/meta/related.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/meta/related.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/meta/schema.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/meta/schema.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/meta/summary.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/meta/summary.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/meta/sustainability.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/meta/sustainability.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/metaresource.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/metaresource.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/metaresources.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/metaresources.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/prose.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/prose.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/reference.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/reference.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/disallowed_identifier.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/disallowed_identifier.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/invalid_identifier.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/invalid_identifier.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/missing_prefix.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/missing_prefix.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/resolve_errors/missing_providers.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/resolve_errors/missing_providers.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/resource.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/resource.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/templates/resources.html` & `bioregistry-0.9.91/src/bioregistry/app/templates/resources.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/test.py` & `bioregistry-0.9.91/src/bioregistry/app/test.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/ui.py` & `bioregistry-0.9.91/src/bioregistry/app/ui.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/app/utils.py` & `bioregistry-0.9.91/src/bioregistry/app/utils.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/benchmarks/cli.py` & `bioregistry-0.9.91/src/bioregistry/benchmarks/cli.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/benchmarks/curie_parsing.py` & `bioregistry-0.9.91/src/bioregistry/benchmarks/curie_parsing.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/benchmarks/curie_validation.py` & `bioregistry-0.9.91/src/bioregistry/benchmarks/curie_validation.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/benchmarks/uri_parsing.py` & `bioregistry-0.9.91/src/bioregistry/benchmarks/uri_parsing.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/bibliometrics.py` & `bioregistry-0.9.91/src/bioregistry/bibliometrics.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/cli.py` & `bioregistry-0.9.91/src/bioregistry/cli.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/collection_api.py` & `bioregistry-0.9.91/src/bioregistry/collection_api.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/compare.py` & `bioregistry-0.9.91/src/bioregistry/compare.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/constants.py` & `bioregistry-0.9.91/src/bioregistry/constants.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/add_co_providers.py` & `bioregistry-0.9.91/src/bioregistry/curation/add_co_providers.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/add_descriptions_from_gs.py` & `bioregistry-0.9.91/src/bioregistry/curation/add_descriptions_from_gs.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/add_examples.py` & `bioregistry-0.9.91/src/bioregistry/curation/add_examples.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/add_examples_from_javert.py` & `bioregistry-0.9.91/src/bioregistry/curation/add_examples_from_javert.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/add_examples_from_ols.py` & `bioregistry-0.9.91/src/bioregistry/curation/add_examples_from_ols.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/add_ontology_regexes.py` & `bioregistry-0.9.91/src/bioregistry/curation/add_ontology_regexes.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/bulk_import.py` & `bioregistry-0.9.91/src/bioregistry/curation/bulk_import.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/clean_licenses.py` & `bioregistry-0.9.91/src/bioregistry/curation/clean_licenses.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/clean_name_suffixes.py` & `bioregistry-0.9.91/src/bioregistry/curation/clean_name_suffixes.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/clean_publications.py` & `bioregistry-0.9.91/src/bioregistry/curation/clean_publications.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/cleanup_authors.py` & `bioregistry-0.9.91/src/bioregistry/curation/cleanup_authors.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/deprecation_diff.py` & `bioregistry-0.9.91/src/bioregistry/curation/deprecation_diff.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/enrich_publications.py` & `bioregistry-0.9.91/src/bioregistry/curation/enrich_publications.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/import_pc_semiautomatic.py` & `bioregistry-0.9.91/src/bioregistry/curation/import_pc_semiautomatic.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/make_description_curation_sheet.py` & `bioregistry-0.9.91/src/bioregistry/curation/make_description_curation_sheet.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/map_bartoc_via_wikidata.py` & `bioregistry-0.9.91/src/bioregistry/curation/map_bartoc_via_wikidata.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/map_re3data_via_fairsharing.py` & `bioregistry-0.9.91/src/bioregistry/curation/map_re3data_via_fairsharing.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/rename_metaprefix.py` & `bioregistry-0.9.91/src/bioregistry/curation/rename_metaprefix.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/review_pc.py` & `bioregistry-0.9.91/src/bioregistry/curation/review_pc.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/suggest_author_curation.py` & `bioregistry-0.9.91/src/bioregistry/curation/suggest_author_curation.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/curation/suggest_uniprot_providers.py` & `bioregistry-0.9.91/src/bioregistry/curation/suggest_uniprot_providers.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/README.md` & `bioregistry-0.9.91/src/bioregistry/data/README.md`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/bioregistry.json` & `bioregistry-0.9.91/src/bioregistry/data/bioregistry.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/collections.json` & `bioregistry-0.9.91/src/bioregistry/data/collections.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/contexts.json` & `bioregistry-0.9.91/src/bioregistry/data/contexts.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/aberowl/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/aberowl/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/aberowl/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/aberowl/processed.json`

 * *Files 0% similar despite different names*

#### Pretty-printed

 * *Similarity: 0.9996879334257976%*

 * *Differences: {"'CMO'": "{'download_owl': 'http://aber-owl.net/media/ontologies/CMO/94/cmo.owl'}",*

 * * "'DOID'": "{'download_owl': 'http://aber-owl.net/media/ontologies/DOID/672/doid.owl'}",*

 * * "'FYPO'": "{'download_owl': 'http://aber-owl.net/media/ontologies/FYPO/443/fypo.owl'}",*

 * * "'GNO'": "{'download_owl': 'http://aber-owl.net/media/ontologies/GNO/20/gno.owl'}",*

 * * "'MP'": "{'download_owl': 'http://aber-owl.net/media/ontologies/MP/98/mp.owl'}",*

 * * "'OBI'": "{'download_owl': 'http://aber-owl.net/media/ontologies/OBI/51/obi.owl'} […]*

```diff
@@ -1338,15 +1338,15 @@
     },
     "CMF": {
         "name": "CranioMaxilloFacial ontology",
         "prefix": "CMF"
     },
     "CMO": {
         "description": "Morphological and physiological measurement records generated from clinical and model organism research and health programs.",
-        "download_owl": "http://aber-owl.net/media/ontologies/CMO/93/cmo.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/CMO/94/cmo.owl",
         "homepage": "http://rgd.mcw.edu/rgdweb/ontology/search.html",
         "name": "Clinical measurement ontology",
         "prefix": "CMO"
     },
     "CMOBET": {
         "name": "agenbola",
         "prefix": "CMOBET"
@@ -1998,15 +1998,15 @@
         "download_owl": "http://aber-owl.net/media/ontologies/DOCCC/3/doccc.owl",
         "name": "Diagnosis Ontology of Clinical Care Classification",
         "prefix": "DOCCC",
         "version": "20131004"
     },
     "DOID": {
         "description": "An ontology for describing the classification of human diseases organized by etiology.",
-        "download_owl": "http://aber-owl.net/media/ontologies/DOID/671/doid.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/DOID/672/doid.owl",
         "homepage": "http://www.disease-ontology.org",
         "name": "Human Disease Ontology",
         "prefix": "DOID"
     },
     "DOIDSH": {
         "download_owl": "http://aber-owl.net/media/ontologies/DOIDSH/3/doidsh.owl",
         "name": "Diseases Ontology - Shrinked version",
@@ -2936,15 +2936,15 @@
         "download_owl": "http://aber-owl.net/media/ontologies/FUNCTIONALH/3/functionalh.owl",
         "name": "NeuroMorpho.Org cell types function",
         "prefix": "FUNCTIONALH",
         "version": "2.0"
     },
     "FYPO": {
         "description": "FYPO is a formal ontology of phenotypes observed in fission yeast.",
-        "download_owl": "http://aber-owl.net/media/ontologies/FYPO/442/fypo.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/FYPO/443/fypo.owl",
         "homepage": "https://github.com/pombase/fypo",
         "name": "Fission Yeast Phenotype Ontology",
         "prefix": "FYPO"
     },
     "FaBiO": {
         "description": "This file is imported by the vivo-core-public-1.5.owl file. It contains terms relating to documents from the http://purl.org/spar/fabio/ namespace that are included in the VIVO ontology. FaBiO is a FRBR-aligned Bibliographic Ontology that is part of the Semantic Publishing and Referencing (SPAR) suite of ontologies.",
         "download_owl": "http://aber-owl.net/media/ontologies/FaBiO/1/fabio.owl",
@@ -3211,15 +3211,15 @@
         "download_owl": "http://aber-owl.net/media/ontologies/GMO/6/gmo.owl",
         "name": "Growth Medium Ontology",
         "prefix": "GMO",
         "version": "0.16 Beta"
     },
     "GNO": {
         "description": "GlyTouCan provides stable accessions for glycans described at varyious degrees of characterization, including compositions (no linkage) and topologies (no carbon bond positions or anomeric configurations). GNOme organizes these stable accessions for interative browsing, for text-based searching, and for automated reasoning with well-defined characterization levels.",
-        "download_owl": "http://aber-owl.net/media/ontologies/GNO/19/gno.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/GNO/20/gno.owl",
         "homepage": "https://gnome.glyomics.org/",
         "name": "Glycan Naming and Subsumption Ontology (GNOme)",
         "prefix": "GNO"
     },
     "GO": {
         "description": "An ontology for describing the function of genes and gene products",
         "download_owl": "http://aber-owl.net/media/ontologies/GO/1780/go.owl",
@@ -5393,15 +5393,15 @@
     },
     "MOVIE": {
         "name": "Nonton Gratis Movie Online",
         "prefix": "MOVIE"
     },
     "MP": {
         "description": "Standard terms for annotating mammalian phenotypic data.",
-        "download_owl": "http://aber-owl.net/media/ontologies/MP/97/mp.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/MP/98/mp.owl",
         "homepage": "http://www.informatics.jax.org/searches/MP_form.shtml",
         "name": "Mammalian Phenotype Ontology",
         "prefix": "MP"
     },
     "MPATH": {
         "description": "A structured controlled vocabulary of mutant and transgenic mouse pathology phenotypes",
         "download_owl": "http://aber-owl.net/media/ontologies/MPATH/173/mpath.owl",
@@ -6052,15 +6052,15 @@
         "download_owl": "http://aber-owl.net/media/ontologies/OBCS/15/obcs.owl",
         "homepage": "https://github.com/obcs/obcs",
         "name": "Ontology of Biological and Clinical Statistics",
         "prefix": "OBCS"
     },
     "OBI": {
         "description": "An integrated ontology for the description of life-science and clinical investigations",
-        "download_owl": "http://aber-owl.net/media/ontologies/OBI/50/obi.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/OBI/51/obi.owl",
         "homepage": "http://obi-ontology.org",
         "name": "Ontology for Biomedical Investigations",
         "prefix": "OBI"
     },
     "OBIB": {
         "description": "An ontology built for annotation and modeling of biobank repository and biobanking administration",
         "download_owl": "http://aber-owl.net/media/ontologies/OBIB/15/obib.owl",
@@ -6377,15 +6377,15 @@
         "download_owl": "http://aber-owl.net/media/ontologies/OMIT/41/omit.owl",
         "homepage": "http://omit.cis.usouthal.edu/",
         "name": "Ontology for MIRNA Target",
         "prefix": "OMIT"
     },
     "OMO": {
         "description": "An ontology specifies terms that are used to annotate ontology terms for all OBO ontologies. The ontology was developed as part of Information Artifact Ontology (IAO).",
-        "download_owl": "http://aber-owl.net/media/ontologies/OMO/6/omo.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/OMO/7/omo.owl",
         "homepage": "https://github.com/information-artifact-ontology/ontology-metadata",
         "name": "OBO Metadata Ontology",
         "prefix": "OMO"
     },
     "OMP": {
         "description": "An ontology of phenotypes covering microbes",
         "download_owl": "http://aber-owl.net/media/ontologies/OMP/56/omp.owl",
@@ -7105,15 +7105,15 @@
         "download_owl": "http://aber-owl.net/media/ontologies/PEAO/1/peao.owl",
         "homepage": "https://bitbucket.org/PlantExpAssay/ontology",
         "name": "Plant Experimental Assay Ontology",
         "prefix": "PEAO"
     },
     "PECO": {
         "description": "A structured, controlled vocabulary which describes the treatments, growing conditions, and/or study types used in plant biology experiments.",
-        "download_owl": "http://aber-owl.net/media/ontologies/PECO/28/peco.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/PECO/29/peco.owl",
         "homepage": "http://planteome.org/",
         "name": "Plant Experimental Conditions Ontology",
         "prefix": "PECO"
     },
     "PEDTERM": {
         "description": "Terms associated with pediatrics, representing information related to child health and development from pre-birth through 21 years of age; contributed by the National Institute of Child Health and Human Development.",
         "download_owl": "http://aber-owl.net/media/ontologies/PEDTERM/5/pedterm.owl",
@@ -8941,15 +8941,15 @@
     },
     "TYSON": {
         "name": "ZTWOEM",
         "prefix": "TYSON"
     },
     "UBERON": {
         "description": "An integrated cross-species anatomy ontology covering animals and bridging multiple species-specific ontologies",
-        "download_owl": "http://aber-owl.net/media/ontologies/UBERON/293/uberon.owl",
+        "download_owl": "http://aber-owl.net/media/ontologies/UBERON/294/uberon.owl",
         "homepage": "http://uberon.org",
         "name": "Uberon multi-species anatomy ontology",
         "prefix": "UBERON"
     },
     "UGANDAFACILITIES": {
         "description": "This is the list of health facilities registered under VODANA Project.",
         "name": "UGANDAFACILITIES",
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/aberowl/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/aberowl/raw.json`

 * *Files 0% similar despite different names*

#### Pretty-printed

 * *Similarity: 0.8117184183087254%*

 * *Differences: {'1000': "{'acronym': 'MPATH', 'name': 'Mouse pathology ontology', 'submission': {'id': 3775, "*

 * *         "'download_url': 'media/ontologies/MPATH/173/mpath.owl', 'submission_id': 173, 'domain': "*

 * *         "'health', 'description': 'A structured controlled vocabulary of mutant and transgenic "*

 * *         "mouse pathology phenotypes', 'documentation': None, 'products': [OrderedDict([('id', "*

 * *         "'mpath.owl'), ('ontology_purl', 'http://purl.obolibrary.org/obo/mpath.owl')])], 'taxon': "*

 * *         "{replace:  […]*

```diff
@@ -46,44 +46,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2009-05-04"
         },
         "topics": null
     },
     {
-        "acronym": "ACVD_ONTOLOGY",
-        "name": "Atherosclerotic Cerebrovascular Disease Ontology",
+        "acronym": "CEI-10",
+        "name": "CEI-10_Garrahan (test)",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2023-06-08T08:50:54Z",
-            "date_released": "2023-06-08T00:00:00Z",
-            "description": "The ontology describes the related aspects of atherosclerotic cerebrovascular disease, and provides a clear set of cross-lingual concepts and terms with an explicit hierarchical structure, helping scientific researchers to quickly retrieve relevant medical literature, assisting data scientists to efficiently identify relevant contents in electronic health records, and providing a clear domain framework for academic reference. The current ontology included 10 top-level classes, respectively clinical manifestation, comorbidity, complication, diagnosis, model of atherosclerotic cerebrovascular disease, pathogenesis, prevention, rehabilitation, risk factor, and treatment. Totally, there are 1715 concepts in the 11-level ontology, covering 4588 Chinese terms, 6617 English terms, and 972 definitions.",
+            "date_created": "2018-03-22T02:20:47Z",
+            "date_released": "2018-03-06T00:00:00Z",
+            "description": "Es la Ontologia del Hospital Garrahan de Pediatria",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ACVD_ONTOLOGY/2/acvd_ontology.owl",
+            "download_url": "media/ontologies/CEI-10/8/cei-10.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5869,
+            "id": 1349,
             "indexed": true,
-            "max_children": 93,
-            "max_depth": 11,
+            "max_children": 35,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 1715,
+            "nb_classes": 14502,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 8,
             "taxon": null,
-            "version": "1.0"
+            "version": "0"
         },
         "topics": null
     },
     {
         "acronym": "SNOMED_ANATOMY",
         "name": "SNOMED Anatomy",
         "species": null,
@@ -114,44 +114,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.2"
         },
         "topics": null
     },
     {
-        "acronym": "ADAR",
-        "name": "Autism DSM-ADI-R ontology",
+        "acronym": "ACVD_ONTOLOGY",
+        "name": "Atherosclerotic Cerebrovascular Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2015-04-07T23:36:38Z",
-            "date_released": "2015-03-27T00:00:00Z",
-            "description": "The autism DSM-ADI-R (ADAR) ontology extends an ontology by Tu et al. with additional SWRL rules to infer phenotypes from ADI-R items.\r\nIt includes OWL class definitions representing DSM IV diagnostic criteria for autistic disorder and ASD criteria for Autism Spectrum Disorder\r\nOWL Reasoners can thus infer autism-related phenotypes from ADI-R questionnaire results \r\n\r\nTu SW, Tennakoon L, Connor MO, Das A. Using an Integrated Ontology and Information Model for Querying and Reasoning about Phenotypes\u202f: The Case of Autism. AMIA Annu Symp Proc 2008;727\u201331\r\n",
+            "date_created": "2023-06-08T08:50:54Z",
+            "date_released": "2023-06-08T00:00:00Z",
+            "description": "The ontology describes the related aspects of atherosclerotic cerebrovascular disease, and provides a clear set of cross-lingual concepts and terms with an explicit hierarchical structure, helping scientific researchers to quickly retrieve relevant medical literature, assisting data scientists to efficiently identify relevant contents in electronic health records, and providing a clear domain framework for academic reference. The current ontology included 10 top-level classes, respectively clinical manifestation, comorbidity, complication, diagnosis, model of atherosclerotic cerebrovascular disease, pathogenesis, prevention, rehabilitation, risk factor, and treatment. Totally, there are 1715 concepts in the 11-level ontology, covering 4588 Chinese terms, 6617 English terms, and 972 definitions.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ADAR/1/adar.owl",
+            "download_url": "media/ontologies/ACVD_ONTOLOGY/2/acvd_ontology.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 453,
+            "id": 5869,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 13,
+            "max_children": 93,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 791,
+            "nb_classes": 1715,
             "nb_inconsistent": 0,
-            "nb_individuals": 646,
-            "nb_properties": 1448,
+            "nb_individuals": 0,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": "1.0"
         },
         "topics": null
     },
     {
         "acronym": "LINKALTERNATIF",
         "name": "santikumala",
         "species": null,
@@ -402,48 +402,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "Generated 2013-03-28T19:01:04.897+01:00"
         },
         "topics": null
     },
     {
-        "acronym": "CEI-10",
-        "name": "CEI-10_Garrahan (test)",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2018-03-22T02:20:47Z",
-            "date_released": "2018-03-06T00:00:00Z",
-            "description": "Es la Ontologia del Hospital Garrahan de Pediatria",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/CEI-10/8/cei-10.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 1349,
-            "indexed": true,
-            "max_children": 35,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 14502,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 8,
-            "taxon": null,
-            "version": "0"
-        },
-        "topics": null
-    },
-    {
         "acronym": "APATREATMENT",
         "name": "APA Treatment Cluster",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 48,
             "classifiable": false,
@@ -1311,14 +1277,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
+        "acronym": "ADAR",
+        "name": "Autism DSM-ADI-R ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2015-04-07T23:36:38Z",
+            "date_released": "2015-03-27T00:00:00Z",
+            "description": "The autism DSM-ADI-R (ADAR) ontology extends an ontology by Tu et al. with additional SWRL rules to infer phenotypes from ADI-R items.\r\nIt includes OWL class definitions representing DSM IV diagnostic criteria for autistic disorder and ASD criteria for Autism Spectrum Disorder\r\nOWL Reasoners can thus infer autism-related phenotypes from ADI-R questionnaire results \r\n\r\nTu SW, Tennakoon L, Connor MO, Das A. Using an Integrated Ontology and Information Model for Querying and Reasoning about Phenotypes\u202f: The Case of Autism. AMIA Annu Symp Proc 2008;727\u201331\r\n",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/ADAR/1/adar.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 453,
+            "indexed": true,
+            "max_children": 14,
+            "max_depth": 13,
+            "md5sum": null,
+            "nb_classes": 791,
+            "nb_inconsistent": 0,
+            "nb_individuals": 646,
+            "nb_properties": 1448,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "MEDDRA",
         "name": "Medical Dictionary for Regulatory Activities Terminology (MedDRA)",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 15,
             "classifiable": false,
@@ -1739,44 +1739,44 @@
             "submission_id": 9,
             "taxon": null,
             "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "GALEN",
-        "name": "Galen Ontology",
+        "acronym": "GBOL",
+        "name": "GBOL",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2007-01-16T21:51:57Z",
-            "date_released": "2007-01-09T08:00:00Z",
-            "description": "A translation of the full Galen ontology (from the OpenGALEN project) into the OWL description logic.",
-            "documentation": null,
+            "date_created": "2017-11-28T14:53:41Z",
+            "date_released": "2017-09-05T00:00:00Z",
+            "description": " Genome Biology Ontology Language (GBOL) ",
+            "documentation": "http://gbol.life",
             "domain": null,
-            "download_url": "media/ontologies/GALEN/1/galen.owl",
+            "download_url": "media/ontologies/GBOL/1/gbol.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.co-ode.org/galen/",
-            "id": 419,
+            "home_page": "http://gbol.life",
+            "id": 881,
             "indexed": true,
-            "max_children": 1492,
-            "max_depth": 24,
+            "max_children": 34,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 23141,
+            "nb_classes": 271,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 950,
+            "nb_individuals": 805,
+            "nb_properties": 142,
             "products": null,
-            "publication": null,
+            "publication": "https://www.biorxiv.org/content/early/2017/09/05/184747",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": "1"
         },
         "topics": null
     },
     {
         "acronym": "CU-VO",
         "name": "Venom Ontology",
         "species": null,
@@ -3014,180 +3014,146 @@
         "submission": null,
         "topics": [
             "restaurant",
             "fast food"
         ]
     },
     {
-        "acronym": "DECIDE",
-        "name": "DECIDE ONTOLOGY",
+        "acronym": "DRPSNPTO",
+        "name": "Dementia-Related Psychotic Symptoms Non-Pharmacological Treatment Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2022-12-03T23:35:27Z",
-            "date_released": "2022-12-03T00:00:00Z",
-            "description": "Ontology is developed  for decision support tools for different animal species as a part of DECIDE project (https://decideproject.eu).",
+            "date_created": "2021-03-08T23:41:27Z",
+            "date_released": "2021-03-08T00:00:00Z",
+            "description": "The DRPSNPTO ontology represents the domain knowledge specific to non-pharmacological treatment of psychotic symptoms in dementia in the long-term care setting.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DECIDE/1/decide.owl",
+            "download_url": "media/ontologies/DRPSNPTO/1/drpsnpto.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/decide-project-eu",
-            "id": 5528,
+            "home_page": null,
+            "id": 4329,
             "indexed": true,
-            "max_children": 15,
-            "max_depth": 4,
+            "max_children": 26,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 19,
+            "nb_classes": 610,
             "nb_inconsistent": 0,
-            "nb_individuals": 25,
-            "nb_properties": 14,
+            "nb_individuals": 0,
+            "nb_properties": 60,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "GAMUTS",
-        "name": "Radiology Gamuts Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 10,
-            "classifiable": true,
-            "date_created": "2020-09-30T20:49:35Z",
-            "date_released": "2018-10-15T00:00:00Z",
-            "description": "In radiology, a \"gamut\" is the set of conditions that can cause a specified imaging finding.  The Radiology Gamuts Ontology (RGO) links more than 16,000 diseases and imaging findings.  The causal relation (may_cause; inverse, may_be_caused_by) connects findings and possible causes.\r\n\r\nThis work is made available under provisions of the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) license.",
-            "documentation": "https://www.gamuts.net/about.php",
-            "domain": null,
-            "download_url": "media/ontologies/GAMUTS/24/gamuts.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.gamuts.net/",
-            "id": 3967,
-            "indexed": true,
-            "max_children": 12471,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 18001,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": "https://www.gamuts.net/about.php",
-            "publications": null,
-            "submission_id": 24,
-            "taxon": null,
-            "version": "0.92"
+            "version": "v1.0"
         },
         "topics": null
     },
     {
-        "acronym": "DRPSNPTO",
-        "name": "Dementia-Related Psychotic Symptoms Non-Pharmacological Treatment Ontology",
+        "acronym": "GALEN",
+        "name": "Galen Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-03-08T23:41:27Z",
-            "date_released": "2021-03-08T00:00:00Z",
-            "description": "The DRPSNPTO ontology represents the domain knowledge specific to non-pharmacological treatment of psychotic symptoms in dementia in the long-term care setting.",
+            "date_created": "2007-01-16T21:51:57Z",
+            "date_released": "2007-01-09T08:00:00Z",
+            "description": "A translation of the full Galen ontology (from the OpenGALEN project) into the OWL description logic.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DRPSNPTO/1/drpsnpto.owl",
+            "download_url": "media/ontologies/GALEN/1/galen.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4329,
+            "home_page": "http://www.co-ode.org/galen/",
+            "id": 419,
             "indexed": true,
-            "max_children": 26,
-            "max_depth": 8,
+            "max_children": 1492,
+            "max_depth": 24,
             "md5sum": null,
-            "nb_classes": 610,
+            "nb_classes": 23141,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 60,
+            "nb_properties": 950,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "v1.0"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "GB",
-        "name": "GlycanBind",
+        "acronym": "DECIDE",
+        "name": "DECIDE ONTOLOGY",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-06-13T12:04:05Z",
-            "date_released": "2023-06-11T00:00:00Z",
-            "description": "GlycanBind is an ontology for representing glycan and pathogen binding interaction data in RDF.\r\nMany of the terms are based on the data originally contained in the SugarBind database.",
+            "date_created": "2022-12-03T23:35:27Z",
+            "date_released": "2022-12-03T00:00:00Z",
+            "description": "Ontology is developed  for decision support tools for different animal species as a part of DECIDE project (https://decideproject.eu).",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GB/3/gb.owl",
+            "download_url": "media/ontologies/DECIDE/1/decide.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5883,
+            "home_page": "https://github.com/decide-project-eu",
+            "id": 5528,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 2,
+            "max_children": 15,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 13,
+            "nb_classes": 19,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 11,
+            "nb_individuals": 25,
+            "nb_properties": 14,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.1.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "GBOL",
-        "name": "GBOL",
+        "acronym": "GAMUTS",
+        "name": "Radiology Gamuts Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 10,
             "classifiable": true,
-            "date_created": "2017-11-28T14:53:41Z",
-            "date_released": "2017-09-05T00:00:00Z",
-            "description": " Genome Biology Ontology Language (GBOL) ",
-            "documentation": "http://gbol.life",
+            "date_created": "2020-09-30T20:49:35Z",
+            "date_released": "2018-10-15T00:00:00Z",
+            "description": "In radiology, a \"gamut\" is the set of conditions that can cause a specified imaging finding.  The Radiology Gamuts Ontology (RGO) links more than 16,000 diseases and imaging findings.  The causal relation (may_cause; inverse, may_be_caused_by) connects findings and possible causes.\r\n\r\nThis work is made available under provisions of the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) license.",
+            "documentation": "https://www.gamuts.net/about.php",
             "domain": null,
-            "download_url": "media/ontologies/GBOL/1/gbol.owl",
+            "download_url": "media/ontologies/GAMUTS/24/gamuts.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://gbol.life",
-            "id": 881,
+            "home_page": "https://www.gamuts.net/",
+            "id": 3967,
             "indexed": true,
-            "max_children": 34,
-            "max_depth": 6,
+            "max_children": 12471,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 271,
+            "nb_classes": 18001,
             "nb_inconsistent": 0,
-            "nb_individuals": 805,
-            "nb_properties": 142,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://www.biorxiv.org/content/early/2017/09/05/184747",
+            "publication": "https://www.gamuts.net/about.php",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 24,
             "taxon": null,
-            "version": "1"
+            "version": "0.92"
         },
         "topics": null
     },
     {
         "acronym": "NEUROBRG",
         "name": "NeuroBridge Ontology",
         "species": null,
@@ -3258,15 +3224,15 @@
         },
         "topics": null
     },
     {
         "acronym": "FNS-H",
         "name": "FNS-Harmony",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
             "date_created": "2022-10-25T10:32:58Z",
             "date_released": "2022-10-25T00:00:00Z",
             "description": "Application ontology for the FNS-Cloud project ontology PURL: https://purl.org/fns-h",
             "documentation": null,
@@ -3289,14 +3255,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.2"
         },
         "topics": null
     },
     {
+        "acronym": "GB",
+        "name": "GlycanBind",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2023-06-13T12:04:05Z",
+            "date_released": "2023-06-11T00:00:00Z",
+            "description": "GlycanBind is an ontology for representing glycan and pathogen binding interaction data in RDF.\r\nMany of the terms are based on the data originally contained in the SugarBind database.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/GB/3/gb.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 5883,
+            "indexed": true,
+            "max_children": 8,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 13,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 11,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "1.1.1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ICD10",
         "name": "International Classification of Diseases, Version 10",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 6,
             "classifiable": false,
@@ -4435,44 +4435,44 @@
             "submission_id": 9,
             "taxon": null,
             "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "BMS-LM",
-        "name": "BioMedical Study Lifecycle Management",
+        "acronym": "EDAM",
+        "name": "EDAM bioinformatics operations, types of data, data formats, identifiers, and topics",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2023-02-01T11:42:37Z",
-            "date_released": "2022-02-01T00:00:00Z",
-            "description": "Biomedical research data reuse and sharing is essential for fostering research\r\nprogress. To this aim, data producers need to master data management and reporting\r\nthrough standard and rich metadata, as encouraged by open data initiatives such as\r\nthe FAIR (Findable, Accessible, Interoperable, Reusable) guidelines. This helps data re-users to understand and reuse the shared data with confidence. Therefore, dedicated frameworks are required. The provenance reporting throughout a biomedical study lifecycle has been proposed as a way to increase confidence in data while reusing it. The Biomedical Study - Lifecycle Management (BMS-LM) data model has implemented provenance and lifecycle traceability for several multimodal-imaging techniques but this is not enough for data understanding while reusing it. Actually, in the large scope of biomedical research, a multitude of metadata sources, also called Knowledge Organization Systems (KOSs), are available for data annotation. In addition, data producers uses local terminologies or KOSs, containing vernacular\r\nterms for data reporting. The result is a set of heterogeneous KOSs (local and published) with different formats and levels of granularity. To manage the inherent heterogeneity, semantic interoperability is encouraged by the Research Data\r\nManagement (RDM) community. Ontologies, and more specifically top ontologies such as BFO and DOLCE, make explicit the metadata semantics and enhance semantic interoperability. Based on the BMS-LM data model and the BFO top ontology, the BioMedical Study - Lifecycle Management (BMS-LM) core ontology is proposed together with an associated framework for semantic interoperability between heterogeneous KOSs. It is made of four ontological levels: top/core/domain/local and\r\naims to build bridges between local and published KOSs. In this paper, the conversion of the BMS-LM data model to a core ontology is detailed. The implementation of its semantic interoperability in a specific domain context is explained and illustrated with examples from small animal preclinical research.",
-            "documentation": null,
+            "date_created": "2017-10-31T01:49:28Z",
+            "date_released": "2017-10-31T00:00:00Z",
+            "description": "EDAM is a comprehensive ontology of well-established, familiar concepts that are prevalent within bioinformatics and computational biology, including types of data and data identifiers, data formats, operations and topics. EDAM is a simply-structured ontology - essentially a set of concepts with preferred terms and synonyms, definitions, and some additional information - organised into a simple and intuitive hierarchy for convenient use by information curators, software developers and end-users. EDAM is suitable for large-scale semantic annotations and categorization of diverse bioinformatics resources.  EDAM is also suitable for diverse application including for example within workbenches and workflow-management systems, software distributions, and resource registries.",
+            "documentation": "https://edamontologydocs.readthedocs.io",
             "domain": null,
-            "download_url": "media/ontologies/BMS-LM/1/bms-lm.owl",
+            "download_url": "media/ontologies/EDAM/36/edam.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5645,
+            "home_page": "http://edamontology.org",
+            "id": 346,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 9,
+            "max_children": 1074,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 154,
+            "nb_classes": 3294,
             "nb_inconsistent": 0,
-            "nb_individuals": 22,
-            "nb_properties": 98,
+            "nb_individuals": 0,
+            "nb_properties": 12,
             "products": null,
-            "publication": "https://doi.org/10.1016/j.jbi.2022.104007",
+            "publication": "http://doi.org/10.1093/bioinformatics/btt113",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 36,
             "taxon": null,
-            "version": "Version 1.1"
+            "version": "1.19"
         },
         "topics": null
     },
     {
         "acronym": "GVP",
         "name": "Getty Vocabulary Program ontology",
         "species": null,
@@ -4515,44 +4515,44 @@
         "status": "Unknown",
         "submission": null,
         "topics": [
             "hi"
         ]
     },
     {
-        "acronym": "EDAM",
-        "name": "EDAM bioinformatics operations, types of data, data formats, identifiers, and topics",
+        "acronym": "BERO",
+        "name": "Biological and Environmental Research Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2017-10-31T01:49:28Z",
-            "date_released": "2017-10-31T00:00:00Z",
-            "description": "EDAM is a comprehensive ontology of well-established, familiar concepts that are prevalent within bioinformatics and computational biology, including types of data and data identifiers, data formats, operations and topics. EDAM is a simply-structured ontology - essentially a set of concepts with preferred terms and synonyms, definitions, and some additional information - organised into a simple and intuitive hierarchy for convenient use by information curators, software developers and end-users. EDAM is suitable for large-scale semantic annotations and categorization of diverse bioinformatics resources.  EDAM is also suitable for diverse application including for example within workbenches and workflow-management systems, software distributions, and resource registries.",
-            "documentation": "https://edamontologydocs.readthedocs.io",
+            "date_created": "2022-12-23T08:03:39Z",
+            "date_released": "2022-12-23T00:00:00Z",
+            "description": "Biological and Environmental Research Ontology (BERO) is an application ontology developed by the combination of many ontologies (base or subsets) namely : ChEBI, ECOCORE, EDAM, ENVO, GO, MeSH, MOP, NCBITaxon, NCIT, OBI, OMO, PATO, PO and TO.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/EDAM/36/edam.owl",
+            "download_url": "media/ontologies/BERO/3/bero.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://edamontology.org",
-            "id": 346,
+            "home_page": "https://github.com/berkeleybop/bero",
+            "id": 5587,
             "indexed": true,
-            "max_children": 1074,
-            "max_depth": 8,
+            "max_children": 29287,
+            "max_depth": 42,
             "md5sum": null,
-            "nb_classes": 3294,
+            "nb_classes": 392307,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 12,
+            "nb_individuals": 299,
+            "nb_properties": 297,
             "products": null,
-            "publication": "http://doi.org/10.1093/bioinformatics/btt113",
+            "publication": null,
             "publications": null,
-            "submission_id": 36,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.19"
+            "version": "2022-12-23"
         },
         "topics": null
     },
     {
         "acronym": "ISO19115EX",
         "name": "ISO 19115 Extent Information",
         "species": null,
@@ -4659,14 +4659,53 @@
             "submission_id": 1,
             "taxon": null,
             "version": "0.1"
         },
         "topics": null
     },
     {
+        "acronym": "AEO",
+        "name": "Anatomical Entity Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2017-12-12T21:14:44.509585Z",
+            "date_released": "2017-12-12T21:14:44.509563Z",
+            "description": "AEO is an ontology of anatomical structures that expands CARO, the Common Anatomy Reference Ontology",
+            "documentation": null,
+            "domain": "anatomy",
+            "download_url": "media/ontologies/AEO/8/aeo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obophenotype/human-developmental-anatomy-ontology/",
+            "id": 965,
+            "indexed": true,
+            "max_children": 13,
+            "max_depth": 12,
+            "md5sum": "e70b065253604829b4986a83087a3b1c",
+            "nb_classes": 250,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 11,
+            "products": [
+                {
+                    "id": "aeo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/aeo.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 8,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "SEP",
         "name": "Sample Processing and Separation Techniques Ontology",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 3,
             "classifiable": false,
@@ -4709,48 +4748,14 @@
         "name": "Soil Profile classifiers",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "EDAM2",
-        "name": "EDAM2",
-        "species": [],
-        "status": "Classified",
-        "submission": {
-            "avg_children": 6,
-            "classifiable": true,
-            "date_created": "2019-07-11T13:53:03.690040Z",
-            "date_released": "2019-07-11T13:53:03.690061Z",
-            "description": "",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/EDAM2/1/edam2.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2727,
-            "indexed": true,
-            "max_children": 1102,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 3377,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 12,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1"
-        },
-        "topics": []
-    },
-    {
         "acronym": "APASTATISTICAL",
         "name": "APA Statistical Cluster",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 35,
             "classifiable": false,
@@ -4777,44 +4782,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CRENO",
-        "name": "Culture, Race, Ethnicity and Nationality Ontology",
+        "acronym": "BMS-LM",
+        "name": "BioMedical Study Lifecycle Management",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-02-22T20:32:06Z",
-            "date_released": "2023-02-22T00:00:00Z",
-            "description": "This work undertakes the possibility of using the Web Ontology Language (OWL2) to describe concepts related to \"culture\", \"race\", \"ethnicity\", and \"nationality\" - concepts that are sometimes misunderstood/misused due to lack of standardization and biases. The project also attempts to link data relating to these broad concepts to health data, like cancer rates, and shed light on understanding health disparities that impact certain communities. This work is early in development and preliminary. ",
-            "documentation": "https://github.com/UTHealth-Ontology/CRENO/tree/main/doc",
+            "date_created": "2023-02-01T11:42:37Z",
+            "date_released": "2022-02-01T00:00:00Z",
+            "description": "Biomedical research data reuse and sharing is essential for fostering research\r\nprogress. To this aim, data producers need to master data management and reporting\r\nthrough standard and rich metadata, as encouraged by open data initiatives such as\r\nthe FAIR (Findable, Accessible, Interoperable, Reusable) guidelines. This helps data re-users to understand and reuse the shared data with confidence. Therefore, dedicated frameworks are required. The provenance reporting throughout a biomedical study lifecycle has been proposed as a way to increase confidence in data while reusing it. The Biomedical Study - Lifecycle Management (BMS-LM) data model has implemented provenance and lifecycle traceability for several multimodal-imaging techniques but this is not enough for data understanding while reusing it. Actually, in the large scope of biomedical research, a multitude of metadata sources, also called Knowledge Organization Systems (KOSs), are available for data annotation. In addition, data producers uses local terminologies or KOSs, containing vernacular\r\nterms for data reporting. The result is a set of heterogeneous KOSs (local and published) with different formats and levels of granularity. To manage the inherent heterogeneity, semantic interoperability is encouraged by the Research Data\r\nManagement (RDM) community. Ontologies, and more specifically top ontologies such as BFO and DOLCE, make explicit the metadata semantics and enhance semantic interoperability. Based on the BMS-LM data model and the BFO top ontology, the BioMedical Study - Lifecycle Management (BMS-LM) core ontology is proposed together with an associated framework for semantic interoperability between heterogeneous KOSs. It is made of four ontological levels: top/core/domain/local and\r\naims to build bridges between local and published KOSs. In this paper, the conversion of the BMS-LM data model to a core ontology is detailed. The implementation of its semantic interoperability in a specific domain context is explained and illustrated with examples from small animal preclinical research.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CRENO/1/creno.owl",
+            "download_url": "media/ontologies/BMS-LM/1/bms-lm.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/UTHealth-Ontology/CRENO",
-            "id": 5700,
+            "home_page": null,
+            "id": 5645,
             "indexed": true,
-            "max_children": 241,
-            "max_depth": 12,
+            "max_children": 12,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 1106,
+            "nb_classes": 154,
             "nb_inconsistent": 0,
-            "nb_individuals": 32,
-            "nb_properties": 183,
+            "nb_individuals": 22,
+            "nb_properties": 98,
             "products": null,
-            "publication": null,
+            "publication": "https://doi.org/10.1016/j.jbi.2022.104007",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.5"
+            "version": "Version 1.1"
         },
         "topics": null
     },
     {
         "acronym": "HHS",
         "name": "CIDOnto",
         "species": null,
@@ -4853,133 +4858,128 @@
             "submission_id": 1,
             "taxon": null,
             "version": "0.4.5"
         },
         "topics": null
     },
     {
-        "acronym": "BERO",
-        "name": "Biological and Environmental Research Ontology",
-        "species": null,
+        "acronym": "EDAM2",
+        "name": "EDAM2",
+        "species": [],
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2022-12-23T08:03:39Z",
-            "date_released": "2022-12-23T00:00:00Z",
-            "description": "Biological and Environmental Research Ontology (BERO) is an application ontology developed by the combination of many ontologies (base or subsets) namely : ChEBI, ECOCORE, EDAM, ENVO, GO, MeSH, MOP, NCBITaxon, NCIT, OBI, OMO, PATO, PO and TO.",
+            "date_created": "2019-07-11T13:53:03.690040Z",
+            "date_released": "2019-07-11T13:53:03.690061Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BERO/3/bero.owl",
+            "download_url": "media/ontologies/EDAM2/1/edam2.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/berkeleybop/bero",
-            "id": 5587,
+            "home_page": null,
+            "id": 2727,
             "indexed": true,
-            "max_children": 29287,
-            "max_depth": 42,
+            "max_children": 1102,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 392307,
+            "nb_classes": 3377,
             "nb_inconsistent": 0,
-            "nb_individuals": 299,
-            "nb_properties": 297,
+            "nb_individuals": 0,
+            "nb_properties": 12,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2022-12-23"
+            "version": "1"
         },
-        "topics": null
+        "topics": []
     },
     {
         "acronym": "ANC",
         "name": "ANC",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "AEO",
-        "name": "Anatomical Entity Ontology",
+        "acronym": "BIODIVTHES",
+        "name": "Biodiversity Thesaurus",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-12-12T21:14:44.509585Z",
-            "date_released": "2017-12-12T21:14:44.509563Z",
-            "description": "AEO is an ontology of anatomical structures that expands CARO, the Common Anatomy Reference Ontology",
-            "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/AEO/8/aeo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/human-developmental-anatomy-ontology/",
-            "id": 965,
+            "date_created": "2023-04-07T11:08:59Z",
+            "date_released": "2023-04-07T00:00:00Z",
+            "description": "This bilingual thesaurus organizes the key concepts of biodiversity sciences, in their basic and applied ecological components. It uses polyhierarchy and includes 818 reciprocal associative relationships. The 1654 French and English descriptors (designating 827 concepts) are enriched with a large number of synonyms (1860) and hidden variants (7020) in both languages. Concepts are grouped into 82 collections, by semantic categories, thematic fields and EBV classes (Essential Biodiversity Variables). Definitions are given with their sources. This resource is aligned with the international thesauri AGROVOC, GEMET (GEneral Multilingual Environmental Thesaurus) and EnvThes (Environmental Thesaurus), as well as with the ontology ENVO (Environment Ontology).",
+            "documentation": "https://hal.archives-ouvertes.fr/hal-02907484",
+            "domain": null,
+            "download_url": "media/ontologies/BIODIVTHES/1/biodivthes.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://www.loterre.fr/skosmos/BLH/en/",
+            "id": 5762,
             "indexed": true,
-            "max_children": 13,
-            "max_depth": 12,
-            "md5sum": "e70b065253604829b4986a83087a3b1c",
-            "nb_classes": 250,
+            "max_children": 4,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 11,
-            "products": [
-                {
-                    "id": "aeo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/aeo.owl"
-                }
-            ],
+            "nb_individuals": 915,
+            "nb_properties": 2,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "2.1"
         },
         "topics": null
     },
     {
         "acronym": "PRANAYTC",
         "name": "pranaytc",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "BIODIVTHES",
-        "name": "Biodiversity Thesaurus",
+        "acronym": "CRENO",
+        "name": "Culture, Race, Ethnicity and Nationality Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2023-04-07T11:08:59Z",
-            "date_released": "2023-04-07T00:00:00Z",
-            "description": "This bilingual thesaurus organizes the key concepts of biodiversity sciences, in their basic and applied ecological components. It uses polyhierarchy and includes 818 reciprocal associative relationships. The 1654 French and English descriptors (designating 827 concepts) are enriched with a large number of synonyms (1860) and hidden variants (7020) in both languages. Concepts are grouped into 82 collections, by semantic categories, thematic fields and EBV classes (Essential Biodiversity Variables). Definitions are given with their sources. This resource is aligned with the international thesauri AGROVOC, GEMET (GEneral Multilingual Environmental Thesaurus) and EnvThes (Environmental Thesaurus), as well as with the ontology ENVO (Environment Ontology).",
-            "documentation": "https://hal.archives-ouvertes.fr/hal-02907484",
+            "date_created": "2023-02-22T20:32:06Z",
+            "date_released": "2023-02-22T00:00:00Z",
+            "description": "This work undertakes the possibility of using the Web Ontology Language (OWL2) to describe concepts related to \"culture\", \"race\", \"ethnicity\", and \"nationality\" - concepts that are sometimes misunderstood/misused due to lack of standardization and biases. The project also attempts to link data relating to these broad concepts to health data, like cancer rates, and shed light on understanding health disparities that impact certain communities. This work is early in development and preliminary. ",
+            "documentation": "https://github.com/UTHealth-Ontology/CRENO/tree/main/doc",
             "domain": null,
-            "download_url": "media/ontologies/BIODIVTHES/1/biodivthes.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://www.loterre.fr/skosmos/BLH/en/",
-            "id": 5762,
+            "download_url": "media/ontologies/CRENO/1/creno.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/UTHealth-Ontology/CRENO",
+            "id": 5700,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 1,
+            "max_children": 241,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 4,
+            "nb_classes": 1106,
             "nb_inconsistent": 0,
-            "nb_individuals": 915,
-            "nb_properties": 2,
+            "nb_individuals": 32,
+            "nb_properties": 183,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2.1"
+            "version": "0.5"
         },
         "topics": null
     },
     {
         "acronym": "FPLX",
         "name": "FamPlex",
         "species": null,
@@ -5056,82 +5056,14 @@
         "status": "Unknown",
         "submission": null,
         "topics": [
             "sgresdgs"
         ]
     },
     {
-        "acronym": "EDAM-BIOIMAGING",
-        "name": "EDAM Bioimaging Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2017-04-28T05:59:52Z",
-            "date_released": "2017-04-28T00:00:00Z",
-            "description": "Bioimage informatics operations, types of data, data formats, and bioimaging topics\r\nEDAM-bioimaging is an extension to the EDAM ontology (edamontology.org) for bioimage analysis, bioimage informatics, and bioimaging, developed in collaboration with partners from NEUBIAS Cost Action CA15124 (neubias.org) and EuroBioImaging PP2.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/EDAM-BIOIMAGING/2/edam-bioimaging.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://webprotege.stanford.edu/#projects/2ce704bf-83ed-4d2e-985f-84c4841fac71/edit/Classes",
-            "id": 614,
-            "indexed": true,
-            "max_children": 21,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 211,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 12,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "alpha02"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "EDAMT",
-        "name": "EDAM Topics Subset",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2014-10-06T12:39:31Z",
-            "date_released": "2014-10-06T00:00:00Z",
-            "description": "An ontology of bioinformatics, biological and biomedical topics (EDAM Topics) ",
-            "documentation": "edamontology.org",
-            "domain": null,
-            "download_url": "media/ontologies/EDAMT/1/edamt.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "edamontology.org",
-            "id": 533,
-            "indexed": true,
-            "max_children": 35,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 226,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 12,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.4"
-        },
-        "topics": null
-    },
-    {
         "acronym": "N-CDOH",
         "name": "Non Clinical Determinants of Health",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -5212,14 +5144,48 @@
         "status": "Unknown",
         "submission": null,
         "topics": [
             "Python Topic"
         ]
     },
     {
+        "acronym": "EDAM-BIOIMAGING",
+        "name": "EDAM Bioimaging Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2017-04-28T05:59:52Z",
+            "date_released": "2017-04-28T00:00:00Z",
+            "description": "Bioimage informatics operations, types of data, data formats, and bioimaging topics\r\nEDAM-bioimaging is an extension to the EDAM ontology (edamontology.org) for bioimage analysis, bioimage informatics, and bioimaging, developed in collaboration with partners from NEUBIAS Cost Action CA15124 (neubias.org) and EuroBioImaging PP2.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/EDAM-BIOIMAGING/2/edam-bioimaging.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://webprotege.stanford.edu/#projects/2ce704bf-83ed-4d2e-985f-84c4841fac71/edit/Classes",
+            "id": 614,
+            "indexed": true,
+            "max_children": 21,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 211,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 12,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "alpha02"
+        },
+        "topics": null
+    },
+    {
         "acronym": "HHEARP",
         "name": "HHEAR Properties",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -5246,22 +5212,68 @@
             "submission_id": 3,
             "taxon": null,
             "version": "0.0.3"
         },
         "topics": null
     },
     {
+        "acronym": "EDAMT",
+        "name": "EDAM Topics Subset",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2014-10-06T12:39:31Z",
+            "date_released": "2014-10-06T00:00:00Z",
+            "description": "An ontology of bioinformatics, biological and biomedical topics (EDAM Topics) ",
+            "documentation": "edamontology.org",
+            "domain": null,
+            "download_url": "media/ontologies/EDAMT/1/edamt.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "edamontology.org",
+            "id": 533,
+            "indexed": true,
+            "max_children": 35,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 226,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 12,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.4"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ADMF",
         "name": "alzhaimer dessiese main factors",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
+        "acronym": "KETO",
+        "name": "Kerala Tourism Ontology",
+        "species": [
+            "Tourism"
+        ],
+        "status": "Unknown",
+        "submission": null,
+        "topics": [
+            "Tourism"
+        ]
+    },
+    {
         "acronym": "HHEAR",
         "name": "Human Health Exposure Analysis Resource",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 21,
             "classifiable": true,
@@ -5288,26 +5300,14 @@
             "submission_id": 8,
             "taxon": null,
             "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "KETO",
-        "name": "Kerala Tourism Ontology",
-        "species": [
-            "Tourism"
-        ],
-        "status": "Unknown",
-        "submission": null,
-        "topics": [
-            "Tourism"
-        ]
-    },
-    {
         "acronym": "ETHIOPIADISEASES",
         "name": "EthiopiaDiseasesList",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -5356,220 +5356,285 @@
         "name": "SNOMED CT SPA",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "BIBFRAME",
-        "name": "BIBFRAME 2.0",
+        "acronym": "AHOL",
+        "name": "Animal Health Ontology for Livestock",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 9,
             "classifiable": true,
-            "date_created": "2017-06-09T01:00:14Z",
-            "date_released": "2017-06-09T01:00:14Z",
-            "description": "Initiated by the Library of Congress, BIBFRAME provides a foundation for the future of bibliographic description, both on the web, and in the broader networked world. In addition to being a replacement for MARC, BIBFRAME serves as a general model for expressing and connecting bibliographic data. A major focus of the initiative will be to determine a transition path for the MARC 21 formats while preserving a robust data exchange that has supported resource sharing and cataloging cost savings in recent decades.",
-            "documentation": "https://www.loc.gov/bibframe/docs/index.html",
+            "date_created": "2018-10-23T11:46:11Z",
+            "date_released": "2018-10-23T00:00:00Z",
+            "description": "AHOL (Animal Health Ontology for Livestock) is an ontology of characteristics defining health issues of livestock in their environment (EOL) linked to their phenotypes (ATOL). AHOL aims to: - provide a reference ontology of health traits of farm animals for the international scientific and educational - communities, farmers, etc.; - deliver this reference ontology in a language which can be used by computers in order to support database management, semantic analysis and modeling; - represent traits as generic as possible for livestock vertebrates; - make the AHOL ontology as operational as possible and closely related to measurement techniques; - structure the ontology in relation to animal production ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BIBFRAME/3/bibframe.owl",
+            "download_url": "media/ontologies/AHOL/1/ahol.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.loc.gov/bibframe/",
-            "id": 543,
+            "home_page": "http://147.99.156.182/ontologies/visualisation/ontologie/",
+            "id": 1943,
             "indexed": true,
-            "max_children": 75,
-            "max_depth": 2,
+            "max_children": 79,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 188,
+            "nb_classes": 339,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 195,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2017-05-04T14:49:12.796-04:00"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "FALL",
-        "name": "MyOntoServiceFull_FallDetection",
+        "acronym": "AHSO",
+        "name": "Animal Health Surveillance Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 7,
-            "classifiable": false,
-            "date_created": "2015-03-19T13:18:52Z",
-            "date_released": "2015-03-19T00:00:00Z",
-            "description": "fall detection scenario",
-            "documentation": null,
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2018-07-31T06:29:33Z",
+            "date_released": "2018-07-30T00:00:00Z",
+            "description": "(Work in progress - please contact ahso@datadrivensurveillance.org before using this ontology)\r\n\r\nThis is an ontology for health surveillance, assuming that surveillance data is recorded at three main levels:\r\n\r\nSAMPLE data - data about specific collected samples; OBSERVATIONS, which can aggregate several samples; and OBSERVATIONAL CONTEXT, which is the context in which the observations were made (for instance clinical observation, surveillance sampling, etc). \r\n\r\nHEALTH EVENT is the occurrence in the real World to be modeled. This occurrence is an abstract concept, as it generally cannot be determined what are its boundaries in time, space, or population units. \r\n\r\nWe assume that several OBSERVATIONS are derived from each health event, and recorded in the same or different databases - for instance clinical journal, laboratory database, abattoir. At the same time or not. \r\n\r\nAHSO attempts to model these time specific observations. The ontology is focused on the modeling of the animal and health information connected to a single observation, made at a specific point on time. All the information recorded is relative to that specific point in time.\r\n\r\nFor instance, an animal belongs to a herd, which belongs to an owner, AT THE TIME OF THE OBSERVATION. We do not attempt to model how that animal changed herds on time. Similarly, we do not model clinical progression. It is up to those using the data to create definitions of when two observations should be connected to the same event.",
+            "documentation": "https://github.com/SVA-SE/AHSO",
             "domain": null,
-            "download_url": "media/ontologies/FALL/1/fall.owl",
+            "download_url": "media/ontologies/AHSO/1/ahso.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 148,
-            "indexed": false,
-            "max_children": 35,
-            "max_depth": 2,
+            "home_page": "http://datadrivensurveillance.org/ahso",
+            "id": 1706,
+            "indexed": true,
+            "max_children": 9,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 51,
+            "nb_classes": 38,
             "nb_inconsistent": 0,
-            "nb_individuals": 58,
-            "nb_properties": 40,
+            "nb_individuals": 0,
+            "nb_properties": 31,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "0.1.0"
         },
         "topics": null
     },
     {
         "acronym": "PATAXO",
         "name": "Technical Taxonomies for Patents",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
+        "acronym": "NIFSUBCELL",
+        "name": "Neuroscience Information Framework (NIF) Subcellular Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": false,
+            "date_created": "2012-07-27T22:30:28Z",
+            "date_released": "2012-05-31T07:00:00Z",
+            "description": "http://ontology.neuinfo.org/NIF/BiomaterialEntities/NIF-Subcellular.owl",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/NIFSUBCELL/1/nifsubcell.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 384,
+            "indexed": false,
+            "max_children": 936,
+            "max_depth": 14,
+            "md5sum": null,
+            "nb_classes": 2801,
+            "nb_inconsistent": 0,
+            "nb_individuals": 102,
+            "nb_properties": 87,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.3 (May 2012)"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ISO19115PR",
         "name": "ISO 19115 Codelists",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "TMO",
-        "name": "Translational Medicine Ontology",
+        "acronym": "BHN",
+        "name": "Biologie Hors Nomenclature",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 2,
-            "classifiable": false,
-            "date_created": "2016-05-17T01:27:01Z",
-            "date_released": "2016-05-17T01:27:01Z",
-            "description": "This project focuses on the development of a high level patient-centric ontology for the pharmaceutical industry. The ontology should enable silos in discovery research, hypothesis management, experimental studies, compounds, formulation, drug development, market size, competitive data, population data, etc. to be brought together. This would enable scientists to answer new questions, and to answer existing scientific questions more quickly. This will help pharmaceutical companies to model patient-centric information, which is essential for the tailoring of drugs, and for early detection of compounds that may have sub-optimal safety profiles. The ontology should link to existing publicly available domain ontologies.",
-            "documentation": "https://github.com/micheldumontier/translationalmedicineontology",
+            "avg_children": 21,
+            "classifiable": true,
+            "date_created": "2013-11-19T16:05:53Z",
+            "date_released": "2011-02-11T00:00:00Z",
+            "description": "L'activit\u00e9 innovante de biologie et d'anatomo-pathologie r\u00e9alis\u00e9e notamment dans les Centres Hospitalo-Universitaires est habituellement appel\u00e9e \"activit\u00e9 hors nomenclature\" (BHN pour la biologie hors nomenclature et PHN pour l'anatomo-pathologie hors nomenclature). Ce caract\u00e8re \"hors nomenclature\" signifie que l'assurance maladie n'a pas encore int\u00e9gr\u00e9 ces actes dans la Nomenclature des Actes de Biologie M\u00e9dicale (NABM) ou la Nomenclature G\u00e9n\u00e9rale des Actes Professionnels (NGAP).",
+            "documentation": "http://www.chu-montpellier.fr/publication/inter_pub/R300/rubrique.jsp",
             "domain": null,
-            "download_url": "media/ontologies/TMO/13/tmo.owl",
+            "download_url": "media/ontologies/BHN/1/bhn.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://esw.w3.org/HCLSIG/PharmaOntology",
-            "id": 11,
-            "indexed": false,
-            "max_children": 14,
-            "max_depth": 5,
+            "home_page": "http://www.chu-montpellier.fr/publication/inter_pub/R300/rubrique.jsp",
+            "id": 155,
+            "indexed": true,
+            "max_children": 148,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 225,
+            "nb_classes": 2534,
             "nb_inconsistent": 0,
-            "nb_individuals": 15,
-            "nb_properties": 22,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102889/",
+            "publication": null,
             "publications": null,
-            "submission_id": 13,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.0.2"
+            "version": "3.2"
         },
         "topics": null
     },
     {
-        "acronym": "GENEPIO",
-        "name": "Genomic Epidemiology Ontology",
-        "species": null,
+        "acronym": "GDPR",
+        "name": "Data Protection",
+        "species": [
+            "Data"
+        ],
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2023-07-17T12:28:12.624341Z",
-            "date_released": "2023-07-17T12:28:12.624324Z",
-            "description": "The Genomic Epidemiology Ontology (GenEpiO) covers vocabulary necessary to identify, document and research foodborne pathogens and associated outbreaks.",
+            "date_created": "2022-04-18T19:27:19.637110Z",
+            "date_released": "2022-04-18T19:27:19.637132Z",
+            "description": "",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/GENEPIO/23/genepio.owl",
+            "domain": null,
+            "download_url": "media/ontologies/GDPR/1/gdpr.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://genepio.org/",
-            "id": 5947,
+            "home_page": null,
+            "id": 5039,
             "indexed": true,
-            "max_children": 155,
-            "max_depth": 19,
-            "md5sum": "8dbf835a058c554d39b0fc407fa360a9",
-            "nb_classes": 7221,
+            "max_children": 14,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 55,
             "nb_inconsistent": 0,
-            "nb_individuals": 945,
-            "nb_properties": 64,
-            "products": [
-                {
-                    "homepage": "http://genepio.github.io/genepio/",
-                    "id": "genepio.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/genepio.owl"
-                }
-            ],
+            "nb_individuals": 7,
+            "nb_properties": 11,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 23,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1"
         },
-        "topics": null
+        "topics": [
+            "Personal Data"
+        ]
     },
     {
-        "acronym": "GENO",
-        "name": "Genotype Ontology",
+        "acronym": "BCO",
+        "name": "Biological Collections Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-02-27T12:22:42.600169Z",
-            "date_released": "2023-02-27T12:22:42.600154Z",
-            "description": "An integrated ontology for representing the genetic variations described in genotypes, and their causal relationships to phenotype and diseases.",
+            "date_created": "2021-11-15T12:16:26.519969Z",
+            "date_released": "2021-11-15T12:16:26.519946Z",
+            "description": "An ontology to support the interoperability of biodiversity data, including data on museum collections, environmental/metagenomic samples, and ecological surveys.",
             "documentation": null,
-            "domain": "biological systems",
-            "download_url": "media/ontologies/GENO/27/geno.owl",
+            "domain": "biodiversity collections",
+            "download_url": "media/ontologies/BCO/12/bco.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/monarch-initiative/GENO-ontology/",
-            "id": 5680,
+            "home_page": "https://github.com/tucotuco/bco",
+            "id": 4685,
             "indexed": true,
-            "max_children": 49,
-            "max_depth": 13,
-            "md5sum": "8304b1065272a23ff990b937de5e1859",
-            "nb_classes": 424,
+            "max_children": 39,
+            "max_depth": 31,
+            "md5sum": "75a0edb975d0954c16269bde4263203b",
+            "nb_classes": 254,
             "nb_inconsistent": 0,
-            "nb_individuals": 21,
-            "nb_properties": 130,
+            "nb_individuals": 64,
+            "nb_properties": 112,
             "products": [
                 {
-                    "id": "geno.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/geno.owl",
-                    "title": "GENO"
+                    "id": "bco.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bco.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 27,
+            "submission_id": 12,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "CST",
         "name": "Cancer Staging Terms",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
+        "acronym": "ONSTR",
+        "name": "Ontology for Newborn Screening Follow-up and Translational Research",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": false,
+            "date_created": "2014-09-04T13:42:34Z",
+            "date_released": "2014-09-03T00:00:00Z",
+            "description": "Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. \r\nONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC).\r\n\r\nONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies.\r\n\r\nFor latest release notes please see: \r\nhttp://onstr.googlecode.com/svn/tags/currentRelease/2013-05-02/\r\n\r\nNBSDC project page: https://nbsdc.org  (under construction).\r\n\r\n",
+            "documentation": "http://onstr.googlecode.com/svn/docs/",
+            "domain": null,
+            "download_url": "media/ontologies/ONSTR/5/onstr.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://code.google.com/p/onstr/",
+            "id": 488,
+            "indexed": false,
+            "max_children": 25,
+            "max_depth": 34,
+            "md5sum": null,
+            "nb_classes": 1865,
+            "nb_inconsistent": 0,
+            "nb_individuals": 83,
+            "nb_properties": 48,
+            "products": null,
+            "publication": "https://nbsdc.org/publications.php",
+            "publications": null,
+            "submission_id": 5,
+            "taxon": null,
+            "version": "0.4"
+        },
+        "topics": null
+    },
+    {
         "acronym": "MISINFO",
         "name": "COVID-19 misinformation",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -5578,56 +5643,96 @@
         "name": "agenbola",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "ASSAYNODE",
-        "name": "Assaynode",
+        "acronym": "FALL",
+        "name": "MyOntoServiceFull_FallDetection",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2023-01-19T17:40:37Z",
-            "date_released": "2023-01-18T00:00:00Z",
-            "description": "An ontology providing controlled vocabularies for Sequence Analysis and sequencing instrument model.",
+            "avg_children": 7,
+            "classifiable": false,
+            "date_created": "2015-03-19T13:18:52Z",
+            "date_released": "2015-03-19T00:00:00Z",
+            "description": "fall detection scenario",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ASSAYNODE/2/assaynode.owl",
+            "download_url": "media/ontologies/FALL/1/fall.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5623,
-            "indexed": true,
-            "max_children": 12,
-            "max_depth": 4,
+            "id": 148,
+            "indexed": false,
+            "max_children": 35,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 48,
+            "nb_classes": 51,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 58,
+            "nb_properties": 40,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.0"
+            "version": "1"
         },
         "topics": null
     },
     {
         "acronym": "PRE-CLINICAL",
         "name": "taimoor Ontology",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
+        "acronym": "GENO",
+        "name": "Genotype Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-02-27T12:22:42.600169Z",
+            "date_released": "2023-02-27T12:22:42.600154Z",
+            "description": "An integrated ontology for representing the genetic variations described in genotypes, and their causal relationships to phenotype and diseases.",
+            "documentation": null,
+            "domain": "biological systems",
+            "download_url": "media/ontologies/GENO/27/geno.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/monarch-initiative/GENO-ontology/",
+            "id": 5680,
+            "indexed": true,
+            "max_children": 49,
+            "max_depth": 13,
+            "md5sum": "8304b1065272a23ff990b937de5e1859",
+            "nb_classes": 424,
+            "nb_inconsistent": 0,
+            "nb_individuals": 21,
+            "nb_properties": 130,
+            "products": [
+                {
+                    "id": "geno.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/geno.owl",
+                    "title": "GENO"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 27,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "VPS",
         "name": "vps windows",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -5636,14 +5741,82 @@
         "name": "Cipto Junaedy",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
+        "acronym": "TMO",
+        "name": "Translational Medicine Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": false,
+            "date_created": "2016-05-17T01:27:01Z",
+            "date_released": "2016-05-17T01:27:01Z",
+            "description": "This project focuses on the development of a high level patient-centric ontology for the pharmaceutical industry. The ontology should enable silos in discovery research, hypothesis management, experimental studies, compounds, formulation, drug development, market size, competitive data, population data, etc. to be brought together. This would enable scientists to answer new questions, and to answer existing scientific questions more quickly. This will help pharmaceutical companies to model patient-centric information, which is essential for the tailoring of drugs, and for early detection of compounds that may have sub-optimal safety profiles. The ontology should link to existing publicly available domain ontologies.",
+            "documentation": "https://github.com/micheldumontier/translationalmedicineontology",
+            "domain": null,
+            "download_url": "media/ontologies/TMO/13/tmo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://esw.w3.org/HCLSIG/PharmaOntology",
+            "id": 11,
+            "indexed": false,
+            "max_children": 14,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 225,
+            "nb_inconsistent": 0,
+            "nb_individuals": 15,
+            "nb_properties": 22,
+            "products": null,
+            "publication": "http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102889/",
+            "publications": null,
+            "submission_id": 13,
+            "taxon": null,
+            "version": "1.0.2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "HCODONONT",
+        "name": "HGeneCodonOntology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 48,
+            "classifiable": false,
+            "date_created": "2016-09-02T22:21:19Z",
+            "date_released": "2016-09-02T00:00:00Z",
+            "description": "This ontology contains the information of the frequency of usage and the relative frequency of usage of Human Gene Codons, to be used to the operations by The Project Of HIV Ontology.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HCODONONT/4/hcodonont.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://blog.51.ca/u-345129/",
+            "id": 235,
+            "indexed": false,
+            "max_children": 94,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 98,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 2,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "2"
+        },
+        "topics": null
+    },
+    {
         "acronym": "TEST_IDKWHATIMDO",
         "name": "Tanya ",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -5678,78 +5851,78 @@
             "submission_id": 3,
             "taxon": null,
             "version": "1.2.0"
         },
         "topics": null
     },
     {
-        "acronym": "bibliographic",
-        "name": "Bibliographic",
+        "acronym": "BIBFRAME",
+        "name": "BIBFRAME 2.0",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 9,
             "classifiable": true,
-            "date_created": "2012-08-31T01:25:04Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-public-1.5.owl\r\n\r\nThis file contains selected terms from the bibliographic ontology (see http://bibliontology.com/) which relate to information resources.",
-            "documentation": null,
+            "date_created": "2017-06-09T01:00:14Z",
+            "date_released": "2017-06-09T01:00:14Z",
+            "description": "Initiated by the Library of Congress, BIBFRAME provides a foundation for the future of bibliographic description, both on the web, and in the broader networked world. In addition to being a replacement for MARC, BIBFRAME serves as a general model for expressing and connecting bibliographic data. A major focus of the initiative will be to determine a transition path for the MARC 21 formats while preserving a robust data exchange that has supported resource sharing and cataloging cost savings in recent decades.",
+            "documentation": "https://www.loc.gov/bibframe/docs/index.html",
             "domain": null,
-            "download_url": "media/ontologies/bibliographic/1/bibliographic.owl",
+            "download_url": "media/ontologies/BIBFRAME/3/bibframe.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 448,
+            "home_page": "https://www.loc.gov/bibframe/",
+            "id": 543,
             "indexed": true,
-            "max_children": 19,
-            "max_depth": 4,
+            "max_children": 75,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 63,
+            "nb_classes": 188,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 77,
+            "nb_properties": 195,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.5"
+            "version": "2017-05-04T14:49:12.796-04:00"
         },
         "topics": null
     },
     {
-        "acronym": "BIBLIOTEK-O",
-        "name": "The Biblioteko Ontology",
+        "acronym": "HENECON",
+        "name": "Head and Neck Cancer Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 9,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2017-07-12T01:02:14Z",
-            "date_released": "2017-07-12T01:02:14Z",
-            "description": "An ontology recommendation for modeling bibliographic metadata",
+            "date_created": "2023-03-21T12:22:36Z",
+            "date_released": "2020-04-14T00:00:00Z",
+            "description": "HeNeCOn (Head and Neck Cancer Ontology) is the first ontology specifically focused on the Head and Neck Cancer (HNC) domain. It provides a clinically reliable data model for HNC processes during all phases of the disease: prognosis, treatment, and follow-up. The ontology is established based on an annotated dataset to provide formal reference descriptions, contains  170 clinical variables organized into a taxonomy, and is expanded and mapped to formalize and integrate multiple databases into the HeNeCOn ontology. The outcomes of this iterative process were reviewed and validated by clinicians and statisticians. HeNeCOn, with roughly 500 classes, is a very specific ontology which includes a taxonomy with a hierarchical structure, semantic definitions of 283 medical terms and detailed relations between them, which makes it a reference data model. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BIBLIOTEK-O/3/bibliotek-o.owl",
+            "download_url": "media/ontologies/HENECON/1/henecon.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://bibliotek-o.org/",
-            "id": 471,
+            "home_page": null,
+            "id": 5753,
             "indexed": true,
-            "max_children": 140,
-            "max_depth": 3,
+            "max_children": 81,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 278,
+            "nb_classes": 1933,
             "nb_inconsistent": 0,
-            "nb_individuals": 30,
-            "nb_properties": 24,
+            "nb_individuals": 1472,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "Version 1.1.0"
+            "version": "1.0"
         },
         "topics": null
     },
     {
         "acronym": "BCS7",
         "name": "Breast Cancer Staging - 7th Edition",
         "species": null,
@@ -5770,52 +5943,14 @@
         "name": "Simple Measurement Ontology",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "GDPR",
-        "name": "Data Protection",
-        "species": [
-            "Data"
-        ],
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2022-04-18T19:27:19.637110Z",
-            "date_released": "2022-04-18T19:27:19.637132Z",
-            "description": "",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/GDPR/1/gdpr.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5039,
-            "indexed": true,
-            "max_children": 14,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 55,
-            "nb_inconsistent": 0,
-            "nb_individuals": 7,
-            "nb_properties": 11,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1"
-        },
-        "topics": [
-            "Personal Data"
-        ]
-    },
-    {
         "acronym": "PRIDE",
         "name": "PRIDE Controlled Vocabulary",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -5850,19 +5985,58 @@
             "submission_id": 2,
             "taxon": null,
             "version": "2020"
         },
         "topics": null
     },
     {
-        "acronym": "ISO19115CON",
-        "name": "ISO 19115 Constraints Information",
+        "acronym": "AISM",
+        "name": "Ontology for the Anatomy of the Insect SkeletoMuscular system (AISM)",
         "species": null,
-        "status": "Unknown",
-        "submission": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-04-17T12:00:07.519273Z",
+            "date_released": "2023-04-17T12:00:07.519252Z",
+            "description": "The AISM contains terms used in insect biodiversity research for describing structures of the exoskeleton and the skeletomuscular system. It aims to serve as the basic backbone of generalized terms to be expanded with order-specific terminology.",
+            "documentation": null,
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/AISM/13/aism.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/insect-morphology/aism",
+            "id": 5765,
+            "indexed": true,
+            "max_children": 270,
+            "max_depth": 41,
+            "md5sum": "e79c22d3c423e72b01c45968074810fe",
+            "nb_classes": 7443,
+            "nb_inconsistent": 0,
+            "nb_individuals": 18,
+            "nb_properties": 306,
+            "products": [
+                {
+                    "id": "aism.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/aism.owl"
+                },
+                {
+                    "id": "aism.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/aism.obo"
+                },
+                {
+                    "id": "aism.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/aism.json"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 13,
+            "taxon": null,
+            "version": null
+        },
         "topics": null
     },
     {
         "acronym": "HIVMT",
         "name": "HIVMutation",
         "species": null,
         "status": "Unloadable",
@@ -5892,195 +6066,53 @@
             "submission_id": 6,
             "taxon": null,
             "version": "01"
         },
         "topics": null
     },
     {
-        "acronym": "BCO",
-        "name": "Biological Collections Ontology",
+        "acronym": "ALLERGYDETECTOR",
+        "name": "Allergy Detector II",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-11-15T12:16:26.519969Z",
-            "date_released": "2021-11-15T12:16:26.519946Z",
-            "description": "An ontology to support the interoperability of biodiversity data, including data on museum collections, environmental/metagenomic samples, and ecological surveys.",
-            "documentation": null,
-            "domain": "biodiversity collections",
-            "download_url": "media/ontologies/BCO/12/bco.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/tucotuco/bco",
-            "id": 4685,
-            "indexed": true,
-            "max_children": 39,
-            "max_depth": 31,
-            "md5sum": "75a0edb975d0954c16269bde4263203b",
-            "nb_classes": 254,
-            "nb_inconsistent": 0,
-            "nb_individuals": 64,
-            "nb_properties": 112,
-            "products": [
-                {
-                    "id": "bco.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bco.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 12,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NIFSUBCELL",
-        "name": "Neuroscience Information Framework (NIF) Subcellular Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": false,
-            "date_created": "2012-07-27T22:30:28Z",
-            "date_released": "2012-05-31T07:00:00Z",
-            "description": "http://ontology.neuinfo.org/NIF/BiomaterialEntities/NIF-Subcellular.owl",
+            "date_created": "2015-05-02T00:22:05Z",
+            "date_released": "2015-05-01T00:00:00Z",
+            "description": "Allergy Detection Ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NIFSUBCELL/1/nifsubcell.owl",
+            "download_url": "media/ontologies/ALLERGYDETECTOR/1/allergydetector.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 384,
-            "indexed": false,
-            "max_children": 936,
-            "max_depth": 14,
+            "id": 740,
+            "indexed": true,
+            "max_children": 4,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 2801,
+            "nb_classes": 12,
             "nb_inconsistent": 0,
-            "nb_individuals": 102,
-            "nb_properties": 87,
+            "nb_individuals": 161,
+            "nb_properties": 20,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.3 (May 2012)"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ONSTR",
-        "name": "Ontology for Newborn Screening Follow-up and Translational Research",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": false,
-            "date_created": "2014-09-04T13:42:34Z",
-            "date_released": "2014-09-03T00:00:00Z",
-            "description": "Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. \r\nONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC).\r\n\r\nONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies.\r\n\r\nFor latest release notes please see: \r\nhttp://onstr.googlecode.com/svn/tags/currentRelease/2013-05-02/\r\n\r\nNBSDC project page: https://nbsdc.org  (under construction).\r\n\r\n",
-            "documentation": "http://onstr.googlecode.com/svn/docs/",
-            "domain": null,
-            "download_url": "media/ontologies/ONSTR/5/onstr.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://code.google.com/p/onstr/",
-            "id": 488,
-            "indexed": false,
-            "max_children": 25,
-            "max_depth": 34,
-            "md5sum": null,
-            "nb_classes": 1865,
-            "nb_inconsistent": 0,
-            "nb_individuals": 83,
-            "nb_properties": 48,
-            "products": null,
-            "publication": "https://nbsdc.org/publications.php",
-            "publications": null,
-            "submission_id": 5,
-            "taxon": null,
-            "version": "0.4"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BFO",
-        "name": "Basic Formal Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2019-09-02T12:00:03.790386Z",
-            "date_released": "2019-09-02T12:00:03.790326Z",
-            "description": "The upper level ontology upon which OBO Foundry ontologies are built.",
-            "documentation": null,
-            "domain": "upper",
-            "download_url": "media/ontologies/BFO/5/bfo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://ifomis.org/bfo/",
-            "id": 2877,
-            "indexed": true,
-            "max_children": 4,
-            "max_depth": 6,
-            "md5sum": "738838092081290736ec3a1485cd07cb",
-            "nb_classes": 36,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": [
-                {
-                    "id": "bfo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bfo.owl"
-                },
-                {
-                    "id": "bfo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bfo.obo"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 5,
-            "taxon": null,
-            "version": null
+            "version": "0.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "BHN",
-        "name": "Biologie Hors Nomenclature",
+        "acronym": "ISO19115CON",
+        "name": "ISO 19115 Constraints Information",
         "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 21,
-            "classifiable": true,
-            "date_created": "2013-11-19T16:05:53Z",
-            "date_released": "2011-02-11T00:00:00Z",
-            "description": "L'activit\u00e9 innovante de biologie et d'anatomo-pathologie r\u00e9alis\u00e9e notamment dans les Centres Hospitalo-Universitaires est habituellement appel\u00e9e \"activit\u00e9 hors nomenclature\" (BHN pour la biologie hors nomenclature et PHN pour l'anatomo-pathologie hors nomenclature). Ce caract\u00e8re \"hors nomenclature\" signifie que l'assurance maladie n'a pas encore int\u00e9gr\u00e9 ces actes dans la Nomenclature des Actes de Biologie M\u00e9dicale (NABM) ou la Nomenclature G\u00e9n\u00e9rale des Actes Professionnels (NGAP).",
-            "documentation": "http://www.chu-montpellier.fr/publication/inter_pub/R300/rubrique.jsp",
-            "domain": null,
-            "download_url": "media/ontologies/BHN/1/bhn.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.chu-montpellier.fr/publication/inter_pub/R300/rubrique.jsp",
-            "id": 155,
-            "indexed": true,
-            "max_children": 148,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 2534,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "3.2"
-        },
+        "status": "Unknown",
+        "submission": null,
         "topics": null
     },
     {
         "acronym": "AGROMOP",
         "name": "Control de Cacao",
         "species": null,
         "status": "Classified",
@@ -6110,239 +6142,207 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "AHOL",
-        "name": "Animal Health Ontology for Livestock",
+        "acronym": "GENEPIO",
+        "name": "Genomic Epidemiology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 9,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-10-23T11:46:11Z",
-            "date_released": "2018-10-23T00:00:00Z",
-            "description": "AHOL (Animal Health Ontology for Livestock) is an ontology of characteristics defining health issues of livestock in their environment (EOL) linked to their phenotypes (ATOL). AHOL aims to: - provide a reference ontology of health traits of farm animals for the international scientific and educational - communities, farmers, etc.; - deliver this reference ontology in a language which can be used by computers in order to support database management, semantic analysis and modeling; - represent traits as generic as possible for livestock vertebrates; - make the AHOL ontology as operational as possible and closely related to measurement techniques; - structure the ontology in relation to animal production ",
+            "date_created": "2023-07-17T12:28:12.624341Z",
+            "date_released": "2023-07-17T12:28:12.624324Z",
+            "description": "The Genomic Epidemiology Ontology (GenEpiO) covers vocabulary necessary to identify, document and research foodborne pathogens and associated outbreaks.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/AHOL/1/ahol.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/GENEPIO/23/genepio.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://147.99.156.182/ontologies/visualisation/ontologie/",
-            "id": 1943,
+            "home_page": "http://genepio.org/",
+            "id": 5947,
             "indexed": true,
-            "max_children": 79,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 339,
+            "max_children": 155,
+            "max_depth": 19,
+            "md5sum": "8dbf835a058c554d39b0fc407fa360a9",
+            "nb_classes": 7221,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 7,
-            "products": null,
+            "nb_individuals": 945,
+            "nb_properties": 64,
+            "products": [
+                {
+                    "homepage": "http://genepio.github.io/genepio/",
+                    "id": "genepio.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/genepio.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 23,
             "taxon": null,
-            "version": "1.0"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HMIS033B",
-        "name": "HMIS 033",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "HCODONONT",
-        "name": "HGeneCodonOntology",
+        "acronym": "ASSAYNODE",
+        "name": "Assaynode",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 48,
-            "classifiable": false,
-            "date_created": "2016-09-02T22:21:19Z",
-            "date_released": "2016-09-02T00:00:00Z",
-            "description": "This ontology contains the information of the frequency of usage and the relative frequency of usage of Human Gene Codons, to be used to the operations by The Project Of HIV Ontology.",
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2023-01-19T17:40:37Z",
+            "date_released": "2023-01-18T00:00:00Z",
+            "description": "An ontology providing controlled vocabularies for Sequence Analysis and sequencing instrument model.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HCODONONT/4/hcodonont.owl",
+            "download_url": "media/ontologies/ASSAYNODE/2/assaynode.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://blog.51.ca/u-345129/",
-            "id": 235,
-            "indexed": false,
-            "max_children": 94,
-            "max_depth": 1,
+            "home_page": null,
+            "id": 5623,
+            "indexed": true,
+            "max_children": 12,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 98,
+            "nb_classes": 48,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "AHSO",
-        "name": "Animal Health Surveillance Ontology",
+        "acronym": "bibliographic",
+        "name": "Bibliographic",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-07-31T06:29:33Z",
-            "date_released": "2018-07-30T00:00:00Z",
-            "description": "(Work in progress - please contact ahso@datadrivensurveillance.org before using this ontology)\r\n\r\nThis is an ontology for health surveillance, assuming that surveillance data is recorded at three main levels:\r\n\r\nSAMPLE data - data about specific collected samples; OBSERVATIONS, which can aggregate several samples; and OBSERVATIONAL CONTEXT, which is the context in which the observations were made (for instance clinical observation, surveillance sampling, etc). \r\n\r\nHEALTH EVENT is the occurrence in the real World to be modeled. This occurrence is an abstract concept, as it generally cannot be determined what are its boundaries in time, space, or population units. \r\n\r\nWe assume that several OBSERVATIONS are derived from each health event, and recorded in the same or different databases - for instance clinical journal, laboratory database, abattoir. At the same time or not. \r\n\r\nAHSO attempts to model these time specific observations. The ontology is focused on the modeling of the animal and health information connected to a single observation, made at a specific point on time. All the information recorded is relative to that specific point in time.\r\n\r\nFor instance, an animal belongs to a herd, which belongs to an owner, AT THE TIME OF THE OBSERVATION. We do not attempt to model how that animal changed herds on time. Similarly, we do not model clinical progression. It is up to those using the data to create definitions of when two observations should be connected to the same event.",
-            "documentation": "https://github.com/SVA-SE/AHSO",
+            "date_created": "2012-08-31T01:25:04Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-public-1.5.owl\r\n\r\nThis file contains selected terms from the bibliographic ontology (see http://bibliontology.com/) which relate to information resources.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/AHSO/1/ahso.owl",
+            "download_url": "media/ontologies/bibliographic/1/bibliographic.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://datadrivensurveillance.org/ahso",
-            "id": 1706,
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 448,
             "indexed": true,
-            "max_children": 9,
+            "max_children": 19,
             "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 38,
+            "nb_classes": 63,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 31,
+            "nb_properties": 77,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.1.0"
+            "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "HENECON",
-        "name": "Head and Neck Cancer Ontology",
+        "acronym": "HMIS033B",
+        "name": "HMIS 033",
         "species": null,
-        "status": "Unloadable",
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "BIBLIOTEK-O",
+        "name": "The Biblioteko Ontology",
+        "species": null,
+        "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 9,
             "classifiable": true,
-            "date_created": "2023-03-21T12:22:36Z",
-            "date_released": "2020-04-14T00:00:00Z",
-            "description": "HeNeCOn (Head and Neck Cancer Ontology) is the first ontology specifically focused on the Head and Neck Cancer (HNC) domain. It provides a clinically reliable data model for HNC processes during all phases of the disease: prognosis, treatment, and follow-up. The ontology is established based on an annotated dataset to provide formal reference descriptions, contains  170 clinical variables organized into a taxonomy, and is expanded and mapped to formalize and integrate multiple databases into the HeNeCOn ontology. The outcomes of this iterative process were reviewed and validated by clinicians and statisticians. HeNeCOn, with roughly 500 classes, is a very specific ontology which includes a taxonomy with a hierarchical structure, semantic definitions of 283 medical terms and detailed relations between them, which makes it a reference data model. ",
+            "date_created": "2017-07-12T01:02:14Z",
+            "date_released": "2017-07-12T01:02:14Z",
+            "description": "An ontology recommendation for modeling bibliographic metadata",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HENECON/1/henecon.owl",
+            "download_url": "media/ontologies/BIBLIOTEK-O/3/bibliotek-o.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5753,
+            "home_page": "http://bibliotek-o.org/",
+            "id": 471,
             "indexed": true,
-            "max_children": 81,
-            "max_depth": 6,
+            "max_children": 140,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 1933,
+            "nb_classes": 278,
             "nb_inconsistent": 0,
-            "nb_individuals": 1472,
-            "nb_properties": 0,
+            "nb_individuals": 30,
+            "nb_properties": 24,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.0"
+            "version": "Version 1.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "AISM",
-        "name": "Ontology for the Anatomy of the Insect SkeletoMuscular system (AISM)",
+        "acronym": "BFO",
+        "name": "Basic Formal Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-04-17T12:00:07.519273Z",
-            "date_released": "2023-04-17T12:00:07.519252Z",
-            "description": "The AISM contains terms used in insect biodiversity research for describing structures of the exoskeleton and the skeletomuscular system. It aims to serve as the basic backbone of generalized terms to be expanded with order-specific terminology.",
+            "date_created": "2019-09-02T12:00:03.790386Z",
+            "date_released": "2019-09-02T12:00:03.790326Z",
+            "description": "The upper level ontology upon which OBO Foundry ontologies are built.",
             "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/AISM/13/aism.owl",
+            "domain": "upper",
+            "download_url": "media/ontologies/BFO/5/bfo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/insect-morphology/aism",
-            "id": 5765,
+            "home_page": "http://ifomis.org/bfo/",
+            "id": 2877,
             "indexed": true,
-            "max_children": 270,
-            "max_depth": 41,
-            "md5sum": "e79c22d3c423e72b01c45968074810fe",
-            "nb_classes": 7443,
+            "max_children": 4,
+            "max_depth": 6,
+            "md5sum": "738838092081290736ec3a1485cd07cb",
+            "nb_classes": 36,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 306,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": [
                 {
-                    "id": "aism.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/aism.owl"
-                },
-                {
-                    "id": "aism.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/aism.obo"
+                    "id": "bfo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bfo.owl"
                 },
                 {
-                    "id": "aism.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/aism.json"
+                    "id": "bfo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bfo.obo"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 13,
+            "submission_id": 5,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ALLERGYDETECTOR",
-        "name": "Allergy Detector II",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2015-05-02T00:22:05Z",
-            "date_released": "2015-05-01T00:00:00Z",
-            "description": "Allergy Detection Ontology.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ALLERGYDETECTOR/1/allergydetector.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 740,
-            "indexed": true,
-            "max_children": 4,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 12,
-            "nb_inconsistent": 0,
-            "nb_individuals": 161,
-            "nb_properties": 20,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "0.0.1"
-        },
-        "topics": null
-    },
-    {
         "acronym": "CGBOARD",
         "name": "Cg Board 10th Time Table 2019",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -6423,86 +6423,14 @@
         "name": "nidhiagarwal",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "tfernandes",
-        "name": "Thiago",
-        "species": [
-            "Aspergillus"
-        ],
-        "status": "Unknown",
-        "submission": null,
-        "topics": []
-    },
-    {
-        "acronym": "MTO",
-        "name": "MyTestOntology",
-        "species": [
-            "TestSpecies"
-        ],
-        "status": "Unknown",
-        "submission": null,
-        "topics": [
-            "TestTopic"
-        ]
-    },
-    {
-        "acronym": "VODANA-UG",
-        "name": "VODANA-Uganda",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "HPV",
-        "name": "HPV Cancer Dictionary",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "GWAS_EFO_SKOS",
-        "name": "GWAS view",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2012-02-03T15:55:28Z",
-            "date_released": "2012-02-03T08:00:00Z",
-            "description": "This is an experimental view that uses the Simple Knowledge Organisation System (SKOS) to represent a view over EFO terms. These EFO terms are a subset of terms that are being used to annotate data for the Genome Wide Association Studies (GWAS) curators. ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/GWAS_EFO_SKOS/1/gwas_efo_skos.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 119,
-            "indexed": true,
-            "max_children": 1813,
-            "max_depth": 14,
-            "md5sum": null,
-            "nb_classes": 19787,
-            "nb_inconsistent": 0,
-            "nb_individuals": 365,
-            "nb_properties": 76,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "unknown"
-        },
-        "topics": null
-    },
-    {
         "acronym": "ADCAD",
         "name": "Arctic Data Center Academic Disciplines Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
@@ -6529,358 +6457,83 @@
             "submission_id": 1,
             "taxon": null,
             "version": "Version 1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "NDF-RT",
-        "name": "National Drug File \u2013 Reference Terminology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2012-01-20T06:47:26Z",
-            "date_released": "2011-10-14T07:00:00Z",
-            "description": "Logically inferred version in OWL of the National Drug File \u2013 Reference Terminology (NDF-RT). NDF-RT is the terminology used by FDA and the FedMed collaboration to code these essential pharmacologic properties of medications: Mechanism of Action, Physiologic Effect, Structural Class. ",
-            "documentation": "http://evs.nci.nih.gov/ftp1/NDF-RT/",
-            "domain": null,
-            "download_url": "media/ontologies/NDF-RT/1/ndf-rt.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 643,
-            "indexed": true,
-            "max_children": 5507,
-            "max_depth": 13,
-            "md5sum": null,
-            "nb_classes": 46046,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 55,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "NDF-RT2 [Public Edition]"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NEGO",
-        "name": "Nutritional Epidemiological STandard",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2018-08-04T13:36:06Z",
-            "date_released": "2018-08-04T00:00:00Z",
-            "description": "It represents standards for nutritional epidemiological studies in ontology format. \r\nIn the strict sense, it is not an ontology. Its hierarchy follows structure of each standard, not OWL rules.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NEGO/1/nego.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/cyang0128/Nutritional-epidemiological-ontologies",
-            "id": 1725,
-            "indexed": true,
-            "max_children": 17,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 129,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": "https://academic.oup.com/jn/article/148/2/285/4913031",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "V1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NETFLIX",
-        "name": "Netflix Class Ontology ",
+        "acronym": "GEO",
+        "name": "Geographical Entity Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-28T19:29:32Z",
-            "date_released": "2023-06-28T00:00:00Z",
-            "description": "Learning how to biuld ontologies during UHC 2023 Summer Institute",
+            "date_created": "2023-05-22T12:41:28.663533Z",
+            "date_released": "2023-05-22T12:41:28.663518Z",
+            "description": "An ontology of geographical entities",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NETFLIX/1/netflix.owl",
+            "domain": "environment",
+            "download_url": "media/ontologies/GEO/5/geo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5914,
+            "home_page": "https://github.com/ufbmi/geographical-entity-ontology/wiki",
+            "id": 5818,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 20,
+            "max_children": 6,
+            "max_depth": 9,
+            "md5sum": "2ec5ae0760f0817e921d64c686d8ca63",
+            "nb_classes": 90,
             "nb_inconsistent": 0,
-            "nb_individuals": 21,
-            "nb_properties": 5,
-            "products": null,
+            "nb_individuals": 4,
+            "nb_properties": 49,
+            "products": [
+                {
+                    "id": "geo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/geo.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 5,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NIGO",
-        "name": "Neural-Immune Gene Ontology",
+        "acronym": "geopolitical",
+        "name": "Geopolitical",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2010-10-29T00:24:54Z",
-            "date_released": "2010-09-12T07:00:00Z",
-            "description": "The Neural-Immune Gene Ontology (NIGO) is a subset of GO directed for neurological and immunological systems. NIGO was created by clipping those GO terms that are not associated to any gene in human, rat and mouse, and by clipping terms not found to be relevant to the neural and/or immune domains.",
-            "documentation": "http://bioinfo.bgu.ac.il/rubin/supplementary/NIGO/Supplementary.html",
-            "domain": null,
-            "download_url": "media/ontologies/NIGO/2/nigo.obo",
-            "has_ontology_language": "OBO",
-            "home_page": null,
-            "id": 526,
-            "indexed": true,
-            "max_children": 191,
-            "max_depth": 13,
-            "md5sum": null,
-            "nb_classes": 4842,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": null,
-            "publication": "http://www.biomedcentral.com/1471-2105/11/458",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "v1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NIHSS",
-        "name": "National Institutes of Health Stroke Scale Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 9,
-            "classifiable": true,
-            "date_created": "2016-09-27T01:09:01Z",
-            "date_released": "2016-09-27T01:09:01Z",
-            "description": "Domain ontology for National Institutes of Health Stroke Scale (NIHSS). Scores range from 0-42. Patients are given more points for greater deficiencies. A score of 0 indicates that the test is normal.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NIHSS/11/nihss.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 183,
-            "indexed": true,
-            "max_children": 15,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 18,
-            "nb_inconsistent": 0,
-            "nb_individuals": 106,
-            "nb_properties": 22,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 11,
-            "taxon": null,
-            "version": "2016/09"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NIO",
-        "name": "Neuropsychological Integrative Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2020-12-23T13:16:47Z",
-            "date_released": "2020-03-24T00:00:00Z",
-            "description": "An integrative ontology of the four main domains of neuropsychological diseases, brain areas, cognitive functions and neuropsychological tests",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NIO/2/nio.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4152,
-            "indexed": true,
-            "max_children": 150,
-            "max_depth": 20,
-            "md5sum": null,
-            "nb_classes": 3303,
-            "nb_inconsistent": 0,
-            "nb_individuals": 917,
-            "nb_properties": 168,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "1.1.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "GECCO",
-        "name": "COVID-19 Common Datamodel",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "NIST_GEL",
-        "name": "NIST Genome Editing Lexicon",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2022-01-06T17:01:17Z",
-            "date_released": "2020-11-23T00:00:00Z",
-            "description": "Genome editing technology is a fast-growing and rapidly advancing global bioscience field with applications in many biotechnology sectors. Genome editing is used to modify the nucleic acids of a genetic code, which can be composed of DNA or RNA, in a site-specific manner. Modifications can include insertion, deletion or alteration of nucleic acids. The technology operates by biochemical principles generally applicable to every kind of cell. Examples of genome editing technology applications with global significance include human cell-based therapeutics, agriculture, microbial based therapeutics, synthetic biology and biomanufacturing. While genome editing technology is being actively utilized, there is a need for international standardization in terms and definitions for this field, so as to enhance interpretation and communication of concepts, data and results. This document has been developed to provide a unified standard set of terms and definitions that serve the needs of biotechnology stakeholders and act as a reference for genome editing technology.",
-            "documentation": "https://www.nist.gov/programs-projects/nist-genome-editing-lexicon",
-            "domain": null,
-            "download_url": "media/ontologies/NIST_GEL/7/nist_gel.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.nist.gov/programs-projects/nist-genome-editing-lexicon",
-            "id": 4822,
-            "indexed": true,
-            "max_children": 9,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 52,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 9,
-            "products": null,
-            "publication": "https://www.iso.org/standard/80679.html?browse=tc",
-            "publications": null,
-            "submission_id": 7,
-            "taxon": null,
-            "version": "2.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NMDCO",
-        "name": "NMDC Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2021-09-22T01:16:36Z",
-            "date_released": "2021-09-22T01:16:28Z",
-            "description": "The NMDC Ontology (NMDCO) is used by the National Microbiome Data Collaborative (NMDC, https://microbiomedata.org) to annotate multi-omic microbiome metadata. It merges a number of ontologies relevant to the NMDC (such as the plant and environment ontologies) into a single ontology.\r\nThe ontology can be download from Github at https://github.com/microbiomedata/nmdc-ontology.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NMDCO/4/nmdco.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/microbiomedata/nmdc-ontology",
-            "id": 4613,
-            "indexed": true,
-            "max_children": 91,
-            "max_depth": 35,
-            "md5sum": null,
-            "nb_classes": 7766,
-            "nb_inconsistent": 1,
-            "nb_individuals": 49,
-            "nb_properties": 604,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "2021-09-21"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ADHER_INTCARE_SP",
-        "name": "Adherence and Integrated Care in Spanish",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2019-06-17T11:40:13Z",
-            "date_released": "2019-04-26T00:00:00Z",
-            "description": "This ontology includes concepts in Spanish related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu.",
+            "date_created": "2012-08-31T01:17:12Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-public-1.5.owl\r\n\r\nThis is the geopolitical ontology from the Food and Agriculture Organization of the United Nations",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ADHER_INTCARE_SP/6/adher_intcare_sp.owl",
+            "download_url": "media/ontologies/geopolitical/1/geopolitical.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2661,
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 44,
             "indexed": true,
-            "max_children": 23,
-            "max_depth": 7,
+            "max_children": 4,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 265,
+            "nb_classes": 12,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 12,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 6,
-            "taxon": null,
-            "version": "Spanish 052319"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ADMIN",
-        "name": "Nurse Administrator",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2015-09-24T02:12:30Z",
-            "date_released": "2015-09-24T00:00:00Z",
-            "description": "This is an ontology showing a nurse administrator's process domain",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ADMIN/1/admin.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 267,
-            "indexed": true,
-            "max_children": 10,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 44,
-            "nb_inconsistent": 0,
-            "nb_individuals": 63,
-            "nb_properties": 42,
+            "nb_properties": 95,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "version 1.1"
         },
         "topics": null
     },
     {
         "acronym": "ADO",
         "name": "Alzheimer's Disease Ontology",
         "species": null,
@@ -6916,14 +6569,36 @@
             "submission_id": 7,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "tfernandes",
+        "name": "Thiago",
+        "species": [
+            "Aspergillus"
+        ],
+        "status": "Unknown",
+        "submission": null,
+        "topics": []
+    },
+    {
+        "acronym": "MTO",
+        "name": "MyTestOntology",
+        "species": [
+            "TestSpecies"
+        ],
+        "status": "Unknown",
+        "submission": null,
+        "topics": [
+            "TestTopic"
+        ]
+    },
+    {
         "acronym": "ADW",
         "name": "Animal Natural History and Life History Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 6,
             "classifiable": true,
@@ -7098,14 +6773,22 @@
             "submission_id": 5,
             "taxon": null,
             "version": "1.0.4"
         },
         "topics": null
     },
     {
+        "acronym": "VODANA-UG",
+        "name": "VODANA-Uganda",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
         "acronym": "ATOM",
         "name": "Atlas Ontology Model",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -7132,14 +6815,22 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2023-04-27"
         },
         "topics": null
     },
     {
+        "acronym": "HPV",
+        "name": "HPV Cancer Dictionary",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
         "acronym": "CARELEX",
         "name": "Content Archive Resource Exchange Lexicon",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -7278,432 +6969,405 @@
             "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "GEO",
-        "name": "Geographical Entity Ontology",
+        "acronym": "NIST_GEL",
+        "name": "NIST Genome Editing Lexicon",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-05-22T12:41:28.663533Z",
-            "date_released": "2023-05-22T12:41:28.663518Z",
-            "description": "An ontology of geographical entities",
-            "documentation": null,
-            "domain": "environment",
-            "download_url": "media/ontologies/GEO/5/geo.owl",
+            "date_created": "2022-01-06T17:01:17Z",
+            "date_released": "2020-11-23T00:00:00Z",
+            "description": "Genome editing technology is a fast-growing and rapidly advancing global bioscience field with applications in many biotechnology sectors. Genome editing is used to modify the nucleic acids of a genetic code, which can be composed of DNA or RNA, in a site-specific manner. Modifications can include insertion, deletion or alteration of nucleic acids. The technology operates by biochemical principles generally applicable to every kind of cell. Examples of genome editing technology applications with global significance include human cell-based therapeutics, agriculture, microbial based therapeutics, synthetic biology and biomanufacturing. While genome editing technology is being actively utilized, there is a need for international standardization in terms and definitions for this field, so as to enhance interpretation and communication of concepts, data and results. This document has been developed to provide a unified standard set of terms and definitions that serve the needs of biotechnology stakeholders and act as a reference for genome editing technology.",
+            "documentation": "https://www.nist.gov/programs-projects/nist-genome-editing-lexicon",
+            "domain": null,
+            "download_url": "media/ontologies/NIST_GEL/7/nist_gel.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ufbmi/geographical-entity-ontology/wiki",
-            "id": 5818,
+            "home_page": "https://www.nist.gov/programs-projects/nist-genome-editing-lexicon",
+            "id": 4822,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 9,
-            "md5sum": "2ec5ae0760f0817e921d64c686d8ca63",
-            "nb_classes": 90,
+            "max_children": 9,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 52,
             "nb_inconsistent": 0,
-            "nb_individuals": 4,
-            "nb_properties": 49,
-            "products": [
-                {
-                    "id": "geo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/geo.owl"
-                }
-            ],
-            "publication": null,
+            "nb_individuals": 0,
+            "nb_properties": 9,
+            "products": null,
+            "publication": "https://www.iso.org/standard/80679.html?browse=tc",
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 7,
             "taxon": null,
-            "version": null
+            "version": "2.1"
         },
         "topics": null
     },
     {
-        "acronym": "geopolitical",
-        "name": "Geopolitical",
+        "acronym": "NMDCO",
+        "name": "NMDC Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2012-08-31T01:17:12Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-public-1.5.owl\r\n\r\nThis is the geopolitical ontology from the Food and Agriculture Organization of the United Nations",
+            "date_created": "2021-09-22T01:16:36Z",
+            "date_released": "2021-09-22T01:16:28Z",
+            "description": "The NMDC Ontology (NMDCO) is used by the National Microbiome Data Collaborative (NMDC, https://microbiomedata.org) to annotate multi-omic microbiome metadata. It merges a number of ontologies relevant to the NMDC (such as the plant and environment ontologies) into a single ontology.\r\nThe ontology can be download from Github at https://github.com/microbiomedata/nmdc-ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/geopolitical/1/geopolitical.owl",
+            "download_url": "media/ontologies/NMDCO/4/nmdco.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 44,
+            "home_page": "https://github.com/microbiomedata/nmdc-ontology",
+            "id": 4613,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 2,
+            "max_children": 91,
+            "max_depth": 35,
             "md5sum": null,
-            "nb_classes": 12,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 95,
+            "nb_classes": 7766,
+            "nb_inconsistent": 1,
+            "nb_individuals": 49,
+            "nb_properties": 604,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "version 1.1"
+            "version": "2021-09-21"
         },
         "topics": null
     },
     {
-        "acronym": "cd",
-        "name": "ashuu",
-        "species": [
-            "corpus"
-        ],
-        "status": "Unknown",
-        "submission": null,
-        "topics": [
-            "nlp"
-        ]
-    },
-    {
-        "acronym": "AGROCYMAC",
-        "name": "Control y Monitoreo de afecciones en los cultivos",
+        "acronym": "ADHER_INTCARE_SP",
+        "name": "Adherence and Integrated Care in Spanish",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2019-04-03T21:04:23Z",
-            "date_released": "2019-02-17T00:00:00Z",
-            "description": "Agrocymac es una ontolog\u00eda creada para estructurar informaci\u00f3n sobre el control y monitoreo de plagas, malezas y enfermedades que afectan espec\u00edficamente a los cultivos de Arroz, Caf\u00e9 y Cacao.",
+            "date_created": "2019-06-17T11:40:13Z",
+            "date_released": "2019-04-26T00:00:00Z",
+            "description": "This ontology includes concepts in Spanish related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/AGROCYMAC/2/agrocymac.owl",
+            "download_url": "media/ontologies/ADHER_INTCARE_SP/6/adher_intcare_sp.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 2441,
+            "id": 2661,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
+            "max_children": 23,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 13,
+            "nb_classes": 265,
             "nb_inconsistent": 0,
-            "nb_individuals": 148,
-            "nb_properties": 9,
+            "nb_individuals": 0,
+            "nb_properties": 12,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 6,
             "taxon": null,
-            "version": null
+            "version": "Spanish 052319"
         },
         "topics": null
     },
     {
-        "acronym": "AIPBC",
-        "name": "Aromatase_Inhibitor_Pathway_Breast_Cell_Pharmacodynamics",
-        "species": [],
+        "acronym": "ADMIN",
+        "name": "Nurse Administrator",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2019-10-01T07:27:18.316017Z",
-            "date_released": "2019-10-01T07:27:18.316047Z",
-            "description": "",
+            "date_created": "2015-09-24T02:12:30Z",
+            "date_released": "2015-09-24T00:00:00Z",
+            "description": "This is an ontology showing a nurse administrator's process domain",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/AIPBC/1/aipbc.owl",
+            "download_url": "media/ontologies/ADMIN/1/admin.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 2974,
+            "id": 267,
             "indexed": true,
-            "max_children": 15,
-            "max_depth": 5,
+            "max_children": 10,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 69,
+            "nb_classes": 44,
             "nb_inconsistent": 0,
-            "nb_individuals": 72,
-            "nb_properties": 55,
+            "nb_individuals": 63,
+            "nb_properties": 42,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
-        "topics": []
-    },
-    {
-        "acronym": "HMADO",
-        "name": "Human Microbiome Associated Disease Ontology ",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "WHEAT",
-        "name": "walgreens wheat ridge co",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
         "topics": null
     },
     {
-        "acronym": "NONRCTO",
-        "name": "Non-Randomized Controlled Trials Ontology",
+        "acronym": "NDF-RT",
+        "name": "National Drug File \u2013 Reference Terminology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2012-02-24T23:41:16Z",
-            "date_released": "2012-01-18T08:00:00Z",
-            "description": "We built an ontology to help the systematic review and meta-analysis process of non randomized clinical trials.",
-            "documentation": null,
+            "date_created": "2012-01-20T06:47:26Z",
+            "date_released": "2011-10-14T07:00:00Z",
+            "description": "Logically inferred version in OWL of the National Drug File \u2013 Reference Terminology (NDF-RT). NDF-RT is the terminology used by FDA and the FedMed collaboration to code these essential pharmacologic properties of medications: Mechanism of Action, Physiologic Effect, Structural Class. ",
+            "documentation": "http://evs.nci.nih.gov/ftp1/NDF-RT/",
             "domain": null,
-            "download_url": "media/ontologies/NONRCTO/1/nonrcto.owl",
+            "download_url": "media/ontologies/NDF-RT/1/ndf-rt.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 292,
+            "id": 643,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 5,
+            "max_children": 5507,
+            "max_depth": 13,
             "md5sum": null,
-            "nb_classes": 156,
-            "nb_inconsistent": 5,
-            "nb_individuals": 31,
-            "nb_properties": 9,
+            "nb_classes": 46046,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 55,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2.0"
+            "version": "NDF-RT2 [Public Edition]"
         },
         "topics": null
     },
     {
-        "acronym": "AI-RHEUM",
-        "name": "Artificial Intelligence Rheumatology Consultant System Ontology",
+        "acronym": "NEGO",
+        "name": "Nutritional Epidemiological STandard",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": null,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-02-06T21:25:57Z",
-            "date_released": "2017-02-06T21:25:57Z",
-            "description": "AI/RHEUM is used for the diagnosis of rheumatologic diseases. AI/RHEUM contains findings, such as clinical signs, symptoms, laboratory test results, radiologic observations, tissue biopsy results, and intermediate diagnosis hypotheses. Findings and hypotheses, which include definitions, are used to reach diagnostic conclusions with definite, probable, or possible certainty. AI/RHEUM is used by clinicians and informatics researchers.",
+            "date_created": "2018-08-04T13:36:06Z",
+            "date_released": "2018-08-04T00:00:00Z",
+            "description": "It represents standards for nutritional epidemiological studies in ontology format. \r\nIn the strict sense, it is not an ontology. Its hierarchy follows structure of each standard, not OWL rules.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/AI-RHEUM/9/ai-rheum.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": null,
-            "id": 279,
+            "download_url": "media/ontologies/NEGO/1/nego.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/cyang0128/Nutritional-epidemiological-ontologies",
+            "id": 1725,
             "indexed": true,
-            "max_children": 81,
-            "max_depth": 4,
+            "max_children": 17,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 808,
+            "nb_classes": 129,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "https://academic.oup.com/jn/article/148/2/285/4913031",
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2016AB"
+            "version": "V1.0"
         },
         "topics": null
     },
     {
-        "acronym": "AMINO-ACID",
-        "name": "Amino Acid Ontology",
+        "acronym": "NETFLIX",
+        "name": "Netflix Class Ontology ",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2010-07-02T17:57:00Z",
-            "date_released": "2007-01-10T08:00:00Z",
-            "description": "An ontology of amino acids and their properties. Inferred version.",
+            "date_created": "2023-06-28T19:29:32Z",
+            "date_released": "2023-06-28T00:00:00Z",
+            "description": "Learning how to biuld ontologies during UHC 2023 Summer Institute",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/AMINO-ACID/2/amino-acid.owl",
+            "download_url": "media/ontologies/NETFLIX/1/netflix.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.co-ode.org/ontologies/amino-acid/2009/02/16/",
-            "id": 258,
+            "home_page": null,
+            "id": 5914,
             "indexed": true,
-            "max_children": 9,
+            "max_children": 7,
             "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 46,
+            "nb_classes": 20,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_individuals": 21,
+            "nb_properties": 5,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "classified"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TMD",
-        "name": "Tam Minh Duogn Pharmacy",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "BIM",
-        "name": "Biomedical Image Ontology",
+        "acronym": "NIGO",
+        "name": "Neural-Immune Gene Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2015-10-08T22:02:57Z",
-            "date_released": "2014-09-26T00:00:00Z",
-            "description": "Biomedical images published within the scientific literature play a central role in reporting and facilitating life science discoveries. Existing ontologies and vocabularies describing biomedical images, particularly sequence images, do not provide sufficient semantic representation for image annotations generated automatically and/or semi-automatically. We present an open ontology for the annotation of biomedical images (BIM) scripted in OWL/RDF. The BIM ontology provides semantic vocabularies to describe the manually curated image annotations as well as annotations generated by online bioinformatics services using content extracted from images by the Semantic Enrichment of Biomedical Images (SEBI) system. The BIM ontology is represented in three parts; (i) image vocabularies - which holds vocabularies for the annotation of an image and/or region of interests (ROI) inside an image, as well as vocabularies to represent the pre and post processing states of an image, (ii) text entities - covers annotations from the text that are associated with an image (e.g. image captions) and provides semantic representation for NLP algorithm outputs, (iii) a provenance model  - that contributes towards the maintenance of annotation versioning. To illustrate the BIM ontology\u2019s utility, we provide three annotation cases generated by BIM in conjunction with the SEBI image annotation engine.",
-            "documentation": "http://cbakerlab.unbsj.ca/unbvps/BIM",
+            "date_created": "2010-10-29T00:24:54Z",
+            "date_released": "2010-09-12T07:00:00Z",
+            "description": "The Neural-Immune Gene Ontology (NIGO) is a subset of GO directed for neurological and immunological systems. NIGO was created by clipping those GO terms that are not associated to any gene in human, rat and mouse, and by clipping terms not found to be relevant to the neural and/or immune domains.",
+            "documentation": "http://bioinfo.bgu.ac.il/rubin/supplementary/NIGO/Supplementary.html",
             "domain": null,
-            "download_url": "media/ontologies/BIM/6/bim.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://sites.google.com/site/bimontology/",
-            "id": 559,
+            "download_url": "media/ontologies/NIGO/2/nigo.obo",
+            "has_ontology_language": "OBO",
+            "home_page": null,
+            "id": 526,
             "indexed": true,
-            "max_children": 50,
-            "max_depth": 3,
+            "max_children": 191,
+            "max_depth": 13,
             "md5sum": null,
-            "nb_classes": 125,
+            "nb_classes": 4842,
             "nb_inconsistent": 0,
-            "nb_individuals": 34,
-            "nb_properties": 73,
+            "nb_individuals": 0,
+            "nb_properties": 4,
             "products": null,
-            "publication": "https://code.google.com/p/bio-ont/",
+            "publication": "http://www.biomedcentral.com/1471-2105/11/458",
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.3"
+            "version": "v1"
         },
         "topics": null
     },
     {
-        "acronym": "BIRTHDAYH",
-        "name": "NeuroMorpho.Org cell types birthdate",
+        "acronym": "NIHSS",
+        "name": "National Institutes of Health Stroke Scale Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 9,
             "classifiable": true,
-            "date_created": "2017-03-31T01:00:16Z",
-            "date_released": "2017-03-31T01:00:16Z",
-            "description": "A hierarchy of neuron types based on its birthdate. ",
+            "date_created": "2016-09-27T01:09:01Z",
+            "date_released": "2016-09-27T01:09:01Z",
+            "description": "Domain ontology for National Institutes of Health Stroke Scale (NIHSS). Scores range from 0-42. Patients are given more points for greater deficiencies. A score of 0 indicates that the test is normal.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BIRTHDAYH/4/birthdayh.owl",
+            "download_url": "media/ontologies/NIHSS/11/nihss.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 652,
+            "id": 183,
             "indexed": true,
-            "max_children": 13,
+            "max_children": 15,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 15,
+            "nb_classes": 18,
             "nb_inconsistent": 0,
-            "nb_individuals": 13,
-            "nb_properties": 0,
+            "nb_individuals": 106,
+            "nb_properties": 22,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 11,
             "taxon": null,
-            "version": "2.0"
+            "version": "2016/09"
         },
         "topics": null
     },
     {
-        "acronym": "BLM",
-        "name": "BioLink Model",
+        "acronym": "NIO",
+        "name": "Neuropsychological Integrative Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-12-23T02:00:47Z",
-            "date_released": "2021-12-23T02:00:45Z",
-            "description": "A high level datamodel of biological entities (genes, diseases, phenotypes, pathways, individuals, substances, etc) and their associations",
-            "documentation": "https://github.com/biolink/biolink-model",
+            "date_created": "2020-12-23T13:16:47Z",
+            "date_released": "2020-03-24T00:00:00Z",
+            "description": "An integrative ontology of the four main domains of neuropsychological diseases, brain areas, cognitive functions and neuropsychological tests",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BLM/79/blm.owl",
+            "download_url": "media/ontologies/NIO/2/nio.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://biolink.github.io/biolink-model/",
-            "id": 4794,
+            "home_page": null,
+            "id": 4152,
             "indexed": true,
-            "max_children": 46,
-            "max_depth": 7,
+            "max_children": 150,
+            "max_depth": 20,
             "md5sum": null,
-            "nb_classes": 292,
+            "nb_classes": 3303,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 443,
+            "nb_individuals": 917,
+            "nb_properties": 168,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 79,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "1.1.1"
         },
         "topics": null
     },
     {
-        "acronym": "BPT",
-        "name": "Biological Pathway Taxonomy",
-        "species": [
-            "human",
-            "mice",
-            "rat"
-        ],
-        "status": "Classified",
+        "acronym": "GECCO",
+        "name": "COVID-19 Common Datamodel",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "GWAS_EFO_SKOS",
+        "name": "GWAS view",
+        "species": null,
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 19,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2022-03-29T16:57:06.962723Z",
-            "date_released": "2022-03-29T16:57:06.962743Z",
-            "description": "Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage.\r\n\r\n\uf0e0\uf0e0\uf0e0 There are 3 types of entities in BPT \u2013 pathway, group, and folder of pathways and groups. \r\n\uf0e0\uf0e0\uf0e0 Group is the list of genes and proteins associated with biological function and diseases. \r\n\uf0e0\uf0e0\uf0e0 Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control.\r\n\r\n\uf0e0\uf0e0\uf0e0 BPT pathway entity includes several properties such as \r\n link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. \uf0e0\uf0e0\uf0e0 The link can be found among synonymes. List of members for each model can be provided by request. Property \u201cnetwork\u201d describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (\u201csignaling\u201d).\r\n\r\n\uf0e0\uf0e0\uf0e0 The index folder contains all pathways and groups sorted alphabetically.",
-            "documentation": "https://mammal-profservices.pathwaystudio.com/",
+            "date_created": "2012-02-03T15:55:28Z",
+            "date_released": "2012-02-03T08:00:00Z",
+            "description": "This is an experimental view that uses the Simple Knowledge Organisation System (SKOS) to represent a view over EFO terms. These EFO terms are a subset of terms that are being used to annotate data for the Genome Wide Association Studies (GWAS) curators. ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BPT/2/bpt.obo",
-            "has_ontology_language": "OBO",
+            "download_url": "media/ontologies/GWAS_EFO_SKOS/1/gwas_efo_skos.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4992,
+            "id": 119,
             "indexed": true,
-            "max_children": 438,
-            "max_depth": 6,
+            "max_children": 1813,
+            "max_depth": 14,
             "md5sum": null,
-            "nb_classes": 3875,
+            "nb_classes": 19787,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 365,
+            "nb_properties": 76,
             "products": null,
-            "publication": "Nesterova, A.P., Klimov, E.A., Zharkova, M., Sozin, S., Sobolev, V., Ivanikova, N.V., Shkrob, M., Yuryev, A., 2020. Introduction, in: Disease Pathways. Elsevier, pp. 3\u201332. https://doi.org/10.1016/b978-0-12-817086-1.00001-4",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "BPT.properties.2022"
+            "version": "unknown"
         },
+        "topics": null
+    },
+    {
+        "acronym": "cd",
+        "name": "ashuu",
+        "species": [
+            "corpus"
+        ],
+        "status": "Unknown",
+        "submission": null,
         "topics": [
-            "Cell signaling pathways"
+            "nlp"
         ]
     },
     {
         "acronym": "building",
         "name": "Building and Sub Elements",
         "species": [
             "wall base ceiling room floor building"
@@ -7776,14 +7440,80 @@
             "submission_id": 17,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "HMADO",
+        "name": "Human Microbiome Associated Disease Ontology ",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "WHEAT",
+        "name": "walgreens wheat ridge co",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "NONRCTO",
+        "name": "Non-Randomized Controlled Trials Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2012-02-24T23:41:16Z",
+            "date_released": "2012-01-18T08:00:00Z",
+            "description": "We built an ontology to help the systematic review and meta-analysis process of non randomized clinical trials.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/NONRCTO/1/nonrcto.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 292,
+            "indexed": true,
+            "max_children": 8,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 156,
+            "nb_inconsistent": 5,
+            "nb_individuals": 31,
+            "nb_properties": 9,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "2.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TMD",
+        "name": "Tam Minh Duogn Pharmacy",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "QUDT2-1",
+        "name": "Quantities, Units, Dimensions and Types (QUDT Version 2.1)",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
         "acronym": "CCCNO",
         "name": "CCC Nursing Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
@@ -7810,22 +7540,192 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "QUDT2-1",
-        "name": "Quantities, Units, Dimensions and Types (QUDT Version 2.1)",
+        "acronym": "GFFO",
+        "name": "Genetic Feature Format Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 1,
+            "classifiable": true,
+            "date_created": "2022-03-28T15:58:06Z",
+            "date_released": "2022-03-28T00:00:00Z",
+            "description": "An ontology to represent genome feature annotations in gff3 format.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/GFFO/1/gffo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/mpievolbio-scicomp/GenomeFeatureFormatOntology",
+            "id": 4987,
+            "indexed": true,
+            "max_children": 1,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 1,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 33,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "0.0.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ISO19110",
+        "name": "ISO 19110 Methodology for Feature Cataloguing",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": false,
+            "date_created": "2017-07-22T14:32:20Z",
+            "date_released": "2017-07-22T00:00:00Z",
+            "description": "ISO 19110:2005 Methodology for feature cataloguing",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/ISO19110/1/iso19110.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 69,
+            "indexed": false,
+            "max_children": 22,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 36,
+            "nb_inconsistent": 0,
+            "nb_individuals": 5,
+            "nb_properties": 57,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "2005-02-15"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ONTODRC",
+        "name": "ONTODRC",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
+        "acronym": "AGROCYMAC",
+        "name": "Control y Monitoreo de afecciones en los cultivos",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2019-04-03T21:04:23Z",
+            "date_released": "2019-02-17T00:00:00Z",
+            "description": "Agrocymac es una ontolog\u00eda creada para estructurar informaci\u00f3n sobre el control y monitoreo de plagas, malezas y enfermedades que afectan espec\u00edficamente a los cultivos de Arroz, Caf\u00e9 y Cacao.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/AGROCYMAC/2/agrocymac.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 2441,
+            "indexed": true,
+            "max_children": 5,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 13,
+            "nb_inconsistent": 0,
+            "nb_individuals": 148,
+            "nb_properties": 9,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BIRTHDAYH",
+        "name": "NeuroMorpho.Org cell types birthdate",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2017-03-31T01:00:16Z",
+            "date_released": "2017-03-31T01:00:16Z",
+            "description": "A hierarchy of neuron types based on its birthdate. ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/BIRTHDAYH/4/birthdayh.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 652,
+            "indexed": true,
+            "max_children": 13,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 15,
+            "nb_inconsistent": 0,
+            "nb_individuals": 13,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "2.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BLM",
+        "name": "BioLink Model",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2021-12-23T02:00:47Z",
+            "date_released": "2021-12-23T02:00:45Z",
+            "description": "A high level datamodel of biological entities (genes, diseases, phenotypes, pathways, individuals, substances, etc) and their associations",
+            "documentation": "https://github.com/biolink/biolink-model",
+            "domain": null,
+            "download_url": "media/ontologies/BLM/79/blm.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://biolink.github.io/biolink-model/",
+            "id": 4794,
+            "indexed": true,
+            "max_children": 46,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 292,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 443,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 79,
+            "taxon": null,
+            "version": "1.0.0"
+        },
+        "topics": null
+    },
+    {
         "acronym": "CCF",
         "name": "Human Reference Atlas Common Coordinate Framework Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
@@ -7852,14 +7752,54 @@
             "submission_id": 14,
             "taxon": null,
             "version": "2.2.1"
         },
         "topics": null
     },
     {
+        "acronym": "BPT",
+        "name": "Biological Pathway Taxonomy",
+        "species": [
+            "human",
+            "mice",
+            "rat"
+        ],
+        "status": "Classified",
+        "submission": {
+            "avg_children": 19,
+            "classifiable": true,
+            "date_created": "2022-03-29T16:57:06.962723Z",
+            "date_released": "2022-03-29T16:57:06.962743Z",
+            "description": "Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage.\r\n\r\n\uf0e0\uf0e0\uf0e0 There are 3 types of entities in BPT \u2013 pathway, group, and folder of pathways and groups. \r\n\uf0e0\uf0e0\uf0e0 Group is the list of genes and proteins associated with biological function and diseases. \r\n\uf0e0\uf0e0\uf0e0 Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control.\r\n\r\n\uf0e0\uf0e0\uf0e0 BPT pathway entity includes several properties such as \r\n link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. \uf0e0\uf0e0\uf0e0 The link can be found among synonymes. List of members for each model can be provided by request. Property \u201cnetwork\u201d describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (\u201csignaling\u201d).\r\n\r\n\uf0e0\uf0e0\uf0e0 The index folder contains all pathways and groups sorted alphabetically.",
+            "documentation": "https://mammal-profservices.pathwaystudio.com/",
+            "domain": null,
+            "download_url": "media/ontologies/BPT/2/bpt.obo",
+            "has_ontology_language": "OBO",
+            "home_page": null,
+            "id": 4992,
+            "indexed": true,
+            "max_children": 438,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 3875,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": "Nesterova, A.P., Klimov, E.A., Zharkova, M., Sozin, S., Sobolev, V., Ivanikova, N.V., Shkrob, M., Yuryev, A., 2020. Introduction, in: Disease Pathways. Elsevier, pp. 3\u201332. https://doi.org/10.1016/b978-0-12-817086-1.00001-4",
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "BPT.properties.2022"
+        },
+        "topics": [
+            "Cell signaling pathways"
+        ]
+    },
+    {
         "acronym": "CCFO",
         "name": "Cardiac-centered Frailty Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -7920,14 +7860,150 @@
             "submission_id": 4,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
+        "acronym": "BIM",
+        "name": "Biomedical Image Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 6,
+            "classifiable": true,
+            "date_created": "2015-10-08T22:02:57Z",
+            "date_released": "2014-09-26T00:00:00Z",
+            "description": "Biomedical images published within the scientific literature play a central role in reporting and facilitating life science discoveries. Existing ontologies and vocabularies describing biomedical images, particularly sequence images, do not provide sufficient semantic representation for image annotations generated automatically and/or semi-automatically. We present an open ontology for the annotation of biomedical images (BIM) scripted in OWL/RDF. The BIM ontology provides semantic vocabularies to describe the manually curated image annotations as well as annotations generated by online bioinformatics services using content extracted from images by the Semantic Enrichment of Biomedical Images (SEBI) system. The BIM ontology is represented in three parts; (i) image vocabularies - which holds vocabularies for the annotation of an image and/or region of interests (ROI) inside an image, as well as vocabularies to represent the pre and post processing states of an image, (ii) text entities - covers annotations from the text that are associated with an image (e.g. image captions) and provides semantic representation for NLP algorithm outputs, (iii) a provenance model  - that contributes towards the maintenance of annotation versioning. To illustrate the BIM ontology\u2019s utility, we provide three annotation cases generated by BIM in conjunction with the SEBI image annotation engine.",
+            "documentation": "http://cbakerlab.unbsj.ca/unbvps/BIM",
+            "domain": null,
+            "download_url": "media/ontologies/BIM/6/bim.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://sites.google.com/site/bimontology/",
+            "id": 559,
+            "indexed": true,
+            "max_children": 50,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 125,
+            "nb_inconsistent": 0,
+            "nb_individuals": 34,
+            "nb_properties": 73,
+            "products": null,
+            "publication": "https://code.google.com/p/bio-ont/",
+            "publications": null,
+            "submission_id": 6,
+            "taxon": null,
+            "version": "1.3"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "AIPBC",
+        "name": "Aromatase_Inhibitor_Pathway_Breast_Cell_Pharmacodynamics",
+        "species": [],
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2019-10-01T07:27:18.316017Z",
+            "date_released": "2019-10-01T07:27:18.316047Z",
+            "description": "",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/AIPBC/1/aipbc.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 2974,
+            "indexed": true,
+            "max_children": 15,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 69,
+            "nb_inconsistent": 0,
+            "nb_individuals": 72,
+            "nb_properties": 55,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": []
+    },
+    {
+        "acronym": "AI-RHEUM",
+        "name": "Artificial Intelligence Rheumatology Consultant System Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": null,
+            "classifiable": true,
+            "date_created": "2017-02-06T21:25:57Z",
+            "date_released": "2017-02-06T21:25:57Z",
+            "description": "AI/RHEUM is used for the diagnosis of rheumatologic diseases. AI/RHEUM contains findings, such as clinical signs, symptoms, laboratory test results, radiologic observations, tissue biopsy results, and intermediate diagnosis hypotheses. Findings and hypotheses, which include definitions, are used to reach diagnostic conclusions with definite, probable, or possible certainty. AI/RHEUM is used by clinicians and informatics researchers.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/AI-RHEUM/9/ai-rheum.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": null,
+            "id": 279,
+            "indexed": true,
+            "max_children": 81,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 808,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 9,
+            "taxon": null,
+            "version": "2016AB"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "AMINO-ACID",
+        "name": "Amino Acid Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2010-07-02T17:57:00Z",
+            "date_released": "2007-01-10T08:00:00Z",
+            "description": "An ontology of amino acids and their properties. Inferred version.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/AMINO-ACID/2/amino-acid.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.co-ode.org/ontologies/amino-acid/2009/02/16/",
+            "id": 258,
+            "indexed": true,
+            "max_children": 9,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 46,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 6,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "classified"
+        },
+        "topics": null
+    },
+    {
         "acronym": "CHIRO",
         "name": "CHEBI Integrated Role Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -7999,90 +8075,14 @@
             "submission_id": 6,
             "taxon": null,
             "version": "m4m.25-2"
         },
         "topics": null
     },
     {
-        "acronym": "ISO19110",
-        "name": "ISO 19110 Methodology for Feature Cataloguing",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": false,
-            "date_created": "2017-07-22T14:32:20Z",
-            "date_released": "2017-07-22T00:00:00Z",
-            "description": "ISO 19110:2005 Methodology for feature cataloguing",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ISO19110/1/iso19110.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 69,
-            "indexed": false,
-            "max_children": 22,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 36,
-            "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 57,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "2005-02-15"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ONTODRC",
-        "name": "ONTODRC",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "GFFO",
-        "name": "Genetic Feature Format Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2022-03-28T15:58:06Z",
-            "date_released": "2022-03-28T00:00:00Z",
-            "description": "An ontology to represent genome feature annotations in gff3 format.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/GFFO/1/gffo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/mpievolbio-scicomp/GenomeFeatureFormatOntology",
-            "id": 4987,
-            "indexed": true,
-            "max_children": 1,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 1,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 33,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "0.0.1"
-        },
-        "topics": null
-    },
-    {
         "acronym": "TEST1234",
         "name": "test123",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -8163,44 +8163,49 @@
         "name": "Sewa Mobil Jogja",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "CDPEO",
-        "name": "Chronic Disease Patient Education Ontology",
+        "acronym": "BCGO",
+        "name": "Beta Cell Genomics Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2019-12-18T12:12:32Z",
-            "date_released": "2019-12-18T00:00:00Z",
-            "description": "This ontology is used to recommend personalized education materials to chronic disease patients. The constructed ontology mainly consists of two levels. Level 1 includes 5 terms: demographic, disease, vital sign, lifestyle and medication, which describe the characteristics contained in the patient data, as well as the topics of education materials. Leve 2 contains the detailed elements for each Level 1 class. The patient profile class is used to generate the instance of patient data, using object properties to connect to the specific characteristic class. The patient data will be mapped to a vector space generated from the Level 2 classes. SWRL rules are added to implement the semantic logic of mapping from patient original data to the ontology vector space. The pulblication of the ontology is in preperation.",
+            "date_created": "2017-12-12T21:15:44.000724Z",
+            "date_released": "2017-12-12T21:15:44.000702Z",
+            "description": "An application ontology built for beta cell genomics studies.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CDPEO/1/cdpeo.owl",
+            "download_url": "media/ontologies/BCGO/3/bcgo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3202,
+            "home_page": "https://github.com/obi-bcgo/bcgo",
+            "id": 968,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 40,
+            "max_children": 129,
+            "max_depth": 15,
+            "md5sum": "98f009c856256cc020120c8bf07b3ff8",
+            "nb_classes": 2270,
             "nb_inconsistent": 0,
-            "nb_individuals": 32,
-            "nb_properties": 31,
-            "products": null,
+            "nb_individuals": 46,
+            "nb_properties": 101,
+            "products": [
+                {
+                    "id": "bcgo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bcgo.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.0.0"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "CEDARVS",
         "name": "CEDAR Value Sets",
         "species": null,
@@ -8308,14 +8313,38 @@
             "submission_id": 2,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
+        "acronym": "SEWABUSJOGJA47",
+        "name": "Sewa Bus Jogja 3",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "SEWAHIACEJOGJA32",
+        "name": "Sewa Hiace Jogja Murah",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "SEWABUSJOGJA24",
+        "name": "Sewa Bus Jogja 6",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
         "acronym": "APAONTO",
         "name": "Psychology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 6037,
             "classifiable": true,
@@ -8342,14 +8371,99 @@
             "submission_id": 4,
             "taxon": null,
             "version": "4"
         },
         "topics": null
     },
     {
+        "acronym": "APO",
+        "name": "Ascomycete phenotype ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 8,
+            "classifiable": true,
+            "date_created": "2023-05-01T12:00:10.668573Z",
+            "date_released": "2023-05-01T12:00:10.668557Z",
+            "description": "A structured controlled vocabulary for the phenotypes of Ascomycete fungi",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/APO/46/apo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.yeastgenome.org/",
+            "id": 5784,
+            "indexed": true,
+            "max_children": 314,
+            "max_depth": 8,
+            "md5sum": "54f5b25a28869fff29840e6c0eb881e8",
+            "nb_classes": 619,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": [
+                {
+                    "id": "apo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/apo.owl"
+                },
+                {
+                    "id": "apo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/apo.obo"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/20157474",
+                    "title": "New mutant phenotype data curation system in the Saccharomyces Genome Database"
+                }
+            ],
+            "submission_id": 46,
+            "taxon": {
+                "id": "NCBITaxon:4890",
+                "label": "Ascomycota"
+            },
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BCTEO",
+        "name": "Bone and Cartilage Tissue Engineering Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2014-06-18T08:16:24Z",
+            "date_released": "2013-09-02T07:00:00Z",
+            "description": "Ontology that describes the field of Tissue Engineering for what concerns bone and cartilage tissues.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/BCTEO/4/bcteo.obo",
+            "has_ontology_language": "OBO",
+            "home_page": null,
+            "id": 298,
+            "indexed": true,
+            "max_children": 67,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 253,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 4,
+            "products": null,
+            "publication": "Federica Viti, Silvia Scaglione, Alessandro Orro, Luciano Milanesi \"Guidelines for managing data and processes in bone and cartilage tissue engineering\", BMC Bioinformatics, in press.",
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "releases/2013-08"
+        },
+        "topics": null
+    },
+    {
         "acronym": "CBLAYERH",
         "name": "NeuroMorpho.Org Cerebellum layers",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -8376,14 +8490,60 @@
             "submission_id": 2,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
+        "acronym": "AIDSCLIN",
+        "name": "AIDSClinicOntology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 48,
+            "classifiable": false,
+            "date_created": "2016-09-02T19:17:24Z",
+            "date_released": "2016-06-23T00:00:00Z",
+            "description": "This new ontology will cover the knowledge on AIDS and the relevant clinic activities, which imports the HIV ontology at the site https://bioportal.bioontology.org/ontologies/HIVO004 as its main or basic concept system. \r\nThe new ontology is named AIDSClinic001.owl\r\n",
+            "documentation": "http://blog.51.ca/u-345129/",
+            "domain": null,
+            "download_url": "media/ontologies/AIDSCLIN/4/aidsclin.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://bhf.miforum.net/english/index_english.htm",
+            "id": 678,
+            "indexed": false,
+            "max_children": 94,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 98,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 2,
+            "products": null,
+            "publication": "http://bhf.miforum.net/english/index_english.htm",
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "001"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ABS",
+        "name": "Biosurveillance Diseases",
+        "species": [
+            "Biosurveillance"
+        ],
+        "status": "Unknown",
+        "submission": null,
+        "topics": [
+            "Anthology of Biosurveillance Diseases"
+        ]
+    },
+    {
         "acronym": "CBO",
         "name": "Cell Behavior Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -8410,38 +8570,14 @@
             "submission_id": 22,
             "taxon": null,
             "version": "Version 1.1.0 added classes for OMERO-Tiff files similar to those for VTK files \nVersion 1.0.0 is the initial release of the Cell Behavior Ontology (CBO). \nVersion 1.0.1 added Journal citation for the Cell Behavior Ontology (CBO). \nVersion 1.0.2 corrected a few annotations that had \"snap:process\" that should be \"span:process\".\nVersion 1.0.3 corrected/changed class \"SpacialExclusive\" to \"SpatialExclusive\" and property \"has_Quality\" to \"has_quality\"."
         },
         "topics": null
     },
     {
-        "acronym": "SEWABUSJOGJA47",
-        "name": "Sewa Bus Jogja 3",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "SEWAHIACEJOGJA32",
-        "name": "Sewa Hiace Jogja Murah",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "SEWABUSJOGJA24",
-        "name": "Sewa Bus Jogja 6",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
         "acronym": "CCON",
         "name": "Cerrado concepts and plant community dynamics",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -8468,236 +8604,78 @@
             "submission_id": 8,
             "taxon": null,
             "version": "0.9.3"
         },
         "topics": null
     },
     {
-        "acronym": "CDO",
-        "name": "Chinese Diabetes Mellitus Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2023-07-22T16:53:12Z",
-            "date_released": "2022-03-24T00:00:00Z",
-            "description": "Note: The abbreviation of Chinese Diabetes Mellitus Ontology was changed to CDMO.\r\nCDMO is an OWL 2 ontology designed to cover the diabetes mellitus knowledge represented in Chinese language. The ontology is based on BFO and OGMS. In addition, the top hierarchical structure was adapted from SNOMED-CT, and strive for mapping with other medical ontologies in English, such as SNOMED-CT, HPO, DOID, DRON.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/CDO/6/cdo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5966,
-            "indexed": true,
-            "max_children": 196,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 3752,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 182,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 6,
-            "taxon": null,
-            "version": "3.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "APO",
-        "name": "Ascomycete phenotype ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 8,
-            "classifiable": true,
-            "date_created": "2023-05-01T12:00:10.668573Z",
-            "date_released": "2023-05-01T12:00:10.668557Z",
-            "description": "A structured controlled vocabulary for the phenotypes of Ascomycete fungi",
-            "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/APO/46/apo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.yeastgenome.org/",
-            "id": 5784,
-            "indexed": true,
-            "max_children": 314,
-            "max_depth": 8,
-            "md5sum": "54f5b25a28869fff29840e6c0eb881e8",
-            "nb_classes": 619,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": [
-                {
-                    "id": "apo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/apo.owl"
-                },
-                {
-                    "id": "apo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/apo.obo"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/20157474",
-                    "title": "New mutant phenotype data curation system in the Saccharomyces Genome Database"
-                }
-            ],
-            "submission_id": 46,
-            "taxon": {
-                "id": "NCBITaxon:4890",
-                "label": "Ascomycota"
-            },
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ASPECT",
-        "name": "ASPECT: wind energy vAriableS, ParametErs and ConsTants",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-10-15T01:00:21Z",
-            "date_released": "2021-10-15T01:00:19Z",
-            "description": "This is a taxonomy (i.e. controlled vocabulary) of variables, parameters, and constants used in the wind energy community.",
-            "documentation": "https://github.com/DTUWindEnergy/ASPECT-taxonomy",
-            "domain": null,
-            "download_url": "media/ontologies/ASPECT/3/aspect.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://purl.org/aspect",
-            "id": 4583,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 143,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "0.1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "AUDITORYH",
-        "name": "NeuroMorpho.Org auditory functional regions",
+        "acronym": "ENTITYCANDIDATES",
+        "name": "ISO-15926-2_2003_entityMembershipCandidates",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2016-06-05T18:48:16Z",
-            "date_released": "2016-06-05T00:00:00Z",
-            "description": "A hierarchy of functional regions related to auditory system.",
+            "avg_children": 45,
+            "classifiable": false,
+            "date_created": "2016-01-07T20:04:01Z",
+            "date_released": "2016-01-08T00:00:00Z",
+            "description": "ISO entity candidates for oil and gas",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/AUDITORYH/1/auditoryh.owl",
+            "download_url": "media/ontologies/ENTITYCANDIDATES/1/entitycandidates.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 418,
-            "indexed": true,
-            "max_children": 1,
+            "id": 318,
+            "indexed": false,
+            "max_children": 45,
             "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 1,
+            "nb_classes": 45,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 4,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.6"
+            "version": "Created 2008-07-14. Version 1.0."
         },
         "topics": null
     },
     {
-        "acronym": "BCGO",
-        "name": "Beta Cell Genomics Ontology",
+        "acronym": "CDO",
+        "name": "Chinese Diabetes Mellitus Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-12-12T21:15:44.000724Z",
-            "date_released": "2017-12-12T21:15:44.000702Z",
-            "description": "An application ontology built for beta cell genomics studies.",
+            "date_created": "2023-07-22T16:53:12Z",
+            "date_released": "2022-03-24T00:00:00Z",
+            "description": "Note: The abbreviation of Chinese Diabetes Mellitus Ontology was changed to CDMO.\r\nCDMO is an OWL 2 ontology designed to cover the diabetes mellitus knowledge represented in Chinese language. The ontology is based on BFO and OGMS. In addition, the top hierarchical structure was adapted from SNOMED-CT, and strive for mapping with other medical ontologies in English, such as SNOMED-CT, HPO, DOID, DRON.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BCGO/3/bcgo.owl",
+            "download_url": "media/ontologies/CDO/6/cdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obi-bcgo/bcgo",
-            "id": 968,
-            "indexed": true,
-            "max_children": 129,
-            "max_depth": 15,
-            "md5sum": "98f009c856256cc020120c8bf07b3ff8",
-            "nb_classes": 2270,
-            "nb_inconsistent": 0,
-            "nb_individuals": 46,
-            "nb_properties": 101,
-            "products": [
-                {
-                    "id": "bcgo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bcgo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BCTEO",
-        "name": "Bone and Cartilage Tissue Engineering Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2014-06-18T08:16:24Z",
-            "date_released": "2013-09-02T07:00:00Z",
-            "description": "Ontology that describes the field of Tissue Engineering for what concerns bone and cartilage tissues.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/BCTEO/4/bcteo.obo",
-            "has_ontology_language": "OBO",
             "home_page": null,
-            "id": 298,
+            "id": 5966,
             "indexed": true,
-            "max_children": 67,
-            "max_depth": 3,
+            "max_children": 196,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 253,
+            "nb_classes": 3752,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 4,
+            "nb_properties": 182,
             "products": null,
-            "publication": "Federica Viti, Silvia Scaglione, Alessandro Orro, Luciano Milanesi \"Guidelines for managing data and processes in bone and cartilage tissue engineering\", BMC Bioinformatics, in press.",
+            "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 6,
             "taxon": null,
-            "version": "releases/2013-08"
+            "version": "3.0"
         },
         "topics": null
     },
     {
         "acronym": "ADALAB",
         "name": "AdaLab ontology",
         "species": null,
@@ -8762,112 +8740,78 @@
             "submission_id": 6,
             "taxon": null,
             "version": "0.4"
         },
         "topics": null
     },
     {
-        "acronym": "AIDSCLIN",
-        "name": "AIDSClinicOntology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 48,
-            "classifiable": false,
-            "date_created": "2016-09-02T19:17:24Z",
-            "date_released": "2016-06-23T00:00:00Z",
-            "description": "This new ontology will cover the knowledge on AIDS and the relevant clinic activities, which imports the HIV ontology at the site https://bioportal.bioontology.org/ontologies/HIVO004 as its main or basic concept system. \r\nThe new ontology is named AIDSClinic001.owl\r\n",
-            "documentation": "http://blog.51.ca/u-345129/",
-            "domain": null,
-            "download_url": "media/ontologies/AIDSCLIN/4/aidsclin.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://bhf.miforum.net/english/index_english.htm",
-            "id": 678,
-            "indexed": false,
-            "max_children": 94,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 98,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 2,
-            "products": null,
-            "publication": "http://bhf.miforum.net/english/index_english.htm",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "001"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BHO",
-        "name": "Bleeding History Phenotype Ontology",
+        "acronym": "ASPECT",
+        "name": "ASPECT: wind energy vAriableS, ParametErs and ConsTants",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2010-12-22T19:02:22Z",
-            "date_released": "2010-12-22T08:00:00Z",
-            "description": "An application ontology devoted to the standardized recording of phenotypic data related to hemorrhagic disorders.",
-            "documentation": "https://bh.rockefeller.edu/RUBHPSR/Documentation.html",
+            "date_created": "2021-10-15T01:00:21Z",
+            "date_released": "2021-10-15T01:00:19Z",
+            "description": "This is a taxonomy (i.e. controlled vocabulary) of variables, parameters, and constants used in the wind energy community.",
+            "documentation": "https://github.com/DTUWindEnergy/ASPECT-taxonomy",
             "domain": null,
-            "download_url": "media/ontologies/BHO/3/bho.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.rockefeller.edu/ccts/bioinformatics.php",
-            "id": 263,
+            "download_url": "media/ontologies/ASPECT/3/aspect.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://purl.org/aspect",
+            "id": 4583,
             "indexed": true,
-            "max_children": 19,
-            "max_depth": 7,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 544,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 38,
+            "nb_individuals": 143,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.rockefeller.edu/ccts/bioinformatics.php",
+            "publication": null,
             "publications": null,
             "submission_id": 3,
             "taxon": null,
-            "version": "0.4"
+            "version": "0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "BIFO",
-        "name": "Biofilm Ontology",
+        "acronym": "AUDITORYH",
+        "name": "NeuroMorpho.Org auditory functional regions",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2020-10-24T22:45:43Z",
-            "date_released": "2020-10-24T00:00:00Z",
-            "description": "BIFO is intended to provide a biofilm ontology capable of understanding fundamental terminologies the relations between common biofilm terminologies following the guidelines of the Basic Formal Ontology (BFO).",
-            "documentation": "https://github.com/biofilmDB/ontology ",
+            "date_created": "2016-06-05T18:48:16Z",
+            "date_released": "2016-06-05T00:00:00Z",
+            "description": "A hierarchy of functional regions related to auditory system.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BIFO/1/bifo.owl",
+            "download_url": "media/ontologies/AUDITORYH/1/auditoryh.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/biofilmDB/ontology ",
-            "id": 4016,
+            "home_page": null,
+            "id": 418,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 6,
+            "max_children": 1,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 78,
+            "nb_classes": 1,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 15,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://github.com/biofilmDB/ontology ",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "0.6"
         },
         "topics": null
     },
     {
         "acronym": "CARRE",
         "name": "CARRE Risk Factor ontology",
         "species": null,
@@ -8898,14 +8842,48 @@
             "submission_id": 3,
             "taxon": null,
             "version": "0.2"
         },
         "topics": null
     },
     {
+        "acronym": "BHO",
+        "name": "Bleeding History Phenotype Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2010-12-22T19:02:22Z",
+            "date_released": "2010-12-22T08:00:00Z",
+            "description": "An application ontology devoted to the standardized recording of phenotypic data related to hemorrhagic disorders.",
+            "documentation": "https://bh.rockefeller.edu/RUBHPSR/Documentation.html",
+            "domain": null,
+            "download_url": "media/ontologies/BHO/3/bho.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.rockefeller.edu/ccts/bioinformatics.php",
+            "id": 263,
+            "indexed": true,
+            "max_children": 19,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 544,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 38,
+            "products": null,
+            "publication": "http://www.rockefeller.edu/ccts/bioinformatics.php",
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "0.4"
+        },
+        "topics": null
+    },
+    {
         "acronym": "CASE-BASE-ONTO",
         "name": "Case-base ontology for gastric dystemperament in Persian medicine",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
@@ -8932,24 +8910,46 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "ABS",
-        "name": "Biosurveillance Diseases",
-        "species": [
-            "Biosurveillance"
-        ],
-        "status": "Unknown",
-        "submission": null,
-        "topics": [
-            "Anthology of Biosurveillance Diseases"
-        ]
+        "acronym": "BIFO",
+        "name": "Biofilm Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2020-10-24T22:45:43Z",
+            "date_released": "2020-10-24T00:00:00Z",
+            "description": "BIFO is intended to provide a biofilm ontology capable of understanding fundamental terminologies the relations between common biofilm terminologies following the guidelines of the Basic Formal Ontology (BFO).",
+            "documentation": "https://github.com/biofilmDB/ontology ",
+            "domain": null,
+            "download_url": "media/ontologies/BIFO/1/bifo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/biofilmDB/ontology ",
+            "id": 4016,
+            "indexed": true,
+            "max_children": 11,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 78,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 15,
+            "products": null,
+            "publication": "https://github.com/biofilmDB/ontology ",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
     },
     {
         "acronym": "ADALAB-META",
         "name": "AdaLab-meta ontology",
         "species": null,
         "status": "Classified",
         "submission": {
@@ -9012,56 +9012,56 @@
             "submission_id": 1,
             "taxon": null,
             "version": "English 051319"
         },
         "topics": null
     },
     {
-        "acronym": "ENTITYCANDIDATES",
-        "name": "ISO-15926-2_2003_entityMembershipCandidates",
+        "acronym": "HYDROMAX",
+        "name": "hydroxtreme7",
         "species": null,
-        "status": "Unloadable",
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "CDPEO",
+        "name": "Chronic Disease Patient Education Ontology",
+        "species": null,
+        "status": "Classified",
         "submission": {
-            "avg_children": 45,
-            "classifiable": false,
-            "date_created": "2016-01-07T20:04:01Z",
-            "date_released": "2016-01-08T00:00:00Z",
-            "description": "ISO entity candidates for oil and gas",
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2019-12-18T12:12:32Z",
+            "date_released": "2019-12-18T00:00:00Z",
+            "description": "This ontology is used to recommend personalized education materials to chronic disease patients. The constructed ontology mainly consists of two levels. Level 1 includes 5 terms: demographic, disease, vital sign, lifestyle and medication, which describe the characteristics contained in the patient data, as well as the topics of education materials. Leve 2 contains the detailed elements for each Level 1 class. The patient profile class is used to generate the instance of patient data, using object properties to connect to the specific characteristic class. The patient data will be mapped to a vector space generated from the Level 2 classes. SWRL rules are added to implement the semantic logic of mapping from patient original data to the ontology vector space. The pulblication of the ontology is in preperation.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ENTITYCANDIDATES/1/entitycandidates.owl",
+            "download_url": "media/ontologies/CDPEO/1/cdpeo.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 318,
-            "indexed": false,
-            "max_children": 45,
-            "max_depth": 0,
+            "id": 3202,
+            "indexed": true,
+            "max_children": 6,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 45,
+            "nb_classes": 40,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
+            "nb_individuals": 32,
+            "nb_properties": 31,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "Created 2008-07-14. Version 1.0."
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "HYDROMAX",
-        "name": "hydroxtreme7",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
         "acronym": "SEWAELFJOGJA20",
         "name": "Sewa Elf Jogja",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
@@ -9142,78 +9142,180 @@
         "name": "Sewa Bus Jogja 2",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "PSO_2",
-        "name": "PatientSafetyOntologyRevisted",
+        "acronym": "goslim_pir",
+        "name": "Protein Information Resource slim",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
-            "date_created": "2020-06-27T10:07:25Z",
-            "date_released": "2020-06-26T00:00:00Z",
-            "description": "This is a revised version of the Patient Safety Ontology",
+            "date_created": "2009-09-27T17:52:34Z",
+            "date_released": "2009-09-22T07:00:00Z",
+            "description": "Developed by Darren Natale of PIR",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PSO_2/3/pso_2.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/goslim_pir/1/goslim_pir.obo",
+            "has_ontology_language": "OBO",
             "home_page": null,
-            "id": 3729,
+            "id": 472,
             "indexed": true,
-            "max_children": 47,
+            "max_children": 109,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 537,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 5,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.6 (CVS version)"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "goslim_plant",
+        "name": "Plant slim",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2009-09-27T17:40:25Z",
+            "date_released": "2009-09-22T07:00:00Z",
+            "description": "Developed by The Arabidopsis Information Resource (http://www.arabidopsis.org/).",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/goslim_plant/1/goslim_plant.obo",
+            "has_ontology_language": "OBO",
+            "home_page": null,
+            "id": 512,
+            "indexed": true,
+            "max_children": 23,
             "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 291,
+            "nb_classes": 122,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 7,
+            "nb_properties": 5,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "005"
+            "version": "1.738 (CVS version)"
         },
         "topics": null
     },
     {
-        "acronym": "CCONT",
-        "name": "Cell Culture Ontology",
+        "acronym": "CEI_10",
+        "name": "Codificacion De Enfermedades Pediatricas (En Edici\u00f3n)",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2014-07-24T01:00:22Z",
-            "date_released": "2014-07-24T01:00:21Z",
-            "description": "CCONT is an ontology for the formal representation of cell lines and their correspnding culture conditions.",
+            "date_created": "2018-03-22T23:15:27Z",
+            "date_released": "2018-03-06T00:00:00Z",
+            "description": "En edicion por el Hospital de pediatria Garrahan",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CCONT/3/ccont.owl",
+            "download_url": "media/ontologies/CEI_10/16/cei_10.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 83,
+            "id": 1355,
             "indexed": true,
-            "max_children": 1819,
-            "max_depth": 14,
+            "max_children": 1,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 19991,
+            "nb_classes": 1,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 61,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/23144907",
+            "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 16,
             "taxon": null,
-            "version": "1.1"
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "SPTO",
+        "name": "Solanaceae Phenotype Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2017-10-17T01:14:30Z",
+            "date_released": "2017-10-17T01:14:30Z",
+            "description": "Solanaceae crop phenotypes and traits, developed in collaboration with the research community, especially for breeder traits of agronomic importance.",
+            "documentation": "http://solgenomics.net/search/phenotypes/traits",
+            "domain": null,
+            "download_url": "media/ontologies/SPTO/5/spto.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "solgenomics.net",
+            "id": 259,
+            "indexed": true,
+            "max_children": 75,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 442,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 3,
+            "products": null,
+            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/18539779",
+            "publications": null,
+            "submission_id": 5,
+            "taxon": null,
+            "version": "unknown"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PMR",
+        "name": "Physical Medicine and Rehabilitation",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2017-07-15T14:10:09Z",
+            "date_released": "2012-03-06T08:00:00Z",
+            "description": "Knowledge representation related to computer-based decision support in rehabilitation; concepts and relationships in the rehabilitation domain, integrating clinical practice, the ICD (specifically its 11th revision), the clinical investigator record ontology, the ICF and SNOMED CT.",
+            "documentation": "https://www.researchgate.net/publication/235248247_Interoperable_Knowledge_Representation_In_Clinical_Decision_Support_Systems_for_Rehabilitation",
+            "domain": null,
+            "download_url": "media/ontologies/PMR/4/pmr.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://sites.google.com/site/laiasubirats",
+            "id": 193,
+            "indexed": true,
+            "max_children": 293,
+            "max_depth": 15,
+            "md5sum": null,
+            "nb_classes": 1597,
+            "nb_inconsistent": 0,
+            "nb_individuals": 945,
+            "nb_properties": 52,
+            "products": null,
+            "publication": "https://sites.google.com/site/laiasubirats/publications",
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "1.6"
         },
         "topics": null
     },
     {
         "acronym": "CCTOO",
         "name": "Cancer Care: Treatment Outcome Ontology",
         "species": null,
@@ -9325,14 +9427,22 @@
             "submission_id": 3,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "DRMO",
+        "name": "Dental Restorative Material Ontology",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
         "acronym": "CABRO",
         "name": "Computer Assisted Brain Injury Rehabilitation Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -9359,44 +9469,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "0.5"
         },
         "topics": null
     },
     {
-        "acronym": "SSO",
-        "name": "Syndromic Surveillance Ontology",
+        "acronym": "SP_ONTOLOGY",
+        "name": "SmartPITeS Proyect Ontology for Complex Chronic Patients",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 25,
+            "avg_children": 23,
             "classifiable": true,
-            "date_created": "2009-09-19T03:17:07Z",
-            "date_released": "2009-09-18T07:00:00Z",
-            "description": "The SSO encodes agreement among experts about how Emergency Department (ED) chief complaints are grouped into syndromes of public health importance (consensus definitions). ",
+            "date_created": "2023-03-27T08:56:51Z",
+            "date_released": "2023-03-27T00:00:00Z",
+            "description": "The ontology developed for the SmartPITeS project contains decision rules established in clinical practice on which a software developed for clinical decision support for the pharmacological prescription and de-prescription of complex chronic patients is based. Specifically patients with atrial fibrillation under treatment with anticoagulants. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SSO/1/sso.owl",
+            "download_url": "media/ontologies/SP_ONTOLOGY/1/sp_ontology.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 564,
+            "id": 5741,
             "indexed": true,
-            "max_children": 88,
-            "max_depth": 2,
+            "max_children": 59,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 176,
-            "nb_inconsistent": 0,
-            "nb_individuals": 158,
-            "nb_properties": 32,
+            "nb_classes": 334,
+            "nb_inconsistent": 36,
+            "nb_individuals": 2,
+            "nb_properties": 10,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PSDS",
+        "name": "Plant Structure Development Stage",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2013-08-22T01:06:01Z",
+            "date_released": "2013-08-21T07:00:00Z",
+            "description": "REPLACED BY: Plant Ontology (PO).  A controlled vocabulary of growth and developmental stages in various plants. Note that this has been subsumed into the Plant Ontology (PO).\r\nThis file is created by filtering plant_ontology_assert.obo to contain only terms from the plant structure development stage branch of the PO. For more information, please see: http://palea.cgrb.oregonstate.edu/viewsvn/Poc/tags/live/",
+            "documentation": "http://www.plantontology.org/docs/docs.html",
+            "domain": null,
+            "download_url": "media/ontologies/PSDS/6/psds.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.plantontology.org",
+            "id": 343,
+            "indexed": true,
+            "max_children": 20,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 345,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 10,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 6,
+            "taxon": null,
+            "version": "Release #20"
         },
         "topics": null
     },
     {
         "acronym": "CAO",
         "name": "Clusters of Orthologous Groups (COG) Analysis Ontology",
         "species": null,
@@ -9427,363 +9571,400 @@
             "submission_id": 3,
             "taxon": null,
             "version": "1.4"
         },
         "topics": null
     },
     {
-        "acronym": "PTO",
-        "name": "Plant Trait Ontology",
+        "acronym": "RCTONT",
+        "name": "Randomized Controlled Trials Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-11-10T02:14:53Z",
-            "date_released": "2017-11-10T02:14:53Z",
-            "description": "A controlled vocabulary to describe phenotypic traits in plants. Each trait is a distinguishable feature, characteristic, quality or phenotypic feature of a developing or mature plant, or a plant part.",
+            "date_created": "2011-10-15T01:34:43Z",
+            "date_released": "2010-11-01T07:00:00Z",
+            "description": "We developed and evaluated an ontology\r\nspecifically for Randomized Controlled Trials in order to facilitate the production of systematic reviews and metaanalysis.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PTO/427/pto.owl",
+            "download_url": "media/ontologies/RCTONT/1/rctont.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.planteome.org",
-            "id": 809,
+            "home_page": null,
+            "id": 126,
             "indexed": true,
-            "max_children": 34,
-            "max_depth": 12,
+            "max_children": 13,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 1433,
+            "nb_classes": 187,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_individuals": 45,
+            "nb_properties": 50,
             "products": null,
-            "publication": null,
+            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/20953737",
             "publications": null,
-            "submission_id": 427,
+            "submission_id": 1,
             "taxon": null,
-            "version": "September 2016"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "DRMO",
-        "name": "Dental Restorative Material Ontology",
+        "acronym": "BKO",
+        "name": "Bricks ontology",
         "species": null,
-        "status": "Unknown",
-        "submission": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2020-07-21T12:56:52Z",
+            "date_released": "2020-07-21T00:00:00Z",
+            "description": "The BricKs Ontology (BKO) provides Systems Biology Graphical Notation representations of recurring concepts in molecular networks. These representations, called bricks, are associated in the ontology with terms describing the concepts. Part of these terms are borrowed from the Gene Ontology and the Systems Biology Ontology. The bricks can be used for both the template-based construction of molecular networks maps as well as for their annotation.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/BKO/1/bko.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://sbgnbricks.org",
+            "id": 3798,
+            "indexed": true,
+            "max_children": 35,
+            "max_depth": 9,
+            "md5sum": null,
+            "nb_classes": 669,
+            "nb_inconsistent": 0,
+            "nb_individuals": 35,
+            "nb_properties": 3,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "13:07:2018 10:45"
+        },
         "topics": null
     },
     {
-        "acronym": "GLOMA_ONTOLOGY",
-        "name": "lanxin huang",
+        "acronym": "goslim_goa",
+        "name": "GOA and whole proteome analysis",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-09-22T02:01:30Z",
-            "date_released": "2022-09-21T00:00:00Z",
-            "description": "\u4e00\u4e2a\u4e2d\u6587\u8111\u80f6\u8d28\u7624\u9884\u540e\u672c\u4f53",
-            "documentation": null,
+            "date_created": "2009-09-27T17:34:48Z",
+            "date_released": "2009-09-22T07:00:00Z",
+            "description": "Developed by N. Mulder, M. Pruess",
+            "documentation": "http://www.geneontology.org/GO.slims.shtml",
             "domain": null,
-            "download_url": "media/ontologies/GLOMA_ONTOLOGY/2/gloma_ontology.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/goslim_goa/1/goslim_goa.obo",
+            "has_ontology_language": "OBO",
             "home_page": null,
-            "id": 5381,
+            "id": 586,
             "indexed": true,
-            "max_children": 8,
+            "max_children": 11,
             "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 39,
+            "nb_classes": 81,
             "nb_inconsistent": 0,
-            "nb_individuals": 182,
-            "nb_properties": 19,
+            "nb_individuals": 0,
+            "nb_properties": 5,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.0"
+            "version": "1.761 (CVS version) "
         },
         "topics": null
     },
     {
-        "acronym": "PTRANS",
-        "name": "Pathogen Transmission Ontology",
+        "acronym": "BMHO",
+        "name": "Brazilian Mental Health Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2010-12-15T02:10:23Z",
-            "date_released": "2010-12-14T08:00:00Z",
-            "description": "This vocabulary describes a process that is the means of how a pathogen is transmitted from one host, reservoir, or source to another host. This transmission may occur either directly or indirectly and may involve animate vectors or inanimate vehicles.",
+            "date_created": "2019-07-10T21:49:16Z",
+            "date_released": "2019-07-10T00:00:00Z",
+            "description": "Esta ontologia foi desenvolvida para auxiliar na gest\u00e3o da rede de aten\u00e7\u00e3o em sa\u00fade mental no contexto do Brasil, a fim de proporcionar uma estrutura sem\u00e2ntica que pode ser utilizada para cria\u00e7\u00e3o de ferramentas computacionais inteligentes, permitindo a integra\u00e7\u00e3o e interoperabilidade de bases de dados independentes e possibilitando a gera\u00e7\u00e3o de indicadores relevantes para a sa\u00fade mental, visando melhorar a tomada de decis\u00e3o e o planejamento p\u00fablico, aumentando a efici\u00eancia e efic\u00e1cia dos servi\u00e7os prestados \u00e0 comunidade.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PTRANS/14/ptrans.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://diseaseontology.sourceforge.net/",
-            "id": 519,
+            "download_url": "media/ontologies/BMHO/7/bmho.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://saudeconectada.org/",
+            "id": 2744,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 6,
+            "max_children": 19,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 28,
+            "nb_classes": 280,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 21,
+            "nb_properties": 68,
             "products": null,
-            "publication": null,
+            "publication": "https://www.sciencedirect.com/science/article/pii/S187705091831651X",
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 7,
             "taxon": null,
-            "version": "unknown"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "goslim_goa",
-        "name": "GOA and whole proteome analysis",
+        "acronym": "SPO",
+        "name": "Skin Physiology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2009-09-27T17:34:48Z",
-            "date_released": "2009-09-22T07:00:00Z",
-            "description": "Developed by N. Mulder, M. Pruess",
-            "documentation": "http://www.geneontology.org/GO.slims.shtml",
+            "date_created": "2008-09-19T05:35:09Z",
+            "date_released": "2008-09-19T07:00:00Z",
+            "description": null,
+            "documentation": "www.lboro.ac.uk",
             "domain": null,
-            "download_url": "media/ontologies/goslim_goa/1/goslim_goa.obo",
-            "has_ontology_language": "OBO",
-            "home_page": null,
-            "id": 586,
+            "download_url": "media/ontologies/SPO/1/spo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "www.lboro.ac.uk",
+            "id": 289,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 3,
+            "max_children": 16,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 81,
+            "nb_classes": 339,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 5,
+            "nb_properties": 34,
             "products": null,
-            "publication": null,
+            "publication": "www.lboro.ac.uk",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.761 (CVS version) "
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "goslim_pir",
-        "name": "Protein Information Resource slim",
+        "acronym": "BERKAH",
+        "name": "berkah",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "CCONT",
+        "name": "Cell Culture Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
-            "date_created": "2009-09-27T17:52:34Z",
-            "date_released": "2009-09-22T07:00:00Z",
-            "description": "Developed by Darren Natale of PIR",
+            "date_created": "2014-07-24T01:00:22Z",
+            "date_released": "2014-07-24T01:00:21Z",
+            "description": "CCONT is an ontology for the formal representation of cell lines and their correspnding culture conditions.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/goslim_pir/1/goslim_pir.obo",
-            "has_ontology_language": "OBO",
+            "download_url": "media/ontologies/CCONT/3/ccont.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 472,
+            "id": 83,
             "indexed": true,
-            "max_children": 109,
-            "max_depth": 5,
+            "max_children": 1819,
+            "max_depth": 14,
             "md5sum": null,
-            "nb_classes": 537,
+            "nb_classes": 19991,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 5,
+            "nb_properties": 61,
             "products": null,
-            "publication": null,
+            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/23144907",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.6 (CVS version)"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "goslim_plant",
-        "name": "Plant slim",
+        "acronym": "PTO",
+        "name": "Plant Trait Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2009-09-27T17:40:25Z",
-            "date_released": "2009-09-22T07:00:00Z",
-            "description": "Developed by The Arabidopsis Information Resource (http://www.arabidopsis.org/).",
+            "date_created": "2017-11-10T02:14:53Z",
+            "date_released": "2017-11-10T02:14:53Z",
+            "description": "A controlled vocabulary to describe phenotypic traits in plants. Each trait is a distinguishable feature, characteristic, quality or phenotypic feature of a developing or mature plant, or a plant part.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/goslim_plant/1/goslim_plant.obo",
-            "has_ontology_language": "OBO",
-            "home_page": null,
-            "id": 512,
+            "download_url": "media/ontologies/PTO/427/pto.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.planteome.org",
+            "id": 809,
             "indexed": true,
-            "max_children": 23,
-            "max_depth": 4,
+            "max_children": 34,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 122,
+            "nb_classes": 1433,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 5,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 427,
             "taxon": null,
-            "version": "1.738 (CVS version)"
+            "version": "September 2016"
         },
         "topics": null
     },
     {
-        "acronym": "SSE",
-        "name": "Surgical Secondary Events",
+        "acronym": "SSO",
+        "name": "Syndromic Surveillance Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 12,
+            "avg_children": 25,
             "classifiable": true,
-            "date_created": "2013-07-17T15:40:19Z",
-            "date_released": "2013-07-11T07:00:00Z",
-            "description": "Memorial Sloan-Kettering Cancer Center Surgical Secondary Events Ontology (Adverse Events)",
+            "date_created": "2009-09-19T03:17:07Z",
+            "date_released": "2009-09-18T07:00:00Z",
+            "description": "The SSO encodes agreement among experts about how Emergency Department (ED) chief complaints are grouped into syndromes of public health importance (consensus definitions). ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SSE/2/sse.owl",
+            "download_url": "media/ontologies/SSO/1/sso.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.mskcc.org/sse",
-            "id": 194,
+            "home_page": null,
+            "id": 564,
             "indexed": true,
-            "max_children": 48,
+            "max_children": 88,
             "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 244,
+            "nb_classes": 176,
             "nb_inconsistent": 0,
-            "nb_individuals": 1323,
-            "nb_properties": 19,
+            "nb_individuals": 158,
+            "nb_properties": 32,
             "products": null,
-            "publication": "http://www.mskcc.org/cancer-care/healthcare-professionals/surgical-secondary-events-system/references",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "RCTONT",
-        "name": "Randomized Controlled Trials Ontology",
+        "acronym": "GLOMA_ONTOLOGY",
+        "name": "lanxin huang",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2011-10-15T01:34:43Z",
-            "date_released": "2010-11-01T07:00:00Z",
-            "description": "We developed and evaluated an ontology\r\nspecifically for Randomized Controlled Trials in order to facilitate the production of systematic reviews and metaanalysis.",
+            "date_created": "2022-09-22T02:01:30Z",
+            "date_released": "2022-09-21T00:00:00Z",
+            "description": "\u4e00\u4e2a\u4e2d\u6587\u8111\u80f6\u8d28\u7624\u9884\u540e\u672c\u4f53",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/RCTONT/1/rctont.owl",
+            "download_url": "media/ontologies/GLOMA_ONTOLOGY/2/gloma_ontology.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 126,
+            "id": 5381,
             "indexed": true,
-            "max_children": 13,
-            "max_depth": 5,
+            "max_children": 8,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 187,
+            "nb_classes": 39,
             "nb_inconsistent": 0,
-            "nb_individuals": 45,
-            "nb_properties": 50,
+            "nb_individuals": 182,
+            "nb_properties": 19,
             "products": null,
-            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/20953737",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2.0"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "SSN",
-        "name": "Semantic Sensor Network Ontology",
+        "acronym": "LATESTGOVTJOBS",
+        "name": "Latest Govt Jobs",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "PREO",
+        "name": "Presence Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-07-11T20:04:08Z",
-            "date_released": "2017-04-17T00:00:00Z",
-            "description": "This ontology describes sensors, actuators and observations, and related concepts. It does not describe domain concepts, time, locations, etc. these are intended to be included from other ontologies via OWL imports.",
-            "documentation": "https://w3c.github.io/sdw/ssn-usage/",
+            "date_created": "2019-07-27T22:09:56Z",
+            "date_released": "2019-07-27T00:00:00Z",
+            "description": "The Presence Ontology is a systematic vocabulary of terms with defined relationships that models the encounters taking place every day among providers, patients, and family members or friends in environments such as hospitals and clinics. The Presence Ontology provides a conceptual model for the human experience in medicine. This is a preliminary approach, but further use of methods we have developed here may aid in providing clarity and consensus to topics in healthcare. The ontology is also a model for interdisciplinary collaboration, as it was developed in conjunction with experts from bioinformatics, medicine, anthropology, business, and communication sciences through multiple iterative stages.",
+            "documentation": "http://onto-apps.stanford.edu/presence",
             "domain": null,
-            "download_url": "media/ontologies/SSN/2/ssn.owl",
+            "download_url": "media/ontologies/PREO/1/preo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.w3.org/TR/vocab-ssn/",
-            "id": 103,
+            "home_page": "http://med.stanford.edu/presence/about.html",
+            "id": 2787,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 1,
+            "max_children": 15,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 23,
-            "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 38,
+            "nb_classes": 276,
+            "nb_inconsistent": 3,
+            "nb_individuals": 0,
+            "nb_properties": 24,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "New modular version of the SSN ontology. \n\nThis ontology was originally developed in 2009-2011 by the W3C Semantic Sensor Networks Incubator Group (SSN-XG). For more information on the group's activities see: http://www.w3.org/2005/Incubator/ssn/. The ontology was revised and modularized in 2015-2017 by the W3C/OGC Spatial Data on the Web Working Group, see: https://www.w3.org/2015/spatial/wiki/Semantic_Sensor_Network_Ontology. \n\nIn particular, (a) the scope is extended to include actuation and sampling; (b) the core concepts and properties are factored out into the SOSA ontology. The SSN ontology imports SOSA and adds formal axiomatization consistent with the text definitions in SOSA, and adds classes and properties to accommodate the scope of the original SSN ontology. "
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "PROCO",
-        "name": "Process Chemistry Ontology",
+        "acronym": "PTRANS",
+        "name": "Pathogen Transmission Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2022-06-27T12:51:00.253697Z",
-            "date_released": "2022-06-27T12:51:00.253685Z",
-            "description": "PROCO covers process chemistry, the chemical field concerned with scaling up laboratory syntheses to commercially viable processes.",
+            "date_created": "2010-12-15T02:10:23Z",
+            "date_released": "2010-12-14T08:00:00Z",
+            "description": "This vocabulary describes a process that is the means of how a pathogen is transmitted from one host, reservoir, or source to another host. This transmission may occur either directly or indirectly and may involve animate vectors or inanimate vehicles.",
             "documentation": null,
-            "domain": "chemistry and biochemistry",
-            "download_url": "media/ontologies/PROCO/1/proco.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/proco-ontology/PROCO",
-            "id": 5180,
+            "domain": null,
+            "download_url": "media/ontologies/PTRANS/14/ptrans.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://diseaseontology.sourceforge.net/",
+            "id": 519,
             "indexed": true,
-            "max_children": 33,
-            "max_depth": 15,
-            "md5sum": "97684b32ed8820ea7bf2088dc1fa2998",
-            "nb_classes": 922,
+            "max_children": 4,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 28,
             "nb_inconsistent": 0,
-            "nb_individuals": 17,
-            "nb_properties": 58,
-            "products": [
-                {
-                    "id": "proco.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/proco.owl"
-                }
-            ],
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 14,
             "taxon": null,
-            "version": null
+            "version": "unknown"
         },
         "topics": null
     },
     {
         "acronym": "PROPREO",
         "name": "Proteomics Data and Process Provenance Ontology",
         "species": null,
@@ -9814,14 +9995,48 @@
             "submission_id": 2,
             "taxon": null,
             "version": "1.1"
         },
         "topics": null
     },
     {
+        "acronym": "PSO_2",
+        "name": "PatientSafetyOntologyRevisted",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2020-06-27T10:07:25Z",
+            "date_released": "2020-06-26T00:00:00Z",
+            "description": "This is a revised version of the Patient Safety Ontology",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PSO_2/3/pso_2.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3729,
+            "indexed": true,
+            "max_children": 47,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 291,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 7,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "005"
+        },
+        "topics": null
+    },
+    {
         "acronym": "goslim_pombe",
         "name": "Schizosaccharomyces pombe slim",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
@@ -9848,22 +10063,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.6 (CVS version)"
         },
         "topics": null
     },
     {
-        "acronym": "BERKAH",
-        "name": "berkah",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
         "acronym": "provenance",
         "name": "Provenance",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 1,
             "classifiable": true,
@@ -9924,95 +10131,14 @@
             "submission_id": 124,
             "taxon": null,
             "version": "Recommendation version 2013-04-30"
         },
         "topics": null
     },
     {
-        "acronym": "PSDO",
-        "name": "Performance Summary Display Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 16,
-            "classifiable": true,
-            "date_created": "2023-02-27T15:11:55.660192Z",
-            "date_released": "2023-02-27T15:11:55.660176Z",
-            "description": "Ontology to reproducibly study visualizations of clinical performance",
-            "documentation": null,
-            "domain": "information",
-            "download_url": "media/ontologies/PSDO/2/psdo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/Display-Lab/psdo",
-            "id": 5690,
-            "indexed": true,
-            "max_children": 48,
-            "max_depth": 15,
-            "md5sum": "8463b5e4ae7271004c964aa5020345c4",
-            "nb_classes": 1123,
-            "nb_inconsistent": 21,
-            "nb_individuals": 25,
-            "nb_properties": 83,
-            "products": [
-                {
-                    "id": "psdo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/psdo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PSDS",
-        "name": "Plant Structure Development Stage",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2013-08-22T01:06:01Z",
-            "date_released": "2013-08-21T07:00:00Z",
-            "description": "REPLACED BY: Plant Ontology (PO).  A controlled vocabulary of growth and developmental stages in various plants. Note that this has been subsumed into the Plant Ontology (PO).\r\nThis file is created by filtering plant_ontology_assert.obo to contain only terms from the plant structure development stage branch of the PO. For more information, please see: http://palea.cgrb.oregonstate.edu/viewsvn/Poc/tags/live/",
-            "documentation": "http://www.plantontology.org/docs/docs.html",
-            "domain": null,
-            "download_url": "media/ontologies/PSDS/6/psds.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.plantontology.org",
-            "id": 343,
-            "indexed": true,
-            "max_children": 20,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 345,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 10,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 6,
-            "taxon": null,
-            "version": "Release #20"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "LATESTGOVTJOBS",
-        "name": "Latest Govt Jobs",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
         "acronym": "pseudo",
         "name": "Pseudogene",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -10107,78 +10233,49 @@
             "submission_id": 45,
             "taxon": null,
             "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "BKO",
-        "name": "Bricks ontology",
+        "acronym": "PSDO",
+        "name": "Performance Summary Display Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 16,
             "classifiable": true,
-            "date_created": "2020-07-21T12:56:52Z",
-            "date_released": "2020-07-21T00:00:00Z",
-            "description": "The BricKs Ontology (BKO) provides Systems Biology Graphical Notation representations of recurring concepts in molecular networks. These representations, called bricks, are associated in the ontology with terms describing the concepts. Part of these terms are borrowed from the Gene Ontology and the Systems Biology Ontology. The bricks can be used for both the template-based construction of molecular networks maps as well as for their annotation.",
+            "date_created": "2023-02-27T15:11:55.660192Z",
+            "date_released": "2023-02-27T15:11:55.660176Z",
+            "description": "Ontology to reproducibly study visualizations of clinical performance",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/BKO/1/bko.owl",
+            "domain": "information",
+            "download_url": "media/ontologies/PSDO/2/psdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://sbgnbricks.org",
-            "id": 3798,
+            "home_page": "https://github.com/Display-Lab/psdo",
+            "id": 5690,
             "indexed": true,
-            "max_children": 35,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 669,
-            "nb_inconsistent": 0,
-            "nb_individuals": 35,
-            "nb_properties": 3,
-            "products": null,
+            "max_children": 48,
+            "max_depth": 15,
+            "md5sum": "8463b5e4ae7271004c964aa5020345c4",
+            "nb_classes": 1123,
+            "nb_inconsistent": 21,
+            "nb_individuals": 25,
+            "nb_properties": 83,
+            "products": [
+                {
+                    "id": "psdo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/psdo.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "13:07:2018 10:45"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BMHO",
-        "name": "Brazilian Mental Health Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2019-07-10T21:49:16Z",
-            "date_released": "2019-07-10T00:00:00Z",
-            "description": "Esta ontologia foi desenvolvida para auxiliar na gest\u00e3o da rede de aten\u00e7\u00e3o em sa\u00fade mental no contexto do Brasil, a fim de proporcionar uma estrutura sem\u00e2ntica que pode ser utilizada para cria\u00e7\u00e3o de ferramentas computacionais inteligentes, permitindo a integra\u00e7\u00e3o e interoperabilidade de bases de dados independentes e possibilitando a gera\u00e7\u00e3o de indicadores relevantes para a sa\u00fade mental, visando melhorar a tomada de decis\u00e3o e o planejamento p\u00fablico, aumentando a efici\u00eancia e efic\u00e1cia dos servi\u00e7os prestados \u00e0 comunidade.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/BMHO/7/bmho.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://saudeconectada.org/",
-            "id": 2744,
-            "indexed": true,
-            "max_children": 19,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 280,
-            "nb_inconsistent": 0,
-            "nb_individuals": 21,
-            "nb_properties": 68,
-            "products": null,
-            "publication": "https://www.sciencedirect.com/science/article/pii/S187705091831651X",
-            "publications": null,
-            "submission_id": 7,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "CDAO",
         "name": "Comparative Data Analysis Ontology",
         "species": null,
@@ -10219,14 +10316,121 @@
             "submission_id": 10,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "SSE",
+        "name": "Surgical Secondary Events",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 12,
+            "classifiable": true,
+            "date_created": "2013-07-17T15:40:19Z",
+            "date_released": "2013-07-11T07:00:00Z",
+            "description": "Memorial Sloan-Kettering Cancer Center Surgical Secondary Events Ontology (Adverse Events)",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/SSE/2/sse.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.mskcc.org/sse",
+            "id": 194,
+            "indexed": true,
+            "max_children": 48,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 244,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1323,
+            "nb_properties": 19,
+            "products": null,
+            "publication": "http://www.mskcc.org/cancer-care/healthcare-professionals/surgical-secondary-events-system/references",
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "SSN",
+        "name": "Semantic Sensor Network Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2017-07-11T20:04:08Z",
+            "date_released": "2017-04-17T00:00:00Z",
+            "description": "This ontology describes sensors, actuators and observations, and related concepts. It does not describe domain concepts, time, locations, etc. these are intended to be included from other ontologies via OWL imports.",
+            "documentation": "https://w3c.github.io/sdw/ssn-usage/",
+            "domain": null,
+            "download_url": "media/ontologies/SSN/2/ssn.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.w3.org/TR/vocab-ssn/",
+            "id": 103,
+            "indexed": true,
+            "max_children": 16,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 23,
+            "nb_inconsistent": 0,
+            "nb_individuals": 2,
+            "nb_properties": 38,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "New modular version of the SSN ontology. \n\nThis ontology was originally developed in 2009-2011 by the W3C Semantic Sensor Networks Incubator Group (SSN-XG). For more information on the group's activities see: http://www.w3.org/2005/Incubator/ssn/. The ontology was revised and modularized in 2015-2017 by the W3C/OGC Spatial Data on the Web Working Group, see: https://www.w3.org/2015/spatial/wiki/Semantic_Sensor_Network_Ontology. \n\nIn particular, (a) the scope is extended to include actuation and sampling; (b) the core concepts and properties are factored out into the SOSA ontology. The SSN ontology imports SOSA and adds formal axiomatization consistent with the text definitions in SOSA, and adds classes and properties to accommodate the scope of the original SSN ontology. "
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PROCO",
+        "name": "Process Chemistry Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2022-06-27T12:51:00.253697Z",
+            "date_released": "2022-06-27T12:51:00.253685Z",
+            "description": "PROCO covers process chemistry, the chemical field concerned with scaling up laboratory syntheses to commercially viable processes.",
+            "documentation": null,
+            "domain": "chemistry and biochemistry",
+            "download_url": "media/ontologies/PROCO/1/proco.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/proco-ontology/PROCO",
+            "id": 5180,
+            "indexed": true,
+            "max_children": 33,
+            "max_depth": 15,
+            "md5sum": "97684b32ed8820ea7bf2088dc1fa2998",
+            "nb_classes": 922,
+            "nb_inconsistent": 0,
+            "nb_individuals": 17,
+            "nb_properties": 58,
+            "products": [
+                {
+                    "id": "proco.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/proco.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "CDNO",
         "name": "Compositional Dietary Nutrition Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -10303,547 +10507,547 @@
             "submission_id": 7,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CEI_10",
-        "name": "Codificacion De Enfermedades Pediatricas (En Edici\u00f3n)",
+        "acronym": "SEWAELFJOGJA53",
+        "name": "Sewa Elf Jogja 6",
         "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "DOIDSH",
+        "name": "Diseases Ontology - Shrinked version",
+        "species": [],
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 8,
             "classifiable": true,
-            "date_created": "2018-03-22T23:15:27Z",
-            "date_released": "2018-03-06T00:00:00Z",
-            "description": "En edicion por el Hospital de pediatria Garrahan",
+            "date_created": "2018-09-25T08:28:52.527521Z",
+            "date_released": "2018-09-25T08:28:52.527555Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CEI_10/16/cei_10.owl",
+            "download_url": "media/ontologies/DOIDSH/3/doidsh.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 1355,
+            "id": 1857,
             "indexed": true,
-            "max_children": 1,
-            "max_depth": 1,
+            "max_children": 109,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 1,
+            "nb_classes": 1326,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 6,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 16,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1"
+            "version": "0.3"
         },
-        "topics": null
+        "topics": []
     },
     {
-        "acronym": "SPO",
-        "name": "Skin Physiology Ontology",
+        "acronym": "DPO",
+        "name": "Drosophila Phenotype Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2008-09-19T05:35:09Z",
-            "date_released": "2008-09-19T07:00:00Z",
-            "description": null,
-            "documentation": "www.lboro.ac.uk",
-            "domain": null,
-            "download_url": "media/ontologies/SPO/1/spo.owl",
+            "date_created": "2023-07-10T12:17:34.942536Z",
+            "date_released": "2023-07-10T12:17:34.942504Z",
+            "description": "An ontology of commonly encountered and/or high level Drosophila phenotypes.",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/DPO/27/dpo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.lboro.ac.uk",
-            "id": 289,
+            "home_page": "http://purl.obolibrary.org/obo/fbcv",
+            "id": 5929,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 339,
+            "max_children": 95,
+            "max_depth": 16,
+            "md5sum": "3ed2776509d410ccc01fd9457ce0839c",
+            "nb_classes": 1423,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 34,
-            "products": null,
-            "publication": "www.lboro.ac.uk",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "2"
+            "nb_individuals": 41,
+            "nb_properties": 160,
+            "products": [
+                {
+                    "id": "dpo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/dpo.owl"
+                },
+                {
+                    "id": "dpo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/dpo.obo"
+                },
+                {
+                    "id": "dpo.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/dpo.json"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24138933",
+                    "title": "The Drosophila phenotype ontology."
+                }
+            ],
+            "submission_id": 27,
+            "taxon": {
+                "id": "NCBITaxon:7227",
+                "label": "Drosophila"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SP_ONTOLOGY",
-        "name": "SmartPITeS Proyect Ontology for Complex Chronic Patients",
+        "acronym": "bioscience",
+        "name": "123",
+        "species": [
+            "1234"
+        ],
+        "status": "Unknown",
+        "submission": null,
+        "topics": [
+            "science"
+        ]
+    },
+    {
+        "acronym": "ICF-NoCodeLabel",
+        "name": "ICF with label-without-code",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 23,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2023-03-27T08:56:51Z",
-            "date_released": "2023-03-27T00:00:00Z",
-            "description": "The ontology developed for the SmartPITeS project contains decision rules established in clinical practice on which a software developed for clinical decision support for the pharmacological prescription and de-prescription of complex chronic patients is based. Specifically patients with atrial fibrillation under treatment with anticoagulants. ",
+            "date_created": "2012-04-24T02:41:18Z",
+            "date_released": "2012-04-23T07:00:00Z",
+            "description": "This view contains ICF 1.01 ontology with rdfs:label values that do not contain the codes",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SP_ONTOLOGY/1/sp_ontology.owl",
+            "download_url": "media/ontologies/ICF-NoCodeLabel/1/icf-nocodelabel.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5741,
+            "id": 500,
             "indexed": true,
-            "max_children": 59,
-            "max_depth": 5,
+            "max_children": 20,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 334,
-            "nb_inconsistent": 36,
-            "nb_individuals": 2,
-            "nb_properties": 10,
+            "nb_classes": 1596,
+            "nb_inconsistent": 0,
+            "nb_individuals": 4807,
+            "nb_properties": 66,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "SPTO",
-        "name": "Solanaceae Phenotype Ontology",
+        "acronym": "WB-PHENOTYPE",
+        "name": "C. elegans Phenotype Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-10-17T01:14:30Z",
-            "date_released": "2017-10-17T01:14:30Z",
-            "description": "Solanaceae crop phenotypes and traits, developed in collaboration with the research community, especially for breeder traits of agronomic importance.",
-            "documentation": "http://solgenomics.net/search/phenotypes/traits",
+            "date_created": "2017-11-23T03:44:14Z",
+            "date_released": "2017-11-23T03:44:14Z",
+            "description": "A structured controlled vocabulary of <i>Caenorhabditis elegans</i> phenotypes",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SPTO/5/spto.obo",
+            "download_url": "media/ontologies/WB-PHENOTYPE/267/wb-phenotype.obo",
             "has_ontology_language": "OBO",
-            "home_page": "solgenomics.net",
-            "id": 259,
+            "home_page": "http://www.wormbase.org/",
+            "id": 852,
             "indexed": true,
-            "max_children": 75,
-            "max_depth": 5,
+            "max_children": 29,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 442,
+            "nb_classes": 2470,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 3,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/18539779",
+            "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 267,
             "taxon": null,
-            "version": "unknown"
+            "version": "releases/2017-11-22"
         },
         "topics": null
     },
     {
-        "acronym": "PMR",
-        "name": "Physical Medicine and Rehabilitation",
+        "acronym": "DILIo",
+        "name": "Drug-Induce Liver Injury (combined)",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-07-15T14:10:09Z",
-            "date_released": "2012-03-06T08:00:00Z",
-            "description": "Knowledge representation related to computer-based decision support in rehabilitation; concepts and relationships in the rehabilitation domain, integrating clinical practice, the ICD (specifically its 11th revision), the clinical investigator record ontology, the ICF and SNOMED CT.",
-            "documentation": "https://www.researchgate.net/publication/235248247_Interoperable_Knowledge_Representation_In_Clinical_Decision_Support_Systems_for_Rehabilitation",
+            "date_created": "2013-05-09T20:17:00Z",
+            "date_released": "2013-05-09T07:00:00Z",
+            "description": "A SNOMED subset of terms related to histological and clinical descriptions of Drug Induced Liver Injury (DILI). See Wang et al., \"A Unifying Ontology to Integrate Histological and Clinical Observations for Drug-Induce Liver Injury\" The American Journal of Pathology, 182:4,1180-1187 (2013). \r\n\r\nThis view contains both of the DILIo major branches; SNOMED:Body Structure and SNOMED:Clinical finding\r\n\r\nSome of the class names have been changed to correspond to the preferred names in SNOMED of 7/31/2011.\r\nIn DILIo publication -> in this file;\r\nDisorder of liver -> Disease of liver;\r\nFinding of general form of body -> General form of body - finding;\r\nFinding of size of liver -> size of liver - finding;\r\nSubacute necrosis of liver -> subacute hepatic necrosis",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PMR/4/pmr.owl",
+            "download_url": "media/ontologies/DILIo/1/dilio.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://sites.google.com/site/laiasubirats",
-            "id": 193,
+            "home_page": null,
+            "id": 373,
             "indexed": true,
-            "max_children": 293,
-            "max_depth": 15,
+            "max_children": 11,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 1597,
+            "nb_classes": 200,
             "nb_inconsistent": 0,
-            "nb_individuals": 945,
-            "nb_properties": 52,
-            "products": null,
-            "publication": "https://sites.google.com/site/laiasubirats/publications",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "1.6"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PREO",
-        "name": "Presence Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2019-07-27T22:09:56Z",
-            "date_released": "2019-07-27T00:00:00Z",
-            "description": "The Presence Ontology is a systematic vocabulary of terms with defined relationships that models the encounters taking place every day among providers, patients, and family members or friends in environments such as hospitals and clinics. The Presence Ontology provides a conceptual model for the human experience in medicine. This is a preliminary approach, but further use of methods we have developed here may aid in providing clarity and consensus to topics in healthcare. The ontology is also a model for interdisciplinary collaboration, as it was developed in conjunction with experts from bioinformatics, medicine, anthropology, business, and communication sciences through multiple iterative stages.",
-            "documentation": "http://onto-apps.stanford.edu/presence",
-            "domain": null,
-            "download_url": "media/ontologies/PREO/1/preo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://med.stanford.edu/presence/about.html",
-            "id": 2787,
-            "indexed": true,
-            "max_children": 15,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 276,
-            "nb_inconsistent": 3,
             "nb_individuals": 0,
-            "nb_properties": 24,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "SEWAELFJOGJA53",
-        "name": "Sewa Elf Jogja 6",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "ICHOM-PROMS-PCB",
-        "name": "ICHOM Set Pregnancy and Childbirth",
+        "acronym": "DILON",
+        "name": "Dietary lifestyle ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-05-10T13:18:01Z",
-            "date_released": "2023-05-08T00:00:00Z",
-            "description": "The ICHOM Set of Patient-Centered Outcome Measures for Pregnancy And Childbirth is the result of hard work by a group of leading physicians, measurement experts and patients. It is a recommendation of the outcomes that matter most to persons experiencing Pregnancy And Childbirth. ",
+            "date_created": "2022-03-11T13:25:18Z",
+            "date_released": "2021-11-02T00:00:00Z",
+            "description": "DILON was created to represent dietary lifestyle data.  Concepts were pulled from Korean dietary assessment scales and English assessment scales. Concepts are labeled in English and Korean translations are also provided.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICHOM-PROMS-PCB/3/ichom-proms-pcb.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/DILON/2/dilon.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5814,
+            "id": 4955,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 13,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 262,
             "nb_inconsistent": 0,
-            "nb_individuals": 106,
-            "nb_properties": 0,
+            "nb_individuals": 513,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "goslim_yeast",
-        "name": "Yeast slim",
+        "acronym": "DRANPTO",
+        "name": "Dementia-Related Agitation Non-Pharmacological Treatment Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 9,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2009-09-27T18:02:38Z",
-            "date_released": "2009-09-22T07:00:00Z",
-            "description": "Developed by Saccharomyces Genome Database",
+            "date_created": "2020-06-06T23:41:06Z",
+            "date_released": "2020-06-06T00:00:00Z",
+            "description": "DRANPTO is developed for representing the domain knowledge specific to non-pharmacological intervention for agitation in dementia, particularly for the long-term care setting.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/goslim_yeast/1/goslim_yeast.obo",
-            "has_ontology_language": "OBO",
+            "download_url": "media/ontologies/DRANPTO/1/dranpto.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 399,
+            "id": 3664,
             "indexed": true,
-            "max_children": 41,
-            "max_depth": 2,
+            "max_children": 30,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 118,
+            "nb_classes": 569,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 5,
+            "nb_properties": 48,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.729 (CVS version)"
+            "version": "Version 1.0"
         },
         "topics": null
     },
     {
-        "acronym": "bioscience",
-        "name": "123",
-        "species": [
-            "1234"
-        ],
+        "acronym": "SEWAELFJOGJA31",
+        "name": "Sewa Elf Jogja 7",
+        "species": null,
         "status": "Unknown",
         "submission": null,
-        "topics": [
-            "science"
-        ]
+        "topics": null
     },
     {
-        "acronym": "WBPHENOTYPE",
-        "name": "C. elegans phenotype",
+        "acronym": "WISATAJOGJA29",
+        "name": "Paket Wisata Jogja 6",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "DIAB",
+        "name": "BioMedBridges Diabetes Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 62,
             "classifiable": true,
-            "date_created": "2023-07-17T15:03:54.422715Z",
-            "date_released": "2023-07-17T15:03:54.422692Z",
-            "description": "A structured controlled vocabulary of <i>Caenorhabditis elegans</i> phenotypes",
+            "date_created": "2015-11-27T13:59:57Z",
+            "date_released": "2015-11-27T00:00:00Z",
+            "description": "Diabetes ontology",
             "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/WBPHENOTYPE/76/wbphenotype.owl",
+            "domain": null,
+            "download_url": "media/ontologies/DIAB/73/diab.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/c-elegans-phenotype-ontology",
-            "id": 5952,
+            "home_page": null,
+            "id": 552,
             "indexed": true,
-            "max_children": 218,
-            "max_depth": 33,
-            "md5sum": "83e55363b92dca78a07a3e2900f58578",
-            "nb_classes": 12728,
+            "max_children": 365,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 375,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 292,
-            "products": [
-                {
-                    "id": "wbphenotype.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/wbphenotype.owl"
-                },
-                {
-                    "id": "wbphenotype.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/wbphenotype.obo"
-                },
-                {
-                    "id": "wbphenotype/wbphenotype-base.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/wbphenotype/wbphenotype-base.owl"
-                }
-            ],
+            "nb_individuals": 0,
+            "nb_properties": 4,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/21261995",
-                    "title": "Worm Phenotype Ontology: integrating phenotype data within and beyond the C. elegans community."
-                }
-            ],
-            "submission_id": 76,
-            "taxon": {
-                "id": "NCBITaxon:6237",
-                "label": "Caenorhabditis"
-            },
-            "version": null
+            "publications": null,
+            "submission_id": 73,
+            "taxon": null,
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "DMTO",
-        "name": "Diabetes Mellitus Treatment Ontology",
+        "acronym": "CHD",
+        "name": "Congenital Heart Defects Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 30,
             "classifiable": true,
-            "date_created": "2017-05-10T22:33:48Z",
-            "date_released": "2017-05-10T00:00:00Z",
-            "description": "DMTO is an OWL 2 ontology for creating customized treatment plans for diabetic patients. The ontology is based on BFO and OGMS. In addition, it extends the DDO ontology by adding the treatment classes and axioms to the existing diagnosis part. ",
+            "date_created": "2013-06-03T09:53:00Z",
+            "date_released": "2013-05-23T07:00:00Z",
+            "description": "An ontology that describes the Congenital Heart Defects data.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DMTO/5/dmto.owl",
+            "download_url": "media/ontologies/CHD/5/chd.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 113,
+            "home_page": "http://homes.esat.kuleuven.be/~bioiuser/chdwiki",
+            "id": 237,
             "indexed": true,
-            "max_children": 91,
-            "max_depth": 19,
+            "max_children": 338,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 10700,
+            "nb_classes": 511,
             "nb_inconsistent": 0,
-            "nb_individuals": 63,
-            "nb_properties": 315,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 5,
             "taxon": null,
-            "version": "2015-11-20"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "DOCCC",
-        "name": "Diagnosis Ontology of Clinical Care Classification",
+        "acronym": "IFAR",
+        "name": "Fanconi Anemia Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2013-10-04T02:42:45Z",
-            "date_released": "2013-10-04T00:00:00Z",
-            "description": "Diagnosis Ontology of Clinical Care Classification (CCC) ",
-            "documentation": null,
+            "date_created": "2014-02-18T21:35:25Z",
+            "date_released": "2013-06-26T07:00:00Z",
+            "description": "An application ontology devoted to the standardized recording of data related to Fanconi Anemia (FA). This ontology was created using an OWL file provided by Dr. Ada Hamish (and Francois Schiettecatte) at the Centers for Mendelian Genetics, with their permission. Their original ontology can be found at: http://phenodb.net/help/features. Modifications were made using HPO, OMIM, NCI, and SNOMED. Novel classes pertaining to FA were added and defined as appropriate. ",
+            "documentation": "http://lab.rockefeller.edu/smogorzewska/ifar/",
             "domain": null,
-            "download_url": "media/ontologies/DOCCC/3/doccc.owl",
+            "download_url": "media/ontologies/IFAR/5/ifar.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 594,
+            "home_page": "http://lab.rockefeller.edu/smogorzewska/ifar/",
+            "id": 720,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 4,
+            "max_children": 80,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 204,
+            "nb_classes": 4530,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "http://lab.rockefeller.edu/smogorzewska/ifarpublications",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 5,
             "taxon": null,
-            "version": "20131004"
+            "version": "3"
         },
         "topics": null
     },
     {
-        "acronym": "DREAMDNPTO",
-        "name": "Dementia-Related Emotional And Mood Disturbance Non-Pharmacological Treatment Ontology",
+        "acronym": "SEWAHIACEJOGJA55",
+        "name": "Sewa Hiace Jogja 6",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "WISATAJOGJA52",
+        "name": "Paket Wisata Jogja 5",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "ICHOM-PROMS-PCB",
+        "name": "ICHOM Set Pregnancy and Childbirth",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-06-13T03:21:40Z",
-            "date_released": "2022-06-12T00:00:00Z",
-            "description": "DREAMDNPTO represents the domain knowledge specific to the non-pharmacological treatment of emotional and mood disturbances (e.g., depression and anxiety in dementia) in people with dementia living in the long-term care setting.",
+            "date_created": "2023-05-10T13:18:01Z",
+            "date_released": "2023-05-08T00:00:00Z",
+            "description": "The ICHOM Set of Patient-Centered Outcome Measures for Pregnancy And Childbirth is the result of hard work by a group of leading physicians, measurement experts and patients. It is a recommendation of the outcomes that matter most to persons experiencing Pregnancy And Childbirth. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DREAMDNPTO/1/dreamdnpto.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/ICHOM-PROMS-PCB/3/ichom-proms-pcb.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 5146,
+            "id": 5814,
             "indexed": true,
-            "max_children": 50,
-            "max_depth": 9,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 1258,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 70,
+            "nb_individuals": 106,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "v1.0"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HIPLAYERH",
-        "name": "NeuroMorpho.Org Hippocampus layers",
+        "acronym": "goslim_yeast",
+        "name": "Yeast slim",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 9,
             "classifiable": true,
-            "date_created": "2016-08-29T19:38:20Z",
-            "date_released": "2016-08-29T00:00:00Z",
-            "description": "A hierarchy of layers in the Hippocampus.",
+            "date_created": "2009-09-27T18:02:38Z",
+            "date_released": "2009-09-22T07:00:00Z",
+            "description": "Developed by Saccharomyces Genome Database",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HIPLAYERH/2/hiplayerh.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/goslim_yeast/1/goslim_yeast.obo",
+            "has_ontology_language": "OBO",
             "home_page": null,
-            "id": 709,
+            "id": 399,
             "indexed": true,
-            "max_children": 9,
+            "max_children": 41,
             "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 15,
+            "nb_classes": 118,
             "nb_inconsistent": 0,
-            "nb_individuals": 16,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 5,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2.0"
+            "version": "1.729 (CVS version)"
         },
         "topics": null
     },
     {
-        "acronym": "SEWAELFJOGJA31",
-        "name": "Sewa Elf Jogja 7",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "WISATAJOGJA29",
-        "name": "Paket Wisata Jogja 6",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "ICO",
-        "name": "Informed Consent Ontology",
+        "acronym": "DOID",
+        "name": "Human Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-04-05T13:29:30.838973Z",
-            "date_released": "2021-04-05T13:29:30.838916Z",
-            "description": "An ontology of clinical informed consents",
+            "date_created": "2023-07-31T12:17:52.110206Z",
+            "date_released": "2023-07-31T12:17:52.110184Z",
+            "description": "An ontology for describing the classification of human diseases organized by etiology.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ICO/16/ico.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/DOID/672/doid.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ICO-ontology/ICO",
-            "id": 4381,
+            "home_page": "http://www.disease-ontology.org",
+            "id": 5968,
             "indexed": true,
-            "max_children": 84,
-            "max_depth": 14,
-            "md5sum": "9999be93cec4952c3945885b4ce7131e",
-            "nb_classes": 963,
+            "max_children": 2491,
+            "max_depth": 34,
+            "md5sum": "3746e5eac45ed0f310961ef2a103ee39",
+            "nb_classes": 18287,
             "nb_inconsistent": 0,
-            "nb_individuals": 26,
-            "nb_properties": 97,
+            "nb_individuals": 0,
+            "nb_properties": 46,
             "products": [
                 {
-                    "id": "ico.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ico.owl"
+                    "id": "doid.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/doid.owl",
+                    "title": "Disease Ontology, OWL format. This file contains DO's is_a asserted hierarchy plus equivalent axioms to other OBO Foundry ontologies."
+                },
+                {
+                    "id": "doid.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/doid.obo",
+                    "title": "Disease Ontology, OBO format. This file omits the equivalent axioms."
                 }
             ],
             "publication": null,
-            "publications": null,
-            "submission_id": 16,
-            "taxon": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/25348409",
+                    "title": "Disease Ontology 2015 update: an expanded and updated database of human diseases for linking biomedical knowledge through disease data"
+                },
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34755882",
+                    "title": "The Human Disease Ontology 2022 update"
+                }
+            ],
+            "submission_id": 672,
+            "taxon": {
+                "id": "NCBITaxon:9606",
+                "label": "Homo sapiens"
+            },
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "VBO",
         "name": "Vertebrate Breed Ontology",
@@ -10940,341 +11144,110 @@
             "submission_id": 6,
             "taxon": null,
             "version": "Version 1.91"
         },
         "topics": null
     },
     {
-        "acronym": "SEWAHIACEJOGJA55",
-        "name": "Sewa Hiace Jogja 6",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "WISATAJOGJA52",
-        "name": "Paket Wisata Jogja 5",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "VDOT",
-        "name": "Viral Disease Ontology Trunk",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-01-12T02:14:15Z",
-            "date_released": "2021-01-12T02:14:11Z",
-            "description": "VDOT is a middle-layer ontology for the health care domain with special emphasis on viral infections and transplantation medicine.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/VDOT/35/vdot.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4168,
-            "indexed": true,
-            "max_children": 14,
-            "max_depth": 16,
-            "md5sum": null,
-            "nb_classes": 519,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 41,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 35,
-            "taxon": null,
-            "version": "Initial Version\nrelease date 28/02/2017"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "WB-PHENOTYPE",
-        "name": "C. elegans Phenotype Vocabulary",
+        "acronym": "WC",
+        "name": "Whole-cell modeling ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2017-11-23T03:44:14Z",
-            "date_released": "2017-11-23T03:44:14Z",
-            "description": "A structured controlled vocabulary of <i>Caenorhabditis elegans</i> phenotypes",
+            "date_created": "2020-04-15T01:34:50Z",
+            "date_released": "2020-04-15T01:34:48Z",
+            "description": "Ontology for whole-cell (WC) modeling\r\n\r\nSee https://wholecell.org for more information about WC modeling",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/WB-PHENOTYPE/267/wb-phenotype.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.wormbase.org/",
-            "id": 852,
-            "indexed": true,
-            "max_children": 29,
-            "max_depth": 11,
-            "md5sum": null,
-            "nb_classes": 2470,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 267,
-            "taxon": null,
-            "version": "releases/2017-11-22"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "CO-WHEAT",
-        "name": "Wheat Trait Ontology ",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2010-11-10T00:32:47Z",
-            "date_released": "2010-08-11T07:00:00Z",
-            "description": "This ontology defines traits of the International Wheat Information System (IWIS) database and wheat descriptor.",
-            "documentation": "http://ontology.generationcp.org/",
-            "domain": null,
-            "download_url": "media/ontologies/CO-WHEAT/1/co-wheat.obo",
+            "download_url": "media/ontologies/WC/13/wc.obo",
             "has_ontology_language": "OBO",
-            "home_page": "http://cropontology.org/",
-            "id": 637,
+            "home_page": "https://github.com/karrlab/wc_onto",
+            "id": 3551,
             "indexed": true,
-            "max_children": 37,
+            "max_children": 35,
             "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 176,
+            "nb_classes": 272,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
-            "publication": "http://pantheon.generationcp.org/",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "08.11.2010"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "CRO",
-        "name": "Contributor Role Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2019-12-16T13:38:17.647429Z",
-            "date_released": "2019-12-16T13:38:17.647395Z",
-            "description": "A classification of the diverse roles performed in the work leading to a published research output in the sciences. Its purpose to provide transparency in contributions to scholarly published work, to enable improved systems of attribution, credit, and accountability.",
-            "documentation": null,
-            "domain": "scholarly contribution roles",
-            "download_url": "media/ontologies/CRO/5/cro.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/data2health/contributor-role-ontology",
-            "id": 3171,
-            "indexed": true,
-            "max_children": 32,
-            "max_depth": 7,
-            "md5sum": "0ee9b321e6f69fcf1428ec7ccda6ad65",
-            "nb_classes": 124,
-            "nb_inconsistent": 0,
-            "nb_individuals": 21,
-            "nb_properties": 5,
-            "products": [
-                {
-                    "id": "cro.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cro.owl",
-                    "title": "CRO"
-                }
-            ],
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 13,
             "taxon": null,
-            "version": null
+            "version": "0.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "CSEO",
-        "name": "Cigarette Smoke Exposure Ontology",
+        "acronym": "WCM",
+        "name": "Whole-cell modeling ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2014-07-10T18:38:44Z",
-            "date_released": "2014-07-10T00:00:00Z",
-            "description": "The CSEO has potential to become a widely used standard within the academic and industrial community. Mainly because of the emerging need of systems toxicology to controlled vocabularies and also the lack of suitable ontologies for this domain, the CSEO prepares the ground for integrative systems-based research in the exposure science.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/CSEO/2/cseo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://publicwiki-01.fraunhofer.de/CSEO-Wiki/index.php/Main_Page",
-            "id": 473,
-            "indexed": true,
-            "max_children": 413,
-            "max_depth": 21,
-            "md5sum": null,
-            "nb_classes": 20085,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 91,
-            "products": null,
-            "publication": "http://www.jbiomedsem.com/content/5/1/31/abstract",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "CSO",
-        "name": "Clinical Study Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2015-09-23T20:49:16Z",
-            "date_released": "2015-09-23T00:00:00Z",
-            "description": "Ontology to describe general clinical studies.",
+            "date_created": "2019-02-05T02:22:31Z",
+            "date_released": "2019-02-05T02:22:30Z",
+            "description": "Ontology for whole-cell (WC) modeling\r\n\r\nSee https://wholecell.org for more information about WC modeling",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CSO/1/cso.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/WCM/4/wcm.obo",
+            "has_ontology_language": "OBO",
             "home_page": null,
-            "id": 683,
+            "id": 2241,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 2,
+            "max_children": 35,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 37,
+            "nb_classes": 112,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "CVDO",
-        "name": "Cardiovascular Disease Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2023-03-13T12:17:24.456221Z",
-            "date_released": "2023-03-13T12:17:24.456204Z",
-            "description": "An ontology to describe entities related to cardiovascular diseases",
-            "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/CVDO/4/cvdo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/OpenLHS/CVDO",
-            "id": 5715,
-            "indexed": true,
-            "max_children": 38,
-            "max_depth": 14,
-            "md5sum": "232e5a661f51a6677e87e96dafab9658",
-            "nb_classes": 518,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 78,
-            "products": [
-                {
-                    "id": "cvdo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cvdo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
             "submission_id": 4,
             "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "DATACITE-VOCAB",
-        "name": "DATACITE V4.4",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2022-05-14T01:03:36Z",
-            "date_released": "2022-05-14T01:03:34Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Datacite Controlled Vocabulary. This controlled vocabulary is produced based on description of Datacite Schema V4.4. The work of creating this controlled vocabulary is part of FAIRware project which if funded by RoRi.\r\n\r\nsheet2rdf is used to build Datacite Controlled Vocabulary, while PURL, is used to persist and resolve identifiers via BioPortal for the vocabulary terms.",
-            "documentation": "https://github.com/metadatacenter/datacite-controlled-vocabulary",
-            "domain": null,
-            "download_url": "media/ontologies/DATACITE-VOCAB/5/datacite-vocab.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/metadatacenter/datacite-controlled-vocabulary",
-            "id": 5081,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 132,
-            "nb_properties": 0,
-            "products": null,
-            "publication": "https://github.com/metadatacenter/datacite-controlled-vocabulary",
-            "publications": null,
-            "submission_id": 5,
-            "taxon": null,
-            "version": "0.1.0"
+            "version": "0.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "DIAB",
-        "name": "BioMedBridges Diabetes Ontology",
+        "acronym": "CELEGANH",
+        "name": "NeuroMorpho.Org C. elegans regions",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 62,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2015-11-27T13:59:57Z",
-            "date_released": "2015-11-27T00:00:00Z",
-            "description": "Diabetes ontology",
+            "date_created": "2016-08-29T19:27:44Z",
+            "date_released": "2016-06-05T00:00:00Z",
+            "description": "A hierarchy of C. elegans cell and anatomy & Organs.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DIAB/73/diab.owl",
+            "download_url": "media/ontologies/CELEGANH/2/celeganh.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 552,
+            "id": 486,
             "indexed": true,
-            "max_children": 365,
-            "max_depth": 2,
+            "max_children": 4,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 375,
+            "nb_classes": 11,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
+            "nb_individuals": 6,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 73,
+            "submission_id": 2,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
         "acronym": "DINTO",
@@ -11341,44 +11314,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": []
     },
     {
-        "acronym": "WC",
-        "name": "Whole-cell modeling ontology",
+        "acronym": "HIVO004",
+        "name": "HIVOntologymain.owl",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2020-04-15T01:34:50Z",
-            "date_released": "2020-04-15T01:34:48Z",
-            "description": "Ontology for whole-cell (WC) modeling\r\n\r\nSee https://wholecell.org for more information about WC modeling",
+            "date_created": "2016-08-31T19:17:56Z",
+            "date_released": "2016-08-31T00:00:00Z",
+            "description": "A Project Of Biomedical Ontology for Systems Biomedicine\r\n\r\nThe Project includes so far a main ontology (HIVOnt0**.owl) which is imported by two sub ontologies AIDSClinic0**.owl and HumanGeneCodon0**.owl.\r\n\r\nThe goals of the Project of HIV Ontology(PHIVO) for Knowledge Integrations include \r\n \r\n1. Providing AIDS clinics and HIV virology and the future biomedicines, including the Systems Biomedicine(SBM), the Precision Biomedicine(PBM) and the Knowledge-Integrated Biomedicine(KIMB), with the knowledge resources expressed in the way of ontology, which is able to be operated on engineering ground. \r\n2. Carrying out the methodology researches for the goals above. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/WC/13/wc.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "https://github.com/karrlab/wc_onto",
-            "id": 3551,
+            "download_url": "media/ontologies/HIVO004/16/hivo004.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://blog.51.ca/u-345129/",
+            "id": 28,
             "indexed": true,
-            "max_children": 35,
-            "max_depth": 6,
+            "max_children": 36,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 272,
+            "nb_classes": 1295,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 94,
+            "nb_properties": 204,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 13,
+            "submission_id": 16,
             "taxon": null,
-            "version": "0.0.1"
+            "version": "016"
         },
         "topics": null
     },
     {
         "acronym": "NCRO",
         "name": "Non-Coding RNA Ontology",
         "species": null,
@@ -11415,146 +11388,157 @@
             "submission_id": 13,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "WCM",
-        "name": "Whole-cell modeling ontology",
+        "acronym": "CRO",
+        "name": "Contributor Role Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2019-02-05T02:22:31Z",
-            "date_released": "2019-02-05T02:22:30Z",
-            "description": "Ontology for whole-cell (WC) modeling\r\n\r\nSee https://wholecell.org for more information about WC modeling",
+            "date_created": "2019-12-16T13:38:17.647429Z",
+            "date_released": "2019-12-16T13:38:17.647395Z",
+            "description": "A classification of the diverse roles performed in the work leading to a published research output in the sciences. Its purpose to provide transparency in contributions to scholarly published work, to enable improved systems of attribution, credit, and accountability.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/WCM/4/wcm.obo",
-            "has_ontology_language": "OBO",
-            "home_page": null,
-            "id": 2241,
+            "domain": "scholarly contribution roles",
+            "download_url": "media/ontologies/CRO/5/cro.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/data2health/contributor-role-ontology",
+            "id": 3171,
             "indexed": true,
-            "max_children": 35,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 112,
+            "max_children": 32,
+            "max_depth": 7,
+            "md5sum": "0ee9b321e6f69fcf1428ec7ccda6ad65",
+            "nb_classes": 124,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
+            "nb_individuals": 21,
+            "nb_properties": 5,
+            "products": [
+                {
+                    "id": "cro.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cro.owl",
+                    "title": "CRO"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 5,
             "taxon": null,
-            "version": "0.0.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CELEGANH",
-        "name": "NeuroMorpho.Org C. elegans regions",
+        "acronym": "DREAMDNPTO",
+        "name": "Dementia-Related Emotional And Mood Disturbance Non-Pharmacological Treatment Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2016-08-29T19:27:44Z",
-            "date_released": "2016-06-05T00:00:00Z",
-            "description": "A hierarchy of C. elegans cell and anatomy & Organs.",
+            "date_created": "2022-06-13T03:21:40Z",
+            "date_released": "2022-06-12T00:00:00Z",
+            "description": "DREAMDNPTO represents the domain knowledge specific to the non-pharmacological treatment of emotional and mood disturbances (e.g., depression and anxiety in dementia) in people with dementia living in the long-term care setting.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CELEGANH/2/celeganh.owl",
+            "download_url": "media/ontologies/DREAMDNPTO/1/dreamdnpto.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 486,
+            "id": 5146,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 4,
+            "max_children": 50,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 11,
+            "nb_classes": 1258,
             "nb_inconsistent": 0,
-            "nb_individuals": 6,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 70,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2.0"
+            "version": "v1.0"
         },
         "topics": null
     },
     {
-        "acronym": "CHD",
-        "name": "Congenital Heart Defects Ontology",
+        "acronym": "HIPLAYERH",
+        "name": "NeuroMorpho.Org Hippocampus layers",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 30,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2013-06-03T09:53:00Z",
-            "date_released": "2013-05-23T07:00:00Z",
-            "description": "An ontology that describes the Congenital Heart Defects data.",
+            "date_created": "2016-08-29T19:38:20Z",
+            "date_released": "2016-08-29T00:00:00Z",
+            "description": "A hierarchy of layers in the Hippocampus.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CHD/5/chd.owl",
+            "download_url": "media/ontologies/HIPLAYERH/2/hiplayerh.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://homes.esat.kuleuven.be/~bioiuser/chdwiki",
-            "id": 237,
+            "home_page": null,
+            "id": 709,
             "indexed": true,
-            "max_children": 338,
+            "max_children": 9,
             "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 511,
+            "nb_classes": 15,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 16,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "HIVO004",
-        "name": "HIVOntologymain.owl",
+        "acronym": "ICO",
+        "name": "Informed Consent Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-08-31T19:17:56Z",
-            "date_released": "2016-08-31T00:00:00Z",
-            "description": "A Project Of Biomedical Ontology for Systems Biomedicine\r\n\r\nThe Project includes so far a main ontology (HIVOnt0**.owl) which is imported by two sub ontologies AIDSClinic0**.owl and HumanGeneCodon0**.owl.\r\n\r\nThe goals of the Project of HIV Ontology(PHIVO) for Knowledge Integrations include \r\n \r\n1. Providing AIDS clinics and HIV virology and the future biomedicines, including the Systems Biomedicine(SBM), the Precision Biomedicine(PBM) and the Knowledge-Integrated Biomedicine(KIMB), with the knowledge resources expressed in the way of ontology, which is able to be operated on engineering ground. \r\n2. Carrying out the methodology researches for the goals above. ",
+            "date_created": "2021-04-05T13:29:30.838973Z",
+            "date_released": "2021-04-05T13:29:30.838916Z",
+            "description": "An ontology of clinical informed consents",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HIVO004/16/hivo004.owl",
+            "download_url": "media/ontologies/ICO/16/ico.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://blog.51.ca/u-345129/",
-            "id": 28,
+            "home_page": "https://github.com/ICO-ontology/ICO",
+            "id": 4381,
             "indexed": true,
-            "max_children": 36,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 1295,
+            "max_children": 84,
+            "max_depth": 14,
+            "md5sum": "9999be93cec4952c3945885b4ce7131e",
+            "nb_classes": 963,
             "nb_inconsistent": 0,
-            "nb_individuals": 94,
-            "nb_properties": 204,
-            "products": null,
+            "nb_individuals": 26,
+            "nb_properties": 97,
+            "products": [
+                {
+                    "id": "ico.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ico.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
             "submission_id": 16,
             "taxon": null,
-            "version": "016"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "DRON",
         "name": "The Drug Ontology",
         "species": null,
@@ -11668,112 +11652,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "HRAVS",
-        "name": "Human Reference Atlas Value Set",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2023-07-14T02:42:30Z",
-            "date_released": "2023-07-13T00:00:00Z",
-            "description": "The value set for the Human Reference Atlas metadata records as part of the Human BioMolecular Atlas Program (HuBMAP) (https://hubmapconsortium.org)",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HRAVS/13/hravs.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5957,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 520,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 13,
-            "taxon": null,
-            "version": "1.4.6"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "DILIo",
-        "name": "Drug-Induce Liver Injury (combined)",
+        "acronym": "VDOT",
+        "name": "Viral Disease Ontology Trunk",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2013-05-09T20:17:00Z",
-            "date_released": "2013-05-09T07:00:00Z",
-            "description": "A SNOMED subset of terms related to histological and clinical descriptions of Drug Induced Liver Injury (DILI). See Wang et al., \"A Unifying Ontology to Integrate Histological and Clinical Observations for Drug-Induce Liver Injury\" The American Journal of Pathology, 182:4,1180-1187 (2013). \r\n\r\nThis view contains both of the DILIo major branches; SNOMED:Body Structure and SNOMED:Clinical finding\r\n\r\nSome of the class names have been changed to correspond to the preferred names in SNOMED of 7/31/2011.\r\nIn DILIo publication -> in this file;\r\nDisorder of liver -> Disease of liver;\r\nFinding of general form of body -> General form of body - finding;\r\nFinding of size of liver -> size of liver - finding;\r\nSubacute necrosis of liver -> subacute hepatic necrosis",
+            "date_created": "2021-01-12T02:14:15Z",
+            "date_released": "2021-01-12T02:14:11Z",
+            "description": "VDOT is a middle-layer ontology for the health care domain with special emphasis on viral infections and transplantation medicine.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DILIo/1/dilio.owl",
+            "download_url": "media/ontologies/VDOT/35/vdot.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 373,
+            "id": 4168,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 9,
+            "max_children": 14,
+            "max_depth": 16,
             "md5sum": null,
-            "nb_classes": 200,
+            "nb_classes": 519,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 41,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 35,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "Initial Version\nrelease date 28/02/2017"
         },
         "topics": null
     },
     {
-        "acronym": "DILON",
-        "name": "Dietary lifestyle ontology",
+        "acronym": "CSEO",
+        "name": "Cigarette Smoke Exposure Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-03-11T13:25:18Z",
-            "date_released": "2021-11-02T00:00:00Z",
-            "description": "DILON was created to represent dietary lifestyle data.  Concepts were pulled from Korean dietary assessment scales and English assessment scales. Concepts are labeled in English and Korean translations are also provided.",
+            "date_created": "2014-07-10T18:38:44Z",
+            "date_released": "2014-07-10T00:00:00Z",
+            "description": "The CSEO has potential to become a widely used standard within the academic and industrial community. Mainly because of the emerging need of systems toxicology to controlled vocabularies and also the lack of suitable ontologies for this domain, the CSEO prepares the ground for integrative systems-based research in the exposure science.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DILON/2/dilon.owl",
+            "download_url": "media/ontologies/CSEO/2/cseo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4955,
+            "home_page": "https://publicwiki-01.fraunhofer.de/CSEO-Wiki/index.php/Main_Page",
+            "id": 473,
             "indexed": true,
-            "max_children": 13,
-            "max_depth": 6,
+            "max_children": 413,
+            "max_depth": 21,
             "md5sum": null,
-            "nb_classes": 262,
+            "nb_classes": 20085,
             "nb_inconsistent": 0,
-            "nb_individuals": 513,
-            "nb_properties": 7,
+            "nb_individuals": 0,
+            "nb_properties": 91,
             "products": null,
-            "publication": null,
+            "publication": "http://www.jbiomedsem.com/content/5/1/31/abstract",
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "2"
+            "version": "1.0"
         },
         "topics": null
     },
     {
         "acronym": "DLORO",
         "name": "Dependency Layered Ontology for Radiation Oncology",
         "species": null,
@@ -11804,198 +11754,227 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DOID",
-        "name": "Human Disease Ontology",
+        "acronym": "CSO",
+        "name": "Clinical Study Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2023-07-24T12:19:55.812631Z",
-            "date_released": "2023-07-24T12:19:55.812610Z",
-            "description": "An ontology for describing the classification of human diseases organized by etiology.",
+            "date_created": "2015-09-23T20:49:16Z",
+            "date_released": "2015-09-23T00:00:00Z",
+            "description": "Ontology to describe general clinical studies.",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/DOID/671/doid.owl",
+            "domain": null,
+            "download_url": "media/ontologies/CSO/1/cso.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.disease-ontology.org",
-            "id": 5959,
+            "home_page": null,
+            "id": 683,
             "indexed": true,
-            "max_children": 2491,
-            "max_depth": 34,
-            "md5sum": "7e6bfec5d7dbf480df1f092ada02971d",
-            "nb_classes": 18287,
+            "max_children": 11,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 37,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 46,
-            "products": [
-                {
-                    "id": "doid.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/doid.owl",
-                    "title": "Disease Ontology, OWL format. This file contains DO's is_a asserted hierarchy plus equivalent axioms to other OBO Foundry ontologies."
-                },
-                {
-                    "id": "doid.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/doid.obo",
-                    "title": "Disease Ontology, OBO format. This file omits the equivalent axioms."
-                }
-            ],
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/25348409",
-                    "title": "Disease Ontology 2015 update: an expanded and updated database of human diseases for linking biomedical knowledge through disease data"
-                },
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34755882",
-                    "title": "The Human Disease Ontology 2022 update"
-                }
-            ],
-            "submission_id": 671,
-            "taxon": {
-                "id": "NCBITaxon:9606",
-                "label": "Homo sapiens"
-            },
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DOIDSH",
-        "name": "Diseases Ontology - Shrinked version",
-        "species": [],
+        "acronym": "VRD",
+        "name": "Vaccineresearchdata",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 8,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2018-09-25T08:28:52.527521Z",
-            "date_released": "2018-09-25T08:28:52.527555Z",
-            "description": "",
+            "date_created": "2023-04-17T18:43:16Z",
+            "date_released": "2023-04-17T00:00:00Z",
+            "description": "This is test",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DOIDSH/3/doidsh.owl",
+            "download_url": "media/ontologies/VRD/1/vrd.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 1857,
+            "id": 5776,
             "indexed": true,
-            "max_children": 109,
-            "max_depth": 10,
+            "max_children": 7,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 1326,
+            "nb_classes": 13,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.3"
+            "version": "2.0"
         },
-        "topics": []
+        "topics": null
     },
     {
-        "acronym": "CIE-10",
-        "name": "CIE-10",
+        "acronym": "CO-WHEAT",
+        "name": "Wheat Trait Ontology ",
         "species": null,
-        "status": "Unknown",
-        "submission": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2010-11-10T00:32:47Z",
+            "date_released": "2010-08-11T07:00:00Z",
+            "description": "This ontology defines traits of the International Wheat Information System (IWIS) database and wheat descriptor.",
+            "documentation": "http://ontology.generationcp.org/",
+            "domain": null,
+            "download_url": "media/ontologies/CO-WHEAT/1/co-wheat.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://cropontology.org/",
+            "id": 637,
+            "indexed": true,
+            "max_children": 37,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 176,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": "http://pantheon.generationcp.org/",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "08.11.2010"
+        },
         "topics": null
     },
     {
-        "acronym": "DPO",
-        "name": "Drosophila Phenotype Ontology",
+        "acronym": "DOCCC",
+        "name": "Diagnosis Ontology of Clinical Care Classification",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2023-07-10T12:17:34.942536Z",
-            "date_released": "2023-07-10T12:17:34.942504Z",
-            "description": "An ontology of commonly encountered and/or high level Drosophila phenotypes.",
+            "date_created": "2013-10-04T02:42:45Z",
+            "date_released": "2013-10-04T00:00:00Z",
+            "description": "Diagnosis Ontology of Clinical Care Classification (CCC) ",
             "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/DPO/27/dpo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/DOCCC/3/doccc.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://purl.obolibrary.org/obo/fbcv",
-            "id": 5929,
+            "home_page": null,
+            "id": 594,
             "indexed": true,
-            "max_children": 95,
-            "max_depth": 16,
-            "md5sum": "3ed2776509d410ccc01fd9457ce0839c",
-            "nb_classes": 1423,
+            "max_children": 17,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 204,
             "nb_inconsistent": 0,
-            "nb_individuals": 41,
-            "nb_properties": 160,
-            "products": [
-                {
-                    "id": "dpo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/dpo.owl"
-                },
-                {
-                    "id": "dpo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/dpo.obo"
-                },
-                {
-                    "id": "dpo.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/dpo.json"
-                }
-            ],
+            "nb_individuals": 0,
+            "nb_properties": 1,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24138933",
-                    "title": "The Drosophila phenotype ontology."
-                }
-            ],
-            "submission_id": 27,
-            "taxon": {
-                "id": "NCBITaxon:7227",
-                "label": "Drosophila"
-            },
-            "version": null
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "20131004"
         },
         "topics": null
     },
     {
-        "acronym": "DRANPTO",
-        "name": "Dementia-Related Agitation Non-Pharmacological Treatment Ontology",
+        "acronym": "DATACITE-VOCAB",
+        "name": "DATACITE V4.4",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2020-06-06T23:41:06Z",
-            "date_released": "2020-06-06T00:00:00Z",
-            "description": "DRANPTO is developed for representing the domain knowledge specific to non-pharmacological intervention for agitation in dementia, particularly for the long-term care setting.",
-            "documentation": null,
+            "date_created": "2022-05-14T01:03:36Z",
+            "date_released": "2022-05-14T01:03:34Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Datacite Controlled Vocabulary. This controlled vocabulary is produced based on description of Datacite Schema V4.4. The work of creating this controlled vocabulary is part of FAIRware project which if funded by RoRi.\r\n\r\nsheet2rdf is used to build Datacite Controlled Vocabulary, while PURL, is used to persist and resolve identifiers via BioPortal for the vocabulary terms.",
+            "documentation": "https://github.com/metadatacenter/datacite-controlled-vocabulary",
             "domain": null,
-            "download_url": "media/ontologies/DRANPTO/1/dranpto.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3664,
+            "download_url": "media/ontologies/DATACITE-VOCAB/5/datacite-vocab.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/metadatacenter/datacite-controlled-vocabulary",
+            "id": 5081,
             "indexed": true,
-            "max_children": 30,
-            "max_depth": 10,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 569,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 48,
+            "nb_individuals": 132,
+            "nb_properties": 0,
             "products": null,
+            "publication": "https://github.com/metadatacenter/datacite-controlled-vocabulary",
+            "publications": null,
+            "submission_id": 5,
+            "taxon": null,
+            "version": "0.1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "CIE-10",
+        "name": "CIE-10",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "CVDO",
+        "name": "Cardiovascular Disease Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-03-13T12:17:24.456221Z",
+            "date_released": "2023-03-13T12:17:24.456204Z",
+            "description": "An ontology to describe entities related to cardiovascular diseases",
+            "documentation": null,
+            "domain": "health",
+            "download_url": "media/ontologies/CVDO/4/cvdo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/OpenLHS/CVDO",
+            "id": 5715,
+            "indexed": true,
+            "max_children": 38,
+            "max_depth": 14,
+            "md5sum": "232e5a661f51a6677e87e96dafab9658",
+            "nb_classes": 518,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 78,
+            "products": [
+                {
+                    "id": "cvdo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cvdo.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "Version 1.0"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "DMDSONT",
         "name": "Diabetes Mellitus Diagnosis and Support Ontology ",
         "species": null,
@@ -12026,14 +12005,69 @@
             "submission_id": 1,
             "taxon": null,
             "version": "ver 1.0"
         },
         "topics": null
     },
     {
+        "acronym": "WBPHENOTYPE",
+        "name": "C. elegans phenotype",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-07-17T15:03:54.422715Z",
+            "date_released": "2023-07-17T15:03:54.422692Z",
+            "description": "A structured controlled vocabulary of <i>Caenorhabditis elegans</i> phenotypes",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/WBPHENOTYPE/76/wbphenotype.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obophenotype/c-elegans-phenotype-ontology",
+            "id": 5952,
+            "indexed": true,
+            "max_children": 218,
+            "max_depth": 33,
+            "md5sum": "83e55363b92dca78a07a3e2900f58578",
+            "nb_classes": 12728,
+            "nb_inconsistent": 0,
+            "nb_individuals": 18,
+            "nb_properties": 292,
+            "products": [
+                {
+                    "id": "wbphenotype.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/wbphenotype.owl"
+                },
+                {
+                    "id": "wbphenotype.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/wbphenotype.obo"
+                },
+                {
+                    "id": "wbphenotype/wbphenotype-base.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/wbphenotype/wbphenotype-base.owl"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/21261995",
+                    "title": "Worm Phenotype Ontology: integrating phenotype data within and beyond the C. elegans community."
+                }
+            ],
+            "submission_id": 76,
+            "taxon": {
+                "id": "NCBITaxon:6237",
+                "label": "Caenorhabditis"
+            },
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "VODANADISEASES",
         "name": "VODANADiseases",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -12060,258 +12094,219 @@
             "submission_id": 3,
             "taxon": null,
             "version": "2.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "VRD",
-        "name": "Vaccineresearchdata",
+        "acronym": "DMTO",
+        "name": "Diabetes Mellitus Treatment Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-04-17T18:43:16Z",
-            "date_released": "2023-04-17T00:00:00Z",
-            "description": "This is test",
+            "date_created": "2017-05-10T22:33:48Z",
+            "date_released": "2017-05-10T00:00:00Z",
+            "description": "DMTO is an OWL 2 ontology for creating customized treatment plans for diabetic patients. The ontology is based on BFO and OGMS. In addition, it extends the DDO ontology by adding the treatment classes and axioms to the existing diagnosis part. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/VRD/1/vrd.owl",
+            "download_url": "media/ontologies/DMTO/5/dmto.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5776,
+            "id": 113,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 2,
+            "max_children": 91,
+            "max_depth": 19,
             "md5sum": null,
-            "nb_classes": 13,
+            "nb_classes": 10700,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 63,
+            "nb_properties": 315,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 5,
             "taxon": null,
-            "version": "2.0"
+            "version": "2015-11-20"
         },
         "topics": null
     },
     {
-        "acronym": "IFAR",
-        "name": "Fanconi Anemia Ontology",
+        "acronym": "HRAVS",
+        "name": "Human Reference Atlas Value Set",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2014-02-18T21:35:25Z",
-            "date_released": "2013-06-26T07:00:00Z",
-            "description": "An application ontology devoted to the standardized recording of data related to Fanconi Anemia (FA). This ontology was created using an OWL file provided by Dr. Ada Hamish (and Francois Schiettecatte) at the Centers for Mendelian Genetics, with their permission. Their original ontology can be found at: http://phenodb.net/help/features. Modifications were made using HPO, OMIM, NCI, and SNOMED. Novel classes pertaining to FA were added and defined as appropriate. ",
-            "documentation": "http://lab.rockefeller.edu/smogorzewska/ifar/",
+            "date_created": "2023-07-14T02:42:30Z",
+            "date_released": "2023-07-13T00:00:00Z",
+            "description": "The value set for the Human Reference Atlas metadata records as part of the Human BioMolecular Atlas Program (HuBMAP) (https://hubmapconsortium.org)",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IFAR/5/ifar.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://lab.rockefeller.edu/smogorzewska/ifar/",
-            "id": 720,
+            "download_url": "media/ontologies/HRAVS/13/hravs.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5957,
             "indexed": true,
-            "max_children": 80,
-            "max_depth": 9,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 4530,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 520,
             "nb_properties": 0,
             "products": null,
-            "publication": "http://lab.rockefeller.edu/smogorzewska/ifarpublications",
+            "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 13,
             "taxon": null,
-            "version": "3"
+            "version": "1.4.6"
         },
         "topics": null
     },
     {
-        "acronym": "ICF-NoCodeLabel",
-        "name": "ICF with label-without-code",
+        "acronym": "CN",
+        "name": "computer network",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2012-04-24T02:41:18Z",
-            "date_released": "2012-04-23T07:00:00Z",
-            "description": "This view contains ICF 1.01 ontology with rdfs:label values that do not contain the codes",
+            "date_created": "2014-09-25T12:09:00Z",
+            "date_released": "2014-09-18T00:00:00Z",
+            "description": "Computer networks ontology ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICF-NoCodeLabel/1/icf-nocodelabel.owl",
+            "download_url": "media/ontologies/CN/1/cn.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 500,
+            "id": 540,
             "indexed": true,
-            "max_children": 20,
+            "max_children": 34,
             "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 1596,
+            "nb_classes": 537,
             "nb_inconsistent": 0,
-            "nb_individuals": 4807,
-            "nb_properties": 66,
+            "nb_individuals": 2,
+            "nb_properties": 38,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "unknown"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CODO",
-        "name": "An Ontology for Collection and Analysis of COviD-19 Data",
+        "acronym": "LCGFT_SKOS",
+        "name": "Library of Congress Genre/Form Terms (SKOS)",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2020-10-08T18:14:48Z",
-            "date_released": "2020-09-25T00:00:00Z",
-            "description": "This COVID-19 Ontology is a data model for publishing COVID-19 data on the Web as a Knowledge Graph. \r\n\r\nThe primary focuses of the model are (1) COVID-19 cases: the data for COVID-19 cases (e.g., active, recovered, deceased, migrated) on daily basis across the geo-location (district, state (province), and country), available resources and requirements; \r\n(2) COVID-19 patient data: nationality, symptom, suspected level of COVID -19, treatment facility, COVID-19 clinical facility, patient's travel history, inter-personal relationships between patients, suspected transmission reason, tracking of patient test results, etc. ",
-            "documentation": "https://isibang.ac.in/ns/codo",
-            "domain": null,
-            "download_url": "media/ontologies/CODO/4/codo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/biswanathdutta/CODO",
-            "id": 3983,
-            "indexed": true,
-            "max_children": 13,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 90,
-            "nb_inconsistent": 0,
-            "nb_individuals": 271,
-            "nb_properties": 73,
-            "products": null,
-            "publication": "https://isibang.ac.in/ns/codo",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "1.3"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "COGAT",
-        "name": "Cognitive Atlas Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": null,
-            "classifiable": true,
-            "date_created": "2016-08-15T22:55:22Z",
-            "date_released": "2012-04-03T07:00:00Z",
-            "description": "The Cognitive Atlas is a collaborative knowledge building project that aims to develop an ontology that characterizes the state of current thought in cognitive science.  It defines a set of mental concepts along with a set of mental tasks, and the measurement relations between those classes.",
+            "date_created": "2018-03-20T22:17:17Z",
+            "date_released": "2018-03-20T00:00:00Z",
+            "description": "The Library of Congress Genre/Form Terms for Library and Archival Materials (LCGFT) is a thesaurus that describes what a work is versus what it is about.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/COGAT/7/cogat.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.cognitiveatlas.org",
-            "id": 79,
+            "download_url": "media/ontologies/LCGFT_SKOS/2/lcgft_skos.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 1353,
             "indexed": true,
-            "max_children": null,
-            "max_depth": 7,
+            "max_children": 3,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 3639,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_individuals": 2222,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.3.1"
+            "version": "using output of skosify"
         },
         "topics": null
     },
     {
-        "acronym": "CIDO",
-        "name": "Coronavirus Infectious Disease Ontology",
+        "acronym": "INO",
+        "name": "Interaction Network Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-01-30T12:17:14.821954Z",
-            "date_released": "2023-01-30T12:17:14.821935Z",
-            "description": "The Coronavirus Infectious Disease Ontology (CIDO) aims to ontologically represent and standardize various aspects of coronavirus infectious diseases, including their etiology, transmission, epidemiology, pathogenesis, diagnosis, prevention, and treatment.",
+            "date_created": "2022-03-28T12:21:05.529510Z",
+            "date_released": "2022-03-28T12:21:05.529499Z",
+            "description": "An ontology of interactions and interaction networks",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/CIDO/33/cido.owl",
+            "domain": null,
+            "download_url": "media/ontologies/INO/32/ino.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/cido-ontology/cido",
-            "id": 5626,
+            "home_page": "https://github.com/INO-ontology/ino",
+            "id": 4978,
             "indexed": true,
-            "max_children": 5749,
-            "max_depth": 38,
-            "md5sum": "ed8d5b61a188602622cf664962dd21e1",
-            "nb_classes": 31620,
+            "max_children": 43,
+            "max_depth": 12,
+            "md5sum": "7d35b715a7e2412851c6cd63dd6082c7",
+            "nb_classes": 328,
             "nb_inconsistent": 0,
-            "nb_individuals": 458,
-            "nb_properties": 384,
+            "nb_individuals": 0,
+            "nb_properties": 59,
             "products": [
                 {
-                    "id": "cido.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cido.owl"
+                    "id": "ino.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ino.owl"
                 }
             ],
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/36271389",
-                    "title": "A comprehensive update on CIDO: the community-based coronavirus infectious disease ontology"
-                }
-            ],
-            "submission_id": 33,
+            "publications": null,
+            "submission_id": 32,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "COHSI2STUDY",
-        "name": "cohsi2study",
+        "acronym": "INSNAME",
+        "name": "VODANAINSNAMES",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-09-12T05:59:40Z",
-            "date_released": "2021-11-26T00:00:00Z",
-            "description": "These ontologies are created under the CoHSI2 Study.",
+            "date_created": "2021-03-30T05:33:23Z",
+            "date_released": "2021-03-26T00:00:00Z",
+            "description": "This is the list of facilities participating in FAIRification of data.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/COHSI2STUDY/27/cohsi2study.skos",
+            "download_url": "media/ontologies/INSNAME/5/insname.skos",
             "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 5352,
+            "id": 4373,
             "indexed": true,
             "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
             "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 83,
+            "nb_individuals": 24,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 27,
+            "submission_id": 5,
             "taxon": null,
-            "version": "0.2.8"
+            "version": "5"
         },
         "topics": null
     },
     {
         "acronym": "a",
         "name": "z",
         "species": [
@@ -12356,168 +12351,117 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": []
     },
     {
-        "acronym": "CL",
-        "name": "Cell Ontology",
+        "acronym": "BIRTHONTO",
+        "name": "BirthOnto: Birth Data Analysis",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2023-07-24T12:16:17.602492Z",
-            "date_released": "2023-07-24T12:16:17.602472Z",
-            "description": "The Cell Ontology is a structured controlled vocabulary for cell types in animals.",
-            "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/CL/98/cl.owl",
+            "date_created": "2023-03-29T05:22:54Z",
+            "date_released": "2023-03-28T00:00:00Z",
+            "description": "US Birth ontology is a formal representation of the concepts, properties, and relationships that are relevant to the domain of US birth data\r\n\r\nCompetency questions are:\r\n1. What is the total number of live births in the United States for a given year?\r\n\r\n2. What is the distribution of birth weight for babies born to mothers of different ages?\r\n\r\n3. What is the prevalence of preterm births among babies born to mothers who smoke during pregnancy?\r\n\r\n4. What is the relationship between maternal age and the likelihood of having a multiple birth?\r\n\r\n5. What is the difference in birth outcomes between mothers who receive prenatal care and those who do not?\r\n\r\n6. What is the trend in the rate of low birth weight babies over time?\r\n\r\n7. What is the impact of socioeconomic status on birth outcomes, such as birth weight and gestational age?\r\n\r\n8. What is the relationship between the mother's race and ethnicity and the likelihood of having a preterm birth?\r\n\r\n9. What are the common risk factors for adverse birth outcomes, and how do they interact with each other?\r\n\r\nUS Birth ontology can be used to facilitate data integration, analysis, and sharing across different data sources and systems. It can help to ensure that data is consistent, interoperable, and reusable, and can facilitate data discovery and reuse. The ontology can be used by researchers, healthcare professionals, and policymakers to ask and answer questions about birth outcomes, risk factors, and trends in the US birth data.",
+            "documentation": "https://koolgax99.github.io/abhilash-US-birth-onto/index-en.html",
+            "domain": null,
+            "download_url": "media/ontologies/BIRTHONTO/1/birthonto.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://obophenotype.github.io/cell-ontology/",
-            "id": 5958,
+            "home_page": "https://koolgax99.github.io/abhilash-US-birth-onto/index-en.html",
+            "id": 5750,
             "indexed": true,
-            "max_children": 395,
-            "max_depth": 32,
-            "md5sum": "3c01d6205178993e34652f662d746e2d",
-            "nb_classes": 16047,
+            "max_children": 16,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 87,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 302,
-            "products": [
-                {
-                    "description": "Complete ontology, plus inter-ontology axioms, and imports modules",
-                    "format": "owl-rdf/xml",
-                    "id": "cl.owl",
-                    "is_canonical": true,
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cl.owl",
-                    "title": "Main CL OWL edition",
-                    "uses": [
-                        "uberon",
-                        "chebi",
-                        "go",
-                        "pr",
-                        "pato",
-                        "ncbitaxon",
-                        "ro"
-                    ]
-                },
-                {
-                    "derived_from": "cl.owl",
-                    "description": "Complete ontology, plus inter-ontology axioms, and imports modules merged in",
-                    "format": "obo",
-                    "id": "cl.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cl.obo",
-                    "title": "CL obo format edition"
-                },
-                {
-                    "description": "Basic version, no inter-ontology axioms",
-                    "format": "obo",
-                    "id": "cl/cl-basic.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cl/cl-basic.obo",
-                    "title": "Basic CL"
-                },
-                {
-                    "description": "complete CL but with no imports or external axioms",
-                    "id": "cl/cl-base.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cl/cl-base.owl",
-                    "title": "CL base module"
-                }
-            ],
+            "nb_individuals": 590,
+            "nb_properties": 71,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/27377652",
-                    "title": "The Cell Ontology 2016: enhanced content, modularization, and ontology interoperability."
-                }
-            ],
-            "submission_id": 98,
-            "taxon": {
-                "id": "NCBITaxon:33208",
-                "label": "Metazoa"
-            },
-            "version": null
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "CLAO",
-        "name": "Collembola Anatomy Ontology",
+        "acronym": "CHMO",
+        "name": "Chemical Methods Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-10-25T13:30:39.044156Z",
-            "date_released": "2021-10-25T13:30:39.044139Z",
-            "description": "CLAO is an ontology of anatomical terms employed in morphological descriptions for the Class Collembola (Arthropoda: Hexapoda).",
+            "date_created": "2023-05-29T12:18:01.801012Z",
+            "date_released": "2023-05-29T12:18:01.800996Z",
+            "description": "CHMO, the chemical methods ontology, describes methods used to",
             "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/CLAO/2/clao.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/CHMO/31/chmo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/luis-gonzalez-m/Collembola",
-            "id": 4513,
+            "home_page": "https://github.com/rsc-ontologies/rsc-cmo",
+            "id": 5827,
             "indexed": true,
-            "max_children": 340,
-            "max_depth": 14,
-            "md5sum": "99c04b1e9d00640957ebcd96bb020c23",
-            "nb_classes": 1516,
+            "max_children": 125,
+            "max_depth": 12,
+            "md5sum": "cb7d85fd0794f6f88318c8dc57274d5e",
+            "nb_classes": 3101,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 26,
+            "nb_properties": 27,
             "products": [
                 {
-                    "id": "clao.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/clao.owl"
-                },
-                {
-                    "id": "clao.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/clao.obo"
+                    "id": "chmo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/chmo.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 31,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "COHSI2TERMSWEEKX",
-        "name": "cohsi2termsweekx",
+        "acronym": "CODO",
+        "name": "An Ontology for Collection and Analysis of COviD-19 Data",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2022-09-11T20:59:45Z",
-            "date_released": "2022-09-11T00:00:00Z",
-            "description": "Vocabularies are created to descibe week X report for CoHSI2 study",
-            "documentation": null,
+            "date_created": "2020-10-08T18:14:48Z",
+            "date_released": "2020-09-25T00:00:00Z",
+            "description": "This COVID-19 Ontology is a data model for publishing COVID-19 data on the Web as a Knowledge Graph. \r\n\r\nThe primary focuses of the model are (1) COVID-19 cases: the data for COVID-19 cases (e.g., active, recovered, deceased, migrated) on daily basis across the geo-location (district, state (province), and country), available resources and requirements; \r\n(2) COVID-19 patient data: nationality, symptom, suspected level of COVID -19, treatment facility, COVID-19 clinical facility, patient's travel history, inter-personal relationships between patients, suspected transmission reason, tracking of patient test results, etc. ",
+            "documentation": "https://isibang.ac.in/ns/codo",
             "domain": null,
-            "download_url": "media/ontologies/COHSI2TERMSWEEKX/1/cohsi2termsweekx.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5348,
+            "download_url": "media/ontologies/CODO/4/codo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/biswanathdutta/CODO",
+            "id": 3983,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 13,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 90,
             "nb_inconsistent": 0,
-            "nb_individuals": 32,
-            "nb_properties": 0,
+            "nb_individuals": 271,
+            "nb_properties": 73,
             "products": null,
-            "publication": null,
+            "publication": "https://isibang.ac.in/ns/codo",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": null
+            "version": "1.3"
         },
         "topics": null
     },
     {
         "acronym": "COID",
         "name": "Combined Ontology for Inflammatory Diseases",
         "species": null,
@@ -12582,668 +12526,677 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "DUO",
-        "name": "Data Use Ontology",
+        "acronym": "CLO",
+        "name": "Cell Line Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2021-03-01T13:17:10.013270Z",
-            "date_released": "2021-03-01T13:17:10.013258Z",
-            "description": "DUO is an ontology which represent data use conditions.",
+            "date_created": "2022-03-21T12:16:18.480526Z",
+            "date_released": "2022-03-21T12:16:18.480515Z",
+            "description": "An ontology to standardize and integrate cell line information and to support computer-assisted reasoning.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DUO/9/duo.owl",
+            "download_url": "media/ontologies/CLO/6/clo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/EBISPOT/DUO",
-            "id": 4303,
+            "home_page": "http://www.clo-ontology.org",
+            "id": 4961,
             "indexed": true,
-            "max_children": 39,
-            "max_depth": 11,
-            "md5sum": "b150506aa3a0fece85eef888f09c0388",
-            "nb_classes": 300,
+            "max_children": 21698,
+            "max_depth": 36,
+            "md5sum": "ea58a1010b7e745702a8397a526b3a33",
+            "nb_classes": 43325,
             "nb_inconsistent": 0,
-            "nb_individuals": 22,
-            "nb_properties": 51,
+            "nb_individuals": 26,
+            "nb_properties": 249,
             "products": [
                 {
-                    "id": "duo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/duo.owl"
+                    "id": "clo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/clo.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 6,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LBO",
-        "name": "Livestock Breed Ontology",
+        "acronym": "INTO",
+        "name": "Indian Terrorism Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 81,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-04-12T20:14:30Z",
-            "date_released": "2017-04-12T00:00:00Z",
-            "description": "A vocabulary for buffalo, cattle, chicken, goat, horse, pig, and sheep breeds.",
+            "date_created": "2021-06-07T18:00:22Z",
+            "date_released": "2021-06-07T00:00:00Z",
+            "description": "Creating the largest comprehensive knowledge base of terrorism and related activities, people and agencies involved, and extremist movements; and providing a platform to the society, the government, and the military personnel in order to combat the evolving threat of the global menace of terrorism.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LBO/9/lbo.obo",
-            "has_ontology_language": "OBO",
+            "download_url": "media/ontologies/INTO/1/into.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 611,
+            "id": 4615,
             "indexed": true,
-            "max_children": 291,
-            "max_depth": 3,
+            "max_children": 16,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 1064,
+            "nb_classes": 261,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 929,
+            "nb_properties": 22,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2.2"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "LC-CARRIERS",
-        "name": "Library of Congress Carriers Scheme",
+        "acronym": "CIDO",
+        "name": "Coronavirus Infectious Disease Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2018-09-09T23:13:05Z",
-            "date_released": "2018-05-07T00:00:00Z",
-            "description": "Carriers Scheme is derived from a controlled list of values representing the format of the storage medium and its media type, which is principally used in RDA cataloging.",
+            "date_created": "2023-01-30T12:17:14.821954Z",
+            "date_released": "2023-01-30T12:17:14.821935Z",
+            "description": "The Coronavirus Infectious Disease Ontology (CIDO) aims to ontologically represent and standardize various aspects of coronavirus infectious diseases, including their etiology, transmission, epidemiology, pathogenesis, diagnosis, prevention, and treatment.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/LC-CARRIERS/3/lc-carriers.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/vocabulary/carriers.html",
-            "id": 1819,
+            "domain": "health",
+            "download_url": "media/ontologies/CIDO/33/cido.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/cido-ontology/cido",
+            "id": 5626,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 6,
+            "max_children": 5749,
+            "max_depth": 38,
+            "md5sum": "ed8d5b61a188602622cf664962dd21e1",
+            "nb_classes": 31620,
             "nb_inconsistent": 0,
-            "nb_individuals": 56,
-            "nb_properties": 0,
-            "products": null,
+            "nb_individuals": 458,
+            "nb_properties": 384,
+            "products": [
+                {
+                    "id": "cido.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cido.owl"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 3,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/36271389",
+                    "title": "A comprehensive update on CIDO: the community-based coronavirus infectious disease ontology"
+                }
+            ],
+            "submission_id": 33,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "EBP",
-        "name": "EmpowerBP",
+        "acronym": "INBIO",
+        "name": "Invasion Biology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 10,
             "classifiable": true,
-            "date_created": "2018-06-15T15:25:53Z",
-            "date_released": "2018-02-21T00:00:00Z",
-            "description": "Clinical practice guidelines, behaviour change theories, and associated behaviour change strategies for the management of hypertension.",
+            "date_created": "2023-02-02T10:32:47Z",
+            "date_released": "2023-02-02T00:00:00Z",
+            "description": "The Invasion Biology Ontology (INBIO) contains terms and concepts relevant in the field of invasion biology, which is a research area dealing with the translocation, establishment, spread, impact and management of species outside of their native ranges, where they are called non-native or alien species. This first version of the ontology covers terms and concepts needed to describe twelve major invasion hypotheses building the hierarchical hypothesis network (see also Jeschke JM, Heger T (Eds) (2018) Invasion Biology: Hypotheses and Evidence. CABI, Wallingford, UK).",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/EBP/2/ebp.owl",
+            "download_url": "media/ontologies/INBIO/2/inbio.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 1582,
+            "id": 5644,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 3,
+            "max_children": 17,
+            "max_depth": 30,
             "md5sum": null,
-            "nb_classes": 50,
-            "nb_inconsistent": 0,
-            "nb_individuals": 321,
-            "nb_properties": 26,
+            "nb_classes": 458,
+            "nb_inconsistent": 17,
+            "nb_individuals": 18,
+            "nb_properties": 92,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "2.5"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "EDDA",
-        "name": "EDDA Study Designs Taxonomy",
+        "acronym": "COB",
+        "name": "Core Ontology for Biology and Biomedicine",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2016-07-16T15:20:06Z",
-            "date_released": "2016-07-16T00:00:00Z",
-            "description": "The EDDA Study Designs Taxonomy (v2.0) was developed by the Evidence in Documents, Discovery, and Analytics (EDDA) Group: Tanja Bekhuis (Principal Scientist); Eugene Tseytlin (Systems Developer); Ashleigh Faith (Taxonomist); Faina Linkov (Epidemiologist). The EDDA Group is a division of TCB Research & Indexing LLC. This work was made possible, in part, by the US National Library of Medicine, National Institutes of Health, grant no. R00LM010943. \r\n\r\nFoundational research is described in Bekhuis T, Demner Fushman D, Crowley RS. Comparative effectiveness research designs: an analysis of terms and coverage in Medical Subject Headings (MeSH) and Emtree. Journal of the Medical Library Association (JMLA). 2013 April;101(2):92-100. PMC3634392. \r\n\r\nCoverage of the terminology appearing in JMLA was extended with terms from MeSH, NCI Thesaurus (NCI), Emtree, the HTA Database Canadian Repository [international repository for health technology assessment], and Robert Sandieson's synonym ring for research synthesis. Collected terms were enriched with terms from the NCI Metathesaurus. Variants include synonyms for preferred terms, singular and plural forms, and American and British spellings. Definitions, if they exist, are mainly from MeSH, NCI, Emtree, and medical dictionaries.\r\n\r\nA PDF of this taxonomy is available in ResearchGate DOI: 10.13140/RG.2.1.3769.2406/1.\r\n\r\nThe EDDA Study Designs Taxonomy by Tanja Bekhuis and Eugene Tseytlin is licensed under a Creative Commons Attribution\u2013NonCommercial\u2013ShareAlike 4.0 International License (CC BY-NC-SA 4.0). For information about this license, see https://creativecommons.org/licenses/by-nc-sa/4.0/.",
+            "date_created": "2023-05-15T12:22:54.016356Z",
+            "date_released": "2023-05-15T12:22:54.016340Z",
+            "description": "COB brings together key terms from a wide range of OBO projects to improve interoperability.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/EDDA/11/edda.owl",
+            "domain": "upper",
+            "download_url": "media/ontologies/COB/10/cob.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 57,
+            "home_page": "https://obofoundry.org/COB/",
+            "id": 5807,
             "indexed": true,
-            "max_children": 93,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 502,
+            "max_children": 14,
+            "max_depth": 4,
+            "md5sum": "d8416e75f8f27bf27a032cbcc7836be4",
+            "nb_classes": 74,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": null,
+            "nb_properties": 23,
+            "products": [
+                {
+                    "description": "Core Ontology for Biology and Biomedicine, main ontology",
+                    "id": "cob.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cob.owl",
+                    "title": "COB"
+                },
+                {
+                    "description": "base module for COB",
+                    "id": "cob/cob-base.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cob/cob-base.owl",
+                    "title": "COB base module"
+                },
+                {
+                    "id": "cob/cob-to-external.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cob/cob-to-external.owl",
+                    "title": "COB to external",
+                    "type": "BridgeOntology"
+                },
+                {
+                    "description": "demo of COB including subsets of other ontologies (Experimental, for demo purposes only)",
+                    "id": "cob/products/demo-cob.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cob/products/demo-cob.owl",
+                    "status": "alpha",
+                    "title": "COB demo ontology (experimental)"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 10,
             "taxon": null,
-            "version": "2.0"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LC-CONTENT",
-        "name": "Library of Congress Content Types Scheme",
+        "acronym": "CL",
+        "name": "Cell Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-04-19T01:07:20Z",
-            "date_released": "2021-04-19T01:07:18Z",
-            "description": null,
+            "date_created": "2023-07-24T12:16:17.602492Z",
+            "date_released": "2023-07-24T12:16:17.602472Z",
+            "description": "The Cell Ontology is a structured controlled vocabulary for cell types in animals.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/LC-CONTENT/157/lc-content.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/vocabulary/contentTypes.html",
-            "id": 4416,
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/CL/98/cl.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://obophenotype.github.io/cell-ontology/",
+            "id": 5958,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 4,
+            "max_children": 395,
+            "max_depth": 32,
+            "md5sum": "3c01d6205178993e34652f662d746e2d",
+            "nb_classes": 16047,
             "nb_inconsistent": 0,
-            "nb_individuals": 26,
-            "nb_properties": 0,
-            "products": null,
+            "nb_individuals": 18,
+            "nb_properties": 302,
+            "products": [
+                {
+                    "description": "Complete ontology, plus inter-ontology axioms, and imports modules",
+                    "format": "owl-rdf/xml",
+                    "id": "cl.owl",
+                    "is_canonical": true,
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cl.owl",
+                    "title": "Main CL OWL edition",
+                    "uses": [
+                        "uberon",
+                        "chebi",
+                        "go",
+                        "pr",
+                        "pato",
+                        "ncbitaxon",
+                        "ro"
+                    ]
+                },
+                {
+                    "derived_from": "cl.owl",
+                    "description": "Complete ontology, plus inter-ontology axioms, and imports modules merged in",
+                    "format": "obo",
+                    "id": "cl.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cl.obo",
+                    "title": "CL obo format edition"
+                },
+                {
+                    "description": "Basic version, no inter-ontology axioms",
+                    "format": "obo",
+                    "id": "cl/cl-basic.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cl/cl-basic.obo",
+                    "title": "Basic CL"
+                },
+                {
+                    "description": "complete CL but with no imports or external axioms",
+                    "id": "cl/cl-base.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cl/cl-base.owl",
+                    "title": "CL base module"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 157,
-            "taxon": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/27377652",
+                    "title": "The Cell Ontology 2016: enhanced content, modularization, and ontology interoperability."
+                }
+            ],
+            "submission_id": 98,
+            "taxon": {
+                "id": "NCBITaxon:33208",
+                "label": "Metazoa"
+            },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LCDGT",
-        "name": "Library of Congress Demographic Group Terms",
+        "acronym": "EDDA",
+        "name": "EDDA Study Designs Taxonomy",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2018-03-27T16:53:41Z",
-            "date_released": "2018-01-01T00:00:00Z",
-            "description": "The Library of Congress Demographic Group Terms (LCDGT) is designed to describe the characteristics of the intended audiences of resources, and also the creators of, and contributors to, those resources.",
+            "date_created": "2016-07-16T15:20:06Z",
+            "date_released": "2016-07-16T00:00:00Z",
+            "description": "The EDDA Study Designs Taxonomy (v2.0) was developed by the Evidence in Documents, Discovery, and Analytics (EDDA) Group: Tanja Bekhuis (Principal Scientist); Eugene Tseytlin (Systems Developer); Ashleigh Faith (Taxonomist); Faina Linkov (Epidemiologist). The EDDA Group is a division of TCB Research & Indexing LLC. This work was made possible, in part, by the US National Library of Medicine, National Institutes of Health, grant no. R00LM010943. \r\n\r\nFoundational research is described in Bekhuis T, Demner Fushman D, Crowley RS. Comparative effectiveness research designs: an analysis of terms and coverage in Medical Subject Headings (MeSH) and Emtree. Journal of the Medical Library Association (JMLA). 2013 April;101(2):92-100. PMC3634392. \r\n\r\nCoverage of the terminology appearing in JMLA was extended with terms from MeSH, NCI Thesaurus (NCI), Emtree, the HTA Database Canadian Repository [international repository for health technology assessment], and Robert Sandieson's synonym ring for research synthesis. Collected terms were enriched with terms from the NCI Metathesaurus. Variants include synonyms for preferred terms, singular and plural forms, and American and British spellings. Definitions, if they exist, are mainly from MeSH, NCI, Emtree, and medical dictionaries.\r\n\r\nA PDF of this taxonomy is available in ResearchGate DOI: 10.13140/RG.2.1.3769.2406/1.\r\n\r\nThe EDDA Study Designs Taxonomy by Tanja Bekhuis and Eugene Tseytlin is licensed under a Creative Commons Attribution\u2013NonCommercial\u2013ShareAlike 4.0 International License (CC BY-NC-SA 4.0). For information about this license, see https://creativecommons.org/licenses/by-nc-sa/4.0/.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LCDGT/2/lcdgt.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/authorities/demographicTerms.html",
-            "id": 1639,
+            "download_url": "media/ontologies/EDDA/11/edda.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 57,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 93,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 4,
+            "nb_classes": 502,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 4,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 11,
             "taxon": null,
-            "version": null
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "OMIT",
-        "name": "Ontology for MIRNA Target",
+        "acronym": "CLAO",
+        "name": "Collembola Anatomy Ontology",
         "species": null,
-        "status": "Incoherent",
+        "status": "Classified",
         "submission": {
-            "avg_children": 2225,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2017-12-13T10:42:10.368096Z",
-            "date_released": "2017-12-13T10:42:10.368073Z",
-            "description": "Ontology to establish data exchange standards and common data elements in the microRNA (miR) domain",
+            "date_created": "2021-10-25T13:30:39.044156Z",
+            "date_released": "2021-10-25T13:30:39.044139Z",
+            "description": "CLAO is an ontology of anatomical terms employed in morphological descriptions for the Class Collembola (Arthropoda: Hexapoda).",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OMIT/41/omit.owl",
+            "domain": "anatomy",
+            "download_url": "media/ontologies/CLAO/2/clao.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://omit.cis.usouthal.edu/",
-            "id": 1045,
+            "home_page": "https://github.com/luis-gonzalez-m/Collembola",
+            "id": 4513,
             "indexed": true,
-            "max_children": 59874,
-            "max_depth": 18,
-            "md5sum": "61894914b3b84db9ace969d26854349c",
-            "nb_classes": 90894,
-            "nb_inconsistent": 2493,
-            "nb_individuals": 18,
-            "nb_properties": 44,
+            "max_children": 340,
+            "max_depth": 14,
+            "md5sum": "99c04b1e9d00640957ebcd96bb020c23",
+            "nb_classes": 1516,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 26,
             "products": [
                 {
-                    "id": "omit.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/omit.owl"
+                    "id": "clao.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/clao.owl"
+                },
+                {
+                    "id": "clao.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/clao.obo"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 41,
+            "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LCGFT",
-        "name": "Library of Congress Genre/Form Thesaurus for Library and Archival Materials",
+        "acronym": "COHSI2TERMSWEEKX",
+        "name": "cohsi2termsweekx",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 12,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-09-06T18:47:49Z",
-            "date_released": "2017-09-05T00:00:00Z",
-            "description": "The Library of Congress Genre/Form Terms for Library and Archival Materials (LCGFT) is a thesaurus that describes what a work is versus what it is about. For instance, the subject heading Horror films, with appropriate subdivisions, would be assigned to a book about horror films. A cataloger assigning headings to the movie The Texas Chainsaw Massacre would also use Horror films, but it would be a genre/form term since the movie is a horror film, not a movie about horror films. The thesaurus combines both genres and forms. Form is defined as a characteristic of works with a particular format and/or purpose. A \"short\" is a particular form, for example, as is \"animation.\" Genre refers to categories of works that are characterized by similar plots, themes, settings, situations, and characters. Examples of genres are westerns and thrillers. In the term Horror films\"horror\" is the genre and \"films\" is the form.",
+            "date_created": "2022-09-11T20:59:45Z",
+            "date_released": "2022-09-11T00:00:00Z",
+            "description": "Vocabularies are created to descibe week X report for CoHSI2 study",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LCGFT/3/lcgft.skos",
+            "download_url": "media/ontologies/COHSI2TERMSWEEKX/1/cohsi2termsweekx.skos",
             "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/authorities/genreForms.html",
-            "id": 404,
+            "home_page": null,
+            "id": 5348,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 0,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 12,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 2181,
+            "nb_individuals": 32,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2017-09-05"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LCGFT_SKOS",
-        "name": "Library of Congress Genre/Form Terms (SKOS)",
+        "acronym": "ECG",
+        "name": "Electrocardiography Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2018-03-20T22:17:17Z",
-            "date_released": "2018-03-20T00:00:00Z",
-            "description": "The Library of Congress Genre/Form Terms for Library and Archival Materials (LCGFT) is a thesaurus that describes what a work is versus what it is about.",
+            "date_created": "2014-09-12T14:50:59Z",
+            "date_released": "2014-09-09T07:00:00Z",
+            "description": "The Electrocardiography (ECG) Ontology is a Driving Biological Project of the NCBO. The ECG Ontology will contain terms for describing electrocardiograms, their capture method(s) and their waveforms.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LCGFT_SKOS/2/lcgft_skos.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 1353,
+            "download_url": "media/ontologies/ECG/12/ecg.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.cvrgrid.org",
+            "id": 755,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
+            "max_children": 109,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 4,
+            "nb_classes": 1124,
             "nb_inconsistent": 0,
-            "nb_individuals": 2222,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 72,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 12,
             "taxon": null,
-            "version": "using output of skosify"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "INO",
-        "name": "Interaction Network Ontology",
+        "acronym": "OMIT",
+        "name": "Ontology for MIRNA Target",
         "species": null,
-        "status": "Classified",
+        "status": "Incoherent",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2225,
             "classifiable": true,
-            "date_created": "2022-03-28T12:21:05.529510Z",
-            "date_released": "2022-03-28T12:21:05.529499Z",
-            "description": "An ontology of interactions and interaction networks",
+            "date_created": "2017-12-13T10:42:10.368096Z",
+            "date_released": "2017-12-13T10:42:10.368073Z",
+            "description": "Ontology to establish data exchange standards and common data elements in the microRNA (miR) domain",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INO/32/ino.owl",
+            "download_url": "media/ontologies/OMIT/41/omit.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/INO-ontology/ino",
-            "id": 4978,
+            "home_page": "http://omit.cis.usouthal.edu/",
+            "id": 1045,
             "indexed": true,
-            "max_children": 43,
-            "max_depth": 12,
-            "md5sum": "7d35b715a7e2412851c6cd63dd6082c7",
-            "nb_classes": 328,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 59,
+            "max_children": 59874,
+            "max_depth": 18,
+            "md5sum": "61894914b3b84db9ace969d26854349c",
+            "nb_classes": 90894,
+            "nb_inconsistent": 2493,
+            "nb_individuals": 18,
+            "nb_properties": 44,
             "products": [
                 {
-                    "id": "ino.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ino.owl"
+                    "id": "omit.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/omit.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 32,
+            "submission_id": 41,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "INSNAME",
-        "name": "VODANAINSNAMES",
+        "acronym": "LCDGT",
+        "name": "Library of Congress Demographic Group Terms",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-03-30T05:33:23Z",
-            "date_released": "2021-03-26T00:00:00Z",
-            "description": "This is the list of facilities participating in FAIRification of data.",
+            "date_created": "2018-03-27T16:53:41Z",
+            "date_released": "2018-01-01T00:00:00Z",
+            "description": "The Library of Congress Demographic Group Terms (LCDGT) is designed to describe the characteristics of the intended audiences of resources, and also the creators of, and contributors to, those resources.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INSNAME/5/insname.skos",
+            "download_url": "media/ontologies/LCDGT/2/lcdgt.skos",
             "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4373,
+            "home_page": "http://id.loc.gov/authorities/demographicTerms.html",
+            "id": 1639,
             "indexed": true,
             "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 24,
+            "nb_individuals": 1,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 2,
             "taxon": null,
-            "version": "5"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "INTO",
-        "name": "Indian Terrorism Ontology",
+        "acronym": "EBP",
+        "name": "EmpowerBP",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-06-07T18:00:22Z",
-            "date_released": "2021-06-07T00:00:00Z",
-            "description": "Creating the largest comprehensive knowledge base of terrorism and related activities, people and agencies involved, and extremist movements; and providing a platform to the society, the government, and the military personnel in order to combat the evolving threat of the global menace of terrorism.",
+            "date_created": "2018-06-15T15:25:53Z",
+            "date_released": "2018-02-21T00:00:00Z",
+            "description": "Clinical practice guidelines, behaviour change theories, and associated behaviour change strategies for the management of hypertension.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INTO/1/into.owl",
+            "download_url": "media/ontologies/EBP/2/ebp.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4615,
+            "id": 1582,
             "indexed": true,
             "max_children": 16,
-            "max_depth": 8,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 261,
+            "nb_classes": 50,
             "nb_inconsistent": 0,
-            "nb_individuals": 929,
-            "nb_properties": 22,
+            "nb_individuals": 321,
+            "nb_properties": 26,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": "2.5"
         },
         "topics": null
     },
     {
-        "acronym": "BIRNLEX",
-        "name": "Biomedical Informatics Research Network Project Lexicon",
+        "acronym": "LCGFT",
+        "name": "Library of Congress Genre/Form Thesaurus for Library and Archival Materials",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 12,
             "classifiable": true,
-            "date_created": "2008-03-25T00:39:39Z",
-            "date_released": "2007-08-08T07:00:00Z",
-            "description": "The BIRN Project lexicon will provide entities for data and database annotation for the BIRN project, covering anatomy, disease, data collection, project management and experimental design.",
+            "date_created": "2017-09-06T18:47:49Z",
+            "date_released": "2017-09-05T00:00:00Z",
+            "description": "The Library of Congress Genre/Form Terms for Library and Archival Materials (LCGFT) is a thesaurus that describes what a work is versus what it is about. For instance, the subject heading Horror films, with appropriate subdivisions, would be assigned to a book about horror films. A cataloger assigning headings to the movie The Texas Chainsaw Massacre would also use Horror films, but it would be a genre/form term since the movie is a horror film, not a movie about horror films. The thesaurus combines both genres and forms. Form is defined as a characteristic of works with a particular format and/or purpose. A \"short\" is a particular form, for example, as is \"animation.\" Genre refers to categories of works that are characterized by similar plots, themes, settings, situations, and characters. Examples of genres are westerns and thrillers. In the term Horror films\"horror\" is the genre and \"films\" is the form.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BIRNLEX/1/birnlex.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://xwiki.nbirn.net:8080/xwiki/bin/view/%20BIRN-OTF-Public/Home",
-            "id": 34,
-            "indexed": true,
-            "max_children": 66,
-            "max_depth": 40,
-            "md5sum": null,
-            "nb_classes": 3580,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 9,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.3.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BIRTHONTO",
-        "name": "BirthOnto: Birth Data Analysis",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 6,
-            "classifiable": true,
-            "date_created": "2023-03-29T05:22:54Z",
-            "date_released": "2023-03-28T00:00:00Z",
-            "description": "US Birth ontology is a formal representation of the concepts, properties, and relationships that are relevant to the domain of US birth data\r\n\r\nCompetency questions are:\r\n1. What is the total number of live births in the United States for a given year?\r\n\r\n2. What is the distribution of birth weight for babies born to mothers of different ages?\r\n\r\n3. What is the prevalence of preterm births among babies born to mothers who smoke during pregnancy?\r\n\r\n4. What is the relationship between maternal age and the likelihood of having a multiple birth?\r\n\r\n5. What is the difference in birth outcomes between mothers who receive prenatal care and those who do not?\r\n\r\n6. What is the trend in the rate of low birth weight babies over time?\r\n\r\n7. What is the impact of socioeconomic status on birth outcomes, such as birth weight and gestational age?\r\n\r\n8. What is the relationship between the mother's race and ethnicity and the likelihood of having a preterm birth?\r\n\r\n9. What are the common risk factors for adverse birth outcomes, and how do they interact with each other?\r\n\r\nUS Birth ontology can be used to facilitate data integration, analysis, and sharing across different data sources and systems. It can help to ensure that data is consistent, interoperable, and reusable, and can facilitate data discovery and reuse. The ontology can be used by researchers, healthcare professionals, and policymakers to ask and answer questions about birth outcomes, risk factors, and trends in the US birth data.",
-            "documentation": "https://koolgax99.github.io/abhilash-US-birth-onto/index-en.html",
-            "domain": null,
-            "download_url": "media/ontologies/BIRTHONTO/1/birthonto.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://koolgax99.github.io/abhilash-US-birth-onto/index-en.html",
-            "id": 5750,
+            "download_url": "media/ontologies/LCGFT/3/lcgft.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://id.loc.gov/authorities/genreForms.html",
+            "id": 404,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 3,
+            "max_children": 12,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 87,
+            "nb_classes": 12,
             "nb_inconsistent": 0,
-            "nb_individuals": 590,
-            "nb_properties": 71,
+            "nb_individuals": 2181,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.1"
+            "version": "2017-09-05"
         },
         "topics": null
     },
     {
-        "acronym": "CHMO",
-        "name": "Chemical Methods Ontology",
+        "acronym": "CLYH",
+        "name": "Clytia hemisphaerica Development and Anatomy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2023-05-29T12:18:01.801012Z",
-            "date_released": "2023-05-29T12:18:01.800996Z",
-            "description": "CHMO, the chemical methods ontology, describes methods used to",
+            "date_created": "2020-06-01T14:09:54.824035Z",
+            "date_released": "2020-06-01T14:09:54.824025Z",
+            "description": "The Clytia hemisphaerica Development and Anatomy Ontology (CLYH) describes the anatomical and developmental features of the Clytia hemisphaerica life cycle.",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/CHMO/31/chmo.owl",
+            "domain": "anatomy",
+            "download_url": "media/ontologies/CLYH/3/clyh.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/rsc-ontologies/rsc-cmo",
-            "id": 5827,
+            "home_page": "https://github.com/EBISPOT/clyh_ontology",
+            "id": 3642,
             "indexed": true,
-            "max_children": 125,
-            "max_depth": 12,
-            "md5sum": "cb7d85fd0794f6f88318c8dc57274d5e",
-            "nb_classes": 3101,
+            "max_children": 76,
+            "max_depth": 5,
+            "md5sum": "3f6dcba5ce39635cb45cc5fbfaee3c91",
+            "nb_classes": 265,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 27,
+            "nb_properties": 5,
             "products": [
                 {
-                    "id": "chmo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/chmo.owl"
+                    "id": "clyh.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/clyh.owl"
+                },
+                {
+                    "id": "clyh.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/clyh.obo"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 31,
+            "submission_id": 3,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CLO",
-        "name": "Cell Line Ontology",
+        "acronym": "LC-MEDIA",
+        "name": "Library of Congress Media Types Scheme",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-03-21T12:16:18.480526Z",
-            "date_released": "2022-03-21T12:16:18.480515Z",
-            "description": "An ontology to standardize and integrate cell line information and to support computer-assisted reasoning.",
+            "date_created": "2020-07-03T01:05:13Z",
+            "date_released": "2020-07-03T01:05:12Z",
+            "description": "Media Types Scheme is derived from a controlled list of values reflecting the general type of intermediation device required to view, play, run, etc., the content of a resource, which is principally used in RDA cataloging.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CLO/6/clo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.clo-ontology.org",
-            "id": 4961,
+            "download_url": "media/ontologies/LC-MEDIA/17/lc-media.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://id.loc.gov/vocabulary/mediaTypes.html",
+            "id": 3749,
             "indexed": true,
-            "max_children": 21698,
-            "max_depth": 36,
-            "md5sum": "ea58a1010b7e745702a8397a526b3a33",
-            "nb_classes": 43325,
+            "max_children": 4,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 6,
             "nb_inconsistent": 0,
-            "nb_individuals": 26,
-            "nb_properties": 249,
-            "products": [
-                {
-                    "id": "clo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/clo.owl"
-                }
-            ],
+            "nb_individuals": 11,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 17,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "D1245",
         "name": "Data Center",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "CLYH",
-        "name": "Clytia hemisphaerica Development and Anatomy Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2020-06-01T14:09:54.824035Z",
-            "date_released": "2020-06-01T14:09:54.824025Z",
-            "description": "The Clytia hemisphaerica Development and Anatomy Ontology (CLYH) describes the anatomical and developmental features of the Clytia hemisphaerica life cycle.",
-            "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/CLYH/3/clyh.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/EBISPOT/clyh_ontology",
-            "id": 3642,
-            "indexed": true,
-            "max_children": 76,
-            "max_depth": 5,
-            "md5sum": "3f6dcba5ce39635cb45cc5fbfaee3c91",
-            "nb_classes": 265,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 5,
-            "products": [
-                {
-                    "id": "clyh.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/clyh.owl"
-                },
-                {
-                    "id": "clyh.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/clyh.obo"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
         "acronym": "ECAO",
         "name": "The Echinoderm Anatomy and Development Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
@@ -13279,759 +13232,636 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ECG",
-        "name": "Electrocardiography Ontology",
+        "acronym": "COGAT",
+        "name": "Cognitive Atlas Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": null,
             "classifiable": true,
-            "date_created": "2014-09-12T14:50:59Z",
-            "date_released": "2014-09-09T07:00:00Z",
-            "description": "The Electrocardiography (ECG) Ontology is a Driving Biological Project of the NCBO. The ECG Ontology will contain terms for describing electrocardiograms, their capture method(s) and their waveforms.",
+            "date_created": "2016-08-15T22:55:22Z",
+            "date_released": "2012-04-03T07:00:00Z",
+            "description": "The Cognitive Atlas is a collaborative knowledge building project that aims to develop an ontology that characterizes the state of current thought in cognitive science.  It defines a set of mental concepts along with a set of mental tasks, and the measurement relations between those classes.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ECG/12/ecg.owl",
+            "download_url": "media/ontologies/COGAT/7/cogat.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.cvrgrid.org",
-            "id": 755,
+            "home_page": "http://www.cognitiveatlas.org",
+            "id": 79,
             "indexed": true,
-            "max_children": 109,
-            "max_depth": 11,
+            "max_children": null,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 1124,
+            "nb_classes": 3639,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 72,
+            "nb_properties": 6,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
+            "submission_id": 7,
             "taxon": null,
-            "version": "1"
+            "version": "0.3.1"
         },
         "topics": null
     },
     {
-        "acronym": "gr",
-        "name": "gr_ontology",
-        "species": [],
+        "acronym": "LCMPT",
+        "name": "Library of Congress Medium of Performance Thesaurus for Music",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 10,
             "classifiable": true,
-            "date_created": "2022-08-12T09:40:35.502078Z",
-            "date_released": "2022-08-12T09:40:35.502091Z",
-            "description": "",
+            "date_created": "2017-09-06T18:52:09Z",
+            "date_released": "2017-08-31T00:00:00Z",
+            "description": "Vocabulary that provides terminology to describe the instruments, voices, etc., used in the performance of musical works.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/gr/1/gr.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5276,
+            "download_url": "media/ontologies/LCMPT/3/lcmpt.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://id.loc.gov/authorities/performanceMediums.html",
+            "id": 747,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 4,
+            "max_children": 10,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 105,
+            "nb_classes": 10,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 849,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "juhiygh"
+            "version": "31 Aug 2017"
         },
-        "topics": []
+        "topics": null
     },
     {
-        "acronym": "CMDO",
-        "name": "Clinical MetaData Ontology",
+        "acronym": "IPD",
+        "name": "injury process description ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-07-12T09:44:05Z",
-            "date_released": "2019-02-26T00:00:00Z",
-            "description": "The Clinical MetaData Ontology is designed to be used to classify Common Data Elements (CDEs) from different studies and clinical documents. CDEs can be efficiently searched, shared and reused by using our ontology.",
+            "date_created": "2023-01-22T10:09:37Z",
+            "date_released": "2023-01-22T00:00:00Z",
+            "description": "IPD-Onto divides product injury  into five unified classes (host, vector, agent, environment, and consequences) and constructs the accident process through the object properties (lead to, touch, accident, etc.)IPD-Onto is used to realise the transformation and reorganisation from product injury to injury information, to reproduce the process of accident occurrence, and realise human-computer interactions to meet the needs of data query.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CMDO/10/cmdo.owl",
+            "download_url": "media/ontologies/IPD/2/ipd.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.snubi.org/software/cmo/",
-            "id": 2743,
+            "home_page": null,
+            "id": 5625,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 7,
+            "max_children": 5,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 188,
+            "nb_classes": 8,
             "nb_inconsistent": 0,
-            "nb_individuals": 151,
-            "nb_properties": 3,
+            "nb_individuals": 5,
+            "nb_properties": 20,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 10,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1.4"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "CMO",
-        "name": "Clinical measurement ontology",
+        "acronym": "LABO",
+        "name": "clinical LABoratory Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 20,
             "classifiable": true,
-            "date_created": "2023-07-17T12:16:30.553077Z",
-            "date_released": "2023-07-17T12:16:30.553057Z",
-            "description": "Morphological and physiological measurement records generated from clinical and model organism research and health programs.",
+            "date_created": "2022-12-12T12:33:18.402360Z",
+            "date_released": "2022-12-12T12:33:18.402343Z",
+            "description": "LABO is an ontology of informational entities formalizing clinical laboratory tests prescriptions and reporting documents.",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/CMO/93/cmo.owl",
+            "domain": "information",
+            "download_url": "media/ontologies/LABO/3/labo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://rgd.mcw.edu/rgdweb/ontology/search.html",
-            "id": 5944,
+            "home_page": "https://github.com/OpenLHS/LABO",
+            "id": 5535,
             "indexed": true,
-            "max_children": 111,
-            "max_depth": 11,
-            "md5sum": "667ce5f7e4ef0eded2264293d33ce594",
-            "nb_classes": 3927,
-            "nb_inconsistent": 0,
+            "max_children": 31,
+            "max_depth": 9,
+            "md5sum": "b29d5c3eccf778158c823b80df36a782",
+            "nb_classes": 169,
+            "nb_inconsistent": 31,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 33,
             "products": [
                 {
-                    "id": "cmo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cmo.owl"
-                },
-                {
-                    "id": "cmo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cmo.obo"
+                    "id": "labo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/labo.owl"
                 }
             ],
             "publication": null,
             "publications": [
                 {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/22654893",
-                    "title": "Three ontologies to define phenotype measurement data."
-                },
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24103152",
-                    "title": "The clinical measurement, measurement method and experimental condition ontologies: expansion, improvements and new applications."
+                    "id": "https://doi.org/10.5281/zenodo.6522019",
+                    "title": "LABO: An Ontology for Laboratory Test Prescription and Reporting"
                 }
             ],
-            "submission_id": 93,
-            "taxon": null,
+            "submission_id": 3,
+            "taxon": {
+                "id": "NCBITaxon:9606",
+                "label": "Homo sapiens"
+            },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CMPO",
-        "name": "Cellular microscopy phenotype ontology",
+        "acronym": "DUO",
+        "name": "Data Use Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2015-06-25T01:01:45Z",
-            "date_released": "2015-06-25T01:01:45Z",
-            "description": "The Cellular Microscopy Phenotype Ontology (CMPO) is a species neutral ontology for describing general phenotypic observations relating to the whole cell, cellular components, cellular processes and cell populations. CMPO is an application ontology developed in OWL that contains precomposed phenotypes descriptions that are defined using the Gene Ontology (GO) and the Phenotype Trait Ontology (PATO).",
+            "date_created": "2021-03-01T13:17:10.013270Z",
+            "date_released": "2021-03-01T13:17:10.013258Z",
+            "description": "DUO is an ontology which represent data use conditions.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CMPO/14/cmpo.owl",
+            "download_url": "media/ontologies/DUO/9/duo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/EBISPOT/CMPO",
-            "id": 484,
+            "home_page": "https://github.com/EBISPOT/DUO",
+            "id": 4303,
             "indexed": true,
-            "max_children": 67,
-            "max_depth": 12,
-            "md5sum": null,
-            "nb_classes": 813,
+            "max_children": 39,
+            "max_depth": 11,
+            "md5sum": "b150506aa3a0fece85eef888f09c0388",
+            "nb_classes": 300,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 30,
-            "products": null,
+            "nb_individuals": 22,
+            "nb_properties": 51,
+            "products": [
+                {
+                    "id": "duo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/duo.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 9,
             "taxon": null,
-            "version": "1.3"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "cmt",
-        "name": "cmt",
-        "species": [
-            "cmt"
-        ],
+        "acronym": "LBO",
+        "name": "Livestock Breed Ontology",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 81,
             "classifiable": true,
-            "date_created": "2019-10-28T14:29:05.297261Z",
-            "date_released": "2019-10-28T14:29:05.297273Z",
-            "description": "OAEI Conference 2019",
-            "documentation": "Yes",
+            "date_created": "2017-04-12T20:14:30Z",
+            "date_released": "2017-04-12T00:00:00Z",
+            "description": "A vocabulary for buffalo, cattle, chicken, goat, horse, pig, and sheep breeds.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/cmt/1/cmt.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://gite.lirmm.fr/opendata/yampp-online/raw/4606498a7c3aa8f3704898a4cc3f513c7a79ba2f/cmt.owl",
-            "id": 3036,
+            "download_url": "media/ontologies/LBO/9/lbo.obo",
+            "has_ontology_language": "OBO",
+            "home_page": null,
+            "id": 611,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 4,
+            "max_children": 291,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 30,
+            "nb_classes": 1064,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 49,
+            "nb_properties": 0,
             "products": null,
-            "publication": "Yes",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 9,
             "taxon": null,
-            "version": "1.0"
+            "version": "2.2"
         },
-        "topics": [
-            "cmt"
-        ]
+        "topics": null
     },
     {
-        "acronym": "CN",
-        "name": "computer network",
+        "acronym": "BIRNLEX",
+        "name": "Biomedical Informatics Research Network Project Lexicon",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2014-09-25T12:09:00Z",
-            "date_released": "2014-09-18T00:00:00Z",
-            "description": "Computer networks ontology ",
+            "date_created": "2008-03-25T00:39:39Z",
+            "date_released": "2007-08-08T07:00:00Z",
+            "description": "The BIRN Project lexicon will provide entities for data and database annotation for the BIRN project, covering anatomy, disease, data collection, project management and experimental design.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CN/1/cn.owl",
+            "download_url": "media/ontologies/BIRNLEX/1/birnlex.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 540,
+            "home_page": "http://xwiki.nbirn.net:8080/xwiki/bin/view/%20BIRN-OTF-Public/Home",
+            "id": 34,
             "indexed": true,
-            "max_children": 34,
-            "max_depth": 8,
+            "max_children": 66,
+            "max_depth": 40,
             "md5sum": null,
-            "nb_classes": 537,
+            "nb_classes": 3580,
             "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 38,
+            "nb_individuals": 0,
+            "nb_properties": 9,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.3.1"
         },
         "topics": null
     },
     {
-        "acronym": "INTVEETYPE",
-        "name": "Interviewee Type",
+        "acronym": "COHSI2STUDY",
+        "name": "cohsi2study",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-12-07T19:53:27Z",
-            "date_released": "2022-12-07T00:00:00Z",
-            "description": "This class refers to the different types of Interviewees.",
+            "date_created": "2022-09-12T05:59:40Z",
+            "date_released": "2021-11-26T00:00:00Z",
+            "description": "These ontologies are created under the CoHSI2 Study.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INTVEETYPE/1/intveetype.skos",
+            "download_url": "media/ontologies/COHSI2STUDY/27/cohsi2study.skos",
             "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 5549,
+            "id": 5352,
             "indexed": true,
             "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
             "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 6,
+            "nb_individuals": 83,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 27,
             "taxon": null,
-            "version": null
+            "version": "0.2.8"
         },
         "topics": null
     },
     {
-        "acronym": "HMSG",
-        "name": "Hack Mudah Slot Games",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "IPD",
-        "name": "injury process description ontology",
+        "acronym": "ILLNESSINJURY",
+        "name": "Illness and Injury",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 10,
             "classifiable": true,
-            "date_created": "2023-01-22T10:09:37Z",
-            "date_released": "2023-01-22T00:00:00Z",
-            "description": "IPD-Onto divides product injury  into five unified classes (host, vector, agent, environment, and consequences) and constructs the accident process through the object properties (lead to, touch, accident, etc.)IPD-Onto is used to realise the transformation and reorganisation from product injury to injury information, to reproduce the process of accident occurrence, and realise human-computer interactions to meet the needs of data query.",
+            "date_created": "2018-07-22T21:52:48Z",
+            "date_released": "2018-07-22T00:00:00Z",
+            "description": "This list of Illness and Injury (Morbidity and Mortality) classifications is an abbreviated version of the World Health Organization's (WHO) Startup Mortality List (ICD-10-SMoL).  ICD-10-SMoL is the WHO's application of ICD-10 for low-resource settings initial cause of death collection.\r\n\r\nICD-10-SMoL was chosen because it is a simplified version of the ICD-10.\r\nICD (International Classification of Diseases) \"is the foundation for the identification of health trends and statistics globally, and the international standard for reporting diseases and health conditions. It is the diagnostic classification standard for all clinical and research purposes. ICD defines the universe of diseases, disorders, injuries and other related health conditions\" [http://www.who.int/classifications/icd/en/].\r\n\r\nVersion 1.0 of this ontology was compiled for the Canadian Writing Research Collaboratory (CWRC) ontology project and reflects the need to have causes of death and health issues organized in a way to allow for analysis of cultural data.  A simplified list was deemed sufficient since the domain under study (cultural documents) will not have detailed medical information with regards to cause of death or health issues.  But instead of accepting a list of general usage terms for causes of death or health issues that appear in some sources (e.g. Wikipedia, Wikidatak, etc.), a standardized, medical classification was selected so that it could be of use for future analysis regarding questions involving the intersection of health and culture.\r\n\r\nICD-10-SMoL was chosen since it was the latest version of this list (June, 2018).  A stable version of ICD-11 was to be available in June 2018 but there was no ICD-11-SMoL available at this time.\r\n\r\nTo obtain the ICD10-SMoL:\r\nhttp://www.who.int/healthinfo/civil_registration/ICD_10_SMoL.pdf?ua=1",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IPD/2/ipd.owl",
+            "download_url": "media/ontologies/ILLNESSINJURY/1/illnessinjury.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5625,
+            "id": 1670,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 3,
+            "max_children": 20,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 8,
+            "nb_classes": 21,
             "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 20,
+            "nb_individuals": 141,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "LABO",
-        "name": "clinical LABoratory Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 20,
-            "classifiable": true,
-            "date_created": "2022-12-12T12:33:18.402360Z",
-            "date_released": "2022-12-12T12:33:18.402343Z",
-            "description": "LABO is an ontology of informational entities formalizing clinical laboratory tests prescriptions and reporting documents.",
-            "documentation": null,
-            "domain": "information",
-            "download_url": "media/ontologies/LABO/3/labo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/OpenLHS/LABO",
-            "id": 5535,
-            "indexed": true,
-            "max_children": 31,
-            "max_depth": 9,
-            "md5sum": "b29d5c3eccf778158c823b80df36a782",
-            "nb_classes": 169,
-            "nb_inconsistent": 31,
-            "nb_individuals": 0,
-            "nb_properties": 33,
-            "products": [
-                {
-                    "id": "labo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/labo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://doi.org/10.5281/zenodo.6522019",
-                    "title": "LABO: An Ontology for Laboratory Test Prescription and Reporting"
-                }
-            ],
-            "submission_id": 3,
-            "taxon": {
-                "id": "NCBITaxon:9606",
-                "label": "Homo sapiens"
-            },
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "LC-MEDIA",
-        "name": "Library of Congress Media Types Scheme",
+        "acronym": "LC-CARRIERS",
+        "name": "Library of Congress Carriers Scheme",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2020-07-03T01:05:13Z",
-            "date_released": "2020-07-03T01:05:12Z",
-            "description": "Media Types Scheme is derived from a controlled list of values reflecting the general type of intermediation device required to view, play, run, etc., the content of a resource, which is principally used in RDA cataloging.",
+            "date_created": "2018-09-09T23:13:05Z",
+            "date_released": "2018-05-07T00:00:00Z",
+            "description": "Carriers Scheme is derived from a controlled list of values representing the format of the storage medium and its media type, which is principally used in RDA cataloging.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LC-MEDIA/17/lc-media.skos",
+            "download_url": "media/ontologies/LC-CARRIERS/3/lc-carriers.skos",
             "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/vocabulary/mediaTypes.html",
-            "id": 3749,
+            "home_page": "http://id.loc.gov/vocabulary/carriers.html",
+            "id": 1819,
             "indexed": true,
             "max_children": 4,
             "max_depth": 1,
             "md5sum": null,
             "nb_classes": 6,
             "nb_inconsistent": 0,
-            "nb_individuals": 11,
+            "nb_individuals": 56,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 17,
+            "submission_id": 3,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LCMPT",
-        "name": "Library of Congress Medium of Performance Thesaurus for Music",
+        "acronym": "HMSG",
+        "name": "Hack Mudah Slot Games",
         "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 10,
-            "classifiable": true,
-            "date_created": "2017-09-06T18:52:09Z",
-            "date_released": "2017-08-31T00:00:00Z",
-            "description": "Vocabulary that provides terminology to describe the instruments, voices, etc., used in the performance of musical works.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/LCMPT/3/lcmpt.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/authorities/performanceMediums.html",
-            "id": 747,
-            "indexed": true,
-            "max_children": 10,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 10,
-            "nb_inconsistent": 0,
-            "nb_individuals": 849,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "31 Aug 2017"
-        },
+        "status": "Unknown",
+        "submission": null,
         "topics": null
     },
     {
-        "acronym": "ILLNESSINJURY",
-        "name": "Illness and Injury",
+        "acronym": "LC-CONTENT",
+        "name": "Library of Congress Content Types Scheme",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 10,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2018-07-22T21:52:48Z",
-            "date_released": "2018-07-22T00:00:00Z",
-            "description": "This list of Illness and Injury (Morbidity and Mortality) classifications is an abbreviated version of the World Health Organization's (WHO) Startup Mortality List (ICD-10-SMoL).  ICD-10-SMoL is the WHO's application of ICD-10 for low-resource settings initial cause of death collection.\r\n\r\nICD-10-SMoL was chosen because it is a simplified version of the ICD-10.\r\nICD (International Classification of Diseases) \"is the foundation for the identification of health trends and statistics globally, and the international standard for reporting diseases and health conditions. It is the diagnostic classification standard for all clinical and research purposes. ICD defines the universe of diseases, disorders, injuries and other related health conditions\" [http://www.who.int/classifications/icd/en/].\r\n\r\nVersion 1.0 of this ontology was compiled for the Canadian Writing Research Collaboratory (CWRC) ontology project and reflects the need to have causes of death and health issues organized in a way to allow for analysis of cultural data.  A simplified list was deemed sufficient since the domain under study (cultural documents) will not have detailed medical information with regards to cause of death or health issues.  But instead of accepting a list of general usage terms for causes of death or health issues that appear in some sources (e.g. Wikipedia, Wikidatak, etc.), a standardized, medical classification was selected so that it could be of use for future analysis regarding questions involving the intersection of health and culture.\r\n\r\nICD-10-SMoL was chosen since it was the latest version of this list (June, 2018).  A stable version of ICD-11 was to be available in June 2018 but there was no ICD-11-SMoL available at this time.\r\n\r\nTo obtain the ICD10-SMoL:\r\nhttp://www.who.int/healthinfo/civil_registration/ICD_10_SMoL.pdf?ua=1",
+            "date_created": "2021-04-19T01:07:20Z",
+            "date_released": "2021-04-19T01:07:18Z",
+            "description": null,
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ILLNESSINJURY/1/illnessinjury.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 1670,
+            "download_url": "media/ontologies/LC-CONTENT/157/lc-content.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://id.loc.gov/vocabulary/contentTypes.html",
+            "id": 4416,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 2,
+            "max_children": 4,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 21,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 141,
+            "nb_individuals": 26,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "INBIO",
-        "name": "Invasion Biology Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 10,
-            "classifiable": true,
-            "date_created": "2023-02-02T10:32:47Z",
-            "date_released": "2023-02-02T00:00:00Z",
-            "description": "The Invasion Biology Ontology (INBIO) contains terms and concepts relevant in the field of invasion biology, which is a research area dealing with the translocation, establishment, spread, impact and management of species outside of their native ranges, where they are called non-native or alien species. This first version of the ontology covers terms and concepts needed to describe twelve major invasion hypotheses building the hierarchical hypothesis network (see also Jeschke JM, Heger T (Eds) (2018) Invasion Biology: Hypotheses and Evidence. CABI, Wallingford, UK).",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/INBIO/2/inbio.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5644,
-            "indexed": true,
-            "max_children": 17,
-            "max_depth": 30,
-            "md5sum": null,
-            "nb_classes": 458,
-            "nb_inconsistent": 17,
-            "nb_individuals": 18,
-            "nb_properties": 92,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
+            "submission_id": 157,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "COB",
-        "name": "Core Ontology for Biology and Biomedicine",
+        "acronym": "CMO",
+        "name": "Clinical measurement ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-05-15T12:22:54.016356Z",
-            "date_released": "2023-05-15T12:22:54.016340Z",
-            "description": "COB brings together key terms from a wide range of OBO projects to improve interoperability.",
+            "date_created": "2023-07-31T12:16:43.961454Z",
+            "date_released": "2023-07-31T12:16:43.961439Z",
+            "description": "Morphological and physiological measurement records generated from clinical and model organism research and health programs.",
             "documentation": null,
-            "domain": "upper",
-            "download_url": "media/ontologies/COB/10/cob.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/CMO/94/cmo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://obofoundry.org/COB/",
-            "id": 5807,
+            "home_page": "http://rgd.mcw.edu/rgdweb/ontology/search.html",
+            "id": 5967,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 4,
-            "md5sum": "d8416e75f8f27bf27a032cbcc7836be4",
-            "nb_classes": 74,
+            "max_children": 111,
+            "max_depth": 11,
+            "md5sum": "9a46976c49fa2da406b4725907815f82",
+            "nb_classes": 3927,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 23,
+            "nb_properties": 1,
             "products": [
                 {
-                    "description": "Core Ontology for Biology and Biomedicine, main ontology",
-                    "id": "cob.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cob.owl",
-                    "title": "COB"
+                    "id": "cmo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cmo.owl"
                 },
                 {
-                    "description": "base module for COB",
-                    "id": "cob/cob-base.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cob/cob-base.owl",
-                    "title": "COB base module"
-                },
+                    "id": "cmo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cmo.obo"
+                }
+            ],
+            "publication": null,
+            "publications": [
                 {
-                    "id": "cob/cob-to-external.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cob/cob-to-external.owl",
-                    "title": "COB to external",
-                    "type": "BridgeOntology"
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/22654893",
+                    "title": "Three ontologies to define phenotype measurement data."
                 },
                 {
-                    "description": "demo of COB including subsets of other ontologies (Experimental, for demo purposes only)",
-                    "id": "cob/products/demo-cob.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cob/products/demo-cob.owl",
-                    "status": "alpha",
-                    "title": "COB demo ontology (experimental)"
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24103152",
+                    "title": "The clinical measurement, measurement method and experimental condition ontologies: expansion, improvements and new applications."
                 }
             ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 10,
+            "submission_id": 94,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HECON",
-        "name": "HECON: Health Condition Evolution Ontology",
-        "species": null,
+        "acronym": "gr",
+        "name": "gr_ontology",
+        "species": [],
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-04-22T01:59:19Z",
-            "date_released": "2022-04-22T00:00:00Z",
-            "description": "Health Condition Evolution Ontology (HECON), is a formal model representation of the evolution of medical events over time.",
+            "date_created": "2022-08-12T09:40:35.502078Z",
+            "date_released": "2022-08-12T09:40:35.502091Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HECON/1/hecon.owl",
+            "download_url": "media/ontologies/gr/1/gr.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://kmi.open.ac.uk/conrad/HECON",
-            "id": 5051,
+            "home_page": null,
+            "id": 5276,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 3,
+            "max_children": 12,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 16,
+            "nb_classes": 105,
             "nb_inconsistent": 0,
-            "nb_individuals": 21,
-            "nb_properties": 10,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.1"
+            "version": "juhiygh"
         },
-        "topics": null
+        "topics": []
     },
     {
-        "acronym": "HEIO",
-        "name": "Regional Healthcare System Interoperability and Information Exchange Measurement Ontology",
+        "acronym": "CMDO",
+        "name": "Clinical MetaData Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2015-04-17T14:27:05Z",
-            "date_released": "2015-04-09T00:00:00Z",
-            "description": "The purpose of this ontology is to develop a model for regional healthcare system interoperability and information exchange quantification (measured by the electronic health information exchange (eHIE) indicator). The eHIE is hypothesized as a leading measure of regional healthcare system integration.\r\nA publication associated with this ontology is currently under review. ",
+            "date_created": "2019-07-12T09:44:05Z",
+            "date_released": "2019-02-26T00:00:00Z",
+            "description": "The Clinical MetaData Ontology is designed to be used to classify Common Data Elements (CDEs) from different studies and clinical documents. CDEs can be efficiently searched, shared and reused by using our ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HEIO/17/heio.owl",
+            "download_url": "media/ontologies/CMDO/10/cmdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 17,
+            "home_page": "http://www.snubi.org/software/cmo/",
+            "id": 2743,
             "indexed": true,
-            "max_children": 36,
-            "max_depth": 3,
+            "max_children": 20,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 125,
+            "nb_classes": 188,
             "nb_inconsistent": 0,
-            "nb_individuals": 1605,
-            "nb_properties": 47,
+            "nb_individuals": 151,
+            "nb_properties": 3,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 17,
+            "submission_id": 10,
             "taxon": null,
-            "version": "Version 1.15"
+            "version": "1.1.4"
         },
         "topics": null
     },
     {
-        "acronym": "HELIFIT",
-        "name": "HeLiFit-Ontology",
+        "acronym": "CMPO",
+        "name": "Cellular microscopy phenotype ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 11,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-12-13T11:48:34Z",
-            "date_released": "2022-12-13T00:00:00Z",
-            "description": "This an ontology for healthcare.",
-            "documentation": "chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/http://wrap.warwick.ac.uk/162747/1/WRAP-toward-symbolic-AI-approach-WHO-ACSM-physical-activity-sedentary-behavior-guidelines-Pecchia-2022.pdf",
+            "date_created": "2015-06-25T01:01:45Z",
+            "date_released": "2015-06-25T01:01:45Z",
+            "description": "The Cellular Microscopy Phenotype Ontology (CMPO) is a species neutral ontology for describing general phenotypic observations relating to the whole cell, cellular components, cellular processes and cell populations. CMPO is an application ontology developed in OWL that contains precomposed phenotypes descriptions that are defined using the Gene Ontology (GO) and the Phenotype Trait Ontology (PATO).",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HELIFIT/1/helifit.owl",
+            "download_url": "media/ontologies/CMPO/14/cmpo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5857,
+            "home_page": "https://github.com/EBISPOT/CMPO",
+            "id": 484,
             "indexed": true,
-            "max_children": 111,
-            "max_depth": 9,
+            "max_children": 67,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 487,
+            "nb_classes": 813,
             "nb_inconsistent": 0,
-            "nb_individuals": 152,
+            "nb_individuals": 0,
             "nb_properties": 30,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 14,
             "taxon": null,
-            "version": ""
+            "version": "1.3"
         },
         "topics": null
     },
     {
-        "acronym": "HGNC-NR",
-        "name": "HGNC New Releases Ontology",
-        "species": null,
+        "acronym": "cmt",
+        "name": "cmt",
+        "species": [
+            "cmt"
+        ],
         "status": "Classified",
         "submission": {
-            "avg_children": 6981,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-20T19:11:07Z",
-            "date_released": "2023-06-20T00:00:00Z",
-            "description": "HUGO Gene Nomenclature Committee Ontology with recurring uploads",
-            "documentation": null,
+            "date_created": "2019-10-28T14:29:05.297261Z",
+            "date_released": "2019-10-28T14:29:05.297273Z",
+            "description": "OAEI Conference 2019",
+            "documentation": "Yes",
             "domain": null,
-            "download_url": "media/ontologies/HGNC-NR/7/hgnc-nr.owl",
+            "download_url": "media/ontologies/cmt/1/cmt.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5898,
+            "home_page": "https://gite.lirmm.fr/opendata/yampp-online/raw/4606498a7c3aa8f3704898a4cc3f513c7a79ba2f/cmt.owl",
+            "id": 3036,
             "indexed": true,
-            "max_children": 19270,
-            "max_depth": 3,
+            "max_children": 8,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 48867,
+            "nb_classes": 30,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 49,
             "products": null,
-            "publication": null,
+            "publication": "Yes",
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2023-06-20"
+            "version": "1.0"
         },
-        "topics": null
+        "topics": [
+            "cmt"
+        ]
     },
     {
-        "acronym": "HIO",
-        "name": "Hearing Impairment Ontology",
+        "acronym": "INTVEETYPE",
+        "name": "Interviewee Type",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-11-01T01:10:10Z",
-            "date_released": "2019-11-01T01:10:08Z",
-            "description": "The Hearing Impairment Ontology (HIO) is a common controlled hearing impairment (HI) vocabulary, which is expected to enhance collaborative research. This ontology is currently the most comprehensive and standardized human- and machine-readable resource that unambiguously defines HI concepts and terminology for researchers, patients and clinicians in order to help process, reuse and re-apply existing HI knowledge in biomedical research and health-care systems. In the context of big data analytics, this ontology may facilitate a retrospective data harmonization and contribute to mapping HI datasets to functional knowledge to enable the subsequent HI research translation into clinical applications and policy guidelines. The HIO will allow researchers, clinicians and patients to readily access standardized HI-related knowledge in a single location and promote HI data integration, interoperability and sharing, including epidemiological, socio-environmental, biomedical, genetic and phenotypic datasets.",
+            "date_created": "2022-12-07T19:53:27Z",
+            "date_released": "2022-12-07T00:00:00Z",
+            "description": "This class refers to the different types of Interviewees.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HIO/3/hio.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/INTVEETYPE/1/intveetype.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 3067,
+            "id": 5549,
             "indexed": true,
-            "max_children": 26,
-            "max_depth": 8,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 496,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 46,
+            "nb_individuals": 6,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "HFO",
         "name": "Heart Failure Ontology",
         "species": null,
@@ -14062,511 +13892,511 @@
             "submission_id": 1,
             "taxon": null,
             "version": "version 1.0"
         },
         "topics": null
     },
     {
-        "acronym": "SERVICE",
-        "name": "Rafter Loft Conversions Ltd",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "RGD",
-        "name": "RRB Group D",
-        "species": null,
-        "status": "Unknown",
-        "submission": null,
-        "topics": null
-    },
-    {
-        "acronym": "HGNC",
-        "name": "HUGO Gene Nomenclature",
+        "acronym": "MAT_TEST_CASE2",
+        "name": "Maternity_Test_Case2",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 866,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-02-21T00:56:52Z",
-            "date_released": "2010-01-07T00:00:00Z",
-            "description": "Human Genome Organisation (HUGO) Gene Nomenclature Committee at the European Bioinformatics Institute approves a gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database.",
-            "documentation": "http://www.genenames.org/",
+            "date_created": "2021-02-20T16:21:55Z",
+            "date_released": "2021-02-20T00:00:00Z",
+            "description": "Testing",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HGNC/1/hgnc.owl",
+            "download_url": "media/ontologies/MAT_TEST_CASE2/1/mat_test_case2.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.genenames.org/",
-            "id": 1261,
+            "home_page": null,
+            "id": 4296,
             "indexed": true,
-            "max_children": 19361,
+            "max_children": 22,
             "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 32918,
+            "nb_classes": 56,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 8,
             "products": null,
-            "publication": "http://www.genenames.org/",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "July2010"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MACROALGAETRAITS",
-        "name": "Macroalgae Traits Thesaurus",
+        "acronym": "HCPCS",
+        "name": "Healthcare Common Procedure Coding System",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2023-04-10T19:33:48Z",
-            "date_released": "2023-04-10T00:00:00Z",
-            "description": "The Macroalgae Traits Thesaurus contains several concepts on demographic and functional traits. It has been developed and published by LifeWatch Italy, the Italian node of the e-science European infrastructure for biodiversity and ecosystem research (LifeWatch ERIC). It reflects the agreement of a scientific expert community to fix semantic properties (e.g. label, definition) of approximately 100 traits.",
-            "documentation": "http://www.lifewatchitaly.eu/web/lifewatch-italia/reports",
+            "date_created": "2017-02-06T21:25:59Z",
+            "date_released": "2017-02-06T21:25:59Z",
+            "description": "Healthcare Common Procedure Coding System",
+            "documentation": "http://cms.hhs.gov",
             "domain": null,
-            "download_url": "media/ontologies/MACROALGAETRAITS/1/macroalgaetraits.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://thesauri.lifewatchitaly.eu/Macroalgae/index.php",
-            "id": 5763,
+            "download_url": "media/ontologies/HCPCS/9/hcpcs.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://cms.hhs.gov",
+            "id": 331,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 5946,
             "nb_inconsistent": 0,
-            "nb_individuals": 127,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 20,
             "products": null,
-            "publication": null,
+            "publication": "http://cms.hhs.gov",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 9,
             "taxon": null,
-            "version": null
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "MAT",
-        "name": "Minimal Anatomical Terminology",
+        "acronym": "MCBCC",
+        "name": "Breast Tissue Cell Lines Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 153,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2015-08-21T01:06:11Z",
-            "date_released": "2015-08-21T01:06:11Z",
-            "description": "Minimal set of terms for anatomy",
+            "date_created": "2010-07-30T03:59:39Z",
+            "date_released": "2010-07-30T07:00:00Z",
+            "description": "Contains a comprehensive list of cell lines derived from breast tissue, both normal and pathological. The ontology in built in OWL with cross relation to classes- genetic variation, pathological condition, genes, chemicals and drugs. The relations built enable semantic query across different classes",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MAT/3/mat.obo",
-            "has_ontology_language": "OBO",
-            "home_page": null,
-            "id": 189,
+            "download_url": "media/ontologies/MCBCC/2/mcbcc.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "www.molecularconnections.com",
+            "id": 457,
             "indexed": true,
-            "max_children": 457,
-            "max_depth": 1,
+            "max_children": 153,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 2166,
+            "nb_classes": 385,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 19,
+            "nb_properties": 27,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "MAT_TEST_CASE2",
-        "name": "Maternity_Test_Case2",
+        "acronym": "HNS",
+        "name": "HNS_Ontolgoy",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-02-20T16:21:55Z",
-            "date_released": "2021-02-20T00:00:00Z",
-            "description": "Testing",
-            "documentation": null,
+            "date_created": "2020-05-05T19:45:08Z",
+            "date_released": "2020-05-05T00:00:00Z",
+            "description": "The purpose of this research is to develop an ontology with subsequent testing and evaluation, for identifying utility and value. The domain that has been chosen is human nervous system (HNS) disorders. It is hypothesized here that an ontology-based patient records management system is more effective in meeting and addressing complex information needs of health-care personnel. Therefore, this study has been based on the premise that developing an ontology and using it as a component of the search interface in hospital records management systems will lead to more efficient and effective management of health-care. It is proposed here to develop an ontology of the domain of HNS disorders using a standard vocabulary such as MeSH or SNOMED CT. The principal classes of an ontology include facet analysis for arranging concepts based on their common characteristics to build mutually exclusive classes. We combine faceted theory with description logic, which helps us to better query and retrieve data by implementing an ontological model. Prot\u00e9g\u00e9 5.2.0 was used as ontology editor. The use of ontologies for domain modelling will be of acute help to doctors for searching patient records. In this paper we show how the faceted approach helps us to build a flexible model and retrieve better information. We use the medical domain as a case study to show examples and implementation.",
+            "documentation": "https://github.com/subhashishhh/HNS-Ontology",
             "domain": null,
-            "download_url": "media/ontologies/MAT_TEST_CASE2/1/mat_test_case2.owl",
+            "download_url": "media/ontologies/HNS/1/hns.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4296,
+            "home_page": "https://github.com/subhashishhh/HNS-Ontology/blob/master/HNS_ontology.owl",
+            "id": 3609,
             "indexed": true,
-            "max_children": 22,
-            "max_depth": 3,
+            "max_children": 28,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 56,
+            "nb_classes": 546,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 8,
+            "nb_individuals": 142,
+            "nb_properties": 15,
             "products": null,
-            "publication": null,
+            "publication": "https://www.ergon-verlag.de/isko_ko/downloads/ko_46_2019_3_c.pdf",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "4.0"
         },
         "topics": null
     },
     {
-        "acronym": "MAXO",
-        "name": "Medical Action Ontology",
+        "acronym": "HOOM",
+        "name": "HPO - ORDO Ontological Module",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 16598,
             "classifiable": true,
-            "date_created": "2023-06-12T14:17:30.870326Z",
-            "date_released": "2023-06-12T14:17:30.870310Z",
-            "description": "The Medical Action Ontology (MAxO) provides a broad view of medical actions and includes terms for medical procedures, interventions, therapies, treatments, and recommendations.",
+            "date_created": "2022-12-08T02:17:20Z",
+            "date_released": "2022-12-08T02:16:38Z",
+            "description": "Orphanet provides phenotypic annotations of the rare diseases in the Orphanet nomenclature using the Human Phenotype Ontology (HPO). HOOM is a module that qualifies the annotation between a clinical entity and phenotypic abnormalities according to a frequency and by integrating the notion of diagnostic criterion. In ORDO a clinical entity is either a group of rare disorders, a rare disorder or a subtype of disorder. The phenomes branch of ORDO has been refactored as a logical import of HPO, and the HPO-ORDO phenotype disease-annotations have been provided in a series of triples in OBAN format in which associations, frequency and provenance are modeled.\r\n\r\nHOOM is provided as an OWL (Ontologies Web Languages) file, using OBAN, the Orphanet Rare Disease Ontology (ORDO), and HPO ontological models.\r\n\r\nHOOM provides extra possibilities for researchers, pharmaceutical companies and others wishing to co-analyse rare and common disease phenotype associations, or re-use the integrated ontologies in genomic variants repositories or match-making tools. ",
+            "documentation": "https://www.orphadata.com/hoom/",
+            "domain": null,
+            "download_url": "media/ontologies/HOOM/7/hoom.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.orphadata.com",
+            "id": 5540,
+            "indexed": true,
+            "max_children": 114800,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 132786,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1,
+            "nb_properties": 12,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 7,
+            "taxon": null,
+            "version": "1.9"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "SERVICE",
+        "name": "Rafter Loft Conversions Ltd",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "RGD",
+        "name": "RRB Group D",
+        "species": null,
+        "status": "Unknown",
+        "submission": null,
+        "topics": null
+    },
+    {
+        "acronym": "MCCL",
+        "name": "Cell Line Ontology [by Mahadevan]",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2014-08-05T04:29:27Z",
+            "date_released": "2014-08-05T00:00:00Z",
+            "description": "Ontologies provide a common platform of controlled vocabulary for researchers who need to share information across a particular domain, inclusive of machine-interpretable definitions of basic concepts in the domain and relations among them. The need for ontologies that provide a systematic arrangement of available data and allow cross talk across various related domains is gaining momentum. In this backdrop, we have developed a comprehensive ontology on primary and established cell lines-both normal and pathologic. It covers around 400 cell lines. This ontology has been built to include the major domains in the field of biology like anatomy, bio-molecules, chemicals and drugs, pathological conditions and genetic variations around the cell lines. An extensive network of relations has been built across these concepts to enable different combinations of queries. The ontology covers all cell lines from major sources like ATCC, DSMZ, ECACC, ICLC etc. and is built in OWL format.",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/MAXO/38/maxo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/MCCL/2/mccl.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/monarch-initiative/MAxO",
-            "id": 5861,
+            "home_page": "http://www.molecularconnections.com",
+            "id": 328,
             "indexed": true,
-            "max_children": 119,
-            "max_depth": 34,
-            "md5sum": "4848936305ffeadec05a3bf9a309128d",
-            "nb_classes": 13851,
+            "max_children": 2155,
+            "max_depth": 9,
+            "md5sum": null,
+            "nb_classes": 5475,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 286,
-            "products": [
-                {
-                    "id": "maxo.owl",
-                    "name": "Medical Action Ontology main release in OWL format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo.owl"
-                },
-                {
-                    "id": "maxo.obo",
-                    "name": "Medical Action Ontology additional release in OBO format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo.obo"
-                },
-                {
-                    "id": "maxo.json",
-                    "name": "Medical Action Ontology additional release in OBOJSon format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo.json"
-                },
-                {
-                    "id": "maxo/maxo-base.owl",
-                    "name": "Medical Action Ontology main release in OWL format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo/maxo-base.owl"
-                },
-                {
-                    "id": "maxo/maxo-base.obo",
-                    "name": "Medical Action Ontology additional release in OBO format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo/maxo-base.obo"
-                },
-                {
-                    "id": "maxo/maxo-base.json",
-                    "name": "Medical Action Ontology additional release in OBOJSon format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo/maxo-base.json"
-                }
-            ],
+            "nb_individuals": 0,
+            "nb_properties": 17,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 38,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "MCHVODANATERMS",
-        "name": "MCHVODANATERMS",
+        "acronym": "LUNGMAP_H_CELL",
+        "name": "Cell Ontology for Human Lung Maturation",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2022-02-10T18:12:22Z",
-            "date_released": "2022-02-10T00:00:00Z",
-            "description": "Ontology containing the controlled terms and semantic properties used in the Maternal Child Health Care generic template used within different facilities in VODANA ",
+            "date_created": "2019-07-28T22:06:22Z",
+            "date_released": "2019-07-28T00:00:00Z",
+            "description": "Cell Ontology for Human Lung Maturation (LungMAP Human Anatomy (LMHA))\r\n\r\nDeveloped by Susan E Wert, Gail H. Deutsch, Helen Pan, and the NHLBI Molecular Atlas of Lung Development Program Consortium Ontology Subcommittee",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MCHVODANATERMS/5/mchvodanaterms.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4904,
+            "download_url": "media/ontologies/LUNGMAP_H_CELL/6/lungmap_h_cell.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.lungmap.net/breath-ontology-browser/",
+            "id": 2789,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 10,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 80,
             "nb_inconsistent": 0,
-            "nb_individuals": 27,
-            "nb_properties": 9,
+            "nb_individuals": 1,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 6,
             "taxon": null,
-            "version": "3.0"
+            "version": "3.2"
         },
         "topics": null
     },
     {
-        "acronym": "HMISLAB0115",
-        "name": "HMIS LAB 011 5",
+        "acronym": "MCCV",
+        "name": "Microbial Culture Collection Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2021-11-12T15:01:37Z",
-            "date_released": "2021-11-12T00:00:00Z",
-            "description": "Assignment for group 5 of DSIP",
+            "date_created": "2016-03-14T09:09:37Z",
+            "date_released": "2015-04-08T00:00:00Z",
+            "description": "Structured controlled vocabulary for describing meta information of microbial calture collection maintained in biological research centers",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HMISLAB0115/1/hmislab0115.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/MCCV/10/mccv.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4693,
+            "id": 275,
             "indexed": true,
-            "max_children": 2,
+            "max_children": 12,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 20,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 59,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 10,
             "taxon": null,
-            "version": null
+            "version": "Version 0.96 beta"
         },
         "topics": null
     },
     {
-        "acronym": "HNS",
-        "name": "HNS_Ontolgoy",
+        "acronym": "HL7",
+        "name": "Health Level Seven Reference Implementation Model, Version 3",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2020-05-05T19:45:08Z",
-            "date_released": "2020-05-05T00:00:00Z",
-            "description": "The purpose of this research is to develop an ontology with subsequent testing and evaluation, for identifying utility and value. The domain that has been chosen is human nervous system (HNS) disorders. It is hypothesized here that an ontology-based patient records management system is more effective in meeting and addressing complex information needs of health-care personnel. Therefore, this study has been based on the premise that developing an ontology and using it as a component of the search interface in hospital records management systems will lead to more efficient and effective management of health-care. It is proposed here to develop an ontology of the domain of HNS disorders using a standard vocabulary such as MeSH or SNOMED CT. The principal classes of an ontology include facet analysis for arranging concepts based on their common characteristics to build mutually exclusive classes. We combine faceted theory with description logic, which helps us to better query and retrieve data by implementing an ontological model. Prot\u00e9g\u00e9 5.2.0 was used as ontology editor. The use of ontologies for domain modelling will be of acute help to doctors for searching patient records. In this paper we show how the faceted approach helps us to build a flexible model and retrieve better information. We use the medical domain as a case study to show examples and implementation.",
-            "documentation": "https://github.com/subhashishhh/HNS-Ontology",
+            "date_created": "2017-02-06T21:26:43Z",
+            "date_released": "2017-02-06T21:26:43Z",
+            "description": "This version is the first update to Normative RIM, Release 3. It is based on changes approved in Harmonization in November 2010. This release of the RIM is bound to HL7 Abstract Data Types Release 2.",
+            "documentation": "http://www.hl7.org",
             "domain": null,
-            "download_url": "media/ontologies/HNS/1/hns.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/subhashishhh/HNS-Ontology/blob/master/HNS_ontology.owl",
-            "id": 3609,
+            "download_url": "media/ontologies/HL7/9/hl7.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.hl7.org",
+            "id": 80,
             "indexed": true,
-            "max_children": 28,
-            "max_depth": 7,
+            "max_children": 337,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 546,
+            "nb_classes": 10235,
             "nb_inconsistent": 0,
-            "nb_individuals": 142,
-            "nb_properties": 15,
+            "nb_individuals": 0,
+            "nb_properties": 63,
             "products": null,
-            "publication": "https://www.ergon-verlag.de/isko_ko/downloads/ko_46_2019_3_c.pdf",
+            "publication": "http://www.hl7.org",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 9,
             "taxon": null,
-            "version": "4.0"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "HO",
-        "name": "Histological Ontology",
+        "acronym": "HOM-UCSF_UCareDispostion",
+        "name": "Health Ontology Mapper - UCSF_UCareDischargeDisposition",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2017-10-06T17:45:42Z",
-            "date_released": "2017-10-06T00:00:00Z",
-            "description": "A Histological Ontology of the Human Cardiovascular System v. 1.1",
+            "date_created": "2010-11-22T02:33:52Z",
+            "date_released": "2010-11-21T08:00:00Z",
+            "description": "All Disposition/Discharge Codes and Values listed in the UCSF- UCare patient source data",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HO/78/ho.owl",
+            "download_url": "media/ontologies/HOM-UCSF_UCareDispostion/1/hom-ucsf_ucaredispostion.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 184,
+            "id": 516,
             "indexed": true,
-            "max_children": 46,
-            "max_depth": 5,
+            "max_children": 1,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 213,
-            "nb_inconsistent": 4,
-            "nb_individuals": 3,
-            "nb_properties": 39,
+            "nb_classes": 1,
+            "nb_inconsistent": 0,
+            "nb_individuals": 150,
+            "nb_properties": 2,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 78,
+            "submission_id": 1,
             "taxon": null,
             "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "LHO",
-        "name": "LiveStockHealthOntology",
+        "acronym": "HCOT",
+        "name": "HealthCare Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-07-02T00:35:45Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "The Live Stock Health Ontology (LHO) is an extension of the DECIDE ontology. LHO follows OBO Foundry guidelines and contains information related to the health and well-being of livestock animal species such as cattle. Furthermore, it represents the explicit structured and standardized framework for specific domain knowledge and data integration of livestock species. During the development process, knowledge can be retrieved from domain experts, and after the retrieving process, the first prototype is designed for reasoning and planning of data related to cattle. Moreover, LHO contains information regarding farms Identifications, their samples, geolocation, breed, diagnostic tests, Pathogen identifications, and Pathogen results for specific bacteria such as Histophilus somni (HS), Mannheimia haemolytica (MH), Pasteurella multocida (PM), Bovine coronavirus (BCV) bacterial mycoplasma and aerobic culture results for the respiratory track in cattle. This information is represented in the form of concepts of classes, subclasses, and their relationship in the form of object and data properties.",
+            "date_created": "2018-12-06T05:21:15Z",
+            "date_released": "2018-12-06T00:00:00Z",
+            "description": "It is a sensor based health care ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LHO/1/lho.owl",
+            "download_url": "media/ontologies/HCOT/1/hcot.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5918,
+            "id": 2073,
             "indexed": true,
-            "max_children": 29,
+            "max_children": 6,
             "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 40,
+            "nb_classes": 49,
             "nb_inconsistent": 0,
-            "nb_individuals": 25,
-            "nb_properties": 22,
+            "nb_individuals": 0,
+            "nb_properties": 8,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.0.0"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TODZ",
-        "name": "Test Ontology",
-        "species": [
-            "universal"
-        ],
-        "status": "Unknown",
-        "submission": null,
-        "topics": [
-            "This is only an ontology for test :)"
-        ]
-    },
-    {
-        "acronym": "MCBCC",
-        "name": "Breast Tissue Cell Lines Ontology",
+        "acronym": "HOM-ICD9",
+        "name": "ICD9 for Health Ontology Mapper",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 109,
             "classifiable": true,
-            "date_created": "2010-07-30T03:59:39Z",
-            "date_released": "2010-07-30T07:00:00Z",
-            "description": "Contains a comprehensive list of cell lines derived from breast tissue, both normal and pathological. The ontology in built in OWL with cross relation to classes- genetic variation, pathological condition, genes, chemicals and drugs. The relations built enable semantic query across different classes",
+            "date_created": "2011-04-26T22:58:45Z",
+            "date_released": "2011-04-26T07:00:00Z",
+            "description": "This is the ICD9 ontology with a different hierarchical view, but based on the same nomenclature and code structure. \r\nUseful for researchers that look for a disease by ICD9 code.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MCBCC/2/mcbcc.owl",
+            "download_url": "media/ontologies/HOM-ICD9/4/hom-icd9.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.molecularconnections.com",
-            "id": 457,
+            "home_page": null,
+            "id": 768,
             "indexed": true,
-            "max_children": 153,
-            "max_depth": 5,
+            "max_children": 947,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 385,
+            "nb_classes": 22202,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 27,
+            "nb_properties": 10,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 4,
             "taxon": null,
             "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "HOOM",
-        "name": "HPO - ORDO Ontological Module",
+        "acronym": "HCDR",
+        "name": "Homelessness and Clinical Data Recording",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 16598,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-12-08T02:17:20Z",
-            "date_released": "2022-12-08T02:16:38Z",
-            "description": "Orphanet provides phenotypic annotations of the rare diseases in the Orphanet nomenclature using the Human Phenotype Ontology (HPO). HOOM is a module that qualifies the annotation between a clinical entity and phenotypic abnormalities according to a frequency and by integrating the notion of diagnostic criterion. In ORDO a clinical entity is either a group of rare disorders, a rare disorder or a subtype of disorder. The phenomes branch of ORDO has been refactored as a logical import of HPO, and the HPO-ORDO phenotype disease-annotations have been provided in a series of triples in OBAN format in which associations, frequency and provenance are modeled.\r\n\r\nHOOM is provided as an OWL (Ontologies Web Languages) file, using OBAN, the Orphanet Rare Disease Ontology (ORDO), and HPO ontological models.\r\n\r\nHOOM provides extra possibilities for researchers, pharmaceutical companies and others wishing to co-analyse rare and common disease phenotype associations, or re-use the integrated ontologies in genomic variants repositories or match-making tools. ",
-            "documentation": "https://www.orphadata.com/hoom/",
+            "date_created": "2020-01-21T21:56:18Z",
+            "date_released": "2020-01-21T00:00:00Z",
+            "description": "This ontology utilizes the European Typology of Homelessness and Housing Exclusion (ETHOS), a respected conceptual framework used to define degrees of housing insecurity and homelessness internationally, to conceptualize, define, and link homelessness in SNOMED, Read CTV3, and Read 2 codes to clinical data extraction. The ontology uses the ETHOS as a general framework, leading to endpoints represented by codes from across the three databases. This is useful for researchers and medical professionals seeking to understand how homelessness is represented in clinical data, how homelessness is defined internationally, and what types of services are available for homeless individuals. ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HOOM/7/hoom.owl",
+            "download_url": "media/ontologies/HCDR/1/hcdr.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com",
-            "id": 5540,
+            "home_page": null,
+            "id": 3298,
             "indexed": true,
-            "max_children": 114800,
-            "max_depth": 2,
+            "max_children": 6,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 132786,
+            "nb_classes": 119,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 12,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.9"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "MCCL",
-        "name": "Cell Line Ontology [by Mahadevan]",
+        "acronym": "COLAO",
+        "name": "Coleoptera Anatomy Ontology (COLAO)",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2014-08-05T04:29:27Z",
-            "date_released": "2014-08-05T00:00:00Z",
-            "description": "Ontologies provide a common platform of controlled vocabulary for researchers who need to share information across a particular domain, inclusive of machine-interpretable definitions of basic concepts in the domain and relations among them. The need for ontologies that provide a systematic arrangement of available data and allow cross talk across various related domains is gaining momentum. In this backdrop, we have developed a comprehensive ontology on primary and established cell lines-both normal and pathologic. It covers around 400 cell lines. This ontology has been built to include the major domains in the field of biology like anatomy, bio-molecules, chemicals and drugs, pathological conditions and genetic variations around the cell lines. An extensive network of relations has been built across these concepts to enable different combinations of queries. The ontology covers all cell lines from major sources like ATCC, DSMZ, ECACC, ICLC etc. and is built in OWL format.",
+            "date_created": "2023-04-10T12:17:14.222042Z",
+            "date_released": "2023-04-10T12:17:14.222028Z",
+            "description": "The Coleoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of beetles in biodiversity research. It has been built using the Ontology Develoment Kit, with the Ontology for the Anatomy of the Insect Skeleto-Muscular system (AISM) as a backbone.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MCCL/2/mccl.owl",
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/COLAO/7/colao.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.molecularconnections.com",
-            "id": 328,
+            "home_page": "https://github.com/insect-morphology/colao",
+            "id": 5754,
             "indexed": true,
-            "max_children": 2155,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 5475,
+            "max_children": 96,
+            "max_depth": 7,
+            "md5sum": "74cc86368f3655c9514ac051ec2ecbe0",
+            "nb_classes": 905,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 17,
-            "products": null,
+            "nb_properties": 31,
+            "products": [
+                {
+                    "id": "colao.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/colao.owl"
+                },
+                {
+                    "id": "colao.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/colao.obo"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 7,
             "taxon": null,
-            "version": "2"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "HP",
         "name": "Human Phenotype Ontology",
         "species": null,
@@ -14754,14 +14584,201 @@
                 "label": "Homo sapiens"
             },
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "HECON",
+        "name": "HECON: Health Condition Evolution Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2022-04-22T01:59:19Z",
+            "date_released": "2022-04-22T00:00:00Z",
+            "description": "Health Condition Evolution Ontology (HECON), is a formal model representation of the evolution of medical events over time.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HECON/1/hecon.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://kmi.open.ac.uk/conrad/HECON",
+            "id": 5051,
+            "indexed": true,
+            "max_children": 7,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 16,
+            "nb_inconsistent": 0,
+            "nb_individuals": 21,
+            "nb_properties": 10,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "0.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "HGNC-NR",
+        "name": "HGNC New Releases Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 6981,
+            "classifiable": true,
+            "date_created": "2023-06-20T19:11:07Z",
+            "date_released": "2023-06-20T00:00:00Z",
+            "description": "HUGO Gene Nomenclature Committee Ontology with recurring uploads",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HGNC-NR/7/hgnc-nr.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 5898,
+            "indexed": true,
+            "max_children": 19270,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 48867,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 7,
+            "taxon": null,
+            "version": "2023-06-20"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "HIO",
+        "name": "Hearing Impairment Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2019-11-01T01:10:10Z",
+            "date_released": "2019-11-01T01:10:08Z",
+            "description": "The Hearing Impairment Ontology (HIO) is a common controlled hearing impairment (HI) vocabulary, which is expected to enhance collaborative research. This ontology is currently the most comprehensive and standardized human- and machine-readable resource that unambiguously defines HI concepts and terminology for researchers, patients and clinicians in order to help process, reuse and re-apply existing HI knowledge in biomedical research and health-care systems. In the context of big data analytics, this ontology may facilitate a retrospective data harmonization and contribute to mapping HI datasets to functional knowledge to enable the subsequent HI research translation into clinical applications and policy guidelines. The HIO will allow researchers, clinicians and patients to readily access standardized HI-related knowledge in a single location and promote HI data integration, interoperability and sharing, including epidemiological, socio-environmental, biomedical, genetic and phenotypic datasets.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HIO/3/hio.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3067,
+            "indexed": true,
+            "max_children": 26,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 496,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 46,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TODZ",
+        "name": "Test Ontology",
+        "species": [
+            "universal"
+        ],
+        "status": "Unknown",
+        "submission": null,
+        "topics": [
+            "This is only an ontology for test :)"
+        ]
+    },
+    {
+        "acronym": "HOM",
+        "name": "Homology Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2017-12-13T01:41:29.177764Z",
+            "date_released": "2017-12-13T01:41:29.177732Z",
+            "description": "This ontology represents concepts related to homology, as well as other concepts used to describe similarity and non-homology.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HOM/7/hom.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://bioinfo.unil.ch",
+            "id": 1005,
+            "indexed": true,
+            "max_children": 13,
+            "max_depth": 6,
+            "md5sum": "d3f7a3aabf82ce1f5cfc2112875eeba9",
+            "nb_classes": 66,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": [
+                {
+                    "id": "hom.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/hom.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 7,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "HOME",
+        "name": "Health Ontology for Minority Equity",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 21,
+            "classifiable": true,
+            "date_created": "2022-05-31T18:12:36Z",
+            "date_released": "2022-05-31T00:00:00Z",
+            "description": "An ontology to record the physical, emotional and psychological effects resulting from differences in treatment that citizens receive based on their identity when entering healthcare institutions. We present the first version of such a Health Ontology for Minority Equity (HOME)",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HOME/1/home.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 5124,
+            "indexed": true,
+            "max_children": 29,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 82,
+            "nb_inconsistent": 29,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "LPT",
         "name": "Livestock Product Trait Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -14788,14 +14805,48 @@
             "submission_id": 3,
             "taxon": null,
             "version": "1.4"
         },
         "topics": null
     },
     {
+        "acronym": "HELIFIT",
+        "name": "HeLiFit-Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 11,
+            "classifiable": true,
+            "date_created": "2022-12-13T11:48:34Z",
+            "date_released": "2022-12-13T00:00:00Z",
+            "description": "This an ontology for healthcare.",
+            "documentation": "chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/http://wrap.warwick.ac.uk/162747/1/WRAP-toward-symbolic-AI-approach-WHO-ACSM-physical-activity-sedentary-behavior-guidelines-Pecchia-2022.pdf",
+            "domain": null,
+            "download_url": "media/ontologies/HELIFIT/1/helifit.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 5857,
+            "indexed": true,
+            "max_children": 111,
+            "max_depth": 9,
+            "md5sum": null,
+            "nb_classes": 487,
+            "nb_inconsistent": 0,
+            "nb_individuals": 152,
+            "nb_properties": 30,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": ""
+        },
+        "topics": null
+    },
+    {
         "acronym": "AD-DROP",
         "name": "Alzheimer Disease Relevance Ontology by Process",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -14890,121 +14941,44 @@
             "submission_id": 8,
             "taxon": null,
             "version": "2023-03-21"
         },
         "topics": null
     },
     {
-        "acronym": "LSHO",
-        "name": "LiveStockHealthOntologyCattleUseCase ",
+        "acronym": "HGNC",
+        "name": "HUGO Gene Nomenclature",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 18,
+            "avg_children": 866,
             "classifiable": true,
-            "date_created": "2023-06-25T21:42:39Z",
-            "date_released": "2023-06-25T00:00:00Z",
-            "description": "Ontology is the formal and explicit description of a data model within an application domain in which relationships occur between individuals, making it easy to reuse and knowledge sharing. LSHO contains the information related to the health and well-being of livestock animal species such as cattle. Furthermore, it represents the explicit structured and standardized framework for specific domain knowledge and data integration of livestock species. During the development process, knowledge can be retrieved from domain experts, and after the retrieving process, the first prototype is designed for reasoning and planning of data related to cattle. Moreover, LSHO contains information regarding farms Identifications, their samples, geolocation, breed, diagnostic tests, Pathogen identifications, and Pathogen results for specific bacteria such as Histophilus somni (HS), Mannheimia haemolytica (MH), Pasteurella multocida (PM), Bovine coronavirus (BCV) bacterial mycoplasma and aerobic culture results for the respiratory track . This information is represented in the form of concepts of classes, subclasses, and their relationship in the form of object and data properties.",
-            "documentation": null,
+            "date_created": "2018-02-21T00:56:52Z",
+            "date_released": "2010-01-07T00:00:00Z",
+            "description": "Human Genome Organisation (HUGO) Gene Nomenclature Committee at the European Bioinformatics Institute approves a gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database.",
+            "documentation": "http://www.genenames.org/",
             "domain": null,
-            "download_url": "media/ontologies/LSHO/1/lsho.owl",
+            "download_url": "media/ontologies/HGNC/1/hgnc.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5897,
+            "home_page": "http://www.genenames.org/",
+            "id": 1261,
             "indexed": true,
-            "max_children": 18,
-            "max_depth": 1,
+            "max_children": 19361,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 18,
+            "nb_classes": 32918,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 13,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "LUNGMAP_H_CELL",
-        "name": "Cell Ontology for Human Lung Maturation",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2019-07-28T22:06:22Z",
-            "date_released": "2019-07-28T00:00:00Z",
-            "description": "Cell Ontology for Human Lung Maturation (LungMAP Human Anatomy (LMHA))\r\n\r\nDeveloped by Susan E Wert, Gail H. Deutsch, Helen Pan, and the NHLBI Molecular Atlas of Lung Development Program Consortium Ontology Subcommittee",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/LUNGMAP_H_CELL/6/lungmap_h_cell.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.lungmap.net/breath-ontology-browser/",
-            "id": 2789,
-            "indexed": true,
-            "max_children": 10,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 80,
-            "nb_inconsistent": 0,
-            "nb_individuals": 1,
             "nb_properties": 0,
             "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 6,
-            "taxon": null,
-            "version": "3.2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "COLAO",
-        "name": "Coleoptera Anatomy Ontology (COLAO)",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2023-04-10T12:17:14.222042Z",
-            "date_released": "2023-04-10T12:17:14.222028Z",
-            "description": "The Coleoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of beetles in biodiversity research. It has been built using the Ontology Develoment Kit, with the Ontology for the Anatomy of the Insect Skeleto-Muscular system (AISM) as a backbone.",
-            "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/COLAO/7/colao.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/insect-morphology/colao",
-            "id": 5754,
-            "indexed": true,
-            "max_children": 96,
-            "max_depth": 7,
-            "md5sum": "74cc86368f3655c9514ac051ec2ecbe0",
-            "nb_classes": 905,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 31,
-            "products": [
-                {
-                    "id": "colao.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/colao.owl"
-                },
-                {
-                    "id": "colao.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/colao.obo"
-                }
-            ],
-            "publication": null,
+            "publication": "http://www.genenames.org/",
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "July2010"
         },
         "topics": null
     },
     {
         "acronym": "COLL",
         "name": "Collembola Anatomy Ontology",
         "species": null,
@@ -15062,362 +15036,388 @@
             "nb_classes": 65,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 7,
             "products": null,
             "publication": "http://comodi.sems.uni-rostock.de",
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 6,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "MAXO",
+        "name": "Medical Action Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-06-12T14:17:30.870326Z",
+            "date_released": "2023-06-12T14:17:30.870310Z",
+            "description": "The Medical Action Ontology (MAxO) provides a broad view of medical actions and includes terms for medical procedures, interventions, therapies, treatments, and recommendations.",
+            "documentation": null,
+            "domain": "health",
+            "download_url": "media/ontologies/MAXO/38/maxo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/monarch-initiative/MAxO",
+            "id": 5861,
+            "indexed": true,
+            "max_children": 119,
+            "max_depth": 34,
+            "md5sum": "4848936305ffeadec05a3bf9a309128d",
+            "nb_classes": 13851,
+            "nb_inconsistent": 0,
+            "nb_individuals": 18,
+            "nb_properties": 286,
+            "products": [
+                {
+                    "id": "maxo.owl",
+                    "name": "Medical Action Ontology main release in OWL format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo.owl"
+                },
+                {
+                    "id": "maxo.obo",
+                    "name": "Medical Action Ontology additional release in OBO format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo.obo"
+                },
+                {
+                    "id": "maxo.json",
+                    "name": "Medical Action Ontology additional release in OBOJSon format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo.json"
+                },
+                {
+                    "id": "maxo/maxo-base.owl",
+                    "name": "Medical Action Ontology main release in OWL format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo/maxo-base.owl"
+                },
+                {
+                    "id": "maxo/maxo-base.obo",
+                    "name": "Medical Action Ontology additional release in OBO format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo/maxo-base.obo"
+                },
+                {
+                    "id": "maxo/maxo-base.json",
+                    "name": "Medical Action Ontology additional release in OBOJSon format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/maxo/maxo-base.json"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 38,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HL7",
-        "name": "Health Level Seven Reference Implementation Model, Version 3",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 6,
-            "classifiable": true,
-            "date_created": "2017-02-06T21:26:43Z",
-            "date_released": "2017-02-06T21:26:43Z",
-            "description": "This version is the first update to Normative RIM, Release 3. It is based on changes approved in Harmonization in November 2010. This release of the RIM is bound to HL7 Abstract Data Types Release 2.",
-            "documentation": "http://www.hl7.org",
-            "domain": null,
-            "download_url": "media/ontologies/HL7/9/hl7.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.hl7.org",
-            "id": 80,
-            "indexed": true,
-            "max_children": 337,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 10235,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 63,
-            "products": null,
-            "publication": "http://www.hl7.org",
-            "publications": null,
-            "submission_id": 9,
-            "taxon": null,
-            "version": "2016AB"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "GRO",
-        "name": "Gene Regulation Ontology",
+        "acronym": "MCHVODANATERMS",
+        "name": "MCHVODANATERMS",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2016-01-06T21:29:49Z",
-            "date_released": "2015-10-10T01:04:37Z",
-            "description": "The Gene Regulation Ontology (GRO) is a conceptual model for the domain of gene regulation. It covers processes  that are linked to the regulation of gene expression as well as physical entities that are involved in these processes (such as genes and transcription factors) in terms of ontology classes and semantic relations between classes.",
-            "documentation": "http://www.ebi.ac.uk/Rebholz-srv/GRO/GRO.html",
+            "date_created": "2022-02-10T18:12:22Z",
+            "date_released": "2022-02-10T00:00:00Z",
+            "description": "Ontology containing the controlled terms and semantic properties used in the Maternal Child Health Care generic template used within different facilities in VODANA ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GRO/5/gro.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.ebi.ac.uk/Rebholz-srv/GRO/GRO.html",
-            "id": 356,
+            "download_url": "media/ontologies/MCHVODANATERMS/5/mchvodanaterms.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 4904,
             "indexed": true,
-            "max_children": 21,
-            "max_depth": 11,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 507,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 4,
-            "nb_properties": 33,
+            "nb_individuals": 27,
+            "nb_properties": 9,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 5,
             "taxon": null,
-            "version": "0.5"
+            "version": "3.0"
         },
         "topics": null
     },
     {
-        "acronym": "HCDR",
-        "name": "Homelessness and Clinical Data Recording",
+        "acronym": "HMISLAB0115",
+        "name": "HMIS LAB 011 5",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2020-01-21T21:56:18Z",
-            "date_released": "2020-01-21T00:00:00Z",
-            "description": "This ontology utilizes the European Typology of Homelessness and Housing Exclusion (ETHOS), a respected conceptual framework used to define degrees of housing insecurity and homelessness internationally, to conceptualize, define, and link homelessness in SNOMED, Read CTV3, and Read 2 codes to clinical data extraction. The ontology uses the ETHOS as a general framework, leading to endpoints represented by codes from across the three databases. This is useful for researchers and medical professionals seeking to understand how homelessness is represented in clinical data, how homelessness is defined internationally, and what types of services are available for homeless individuals. ",
+            "date_created": "2021-11-12T15:01:37Z",
+            "date_released": "2021-11-12T00:00:00Z",
+            "description": "Assignment for group 5 of DSIP",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HCDR/1/hcdr.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/HMISLAB0115/1/hmislab0115.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 3298,
+            "id": 4693,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 6,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 119,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 18,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HOIP",
-        "name": "Homeostasis imbalance process ontology",
+        "acronym": "LSHO",
+        "name": "LiveStockHealthOntologyCattleUseCase ",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 18,
             "classifiable": true,
-            "date_created": "2023-06-07T05:24:33Z",
-            "date_released": "2023-05-29T00:00:00Z",
-            "description": "Homeostasis imbalance process ontology (HoIP) is an ontology that systematizes a wide variety of terms involving homeostasis imbalance courses and processes.\r\nHoIP focuses on the course of COVID-19 infectious processes and cellular senescence. \r\n\r\n",
+            "date_created": "2023-06-25T21:42:39Z",
+            "date_released": "2023-06-25T00:00:00Z",
+            "description": "Ontology is the formal and explicit description of a data model within an application domain in which relationships occur between individuals, making it easy to reuse and knowledge sharing. LSHO contains the information related to the health and well-being of livestock animal species such as cattle. Furthermore, it represents the explicit structured and standardized framework for specific domain knowledge and data integration of livestock species. During the development process, knowledge can be retrieved from domain experts, and after the retrieving process, the first prototype is designed for reasoning and planning of data related to cattle. Moreover, LSHO contains information regarding farms Identifications, their samples, geolocation, breed, diagnostic tests, Pathogen identifications, and Pathogen results for specific bacteria such as Histophilus somni (HS), Mannheimia haemolytica (MH), Pasteurella multocida (PM), Bovine coronavirus (BCV) bacterial mycoplasma and aerobic culture results for the respiratory track . This information is represented in the form of concepts of classes, subclasses, and their relationship in the form of object and data properties.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HOIP/64/hoip.owl",
+            "download_url": "media/ontologies/LSHO/1/lsho.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/yuki-yamagata/hoip",
-            "id": 5871,
+            "home_page": null,
+            "id": 5897,
             "indexed": true,
-            "max_children": 828,
-            "max_depth": 19,
+            "max_children": 18,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 13389,
+            "nb_classes": 18,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 195,
+            "nb_properties": 13,
             "products": null,
-            "publication": "https://doi.org/10.1101/2023.03.09.531883",
+            "publication": null,
             "publications": null,
-            "submission_id": 64,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2023/05/29"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "HOM",
-        "name": "Homology Ontology",
+        "acronym": "HEIO",
+        "name": "Regional Healthcare System Interoperability and Information Exchange Measurement Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2017-12-13T01:41:29.177764Z",
-            "date_released": "2017-12-13T01:41:29.177732Z",
-            "description": "This ontology represents concepts related to homology, as well as other concepts used to describe similarity and non-homology.",
+            "date_created": "2015-04-17T14:27:05Z",
+            "date_released": "2015-04-09T00:00:00Z",
+            "description": "The purpose of this ontology is to develop a model for regional healthcare system interoperability and information exchange quantification (measured by the electronic health information exchange (eHIE) indicator). The eHIE is hypothesized as a leading measure of regional healthcare system integration.\r\nA publication associated with this ontology is currently under review. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HOM/7/hom.owl",
+            "download_url": "media/ontologies/HEIO/17/heio.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://bioinfo.unil.ch",
-            "id": 1005,
+            "home_page": null,
+            "id": 17,
             "indexed": true,
-            "max_children": 13,
-            "max_depth": 6,
-            "md5sum": "d3f7a3aabf82ce1f5cfc2112875eeba9",
-            "nb_classes": 66,
+            "max_children": 36,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 125,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": [
-                {
-                    "id": "hom.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/hom.owl"
-                }
-            ],
+            "nb_individuals": 1605,
+            "nb_properties": 47,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 17,
             "taxon": null,
-            "version": null
+            "version": "Version 1.15"
         },
         "topics": null
     },
     {
-        "acronym": "HOME",
-        "name": "Health Ontology for Minority Equity",
+        "acronym": "MACROALGAETRAITS",
+        "name": "Macroalgae Traits Thesaurus",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 21,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-05-31T18:12:36Z",
-            "date_released": "2022-05-31T00:00:00Z",
-            "description": "An ontology to record the physical, emotional and psychological effects resulting from differences in treatment that citizens receive based on their identity when entering healthcare institutions. We present the first version of such a Health Ontology for Minority Equity (HOME)",
-            "documentation": null,
+            "date_created": "2023-04-10T19:33:48Z",
+            "date_released": "2023-04-10T00:00:00Z",
+            "description": "The Macroalgae Traits Thesaurus contains several concepts on demographic and functional traits. It has been developed and published by LifeWatch Italy, the Italian node of the e-science European infrastructure for biodiversity and ecosystem research (LifeWatch ERIC). It reflects the agreement of a scientific expert community to fix semantic properties (e.g. label, definition) of approximately 100 traits.",
+            "documentation": "http://www.lifewatchitaly.eu/web/lifewatch-italia/reports",
             "domain": null,
-            "download_url": "media/ontologies/HOME/1/home.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5124,
+            "download_url": "media/ontologies/MACROALGAETRAITS/1/macroalgaetraits.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://thesauri.lifewatchitaly.eu/Macroalgae/index.php",
+            "id": 5763,
             "indexed": true,
-            "max_children": 29,
-            "max_depth": 6,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 82,
-            "nb_inconsistent": 29,
-            "nb_individuals": 0,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 127,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HOM-ICD9",
-        "name": "ICD9 for Health Ontology Mapper",
+        "acronym": "HO",
+        "name": "Histological Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 109,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2011-04-26T22:58:45Z",
-            "date_released": "2011-04-26T07:00:00Z",
-            "description": "This is the ICD9 ontology with a different hierarchical view, but based on the same nomenclature and code structure. \r\nUseful for researchers that look for a disease by ICD9 code.",
+            "date_created": "2017-10-06T17:45:42Z",
+            "date_released": "2017-10-06T00:00:00Z",
+            "description": "A Histological Ontology of the Human Cardiovascular System v. 1.1",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HOM-ICD9/4/hom-icd9.owl",
+            "download_url": "media/ontologies/HO/78/ho.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 768,
+            "id": 184,
             "indexed": true,
-            "max_children": 947,
-            "max_depth": 6,
+            "max_children": 46,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 22202,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 10,
+            "nb_classes": 213,
+            "nb_inconsistent": 4,
+            "nb_individuals": 3,
+            "nb_properties": 39,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 78,
             "taxon": null,
-            "version": "2"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "HOM-UCSF_UCareDispostion",
-        "name": "Health Ontology Mapper - UCSF_UCareDischargeDisposition",
+        "acronym": "LHO",
+        "name": "LiveStockHealthOntology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2010-11-22T02:33:52Z",
-            "date_released": "2010-11-21T08:00:00Z",
-            "description": "All Disposition/Discharge Codes and Values listed in the UCSF- UCare patient source data",
+            "date_created": "2023-07-02T00:35:45Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "The Live Stock Health Ontology (LHO) is an extension of the DECIDE ontology. LHO follows OBO Foundry guidelines and contains information related to the health and well-being of livestock animal species such as cattle. Furthermore, it represents the explicit structured and standardized framework for specific domain knowledge and data integration of livestock species. During the development process, knowledge can be retrieved from domain experts, and after the retrieving process, the first prototype is designed for reasoning and planning of data related to cattle. Moreover, LHO contains information regarding farms Identifications, their samples, geolocation, breed, diagnostic tests, Pathogen identifications, and Pathogen results for specific bacteria such as Histophilus somni (HS), Mannheimia haemolytica (MH), Pasteurella multocida (PM), Bovine coronavirus (BCV) bacterial mycoplasma and aerobic culture results for the respiratory track in cattle. This information is represented in the form of concepts of classes, subclasses, and their relationship in the form of object and data properties.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HOM-UCSF_UCareDispostion/1/hom-ucsf_ucaredispostion.owl",
+            "download_url": "media/ontologies/LHO/1/lho.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 516,
+            "id": 5918,
             "indexed": true,
-            "max_children": 1,
-            "max_depth": 0,
+            "max_children": 29,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 1,
+            "nb_classes": 40,
             "nb_inconsistent": 0,
-            "nb_individuals": 150,
-            "nb_properties": 2,
+            "nb_individuals": 25,
+            "nb_properties": 22,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "HCOT",
-        "name": "HealthCare Ontology",
+        "acronym": "GRO",
+        "name": "Gene Regulation Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-12-06T05:21:15Z",
-            "date_released": "2018-12-06T00:00:00Z",
-            "description": "It is a sensor based health care ontology.",
-            "documentation": null,
+            "date_created": "2016-01-06T21:29:49Z",
+            "date_released": "2015-10-10T01:04:37Z",
+            "description": "The Gene Regulation Ontology (GRO) is a conceptual model for the domain of gene regulation. It covers processes  that are linked to the regulation of gene expression as well as physical entities that are involved in these processes (such as genes and transcription factors) in terms of ontology classes and semantic relations between classes.",
+            "documentation": "http://www.ebi.ac.uk/Rebholz-srv/GRO/GRO.html",
             "domain": null,
-            "download_url": "media/ontologies/HCOT/1/hcot.owl",
+            "download_url": "media/ontologies/GRO/5/gro.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2073,
+            "home_page": "http://www.ebi.ac.uk/Rebholz-srv/GRO/GRO.html",
+            "id": 356,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 4,
+            "max_children": 21,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 49,
+            "nb_classes": 507,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 8,
+            "nb_individuals": 4,
+            "nb_properties": 33,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 5,
             "taxon": null,
-            "version": null
+            "version": "0.5"
         },
         "topics": null
     },
     {
-        "acronym": "HCPCS",
-        "name": "Healthcare Common Procedure Coding System",
+        "acronym": "MAT",
+        "name": "Minimal Anatomical Terminology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 153,
             "classifiable": true,
-            "date_created": "2017-02-06T21:25:59Z",
-            "date_released": "2017-02-06T21:25:59Z",
-            "description": "Healthcare Common Procedure Coding System",
-            "documentation": "http://cms.hhs.gov",
+            "date_created": "2015-08-21T01:06:11Z",
+            "date_released": "2015-08-21T01:06:11Z",
+            "description": "Minimal set of terms for anatomy",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HCPCS/9/hcpcs.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://cms.hhs.gov",
-            "id": 331,
+            "download_url": "media/ontologies/MAT/3/mat.obo",
+            "has_ontology_language": "OBO",
+            "home_page": null,
+            "id": 189,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 457,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 5946,
+            "nb_classes": 2166,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 20,
+            "nb_properties": 19,
             "products": null,
-            "publication": "http://cms.hhs.gov",
+            "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 3,
             "taxon": null,
-            "version": "2016AB"
+            "version": "1.1"
         },
         "topics": null
     },
     {
         "acronym": "HDMPONTO",
         "name": "HierarchicalDMProcessOnto",
         "species": null,
@@ -15448,44 +15448,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MCCV",
-        "name": "Microbial Culture Collection Vocabulary",
+        "acronym": "HOIP",
+        "name": "Homeostasis imbalance process ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2016-03-14T09:09:37Z",
-            "date_released": "2015-04-08T00:00:00Z",
-            "description": "Structured controlled vocabulary for describing meta information of microbial calture collection maintained in biological research centers",
+            "date_created": "2023-06-07T05:24:33Z",
+            "date_released": "2023-05-29T00:00:00Z",
+            "description": "Homeostasis imbalance process ontology (HoIP) is an ontology that systematizes a wide variety of terms involving homeostasis imbalance courses and processes.\r\nHoIP focuses on the course of COVID-19 infectious processes and cellular senescence. \r\n\r\n",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MCCV/10/mccv.owl",
+            "download_url": "media/ontologies/HOIP/64/hoip.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 275,
+            "home_page": "https://github.com/yuki-yamagata/hoip",
+            "id": 5871,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 1,
+            "max_children": 828,
+            "max_depth": 19,
             "md5sum": null,
-            "nb_classes": 20,
+            "nb_classes": 13389,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 59,
+            "nb_properties": 195,
             "products": null,
-            "publication": null,
+            "publication": "https://doi.org/10.1101/2023.03.09.531883",
             "publications": null,
-            "submission_id": 10,
+            "submission_id": 64,
             "taxon": null,
-            "version": "Version 0.96 beta"
+            "version": "2023/05/29"
         },
         "topics": null
     },
     {
         "acronym": "REPOS",
         "name": "repository",
         "species": null,
@@ -15498,44 +15498,44 @@
         "name": "CranioMaxilloFacial ontology",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "MOMCARE",
-        "name": "MomCareVocabulary",
+        "acronym": "OCIMIDO",
+        "name": "Ocular Immune-Mediated Inflammatory Diseases Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-05-17T15:07:31Z",
-            "date_released": "2023-05-17T00:00:00Z",
-            "description": "MomCare Vocabulary",
-            "documentation": null,
+            "date_created": "2021-06-16T14:23:13Z",
+            "date_released": "2020-07-07T00:00:00Z",
+            "description": "Application ontology for Ocular Immune-mediated Inflammatory Diseases, built from domain experts in ophthalmology, clinical guidelines, and enhanced with patient-preferred terms.",
+            "documentation": "https://sap218.github.io/ocimido/",
             "domain": null,
-            "download_url": "media/ontologies/MOMCARE/1/momcare.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5824,
+            "download_url": "media/ontologies/OCIMIDO/2/ocimido.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/sap218/ocimido",
+            "id": 4604,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 47,
+            "max_depth": 17,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 661,
             "nb_inconsistent": 0,
-            "nb_individuals": 265,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 9,
             "products": null,
-            "publication": null,
+            "publication": "https://doi.org/10.1016/j.compbiomed.2021.104542",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": "1.2"
         },
         "topics": null
     },
     {
         "acronym": "IOBC",
         "name": "Interlinking Ontology for Biological Concepts",
         "species": null,
@@ -15574,356 +15574,182 @@
         "name": "Cross-linker reagents ontology",
         "species": null,
         "status": "Unknown",
         "submission": null,
         "topics": null
     },
     {
-        "acronym": "OCCO",
-        "name": "Occupation Ontology",
+        "acronym": "TERRADCAT_AP",
+        "name": "Terra DCAT Application Profile",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-04-16T01:19:21Z",
-            "date_released": "2023-04-16T01:19:18Z",
-            "description": "This Occupaton Ontology (OccO) is an ontology in the domain of human occupations. Current OccO version is developed based on the US Bureau of Labor Statistics Standard Occupation Classification (SOC) and the related O*Net System. The OccO development follows the principles of the Open Biological and Biomedical (OBO) Foundry.",
-            "documentation": "https://github.com/Occupation-Ontology/OccO",
+            "date_created": "2021-09-17T21:32:53Z",
+            "date_released": "2021-09-17T00:00:00Z",
+            "description": "Terra DCAT AP is the Broad Data Sciences Platform's application profile.  It is an RDF vocabulary which extends the W3C's Data Catalog Vocabulary (https://www.w3.org/TR/vocab-dcat/)",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OCCO/4/occo.owl",
+            "download_url": "media/ontologies/TERRADCAT_AP/2/terradcat_ap.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://hegroup.org/",
-            "id": 5774,
+            "home_page": "https://github.com/DataBiosphere/terra-interoperability-model",
+            "id": 4584,
             "indexed": true,
-            "max_children": 289,
-            "max_depth": 11,
+            "max_children": 36,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 1774,
+            "nb_classes": 110,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 3,
+            "nb_individuals": 21,
+            "nb_properties": 101,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0.65"
+            "version": "alpha"
         },
         "topics": null
     },
     {
-        "acronym": "NMR",
-        "name": "NMR-Controlled Vocabulary",
+        "acronym": "ORCS",
+        "name": "Ontology for Representing CDM Semantics",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2019-11-13T02:13:44Z",
-            "date_released": "2019-11-13T02:13:41Z",
-            "description": "This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors.",
-            "documentation": "http://www.cosmos-fp7.eu/system/files/presentation/COSMOS%20DELIVERABLE%202.4_0.pdf",
-            "domain": null,
-            "download_url": "media/ontologies/NMR/12/nmr.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://nmrml.org/cv/",
-            "id": 3116,
-            "indexed": true,
-            "max_children": 28,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 788,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 12,
-            "taxon": null,
-            "version": "1.1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NCIt-Activity",
-        "name": "Activity-View",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2009-11-19T07:53:31Z",
-            "date_released": "2009-11-18T08:00:00Z",
-            "description": "Activity subtree of NCIt.",
+            "date_created": "2021-11-18T17:47:11Z",
+            "date_released": "2021-11-18T00:00:00Z",
+            "description": "The Ontology for Representing CDM Semantics (ORCS) is an application ontology that was developed to represent the semantics underlying the National Patient-Centered Outcomes Research Network's (PCORnet) Common Data Model (CDM). ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NCIt-Activity/1/ncit-activity.owl",
+            "download_url": "media/ontologies/ORCS/1/orcs.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 654,
+            "home_page": "https://github.com/ufbmi/ORCS",
+            "id": 4719,
             "indexed": true,
-            "max_children": 231,
-            "max_depth": 11,
+            "max_children": 53,
+            "max_depth": 13,
             "md5sum": null,
-            "nb_classes": 3816,
+            "nb_classes": 861,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 68,
+            "nb_properties": 424,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "09.0"
+            "version": "2021-11-13"
         },
         "topics": null
     },
     {
-        "acronym": "NCIT_NEOPLASM",
-        "name": "Neoplasm",
+        "acronym": "MIDO",
+        "name": "Medical Imaging and Diagnostic Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2016-06-22T06:54:47Z",
-            "date_released": "2016-06-22T00:00:00Z",
-            "description": "The NCIt Neoplasm provides a core reference set of NCIt neoplasm classification concepts that is designed to facilitate consistent coding, analysis, and data sharing across a broad range of NCI and related resources.",
+            "date_created": "2022-04-25T23:32:03Z",
+            "date_released": "2022-04-25T00:00:00Z",
+            "description": "MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation\r\n",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NCIT_NEOPLASM/1/ncit_neoplasm.owl",
+            "download_url": "media/ontologies/MIDO/1/mido.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 264,
+            "id": 5050,
             "indexed": true,
-            "max_children": 3512,
-            "max_depth": 15,
+            "max_children": 173,
+            "max_depth": 23,
             "md5sum": null,
-            "nb_classes": 11867,
+            "nb_classes": 51838,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 173,
+            "nb_individuals": 24138,
+            "nb_properties": 250,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "16.05e"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OCD",
-        "name": "OCD ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2022-12-21T15:44:05Z",
-            "date_released": "2022-12-21T00:00:00Z",
-            "description": "The Obsessive Compulsive Disorder Ontology (OCD) is a domain-specific ontology for representing the knowledge of OCD.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OCD/7/ocd.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5588,
-            "indexed": true,
-            "max_children": 78,
-            "max_depth": 13,
-            "md5sum": null,
-            "nb_classes": 1327,
-            "nb_inconsistent": 0,
-            "nb_individuals": 81,
-            "nb_properties": 20,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 7,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OCE",
-        "name": "Ontology of Chemical Elements",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2020-02-29T02:11:01Z",
-            "date_released": "2020-02-29T02:11:00Z",
-            "description": "The Ontology of Chemical Elements (OCE) is a community-based ontology in the domain of chemical elements. OCE systematically represents and annotates the known 118 chemical elements and their charactistics and relations with other entities. OCE imports the basic chemical element information from ChEBI and adds more classes, annotations, and axioms. OCE specifically focuses on adding all the related information from the Periodic Table of Chemical Elements. ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OCE/2/oce.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ontologyofchemicalelements/OCE",
-            "id": 3385,
-            "indexed": true,
-            "max_children": 23,
-            "max_depth": 12,
-            "md5sum": null,
-            "nb_classes": 249,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 7,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "Vision Release: 1.0.29"
+            "version": "2015-11-20"
         },
         "topics": null
     },
     {
-        "acronym": "suicideo",
-        "name": "suicideonto",
+        "acronym": "PAYMENTMODE",
+        "name": "Payment Mode",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2013-08-26T06:09:58Z",
-            "date_released": "2013-08-26T07:00:00Z",
-            "description": "ontology for suicidology ",
+            "date_created": "2022-12-07T20:11:21Z",
+            "date_released": "2022-12-07T00:00:00Z",
+            "description": "This class refers to the different payment methods.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/suicideo/1/suicideo.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/PAYMENTMODE/1/paymentmode.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 665,
+            "id": 5550,
             "indexed": true,
-            "max_children": 22,
-            "max_depth": 6,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 290,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 50,
-            "nb_properties": 14,
+            "nb_individuals": 7,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OCIMIDO",
-        "name": "Ocular Immune-Mediated Inflammatory Diseases Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2021-06-16T14:23:13Z",
-            "date_released": "2020-07-07T00:00:00Z",
-            "description": "Application ontology for Ocular Immune-mediated Inflammatory Diseases, built from domain experts in ophthalmology, clinical guidelines, and enhanced with patient-preferred terms.",
-            "documentation": "https://sap218.github.io/ocimido/",
-            "domain": null,
-            "download_url": "media/ontologies/OCIMIDO/2/ocimido.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/sap218/ocimido",
-            "id": 4604,
-            "indexed": true,
-            "max_children": 47,
-            "max_depth": 17,
-            "md5sum": null,
-            "nb_classes": 661,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 9,
-            "products": null,
-            "publication": "https://doi.org/10.1016/j.compbiomed.2021.104542",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "1.2"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ORCS",
-        "name": "Ontology for Representing CDM Semantics",
+        "acronym": "VANDF",
+        "name": "Veterans Health Administration National Drug File",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2021-11-18T17:47:11Z",
-            "date_released": "2021-11-18T00:00:00Z",
-            "description": "The Ontology for Representing CDM Semantics (ORCS) is an application ontology that was developed to represent the semantics underlying the National Patient-Centered Outcomes Research Network's (PCORnet) Common Data Model (CDM). ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ORCS/1/orcs.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ufbmi/ORCS",
-            "id": 4719,
-            "indexed": true,
-            "max_children": 53,
-            "max_depth": 13,
-            "md5sum": null,
-            "nb_classes": 861,
-            "nb_inconsistent": 0,
-            "nb_individuals": 68,
-            "nb_properties": 424,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "2021-11-13"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ordo",
-        "name": "Orphanet Rare Disease Ontology",
-        "species": [
-            "Human"
-        ],
-        "status": "Classified",
-        "submission": {
-            "avg_children": 14,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2019-01-25T18:57:56.313507Z",
-            "date_released": "2019-01-25T18:57:56.313542Z",
-            "description": "",
+            "date_created": "2017-02-06T21:26:41Z",
+            "date_released": "2017-02-06T21:26:41Z",
+            "description": "NDF includes information on clinical drugs, drug classes, ingredients and National Drug Code (NDC) Directory codes.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ordo/1/ordo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.ebi.ac.uk/ols/ontologies/ordo",
-            "id": 2197,
+            "download_url": "media/ontologies/VANDF/9/vandf.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.pbm.va.gov/NationalFormulary.aspx",
+            "id": 545,
             "indexed": true,
-            "max_children": 3864,
-            "max_depth": 11,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 14062,
+            "nb_classes": 26295,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 18,
+            "nb_properties": 20,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 9,
             "taxon": null,
-            "version": "2.7"
+            "version": "2016AB"
         },
-        "topics": [
-            "rare disease"
-        ]
+        "topics": null
     },
     {
         "acronym": "WEAVE",
         "name": "WEAVE: Wind Energy ActiVitiEs",
         "species": null,
         "status": "Classified",
         "submission": {
@@ -16020,48 +15846,14 @@
             "submission_id": 225,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PAYMENTMODE",
-        "name": "Payment Mode",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2022-12-07T20:11:21Z",
-            "date_released": "2022-12-07T00:00:00Z",
-            "description": "This class refers to the different payment methods.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PAYMENTMODE/1/paymentmode.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5550,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 7,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
         "acronym": "PBO",
         "name": "Plant Breeding Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -16294,44 +16086,44 @@
             "submission_id": 11,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "MIDO",
-        "name": "Medical Imaging and Diagnostic Ontology",
+        "acronym": "MOMCARE",
+        "name": "MomCareVocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-04-25T23:32:03Z",
-            "date_released": "2022-04-25T00:00:00Z",
-            "description": "MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation\r\n",
+            "date_created": "2023-05-17T15:07:31Z",
+            "date_released": "2023-05-17T00:00:00Z",
+            "description": "MomCare Vocabulary",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MIDO/1/mido.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/MOMCARE/1/momcare.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 5050,
+            "id": 5824,
             "indexed": true,
-            "max_children": 173,
-            "max_depth": 23,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 51838,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 24138,
-            "nb_properties": 250,
+            "nb_individuals": 265,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2015-11-20"
+            "version": "1.1"
         },
         "topics": null
     },
     {
         "acronym": "STATO",
         "name": "The Statistical Methods Ontology",
         "species": null,
@@ -16367,44 +16159,44 @@
             "submission_id": 13,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "STO-DRAFT",
-        "name": "The Stroke Ontology",
+        "acronym": "NCIt-Activity",
+        "name": "Activity-View",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 18,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2018-08-07T13:36:29Z",
-            "date_released": "2018-08-07T00:00:00Z",
-            "description": "The Stroke Ontology (STO) covers the knowledge domain of stroke based on expert views and translational continuum of stroke research. ",
+            "date_created": "2009-11-19T07:53:31Z",
+            "date_released": "2009-11-18T08:00:00Z",
+            "description": "Activity subtree of NCIt.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/STO-DRAFT/1/sto-draft.owl",
+            "download_url": "media/ontologies/NCIt-Activity/1/ncit-activity.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 1739,
+            "id": 654,
             "indexed": true,
-            "max_children": 1676,
-            "max_depth": 5,
+            "max_children": 231,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 1712,
+            "nb_classes": 3816,
             "nb_inconsistent": 0,
-            "nb_individuals": 69,
-            "nb_properties": 29,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2017-11-01"
+            "version": "09.0"
         },
         "topics": null
     },
     {
         "acronym": "STMSO",
         "name": "symptomatic treatment of multiple sclerosis ontology",
         "species": null,
@@ -16435,78 +16227,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TEST_CBI",
-        "name": "test_cbi",
+        "acronym": "MEDLINEPLUS",
+        "name": "MedlinePlus Health Topics",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 21,
             "classifiable": true,
-            "date_created": "2021-02-02T18:25:06Z",
-            "date_released": "2021-02-02T00:00:00Z",
-            "description": "Testing CBI ontology",
-            "documentation": null,
+            "date_created": "2017-02-06T21:26:12Z",
+            "date_released": "2017-02-06T21:26:11Z",
+            "description": "MedlinePlus Health Topics;National Library of Medicine;June 14, 2008;Bethesda, MD",
+            "documentation": "http://www.nlm.nih.gov/medlineplus/xmldescription.html",
             "domain": null,
-            "download_url": "media/ontologies/TEST_CBI/1/test_cbi.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4251,
+            "download_url": "media/ontologies/MEDLINEPLUS/8/medlineplus.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.nlm.nih.gov/medlineplus/xmldescription.html",
+            "id": 694,
             "indexed": true,
-            "max_children": 3,
+            "max_children": 117,
             "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 4,
+            "nb_classes": 2146,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 13,
             "products": null,
-            "publication": null,
+            "publication": "http://www.nlm.nih.gov/medlineplus/xmldescription.html",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 8,
             "taxon": null,
-            "version": null
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "STY",
-        "name": "Semantic Types Ontology",
+        "acronym": "WSIO",
+        "name": "Web-Service Interaction Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-02-06T21:25:36Z",
-            "date_released": "2017-02-06T21:25:35Z",
-            "description": "UMLS Semantic Network\r\nThe Semantic Network consists of (1) a set of broad subject categories, or Semantic Types, that provide a consistent categorization of all concepts represented in the UMLS Metathesaurus, and (2) a set of useful and important relationships, or Semantic Relations, that exist between Semantic Types. This section of the documentation provides an overview of the Semantic Network, and describes the files of the Semantic Network. Sample records illustrate structure and content of these files.\r\n\r\nThe Semantic Network is distributed as one of the UMLS Knowledge Sources and as an open source resource available on the Semantic Network Web site, subject to these terms and conditions.",
-            "documentation": null,
+            "date_created": "2014-06-19T16:59:47Z",
+            "date_released": "2014-06-20T01:03:00Z",
+            "description": "Web Service Interaction Ontology (WSIO) enables automated interaction with more complex Web services that are typical for example within life sciences. WSIO is however independent of the application domain and relevant for both SOAP and REST Web services, and for batch execution engines in general. If the interaction scenario with a Web service is nontrivial (and incorporates session handling), annotation with WSIO concepts will enable automatic generation of client programs, scripts, or interactive applications with a graphical user interface. WSIO also enables automation of different ways of data transfer and data un-/compression or en-/decoding. We strongly discourage providers from developing complex interaction and data-transfer/compression scenarios, however when needed, WSIO may enable smooth automated interaction with them. Future versions will support more interaction scenarios. WSIO aims to serve also as a means to standardise the complex interaction scenarios primarily within both SOAP and REST Web services, and secondarily to apply also to batch execution infrastructure in general.",
+            "documentation": "http://wsio.org",
             "domain": null,
-            "download_url": "media/ontologies/STY/8/sty.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "https://semanticnetwork.nlm.nih.gov/",
-            "id": 467,
+            "download_url": "media/ontologies/WSIO/4/wsio.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://wsio.org",
+            "id": 374,
             "indexed": true,
-            "max_children": 9,
-            "max_depth": 7,
+            "max_children": 6,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 127,
+            "nb_classes": 31,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 11,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 4,
             "taxon": null,
-            "version": "2016AB"
+            "version": "beta2"
         },
         "topics": null
     },
     {
         "acronym": "SUGARBIND",
         "name": "SugarBind",
         "species": null,
@@ -16660,44 +16452,54 @@
                 "label": "Xenopus"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TERRADCAT_AP",
-        "name": "Terra DCAT Application Profile",
+        "acronym": "MCRO",
+        "name": "Model Card Report Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2021-09-17T21:32:53Z",
-            "date_released": "2021-09-17T00:00:00Z",
-            "description": "Terra DCAT AP is the Broad Data Sciences Platform's application profile.  It is an RDF vocabulary which extends the W3C's Data Catalog Vocabulary (https://www.w3.org/TR/vocab-dcat/)",
+            "date_created": "2023-07-03T12:59:38.658140Z",
+            "date_released": "2023-07-03T12:59:38.658129Z",
+            "description": "An ontology representing the structure of model card reports - reports that describe basic characteristics of machine learning models for the public and consumers.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/TERRADCAT_AP/2/terradcat_ap.owl",
+            "domain": "information technology",
+            "download_url": "media/ontologies/MCRO/1/mcro.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/DataBiosphere/terra-interoperability-model",
-            "id": 4584,
+            "home_page": "https://github.com/UTHealth-Ontology/MCRO",
+            "id": 5906,
             "indexed": true,
-            "max_children": 36,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 110,
+            "max_children": 155,
+            "max_depth": 11,
+            "md5sum": "86571d43ce2c0613b10f43018431cfa6",
+            "nb_classes": 938,
             "nb_inconsistent": 0,
-            "nb_individuals": 21,
-            "nb_properties": 101,
-            "products": null,
+            "nb_individuals": 24,
+            "nb_properties": 145,
+            "products": [
+                {
+                    "id": "mcro.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mcro.owl"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 2,
+            "publications": [
+                {
+                    "id": "https://doi.org/10.1186/s12859-022-04797-6",
+                    "title": "Toward a standard formal semantic representation of the model card report"
+                }
+            ],
+            "submission_id": 1,
             "taxon": null,
-            "version": "alpha"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "TEST_A",
         "name": "TEST_A",
         "species": null,
@@ -16728,44 +16530,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TESTEX",
-        "name": "test1",
+        "acronym": "suicideo",
+        "name": "suicideonto",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-04-21T18:26:34Z",
-            "date_released": "2021-04-21T00:00:00Z",
-            "description": "first ontology creation format",
+            "date_created": "2013-08-26T06:09:58Z",
+            "date_released": "2013-08-26T07:00:00Z",
+            "description": "ontology for suicidology ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TESTEX/1/testex.owl",
+            "download_url": "media/ontologies/suicideo/1/suicideo.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4429,
+            "id": 665,
             "indexed": true,
-            "max_children": 28,
-            "max_depth": 4,
+            "max_children": 22,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 102,
+            "nb_classes": 290,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 50,
+            "nb_properties": 14,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.0.1"
+            "version": "2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "OCD",
+        "name": "OCD ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2022-12-21T15:44:05Z",
+            "date_released": "2022-12-21T00:00:00Z",
+            "description": "The Obsessive Compulsive Disorder Ontology (OCD) is a domain-specific ontology for representing the knowledge of OCD.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/OCD/7/ocd.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 5588,
+            "indexed": true,
+            "max_children": 78,
+            "max_depth": 13,
+            "md5sum": null,
+            "nb_classes": 1327,
+            "nb_inconsistent": 0,
+            "nb_individuals": 81,
+            "nb_properties": 20,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 7,
+            "taxon": null,
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "MEDO",
         "name": "Mouse Experimental Design Ontology",
         "species": null,
@@ -16864,82 +16700,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "usecase",
-        "name": "usecase",
-        "species": [
-            "usecase"
-        ],
+        "acronym": "OCE",
+        "name": "Ontology of Chemical Elements",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2020-10-17T11:32:38.389268Z",
-            "date_released": "2020-10-17T11:32:38.389282Z",
-            "description": "",
+            "date_created": "2020-02-29T02:11:01Z",
+            "date_released": "2020-02-29T02:11:00Z",
+            "description": "The Ontology of Chemical Elements (OCE) is a community-based ontology in the domain of chemical elements. OCE systematically represents and annotates the known 118 chemical elements and their charactistics and relations with other entities. OCE imports the basic chemical element information from ChEBI and adds more classes, annotations, and axioms. OCE specifically focuses on adding all the related information from the Periodic Table of Chemical Elements. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/usecase/2/usecase.owl",
+            "download_url": "media/ontologies/OCE/2/oce.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3986,
+            "home_page": "https://github.com/ontologyofchemicalelements/OCE",
+            "id": 3385,
             "indexed": true,
-            "max_children": 29,
-            "max_depth": 4,
+            "max_children": 23,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 245,
+            "nb_classes": 249,
             "nb_inconsistent": 0,
-            "nb_individuals": 583,
-            "nb_properties": 81,
+            "nb_individuals": 0,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "Version2"
+            "version": "Vision Release: 1.0.29"
         },
-        "topics": [
-            "usecase"
-        ]
+        "topics": null
     },
     {
-        "acronym": "VANDF",
-        "name": "Veterans Health Administration National Drug File",
+        "acronym": "OCHV",
+        "name": "Ontology of Consumer Health Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-02-06T21:26:41Z",
-            "date_released": "2017-02-06T21:26:41Z",
-            "description": "NDF includes information on clinical drugs, drug classes, ingredients and National Drug Code (NDC) Directory codes.",
+            "date_created": "2016-01-21T05:00:15Z",
+            "date_released": "2016-01-21T00:00:00Z",
+            "description": "A SKOS-encoded implementation of  the \"Open Access, Collaborative Consumer Health Vocabulary Initiative\" by the University of Utah (consumerhealthvocab.org and layhealthinformatics.com)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/VANDF/9/vandf.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.pbm.va.gov/NationalFormulary.aspx",
-            "id": 545,
+            "download_url": "media/ontologies/OCHV/1/ochv.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 481,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 4,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 26295,
+            "nb_classes": 5,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 20,
+            "nb_individuals": 87879,
+            "nb_properties": 19,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2016AB"
+            "version": "1"
         },
         "topics": null
     },
     {
         "acronym": "VARIO",
         "name": "Variation Ontology",
         "species": null,
@@ -17013,88 +16845,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "v0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "WSIO",
-        "name": "Web-Service Interaction Ontology",
+        "acronym": "TEST-M4M20-PAV",
+        "name": "test-m4m20-subjects-pav",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2014-06-19T16:59:47Z",
-            "date_released": "2014-06-20T01:03:00Z",
-            "description": "Web Service Interaction Ontology (WSIO) enables automated interaction with more complex Web services that are typical for example within life sciences. WSIO is however independent of the application domain and relevant for both SOAP and REST Web services, and for batch execution engines in general. If the interaction scenario with a Web service is nontrivial (and incorporates session handling), annotation with WSIO concepts will enable automatic generation of client programs, scripts, or interactive applications with a graphical user interface. WSIO also enables automation of different ways of data transfer and data un-/compression or en-/decoding. We strongly discourage providers from developing complex interaction and data-transfer/compression scenarios, however when needed, WSIO may enable smooth automated interaction with them. Future versions will support more interaction scenarios. WSIO aims to serve also as a means to standardise the complex interaction scenarios primarily within both SOAP and REST Web services, and secondarily to apply also to batch execution infrastructure in general.",
-            "documentation": "http://wsio.org",
+            "date_created": "2022-06-08T17:47:46Z",
+            "date_released": "2022-06-08T00:00:00Z",
+            "description": "Test",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/WSIO/4/wsio.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://wsio.org",
-            "id": 374,
+            "download_url": "media/ontologies/TEST-M4M20-PAV/1/test-m4m20-pav.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5150,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 2,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 31,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 11,
+            "nb_individuals": 2,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 1,
             "taxon": null,
-            "version": "beta2"
+            "version": "0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "MCRO",
-        "name": "Model Card Report Ontology",
+        "acronym": "STY",
+        "name": "Semantic Types Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-07-03T12:59:38.658140Z",
-            "date_released": "2023-07-03T12:59:38.658129Z",
-            "description": "An ontology representing the structure of model card reports - reports that describe basic characteristics of machine learning models for the public and consumers.",
+            "date_created": "2017-02-06T21:25:36Z",
+            "date_released": "2017-02-06T21:25:35Z",
+            "description": "UMLS Semantic Network\r\nThe Semantic Network consists of (1) a set of broad subject categories, or Semantic Types, that provide a consistent categorization of all concepts represented in the UMLS Metathesaurus, and (2) a set of useful and important relationships, or Semantic Relations, that exist between Semantic Types. This section of the documentation provides an overview of the Semantic Network, and describes the files of the Semantic Network. Sample records illustrate structure and content of these files.\r\n\r\nThe Semantic Network is distributed as one of the UMLS Knowledge Sources and as an open source resource available on the Semantic Network Web site, subject to these terms and conditions.",
             "documentation": null,
-            "domain": "information technology",
-            "download_url": "media/ontologies/MCRO/1/mcro.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/UTHealth-Ontology/MCRO",
-            "id": 5906,
+            "domain": null,
+            "download_url": "media/ontologies/STY/8/sty.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "https://semanticnetwork.nlm.nih.gov/",
+            "id": 467,
             "indexed": true,
-            "max_children": 155,
-            "max_depth": 11,
-            "md5sum": "86571d43ce2c0613b10f43018431cfa6",
-            "nb_classes": 938,
+            "max_children": 9,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 127,
             "nb_inconsistent": 0,
-            "nb_individuals": 24,
-            "nb_properties": 145,
-            "products": [
-                {
-                    "id": "mcro.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mcro.owl"
-                }
-            ],
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://doi.org/10.1186/s12859-022-04797-6",
-                    "title": "Toward a standard formal semantic representation of the model card report"
-                }
-            ],
-            "submission_id": 1,
+            "publications": null,
+            "submission_id": 8,
             "taxon": null,
-            "version": null
+            "version": "2016AB"
         },
         "topics": null
     },
     {
         "acronym": "WWECA",
         "name": "Women with Epilepsy of Child-bearing Age Ontology",
         "species": null,
@@ -17125,80 +16947,84 @@
             "submission_id": 3,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "OCHV",
-        "name": "Ontology of Consumer Health Vocabulary",
+        "acronym": "NCIT_NEOPLASM",
+        "name": "Neoplasm",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-01-21T05:00:15Z",
-            "date_released": "2016-01-21T00:00:00Z",
-            "description": "A SKOS-encoded implementation of  the \"Open Access, Collaborative Consumer Health Vocabulary Initiative\" by the University of Utah (consumerhealthvocab.org and layhealthinformatics.com)",
+            "date_created": "2016-06-22T06:54:47Z",
+            "date_released": "2016-06-22T00:00:00Z",
+            "description": "The NCIt Neoplasm provides a core reference set of NCIt neoplasm classification concepts that is designed to facilitate consistent coding, analysis, and data sharing across a broad range of NCI and related resources.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OCHV/1/ochv.owl",
+            "download_url": "media/ontologies/NCIT_NEOPLASM/1/ncit_neoplasm.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 481,
+            "id": 264,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 1,
+            "max_children": 3512,
+            "max_depth": 15,
             "md5sum": null,
-            "nb_classes": 5,
+            "nb_classes": 11867,
             "nb_inconsistent": 0,
-            "nb_individuals": 87879,
-            "nb_properties": 19,
+            "nb_individuals": 0,
+            "nb_properties": 173,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "16.05e"
         },
         "topics": null
     },
     {
-        "acronym": "WHOFRE",
-        "name": "WHOART, French Translation",
-        "species": null,
+        "acronym": "ordo",
+        "name": "Orphanet Rare Disease Ontology",
+        "species": [
+            "Human"
+        ],
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 14,
             "classifiable": true,
-            "date_created": "2017-02-06T21:26:46Z",
-            "date_released": "2017-02-06T21:26:46Z",
-            "description": "WHOART, French Translation",
-            "documentation": "http://www.who.int/",
+            "date_created": "2019-01-25T18:57:56.313507Z",
+            "date_released": "2019-01-25T18:57:56.313542Z",
+            "description": "",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/WHOFRE/9/whofre.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.who.int/",
-            "id": 314,
+            "download_url": "media/ontologies/ordo/1/ordo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.ebi.ac.uk/ols/ontologies/ordo",
+            "id": 2197,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 3864,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 1827,
+            "nb_classes": 14062,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 18,
             "products": null,
-            "publication": "http://www.who.int/",
+            "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2016AB"
+            "version": "2.7"
         },
-        "topics": null
+        "topics": [
+            "rare disease"
+        ]
     },
     {
         "acronym": "PESONT",
         "name": "Preposition_enabled Spatial Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
@@ -17377,44 +17203,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MEDLINEPLUS",
-        "name": "MedlinePlus Health Topics",
+        "acronym": "NMR",
+        "name": "NMR-Controlled Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 21,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-02-06T21:26:12Z",
-            "date_released": "2017-02-06T21:26:11Z",
-            "description": "MedlinePlus Health Topics;National Library of Medicine;June 14, 2008;Bethesda, MD",
-            "documentation": "http://www.nlm.nih.gov/medlineplus/xmldescription.html",
+            "date_created": "2019-11-13T02:13:44Z",
+            "date_released": "2019-11-13T02:13:41Z",
+            "description": "This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors.",
+            "documentation": "http://www.cosmos-fp7.eu/system/files/presentation/COSMOS%20DELIVERABLE%202.4_0.pdf",
             "domain": null,
-            "download_url": "media/ontologies/MEDLINEPLUS/8/medlineplus.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.nlm.nih.gov/medlineplus/xmldescription.html",
-            "id": 694,
+            "download_url": "media/ontologies/NMR/12/nmr.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://nmrml.org/cv/",
+            "id": 3116,
             "indexed": true,
-            "max_children": 117,
-            "max_depth": 2,
+            "max_children": 28,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 2146,
+            "nb_classes": 788,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.nlm.nih.gov/medlineplus/xmldescription.html",
+            "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 12,
             "taxon": null,
-            "version": "2016AB"
+            "version": "1.1.0"
         },
         "topics": null
     },
     {
         "acronym": "PMDO",
         "name": "Parkinson and Movement Disorder Ontology",
         "species": null,
@@ -17581,44 +17407,44 @@
             "submission_id": 11,
             "taxon": null,
             "version": "2.4"
         },
         "topics": null
     },
     {
-        "acronym": "TEST-M4M20-PAV",
-        "name": "test-m4m20-subjects-pav",
+        "acronym": "STO-DRAFT",
+        "name": "The Stroke Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 18,
             "classifiable": true,
-            "date_created": "2022-06-08T17:47:46Z",
-            "date_released": "2022-06-08T00:00:00Z",
-            "description": "Test",
+            "date_created": "2018-08-07T13:36:29Z",
+            "date_released": "2018-08-07T00:00:00Z",
+            "description": "The Stroke Ontology (STO) covers the knowledge domain of stroke based on expert views and translational continuum of stroke research. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TEST-M4M20-PAV/1/test-m4m20-pav.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/STO-DRAFT/1/sto-draft.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5150,
+            "id": 1739,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 1676,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 1712,
             "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 0,
+            "nb_individuals": 69,
+            "nb_properties": 29,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.1.0"
+            "version": "2017-11-01"
         },
         "topics": null
     },
     {
         "acronym": "MIO",
         "name": "Myocardial Infarction Ontology",
         "species": null,
@@ -17649,44 +17475,388 @@
             "submission_id": 3,
             "taxon": null,
             "version": "V1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "DTO",
-        "name": "Drug Target Ontology",
+        "acronym": "OCCO",
+        "name": "Occupation Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-12-08T02:01:35Z",
-            "date_released": "2017-12-08T02:01:34Z",
-            "description": "Drug Target Ontology (DTO) is being developed at the University of Miami in the research group of Stephan Sch\u00fcrer. DTO is supported by grant U54CA189205 (Illuminating the Druggable Genome Knowledge Management Center, Tudor Oprea, PI) awarded by the NCI through the NIH Common Fund. It is a component of the Illuminating the Druggable Genome (IDG) project (https://commonfund.nih.gov/idg). The DTO project develops a novel semantic framework to formalize knowledge about drug targets with a focus on the current IDG protein families. The DTO is developed as a reference for drug targets with the longer-term goal to create a community standard that will facilitate the integration of diverse drug discovery information from numerous heterogeneous resources. The first version of the DTO consists of asserted class hierarchies of the four IDG protein families, GPCRs, kinases, ion channels, and nuclear hormone receptors. Protein classes are linked to tissue and disease via different levels of confidence. DTO also contains drug target development level classifications developed in the IDG project (http://targetcentral.ws/), and functional and qualitative annotations and classifications for kinase proteins, GPCR ligands and ion channels. DTO is modeled in OWL2-DL to enable further classification by inference reasoning and SPARQL queries. DTO is implemented following a modularization approach. DTO is used as the organizational framework for drug targets in the IDG PHAROS User Interface Portal (https://pharos.nih.gov) and also the Tin-X Target Importance and Novelty Explorer (http://newdrugtargets.org).",
-            "documentation": "http://drugtargetontology.org/",
+            "date_created": "2023-04-16T01:19:21Z",
+            "date_released": "2023-04-16T01:19:18Z",
+            "description": "This Occupaton Ontology (OccO) is an ontology in the domain of human occupations. Current OccO version is developed based on the US Bureau of Labor Statistics Standard Occupation Classification (SOC) and the related O*Net System. The OccO development follows the principles of the Open Biological and Biomedical (OBO) Foundry.",
+            "documentation": "https://github.com/Occupation-Ontology/OccO",
             "domain": null,
-            "download_url": "media/ontologies/DTO/4/dto.owl",
+            "download_url": "media/ontologies/OCCO/4/occo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://drugtargetontology.org/",
-            "id": 937,
+            "home_page": "https://hegroup.org/",
+            "id": 5774,
             "indexed": true,
-            "max_children": 636,
+            "max_children": 289,
             "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 10075,
+            "nb_classes": 1774,
             "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 42,
+            "nb_individuals": 18,
+            "nb_properties": 3,
             "products": null,
-            "publication": "http://drugtargetontology.org/",
+            "publication": null,
             "publications": null,
             "submission_id": 4,
             "taxon": null,
-            "version": "1.1"
+            "version": "1.0.65"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "usecase",
+        "name": "usecase",
+        "species": [
+            "usecase"
+        ],
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2020-10-17T11:32:38.389268Z",
+            "date_released": "2020-10-17T11:32:38.389282Z",
+            "description": "",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/usecase/2/usecase.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3986,
+            "indexed": true,
+            "max_children": 29,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 245,
+            "nb_inconsistent": 0,
+            "nb_individuals": 583,
+            "nb_properties": 81,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "Version2"
+        },
+        "topics": [
+            "usecase"
+        ]
+    },
+    {
+        "acronym": "TESTEX",
+        "name": "test1",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 15,
+            "classifiable": true,
+            "date_created": "2021-04-21T18:26:34Z",
+            "date_released": "2021-04-21T00:00:00Z",
+            "description": "first ontology creation format",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/TESTEX/1/testex.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4429,
+            "indexed": true,
+            "max_children": 28,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 102,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "0.0.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TEST_CBI",
+        "name": "test_cbi",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 1,
+            "classifiable": true,
+            "date_created": "2021-02-02T18:25:06Z",
+            "date_released": "2021-02-02T00:00:00Z",
+            "description": "Testing CBI ontology",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/TEST_CBI/1/test_cbi.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4251,
+            "indexed": true,
+            "max_children": 3,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 4,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "WHOFRE",
+        "name": "WHOART, French Translation",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 0,
+            "classifiable": true,
+            "date_created": "2017-02-06T21:26:46Z",
+            "date_released": "2017-02-06T21:26:46Z",
+            "description": "WHOART, French Translation",
+            "documentation": "http://www.who.int/",
+            "domain": null,
+            "download_url": "media/ontologies/WHOFRE/9/whofre.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.who.int/",
+            "id": 314,
+            "indexed": true,
+            "max_children": 0,
+            "max_depth": 0,
+            "md5sum": null,
+            "nb_classes": 1827,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
+            "products": null,
+            "publication": "http://www.who.int/",
+            "publications": null,
+            "submission_id": 9,
+            "taxon": null,
+            "version": "2016AB"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "DOREMUS-KEYS",
+        "name": "Doremus List of Keys",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2018-07-02T16:14:57Z",
+            "date_released": "2018-06-30T00:00:00Z",
+            "description": "The sum of relations, melodic and harmonic, existing between the tones of a scale or musical system.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/DOREMUS-KEYS/3/doremus-keys.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 1617,
+            "indexed": true,
+            "max_children": 3,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 3,
+            "nb_inconsistent": 0,
+            "nb_individuals": 60,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "DPCO",
+        "name": "Diabetes Pharmacology Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2022-12-18T15:38:21Z",
+            "date_released": "2022-10-31T00:00:00Z",
+            "description": "The Diabetes Pharmacology Ontology contains the classes needed to describe antihyperglycemic therapies, pharmacotherapy factors, and diagnostic factors.",
+            "documentation": "https://tetherless-world.github.io/diabetes-pharmacology-ontology/docs/DiabetesPharmacologyOntology/index-en.html",
+            "domain": null,
+            "download_url": "media/ontologies/DPCO/2/dpco.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://tetherless-world.github.io/diabetes-pharmacology-ontology/",
+            "id": 5570,
+            "indexed": true,
+            "max_children": 19,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 273,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 6,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "DSEO",
+        "name": "Data Science Education Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 0,
+            "classifiable": true,
+            "date_created": "2017-07-17T20:21:47Z",
+            "date_released": "2017-02-21T00:00:00Z",
+            "description": "Simple ISA hierarchical  taxonomy denoting subfields of study in data science for use in educational applications. ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/DSEO/2/dseo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://bigdatau.org/about_erudite",
+            "id": 691,
+            "indexed": true,
+            "max_children": 0,
+            "max_depth": 0,
+            "md5sum": null,
+            "nb_classes": 0,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "COVID19-IBO",
+        "name": "Covid19 Impact on Banking Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2021-06-07T11:29:25Z",
+            "date_released": "2021-06-07T00:00:00Z",
+            "description": "Covid19 Impact on Banking Ontology (Covid19-IBO) provides semantic information about the impact of the Covid-19 on the banking sector of India",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/COVID19-IBO/6/covid19-ibo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4651,
+            "indexed": true,
+            "max_children": 20,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 159,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 34,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 6,
+            "taxon": null,
+            "version": "1.5"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "COVID-19-ONT-PM",
+        "name": "COVID-19OntologyInPatternMedicine",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2022-04-05T03:13:03Z",
+            "date_released": "2021-08-20T00:00:00Z",
+            "description": "This ontology focuses on the aspects of the scientific findings in the COVID-19 fields in order to help the efforts against the outbreak of the pandemic. \r\nIt is expected, through the ontology,  to understand the medical conditions and make medical decisions in more integrated or systematic ways, by the engineerized or calculatable methods, rahter than the experience-based ways, usually used in the Traditional Information-Pieces-Accumulation Medicine(TIPAM).\r\nThe researches of the ontology involve the exploratory works of the Pattern Medicine(PM) and so called the Generalized Biomedical Dynamics(GBMD) , especially concentrating to the relations between the molecular and clinic levels.\r\n\r\nPlease pay attention to the background researches or the pilot project related to this ontolgy, and the introduction and publications, as well. \r\nThe relative artcle under going is at stie:\r\n\r\nhttps://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
+            "documentation": "https://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
+            "domain": null,
+            "download_url": "media/ontologies/COVID-19-ONT-PM/9/covid-19-ont-pm.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
+            "id": 5008,
+            "indexed": true,
+            "max_children": 39,
+            "max_depth": 9,
+            "md5sum": null,
+            "nb_classes": 299,
+            "nb_inconsistent": 0,
+            "nb_individuals": 6,
+            "nb_properties": 11,
+            "products": null,
+            "publication": "https://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
+            "publications": null,
+            "submission_id": 9,
+            "taxon": null,
+            "version": "007"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "COVIDCRFRAPID",
+        "name": "WHO COVID-19 Rapid Version CRF semantic data model",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2020-06-24T14:46:48Z",
+            "date_released": "2020-06-24T00:00:00Z",
+            "description": "This is a semantic data model for the WHO's COVID-19 case record form RAPID version from April 8 2020. It aims at providing semantic references to the questions and answers of the form.\r\n\r\nChanges in the version 1.1.4:\r\n- Corrected: Urea_(BUN) to remove the \"()\", which may cause resolution and validation issues;\r\n\r\nChanges in the version 1.1.3:\r\n- Corrected: Chloroquine/hydroxychloroquine to remove the \"/\", which may cause resolution issues;\r\n\r\nChanges in version 1.1.2:\r\n- Corrected: Cough with haemoptysis because the IRI was mispelled;\r\n- Corrected: CPAP/NIV Mask to remove the \"/\", which may cause resolution issues;\r\n- Added: missing \"part of\" property for section 1g.\r\n- Added: \"has other measurement unit label\" property for the laboratory results that may use a different unit than specified in the CRF;\r\n- Added: missing \"has value\" for the Which coronavirus question;\r\n- Added: missing possibility to enter the name of a specific antibiotic in section 3c of the CRF;\r\n- Added: missing question on testing the presence of other pathogenic of public health interest from section 3a of the CRF.\r\n\r\nChanges in version 1.1.1:\r\n- Added: Facility Name to identify the care facility in which the case is reported from.\r\n\r\nChanges in version 1.1.0:\r\n- This version covers the modifications introduced by the WHO COVID-19 Rapid CRF from April 8, 2020. The model maintains backward compatibility with the WHO COVID-19 Rapid CRF from March 23, 2020.",
+            "documentation": "https://vodan-ontology.github.io",
+            "domain": null,
+            "download_url": "media/ontologies/COVIDCRFRAPID/12/covidcrfrapid.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://purl.org/vodan/whocovid19crfsemdatamodel/",
+            "id": 3725,
+            "indexed": true,
+            "max_children": 91,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 398,
+            "nb_inconsistent": 0,
+            "nb_individuals": 333,
+            "nb_properties": 6,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 12,
+            "taxon": null,
+            "version": "1.1.4"
         },
         "topics": null
     },
     {
         "acronym": "EFO",
         "name": "Experimental Factor Ontology",
         "species": null,
@@ -17819,14 +17989,82 @@
             "submission_id": 3,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "DTO",
+        "name": "Drug Target Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2017-12-08T02:01:35Z",
+            "date_released": "2017-12-08T02:01:34Z",
+            "description": "Drug Target Ontology (DTO) is being developed at the University of Miami in the research group of Stephan Sch\u00fcrer. DTO is supported by grant U54CA189205 (Illuminating the Druggable Genome Knowledge Management Center, Tudor Oprea, PI) awarded by the NCI through the NIH Common Fund. It is a component of the Illuminating the Druggable Genome (IDG) project (https://commonfund.nih.gov/idg). The DTO project develops a novel semantic framework to formalize knowledge about drug targets with a focus on the current IDG protein families. The DTO is developed as a reference for drug targets with the longer-term goal to create a community standard that will facilitate the integration of diverse drug discovery information from numerous heterogeneous resources. The first version of the DTO consists of asserted class hierarchies of the four IDG protein families, GPCRs, kinases, ion channels, and nuclear hormone receptors. Protein classes are linked to tissue and disease via different levels of confidence. DTO also contains drug target development level classifications developed in the IDG project (http://targetcentral.ws/), and functional and qualitative annotations and classifications for kinase proteins, GPCR ligands and ion channels. DTO is modeled in OWL2-DL to enable further classification by inference reasoning and SPARQL queries. DTO is implemented following a modularization approach. DTO is used as the organizational framework for drug targets in the IDG PHAROS User Interface Portal (https://pharos.nih.gov) and also the Tin-X Target Importance and Novelty Explorer (http://newdrugtargets.org).",
+            "documentation": "http://drugtargetontology.org/",
+            "domain": null,
+            "download_url": "media/ontologies/DTO/4/dto.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://drugtargetontology.org/",
+            "id": 937,
+            "indexed": true,
+            "max_children": 636,
+            "max_depth": 11,
+            "md5sum": null,
+            "nb_classes": 10075,
+            "nb_inconsistent": 0,
+            "nb_individuals": 3,
+            "nb_properties": 42,
+            "products": null,
+            "publication": "http://drugtargetontology.org/",
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "DDO",
+        "name": "Diabetes Mellitus Diagnosis Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2015-12-23T11:49:12Z",
+            "date_released": "2015-12-20T00:00:00Z",
+            "description": "An ontology for diagnosis of diabetes containing the diabetes related complications, symptoms, drugs, lab tests, etc.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/DDO/3/ddo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 317,
+            "indexed": true,
+            "max_children": 62,
+            "max_depth": 18,
+            "md5sum": null,
+            "nb_classes": 6444,
+            "nb_inconsistent": 0,
+            "nb_individuals": 18,
+            "nb_properties": 48,
+            "products": null,
+            "publication": "http://link.springer.com/article/10.1186/s40535-016-0021-2?wt_mc=Internal.Event.1.SEM.ArticleAuthorIncrementalIssue",
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "2015-11-20"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ENVO",
         "name": "Environment Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -17929,14 +18167,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.53"
         },
         "topics": null
     },
     {
+        "acronym": "EGO",
+        "name": "Epigenome Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2016-07-03T23:00:59Z",
+            "date_released": "2016-06-30T00:00:00Z",
+            "description": "The Epigenome Ontology (EGO) a biomedical ontology for integrative epigenome knowledge representation and data analysis.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/EGO/1/ego.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/EGO-ontology",
+            "id": 18,
+            "indexed": true,
+            "max_children": 78,
+            "max_depth": 16,
+            "md5sum": null,
+            "nb_classes": 2952,
+            "nb_inconsistent": 0,
+            "nb_individuals": 178,
+            "nb_properties": 111,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "Arbor version; 1.0.1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "EPIP",
         "name": "Epigenetic Process",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 12,
             "classifiable": true,
@@ -17997,82 +18269,14 @@
             "submission_id": 8,
             "taxon": null,
             "version": "5.0"
         },
         "topics": null
     },
     {
-        "acronym": "DDO",
-        "name": "Diabetes Mellitus Diagnosis Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2015-12-23T11:49:12Z",
-            "date_released": "2015-12-20T00:00:00Z",
-            "description": "An ontology for diagnosis of diabetes containing the diabetes related complications, symptoms, drugs, lab tests, etc.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/DDO/3/ddo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 317,
-            "indexed": true,
-            "max_children": 62,
-            "max_depth": 18,
-            "md5sum": null,
-            "nb_classes": 6444,
-            "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 48,
-            "products": null,
-            "publication": "http://link.springer.com/article/10.1186/s40535-016-0021-2?wt_mc=Internal.Event.1.SEM.ArticleAuthorIncrementalIssue",
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "2015-11-20"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "EGO",
-        "name": "Epigenome Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2016-07-03T23:00:59Z",
-            "date_released": "2016-06-30T00:00:00Z",
-            "description": "The Epigenome Ontology (EGO) a biomedical ontology for integrative epigenome knowledge representation and data analysis.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/EGO/1/ego.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/EGO-ontology",
-            "id": 18,
-            "indexed": true,
-            "max_children": 78,
-            "max_depth": 16,
-            "md5sum": null,
-            "nb_classes": 2952,
-            "nb_inconsistent": 0,
-            "nb_individuals": 178,
-            "nb_properties": 111,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "Arbor version; 1.0.1"
-        },
-        "topics": null
-    },
-    {
         "acronym": "EPO",
         "name": "Early Pregnancy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -18457,282 +18661,406 @@
                 "label": "Dictyostelium discoideum"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DOREMUS-KEYS",
-        "name": "Doremus List of Keys",
+        "acronym": "RXNO",
+        "name": "Name Reaction Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2022-07-11T12:46:59.513218Z",
+            "date_released": "2022-07-11T12:46:59.513200Z",
+            "description": "Connects organic name reactions to their roles in an organic synthesis and to processes in MOP",
+            "documentation": null,
+            "domain": "chemistry and biochemistry",
+            "download_url": "media/ontologies/RXNO/40/rxno.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/rsc-ontologies/rxno",
+            "id": 5222,
+            "indexed": true,
+            "max_children": 53,
+            "max_depth": 9,
+            "md5sum": "87b6a75767c6f56a71a68b240352f92b",
+            "nb_classes": 1019,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 14,
+            "products": [
+                {
+                    "id": "rxno.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/rxno.owl",
+                    "title": "Name Reaction Ontology"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 40,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "SCTO",
+        "name": "SNOMED CT Standard Ontology Based on  the Ontology for General Medical Science",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-07-02T16:14:57Z",
-            "date_released": "2018-06-30T00:00:00Z",
-            "description": "The sum of relations, melodic and harmonic, existing between the tones of a scale or musical system.",
+            "date_created": "2017-11-21T12:30:34Z",
+            "date_released": "2017-11-21T00:00:00Z",
+            "description": "The Systematized Nomenclature of Medicine Clinical Term Upper-Level Ontology (SCTO) attempts to create a basic OWL 2 ontology for SNOMED CT terminology. The ontology is built based on the Ontology of General Medical Science (OGMS). This ontology implements the SNOMED CT concept model by adding the appropriate axioms to its top-level concepts. The resulting ontology can prevent redundancies and inconsistencies in this large terminology. Moreover, the terminology maintenance can become more accurate. Finally, SNOMED CT can be used in a healthcare environment to create more complex post-coordinated expressions and constraints in a more accurate manner.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DOREMUS-KEYS/3/doremus-keys.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/SCTO/14/scto.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 1617,
+            "id": 846,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
+            "max_children": 44,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 3,
+            "nb_classes": 304,
             "nb_inconsistent": 0,
-            "nb_individuals": 60,
-            "nb_properties": 0,
+            "nb_individuals": 18,
+            "nb_properties": 8,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 14,
             "taxon": null,
-            "version": null
+            "version": "21-11-2017"
         },
         "topics": null
     },
     {
-        "acronym": "DPCO",
-        "name": "Diabetes Pharmacology Ontology",
+        "acronym": "GANGLIONH",
+        "name": "NeuroMorpho.Org Ganglion cell type",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-12-18T15:38:21Z",
-            "date_released": "2022-10-31T00:00:00Z",
-            "description": "The Diabetes Pharmacology Ontology contains the classes needed to describe antihyperglycemic therapies, pharmacotherapy factors, and diagnostic factors.",
-            "documentation": "https://tetherless-world.github.io/diabetes-pharmacology-ontology/docs/DiabetesPharmacologyOntology/index-en.html",
+            "date_created": "2016-08-29T19:35:46Z",
+            "date_released": "2016-08-29T00:00:00Z",
+            "description": "A hierarchy of Ganglion cell types based on Retinal function.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DPCO/2/dpco.owl",
+            "download_url": "media/ontologies/GANGLIONH/2/ganglionh.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://tetherless-world.github.io/diabetes-pharmacology-ontology/",
-            "id": 5570,
+            "home_page": null,
+            "id": 231,
             "indexed": true,
-            "max_children": 19,
-            "max_depth": 7,
+            "max_children": 4,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 273,
+            "nb_classes": 12,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_individuals": 6,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.0"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "DSEO",
-        "name": "Data Science Education Ontology",
+        "acronym": "EUPATH",
+        "name": "VEuPathDB ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-07-17T20:21:47Z",
-            "date_released": "2017-02-21T00:00:00Z",
-            "description": "Simple ISA hierarchical  taxonomy denoting subfields of study in data science for use in educational applications. ",
+            "date_created": "2023-06-05T12:59:47.034684Z",
+            "date_released": "2023-06-05T12:59:47.034673Z",
+            "description": "An ontology is developed to support Eukaryotic Pathogen, Host & Vector Genomics Resource (VEuPathDB; https://veupathdb.org).",
+            "documentation": null,
+            "domain": "organisms",
+            "download_url": "media/ontologies/EUPATH/20/eupath.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/VEuPathDB-ontology/VEuPathDB-ontology",
+            "id": 5839,
+            "indexed": true,
+            "max_children": 348,
+            "max_depth": 25,
+            "md5sum": "2236f2639f45c7d3dabd63823436e888",
+            "nb_classes": 5110,
+            "nb_inconsistent": 0,
+            "nb_individuals": 92,
+            "nb_properties": 83,
+            "products": [
+                {
+                    "id": "eupath.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/eupath.owl"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://doi.org/10.5281/zenodo.6685957",
+                    "title": "Malaria study data integration and information retrieval based on OBO Foundry ontologies."
+                }
+            ],
+            "submission_id": 20,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "event",
+        "name": "Event",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2012-08-31T10:05:03Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-public-1.5.owl.  This file contains terms from the http://purl.org/NET/c4dm/event.owl# namespace that are included in the vivo ontology",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DSEO/2/dseo.owl",
+            "download_url": "media/ontologies/event/1/event.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://bigdatau.org/about_erudite",
-            "id": 691,
+            "home_page": "http://vitro.mannlib.cornell.edu/ns/vitro/0.7#shortDef",
+            "id": 590,
             "indexed": true,
-            "max_children": 0,
+            "max_children": 7,
             "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 0,
+            "nb_classes": 8,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 16,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.0"
+            "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "COVID19-IBO",
-        "name": "Covid19 Impact on Banking Ontology",
+        "acronym": "GYRISULCIH",
+        "name": "NeuroMorpho.Org Cortical Gyri & Sulci",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 11,
             "classifiable": true,
-            "date_created": "2021-06-07T11:29:25Z",
-            "date_released": "2021-06-07T00:00:00Z",
-            "description": "Covid19 Impact on Banking Ontology (Covid19-IBO) provides semantic information about the impact of the Covid-19 on the banking sector of India",
+            "date_created": "2017-03-31T01:04:29Z",
+            "date_released": "2017-03-31T01:04:29Z",
+            "description": "A hierarchy of Neocortex gyri and sulci in Primates.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/COVID19-IBO/6/covid19-ibo.owl",
+            "download_url": "media/ontologies/GYRISULCIH/3/gyrisulcih.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4651,
+            "id": 557,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 6,
+            "max_children": 21,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 159,
+            "nb_classes": 22,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 34,
+            "nb_individuals": 27,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.5"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "COVID-19-ONT-PM",
-        "name": "COVID-19OntologyInPatternMedicine",
+        "acronym": "FAST-TITLE",
+        "name": "FAST (Faceted Application of Subject Terminology) Title Facet",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2022-04-05T03:13:03Z",
-            "date_released": "2021-08-20T00:00:00Z",
-            "description": "This ontology focuses on the aspects of the scientific findings in the COVID-19 fields in order to help the efforts against the outbreak of the pandemic. \r\nIt is expected, through the ontology,  to understand the medical conditions and make medical decisions in more integrated or systematic ways, by the engineerized or calculatable methods, rahter than the experience-based ways, usually used in the Traditional Information-Pieces-Accumulation Medicine(TIPAM).\r\nThe researches of the ontology involve the exploratory works of the Pattern Medicine(PM) and so called the Generalized Biomedical Dynamics(GBMD) , especially concentrating to the relations between the molecular and clinic levels.\r\n\r\nPlease pay attention to the background researches or the pilot project related to this ontolgy, and the introduction and publications, as well. \r\nThe relative artcle under going is at stie:\r\n\r\nhttps://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
-            "documentation": "https://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
+            "date_created": "2017-09-06T00:04:12Z",
+            "date_released": "2017-04-14T00:00:00Z",
+            "description": "FAST (Faceted Application of Subject Terminology) is an enumerative, faceted subject heading schema derived from the Library of Congress Subject Headings (LCSH). The purpose of adapting the LCSH with a simplified syntax to create FAST is to retain the very rich vocabulary of LCSH while making the schema easier to understand, control, apply, and use.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/COVID-19-ONT-PM/9/covid-19-ont-pm.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
-            "id": 5008,
+            "download_url": "media/ontologies/FAST-TITLE/1/fast-title.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://www.oclc.org/research/themes/data-science/fast/download.html",
+            "id": 262,
             "indexed": true,
-            "max_children": 39,
-            "max_depth": 9,
+            "max_children": 3,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 299,
+            "nb_classes": 3,
             "nb_inconsistent": 0,
-            "nb_individuals": 6,
-            "nb_properties": 11,
+            "nb_individuals": 65800,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://blog.51.ca/u-345129/2020/11/13/the-preliminary-researches-on-the-data-dynamics-of-covid-19/",
+            "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 1,
             "taxon": null,
-            "version": "007"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "COVIDCRFRAPID",
-        "name": "WHO COVID-19 Rapid Version CRF semantic data model",
+        "acronym": "SEPIO",
+        "name": "Scientific Evidence and Provenance Information Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2020-06-24T14:46:48Z",
-            "date_released": "2020-06-24T00:00:00Z",
-            "description": "This is a semantic data model for the WHO's COVID-19 case record form RAPID version from April 8 2020. It aims at providing semantic references to the questions and answers of the form.\r\n\r\nChanges in the version 1.1.4:\r\n- Corrected: Urea_(BUN) to remove the \"()\", which may cause resolution and validation issues;\r\n\r\nChanges in the version 1.1.3:\r\n- Corrected: Chloroquine/hydroxychloroquine to remove the \"/\", which may cause resolution issues;\r\n\r\nChanges in version 1.1.2:\r\n- Corrected: Cough with haemoptysis because the IRI was mispelled;\r\n- Corrected: CPAP/NIV Mask to remove the \"/\", which may cause resolution issues;\r\n- Added: missing \"part of\" property for section 1g.\r\n- Added: \"has other measurement unit label\" property for the laboratory results that may use a different unit than specified in the CRF;\r\n- Added: missing \"has value\" for the Which coronavirus question;\r\n- Added: missing possibility to enter the name of a specific antibiotic in section 3c of the CRF;\r\n- Added: missing question on testing the presence of other pathogenic of public health interest from section 3a of the CRF.\r\n\r\nChanges in version 1.1.1:\r\n- Added: Facility Name to identify the care facility in which the case is reported from.\r\n\r\nChanges in version 1.1.0:\r\n- This version covers the modifications introduced by the WHO COVID-19 Rapid CRF from April 8, 2020. The model maintains backward compatibility with the WHO COVID-19 Rapid CRF from March 23, 2020.",
-            "documentation": "https://vodan-ontology.github.io",
+            "date_created": "2023-07-03T15:37:01.486619Z",
+            "date_released": "2023-07-03T15:37:01.486604Z",
+            "description": "An ontology for representing the provenance of scientific claims and the evidence that supports them.",
+            "documentation": null,
+            "domain": "investigations",
+            "download_url": "media/ontologies/SEPIO/4/sepio.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/monarch-initiative/SEPIO-ontology",
+            "id": 5910,
+            "indexed": true,
+            "max_children": 11,
+            "max_depth": 11,
+            "md5sum": "8fb50168990bedd0dd8fd3e84f994ae3",
+            "nb_classes": 127,
+            "nb_inconsistent": 0,
+            "nb_individuals": 21,
+            "nb_properties": 211,
+            "products": [
+                {
+                    "id": "sepio.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/sepio.owl",
+                    "title": "SEPIO"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "H1_NMOABA_1_0",
+        "name": "NeuroMorpho.Org Allen Brain Atlas ontology v.1.0",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2011-04-24T23:42:51Z",
+            "date_released": "2011-04-24T07:00:00Z",
+            "description": "Light version of the Allen Brain Atlas ontology that is customized for the NeuroMorpho.Org brain regions. The ontology includes NeuroMorpho.Org rodent data instantiated for direct matching classes.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/COVIDCRFRAPID/12/covidcrfrapid.owl",
+            "download_url": "media/ontologies/H1_NMOABA_1_0/1/h1_nmoaba_1_0.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://purl.org/vodan/whocovid19crfsemdatamodel/",
-            "id": 3725,
+            "home_page": "www.neuromorpho.org",
+            "id": 510,
             "indexed": true,
-            "max_children": 91,
-            "max_depth": 7,
+            "max_children": 17,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 398,
+            "nb_classes": 654,
             "nb_inconsistent": 0,
-            "nb_individuals": 333,
-            "nb_properties": 6,
+            "nb_individuals": 45,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.1.4"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "RDO",
-        "name": "Reusable Biodiversity Ontology",
+        "acronym": "H3_HIPSeptotemporalAxis",
+        "name": "NeuroMorpho.Org longitudinal hippocampal regions",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 89,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2021-06-02T12:26:11Z",
-            "date_released": "2021-06-02T00:00:00Z",
-            "description": "Facilitating Biodiversity information ",
+            "date_created": "2011-09-16T16:27:00Z",
+            "date_released": "2011-09-16T07:00:00Z",
+            "description": "longitudinal partitioning of rodent hippocampus",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/RDO/1/rdo.owl",
+            "download_url": "media/ontologies/H3_HIPSeptotemporalAxis/2/h3_hipseptotemporalaxis.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4647,
+            "home_page": "www.neuromorpho.org",
+            "id": 506,
             "indexed": true,
-            "max_children": 108,
-            "max_depth": 7,
+            "max_children": 2,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 362,
-            "nb_inconsistent": 108,
-            "nb_individuals": 3,
-            "nb_properties": 13,
+            "nb_classes": 6,
+            "nb_inconsistent": 0,
+            "nb_individuals": 2,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "REGN_BRO",
-        "name": "REGN_BRO",
+        "acronym": "HAAURAADO",
+        "name": "Halocynthia aurantium Anatomy and Development Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 85,
             "classifiable": true,
-            "date_created": "2023-01-12T14:19:05Z",
-            "date_released": "2023-01-12T00:00:00Z",
-            "description": "A controlled terminology of resources, which is used to improve the sensitivity and specificity of web searches.",
+            "date_created": "2016-08-04T19:01:10Z",
+            "date_released": "2015-08-06T00:00:00Z",
+            "description": "The first ontology describing the anatomy and the development of Halocynthia aurantium. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/REGN_BRO/1/regn_bro.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5610,
+            "download_url": "media/ontologies/HAAURAADO/2/haauraado.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.aniseed.cnrs.fr/",
+            "id": 358,
             "indexed": true,
-            "max_children": 65,
-            "max_depth": 8,
+            "max_children": 355,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 486,
+            "nb_classes": 856,
             "nb_inconsistent": 0,
-            "nb_individuals": 80,
-            "nb_properties": 30,
+            "nb_individuals": 0,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "FMA",
         "name": "Foundational Model of Anatomy Ontology (subset)",
         "species": null,
@@ -18781,430 +19109,350 @@
                 "label": "Homo sapiens"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "REGN_GO",
-        "name": "REGN_GO",
+        "acronym": "XEO",
+        "name": "XEML Environment Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-01-12T14:35:54Z",
-            "date_released": "2023-01-12T00:00:00Z",
-            "description": "Provides structured controlled vocabularies for the annotation of gene products with respect to their molecular function, cellular component, and biological role. The Gene Ontology consists of three Vocabularies.",
+            "date_created": "2014-06-25T08:07:43Z",
+            "date_released": "2012-11-09T08:00:00Z",
+            "description": "XeO has been created to help plant scientists in documenting and sharing metadata describing the abiotic environment.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/REGN_GO/1/regn_go.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5611,
+            "download_url": "media/ontologies/XEO/4/xeo.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://xeo.codeplex.com/",
+            "id": 252,
             "indexed": true,
-            "max_children": 7818,
-            "max_depth": 12,
+            "max_children": 12,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 51063,
+            "nb_classes": 145,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 9,
+            "nb_properties": 4,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "2023-01-01"
+            "version": "1.6"
         },
         "topics": null
     },
     {
-        "acronym": "RXNO",
-        "name": "Name Reaction Ontology",
+        "acronym": "GBM",
+        "name": "Glioblastoma",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2022-07-11T12:46:59.513218Z",
-            "date_released": "2022-07-11T12:46:59.513200Z",
-            "description": "Connects organic name reactions to their roles in an organic synthesis and to processes in MOP",
+            "date_created": "2015-08-27T21:56:32Z",
+            "date_released": "2015-08-27T00:00:00Z",
+            "description": "Ontology of Glioblastoma",
             "documentation": null,
-            "domain": "chemistry and biochemistry",
-            "download_url": "media/ontologies/RXNO/40/rxno.owl",
+            "domain": null,
+            "download_url": "media/ontologies/GBM/1/gbm.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/rsc-ontologies/rxno",
-            "id": 5222,
+            "home_page": null,
+            "id": 48,
             "indexed": true,
-            "max_children": 53,
-            "max_depth": 9,
-            "md5sum": "87b6a75767c6f56a71a68b240352f92b",
-            "nb_classes": 1019,
+            "max_children": 22,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 109,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 14,
-            "products": [
-                {
-                    "id": "rxno.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/rxno.owl",
-                    "title": "Name Reaction Ontology"
-                }
-            ],
+            "nb_properties": 38,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 40,
+            "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SCTO",
-        "name": "SNOMED CT Standard Ontology Based on  the Ontology for General Medical Science",
+        "acronym": "RADLEX",
+        "name": "Radiology Lexicon",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-11-21T12:30:34Z",
-            "date_released": "2017-11-21T00:00:00Z",
-            "description": "The Systematized Nomenclature of Medicine Clinical Term Upper-Level Ontology (SCTO) attempts to create a basic OWL 2 ontology for SNOMED CT terminology. The ontology is built based on the Ontology of General Medical Science (OGMS). This ontology implements the SNOMED CT concept model by adding the appropriate axioms to its top-level concepts. The resulting ontology can prevent redundancies and inconsistencies in this large terminology. Moreover, the terminology maintenance can become more accurate. Finally, SNOMED CT can be used in a healthcare environment to create more complex post-coordinated expressions and constraints in a more accurate manner.",
-            "documentation": null,
+            "date_created": "2016-11-16T16:50:31Z",
+            "date_released": "2016-11-16T00:00:00Z",
+            "description": "RadLex is a controlled terminology for radiology-a single unified source of radiology terms for radiology practice, education, and research. For license information see : <a href=\"http://www.rsna.org/uploadedFiles/RSNA/Content/Informatics/RadLex_License_Agreement_and_Terms_of_Use_V2_Final.pdf\">http://www.rsna.org/uploadedFiles/RSNA/Content/Informatics/RadLex_License_Agreement_and_Terms_of_Use_V2_Final.pdf</a>\r\n\r\nConverted to use OWL format submissions",
+            "documentation": "https://docs.google.com/document/d/10zRIBkXyj1eLt3LS_A7w3gSGRedXXoYvucH6H4trCZY/edit?hl=en#",
             "domain": null,
-            "download_url": "media/ontologies/SCTO/14/scto.owl",
+            "download_url": "media/ontologies/RADLEX/37/radlex.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 846,
+            "home_page": "http://www.rsna.org/RadLex.aspx",
+            "id": 415,
             "indexed": true,
-            "max_children": 44,
-            "max_depth": 12,
+            "max_children": 747,
+            "max_depth": 20,
             "md5sum": null,
-            "nb_classes": 304,
+            "nb_classes": 46433,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 8,
+            "nb_individuals": 46938,
+            "nb_properties": 91,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 37,
             "taxon": null,
-            "version": "21-11-2017"
+            "version": "3.14"
         },
         "topics": null
     },
     {
-        "acronym": "GANGLIONH",
-        "name": "NeuroMorpho.Org Ganglion cell type",
+        "acronym": "RADXTT-MVREASONS",
+        "name": "RADx Tiger Team Missing Value Reasons",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2016-08-29T19:35:46Z",
-            "date_released": "2016-08-29T00:00:00Z",
-            "description": "A hierarchy of Ganglion cell types based on Retinal function.",
-            "documentation": null,
+            "date_created": "2022-04-15T01:12:13Z",
+            "date_released": "2022-04-15T01:12:12Z",
+            "description": "These concepts define the different reasons values might be missing from the RADx data record. The concepts have been approved by the RADx Executive Committee to fill in fields that do not have a value in them. \r\n\r\nThe concepts in this vocabulary were developed by the RADx Missing Values Tiger Team. The protocols for managing these concepts in the RADx Data Hub environment are still being developed and will be improved and described as part of the RADx program.",
+            "documentation": "https://github.com/graybealski/excel2rdf-template",
             "domain": null,
-            "download_url": "media/ontologies/GANGLIONH/2/ganglionh.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/RADXTT-MVREASONS/9/radxtt-mvreasons.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 231,
+            "id": 5040,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 2,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 12,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 6,
+            "nb_individuals": 26,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 9,
             "taxon": null,
-            "version": "2.0"
+            "version": "0.1.1"
         },
         "topics": null
     },
     {
-        "acronym": "GBM",
-        "name": "Glioblastoma",
+        "acronym": "SENSO",
+        "name": "Sensitive Data Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2015-08-27T21:56:32Z",
-            "date_released": "2015-08-27T00:00:00Z",
-            "description": "Ontology of Glioblastoma",
-            "documentation": null,
+            "date_created": "2021-10-14T00:13:13Z",
+            "date_released": "2021-10-13T00:00:00Z",
+            "description": "The SENSO ontology is designed to provide classification terms for labeling sensitive data for data archival and sharing.",
+            "documentation": "https://github.com/DataONEorg/sem-prov-ontologies",
             "domain": null,
-            "download_url": "media/ontologies/GBM/1/gbm.owl",
+            "download_url": "media/ontologies/SENSO/1/senso.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 48,
+            "home_page": "https://github.com/DataONEorg/sem-prov-ontologies",
+            "id": 4650,
             "indexed": true,
-            "max_children": 22,
-            "max_depth": 4,
+            "max_children": 3,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 109,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 38,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Version 0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "GYRISULCIH",
-        "name": "NeuroMorpho.Org Cortical Gyri & Sulci",
+        "acronym": "SHR",
+        "name": "Student Health Record Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 11,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-03-31T01:04:29Z",
-            "date_released": "2017-03-31T01:04:29Z",
-            "description": "A hierarchy of Neocortex gyri and sulci in Primates.",
+            "date_created": "2015-01-09T12:36:20Z",
+            "date_released": "2011-09-29T07:00:00Z",
+            "description": "The goal of SHR project is to develop an ontology for the description of student health records. Student health records are created for entering college students in order to provide better health services will be formed. This file contains various sections such as history of disease, family history of disease, public examinations and ... .",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GYRISULCIH/3/gyrisulcih.owl",
+            "download_url": "media/ontologies/SHR/3/shr.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 557,
+            "home_page": "http://www.shojaee.com/shr/",
+            "id": 487,
             "indexed": true,
-            "max_children": 21,
-            "max_depth": 1,
+            "max_children": 23,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 22,
+            "nb_classes": 295,
             "nb_inconsistent": 0,
-            "nb_individuals": 27,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 38,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 3,
             "taxon": null,
-            "version": "2.0"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "H1_NMOABA_1_0",
-        "name": "NeuroMorpho.Org Allen Brain Atlas ontology v.1.0",
+        "acronym": "FALLS",
+        "name": "falls prevention ",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2011-04-24T23:42:51Z",
-            "date_released": "2011-04-24T07:00:00Z",
-            "description": "Light version of the Allen Brain Atlas ontology that is customized for the NeuroMorpho.Org brain regions. The ontology includes NeuroMorpho.Org rodent data instantiated for direct matching classes.",
+            "date_created": "2022-10-20T07:58:16Z",
+            "date_released": "2022-10-20T00:00:00Z",
+            "description": "This ontology is about a falls prevention intervention record. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/H1_NMOABA_1_0/1/h1_nmoaba_1_0.owl",
+            "download_url": "media/ontologies/FALLS/1/falls.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.neuromorpho.org",
-            "id": 510,
+            "home_page": null,
+            "id": 5443,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 9,
+            "max_children": 7,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 654,
+            "nb_classes": 30,
             "nb_inconsistent": 0,
-            "nb_individuals": 45,
-            "nb_properties": 0,
+            "nb_individuals": 17,
+            "nb_properties": 38,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "0.1"
         },
         "topics": null
     },
     {
-        "acronym": "H3_HIPSeptotemporalAxis",
-        "name": "NeuroMorpho.Org longitudinal hippocampal regions",
+        "acronym": "RDO",
+        "name": "Reusable Biodiversity Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 89,
             "classifiable": true,
-            "date_created": "2011-09-16T16:27:00Z",
-            "date_released": "2011-09-16T07:00:00Z",
-            "description": "longitudinal partitioning of rodent hippocampus",
+            "date_created": "2021-06-02T12:26:11Z",
+            "date_released": "2021-06-02T00:00:00Z",
+            "description": "Facilitating Biodiversity information ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/H3_HIPSeptotemporalAxis/2/h3_hipseptotemporalaxis.owl",
+            "download_url": "media/ontologies/RDO/1/rdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.neuromorpho.org",
-            "id": 506,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 6,
-            "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HAAURAADO",
-        "name": "Halocynthia aurantium Anatomy and Development Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 85,
-            "classifiable": true,
-            "date_created": "2016-08-04T19:01:10Z",
-            "date_released": "2015-08-06T00:00:00Z",
-            "description": "The first ontology describing the anatomy and the development of Halocynthia aurantium. ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HAAURAADO/2/haauraado.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.aniseed.cnrs.fr/",
-            "id": 358,
+            "home_page": null,
+            "id": 4647,
             "indexed": true,
-            "max_children": 355,
-            "max_depth": 2,
+            "max_children": 108,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 856,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 7,
+            "nb_classes": 362,
+            "nb_inconsistent": 108,
+            "nb_individuals": 3,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SEPIO",
-        "name": "Scientific Evidence and Provenance Information Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2023-07-03T15:37:01.486619Z",
-            "date_released": "2023-07-03T15:37:01.486604Z",
-            "description": "An ontology for representing the provenance of scientific claims and the evidence that supports them.",
-            "documentation": null,
-            "domain": "investigations",
-            "download_url": "media/ontologies/SEPIO/4/sepio.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/monarch-initiative/SEPIO-ontology",
-            "id": 5910,
-            "indexed": true,
-            "max_children": 11,
-            "max_depth": 11,
-            "md5sum": "8fb50168990bedd0dd8fd3e84f994ae3",
-            "nb_classes": 127,
-            "nb_inconsistent": 0,
-            "nb_individuals": 21,
-            "nb_properties": 211,
-            "products": [
-                {
-                    "id": "sepio.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/sepio.owl",
-                    "title": "SEPIO"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 4,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "SHR",
-        "name": "Student Health Record Ontology",
+        "acronym": "REGN_BRO",
+        "name": "REGN_BRO",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2015-01-09T12:36:20Z",
-            "date_released": "2011-09-29T07:00:00Z",
-            "description": "The goal of SHR project is to develop an ontology for the description of student health records. Student health records are created for entering college students in order to provide better health services will be formed. This file contains various sections such as history of disease, family history of disease, public examinations and ... .",
+            "date_created": "2023-01-12T14:19:05Z",
+            "date_released": "2023-01-12T00:00:00Z",
+            "description": "A controlled terminology of resources, which is used to improve the sensitivity and specificity of web searches.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SHR/3/shr.owl",
+            "download_url": "media/ontologies/REGN_BRO/1/regn_bro.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.shojaee.com/shr/",
-            "id": 487,
+            "home_page": null,
+            "id": 5610,
             "indexed": true,
-            "max_children": 23,
-            "max_depth": 4,
+            "max_children": 65,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 295,
+            "nb_classes": 486,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 38,
+            "nb_individuals": 80,
+            "nb_properties": 30,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
             "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "XEO",
-        "name": "XEML Environment Ontology",
+        "acronym": "FDT-O",
+        "name": "FAIR Data Train Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2014-06-25T08:07:43Z",
-            "date_released": "2012-11-09T08:00:00Z",
-            "description": "XeO has been created to help plant scientists in documenting and sharing metadata describing the abiotic environment.",
+            "date_created": "2022-12-01T02:05:01Z",
+            "date_released": "2022-12-01T02:04:59Z",
+            "description": "The FAIR Data Train Ontology provides the definitions of concepts and relations used in the FAIR Data Train environment.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/XEO/4/xeo.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://xeo.codeplex.com/",
-            "id": 252,
+            "download_url": "media/ontologies/FDT-O/9/fdt-o.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 5529,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 5,
+            "max_children": 6,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 145,
+            "nb_classes": 47,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
+            "nb_individuals": 19,
+            "nb_properties": 14,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 9,
             "taxon": null,
-            "version": "1.6"
+            "version": "- Refactored FHIR API and FHIR Train as sub-classes of REST API and REST Train, respectively."
         },
         "topics": null
     },
     {
         "acronym": "SD3",
         "name": "Simulation Delivery and Documentation Deviations",
         "species": null,
@@ -19274,112 +19522,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SDPV",
-        "name": "Syndrome Differentiation of Psoriasis Vulgaris in Traditional Chinese Medicine",
+        "acronym": "FGNHNS",
+        "name": "FoodGroupNHNS",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 11,
             "classifiable": true,
-            "date_created": "2021-01-20T00:06:23Z",
-            "date_released": "2021-01-19T00:00:00Z",
-            "description": "an ontology for syndrome differentiation and treatment of Psoriasis Vulgaris in Traditional Chinese Medicine",
+            "date_created": "2023-07-24T23:46:21Z",
+            "date_released": "2022-09-06T00:00:00Z",
+            "description": "Ontology of food groups of the National Health and Nutrition Examination Survey issued by the Ministry of Health, Labour and Welfare. Foods are divided into major, medium, and minor categories, and the English and Japanese names of foods belonging to the minor categories are noted.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SDPV/1/sdpv.owl",
+            "download_url": "media/ontologies/FGNHNS/12/fgnhns.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4215,
+            "id": 5965,
             "indexed": true,
-            "max_children": 9,
-            "max_depth": 8,
+            "max_children": 124,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 99,
+            "nb_classes": 1779,
             "nb_inconsistent": 0,
-            "nb_individuals": 275,
-            "nb_properties": 56,
+            "nb_individuals": 1777,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 12,
             "taxon": null,
-            "version": "Version 1.0"
+            "version": "0.161"
         },
         "topics": null
     },
     {
-        "acronym": "SEDI",
-        "name": "Semantic DICOM Ontology",
+        "acronym": "EVI",
+        "name": "Evidence Graph Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 49,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2015-06-10T15:12:14Z",
-            "date_released": "2015-06-10T00:00:00Z",
-            "description": "An ontology for DICOM as used in the SeDI project.",
-            "documentation": null,
+            "date_created": "2023-05-16T01:07:25Z",
+            "date_released": "2023-05-16T01:07:23Z",
+            "description": "The Evidence Graph ontology extends core concepts from the W3C Provenance Ontology PROV-O and Bioschemas' Profiles to describe evidence for correctness of findings in biomedical publications. The semantic data model in EVI is expressed using OWL2 Web Ontology Language.",
+            "documentation": "https://evidencegraph.github.io/EVI/",
             "domain": null,
-            "download_url": "media/ontologies/SEDI/6/sedi.owl",
+            "download_url": "media/ontologies/EVI/7/evi.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://semantic-dicom.org",
-            "id": 325,
-            "indexed": true,
-            "max_children": 490,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 1423,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4606,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 6,
-            "taxon": null,
-            "version": "0.6 alpha"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ETHANC",
-        "name": "ETHANC",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-08-24T16:46:57Z",
-            "date_released": "2021-07-05T00:00:00Z",
-            "description": "Antenatal care registry for Ethiopia",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ETHANC/16/ethanc.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4646,
+            "home_page": "https://evidencegraph.github.io/EVI/",
+            "id": 5815,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 14,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 31,
             "nb_inconsistent": 0,
-            "nb_individuals": 30,
-            "nb_properties": 12,
+            "nb_individuals": 3,
+            "nb_properties": 39,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 16,
+            "submission_id": 7,
             "taxon": null,
-            "version": "14"
+            "version": "1.1"
         },
         "topics": null
     },
     {
         "acronym": "ETHOPD",
         "name": "Ethiopian OPD",
         "species": null,
@@ -19410,122 +19624,53 @@
             "submission_id": 1,
             "taxon": null,
             "version": "0.3"
         },
         "topics": null
     },
     {
-        "acronym": "EUPATH",
-        "name": "VEuPathDB ontology",
+        "acronym": "FIX",
+        "name": "Physico-chemical methods and properties",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-05T12:59:47.034684Z",
-            "date_released": "2023-06-05T12:59:47.034673Z",
-            "description": "An ontology is developed to support Eukaryotic Pathogen, Host & Vector Genomics Resource (VEuPathDB; https://veupathdb.org).",
+            "date_created": "2020-04-13T18:02:48.973373Z",
+            "date_released": "2020-04-13T18:02:48.973319Z",
+            "description": "An ontology of physico-chemical methods and properties.",
             "documentation": null,
-            "domain": "organisms",
-            "download_url": "media/ontologies/EUPATH/20/eupath.owl",
+            "domain": null,
+            "download_url": "media/ontologies/FIX/119/fix.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/VEuPathDB-ontology/VEuPathDB-ontology",
-            "id": 5839,
+            "home_page": "",
+            "id": 3534,
             "indexed": true,
-            "max_children": 348,
-            "max_depth": 25,
-            "md5sum": "2236f2639f45c7d3dabd63823436e888",
-            "nb_classes": 5110,
+            "max_children": 45,
+            "max_depth": 12,
+            "md5sum": "e237a5f03e3a5ffd881f72fc12490199",
+            "nb_classes": 1163,
             "nb_inconsistent": 0,
-            "nb_individuals": 92,
-            "nb_properties": 83,
+            "nb_individuals": 0,
+            "nb_properties": 5,
             "products": [
                 {
-                    "id": "eupath.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/eupath.owl"
-                }
-            ],
-            "publication": null,
-            "publications": [
+                    "id": "fix.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fix.owl"
+                },
                 {
-                    "id": "https://doi.org/10.5281/zenodo.6685957",
-                    "title": "Malaria study data integration and information retrieval based on OBO Foundry ontologies."
+                    "id": "fix.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fix.obo"
                 }
             ],
-            "submission_id": 20,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "event",
-        "name": "Event",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2012-08-31T10:05:03Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-public-1.5.owl.  This file contains terms from the http://purl.org/NET/c4dm/event.owl# namespace that are included in the vivo ontology",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/event/1/event.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://vitro.mannlib.cornell.edu/ns/vitro/0.7#shortDef",
-            "id": 590,
-            "indexed": true,
-            "max_children": 7,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 8,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 16,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.5"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "EVI",
-        "name": "Evidence Graph Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2023-05-16T01:07:25Z",
-            "date_released": "2023-05-16T01:07:23Z",
-            "description": "The Evidence Graph ontology extends core concepts from the W3C Provenance Ontology PROV-O and Bioschemas' Profiles to describe evidence for correctness of findings in biomedical publications. The semantic data model in EVI is expressed using OWL2 Web Ontology Language.",
-            "documentation": "https://evidencegraph.github.io/EVI/",
-            "domain": null,
-            "download_url": "media/ontologies/EVI/7/evi.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://evidencegraph.github.io/EVI/",
-            "id": 5815,
-            "indexed": true,
-            "max_children": 14,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 31,
-            "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 39,
-            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 119,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "EXMO",
         "name": "Exercise Medicine Ontology",
         "species": null,
@@ -19633,257 +19778,112 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "EXTRACT",
-        "name": "EXTeRnAl Conditions Taxonomy (EXTRACT)",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2023-05-24T01:05:39Z",
-            "date_released": "2023-05-24T01:05:38Z",
-            "description": "Controlled vocabularies such as taxonomies allow an accurate and controlled approach in describing datasets. One of such controlled vocabulary is Wind Energy Taxonomy of External Conditions. This taxonomy is the result of EERA JP WIND IRPWind Open Data initiative that took place in 2017 in which wind energy experts generated the first version of the taxonomy as an input for defining and structuring wind energy metadata.",
-            "documentation": "https://github.com/DTUWindEnergy/EXTRACT-taxonomy",
-            "domain": null,
-            "download_url": "media/ontologies/EXTRACT/2/extract.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/DTUWindEnergy/EXTRACT-taxonomy",
-            "id": 5833,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 20,
-            "nb_properties": 0,
-            "products": null,
-            "publication": "https://github.com/DTUWindEnergy/EXTRACT-taxonomy",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "v0.1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FaBiO",
-        "name": "FaBiO (bibliographic)",
+        "acronym": "SDPV",
+        "name": "Syndrome Differentiation of Psoriasis Vulgaris in Traditional Chinese Medicine",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2012-08-31T10:12:47Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by the vivo-core-public-1.5.owl file. It contains terms relating to documents from the http://purl.org/spar/fabio/ namespace that are included in the VIVO ontology.  FaBiO is a FRBR-aligned Bibliographic Ontology that is part of the Semantic Publishing and Referencing (SPAR) suite of ontologies.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/FaBiO/1/fabio.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 43,
-            "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 4,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.5"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FALLS",
-        "name": "falls prevention ",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2022-10-20T07:58:16Z",
-            "date_released": "2022-10-20T00:00:00Z",
-            "description": "This ontology is about a falls prevention intervention record. ",
+            "date_created": "2021-01-20T00:06:23Z",
+            "date_released": "2021-01-19T00:00:00Z",
+            "description": "an ontology for syndrome differentiation and treatment of Psoriasis Vulgaris in Traditional Chinese Medicine",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FALLS/1/falls.owl",
+            "download_url": "media/ontologies/SDPV/1/sdpv.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5443,
-            "indexed": true,
-            "max_children": 7,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 30,
-            "nb_inconsistent": 0,
-            "nb_individuals": 17,
-            "nb_properties": 38,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "0.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FAST-TITLE",
-        "name": "FAST (Faceted Application of Subject Terminology) Title Facet",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2017-09-06T00:04:12Z",
-            "date_released": "2017-04-14T00:00:00Z",
-            "description": "FAST (Faceted Application of Subject Terminology) is an enumerative, faceted subject heading schema derived from the Library of Congress Subject Headings (LCSH). The purpose of adapting the LCSH with a simplified syntax to create FAST is to retain the very rich vocabulary of LCSH while making the schema easier to understand, control, apply, and use.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/FAST-TITLE/1/fast-title.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://www.oclc.org/research/themes/data-science/fast/download.html",
-            "id": 262,
+            "id": 4215,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 0,
+            "max_children": 9,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 3,
+            "nb_classes": 99,
             "nb_inconsistent": 0,
-            "nb_individuals": 65800,
-            "nb_properties": 0,
+            "nb_individuals": 275,
+            "nb_properties": 56,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Version 1.0"
         },
         "topics": null
     },
     {
-        "acronym": "FDT-O",
-        "name": "FAIR Data Train Ontology",
+        "acronym": "SEDI",
+        "name": "Semantic DICOM Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 49,
             "classifiable": true,
-            "date_created": "2022-12-01T02:05:01Z",
-            "date_released": "2022-12-01T02:04:59Z",
-            "description": "The FAIR Data Train Ontology provides the definitions of concepts and relations used in the FAIR Data Train environment.",
+            "date_created": "2015-06-10T15:12:14Z",
+            "date_released": "2015-06-10T00:00:00Z",
+            "description": "An ontology for DICOM as used in the SeDI project.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FDT-O/9/fdt-o.owl",
+            "download_url": "media/ontologies/SEDI/6/sedi.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5529,
+            "home_page": "http://semantic-dicom.org",
+            "id": 325,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 6,
+            "max_children": 490,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 47,
+            "nb_classes": 1423,
             "nb_inconsistent": 0,
-            "nb_individuals": 19,
-            "nb_properties": 14,
+            "nb_individuals": 0,
+            "nb_properties": 4606,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 6,
             "taxon": null,
-            "version": "- Refactored FHIR API and FHIR Train as sub-classes of REST API and REST Train, respectively."
+            "version": "0.6 alpha"
         },
         "topics": null
     },
     {
-        "acronym": "FGNHNS",
-        "name": "FoodGroupNHNS",
+        "acronym": "ETHANC",
+        "name": "ETHANC",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 11,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-07-24T23:46:21Z",
-            "date_released": "2022-09-06T00:00:00Z",
-            "description": "Ontology of food groups of the National Health and Nutrition Examination Survey issued by the Ministry of Health, Labour and Welfare. Foods are divided into major, medium, and minor categories, and the English and Japanese names of foods belonging to the minor categories are noted.",
+            "date_created": "2021-08-24T16:46:57Z",
+            "date_released": "2021-07-05T00:00:00Z",
+            "description": "Antenatal care registry for Ethiopia",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FGNHNS/12/fgnhns.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/ETHANC/16/ethanc.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 5965,
+            "id": 4646,
             "indexed": true,
-            "max_children": 124,
-            "max_depth": 5,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 1779,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 1777,
-            "nb_properties": 0,
+            "nb_individuals": 30,
+            "nb_properties": 12,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
-            "taxon": null,
-            "version": "0.161"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FIX",
-        "name": "Physico-chemical methods and properties",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2020-04-13T18:02:48.973373Z",
-            "date_released": "2020-04-13T18:02:48.973319Z",
-            "description": "An ontology of physico-chemical methods and properties.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/FIX/119/fix.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "",
-            "id": 3534,
-            "indexed": true,
-            "max_children": 45,
-            "max_depth": 12,
-            "md5sum": "e237a5f03e3a5ffd881f72fc12490199",
-            "nb_classes": 1163,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 5,
-            "products": [
-                {
-                    "id": "fix.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fix.owl"
-                },
-                {
-                    "id": "fix.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fix.obo"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 119,
+            "submission_id": 16,
             "taxon": null,
-            "version": null
+            "version": "14"
         },
         "topics": null
     },
     {
         "acronym": "MSTDE-FRE",
         "name": "Minimal Standard Terminology of Digestive Endoscopy, French",
         "species": null,
@@ -19914,422 +19914,280 @@
             "submission_id": 9,
             "taxon": null,
             "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "RADLEX",
-        "name": "Radiology Lexicon",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2016-11-16T16:50:31Z",
-            "date_released": "2016-11-16T00:00:00Z",
-            "description": "RadLex is a controlled terminology for radiology-a single unified source of radiology terms for radiology practice, education, and research. For license information see : <a href=\"http://www.rsna.org/uploadedFiles/RSNA/Content/Informatics/RadLex_License_Agreement_and_Terms_of_Use_V2_Final.pdf\">http://www.rsna.org/uploadedFiles/RSNA/Content/Informatics/RadLex_License_Agreement_and_Terms_of_Use_V2_Final.pdf</a>\r\n\r\nConverted to use OWL format submissions",
-            "documentation": "https://docs.google.com/document/d/10zRIBkXyj1eLt3LS_A7w3gSGRedXXoYvucH6H4trCZY/edit?hl=en#",
-            "domain": null,
-            "download_url": "media/ontologies/RADLEX/37/radlex.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.rsna.org/RadLex.aspx",
-            "id": 415,
-            "indexed": true,
-            "max_children": 747,
-            "max_depth": 20,
-            "md5sum": null,
-            "nb_classes": 46433,
-            "nb_inconsistent": 0,
-            "nb_individuals": 46938,
-            "nb_properties": 91,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 37,
-            "taxon": null,
-            "version": "3.14"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "RADXTT-MVREASONS",
-        "name": "RADx Tiger Team Missing Value Reasons",
+        "acronym": "EXTRACT",
+        "name": "EXTeRnAl Conditions Taxonomy (EXTRACT)",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-04-15T01:12:13Z",
-            "date_released": "2022-04-15T01:12:12Z",
-            "description": "These concepts define the different reasons values might be missing from the RADx data record. The concepts have been approved by the RADx Executive Committee to fill in fields that do not have a value in them. \r\n\r\nThe concepts in this vocabulary were developed by the RADx Missing Values Tiger Team. The protocols for managing these concepts in the RADx Data Hub environment are still being developed and will be improved and described as part of the RADx program.",
-            "documentation": "https://github.com/graybealski/excel2rdf-template",
+            "date_created": "2023-05-24T01:05:39Z",
+            "date_released": "2023-05-24T01:05:38Z",
+            "description": "Controlled vocabularies such as taxonomies allow an accurate and controlled approach in describing datasets. One of such controlled vocabulary is Wind Energy Taxonomy of External Conditions. This taxonomy is the result of EERA JP WIND IRPWind Open Data initiative that took place in 2017 in which wind energy experts generated the first version of the taxonomy as an input for defining and structuring wind energy metadata.",
+            "documentation": "https://github.com/DTUWindEnergy/EXTRACT-taxonomy",
             "domain": null,
-            "download_url": "media/ontologies/RADXTT-MVREASONS/9/radxtt-mvreasons.skos",
+            "download_url": "media/ontologies/EXTRACT/2/extract.skos",
             "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5040,
+            "home_page": "https://github.com/DTUWindEnergy/EXTRACT-taxonomy",
+            "id": 5833,
             "indexed": true,
             "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
             "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 26,
+            "nb_individuals": 20,
             "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "https://github.com/DTUWindEnergy/EXTRACT-taxonomy",
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.1.1"
+            "version": "v0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "SENSO",
-        "name": "Sensitive Data Ontology",
+        "acronym": "FaBiO",
+        "name": "FaBiO (bibliographic)",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-10-14T00:13:13Z",
-            "date_released": "2021-10-13T00:00:00Z",
-            "description": "The SENSO ontology is designed to provide classification terms for labeling sensitive data for data archival and sharing.",
-            "documentation": "https://github.com/DataONEorg/sem-prov-ontologies",
+            "date_created": "2012-08-31T10:12:47Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by the vivo-core-public-1.5.owl file. It contains terms relating to documents from the http://purl.org/spar/fabio/ namespace that are included in the VIVO ontology.  FaBiO is a FRBR-aligned Bibliographic Ontology that is part of the Semantic Publishing and Referencing (SPAR) suite of ontologies.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SENSO/1/senso.owl",
+            "download_url": "media/ontologies/FaBiO/1/fabio.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/DataONEorg/sem-prov-ontologies",
-            "id": 4650,
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 43,
             "indexed": true,
             "max_children": 3,
-            "max_depth": 2,
+            "max_depth": 1,
             "md5sum": null,
             "nb_classes": 4,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "Version 0.1.0"
+            "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "skos",
-        "name": "SKOS (concepts)",
+        "acronym": "REGN_GO",
+        "name": "REGN_GO",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2012-08-31T10:33:06Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-1.5.owl.  It contains terms relating to concepts from the  http://www.w3.org/2004/02/skos/core# namespace of the SKOS (Simple Knowledge Organization System) ontology that are included in the vivo ontology.",
+            "date_created": "2023-01-12T14:35:54Z",
+            "date_released": "2023-01-12T00:00:00Z",
+            "description": "Provides structured controlled vocabularies for the annotation of gene products with respect to their molecular function, cellular component, and biological role. The Gene Ontology consists of three Vocabularies.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/skos/1/skos.owl",
+            "download_url": "media/ontologies/REGN_GO/1/regn_go.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 36,
+            "home_page": null,
+            "id": 5611,
             "indexed": true,
-            "max_children": 1,
-            "max_depth": 0,
+            "max_children": 7818,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 1,
+            "nb_classes": 51063,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 3,
+            "nb_properties": 9,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.5"
+            "version": "2023-01-01"
         },
         "topics": null
     },
     {
-        "acronym": "SMASH",
-        "name": "SMASH Ontology",
+        "acronym": "HTO",
+        "name": "Horridge Test Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2015-06-03T01:14:47Z",
-            "date_released": "2015-06-03T01:14:47Z",
-            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.",
+            "date_created": "2022-03-24T17:59:18Z",
+            "date_released": "2022-03-24T00:00:00Z",
+            "description": "This is just a test ontology.  Called the Horridge Test Ontology (HTO)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SMASH/6/smash.owl",
+            "download_url": "media/ontologies/HTO/1/hto.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 436,
+            "id": 4984,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 7,
+            "max_children": 3,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 189,
-            "nb_inconsistent": 2,
-            "nb_individuals": 87,
-            "nb_properties": 144,
+            "nb_classes": 3,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "0.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "SMASHBIOMARKER",
-        "name": "SMASH ontology, Biomarker Ontology",
+        "acronym": "HRDO",
+        "name": "Disease core ontology applied to Rare Diseases",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 13,
             "classifiable": true,
-            "date_created": "2015-06-03T01:14:48Z",
-            "date_released": "2015-06-03T01:14:48Z",
-            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.\r\n\r\nBiomedical measures for users in SMASH ontology.",
+            "date_created": "2014-02-07T22:23:20Z",
+            "date_released": "2013-09-11T00:00:00Z",
+            "description": "This resource was designed during a PhD in medical informatics (funded by INSERM, 2010-2012). Its components are (i) a core ontology consistent with a metamodel (disorders and groups of disorders, genes, clinical signs and their relations) and (ii) an instantiation of this metamodel with Orphanet Data (available on http://orphadata.org).\r\n</ br>\r\nResearch experiments demonstrated (i) efficient classifications generation based on SPARQL Construct, (ii) perspectives in semantic audit of a knowledge base, (iii) semantic comparison with OMIM (www.omim.org) using proximity measurements and (iv) opened perspectives in knowledge sharing (LORD, http://lord.bndmr.fr). \r\n\r\nCurrent production services of Orphanet developed ORDO, released in 2014, an ontology synchronized with their production database. This ontology is now available on Bioportal.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SMASHBIOMARKER/4/smashbiomarker.owl",
+            "download_url": "media/ontologies/HRDO/2/hrdo.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 305,
+            "id": 375,
             "indexed": true,
-            "max_children": 9,
-            "max_depth": 4,
+            "max_children": 3514,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 97,
+            "nb_classes": 13539,
             "nb_inconsistent": 0,
-            "nb_individuals": 29,
-            "nb_properties": 52,
+            "nb_individuals": 0,
+            "nb_properties": 20,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HSPO",
-        "name": "Health & Social Person-centric Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2023-06-09T15:38:35Z",
-            "date_released": "2023-05-25T00:00:00Z",
-            "description": "The HSPO is an ontology that aims to represent and link together information about an individual from multiple domains. This work is motivated by the need to better understand the drivers of health outcomes that may arise from other domains including, but not limited to, the social determinants of health (e.g. ethnicity, transportation, education). This ontology can also describe a group of individuals that share similar characteristics (i.e. a cohort). Through this ontology we attempt to define a number of social contexts or social domains which can be useful to organise information coming from different terminologies such as ICD and SNOMED. \r\n\r\nGit Repository, Documentation and License details: https://github.com/IBM/hspo-ontology\r\nApache License 2.0",
-            "documentation": "https://github.com/IBM/hspo-ontology",
-            "domain": null,
-            "download_url": "media/ontologies/HSPO/3/hspo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/IBM/hspo-ontology",
-            "id": 5870,
-            "indexed": true,
-            "max_children": 5924,
-            "max_depth": 17,
-            "md5sum": null,
-            "nb_classes": 40641,
-            "nb_inconsistent": 0,
-            "nb_individuals": 200,
-            "nb_properties": 96,
-            "products": null,
-            "publication": "https://github.com/IBM/hspo-ontology",
-            "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.1.0"
+            "version": "V2"
         },
         "topics": null
     },
     {
-        "acronym": "HTN",
-        "name": "Hypertension Ontology",
+        "acronym": "SOCPRES",
+        "name": "Social Prescribing Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-06-10T14:40:58.259291Z",
-            "date_released": "2019-06-10T14:40:58.259265Z",
-            "description": "An ontology for representing clinical data about hypertension, intended to support classification of patients according to various diagnostic guidelines",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HTN/1/htn.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/aellenhicks/htn_owl",
-            "id": 2627,
-            "indexed": true,
-            "max_children": 36,
-            "max_depth": 14,
-            "md5sum": "53c4120c49cc8a2ae78d37313e3a898e",
-            "nb_classes": 603,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 263,
-            "products": [
-                {
-                    "id": "htn.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/htn.owl",
-                    "title": "HTN"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "H1_NMOABA_2_0",
-        "name": "NeuroMorpho.Org Allen Brain Atlas ontology  v.2.0",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2011-08-31T15:21:48Z",
-            "date_released": "2011-08-31T07:00:00Z",
-            "description": "Light version of the Allen Brain Atlas ontology that is customized for the NeuroMorpho.Org brain regions. This ontology instantiates 90% of rodent brain regions from NeuroMorpho.Org v5.2.",
+            "date_created": "2018-12-10T17:06:49Z",
+            "date_released": "2018-12-10T00:00:00Z",
+            "description": "The social prescribing ontology specifies concepts associated to social prescribing practices and the well being of social prescribing recipients. The ontology will be primary used to harmonise data related social prescribing processes across various parts of the health system.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/H1_NMOABA_2_0/1/h1_nmoaba_2_0.owl",
+            "download_url": "media/ontologies/SOCPRES/1/socpres.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.neuromorpho.org",
-            "id": 272,
+            "home_page": null,
+            "id": 2074,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 9,
+            "max_children": 7,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 661,
+            "nb_classes": 54,
             "nb_inconsistent": 0,
-            "nb_individuals": 61,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "H2_HIPClassicRegions",
-        "name": "NeuroMorpho.Org hippocampal regions",
+        "acronym": "GENDER",
+        "name": "Gender Identity",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2011-09-16T16:24:33Z",
-            "date_released": "2011-09-16T07:00:00Z",
-            "description": "A classical regional partition hierarchy of cornu ammonis  of rodent hippocampus.",
+            "date_created": "2022-12-07T18:37:29Z",
+            "date_released": "2022-12-07T00:00:00Z",
+            "description": "This vocabulary makes up the different gender identities - Female, Male and Other.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/H2_HIPClassicRegions/2/h2_hipclassicregions.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "www.neuromorpho.org",
-            "id": 377,
+            "download_url": "media/ontologies/GENDER/1/gender.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5545,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 3,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 8,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 7,
+            "nb_individuals": 5,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HAKMSVU",
-        "name": "Hakm",
+        "acronym": "SMASHBIOMARKER",
+        "name": "SMASH ontology, Biomarker Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-11-15T11:58:20Z",
-            "date_released": "2018-11-15T00:00:00Z",
-            "description": "OntologyForSvu",
-            "documentation": "http://SemanticWeb.org/Hakm",
-            "domain": null,
-            "download_url": "media/ontologies/HAKMSVU/2/hakmsvu.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://SemanticWeb.org/Hakm",
-            "id": 2523,
-            "indexed": true,
-            "max_children": 4,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 9,
-            "nb_inconsistent": 0,
-            "nb_individuals": 59,
-            "nb_properties": 4,
-            "products": null,
-            "publication": "http://SemanticWeb.org/Hakm",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "Latest"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HAMIDEHSGH",
-        "name": "Inherited Retinal Dystrophy",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2019-07-11T01:50:19Z",
-            "date_released": "2019-07-10T00:00:00Z",
-            "description": "Inherited retinal dystrophy (IRD) as one of the rare ocular diseases encompasses different types of hereditary retinal degenerations appearing by involvement of the different layers of the posterior ocular segment. In respect of the importance of the classification systems regarding the recorded history of biologic and natural sciences, there is no classification regarding the IRD diagnoses. Therefore, we aimed to present a novel classification in this regard.",
+            "date_created": "2015-06-03T01:14:48Z",
+            "date_released": "2015-06-03T01:14:48Z",
+            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.\r\n\r\nBiomedical measures for users in SMASH ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HAMIDEHSGH/1/hamidehsgh.owl",
+            "download_url": "media/ontologies/SMASHBIOMARKER/4/smashbiomarker.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 2746,
+            "id": 305,
             "indexed": true,
-            "max_children": 76,
-            "max_depth": 5,
+            "max_children": 9,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 407,
+            "nb_classes": 97,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 29,
+            "nb_properties": 52,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "HANCESTRO",
@@ -20435,218 +20293,14 @@
                 "label": "Hymenoptera"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HAROREADO",
-        "name": "Halocynthia roretzi Anatomy and Development Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 85,
-            "classifiable": true,
-            "date_created": "2016-08-04T19:19:34Z",
-            "date_released": "2015-08-06T00:00:00Z",
-            "description": " The first ontology describing the anatomy and the development of Halocynthia roretzi. ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HAROREADO/2/haroreado.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.aniseed.cnrs.fr/",
-            "id": 173,
-            "indexed": true,
-            "max_children": 355,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 856,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 7,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HASSET",
-        "name": "Humanities And Social Science Electronic Thesaurus",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2017-04-27T11:55:28Z",
-            "date_released": "2016-04-14T00:00:00Z",
-            "description": "The Humanities And Social Science Electronic Thesaurus (HASSET) is the leading British English thesaurus for the social sciences.\r\n\r\nHASSET has been compiled at the UK Data Archive for over thirty years. It was developed initially as an information retrieval tool to access data deposited in the UK Data Archive. Its scope has now expanded to cover the collections held by the UK Data Service.\r\n\r\nCopyright University of Essex. All rights reserved. \r\nYou may view the thesaurus online. \r\nPrior to downloading and/or using the full version of HASSET, SKOS-HASSET or any related product, a licence agreement must be completed between the user and the University of Essex.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HASSET/1/hasset.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://hasset.ukdataservice.ac.uk/",
-            "id": 1947,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 7610,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "2.00"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HCIOTO",
-        "name": "HealthCareOntology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2018-12-05T18:40:59Z",
-            "date_released": "2018-12-05T00:00:00Z",
-            "description": "A sensor based health care ontology",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HCIOTO/1/hcioto.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2075,
-            "indexed": true,
-            "max_children": 6,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 49,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 8,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HRDO",
-        "name": "Disease core ontology applied to Rare Diseases",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 13,
-            "classifiable": true,
-            "date_created": "2014-02-07T22:23:20Z",
-            "date_released": "2013-09-11T00:00:00Z",
-            "description": "This resource was designed during a PhD in medical informatics (funded by INSERM, 2010-2012). Its components are (i) a core ontology consistent with a metamodel (disorders and groups of disorders, genes, clinical signs and their relations) and (ii) an instantiation of this metamodel with Orphanet Data (available on http://orphadata.org).\r\n</ br>\r\nResearch experiments demonstrated (i) efficient classifications generation based on SPARQL Construct, (ii) perspectives in semantic audit of a knowledge base, (iii) semantic comparison with OMIM (www.omim.org) using proximity measurements and (iv) opened perspectives in knowledge sharing (LORD, http://lord.bndmr.fr). \r\n\r\nCurrent production services of Orphanet developed ORDO, released in 2014, an ontology synchronized with their production database. This ontology is now available on Bioportal.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HRDO/2/hrdo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 375,
-            "indexed": true,
-            "max_children": 3514,
-            "max_depth": 12,
-            "md5sum": null,
-            "nb_classes": 13539,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 20,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "V2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HTO",
-        "name": "Horridge Test Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2022-03-24T17:59:18Z",
-            "date_released": "2022-03-24T00:00:00Z",
-            "description": "This is just a test ontology.  Called the Horridge Test Ontology (HTO)",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/HTO/1/hto.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4984,
-            "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 3,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "0.0.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SOCPRES",
-        "name": "Social Prescribing Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2018-12-10T17:06:49Z",
-            "date_released": "2018-12-10T00:00:00Z",
-            "description": "The social prescribing ontology specifies concepts associated to social prescribing practices and the well being of social prescribing recipients. The ontology will be primary used to harmonise data related social prescribing processes across various parts of the health system.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/SOCPRES/1/socpres.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2074,
-            "indexed": true,
-            "max_children": 7,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 54,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
         "acronym": "SOHO",
         "name": "Social Determinants of Health Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
@@ -20673,339 +20327,322 @@
             "submission_id": 4,
             "taxon": null,
             "version": "3"
         },
         "topics": null
     },
     {
-        "acronym": "HSAPDV",
-        "name": "Human Developmental Stages",
+        "acronym": "HAROREADO",
+        "name": "Halocynthia roretzi Anatomy and Development Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 85,
             "classifiable": true,
-            "date_created": "2020-03-16T16:46:21.984399Z",
-            "date_released": "2020-03-16T16:46:21.984377Z",
-            "description": "Life cycle stages for Human",
+            "date_created": "2016-08-04T19:19:34Z",
+            "date_released": "2015-08-06T00:00:00Z",
+            "description": " The first ontology describing the anatomy and the development of Halocynthia roretzi. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HSAPDV/11/hsapdv.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/developmental-stage-ontologies/wiki/HsapDv",
-            "id": 3439,
+            "download_url": "media/ontologies/HAROREADO/2/haroreado.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.aniseed.cnrs.fr/",
+            "id": 173,
             "indexed": true,
-            "max_children": 242,
+            "max_children": 355,
             "max_depth": 2,
-            "md5sum": "52181d59df84578ed69214a5cb614036",
-            "nb_classes": 245,
+            "md5sum": null,
+            "nb_classes": 856,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": [
-                {
-                    "id": "hsapdv.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/hsapdv.owl"
-                },
-                {
-                    "id": "hsapdv.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/hsapdv.obo"
-                }
-            ],
+            "nb_properties": 7,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HSO",
-        "name": "Health Surveillance Ontology",
+        "acronym": "HTN",
+        "name": "Hypertension Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-12-20T13:18:30.149730Z",
-            "date_released": "2021-12-20T13:18:30.149714Z",
-            "description": "The health Surveillance Ontology (HSO) focuses on \"surveillance system level data\", that is, data outputs from surveillance activities, such as number of samples collected, cases observed, etc. It aims to support One-Health surveillance, covering animal health, public health and food safety surveillance.",
+            "date_created": "2019-06-10T14:40:58.259291Z",
+            "date_released": "2019-06-10T14:40:58.259265Z",
+            "description": "An ontology for representing clinical data about hypertension, intended to support classification of patients according to various diagnostic guidelines",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/HSO/11/hso.owl",
+            "domain": null,
+            "download_url": "media/ontologies/HTN/1/htn.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://w3id.org/hso",
-            "id": 4772,
+            "home_page": "https://github.com/aellenhicks/htn_owl",
+            "id": 2627,
             "indexed": true,
-            "max_children": 39,
-            "max_depth": 13,
-            "md5sum": "935e7235775b290e8d5cf6500237326e",
-            "nb_classes": 448,
+            "max_children": 36,
+            "max_depth": 14,
+            "md5sum": "53c4120c49cc8a2ae78d37313e3a898e",
+            "nb_classes": 603,
             "nb_inconsistent": 0,
-            "nb_individuals": 192,
-            "nb_properties": 149,
+            "nb_individuals": 0,
+            "nb_properties": 263,
             "products": [
                 {
-                    "homepage": "https://w3id.org/hso",
-                    "id": "hso.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/hso.owl"
+                    "id": "htn.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/htn.owl",
+                    "title": "HTN"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SOPHARM",
-        "name": "Suggested Ontology for Pharmacogenomics",
+        "acronym": "H1_NMOABA_2_0",
+        "name": "NeuroMorpho.Org Allen Brain Atlas ontology  v.2.0",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-03-02T16:12:57Z",
-            "date_released": "2017-03-02T00:00:00Z",
-            "description": "SO-Pharm (Suggested Ontology for Pharmacogenomics) is a domain ontology implemented in OWL-DL, which proposes a formal description of pharmacogenomic knowledge. SO-Pharm articulates different ontologies that represent complementary sub-domains of pharmacogenomics, i.e. related to genotype, phenotype, drugs, and clinical trials. SO-Pharm enables to represent pharmacogenomic relationships between a drug, a genomic variation and a phenotype trait. In addition, it enables to represent patient and more largely panel included in trials, and populations. SO-Pharm enables the representation of measured items on this patients such as results from the observation of a phenotype trait or of genomic variations. SO-Pharm supports knowledge about pharmacogenomic hypothesis, case study, and investigations in pharmacogenomics. SO-Pharm is designed to facilitate data integration and knowledge discovery in pharmacogenomics. In addition it provides a consistent articulation of ontologies of pharmacogenomic sub-domains.",
-            "documentation": "https://members.loria.fr/ACoulet/files/ontology/sopharm/version2.0/foundry_requirements.php",
+            "date_created": "2011-08-31T15:21:48Z",
+            "date_released": "2011-08-31T07:00:00Z",
+            "description": "Light version of the Allen Brain Atlas ontology that is customized for the NeuroMorpho.Org brain regions. This ontology instantiates 90% of rodent brain regions from NeuroMorpho.Org v5.2.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SOPHARM/18/sopharm.owl",
+            "download_url": "media/ontologies/H1_NMOABA_2_0/1/h1_nmoaba_2_0.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://members.loria.fr/ACoulet/files/sopharm2.0_description.php",
-            "id": 692,
+            "home_page": "www.neuromorpho.org",
+            "id": 272,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 4,
+            "max_children": 17,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 117,
+            "nb_classes": 661,
             "nb_inconsistent": 0,
-            "nb_individuals": 17,
-            "nb_properties": 61,
+            "nb_individuals": 61,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://hal.inria.fr/inria-00089824/en/",
+            "publication": null,
             "publications": null,
-            "submission_id": 18,
+            "submission_id": 1,
             "taxon": null,
-            "version": "The new release (March 02, 2017) 2.2 includes concept and properties labels in French.\nImports have been abandoned in favor of simple references to distant URI"
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "SOS",
-        "name": "emerg\u00eancia",
+        "acronym": "H2_HIPClassicRegions",
+        "name": "NeuroMorpho.Org hippocampal regions",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-09-20T17:41:36Z",
-            "date_released": "2021-09-20T00:00:00Z",
-            "description": "Um teste",
+            "date_created": "2011-09-16T16:24:33Z",
+            "date_released": "2011-09-16T07:00:00Z",
+            "description": "A classical regional partition hierarchy of cornu ammonis  of rodent hippocampus.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SOS/1/sos.owl",
+            "download_url": "media/ontologies/H2_HIPClassicRegions/2/h2_hipclassicregions.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4594,
+            "home_page": "www.neuromorpho.org",
+            "id": 377,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 4,
+            "max_children": 3,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 64,
+            "nb_classes": 8,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 7,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "SOY",
-        "name": "Soy Ontology",
+        "acronym": "HAKMSVU",
+        "name": "Hakm",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2013-05-01T01:23:44Z",
-            "date_released": "2013-04-30T07:00:00Z",
-            "description": "Growth, trait and development ontology for soybean",
-            "documentation": null,
+            "date_created": "2018-11-15T11:58:20Z",
+            "date_released": "2018-11-15T00:00:00Z",
+            "description": "OntologyForSvu",
+            "documentation": "http://SemanticWeb.org/Hakm",
             "domain": null,
-            "download_url": "media/ontologies/SOY/2/soy.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "soybase.org",
-            "id": 402,
+            "download_url": "media/ontologies/HAKMSVU/2/hakmsvu.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://SemanticWeb.org/Hakm",
+            "id": 2523,
             "indexed": true,
-            "max_children": 64,
-            "max_depth": 8,
+            "max_children": 4,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 1835,
+            "nb_classes": 9,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 59,
+            "nb_properties": 4,
             "products": null,
-            "publication": null,
+            "publication": "http://SemanticWeb.org/Hakm",
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "unknown"
+            "version": "Latest"
         },
         "topics": null
     },
     {
-        "acronym": "SPD",
-        "name": "Spider Ontology",
+        "acronym": "HSPO",
+        "name": "Health & Social Person-centric Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2019-03-04T15:46:08.307620Z",
-            "date_released": "2019-03-04T15:46:08.307595Z",
-            "description": "An ontology for spider comparative biology including anatomical parts (e.g. leg, claw), behavior (e.g. courtship, combing) and products (i.g. silk, web, borrow).",
-            "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/SPD/25/spd.owl",
+            "date_created": "2023-06-09T15:38:35Z",
+            "date_released": "2023-05-25T00:00:00Z",
+            "description": "The HSPO is an ontology that aims to represent and link together information about an individual from multiple domains. This work is motivated by the need to better understand the drivers of health outcomes that may arise from other domains including, but not limited to, the social determinants of health (e.g. ethnicity, transportation, education). This ontology can also describe a group of individuals that share similar characteristics (i.e. a cohort). Through this ontology we attempt to define a number of social contexts or social domains which can be useful to organise information coming from different terminologies such as ICD and SNOMED. \r\n\r\nGit Repository, Documentation and License details: https://github.com/IBM/hspo-ontology\r\nApache License 2.0",
+            "documentation": "https://github.com/IBM/hspo-ontology",
+            "domain": null,
+            "download_url": "media/ontologies/HSPO/3/hspo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://research.amnh.org/atol/files/",
-            "id": 2323,
+            "home_page": "https://github.com/IBM/hspo-ontology",
+            "id": 5870,
             "indexed": true,
-            "max_children": 278,
-            "max_depth": 10,
-            "md5sum": "b8dd96c7e895da699883f9101956e9ca",
-            "nb_classes": 832,
+            "max_children": 5924,
+            "max_depth": 17,
+            "md5sum": null,
+            "nb_classes": 40641,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": [
-                {
-                    "id": "spd.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/spd.owl"
-                }
-            ],
-            "publication": null,
+            "nb_individuals": 200,
+            "nb_properties": 96,
+            "products": null,
+            "publication": "https://github.com/IBM/hspo-ontology",
             "publications": null,
-            "submission_id": 25,
-            "taxon": {
-                "id": "NCBITaxon:6893",
-                "label": "spiders"
-            },
-            "version": null
+            "submission_id": 3,
+            "taxon": null,
+            "version": "0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "SPINALH",
-        "name": "NeuroMorpho.Org spinal cord regions ",
+        "acronym": "HUGO",
+        "name": "Human Genome Organization Gene Symbols",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 866,
             "classifiable": true,
-            "date_created": "2017-03-31T01:10:37Z",
-            "date_released": "2017-03-31T01:10:37Z",
-            "description": "A hierarchy of Spinal Cord regions in Bilateral species.",
-            "documentation": null,
+            "date_created": "2010-12-21T16:27:14Z",
+            "date_released": "2010-01-07T08:00:00Z",
+            "description": "Human Genome Organisation (HUGO) Gene Nomenclature Committee at the European Bioinformatics Institute approves a gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database.",
+            "documentation": "http://www.genenames.org/",
             "domain": null,
-            "download_url": "media/ontologies/SPINALH/3/spinalh.owl",
+            "download_url": "media/ontologies/HUGO/3/hugo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 542,
+            "home_page": "http://www.genenames.org/",
+            "id": 257,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 3,
+            "max_children": 19361,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 11,
+            "nb_classes": 32918,
             "nb_inconsistent": 0,
-            "nb_individuals": 11,
+            "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "http://www.genenames.org/",
             "publications": null,
             "submission_id": 3,
             "taxon": null,
-            "version": "2.0"
+            "version": "July2010"
         },
         "topics": null
     },
     {
-        "acronym": "GENDER",
-        "name": "Gender Identity",
+        "acronym": "HASSET",
+        "name": "Humanities And Social Science Electronic Thesaurus",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-12-07T18:37:29Z",
-            "date_released": "2022-12-07T00:00:00Z",
-            "description": "This vocabulary makes up the different gender identities - Female, Male and Other.",
+            "date_created": "2017-04-27T11:55:28Z",
+            "date_released": "2016-04-14T00:00:00Z",
+            "description": "The Humanities And Social Science Electronic Thesaurus (HASSET) is the leading British English thesaurus for the social sciences.\r\n\r\nHASSET has been compiled at the UK Data Archive for over thirty years. It was developed initially as an information retrieval tool to access data deposited in the UK Data Archive. Its scope has now expanded to cover the collections held by the UK Data Service.\r\n\r\nCopyright University of Essex. All rights reserved. \r\nYou may view the thesaurus online. \r\nPrior to downloading and/or using the full version of HASSET, SKOS-HASSET or any related product, a licence agreement must be completed between the user and the University of Essex.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GENDER/1/gender.skos",
+            "download_url": "media/ontologies/HASSET/1/hasset.skos",
             "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5545,
+            "home_page": "https://hasset.ukdataservice.ac.uk/",
+            "id": 1947,
             "indexed": true,
             "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
             "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 5,
+            "nb_individuals": 7610,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "2.00"
         },
         "topics": null
     },
     {
-        "acronym": "HUGO",
-        "name": "Human Genome Organization Gene Symbols",
+        "acronym": "HCIOTO",
+        "name": "HealthCareOntology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 866,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2010-12-21T16:27:14Z",
-            "date_released": "2010-01-07T08:00:00Z",
-            "description": "Human Genome Organisation (HUGO) Gene Nomenclature Committee at the European Bioinformatics Institute approves a gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database.",
-            "documentation": "http://www.genenames.org/",
+            "date_created": "2018-12-05T18:40:59Z",
+            "date_released": "2018-12-05T00:00:00Z",
+            "description": "A sensor based health care ontology",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HUGO/3/hugo.owl",
+            "download_url": "media/ontologies/HCIOTO/1/hcioto.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.genenames.org/",
-            "id": 257,
+            "home_page": null,
+            "id": 2075,
             "indexed": true,
-            "max_children": 19361,
-            "max_depth": 2,
+            "max_children": 6,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 32918,
+            "nb_classes": 49,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 8,
             "products": null,
-            "publication": "http://www.genenames.org/",
+            "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "July2010"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "HUPSON",
         "name": "Human Physiology Simulation Ontology",
         "species": null,
@@ -21036,48 +20673,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.1.1"
         },
         "topics": null
     },
     {
-        "acronym": "ISO639-1",
-        "name": "ISO 639-1: Codes for the Representation of Names of Languages - Part 1: Two-letter codes for languages",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2020-10-19T19:42:28Z",
-            "date_released": "2020-10-19T00:00:00Z",
-            "description": "Language codes from ISO 639-1, as expressed in https://www.w3schools.com/tags/ref_language_codes.asp and converted to SKOS using http://labs.sparna.fr/skos-play/convert. ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ISO639-1/1/iso639-1.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5858,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 190,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0.0"
-        },
-        "topics": null
-    },
-    {
         "acronym": "SIBO",
         "name": "Social Insect Behavior Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 16,
             "classifiable": true,
@@ -21147,14 +20750,91 @@
             "submission_id": 93,
             "taxon": null,
             "version": "1.59"
         },
         "topics": []
     },
     {
+        "acronym": "HSAPDV",
+        "name": "Human Developmental Stages",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 85,
+            "classifiable": true,
+            "date_created": "2020-03-16T16:46:21.984399Z",
+            "date_released": "2020-03-16T16:46:21.984377Z",
+            "description": "Life cycle stages for Human",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HSAPDV/11/hsapdv.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obophenotype/developmental-stage-ontologies/wiki/HsapDv",
+            "id": 3439,
+            "indexed": true,
+            "max_children": 242,
+            "max_depth": 2,
+            "md5sum": "52181d59df84578ed69214a5cb614036",
+            "nb_classes": 245,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 4,
+            "products": [
+                {
+                    "id": "hsapdv.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/hsapdv.owl"
+                },
+                {
+                    "id": "hsapdv.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/hsapdv.obo"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 11,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "HAMIDEHSGH",
+        "name": "Inherited Retinal Dystrophy",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2019-07-11T01:50:19Z",
+            "date_released": "2019-07-10T00:00:00Z",
+            "description": "Inherited retinal dystrophy (IRD) as one of the rare ocular diseases encompasses different types of hereditary retinal degenerations appearing by involvement of the different layers of the posterior ocular segment. In respect of the importance of the classification systems regarding the recorded history of biologic and natural sciences, there is no classification regarding the IRD diagnoses. Therefore, we aimed to present a novel classification in this regard.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/HAMIDEHSGH/1/hamidehsgh.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 2746,
+            "indexed": true,
+            "max_children": 76,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 407,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "SITBAC",
         "name": "Situation-Based Access Control Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -21181,350 +20861,364 @@
             "submission_id": 3,
             "taxon": null,
             "version": "1.3"
         },
         "topics": null
     },
     {
-        "acronym": "HCPCS-mod",
-        "name": "HCPCS modifiers",
+        "acronym": "skos",
+        "name": "SKOS (concepts)",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2012-09-18T19:17:54Z",
-            "date_released": "2012-09-18T07:00:00Z",
-            "description": "Modifiers denote that a certain procedure/service has been altered by a particular circumstance, but not changed in its definition, therefore the same code is used and a modifier is added to denote what has been altered.",
+            "date_created": "2012-08-31T10:33:06Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-1.5.owl.  It contains terms relating to concepts from the  http://www.w3.org/2004/02/skos/core# namespace of the SKOS (Simple Knowledge Organization System) ontology that are included in the vivo ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HCPCS-mod/1/hcpcs-mod.owl",
+            "download_url": "media/ontologies/skos/1/skos.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 588,
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 36,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 1,
+            "max_children": 1,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 11,
+            "nb_classes": 1,
             "nb_inconsistent": 0,
-            "nb_individuals": 7,
-            "nb_properties": 18,
+            "nb_individuals": 0,
+            "nb_properties": 3,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "unknown"
+            "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "IBIO",
-        "name": "IndianBiodiversity",
+        "acronym": "SMASH",
+        "name": "SMASH Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 89,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-06-19T19:08:47Z",
-            "date_released": "2021-06-19T00:00:00Z",
-            "description": "Conceptualization of knowledge of Indian Biodiversity domain.",
+            "date_created": "2015-06-03T01:14:47Z",
+            "date_released": "2015-06-03T01:14:47Z",
+            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IBIO/1/ibio.owl",
+            "download_url": "media/ontologies/SMASH/6/smash.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4580,
+            "id": 436,
             "indexed": true,
-            "max_children": 108,
+            "max_children": 10,
             "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 362,
-            "nb_inconsistent": 108,
-            "nb_individuals": 3,
-            "nb_properties": 13,
+            "nb_classes": 189,
+            "nb_inconsistent": 2,
+            "nb_individuals": 87,
+            "nb_properties": 144,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "iceci-countermeasures",
-        "name": "iceci-countermeasures",
+        "acronym": "HSO",
+        "name": "Health Surveillance Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2010-05-07T19:46:00Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI PERSONAL COUNTERMEASURES(S.3) and ENVIRONMENTAL COUNTERMEASURES(S.4)",
+            "date_created": "2021-12-20T13:18:30.149730Z",
+            "date_released": "2021-12-20T13:18:30.149714Z",
+            "description": "The health Surveillance Ontology (HSO) focuses on \"surveillance system level data\", that is, data outputs from surveillance activities, such as number of samples collected, cases observed, etc. It aims to support One-Health surveillance, covering animal health, public health and food safety surveillance.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/iceci-countermeasures/2/iceci-countermeasures.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/HSO/11/hso.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 395,
+            "home_page": "https://w3id.org/hso",
+            "id": 4772,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 29,
+            "max_children": 39,
+            "max_depth": 13,
+            "md5sum": "935e7235775b290e8d5cf6500237326e",
+            "nb_classes": 448,
             "nb_inconsistent": 0,
-            "nb_individuals": 26,
-            "nb_properties": 7,
-            "products": null,
+            "nb_individuals": 192,
+            "nb_properties": 149,
+            "products": [
+                {
+                    "homepage": "https://w3id.org/hso",
+                    "id": "hso.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/hso.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 11,
             "taxon": null,
-            "version": "1.02"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "iceci-descriptor_for_intent",
-        "name": "iceci-descriptor_for_intent",
+        "acronym": "SOPHARM",
+        "name": "Suggested Ontology for Pharmacogenomics",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2010-05-10T22:17:13Z",
-            "date_released": "2010-05-10T07:00:00Z",
-            "description": "ICECI Descriptors for Intent V.1 - V.7",
-            "documentation": null,
+            "date_created": "2017-03-02T16:12:57Z",
+            "date_released": "2017-03-02T00:00:00Z",
+            "description": "SO-Pharm (Suggested Ontology for Pharmacogenomics) is a domain ontology implemented in OWL-DL, which proposes a formal description of pharmacogenomic knowledge. SO-Pharm articulates different ontologies that represent complementary sub-domains of pharmacogenomics, i.e. related to genotype, phenotype, drugs, and clinical trials. SO-Pharm enables to represent pharmacogenomic relationships between a drug, a genomic variation and a phenotype trait. In addition, it enables to represent patient and more largely panel included in trials, and populations. SO-Pharm enables the representation of measured items on this patients such as results from the observation of a phenotype trait or of genomic variations. SO-Pharm supports knowledge about pharmacogenomic hypothesis, case study, and investigations in pharmacogenomics. SO-Pharm is designed to facilitate data integration and knowledge discovery in pharmacogenomics. In addition it provides a consistent articulation of ontologies of pharmacogenomic sub-domains.",
+            "documentation": "https://members.loria.fr/ACoulet/files/ontology/sopharm/version2.0/foundry_requirements.php",
             "domain": null,
-            "download_url": "media/ontologies/iceci-descriptor_for_intent/3/iceci-descriptor_for_intent.owl",
+            "download_url": "media/ontologies/SOPHARM/18/sopharm.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 273,
+            "home_page": "https://members.loria.fr/ACoulet/files/sopharm2.0_description.php",
+            "id": 692,
             "indexed": true,
-            "max_children": 9,
+            "max_children": 51,
             "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 217,
+            "nb_classes": 117,
             "nb_inconsistent": 0,
-            "nb_individuals": 214,
-            "nb_properties": 7,
+            "nb_individuals": 17,
+            "nb_properties": 61,
             "products": null,
-            "publication": null,
+            "publication": "http://hal.inria.fr/inria-00089824/en/",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 18,
             "taxon": null,
-            "version": "1.02"
+            "version": "The new release (March 02, 2017) 2.2 includes concept and properties labels in French.\nImports have been abandoned in favor of simple references to distant URI"
         },
         "topics": null
     },
     {
-        "acronym": "iceci-details_for_activity",
-        "name": "iceci-details_for_activity",
+        "acronym": "SOS",
+        "name": "emerg\u00eancia",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 11,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2010-05-10T22:19:04Z",
-            "date_released": "2010-05-10T07:00:00Z",
-            "description": "ICICI Details for activity O1-O2 and S1-S2",
+            "date_created": "2021-09-20T17:41:36Z",
+            "date_released": "2021-09-20T00:00:00Z",
+            "description": "Um teste",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-details_for_activity/3/iceci-details_for_activity.owl",
+            "download_url": "media/ontologies/SOS/1/sos.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 33,
+            "id": 4594,
             "indexed": true,
-            "max_children": 34,
-            "max_depth": 3,
+            "max_children": 6,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 333,
+            "nb_classes": 64,
             "nb_inconsistent": 0,
-            "nb_individuals": 330,
-            "nb_properties": 7,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.02"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "iceci-details_for_mechanism",
-        "name": "iceci-details_for_mechanism",
+        "acronym": "SOY",
+        "name": "Soy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2010-05-07T20:57:20Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI Details for Mechanism T1 - T4",
+            "date_created": "2013-05-01T01:23:44Z",
+            "date_released": "2013-04-30T07:00:00Z",
+            "description": "Growth, trait and development ontology for soybean",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-details_for_mechanism/2/iceci-details_for_mechanism.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 118,
+            "download_url": "media/ontologies/SOY/2/soy.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "soybase.org",
+            "id": 402,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 4,
+            "max_children": 64,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 159,
+            "nb_classes": 1835,
             "nb_inconsistent": 0,
-            "nb_individuals": 156,
-            "nb_properties": 7,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.02"
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "iceci-details_for_place_of_occurrence",
-        "name": "iceci-details_for_place_of_occurrence",
+        "acronym": "SPD",
+        "name": "Spider Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2010-05-07T20:58:40Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI Details for place of occurrence P1-P7",
+            "date_created": "2019-03-04T15:46:08.307620Z",
+            "date_released": "2019-03-04T15:46:08.307595Z",
+            "description": "An ontology for spider comparative biology including anatomical parts (e.g. leg, claw), behavior (e.g. courtship, combing) and products (i.g. silk, web, borrow).",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/iceci-details_for_place_of_occurrence/2/iceci-details_for_place_of_occurrence.owl",
+            "domain": "anatomy",
+            "download_url": "media/ontologies/SPD/25/spd.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 397,
+            "home_page": "http://research.amnh.org/atol/files/",
+            "id": 2323,
             "indexed": true,
-            "max_children": 24,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 61,
+            "max_children": 278,
+            "max_depth": 10,
+            "md5sum": "b8dd96c7e895da699883f9101956e9ca",
+            "nb_classes": 832,
             "nb_inconsistent": 0,
-            "nb_individuals": 58,
-            "nb_properties": 7,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 4,
+            "products": [
+                {
+                    "id": "spd.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/spd.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "1.02"
+            "submission_id": 25,
+            "taxon": {
+                "id": "NCBITaxon:6893",
+                "label": "spiders"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "iceci-instrument_object_substance",
-        "name": "iceci-instrument_object_substance",
+        "acronym": "SPINALH",
+        "name": "NeuroMorpho.Org spinal cord regions ",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2010-05-07T20:59:44Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI Object/substance producing injury C3",
+            "date_created": "2017-03-31T01:10:37Z",
+            "date_released": "2017-03-31T01:10:37Z",
+            "description": "A hierarchy of Spinal Cord regions in Bilateral species.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-instrument_object_substance/2/iceci-instrument_object_substance.owl",
+            "download_url": "media/ontologies/SPINALH/3/spinalh.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 347,
+            "id": 542,
             "indexed": true,
-            "max_children": 26,
-            "max_depth": 4,
+            "max_children": 5,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 1085,
+            "nb_classes": 11,
             "nb_inconsistent": 0,
-            "nb_individuals": 1082,
-            "nb_properties": 7,
+            "nb_individuals": 11,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.02"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "iceci-intent",
-        "name": "iceci-intent",
+        "acronym": "ISO639-1",
+        "name": "ISO 639-1: Codes for the Representation of Names of Languages - Part 1: Two-letter codes for languages",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2010-05-07T21:08:55Z",
-            "date_released": "2010-04-16T07:00:00Z",
-            "description": "ICECI Intent C1",
+            "date_created": "2020-10-19T19:42:28Z",
+            "date_released": "2020-10-19T00:00:00Z",
+            "description": "Language codes from ISO 639-1, as expressed in https://www.w3schools.com/tags/ref_language_codes.asp and converted to SKOS using http://labs.sparna.fr/skos-play/convert. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-intent/2/iceci-intent.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/ISO639-1/1/iso639-1.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 445,
+            "id": 5858,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 3,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 15,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 12,
-            "nb_properties": 7,
+            "nb_individuals": 190,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.02"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "iceci-place_of_occurrence",
-        "name": "iceci-place_of_occurrence",
+        "acronym": "HCPCS-mod",
+        "name": "HCPCS modifiers",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2010-05-07T21:16:24Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI Place of occurrence C4",
+            "date_created": "2012-09-18T19:17:54Z",
+            "date_released": "2012-09-18T07:00:00Z",
+            "description": "Modifiers denote that a certain procedure/service has been altered by a particular circumstance, but not changed in its definition, therefore the same code is used and a modifier is added to denote what has been altered.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-place_of_occurrence/2/iceci-place_of_occurrence.owl",
+            "download_url": "media/ontologies/HCPCS-mod/1/hcpcs-mod.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 634,
+            "id": 588,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 3,
+            "max_children": 10,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 92,
+            "nb_classes": 11,
             "nb_inconsistent": 0,
-            "nb_individuals": 89,
-            "nb_properties": 7,
+            "nb_individuals": 7,
+            "nb_properties": 18,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.02"
+            "version": "unknown"
         },
         "topics": null
     },
     {
         "acronym": "ICEO",
         "name": "Integrative and Conjugative Element Ontology",
         "species": null,
@@ -21560,14 +21254,82 @@
             "submission_id": 22,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "iceci-activity",
+        "name": "iceci-activity",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2010-05-07T19:37:45Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "ICECI ACTIVITY WHEN INJURED(C.5)",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/iceci-activity/2/iceci-activity.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 466,
+            "indexed": true,
+            "max_children": 10,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 32,
+            "nb_inconsistent": 0,
+            "nb_individuals": 29,
+            "nb_properties": 7,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.02"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ICF",
+        "name": "International Classification of Functioning, Disability and Health",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2012-05-08T23:07:03Z",
+            "date_released": "2012-05-08T07:00:00Z",
+            "description": "The International Classification of Functioning, Disability and Health, known more commonly as ICF, is a classification of health and health-related domains. These domains are classified from body, individual and societal perspectives by means of two lists: a list of body functions and structure, and a list of domains of activity and participation. Since an individual's functioning and disability occurs in a context, the ICF also includes a list of environmental factors.\r\n<br><br>\r\nSource: <a href=\"http://www.who.int/classifications/icf/en/\">http://www.who.int/classifications/icf/en/</a>",
+            "documentation": "http://www.who.int/classifications/icf/en/",
+            "domain": null,
+            "download_url": "media/ontologies/ICF/3/icf.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.who.int/classifications/icf/en/",
+            "id": 96,
+            "indexed": true,
+            "max_children": 20,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 1596,
+            "nb_inconsistent": 0,
+            "nb_individuals": 4807,
+            "nb_properties": 67,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "1.0.2"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ONTODM-CORE",
         "name": "Ontology of Core Data Mining Entities",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -21594,48 +21356,14 @@
             "submission_id": 8,
             "taxon": null,
             "version": "1.0R"
         },
         "topics": null
     },
     {
-        "acronym": "ICD10CM",
-        "name": "International Classification of Diseases, Version 10 - Clinical Modification",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2017-02-06T21:25:46Z",
-            "date_released": "2017-02-06T21:25:46Z",
-            "description": "International Classification of Diseases, 10th Edition, Clinical Modification, 2011_01",
-            "documentation": "http://www.cms.hhs.gov/",
-            "domain": null,
-            "download_url": "media/ontologies/ICD10CM/11/icd10cm.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.cms.hhs.gov/",
-            "id": 332,
-            "indexed": true,
-            "max_children": 34,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 94276,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 22,
-            "products": null,
-            "publication": "http://www.cms.hhs.gov/",
-            "publications": null,
-            "submission_id": 11,
-            "taxon": null,
-            "version": "2016AB"
-        },
-        "topics": null
-    },
-    {
         "acronym": "FOOD_ONTOLOGY",
         "name": "Food Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -21670,384 +21398,282 @@
         "name": "Data Science",
         "species": [],
         "status": "Unknown",
         "submission": null,
         "topics": []
     },
     {
-        "acronym": "ICD10PCS",
-        "name": "International Classification of Diseases, Version 10 - Procedure Coding System",
+        "acronym": "I2SV",
+        "name": "Integration and Implementation Sciences Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-02-06T21:26:01Z",
-            "date_released": "2017-02-06T21:26:01Z",
-            "description": "ICD-10-PCS, 2009",
-            "documentation": "http://www.cms.hhs.gov",
+            "date_created": "2021-04-22T06:37:21Z",
+            "date_released": "2021-04-14T00:00:00Z",
+            "description": "Terms used to describe research concepts, methods and processes for addressing complex societal and environmental problems, especially investigations involving integration across different disciplinary and stakeholder perspectives, as well as implementation of findings into policy and practice change. The terms were originally developed for the Integration and Implementation Insights blog (http://i2Insights.org).",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICD10PCS/10/icd10pcs.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.cms.hhs.gov",
-            "id": 341,
+            "download_url": "media/ontologies/I2SV/5/i2sv.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://i2insights.org/index/integration-and-implementation-sciences-vocabulary/",
+            "id": 4430,
             "indexed": true,
-            "max_children": 33,
-            "max_depth": 6,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 183988,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 19,
+            "nb_individuals": 187,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.cms.hhs.gov",
+            "publication": null,
             "publications": null,
-            "submission_id": 10,
+            "submission_id": 5,
             "taxon": null,
-            "version": "2016AB"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "ICD11-BODYSYSTEM",
-        "name": "Body System Terms from ICD11",
+        "acronym": "I-ADOPT",
+        "name": "I-ADOPT Framework Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2010-04-15T21:54:02Z",
-            "date_released": "2010-04-15T07:00:00Z",
-            "description": "This is a set of body-system terms used in the ICD 11 revision",
-            "documentation": null,
+            "date_created": "2023-07-12T01:06:46Z",
+            "date_released": "2023-07-12T01:06:44Z",
+            "description": "The I-ADOPT Framework is an ontology designed to facilitate interoperability between existing variable description models (including ontologies, taxonomy, and structured controlled vocabularies). One of the challenges in representing semantic descriptions of variables is getting people to agree about what they mean when describing the components that define the variables. The I-ADOPT ontology addresses this by providing core components and their relations that can be applied to define machine-interpretable variable descriptions that re-use FAIR vocabulary terms. This first version of the ontology has been developed by a core group of terminology experts and users from the Research Data Alliance (RDA) InteroperAble Descriptions of Observable Property Terminology (I-ADOPT) Working Group.",
+            "documentation": "https://zenodo.org/record/6520132#.YrcMzXbP1D9",
             "domain": null,
-            "download_url": "media/ontologies/ICD11-BODYSYSTEM/1/icd11-bodysystem.owl",
+            "download_url": "media/ontologies/I-ADOPT/5/i-adopt.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 422,
+            "home_page": "https://www.rd-alliance.org/group/interoperable-descriptions-observable-property-terminology-wg-i-adopt-wg/wiki/i-adopt-0",
+            "id": 5955,
             "indexed": true,
-            "max_children": 15,
-            "max_depth": 3,
+            "max_children": 5,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 30,
+            "nb_classes": 5,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 10,
             "products": null,
-            "publication": null,
+            "publication": "http://ceur-ws.org/Vol-2969/paper10-s4biodiv.pdf",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 5,
             "taxon": null,
-            "version": "1"
+            "version": "1.0.3"
         },
         "topics": null
     },
     {
-        "acronym": "ICD9CM-PROC",
-        "name": "ICD9CM Procedures",
+        "acronym": "iceci-details_for_mechanism",
+        "name": "iceci-details_for_mechanism",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
-            "date_created": "2012-09-18T07:34:15Z",
-            "date_released": "2011-09-16T07:00:00Z",
-            "description": "This view contains only the PROCEDURES sub-tree of the ICD9CM ontology.",
+            "date_created": "2010-05-07T20:57:20Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "ICECI Details for Mechanism T1 - T4",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICD9CM-PROC/1/icd9cm-proc.owl",
+            "download_url": "media/ontologies/iceci-details_for_mechanism/2/iceci-details_for_mechanism.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 699,
+            "id": 118,
             "indexed": true,
-            "max_children": 18,
+            "max_children": 17,
             "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 4664,
+            "nb_classes": 159,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 156,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1"
+            "version": "1.02"
         },
         "topics": null
     },
     {
-        "acronym": "ICDO",
-        "name": "International Classification of Diseases Ontology",
+        "acronym": "iceci-details_for_place_of_occurrence",
+        "name": "iceci-details_for_place_of_occurrence",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2022-11-06T01:19:06Z",
-            "date_released": "2022-11-06T01:19:03Z",
-            "description": "A biomedical ontology for logical representation of the terms and relations related to the International Classification of Diseases (ICD)",
+            "date_created": "2010-05-07T20:58:40Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "ICECI Details for place of occurrence P1-P7",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICDO/12/icdo.owl",
+            "download_url": "media/ontologies/iceci-details_for_place_of_occurrence/2/iceci-details_for_place_of_occurrence.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/icdo/ICDO",
-            "id": 5471,
+            "home_page": null,
+            "id": 397,
             "indexed": true,
-            "max_children": 54,
-            "max_depth": 12,
+            "max_children": 24,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 1388,
+            "nb_classes": 61,
             "nb_inconsistent": 0,
-            "nb_individuals": 4,
-            "nb_properties": 233,
+            "nb_individuals": 58,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0.83"
+            "version": "1.02"
         },
         "topics": null
     },
     {
-        "acronym": "ICD-O-3-M",
-        "name": "ICD-O-3-M International Classification of Diseases for Oncology Morphology",
+        "acronym": "iceci-instrument_object_substance",
+        "name": "iceci-instrument_object_substance",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1080,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2019-06-04T11:01:20Z",
-            "date_released": "2019-05-09T00:00:00Z",
-            "description": "ICD-O-3-M International Classification of Diseases for Oncology, Morphology version: ICD-O-3.1 (2011)\r\n",
+            "date_created": "2010-05-07T20:59:44Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "ICECI Object/substance producing injury C3",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICD-O-3-M/3/icd-o-3-m.owl",
+            "download_url": "media/ontologies/iceci-instrument_object_substance/2/iceci-instrument_object_substance.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 2607,
+            "id": 347,
             "indexed": true,
-            "max_children": 1080,
-            "max_depth": 1,
+            "max_children": 26,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 1080,
+            "nb_classes": 1085,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_individuals": 1082,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "1.02"
         },
         "topics": null
     },
     {
-        "acronym": "ICD-O-3-T",
-        "name": "ICD-O-3-T International Classification of Diseases for Oncology Topography",
+        "acronym": "iceci-intent",
+        "name": "iceci-intent",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-06-04T11:03:42Z",
-            "date_released": "2019-05-13T00:00:00Z",
-            "description": "International Classification of Diseases for Oncology - Topography",
+            "date_created": "2010-05-07T21:08:55Z",
+            "date_released": "2010-04-16T07:00:00Z",
+            "description": "ICECI Intent C1",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICD-O-3-T/2/icd-o-3-t.owl",
+            "download_url": "media/ontologies/iceci-intent/2/iceci-intent.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 2614,
+            "id": 445,
             "indexed": true,
-            "max_children": 70,
-            "max_depth": 2,
+            "max_children": 8,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 401,
+            "nb_classes": 15,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_individuals": 12,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "1.02"
         },
         "topics": null
     },
     {
-        "acronym": "iceci-activity",
-        "name": "iceci-activity",
+        "acronym": "iceci-place_of_occurrence",
+        "name": "iceci-place_of_occurrence",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2010-05-07T19:37:45Z",
+            "date_created": "2010-05-07T21:16:24Z",
             "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI ACTIVITY WHEN INJURED(C.5)",
+            "description": "ICECI Place of occurrence C4",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-activity/2/iceci-activity.owl",
+            "download_url": "media/ontologies/iceci-place_of_occurrence/2/iceci-place_of_occurrence.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 466,
+            "id": 634,
             "indexed": true,
-            "max_children": 10,
+            "max_children": 14,
             "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 32,
+            "nb_classes": 92,
             "nb_inconsistent": 0,
-            "nb_individuals": 29,
+            "nb_individuals": 89,
             "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
             "version": "1.02"
         },
         "topics": null
     },
     {
-        "acronym": "HORD",
-        "name": "Holistic Ontology of Rare Diseases",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 9,
-            "classifiable": true,
-            "date_created": "2017-08-02T10:32:49Z",
-            "date_released": "2017-08-02T00:00:00Z",
-            "description": "This ontology describes the bio-psico-social state of a person of rare diseses in a holistic way. It considers several contents such as disease, psychological, social, environmental state of the person.",
-            "documentation": "https://drive.google.com/file/d/0B2unmdSxW85dTXdVUl81TmlRRGc/view",
-            "domain": null,
-            "download_url": "media/ontologies/HORD/1/hord.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://sites.google.com/site/laiasubirats",
-            "id": 365,
-            "indexed": true,
-            "max_children": 3795,
-            "max_depth": 11,
-            "md5sum": null,
-            "nb_classes": 29656,
-            "nb_inconsistent": 0,
-            "nb_individuals": 5199,
-            "nb_properties": 346,
-            "products": null,
-            "publication": "https://sites.google.com/site/laiasubirats/publications",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ICF",
-        "name": "International Classification of Functioning, Disability and Health",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2012-05-08T23:07:03Z",
-            "date_released": "2012-05-08T07:00:00Z",
-            "description": "The International Classification of Functioning, Disability and Health, known more commonly as ICF, is a classification of health and health-related domains. These domains are classified from body, individual and societal perspectives by means of two lists: a list of body functions and structure, and a list of domains of activity and participation. Since an individual's functioning and disability occurs in a context, the ICF also includes a list of environmental factors.\r\n<br><br>\r\nSource: <a href=\"http://www.who.int/classifications/icf/en/\">http://www.who.int/classifications/icf/en/</a>",
-            "documentation": "http://www.who.int/classifications/icf/en/",
-            "domain": null,
-            "download_url": "media/ontologies/ICF/3/icf.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.who.int/classifications/icf/en/",
-            "id": 96,
-            "indexed": true,
-            "max_children": 20,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 1596,
-            "nb_inconsistent": 0,
-            "nb_individuals": 4807,
-            "nb_properties": 67,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "1.0.2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "I2SV",
-        "name": "Integration and Implementation Sciences Vocabulary",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-04-22T06:37:21Z",
-            "date_released": "2021-04-14T00:00:00Z",
-            "description": "Terms used to describe research concepts, methods and processes for addressing complex societal and environmental problems, especially investigations involving integration across different disciplinary and stakeholder perspectives, as well as implementation of findings into policy and practice change. The terms were originally developed for the Integration and Implementation Insights blog (http://i2Insights.org).",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/I2SV/5/i2sv.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://i2insights.org/index/integration-and-implementation-sciences-vocabulary/",
-            "id": 4430,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 187,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 5,
-            "taxon": null,
-            "version": "1.0.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "I-ADOPT",
-        "name": "I-ADOPT Framework Ontology",
+        "acronym": "ICD10PCS",
+        "name": "International Classification of Diseases, Version 10 - Procedure Coding System",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2023-07-12T01:06:46Z",
-            "date_released": "2023-07-12T01:06:44Z",
-            "description": "The I-ADOPT Framework is an ontology designed to facilitate interoperability between existing variable description models (including ontologies, taxonomy, and structured controlled vocabularies). One of the challenges in representing semantic descriptions of variables is getting people to agree about what they mean when describing the components that define the variables. The I-ADOPT ontology addresses this by providing core components and their relations that can be applied to define machine-interpretable variable descriptions that re-use FAIR vocabulary terms. This first version of the ontology has been developed by a core group of terminology experts and users from the Research Data Alliance (RDA) InteroperAble Descriptions of Observable Property Terminology (I-ADOPT) Working Group.",
-            "documentation": "https://zenodo.org/record/6520132#.YrcMzXbP1D9",
+            "date_created": "2017-02-06T21:26:01Z",
+            "date_released": "2017-02-06T21:26:01Z",
+            "description": "ICD-10-PCS, 2009",
+            "documentation": "http://www.cms.hhs.gov",
             "domain": null,
-            "download_url": "media/ontologies/I-ADOPT/5/i-adopt.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.rd-alliance.org/group/interoperable-descriptions-observable-property-terminology-wg-i-adopt-wg/wiki/i-adopt-0",
-            "id": 5955,
+            "download_url": "media/ontologies/ICD10PCS/10/icd10pcs.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.cms.hhs.gov",
+            "id": 341,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 1,
+            "max_children": 33,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 5,
+            "nb_classes": 183988,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 10,
+            "nb_properties": 19,
             "products": null,
-            "publication": "http://ceur-ws.org/Vol-2969/paper10-s4biodiv.pdf",
+            "publication": "http://www.cms.hhs.gov",
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 10,
             "taxon": null,
-            "version": "1.0.3"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
         "acronym": "IAML-MOP",
         "name": "IAML Medium of Performance Vocabulary",
         "species": null,
@@ -22138,350 +21764,418 @@
             "submission_id": 11,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "IBD",
-        "name": "Indian Biodiversity Ontology",
+        "acronym": "ICD10CM",
+        "name": "International Classification of Diseases, Version 10 - Clinical Modification",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 89,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-06-04T09:17:50Z",
-            "date_released": "2021-06-04T00:00:00Z",
-            "description": "Indian Biodiversity Ontology contains the semantic information about the domain of Indian Biodiversity ",
+            "date_created": "2017-02-06T21:25:46Z",
+            "date_released": "2017-02-06T21:25:46Z",
+            "description": "International Classification of Diseases, 10th Edition, Clinical Modification, 2011_01",
+            "documentation": "http://www.cms.hhs.gov/",
+            "domain": null,
+            "download_url": "media/ontologies/ICD10CM/11/icd10cm.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.cms.hhs.gov/",
+            "id": 332,
+            "indexed": true,
+            "max_children": 34,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 94276,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 22,
+            "products": null,
+            "publication": "http://www.cms.hhs.gov/",
+            "publications": null,
+            "submission_id": 11,
+            "taxon": null,
+            "version": "2016AB"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ICD11-BODYSYSTEM",
+        "name": "Body System Terms from ICD11",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2010-04-15T21:54:02Z",
+            "date_released": "2010-04-15T07:00:00Z",
+            "description": "This is a set of body-system terms used in the ICD 11 revision",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IBD/1/ibd.owl",
+            "download_url": "media/ontologies/ICD11-BODYSYSTEM/1/icd11-bodysystem.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4593,
+            "id": 422,
             "indexed": true,
-            "max_children": 108,
-            "max_depth": 7,
+            "max_children": 15,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 362,
-            "nb_inconsistent": 108,
-            "nb_individuals": 3,
-            "nb_properties": 13,
+            "nb_classes": 30,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "INCENTIVE",
-        "name": "INCENTIVE Community Controlled Vocabulary",
+        "acronym": "ICD9CM-PROC",
+        "name": "ICD9CM Procedures",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
-            "date_created": "2022-06-28T06:42:30Z",
-            "date_released": "2022-06-28T00:00:00Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is INCENTIVE Controlled Vocabulary. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 18 and 22 (M4M.18 and M4M.22) provided to the INCENTIVE community.\r\n\r\nsheet2rdf and OntoStack are used to build and serve INCENTIVE Controlled Vocabulary, while PURL is used to persist identifiers for the vocabulary terms:\r\n\r\nhttp://purl.org/incentive/",
-            "documentation": "https://github.com/fair-data-collective/INCENTIVE-Controlled-Vocabulary",
+            "date_created": "2012-09-18T07:34:15Z",
+            "date_released": "2011-09-16T07:00:00Z",
+            "description": "This view contains only the PROCEDURES sub-tree of the ICD9CM ontology.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INCENTIVE/5/incentive.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/INCENTIVE-Controlled-Vocabulary",
-            "id": 5197,
+            "download_url": "media/ontologies/ICD9CM-PROC/1/icd9cm-proc.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 699,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 1,
+            "max_children": 18,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 10,
+            "nb_classes": 4664,
             "nb_inconsistent": 0,
-            "nb_individuals": 83,
-            "nb_properties": 11,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://github.com/fair-data-collective/INCENTIVE-Controlled-Vocabulary",
+            "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.1.3"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "INCENTIVE-VARS",
-        "name": "INCENTIVE Variables",
+        "acronym": "IBD",
+        "name": "Indian Biodiversity Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 89,
             "classifiable": true,
-            "date_created": "2022-06-29T01:08:57Z",
-            "date_released": "2022-06-29T01:08:56Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is INCENTIVE Variables. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 18 (M4M.18) provided to the INCENTIVE community (https://zenodo.org/communities/incentive/about/).\r\n\r\nsheet2rdf and OntoStack, developed by FAIR Data Collective, are used to build and serve INCENTIVE Variable, while PURL is used to persist identifiers for the vocabulary terms and properties:\r\n\r\nhttp://purl.org/incentive/variables/",
-            "documentation": "https://github.com/fair-data-collective/M4M18-vocabulary",
+            "date_created": "2021-06-04T09:17:50Z",
+            "date_released": "2021-06-04T00:00:00Z",
+            "description": "Indian Biodiversity Ontology contains the semantic information about the domain of Indian Biodiversity ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INCENTIVE-VARS/8/incentive-vars.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/M4M18-vocabulary",
-            "id": 5216,
+            "download_url": "media/ontologies/IBD/1/ibd.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4593,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 1,
+            "max_children": 108,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 10,
-            "nb_inconsistent": 0,
-            "nb_individuals": 83,
-            "nb_properties": 11,
+            "nb_classes": 362,
+            "nb_inconsistent": 108,
+            "nb_individuals": 3,
+            "nb_properties": 13,
             "products": null,
-            "publication": "https://zenodo.org/communities/incentive/about/",
+            "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.1.0"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "INFECTION_TRANS",
-        "name": "Infection Transmission Ontology",
+        "acronym": "HORD",
+        "name": "Holistic Ontology of Rare Diseases",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 9,
             "classifiable": true,
-            "date_created": "2022-07-20T10:15:24Z",
-            "date_released": "2022-07-13T00:00:00Z",
-            "description": "The Infection Transmission Ontology is developed in order to make transmission data reusable and interoperable. \r\nThe ontology captures every step of analysing data from the transmission process, especially for quantification of trans-\r\nmission parameters. It is applicable to animal and human (hospital) data from both experimental and observational stud-\r\nies.\r\n\r\nThe lack of a common terminology in the infection transmission experiments leads to repetition of costly experiments. By developing a domain ontology we can reuse and combine the transmission data, which leads to the improvement of an estimation of transmission parameters and prediction of outcomes of yet unobserved situations.",
-            "documentation": "https://utrechtuniversity.github.io/summer-fair/",
+            "date_created": "2017-08-02T10:32:49Z",
+            "date_released": "2017-08-02T00:00:00Z",
+            "description": "This ontology describes the bio-psico-social state of a person of rare diseses in a holistic way. It considers several contents such as disease, psychological, social, environmental state of the person.",
+            "documentation": "https://drive.google.com/file/d/0B2unmdSxW85dTXdVUl81TmlRRGc/view",
             "domain": null,
-            "download_url": "media/ontologies/INFECTION_TRANS/2/infection_trans.owl",
+            "download_url": "media/ontologies/HORD/1/hord.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://raw.githubusercontent.com/UtrechtUniversity/summer-fair/master/src/create_ontology/map_ontology/infection_trans.owl#",
-            "id": 5250,
+            "home_page": "https://sites.google.com/site/laiasubirats",
+            "id": 365,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 2,
+            "max_children": 3795,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 22,
+            "nb_classes": 29656,
             "nb_inconsistent": 0,
-            "nb_individuals": 11,
-            "nb_properties": 21,
+            "nb_individuals": 5199,
+            "nb_properties": 346,
             "products": null,
-            "publication": null,
+            "publication": "https://sites.google.com/site/laiasubirats/publications",
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.0.2"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "INFRARISK",
-        "name": "InfraRisk Ontology",
+        "acronym": "ICDO",
+        "name": "International Classification of Diseases Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-02-11T12:50:55Z",
-            "date_released": "2016-03-01T00:00:00Z",
-            "description": "The InfraRisk Ontology: Enabling Semantic Interoperability for Critical Infrastructures at Risk from Natural Hazards. This InfraRisk ontology may be used for representing relevant information about natural hazard events and their impact on infrastructure components. Earthquakes, landslides, and other natural hazard events have severe negative socio-economic impacts.",
-            "documentation": "http://dx.doi.org/10.1007/978-3-319-69459-7_31",
+            "date_created": "2022-11-06T01:19:06Z",
+            "date_released": "2022-11-06T01:19:03Z",
+            "description": "A biomedical ontology for logical representation of the terms and relations related to the International Classification of Diseases (ICD)",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INFRARISK/1/infrarisk.owl",
+            "download_url": "media/ontologies/ICDO/12/icdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.infrarisk-fp7.eu",
-            "id": 4274,
+            "home_page": "https://github.com/icdo/ICDO",
+            "id": 5471,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 5,
+            "max_children": 54,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 33,
+            "nb_classes": 1388,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 12,
+            "nb_individuals": 4,
+            "nb_properties": 233,
             "products": null,
-            "publication": "http://dx.doi.org/10.1007/978-3-319-69459-7_31",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 12,
             "taxon": null,
-            "version": "1.0"
+            "version": "1.0.83"
         },
         "topics": null
     },
     {
-        "acronym": "INJURYTYPES",
-        "name": "Injury Types",
+        "acronym": "ICD-O-3-M",
+        "name": "ICD-O-3-M International Classification of Diseases for Oncology Morphology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 1080,
             "classifiable": true,
-            "date_created": "2022-12-07T20:09:23Z",
-            "date_released": "2022-12-07T00:00:00Z",
-            "description": "This class is associated with the different injury types.",
+            "date_created": "2019-06-04T11:01:20Z",
+            "date_released": "2019-05-09T00:00:00Z",
+            "description": "ICD-O-3-M International Classification of Diseases for Oncology, Morphology version: ICD-O-3.1 (2011)\r\n",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INJURYTYPES/1/injurytypes.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/ICD-O-3-M/3/icd-o-3-m.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5548,
+            "id": 2607,
             "indexed": true,
-            "max_children": 2,
+            "max_children": 1080,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 1080,
             "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": null
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "INM-NPI",
-        "name": "Interventionnelle Non M\u00e9dicamenteuse (INM)",
+        "acronym": "ICD-O-3-T",
+        "name": "ICD-O-3-T International Classification of Diseases for Oncology Topography",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
-            "date_created": "2016-07-16T15:11:58Z",
-            "date_released": "2016-02-16T00:00:00Z",
-            "description": "Interventionnelle Non M\u00e9dicamenteuse, Non-Pharmacological Interventions",
+            "date_created": "2019-06-04T11:03:42Z",
+            "date_released": "2019-05-13T00:00:00Z",
+            "description": "International Classification of Diseases for Oncology - Topography",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INM-NPI/3/inm-npi.owl",
+            "download_url": "media/ontologies/ICD-O-3-T/2/icd-o-3-t.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.cepsplatform.eu",
-            "id": 285,
+            "home_page": null,
+            "id": 2614,
             "indexed": true,
-            "max_children": 30,
-            "max_depth": 4,
+            "max_children": 70,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 160,
+            "nb_classes": 401,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2.0"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "INSECTH",
-        "name": "insectH",
+        "acronym": "IBIO",
+        "name": "IndianBiodiversity",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 89,
             "classifiable": true,
-            "date_created": "2017-11-28T02:10:09Z",
-            "date_released": "2017-11-28T02:10:09Z",
-            "description": "NeuroMorpho.Org insect brain regions ontology is compiled to create data annotation standard for depositing insect data. Currently at v6.2, NeuroMorpho.Org hosts over 17k neuronal reconstructions mostly from the fly. The insect ontology follows the standard nomenclature of the insect brain (Ito et al., 2013) and the virtualfly.org.  ",
+            "date_created": "2021-06-19T19:08:47Z",
+            "date_released": "2021-06-19T00:00:00Z",
+            "description": "Conceptualization of knowledge of Indian Biodiversity domain.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INSECTH/4/insecth.owl",
+            "download_url": "media/ontologies/IBIO/1/ibio.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 872,
+            "id": 4580,
             "indexed": true,
-            "max_children": 39,
-            "max_depth": 8,
+            "max_children": 108,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 119,
-            "nb_inconsistent": 0,
-            "nb_individuals": 130,
-            "nb_properties": 0,
+            "nb_classes": 362,
+            "nb_inconsistent": 108,
+            "nb_individuals": 3,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "BIN",
-        "name": "Body in Numbers project terminology",
+        "acronym": "iceci-countermeasures",
+        "name": "iceci-countermeasures",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2019-05-01T16:31:46Z",
-            "date_released": "2019-05-02T00:00:00Z",
-            "description": "This ontology was created as a part of V\u00edt\u011bzslav Vacek's masters thesis and is describing the terminology used within the Body in Numbers (BiN) project that is currently underway (as of 2019-05-01) at the University of West Bohemia, Czech Republic and focuses on the terminology used during the data collection process.",
+            "date_created": "2010-05-07T19:46:00Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "ICECI PERSONAL COUNTERMEASURES(S.3) and ENVIRONMENTAL COUNTERMEASURES(S.4)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BIN/2/bin.owl",
+            "download_url": "media/ontologies/iceci-countermeasures/2/iceci-countermeasures.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 2526,
+            "id": 395,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 11,
+            "max_children": 16,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 138,
+            "nb_classes": 29,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 26,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": "1.02"
         },
         "topics": null
     },
     {
-        "acronym": "BIOBIPON",
-        "name": "Biological part of Bacterial interlocked Process ONtology",
+        "acronym": "iceci-descriptor_for_intent",
+        "name": "iceci-descriptor_for_intent",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2017-04-05T08:49:26Z",
-            "date_released": "2017-04-05T00:00:00Z",
-            "description": "bioBiPON (a sub-ontology of BiPON) aims at organizing the systemic description of\r\nbiological information",
+            "date_created": "2010-05-10T22:17:13Z",
+            "date_released": "2010-05-10T07:00:00Z",
+            "description": "ICECI Descriptors for Intent V.1 - V.7",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BIOBIPON/2/biobipon.owl",
+            "download_url": "media/ontologies/iceci-descriptor_for_intent/3/iceci-descriptor_for_intent.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 270,
+            "id": 273,
             "indexed": true,
-            "max_children": 25,
-            "max_depth": 15,
+            "max_children": 9,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 1556,
+            "nb_classes": 217,
             "nb_inconsistent": 0,
-            "nb_individuals": 767,
-            "nb_properties": 29,
+            "nb_individuals": 214,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 3,
             "taxon": null,
-            "version": null
+            "version": "1.02"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "iceci-details_for_activity",
+        "name": "iceci-details_for_activity",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 11,
+            "classifiable": true,
+            "date_created": "2010-05-10T22:19:04Z",
+            "date_released": "2010-05-10T07:00:00Z",
+            "description": "ICICI Details for activity O1-O2 and S1-S2",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/iceci-details_for_activity/3/iceci-details_for_activity.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 33,
+            "indexed": true,
+            "max_children": 34,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 333,
+            "nb_inconsistent": 0,
+            "nb_individuals": 330,
+            "nb_properties": 7,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "1.02"
         },
         "topics": null
     },
     {
         "acronym": "BIOLINK",
         "name": "Biolink Model",
         "species": null,
@@ -22852,14 +22546,82 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
+        "acronym": "ICF-d4",
+        "name": "ICF-d4. MOBILITY",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2010-05-04T22:35:33Z",
+            "date_released": "2010-05-04T07:00:00Z",
+            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 4: MOBILITY\" (d4) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
+            "documentation": "http://www.who.int/classifications/icf/en/",
+            "domain": null,
+            "download_url": "media/ontologies/ICF-d4/1/icf-d4.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 658,
+            "indexed": true,
+            "max_children": 10,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 126,
+            "nb_inconsistent": 0,
+            "nb_individuals": 2511,
+            "nb_properties": 65,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ICF-d5",
+        "name": "ICF-d5. SELF-CARE",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2010-05-04T22:36:49Z",
+            "date_released": "2010-05-04T07:00:00Z",
+            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 5: SELF-CARE\" (d5) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
+            "documentation": "http://www.who.int/classifications/icf/en/",
+            "domain": null,
+            "download_url": "media/ontologies/ICF-d5/1/icf-d5.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 55,
+            "indexed": true,
+            "max_children": 10,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 66,
+            "nb_inconsistent": 0,
+            "nb_individuals": 2451,
+            "nb_properties": 65,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "MATRELEMENT",
         "name": "Material Element",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -22886,78 +22648,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2.3"
         },
         "topics": null
     },
     {
-        "acronym": "ICF-d4",
-        "name": "ICF-d4. MOBILITY",
+        "acronym": "IMGT-ONTOLOGY",
+        "name": "Immunogenetics Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2010-05-04T22:35:33Z",
-            "date_released": "2010-05-04T07:00:00Z",
-            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 4: MOBILITY\" (d4) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
-            "documentation": "http://www.who.int/classifications/icf/en/",
+            "date_created": "2013-07-17T15:30:29Z",
+            "date_released": "2013-07-17T07:00:00Z",
+            "description": "IMGT-ONTOLOGY is the first ontology for immunogenetics and immunoinformatics. It provides a semantic specification of the terms to be used in immunogenetics and immunoinformatics and manages the related knowledge, thus allowing the standardization for immunogenetics data from genome, proteome, genetics, two-dimensional (2D) and three-dimensional (3D) structures.  \r\nIMGT-ONTOLOGY manages the knowledge through diverse facets relying on seven axioms, \"IDENTIFICATION\", \"CLASSIFICATION\", \"DESCRIPTION\", \"NUMEROTATION\", \"LOCALIZATION\", \"ORIENTATION\" and \"OBTENTION\". These axioms postulate that any object, any process and any relation can be identified, classified, described, numbered, localized and orientated, and the way it is obtained can be characterized. The axioms constitute the Formal IMGT-ONTOLOGY, also designated as IMGT-Kaleidoscope. As the same axioms can be used to generate concepts for multi-scale level approaches, the Formal IMGT-ONTOLOGY represents a paradigm for system biology ontologies, which need to identify, to classify, to describe, to number, to localize and to orientate objects, processes and relations at the molecule, cell, tissue, organ, organism or population levels.\r\nIMGT\u00ae, the international ImMunoGeneTics information system\u00ae, has been built on IMGT-ONTOLOGY.\r\n\r\nThe version 1.0.2 of IMGT-ONTOLOGY includes the concepts of IDENTIFICATION and the concepts of CLASSIFICATION.",
+            "documentation": "http://www.imgt.org/IMGTindex/ontology.php",
             "domain": null,
-            "download_url": "media/ontologies/ICF-d4/1/icf-d4.owl",
+            "download_url": "media/ontologies/IMGT-ONTOLOGY/4/imgt-ontology.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 658,
+            "home_page": "http://www.imgt.org/",
+            "id": 311,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 5,
+            "max_children": 61,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 126,
+            "nb_classes": 1415,
             "nb_inconsistent": 0,
-            "nb_individuals": 2511,
-            "nb_properties": 65,
+            "nb_individuals": 0,
+            "nb_properties": 42,
             "products": null,
-            "publication": null,
+            "publication": "http://www.imgt.org/IMGTindex/ontology.php",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "1"
+            "version": "1.0.2"
         },
         "topics": null
     },
     {
-        "acronym": "ICF-d5",
-        "name": "ICF-d5. SELF-CARE",
+        "acronym": "INBIODIV",
+        "name": "An Ontology for Indian Biodiversity Knowledge Management",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 32,
             "classifiable": true,
-            "date_created": "2010-05-04T22:36:49Z",
-            "date_released": "2010-05-04T07:00:00Z",
-            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 5: SELF-CARE\" (d5) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
-            "documentation": "http://www.who.int/classifications/icf/en/",
+            "date_created": "2021-06-25T20:15:51Z",
+            "date_released": "2021-06-25T00:00:00Z",
+            "description": "InBiodiv-O: An Ontology for Indian Biodiversity Knowledge Management \r\n",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICF-d5/1/icf-d5.owl",
+            "download_url": "media/ontologies/INBIODIV/1/inbiodiv.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 55,
+            "id": 4645,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 5,
+            "max_children": 51,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 66,
-            "nb_inconsistent": 0,
-            "nb_individuals": 2451,
-            "nb_properties": 65,
+            "nb_classes": 628,
+            "nb_inconsistent": 36,
+            "nb_individuals": 3,
+            "nb_properties": 18,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "1.1"
         },
         "topics": null
     },
     {
         "acronym": "MATR",
         "name": "Material",
         "species": null,
@@ -22988,14 +22750,184 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2.3"
         },
         "topics": null
     },
     {
+        "acronym": "INCENTIVE",
+        "name": "INCENTIVE Community Controlled Vocabulary",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2022-06-28T06:42:30Z",
+            "date_released": "2022-06-28T00:00:00Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is INCENTIVE Controlled Vocabulary. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 18 and 22 (M4M.18 and M4M.22) provided to the INCENTIVE community.\r\n\r\nsheet2rdf and OntoStack are used to build and serve INCENTIVE Controlled Vocabulary, while PURL is used to persist identifiers for the vocabulary terms:\r\n\r\nhttp://purl.org/incentive/",
+            "documentation": "https://github.com/fair-data-collective/INCENTIVE-Controlled-Vocabulary",
+            "domain": null,
+            "download_url": "media/ontologies/INCENTIVE/5/incentive.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/INCENTIVE-Controlled-Vocabulary",
+            "id": 5197,
+            "indexed": true,
+            "max_children": 5,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 10,
+            "nb_inconsistent": 0,
+            "nb_individuals": 83,
+            "nb_properties": 11,
+            "products": null,
+            "publication": "https://github.com/fair-data-collective/INCENTIVE-Controlled-Vocabulary",
+            "publications": null,
+            "submission_id": 5,
+            "taxon": null,
+            "version": "0.1.3"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "INCENTIVE-VARS",
+        "name": "INCENTIVE Variables",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2022-06-29T01:08:57Z",
+            "date_released": "2022-06-29T01:08:56Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is INCENTIVE Variables. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 18 (M4M.18) provided to the INCENTIVE community (https://zenodo.org/communities/incentive/about/).\r\n\r\nsheet2rdf and OntoStack, developed by FAIR Data Collective, are used to build and serve INCENTIVE Variable, while PURL is used to persist identifiers for the vocabulary terms and properties:\r\n\r\nhttp://purl.org/incentive/variables/",
+            "documentation": "https://github.com/fair-data-collective/M4M18-vocabulary",
+            "domain": null,
+            "download_url": "media/ontologies/INCENTIVE-VARS/8/incentive-vars.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/M4M18-vocabulary",
+            "id": 5216,
+            "indexed": true,
+            "max_children": 5,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 10,
+            "nb_inconsistent": 0,
+            "nb_individuals": 83,
+            "nb_properties": 11,
+            "products": null,
+            "publication": "https://zenodo.org/communities/incentive/about/",
+            "publications": null,
+            "submission_id": 8,
+            "taxon": null,
+            "version": "0.1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "INFECTION_TRANS",
+        "name": "Infection Transmission Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2022-07-20T10:15:24Z",
+            "date_released": "2022-07-13T00:00:00Z",
+            "description": "The Infection Transmission Ontology is developed in order to make transmission data reusable and interoperable. \r\nThe ontology captures every step of analysing data from the transmission process, especially for quantification of trans-\r\nmission parameters. It is applicable to animal and human (hospital) data from both experimental and observational stud-\r\nies.\r\n\r\nThe lack of a common terminology in the infection transmission experiments leads to repetition of costly experiments. By developing a domain ontology we can reuse and combine the transmission data, which leads to the improvement of an estimation of transmission parameters and prediction of outcomes of yet unobserved situations.",
+            "documentation": "https://utrechtuniversity.github.io/summer-fair/",
+            "domain": null,
+            "download_url": "media/ontologies/INFECTION_TRANS/2/infection_trans.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://raw.githubusercontent.com/UtrechtUniversity/summer-fair/master/src/create_ontology/map_ontology/infection_trans.owl#",
+            "id": 5250,
+            "indexed": true,
+            "max_children": 17,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 22,
+            "nb_inconsistent": 0,
+            "nb_individuals": 11,
+            "nb_properties": 21,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.0.2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "INFRARISK",
+        "name": "InfraRisk Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2021-02-11T12:50:55Z",
+            "date_released": "2016-03-01T00:00:00Z",
+            "description": "The InfraRisk Ontology: Enabling Semantic Interoperability for Critical Infrastructures at Risk from Natural Hazards. This InfraRisk ontology may be used for representing relevant information about natural hazard events and their impact on infrastructure components. Earthquakes, landslides, and other natural hazard events have severe negative socio-economic impacts.",
+            "documentation": "http://dx.doi.org/10.1007/978-3-319-69459-7_31",
+            "domain": null,
+            "download_url": "media/ontologies/INFRARISK/1/infrarisk.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.infrarisk-fp7.eu",
+            "id": 4274,
+            "indexed": true,
+            "max_children": 7,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 33,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1,
+            "nb_properties": 12,
+            "products": null,
+            "publication": "http://dx.doi.org/10.1007/978-3-319-69459-7_31",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "INJURYTYPES",
+        "name": "Injury Types",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2022-12-07T20:09:23Z",
+            "date_released": "2022-12-07T00:00:00Z",
+            "description": "This class is associated with the different injury types.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/INJURYTYPES/1/injurytypes.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5548,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 9,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "PANDA",
         "name": "Probabilistic Knowledge Assembly Ontology",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -23022,14 +22954,48 @@
             "submission_id": 2,
             "taxon": null,
             "version": "0.9"
         },
         "topics": null
     },
     {
+        "acronym": "INM-NPI",
+        "name": "Interventionnelle Non M\u00e9dicamenteuse (INM)",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2016-07-16T15:11:58Z",
+            "date_released": "2016-02-16T00:00:00Z",
+            "description": "Interventionnelle Non M\u00e9dicamenteuse, Non-Pharmacological Interventions",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/INM-NPI/3/inm-npi.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.cepsplatform.eu",
+            "id": 285,
+            "indexed": true,
+            "max_children": 30,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 160,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 2,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "2.0"
+        },
+        "topics": null
+    },
+    {
         "acronym": "MATRCOMPOUND",
         "name": "Material Compound",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -23056,14 +23022,82 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2.3"
         },
         "topics": null
     },
     {
+        "acronym": "INSECTH",
+        "name": "insectH",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2017-11-28T02:10:09Z",
+            "date_released": "2017-11-28T02:10:09Z",
+            "description": "NeuroMorpho.Org insect brain regions ontology is compiled to create data annotation standard for depositing insect data. Currently at v6.2, NeuroMorpho.Org hosts over 17k neuronal reconstructions mostly from the fly. The insect ontology follows the standard nomenclature of the insect brain (Ito et al., 2013) and the virtualfly.org.  ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/INSECTH/4/insecth.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 872,
+            "indexed": true,
+            "max_children": 39,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 119,
+            "nb_inconsistent": 0,
+            "nb_individuals": 130,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "0.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BIN",
+        "name": "Body in Numbers project terminology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2019-05-01T16:31:46Z",
+            "date_released": "2019-05-02T00:00:00Z",
+            "description": "This ontology was created as a part of V\u00edt\u011bzslav Vacek's masters thesis and is describing the terminology used within the Body in Numbers (BiN) project that is currently underway (as of 2019-05-01) at the University of West Bohemia, Czech Republic and focuses on the terminology used during the data collection process.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/BIN/2/bin.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 2526,
+            "indexed": true,
+            "max_children": 8,
+            "max_depth": 11,
+            "md5sum": null,
+            "nb_classes": 138,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "MATRROCK",
         "name": "Material Rock",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -23133,78 +23167,44 @@
             "submission_id": 5,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "IMGT-ONTOLOGY",
-        "name": "Immunogenetics Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2013-07-17T15:30:29Z",
-            "date_released": "2013-07-17T07:00:00Z",
-            "description": "IMGT-ONTOLOGY is the first ontology for immunogenetics and immunoinformatics. It provides a semantic specification of the terms to be used in immunogenetics and immunoinformatics and manages the related knowledge, thus allowing the standardization for immunogenetics data from genome, proteome, genetics, two-dimensional (2D) and three-dimensional (3D) structures.  \r\nIMGT-ONTOLOGY manages the knowledge through diverse facets relying on seven axioms, \"IDENTIFICATION\", \"CLASSIFICATION\", \"DESCRIPTION\", \"NUMEROTATION\", \"LOCALIZATION\", \"ORIENTATION\" and \"OBTENTION\". These axioms postulate that any object, any process and any relation can be identified, classified, described, numbered, localized and orientated, and the way it is obtained can be characterized. The axioms constitute the Formal IMGT-ONTOLOGY, also designated as IMGT-Kaleidoscope. As the same axioms can be used to generate concepts for multi-scale level approaches, the Formal IMGT-ONTOLOGY represents a paradigm for system biology ontologies, which need to identify, to classify, to describe, to number, to localize and to orientate objects, processes and relations at the molecule, cell, tissue, organ, organism or population levels.\r\nIMGT\u00ae, the international ImMunoGeneTics information system\u00ae, has been built on IMGT-ONTOLOGY.\r\n\r\nThe version 1.0.2 of IMGT-ONTOLOGY includes the concepts of IDENTIFICATION and the concepts of CLASSIFICATION.",
-            "documentation": "http://www.imgt.org/IMGTindex/ontology.php",
-            "domain": null,
-            "download_url": "media/ontologies/IMGT-ONTOLOGY/4/imgt-ontology.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.imgt.org/",
-            "id": 311,
-            "indexed": true,
-            "max_children": 61,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 1415,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 42,
-            "products": null,
-            "publication": "http://www.imgt.org/IMGTindex/ontology.php",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "1.0.2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "INBIODIV",
-        "name": "An Ontology for Indian Biodiversity Knowledge Management",
+        "acronym": "BIOBIPON",
+        "name": "Biological part of Bacterial interlocked Process ONtology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 32,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-06-25T20:15:51Z",
-            "date_released": "2021-06-25T00:00:00Z",
-            "description": "InBiodiv-O: An Ontology for Indian Biodiversity Knowledge Management \r\n",
+            "date_created": "2017-04-05T08:49:26Z",
+            "date_released": "2017-04-05T00:00:00Z",
+            "description": "bioBiPON (a sub-ontology of BiPON) aims at organizing the systemic description of\r\nbiological information",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/INBIODIV/1/inbiodiv.owl",
+            "download_url": "media/ontologies/BIOBIPON/2/biobipon.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4645,
+            "id": 270,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 8,
+            "max_children": 25,
+            "max_depth": 15,
             "md5sum": null,
-            "nb_classes": 628,
-            "nb_inconsistent": 36,
-            "nb_individuals": 3,
-            "nb_properties": 18,
+            "nb_classes": 1556,
+            "nb_inconsistent": 0,
+            "nb_individuals": 767,
+            "nb_properties": 29,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "BRO-Resource",
         "name": "BRO Resource View",
         "species": null,
@@ -23235,14 +23235,48 @@
             "submission_id": 3,
             "taxon": null,
             "version": "3.1"
         },
         "topics": null
     },
     {
+        "acronym": "MINERAL",
+        "name": "Material Mineral",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2016-01-07T20:58:51Z",
+            "date_released": "2016-01-08T00:00:00Z",
+            "description": "Material Mineral ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/MINERAL/1/mineral.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 547,
+            "indexed": true,
+            "max_children": 51,
+            "max_depth": 12,
+            "md5sum": null,
+            "nb_classes": 3553,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1226,
+            "nb_properties": 358,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "2.3"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ICF-d6",
         "name": "ICF-d6. DOMESTIC LIFE",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -23269,14 +23303,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
+        "acronym": "MATRROCKIGNEOUS",
+        "name": "Material Rock Igneouus",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2016-01-07T21:27:36Z",
+            "date_released": "2016-01-08T00:00:00Z",
+            "description": "matrRockIgneous",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/MATRROCKIGNEOUS/2/matrrockigneous.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 606,
+            "indexed": true,
+            "max_children": 51,
+            "max_depth": 12,
+            "md5sum": null,
+            "nb_classes": 3535,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1226,
+            "nb_properties": 358,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "2.3"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ICF-d7",
         "name": "ICF-d7. INTERPERSONAL INTERACTIONS AND RELATIONSHIPS",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -23337,112 +23405,146 @@
             "submission_id": 1,
             "taxon": null,
             "version": "0.1"
         },
         "topics": []
     },
     {
-        "acronym": "MINERAL",
-        "name": "Material Mineral",
+        "acronym": "BMT",
+        "name": "Biomedical Topics",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-01-07T20:58:51Z",
-            "date_released": "2016-01-08T00:00:00Z",
-            "description": "Material Mineral ",
+            "date_created": "2014-10-06T13:21:14Z",
+            "date_released": "2014-10-06T00:00:00Z",
+            "description": "An ontology of biological and biomedical topics, created from merging the EDAM Topics sub-ontology with auxiliary topics that are out of scope of EDAM.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MINERAL/1/mineral.owl",
+            "download_url": "media/ontologies/BMT/1/bmt.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 547,
+            "id": 196,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 12,
+            "max_children": 35,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 3553,
+            "nb_classes": 253,
             "nb_inconsistent": 0,
-            "nb_individuals": 1226,
-            "nb_properties": 358,
+            "nb_individuals": 0,
+            "nb_properties": 12,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2.3"
+            "version": "1.4"
         },
         "topics": null
     },
     {
-        "acronym": "BMT",
-        "name": "Biomedical Topics",
+        "acronym": "IRD3",
+        "name": "Inherited retinal dystrophy",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2014-10-06T13:21:14Z",
-            "date_released": "2014-10-06T00:00:00Z",
-            "description": "An ontology of biological and biomedical topics, created from merging the EDAM Topics sub-ontology with auxiliary topics that are out of scope of EDAM.",
+            "date_created": "2023-01-29T06:08:12Z",
+            "date_released": "2023-01-28T00:00:00Z",
+            "description": "nherited retinal dystrophy (IRD) as one of the rare ocular diseases encompasses different types of hereditary retinal degenerations appearing by involvement of the different layers of the posterior ocular segment. In respect of the importance of the classification systems regarding the recorded history of biologic and natural sciences, there is no classification regarding the IRD diagnoses. Therefore, a novel classification for this type of hereditary ocular disease was developed and presented.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BMT/1/bmt.owl",
+            "download_url": "media/ontologies/IRD3/1/ird3.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 196,
+            "id": 5631,
             "indexed": true,
-            "max_children": 35,
-            "max_depth": 5,
+            "max_children": 78,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 253,
+            "nb_classes": 163,
+            "nb_inconsistent": 0,
+            "nb_individuals": 412,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "3"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "IRDG",
+        "name": "Data harmonization ontology for cross border malaria surveillance ",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2018-10-12T11:14:09Z",
+            "date_released": "2018-10-12T00:00:00Z",
+            "description": "Ontology used to describe the data system in harmonized surveillance system of malaria between french Guiana and Brazil. Developed by the IRD and the Fiocruz.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/IRDG/1/irdg.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 1941,
+            "indexed": true,
+            "max_children": 9,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 49,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 12,
+            "nb_properties": 19,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.4"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MATRROCKIGNEOUS",
-        "name": "Material Rock Igneouus",
+        "acronym": "STO",
+        "name": "Stroke Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 18,
             "classifiable": true,
-            "date_created": "2016-01-07T21:27:36Z",
-            "date_released": "2016-01-08T00:00:00Z",
-            "description": "matrRockIgneous",
+            "date_created": "2018-06-17T13:55:16Z",
+            "date_released": "2018-06-17T00:00:00Z",
+            "description": "Stroke ontology covers the knowledge domain of stroke based on expert views and translational continuum of stroke research. The Stroke Ontology (STO) presented here, is the first draft and we hope that the scientific community can contribute to the enrichment of this draft. The ontology can be used a reference for a variety of bioinformatics applications such as annotation of terms, harmonization of experimental data, text-mining, and modeling disease mechanism. \r\n",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MATRROCKIGNEOUS/2/matrrockigneous.owl",
+            "download_url": "media/ontologies/STO/3/sto.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 606,
+            "id": 1581,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 12,
+            "max_children": 1676,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 3535,
+            "nb_classes": 1712,
             "nb_inconsistent": 0,
-            "nb_individuals": 1226,
-            "nb_properties": 358,
+            "nb_individuals": 69,
+            "nb_properties": 29,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 3,
             "taxon": null,
-            "version": "2.3"
+            "version": "2017-11-01"
         },
         "topics": null
     },
     {
         "acronym": "BMTO",
         "name": "Biomedical Topics Ontology",
         "species": null,
@@ -23541,146 +23643,180 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ICF-d840-d859",
-        "name": "ICF-d840-d859. Work and employment",
+        "acronym": "BNO",
+        "name": "Bionutrition Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2010-05-04T22:51:47Z",
-            "date_released": "2010-05-04T07:00:00Z",
-            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 8: MAJOR LIFE AREAS -> Work and employment\" (d840-d859) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
-            "documentation": "http://www.who.int/classifications/icf/en/",
+            "date_created": "2013-08-15T19:10:25Z",
+            "date_released": "2013-08-08T07:00:00Z",
+            "description": "The Bionutrition Ontology relates concepts and terminologies used for human nutrition in a clinical and biomedical setting.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ICF-d840-d859/1/icf-d840-d859.owl",
+            "download_url": "media/ontologies/BNO/2/bno.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 624,
+            "id": 633,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 5,
+            "max_children": 13,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 44,
+            "nb_classes": 100,
             "nb_inconsistent": 0,
-            "nb_individuals": 2429,
-            "nb_properties": 65,
+            "nb_individuals": 0,
+            "nb_properties": 4,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1"
+            "version": "0.2"
         },
         "topics": null
     },
     {
-        "acronym": "ICF-d860-d879",
-        "name": "ICF-d860-d879. Economic life",
+        "acronym": "BP",
+        "name": "BioPAX Ontology of Biological Pathways",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2010-05-04T22:53:02Z",
-            "date_released": "2010-05-04T07:00:00Z",
-            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 8: MAJOR LIFE AREAS -> Economic life\" (d860-d879) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
-            "documentation": "http://www.who.int/classifications/icf/en/",
+            "date_created": "2010-07-29T13:28:09Z",
+            "date_released": "2010-07-02T07:00:00Z",
+            "description": "The BioPAX ontology (www.biopax.org) is a standard language for formally representing biological pathways. BioPAX Level 3 supports the representation of metabolic pathways, signal transduction pathways, protein-protein interaction networks, gene regulatory networks and genetic interactions.",
+            "documentation": "www.biopax.org/release/biopax-level3-documentation.pdf",
             "domain": null,
-            "download_url": "media/ontologies/ICF-d860-d879/1/icf-d860-d879.owl",
+            "download_url": "media/ontologies/BP/1/bp.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 215,
+            "home_page": "www.biopax.org",
+            "id": 274,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 5,
+            "max_children": 15,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 37,
+            "nb_classes": 69,
             "nb_inconsistent": 0,
-            "nb_individuals": 2422,
-            "nb_properties": 65,
+            "nb_individuals": 0,
+            "nb_properties": 96,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "Level3 v1.0"
         },
         "topics": null
     },
     {
-        "acronym": "ICF-d9",
-        "name": "ICF-d9. COMMUNITY, SOCIAL AND CIVIC LIFE",
+        "acronym": "BRO-Activity",
+        "name": "Biomedical Resource Ontology - Activity View",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2010-05-04T22:44:55Z",
+            "date_created": "2010-06-28T03:17:41Z",
+            "date_released": "2010-05-17T07:00:00Z",
+            "description": "This view represents a subset of the BRO containing all the Activity classes that are relevant in the biomedical domain.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/BRO-Activity/2/bro-activity.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.biositemaps.org/",
+            "id": 703,
+            "indexed": true,
+            "max_children": 13,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 21,
+            "nb_inconsistent": 0,
+            "nb_individuals": 3,
+            "nb_properties": 45,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "3.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ICF-d840-d859",
+        "name": "ICF-d840-d859. Work and employment",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2010-05-04T22:51:47Z",
             "date_released": "2010-05-04T07:00:00Z",
-            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 9: COMMUNITY, SOCIAL AND CIVIC LIFE\" (d9) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
+            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 8: MAJOR LIFE AREAS -> Work and employment\" (d840-d859) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
             "documentation": "http://www.who.int/classifications/icf/en/",
             "domain": null,
-            "download_url": "media/ontologies/ICF-d9/1/icf-d9.owl",
+            "download_url": "media/ontologies/ICF-d840-d859/1/icf-d840-d859.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 495,
+            "id": 624,
             "indexed": true,
             "max_children": 10,
             "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 52,
+            "nb_classes": 44,
             "nb_inconsistent": 0,
-            "nb_individuals": 2437,
+            "nb_individuals": 2429,
             "nb_properties": 65,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ICF-NOCM",
-        "name": "ICF without content model",
+        "acronym": "ICF-d860-d879",
+        "name": "ICF-d860-d879. Economic life",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2014-01-30T00:09:43Z",
-            "date_released": "2014-01-29T00:00:00Z",
-            "description": "This is a version of ICF OWL ontology that has the WHO-FIC \"content model\" entities stripped out. Definitions and labels were moved from content model individuals to entity annotations.",
-            "documentation": null,
+            "date_created": "2010-05-04T22:53:02Z",
+            "date_released": "2010-05-04T07:00:00Z",
+            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 8: MAJOR LIFE AREAS -> Economic life\" (d860-d879) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
+            "documentation": "http://www.who.int/classifications/icf/en/",
             "domain": null,
-            "download_url": "media/ontologies/ICF-NOCM/1/icf-nocm.owl",
+            "download_url": "media/ontologies/ICF-d860-d879/1/icf-d860-d879.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 629,
+            "id": 215,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 6,
+            "max_children": 10,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 1557,
+            "nb_classes": 37,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_individuals": 2422,
+            "nb_properties": 65,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.0.2"
+            "version": "1"
         },
         "topics": null
     },
     {
         "acronym": "ICW",
         "name": "International Classification of Wellness",
         "species": null,
@@ -23711,146 +23847,78 @@
             "submission_id": 4,
             "taxon": null,
             "version": "1.2"
         },
         "topics": null
     },
     {
-        "acronym": "ID-AMR",
-        "name": "Infectious Diseases and Antimicrobial Resistance",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-09-30T20:35:58Z",
-            "date_released": "2021-09-30T00:00:00Z",
-            "description": "Collections of terms for the ZonMW ID&AMR programme. These terms are based on vocabularies provided for describing project and data content metadata. This submission was created using sheet2rdf GitHub workflow.",
-            "documentation": "https://github.com/fair-data-collective/zonmw-id-amr-vocabulary",
-            "domain": null,
-            "download_url": "media/ontologies/ID-AMR/35/id-amr.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/zonmw-id-amr-vocabulary",
-            "id": 4636,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 272,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 35,
-            "taxon": null,
-            "version": "1.2.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "IDEM",
-        "name": "IDEM: wInD Energy Models Taxonomy",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-10-05T13:54:52Z",
-            "date_released": "2021-10-05T00:00:00Z",
-            "description": "Controlled vocabularies such as taxonomies allow an accurate and controlled approach in describing datasets. One of such controlled vocabulary is Wind Energy Taxonomy of Models. This taxonomy is the result of EERA JP WIND IRPWind Open Data initiative that took place in 2017 in which wind energy experts generated the first version of the taxonomy as an input for defining and structuring wind energy metadata.\r\n\r\nThe report of this work is available at Zenodo: https://www.zenodo.org/record/1199489#.XSD6haeQ3RY\r\n\r\nIn 2018, the taxonomy of topics was improved during the internal project of DTU Wind Energy titled 'FAIR Digitalization': https://www.zenodo.org/record/1493874#.XSD7TaeQ3RY\r\n\r\nIn 2021, the taxonomy was converted into FAIR machine-actionable controlled vocabulary using sheet2rdf. The controlled vocabulary is served to humans and machines using an instace of OntoStack hosted by DTU Wind Energy. sheet2rdf and OntoStack are developed by FAIR Data Collective.\r\n\r\nAdditionally, in 2021, the persistent URL purl.org/idem has been registered for the taxonomy. This allows us to:\r\n\r\nIf there is a needed to move the taxonomy to another domain (currently under data.windenergy.dtu.dk) URLs will be still resolvable and unchanged\r\nUse short URls as PIDs for the taxonomy terms and properties.",
-            "documentation": "https://github.com/DTUWindEnergy/IDEM-taxonomy",
-            "domain": null,
-            "download_url": "media/ontologies/IDEM/2/idem.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/DTUWindEnergy/IDEM-taxonomy",
-            "id": 4590,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 38,
-            "nb_properties": 0,
-            "products": null,
-            "publication": "https://github.com/DTUWindEnergy/IDEM-taxonomy",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "v0.1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "INTVMODE",
-        "name": "Interview Mode",
+        "acronym": "ICF-d9",
+        "name": "ICF-d9. COMMUNITY, SOCIAL AND CIVIC LIFE",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-12-07T19:44:48Z",
-            "date_released": "2022-12-07T00:00:00Z",
-            "description": "This class refers to the different modes of Interview",
-            "documentation": null,
+            "date_created": "2010-05-04T22:44:55Z",
+            "date_released": "2010-05-04T07:00:00Z",
+            "description": "The \"ACTIVITIES AND PARTICIPATION CHAPTER 9: COMMUNITY, SOCIAL AND CIVIC LIFE\" (d9) sub-tree of the \"International Classification of Functioning, Disability and Health (ICF)\" ontology.",
+            "documentation": "http://www.who.int/classifications/icf/en/",
             "domain": null,
-            "download_url": "media/ontologies/INTVMODE/1/intvmode.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/ICF-d9/1/icf-d9.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5558,
+            "id": 495,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 10,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 52,
             "nb_inconsistent": 0,
-            "nb_individuals": 8,
-            "nb_properties": 0,
+            "nb_individuals": 2437,
+            "nb_properties": 65,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "IRD",
-        "name": "Inherited Retinal Dystrophies",
+        "acronym": "ICF-NOCM",
+        "name": "ICF without content model",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2019-10-21T19:03:19Z",
-            "date_released": "2019-10-21T00:00:00Z",
-            "description": "Inherited retinal dystrophy (IRD) as one of the rare ocular diseases encompasses different types of hereditary retinal degenerations appearing by involvement of the different layers of the posterior ocular segment. In respect of the importance of the classification systems regarding the recorded history of biologic and natural sciences, there is no classification regarding the IRD diagnoses. Therefore, a novel classification for this type of hereditary ocular disease was developed and presented.  ",
+            "date_created": "2014-01-30T00:09:43Z",
+            "date_released": "2014-01-29T00:00:00Z",
+            "description": "This is a version of ICF OWL ontology that has the WHO-FIC \"content model\" entities stripped out. Definitions and labels were moved from content model individuals to entity annotations.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IRD/1/ird.owl",
+            "download_url": "media/ontologies/ICF-NOCM/1/icf-nocm.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 3024,
+            "id": 629,
             "indexed": true,
-            "max_children": 83,
+            "max_children": 20,
             "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 541,
+            "nb_classes": 1557,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 6,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.0.2"
         },
         "topics": null
     },
     {
         "acronym": "IRD1",
         "name": "Inherited retinal disorders",
         "species": null,
@@ -23915,214 +23983,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "IRD3",
-        "name": "Inherited retinal dystrophy",
+        "acronym": "ID-AMR",
+        "name": "Infectious Diseases and Antimicrobial Resistance",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-01-29T06:08:12Z",
-            "date_released": "2023-01-28T00:00:00Z",
-            "description": "nherited retinal dystrophy (IRD) as one of the rare ocular diseases encompasses different types of hereditary retinal degenerations appearing by involvement of the different layers of the posterior ocular segment. In respect of the importance of the classification systems regarding the recorded history of biologic and natural sciences, there is no classification regarding the IRD diagnoses. Therefore, a novel classification for this type of hereditary ocular disease was developed and presented.",
-            "documentation": null,
+            "date_created": "2021-09-30T20:35:58Z",
+            "date_released": "2021-09-30T00:00:00Z",
+            "description": "Collections of terms for the ZonMW ID&AMR programme. These terms are based on vocabularies provided for describing project and data content metadata. This submission was created using sheet2rdf GitHub workflow.",
+            "documentation": "https://github.com/fair-data-collective/zonmw-id-amr-vocabulary",
             "domain": null,
-            "download_url": "media/ontologies/IRD3/1/ird3.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5631,
+            "download_url": "media/ontologies/ID-AMR/35/id-amr.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/zonmw-id-amr-vocabulary",
+            "id": 4636,
             "indexed": true,
-            "max_children": 78,
-            "max_depth": 4,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 163,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 412,
+            "nb_individuals": 272,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 35,
             "taxon": null,
-            "version": "3"
+            "version": "1.2.1"
         },
         "topics": null
     },
     {
-        "acronym": "IRDG",
-        "name": "Data harmonization ontology for cross border malaria surveillance ",
+        "acronym": "IDEM",
+        "name": "IDEM: wInD Energy Models Taxonomy",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2018-10-12T11:14:09Z",
-            "date_released": "2018-10-12T00:00:00Z",
-            "description": "Ontology used to describe the data system in harmonized surveillance system of malaria between french Guiana and Brazil. Developed by the IRD and the Fiocruz.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/IRDG/1/irdg.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 1941,
-            "indexed": true,
-            "max_children": 9,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 49,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 19,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "STO",
-        "name": "Stroke Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 18,
-            "classifiable": true,
-            "date_created": "2018-06-17T13:55:16Z",
-            "date_released": "2018-06-17T00:00:00Z",
-            "description": "Stroke ontology covers the knowledge domain of stroke based on expert views and translational continuum of stroke research. The Stroke Ontology (STO) presented here, is the first draft and we hope that the scientific community can contribute to the enrichment of this draft. The ontology can be used a reference for a variety of bioinformatics applications such as annotation of terms, harmonization of experimental data, text-mining, and modeling disease mechanism. \r\n",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/STO/3/sto.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 1581,
-            "indexed": true,
-            "max_children": 1676,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 1712,
-            "nb_inconsistent": 0,
-            "nb_individuals": 69,
-            "nb_properties": 29,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "2017-11-01"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BNO",
-        "name": "Bionutrition Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2013-08-15T19:10:25Z",
-            "date_released": "2013-08-08T07:00:00Z",
-            "description": "The Bionutrition Ontology relates concepts and terminologies used for human nutrition in a clinical and biomedical setting.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/BNO/2/bno.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 633,
-            "indexed": true,
-            "max_children": 13,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 100,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "0.2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BP",
-        "name": "BioPAX Ontology of Biological Pathways",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2010-07-29T13:28:09Z",
-            "date_released": "2010-07-02T07:00:00Z",
-            "description": "The BioPAX ontology (www.biopax.org) is a standard language for formally representing biological pathways. BioPAX Level 3 supports the representation of metabolic pathways, signal transduction pathways, protein-protein interaction networks, gene regulatory networks and genetic interactions.",
-            "documentation": "www.biopax.org/release/biopax-level3-documentation.pdf",
-            "domain": null,
-            "download_url": "media/ontologies/BP/1/bp.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "www.biopax.org",
-            "id": 274,
-            "indexed": true,
-            "max_children": 15,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 69,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 96,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "Level3 v1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "BRO-Activity",
-        "name": "Biomedical Resource Ontology - Activity View",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2010-06-28T03:17:41Z",
-            "date_released": "2010-05-17T07:00:00Z",
-            "description": "This view represents a subset of the BRO containing all the Activity classes that are relevant in the biomedical domain.",
-            "documentation": null,
+            "date_created": "2021-10-05T13:54:52Z",
+            "date_released": "2021-10-05T00:00:00Z",
+            "description": "Controlled vocabularies such as taxonomies allow an accurate and controlled approach in describing datasets. One of such controlled vocabulary is Wind Energy Taxonomy of Models. This taxonomy is the result of EERA JP WIND IRPWind Open Data initiative that took place in 2017 in which wind energy experts generated the first version of the taxonomy as an input for defining and structuring wind energy metadata.\r\n\r\nThe report of this work is available at Zenodo: https://www.zenodo.org/record/1199489#.XSD6haeQ3RY\r\n\r\nIn 2018, the taxonomy of topics was improved during the internal project of DTU Wind Energy titled 'FAIR Digitalization': https://www.zenodo.org/record/1493874#.XSD7TaeQ3RY\r\n\r\nIn 2021, the taxonomy was converted into FAIR machine-actionable controlled vocabulary using sheet2rdf. The controlled vocabulary is served to humans and machines using an instace of OntoStack hosted by DTU Wind Energy. sheet2rdf and OntoStack are developed by FAIR Data Collective.\r\n\r\nAdditionally, in 2021, the persistent URL purl.org/idem has been registered for the taxonomy. This allows us to:\r\n\r\nIf there is a needed to move the taxonomy to another domain (currently under data.windenergy.dtu.dk) URLs will be still resolvable and unchanged\r\nUse short URls as PIDs for the taxonomy terms and properties.",
+            "documentation": "https://github.com/DTUWindEnergy/IDEM-taxonomy",
             "domain": null,
-            "download_url": "media/ontologies/BRO-Activity/2/bro-activity.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.biositemaps.org/",
-            "id": 703,
+            "download_url": "media/ontologies/IDEM/2/idem.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/DTUWindEnergy/IDEM-taxonomy",
+            "id": 4590,
             "indexed": true,
-            "max_children": 13,
-            "max_depth": 2,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 21,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 45,
+            "nb_individuals": 38,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "https://github.com/DTUWindEnergy/IDEM-taxonomy",
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "3.1"
+            "version": "v0.1.0"
         },
         "topics": null
     },
     {
         "acronym": "CEPH",
         "name": "Cephalopod Ontology",
         "species": null,
@@ -24167,112 +24099,190 @@
                 "label": "Cephalopod"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ACESO",
-        "name": "Adverse Childhood Experiences Ontology",
+        "acronym": "INTVMODE",
+        "name": "Interview Mode",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-02-25T17:40:17Z",
-            "date_released": "2018-12-21T00:00:00Z",
-            "description": "An ontology to help describe data about Adverse Childhood Experiences.",
+            "date_created": "2022-12-07T19:44:48Z",
+            "date_released": "2022-12-07T00:00:00Z",
+            "description": "This class refers to the different modes of Interview",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ACESO/3/aceso.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/INTVMODE/1/intvmode.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 2310,
+            "id": 5558,
             "indexed": true,
-            "max_children": 29,
-            "max_depth": 11,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 296,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 93,
+            "nb_individuals": 8,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ACGT-MO",
-        "name": "Cancer Research and Management ACGT Master Ontology",
+        "acronym": "IRD",
+        "name": "Inherited Retinal Dystrophies",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2010-03-17T10:04:50Z",
-            "date_released": "2010-01-03T08:00:00Z",
-            "description": "The intention of the ACGT Master Ontology (MO) is to represent the domain of cancer research and management in a computationally tractable manner.",
-            "documentation": "http://www.ifomis.org/acgt",
+            "date_created": "2019-10-21T19:03:19Z",
+            "date_released": "2019-10-21T00:00:00Z",
+            "description": "Inherited retinal dystrophy (IRD) as one of the rare ocular diseases encompasses different types of hereditary retinal degenerations appearing by involvement of the different layers of the posterior ocular segment. In respect of the importance of the classification systems regarding the recorded history of biologic and natural sciences, there is no classification regarding the IRD diagnoses. Therefore, a novel classification for this type of hereditary ocular disease was developed and presented.  ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ACGT-MO/2/acgt-mo.owl",
+            "download_url": "media/ontologies/IRD/1/ird.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.ifomis.org/acgt",
-            "id": 571,
+            "home_page": null,
+            "id": 3024,
             "indexed": true,
-            "max_children": 66,
-            "max_depth": 18,
+            "max_children": 83,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 1770,
+            "nb_classes": 541,
             "nb_inconsistent": 0,
-            "nb_individuals": 61,
-            "nb_properties": 260,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.ifomis.org/acgt",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "IDG_GL",
-        "name": "IDG gene list",
+        "acronym": "NEUROTRANSH",
+        "name": "NeuroMorpho.Org NT cell types",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 209,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-02-23T14:05:11Z",
-            "date_released": "2019-03-20T00:00:00Z",
-            "description": "List of genes studied under the IDG project",
+            "date_created": "2017-11-28T02:12:04Z",
+            "date_released": "2017-11-28T02:12:04Z",
+            "description": "A hierarchy of neuron types based on Neurotransmitter.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IDG_GL/5/idg_gl.owl",
+            "download_url": "media/ontologies/NEUROTRANSH/4/neurotransh.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4308,
+            "id": 866,
             "indexed": true,
-            "max_children": 417,
+            "max_children": 7,
             "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 419,
+            "nb_classes": 9,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 7,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 4,
             "taxon": null,
-            "version": "1.0"
+            "version": "2.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NGBO",
+        "name": "Next Generation Biobanking Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2023-01-23T12:29:21.682176Z",
+            "date_released": "2023-01-23T12:29:21.682160Z",
+            "description": "The Next Generation Biobanking Ontology (NGBO) is an open application ontology representing contextual data about omics digital assets in biobank. The ontology focuses on capturing the information about three main activities: wet bench analysis used to generate omics data, bioinformatics analysis used to analyze and interpret data, and data management.",
+            "documentation": null,
+            "domain": "investigations",
+            "download_url": "media/ontologies/NGBO/2/ngbo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/Dalalghamdi/NGBO",
+            "id": 5613,
+            "indexed": true,
+            "max_children": 103,
+            "max_depth": 16,
+            "md5sum": "c489a96bd13b1935d095070856ef519a",
+            "nb_classes": 1515,
+            "nb_inconsistent": 0,
+            "nb_individuals": 2,
+            "nb_properties": 129,
+            "products": [
+                {
+                    "id": "ngbo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ngbo.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "IDOMAL",
+        "name": "Malaria Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 6,
+            "classifiable": true,
+            "date_created": "2020-02-24T17:38:47.582402Z",
+            "date_released": "2020-02-24T17:38:47.582359Z",
+            "description": "An application ontology to cover all aspects of malaria as well as the intervention attempts to control it.",
+            "documentation": null,
+            "domain": "health",
+            "download_url": "media/ontologies/IDOMAL/187/idomal.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.vectorbase.org/ontology-browser",
+            "id": 3366,
+            "indexed": true,
+            "max_children": 516,
+            "max_depth": 17,
+            "md5sum": "33f8a68a6e25a61e1114ae8f364ba3ca",
+            "nb_classes": 3159,
+            "nb_inconsistent": 0,
+            "nb_individuals": 595,
+            "nb_properties": 16,
+            "products": [
+                {
+                    "id": "idomal.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/idomal.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 187,
+            "taxon": null,
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "BCI-O",
         "name": "Brain-Computer Interaction (BCI) Ontology",
         "species": null,
@@ -24371,86 +24381,146 @@
             "submission_id": 9,
             "taxon": null,
             "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "IDO",
-        "name": "Infectious Disease Ontology",
+        "acronym": "IDQA",
+        "name": "Image and Data Quality Assessment Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-12-13T01:53:30.558727Z",
-            "date_released": "2017-12-13T01:53:30.558682Z",
-            "description": "",
+            "date_created": "2013-12-05T22:14:47Z",
+            "date_released": "2013-08-25T07:00:00Z",
+            "description": "Image and Data Quality Assessment for scientific data management",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/IDO/12/ido.owl",
+            "domain": null,
+            "download_url": "media/ontologies/IDQA/8/idqa.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://idqa.wordpress.com",
+            "id": 417,
+            "indexed": true,
+            "max_children": 18,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 260,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 2,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 8,
+            "taxon": null,
+            "version": "v50"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ISO19115DTC",
+        "name": "ISO 19115 Date Type Code",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 9,
+            "classifiable": true,
+            "date_created": "2017-07-17T19:59:10Z",
+            "date_released": "2017-07-17T00:00:00Z",
+            "description": "ISO 19115 Date Type Codes",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/ISO19115DTC/1/iso19115dtc.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.bioontology.org/wiki/index.php/Infectious_Disease_Ontology",
-            "id": 1010,
+            "home_page": null,
+            "id": 754,
             "indexed": true,
-            "max_children": 173,
-            "max_depth": 10,
-            "md5sum": "eebcb7b81a5620908e1ccfbdf666f137",
-            "nb_classes": 519,
+            "max_children": 16,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 19,
             "nb_inconsistent": 0,
-            "nb_individuals": 20,
-            "nb_properties": 39,
-            "products": [
-                {
-                    "id": "ido.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ido.owl"
-                }
-            ],
+            "nb_individuals": 16,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
-            "taxon": {
-                "id": "NCBITaxon:9606",
-                "label": "Homo sapiens"
-            },
-            "version": null
+            "submission_id": 1,
+            "taxon": null,
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "IDOBRU",
-        "name": "Brucellosis Ontology",
+        "acronym": "ACESO",
+        "name": "Adverse Childhood Experiences Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2019-02-25T17:40:17Z",
+            "date_released": "2018-12-21T00:00:00Z",
+            "description": "An ontology to help describe data about Adverse Childhood Experiences.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/ACESO/3/aceso.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 2310,
+            "indexed": true,
+            "max_children": 29,
+            "max_depth": 11,
+            "md5sum": null,
+            "nb_classes": 296,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 93,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NIDM-RESULTS",
+        "name": "NIDM-Results",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2015-03-29T01:04:36Z",
-            "date_released": "2015-03-29T01:04:35Z",
-            "description": "Brucellosis Ontology is a biomedical ontology in the domain of zoonotic disease brucellosis that is caused by Brucella, a facultative intracellular baterium.",
+            "date_created": "2016-06-07T11:31:38Z",
+            "date_released": "2016-06-07T00:00:00Z",
+            "description": "NIDM-Results is a data model that enable sharing neuroimaging results using a common descriptive standard across neuroimaging software. NIDM-Results focuses on mass-univariate studies and is mostly targeted at fMRI but is also suitable for anatomical MRI (with Voxel-Based Morphometry), and Positron Emission Tomography (PET). \r\n\r\nNIDM-Results packs provide a lightweight solution to share maps generated by neuroimaging studies along with their metadata. Export to NIDM-Results is natively implemented in SPM, NeuroVault and CBRAIN and a Python library is ready for  integration in FSL. NeuroVault additionally can import NIDM-Results packs.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IDOBRU/61/idobru.owl",
+            "download_url": "media/ontologies/NIDM-RESULTS/2/nidm-results.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.phidias.us/bbp/idobru/index.php",
-            "id": 456,
+            "home_page": "http://nidm.nidash.org/specs/nidm-results.html",
+            "id": 442,
             "indexed": true,
-            "max_children": 441,
-            "max_depth": 34,
+            "max_children": 36,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 3214,
+            "nb_classes": 161,
             "nb_inconsistent": 0,
-            "nb_individuals": 25,
-            "nb_properties": 158,
+            "nb_individuals": 38,
+            "nb_properties": 149,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 61,
+            "submission_id": 2,
             "taxon": null,
-            "version": "Arbor version; 1.2.92"
+            "version": "Recommendation version 2013-04-30"
         },
         "topics": null
     },
     {
         "acronym": "NCIT",
         "name": "NCI Thesaurus OBO Edition",
         "species": null,
@@ -24577,48 +24647,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "NEIC-ONTO",
-        "name": "Unoffical NeIC Organisation Vocabulary",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2023-01-25T11:28:31Z",
-            "date_released": "2023-01-24T00:00:00Z",
-            "description": "Terms used to describe different aspects of NeIC activities",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NEIC-ONTO/5/neic-onto.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5632,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 22,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 5,
-            "taxon": null,
-            "version": "0.2.2"
-        },
-        "topics": null
-    },
-    {
         "acronym": "NEO",
         "name": "Neurologic Examination Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -24645,14 +24681,48 @@
             "submission_id": 90,
             "taxon": null,
             "version": "49.0"
         },
         "topics": null
     },
     {
+        "acronym": "ACGT-MO",
+        "name": "Cancer Research and Management ACGT Master Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2010-03-17T10:04:50Z",
+            "date_released": "2010-01-03T08:00:00Z",
+            "description": "The intention of the ACGT Master Ontology (MO) is to represent the domain of cancer research and management in a computationally tractable manner.",
+            "documentation": "http://www.ifomis.org/acgt",
+            "domain": null,
+            "download_url": "media/ontologies/ACGT-MO/2/acgt-mo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.ifomis.org/acgt",
+            "id": 571,
+            "indexed": true,
+            "max_children": 66,
+            "max_depth": 18,
+            "md5sum": null,
+            "nb_classes": 1770,
+            "nb_inconsistent": 0,
+            "nb_individuals": 61,
+            "nb_properties": 260,
+            "products": null,
+            "publication": "http://www.ifomis.org/acgt",
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "NLN",
         "name": "NLighten Ontology",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
@@ -24679,44 +24749,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2018-02-28"
         },
         "topics": null
     },
     {
-        "acronym": "NEOMARK3",
-        "name": "Neomark Oral Cancer Ontology, version 3",
+        "acronym": "IDG_GL",
+        "name": "IDG gene list",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 209,
             "classifiable": true,
-            "date_created": "2010-05-21T10:35:05Z",
-            "date_released": "2010-05-17T07:00:00Z",
-            "description": "Ontology that describes the medical information necessary for early detection of the oral cancer reoccurrence extracted from the NeoMark Project.",
+            "date_created": "2021-02-23T14:05:11Z",
+            "date_released": "2019-03-20T00:00:00Z",
+            "description": "List of genes studied under the IDG project",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NEOMARK3/2/neomark3.owl",
+            "download_url": "media/ontologies/IDG_GL/5/idg_gl.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.neomark.eu/portal/",
-            "id": 451,
+            "home_page": null,
+            "id": 4308,
             "indexed": true,
-            "max_children": 41,
+            "max_children": 417,
             "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 55,
-            "nb_inconsistent": 1,
+            "nb_classes": 419,
+            "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 593,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 5,
             "taxon": null,
-            "version": "3.1"
+            "version": "1.0"
         },
         "topics": null
     },
     {
         "acronym": "MOC",
         "name": "Material Organic Compound",
         "species": null,
@@ -24747,610 +24817,498 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2.3"
         },
         "topics": null
     },
     {
-        "acronym": "IDO-COVID-19",
-        "name": "The COVID-19 Infectious Disease Ontology",
+        "acronym": "TRAK",
+        "name": "Taxonomy for Rehabilitation of Knee Conditions",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2020-08-03T07:45:39Z",
-            "date_released": "2020-06-05T00:00:00Z",
-            "description": "The COVID-19 Infectious Disease Ontology (IDO-COVID-19) is an extension of the Infectious Disease Ontology (IDO) and the Virus Infectious Disease Ontology (VIDO). IDO Virus follows OBO Foundry guidelines, employs the Basic Formal Ontology as its starting point, and covers epidemiology, classification, pathogenesis, and treatment of terms used to represent infection by the SARS-CoV-2 virus strain, and the associated COVID-19 disease.\r\n",
-            "documentation": "https://github.com/infectious-disease-ontology-extensions/ido-covid-19",
+            "date_created": "2015-09-14T10:39:19Z",
+            "date_released": "2015-09-14T00:00:00Z",
+            "description": "TRAK (Taxonomy for RehAbilitation of Knee conditions) is an ontology that formally models information relevant for the rehabilitation of knee conditions. TRAK provides the framework that can be used to collect coded data in sufficient detail to support epidemiologic studies so that the most effective treatment components can be identified, new interventions developed and the quality of future randomized control trials improved to incorporate a control intervention that is well defined and reflects clinical practice. ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IDO-COVID-19/2/ido-covid-19.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/infectious-disease-ontology-extensions/ido-covid-19",
-            "id": 3820,
+            "download_url": "media/ontologies/TRAK/5/trak.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.cs.cf.ac.uk/trak",
+            "id": 139,
             "indexed": true,
-            "max_children": 18,
-            "max_depth": 11,
+            "max_children": 43,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 486,
+            "nb_classes": 1619,
             "nb_inconsistent": 0,
-            "nb_individuals": 23,
-            "nb_properties": 43,
+            "nb_individuals": 0,
+            "nb_properties": 16,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 5,
             "taxon": null,
-            "version": "8-3-2020"
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "IDODEN",
-        "name": "Dengue Fever Ontology",
+        "acronym": "ABA-AMB",
+        "name": "Allen Brain Atlas (ABA) Adult Mouse Brain Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 10,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2014-02-17T13:29:24Z",
-            "date_released": "2013-07-12T07:00:00Z",
-            "description": "An ontology for dengue fever.",
-            "documentation": null,
+            "date_created": "2009-08-09T00:18:35Z",
+            "date_released": "2009-06-12T07:00:00Z",
+            "description": "Allen Brain Atlas P56 Mouse Ontology",
+            "documentation": "http://mouse.brain-map.org/atlas/index.html",
             "domain": null,
-            "download_url": "media/ontologies/IDODEN/6/idoden.owl",
+            "download_url": "media/ontologies/ABA-AMB/1/aba-amb.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://code.google.com/p/dengue-fever-ontology/",
-            "id": 299,
+            "home_page": "http://www.brain-map.org",
+            "id": 631,
             "indexed": true,
-            "max_children": 1389,
-            "max_depth": 18,
+            "max_children": 28,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 5019,
-            "nb_inconsistent": 16,
+            "nb_classes": 913,
+            "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 23,
+            "nb_properties": 2,
             "products": null,
-            "publication": null,
+            "publication": "http://www.wiley.com/WileyCDA/WileyTitle/productCd-0470054085.html",
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.15b"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "IDOMAL",
-        "name": "Malaria Ontology",
+        "acronym": "NEOMARK4",
+        "name": "Neomark Oral Cancer Ontology, version 4",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2020-02-24T17:38:47.582402Z",
-            "date_released": "2020-02-24T17:38:47.582359Z",
-            "description": "An application ontology to cover all aspects of malaria as well as the intervention attempts to control it.",
-            "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/IDOMAL/187/idomal.owl",
+            "date_created": "2011-11-11T10:19:57Z",
+            "date_released": "2011-03-24T07:00:00Z",
+            "description": "Ontology that describes the medical information necessary for early detection of the oral cancer reoccurrence extracted from the NeoMark Project.",
+            "documentation": "http://www.neomark.eu",
+            "domain": null,
+            "download_url": "media/ontologies/NEOMARK4/1/neomark4.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.vectorbase.org/ontology-browser",
-            "id": 3366,
+            "home_page": "http://www.neomark.eu",
+            "id": 525,
             "indexed": true,
-            "max_children": 516,
-            "max_depth": 17,
-            "md5sum": "33f8a68a6e25a61e1114ae8f364ba3ca",
-            "nb_classes": 3159,
+            "max_children": 19,
+            "max_depth": 10,
+            "md5sum": null,
+            "nb_classes": 325,
             "nb_inconsistent": 0,
-            "nb_individuals": 595,
-            "nb_properties": 16,
-            "products": [
-                {
-                    "id": "idomal.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/idomal.owl"
-                }
-            ],
-            "publication": null,
+            "nb_individuals": 0,
+            "nb_properties": 27,
+            "products": null,
+            "publication": "http://www.neomark.eu/portal/index.php?option=com_content&view=article&id=86&Itemid=96",
             "publications": null,
-            "submission_id": 187,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "4.1"
         },
         "topics": null
     },
     {
-        "acronym": "IDQA",
-        "name": "Image and Data Quality Assessment Ontology",
+        "acronym": "TGMA",
+        "name": "Mosquito gross anatomy ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 12,
             "classifiable": true,
-            "date_created": "2013-12-05T22:14:47Z",
-            "date_released": "2013-08-25T07:00:00Z",
-            "description": "Image and Data Quality Assessment for scientific data management",
+            "date_created": "2020-03-06T11:58:05.219310Z",
+            "date_released": "2020-03-06T11:58:05.219277Z",
+            "description": "A structured controlled vocabulary of the anatomy of mosquitoes.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/IDQA/8/idqa.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://idqa.wordpress.com",
-            "id": 417,
+            "domain": "anatomy",
+            "download_url": "media/ontologies/TGMA/18/tgma.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.vectorbase.org/ontology-browser",
+            "id": 3410,
             "indexed": true,
-            "max_children": 18,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 260,
+            "max_children": 1122,
+            "max_depth": 7,
+            "md5sum": "7d1320828369919b51c6d7704f430f82",
+            "nb_classes": 1865,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 2,
-            "products": null,
+            "nb_properties": 3,
+            "products": [
+                {
+                    "id": "tgma.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/tgma.owl"
+                },
+                {
+                    "id": "tgma.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/tgma.obo"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 8,
-            "taxon": null,
-            "version": "v50"
+            "submission_id": 18,
+            "taxon": {
+                "id": "NCBITaxon:44484",
+                "label": "Anopheles"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ISO19115DTC",
-        "name": "ISO 19115 Date Type Code",
+        "acronym": "NEIC-ONTO",
+        "name": "Unoffical NeIC Organisation Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 9,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-07-17T19:59:10Z",
-            "date_released": "2017-07-17T00:00:00Z",
-            "description": "ISO 19115 Date Type Codes",
+            "date_created": "2023-01-25T11:28:31Z",
+            "date_released": "2023-01-24T00:00:00Z",
+            "description": "Terms used to describe different aspects of NeIC activities",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ISO19115DTC/1/iso19115dtc.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/NEIC-ONTO/5/neic-onto.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 754,
+            "id": 5632,
             "indexed": true,
-            "max_children": 16,
+            "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 19,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 16,
+            "nb_individuals": 22,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 5,
             "taxon": null,
-            "version": "2.0"
+            "version": "0.2.2"
         },
         "topics": null
     },
     {
-        "acronym": "MGBD",
-        "name": "MGB Drug List",
+        "acronym": "NIAID-GSC-BRC",
+        "name": "NIAID GSC and BRC View",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-07-05T20:33:35Z",
-            "date_released": "2022-06-24T00:00:00Z",
-            "description": "MGB Drug List",
+            "date_created": "2015-05-13T18:51:19Z",
+            "date_released": "2015-05-13T00:00:00Z",
+            "description": "It is a subset of OBI containing all terms  representing NIAID GSCID and BRC Core Metadata. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MGBD/7/mgbd.owl",
+            "download_url": "media/ontologies/NIAID-GSC-BRC/3/niaid-gsc-brc.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5229,
+            "id": 105,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
+            "max_children": 12,
+            "max_depth": 15,
             "md5sum": null,
-            "nb_classes": 10,
+            "nb_classes": 198,
             "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 0,
+            "nb_individuals": 12,
+            "nb_properties": 35,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 3,
             "taxon": null,
-            "version": "0.6"
+            "version": "2015-04-13"
         },
         "topics": null
     },
     {
-        "acronym": "NEOMARK4",
-        "name": "Neomark Oral Cancer Ontology, version 4",
+        "acronym": "NEOMARK3",
+        "name": "Neomark Oral Cancer Ontology, version 3",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2011-11-11T10:19:57Z",
-            "date_released": "2011-03-24T07:00:00Z",
+            "date_created": "2010-05-21T10:35:05Z",
+            "date_released": "2010-05-17T07:00:00Z",
             "description": "Ontology that describes the medical information necessary for early detection of the oral cancer reoccurrence extracted from the NeoMark Project.",
-            "documentation": "http://www.neomark.eu",
-            "domain": null,
-            "download_url": "media/ontologies/NEOMARK4/1/neomark4.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.neomark.eu",
-            "id": 525,
-            "indexed": true,
-            "max_children": 19,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 325,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 27,
-            "products": null,
-            "publication": "http://www.neomark.eu/portal/index.php?option=com_content&view=article&id=86&Itemid=96",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "4.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FOOD_ONT",
-        "name": "FOOD_ONTOLOGY_OWL",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2015-04-27T18:56:49Z",
-            "date_released": "2015-04-01T00:00:00Z",
-            "description": "We built food ontology primarily to support comparative effectiveness researchers studying food components or ingredients data. Food Ontology Knowledgebase (FOKB) is developed in English and Turkish that contains details of food ingredients such as their codes (e-codex or codex numbers) and also side effects of them such as allergy. Initially, the first stage of the research project was funded in 2013 by TUBITAK (The Scientific and Technological Research Council of Turkey- http://www.tubitak.gov.tr/en) \u2013 1512 Progressive Entrepreneurship Support Program (Project No: 2120357, \u201cSemantic-based Personal Safe Food Consumption System for Allergic Individuals via Smart Devices\u201d) by the Semantica Trd. Ltd. Co (http://www.semantica.com.tr/en). The initial proposal of the first stage project is presented in Hindawi Publishing Corporation http://www.hindawi.com- The Scientific World Journal with the title \u201cFoodWiki: Ontology-driven Mobile Safe Food Consumption System\u201d. After that, another project as an extension of the first stage project was funded again by TUBITAK toward the end of 2014. It is titled \u201cArtificial Intelligence-based Mobile System for Conscious Food Consumption through Ontology\u201d and recorded with the project number 3140417 in cooperation with Semantica Internet and Software Services Trd. Ltd. Co. and Ac\u0131badem Hospitals Group[1] in consultation with Asst. Prof. Dr. Duygu \u00c7elik. Copyrights of the Food Ontology Knowledgebase (FOKB) and FoodWiki system are reserved by Semantica Trd. Ltd. Co and Ac\u0131badem Hospitals Group (http://www.acibademinternational.com/)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FOOD_ONT/2/food_ont.owl",
+            "download_url": "media/ontologies/NEOMARK3/2/neomark3.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 504,
+            "home_page": "http://www.neomark.eu/portal/",
+            "id": 451,
             "indexed": true,
-            "max_children": 15,
-            "max_depth": 4,
+            "max_children": 41,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 62,
-            "nb_inconsistent": 0,
-            "nb_individuals": 1740,
-            "nb_properties": 46,
+            "nb_classes": 55,
+            "nb_inconsistent": 1,
+            "nb_individuals": 0,
+            "nb_properties": 593,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "v1.0"
+            "version": "3.1"
         },
         "topics": null
     },
     {
-        "acronym": "NEUDIGS",
-        "name": "Neuroscience Domain Insight Graph",
+        "acronym": "IDO-COVID-19",
+        "name": "The COVID-19 Infectious Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-02-24T05:10:01Z",
-            "date_released": "2017-02-23T00:00:00Z",
-            "description": "Neuroscience Domain Insight Graphs provide a framework for expressing linked open data about ",
-            "documentation": null,
+            "date_created": "2020-08-03T07:45:39Z",
+            "date_released": "2020-06-05T00:00:00Z",
+            "description": "The COVID-19 Infectious Disease Ontology (IDO-COVID-19) is an extension of the Infectious Disease Ontology (IDO) and the Virus Infectious Disease Ontology (VIDO). IDO Virus follows OBO Foundry guidelines, employs the Basic Formal Ontology as its starting point, and covers epidemiology, classification, pathogenesis, and treatment of terms used to represent infection by the SARS-CoV-2 virus strain, and the associated COVID-19 disease.\r\n",
+            "documentation": "https://github.com/infectious-disease-ontology-extensions/ido-covid-19",
             "domain": null,
-            "download_url": "media/ontologies/NEUDIGS/1/neudigs.owl",
+            "download_url": "media/ontologies/IDO-COVID-19/2/ido-covid-19.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://dig.isi.edu",
-            "id": 478,
+            "home_page": "https://github.com/infectious-disease-ontology-extensions/ido-covid-19",
+            "id": 3820,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 1,
+            "max_children": 18,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 23,
+            "nb_classes": 486,
             "nb_inconsistent": 0,
-            "nb_individuals": 8,
-            "nb_properties": 22,
+            "nb_individuals": 23,
+            "nb_properties": 43,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.0.1"
+            "version": "8-3-2020"
         },
         "topics": null
     },
     {
-        "acronym": "NeuroFMA",
-        "name": "NeuroFMA",
+        "acronym": "IDODEN",
+        "name": "Dengue Fever Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 10,
             "classifiable": true,
-            "date_created": "2009-09-18T22:04:17Z",
-            "date_released": "2009-07-14T07:00:00Z",
-            "description": "A view of FMA for neuroanatomy",
+            "date_created": "2014-02-17T13:29:24Z",
+            "date_released": "2013-07-12T07:00:00Z",
+            "description": "An ontology for dengue fever.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NeuroFMA/1/neurofma.owl",
+            "download_url": "media/ontologies/IDODEN/6/idoden.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 424,
+            "home_page": "http://code.google.com/p/dengue-fever-ontology/",
+            "id": 299,
             "indexed": true,
-            "max_children": 43,
+            "max_children": 1389,
             "max_depth": 18,
             "md5sum": null,
-            "nb_classes": 2302,
-            "nb_inconsistent": 0,
-            "nb_individuals": 7340,
-            "nb_properties": 40,
+            "nb_classes": 5019,
+            "nb_inconsistent": 16,
+            "nb_individuals": 0,
+            "nb_properties": 23,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
-            "version": "1"
+            "version": "0.15b"
         },
         "topics": null
     },
     {
-        "acronym": "NEUROTRANSH",
-        "name": "NeuroMorpho.Org NT cell types",
+        "acronym": "THREE-ST",
+        "name": "3-Step Theory of suicide",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-11-28T02:12:04Z",
-            "date_released": "2017-11-28T02:12:04Z",
-            "description": "A hierarchy of neuron types based on Neurotransmitter.",
+            "date_created": "2023-07-13T16:19:30Z",
+            "date_released": "2023-07-07T00:00:00Z",
+            "description": "This ontology contains classes, sub-classes, and relationships that map onto the 3-Step Theory of suicide (3ST). The classes represent the four factors that make up the central tenets of the theory: 1) psychological pain, 2) hopelessness, 3) connectedness, and 4) capacity for suicide. The instances of each subclass are terms that were found in VA clinical notes and indicate presence or absence of the associated 3ST factor. The terms are intended to be used in an NLP pipeline to extract 3ST information from clinical progress notes.\r\nPsychological pain, hopelessness, and capacity for suicide are major risk factors for suicide, whereas connectedness is a protective factor. In brief, the theory holds that suicide only occurs if pain and hopelessness outweigh connectedness and only then if the capacity for suicide is present.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NEUROTRANSH/4/neurotransh.owl",
+            "download_url": "media/ontologies/THREE-ST/4/three-st.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 866,
+            "id": 5953,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 2,
+            "max_children": 9,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 9,
+            "nb_classes": 48,
             "nb_inconsistent": 0,
-            "nb_individuals": 7,
+            "nb_individuals": 8484,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 4,
             "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "NGBO",
-        "name": "Next Generation Biobanking Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2023-01-23T12:29:21.682176Z",
-            "date_released": "2023-01-23T12:29:21.682160Z",
-            "description": "The Next Generation Biobanking Ontology (NGBO) is an open application ontology representing contextual data about omics digital assets in biobank. The ontology focuses on capturing the information about three main activities: wet bench analysis used to generate omics data, bioinformatics analysis used to analyze and interpret data, and data management.",
-            "documentation": null,
-            "domain": "investigations",
-            "download_url": "media/ontologies/NGBO/2/ngbo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/Dalalghamdi/NGBO",
-            "id": 5613,
-            "indexed": true,
-            "max_children": 103,
-            "max_depth": 16,
-            "md5sum": "c489a96bd13b1935d095070856ef519a",
-            "nb_classes": 1515,
-            "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 129,
-            "products": [
-                {
-                    "id": "ngbo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ngbo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": null
+            "version": "1.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "NIAID-GSC-BRC",
-        "name": "NIAID GSC and BRC View",
+        "acronym": "FOOD_ONT",
+        "name": "FOOD_ONTOLOGY_OWL",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2015-05-13T18:51:19Z",
-            "date_released": "2015-05-13T00:00:00Z",
-            "description": "It is a subset of OBI containing all terms  representing NIAID GSCID and BRC Core Metadata. ",
+            "date_created": "2015-04-27T18:56:49Z",
+            "date_released": "2015-04-01T00:00:00Z",
+            "description": "We built food ontology primarily to support comparative effectiveness researchers studying food components or ingredients data. Food Ontology Knowledgebase (FOKB) is developed in English and Turkish that contains details of food ingredients such as their codes (e-codex or codex numbers) and also side effects of them such as allergy. Initially, the first stage of the research project was funded in 2013 by TUBITAK (The Scientific and Technological Research Council of Turkey- http://www.tubitak.gov.tr/en) \u2013 1512 Progressive Entrepreneurship Support Program (Project No: 2120357, \u201cSemantic-based Personal Safe Food Consumption System for Allergic Individuals via Smart Devices\u201d) by the Semantica Trd. Ltd. Co (http://www.semantica.com.tr/en). The initial proposal of the first stage project is presented in Hindawi Publishing Corporation http://www.hindawi.com- The Scientific World Journal with the title \u201cFoodWiki: Ontology-driven Mobile Safe Food Consumption System\u201d. After that, another project as an extension of the first stage project was funded again by TUBITAK toward the end of 2014. It is titled \u201cArtificial Intelligence-based Mobile System for Conscious Food Consumption through Ontology\u201d and recorded with the project number 3140417 in cooperation with Semantica Internet and Software Services Trd. Ltd. Co. and Ac\u0131badem Hospitals Group[1] in consultation with Asst. Prof. Dr. Duygu \u00c7elik. Copyrights of the Food Ontology Knowledgebase (FOKB) and FoodWiki system are reserved by Semantica Trd. Ltd. Co and Ac\u0131badem Hospitals Group (http://www.acibademinternational.com/)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NIAID-GSC-BRC/3/niaid-gsc-brc.owl",
+            "download_url": "media/ontologies/FOOD_ONT/2/food_ont.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 105,
+            "id": 504,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 15,
+            "max_children": 15,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 198,
+            "nb_classes": 62,
             "nb_inconsistent": 0,
-            "nb_individuals": 12,
-            "nb_properties": 35,
+            "nb_individuals": 1740,
+            "nb_properties": 46,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2015-04-13"
+            "version": "v1.0"
         },
         "topics": null
     },
     {
-        "acronym": "NIDM-RESULTS",
-        "name": "NIDM-Results",
+        "acronym": "TP4",
+        "name": "Oualid",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2016-06-07T11:31:38Z",
-            "date_released": "2016-06-07T00:00:00Z",
-            "description": "NIDM-Results is a data model that enable sharing neuroimaging results using a common descriptive standard across neuroimaging software. NIDM-Results focuses on mass-univariate studies and is mostly targeted at fMRI but is also suitable for anatomical MRI (with Voxel-Based Morphometry), and Positron Emission Tomography (PET). \r\n\r\nNIDM-Results packs provide a lightweight solution to share maps generated by neuroimaging studies along with their metadata. Export to NIDM-Results is natively implemented in SPM, NeuroVault and CBRAIN and a Python library is ready for  integration in FSL. NeuroVault additionally can import NIDM-Results packs.",
+            "date_created": "2023-07-02T11:38:02Z",
+            "date_released": "2023-07-02T00:00:00Z",
+            "description": "relation familiales ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NIDM-RESULTS/2/nidm-results.owl",
+            "download_url": "media/ontologies/TP4/1/tp4.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://nidm.nidash.org/specs/nidm-results.html",
-            "id": 442,
+            "home_page": null,
+            "id": 5926,
             "indexed": true,
-            "max_children": 36,
-            "max_depth": 4,
+            "max_children": 4,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 161,
+            "nb_classes": 9,
             "nb_inconsistent": 0,
-            "nb_individuals": 38,
-            "nb_properties": 149,
+            "nb_individuals": 14,
+            "nb_properties": 14,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "Recommendation version 2013-04-30"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TGMA",
-        "name": "Mosquito gross anatomy ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 12,
-            "classifiable": true,
-            "date_created": "2020-03-06T11:58:05.219310Z",
-            "date_released": "2020-03-06T11:58:05.219277Z",
-            "description": "A structured controlled vocabulary of the anatomy of mosquitoes.",
-            "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/TGMA/18/tgma.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.vectorbase.org/ontology-browser",
-            "id": 3410,
-            "indexed": true,
-            "max_children": 1122,
-            "max_depth": 7,
-            "md5sum": "7d1320828369919b51c6d7704f430f82",
-            "nb_classes": 1865,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 3,
-            "products": [
-                {
-                    "id": "tgma.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/tgma.owl"
-                },
-                {
-                    "id": "tgma.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/tgma.obo"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 18,
-            "taxon": {
-                "id": "NCBITaxon:44484",
-                "label": "Anopheles"
-            },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "THREE-ST",
-        "name": "3-Step Theory of suicide",
+        "acronym": "NEUDIGS",
+        "name": "Neuroscience Domain Insight Graph",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2023-07-13T16:19:30Z",
-            "date_released": "2023-07-07T00:00:00Z",
-            "description": "This ontology contains classes, sub-classes, and relationships that map onto the 3-Step Theory of suicide (3ST). The classes represent the four factors that make up the central tenets of the theory: 1) psychological pain, 2) hopelessness, 3) connectedness, and 4) capacity for suicide. The instances of each subclass are terms that were found in VA clinical notes and indicate presence or absence of the associated 3ST factor. The terms are intended to be used in an NLP pipeline to extract 3ST information from clinical progress notes.\r\nPsychological pain, hopelessness, and capacity for suicide are major risk factors for suicide, whereas connectedness is a protective factor. In brief, the theory holds that suicide only occurs if pain and hopelessness outweigh connectedness and only then if the capacity for suicide is present.",
+            "date_created": "2017-02-24T05:10:01Z",
+            "date_released": "2017-02-23T00:00:00Z",
+            "description": "Neuroscience Domain Insight Graphs provide a framework for expressing linked open data about ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/THREE-ST/4/three-st.owl",
+            "download_url": "media/ontologies/NEUDIGS/1/neudigs.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5953,
+            "home_page": "http://dig.isi.edu",
+            "id": 478,
             "indexed": true,
-            "max_children": 9,
-            "max_depth": 4,
+            "max_children": 10,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 48,
+            "nb_classes": 23,
             "nb_inconsistent": 0,
-            "nb_individuals": 8484,
-            "nb_properties": 0,
+            "nb_individuals": 8,
+            "nb_properties": 22,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.0.1"
+            "version": "0.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "TP4",
-        "name": "Oualid",
+        "acronym": "MGBD",
+        "name": "MGB Drug List",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 1,
             "classifiable": true,
-            "date_created": "2023-07-02T11:38:02Z",
-            "date_released": "2023-07-02T00:00:00Z",
-            "description": "relation familiales ",
+            "date_created": "2022-07-05T20:33:35Z",
+            "date_released": "2022-06-24T00:00:00Z",
+            "description": "MGB Drug List",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TP4/1/tp4.owl",
+            "download_url": "media/ontologies/MGBD/7/mgbd.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5926,
+            "id": 5229,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 3,
+            "max_children": 5,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 9,
+            "nb_classes": 10,
             "nb_inconsistent": 0,
-            "nb_individuals": 14,
-            "nb_properties": 14,
+            "nb_individuals": 5,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 7,
             "taxon": null,
-            "version": null
+            "version": "0.6"
         },
         "topics": null
     },
     {
         "acronym": "TP_REL_FAM",
         "name": "Relation familiales",
         "species": null,
@@ -25381,74 +25339,116 @@
             "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TRAK",
-        "name": "Taxonomy for Rehabilitation of Knee Conditions",
+        "acronym": "IDO",
+        "name": "Infectious Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2015-09-14T10:39:19Z",
-            "date_released": "2015-09-14T00:00:00Z",
-            "description": "TRAK (Taxonomy for RehAbilitation of Knee conditions) is an ontology that formally models information relevant for the rehabilitation of knee conditions. TRAK provides the framework that can be used to collect coded data in sufficient detail to support epidemiologic studies so that the most effective treatment components can be identified, new interventions developed and the quality of future randomized control trials improved to incorporate a control intervention that is well defined and reflects clinical practice. ",
+            "date_created": "2017-12-13T01:53:30.558727Z",
+            "date_released": "2017-12-13T01:53:30.558682Z",
+            "description": "",
+            "documentation": null,
+            "domain": "health",
+            "download_url": "media/ontologies/IDO/12/ido.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.bioontology.org/wiki/index.php/Infectious_Disease_Ontology",
+            "id": 1010,
+            "indexed": true,
+            "max_children": 173,
+            "max_depth": 10,
+            "md5sum": "eebcb7b81a5620908e1ccfbdf666f137",
+            "nb_classes": 519,
+            "nb_inconsistent": 0,
+            "nb_individuals": 20,
+            "nb_properties": 39,
+            "products": [
+                {
+                    "id": "ido.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ido.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 12,
+            "taxon": {
+                "id": "NCBITaxon:9606",
+                "label": "Homo sapiens"
+            },
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "IDOBRU",
+        "name": "Brucellosis Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2015-03-29T01:04:36Z",
+            "date_released": "2015-03-29T01:04:35Z",
+            "description": "Brucellosis Ontology is a biomedical ontology in the domain of zoonotic disease brucellosis that is caused by Brucella, a facultative intracellular baterium.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TRAK/5/trak.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.cs.cf.ac.uk/trak",
-            "id": 139,
+            "download_url": "media/ontologies/IDOBRU/61/idobru.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.phidias.us/bbp/idobru/index.php",
+            "id": 456,
             "indexed": true,
-            "max_children": 43,
-            "max_depth": 12,
+            "max_children": 441,
+            "max_depth": 34,
             "md5sum": null,
-            "nb_classes": 1619,
+            "nb_classes": 3214,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 16,
+            "nb_individuals": 25,
+            "nb_properties": 158,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 61,
             "taxon": null,
-            "version": "unknown"
+            "version": "Arbor version; 1.2.92"
         },
         "topics": null
     },
     {
-        "acronym": "ABA-AMB",
-        "name": "Allen Brain Atlas (ABA) Adult Mouse Brain Ontology",
+        "acronym": "NeuroFMA",
+        "name": "NeuroFMA",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2009-08-09T00:18:35Z",
-            "date_released": "2009-06-12T07:00:00Z",
-            "description": "Allen Brain Atlas P56 Mouse Ontology",
-            "documentation": "http://mouse.brain-map.org/atlas/index.html",
+            "date_created": "2009-09-18T22:04:17Z",
+            "date_released": "2009-07-14T07:00:00Z",
+            "description": "A view of FMA for neuroanatomy",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ABA-AMB/1/aba-amb.owl",
+            "download_url": "media/ontologies/NeuroFMA/1/neurofma.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.brain-map.org",
-            "id": 631,
+            "home_page": null,
+            "id": 424,
             "indexed": true,
-            "max_children": 28,
-            "max_depth": 9,
+            "max_children": 43,
+            "max_depth": 18,
             "md5sum": null,
-            "nb_classes": 913,
+            "nb_classes": 2302,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_individuals": 7340,
+            "nb_properties": 40,
             "products": null,
-            "publication": "http://www.wiley.com/WileyCDA/WileyTitle/productCd-0470054085.html",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
@@ -25585,112 +25585,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2.3"
         },
         "topics": null
     },
     {
-        "acronym": "BRO-AreaOfResearch",
-        "name": "Biomedical Resource Ontology - Area of Research View",
+        "acronym": "ITO",
+        "name": "Intelligence Task Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 11,
+            "avg_children": 14,
             "classifiable": true,
-            "date_created": "2010-06-28T03:25:11Z",
-            "date_released": "2010-05-17T07:00:00Z",
-            "description": "This view represents a subset of the BRO containing all the Area of Research classes that are relevant in the biomedical domain.",
+            "date_created": "2022-09-09T08:53:52Z",
+            "date_released": "2022-09-09T00:00:00Z",
+            "description": "The Intelligence Task Ontology and Knowledge Graph (ITO) provides a comprehensive, curated model of artificial intelligence tasks, benchmarks and benchmark results, including the biomedical domain.\r\n\r\nNOTE: The version uploaded to BioPortal has all instances and benchmark data removed because of performance issues with the BioPortal class browser. Please refer to https://github.com/OpenBioLink/ITO/ for downloading the full version of the ontology and dataset.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BRO-AreaOfResearch/2/bro-areaofresearch.owl",
+            "download_url": "media/ontologies/ITO/17/ito.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.biositemaps.org/",
-            "id": 612,
+            "home_page": "https://github.com/OpenBioLink/ITO/",
+            "id": 5346,
             "indexed": true,
-            "max_children": 19,
-            "max_depth": 1,
+            "max_children": 11326,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 22,
+            "nb_classes": 16283,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 45,
+            "nb_individuals": 37222,
+            "nb_properties": 16,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 17,
             "taxon": null,
-            "version": "3.1"
+            "version": "1.12 (PWC export dated 2022-07-30) - internal, just a minor fix"
         },
         "topics": null
     },
     {
-        "acronym": "BRO-Chinese",
-        "name": "Biomedical Resource Ontology with Simplified Chinese annotations.",
+        "acronym": "BRSO",
+        "name": "Biological Resource Schema Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2009-10-06T22:25:54Z",
-            "date_released": "2009-02-24T08:00:00Z",
-            "description": "This view contains the entire BRO with the addition of three DataType properties, i.e., preferredName, shortName and synonym; then for all the classes and properties, Simplified Chinese (and some English) annotations were inserted into the ontology.",
+            "date_created": "2021-12-21T05:34:45Z",
+            "date_released": "2021-12-20T00:00:00Z",
+            "description": "Biological Resource Schema Ontology",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BRO-Chinese/1/bro-chinese.owl",
+            "download_url": "media/ontologies/BRSO/1/brso.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 390,
+            "home_page": "https://github.com/dbcls/brso",
+            "id": 4787,
             "indexed": true,
-            "max_children": 13,
-            "max_depth": 9,
+            "max_children": 14,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 311,
+            "nb_classes": 15,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 39,
+            "nb_individuals": 9,
+            "nb_properties": 8,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "2.5"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "KGCL",
-        "name": "Knowledge Graph Change Language",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2023-04-07T01:19:04Z",
-            "date_released": "2023-04-07T01:19:02Z",
-            "description": "Knowledge Graph Change Language ontology.",
-            "documentation": "https://incatools.github.io/kgcl/",
-            "domain": null,
-            "download_url": "media/ontologies/KGCL/3/kgcl.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/INCATools/kgcl",
-            "id": 5760,
-            "indexed": true,
-            "max_children": 13,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 92,
-            "nb_inconsistent": 0,
-            "nb_individuals": 11,
-            "nb_properties": 32,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "0.1.0"
+            "version": "0.02"
         },
         "topics": null
     },
     {
         "acronym": "BRODMANNH",
         "name": "NeuroMorpho.Org Brodmann areas",
         "species": null,
@@ -25721,14 +25687,82 @@
             "submission_id": 4,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
+        "acronym": "BSAO",
+        "name": "Botryllus schlosseri anatomy and development ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 8,
+            "classifiable": true,
+            "date_created": "2013-08-29T09:04:07Z",
+            "date_released": "2013-06-27T07:00:00Z",
+            "description": "The first ontology describing the anatomy and the development of Botryllus schlosseri.\r\nWorking group : Lucia Manni, Fabio Gasparini, Kohji Hotta, Katherine J. Ishizuka J,  Lorenzo Ricci, Stefano Tiozzo, Ayelet Voskoboynik, Delphine Dauga",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/BSAO/1/bsao.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "",
+            "id": 536,
+            "indexed": true,
+            "max_children": 8,
+            "max_depth": 0,
+            "md5sum": null,
+            "nb_classes": 104,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 6,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "Version 1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BTO_ONTOLOGY",
+        "name": "Brain Tumour Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2020-05-04T09:51:06Z",
+            "date_released": "2016-01-01T00:00:00Z",
+            "description": "The purpose of this work is to develop an ontology-based framework for developing an information retrieval system to cater to specific queries of users. For creating such an ontology, information was obtained from a wide range of information sources involved with brain tumour study and research. The information thus obtained was compiled and analysed to provide a standard, reliable and relevant information base to aid our proposed system. Facet-based methodology has been used for ontology formalization for quite some time. Ontology formalization involves different steps such as identification of the terminology, analysis, synthesis, standardization and ordering. A vast majority of the ontologies being developed nowadays lack flexibility. This becomes a formidable constraint when it comes to interoperability. We found that a facet-based method provides a distinct guideline for the development of a robust and flexible model concerning the domain of brain tumours. Our attempt has been to bridge library and information science and computer science, which itself involved an experimental approach. It was discovered that a faceted approach is really enduring, as it helps in the achievement of properties like navigation, exploration and faceted browsing. Computer-based brain tumour ontology supports the work of researchers towards gathering information on brain tumour research and allows users across the world to intelligently access new scientific information quickly and efficiently.",
+            "documentation": "https://github.com/subhashishhh/BTO-Ontology",
+            "domain": null,
+            "download_url": "media/ontologies/BTO_ONTOLOGY/1/bto_ontology.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/subhashishhh/BTO-Ontology/blob/master/BrainTumour_ontology.owl",
+            "id": 3585,
+            "indexed": true,
+            "max_children": 11,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 28,
+            "nb_inconsistent": 0,
+            "nb_individuals": 146,
+            "nb_properties": 9,
+            "products": null,
+            "publication": "Das, Subhashis, and Sayon Roy. \"Faceted Ontological Model for Brain Tumour Study.\" KO KNOWLEDGE ORGANIZATION 43, no. 1 (2016): 3-12.",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "KISAO",
         "name": "Kinetic Simulation Algorithm Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 17,
             "classifiable": true,
@@ -25760,76 +25794,176 @@
             "submission_id": 36,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "KTAO",
-        "name": "Kidney Tissue Atlas Ontology",
+        "acronym": "CANONT",
+        "name": "Upper-Level Cancer Ontology ",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-02-15T02:10:50Z",
-            "date_released": "2023-02-15T02:10:39Z",
-            "description": "The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is \r\ndeveloped to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP). ",
+            "date_created": "2012-11-20T13:39:03Z",
+            "date_released": "2012-11-20T08:00:00Z",
+            "description": "Providing an upper-level ontology for cancer.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/KTAO/19/ktao.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/KPMP/KTAO",
-            "id": 5677,
+            "download_url": "media/ontologies/CANONT/1/canont.obo",
+            "has_ontology_language": "OBO",
+            "home_page": null,
+            "id": 82,
             "indexed": true,
-            "max_children": 395,
-            "max_depth": 38,
+            "max_children": 4,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 11040,
+            "nb_classes": 56,
             "nb_inconsistent": 0,
-            "nb_individuals": 40,
-            "nb_properties": 387,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 19,
+            "submission_id": 1,
             "taxon": null,
-            "version": "Vision Release: 1.0.114"
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "LC-SUBJSCHEMES",
-        "name": "Library of Congress Subject Schemes",
+        "acronym": "BSPO",
+        "name": "Biological Spatial Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2020-08-19T01:07:11Z",
-            "date_released": "2020-08-19T01:07:09Z",
-            "description": "ubject Schemes contains a list of subject schemes and assigns a URI to each scheme. The purpose of these properties is to permit users to associate controlled subjects with the appropriate subject schemes. These schemes are general subject heading and thesauri, such as the Library of Congress Subject Headings, along with a number of focused subject term lists like Medical Subject Headings or Getty Thesaurus of Geographic Names.",
+            "date_created": "2023-05-29T12:01:49.764370Z",
+            "date_released": "2023-05-29T12:01:49.764354Z",
+            "description": "An ontology for representing spatial concepts, anatomical axes, gradients, regions, planes, sides, and surfaces",
             "documentation": null,
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/BSPO/31/bspo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obophenotype/biological-spatial-ontology",
+            "id": 5826,
+            "indexed": true,
+            "max_children": 23,
+            "max_depth": 9,
+            "md5sum": "e81b08fb6d9f221e321de5391f856abd",
+            "nb_classes": 170,
+            "nb_inconsistent": 0,
+            "nb_individuals": 18,
+            "nb_properties": 172,
+            "products": [
+                {
+                    "id": "bspo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bspo.owl"
+                },
+                {
+                    "id": "bspo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bspo.obo"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/25140222",
+                    "title": "Nose to tail, roots to shoots: spatial descriptors for phenotypic diversity in the Biological Spatial Ontology."
+                }
+            ],
+            "submission_id": 31,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BT",
+        "name": "BioTop Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 0,
+            "classifiable": true,
+            "date_created": "2016-10-23T01:00:32Z",
+            "date_released": "2016-10-23T01:00:31Z",
+            "description": "A top-domain ontology that provides definitions for the foundational entities of biomedicine as a basic vocabulary to unambiguously describe facts in this domain.",
+            "documentation": "http://purl.org/biotop",
             "domain": null,
-            "download_url": "media/ontologies/LC-SUBJSCHEMES/13/lc-subjschemes.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/vocabulary/subjectSchemes.html",
-            "id": 3884,
+            "download_url": "media/ontologies/BT/12/bt.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://purl.org/biotop",
+            "id": 468,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 3,
+            "nb_classes": 390,
             "nb_inconsistent": 0,
-            "nb_individuals": 351,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 83,
             "products": null,
-            "publication": null,
+            "publication": "http://purl.org/biotop",
             "publications": null,
-            "submission_id": 13,
+            "submission_id": 12,
+            "taxon": null,
+            "version": "Last modification\nApr 24, 2012 by Stefan Schulz"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BTO",
+        "name": "BRENDA tissue / enzyme source",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2021-11-01T12:24:47.095060Z",
+            "date_released": "2021-11-01T12:24:47.095037Z",
+            "description": "A structured controlled vocabulary for the source of an enzyme comprising tissues, cell lines, cell types and cell cultures.",
+            "documentation": null,
+            "domain": "anatomy",
+            "download_url": "media/ontologies/BTO/39/bto.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.brenda-enzymes.org",
+            "id": 4653,
+            "indexed": true,
+            "max_children": 2264,
+            "max_depth": 7,
+            "md5sum": "bace9597267f0b286b58d4b5c2039f55",
+            "nb_classes": 6569,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 10,
+            "products": [
+                {
+                    "id": "bto.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bto.owl"
+                },
+                {
+                    "id": "bto.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bto.obo"
+                },
+                {
+                    "id": "bto.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/bto.json"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/21030441",
+                    "title": "The BRENDA Tissue Ontology (BTO): the first all-integrating ontology of all organisms for enzyme sources"
+                }
+            ],
+            "submission_id": 39,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "ISO19115",
@@ -25862,790 +25996,830 @@
             "submission_id": 2,
             "taxon": null,
             "version": "2014"
         },
         "topics": null
     },
     {
-        "acronym": "LDA",
-        "name": "Ontology of Language Disorder in Autism",
+        "acronym": "IXNO",
+        "name": "Interaction Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
-            "date_created": "2009-09-28T15:47:35Z",
-            "date_released": "2008-12-15T08:00:00Z",
-            "description": "We present the short review of language terms used in the domain of autism and populate them into ontology available for consultation and sharing. The language terms were obtained via text mining and automatic retrieval of terms from the corpus of PubMed abstracts.",
+            "date_created": "2011-04-21T16:43:51Z",
+            "date_released": "2011-01-25T08:00:00Z",
+            "description": "The Interaction ontology (IxnO) was developed to enable curation of chemical-gene and chemical-protein interactions for the Comparative Toxicogenomics Database (CTD). CTD is a freely available resource that aims to promote understanding and novel hypothesis development about the effects of the environment on human health.",
+            "documentation": "http://ctd.mdibl.org/reports/CTD_chem_gene_ixn_types.obo",
+            "domain": null,
+            "download_url": "media/ontologies/IXNO/2/ixno.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://ctd.mdibl.org",
+            "id": 164,
+            "indexed": true,
+            "max_children": 24,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 64,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NGSONTO",
+        "name": "NGS ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2017-03-13T16:08:48Z",
+            "date_released": "2016-03-12T00:00:00Z",
+            "description": "The NGSOnto ontology aims at capturing the workflow of all the processes involved in a Next Generation Sequencing, in order to ensure the reproducibility of the entire process, through the use of a controled and specific vocabulary. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LDA/1/lda.owl",
+            "download_url": "media/ontologies/NGSONTO/32/ngsonto.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 403,
+            "id": 238,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 2,
+            "max_children": 7,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 36,
+            "nb_classes": 94,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_properties": 52,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 32,
             "taxon": null,
-            "version": "12-15-2008"
+            "version": "1.2"
         },
         "topics": null
     },
     {
-        "acronym": "ISPO",
-        "name": "Integrated Ontology for Symptom Phenotype Terminologies",
+        "acronym": "JERM",
+        "name": "Just Enough Results Model Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-01-10T08:46:41Z",
-            "date_released": "2023-01-10T00:00:00Z",
-            "description": "ISPO delivers an integrated controlled vocabulary for symptom phenotypes, with both clinical and biomedical literature synonyms in Chinese and English languages, it would enhance the semantic interoperability among heterogeneous medical data sources and clinical decision support systems. Furthermore, symptom ontology would propose a standard vocabulary for teasing the symptom-gene associations and thus become a vital requirement for symptom science and TCM biological discovery.",
+            "date_created": "2017-09-26T09:55:12Z",
+            "date_released": "2017-09-26T00:00:00Z",
+            "description": "The JERM ontology is an application ontology that describes the relationships between the data, models and other research assets that can be shared through the FAIRDOM platform, for sharing Systems and Synthetic Biology experiments. JERM 2 includes a new model for describing Biosamples, developed during the ELIXIR-lead Biosamples club",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ISPO/1/ispo.owl",
+            "download_url": "media/ontologies/JERM/15/jerm.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.tcmkg.com/",
-            "id": 5602,
+            "home_page": "https://fair-dom.org/",
+            "id": 743,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 11,
+            "max_children": 16,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 3148,
+            "nb_classes": 295,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 48,
             "products": null,
-            "publication": null,
+            "publication": "https://fair-dom.org/publications/",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 15,
             "taxon": null,
-            "version": "2023V1"
+            "version": "2"
         },
         "topics": null
     },
     {
-        "acronym": "ITEMAS",
-        "name": "Medical Technology Innovation in healthcare centers",
+        "acronym": "ISO19115MI",
+        "name": "ISO 19115 Metadata Information",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 22,
             "classifiable": true,
-            "date_created": "2017-11-16T09:40:20Z",
-            "date_released": "2017-11-16T00:00:00Z",
-            "description": "The defined ontology is focused on  addressing medical innovation technology in public healthcare institutions. This ontology is currently only in Spanish language. It has been developed the Platform for Innovation Medical and Health Technology (ITEMAS) a network composed by 67 Spanish healthcare organizations (including hospitals, healthcare centers, Ministry of Health, Social Services and Equality and more than 100 organizations focused on medical technology innovation. The ITEMAS Platform  has been by Carlos III National Health Institute (PT13/0006/0036), both co-funded by FEDER from European Regional Development Funds (European Union).",
-            "documentation": "https://drive.google.com/file/d/0B9CVhUu0djsXWkppaXpMdWppeE0/view?usp=sharing",
+            "date_created": "2017-12-11T04:37:37Z",
+            "date_released": "2017-12-11T04:37:37Z",
+            "description": "ISO 19115 Metadata Information section",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ITEMAS/1/itemas.owl",
+            "download_url": "media/ontologies/ISO19115MI/136/iso19115mi.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.itemas.org",
-            "id": 830,
+            "home_page": null,
+            "id": 948,
             "indexed": true,
-            "max_children": 36,
-            "max_depth": 6,
+            "max_children": 22,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 135,
+            "nb_classes": 23,
             "nb_inconsistent": 0,
-            "nb_individuals": 34,
-            "nb_properties": 144,
+            "nb_individuals": 0,
+            "nb_properties": 25,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 136,
             "taxon": null,
-            "version": "1.1."
+            "version": "2014"
         },
         "topics": null
     },
     {
-        "acronym": "ITO",
-        "name": "Intelligence Task Ontology",
+        "acronym": "BRO",
+        "name": "Biomedical Resource Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 14,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2022-09-09T08:53:52Z",
-            "date_released": "2022-09-09T00:00:00Z",
-            "description": "The Intelligence Task Ontology and Knowledge Graph (ITO) provides a comprehensive, curated model of artificial intelligence tasks, benchmarks and benchmark results, including the biomedical domain.\r\n\r\nNOTE: The version uploaded to BioPortal has all instances and benchmark data removed because of performance issues with the BioPortal class browser. Please refer to https://github.com/OpenBioLink/ITO/ for downloading the full version of the ontology and dataset.",
-            "documentation": null,
+            "date_created": "2010-08-31T20:09:49Z",
+            "date_released": "2010-08-31T07:00:00Z",
+            "description": "A controlled terminology of resources, which is used to improve the sensitivity and specificity of web searches.",
+            "documentation": "http://www.ncbcs.org/biositemaps/",
             "domain": null,
-            "download_url": "media/ontologies/ITO/17/ito.owl",
+            "download_url": "media/ontologies/BRO/14/bro.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/OpenBioLink/ITO/",
-            "id": 5346,
+            "home_page": "http://www.ncbcs.org/biositemaps/",
+            "id": 66,
             "indexed": true,
-            "max_children": 11326,
-            "max_depth": 10,
+            "max_children": 65,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 16283,
+            "nb_classes": 488,
             "nb_inconsistent": 0,
-            "nb_individuals": 37222,
-            "nb_properties": 16,
+            "nb_individuals": 52,
+            "nb_properties": 64,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 17,
+            "submission_id": 14,
             "taxon": null,
-            "version": "1.12 (PWC export dated 2022-07-30) - internal, just a minor fix"
+            "version": "3.2.1"
         },
         "topics": null
     },
     {
-        "acronym": "BRSO",
-        "name": "Biological Resource Schema Ontology",
+        "acronym": "CHEAR",
+        "name": "Children's Health Exposure Analysis Resource",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-12-21T05:34:45Z",
-            "date_released": "2021-12-20T00:00:00Z",
-            "description": "Biological Resource Schema Ontology",
+            "date_created": "2017-07-17T22:11:01Z",
+            "date_released": "2017-05-22T00:00:00Z",
+            "description": "Children's health and wellbeing are influenced by interactions between environmental and genetic factors. NIEHS is establishing an infrastructure, the Children's Health Exposure Analysis Resource (CHEAR), to provide the extramural research community access to laboratory and data analyses that add or expand the inclusion of environmental exposures in children's health research. The goal of CHEAR is to provide tools so researchers can assess the full range of environmental exposures which may affect children's health. We anticipate that CHEAR will be used by children's health researchers conducting epidemiological or clinical studies that currently have limited consideration of environmental exposures, or those who have collected exposure data but seek more extensive analyses.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BRSO/1/brso.owl",
+            "download_url": "media/ontologies/CHEAR/5/chear.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/dbcls/brso",
-            "id": 4787,
+            "home_page": null,
+            "id": 38,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 2,
+            "max_children": 190,
+            "max_depth": 34,
             "md5sum": null,
-            "nb_classes": 15,
+            "nb_classes": 10469,
             "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 8,
+            "nb_individuals": 38,
+            "nb_properties": 741,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 5,
             "taxon": null,
-            "version": "0.02"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "BSAO",
-        "name": "Botryllus schlosseri anatomy and development ontology",
+        "acronym": "ISO19115ROLES",
+        "name": "ISO 19115 Role Codes",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 8,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2013-08-29T09:04:07Z",
-            "date_released": "2013-06-27T07:00:00Z",
-            "description": "The first ontology describing the anatomy and the development of Botryllus schlosseri.\r\nWorking group : Lucia Manni, Fabio Gasparini, Kohji Hotta, Katherine J. Ishizuka J,  Lorenzo Ricci, Stefano Tiozzo, Ayelet Voskoboynik, Delphine Dauga",
-            "documentation": null,
+            "date_created": "2017-12-02T16:27:35Z",
+            "date_released": "2015-09-10T00:00:00Z",
+            "description": "Role Codes for use in ISO 19115-1:2014 metadata",
+            "documentation": "http://geosciml.org/doc/vocabularies",
             "domain": null,
-            "download_url": "media/ontologies/BSAO/1/bsao.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "",
-            "id": 536,
+            "download_url": "media/ontologies/ISO19115ROLES/6/iso19115roles.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 900,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 0,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 104,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_individuals": 20,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
-            "version": "Version 1"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "NGSONTO",
-        "name": "NGS ontology",
+        "acronym": "ISO19115TCC",
+        "name": "ISO 19115 Topic Categories",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-03-13T16:08:48Z",
-            "date_released": "2016-03-12T00:00:00Z",
-            "description": "The NGSOnto ontology aims at capturing the workflow of all the processes involved in a Next Generation Sequencing, in order to ensure the reproducibility of the entire process, through the use of a controled and specific vocabulary. ",
+            "date_created": "2017-12-03T00:53:02Z",
+            "date_released": "2017-07-21T00:00:00Z",
+            "description": "ISO 19115 Topic Category Codes",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NGSONTO/32/ngsonto.owl",
+            "download_url": "media/ontologies/ISO19115TCC/10/iso19115tcc.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 238,
+            "id": 902,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 7,
+            "max_children": 3,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 94,
+            "nb_classes": 5,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 52,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 32,
+            "submission_id": 10,
             "taxon": null,
-            "version": "1.2"
+            "version": "Created with TopBraid Composer"
         },
         "topics": null
     },
     {
-        "acronym": "BSPO",
-        "name": "Biological Spatial Ontology",
+        "acronym": "ISO639-2",
+        "name": "ISO 639-2: Codes for the Representation of Names of Languages",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2023-05-29T12:01:49.764370Z",
-            "date_released": "2023-05-29T12:01:49.764354Z",
-            "description": "An ontology for representing spatial concepts, anatomical axes, gradients, regions, planes, sides, and surfaces",
+            "date_created": "2017-12-02T03:04:52Z",
+            "date_released": "2017-12-02T03:04:52Z",
+            "description": "ISO 639-2 is part of the ISO 639 language code family, which provides also a two-character code set (ISO 639-1) for the representation of names of languages. ISO 639-2 contains codes for all languages contained in ISO 639-1 and several hundred additional languages. The ISO 639-2 (Bibliographic) codes were devised for use in bibliographic metadata, e.g., for libraries, information services, and publishers, and ISO 639-2 (Terminology) targets terminology, lexicography, and linguistic applications. The lists are the same except for 20 languages that have different Bibliographic and Terminology codes. The list contains over 500 discrete codes.",
             "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/BSPO/31/bspo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/ISO639-2/3/iso639-2.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/biological-spatial-ontology",
-            "id": 5826,
+            "home_page": "http://id.loc.gov/vocabulary/iso639-2.html",
+            "id": 893,
             "indexed": true,
-            "max_children": 23,
-            "max_depth": 9,
-            "md5sum": "e81b08fb6d9f221e321de5391f856abd",
-            "nb_classes": 170,
+            "max_children": 1,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 1,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 172,
-            "products": [
-                {
-                    "id": "bspo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bspo.owl"
-                },
-                {
-                    "id": "bspo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bspo.obo"
-                }
-            ],
+            "nb_individuals": 1,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/25140222",
-                    "title": "Nose to tail, roots to shoots: spatial descriptors for phenotypic diversity in the Biological Spatial Ontology."
-                }
-            ],
-            "submission_id": 31,
+            "publications": null,
+            "submission_id": 3,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "BT",
-        "name": "BioTop Ontology",
+        "acronym": "ISO-FOOD",
+        "name": "A formal representation of the knowledge within the domain of Isotopes for Food Science",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 12,
             "classifiable": true,
-            "date_created": "2016-10-23T01:00:32Z",
-            "date_released": "2016-10-23T01:00:31Z",
-            "description": "A top-domain ontology that provides definitions for the foundational entities of biomedicine as a basic vocabulary to unambiguously describe facts in this domain.",
-            "documentation": "http://purl.org/biotop",
+            "date_created": "2018-10-22T11:37:53Z",
+            "date_released": "2018-10-22T00:00:00Z",
+            "description": "To link and harmonize different knowledge repositories with respect to isotopic data, we propose an ISO-FOOD ontology as a domain ontology for describing isotopic data within Food Science. The ISO-FOOD ontology consists of metadata and provenance data that needs to be stored together with data elements in order to describe isotopic measurements with all necessary information required for future analysis. The new domain has been linked with existing ontologies, such as Units of Measurements Ontology, Food, Nutrient and the Bibliographic Ontology. So linked ontology consists of several superclasses, and for each of these the relevant subclasses, object properties and data properties are defined. To show how such an ontology can be used in practice, it was populated with 20 isotopic measurements of Slovenian food samples. Describing data in this way offers a powerful technique for organizing and sharing stable isotope data across Food Science. ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BT/12/bt.owl",
+            "download_url": "media/ontologies/ISO-FOOD/1/iso-food.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://purl.org/biotop",
-            "id": 468,
+            "home_page": "http://cs.ijs.si/repository/ISO-FOOD/ISO-FOOD_Ontology.zip",
+            "id": 1946,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 862,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 390,
+            "nb_classes": 1323,
+            "nb_inconsistent": 0,
+            "nb_individuals": 126,
+            "nb_properties": 92,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "BRO-AreaOfResearch",
+        "name": "Biomedical Resource Ontology - Area of Research View",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 11,
+            "classifiable": true,
+            "date_created": "2010-06-28T03:25:11Z",
+            "date_released": "2010-05-17T07:00:00Z",
+            "description": "This view represents a subset of the BRO containing all the Area of Research classes that are relevant in the biomedical domain.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/BRO-AreaOfResearch/2/bro-areaofresearch.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.biositemaps.org/",
+            "id": 612,
+            "indexed": true,
+            "max_children": 19,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 22,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 83,
+            "nb_individuals": 1,
+            "nb_properties": 45,
             "products": null,
-            "publication": "http://purl.org/biotop",
+            "publication": null,
             "publications": null,
-            "submission_id": 12,
+            "submission_id": 2,
             "taxon": null,
-            "version": "Last modification\nApr 24, 2012 by Stefan Schulz"
+            "version": "3.1"
         },
         "topics": null
     },
     {
-        "acronym": "BTO",
-        "name": "BRENDA tissue / enzyme source",
+        "acronym": "CANCO",
+        "name": "Cancer Chemoprevention Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-11-01T12:24:47.095060Z",
-            "date_released": "2021-11-01T12:24:47.095037Z",
-            "description": "A structured controlled vocabulary for the source of an enzyme comprising tissues, cell lines, cell types and cell cultures.",
+            "date_created": "2013-05-01T16:16:39Z",
+            "date_released": "2013-05-01T07:00:00Z",
+            "description": "The Cancer Chemoprevention Ontology constitutes a vocabulary that is able to describe and semantically interconnect the different paradigms of the cancer chemoprevention domain.",
             "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/BTO/39/bto.owl",
+            "domain": null,
+            "download_url": "media/ontologies/CANCO/3/canco.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.brenda-enzymes.org",
-            "id": 4653,
+            "home_page": null,
+            "id": 136,
             "indexed": true,
-            "max_children": 2264,
-            "max_depth": 7,
-            "md5sum": "bace9597267f0b286b58d4b5c2039f55",
-            "nb_classes": 6569,
+            "max_children": 11,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 127,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 10,
-            "products": [
-                {
-                    "id": "bto.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bto.owl"
-                },
-                {
-                    "id": "bto.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bto.obo"
-                },
-                {
-                    "id": "bto.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/bto.json"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/21030441",
-                    "title": "The BRENDA Tissue Ontology (BTO): the first all-integrating ontology of all organisms for enzyme sources"
-                }
-            ],
-            "submission_id": 39,
+            "nb_individuals": 13,
+            "nb_properties": 93,
+            "products": null,
+            "publication": "http://www.semantic-web-journal.net/content/collaborative-development-common-semantic-model-interlinking-cancer-chemoprevention-linked-d",
+            "publications": null,
+            "submission_id": 3,
             "taxon": null,
-            "version": null
+            "version": "0.3"
         },
         "topics": null
     },
     {
-        "acronym": "BRO",
-        "name": "Biomedical Resource Ontology",
+        "acronym": "BRO-Chinese",
+        "name": "Biomedical Resource Ontology with Simplified Chinese annotations.",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2010-08-31T20:09:49Z",
-            "date_released": "2010-08-31T07:00:00Z",
-            "description": "A controlled terminology of resources, which is used to improve the sensitivity and specificity of web searches.",
-            "documentation": "http://www.ncbcs.org/biositemaps/",
+            "date_created": "2009-10-06T22:25:54Z",
+            "date_released": "2009-02-24T08:00:00Z",
+            "description": "This view contains the entire BRO with the addition of three DataType properties, i.e., preferredName, shortName and synonym; then for all the classes and properties, Simplified Chinese (and some English) annotations were inserted into the ontology.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BRO/14/bro.owl",
+            "download_url": "media/ontologies/BRO-Chinese/1/bro-chinese.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.ncbcs.org/biositemaps/",
-            "id": 66,
+            "home_page": null,
+            "id": 390,
             "indexed": true,
-            "max_children": 65,
-            "max_depth": 7,
+            "max_children": 13,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 488,
+            "nb_classes": 311,
             "nb_inconsistent": 0,
-            "nb_individuals": 52,
-            "nb_properties": 64,
+            "nb_individuals": 0,
+            "nb_properties": 39,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 1,
             "taxon": null,
-            "version": "3.2.1"
+            "version": "2.5"
         },
         "topics": null
     },
     {
-        "acronym": "CHEAR",
-        "name": "Children's Health Exposure Analysis Resource",
+        "acronym": "KGCL",
+        "name": "Knowledge Graph Change Language",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-07-17T22:11:01Z",
-            "date_released": "2017-05-22T00:00:00Z",
-            "description": "Children's health and wellbeing are influenced by interactions between environmental and genetic factors. NIEHS is establishing an infrastructure, the Children's Health Exposure Analysis Resource (CHEAR), to provide the extramural research community access to laboratory and data analyses that add or expand the inclusion of environmental exposures in children's health research. The goal of CHEAR is to provide tools so researchers can assess the full range of environmental exposures which may affect children's health. We anticipate that CHEAR will be used by children's health researchers conducting epidemiological or clinical studies that currently have limited consideration of environmental exposures, or those who have collected exposure data but seek more extensive analyses.",
-            "documentation": null,
+            "date_created": "2023-04-07T01:19:04Z",
+            "date_released": "2023-04-07T01:19:02Z",
+            "description": "Knowledge Graph Change Language ontology.",
+            "documentation": "https://incatools.github.io/kgcl/",
             "domain": null,
-            "download_url": "media/ontologies/CHEAR/5/chear.owl",
+            "download_url": "media/ontologies/KGCL/3/kgcl.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 38,
+            "home_page": "https://github.com/INCATools/kgcl",
+            "id": 5760,
             "indexed": true,
-            "max_children": 190,
-            "max_depth": 34,
+            "max_children": 13,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 10469,
+            "nb_classes": 92,
             "nb_inconsistent": 0,
-            "nb_individuals": 38,
-            "nb_properties": 741,
+            "nb_individuals": 11,
+            "nb_properties": 32,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.1"
+            "version": "0.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "BTO_ONTOLOGY",
-        "name": "Brain Tumour Ontology",
+        "acronym": "KTAO",
+        "name": "Kidney Tissue Atlas Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2020-05-04T09:51:06Z",
-            "date_released": "2016-01-01T00:00:00Z",
-            "description": "The purpose of this work is to develop an ontology-based framework for developing an information retrieval system to cater to specific queries of users. For creating such an ontology, information was obtained from a wide range of information sources involved with brain tumour study and research. The information thus obtained was compiled and analysed to provide a standard, reliable and relevant information base to aid our proposed system. Facet-based methodology has been used for ontology formalization for quite some time. Ontology formalization involves different steps such as identification of the terminology, analysis, synthesis, standardization and ordering. A vast majority of the ontologies being developed nowadays lack flexibility. This becomes a formidable constraint when it comes to interoperability. We found that a facet-based method provides a distinct guideline for the development of a robust and flexible model concerning the domain of brain tumours. Our attempt has been to bridge library and information science and computer science, which itself involved an experimental approach. It was discovered that a faceted approach is really enduring, as it helps in the achievement of properties like navigation, exploration and faceted browsing. Computer-based brain tumour ontology supports the work of researchers towards gathering information on brain tumour research and allows users across the world to intelligently access new scientific information quickly and efficiently.",
-            "documentation": "https://github.com/subhashishhh/BTO-Ontology",
+            "date_created": "2023-02-15T02:10:50Z",
+            "date_released": "2023-02-15T02:10:39Z",
+            "description": "The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is \r\ndeveloped to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP). ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/BTO_ONTOLOGY/1/bto_ontology.owl",
+            "download_url": "media/ontologies/KTAO/19/ktao.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/subhashishhh/BTO-Ontology/blob/master/BrainTumour_ontology.owl",
-            "id": 3585,
+            "home_page": "https://github.com/KPMP/KTAO",
+            "id": 5677,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 4,
+            "max_children": 395,
+            "max_depth": 38,
             "md5sum": null,
-            "nb_classes": 28,
+            "nb_classes": 11040,
             "nb_inconsistent": 0,
-            "nb_individuals": 146,
-            "nb_properties": 9,
+            "nb_individuals": 40,
+            "nb_properties": 387,
             "products": null,
-            "publication": "Das, Subhashis, and Sayon Roy. \"Faceted Ontological Model for Brain Tumour Study.\" KO KNOWLEDGE ORGANIZATION 43, no. 1 (2016): 3-12.",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 19,
             "taxon": null,
-            "version": "1"
+            "version": "Vision Release: 1.0.114"
         },
         "topics": null
     },
     {
-        "acronym": "CANONT",
-        "name": "Upper-Level Cancer Ontology ",
+        "acronym": "LC-SUBJSCHEMES",
+        "name": "Library of Congress Subject Schemes",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2012-11-20T13:39:03Z",
-            "date_released": "2012-11-20T08:00:00Z",
-            "description": "Providing an upper-level ontology for cancer.",
+            "date_created": "2020-08-19T01:07:11Z",
+            "date_released": "2020-08-19T01:07:09Z",
+            "description": "ubject Schemes contains a list of subject schemes and assigns a URI to each scheme. The purpose of these properties is to permit users to associate controlled subjects with the appropriate subject schemes. These schemes are general subject heading and thesauri, such as the Library of Congress Subject Headings, along with a number of focused subject term lists like Medical Subject Headings or Getty Thesaurus of Geographic Names.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CANONT/1/canont.obo",
-            "has_ontology_language": "OBO",
-            "home_page": null,
-            "id": 82,
+            "download_url": "media/ontologies/LC-SUBJSCHEMES/13/lc-subjschemes.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://id.loc.gov/vocabulary/subjectSchemes.html",
+            "id": 3884,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 9,
+            "max_children": 3,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 56,
+            "nb_classes": 3,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 351,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 13,
             "taxon": null,
-            "version": "unknown"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "IXNO",
-        "name": "Interaction Ontology",
+        "acronym": "LDA",
+        "name": "Ontology of Language Disorder in Autism",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
-            "date_created": "2011-04-21T16:43:51Z",
-            "date_released": "2011-01-25T08:00:00Z",
-            "description": "The Interaction ontology (IxnO) was developed to enable curation of chemical-gene and chemical-protein interactions for the Comparative Toxicogenomics Database (CTD). CTD is a freely available resource that aims to promote understanding and novel hypothesis development about the effects of the environment on human health.",
-            "documentation": "http://ctd.mdibl.org/reports/CTD_chem_gene_ixn_types.obo",
+            "date_created": "2009-09-28T15:47:35Z",
+            "date_released": "2008-12-15T08:00:00Z",
+            "description": "We present the short review of language terms used in the domain of autism and populate them into ontology available for consultation and sharing. The language terms were obtained via text mining and automatic retrieval of terms from the corpus of PubMed abstracts.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/IXNO/2/ixno.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://ctd.mdibl.org",
-            "id": 164,
+            "download_url": "media/ontologies/LDA/1/lda.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 403,
             "indexed": true,
-            "max_children": 24,
-            "max_depth": 3,
+            "max_children": 14,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 64,
+            "nb_classes": 36,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "12-15-2008"
         },
         "topics": null
     },
     {
-        "acronym": "JERM",
-        "name": "Just Enough Results Model Ontology",
+        "acronym": "ISPO",
+        "name": "Integrated Ontology for Symptom Phenotype Terminologies",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-09-26T09:55:12Z",
-            "date_released": "2017-09-26T00:00:00Z",
-            "description": "The JERM ontology is an application ontology that describes the relationships between the data, models and other research assets that can be shared through the FAIRDOM platform, for sharing Systems and Synthetic Biology experiments. JERM 2 includes a new model for describing Biosamples, developed during the ELIXIR-lead Biosamples club",
+            "date_created": "2023-01-10T08:46:41Z",
+            "date_released": "2023-01-10T00:00:00Z",
+            "description": "ISPO delivers an integrated controlled vocabulary for symptom phenotypes, with both clinical and biomedical literature synonyms in Chinese and English languages, it would enhance the semantic interoperability among heterogeneous medical data sources and clinical decision support systems. Furthermore, symptom ontology would propose a standard vocabulary for teasing the symptom-gene associations and thus become a vital requirement for symptom science and TCM biological discovery.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/JERM/15/jerm.owl",
+            "download_url": "media/ontologies/ISPO/1/ispo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://fair-dom.org/",
-            "id": 743,
+            "home_page": "http://www.tcmkg.com/",
+            "id": 5602,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 7,
+            "max_children": 51,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 295,
+            "nb_classes": 3148,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 48,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://fair-dom.org/publications/",
+            "publication": null,
             "publications": null,
-            "submission_id": 15,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2"
+            "version": "2023V1"
         },
         "topics": null
     },
     {
-        "acronym": "CANCO",
-        "name": "Cancer Chemoprevention Ontology",
+        "acronym": "ITEMAS",
+        "name": "Medical Technology Innovation in healthcare centers",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2013-05-01T16:16:39Z",
-            "date_released": "2013-05-01T07:00:00Z",
-            "description": "The Cancer Chemoprevention Ontology constitutes a vocabulary that is able to describe and semantically interconnect the different paradigms of the cancer chemoprevention domain.",
-            "documentation": null,
+            "date_created": "2017-11-16T09:40:20Z",
+            "date_released": "2017-11-16T00:00:00Z",
+            "description": "The defined ontology is focused on  addressing medical innovation technology in public healthcare institutions. This ontology is currently only in Spanish language. It has been developed the Platform for Innovation Medical and Health Technology (ITEMAS) a network composed by 67 Spanish healthcare organizations (including hospitals, healthcare centers, Ministry of Health, Social Services and Equality and more than 100 organizations focused on medical technology innovation. The ITEMAS Platform  has been by Carlos III National Health Institute (PT13/0006/0036), both co-funded by FEDER from European Regional Development Funds (European Union).",
+            "documentation": "https://drive.google.com/file/d/0B9CVhUu0djsXWkppaXpMdWppeE0/view?usp=sharing",
             "domain": null,
-            "download_url": "media/ontologies/CANCO/3/canco.owl",
+            "download_url": "media/ontologies/ITEMAS/1/itemas.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 136,
+            "home_page": "www.itemas.org",
+            "id": 830,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 8,
+            "max_children": 36,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 127,
+            "nb_classes": 135,
             "nb_inconsistent": 0,
-            "nb_individuals": 13,
-            "nb_properties": 93,
+            "nb_individuals": 34,
+            "nb_properties": 144,
             "products": null,
-            "publication": "http://www.semantic-web-journal.net/content/collaborative-development-common-semantic-model-interlinking-cancer-chemoprevention-linked-d",
+            "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.3"
+            "version": "1.1."
         },
         "topics": null
     },
     {
-        "acronym": "ISO19115MI",
-        "name": "ISO 19115 Metadata Information",
+        "acronym": "PCMO",
+        "name": "Pediatric Consultation and Monitoring Ontology ",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 22,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-12-11T04:37:37Z",
-            "date_released": "2017-12-11T04:37:37Z",
-            "description": "ISO 19115 Metadata Information section",
+            "date_created": "2017-11-22T14:46:45Z",
+            "date_released": "2015-12-25T00:00:00Z",
+            "description": "Pediatric Consultation and Monitoring Ontology consists of 10 main classes, 10 sub-classes, 39 properties (object-type and sub-object-type properties,  data type), numerous OWL individuals and almost 90 SWRL rules (medical cases based on ARTI based diseases for pediatric patients).\r\nAlmost all the rules of the knowledge base consider:\r\n-Body temperature\r\n-Currently observed number of the local, major, and minor symptoms\r\n-Medications used and time of use in recent period (has drug and has drug time properties)\r\n-Presence of medical cold treatment applied in recent period \u21d2 only one of medical case will be inferencing result.\r\nProject Funding: \u201cThis research project was funded in 2014 by the National Science Foundation- The Scientific and Technological Research Council of Turkey (TUBITAK) under 1501 type funding program with the \u20183140417\u2019 grant number. The titled \u201cMobile Pediatric Consultation and Monitoring System\u201d project was successfully completed by the end of 2015. The project was completed by collaboration of the Ac\u0131badem Hospitals Group, Istanbul Aydin University and Semantica Internet and Software Services Trd. Ltd. Company in Turkey\u201d. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ISO19115MI/136/iso19115mi.owl",
+            "download_url": "media/ontologies/PCMO/4/pcmo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 948,
+            "home_page": "http://cmpe.emu.edu.tr/duygu/",
+            "id": 854,
             "indexed": true,
-            "max_children": 22,
-            "max_depth": 1,
+            "max_children": 10,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 23,
+            "nb_classes": 20,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 25,
+            "nb_individuals": 201,
+            "nb_properties": 27,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 136,
+            "submission_id": 4,
             "taxon": null,
-            "version": "2014"
+            "version": "v.4.0"
         },
         "topics": null
     },
     {
-        "acronym": "ISO19115ROLES",
-        "name": "ISO 19115 Role Codes",
-        "species": null,
+        "acronym": "PCPCKB",
+        "name": "PC Parts Compatibility Knowledge Base",
+        "species": [
+            "Ontology"
+        ],
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-12-02T16:27:35Z",
-            "date_released": "2015-09-10T00:00:00Z",
-            "description": "Role Codes for use in ISO 19115-1:2014 metadata",
-            "documentation": "http://geosciml.org/doc/vocabularies",
+            "date_created": "2023-05-05T12:04:34.485363Z",
+            "date_released": "2023-05-05T12:04:34.485394Z",
+            "description": "Still",
+            "documentation": "trust me",
             "domain": null,
-            "download_url": "media/ontologies/ISO19115ROLES/6/iso19115roles.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 900,
+            "download_url": "media/ontologies/PCPCKB/2/pcpckb.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com",
+            "id": 5793,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 5,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 23,
             "nb_inconsistent": 0,
-            "nb_individuals": 20,
-            "nb_properties": 0,
+            "nb_individuals": 6,
+            "nb_properties": 5,
             "products": null,
-            "publication": null,
+            "publication": "https://github.com/Grbson00/OWLOntologyRepo/tree/main/TestOntology",
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2.0"
+            "version": "1.1"
         },
-        "topics": null
+        "topics": [
+            "PC hardware compatibility"
+        ]
     },
     {
-        "acronym": "ISO19115TCC",
-        "name": "ISO 19115 Topic Categories",
+        "acronym": "MARC-LANGUAGES",
+        "name": "MARC List for Languages",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-12-03T00:53:02Z",
-            "date_released": "2017-07-21T00:00:00Z",
-            "description": "ISO 19115 Topic Category Codes",
+            "date_created": "2022-10-18T01:29:53Z",
+            "date_released": "2022-10-18T01:29:51Z",
+            "description": null,
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ISO19115TCC/10/iso19115tcc.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 902,
+            "download_url": "media/ontologies/MARC-LANGUAGES/161/marc-languages.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://id.loc.gov/vocabulary/languages.html",
+            "id": 5414,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 2,
+            "max_children": 4,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 5,
+            "nb_classes": 6,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 486,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 10,
+            "submission_id": 161,
             "taxon": null,
-            "version": "Created with TopBraid Composer"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ISO639-2",
-        "name": "ISO 639-2: Codes for the Representation of Names of Languages",
+        "acronym": "M4M19-VARS",
+        "name": "M4M19 Variables",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-12-02T03:04:52Z",
-            "date_released": "2017-12-02T03:04:52Z",
-            "description": "ISO 639-2 is part of the ISO 639 language code family, which provides also a two-character code set (ISO 639-1) for the representation of names of languages. ISO 639-2 contains codes for all languages contained in ISO 639-1 and several hundred additional languages. The ISO 639-2 (Bibliographic) codes were devised for use in bibliographic metadata, e.g., for libraries, information services, and publishers, and ISO 639-2 (Terminology) targets terminology, lexicography, and linguistic applications. The lists are the same except for 20 languages that have different Bibliographic and Terminology codes. The list contains over 500 discrete codes.",
-            "documentation": null,
+            "date_created": "2021-12-08T03:22:14Z",
+            "date_released": "2021-12-07T00:00:00Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is NICEST-2 Variables. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 19 (M4M.19 funded by EOSC-Nordic ) provided to the NICEST-2 project.\r\n\r\nsheet2rdf and OntoStack, developed by FAIR Data Collective, are used to build and serve NICEST-2 Subjects, while PURL will be used to persist identifiers for the vocabulary terms and properties:\r\n\r\nhttp://purl.org/m4m19/variables/",
+            "documentation": "https://github.com/fair-data-collective/M4M19-variables-vocabulary",
             "domain": null,
-            "download_url": "media/ontologies/ISO639-2/3/iso639-2.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://id.loc.gov/vocabulary/iso639-2.html",
-            "id": 893,
+            "download_url": "media/ontologies/M4M19-VARS/4/m4m19-vars.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/M4M19-variables-vocabulary",
+            "id": 4763,
             "indexed": true,
-            "max_children": 1,
+            "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 1,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
+            "nb_individuals": 22,
             "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "https://github.com/fair-data-collective/M4M19-variables-vocabulary",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 4,
             "taxon": null,
-            "version": null
+            "version": "0.2.0"
         },
         "topics": null
     },
     {
-        "acronym": "ISO-FOOD",
-        "name": "A formal representation of the knowledge within the domain of Isotopes for Food Science",
+        "acronym": "M4M-20-SUBJECTS",
+        "name": "M4M.20 FAIRWare Subjects",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 12,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2018-10-22T11:37:53Z",
-            "date_released": "2018-10-22T00:00:00Z",
-            "description": "To link and harmonize different knowledge repositories with respect to isotopic data, we propose an ISO-FOOD ontology as a domain ontology for describing isotopic data within Food Science. The ISO-FOOD ontology consists of metadata and provenance data that needs to be stored together with data elements in order to describe isotopic measurements with all necessary information required for future analysis. The new domain has been linked with existing ontologies, such as Units of Measurements Ontology, Food, Nutrient and the Bibliographic Ontology. So linked ontology consists of several superclasses, and for each of these the relevant subclasses, object properties and data properties are defined. To show how such an ontology can be used in practice, it was populated with 20 isotopic measurements of Slovenian food samples. Describing data in this way offers a powerful technique for organizing and sharing stable isotope data across Food Science. ",
-            "documentation": null,
+            "date_created": "2022-06-23T11:34:30Z",
+            "date_released": "2022-06-23T00:00:00Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Psychology Subjects. This controlled vocabulary is produced in Metadata 4 Machine (M4M) Workshop 20 (M4M.20 FAIRware Psychology), which one of deliverables of FAIRware project which if funded by RoRi.\r\n\r\nsheet2rdf is used to build and serve Psychology Subjects, while PURL is used to persist identifiers for the vocabulary, which resolve on BioPortal (Stanford's ontology service):\r\n\r\nhttp://purl.org/m4m20/subjects/",
+            "documentation": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Subjects",
             "domain": null,
-            "download_url": "media/ontologies/ISO-FOOD/1/iso-food.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://cs.ijs.si/repository/ISO-FOOD/ISO-FOOD_Ontology.zip",
-            "id": 1946,
+            "download_url": "media/ontologies/M4M-20-SUBJECTS/6/m4m-20-subjects.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Subjects",
+            "id": 5190,
             "indexed": true,
-            "max_children": 862,
-            "max_depth": 5,
+            "max_children": 5,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 1323,
+            "nb_classes": 10,
             "nb_inconsistent": 0,
-            "nb_individuals": 126,
-            "nb_properties": 92,
+            "nb_individuals": 16,
+            "nb_properties": 11,
             "products": null,
-            "publication": null,
+            "publication": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Subjects",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
-            "version": null
+            "version": "0.1.3"
         },
         "topics": null
     },
     {
         "acronym": "LOINC",
         "name": "Logical Observation Identifier Names and Codes",
         "species": null,
@@ -26812,316 +26986,325 @@
             "submission_id": 5,
             "taxon": null,
             "version": "Version 2.0"
         },
         "topics": null
     },
     {
-        "acronym": "LIFO",
-        "name": "Life Ontology",
+        "acronym": "LEPAO",
+        "name": "Lepidoptera Anatomy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2018-03-11T20:47:18Z",
-            "date_released": "2018-03-11T00:00:00Z",
-            "description": "The Life Ontology (LifO) is an ontology of the life of organism. LifO represents the life processes of organisms and related entities and relations. LifO is a general purpose ontology that covers the common features associated with different organisms such as unicellular prokaryotes (e.g., E. coli) and multicellular organisms (e.g., human).",
+            "date_created": "2023-02-20T12:24:46.472930Z",
+            "date_released": "2023-02-20T12:24:46.472915Z",
+            "description": "The Lepidoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of moths and butterflies in biodiversity research. LEPAO is developed in part by BIOfid (The Specialised Information Service Biodiversity Research).",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/LIFO/1/lifo.owl",
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/LEPAO/2/lepao.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/lifeontology/lifo",
-            "id": 1299,
+            "home_page": "https://github.com/insect-morphology/lepao",
+            "id": 5667,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 215,
+            "max_children": 71,
+            "max_depth": 8,
+            "md5sum": "2ebe95b84326be82c7b18cfdae8a2cec",
+            "nb_classes": 822,
             "nb_inconsistent": 0,
-            "nb_individuals": 12,
-            "nb_properties": 98,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 57,
+            "products": [
+                {
+                    "id": "lepao.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/lepao.owl"
+                },
+                {
+                    "id": "lepao.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/lepao.obo"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "Vision Release: 1.0.17"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LION",
-        "name": "LipidOntology",
+        "acronym": "M4M-20-VARIABLES",
+        "name": "M4M.20 FAIRWare Variables",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 30,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2020-07-08T11:58:30Z",
-            "date_released": "2019-11-09T00:00:00Z",
-            "description": "Lipid ontology (LION) associates >50,000 lipid species to biophysical, chemical, and cell biological features. ",
-            "documentation": null,
+            "date_created": "2022-06-24T01:11:20Z",
+            "date_released": "2022-06-24T01:11:19Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Psychology Variables. This controlled vocabulary is produced in Metadata 4 Machine (M4M) Workshop 20 (M4M.20 FAIRware Psychology), which is one of deliverables of FAIRware project funded by RoRi.\r\n\r\nsheet2rdf is used to build and serve Psychology Variables, while PURL is used to persist identifiers for the vocabulary, which resolve on BioPortal (Stanford's ontology service):\r\n\r\nhttp://purl.org/m4m20/variables/",
+            "documentation": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Variables",
             "domain": null,
-            "download_url": "media/ontologies/LION/1/lion.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "www.lipidontology.com",
-            "id": 3767,
+            "download_url": "media/ontologies/M4M-20-VARIABLES/6/m4m-20-variables.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Variables",
+            "id": 5205,
             "indexed": true,
-            "max_children": 2501,
-            "max_depth": 6,
+            "max_children": 5,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 63567,
+            "nb_classes": 10,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 9,
+            "nb_properties": 11,
             "products": null,
-            "publication": "https://doi.org/10.1093/gigascience/giz061",
+            "publication": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Variables",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
-            "version": "1.1"
+            "version": "0.1.3"
         },
         "topics": null
     },
     {
-        "acronym": "LUNGMAP_M_CELL",
-        "name": "Cell Ontology for Mouse Lung Maturation",
+        "acronym": "M4M-21-SUBJECTS",
+        "name": "M4M.21 FAIRWare Subjects",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2019-07-28T22:12:46Z",
-            "date_released": "2019-07-28T00:00:00Z",
-            "description": "Cell Ontology for Mouse Lung Maturation (LungMAP Mouse Anatomy (LMMA))\r\n\r\nDeveloped by Susan E Wert, Gail H. Deutsch, Helen Pan, and the NHLBI Molecular Atlas of Lung Development Program Consortium Ontology Subcommittee",
-            "documentation": null,
+            "date_created": "2022-06-23T14:58:04Z",
+            "date_released": "2022-06-23T00:00:00Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Neuroscience Subjects. This controlled vocabulary is produced in Metadata 4 Machine (M4M) Workshop 21 (M4M.21 FAIRware Neuroscience), which is one of deliverables of FAIRware project funded by RoRi.\r\n\r\nsheet2rdf is used to build and serve Neuroscience Subjects, while PURL is used to persist identifiers for the vocabulary, which resolve on BioPortal (Stanford's ontology service):\r\n\r\nhttp://purl.org/m4m21/subjects/",
+            "documentation": "https://github.com/fair-data-collective/M4M-21-FAIRware-Neuroscience-Subjects",
             "domain": null,
-            "download_url": "media/ontologies/LUNGMAP_M_CELL/5/lungmap_m_cell.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.lungmap.net/breath-ontology-browser/",
-            "id": 2793,
+            "download_url": "media/ontologies/M4M-21-SUBJECTS/4/m4m-21-subjects.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/M4M-21-FAIRware-Neuroscience-Subjects",
+            "id": 5195,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 7,
+            "max_children": 5,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 92,
+            "nb_classes": 10,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_individuals": 9,
+            "nb_properties": 11,
             "products": null,
-            "publication": null,
+            "publication": "https://github.com/fair-data-collective/M4M-21-FAIRware-Neuroscience-Subjects",
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 4,
             "taxon": null,
-            "version": "3.2"
+            "version": "0.1.2"
         },
         "topics": null
     },
     {
-        "acronym": "LUNGMAP-MOUSE",
-        "name": "Anatomic Ontology for Mouse Lung Maturation",
+        "acronym": "MARC-RELATORS",
+        "name": "MARC Code List for Relators",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2019-07-28T22:08:56Z",
-            "date_released": "2019-07-28T00:00:00Z",
-            "description": "Anatomic ontology for Mouse Lung Maturation (LungMAP Mouse Anatomy (LMMA)\r\n\r\nDeveloped by Susan E Wert, Gail H. Deutsch, Helen Pan, and the NHLBI Molecular Atlas of Lung Development Program Consortium Ontology Subcommittee",
+            "date_created": "2023-05-19T01:12:43Z",
+            "date_released": "2023-05-19T01:12:42Z",
+            "description": "Relator terms and their associated codes designate the relationship between a name and a bibliographic resource. The relator codes are three-character lowercase alphabetic strings that serve as identifiers. Either the term or the code may be used as controlled values.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LUNGMAP-MOUSE/4/lungmap-mouse.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://lungmap.net/breath-ontology-browser/",
-            "id": 2816,
+            "download_url": "media/ontologies/MARC-RELATORS/3/marc-relators.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5821,
             "indexed": true,
-            "max_children": 215,
-            "max_depth": 4,
+            "max_children": 3,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 308,
+            "nb_classes": 5,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_individuals": 271,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.2"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "M4M19-SUBS",
-        "name": "M4M19 Subjects",
+        "acronym": "LICO",
+        "name": "Liver Case Ontology (LiCO)",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2021-12-08T00:47:22Z",
-            "date_released": "2021-11-20T00:00:00Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is NICEST-2 Subjects. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 19 (M4M.19 funded by EOSC-Nordic ) provided to the NICEST-2 project.\r\n\r\nsheet2rdf and OntoStack, developed by FAIR Data Collective, are used to build and serve NICEST-2 Subjects, while PURL will be used to persist identifiers for the vocabulary terms and properties:\r\n\r\nhttp://purl.org/m4m19/subjects/",
-            "documentation": "https://github.com/fair-data-collective/M4M19-subjects-vocabulary",
+            "date_created": "2020-11-04T16:59:52Z",
+            "date_released": "2018-08-31T00:00:00Z",
+            "description": "Liver Case Ontology (LiCO) is a case-centric ontology for liver patient cases called lico (Liver Case Ontology) that integrates several ontologies and lexicons developed for sub-disciplines of medicine such as RadLex (Radiology Lexicon), SNOMED CT (Systematized Nomenclature of Medicine, Clinical Terms), LOINC (The Logical Observation Identifier Names and Codes). It is an extension of ONLIRA (Ontology of the Liver for Radiology) that represents the liver and lesions from a radiological perspective.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/M4M19-SUBS/3/m4m19-subs.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/M4M19-subjects-vocabulary",
-            "id": 4760,
+            "download_url": "media/ontologies/LICO/2/lico.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4045,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 24,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 93,
             "nb_inconsistent": 0,
-            "nb_individuals": 23,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 36,
             "products": null,
-            "publication": "https://github.com/fair-data-collective/M4M19-subjects-vocabulary",
+            "publication": "https://vavlab.boun.edu.tr/publications-patents?field_tags_tid%5B%5D=68&field_tags_tid%5B%5D=70",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.1.0"
+            "version": "changes\n* Segments and Regions are modelling as Classes in order to identify which patient (liver) they belong to.\n* Redundant restrictions are removed\n"
         },
         "topics": null
     },
     {
-        "acronym": "M4M19-VARS",
-        "name": "M4M19 Variables",
+        "acronym": "LIFO",
+        "name": "Life Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-12-08T03:22:14Z",
-            "date_released": "2021-12-07T00:00:00Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is NICEST-2 Variables. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 19 (M4M.19 funded by EOSC-Nordic ) provided to the NICEST-2 project.\r\n\r\nsheet2rdf and OntoStack, developed by FAIR Data Collective, are used to build and serve NICEST-2 Subjects, while PURL will be used to persist identifiers for the vocabulary terms and properties:\r\n\r\nhttp://purl.org/m4m19/variables/",
-            "documentation": "https://github.com/fair-data-collective/M4M19-variables-vocabulary",
+            "date_created": "2018-03-11T20:47:18Z",
+            "date_released": "2018-03-11T00:00:00Z",
+            "description": "The Life Ontology (LifO) is an ontology of the life of organism. LifO represents the life processes of organisms and related entities and relations. LifO is a general purpose ontology that covers the common features associated with different organisms such as unicellular prokaryotes (e.g., E. coli) and multicellular organisms (e.g., human).",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/M4M19-VARS/4/m4m19-vars.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/M4M19-variables-vocabulary",
-            "id": 4763,
+            "download_url": "media/ontologies/LIFO/1/lifo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/lifeontology/lifo",
+            "id": 1299,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 16,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 215,
             "nb_inconsistent": 0,
-            "nb_individuals": 22,
-            "nb_properties": 0,
+            "nb_individuals": 12,
+            "nb_properties": 98,
             "products": null,
-            "publication": "https://github.com/fair-data-collective/M4M19-variables-vocabulary",
+            "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.2.0"
+            "version": "Vision Release: 1.0.17"
         },
         "topics": null
     },
     {
-        "acronym": "M4M-20-SUBJECTS",
-        "name": "M4M.20 FAIRWare Subjects",
+        "acronym": "ORDO_FR",
+        "name": "ORDO FRENCH",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2022-06-23T11:34:30Z",
-            "date_released": "2022-06-23T00:00:00Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Psychology Subjects. This controlled vocabulary is produced in Metadata 4 Machine (M4M) Workshop 20 (M4M.20 FAIRware Psychology), which one of deliverables of FAIRware project which if funded by RoRi.\r\n\r\nsheet2rdf is used to build and serve Psychology Subjects, while PURL is used to persist identifiers for the vocabulary, which resolve on BioPortal (Stanford's ontology service):\r\n\r\nhttp://purl.org/m4m20/subjects/",
-            "documentation": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Subjects",
+            "date_created": "2023-07-03T15:06:11Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "This ontology is the french version of ORDO with definition, labels and synonyms in french.\r\nCette ontologie est la version fran\u00e7aise d'ORDO avec les d\u00e9finitions, labels et synonymes en fran\u00e7ais.",
+            "documentation": "https://www.orphadata.com/ordo/",
             "domain": null,
-            "download_url": "media/ontologies/M4M-20-SUBJECTS/6/m4m-20-subjects.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Subjects",
-            "id": 5190,
+            "download_url": "media/ontologies/ORDO_FR/12/ordo_fr.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.orphadata.com/ordo/",
+            "id": 5922,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 1,
+            "max_children": 4370,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 10,
+            "nb_classes": 15447,
             "nb_inconsistent": 0,
-            "nb_individuals": 16,
-            "nb_properties": 11,
+            "nb_individuals": 0,
+            "nb_properties": 13,
             "products": null,
-            "publication": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Subjects",
+            "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 12,
             "taxon": null,
-            "version": "0.1.3"
+            "version": "4.3"
         },
         "topics": null
     },
     {
-        "acronym": "M4M-20-VARIABLES",
-        "name": "M4M.20 FAIRWare Variables",
+        "acronym": "ORDO_NL",
+        "name": "ORDO DUTCH",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2022-06-24T01:11:20Z",
-            "date_released": "2022-06-24T01:11:19Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Psychology Variables. This controlled vocabulary is produced in Metadata 4 Machine (M4M) Workshop 20 (M4M.20 FAIRware Psychology), which is one of deliverables of FAIRware project funded by RoRi.\r\n\r\nsheet2rdf is used to build and serve Psychology Variables, while PURL is used to persist identifiers for the vocabulary, which resolve on BioPortal (Stanford's ontology service):\r\n\r\nhttp://purl.org/m4m20/variables/",
-            "documentation": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Variables",
+            "date_created": "2023-07-03T15:09:44Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "This ontology is the dutch version of ORDO with definition, labels and synonyms in dutch. ",
+            "documentation": "https://www.orphadata.com/ordo/",
             "domain": null,
-            "download_url": "media/ontologies/M4M-20-VARIABLES/6/m4m-20-variables.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Variables",
-            "id": 5205,
+            "download_url": "media/ontologies/ORDO_NL/14/ordo_nl.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.orphadata.com/ordo/",
+            "id": 5919,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 1,
+            "max_children": 4370,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 10,
+            "nb_classes": 15447,
             "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 11,
+            "nb_individuals": 0,
+            "nb_properties": 13,
             "products": null,
-            "publication": "https://github.com/fair-data-collective/M4M-20-FAIRware-Psychology-Variables",
+            "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 14,
             "taxon": null,
-            "version": "0.1.3"
+            "version": "4.3"
         },
         "topics": null
     },
     {
-        "acronym": "M4M-21-SUBJECTS",
-        "name": "M4M.21 FAIRWare Subjects",
+        "acronym": "ORDO_IT",
+        "name": "ORDO ITALIAN",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2022-06-23T14:58:04Z",
-            "date_released": "2022-06-23T00:00:00Z",
-            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is Neuroscience Subjects. This controlled vocabulary is produced in Metadata 4 Machine (M4M) Workshop 21 (M4M.21 FAIRware Neuroscience), which is one of deliverables of FAIRware project funded by RoRi.\r\n\r\nsheet2rdf is used to build and serve Neuroscience Subjects, while PURL is used to persist identifiers for the vocabulary, which resolve on BioPortal (Stanford's ontology service):\r\n\r\nhttp://purl.org/m4m21/subjects/",
-            "documentation": "https://github.com/fair-data-collective/M4M-21-FAIRware-Neuroscience-Subjects",
+            "date_created": "2023-07-03T15:08:27Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "This ontology is the italian version of ORDO with definition, labels and synonyms in italian. ",
+            "documentation": "https://www.orphadata.com/ordo/",
             "domain": null,
-            "download_url": "media/ontologies/M4M-21-SUBJECTS/4/m4m-21-subjects.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/fair-data-collective/M4M-21-FAIRware-Neuroscience-Subjects",
-            "id": 5195,
+            "download_url": "media/ontologies/ORDO_IT/8/ordo_it.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.orphadata.com/ordo/",
+            "id": 5917,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 1,
+            "max_children": 4370,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 10,
+            "nb_classes": 15447,
             "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 11,
+            "nb_individuals": 0,
+            "nb_properties": 13,
             "products": null,
-            "publication": "https://github.com/fair-data-collective/M4M-21-FAIRware-Neuroscience-Subjects",
+            "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 8,
             "taxon": null,
-            "version": "0.1.2"
+            "version": "4.3"
         },
         "topics": null
     },
     {
         "acronym": "M4M-21-VARIABLES",
         "name": "M4M.21 FAIRWare Variables",
         "species": null,
@@ -27301,180 +27484,146 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2015-10-17"
         },
         "topics": null
     },
     {
-        "acronym": "MARC-LANGUAGES",
-        "name": "MARC List for Languages",
+        "acronym": "LION",
+        "name": "LipidOntology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 30,
             "classifiable": true,
-            "date_created": "2022-10-18T01:29:53Z",
-            "date_released": "2022-10-18T01:29:51Z",
-            "description": null,
+            "date_created": "2020-07-08T11:58:30Z",
+            "date_released": "2019-11-09T00:00:00Z",
+            "description": "Lipid ontology (LION) associates >50,000 lipid species to biophysical, chemical, and cell biological features. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MARC-LANGUAGES/161/marc-languages.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://id.loc.gov/vocabulary/languages.html",
-            "id": 5414,
+            "download_url": "media/ontologies/LION/1/lion.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "www.lipidontology.com",
+            "id": 3767,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 1,
+            "max_children": 2501,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 6,
+            "nb_classes": 63567,
             "nb_inconsistent": 0,
-            "nb_individuals": 486,
+            "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "https://doi.org/10.1093/gigascience/giz061",
             "publications": null,
-            "submission_id": 161,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "MARC-RELATORS",
-        "name": "MARC Code List for Relators",
+        "acronym": "LUNGMAP_M_CELL",
+        "name": "Cell Ontology for Mouse Lung Maturation",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-05-19T01:12:43Z",
-            "date_released": "2023-05-19T01:12:42Z",
-            "description": "Relator terms and their associated codes designate the relationship between a name and a bibliographic resource. The relator codes are three-character lowercase alphabetic strings that serve as identifiers. Either the term or the code may be used as controlled values.",
+            "date_created": "2019-07-28T22:12:46Z",
+            "date_released": "2019-07-28T00:00:00Z",
+            "description": "Cell Ontology for Mouse Lung Maturation (LungMAP Mouse Anatomy (LMMA))\r\n\r\nDeveloped by Susan E Wert, Gail H. Deutsch, Helen Pan, and the NHLBI Molecular Atlas of Lung Development Program Consortium Ontology Subcommittee",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MARC-RELATORS/3/marc-relators.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 5821,
-            "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 5,
-            "nb_inconsistent": 0,
-            "nb_individuals": 271,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ORDO_FR",
-        "name": "ORDO FRENCH",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 15,
-            "classifiable": true,
-            "date_created": "2023-07-03T15:06:11Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "This ontology is the french version of ORDO with definition, labels and synonyms in french.\r\nCette ontologie est la version fran\u00e7aise d'ORDO avec les d\u00e9finitions, labels et synonymes en fran\u00e7ais.",
-            "documentation": "https://www.orphadata.com/ordo/",
-            "domain": null,
-            "download_url": "media/ontologies/ORDO_FR/12/ordo_fr.owl",
+            "download_url": "media/ontologies/LUNGMAP_M_CELL/5/lungmap_m_cell.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com/ordo/",
-            "id": 5922,
+            "home_page": "https://www.lungmap.net/breath-ontology-browser/",
+            "id": 2793,
             "indexed": true,
-            "max_children": 4370,
-            "max_depth": 11,
+            "max_children": 12,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 15447,
+            "nb_classes": 92,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_properties": 2,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
+            "submission_id": 5,
             "taxon": null,
-            "version": "4.3"
+            "version": "3.2"
         },
         "topics": null
     },
     {
-        "acronym": "ORDO_IT",
-        "name": "ORDO ITALIAN",
+        "acronym": "LUNGMAP-MOUSE",
+        "name": "Anatomic Ontology for Mouse Lung Maturation",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2023-07-03T15:08:27Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "This ontology is the italian version of ORDO with definition, labels and synonyms in italian. ",
-            "documentation": "https://www.orphadata.com/ordo/",
+            "date_created": "2019-07-28T22:08:56Z",
+            "date_released": "2019-07-28T00:00:00Z",
+            "description": "Anatomic ontology for Mouse Lung Maturation (LungMAP Mouse Anatomy (LMMA)\r\n\r\nDeveloped by Susan E Wert, Gail H. Deutsch, Helen Pan, and the NHLBI Molecular Atlas of Lung Development Program Consortium Ontology Subcommittee",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ORDO_IT/8/ordo_it.owl",
+            "download_url": "media/ontologies/LUNGMAP-MOUSE/4/lungmap-mouse.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com/ordo/",
-            "id": 5917,
+            "home_page": "https://lungmap.net/breath-ontology-browser/",
+            "id": 2816,
             "indexed": true,
-            "max_children": 4370,
-            "max_depth": 11,
+            "max_children": 215,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 15447,
+            "nb_classes": 308,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_properties": 6,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 4,
             "taxon": null,
-            "version": "4.3"
+            "version": "1.2"
         },
         "topics": null
     },
     {
-        "acronym": "ORDO_NL",
-        "name": "ORDO DUTCH",
+        "acronym": "M4M19-SUBS",
+        "name": "M4M19 Subjects",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-07-03T15:09:44Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "This ontology is the dutch version of ORDO with definition, labels and synonyms in dutch. ",
-            "documentation": "https://www.orphadata.com/ordo/",
+            "date_created": "2021-12-08T00:47:22Z",
+            "date_released": "2021-11-20T00:00:00Z",
+            "description": "Controlled vocabularies allow an accurate and controlled approach in describing physical and digital assets (e.g., data). One of such controlled vocabulary is NICEST-2 Subjects. This controlled vocabulary is a result of Metadata 4 Machine (M4M) Workshop 19 (M4M.19 funded by EOSC-Nordic ) provided to the NICEST-2 project.\r\n\r\nsheet2rdf and OntoStack, developed by FAIR Data Collective, are used to build and serve NICEST-2 Subjects, while PURL will be used to persist identifiers for the vocabulary terms and properties:\r\n\r\nhttp://purl.org/m4m19/subjects/",
+            "documentation": "https://github.com/fair-data-collective/M4M19-subjects-vocabulary",
             "domain": null,
-            "download_url": "media/ontologies/ORDO_NL/14/ordo_nl.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com/ordo/",
-            "id": 5919,
+            "download_url": "media/ontologies/M4M19-SUBS/3/m4m19-subs.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/fair-data-collective/M4M19-subjects-vocabulary",
+            "id": 4760,
             "indexed": true,
-            "max_children": 4370,
-            "max_depth": 11,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 15447,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_individuals": 23,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "https://github.com/fair-data-collective/M4M19-subjects-vocabulary",
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 3,
             "taxon": null,
-            "version": "4.3"
+            "version": "0.1.0"
         },
         "topics": null
     },
     {
         "acronym": "ORDO_OBO",
         "name": "Orphanet Rare Diseases Ontology in OBO",
         "species": null,
@@ -27505,261 +27654,275 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "PCMO",
-        "name": "Pediatric Consultation and Monitoring Ontology ",
+        "acronym": "MMUSDV",
+        "name": "Mouse Developmental Stages",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 56,
             "classifiable": true,
-            "date_created": "2017-11-22T14:46:45Z",
-            "date_released": "2015-12-25T00:00:00Z",
-            "description": "Pediatric Consultation and Monitoring Ontology consists of 10 main classes, 10 sub-classes, 39 properties (object-type and sub-object-type properties,  data type), numerous OWL individuals and almost 90 SWRL rules (medical cases based on ARTI based diseases for pediatric patients).\r\nAlmost all the rules of the knowledge base consider:\r\n-Body temperature\r\n-Currently observed number of the local, major, and minor symptoms\r\n-Medications used and time of use in recent period (has drug and has drug time properties)\r\n-Presence of medical cold treatment applied in recent period \u21d2 only one of medical case will be inferencing result.\r\nProject Funding: \u201cThis research project was funded in 2014 by the National Science Foundation- The Scientific and Technological Research Council of Turkey (TUBITAK) under 1501 type funding program with the \u20183140417\u2019 grant number. The titled \u201cMobile Pediatric Consultation and Monitoring System\u201d project was successfully completed by the end of 2015. The project was completed by collaboration of the Ac\u0131badem Hospitals Group, Istanbul Aydin University and Semantica Internet and Software Services Trd. Ltd. Company in Turkey\u201d. ",
+            "date_created": "2020-03-16T16:49:30.789608Z",
+            "date_released": "2020-03-16T16:49:30.789575Z",
+            "description": "Life cycle stages for Mus Musculus",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PCMO/4/pcmo.owl",
+            "download_url": "media/ontologies/MMUSDV/9/mmusdv.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://cmpe.emu.edu.tr/duygu/",
-            "id": 854,
+            "home_page": "https://github.com/obophenotype/developmental-stage-ontologies/wiki/MmusDv",
+            "id": 3441,
             "indexed": true,
-            "max_children": 10,
+            "max_children": 108,
             "max_depth": 2,
+            "md5sum": "5bef72395d853c7f65450e6c2a1fc653",
+            "nb_classes": 113,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 3,
+            "products": [
+                {
+                    "id": "mmusdv.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mmusdv.owl"
+                },
+                {
+                    "id": "mmusdv.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mmusdv.obo"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 9,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PHYLO",
+        "name": "Phylontology_v1.0",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2016-06-06T17:44:08Z",
+            "date_released": "2016-06-01T00:00:00Z",
+            "description": "This phylogenetic ontology represent phylogenetic analyses tasks.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PHYLO/4/phylo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 35,
+            "indexed": true,
+            "max_children": 13,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 20,
+            "nb_classes": 115,
             "nb_inconsistent": 0,
-            "nb_individuals": 201,
-            "nb_properties": 27,
+            "nb_individuals": 709,
+            "nb_properties": 14,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 4,
             "taxon": null,
-            "version": "v.4.0"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "PCPCKB",
-        "name": "PC Parts Compatibility Knowledge Base",
-        "species": [
-            "Ontology"
-        ],
+        "acronym": "PHYLONT",
+        "name": "Phylogenetic Ontology",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-05-05T12:04:34.485363Z",
-            "date_released": "2023-05-05T12:04:34.485394Z",
-            "description": "Still",
-            "documentation": "trust me",
+            "date_created": "2013-03-11T20:07:49Z",
+            "date_released": "2012-10-02T07:00:00Z",
+            "description": "Ontology for Phylogenetic Analysis",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PCPCKB/2/pcpckb.owl",
+            "download_url": "media/ontologies/PHYLONT/7/phylont.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com",
-            "id": 5793,
+            "home_page": null,
+            "id": 705,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 3,
+            "max_children": 33,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 23,
+            "nb_classes": 148,
             "nb_inconsistent": 0,
-            "nb_individuals": 6,
-            "nb_properties": 5,
+            "nb_individuals": 2,
+            "nb_properties": 38,
             "products": null,
-            "publication": "https://github.com/Grbson00/OWLOntologyRepo/tree/main/TestOntology",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 7,
             "taxon": null,
-            "version": "1.1"
+            "version": "ver8"
         },
-        "topics": [
-            "PC hardware compatibility"
-        ]
+        "topics": null
     },
     {
-        "acronym": "LEPAO",
-        "name": "Lepidoptera Anatomy Ontology",
+        "acronym": "TRANS",
+        "name": "Pathogen Transmission Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-02-20T12:24:46.472930Z",
-            "date_released": "2023-02-20T12:24:46.472915Z",
-            "description": "The Lepidoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of moths and butterflies in biodiversity research. LEPAO is developed in part by BIOfid (The Specialised Information Service Biodiversity Research).",
+            "date_created": "2022-10-24T15:49:54.706990Z",
+            "date_released": "2022-10-24T15:49:54.706978Z",
+            "description": "An ontology representing the disease transmission process during which the pathogen is transmitted directly or indirectly from its natural reservoir, a susceptible host or source to a new host.",
             "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/LEPAO/2/lepao.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/TRANS/5/trans.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/insect-morphology/lepao",
-            "id": 5667,
+            "home_page": "https://github.com/DiseaseOntology/PathogenTransmissionOntology",
+            "id": 5434,
             "indexed": true,
-            "max_children": 71,
-            "max_depth": 8,
-            "md5sum": "2ebe95b84326be82c7b18cfdae8a2cec",
-            "nb_classes": 822,
+            "max_children": 7,
+            "max_depth": 7,
+            "md5sum": "621670dbf7a6d87091e4ecd0a3dabaf6",
+            "nb_classes": 35,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 57,
+            "nb_properties": 0,
             "products": [
                 {
-                    "id": "lepao.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/lepao.owl"
+                    "id": "trans.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/trans.owl"
                 },
                 {
-                    "id": "lepao.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/lepao.obo"
+                    "id": "trans.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/trans.obo"
                 }
             ],
             "publication": null,
-            "publications": null,
-            "submission_id": 2,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/19850722",
+                    "title": "GeMInA, Genomic Metadata for Infectious Agents, a geospatial surveillance pathogen database"
+                },
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34755882",
+                    "title": "The Human Disease Ontology 2022 update"
+                }
+            ],
+            "submission_id": 5,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "LICO",
-        "name": "Liver Case Ontology (LiCO)",
+        "acronym": "MIXS",
+        "name": "Minimal Information about any Sequence Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2020-11-04T16:59:52Z",
-            "date_released": "2018-08-31T00:00:00Z",
-            "description": "Liver Case Ontology (LiCO) is a case-centric ontology for liver patient cases called lico (Liver Case Ontology) that integrates several ontologies and lexicons developed for sub-disciplines of medicine such as RadLex (Radiology Lexicon), SNOMED CT (Systematized Nomenclature of Medicine, Clinical Terms), LOINC (The Logical Observation Identifier Names and Codes). It is an extension of ONLIRA (Ontology of the Liver for Radiology) that represents the liver and lesions from a radiological perspective.",
+            "date_created": "2013-04-11T01:13:55Z",
+            "date_released": "2013-04-10T07:00:00Z",
+            "description": "An RDF representation of the MIxS Family of Checklists",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/LICO/2/lico.owl",
+            "download_url": "media/ontologies/MIXS/5/mixs.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4045,
+            "id": 574,
             "indexed": true,
-            "max_children": 24,
-            "max_depth": 4,
+            "max_children": 7,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 93,
+            "nb_classes": 7,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 36,
+            "nb_properties": 59,
             "products": null,
-            "publication": "https://vavlab.boun.edu.tr/publications-patents?field_tags_tid%5B%5D=68&field_tags_tid%5B%5D=70",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 5,
             "taxon": null,
-            "version": "changes\n* Segments and Regions are modelling as Classes in order to identify which patient (liver) they belong to.\n* Redundant restrictions are removed\n"
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "MEPO",
-        "name": "Mapping of Epilepsy Ontologies",
+        "acronym": "MDIM",
+        "name": "Mapping of Drug Names, ICD-11 and MeSH 2021",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5686,
+            "avg_children": 110124,
             "classifiable": true,
-            "date_created": "2021-02-19T09:04:05Z",
+            "date_created": "2021-02-23T11:14:19Z",
             "date_released": "2021-01-15T00:00:00Z",
-            "description": "The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology",
+            "description": "The Mapping of Drug Names, ICD-11 and MeSH 2021 (MDIM) provides owl:sameAs relationships of similar concepts for Drug Names from the DrugBank vocabulary (https://go.drugbank.com/releases/latest#open-data), the International Classification of Diseases Version 11 (https://icd.who.int/) and the Medical Subject Headings (http://id.nlm.nih.gov/mesh/). Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MEPO/6/mepo.owl",
+            "download_url": "media/ontologies/MDIM/2/mdim.owl",
             "has_ontology_language": "OWL",
             "home_page": "https://www.zbmed.de/",
-            "id": 4292,
+            "id": 4297,
             "indexed": true,
-            "max_children": 5686,
+            "max_children": 110124,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 5687,
+            "nb_classes": 110124,
             "nb_inconsistent": 0,
-            "nb_individuals": 747,
+            "nb_individuals": 20932,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
-            "taxon": null,
-            "version": "0.6"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MERA",
-        "name": "Medical Educational Resource Aggregator",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2015-04-26T06:13:32Z",
-            "date_released": "2015-04-26T00:00:00Z",
-            "description": "The educational resource ontology is designed in order to provide structure to resources aggregated by the educational resource aggregator.\r\nThe aim of the educational resource aggregator is to harvest educational resources from 3rd party repositories, present these to the medical expert for annotation and rating, and output the results of the annotation (together with resource metadata) to the CARRE public RDF repository.\r\nThe main parts of this aggregator are: the Resource Retriever, the Resource Rating, the Resource Metadata Processing, and the User Application.\r\nThe educational metadata involve the Resource Metadata processing module.This module is a combination of 3 services that collect data per article, making multiple SPARQL requests to enrich the data and finally store it as a unique identified resource into the local datastore. Then data is transformed into RDF triples in order to be inserted to CARRE educational repository.\r\nThe ontology is developed using the Prot\u00e9g\u00e9 v5.0 , a free, open-source ontology editor from the Stanford University. The ontology is available on CARRE rdf repository among the other ontologies. CARRE url : http://carre.kmi.open.ac.uk/ontology/educational.owl",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MERA/1/mera.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://edu.carre-project.eu",
-            "id": 498,
-            "indexed": true,
-            "max_children": 5,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 5,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 32,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.0.1"
+            "version": "0.2"
         },
         "topics": null
     },
     {
-        "acronym": "MHC",
-        "name": "Major Histocompatibility Complex Ontology",
+        "acronym": "MDRFRE",
+        "name": "MedDRA French",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 10,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2009-04-15T18:19:00Z",
-            "date_released": "2009-04-15T07:00:00Z",
-            "description": "The MaHCO contains terms necessary for describing and categorizing concepts related to MHC, in general, and for a number of model species, and also for humans. ",
-            "documentation": "http://purl.org/stemnet/",
+            "date_created": "2017-02-06T21:26:50Z",
+            "date_released": "2017-02-06T21:26:50Z",
+            "description": "Medical Dictionary for Regulatory Activities Terminology (MedDRA), French Edition",
+            "documentation": "http://www.meddramsso.com",
             "domain": null,
-            "download_url": "media/ontologies/MHC/4/mhc.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://purl.org/stemnet/",
-            "id": 724,
+            "download_url": "media/ontologies/MDRFRE/9/mdrfre.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.meddramsso.com",
+            "id": 753,
             "indexed": true,
-            "max_children": 183,
-            "max_depth": 12,
+            "max_children": 250,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 7929,
+            "nb_classes": 69107,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 11,
+            "nb_properties": 14,
             "products": null,
-            "publication": "http://purl.org/stemnet/",
+            "publication": "http://www.meddramsso.com",
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 9,
             "taxon": null,
-            "version": "1.0.1"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
         "acronym": "MHCRO",
         "name": "MHC Restriction Ontology",
         "species": null,
@@ -27977,404 +28140,185 @@
                 "label": "Anopheles"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MIXS",
-        "name": "Minimal Information about any Sequence Ontology",
+        "acronym": "MDRGER",
+        "name": "MedDRA German",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 41,
             "classifiable": true,
-            "date_created": "2013-04-11T01:13:55Z",
-            "date_released": "2013-04-10T07:00:00Z",
-            "description": "An RDF representation of the MIxS Family of Checklists",
+            "date_created": "2023-01-07T00:18:04Z",
+            "date_released": "2022-11-07T00:00:00Z",
+            "description": "This is the German translation of the Medical Dictionary for Regulatory Activities (MedDRA). ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MIXS/5/mixs.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 574,
+            "download_url": "media/ontologies/MDRGER/5/mdrger.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "https://www.nlm.nih.gov/research/umls/sourcereleasedocs/current/MDR/index.html",
+            "id": 5603,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 0,
+            "max_children": 50022,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 7,
+            "nb_classes": 77301,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 59,
+            "nb_individuals": 77295,
+            "nb_properties": 22,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 5,
             "taxon": null,
-            "version": "unknown"
+            "version": "2022AB"
         },
         "topics": null
     },
     {
-        "acronym": "MIXSCV",
-        "name": "Minimal Information about any Sequence (MIxS) Controlled Vocabularies",
+        "acronym": "MECHANOH",
+        "name": "NeuroMorpho.Org mechanoreception system",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 19,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2012-01-31T20:17:06Z",
-            "date_released": "2012-01-26T08:00:00Z",
-            "description": "Controlled vocabularies for the MIxS family of metadata checklists. See http://gensc.org/gc_wiki/index.php/MIxS for details on the MIxS checklists.",
+            "date_created": "2016-09-17T01:05:53Z",
+            "date_released": "2016-09-17T01:05:53Z",
+            "description": "A hierarchy of regions involved in mechanoreception system.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MIXSCV/2/mixscv.obo",
-            "has_ontology_language": "OBO",
+            "download_url": "media/ontologies/MECHANOH/2/mechanoh.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 239,
+            "id": 391,
             "indexed": true,
-            "max_children": 267,
-            "max_depth": 2,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 519,
+            "nb_classes": 0,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1"
+            "version": "0.6"
         },
         "topics": null
     },
     {
-        "acronym": "MIZZOU_CCO",
-        "name": "Care Coordination Ontology",
+        "acronym": "MESH",
+        "name": "Medical Subject Headings",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": null,
             "classifiable": true,
-            "date_created": "2013-11-05T22:49:24Z",
-            "date_released": "2013-11-30T00:00:00Z",
-            "description": "Care Coordination ontology",
-            "documentation": null,
+            "date_created": "2017-02-06T21:25:55Z",
+            "date_released": "2017-02-06T21:25:54Z",
+            "description": "Medical Subject Headings (MeSH);National Library of Medicine; 2011",
+            "documentation": "http://www.nlm.nih.gov/mesh/meshhome.html",
             "domain": null,
-            "download_url": "media/ontologies/MIZZOU_CCO/8/mizzou_cco.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 721,
+            "download_url": "media/ontologies/MESH/13/mesh.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.nlm.nih.gov/mesh/meshhome.html",
+            "id": 316,
             "indexed": true,
-            "max_children": 27,
-            "max_depth": 6,
+            "max_children": null,
+            "max_depth": 15,
             "md5sum": null,
-            "nb_classes": 400,
+            "nb_classes": 265414,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 8,
-            "taxon": null,
-            "version": "1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MLTX",
-        "name": "Machine Learning Methodology Taxonomy Mapping",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 12,
-            "classifiable": true,
-            "date_created": "2021-09-10T18:25:44Z",
-            "date_released": "2021-09-10T00:00:00Z",
-            "description": "The machine learning methodology taxonomy mapping project is focused on gathering the terminology focused on machine learning and mapping the controlled and users' natural language to form a mapping file across these terminologies. This work was originally associated with dissertation research.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MLTX/1/mltx.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4607,
-            "indexed": true,
-            "max_children": 12,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 13,
-            "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 3,
+            "nb_properties": null,
             "products": null,
-            "publication": null,
+            "publication": "http://www.nlm.nih.gov/mesh/meshhome.html",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 13,
             "taxon": null,
-            "version": "1"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "MCO",
-        "name": "Microbial Conditions Ontology",
+        "acronym": "MF",
+        "name": "Mental Functioning Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-05-20T13:24:27.583312Z",
-            "date_released": "2019-05-20T13:24:27.583288Z",
-            "description": "Microbial Conditions Ontology is an ontology...",
+            "date_created": "2022-07-25T12:15:05.140675Z",
+            "date_released": "2022-07-25T12:15:05.140663Z",
+            "description": "The Mental Functioning Ontology is an overarching ontology for all aspects of mental functioning.",
             "documentation": null,
-            "domain": "experimental conditions",
-            "download_url": "media/ontologies/MCO/2/mco.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/MF/28/mf.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/microbial-conditions-ontology/microbial-conditions-ontology",
-            "id": 2559,
+            "home_page": "https://github.com/jannahastings/mental-functioning-ontology",
+            "id": 5245,
             "indexed": true,
-            "max_children": 179,
-            "max_depth": 21,
-            "md5sum": "489fbeba542c9f4b2717f5a8a08893d4",
-            "nb_classes": 3384,
+            "max_children": 27,
+            "max_depth": 13,
+            "md5sum": "55065d410cc4e8553f02eb7e648c58e5",
+            "nb_classes": 401,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 19,
             "nb_properties": 16,
             "products": [
                 {
-                    "id": "mco.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mco.owl"
-                },
-                {
-                    "id": "mco.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mco.obo"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MMO",
-        "name": "Measurement method ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2020-02-10T17:58:03.540659Z",
-            "date_released": "2020-02-10T17:58:03.540637Z",
-            "description": "A representation of the variety of methods used to make clinical and phenotype measurements. ",
-            "documentation": null,
-            "domain": "clinical",
-            "download_url": "media/ontologies/MMO/64/mmo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MMO:0000000",
-            "id": 3319,
-            "indexed": true,
-            "max_children": 35,
-            "max_depth": 9,
-            "md5sum": "1c78a37b0eafb55642a2167d76f46a20",
-            "nb_classes": 701,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
-            "products": [
-                {
-                    "id": "mmo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mmo.owl"
-                },
-                {
-                    "id": "mmo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mmo.obo"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/22654893",
-                    "title": "Three ontologies to define phenotype measurement data."
-                },
-                {
-                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/24103152",
-                    "title": "The clinical measurement, measurement method and experimental condition ontologies: expansion, improvements and new applications."
-                }
-            ],
-            "submission_id": 64,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MMUSDV",
-        "name": "Mouse Developmental Stages",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 56,
-            "classifiable": true,
-            "date_created": "2020-03-16T16:49:30.789608Z",
-            "date_released": "2020-03-16T16:49:30.789575Z",
-            "description": "Life cycle stages for Mus Musculus",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MMUSDV/9/mmusdv.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/developmental-stage-ontologies/wiki/MmusDv",
-            "id": 3441,
-            "indexed": true,
-            "max_children": 108,
-            "max_depth": 2,
-            "md5sum": "5bef72395d853c7f65450e6c2a1fc653",
-            "nb_classes": 113,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 3,
-            "products": [
-                {
-                    "id": "mmusdv.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mmusdv.owl"
-                },
-                {
-                    "id": "mmusdv.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mmusdv.obo"
+                    "id": "mf.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mf.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 28,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PHYLO",
-        "name": "Phylontology_v1.0",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2016-06-06T17:44:08Z",
-            "date_released": "2016-06-01T00:00:00Z",
-            "description": "This phylogenetic ontology represent phylogenetic analyses tasks.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PHYLO/4/phylo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 35,
-            "indexed": true,
-            "max_children": 13,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 115,
-            "nb_inconsistent": 0,
-            "nb_individuals": 709,
-            "nb_properties": 14,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PHYLONT",
-        "name": "Phylogenetic Ontology",
+        "acronym": "MIXSCV",
+        "name": "Minimal Information about any Sequence (MIxS) Controlled Vocabularies",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 19,
             "classifiable": true,
-            "date_created": "2013-03-11T20:07:49Z",
-            "date_released": "2012-10-02T07:00:00Z",
-            "description": "Ontology for Phylogenetic Analysis",
+            "date_created": "2012-01-31T20:17:06Z",
+            "date_released": "2012-01-26T08:00:00Z",
+            "description": "Controlled vocabularies for the MIxS family of metadata checklists. See http://gensc.org/gc_wiki/index.php/MIxS for details on the MIxS checklists.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PHYLONT/7/phylont.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/MIXSCV/2/mixscv.obo",
+            "has_ontology_language": "OBO",
             "home_page": null,
-            "id": 705,
+            "id": 239,
             "indexed": true,
-            "max_children": 33,
-            "max_depth": 5,
+            "max_children": 267,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 148,
-            "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 38,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 7,
-            "taxon": null,
-            "version": "ver8"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TRANS",
-        "name": "Pathogen Transmission Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2022-10-24T15:49:54.706990Z",
-            "date_released": "2022-10-24T15:49:54.706978Z",
-            "description": "An ontology representing the disease transmission process during which the pathogen is transmitted directly or indirectly from its natural reservoir, a susceptible host or source to a new host.",
-            "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/TRANS/5/trans.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/DiseaseOntology/PathogenTransmissionOntology",
-            "id": 5434,
-            "indexed": true,
-            "max_children": 7,
-            "max_depth": 7,
-            "md5sum": "621670dbf7a6d87091e4ecd0a3dabaf6",
-            "nb_classes": 35,
+            "nb_classes": 519,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
-            "products": [
-                {
-                    "id": "trans.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/trans.owl"
-                },
-                {
-                    "id": "trans.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/trans.obo"
-                }
-            ],
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/19850722",
-                    "title": "GeMInA, Genomic Metadata for Infectious Agents, a geospatial surveillance pathogen database"
-                },
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34755882",
-                    "title": "The Human Disease Ontology 2022 update"
-                }
-            ],
-            "submission_id": 5,
+            "publications": null,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "1"
         },
         "topics": null
     },
     {
         "acronym": "TRON",
         "name": "Tribolium Ontology",
         "species": null,
@@ -28439,14 +28383,48 @@
             "submission_id": 9,
             "taxon": null,
             "version": "2016AB"
         },
         "topics": null
     },
     {
+        "acronym": "MIZZOU_CCO",
+        "name": "Care Coordination Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2013-11-05T22:49:24Z",
+            "date_released": "2013-11-30T00:00:00Z",
+            "description": "Care Coordination ontology",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/MIZZOU_CCO/8/mizzou_cco.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 721,
+            "indexed": true,
+            "max_children": 27,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 400,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 8,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ICPCFRE",
         "name": "ICPC, French Translation",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 0,
             "classifiable": true,
@@ -28473,14 +28451,100 @@
             "submission_id": 9,
             "taxon": null,
             "version": "2016AB"
         },
         "topics": null
     },
     {
+        "acronym": "MLTX",
+        "name": "Machine Learning Methodology Taxonomy Mapping",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 12,
+            "classifiable": true,
+            "date_created": "2021-09-10T18:25:44Z",
+            "date_released": "2021-09-10T00:00:00Z",
+            "description": "The machine learning methodology taxonomy mapping project is focused on gathering the terminology focused on machine learning and mapping the controlled and users' natural language to form a mapping file across these terminologies. This work was originally associated with dissertation research.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/MLTX/1/mltx.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4607,
+            "indexed": true,
+            "max_children": 12,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 13,
+            "nb_inconsistent": 0,
+            "nb_individuals": 3,
+            "nb_properties": 3,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "MMO",
+        "name": "Measurement method ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2020-02-10T17:58:03.540659Z",
+            "date_released": "2020-02-10T17:58:03.540637Z",
+            "description": "A representation of the variety of methods used to make clinical and phenotype measurements. ",
+            "documentation": null,
+            "domain": "clinical",
+            "download_url": "media/ontologies/MMO/64/mmo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=MMO:0000000",
+            "id": 3319,
+            "indexed": true,
+            "max_children": 35,
+            "max_depth": 9,
+            "md5sum": "1c78a37b0eafb55642a2167d76f46a20",
+            "nb_classes": 701,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
+            "products": [
+                {
+                    "id": "mmo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mmo.owl"
+                },
+                {
+                    "id": "mmo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mmo.obo"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/22654893",
+                    "title": "Three ontologies to define phenotype measurement data."
+                },
+                {
+                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/24103152",
+                    "title": "The clinical measurement, measurement method and experimental condition ontologies: expansion, improvements and new applications."
+                }
+            ],
+            "submission_id": 64,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "MFMO",
         "name": "Mammalian Feeding Muscle Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -28588,149 +28652,85 @@
             "submission_id": 39,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MEDRT",
-        "name": "MED-RT",
+        "acronym": "MHC",
+        "name": "Major Histocompatibility Complex Ontology",
         "species": null,
         "status": "Unloadable",
         "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2019-05-01T08:30:07Z",
-            "date_released": "2018-11-05T00:00:00Z",
-            "description": "MED-RT is a replacement and successor to NDF-RT.  Both terminologies are formal ontological representations of medication terminology, pharmacologic classifications, and asserted authoritative relationships between them.\r\n\r\nDescription\r\nMED-RT includes native pharmacologic classification concepts (e.g. mechanisms of action (MoA), physiologic effects (PE), Established Pharmacologic Class (EPC)) and all relationships asserted between concepts in any namespace.\r\n\r\nMED-RT incorporates the following terminologies:\r\n\r\nRxNorm for prescribable medications, active ingredients, and relationships between them\r\nMeSH for indexing chemical structure and therapeutics using its concept hierarchies\r\nSNOMED CT International to aid in mapping between FDA pharmacologic classes and its product and substance hierarchies\r\nMED-RT will maintain the pharmacologic classification relationships from NDF-RT.\r\n\r\nIt also maintains the National Committee on Vital and Health Statistics (NCVHS) standards for describing medication pharmacologic classification: MoA and PE hierarchy concepts. The concepts are components of the Federal Medication Terminologies (FMT) interagency collaboration.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MEDRT/1/medrt.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": null,
-            "id": 2527,
-            "indexed": true,
-            "max_children": 9,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 132,
-            "nb_inconsistent": 0,
-            "nb_individuals": 127,
-            "nb_properties": 18,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "2018AB"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MESH",
-        "name": "Medical Subject Headings",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": null,
+            "avg_children": 10,
             "classifiable": true,
-            "date_created": "2017-02-06T21:25:55Z",
-            "date_released": "2017-02-06T21:25:54Z",
-            "description": "Medical Subject Headings (MeSH);National Library of Medicine; 2011",
-            "documentation": "http://www.nlm.nih.gov/mesh/meshhome.html",
-            "domain": null,
-            "download_url": "media/ontologies/MESH/13/mesh.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.nlm.nih.gov/mesh/meshhome.html",
-            "id": 316,
-            "indexed": true,
-            "max_children": null,
-            "max_depth": 15,
-            "md5sum": null,
-            "nb_classes": 265414,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": null,
-            "products": null,
-            "publication": "http://www.nlm.nih.gov/mesh/meshhome.html",
-            "publications": null,
-            "submission_id": 13,
-            "taxon": null,
-            "version": "2016AB"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MEO",
-        "name": "Metagenome and Microbes Environmental Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 0,
-            "classifiable": false,
-            "date_created": "2017-09-10T04:42:12Z",
-            "date_released": "2017-09-08T00:00:00Z",
-            "description": "An Ontology for organismal habitats (especially focused on microbes)",
-            "documentation": null,
+            "date_created": "2009-04-15T18:19:00Z",
+            "date_released": "2009-04-15T07:00:00Z",
+            "description": "The MaHCO contains terms necessary for describing and categorizing concepts related to MHC, in general, and for a number of model species, and also for humans. ",
+            "documentation": "http://purl.org/stemnet/",
             "domain": null,
-            "download_url": "media/ontologies/MEO/9/meo.owl",
+            "download_url": "media/ontologies/MHC/4/mhc.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://microbedb.jp",
-            "id": 169,
-            "indexed": false,
-            "max_children": 0,
-            "max_depth": 0,
+            "home_page": "http://purl.org/stemnet/",
+            "id": 724,
+            "indexed": true,
+            "max_children": 183,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 0,
+            "nb_classes": 7929,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 11,
             "products": null,
-            "publication": null,
+            "publication": "http://purl.org/stemnet/",
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 4,
             "taxon": null,
-            "version": "0.9"
+            "version": "1.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "MF",
-        "name": "Mental Functioning Ontology",
+        "acronym": "MCO",
+        "name": "Microbial Conditions Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-07-25T12:15:05.140675Z",
-            "date_released": "2022-07-25T12:15:05.140663Z",
-            "description": "The Mental Functioning Ontology is an overarching ontology for all aspects of mental functioning.",
+            "date_created": "2019-05-20T13:24:27.583312Z",
+            "date_released": "2019-05-20T13:24:27.583288Z",
+            "description": "Microbial Conditions Ontology is an ontology...",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/MF/28/mf.owl",
+            "domain": "experimental conditions",
+            "download_url": "media/ontologies/MCO/2/mco.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/jannahastings/mental-functioning-ontology",
-            "id": 5245,
+            "home_page": "https://github.com/microbial-conditions-ontology/microbial-conditions-ontology",
+            "id": 2559,
             "indexed": true,
-            "max_children": 27,
-            "max_depth": 13,
-            "md5sum": "55065d410cc4e8553f02eb7e648c58e5",
-            "nb_classes": 401,
+            "max_children": 179,
+            "max_depth": 21,
+            "md5sum": "489fbeba542c9f4b2717f5a8a08893d4",
+            "nb_classes": 3384,
             "nb_inconsistent": 0,
-            "nb_individuals": 19,
+            "nb_individuals": 0,
             "nb_properties": 16,
             "products": [
                 {
-                    "id": "mf.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mf.owl"
+                    "id": "mco.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mco.owl"
+                },
+                {
+                    "id": "mco.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mco.obo"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 28,
+            "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "PHMAMMADO",
@@ -28763,48 +28763,14 @@
             "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MDIM",
-        "name": "Mapping of Drug Names, ICD-11 and MeSH 2021",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 110124,
-            "classifiable": true,
-            "date_created": "2021-02-23T11:14:19Z",
-            "date_released": "2021-01-15T00:00:00Z",
-            "description": "The Mapping of Drug Names, ICD-11 and MeSH 2021 (MDIM) provides owl:sameAs relationships of similar concepts for Drug Names from the DrugBank vocabulary (https://go.drugbank.com/releases/latest#open-data), the International Classification of Diseases Version 11 (https://icd.who.int/) and the Medical Subject Headings (http://id.nlm.nih.gov/mesh/). Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MDIM/2/mdim.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.zbmed.de/",
-            "id": 4297,
-            "indexed": true,
-            "max_children": 110124,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 110124,
-            "nb_inconsistent": 0,
-            "nb_individuals": 20932,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "0.2"
-        },
-        "topics": null
-    },
-    {
         "acronym": "MDM",
         "name": "Mapping of Drug Names and MeSH 2022",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 44789,
             "classifiable": true,
@@ -28831,296 +28797,146 @@
             "submission_id": 4,
             "taxon": null,
             "version": "0.33"
         },
         "topics": null
     },
     {
-        "acronym": "MDRFRE",
-        "name": "MedDRA French",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 15,
-            "classifiable": true,
-            "date_created": "2017-02-06T21:26:50Z",
-            "date_released": "2017-02-06T21:26:50Z",
-            "description": "Medical Dictionary for Regulatory Activities Terminology (MedDRA), French Edition",
-            "documentation": "http://www.meddramsso.com",
-            "domain": null,
-            "download_url": "media/ontologies/MDRFRE/9/mdrfre.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.meddramsso.com",
-            "id": 753,
-            "indexed": true,
-            "max_children": 250,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 69107,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 14,
-            "products": null,
-            "publication": "http://www.meddramsso.com",
-            "publications": null,
-            "submission_id": 9,
-            "taxon": null,
-            "version": "2016AB"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MDRGER",
-        "name": "MedDRA German",
+        "acronym": "MEDRT",
+        "name": "MED-RT",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 41,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-01-07T00:18:04Z",
-            "date_released": "2022-11-07T00:00:00Z",
-            "description": "This is the German translation of the Medical Dictionary for Regulatory Activities (MedDRA). ",
+            "date_created": "2019-05-01T08:30:07Z",
+            "date_released": "2018-11-05T00:00:00Z",
+            "description": "MED-RT is a replacement and successor to NDF-RT.  Both terminologies are formal ontological representations of medication terminology, pharmacologic classifications, and asserted authoritative relationships between them.\r\n\r\nDescription\r\nMED-RT includes native pharmacologic classification concepts (e.g. mechanisms of action (MoA), physiologic effects (PE), Established Pharmacologic Class (EPC)) and all relationships asserted between concepts in any namespace.\r\n\r\nMED-RT incorporates the following terminologies:\r\n\r\nRxNorm for prescribable medications, active ingredients, and relationships between them\r\nMeSH for indexing chemical structure and therapeutics using its concept hierarchies\r\nSNOMED CT International to aid in mapping between FDA pharmacologic classes and its product and substance hierarchies\r\nMED-RT will maintain the pharmacologic classification relationships from NDF-RT.\r\n\r\nIt also maintains the National Committee on Vital and Health Statistics (NCVHS) standards for describing medication pharmacologic classification: MoA and PE hierarchy concepts. The concepts are components of the Federal Medication Terminologies (FMT) interagency collaboration.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MDRGER/5/mdrger.umls",
+            "download_url": "media/ontologies/MEDRT/1/medrt.umls",
             "has_ontology_language": "UMLS",
-            "home_page": "https://www.nlm.nih.gov/research/umls/sourcereleasedocs/current/MDR/index.html",
-            "id": 5603,
+            "home_page": null,
+            "id": 2527,
             "indexed": true,
-            "max_children": 50022,
+            "max_children": 9,
             "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 77301,
+            "nb_classes": 132,
             "nb_inconsistent": 0,
-            "nb_individuals": 77295,
-            "nb_properties": 22,
+            "nb_individuals": 127,
+            "nb_properties": 18,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2022AB"
+            "version": "2018AB"
         },
         "topics": null
     },
     {
-        "acronym": "MECHANOH",
-        "name": "NeuroMorpho.Org mechanoreception system",
+        "acronym": "MEPO",
+        "name": "Mapping of Epilepsy Ontologies",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 5686,
             "classifiable": true,
-            "date_created": "2016-09-17T01:05:53Z",
-            "date_released": "2016-09-17T01:05:53Z",
-            "description": "A hierarchy of regions involved in mechanoreception system.",
+            "date_created": "2021-02-19T09:04:05Z",
+            "date_released": "2021-01-15T00:00:00Z",
+            "description": "The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MECHANOH/2/mechanoh.owl",
+            "download_url": "media/ontologies/MEPO/6/mepo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 391,
+            "home_page": "https://www.zbmed.de/",
+            "id": 4292,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 5686,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 0,
+            "nb_classes": 5687,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 747,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 6,
             "taxon": null,
             "version": "0.6"
         },
         "topics": null
     },
     {
-        "acronym": "PLANA",
-        "name": "planaria-ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2023-03-27T12:59:31.758713Z",
-            "date_released": "2023-03-27T12:59:31.758703Z",
-            "description": "PLANA, the planarian anatomy ontology, encompasses the anatomy and life cycle stages for both __Schmidtea mediterranea__ biotypes.",
-            "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/PLANA/28/plana.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/planaria-ontology",
-            "id": 5739,
-            "indexed": true,
-            "max_children": 206,
-            "max_depth": 11,
-            "md5sum": "1896425cc842597b0e0011a633b34965",
-            "nb_classes": 876,
-            "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 160,
-            "products": [
-                {
-                    "id": "plana.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/plana.owl"
-                },
-                {
-                    "id": "plana.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/plana.obo"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34318308",
-                    "title": "Planarian Anatomy Ontology: a resource to connect data within and across experimental platforms"
-                }
-            ],
-            "submission_id": 28,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SCHEMAORG",
-        "name": "Schema.org core and all extension vocabularies",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2018-10-11T18:50:58Z",
-            "date_released": "2018-06-15T00:00:00Z",
-            "description": "A collection of schemas that webmasters can use to markup HTML pages in ways recognized by major search providers, and that can also be used for structured data interoperability (e.g. in JSON).",
-            "documentation": "https://schema.org/",
-            "domain": null,
-            "download_url": "media/ontologies/SCHEMAORG/2/schemaorg.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://schema.org/",
-            "id": 1939,
-            "indexed": true,
-            "max_children": 59,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 822,
-            "nb_inconsistent": 0,
-            "nb_individuals": 989,
-            "nb_properties": 680,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "3.4"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SciRes",
-        "name": "Scientific Research Resources",
+        "acronym": "MEO",
+        "name": "Metagenome and Microbes Environmental Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2012-08-31T11:01:57Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-public-1.5.owl.  It contains terms related to scientific research resources that are included in the vivo ontology.",
+            "avg_children": 0,
+            "classifiable": false,
+            "date_created": "2017-09-10T04:42:12Z",
+            "date_released": "2017-09-08T00:00:00Z",
+            "description": "An Ontology for organismal habitats (especially focused on microbes)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SciRes/1/scires.owl",
+            "download_url": "media/ontologies/MEO/9/meo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 648,
-            "indexed": true,
-            "max_children": 7,
-            "max_depth": 4,
+            "home_page": "http://microbedb.jp",
+            "id": 169,
+            "indexed": false,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 20,
+            "nb_classes": 0,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 10,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.5"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TST-VODAN",
-        "name": "VODANA Migrants News Vocabulary",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2021-05-17T14:09:02Z",
-            "date_released": "2021-05-02T00:00:00Z",
-            "description": "migrants ontologies ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/TST-VODAN/3/tst-vodan.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4483,
-            "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 3,
-            "nb_inconsistent": 0,
-            "nb_individuals": 164,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 9,
             "taxon": null,
-            "version": "2.0"
+            "version": "0.9"
         },
         "topics": null
     },
     {
-        "acronym": "CNO_ACRONYM",
-        "name": "CNO_NAME",
+        "acronym": "MERA",
+        "name": "Medical Educational Resource Aggregator",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2016-06-15T06:49:29Z",
-            "date_released": "2016-06-15T00:00:00Z",
-            "description": "test",
+            "date_created": "2015-04-26T06:13:32Z",
+            "date_released": "2015-04-26T00:00:00Z",
+            "description": "The educational resource ontology is designed in order to provide structure to resources aggregated by the educational resource aggregator.\r\nThe aim of the educational resource aggregator is to harvest educational resources from 3rd party repositories, present these to the medical expert for annotation and rating, and output the results of the annotation (together with resource metadata) to the CARRE public RDF repository.\r\nThe main parts of this aggregator are: the Resource Retriever, the Resource Rating, the Resource Metadata Processing, and the User Application.\r\nThe educational metadata involve the Resource Metadata processing module.This module is a combination of 3 services that collect data per article, making multiple SPARQL requests to enrich the data and finally store it as a unique identified resource into the local datastore. Then data is transformed into RDF triples in order to be inserted to CARRE educational repository.\r\nThe ontology is developed using the Prot\u00e9g\u00e9 v5.0 , a free, open-source ontology editor from the Stanford University. The ontology is available on CARRE rdf repository among the other ontologies. CARRE url : http://carre.kmi.open.ac.uk/ontology/educational.owl",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CNO_ACRONYM/1/cno_acronym.owl",
+            "download_url": "media/ontologies/MERA/1/mera.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 593,
+            "home_page": "https://edu.carre-project.eu",
+            "id": 498,
             "indexed": true,
-            "max_children": 27,
-            "max_depth": 7,
+            "max_children": 5,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 240,
+            "nb_classes": 5,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 22,
+            "nb_properties": 32,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "version 0.5"
+            "version": "0.0.1"
         },
         "topics": null
     },
     {
         "acronym": "TTO",
         "name": "Teleost taxonomy ontology",
         "species": null,
@@ -29163,310 +28979,38 @@
                 "label": "Teleostei"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TWCP",
-        "name": "Top Women Chess Players",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2021-12-20T19:15:30Z",
-            "date_released": "2021-12-20T00:00:00Z",
-            "description": "..",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/TWCP/1/twcp.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4786,
-            "indexed": true,
-            "max_children": 10,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 49,
-            "nb_inconsistent": 0,
-            "nb_individuals": 3280,
-            "nb_properties": 18,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TYPON",
-        "name": "Microbial Typing Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2014-09-29T21:45:27Z",
-            "date_released": "2014-09-29T21:45:25Z",
-            "description": "TyPon provides a comprehensive description of the existing microbial typing methods for the identification of bacterial Isolates and their classification. Such a description constitutes an universal format for the exchange of information on the microbial typing field, providing a vehicle for the integration of the numerous disparate online databases. In its current version, TyPon describes most used microbial typing methods but it is, and always will be, a work in progress given the constant advances in the microbial typing field.",
-            "documentation": "https://bitbucket.org/phyloviz/typon/wiki",
-            "domain": null,
-            "download_url": "media/ontologies/TYPON/11/typon.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.phyloviz.net/typon/",
-            "id": 223,
-            "indexed": true,
-            "max_children": 7,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 44,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 46,
-            "products": null,
-            "publication": "https://bitbucket.org/phyloviz/typon/wiki",
-            "publications": null,
-            "submission_id": 11,
-            "taxon": null,
-            "version": "20140929"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "CPRO",
-        "name": "Computer-Based Patient Record Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": false,
-            "date_created": "2013-04-25T00:36:12Z",
-            "date_released": "2013-04-24T07:00:00Z",
-            "description": "A uniform core set of data elements (whose formal semantics are captured in OWL) for use in a Computer-Based Patient Record (CPR)",
-            "documentation": "https://www.researchgate.net/publication/235463154_A_Framework_Ontology_for_Computer-Based_Patient_Record_Systems?ev=prf_pub",
-            "domain": null,
-            "download_url": "media/ontologies/CPRO/3/cpro.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://code.google.com/p/cpr-ontology/",
-            "id": 776,
-            "indexed": false,
-            "max_children": 9,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 134,
-            "nb_inconsistent": 0,
-            "nb_individuals": 32,
-            "nb_properties": 49,
-            "products": null,
-            "publication": "https://www.researchgate.net/publication/235463154_A_Framework_Ontology_for_Computer-Based_Patient_Record_Systems?ev=prf_pub",
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "Last modification: July 31th, 2011 Chimezie Ogbuji"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "CPT-mod",
-        "name": "CPT modifiers",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2012-09-18T19:18:58Z",
-            "date_released": "2012-09-18T07:00:00Z",
-            "description": "Modifiers denote that a certain procedure/service has been altered by a particular circumstance, but not changed in its definition, therefore the same code is used and a modifier is added to denote what has been altered.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/CPT-mod/1/cpt-mod.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 693,
-            "indexed": true,
-            "max_children": 4,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 5,
-            "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 18,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "unknown"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "VIVO",
-        "name": "VIVO Ontology for Researcher Discovery",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2012-07-16T15:51:31Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "The VIVO ontology represents researchers in the context of their experience, outputs, interests, accomplishments, and associated institutions.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/VIVO/2/vivo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 423,
-            "indexed": true,
-            "max_children": 32,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 225,
-            "nb_inconsistent": 0,
-            "nb_individuals": 13,
-            "nb_properties": 406,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "version 1.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PMAPP-PMO",
-        "name": "PMO Precision Medicine Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2020-12-17T02:38:23Z",
-            "date_released": "2020-12-16T00:00:00Z",
-            "description": "The Precision Medicine Ontology (PMO) has been developed as a standardized ontology for human precision medicine with consistent, reusable and sustainable descriptions of human disease terms, genomic molecular, phenotype characteristics and related medical vocabulary disease concepts through collaborative efforts of researchers at Institute of Medical Information, Chinese Academy of Medical Sciences. The Precision Medicine Ontology focuses on the development of a high level patients-centric ontology for the academic research, thus, it systematically integrates biomedicine vocabularies from GO, NCBI Gene, DrugBank, HPO, MeSH, SNOMED CT, etc. This work is supported by the National Key Research and Development Program of China (Precision Medicine Knowledge Base, Grant No. 2016YFC0901901).",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PMAPP-PMO/1/pmapp-pmo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.phoc.org.cn/pmo/",
-            "id": 4141,
-            "indexed": true,
-            "max_children": 469,
-            "max_depth": 14,
-            "md5sum": null,
-            "nb_classes": 76155,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 95,
-            "products": null,
-            "publication": "https://www.aimspress.com/article/10.3934/mbe.2020227/fulltext.html",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "4.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PMD",
-        "name": "Persian Medicine Diseases Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 10,
-            "classifiable": true,
-            "date_created": "2019-06-24T22:00:53Z",
-            "date_released": "2019-06-24T00:00:00Z",
-            "description": "PMD is an ontology that represents classification and related information of Persian medicine diseases. Persian medicine also known as Iranian traditional medicine is one of famous and most ancient forms of traditional medicines. This ontology is based on the top-level ontology GFO (General Formal Ontology)",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PMD/1/pmd.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2702,
-            "indexed": true,
-            "max_children": 62,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 529,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 41,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "RDFS",
-        "name": "The RDF Schema vocabulary",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2020-05-07T08:11:49Z",
-            "date_released": "2014-02-25T00:00:00Z",
-            "description": "RDF Schema provides a data-modelling vocabulary for RDF data",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/RDFS/1/rdfs.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3610,
-            "indexed": true,
-            "max_children": 3,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 7,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 5,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.1"
-        },
-        "topics": null
-    },
-    {
         "acronym": "UBERON",
         "name": "Uberon multi-species anatomy ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-07-03T15:37:39.476830Z",
-            "date_released": "2023-07-03T15:37:39.476814Z",
+            "date_created": "2023-07-31T15:42:09.408898Z",
+            "date_released": "2023-07-31T15:42:09.408886Z",
             "description": "An integrated cross-species anatomy ontology covering animals and bridging multiple species-specific ontologies",
             "documentation": null,
             "domain": "anatomy and development",
-            "download_url": "media/ontologies/UBERON/293/uberon.owl",
+            "download_url": "media/ontologies/UBERON/294/uberon.owl",
             "has_ontology_language": "OWL",
             "home_page": "http://uberon.org",
-            "id": 5911,
+            "id": 5975,
             "indexed": true,
-            "max_children": 1418,
-            "max_depth": 35,
-            "md5sum": "d57bf833b535a0e9b0fbdd422dde5703",
-            "nb_classes": 26209,
+            "max_children": 1419,
+            "max_depth": 39,
+            "md5sum": "e836b2434673bc055e1353b7e22965d7",
+            "nb_classes": 26233,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 319,
+            "nb_individuals": 9,
+            "nb_properties": 321,
             "products": [
                 {
                     "description": "core ontology",
                     "id": "uberon.owl",
                     "is_canonical": true,
                     "ontology_purl": "http://purl.obolibrary.org/obo/uberon.owl",
                     "title": "Uberon",
@@ -29562,169 +29106,344 @@
                     "title": "Uberon, an integrative multi-species anatomy ontology"
                 },
                 {
                     "id": "https://www.ncbi.nlm.nih.gov/pubmed/25009735",
                     "title": "Unification of multi-species vertebrate anatomy ontologies for comparative biology in Uberon"
                 }
             ],
-            "submission_id": 293,
+            "submission_id": 294,
             "taxon": {
                 "id": "NCBITaxon:33208",
                 "label": "Metazoa"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "UGANDA_DISEASES",
-        "name": "Uganda_diseases",
+        "acronym": "RXNORM",
+        "name": "RxNORM",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2021-11-23T22:54:43Z",
-            "date_released": "2021-03-29T00:00:00Z",
-            "description": "This is the seventh version of the Uganda diseases",
+            "date_created": "2017-02-06T21:26:23Z",
+            "date_released": "2017-02-06T21:26:23Z",
+            "description": "RxNorm Vocabulary",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/UGANDA_DISEASES/17/uganda_diseases.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/RXNORM/12/rxnorm.umls",
+            "has_ontology_language": "UMLS",
             "home_page": null,
-            "id": 4728,
+            "id": 353,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 115514,
             "nb_inconsistent": 0,
-            "nb_individuals": 308,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 44,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 17,
+            "submission_id": 12,
             "taxon": null,
-            "version": "7"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "CIDOC-CRM",
-        "name": "CIDOC Conceptual Reference Model",
+        "acronym": "TWCP",
+        "name": "Top Women Chess Players",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-11-22T19:02:04Z",
-            "date_released": "2015-08-19T00:00:00Z",
-            "description": "The CIDOC Conceptual Reference Model (CRM) provides definitions and a formal structure for describing the implicit and explicit concepts and relationships used in cultural heritage documentation.\r\n\r\nThe CIDOC CRM is intended to promote a shared understanding of cultural heritage information by providing a common and extensible semantic framework that any cultural heritage information can be mapped to. It is intended to be a common language for domain experts and implementers to formulate requirements for information systems and to serve as a guide for good practice of conceptual modelling. In this way, it can provide the \"semantic glue\" needed to mediate between different sources of cultural heritage information, such as that published by museums, libraries and archives.",
-            "documentation": "http://www.cidoc-crm.org/Version/version-6.2",
+            "date_created": "2021-12-20T19:15:30Z",
+            "date_released": "2021-12-20T00:00:00Z",
+            "description": "..",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CIDOC-CRM/2/cidoc-crm.owl",
+            "download_url": "media/ontologies/TWCP/1/twcp.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.cidoc-crm.org/",
-            "id": 853,
+            "home_page": null,
+            "id": 4786,
             "indexed": true,
             "max_children": 10,
-            "max_depth": 8,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 85,
+            "nb_classes": 49,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 275,
+            "nb_individuals": 3280,
+            "nb_properties": 18,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "v6.2"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "CIINTEADO",
-        "name": "Ciona intestinalis Anatomy and Development Ontology",
+        "acronym": "TYPON",
+        "name": "Microbial Typing Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 88,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2016-07-29T01:34:35Z",
-            "date_released": "2015-08-07T00:00:00Z",
-            "description": " The first ontology describing the anatomy and the development of Ciona intestinalis, based on the Hotta developmental table. ",
-            "documentation": null,
+            "date_created": "2014-09-29T21:45:27Z",
+            "date_released": "2014-09-29T21:45:25Z",
+            "description": "TyPon provides a comprehensive description of the existing microbial typing methods for the identification of bacterial Isolates and their classification. Such a description constitutes an universal format for the exchange of information on the microbial typing field, providing a vehicle for the integration of the numerous disparate online databases. In its current version, TyPon describes most used microbial typing methods but it is, and always will be, a work in progress given the constant advances in the microbial typing field.",
+            "documentation": "https://bitbucket.org/phyloviz/typon/wiki",
             "domain": null,
-            "download_url": "media/ontologies/CIINTEADO/2/ciinteado.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.aniseed.cnrs.fr/",
-            "id": 359,
+            "download_url": "media/ontologies/TYPON/11/typon.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.phyloviz.net/typon/",
+            "id": 223,
             "indexed": true,
-            "max_children": 371,
-            "max_depth": 2,
+            "max_children": 7,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 880,
+            "nb_classes": 44,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 7,
+            "nb_properties": 46,
             "products": null,
-            "publication": null,
+            "publication": "https://bitbucket.org/phyloviz/typon/wiki",
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 11,
             "taxon": null,
-            "version": "Version 1"
+            "version": "20140929"
         },
         "topics": null
     },
     {
-        "acronym": "CIO",
-        "name": "Confidence Information Ontology",
+        "acronym": "PLANA",
+        "name": "planaria-ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2017-12-12T21:21:15.446087Z",
-            "date_released": "2017-12-12T21:21:15.446064Z",
-            "description": "An ontology to capture confidence information about annotations.",
+            "date_created": "2023-03-27T12:59:31.758713Z",
+            "date_released": "2023-03-27T12:59:31.758703Z",
+            "description": "PLANA, the planarian anatomy ontology, encompasses the anatomy and life cycle stages for both __Schmidtea mediterranea__ biotypes.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/CIO/2/cio.owl",
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/PLANA/28/plana.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/BgeeDB/confidence-information-ontology",
-            "id": 976,
+            "home_page": "https://github.com/obophenotype/planaria-ontology",
+            "id": 5739,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 6,
-            "md5sum": "a3b4298bc3444e8770ae3ad2703c9a7d",
-            "nb_classes": 44,
+            "max_children": 206,
+            "max_depth": 11,
+            "md5sum": "1896425cc842597b0e0011a633b34965",
+            "nb_classes": 876,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 5,
+            "nb_individuals": 18,
+            "nb_properties": 160,
             "products": [
                 {
-                    "id": "cio.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cio.owl"
+                    "id": "plana.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/plana.owl"
                 },
                 {
-                    "id": "cio.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cio.obo"
+                    "id": "plana.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/plana.obo"
                 }
             ],
             "publication": null,
-            "publications": null,
-            "submission_id": 2,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34318308",
+                    "title": "Planarian Anatomy Ontology: a resource to connect data within and across experimental platforms"
+                }
+            ],
+            "submission_id": 28,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "PMD",
+        "name": "Persian Medicine Diseases Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 10,
+            "classifiable": true,
+            "date_created": "2019-06-24T22:00:53Z",
+            "date_released": "2019-06-24T00:00:00Z",
+            "description": "PMD is an ontology that represents classification and related information of Persian medicine diseases. Persian medicine also known as Iranian traditional medicine is one of famous and most ancient forms of traditional medicines. This ontology is based on the top-level ontology GFO (General Formal Ontology)",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PMD/1/pmd.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 2702,
+            "indexed": true,
+            "max_children": 62,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 529,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 41,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TST-VODAN",
+        "name": "VODANA Migrants News Vocabulary",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2021-05-17T14:09:02Z",
+            "date_released": "2021-05-02T00:00:00Z",
+            "description": "migrants ontologies ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/TST-VODAN/3/tst-vodan.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 4483,
+            "indexed": true,
+            "max_children": 3,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 3,
+            "nb_inconsistent": 0,
+            "nb_individuals": 164,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "2.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "CNO_ACRONYM",
+        "name": "CNO_NAME",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2016-06-15T06:49:29Z",
+            "date_released": "2016-06-15T00:00:00Z",
+            "description": "test",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/CNO_ACRONYM/1/cno_acronym.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 593,
+            "indexed": true,
+            "max_children": 27,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 240,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 22,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "version 0.5"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "RDFS",
+        "name": "The RDF Schema vocabulary",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 1,
+            "classifiable": true,
+            "date_created": "2020-05-07T08:11:49Z",
+            "date_released": "2014-02-25T00:00:00Z",
+            "description": "RDF Schema provides a data-modelling vocabulary for RDF data",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/RDFS/1/rdfs.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3610,
+            "indexed": true,
+            "max_children": 3,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 7,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 5,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PMAPP-PMO",
+        "name": "PMO Precision Medicine Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2020-12-17T02:38:23Z",
+            "date_released": "2020-12-16T00:00:00Z",
+            "description": "The Precision Medicine Ontology (PMO) has been developed as a standardized ontology for human precision medicine with consistent, reusable and sustainable descriptions of human disease terms, genomic molecular, phenotype characteristics and related medical vocabulary disease concepts through collaborative efforts of researchers at Institute of Medical Information, Chinese Academy of Medical Sciences. The Precision Medicine Ontology focuses on the development of a high level patients-centric ontology for the academic research, thus, it systematically integrates biomedicine vocabularies from GO, NCBI Gene, DrugBank, HPO, MeSH, SNOMED CT, etc. This work is supported by the National Key Research and Development Program of China (Precision Medicine Knowledge Base, Grant No. 2016YFC0901901).",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PMAPP-PMO/1/pmapp-pmo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.phoc.org.cn/pmo/",
+            "id": 4141,
+            "indexed": true,
+            "max_children": 469,
+            "max_depth": 14,
+            "md5sum": null,
+            "nb_classes": 76155,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 95,
+            "products": null,
+            "publication": "https://www.aimspress.com/article/10.3934/mbe.2020227/fulltext.html",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "4.0"
+        },
+        "topics": null
+    },
+    {
         "acronym": "CIRCUITH",
         "name": "NeuroMorpho.Org Cell type Circuitry",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -29751,14 +29470,48 @@
             "submission_id": 3,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
+        "acronym": "CPRO",
+        "name": "Computer-Based Patient Record Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": false,
+            "date_created": "2013-04-25T00:36:12Z",
+            "date_released": "2013-04-24T07:00:00Z",
+            "description": "A uniform core set of data elements (whose formal semantics are captured in OWL) for use in a Computer-Based Patient Record (CPR)",
+            "documentation": "https://www.researchgate.net/publication/235463154_A_Framework_Ontology_for_Computer-Based_Patient_Record_Systems?ev=prf_pub",
+            "domain": null,
+            "download_url": "media/ontologies/CPRO/3/cpro.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://code.google.com/p/cpr-ontology/",
+            "id": 776,
+            "indexed": false,
+            "max_children": 9,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 134,
+            "nb_inconsistent": 0,
+            "nb_individuals": 32,
+            "nb_properties": 49,
+            "products": null,
+            "publication": "https://www.researchgate.net/publication/235463154_A_Framework_Ontology_for_Computer-Based_Patient_Record_Systems?ev=prf_pub",
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "Last modification: July 31th, 2011 Chimezie Ogbuji"
+        },
+        "topics": null
+    },
+    {
         "acronym": "CISAVIADO",
         "name": "Ciona savignyi Anatomy and Development Ontology ",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 88,
             "classifiable": true,
@@ -29853,14 +29606,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": "v0.1"
         },
         "topics": null
     },
     {
+        "acronym": "CPT-mod",
+        "name": "CPT modifiers",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2012-09-18T19:18:58Z",
+            "date_released": "2012-09-18T07:00:00Z",
+            "description": "Modifiers denote that a certain procedure/service has been altered by a particular circumstance, but not changed in its definition, therefore the same code is used and a modifier is added to denote what has been altered.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/CPT-mod/1/cpt-mod.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 693,
+            "indexed": true,
+            "max_children": 4,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 5,
+            "nb_inconsistent": 0,
+            "nb_individuals": 5,
+            "nb_properties": 18,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "unknown"
+        },
+        "topics": null
+    },
+    {
         "acronym": "PIERO",
         "name": "Enzyme Reaction Ontology for annotating Partial Information of chemical transformation",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
@@ -29887,14 +29674,48 @@
             "submission_id": 10,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "VIVO",
+        "name": "VIVO Ontology for Researcher Discovery",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2012-07-16T15:51:31Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "The VIVO ontology represents researchers in the context of their experience, outputs, interests, accomplishments, and associated institutions.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/VIVO/2/vivo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 423,
+            "indexed": true,
+            "max_children": 32,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 225,
+            "nb_inconsistent": 0,
+            "nb_individuals": 13,
+            "nb_properties": 406,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "version 1.1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "UGANDAFACILITIES",
         "name": "UGANDAFACILITIES",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -29989,112 +29810,78 @@
             "submission_id": 3,
             "taxon": null,
             "version": "0.3"
         },
         "topics": null
     },
     {
-        "acronym": "RXNORM",
-        "name": "RxNORM",
+        "acronym": "UGANDA_DISEASES",
+        "name": "Uganda_diseases",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-02-06T21:26:23Z",
-            "date_released": "2017-02-06T21:26:23Z",
-            "description": "RxNorm Vocabulary",
+            "date_created": "2021-11-23T22:54:43Z",
+            "date_released": "2021-03-29T00:00:00Z",
+            "description": "This is the seventh version of the Uganda diseases",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/RXNORM/12/rxnorm.umls",
-            "has_ontology_language": "UMLS",
+            "download_url": "media/ontologies/UGANDA_DISEASES/17/uganda_diseases.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 353,
+            "id": 4728,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 115514,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 44,
+            "nb_individuals": 308,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
-            "taxon": null,
-            "version": "2016AB"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SAO",
-        "name": "Subcellular Anatomy Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2015-08-22T01:06:37Z",
-            "date_released": "2015-08-22T01:06:37Z",
-            "description": "SAO describes structures from the dimensional range encompassing cellular and subcellular structure, supracellular domains, and macromolecules.",
-            "documentation": "http://ccdb.ucsd.edu/SAO",
-            "domain": null,
-            "download_url": "media/ontologies/SAO/2/sao.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://ccdb.ucsd.edu/SAO",
-            "id": 515,
-            "indexed": true,
-            "max_children": 34,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 764,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 81,
-            "products": null,
-            "publication": "http://ccdb.ucsd.edu/SAO",
-            "publications": null,
-            "submission_id": 2,
+            "submission_id": 17,
             "taxon": null,
-            "version": "Version 1.2"
+            "version": "7"
         },
         "topics": null
     },
     {
-        "acronym": "SCHEMA",
+        "acronym": "SCHEMAORG",
         "name": "Schema.org core and all extension vocabularies",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
-            "date_created": "2017-05-20T00:20:58Z",
-            "date_released": "2017-03-23T00:00:00Z",
+            "date_created": "2018-10-11T18:50:58Z",
+            "date_released": "2018-06-15T00:00:00Z",
             "description": "A collection of schemas that webmasters can use to markup HTML pages in ways recognized by major search providers, and that can also be used for structured data interoperability (e.g. in JSON).",
-            "documentation": "http://schema.org/docs/documents.html",
+            "documentation": "https://schema.org/",
             "domain": null,
-            "download_url": "media/ontologies/SCHEMA/2/schema.owl",
+            "download_url": "media/ontologies/SCHEMAORG/2/schemaorg.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://schema.org/",
-            "id": 123,
+            "home_page": "https://schema.org/",
+            "id": 1939,
             "indexed": true,
-            "max_children": 56,
-            "max_depth": 5,
+            "max_children": 59,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 780,
+            "nb_classes": 822,
             "nb_inconsistent": 0,
-            "nb_individuals": 268,
-            "nb_properties": 0,
+            "nb_individuals": 989,
+            "nb_properties": 680,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "3.2"
+            "version": "3.4"
         },
         "topics": null
     },
     {
         "acronym": "PINO",
         "name": "Pregnancy Information Needs Ontology",
         "species": null,
@@ -30125,520 +29912,547 @@
             "submission_id": 2,
             "taxon": null,
             "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "COKPME",
-        "name": "COKPME - COVID19 Ontology for analyzing the Karnataka Private Medical Establishments Data  ",
+        "acronym": "CIDOC-CRM",
+        "name": "CIDOC Conceptual Reference Model",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-07-09T17:35:14Z",
-            "date_released": "2021-07-09T00:00:00Z",
-            "description": "The ontology will be used by analysts for finding the relative and precautionary measures that can be put in action for controlling the spread. Also, it should be able to handle the competence question like:\r\n\uf0d8 Which properties of the patient should be considered for modeling\r\n\uf0d8 Which properties of patient changes\r\n\uf0d8 Which properties of patient confirm COVID positive\r\n\uf0d8 Do the symptom of the patient change over time ***\r\n\uf0d8 Do the treatment provided depends on the category of patient\r\n\uf0d8 Do other comorbid conditions derive any interestingness ****\r\n\uf0d8 Which symptom is high in a particular district? (Or) Does the geographical location influence the patient category\r\nThese keep change over an iterative process of ontology development",
-            "documentation": null,
+            "date_created": "2017-11-22T19:02:04Z",
+            "date_released": "2015-08-19T00:00:00Z",
+            "description": "The CIDOC Conceptual Reference Model (CRM) provides definitions and a formal structure for describing the implicit and explicit concepts and relationships used in cultural heritage documentation.\r\n\r\nThe CIDOC CRM is intended to promote a shared understanding of cultural heritage information by providing a common and extensible semantic framework that any cultural heritage information can be mapped to. It is intended to be a common language for domain experts and implementers to formulate requirements for information systems and to serve as a guide for good practice of conceptual modelling. In this way, it can provide the \"semantic glue\" needed to mediate between different sources of cultural heritage information, such as that published by museums, libraries and archives.",
+            "documentation": "http://www.cidoc-crm.org/Version/version-6.2",
             "domain": null,
-            "download_url": "media/ontologies/COKPME/1/cokpme.owl",
+            "download_url": "media/ontologies/CIDOC-CRM/2/cidoc-crm.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4644,
+            "home_page": "http://www.cidoc-crm.org/",
+            "id": 853,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 3,
+            "max_children": 10,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 25,
+            "nb_classes": 85,
             "nb_inconsistent": 0,
-            "nb_individuals": 41,
-            "nb_properties": 19,
+            "nb_individuals": 0,
+            "nb_properties": 275,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0"
+            "version": "v6.2"
         },
         "topics": null
     },
     {
-        "acronym": "CRISP",
-        "name": "Computer Retrieval of Information on Scientific Projects Thesaurus",
+        "acronym": "CIINTEADO",
+        "name": "Ciona intestinalis Anatomy and Development Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 88,
             "classifiable": true,
-            "date_created": "2017-02-06T21:25:59Z",
-            "date_released": "2017-02-06T21:25:58Z",
-            "description": "Computer Retrieval of Information on Scientific Projects (CRISP)",
+            "date_created": "2016-07-29T01:34:35Z",
+            "date_released": "2015-08-07T00:00:00Z",
+            "description": " The first ontology describing the anatomy and the development of Ciona intestinalis, based on the Hotta developmental table. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CRISP/9/crisp.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": null,
-            "id": 354,
+            "download_url": "media/ontologies/CIINTEADO/2/ciinteado.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.aniseed.cnrs.fr/",
+            "id": 359,
             "indexed": true,
-            "max_children": 65,
-            "max_depth": 11,
+            "max_children": 371,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 9039,
+            "nb_classes": 880,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2016AB"
+            "version": "Version 1"
         },
         "topics": null
     },
     {
-        "acronym": "COSTART",
-        "name": "Coding Symbols for a Thesaurus of Adverse Reaction Terms",
+        "acronym": "CIO",
+        "name": "Confidence Information Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-02-06T21:25:58Z",
-            "date_released": "2017-02-06T21:25:58Z",
-            "description": "Coding Symbols for Thesaurus of Adverse Reaction Terms (COSTART). 5th ed. Rockville (MD). COSTART is used for coding, filing, and retrieving post-marketing adverse drug and biologic experience reports. COSTART is organized in body system and pathophysiology hierarchies, as well as a separate fetal/neonatal category of less than 20 terms.\r\n\r\nCOSTART has been superseded by the Medical Dictionary for Regulatory Activities (MedDRA) Terminology. For more information about MedDRA in the Metathesaurus, see the MedDRA source synopsis.\r\n\r\nCOSTART was last updated in the Metathesaurus in 1999.",
+            "date_created": "2017-12-12T21:21:15.446087Z",
+            "date_released": "2017-12-12T21:21:15.446064Z",
+            "description": "An ontology to capture confidence information about annotations.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/COSTART/9/costart.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.nlm.nih.gov/research/umls/sourcereleasedocs/current/CST/",
-            "id": 286,
+            "download_url": "media/ontologies/CIO/2/cio.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/BgeeDB/confidence-information-ontology",
+            "id": 976,
             "indexed": true,
-            "max_children": 96,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 1707,
+            "max_children": 4,
+            "max_depth": 6,
+            "md5sum": "a3b4298bc3444e8770ae3ad2703c9a7d",
+            "nb_classes": 44,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
-            "products": null,
+            "nb_properties": 5,
+            "products": [
+                {
+                    "id": "cio.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cio.owl"
+                },
+                {
+                    "id": "cio.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cio.obo"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2016AB"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CSSO",
-        "name": "Clinical Signs and Symptoms Ontology",
+        "acronym": "SciRes",
+        "name": "Scientific Research Resources",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2016-03-31T09:21:30Z",
-            "date_released": "2016-03-31T00:00:00Z",
-            "description": "An ontology for describing clinical signs and symptoms.",
+            "date_created": "2012-08-31T11:01:57Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-public-1.5.owl.  It contains terms related to scientific research resources that are included in the vivo ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CSSO/6/csso.owl",
+            "download_url": "media/ontologies/SciRes/1/scires.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://mdb.bio.titech.ac.jp/csso",
-            "id": 556,
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 648,
             "indexed": true,
-            "max_children": 26,
-            "max_depth": 5,
+            "max_children": 7,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 303,
+            "nb_classes": 20,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 10,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 1,
             "taxon": null,
-            "version": "Version 0.6"
+            "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "CTCAE",
-        "name": "Common Terminology Criteria for Adverse Events",
+        "acronym": "SAO",
+        "name": "Subcellular Anatomy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2011-08-04T19:27:18Z",
-            "date_released": "2010-06-14T07:00:00Z",
-            "description": "A coding system for reporting adverse events that occur in the course of cancer therapy.  It was derived from the Common Toxicity Criteria (CTC) v2.0 and is maintained by the Cancer Therapy Evaluation Program (CTEP) at the National Cancer Institution (NCI).",
-            "documentation": "http://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm",
+            "date_created": "2015-08-22T01:06:37Z",
+            "date_released": "2015-08-22T01:06:37Z",
+            "description": "SAO describes structures from the dimensional range encompassing cellular and subcellular structure, supracellular domains, and macromolecules.",
+            "documentation": "http://ccdb.ucsd.edu/SAO",
             "domain": null,
-            "download_url": "media/ontologies/CTCAE/2/ctcae.owl",
+            "download_url": "media/ontologies/SAO/2/sao.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 379,
+            "home_page": "http://ccdb.ucsd.edu/SAO",
+            "id": 515,
             "indexed": true,
-            "max_children": 117,
-            "max_depth": 4,
+            "max_children": 34,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 3876,
+            "nb_classes": 764,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_properties": 81,
             "products": null,
-            "publication": null,
+            "publication": "http://ccdb.ucsd.edu/SAO",
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "4.03"
+            "version": "Version 1.2"
         },
         "topics": null
     },
     {
-        "acronym": "DCAT-FDC",
-        "name": "Data Catalog Vocabulary (DCAT)",
+        "acronym": "SCHEMA",
+        "name": "Schema.org core and all extension vocabularies",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2021-09-23T09:13:12Z",
-            "date_released": "2021-09-23T00:00:00Z",
-            "description": "DCAT is an RDF vocabulary designed to facilitate interoperability between data catalogs published on the Web. This document defines the schema and provides examples for its use.\r\n\r\nDCAT enables a publisher to describe datasets and data services in a catalog using a standard model and vocabulary that facilitates the consumption and aggregation of metadata from multiple catalogs. This can increase the discoverability of datasets and data services. It also makes it possible to have a decentralized approach to publishing data catalogs and makes federated search for datasets across catalogs in multiple sites possible using the same query mechanism and structure. Aggregated DCAT metadata can serve as a manifest file as part of the digital preservation process.\r\n\r\nThe namespace for DCAT terms is http://www.w3.org/ns/dcat#\r\n\r\nThe suggested prefix for the DCAT namespace is dcat",
-            "documentation": "https://www.w3.org/TR/vocab-dcat-3/",
+            "date_created": "2017-05-20T00:20:58Z",
+            "date_released": "2017-03-23T00:00:00Z",
+            "description": "A collection of schemas that webmasters can use to markup HTML pages in ways recognized by major search providers, and that can also be used for structured data interoperability (e.g. in JSON).",
+            "documentation": "http://schema.org/docs/documents.html",
             "domain": null,
-            "download_url": "media/ontologies/DCAT-FDC/1/dcat-fdc.owl",
+            "download_url": "media/ontologies/SCHEMA/2/schema.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.w3.org/TR/vocab-dcat-3/",
-            "id": 4598,
+            "home_page": "http://schema.org/",
+            "id": 123,
             "indexed": true,
-            "max_children": 15,
-            "max_depth": 4,
+            "max_children": 56,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 52,
+            "nb_classes": 780,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 88,
+            "nb_individuals": 268,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://www.w3.org/TR/vocab-dcat-3/",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "Questa \u00e8 una copia aggiornata del vocabolario DCAT v2.0 disponibile in https://www.w3.org/ns/dcat.ttl"
+            "version": "3.2"
         },
         "topics": null
     },
     {
-        "acronym": "CWD",
-        "name": "Consumer Wearable Device",
+        "acronym": "COKPME",
+        "name": "COKPME - COVID19 Ontology for analyzing the Karnataka Private Medical Establishments Data  ",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-04-19T16:53:22Z",
-            "date_released": "2018-04-19T00:00:00Z",
-            "description": "Consumer Wearable Devices (CWD) enable passive, low-burden, longitudinal collection of data relevant to health behaviors, human performance, and biometrics.   Such data are increasingly used to tailor clinical and public health interventions and as outcome measures in clinical trials.  The uptake of CWD in research applications warrants an examination of the data collected from these products.  This ontology was developed based on a review of publicly available documentation and data sets from three CWD vendors and is open for researchers to modify + extend as new features become available.  \r\n\r\n\r\n\r\n\r\n",
+            "date_created": "2021-07-09T17:35:14Z",
+            "date_released": "2021-07-09T00:00:00Z",
+            "description": "The ontology will be used by analysts for finding the relative and precautionary measures that can be put in action for controlling the spread. Also, it should be able to handle the competence question like:\r\n\uf0d8 Which properties of the patient should be considered for modeling\r\n\uf0d8 Which properties of patient changes\r\n\uf0d8 Which properties of patient confirm COVID positive\r\n\uf0d8 Do the symptom of the patient change over time ***\r\n\uf0d8 Do the treatment provided depends on the category of patient\r\n\uf0d8 Do other comorbid conditions derive any interestingness ****\r\n\uf0d8 Which symptom is high in a particular district? (Or) Does the geographical location influence the patient category\r\nThese keep change over an iterative process of ontology development",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CWD/1/cwd.owl",
+            "download_url": "media/ontologies/COKPME/1/cokpme.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 1423,
+            "id": 4644,
             "indexed": true,
-            "max_children": 9,
-            "max_depth": 4,
+            "max_children": 6,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 46,
+            "nb_classes": 25,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 41,
+            "nb_properties": 19,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "CYTO",
-        "name": "CYTOKINE",
+        "acronym": "CRISP",
+        "name": "Computer Retrieval of Information on Scientific Projects Thesaurus",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 158,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2015-06-24T19:16:39Z",
-            "date_released": "2015-06-24T00:00:00Z",
-            "description": "It is flat ontology showing cytokine synonyms. It has been produced by merging cytokine branches, using SOMA (Synonym Ontology Mapping Approach) method, of 10 Bioportal ontologies.",
+            "date_created": "2017-02-06T21:25:59Z",
+            "date_released": "2017-02-06T21:25:58Z",
+            "description": "Computer Retrieval of Information on Scientific Projects (CRISP)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CYTO/27/cyto.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/CRISP/9/crisp.umls",
+            "has_ontology_language": "UMLS",
             "home_page": null,
-            "id": 268,
+            "id": 354,
             "indexed": true,
-            "max_children": 315,
-            "max_depth": 1,
+            "max_children": 65,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 317,
+            "nb_classes": 9039,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 3,
+            "nb_individuals": 0,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 27,
+            "submission_id": 9,
             "taxon": null,
-            "version": "Cytokines"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "DATASET",
-        "name": "dataset",
+        "acronym": "COSTART",
+        "name": "Coding Symbols for a Thesaurus of Adverse Reaction Terms",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2019-10-11T08:01:17Z",
-            "date_released": "2019-10-11T00:00:00Z",
-            "description": "You can choose what Dataset you want",
+            "date_created": "2017-02-06T21:25:58Z",
+            "date_released": "2017-02-06T21:25:58Z",
+            "description": "Coding Symbols for Thesaurus of Adverse Reaction Terms (COSTART). 5th ed. Rockville (MD). COSTART is used for coding, filing, and retrieving post-marketing adverse drug and biologic experience reports. COSTART is organized in body system and pathophysiology hierarchies, as well as a separate fetal/neonatal category of less than 20 terms.\r\n\r\nCOSTART has been superseded by the Medical Dictionary for Regulatory Activities (MedDRA) Terminology. For more information about MedDRA in the Metathesaurus, see the MedDRA source synopsis.\r\n\r\nCOSTART was last updated in the Metathesaurus in 1999.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DATASET/1/dataset.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3005,
+            "download_url": "media/ontologies/COSTART/9/costart.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.nlm.nih.gov/research/umls/sourcereleasedocs/current/CST/",
+            "id": 286,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 3,
+            "max_children": 96,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 15,
+            "nb_classes": 1707,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 9,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "DC",
-        "name": "Dublin Core",
+        "acronym": "CTCAE",
+        "name": "Common Terminology Criteria for Adverse Events",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 11,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-02-18T02:01:47Z",
-            "date_released": "2017-02-18T02:01:47Z",
-            "description": "The Dublin Core Schema is a small set of vocabulary terms that can be used to describe several kinds of resources. Dublin Core Metadata may be used for multiple purposes, from simple resource description, to combining metadata vocabularies of different metadata standards, to providing interoperability for metadata vocabularies in the Linked Data cloud and Semantic Web implementations.",
-            "documentation": "http://dublincore.org/specifications",
+            "date_created": "2011-08-04T19:27:18Z",
+            "date_released": "2010-06-14T07:00:00Z",
+            "description": "A coding system for reporting adverse events that occur in the course of cancer therapy.  It was derived from the Common Toxicity Criteria (CTC) v2.0 and is maintained by the Cancer Therapy Evaluation Program (CTEP) at the National Cancer Institution (NCI).",
+            "documentation": "http://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm",
             "domain": null,
-            "download_url": "media/ontologies/DC/3/dc.owl",
+            "download_url": "media/ontologies/CTCAE/2/ctcae.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://dublincore.org",
-            "id": 244,
+            "home_page": null,
+            "id": 379,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 0,
+            "max_children": 117,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 11,
+            "nb_classes": 3876,
             "nb_inconsistent": 0,
-            "nb_individuals": 26,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "4.03"
         },
         "topics": null
     },
     {
-        "acronym": "DCAT",
-        "name": "Data Catalog Vocabulary",
+        "acronym": "CWD",
+        "name": "Consumer Wearable Device",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-12-07T20:14:08Z",
-            "date_released": "2014-01-16T00:00:00Z",
-            "description": "DCAT is an RDF vocabulary designed to facilitate interoperability between data catalogs published on the Web.",
+            "date_created": "2018-04-19T16:53:22Z",
+            "date_released": "2018-04-19T00:00:00Z",
+            "description": "Consumer Wearable Devices (CWD) enable passive, low-burden, longitudinal collection of data relevant to health behaviors, human performance, and biometrics.   Such data are increasingly used to tailor clinical and public health interventions and as outcome measures in clinical trials.  The uptake of CWD in research applications warrants an examination of the data collected from these products.  This ontology was developed based on a review of publicly available documentation and data sets from three CWD vendors and is open for researchers to modify + extend as new features become available.  \r\n\r\n\r\n\r\n\r\n",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DCAT/2/dcat.owl",
+            "download_url": "media/ontologies/CWD/1/cwd.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.w3.org/TR/vocab-dcat/",
-            "id": 929,
+            "home_page": null,
+            "id": 1423,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
+            "max_children": 9,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 15,
+            "nb_classes": 46,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 15,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DCCDFV",
-        "name": "Dublin Core Collection Description Frequency Vocabulary",
+        "acronym": "DCAT-FDC",
+        "name": "Data Catalog Vocabulary (DCAT)",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-02-03T22:52:57Z",
-            "date_released": "2018-02-03T00:00:00Z",
-            "description": "The Collection Description Accrual Policy Vocabulary provides a set of terms that can be used as values of the dcterms:accrualPolicy property in descriptions of collections.",
-            "documentation": null,
+            "date_created": "2021-09-23T09:13:12Z",
+            "date_released": "2021-09-23T00:00:00Z",
+            "description": "DCAT is an RDF vocabulary designed to facilitate interoperability between data catalogs published on the Web. This document defines the schema and provides examples for its use.\r\n\r\nDCAT enables a publisher to describe datasets and data services in a catalog using a standard model and vocabulary that facilitates the consumption and aggregation of metadata from multiple catalogs. This can increase the discoverability of datasets and data services. It also makes it possible to have a decentralized approach to publishing data catalogs and makes federated search for datasets across catalogs in multiple sites possible using the same query mechanism and structure. Aggregated DCAT metadata can serve as a manifest file as part of the digital preservation process.\r\n\r\nThe namespace for DCAT terms is http://www.w3.org/ns/dcat#\r\n\r\nThe suggested prefix for the DCAT namespace is dcat",
+            "documentation": "https://www.w3.org/TR/vocab-dcat-3/",
             "domain": null,
-            "download_url": "media/ontologies/DCCDFV/1/dccdfv.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://dublincore.org/groups/collections/accrual-policy/",
-            "id": 1218,
+            "download_url": "media/ontologies/DCAT-FDC/1/dcat-fdc.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.w3.org/TR/vocab-dcat-3/",
+            "id": 4598,
             "indexed": true,
-            "max_children": 1,
-            "max_depth": 1,
+            "max_children": 15,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 1,
+            "nb_classes": 52,
             "nb_inconsistent": 0,
-            "nb_individuals": 17,
-            "nb_properties": 0,
+            "nb_individuals": 18,
+            "nb_properties": 88,
             "products": null,
-            "publication": null,
+            "publication": "https://www.w3.org/TR/vocab-dcat-3/",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Questa \u00e8 una copia aggiornata del vocabolario DCAT v2.0 disponibile in https://www.w3.org/ns/dcat.ttl"
         },
         "topics": null
     },
     {
-        "acronym": "dcelements",
-        "name": "dcelements",
+        "acronym": "MOD",
+        "name": "Protein modification",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2012-08-31T09:41:04Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-public-1.5.owl.  It contains selected terms from the  \thttp://purl.org/dc/elements/1.1/ namespace that are included in the vivo core ontology.",
+            "date_created": "2022-06-13T12:36:58.869666Z",
+            "date_released": "2022-06-13T12:36:58.869649Z",
+            "description": "PSI-MOD is an ontology consisting of terms that describe protein chemical modifications",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/dcelements/1/dcelements.owl",
+            "domain": "chemistry and biochemistry",
+            "download_url": "media/ontologies/MOD/122/mod.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 204,
+            "home_page": "http://www.psidev.info/MOD",
+            "id": 5135,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 0,
+            "max_children": 121,
+            "max_depth": 7,
+            "md5sum": "1a6f38326f5d2d4518ec076549622e04",
+            "nb_classes": 2098,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
-            "products": null,
+            "nb_properties": 4,
+            "products": [
+                {
+                    "description": "PSI-MOD Ontology, OWL format",
+                    "id": "mod.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mod.owl",
+                    "title": "PSI-MOD.owl"
+                },
+                {
+                    "description": "PSI-MOD Ontology, OBO format",
+                    "id": "mod.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mod.obo",
+                    "title": "PSI-MOD.obo"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 1,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/18688235",
+                    "title": "The PSI-MOD community standard for representation of protein modification data"
+                }
+            ],
+            "submission_id": 122,
             "taxon": null,
-            "version": "1.5"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DCMITYPE",
-        "name": "DCMI Type Vocabulary",
+        "acronym": "MODSCI",
+        "name": "Modern Science Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 8,
             "classifiable": true,
-            "date_created": "2017-07-18T15:06:36Z",
-            "date_released": "2014-12-02T00:00:00Z",
-            "description": "The DCMI Type Vocabulary provides a general, cross-domain list of approved terms that may be used as values for the Resource Type element to identify the genre of a resource",
-            "documentation": null,
+            "date_created": "2019-10-08T21:26:09Z",
+            "date_released": "2019-10-01T00:00:00Z",
+            "description": "ModSci is a reference ontology for modelling different types of modern sciences and related entities, such as scientific discoveries, renowned scientists, instruments, phenomena ... etc.",
+            "documentation": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
             "domain": null,
-            "download_url": "media/ontologies/DCMITYPE/4/dcmitype.owl",
+            "download_url": "media/ontologies/MODSCI/3/modsci.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://dublincore.org/documents/dcmi-type-vocabulary",
-            "id": 497,
+            "home_page": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
+            "id": 3006,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 37,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 0,
+            "nb_classes": 193,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 25,
+            "nb_properties": 16,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 3,
             "taxon": null,
-            "version": null
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "DCT",
-        "name": "DC Terms",
+        "acronym": "CSSO",
+        "name": "Clinical Signs and Symptoms Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-12-09T22:26:03Z",
-            "date_released": "2017-12-09T00:00:00Z",
-            "description": "Specification of all metadata terms maintained by the Dublin Core Metadata Initiative. Included are the fifteen terms of the Dublin Core Metadata Element Set",
-            "documentation": "http://dublincore.org/2012/06/14/dcterms",
+            "date_created": "2016-03-31T09:21:30Z",
+            "date_released": "2016-03-31T00:00:00Z",
+            "description": "An ontology for describing clinical signs and symptoms.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DCT/1/dct.owl",
+            "download_url": "media/ontologies/CSSO/6/csso.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 946,
+            "home_page": "http://mdb.bio.titech.ac.jp/csso",
+            "id": 556,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 3,
+            "max_children": 26,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 25,
+            "nb_classes": 303,
             "nb_inconsistent": 0,
-            "nb_individuals": 100,
-            "nb_properties": 24,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
-            "version": null
+            "version": "Version 0.6"
         },
         "topics": null
     },
     {
         "acronym": "MFOMD",
         "name": "Mental Disease Ontology",
         "species": null,
@@ -30674,116 +30488,14 @@
             "submission_id": 3,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "dcterms",
-        "name": "dcterms",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 0,
-            "classifiable": true,
-            "date_created": "2012-08-31T09:58:46Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-public-1.5.owl. It contains terms from the http://purl.org/dc/terms/ namespace that are included in the vivo ontology.\r\n\r\n",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/dcterms/1/dcterms.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 491,
-            "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 0,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 5,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.5"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MNV",
-        "name": "VODANA Migrants",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-07-04T18:42:51Z",
-            "date_released": "2021-07-21T00:00:00Z",
-            "description": "migrants ontologies",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MNV/1/mnv.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4603,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 232,
-            "nb_properties": 21,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MOBLAYERH",
-        "name": "NeuroMorpho.Org Main Olfactory Bulb layers",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2016-08-29T19:39:11Z",
-            "date_released": "2016-08-29T00:00:00Z",
-            "description": "A hierarchy of layers in the Main Olfactory Bulb of mammals.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MOBLAYERH/2/moblayerh.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 124,
-            "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 8,
-            "nb_inconsistent": 0,
-            "nb_individuals": 11,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
         "acronym": "ONTODT",
         "name": "Ontology of Datatypes",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -30810,100 +30522,14 @@
             "submission_id": 8,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "MOD",
-        "name": "Protein modification",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 6,
-            "classifiable": true,
-            "date_created": "2022-06-13T12:36:58.869666Z",
-            "date_released": "2022-06-13T12:36:58.869649Z",
-            "description": "PSI-MOD is an ontology consisting of terms that describe protein chemical modifications",
-            "documentation": null,
-            "domain": "chemistry and biochemistry",
-            "download_url": "media/ontologies/MOD/122/mod.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.psidev.info/MOD",
-            "id": 5135,
-            "indexed": true,
-            "max_children": 121,
-            "max_depth": 7,
-            "md5sum": "1a6f38326f5d2d4518ec076549622e04",
-            "nb_classes": 2098,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": [
-                {
-                    "description": "PSI-MOD Ontology, OWL format",
-                    "id": "mod.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mod.owl",
-                    "title": "PSI-MOD.owl"
-                },
-                {
-                    "description": "PSI-MOD Ontology, OBO format",
-                    "id": "mod.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mod.obo",
-                    "title": "PSI-MOD.obo"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/18688235",
-                    "title": "The PSI-MOD community standard for representation of protein modification data"
-                }
-            ],
-            "submission_id": 122,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "MODSCI",
-        "name": "Modern Science Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 8,
-            "classifiable": true,
-            "date_created": "2019-10-08T21:26:09Z",
-            "date_released": "2019-10-01T00:00:00Z",
-            "description": "ModSci is a reference ontology for modelling different types of modern sciences and related entities, such as scientific discoveries, renowned scientists, instruments, phenomena ... etc.",
-            "documentation": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
-            "domain": null,
-            "download_url": "media/ontologies/MODSCI/3/modsci.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
-            "id": 3006,
-            "indexed": true,
-            "max_children": 37,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 193,
-            "nb_inconsistent": 0,
-            "nb_individuals": 25,
-            "nb_properties": 16,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
         "acronym": "MONDO",
         "name": "Mondo Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -30970,736 +30596,669 @@
                 "label": "Metazoa"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SDO",
-        "name": "Sleep Domain Ontology",
+        "acronym": "DC",
+        "name": "Dublin Core",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 11,
             "classifiable": true,
-            "date_created": "2015-06-24T00:11:29Z",
-            "date_released": "2015-06-23T00:00:00Z",
-            "description": "An application ontology for the domain of Sleep Medicine. \r\n",
-            "documentation": "https://mimi.case.edu/concepts",
+            "date_created": "2017-02-18T02:01:47Z",
+            "date_released": "2017-02-18T02:01:47Z",
+            "description": "The Dublin Core Schema is a small set of vocabulary terms that can be used to describe several kinds of resources. Dublin Core Metadata may be used for multiple purposes, from simple resource description, to combining metadata vocabularies of different metadata standards, to providing interoperability for metadata vocabularies in the Linked Data cloud and Semantic Web implementations.",
+            "documentation": "http://dublincore.org/specifications",
             "domain": null,
-            "download_url": "media/ontologies/SDO/4/sdo.owl",
+            "download_url": "media/ontologies/DC/3/dc.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://mimi.case.edu/concepts",
-            "id": 734,
+            "home_page": "http://dublincore.org",
+            "id": 244,
             "indexed": true,
-            "max_children": 48,
-            "max_depth": 19,
+            "max_children": 11,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 1382,
+            "nb_classes": 11,
             "nb_inconsistent": 0,
-            "nb_individuals": 59,
-            "nb_properties": 100,
+            "nb_individuals": 26,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 3,
             "taxon": null,
-            "version": "Last modification: Feb. 11, 2013 Sivaram Arabandi"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CTENO",
-        "name": "Ctenophore Ontology",
+        "acronym": "DCAT",
+        "name": "Data Catalog Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-12-12T21:37:03.413382Z",
-            "date_released": "2017-12-12T21:37:03.413359Z",
-            "description": "An anatomical and developmental ontology for ctenophores (Comb Jellies)",
+            "date_created": "2017-12-07T20:14:08Z",
+            "date_released": "2014-01-16T00:00:00Z",
+            "description": "DCAT is an RDF vocabulary designed to facilitate interoperability between data catalogs published on the Web.",
             "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/CTENO/2/cteno.owl",
+            "domain": null,
+            "download_url": "media/ontologies/DCAT/2/dcat.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/ctenophore-ontology",
-            "id": 980,
+            "home_page": "https://www.w3.org/TR/vocab-dcat/",
+            "id": 929,
             "indexed": true,
-            "max_children": 37,
-            "max_depth": 13,
-            "md5sum": "94d13aabf97c2086624d26c5944086ff",
-            "nb_classes": 648,
+            "max_children": 5,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 15,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 85,
-            "products": [
-                {
-                    "id": "cteno.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cteno.owl"
-                }
-            ],
+            "nb_individuals": 1,
+            "nb_properties": 15,
+            "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
-            "taxon": {
-                "id": "NCBITaxon:10197",
-                "label": "Ctenophore"
-            },
+            "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CTO",
-        "name": "CTO: Core Ontology of Clinical Trials",
+        "acronym": "DCCDFV",
+        "name": "Dublin Core Collection Description Frequency Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2021-10-25T13:31:28.118949Z",
-            "date_released": "2021-10-25T13:31:28.118924Z",
-            "description": "The core Ontology of Clinical Trials (CTO) will serve as a structured resource integrating basic terms and concepts in the context of clinical trials. Thereby covering clinicaltrails.gov. CoreCTO will serve as a basic ontology to generate extended versions for specific applications such as annotation of variables in study documents from clinical trials.",
+            "date_created": "2018-02-03T22:52:57Z",
+            "date_released": "2018-02-03T00:00:00Z",
+            "description": "The Collection Description Accrual Policy Vocabulary provides a set of terms that can be used as values of the dcterms:accrualPolicy property in descriptions of collections.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CTO/5/cto.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ClinicalTrialOntology/CTO/",
-            "id": 4515,
+            "download_url": "media/ontologies/DCCDFV/1/dccdfv.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://dublincore.org/groups/collections/accrual-policy/",
+            "id": 1218,
             "indexed": true,
-            "max_children": 19,
-            "max_depth": 12,
-            "md5sum": "5dc0f3ca86ebbdba22063a4190aaa6b9",
-            "nb_classes": 299,
+            "max_children": 1,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 1,
             "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 18,
-            "products": [
-                {
-                    "id": "cto.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/cto.owl"
-                }
-            ],
+            "nb_individuals": 17,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "CTO-NDD",
-        "name": "Clinical Trials Ontology - Neurodegenerative Diseases",
+        "acronym": "dcelements",
+        "name": "dcelements",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2020-05-26T12:26:57Z",
-            "date_released": "2020-05-26T00:00:00Z",
-            "description": "The CTO-NDD stands for Clinical Trial Ontology-Neurodegenerative Diseases, which was built with the purpose of organizing the knowledge domain of clinical trials in neurodegeneration field. This resource has been created for use in the IMI-funded AETIONOMY project (www.aetionomy.org). ",
+            "date_created": "2012-08-31T09:41:04Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-public-1.5.owl.  It contains selected terms from the  \thttp://purl.org/dc/elements/1.1/ namespace that are included in the vivo core ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CTO-NDD/1/cto-ndd.owl",
+            "download_url": "media/ontologies/dcelements/1/dcelements.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3653,
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 204,
             "indexed": true,
-            "max_children": 18,
-            "max_depth": 13,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 377,
+            "nb_classes": 0,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.6"
+            "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "CVAO",
-        "name": "STO",
+        "acronym": "DCMITYPE",
+        "name": "DCMI Type Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 18,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2017-11-28T12:33:04Z",
-            "date_released": "2017-11-28T00:00:00Z",
-            "description": "Stroke ontology",
+            "date_created": "2017-07-18T15:06:36Z",
+            "date_released": "2014-12-02T00:00:00Z",
+            "description": "The DCMI Type Vocabulary provides a general, cross-domain list of approved terms that may be used as values for the Resource Type element to identify the genre of a resource",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/CVAO/2/cvao.owl",
+            "download_url": "media/ontologies/DCMITYPE/4/dcmitype.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 874,
+            "home_page": "http://dublincore.org/documents/dcmi-type-vocabulary",
+            "id": 497,
             "indexed": true,
-            "max_children": 1676,
-            "max_depth": 5,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 1712,
+            "nb_classes": 0,
             "nb_inconsistent": 0,
-            "nb_individuals": 69,
-            "nb_properties": 29,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 4,
             "taxon": null,
-            "version": "STO_V1.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ModSci",
-        "name": "Modern Science Ontology",
-        "species": [],
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 6,
-            "classifiable": true,
-            "date_created": "2019-12-19T14:12:41.595572Z",
-            "date_released": "2019-12-19T14:12:41.595604Z",
-            "description": "Modern Science Ontology (ModSci), for modeling relationships between modern science branches and related entities, such as scientific discoveries, phenomena, renowned scientists, instruments, etc. ModSci is an upper ontology that provides a unifying framework for the various domain ontologies that make up the Science Knowledge Graph Ontology Suite. The Systematic Approach for Building Ontologies (SABiO) has been followed in the development of ModSci. ModSci is a poly-hierarchical ontology that can be utilized in semantic web applications. ModSci powers two projects for semantically representing scholarly information: Open Research Knowledge Graph (for the categorization of research papers) and OpenResearch.org (for the categorization of the information about scientific events, research projects, scientific papers, publishers, and journals).",
-            "documentation": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
-            "domain": null,
-            "download_url": "media/ontologies/ModSci/2/modsci.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
-            "id": 3187,
-            "indexed": true,
-            "max_children": 38,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 264,
-            "nb_inconsistent": 0,
-            "nb_individuals": 130,
-            "nb_properties": 103,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "1.1"
-        },
-        "topics": [
-            "ModSci models relationships between modern science branches and related entities",
-            "such as scientific discoveries",
-            "renowned scientists",
-            "instruments",
-            "phenomena ... etc."
-        ]
-    },
-    {
-        "acronym": "DATACITE",
-        "name": "SPAR DataCite Ontology",
+        "acronym": "DCT",
+        "name": "DC Terms",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-09-16T01:03:32Z",
-            "date_released": "2022-09-16T01:03:27Z",
-            "description": "The DataCite Ontology (DataCite) is an ontology that enables the metadata properties of the DataCite Metadata Schema Specification (i.e., a list of metadata properties for the accurate and consistent identification of a resource for citation and retrieval purposes) to be described in RDF.",
-            "documentation": null,
+            "date_created": "2017-12-09T22:26:03Z",
+            "date_released": "2017-12-09T00:00:00Z",
+            "description": "Specification of all metadata terms maintained by the Dublin Core Metadata Initiative. Included are the fifteen terms of the Dublin Core Metadata Element Set",
+            "documentation": "http://dublincore.org/2012/06/14/dcterms",
             "domain": null,
-            "download_url": "media/ontologies/DATACITE/2/datacite.owl",
+            "download_url": "media/ontologies/DCT/1/dct.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.sparontologies.net/ontologies/datacite",
-            "id": 5364,
+            "home_page": null,
+            "id": 946,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 5,
+            "max_children": 16,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 21,
+            "nb_classes": 25,
             "nb_inconsistent": 0,
-            "nb_individuals": 70,
-            "nb_properties": 14,
+            "nb_individuals": 100,
+            "nb_properties": 24,
             "products": null,
-            "publication": "http://dx.doi.org/10.6084/m9.figshare.2075356",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.2"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DERMLEX",
-        "name": "Dermatology Lexicon",
+        "acronym": "dcterms",
+        "name": "dcterms",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2009-04-17T19:12:06Z",
-            "date_released": "2009-04-30T07:00:00Z",
-            "description": "Version 1.0 of the American Academy of Dermatology's DermLex: The Dermatology Lexicon, a standardized terminology of dermatologic diagnoses, therapies, procedures, and laboratory tests.",
+            "date_created": "2012-08-31T09:58:46Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-public-1.5.owl. It contains terms from the http://purl.org/dc/terms/ namespace that are included in the vivo ontology.\r\n\r\n",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DERMLEX/4/dermlex.owl",
+            "download_url": "media/ontologies/dcterms/1/dcterms.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.aad.org/research/lexicon/",
-            "id": 462,
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 491,
             "indexed": true,
-            "max_children": 164,
-            "max_depth": 10,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 6106,
+            "nb_classes": 0,
             "nb_inconsistent": 0,
-            "nb_individuals": 6101,
-            "nb_properties": 18,
+            "nb_individuals": 0,
+            "nb_properties": 5,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "1.5"
         },
         "topics": null
     },
     {
-        "acronym": "DERMO",
-        "name": "Human Dermatological Disease Ontology",
+        "acronym": "MNV",
+        "name": "VODANA Migrants",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2015-10-13T01:01:39Z",
-            "date_released": "2015-10-13T01:01:38Z",
-            "description": "Ontology of human dermatologic disease",
+            "date_created": "2021-07-04T18:42:51Z",
+            "date_released": "2021-07-21T00:00:00Z",
+            "description": "migrants ontologies",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DERMO/10/dermo.obo",
-            "has_ontology_language": "OBO",
+            "download_url": "media/ontologies/MNV/1/mnv.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 749,
+            "id": 4603,
             "indexed": true,
-            "max_children": 25,
-            "max_depth": 11,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 3521,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
+            "nb_individuals": 232,
+            "nb_properties": 21,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 10,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.19"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "DFO",
-        "name": "Databank Family Ontology",
+        "acronym": "MOBLAYERH",
+        "name": "NeuroMorpho.Org Main Olfactory Bulb layers",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-10-08T12:09:54Z",
-            "date_released": "2021-10-08T00:00:00Z",
-            "description": "The Databank Family Ontology describes groups of data banks with similar record prompts and originators.",
-            "documentation": "https://github.com/molgenis/ontologies/tree/main/dfo",
+            "date_created": "2016-08-29T19:39:11Z",
+            "date_released": "2016-08-29T00:00:00Z",
+            "description": "A hierarchy of layers in the Main Olfactory Bulb of mammals.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DFO/1/dfo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://w3id.org/dfo",
-            "id": 4582,
+            "download_url": "media/ontologies/MOBLAYERH/2/moblayerh.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 124,
             "indexed": true,
-            "max_children": 10,
+            "max_children": 5,
             "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 12,
+            "nb_classes": 8,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 11,
             "nb_properties": 0,
             "products": null,
-            "publication": "forthcoming",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.1"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "DIAGONT",
-        "name": "Diagnostic Ontology",
+        "acronym": "SDO",
+        "name": "Sleep Domain Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2012-02-27T21:40:30Z",
-            "date_released": "2011-12-15T08:00:00Z",
-            "description": "We built a computational diagnostic ontology containing 91 elements, including classes and sub-classes, which are required to conduct SR-MA (Systematic Review - Meta Analysis) for diagnostic studies, that will assist in standardized reporting of diagnostic articles. We also report high percentage of agreement among five observers as a result of the inter-observer agreement that we conducted among them to tag 13 articles using the diagnostic ontology. Moreover, we extend our existing repository CERR-N (Center of Excellence in Research Reporting in Neurosurgery) to include diagnostic studies.",
-            "documentation": null,
+            "date_created": "2015-06-24T00:11:29Z",
+            "date_released": "2015-06-23T00:00:00Z",
+            "description": "An application ontology for the domain of Sleep Medicine. \r\n",
+            "documentation": "https://mimi.case.edu/concepts",
             "domain": null,
-            "download_url": "media/ontologies/DIAGONT/1/diagont.owl",
+            "download_url": "media/ontologies/SDO/4/sdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 650,
+            "home_page": "https://mimi.case.edu/concepts",
+            "id": 734,
             "indexed": true,
-            "max_children": 9,
-            "max_depth": 5,
+            "max_children": 48,
+            "max_depth": 19,
             "md5sum": null,
-            "nb_classes": 96,
-            "nb_inconsistent": 12,
-            "nb_individuals": 6,
-            "nb_properties": 9,
+            "nb_classes": 1382,
+            "nb_inconsistent": 0,
+            "nb_individuals": 59,
+            "nb_properties": 100,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "2.0"
+            "version": "Last modification: Feb. 11, 2013 Sivaram Arabandi"
         },
         "topics": null
     },
     {
-        "acronym": "DISDRIV",
-        "name": "Disease Drivers Ontology",
+        "acronym": "CTENO",
+        "name": "Ctenophore Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-11-22T12:21:05.058461Z",
-            "date_released": "2021-11-22T12:21:05.058450Z",
-            "description": "Ontology for drivers and triggers of human diseases, built to classify ExO ontology exposure stressors. An application ontology. Built in collaboration with EnvO, ExO, ECTO and ChEBI.",
+            "date_created": "2017-12-12T21:37:03.413382Z",
+            "date_released": "2017-12-12T21:37:03.413359Z",
+            "description": "An anatomical and developmental ontology for ctenophores (Comb Jellies)",
             "documentation": null,
-            "domain": "disease",
-            "download_url": "media/ontologies/DISDRIV/1/disdriv.owl",
+            "domain": "anatomy",
+            "download_url": "media/ontologies/CTENO/2/cteno.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.disease-ontology.org",
-            "id": 4702,
+            "home_page": "https://github.com/obophenotype/ctenophore-ontology",
+            "id": 980,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 4,
-            "md5sum": "57619032fc0d43b45fe7b2951c8ee8e2",
-            "nb_classes": 13,
+            "max_children": 37,
+            "max_depth": 13,
+            "md5sum": "94d13aabf97c2086624d26c5944086ff",
+            "nb_classes": 648,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 3,
+            "nb_properties": 85,
             "products": [
                 {
-                    "id": "disdriv.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/disdriv.owl"
+                    "id": "cteno.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cteno.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": {
-                "id": "NCBITaxon:9606",
-                "label": "Homo sapiens"
+                "id": "NCBITaxon:10197",
+                "label": "Ctenophore"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DLO",
-        "name": "Deep Learning Ontology",
+        "acronym": "CTO",
+        "name": "CTO: Core Ontology of Clinical Trials",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-07-27T21:03:23Z",
-            "date_released": "2021-07-23T00:00:00Z",
-            "description": "Taxonomy of Deep Learning networks and layers. TODO: partonomy",
+            "date_created": "2021-10-25T13:31:28.118949Z",
+            "date_released": "2021-10-25T13:31:28.118924Z",
+            "description": "The core Ontology of Clinical Trials (CTO) will serve as a structured resource integrating basic terms and concepts in the context of clinical trials. Thereby covering clinicaltrails.gov. CoreCTO will serve as a basic ontology to generate extended versions for specific applications such as annotation of variables in study documents from clinical trials.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DLO/2/dlo.owl",
+            "download_url": "media/ontologies/CTO/5/cto.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/turbomam/deep-learning-ontology",
-            "id": 4637,
+            "home_page": "https://github.com/ClinicalTrialOntology/CTO/",
+            "id": 4515,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 61,
+            "max_children": 19,
+            "max_depth": 12,
+            "md5sum": "5dc0f3ca86ebbdba22063a4190aaa6b9",
+            "nb_classes": 299,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
+            "nb_individuals": 5,
+            "nb_properties": 18,
+            "products": [
+                {
+                    "id": "cto.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/cto.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 5,
             "taxon": null,
-            "version": "2021-07-27"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MRO",
-        "name": "MHC Restriction Ontology",
-        "species": null,
-        "status": "Classified",
+        "acronym": "ModSci",
+        "name": "Modern Science Ontology",
+        "species": [],
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 50,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2023-07-10T12:36:58.633533Z",
-            "date_released": "2023-07-10T12:36:58.633515Z",
-            "description": "An ontology for Major Histocompatibility Complex (MHC) restriction in experiments",
-            "documentation": null,
-            "domain": "chemistry and biochemistry",
-            "download_url": "media/ontologies/MRO/53/mro.owl",
+            "date_created": "2019-12-19T14:12:41.595572Z",
+            "date_released": "2019-12-19T14:12:41.595604Z",
+            "description": "Modern Science Ontology (ModSci), for modeling relationships between modern science branches and related entities, such as scientific discoveries, phenomena, renowned scientists, instruments, etc. ModSci is an upper ontology that provides a unifying framework for the various domain ontologies that make up the Science Knowledge Graph Ontology Suite. The Systematic Approach for Building Ontologies (SABiO) has been followed in the development of ModSci. ModSci is a poly-hierarchical ontology that can be utilized in semantic web applications. ModSci powers two projects for semantically representing scholarly information: Open Research Knowledge Graph (for the categorization of research papers) and OpenResearch.org (for the categorization of the information about scientific events, research projects, scientific papers, publishers, and journals).",
+            "documentation": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
+            "domain": null,
+            "download_url": "media/ontologies/ModSci/2/modsci.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/IEDB/MRO",
-            "id": 5935,
+            "home_page": "https://saidfathalla.github.io/Science-knowledge-graph-ontologies/doc/ModSci_doc/index-en.html",
+            "id": 3187,
             "indexed": true,
-            "max_children": 4732,
-            "max_depth": 8,
-            "md5sum": "09fcc5fc62064bca6b3938036dcdb046",
-            "nb_classes": 51390,
+            "max_children": 38,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 264,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 21,
-            "products": [
-                {
-                    "id": "mro.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mro.owl"
-                }
-            ],
+            "nb_individuals": 130,
+            "nb_properties": 103,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 53,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "1.1"
         },
-        "topics": null
+        "topics": [
+            "ModSci models relationships between modern science branches and related entities",
+            "such as scientific discoveries",
+            "renowned scientists",
+            "instruments",
+            "phenomena ... etc."
+        ]
     },
     {
-        "acronym": "MS",
-        "name": "Mass spectrometry ontology",
+        "acronym": "CTO-NDD",
+        "name": "Clinical Trials Ontology - Neurodegenerative Diseases",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 11,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-07-17T12:31:28.754196Z",
-            "date_released": "2023-07-17T12:31:28.754184Z",
-            "description": "A structured controlled vocabulary for the annotation of experiments concerned with proteomics mass spectrometry.",
+            "date_created": "2020-05-26T12:26:57Z",
+            "date_released": "2020-05-26T00:00:00Z",
+            "description": "The CTO-NDD stands for Clinical Trial Ontology-Neurodegenerative Diseases, which was built with the purpose of organizing the knowledge domain of clinical trials in neurodegeneration field. This resource has been created for use in the IMI-funded AETIONOMY project (www.aetionomy.org). ",
             "documentation": null,
-            "domain": "investigations",
-            "download_url": "media/ontologies/MS/185/ms.owl",
+            "domain": null,
+            "download_url": "media/ontologies/CTO-NDD/1/cto-ndd.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.psidev.info/groups/controlled-vocabularies",
-            "id": 5948,
+            "home_page": null,
+            "id": 3653,
             "indexed": true,
-            "max_children": 1350,
-            "max_depth": 37,
-            "md5sum": "47a02cb81dd3bc8f039f99b02c932b5f",
-            "nb_classes": 14964,
-            "nb_inconsistent": 16,
-            "nb_individuals": 14,
-            "nb_properties": 369,
-            "products": [
-                {
-                    "id": "ms.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ms.obo"
-                },
-                {
-                    "id": "ms.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ms.owl"
-                }
-            ],
+            "max_children": 18,
+            "max_depth": 13,
+            "md5sum": null,
+            "nb_classes": 377,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/23482073",
-                    "title": "The HUPO proteomics standards initiative- mass spectrometry controlled vocabulary."
-                }
-            ],
-            "submission_id": 185,
+            "publications": null,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "0.6"
         },
         "topics": null
     },
     {
-        "acronym": "MSR",
-        "name": "Mammography screening recommendation",
+        "acronym": "CVAO",
+        "name": "STO",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 18,
             "classifiable": true,
-            "date_created": "2022-12-04T16:15:18Z",
-            "date_released": "2022-12-04T00:00:00Z",
-            "description": "This is an ontology that represents a variety of entities and classifications that share the knowledge of the recommendations provided by the guideline from the American Cancer Society on Breast Cancer Screening for Women at Average Risk.\r\n\r\nThis ontology was developed as part of an ontology course on UdelaR Univeristy. Mentored by Regina Motz and Edelweis Rohrer",
+            "date_created": "2017-11-28T12:33:04Z",
+            "date_released": "2017-11-28T00:00:00Z",
+            "description": "Stroke ontology",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MSR/1/msr.owl",
+            "download_url": "media/ontologies/CVAO/2/cvao.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5527,
+            "id": 874,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 4,
+            "max_children": 1676,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 16,
+            "nb_classes": 1712,
             "nb_inconsistent": 0,
-            "nb_individuals": 17,
-            "nb_properties": 13,
+            "nb_individuals": 69,
+            "nb_properties": 29,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.0.1"
+            "version": "STO_V1.1"
         },
         "topics": null
     },
     {
-        "acronym": "MSTDE",
-        "name": "Minimal Standard Terminology of Digestive Endoscopy",
+        "acronym": "CYTO",
+        "name": "CYTOKINE",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 0,
+            "avg_children": 158,
             "classifiable": true,
-            "date_created": "2017-02-06T21:27:00Z",
-            "date_released": "2017-02-06T21:27:00Z",
-            "description": "Metathesaurus Version of Minimal Standard Terminology Digestive Endoscopy. Bethesda, MD: National Library of Medicine, 2001.",
+            "date_created": "2015-06-24T19:16:39Z",
+            "date_released": "2015-06-24T00:00:00Z",
+            "description": "It is flat ontology showing cytokine synonyms. It has been produced by merging cytokine branches, using SOMA (Synonym Ontology Mapping Approach) method, of 10 Bioportal ontologies.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MSTDE/9/mstde.umls",
-            "has_ontology_language": "UMLS",
+            "download_url": "media/ontologies/CYTO/27/cyto.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 393,
+            "id": 268,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 315,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 1826,
+            "nb_classes": 317,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 30,
+            "nb_individuals": 1,
+            "nb_properties": 3,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 27,
             "taxon": null,
-            "version": "2016AB"
+            "version": "Cytokines"
         },
         "topics": null
     },
     {
-        "acronym": "MOP",
-        "name": "Molecular Process Ontology",
+        "acronym": "DATASET",
+        "name": "dataset",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-05-23T12:23:49.509763Z",
-            "date_released": "2022-05-23T12:23:49.509737Z",
-            "description": "Processes at the molecular level",
+            "date_created": "2019-10-11T08:01:17Z",
+            "date_released": "2019-10-11T00:00:00Z",
+            "description": "You can choose what Dataset you want",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MOP/8/mop.owl",
+            "download_url": "media/ontologies/DATASET/1/dataset.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/rsc-ontologies/rxno",
-            "id": 5086,
+            "home_page": null,
+            "id": 3005,
             "indexed": true,
-            "max_children": 168,
-            "max_depth": 13,
-            "md5sum": "ad8dbe128e9f1e78e1b70ca0c2109e2b",
-            "nb_classes": 3686,
+            "max_children": 12,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 15,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 11,
-            "products": [
-                {
-                    "id": "mop.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mop.owl",
-                    "title": "Molecular Process Ontology"
-                }
-            ],
+            "nb_properties": 2,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "MOSAIC",
-        "name": "The MOSAiC Ontology",
+        "acronym": "DATACITE",
+        "name": "SPAR DataCite Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 19,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-09-11T02:16:15Z",
-            "date_released": "2021-09-10T00:00:00Z",
-            "description": "Ontology to support Semantic Annotation of MOSAiC datasets housed in the Arctic Data Center (https://arcticdata.io).",
-            "documentation": "https://github.com/DataONEorg/sem-prov-ontologies",
+            "date_created": "2022-09-16T01:03:32Z",
+            "date_released": "2022-09-16T01:03:27Z",
+            "description": "The DataCite Ontology (DataCite) is an ontology that enables the metadata properties of the DataCite Metadata Schema Specification (i.e., a list of metadata properties for the accurate and consistent identification of a resource for citation and retrieval purposes) to be described in RDF.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MOSAIC/4/mosaic.owl",
+            "download_url": "media/ontologies/DATACITE/2/datacite.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/DataONEorg/sem-prov-ontologies",
-            "id": 4618,
+            "home_page": "http://www.sparontologies.net/ontologies/datacite",
+            "id": 5364,
             "indexed": true,
-            "max_children": 203,
-            "max_depth": 3,
+            "max_children": 5,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 242,
+            "nb_classes": 21,
             "nb_inconsistent": 0,
-            "nb_individuals": 9414,
-            "nb_properties": 36,
+            "nb_individuals": 70,
+            "nb_properties": 14,
             "products": null,
-            "publication": null,
+            "publication": "http://dx.doi.org/10.6084/m9.figshare.2075356",
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 2,
             "taxon": null,
-            "version": "Version 1.0.1"
+            "version": "1.2"
         },
         "topics": null
     },
     {
         "acronym": "MP",
         "name": "Mammalian Phenotype Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-06-26T12:34:36.563436Z",
-            "date_released": "2023-06-26T12:34:36.563416Z",
+            "date_created": "2023-07-31T12:53:37.803541Z",
+            "date_released": "2023-07-31T12:53:37.803524Z",
             "description": "Standard terms for annotating mammalian phenotypic data.",
             "documentation": null,
             "domain": "phenotype",
-            "download_url": "media/ontologies/MP/97/mp.owl",
+            "download_url": "media/ontologies/MP/98/mp.owl",
             "has_ontology_language": "OWL",
             "home_page": "http://www.informatics.jax.org/searches/MP_form.shtml",
-            "id": 5889,
+            "id": 5971,
             "indexed": true,
-            "max_children": 642,
-            "max_depth": 35,
-            "md5sum": "b01d35e08a61f9c18c5a7ef6d7043ab8",
-            "nb_classes": 33484,
+            "max_children": 498,
+            "max_depth": 39,
+            "md5sum": "afd5a618b1cda6df1eb49c7a11e44aac",
+            "nb_classes": 33685,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 312,
+            "nb_individuals": 9,
+            "nb_properties": 313,
             "products": [
                 {
                     "description": "The main ontology in OWL. Contains all MP terms and links to other OBO ontologies.",
                     "id": "mp.owl",
                     "ontology_purl": "http://purl.obolibrary.org/obo/mp.owl",
                     "page": "https://github.com/mgijax/mammalian-phenotype-ontology/releases/tag/current",
                     "title": "MP (OWL edition)"
@@ -31729,66 +31288,24 @@
             "publication": null,
             "publications": [
                 {
                     "id": "https://www.ncbi.nlm.nih.gov/pubmed/22961259",
                     "title": "The Mammalian Phenotype Ontology as a unifying standard for experimental and high-throughput phenotyping data"
                 }
             ],
-            "submission_id": 97,
+            "submission_id": 98,
             "taxon": {
                 "id": "NCBITaxon:40674",
                 "label": "Mammalia"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MPATH",
-        "name": "Mouse pathology ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2020-07-20T12:31:43.034163Z",
-            "date_released": "2020-07-20T12:31:43.034149Z",
-            "description": "A structured controlled vocabulary of mutant and transgenic mouse pathology phenotypes",
-            "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/MPATH/173/mpath.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.pathbase.net",
-            "id": 3775,
-            "indexed": true,
-            "max_children": 51,
-            "max_depth": 10,
-            "md5sum": "9197c3b190a0b804419e7d82097e22f7",
-            "nb_classes": 891,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 2,
-            "products": [
-                {
-                    "id": "mpath.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/mpath.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 173,
-            "taxon": {
-                "id": "NCBITaxon:10088",
-                "label": "Mus"
-            },
-            "version": null
-        },
-        "topics": null
-    },
-    {
         "acronym": "MPIO",
         "name": "Minimum PDDI Information Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -31990,717 +31507,680 @@
             "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MOOCULADO",
-        "name": "Molgula oculata Anatomy and Development Ontology",
+        "acronym": "MRO",
+        "name": "MHC Restriction Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 85,
+            "avg_children": 50,
             "classifiable": true,
-            "date_created": "2016-08-04T22:42:01Z",
-            "date_released": "2015-08-06T00:00:00Z",
-            "description": "The first ontology describing the anatomy and the development of Molgula oculata. ",
+            "date_created": "2023-07-10T12:36:58.633533Z",
+            "date_released": "2023-07-10T12:36:58.633515Z",
+            "description": "An ontology for Major Histocompatibility Complex (MHC) restriction in experiments",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/MOOCULADO/2/mooculado.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.aniseed.cnrs.fr/",
-            "id": 670,
+            "domain": "chemistry and biochemistry",
+            "download_url": "media/ontologies/MRO/53/mro.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/IEDB/MRO",
+            "id": 5935,
             "indexed": true,
-            "max_children": 355,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 856,
+            "max_children": 4732,
+            "max_depth": 8,
+            "md5sum": "09fcc5fc62064bca6b3938036dcdb046",
+            "nb_classes": 51390,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 7,
-            "products": null,
+            "nb_properties": 21,
+            "products": [
+                {
+                    "id": "mro.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mro.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 53,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DDPHENO",
-        "name": "Dictyostelium discoideum phenotype ontology",
+        "acronym": "MS",
+        "name": "Mass spectrometry ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 11,
             "classifiable": true,
-            "date_created": "2023-02-20T12:09:44.091071Z",
-            "date_released": "2023-02-20T12:09:44.091047Z",
-            "description": "A structured controlled vocabulary of phenotypes of the slime-mould <i>Dictyostelium discoideum</i>.",
+            "date_created": "2023-07-17T12:31:28.754196Z",
+            "date_released": "2023-07-17T12:31:28.754184Z",
+            "description": "A structured controlled vocabulary for the annotation of experiments concerned with proteomics mass spectrometry.",
             "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/DDPHENO/164/ddpheno.owl",
+            "domain": "investigations",
+            "download_url": "media/ontologies/MS/185/ms.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://dictybase.org/",
-            "id": 5661,
+            "home_page": "http://www.psidev.info/groups/controlled-vocabularies",
+            "id": 5948,
             "indexed": true,
-            "max_children": 299,
-            "max_depth": 11,
-            "md5sum": "c70007521fbd3f9c088428018bc3d798",
-            "nb_classes": 1259,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "max_children": 1350,
+            "max_depth": 37,
+            "md5sum": "47a02cb81dd3bc8f039f99b02c932b5f",
+            "nb_classes": 14964,
+            "nb_inconsistent": 16,
+            "nb_individuals": 14,
+            "nb_properties": 369,
             "products": [
                 {
-                    "id": "ddpheno.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ddpheno.owl"
+                    "id": "ms.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ms.obo"
                 },
                 {
-                    "id": "ddpheno.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ddpheno.obo"
+                    "id": "ms.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ms.owl"
                 }
             ],
             "publication": null,
             "publications": [
                 {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/31840793",
-                    "title": "dictyBase and the Dicty Stock Center (version 2.0) - a progress report"
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/23482073",
+                    "title": "The HUPO proteomics standards initiative- mass spectrometry controlled vocabulary."
                 }
             ],
-            "submission_id": 164,
-            "taxon": {
-                "id": "NCBITaxon:44689",
-                "label": "Dictyostelium discoideum"
-            },
+            "submission_id": 185,
+            "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "DEB",
-        "name": "Devices, Experimental scaffolds and Biomaterials Ontology",
+        "acronym": "MSR",
+        "name": "Mammography screening recommendation",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-06-02T17:54:15Z",
-            "date_released": "2021-06-02T00:00:00Z",
-            "description": "An ontology developed to facilitate information curation in the area of medical devices, experimental scaffolds and biomaterials. The ontology was designed to complement existing medical vocabularies, with emphasis on material processing, structures and features associated with biomaterials and experimental scaffolds. It is used as part of an automated text mining pipeline which populates the database of Experimental Biomaterials and their Biological Effect (DEBBIE). \r\nDEBBIE is an EU Horizon 2020 funded project. For more information visit https://projectdebbie.github.io/ ",
-            "documentation": "https://github.com/ProjectDebbie/Ontology_DEB",
+            "date_created": "2022-12-04T16:15:18Z",
+            "date_released": "2022-12-04T00:00:00Z",
+            "description": "This is an ontology that represents a variety of entities and classifications that share the knowledge of the recommendations provided by the guideline from the American Cancer Society on Breast Cancer Screening for Women at Average Risk.\r\n\r\nThis ontology was developed as part of an ontology course on UdelaR Univeristy. Mentored by Regina Motz and Edelweis Rohrer",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DEB/7/deb.owl",
+            "download_url": "media/ontologies/MSR/1/msr.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://projectdebbie.github.io/",
-            "id": 4638,
+            "home_page": null,
+            "id": 5527,
             "indexed": true,
-            "max_children": 56,
-            "max_depth": 5,
+            "max_children": 5,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 601,
+            "nb_classes": 16,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 12,
+            "nb_individuals": 17,
+            "nb_properties": 13,
             "products": null,
-            "publication": "https://onlinelibrary.wiley.com/doi/10.1002/adfm.201909910",
+            "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 1,
             "taxon": null,
-            "version": "4.0"
+            "version": "0.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "DELME1",
-        "name": "deleteme1",
+        "acronym": "MOP",
+        "name": "Molecular Process Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2020-03-27T06:54:48Z",
-            "date_released": "2020-03-26T00:00:00Z",
-            "description": "Test Ontology, Delete on sight",
+            "date_created": "2022-05-23T12:23:49.509763Z",
+            "date_released": "2022-05-23T12:23:49.509737Z",
+            "description": "Processes at the molecular level",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/DELME1/1/delme1.owl",
+            "download_url": "media/ontologies/MOP/8/mop.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3496,
+            "home_page": "https://github.com/rsc-ontologies/rxno",
+            "id": 5086,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 35,
+            "max_children": 168,
+            "max_depth": 13,
+            "md5sum": "ad8dbe128e9f1e78e1b70ca0c2109e2b",
+            "nb_classes": 3686,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
+            "nb_properties": 11,
+            "products": [
+                {
+                    "id": "mop.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mop.owl",
+                    "title": "Molecular Process Ontology"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 8,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "STUFF",
-        "name": "Stuff Ontology",
+        "acronym": "MPATH",
+        "name": "Mouse pathology ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 10,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-07-16T01:33:44Z",
-            "date_released": "2016-07-16T01:33:44Z",
-            "description": "a core ontology of types of macroscopic stuff (types of masses/matter)",
-            "documentation": "http://www.meteck.org/stuff.html",
-            "domain": null,
-            "download_url": "media/ontologies/STUFF/3/stuff.owl",
+            "date_created": "2020-07-20T12:31:43.034163Z",
+            "date_released": "2020-07-20T12:31:43.034149Z",
+            "description": "A structured controlled vocabulary of mutant and transgenic mouse pathology phenotypes",
+            "documentation": null,
+            "domain": "health",
+            "download_url": "media/ontologies/MPATH/173/mpath.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.meteck.org/stuff.html",
-            "id": 475,
+            "home_page": "http://www.pathbase.net",
+            "id": 3775,
             "indexed": true,
-            "max_children": 506,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 1210,
+            "max_children": 51,
+            "max_depth": 10,
+            "md5sum": "9197c3b190a0b804419e7d82097e22f7",
+            "nb_classes": 891,
             "nb_inconsistent": 0,
-            "nb_individuals": 2622,
-            "nb_properties": 72,
-            "products": null,
-            "publication": "EKAW 2014 paper \"A Core Ontology of Macroscopic Stuff\"",
+            "nb_individuals": 0,
+            "nb_properties": 2,
+            "products": [
+                {
+                    "id": "mpath.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/mpath.owl"
+                }
+            ],
+            "publication": null,
             "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "1.5"
+            "submission_id": 173,
+            "taxon": {
+                "id": "NCBITaxon:10088",
+                "label": "Mus"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MYFACILITYNAME",
-        "name": "MY FACILITY NAME",
+        "acronym": "MOOCULADO",
+        "name": "Molgula oculata Anatomy and Development Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 85,
             "classifiable": true,
-            "date_created": "2021-03-27T18:26:10Z",
-            "date_released": "2021-03-01T00:00:00Z",
-            "description": "This is the first vocabulary",
+            "date_created": "2016-08-04T22:42:01Z",
+            "date_released": "2015-08-06T00:00:00Z",
+            "description": "The first ontology describing the anatomy and the development of Molgula oculata. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MYFACILITYNAME/6/myfacilityname.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4372,
+            "download_url": "media/ontologies/MOOCULADO/2/mooculado.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.aniseed.cnrs.fr/",
+            "id": 670,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 355,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 856,
             "nb_inconsistent": 0,
-            "nb_individuals": 13,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 2,
             "taxon": null,
-            "version": "5"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ZFA",
-        "name": "Zebrafish anatomy and development ontology",
+        "acronym": "DDPHENO",
+        "name": "Dictyostelium discoideum phenotype ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-12-12T13:58:29.966560Z",
-            "date_released": "2022-12-12T13:58:29.966542Z",
-            "description": "A structured controlled vocabulary of the anatomy and development of the Zebrafish",
+            "date_created": "2023-02-20T12:09:44.091071Z",
+            "date_released": "2023-02-20T12:09:44.091047Z",
+            "description": "A structured controlled vocabulary of phenotypes of the slime-mould <i>Dictyostelium discoideum</i>.",
             "documentation": null,
             "domain": "anatomy and development",
-            "download_url": "media/ontologies/ZFA/87/zfa.owl",
+            "download_url": "media/ontologies/DDPHENO/164/ddpheno.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://wiki.zfin.org/display/general/Anatomy+Atlases+and+Resources",
-            "id": 5539,
+            "home_page": "http://dictybase.org/",
+            "id": 5661,
             "indexed": true,
-            "max_children": 234,
+            "max_children": 299,
             "max_depth": 11,
-            "md5sum": "88f85dce669a098b8ad836801eeafac4",
-            "nb_classes": 3271,
+            "md5sum": "c70007521fbd3f9c088428018bc3d798",
+            "nb_classes": 1259,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 10,
+            "nb_properties": 0,
             "products": [
                 {
-                    "id": "zfa.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/zfa.owl"
+                    "id": "ddpheno.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ddpheno.owl"
                 },
                 {
-                    "id": "zfa.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/zfa.obo"
+                    "id": "ddpheno.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ddpheno.obo"
                 }
             ],
             "publication": null,
             "publications": [
                 {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24568621",
-                    "title": "The zebrafish anatomy and stage ontologies: representing the anatomy and development of Danio rerio."
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/31840793",
+                    "title": "dictyBase and the Dicty Stock Center (version 2.0) - a progress report"
                 }
             ],
-            "submission_id": 87,
+            "submission_id": 164,
             "taxon": {
-                "id": "NCBITaxon:7954",
-                "label": "Danio"
+                "id": "NCBITaxon:44689",
+                "label": "Dictyostelium discoideum"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NAMBOBI",
-        "name": "NAMBOBI'S ONTOLOGY",
+        "acronym": "DEB",
+        "name": "Devices, Experimental scaffolds and Biomaterials Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2021-05-24T13:35:45Z",
-            "date_released": "2021-05-24T00:00:00Z",
-            "description": "OUT PATIENT REGISTER FOR UGANDA",
-            "documentation": null,
+            "date_created": "2021-06-02T17:54:15Z",
+            "date_released": "2021-06-02T00:00:00Z",
+            "description": "An ontology developed to facilitate information curation in the area of medical devices, experimental scaffolds and biomaterials. The ontology was designed to complement existing medical vocabularies, with emphasis on material processing, structures and features associated with biomaterials and experimental scaffolds. It is used as part of an automated text mining pipeline which populates the database of Experimental Biomaterials and their Biological Effect (DEBBIE). \r\nDEBBIE is an EU Horizon 2020 funded project. For more information visit https://projectdebbie.github.io/ ",
+            "documentation": "https://github.com/ProjectDebbie/Ontology_DEB",
             "domain": null,
-            "download_url": "media/ontologies/NAMBOBI/1/nambobi.owl",
+            "download_url": "media/ontologies/DEB/7/deb.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4494,
+            "home_page": "https://projectdebbie.github.io/",
+            "id": 4638,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 56,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 601,
             "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 12,
             "products": null,
-            "publication": null,
+            "publication": "https://onlinelibrary.wiley.com/doi/10.1002/adfm.201909910",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 7,
             "taxon": null,
-            "version": null
+            "version": "4.0"
         },
         "topics": null
     },
     {
-        "acronym": "NANDO",
-        "name": "Nanbyo Disease Ontology",
+        "acronym": "DLO",
+        "name": "Deep Learning Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-07-28T00:40:36Z",
+            "date_created": "2021-07-27T21:03:23Z",
             "date_released": "2021-07-23T00:00:00Z",
-            "description": "Creating a comprehensive hierarchical controlled vocabulary for intractable and rare disease (i.e., nanbyo) representation in Japan.\r\n",
-            "documentation": "https://nanbyodata.jp",
+            "description": "Taxonomy of Deep Learning networks and layers. TODO: partonomy",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NANDO/14/nando.owl",
+            "download_url": "media/ontologies/DLO/2/dlo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://nanbyodata.jp",
-            "id": 4589,
+            "home_page": "https://github.com/turbomam/deep-learning-ontology",
+            "id": 4637,
             "indexed": true,
-            "max_children": 83,
-            "max_depth": 7,
+            "max_children": 16,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 2732,
+            "nb_classes": 61,
             "nb_inconsistent": 0,
-            "nb_individuals": 2410,
-            "nb_properties": 1,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "2021-07-27"
         },
         "topics": null
     },
     {
-        "acronym": "NCCO",
-        "name": "Nursing Care Coordination Ontology",
+        "acronym": "MSTDE",
+        "name": "Minimal Standard Terminology of Digestive Endoscopy",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2013-11-06T03:01:55Z",
-            "date_released": "2013-11-05T00:00:00Z",
-            "description": "Nursing Care Coordination Ontology contains activities that nurses use while coordinating care among patients.",
+            "date_created": "2017-02-06T21:27:00Z",
+            "date_released": "2017-02-06T21:27:00Z",
+            "description": "Metathesaurus Version of Minimal Standard Terminology Digestive Endoscopy. Bethesda, MD: National Library of Medicine, 2001.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NCCO/1/ncco.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/MSTDE/9/mstde.umls",
+            "has_ontology_language": "UMLS",
             "home_page": null,
-            "id": 700,
+            "id": 393,
             "indexed": true,
-            "max_children": 27,
-            "max_depth": 6,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 400,
+            "nb_classes": 1826,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 30,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 9,
             "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "EHDAA2",
-        "name": "Human developmental anatomy, abstract",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 19,
-            "classifiable": true,
-            "date_created": "2017-12-13T00:50:09.444943Z",
-            "date_released": "2017-12-13T00:50:09.444922Z",
-            "description": "A structured controlled vocabulary of stage-specific anatomical structures of the developing human.",
-            "documentation": null,
-            "domain": "anatomy",
-            "download_url": "media/ontologies/EHDAA2/33/ehdaa2.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://genex.hgu.mrc.ac.uk/",
-            "id": 988,
-            "indexed": true,
-            "max_children": 361,
-            "max_depth": 12,
-            "md5sum": "5e116ce6bac13c46b11b1fa36e0a089f",
-            "nb_classes": 2734,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 9,
-            "products": [
-                {
-                    "id": "ehdaa2.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ehdaa2.owl"
-                },
-                {
-                    "id": "ehdaa2.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ehdaa2.obo"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/22973865",
-                    "title": "A new ontology (structured hierarchy) of human developmental anatomy for the first 7 weeks (Carnegie stages 1-20)."
-                }
-            ],
-            "submission_id": 33,
-            "taxon": {
-                "id": "NCBITaxon:9606",
-                "label": "Homo sapiens"
-            },
-            "version": null
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "ELD",
-        "name": "EthiopianListOfDiseases",
+        "acronym": "DELME1",
+        "name": "deleteme1",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-11-01T10:17:06Z",
-            "date_released": "2021-07-23T00:00:00Z",
-            "description": "This is the list of Ethiopian Diseases approved by MOH Ethiopia",
+            "date_created": "2020-03-27T06:54:48Z",
+            "date_released": "2020-03-26T00:00:00Z",
+            "description": "Test Ontology, Delete on sight",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ELD/5/eld.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/DELME1/1/delme1.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4666,
+            "id": 3496,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 4,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 35,
             "nb_inconsistent": 0,
-            "nb_individuals": 2077,
+            "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 1,
             "taxon": null,
-            "version": "4"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ELIG",
-        "name": "Eligibility Feature Hierarchy",
+        "acronym": "DFO",
+        "name": "Databank Family Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2013-05-07T20:09:37Z",
-            "date_released": "2013-05-07T07:00:00Z",
-            "description": "We derived a set of 1,437 eligibility features that were organized into a feature hierarchy using 80 breast cancer trials. ",
-            "documentation": null,
+            "date_created": "2021-10-08T12:09:54Z",
+            "date_released": "2021-10-08T00:00:00Z",
+            "description": "The Databank Family Ontology describes groups of data banks with similar record prompts and originators.",
+            "documentation": "https://github.com/molgenis/ontologies/tree/main/dfo",
             "domain": null,
-            "download_url": "media/ontologies/ELIG/1/elig.owl",
+            "download_url": "media/ontologies/DFO/1/dfo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 579,
+            "home_page": "https://w3id.org/dfo",
+            "id": 4582,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 5,
+            "max_children": 10,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 29,
+            "nb_classes": 12,
             "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 3,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "forthcoming",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "v. 1.0"
+            "version": "0.1"
         },
         "topics": null
     },
     {
-        "acronym": "NCI",
-        "name": "human anatomy",
+        "acronym": "DERMLEX",
+        "name": "Dermatology Lexicon",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2018-02-16T01:04:46Z",
-            "date_released": "2018-02-15T00:00:00Z",
-            "description": "human anatomy",
+            "date_created": "2009-04-17T19:12:06Z",
+            "date_released": "2009-04-30T07:00:00Z",
+            "description": "Version 1.0 of the American Academy of Dermatology's DermLex: The Dermatology Lexicon, a standardized terminology of dermatologic diagnoses, therapies, procedures, and laboratory tests.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NCI/1/nci.owl",
+            "download_url": "media/ontologies/DERMLEX/4/dermlex.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 1243,
+            "home_page": "http://www.aad.org/research/lexicon/",
+            "id": 462,
             "indexed": true,
-            "max_children": 125,
-            "max_depth": 11,
+            "max_children": 164,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 3304,
+            "nb_classes": 6106,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_individuals": 6101,
+            "nb_properties": 18,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": null
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ELTER_CL",
-        "name": "eLTER Controlled Lists",
+        "acronym": "DERMO",
+        "name": "Human Dermatological Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-07-15T01:05:03Z",
-            "date_released": "2023-07-15T01:05:00Z",
-            "description": "eLTER Controlled Lists are used in different environmental projects related to the eLTER community.",
+            "date_created": "2015-10-13T01:01:39Z",
+            "date_released": "2015-10-13T01:01:38Z",
+            "description": "Ontology of human dermatologic disease",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ELTER_CL/35/elter_cl.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/DERMO/10/dermo.obo",
+            "has_ontology_language": "OBO",
             "home_page": null,
-            "id": 5956,
+            "id": 749,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 25,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 3521,
             "nb_inconsistent": 0,
-            "nb_individuals": 699,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 4,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 35,
+            "submission_id": 10,
             "taxon": null,
-            "version": "1.0.8"
+            "version": "1.19"
         },
         "topics": null
     },
     {
-        "acronym": "NCBI_NMOsp_1_0_0",
-        "name": "NeuroMorpho.Org Species Ontology v.1.0.0",
+        "acronym": "DIAGONT",
+        "name": "Diagnostic Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2010-06-14T02:47:12Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "A lighter version of the NCBI taxonomy that is customized for the NeuroMorpho.Org species.",
+            "date_created": "2012-02-27T21:40:30Z",
+            "date_released": "2011-12-15T08:00:00Z",
+            "description": "We built a computational diagnostic ontology containing 91 elements, including classes and sub-classes, which are required to conduct SR-MA (Systematic Review - Meta Analysis) for diagnostic studies, that will assist in standardized reporting of diagnostic articles. We also report high percentage of agreement among five observers as a result of the inter-observer agreement that we conducted among them to tag 13 articles using the diagnostic ontology. Moreover, we extend our existing repository CERR-N (Center of Excellence in Research Reporting in Neurosurgery) to include diagnostic studies.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NCBI_NMOsp_1_0_0/1/ncbi_nmosp_1_0_0.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "",
-            "id": 529,
+            "download_url": "media/ontologies/DIAGONT/1/diagont.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 650,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 36,
+            "max_children": 9,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 438,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_classes": 96,
+            "nb_inconsistent": 12,
+            "nb_individuals": 6,
+            "nb_properties": 9,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "NCBI_NMOsp_1_2_2",
-        "name": "NeuroMorpho.Org Species Ontology v.1.2.2",
+        "acronym": "DISDRIV",
+        "name": "Disease Drivers Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2010-08-16T01:31:08Z",
-            "date_released": "2010-08-15T07:00:00Z",
-            "description": "A lighter version of the NCBI taxonomy that is customized for the NeuroMorpho.Org species. The ontology also include NIFSTD, Rat strain and Jax mice terms that are arranged in a single parent hierarchical tree.",
+            "date_created": "2021-11-22T12:21:05.058461Z",
+            "date_released": "2021-11-22T12:21:05.058450Z",
+            "description": "Ontology for drivers and triggers of human diseases, built to classify ExO ontology exposure stressors. An application ontology. Built in collaboration with EnvO, ExO, ECTO and ChEBI.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NCBI_NMOsp_1_2_2/1/ncbi_nmosp_1_2_2.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "",
-            "id": 490,
+            "domain": "disease",
+            "download_url": "media/ontologies/DISDRIV/1/disdriv.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.disease-ontology.org",
+            "id": 4702,
             "indexed": true,
-            "max_children": 410,
-            "max_depth": 36,
-            "md5sum": null,
-            "nb_classes": 738,
+            "max_children": 5,
+            "max_depth": 4,
+            "md5sum": "57619032fc0d43b45fe7b2951c8ee8e2",
+            "nb_classes": 13,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 1,
-            "products": null,
+            "nb_properties": 3,
+            "products": [
+                {
+                    "id": "disdriv.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/disdriv.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
             "submission_id": 1,
-            "taxon": null,
-            "version": "1.2.2"
+            "taxon": {
+                "id": "NCBITaxon:9606",
+                "label": "Homo sapiens"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NATPRO",
-        "name": "Natural Products Ontology",
+        "acronym": "MOSAIC",
+        "name": "The MOSAiC Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 19,
             "classifiable": true,
-            "date_created": "2012-02-08T11:39:01Z",
-            "date_released": "2012-02-08T08:00:00Z",
-            "description": "An ontology for describing biological activities of natural products.",
-            "documentation": null,
+            "date_created": "2021-09-11T02:16:15Z",
+            "date_released": "2021-09-10T00:00:00Z",
+            "description": "Ontology to support Semantic Annotation of MOSAiC datasets housed in the Arctic Data Center (https://arcticdata.io).",
+            "documentation": "https://github.com/DataONEorg/sem-prov-ontologies",
             "domain": null,
-            "download_url": "media/ontologies/NATPRO/1/natpro.owl",
+            "download_url": "media/ontologies/MOSAIC/4/mosaic.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 736,
+            "home_page": "https://github.com/DataONEorg/sem-prov-ontologies",
+            "id": 4618,
             "indexed": true,
-            "max_children": 204,
-            "max_depth": 19,
+            "max_children": 203,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 9465,
+            "nb_classes": 242,
             "nb_inconsistent": 0,
-            "nb_individuals": 22012,
-            "nb_properties": 64,
+            "nb_individuals": 9414,
+            "nb_properties": 36,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "unknown"
+            "version": "Version 1.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "NBO",
-        "name": "Neuro Behavior Ontology",
+        "acronym": "STUFF",
+        "name": "Stuff Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 10,
             "classifiable": true,
-            "date_created": "2023-07-10T12:45:42.038387Z",
-            "date_released": "2023-07-10T12:45:42.038374Z",
-            "description": "An ontology of human and animal behaviours and behavioural phenotypes",
-            "documentation": null,
-            "domain": "biological systems",
-            "download_url": "media/ontologies/NBO/38/nbo.owl",
+            "date_created": "2016-07-16T01:33:44Z",
+            "date_released": "2016-07-16T01:33:44Z",
+            "description": "a core ontology of types of macroscopic stuff (types of masses/matter)",
+            "documentation": "http://www.meteck.org/stuff.html",
+            "domain": null,
+            "download_url": "media/ontologies/STUFF/3/stuff.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obo-behavior/behavior-ontology/",
-            "id": 5936,
+            "home_page": "http://www.meteck.org/stuff.html",
+            "id": 475,
             "indexed": true,
-            "max_children": 106,
-            "max_depth": 35,
-            "md5sum": "7236c44b04d61e3cdb1b52a0aa0803c7",
-            "nb_classes": 4548,
+            "max_children": 506,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 1210,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 239,
-            "products": [
-                {
-                    "id": "nbo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/nbo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24177753",
-                    "title": "Analyzing gene expression data in mice with the Neuro Behavior Ontology"
-                }
-            ],
-            "submission_id": 38,
+            "nb_individuals": 2622,
+            "nb_properties": 72,
+            "products": null,
+            "publication": "EKAW 2014 paper \"A Core Ontology of Macroscopic Stuff\"",
+            "publications": null,
+            "submission_id": 3,
             "taxon": null,
-            "version": null
+            "version": "1.5"
         },
         "topics": null
     },
     {
         "acronym": "XPO",
         "name": "Xenopus Phenotype Ontology",
         "species": null,
@@ -32748,118 +32228,14 @@
                 "label": "Xenopus"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NDDO",
-        "name": "Neurodegenerative Disease Data Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 6,
-            "classifiable": true,
-            "date_created": "2019-06-04T01:08:58Z",
-            "date_released": "2019-06-04T01:08:49Z",
-            "description": "Neurodegenerative Disease Data Ontology (NDDO) is an ontology for describing data on neurodegenerative disease patients.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/NDDO/10/nddo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.ontodm.com/doku.php?id=nddo",
-            "id": 2617,
-            "indexed": true,
-            "max_children": 143,
-            "max_depth": 13,
-            "md5sum": null,
-            "nb_classes": 1731,
-            "nb_inconsistent": 0,
-            "nb_individuals": 53,
-            "nb_properties": 100,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 10,
-            "taxon": null,
-            "version": "0.2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ENM",
-        "name": "eNanoMapper",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2019-11-30T14:14:21.900145Z",
-            "date_released": "2019-11-30T14:14:21.900195Z",
-            "description": "",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ENM/17/enm.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3140,
-            "indexed": true,
-            "max_children": 1427,
-            "max_depth": 12,
-            "md5sum": null,
-            "nb_classes": 12751,
-            "nb_inconsistent": 0,
-            "nb_individuals": 464,
-            "nb_properties": 5,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 17,
-            "taxon": null,
-            "version": "6.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ENMT",
-        "name": "eNanoMapper",
-        "species": [],
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2019-11-30T13:22:16.788076Z",
-            "date_released": "2019-11-30T13:22:16.788174Z",
-            "description": "The eNanoMapper ontology covers the full scope of terminology needed to support research into nanomaterial safety. It builds on multiple pre-existing external ontologies such as the NanoParticle Ontology.",
-            "documentation": "http://enanomapper.net/ontology",
-            "domain": null,
-            "download_url": "media/ontologies/ENMT/1/enmt.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://enanomapper.net/",
-            "id": 3139,
-            "indexed": true,
-            "max_children": 1427,
-            "max_depth": 12,
-            "md5sum": null,
-            "nb_classes": 12751,
-            "nb_inconsistent": 0,
-            "nb_individuals": 464,
-            "nb_properties": 5,
-            "products": null,
-            "publication": "http://enanomapper.net/library",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "6.0"
-        },
-        "topics": [
-            "nanomaterials"
-        ]
-    },
-    {
         "acronym": "MWLA",
         "name": "Medical Web Lifestyle Aggregator",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 16,
             "classifiable": true,
@@ -32988,14 +32364,58 @@
             "submission_id": 2,
             "taxon": null,
             "version": "1.2"
         },
         "topics": null
     },
     {
+        "acronym": "NBO",
+        "name": "Neuro Behavior Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2023-07-10T12:45:42.038387Z",
+            "date_released": "2023-07-10T12:45:42.038374Z",
+            "description": "An ontology of human and animal behaviours and behavioural phenotypes",
+            "documentation": null,
+            "domain": "biological systems",
+            "download_url": "media/ontologies/NBO/38/nbo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obo-behavior/behavior-ontology/",
+            "id": 5936,
+            "indexed": true,
+            "max_children": 106,
+            "max_depth": 35,
+            "md5sum": "7236c44b04d61e3cdb1b52a0aa0803c7",
+            "nb_classes": 4548,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 239,
+            "products": [
+                {
+                    "id": "nbo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/nbo.owl"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24177753",
+                    "title": "Analyzing gene expression data in mice with the Neuro Behavior Ontology"
+                }
+            ],
+            "submission_id": 38,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "NCBITAXON",
         "name": "NCBI organismal classification",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 11,
             "classifiable": true,
@@ -33045,46 +32465,114 @@
             "submission_id": 16,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NCBITAXONSH",
-        "name": "NCBI Taxonomy Ontology - Shrinked version",
-        "species": [],
+        "acronym": "NAMBOBI",
+        "name": "NAMBOBI'S ONTOLOGY",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 149,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2018-10-08T08:35:17.291706Z",
-            "date_released": "2018-10-08T08:35:17.291738Z",
-            "description": "",
+            "date_created": "2021-05-24T13:35:45Z",
+            "date_released": "2021-05-24T00:00:00Z",
+            "description": "OUT PATIENT REGISTER FOR UGANDA",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NCBITAXONSH/3/ncbitaxonsh.owl",
+            "download_url": "media/ontologies/NAMBOBI/1/nambobi.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 1881,
+            "id": 4494,
             "indexed": true,
-            "max_children": 33808,
-            "max_depth": 35,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 356489,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 9,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NANDO",
+        "name": "Nanbyo Disease Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2021-07-28T00:40:36Z",
+            "date_released": "2021-07-23T00:00:00Z",
+            "description": "Creating a comprehensive hierarchical controlled vocabulary for intractable and rare disease (i.e., nanbyo) representation in Japan.\r\n",
+            "documentation": "https://nanbyodata.jp",
+            "domain": null,
+            "download_url": "media/ontologies/NANDO/14/nando.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://nanbyodata.jp",
+            "id": 4589,
+            "indexed": true,
+            "max_children": 83,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 2732,
+            "nb_inconsistent": 0,
+            "nb_individuals": 2410,
+            "nb_properties": 1,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 14,
+            "taxon": null,
+            "version": "1.0.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NCCO",
+        "name": "Nursing Care Coordination Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2013-11-06T03:01:55Z",
+            "date_released": "2013-11-05T00:00:00Z",
+            "description": "Nursing Care Coordination Ontology contains activities that nurses use while coordinating care among patients.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/NCCO/1/ncco.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 700,
+            "indexed": true,
+            "max_children": 27,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 400,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.3"
+            "version": "1.0"
         },
-        "topics": []
+        "topics": null
     },
     {
         "acronym": "NCCNEHR",
         "name": "NCCN EHR Oncology Categories",
         "species": null,
         "status": "Classified",
         "submission": {
@@ -33312,550 +32800,522 @@
             "submission_id": 1,
             "taxon": null,
             "version": "Version1"
         },
         "topics": null
     },
     {
-        "acronym": "TCO",
-        "name": "Thyroid Cancer Ontology",
+        "acronym": "MYFACILITYNAME",
+        "name": "MY FACILITY NAME",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 19,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2020-07-03T01:27:51Z",
-            "date_released": "2020-07-02T00:00:00Z",
-            "description": "Thyroid cancer ontology (TCO) contains 578 concepts organized under 6 biological viewpoints, i.e., Anatomy and Histoembryology, Basic Information of Patient, Cellular or Molecular Interactions, Clinical Aspects, Etiology, Pathology",
+            "date_created": "2021-03-27T18:26:10Z",
+            "date_released": "2021-03-01T00:00:00Z",
+            "description": "This is the first vocabulary",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TCO/2/tco.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/MYFACILITYNAME/6/myfacilityname.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 3743,
+            "id": 4372,
             "indexed": true,
-            "max_children": 510,
-            "max_depth": 7,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 578,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 3,
+            "nb_individuals": 13,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 6,
             "taxon": null,
-            "version": "1.0"
+            "version": "5"
         },
         "topics": null
     },
     {
-        "acronym": "TDWGSPEC",
-        "name": "TDWG Specimen LSID Ontology",
+        "acronym": "ZFA",
+        "name": "Zebrafish anatomy and development ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2017-02-16T23:29:08Z",
-            "date_released": "2015-07-01T00:00:00Z",
-            "description": "Biodiversity Information Standards (TDWG) is an international not-for-profit group that develops standards and protocols for sharing biodiversity data. They have a detailed collection of vocabularies and ontologies for studies of biological organisms.",
+            "date_created": "2022-12-12T13:58:29.966560Z",
+            "date_released": "2022-12-12T13:58:29.966542Z",
+            "description": "A structured controlled vocabulary of the anatomy and development of the Zebrafish",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/TDWGSPEC/1/tdwgspec.owl",
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/ZFA/87/zfa.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 562,
+            "home_page": "https://wiki.zfin.org/display/general/Anatomy+Atlases+and+Resources",
+            "id": 5539,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 9,
+            "max_children": 234,
+            "max_depth": 11,
+            "md5sum": "88f85dce669a098b8ad836801eeafac4",
+            "nb_classes": 3271,
             "nb_inconsistent": 0,
-            "nb_individuals": 7,
-            "nb_properties": 31,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 10,
+            "products": [
+                {
+                    "id": "zfa.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/zfa.owl"
+                },
+                {
+                    "id": "zfa.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/zfa.obo"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "0.3"
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/24568621",
+                    "title": "The zebrafish anatomy and stage ontologies: representing the anatomy and development of Danio rerio."
+                }
+            ],
+            "submission_id": 87,
+            "taxon": {
+                "id": "NCBITaxon:7954",
+                "label": "Danio"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "FAST-TOPICAL",
-        "name": "FAST (Faceted Application of Subject Terminology) Topical Facet",
+        "acronym": "NCI",
+        "name": "human anatomy",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2017-09-06T20:47:53Z",
-            "date_released": "2017-04-14T00:00:00Z",
-            "description": "AST is an enumerative, faceted subject heading schema derived from the Library of Congress Subject Headings (LCSH). The purpose of adapting the LCSH with a simplified syntax to create FAST is to retain the very rich vocabulary of LCSH while making the schema easier to understand, control, apply, and use.",
+            "date_created": "2018-02-16T01:04:46Z",
+            "date_released": "2018-02-15T00:00:00Z",
+            "description": "human anatomy",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FAST-TOPICAL/1/fast-topical.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/NCI/1/nci.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 411,
+            "id": 1243,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 0,
+            "max_children": 125,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 3,
+            "nb_classes": 3304,
             "nb_inconsistent": 0,
-            "nb_individuals": 583314,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 2,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "EPIO",
-        "name": "Epilepsy Ontology",
+        "acronym": "ELTER_CL",
+        "name": "eLTER Controlled Lists",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-12-26T12:19:57.770929Z",
-            "date_released": "2022-12-26T12:19:57.770900Z",
-            "description": "A application driven Epilepsy Ontology with official terms from the ILAE.",
+            "date_created": "2023-07-15T01:05:03Z",
+            "date_released": "2023-07-15T01:05:00Z",
+            "description": "eLTER Controlled Lists are used in different environmental projects related to the eLTER community.",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/EPIO/3/epio.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/SCAI-BIO/EpilepsyOntology",
-            "id": 5581,
+            "domain": null,
+            "download_url": "media/ontologies/ELTER_CL/35/elter_cl.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5956,
             "indexed": true,
-            "max_children": 41,
-            "max_depth": 16,
-            "md5sum": "34876cf99fbb5d7a831ddc86f0d0d93e",
-            "nb_classes": 1916,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 12,
-            "products": [
-                {
-                    "id": "epio.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/epio.owl"
-                },
-                {
-                    "id": "EPIO_merged.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/EPIO_merged.owl"
-                }
-            ],
+            "nb_individuals": 699,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 35,
             "taxon": null,
-            "version": null
+            "version": "1.0.8"
         },
         "topics": null
     },
     {
-        "acronym": "FALDO",
-        "name": "Feature Annotation Location Description Ontology ",
+        "acronym": "NCBI_NMOsp_1_0_0",
+        "name": "NeuroMorpho.Org Species Ontology v.1.0.0",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-11-03T15:46:37Z",
-            "date_released": "2015-06-20T00:00:00Z",
-            "description": "FALDO is the Feature Annotation Location Description Ontology. It is a simple ontology to describe sequence feature positions and regions as found in GFF3, DBBJ, EMBL, GenBank files, UniProt, and many other bioinformatics resources.\r\n\r\nThe aim of this ontology is to describe the position of a sequence region or a feature. It does not aim to describe features or regions itself, but instead depends on resources such as the Sequence Ontology or the UniProt core ontolgy.",
-            "documentation": "https://github.com/OBF/FALDO",
+            "date_created": "2010-06-14T02:47:12Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "A lighter version of the NCBI taxonomy that is customized for the NeuroMorpho.Org species.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FALDO/1/faldo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://biohackathon.org/resource/faldo",
-            "id": 783,
+            "download_url": "media/ontologies/NCBI_NMOsp_1_0_0/1/ncbi_nmosp_1_0_0.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "",
+            "id": 529,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
+            "max_children": 51,
+            "max_depth": 36,
             "md5sum": null,
-            "nb_classes": 19,
+            "nb_classes": 438,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 10,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0067-z",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "Created at the Biohackathon 2012 and 2013"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "NERO",
-        "name": "Named Entity Recognition Ontology ",
+        "acronym": "NCBI_NMOsp_1_2_2",
+        "name": "NeuroMorpho.Org Species Ontology v.1.2.2",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2021-04-28T17:37:55Z",
-            "date_released": "2021-04-28T00:00:00Z",
-            "description": "Named Entity Recognition Ontology (NERO) developed specifically for describing textual entities in biomedical texts, which accounts for diverse levels of ambiguity, bridging the scientific sublanguages of molecular biology, genetics, biochemistry, and medicine. The annotation of natural science texts is more challenging than in other domains. Biomedical language is replete with ambiguity distinct from that observed in news articles or informal text online. In biomedical texts, alternative meanings are not always clearly separated.\r\n\r\nFor example, in some biomedical contexts, the words for a named entity may refer to a gene or a protein with nearly equal probability; for example, \u201ca mutant hemoglobin \u03b1_2\u201d can refer to either a gene or a protein. If the author meant gene-or-protein A, and we force an annotator to choose either interpretation gene A or protein A, the resulting annotation is of limited utility because the choice between gene and protein is random if the meanings are equally likely based on context. NERO attempts to minimize unwarranted, arbitrary annotative semantic label assignments for textual entities. \r\n\r\nNERO defines ambiguous concepts, such as GeneOrProtein, which subsumes both Gene and Protein using the following axiom: EquivalentTo: \u2018Gene\u2019 or \u2018Protein.\u2019 There are no biological entities that are either a gene or a protein, but there are lexical entities that can correspond to either or both of these entities.\r\n",
+            "date_created": "2010-08-16T01:31:08Z",
+            "date_released": "2010-08-15T07:00:00Z",
+            "description": "A lighter version of the NCBI taxonomy that is customized for the NeuroMorpho.Org species. The ontology also include NIFSTD, Rat strain and Jax mice terms that are arranged in a single parent hierarchical tree.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NERO/1/nero.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4453,
+            "download_url": "media/ontologies/NCBI_NMOsp_1_2_2/1/ncbi_nmosp_1_2_2.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "",
+            "id": 490,
             "indexed": true,
-            "max_children": 21,
-            "max_depth": 6,
+            "max_children": 410,
+            "max_depth": 36,
             "md5sum": null,
-            "nb_classes": 63,
+            "nb_classes": 738,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 27,
+            "nb_properties": 1,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "The main developer of NERO is Prof. Robert Stevens, the University of Manchester"
+            "version": "1.2.2"
         },
         "topics": null
     },
     {
-        "acronym": "NEST",
-        "name": "Nutritional Epidemiological STandards",
-        "species": null,
+        "acronym": "NCBITAXONSH",
+        "name": "NCBI Taxonomy Ontology - Shrinked version",
+        "species": [],
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 149,
             "classifiable": true,
-            "date_created": "2018-08-27T09:13:11Z",
-            "date_released": "2018-08-20T00:00:00Z",
-            "description": "It represents standards for nutritional epidemiological studies in ontology format. In the strict sense, it is not an ontology. Its hierarchy follows structure of each standard, not OWL rules.",
+            "date_created": "2018-10-08T08:35:17.291706Z",
+            "date_released": "2018-10-08T08:35:17.291738Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NEST/5/nest.owl",
+            "download_url": "media/ontologies/NCBITAXONSH/3/ncbitaxonsh.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/cyang0128/Nutritional-epidemiological-ontologies",
-            "id": 1778,
+            "home_page": null,
+            "id": 1881,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 7,
+            "max_children": 33808,
+            "max_depth": 35,
             "md5sum": null,
-            "nb_classes": 249,
+            "nb_classes": 356489,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
-            "publication": "https://www.ncbi.nlm.nih.gov/pubmed/28916566",
+            "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 3,
             "taxon": null,
-            "version": "V1.2"
+            "version": "0.3"
         },
-        "topics": null
+        "topics": []
     },
     {
-        "acronym": "T123",
-        "name": "Test123",
-        "species": [],
+        "acronym": "NATPRO",
+        "name": "Natural Products Ontology",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-02-08T09:41:23.498758Z",
-            "date_released": "2023-02-08T09:41:23.498777Z",
-            "description": "",
+            "date_created": "2012-02-08T11:39:01Z",
+            "date_released": "2012-02-08T08:00:00Z",
+            "description": "An ontology for describing biological activities of natural products.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/T123/1/t123.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/NATPRO/1/natpro.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5648,
+            "id": 736,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 204,
+            "max_depth": 19,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 9465,
             "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 0,
+            "nb_individuals": 22012,
+            "nb_properties": 64,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.1.0"
+            "version": "unknown"
         },
-        "topics": []
+        "topics": null
     },
     {
-        "acronym": "TAXRANK",
-        "name": "Taxonomic rank vocabulary",
+        "acronym": "NDDO",
+        "name": "Neurodegenerative Disease Data Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 30,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2017-12-13T11:31:32.307539Z",
-            "date_released": "2017-12-13T11:31:32.307517Z",
-            "description": "A vocabulary of taxonomic ranks (species, family, phylum, etc)",
+            "date_created": "2019-06-04T01:08:58Z",
+            "date_released": "2019-06-04T01:08:49Z",
+            "description": "Neurodegenerative Disease Data Ontology (NDDO) is an ontology for describing data on neurodegenerative disease patients.",
             "documentation": null,
-            "domain": "taxonomy",
-            "download_url": "media/ontologies/TAXRANK/10/taxrank.owl",
+            "domain": null,
+            "download_url": "media/ontologies/NDDO/10/nddo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.phenoscape.org/wiki/Taxonomic_Rank_Vocabulary",
-            "id": 1071,
+            "home_page": "http://www.ontodm.com/doku.php?id=nddo",
+            "id": 2617,
             "indexed": true,
-            "max_children": 60,
-            "max_depth": 2,
-            "md5sum": "b5540d3672dcd2d46c5286fc790a52d9",
-            "nb_classes": 62,
+            "max_children": 143,
+            "max_depth": 13,
+            "md5sum": null,
+            "nb_classes": 1731,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": [
-                {
-                    "id": "taxrank.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/taxrank.owl"
-                },
-                {
-                    "id": "taxrank.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/taxrank.obo"
-                }
-            ],
+            "nb_individuals": 53,
+            "nb_properties": 100,
+            "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 10,
             "taxon": null,
-            "version": null
+            "version": "0.2"
         },
         "topics": null
     },
     {
-        "acronym": "FAO",
-        "name": "Fungal gross anatomy",
+        "acronym": "EHDAA2",
+        "name": "Human developmental anatomy, abstract",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 19,
             "classifiable": true,
-            "date_created": "2020-05-11T15:15:58.939049Z",
-            "date_released": "2020-05-11T15:15:58.939040Z",
-            "description": "A structured controlled vocabulary for the anatomy of fungi.",
+            "date_created": "2017-12-13T00:50:09.444943Z",
+            "date_released": "2017-12-13T00:50:09.444922Z",
+            "description": "A structured controlled vocabulary of stage-specific anatomical structures of the developing human.",
             "documentation": null,
             "domain": "anatomy",
-            "download_url": "media/ontologies/FAO/28/fao.owl",
+            "download_url": "media/ontologies/EHDAA2/33/ehdaa2.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/fungal-anatomy-ontology/",
-            "id": 3593,
+            "home_page": "http://genex.hgu.mrc.ac.uk/",
+            "id": 988,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 8,
-            "md5sum": "7484178e9244528982598c717ff00236",
-            "nb_classes": 120,
+            "max_children": 361,
+            "max_depth": 12,
+            "md5sum": "5e116ce6bac13c46b11b1fa36e0a089f",
+            "nb_classes": 2734,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 3,
+            "nb_properties": 9,
             "products": [
                 {
-                    "id": "fao.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fao.owl"
+                    "id": "ehdaa2.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ehdaa2.owl"
                 },
                 {
-                    "id": "fao.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fao.obo"
+                    "id": "ehdaa2.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ehdaa2.obo"
                 }
             ],
             "publication": null,
-            "publications": null,
-            "submission_id": 28,
+            "publications": [
+                {
+                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/22973865",
+                    "title": "A new ontology (structured hierarchy) of human developmental anatomy for the first 7 weeks (Carnegie stages 1-20)."
+                }
+            ],
+            "submission_id": 33,
             "taxon": {
-                "id": "NCBITaxon:4751",
-                "label": "Fungal"
+                "id": "NCBITaxon:9606",
+                "label": "Homo sapiens"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NMOBR",
-        "name": "NeuroMorpho.Org brain region ontologies",
+        "acronym": "ENM",
+        "name": "eNanoMapper",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 5,
             "classifiable": true,
-            "date_created": "2017-11-28T02:12:08Z",
-            "date_released": "2017-11-28T02:12:08Z",
-            "description": "The species, brain regions and cell types are integrated into a single ontology for enabling OntoSearch functionality to mine the latest v7.0 release of NeuroMorpho.Org.",
+            "date_created": "2019-11-30T14:14:21.900145Z",
+            "date_released": "2019-11-30T14:14:21.900195Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NMOBR/32/nmobr.owl",
+            "download_url": "media/ontologies/ENM/17/enm.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 879,
-            "indexed": true,
-            "max_children": 408,
-            "max_depth": 40,
-            "md5sum": null,
-            "nb_classes": 3250,
-            "nb_inconsistent": 0,
-            "nb_individuals": 1968,
-            "nb_properties": 2,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 32,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FAST-CHRONO",
-        "name": "FAST (Faceted Access of Subject Terminology) Chronological Facet",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2018-05-07T23:55:15Z",
-            "date_released": "2018-05-07T00:00:00Z",
-            "description": "FAST is an enumerative, faceted subject heading schema derived from the Library of Congress Subject Headings (LCSH). The purpose of adapting the LCSH with a simplified syntax to create FAST is to retain the very rich vocabulary of LCSH while making the schema easier to understand, control, apply, and use. This facet includes chronological terms derived from LCSH.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/FAST-CHRONO/1/fast-chrono.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 1477,
+            "id": 3140,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
+            "max_children": 1427,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 3,
+            "nb_classes": 12751,
             "nb_inconsistent": 0,
-            "nb_individuals": 681,
-            "nb_properties": 0,
+            "nb_individuals": 464,
+            "nb_properties": 5,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 17,
             "taxon": null,
-            "version": null
+            "version": "6.0"
         },
         "topics": null
     },
     {
-        "acronym": "ATOL",
-        "name": "Animal Trait Ontology for Livestock",
-        "species": null,
-        "status": "Classified",
+        "acronym": "ENMT",
+        "name": "eNanoMapper",
+        "species": [],
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2014-10-22T08:53:10Z",
-            "date_released": "2014-10-22T00:00:00Z",
-            "description": "ATOL (Animal Trait Ontology for Livestock) is an ontology of characteristics defining phenotypes of livestock in their environment (EOL).\r\nATOL aims to:\r\n- provide a reference ontology of phenotypic traits of farm animals for the international scientific and educational - communities, farmers, etc.;\r\n- deliver this reference ontology in a language which can be used by computers in order to support database management, semantic analysis and modeling;\r\n- represent traits as generic as possible for livestock vertebrates;\r\n- make the ATOL ontology as operational as possible and closely related to measurement techniques;\r\n- structure the ontology in relation to animal production\r\n\r\n",
-            "documentation": null,
+            "date_created": "2019-11-30T13:22:16.788076Z",
+            "date_released": "2019-11-30T13:22:16.788174Z",
+            "description": "The eNanoMapper ontology covers the full scope of terminology needed to support research into nanomaterial safety. It builds on multiple pre-existing external ontologies such as the NanoParticle Ontology.",
+            "documentation": "http://enanomapper.net/ontology",
             "domain": null,
-            "download_url": "media/ontologies/ATOL/1/atol.owl",
+            "download_url": "media/ontologies/ENMT/1/enmt.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.atol-ontology.com",
-            "id": 660,
+            "home_page": "http://enanomapper.net/",
+            "id": 3139,
             "indexed": true,
-            "max_children": 35,
-            "max_depth": 11,
+            "max_children": 1427,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 2098,
+            "nb_classes": 12751,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 464,
+            "nb_properties": 5,
             "products": null,
-            "publication": null,
+            "publication": "http://enanomapper.net/library",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "6.0"
         },
-        "topics": null
+        "topics": [
+            "nanomaterials"
+        ]
     },
     {
-        "acronym": "ENVS_VARIABLES",
-        "name": "ENVS variables",
+        "acronym": "ELD",
+        "name": "EthiopianListOfDiseases",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-09-28T01:03:43Z",
-            "date_released": "2021-09-28T01:03:42Z",
-            "description": "Quantitative and qualitative variables describing environmental state",
-            "documentation": "https://github.com/m4m-dk/workshop-2-vocabulary",
+            "date_created": "2021-11-01T10:17:06Z",
+            "date_released": "2021-07-23T00:00:00Z",
+            "description": "This is the list of Ethiopian Diseases approved by MOH Ethiopia",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ENVS_VARIABLES/2/envs_variables.skos",
+            "download_url": "media/ontologies/ELD/5/eld.skos",
             "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/m4m-dk/workshop-2-vocabulary",
-            "id": 4576,
+            "home_page": null,
+            "id": 4666,
             "indexed": true,
             "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
             "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 12,
+            "nb_individuals": 2077,
             "nb_properties": 0,
             "products": null,
-            "publication": "https://github.com/m4m-dk/workshop-2-vocabulary",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 5,
             "taxon": null,
-            "version": "0.1.1"
+            "version": "4"
         },
         "topics": null
     },
     {
-        "acronym": "ENVTHES",
-        "name": "Thesaurus for long-term ecological research, monitoring, experiments",
+        "acronym": "ELIG",
+        "name": "Eligibility Feature Hierarchy",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-05-26T00:17:47Z",
-            "date_released": "2022-11-22T00:00:00Z",
-            "description": "EnvThes",
+            "date_created": "2013-05-07T20:09:37Z",
+            "date_released": "2013-05-07T07:00:00Z",
+            "description": "We derived a set of 1,437 eligibility features that were organized into a feature hierarchy using 80 breast cancer trials. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ENVTHES/23/envthes.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/ELIG/1/elig.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5831,
+            "id": 579,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 6,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 29,
             "nb_inconsistent": 0,
-            "nb_individuals": 5765,
-            "nb_properties": 0,
+            "nb_individuals": 2,
+            "nb_properties": 3,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 23,
+            "submission_id": 1,
             "taxon": null,
-            "version": "5.0.1"
+            "version": "v. 1.0"
         },
         "topics": null
     },
     {
         "acronym": "EO",
         "name": "Ethnicity Ontoloy",
         "species": null,
@@ -33886,14 +33346,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
+        "acronym": "TEDDY",
+        "name": "Terminology for the Description of Dynamics",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2014-04-25T15:51:26Z",
+            "date_released": "2014-04-25T15:51:25Z",
+            "description": "The TErminology for the Description of DYnamics (TEDDY) project aims to provide an ontology for dynamical behaviours, observable dynamical phenomena, and control elements of bio-models and biological systems in Systems Biology and Synthetic Biology.",
+            "documentation": "http://identifiers.org/combine.specifications/teddy",
+            "domain": null,
+            "download_url": "media/ontologies/TEDDY/11/teddy.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://teddyontology.sourceforge.net/",
+            "id": 764,
+            "indexed": true,
+            "max_children": 21,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 167,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 15,
+            "products": null,
+            "publication": "http://www.nature.com/msb/journal/v7/n1/full/msb201177.html",
+            "publications": null,
+            "submission_id": 11,
+            "taxon": null,
+            "version": "rel-2014-04-24 (inferred)"
+        },
+        "topics": null
+    },
+    {
         "acronym": "EO1",
         "name": "Example Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -33920,14 +33414,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "TCDO",
+        "name": "Traditional Chinese Drug Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2021-06-16T08:23:46Z",
+            "date_released": "2021-05-30T00:00:00Z",
+            "description": "More than 400 popular TCD decoction pieces (TCD-DP) were selected from authoritative textbooks, standards and terminologies, as well as their crudes, Chinese medicinal material (CMM). The TCD-DP entities are categorized in 23 upper level terms by their clinical effects. Their properties like nature, flavor, toxicity and meridian entries are also defined formally by logical relations. CMM\u2019s species is mapped to the NCBITaxon taxonomy ontology, which establishes a link between traditional medicine and modern biomedical science. TCDO contains over 1,500 terms, including terms drawn from existing ontologies and more than 1,000 TCD-specific terms, labeled in both Chinese and English.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/TCDO/3/tcdo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/TCMOntology/TCDO",
+            "id": 4667,
+            "indexed": true,
+            "max_children": 435,
+            "max_depth": 22,
+            "md5sum": null,
+            "nb_classes": 1988,
+            "nb_inconsistent": 0,
+            "nb_individuals": 4,
+            "nb_properties": 42,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "0.05"
+        },
+        "topics": null
+    },
+    {
         "acronym": "EOL",
         "name": "Environment Ontology for Livestock",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -33988,112 +33516,260 @@
             "submission_id": 4,
             "taxon": null,
             "version": "1.3"
         },
         "topics": null
     },
     {
-        "acronym": "NMOBR_2",
-        "name": "NeuroMorpho.Org Hierarchies for browsing",
+        "acronym": "EPILONT",
+        "name": "Epilepsy Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2011-07-18T16:54:20Z",
+            "date_released": "2011-07-18T07:00:00Z",
+            "description": "Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE).",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/EPILONT/2/epilont.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "www.epilepsiae.eu",
+            "id": 86,
+            "indexed": true,
+            "max_children": 28,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 138,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 10,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "V1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NERO",
+        "name": "Named Entity Recognition Ontology ",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2021-04-28T17:37:55Z",
+            "date_released": "2021-04-28T00:00:00Z",
+            "description": "Named Entity Recognition Ontology (NERO) developed specifically for describing textual entities in biomedical texts, which accounts for diverse levels of ambiguity, bridging the scientific sublanguages of molecular biology, genetics, biochemistry, and medicine. The annotation of natural science texts is more challenging than in other domains. Biomedical language is replete with ambiguity distinct from that observed in news articles or informal text online. In biomedical texts, alternative meanings are not always clearly separated.\r\n\r\nFor example, in some biomedical contexts, the words for a named entity may refer to a gene or a protein with nearly equal probability; for example, \u201ca mutant hemoglobin \u03b1_2\u201d can refer to either a gene or a protein. If the author meant gene-or-protein A, and we force an annotator to choose either interpretation gene A or protein A, the resulting annotation is of limited utility because the choice between gene and protein is random if the meanings are equally likely based on context. NERO attempts to minimize unwarranted, arbitrary annotative semantic label assignments for textual entities. \r\n\r\nNERO defines ambiguous concepts, such as GeneOrProtein, which subsumes both Gene and Protein using the following axiom: EquivalentTo: \u2018Gene\u2019 or \u2018Protein.\u2019 There are no biological entities that are either a gene or a protein, but there are lexical entities that can correspond to either or both of these entities.\r\n",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/NERO/1/nero.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4453,
+            "indexed": true,
+            "max_children": 21,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 63,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 27,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "The main developer of NERO is Prof. Robert Stevens, the University of Manchester"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NEST",
+        "name": "Nutritional Epidemiological STandards",
+        "species": null,
+        "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-11-28T02:12:11Z",
-            "date_released": "2017-11-28T02:12:11Z",
-            "description": "NeuroMorpho.Org v7.0 Ontologies for human readability.",
+            "date_created": "2018-08-27T09:13:11Z",
+            "date_released": "2018-08-20T00:00:00Z",
+            "description": "It represents standards for nutritional epidemiological studies in ontology format. In the strict sense, it is not an ontology. Its hierarchy follows structure of each standard, not OWL rules.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NMOBR_2/6/nmobr_2.owl",
+            "download_url": "media/ontologies/NEST/5/nest.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/cyang0128/Nutritional-epidemiological-ontologies",
+            "id": 1778,
+            "indexed": true,
+            "max_children": 17,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 249,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": "https://www.ncbi.nlm.nih.gov/pubmed/28916566",
+            "publications": null,
+            "submission_id": 5,
+            "taxon": null,
+            "version": "V1.2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NMOBR",
+        "name": "NeuroMorpho.Org brain region ontologies",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2017-11-28T02:12:08Z",
+            "date_released": "2017-11-28T02:12:08Z",
+            "description": "The species, brain regions and cell types are integrated into a single ontology for enabling OntoSearch functionality to mine the latest v7.0 release of NeuroMorpho.Org.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/NMOBR/32/nmobr.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 865,
+            "id": 879,
             "indexed": true,
             "max_children": 408,
             "max_depth": 40,
             "md5sum": null,
-            "nb_classes": 2394,
+            "nb_classes": 3250,
             "nb_inconsistent": 0,
-            "nb_individuals": 1117,
+            "nb_individuals": 1968,
             "nb_properties": 2,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 32,
             "taxon": null,
-            "version": "2.2"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "NMOCT",
-        "name": "NeuroMorpho.Org cell type ontologies",
-        "species": null,
-        "status": "Unloadable",
+        "acronym": "T123",
+        "name": "Test123",
+        "species": [],
+        "status": "Classified",
         "submission": {
-            "avg_children": 10,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-11-28T02:12:12Z",
-            "date_released": "2017-11-28T02:12:12Z",
-            "description": "A hierarchy of several neuron description dimensions including circuit type, morphology, neurotransmitter, and functional properties. ",
+            "date_created": "2023-02-08T09:41:23.498758Z",
+            "date_released": "2023-02-08T09:41:23.498777Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NMOCT/5/nmoct.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/T123/1/t123.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 882,
+            "id": 5648,
             "indexed": true,
-            "max_children": 251,
-            "max_depth": 4,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 428,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 413,
+            "nb_individuals": 9,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2.0"
+            "version": "0.1.0"
+        },
+        "topics": []
+    },
+    {
+        "acronym": "FAO",
+        "name": "Fungal gross anatomy",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2020-05-11T15:15:58.939049Z",
+            "date_released": "2020-05-11T15:15:58.939040Z",
+            "description": "A structured controlled vocabulary for the anatomy of fungi.",
+            "documentation": null,
+            "domain": "anatomy",
+            "download_url": "media/ontologies/FAO/28/fao.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obophenotype/fungal-anatomy-ontology/",
+            "id": 3593,
+            "indexed": true,
+            "max_children": 11,
+            "max_depth": 8,
+            "md5sum": "7484178e9244528982598c717ff00236",
+            "nb_classes": 120,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 3,
+            "products": [
+                {
+                    "id": "fao.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fao.owl"
+                },
+                {
+                    "id": "fao.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fao.obo"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 28,
+            "taxon": {
+                "id": "NCBITaxon:4751",
+                "label": "Fungal"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "EPILONT",
-        "name": "Epilepsy Ontology",
+        "acronym": "FAST-CHRONO",
+        "name": "FAST (Faceted Access of Subject Terminology) Chronological Facet",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2011-07-18T16:54:20Z",
-            "date_released": "2011-07-18T07:00:00Z",
-            "description": "Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE).",
+            "date_created": "2018-05-07T23:55:15Z",
+            "date_released": "2018-05-07T00:00:00Z",
+            "description": "FAST is an enumerative, faceted subject heading schema derived from the Library of Congress Subject Headings (LCSH). The purpose of adapting the LCSH with a simplified syntax to create FAST is to retain the very rich vocabulary of LCSH while making the schema easier to understand, control, apply, and use. This facet includes chronological terms derived from LCSH.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/EPILONT/2/epilont.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "www.epilepsiae.eu",
-            "id": 86,
+            "download_url": "media/ontologies/FAST-CHRONO/1/fast-chrono.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 1477,
             "indexed": true,
-            "max_children": 28,
-            "max_depth": 4,
+            "max_children": 3,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 138,
+            "nb_classes": 3,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 10,
+            "nb_individuals": 681,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "V1"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "FAST-EVENT",
         "name": "FAST (Faceted Application of Subject Terminology) Event",
         "species": null,
@@ -34124,14 +33800,150 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "ATOL",
+        "name": "Animal Trait Ontology for Livestock",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2014-10-22T08:53:10Z",
+            "date_released": "2014-10-22T00:00:00Z",
+            "description": "ATOL (Animal Trait Ontology for Livestock) is an ontology of characteristics defining phenotypes of livestock in their environment (EOL).\r\nATOL aims to:\r\n- provide a reference ontology of phenotypic traits of farm animals for the international scientific and educational - communities, farmers, etc.;\r\n- deliver this reference ontology in a language which can be used by computers in order to support database management, semantic analysis and modeling;\r\n- represent traits as generic as possible for livestock vertebrates;\r\n- make the ATOL ontology as operational as possible and closely related to measurement techniques;\r\n- structure the ontology in relation to animal production\r\n\r\n",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/ATOL/1/atol.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "www.atol-ontology.com",
+            "id": 660,
+            "indexed": true,
+            "max_children": 35,
+            "max_depth": 11,
+            "md5sum": null,
+            "nb_classes": 2098,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NMOBR_2",
+        "name": "NeuroMorpho.Org Hierarchies for browsing",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2017-11-28T02:12:11Z",
+            "date_released": "2017-11-28T02:12:11Z",
+            "description": "NeuroMorpho.Org v7.0 Ontologies for human readability.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/NMOBR_2/6/nmobr_2.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 865,
+            "indexed": true,
+            "max_children": 408,
+            "max_depth": 40,
+            "md5sum": null,
+            "nb_classes": 2394,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1117,
+            "nb_properties": 2,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 6,
+            "taxon": null,
+            "version": "2.2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ENVS_VARIABLES",
+        "name": "ENVS variables",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2021-09-28T01:03:43Z",
+            "date_released": "2021-09-28T01:03:42Z",
+            "description": "Quantitative and qualitative variables describing environmental state",
+            "documentation": "https://github.com/m4m-dk/workshop-2-vocabulary",
+            "domain": null,
+            "download_url": "media/ontologies/ENVS_VARIABLES/2/envs_variables.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/m4m-dk/workshop-2-vocabulary",
+            "id": 4576,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 12,
+            "nb_properties": 0,
+            "products": null,
+            "publication": "https://github.com/m4m-dk/workshop-2-vocabulary",
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "0.1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "NMOCT",
+        "name": "NeuroMorpho.Org cell type ontologies",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 10,
+            "classifiable": true,
+            "date_created": "2017-11-28T02:12:12Z",
+            "date_released": "2017-11-28T02:12:12Z",
+            "description": "A hierarchy of several neuron description dimensions including circuit type, morphology, neurotransmitter, and functional properties. ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/NMOCT/5/nmoct.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 882,
+            "indexed": true,
+            "max_children": 251,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 428,
+            "nb_inconsistent": 0,
+            "nb_individuals": 413,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 5,
+            "taxon": null,
+            "version": "2.0"
+        },
+        "topics": null
+    },
+    {
         "acronym": "FASTO",
         "name": "HL7 FHIR and SSN ontology based Type 1 Diabetes Mellitus Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -34158,44 +33970,112 @@
             "submission_id": 2,
             "taxon": null,
             "version": "2015-11-20"
         },
         "topics": null
     },
     {
-        "acronym": "FAST-EVENT-SKOS",
-        "name": "FAST (Faceted Application of Subject Terminology) Event Facet",
+        "acronym": "ENVTHES",
+        "name": "Thesaurus for long-term ecological research, monitoring, experiments",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-08-30T15:51:48Z",
-            "date_released": "2017-04-14T00:00:00Z",
-            "description": "This is a test of submitting FAST-EVENT with Format dropdown set to SKOS (instead of OWL)",
+            "date_created": "2023-05-26T00:17:47Z",
+            "date_released": "2022-11-22T00:00:00Z",
+            "description": "EnvThes",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FAST-EVENT-SKOS/1/fast-event-skos.skos",
+            "download_url": "media/ontologies/ENVTHES/23/envthes.skos",
             "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 187,
+            "id": 5831,
             "indexed": true,
-            "max_children": 4,
-            "max_depth": 0,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 4,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 15696,
+            "nb_individuals": 5765,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
+            "submission_id": 23,
+            "taxon": null,
+            "version": "5.0.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TCO",
+        "name": "Thyroid Cancer Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 19,
+            "classifiable": true,
+            "date_created": "2020-07-03T01:27:51Z",
+            "date_released": "2020-07-02T00:00:00Z",
+            "description": "Thyroid cancer ontology (TCO) contains 578 concepts organized under 6 biological viewpoints, i.e., Anatomy and Histoembryology, Basic Information of Patient, Cellular or Molecular Interactions, Clinical Aspects, Etiology, Pathology",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/TCO/2/tco.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3743,
+            "indexed": true,
+            "max_children": 510,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 578,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 3,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TDWGSPEC",
+        "name": "TDWG Specimen LSID Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2017-02-16T23:29:08Z",
+            "date_released": "2015-07-01T00:00:00Z",
+            "description": "Biodiversity Information Standards (TDWG) is an international not-for-profit group that develops standards and protocols for sharing biodiversity data. They have a detailed collection of vocabularies and ontologies for studies of biological organisms.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/TDWGSPEC/1/tdwgspec.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 562,
+            "indexed": true,
+            "max_children": 7,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 9,
+            "nb_inconsistent": 0,
+            "nb_individuals": 7,
+            "nb_properties": 31,
+            "products": null,
+            "publication": null,
+            "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "0.3"
         },
         "topics": null
     },
     {
         "acronym": "FAST-FORMGENRE",
         "name": "FAST (Faceted Application of Subject Terminology) FormGenre facet",
         "species": null,
@@ -34260,326 +34140,266 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TEDDY",
-        "name": "Terminology for the Description of Dynamics",
+        "acronym": "FAST-EVENT-SKOS",
+        "name": "FAST (Faceted Application of Subject Terminology) Event Facet",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2014-04-25T15:51:26Z",
-            "date_released": "2014-04-25T15:51:25Z",
-            "description": "The TErminology for the Description of DYnamics (TEDDY) project aims to provide an ontology for dynamical behaviours, observable dynamical phenomena, and control elements of bio-models and biological systems in Systems Biology and Synthetic Biology.",
-            "documentation": "http://identifiers.org/combine.specifications/teddy",
+            "date_created": "2017-08-30T15:51:48Z",
+            "date_released": "2017-04-14T00:00:00Z",
+            "description": "This is a test of submitting FAST-EVENT with Format dropdown set to SKOS (instead of OWL)",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TEDDY/11/teddy.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://teddyontology.sourceforge.net/",
-            "id": 764,
+            "download_url": "media/ontologies/FAST-EVENT-SKOS/1/fast-event-skos.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 187,
             "indexed": true,
-            "max_children": 21,
-            "max_depth": 7,
+            "max_children": 4,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 167,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 15,
+            "nb_individuals": 15696,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.nature.com/msb/journal/v7/n1/full/msb201177.html",
+            "publication": null,
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 1,
             "taxon": null,
-            "version": "rel-2014-04-24 (inferred)"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "TCDO",
-        "name": "Traditional Chinese Drug Ontology",
+        "acronym": "FAST-TOPICAL",
+        "name": "FAST (Faceted Application of Subject Terminology) Topical Facet",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-06-16T08:23:46Z",
-            "date_released": "2021-05-30T00:00:00Z",
-            "description": "More than 400 popular TCD decoction pieces (TCD-DP) were selected from authoritative textbooks, standards and terminologies, as well as their crudes, Chinese medicinal material (CMM). The TCD-DP entities are categorized in 23 upper level terms by their clinical effects. Their properties like nature, flavor, toxicity and meridian entries are also defined formally by logical relations. CMM\u2019s species is mapped to the NCBITaxon taxonomy ontology, which establishes a link between traditional medicine and modern biomedical science. TCDO contains over 1,500 terms, including terms drawn from existing ontologies and more than 1,000 TCD-specific terms, labeled in both Chinese and English.",
+            "date_created": "2017-09-06T20:47:53Z",
+            "date_released": "2017-04-14T00:00:00Z",
+            "description": "AST is an enumerative, faceted subject heading schema derived from the Library of Congress Subject Headings (LCSH). The purpose of adapting the LCSH with a simplified syntax to create FAST is to retain the very rich vocabulary of LCSH while making the schema easier to understand, control, apply, and use.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TCDO/3/tcdo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/TCMOntology/TCDO",
-            "id": 4667,
+            "download_url": "media/ontologies/FAST-TOPICAL/1/fast-topical.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 411,
             "indexed": true,
-            "max_children": 435,
-            "max_depth": 22,
+            "max_children": 3,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 1988,
+            "nb_classes": 3,
             "nb_inconsistent": 0,
-            "nb_individuals": 4,
-            "nb_properties": 42,
+            "nb_individuals": 583314,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.05"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "EP",
-        "name": "Cardiac Electrophysiology Ontology",
+        "acronym": "EPIO",
+        "name": "Epilepsy Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2009-04-28T02:50:51Z",
-            "date_released": "2008-12-10T08:00:00Z",
-            "description": "The Cardiac Electrophysiology Ontology contains terms describing single-channel electrophysiological experiments and data obtained using voltage-clamp, current clamp and fluorescence imaging techniques applied at the cell level and multi-channel fluorescence imaging techniques applied at the cell, tissue and whole heart levels.",
+            "date_created": "2022-12-26T12:19:57.770929Z",
+            "date_released": "2022-12-26T12:19:57.770900Z",
+            "description": "A application driven Epilepsy Ontology with official terms from the ILAE.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/EP/2/ep.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/EPIO/3/epio.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.cvrgrid.org",
-            "id": 560,
+            "home_page": "https://github.com/SCAI-BIO/EpilepsyOntology",
+            "id": 5581,
             "indexed": true,
-            "max_children": 25,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 232,
+            "max_children": 41,
+            "max_depth": 16,
+            "md5sum": "34876cf99fbb5d7a831ddc86f0d0d93e",
+            "nb_classes": 1916,
             "nb_inconsistent": 0,
             "nb_individuals": 1,
-            "nb_properties": 9,
-            "products": null,
+            "nb_properties": 12,
+            "products": [
+                {
+                    "id": "epio.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/epio.owl"
+                },
+                {
+                    "id": "EPIO_merged.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/EPIO_merged.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OBOREL",
-        "name": "Relations Ontology",
+        "acronym": "FALDO",
+        "name": "Feature Annotation Location Description Ontology ",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-11-28T16:18:50Z",
-            "date_released": "2017-10-02T00:00:00Z",
-            "description": "A collection of relations intended primarily for standardization across ontologies in the OBO Foundry and wider OBO library. It incorporates ROCore upper-level relations such as part of as well as biology-specific relationship types such as develops from.",
-            "documentation": null,
+            "date_created": "2017-11-03T15:46:37Z",
+            "date_released": "2015-06-20T00:00:00Z",
+            "description": "FALDO is the Feature Annotation Location Description Ontology. It is a simple ontology to describe sequence feature positions and regions as found in GFF3, DBBJ, EMBL, GenBank files, UniProt, and many other bioinformatics resources.\r\n\r\nThe aim of this ontology is to describe the position of a sequence region or a feature. It does not aim to describe features or regions itself, but instead depends on resources such as the Sequence Ontology or the UniProt core ontolgy.",
+            "documentation": "https://github.com/OBF/FALDO",
             "domain": null,
-            "download_url": "media/ontologies/OBOREL/10/oborel.owl",
+            "download_url": "media/ontologies/FALDO/1/faldo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/oborel/obo-relations/",
-            "id": 873,
+            "home_page": "http://biohackathon.org/resource/faldo",
+            "id": 783,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 10,
+            "max_children": 5,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 79,
+            "nb_classes": 19,
             "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 528,
+            "nb_individuals": 0,
+            "nb_properties": 10,
             "products": null,
-            "publication": null,
+            "publication": "https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-016-0067-z",
             "publications": null,
-            "submission_id": 10,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Created at the Biohackathon 2012 and 2013"
         },
         "topics": null
     },
     {
-        "acronym": "OBI",
-        "name": "Ontology for Biomedical Investigations",
+        "acronym": "TAXRANK",
+        "name": "Taxonomic rank vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 30,
             "classifiable": true,
-            "date_created": "2023-03-27T12:55:11.532900Z",
-            "date_released": "2023-03-27T12:55:11.532882Z",
-            "description": "An integrated ontology for the description of life-science and clinical investigations",
+            "date_created": "2017-12-13T11:31:32.307539Z",
+            "date_released": "2017-12-13T11:31:32.307517Z",
+            "description": "A vocabulary of taxonomic ranks (species, family, phylum, etc)",
             "documentation": null,
-            "domain": "investigations",
-            "download_url": "media/ontologies/OBI/50/obi.owl",
+            "domain": "taxonomy",
+            "download_url": "media/ontologies/TAXRANK/10/taxrank.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://obi-ontology.org",
-            "id": 5738,
+            "home_page": "https://www.phenoscape.org/wiki/Taxonomic_Rank_Vocabulary",
+            "id": 1071,
             "indexed": true,
-            "max_children": 106,
-            "max_depth": 18,
-            "md5sum": "59ccf5813ee7393cacda3d96aebb665e",
-            "nb_classes": 4866,
+            "max_children": 60,
+            "max_depth": 2,
+            "md5sum": "b5540d3672dcd2d46c5286fc790a52d9",
+            "nb_classes": 62,
             "nb_inconsistent": 0,
-            "nb_individuals": 306,
-            "nb_properties": 98,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": [
                 {
-                    "description": "The full version of OBI in OWL format",
-                    "id": "obi.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/obi.owl",
-                    "title": "OBI"
-                },
-                {
-                    "description": "The OBO-format version of OBI",
-                    "id": "obi.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/obi.obo",
-                    "title": "OBI in OBO"
-                },
-                {
-                    "description": "A collection of important high-level terms and their relations from OBI and other ontologies",
-                    "id": "obi/obi_core.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/obi/obi_core.owl",
-                    "title": "OBI Core"
+                    "id": "taxrank.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/taxrank.owl"
                 },
                 {
-                    "description": "Base module for OBI",
-                    "id": "obi/obi-base.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/obi/obi-base.owl",
-                    "title": "OBI Base module"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/27128319",
-                    "title": "The Ontology for Biomedical Investigations"
-                }
-            ],
-            "submission_id": 50,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OBIB",
-        "name": "Ontology for Biobanking",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2023-05-15T15:17:32.258356Z",
-            "date_released": "2023-05-15T15:17:32.258343Z",
-            "description": "An ontology built for annotation and modeling of biobank repository and biobanking administration",
-            "documentation": null,
-            "domain": "investigations",
-            "download_url": "media/ontologies/OBIB/15/obib.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/biobanking/biobanking",
-            "id": 5811,
-            "indexed": true,
-            "max_children": 129,
-            "max_depth": 16,
-            "md5sum": "d834e37495f377aeaab15271c0cb18bb",
-            "nb_classes": 1804,
-            "nb_inconsistent": 0,
-            "nb_individuals": 226,
-            "nb_properties": 93,
-            "products": [
-                {
-                    "id": "obib.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/obib.owl"
+                    "id": "taxrank.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/taxrank.obo"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 15,
+            "submission_id": 10,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NOMEN",
-        "name": "NOMEN - A nomenclatural ontology for biological names",
+        "acronym": "EP",
+        "name": "Cardiac Electrophysiology Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-04-24T12:46:34.715261Z",
-            "date_released": "2023-04-24T12:46:34.715249Z",
-            "description": "NOMEN is a nomenclatural ontology for biological names (not concepts).  It encodes the goverened rules of nomenclature.",
+            "date_created": "2009-04-28T02:50:51Z",
+            "date_released": "2008-12-10T08:00:00Z",
+            "description": "The Cardiac Electrophysiology Ontology contains terms describing single-channel electrophysiological experiments and data obtained using voltage-clamp, current clamp and fluorescence imaging techniques applied at the cell level and multi-channel fluorescence imaging techniques applied at the cell, tissue and whole heart levels.",
             "documentation": null,
-            "domain": "information",
-            "download_url": "media/ontologies/NOMEN/8/nomen.owl",
+            "domain": null,
+            "download_url": "media/ontologies/EP/2/ep.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/SpeciesFileGroup/nomen",
-            "id": 5780,
+            "home_page": "http://www.cvrgrid.org",
+            "id": 560,
             "indexed": true,
-            "max_children": 22,
-            "max_depth": 8,
-            "md5sum": "228bf60a5bf213cf567ca6798d288e54",
-            "nb_classes": 263,
+            "max_children": 25,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 232,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 112,
-            "products": [
-                {
-                    "description": "core ontology",
-                    "id": "nomen.owl",
-                    "is_canonical": true,
-                    "ontology_purl": "http://purl.obolibrary.org/obo/nomen.owl",
-                    "title": "NOMEN",
-                    "type": "owl:Ontology"
-                }
-            ],
+            "nb_individuals": 1,
+            "nb_properties": 9,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "OBI_BCGO",
-        "name": "Beta Cell Genomics Ontology",
+        "acronym": "OBOREL",
+        "name": "Relations Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2015-08-21T01:07:04Z",
-            "date_released": "2015-08-21T01:07:04Z",
-            "description": "Beta Cell Genomics Ontology (BCGO) is an application ontology built for the Beta Cell Genomics database (http://genomics.betacell.org/gbco/) aiming to support database annotation, complicated semantic queries, and automated cell type classification. The ontology is developed using Basic Formal Ontology (BFO) as upper ontology, Ontology for Biomedical Investigations (OBI) as ontology framework and integrated subsets of multiple OBO Foundry (candidate) ontologies. Current the BCGO contains 2383 classes including terms referencing to 24 various OBO Foundry ontologies including CL, CLO, UBERON, GO, PRO, UO, etc.",
+            "date_created": "2017-11-28T16:18:50Z",
+            "date_released": "2017-10-02T00:00:00Z",
+            "description": "A collection of relations intended primarily for standardization across ontologies in the OBO Foundry and wider OBO library. It incorporates ROCore upper-level relations such as part of as well as biology-specific relationship types such as develops from.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OBI_BCGO/14/obi_bcgo.owl",
+            "download_url": "media/ontologies/OBOREL/10/oborel.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obi-bcgo/bcgo",
-            "id": 679,
+            "home_page": "https://github.com/oborel/obo-relations/",
+            "id": 873,
             "indexed": true,
-            "max_children": 129,
-            "max_depth": 18,
+            "max_children": 7,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 2270,
+            "nb_classes": 79,
             "nb_inconsistent": 0,
-            "nb_individuals": 46,
-            "nb_properties": 107,
+            "nb_individuals": 5,
+            "nb_properties": 528,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 10,
             "taxon": null,
-            "version": "2015-07-08"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "NPHO",
         "name": "Nightshade Phenotype Ontology",
         "species": null,
@@ -34712,150 +34532,14 @@
             "submission_id": 11,
             "taxon": null,
             "version": "2013-10-25"
         },
         "topics": null
     },
     {
-        "acronym": "OBI_IEDB_view",
-        "name": "IEDB View",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2012-01-27T20:31:42Z",
-            "date_released": "2012-01-27T08:00:00Z",
-            "description": "OBI view for IMMUNE EPITOPE DATABASE (IEDB) community. The OBI terms used by IEDB were tagged by inSubSet annotation with the value \"IEDB\". The IEDB friendly labels were provided as IEDB alternative term.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OBI_IEDB_view/1/obi_iedb_view.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 42,
-            "indexed": true,
-            "max_children": 332,
-            "max_depth": 40,
-            "md5sum": null,
-            "nb_classes": 3519,
-            "nb_inconsistent": 0,
-            "nb_individuals": 149,
-            "nb_properties": 77,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "2011-12-13"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OBI_IEE",
-        "name": "The Ontology for Biomedical Investigation based Inner Ear Electrophysiology ",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2019-08-31T21:05:54Z",
-            "date_released": "2019-08-31T00:00:00Z",
-            "description": "Inner Ear Electrophysiology (IEE) is an application ontology created to describe electrophysiology and biophysical data collections of the anatomical entities and cells (e.g., outer hair cells) of the inner ear.  The ontology is developed with The Ontology for Biomedical Investigations (OBI) and integrated subsets of multiple OBO ontologies, especially, PATO, FMA, CHEBI, and OBA.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OBI_IEE/1/obi_iee.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2894,
-            "indexed": true,
-            "max_children": 78,
-            "max_depth": 17,
-            "md5sum": null,
-            "nb_classes": 3507,
-            "nb_inconsistent": 0,
-            "nb_individuals": 300,
-            "nb_properties": 94,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "2018-05-23"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OBIWS",
-        "name": "Bioinformatics Web Service Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2012-12-18T23:13:11Z",
-            "date_released": "2012-12-18T08:00:00Z",
-            "description": "The Bioinformatics Web Services ontology (OBIws) is an ontology that extends the Ontology for Biomedical Investigations (OBI) to build an Ontology that supports consistent annotation of Bioinformatics Web services. ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OBIWS/2/obiws.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://code.google.com/p/obi-webservice/",
-            "id": 185,
-            "indexed": true,
-            "max_children": 34,
-            "max_depth": 33,
-            "md5sum": null,
-            "nb_classes": 228,
-            "nb_inconsistent": 0,
-            "nb_individuals": 34,
-            "nb_properties": 27,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "Release v1.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OBOE",
-        "name": "The Extensible Observation Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 19,
-            "classifiable": true,
-            "date_created": "2019-09-18T01:59:47Z",
-            "date_released": "2019-09-17T00:00:00Z",
-            "description": "The Extensible Observation Ontology (OBOE) is a formal ontology for capturing the semantics of scientific observation and measurement. The ontology supports researchers to add detailed semantic annotations to scientific data, thereby clarifying the inherent meaning of scientific observations. The main concepts in OBOE include:\r\n\r\n- Observation: an event in which one or more measurements are taken\r\n- Measurement: the measured value of a property for a specific object or phenomenon (e.g., 3.2)\r\n- Entity: an object or phenomenon on which measurements are made (e.g., Quercus rubrum)\r\n- Characteristic: the property being measured (e.g., VolumetricDensity)\r\n- Standard: units and controlled vocabularies for interpreting measured values (e.g., g/cm^3)\r\n- Protocol: the procedures followed to obtain measurements (e.g., DensityProtocol2014)\r\n\r\nOBOE can characterize the context of an observation (e.g., space and time), as well as dependencies such as nested experimental observations. It includes an extensive set of unit definitions (e.g., grams of carbon per liter of seawater), and can facilitate automatic unit conversions (e.g., pounds to kilograms). OBOE can be easily extended to create Entities and Characteristics for specific research domains, making it both broadly applicable and highly customizable.\r\n\r\nOBOE is being used to improve data interpretation, facilitate reuse, and create precise and effective systems for data search and discovery.",
-            "documentation": "https://github.com/NCEAS/oboe",
-            "domain": null,
-            "download_url": "media/ontologies/OBOE/4/oboe.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/NCEAS/oboe",
-            "id": 2937,
-            "indexed": true,
-            "max_children": 70,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 292,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 24,
-            "products": null,
-            "publication": "Madin, J., S. Bowers, M. Schildhauer, S. Krivov, D. Pennington, and F. Villa. 2007. An ontology for describing and synthesizing ecological observation data. Ecological Informatics 2:279\u2013296. doi:10.1016/j.ecoinf.2007.05.004",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "Version 1.2"
-        },
-        "topics": null
-    },
-    {
         "acronym": "NPO",
         "name": "NanoParticle Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -35096,14 +34780,116 @@
             "submission_id": 4,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "OBI_IEE",
+        "name": "The Ontology for Biomedical Investigation based Inner Ear Electrophysiology ",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2019-08-31T21:05:54Z",
+            "date_released": "2019-08-31T00:00:00Z",
+            "description": "Inner Ear Electrophysiology (IEE) is an application ontology created to describe electrophysiology and biophysical data collections of the anatomical entities and cells (e.g., outer hair cells) of the inner ear.  The ontology is developed with The Ontology for Biomedical Investigations (OBI) and integrated subsets of multiple OBO ontologies, especially, PATO, FMA, CHEBI, and OBA.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/OBI_IEE/1/obi_iee.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 2894,
+            "indexed": true,
+            "max_children": 78,
+            "max_depth": 17,
+            "md5sum": null,
+            "nb_classes": 3507,
+            "nb_inconsistent": 0,
+            "nb_individuals": 300,
+            "nb_properties": 94,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "2018-05-23"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "OBIWS",
+        "name": "Bioinformatics Web Service Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2012-12-18T23:13:11Z",
+            "date_released": "2012-12-18T08:00:00Z",
+            "description": "The Bioinformatics Web Services ontology (OBIws) is an ontology that extends the Ontology for Biomedical Investigations (OBI) to build an Ontology that supports consistent annotation of Bioinformatics Web services. ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/OBIWS/2/obiws.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://code.google.com/p/obi-webservice/",
+            "id": 185,
+            "indexed": true,
+            "max_children": 34,
+            "max_depth": 33,
+            "md5sum": null,
+            "nb_classes": 228,
+            "nb_inconsistent": 0,
+            "nb_individuals": 34,
+            "nb_properties": 27,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "Release v1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "OBOE",
+        "name": "The Extensible Observation Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 19,
+            "classifiable": true,
+            "date_created": "2019-09-18T01:59:47Z",
+            "date_released": "2019-09-17T00:00:00Z",
+            "description": "The Extensible Observation Ontology (OBOE) is a formal ontology for capturing the semantics of scientific observation and measurement. The ontology supports researchers to add detailed semantic annotations to scientific data, thereby clarifying the inherent meaning of scientific observations. The main concepts in OBOE include:\r\n\r\n- Observation: an event in which one or more measurements are taken\r\n- Measurement: the measured value of a property for a specific object or phenomenon (e.g., 3.2)\r\n- Entity: an object or phenomenon on which measurements are made (e.g., Quercus rubrum)\r\n- Characteristic: the property being measured (e.g., VolumetricDensity)\r\n- Standard: units and controlled vocabularies for interpreting measured values (e.g., g/cm^3)\r\n- Protocol: the procedures followed to obtain measurements (e.g., DensityProtocol2014)\r\n\r\nOBOE can characterize the context of an observation (e.g., space and time), as well as dependencies such as nested experimental observations. It includes an extensive set of unit definitions (e.g., grams of carbon per liter of seawater), and can facilitate automatic unit conversions (e.g., pounds to kilograms). OBOE can be easily extended to create Entities and Characteristics for specific research domains, making it both broadly applicable and highly customizable.\r\n\r\nOBOE is being used to improve data interpretation, facilitate reuse, and create precise and effective systems for data search and discovery.",
+            "documentation": "https://github.com/NCEAS/oboe",
+            "domain": null,
+            "download_url": "media/ontologies/OBOE/4/oboe.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/NCEAS/oboe",
+            "id": 2937,
+            "indexed": true,
+            "max_children": 70,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 292,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 24,
+            "products": null,
+            "publication": "Madin, J., S. Bowers, M. Schildhauer, S. Krivov, D. Pennington, and F. Villa. 2007. An ontology for describing and synthesizing ecological observation data. Ecological Informatics 2:279\u2013296. doi:10.1016/j.ecoinf.2007.05.004",
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "Version 1.2"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ZFS",
         "name": "Zebrafish developmental stages ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 27,
             "classifiable": true,
@@ -35301,1264 +35087,1341 @@
             "submission_id": 15,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OGG",
-        "name": "The Ontology of Genes and Genomes",
+        "acronym": "OBIB",
+        "name": "Ontology for Biobanking",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 810,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2017-12-13T10:10:30.509274Z",
-            "date_released": "2017-12-13T10:10:30.509253Z",
-            "description": "A formal ontology of genes and genomes of biological organisms.",
+            "date_created": "2023-05-15T15:17:32.258356Z",
+            "date_released": "2023-05-15T15:17:32.258343Z",
+            "description": "An ontology built for annotation and modeling of biobank repository and biobanking administration",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OGG/4/ogg.owl",
+            "domain": "investigations",
+            "download_url": "media/ontologies/OBIB/15/obib.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://bitbucket.org/hegroup/ogg",
-            "id": 1039,
+            "home_page": "https://github.com/biobanking/biobanking",
+            "id": 5811,
             "indexed": true,
-            "max_children": 20710,
-            "max_depth": 12,
-            "md5sum": "694a56219122d68e6b29c7c0be9ec052",
-            "nb_classes": 69689,
+            "max_children": 129,
+            "max_depth": 16,
+            "md5sum": "d834e37495f377aeaab15271c0cb18bb",
+            "nb_classes": 1804,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 81,
+            "nb_individuals": 226,
+            "nb_properties": 93,
             "products": [
                 {
-                    "id": "ogg.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ogg.owl"
+                    "id": "obib.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/obib.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 15,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OGG-MM",
-        "name": "Ontology of Genes and Genomes - Mouse",
+        "acronym": "NOMEN",
+        "name": "NOMEN - A nomenclatural ontology for biological names",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1876,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-12-02T19:21:57Z",
-            "date_released": "2016-12-02T00:00:00Z",
-            "description": "OGG-Mm is the OGG Mus musculus (mouse) subset. The OGG (Ontology of Genes and Genomes) is a formal ontology of genes and genomes of biological organisms. OGG is developed by following OBO Foundry principles and aligning with the BFO top ontology. ",
+            "date_created": "2023-04-24T12:46:34.715261Z",
+            "date_released": "2023-04-24T12:46:34.715249Z",
+            "description": "NOMEN is a nomenclatural ontology for biological names (not concepts).  It encodes the goverened rules of nomenclature.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OGG-MM/2/ogg-mm.owl",
+            "domain": "information",
+            "download_url": "media/ontologies/NOMEN/8/nomen.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://bitbucket.org/hegroup/ogg-mouse",
-            "id": 580,
+            "home_page": "https://github.com/SpeciesFileGroup/nomen",
+            "id": 5780,
             "indexed": true,
-            "max_children": 29666,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 69433,
+            "max_children": 22,
+            "max_depth": 8,
+            "md5sum": "228bf60a5bf213cf567ca6798d288e54",
+            "nb_classes": 263,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 81,
-            "products": null,
+            "nb_properties": 112,
+            "products": [
+                {
+                    "description": "core ontology",
+                    "id": "nomen.owl",
+                    "is_canonical": true,
+                    "ontology_purl": "http://purl.obolibrary.org/obo/nomen.owl",
+                    "title": "NOMEN",
+                    "type": "owl:Ontology"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 8,
             "taxon": null,
-            "version": "Vision Release: 1.0.63"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OGMD",
-        "name": "Ontology of Glucose Metabolism Disorder",
+        "acronym": "OBI_BCGO",
+        "name": "Beta Cell Genomics Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-09-28T01:08:51Z",
-            "date_released": "2017-09-28T01:08:51Z",
-            "description": "Including the disease names. phenotypes and their classifications involved in Glucose Metabolism Disorder, Diabetes. (OBO and OWL format are available in sourceforge.)\r\nOWL file is downloadable as the different view here. ",
+            "date_created": "2015-08-21T01:07:04Z",
+            "date_released": "2015-08-21T01:07:04Z",
+            "description": "Beta Cell Genomics Ontology (BCGO) is an application ontology built for the Beta Cell Genomics database (http://genomics.betacell.org/gbco/) aiming to support database annotation, complicated semantic queries, and automated cell type classification. The ontology is developed using Basic Formal Ontology (BFO) as upper ontology, Ontology for Biomedical Investigations (OBI) as ontology framework and integrated subsets of multiple OBO Foundry (candidate) ontologies. Current the BCGO contains 2383 classes including terms referencing to 24 various OBO Foundry ontologies including CL, CLO, UBERON, GO, PRO, UO, etc.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OGMD/33/ogmd.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://sourceforge.net/projects/ogmd/",
-            "id": 104,
+            "download_url": "media/ontologies/OBI_BCGO/14/obi_bcgo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obi-bcgo/bcgo",
+            "id": 679,
             "indexed": true,
-            "max_children": 15,
-            "max_depth": 6,
+            "max_children": 129,
+            "max_depth": 18,
             "md5sum": null,
-            "nb_classes": 134,
+            "nb_classes": 2270,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 46,
+            "nb_properties": 107,
             "products": null,
-            "publication": "http://www.med.kobe-u.ac.jp/journal/contents/55/E53.pdf",
+            "publication": null,
             "publications": null,
-            "submission_id": 33,
+            "submission_id": 14,
             "taxon": null,
-            "version": "1"
+            "version": "2015-07-08"
         },
         "topics": null
     },
     {
-        "acronym": "OGMS",
-        "name": "Ontology for General Medical Science",
+        "acronym": "OBI",
+        "name": "Ontology for Biomedical Investigations",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-10-25T15:58:26.977343Z",
-            "date_released": "2021-10-25T15:58:26.977322Z",
-            "description": "An ontology for representing treatment of disease and diagnosis and on carcinomas and other pathological entities",
+            "date_created": "2023-07-31T13:19:25.761348Z",
+            "date_released": "2023-07-31T13:19:25.761338Z",
+            "description": "An integrated ontology for the description of life-science and clinical investigations",
             "documentation": null,
-            "domain": "medicine",
-            "download_url": "media/ontologies/OGMS/19/ogms.owl",
+            "domain": "investigations",
+            "download_url": "media/ontologies/OBI/51/obi.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/OGMS/ogms",
-            "id": 4544,
+            "home_page": "http://obi-ontology.org",
+            "id": 5972,
             "indexed": true,
-            "max_children": 12,
-            "max_depth": 11,
-            "md5sum": "d988a3b17e385b51878a591bbd496aa5",
-            "nb_classes": 187,
+            "max_children": 107,
+            "max_depth": 18,
+            "md5sum": "8c63c379b5cfddb67bc82ac6892a591b",
+            "nb_classes": 4887,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 0,
+            "nb_individuals": 306,
+            "nb_properties": 98,
             "products": [
                 {
-                    "id": "ogms.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ogms.owl"
+                    "description": "The full version of OBI in OWL format",
+                    "id": "obi.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/obi.owl",
+                    "title": "OBI"
+                },
+                {
+                    "description": "The OBO-format version of OBI",
+                    "id": "obi.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/obi.obo",
+                    "title": "OBI in OBO"
+                },
+                {
+                    "description": "A collection of important high-level terms and their relations from OBI and other ontologies",
+                    "id": "obi/obi_core.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/obi/obi_core.owl",
+                    "title": "OBI Core"
+                },
+                {
+                    "description": "Base module for OBI",
+                    "id": "obi/obi-base.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/obi/obi-base.owl",
+                    "title": "OBI Base module"
                 }
             ],
             "publication": null,
-            "publications": null,
-            "submission_id": 19,
-            "taxon": {
-                "id": "NCBITaxon:9606",
-                "label": "Homo sapiens"
-            },
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/27128319",
+                    "title": "The Ontology for Biomedical Investigations"
+                }
+            ],
+            "submission_id": 51,
+            "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OGR",
-        "name": "Ontology of Geographical Region",
+        "acronym": "OBI_IEDB_view",
+        "name": "IEDB View",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2009-05-20T02:55:37Z",
-            "date_released": "2008-03-08T08:00:00Z",
-            "description": "This OWL ontology classified the  geograhical regions related vocabularies extracted from UMLS. It used with other ontologies to represent the genetic susceptibility factors of diabetes.",
+            "date_created": "2012-01-27T20:31:42Z",
+            "date_released": "2012-01-27T08:00:00Z",
+            "description": "OBI view for IMMUNE EPITOPE DATABASE (IEDB) community. The OBI terms used by IEDB were tagged by inSubSet annotation with the value \"IEDB\". The IEDB friendly labels were provided as IEDB alternative term.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OGR/2/ogr.owl",
+            "download_url": "media/ontologies/OBI_IEDB_view/1/obi_iedb_view.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 329,
+            "id": 42,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 2,
+            "max_children": 332,
+            "max_depth": 40,
             "md5sum": null,
-            "nb_classes": 39,
+            "nb_classes": 3519,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 149,
+            "nb_properties": 77,
             "products": null,
-            "publication": "http://www.med.kobe-u.ac.jp/journal/contents/55/E53.pdf",
+            "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": "2011-12-13"
         },
         "topics": null
     },
     {
-        "acronym": "OCMR",
-        "name": "Ontology of Chinese Medicine for Rheumatism",
+        "acronym": "ODNAE",
+        "name": "Ontology of Drug Neuropathy Adverse Events",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2017-04-21T01:08:03Z",
-            "date_released": "2017-04-21T01:08:03Z",
-            "description": "OCMR is a biomedical ontology that represents anti-rheumatism traditional Chinese medicines and related information.",
+            "date_created": "2016-05-17T01:09:11Z",
+            "date_released": "2016-05-17T01:09:11Z",
+            "description": "ODNAE is an extension of OAE with a focus on drug-associated neuropathy adverse events.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OCMR/3/ocmr.owl",
+            "download_url": "media/ontologies/ODNAE/3/odnae.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/biomedontology/ocmr",
-            "id": 599,
+            "home_page": "https://github.com/odnae/odnae",
+            "id": 463,
             "indexed": true,
-            "max_children": 165,
-            "max_depth": 26,
+            "max_children": 213,
+            "max_depth": 22,
             "md5sum": null,
-            "nb_classes": 3359,
+            "nb_classes": 1544,
             "nb_inconsistent": 0,
             "nb_individuals": 10,
-            "nb_properties": 112,
+            "nb_properties": 90,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 3,
             "taxon": null,
-            "version": "Vision Release; 1.0.11"
+            "version": "Vision Release; 1.0.13"
         },
         "topics": null
     },
     {
-        "acronym": "OCVDAE",
-        "name": "Ontology of Cardiovascular Drug Adverse Events",
+        "acronym": "OFSMR",
+        "name": "Open Predictive Microbiology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2016-12-29T23:50:23Z",
-            "date_released": "2016-12-29T00:00:00Z",
-            "description": "OCVDAE is a biomedical ontology of cardiovascular disease drug-associated adverse events",
+            "date_created": "2017-11-22T10:55:11Z",
+            "date_released": "2015-11-23T00:00:00Z",
+            "description": "Food-matrix ontology for Open Food Safety Model Repository",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OCVDAE/2/ocvdae.owl",
+            "download_url": "media/ontologies/OFSMR/19/ofsmr.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 748,
+            "home_page": "https://sites.google.com/site/openfsmr/open-fsmr-detailfilter",
+            "id": 843,
             "indexed": true,
-            "max_children": 58,
-            "max_depth": 17,
+            "max_children": 14,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 2948,
+            "nb_classes": 156,
             "nb_inconsistent": 0,
-            "nb_individuals": 89,
-            "nb_properties": 141,
+            "nb_individuals": 0,
+            "nb_properties": 2,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 19,
             "taxon": null,
-            "version": "Vision Release; 1.0.4"
+            "version": "0.4"
         },
         "topics": null
     },
     {
-        "acronym": "ODAE",
-        "name": "Ontology of Drug Adverse Events",
+        "acronym": "OGG",
+        "name": "The Ontology of Genes and Genomes",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 810,
             "classifiable": true,
-            "date_created": "2019-01-12T18:33:03Z",
-            "date_released": "2019-01-12T00:00:00Z",
-            "description": "The Ontology of Drug Adverse Events (ODAE) is a biomedical ontology in the area of drug adverse events, developed by following OBO Foundry principles (e.g., openness, collaboration). ",
+            "date_created": "2017-12-13T10:10:30.509274Z",
+            "date_released": "2017-12-13T10:10:30.509253Z",
+            "description": "A formal ontology of genes and genomes of biological organisms.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ODAE/1/odae.owl",
+            "download_url": "media/ontologies/OGG/4/ogg.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ODAE-ontology/ODAE",
-            "id": 2175,
+            "home_page": "https://bitbucket.org/hegroup/ogg",
+            "id": 1039,
             "indexed": true,
-            "max_children": 57,
-            "max_depth": 15,
-            "md5sum": null,
-            "nb_classes": 3672,
+            "max_children": 20710,
+            "max_depth": 12,
+            "md5sum": "694a56219122d68e6b29c7c0be9ec052",
+            "nb_classes": 69689,
             "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 130,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 81,
+            "products": [
+                {
+                    "id": "ogg.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ogg.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "Vision Release: 1.0.08"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ODHT",
-        "name": "Digital Health Technologies",
+        "acronym": "ONL-DP",
+        "name": "Dataset processing",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2022-09-19T15:54:26Z",
-            "date_released": "2022-08-30T00:00:00Z",
-            "description": "An ontology of digital health technologies (DHTs) with a specific focus on DHT use in clinical trials.",
-            "documentation": null,
+            "date_created": "2013-07-10T05:50:13Z",
+            "date_released": "2013-07-10T07:00:00Z",
+            "description": "This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging.\r\nIf covers the domain of Datasets and the processing of datasets. It includes a detailed taxonomy of datasets in the area of neuroimaging (and especially MR imaging) as well as a taxonomy of medical image processing.",
+            "documentation": "http://neurolog.unice.fr/ontoneurolog/v3.0/OntoSpec-documentation.zip",
             "domain": null,
-            "download_url": "media/ontologies/ODHT/2/odht.owl",
+            "download_url": "media/ontologies/ONL-DP/1/onl-dp.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5362,
+            "home_page": "http://neurolog.unice.fr/ontoneurolog/v3.0/Documentation_OntoNeuroLOGv3.pdf",
+            "id": 596,
             "indexed": true,
-            "max_children": 9,
-            "max_depth": 3,
+            "max_children": 23,
+            "max_depth": 13,
             "md5sum": null,
-            "nb_classes": 39,
+            "nb_classes": 541,
             "nb_inconsistent": 0,
-            "nb_individuals": 781,
-            "nb_properties": 6,
+            "nb_individuals": 0,
+            "nb_properties": 220,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.2"
+            "version": "V1"
         },
         "topics": null
     },
     {
-        "acronym": "OGMD_OWL",
-        "name": "Ontology of Glucose Metabolism Disorder OWL View",
+        "acronym": "ONLIRA",
+        "name": "Ontology of Liver for Radiology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
             "avg_children": 3,
-            "classifiable": false,
-            "date_created": "2011-10-20T14:39:19Z",
-            "date_released": "2011-10-10T07:00:00Z",
-            "description": "OWL file of the ontology",
-            "documentation": null,
+            "classifiable": true,
+            "date_created": "2016-02-02T00:39:20Z",
+            "date_released": "2015-01-26T18:18:13Z",
+            "description": "ONLIRA (Ontology of the Liver for Radiology) focuses on a semantic specification of imaging observations of CT scans for the liver. ONLIRA extends RadLex with semantic relationships that describe and relate the concepts. Thus, automated processing tasks, such as identifying similar patients, are supported.",
+            "documentation": "http://vavlab.ee.boun.edu.tr/pages.php?p=research/CARERA/carera.html",
             "domain": null,
-            "download_url": "media/ontologies/OGMD_OWL/1/ogmd_owl.owl",
+            "download_url": "media/ontologies/ONLIRA/4/onlira.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://sourceforge.net/projects/ogmd/",
-            "id": 277,
-            "indexed": false,
-            "max_children": 16,
-            "max_depth": 12,
+            "home_page": "http://vavlab.ee.boun.edu.tr/pages.php?p=research/CARERA/carera.html",
+            "id": 431,
+            "indexed": true,
+            "max_children": 9,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 175,
+            "nb_classes": 56,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_properties": 49,
             "products": null,
-            "publication": "http://www.med.kobe-u.ac.jp/journal/contents/55/E53.pdf",
+            "publication": "http://vavlab.ee.boun.edu.tr/pages.php?p=research/CARERA/carera.html",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "3.0"
+            "version": "changes\n* Segments and Regions are modelling as Classes in order to identify which patient (liver) they belong to.\n* Redundant restrictions are removed\n"
         },
         "topics": null
     },
     {
-        "acronym": "ONTODM-ALGORITHM",
-        "name": "OntoDM-core/algorithms",
+        "acronym": "ONL-MR-DA",
+        "name": "Magnetic Resonance Dataset Acquisition Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 19,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2022-08-30T14:21:25Z",
-            "date_released": "2022-06-06T00:00:00Z",
-            "description": "OntoDM-algorithms is an ontology for data mining and machine learning entities.",
-            "documentation": null,
+            "date_created": "2013-07-10T05:53:55Z",
+            "date_released": "2013-07-10T07:00:00Z",
+            "description": "This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeneous resources in neuroimaging.\r\nIf covers the domain of Magnetic Resonance Imaging (MRI) dataset acquisition, i.e. MRI protocols, and MRI sequence parameters. In particular, it includes a multi-axial classification of MR sequences.",
+            "documentation": "http://neurolog.unice.fr/ontoneurolog/v3.0/OntoSpec-documentation.zip",
             "domain": null,
-            "download_url": "media/ontologies/ONTODM-ALGORITHM/2/ontodm-algorithm.owl",
+            "download_url": "media/ontologies/ONL-MR-DA/1/onl-mr-da.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5332,
+            "home_page": "http://neurolog.unice.fr/ontoneurolog/v3.0/Documentation_OntoNeuroLOGv3.pdf",
+            "id": 461,
             "indexed": true,
-            "max_children": 31,
-            "max_depth": 11,
+            "max_children": 38,
+            "max_depth": 13,
             "md5sum": null,
-            "nb_classes": 377,
-            "nb_inconsistent": 31,
-            "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_classes": 702,
+            "nb_inconsistent": 0,
+            "nb_individuals": 42,
+            "nb_properties": 244,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.1"
+            "version": "V1"
         },
         "topics": null
     },
     {
-        "acronym": "PROCCHEMICAL",
-        "name": "Chemical Process",
+        "acronym": "ORDO_ES",
+        "name": "ORDO SPANISH",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2016-01-07T21:36:12Z",
-            "date_released": "2016-01-08T00:00:00Z",
-            "description": "Chemical process",
-            "documentation": null,
+            "date_created": "2023-07-03T15:05:26Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "This ontology is the spanish version of ORDO with definition, labels and synonyms in spanish.",
+            "documentation": "https://www.orphadata.com/ordo/",
             "domain": null,
-            "download_url": "media/ontologies/PROCCHEMICAL/1/procchemical.owl",
+            "download_url": "media/ontologies/ORDO_ES/11/ordo_es.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 398,
+            "home_page": "https://www.orphadata.com/ordo/",
+            "id": 5925,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 12,
+            "max_children": 4370,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 3535,
+            "nb_classes": 15447,
             "nb_inconsistent": 0,
-            "nb_individuals": 1226,
-            "nb_properties": 358,
+            "nb_individuals": 0,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 11,
             "taxon": null,
-            "version": "2.3"
+            "version": "4.3"
         },
         "topics": null
     },
     {
-        "acronym": "ONTODM-KDD",
-        "name": "Ontology of Data Mining Investigations",
+        "acronym": "ONL-MSA",
+        "name": "Mental State Assessment",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 9,
             "classifiable": true,
-            "date_created": "2012-06-06T12:37:22Z",
-            "date_released": "2012-06-06T07:00:00Z",
-            "description": "OntoDM-KDD is an ontology for representing data mining investigations. Its goal is to allow the representation of knowledge discovery processes and be general enough to represent the data mining investigations. The ontology is based on the CRISP-DM process methodology.",
-            "documentation": "www.ontodm.com",
+            "date_created": "2013-07-09T15:22:06Z",
+            "date_released": "2013-07-09T07:00:00Z",
+            "description": "This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging.\r\nThis MSA ontology module covers the field of mental state assessments, i.e. instruments, instrument variables, assessments, and resulting scores. It includes a generic domain core ontology, that provides a general model of such entities and a general taxonomy of behavioural, neurosychological and neuroclinical instruments, that can be easily extended to model any particular kind of instrument. It also includes such extensions for 8 relatively standard instruments, namely: (1) the Beck-depression-inventory-(BDI-II), (2) the Expanded-Disability-Status-Scale, (3) the Controlled-oral-word-association-test, (4) the Free-and-Cued-Selective-Reminding-Test-with-Immediate-Recall-16-item-version-(The-Gr\u00f6ber-and-Buschke-test), (5) the Mini-Mental-State, (6) the Stroop-color-and-word-test, (7) the Trail-making-test-(TMT), (8) the Wechsler-Adult-Intelligence-Scale-third-edition, (9) the Clinical-Dementia-Rating-scale, (10) the Category-verbal-fluency, (11) the Rey-Osterrieth-Complex-Figure-Test-(CFT).\r\n",
+            "documentation": "http://neurolog.unice.fr/ontoneurolog/v3.0/OntoSpec-documentation.zip",
             "domain": null,
-            "download_url": "media/ontologies/ONTODM-KDD/1/ontodm-kdd.owl",
+            "download_url": "media/ontologies/ONL-MSA/1/onl-msa.owl",
             "has_ontology_language": "OWL",
-            "home_page": "www.ontodm.com",
-            "id": 61,
+            "home_page": "http://neurolog.unice.fr/ontoneurolog/v3.0/Documentation_OntoNeuroLOGv3.pdf",
+            "id": 730,
             "indexed": true,
-            "max_children": 25,
-            "max_depth": 4,
+            "max_children": 119,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 264,
-            "nb_inconsistent": 10,
-            "nb_individuals": 0,
-            "nb_properties": 34,
+            "nb_classes": 769,
+            "nb_inconsistent": 0,
+            "nb_individuals": 581,
+            "nb_properties": 249,
             "products": null,
-            "publication": "www.ontodm.com",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.1"
+            "version": "v1"
         },
         "topics": null
     },
     {
-        "acronym": "ONTOKBCF",
-        "name": "Ontological Knowledge Base Model for Cystic Fibrosis",
+        "acronym": "ONL-TASKS",
+        "name": "Cognitive Reserves Assessment Tasks",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2013-01-16T18:11:04Z",
-            "date_released": "2013-01-14T08:00:00Z",
-            "description": "OntoKBCF is an ontological knowledge base model for cystic fibrosis. There are molecular genetic information (i.e. gene mutations) and health information included in OntoKBCF. The purposes of OntoKBCF include management of molecular genetic information and health information and embedding OntoKBCF into EHR settings.",
+            "date_created": "2020-10-15T08:18:04Z",
+            "date_released": "2020-10-15T00:00:00Z",
+            "description": "This ontology is an extension of the OntoNeuroLOG ontology, developed in the context of a project supported by Russian Foundation for Basic Research (19-29-01017 grant). The ontology organizes tests that are used for assessing cognitive reserves with the focus on keywords that can be employed in mining of PubMed publications.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ONTOKBCF/3/ontokbcf.owl",
+            "download_url": "media/ontologies/ONL-TASKS/1/onl-tasks.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 84,
+            "id": 4061,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 9,
+            "max_children": 11,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 408,
+            "nb_classes": 47,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 35,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "http://ceur-ws.org/Vol-2648/paper3.pdf",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1301"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ORDO_DE",
-        "name": "ORDO GERMAN",
+        "acronym": "ONS",
+        "name": "Ontology for Nutritional Studies",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-07-03T15:03:41Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "This ontology is the german version of ORDO with definition, labels and synonyms in german.",
-            "documentation": "https://www.orphadata.com/ordo/",
-            "domain": null,
-            "download_url": "media/ontologies/ORDO_DE/13/ordo_de.owl",
+            "date_created": "2023-06-05T13:49:55.557264Z",
+            "date_released": "2023-06-05T13:49:55.557247Z",
+            "description": "An ontology for description of concepts in the nutritional studies domain.",
+            "documentation": null,
+            "domain": "diet, metabolomics, and nutrition",
+            "download_url": "media/ontologies/ONS/25/ons.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com/ordo/",
-            "id": 5921,
+            "home_page": "https://github.com/enpadasi/Ontology-for-Nutritional-Studies",
+            "id": 5846,
             "indexed": true,
-            "max_children": 4370,
-            "max_depth": 11,
-            "md5sum": null,
-            "nb_classes": 15447,
+            "max_children": 154,
+            "max_depth": 24,
+            "md5sum": "5aa51bb45d4e7317c8d4b46593bd4130",
+            "nb_classes": 6056,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 13,
-            "products": null,
+            "nb_individuals": 133,
+            "nb_properties": 117,
+            "products": [
+                {
+                    "id": "ons.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ons.owl",
+                    "title": "ONS latest release"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 13,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/29736190",
+                    "title": "ONS: an ontology for a standardized description of interventions and observational studies in nutrition"
+                }
+            ],
+            "submission_id": 25,
             "taxon": null,
-            "version": "4.3"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ORDO_ES",
-        "name": "ORDO SPANISH",
-        "species": null,
+        "acronym": "ONTOAD",
+        "name": "Bilingual Ontology of Alzheimer's Disease and Related Diseases",
+        "species": [],
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-07-03T15:05:26Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "This ontology is the spanish version of ORDO with definition, labels and synonyms in spanish.",
-            "documentation": "https://www.orphadata.com/ordo/",
+            "date_created": "2013-12-02T18:00:14Z",
+            "date_released": "2013-08-10T00:00:00Z",
+            "description": "OntoAD is a bilingual (English-French) domain ontology for modeling knowledge about Alzheimer's Disease and Related Syndromes.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ORDO_ES/11/ordo_es.owl",
+            "download_url": "media/ontologies/ONTOAD/2/ontoad.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com/ordo/",
-            "id": 5925,
+            "home_page": null,
+            "id": 26,
             "indexed": true,
-            "max_children": 4370,
-            "max_depth": 11,
+            "max_children": 173,
+            "max_depth": 34,
             "md5sum": null,
-            "nb_classes": 15447,
+            "nb_classes": 5899,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_individuals": 2465,
+            "nb_properties": 182,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 2,
             "taxon": null,
-            "version": "4.3"
+            "version": "1.0"
         },
-        "topics": null
+        "topics": []
     },
     {
-        "acronym": "TXPO",
-        "name": "Toxic Process Ontology",
+        "acronym": "OGMD_OWL",
+        "name": "Ontology of Glucose Metabolism Disorder OWL View",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
             "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2022-12-12T13:55:32.258325Z",
-            "date_released": "2022-12-12T13:55:32.258303Z",
-            "description": "TOXic Process Ontology (TXPO) systematizes a wide variety of terms involving toxicity courses and processes. The first version of TXPO focuses on liver toxicity.",
+            "classifiable": false,
+            "date_created": "2011-10-20T14:39:19Z",
+            "date_released": "2011-10-10T07:00:00Z",
+            "description": "OWL file of the ontology",
             "documentation": null,
-            "domain": "chemistry and biochemistry",
-            "download_url": "media/ontologies/TXPO/28/txpo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/OGMD_OWL/1/ogmd_owl.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://toxpilot.nibiohn.go.jp/",
-            "id": 5538,
-            "indexed": true,
-            "max_children": 723,
-            "max_depth": 17,
-            "md5sum": "fa876b76cc85363e179460a91cf6992a",
-            "nb_classes": 9407,
+            "home_page": "https://sourceforge.net/projects/ogmd/",
+            "id": 277,
+            "indexed": false,
+            "max_children": 16,
+            "max_depth": 12,
+            "md5sum": null,
+            "nb_classes": 175,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 82,
-            "products": [
-                {
-                    "id": "txpo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/txpo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/32883995",
-                    "title": "Ontological approach to the knowledge systematization of a toxic process and toxic course representation framework for early drug risk management"
-                }
-            ],
-            "submission_id": 28,
+            "nb_properties": 2,
+            "products": null,
+            "publication": "http://www.med.kobe-u.ac.jp/journal/contents/55/E53.pdf",
+            "publications": null,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "3.0"
         },
         "topics": null
     },
     {
-        "acronym": "ODNAE",
-        "name": "Ontology of Drug Neuropathy Adverse Events",
+        "acronym": "PROCCHEMICAL",
+        "name": "Chemical Process",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-05-17T01:09:11Z",
-            "date_released": "2016-05-17T01:09:11Z",
-            "description": "ODNAE is an extension of OAE with a focus on drug-associated neuropathy adverse events.",
+            "date_created": "2016-01-07T21:36:12Z",
+            "date_released": "2016-01-08T00:00:00Z",
+            "description": "Chemical process",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ODNAE/3/odnae.owl",
+            "download_url": "media/ontologies/PROCCHEMICAL/1/procchemical.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/odnae/odnae",
-            "id": 463,
+            "home_page": null,
+            "id": 398,
             "indexed": true,
-            "max_children": 213,
-            "max_depth": 22,
+            "max_children": 51,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 1544,
+            "nb_classes": 3535,
             "nb_inconsistent": 0,
-            "nb_individuals": 10,
-            "nb_properties": 90,
+            "nb_individuals": 1226,
+            "nb_properties": 358,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "Vision Release; 1.0.13"
+            "version": "2.3"
         },
         "topics": null
     },
     {
-        "acronym": "ONE",
-        "name": "Ontology for Nutritional Epidemiology",
+        "acronym": "ONTOAVIDA",
+        "name": "OntoAvida: ontology for Avida digital evolution platform",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 23,
             "classifiable": true,
-            "date_created": "2021-03-29T12:45:05.324325Z",
-            "date_released": "2021-03-29T12:45:05.324311Z",
-            "description": "An ontology to standardize research output of nutritional epidemiologic studies.",
+            "date_created": "2022-03-21T12:32:23.245416Z",
+            "date_released": "2022-03-21T12:32:23.245402Z",
+            "description": "OntoAvida develops an integrated vocabulary for the description of the most widely-used computational approach for studying evolution using digital organisms (i.e., self-replicating computer programs that evolve within a user-defined computational environment).",
             "documentation": null,
-            "domain": "nutritional epidemiology, observational studies, dietary surveys",
-            "download_url": "media/ontologies/ONE/9/one.owl",
+            "domain": "simulation",
+            "download_url": "media/ontologies/ONTOAVIDA/3/ontoavida.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/cyang0128/Nutritional-epidemiologic-ontologies",
-            "id": 4370,
+            "home_page": "https://gitlab.com/fortunalab/ontoavida",
+            "id": 4965,
             "indexed": true,
-            "max_children": 81,
-            "max_depth": 17,
-            "md5sum": "b7d85b1d9b5edc5cdfb549627e488a34",
-            "nb_classes": 3835,
+            "max_children": 512,
+            "max_depth": 4,
+            "md5sum": "8f3cd2b5b6d520fd8f81d33edc237dc6",
+            "nb_classes": 675,
             "nb_inconsistent": 0,
-            "nb_individuals": 303,
-            "nb_properties": 102,
+            "nb_individuals": 0,
+            "nb_properties": 22,
             "products": [
                 {
-                    "id": "one.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/one.owl"
+                    "id": "ontoavida.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ontoavida.owl"
+                },
+                {
+                    "id": "ontoavida.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ontoavida.obo"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 9,
+            "submission_id": 3,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OFSMR",
-        "name": "Open Predictive Microbiology Ontology",
+        "acronym": "OGG-MM",
+        "name": "Ontology of Genes and Genomes - Mouse",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 1876,
             "classifiable": true,
-            "date_created": "2017-11-22T10:55:11Z",
-            "date_released": "2015-11-23T00:00:00Z",
-            "description": "Food-matrix ontology for Open Food Safety Model Repository",
+            "date_created": "2016-12-02T19:21:57Z",
+            "date_released": "2016-12-02T00:00:00Z",
+            "description": "OGG-Mm is the OGG Mus musculus (mouse) subset. The OGG (Ontology of Genes and Genomes) is a formal ontology of genes and genomes of biological organisms. OGG is developed by following OBO Foundry principles and aligning with the BFO top ontology. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OFSMR/19/ofsmr.owl",
+            "download_url": "media/ontologies/OGG-MM/2/ogg-mm.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://sites.google.com/site/openfsmr/open-fsmr-detailfilter",
-            "id": 843,
+            "home_page": "https://bitbucket.org/hegroup/ogg-mouse",
+            "id": 580,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 7,
+            "max_children": 29666,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 156,
+            "nb_classes": 69433,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_properties": 81,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 19,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.4"
+            "version": "Vision Release: 1.0.63"
         },
         "topics": null
     },
     {
-        "acronym": "ONL-DP",
-        "name": "Dataset processing",
+        "acronym": "OGMD",
+        "name": "Ontology of Glucose Metabolism Disorder",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2013-07-10T05:50:13Z",
-            "date_released": "2013-07-10T07:00:00Z",
-            "description": "This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging.\r\nIf covers the domain of Datasets and the processing of datasets. It includes a detailed taxonomy of datasets in the area of neuroimaging (and especially MR imaging) as well as a taxonomy of medical image processing.",
-            "documentation": "http://neurolog.unice.fr/ontoneurolog/v3.0/OntoSpec-documentation.zip",
+            "date_created": "2017-09-28T01:08:51Z",
+            "date_released": "2017-09-28T01:08:51Z",
+            "description": "Including the disease names. phenotypes and their classifications involved in Glucose Metabolism Disorder, Diabetes. (OBO and OWL format are available in sourceforge.)\r\nOWL file is downloadable as the different view here. ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ONL-DP/1/onl-dp.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://neurolog.unice.fr/ontoneurolog/v3.0/Documentation_OntoNeuroLOGv3.pdf",
-            "id": 596,
+            "download_url": "media/ontologies/OGMD/33/ogmd.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://sourceforge.net/projects/ogmd/",
+            "id": 104,
             "indexed": true,
-            "max_children": 23,
-            "max_depth": 13,
+            "max_children": 15,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 541,
+            "nb_classes": 134,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 220,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "http://www.med.kobe-u.ac.jp/journal/contents/55/E53.pdf",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 33,
             "taxon": null,
-            "version": "V1"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ONLIRA",
-        "name": "Ontology of Liver for Radiology",
+        "acronym": "OGMS",
+        "name": "Ontology for General Medical Science",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2016-02-02T00:39:20Z",
-            "date_released": "2015-01-26T18:18:13Z",
-            "description": "ONLIRA (Ontology of the Liver for Radiology) focuses on a semantic specification of imaging observations of CT scans for the liver. ONLIRA extends RadLex with semantic relationships that describe and relate the concepts. Thus, automated processing tasks, such as identifying similar patients, are supported.",
-            "documentation": "http://vavlab.ee.boun.edu.tr/pages.php?p=research/CARERA/carera.html",
-            "domain": null,
-            "download_url": "media/ontologies/ONLIRA/4/onlira.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://vavlab.ee.boun.edu.tr/pages.php?p=research/CARERA/carera.html",
-            "id": 431,
-            "indexed": true,
-            "max_children": 9,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 56,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 49,
-            "products": null,
-            "publication": "http://vavlab.ee.boun.edu.tr/pages.php?p=research/CARERA/carera.html",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "changes\n* Segments and Regions are modelling as Classes in order to identify which patient (liver) they belong to.\n* Redundant restrictions are removed\n"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "OGI",
-        "name": "Ontology for genetic interval",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2015-08-21T01:07:21Z",
-            "date_released": "2015-08-21T01:07:21Z",
-            "description": "Using BFO as its framwork, OGI formalized the genomic element by defining an upper class 'genetic interval'.\r\n\r\nThe definition of 'genetic interval' is \"the spatial continuous physical entity which contains ordered genomic sets(DNA, RNA, Allele, Marker,etc.) between and including two points (Nucleic_Acid_Base_Residue) on a chromosome or RNA molecule which must have a liner primary sequence sturcture.",
+            "date_created": "2021-10-25T15:58:26.977343Z",
+            "date_released": "2021-10-25T15:58:26.977322Z",
+            "description": "An ontology for representing treatment of disease and diagnosis and on carcinomas and other pathological entities",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OGI/18/ogi.owl",
+            "domain": "medicine",
+            "download_url": "media/ontologies/OGMS/19/ogms.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://code.google.com/p/ontology-for-genetic-interval/",
-            "id": 438,
+            "home_page": "https://github.com/OGMS/ogms",
+            "id": 4544,
             "indexed": true,
-            "max_children": 13,
+            "max_children": 12,
             "max_depth": 11,
-            "md5sum": null,
-            "nb_classes": 184,
+            "md5sum": "d988a3b17e385b51878a591bbd496aa5",
+            "nb_classes": 187,
             "nb_inconsistent": 0,
-            "nb_individuals": 23,
-            "nb_properties": 59,
-            "products": null,
+            "nb_individuals": 18,
+            "nb_properties": 0,
+            "products": [
+                {
+                    "id": "ogms.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ogms.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 18,
-            "taxon": null,
-            "version": "2.0"
+            "submission_id": 19,
+            "taxon": {
+                "id": "NCBITaxon:9606",
+                "label": "Homo sapiens"
+            },
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ONL-MR-DA",
-        "name": "Magnetic Resonance Dataset Acquisition Ontology",
+        "acronym": "OGR",
+        "name": "Ontology of Geographical Region",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2013-07-10T05:53:55Z",
-            "date_released": "2013-07-10T07:00:00Z",
-            "description": "This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeneous resources in neuroimaging.\r\nIf covers the domain of Magnetic Resonance Imaging (MRI) dataset acquisition, i.e. MRI protocols, and MRI sequence parameters. In particular, it includes a multi-axial classification of MR sequences.",
-            "documentation": "http://neurolog.unice.fr/ontoneurolog/v3.0/OntoSpec-documentation.zip",
+            "date_created": "2009-05-20T02:55:37Z",
+            "date_released": "2008-03-08T08:00:00Z",
+            "description": "This OWL ontology classified the  geograhical regions related vocabularies extracted from UMLS. It used with other ontologies to represent the genetic susceptibility factors of diabetes.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ONL-MR-DA/1/onl-mr-da.owl",
+            "download_url": "media/ontologies/OGR/2/ogr.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://neurolog.unice.fr/ontoneurolog/v3.0/Documentation_OntoNeuroLOGv3.pdf",
-            "id": 461,
+            "home_page": null,
+            "id": 329,
             "indexed": true,
-            "max_children": 38,
-            "max_depth": 13,
+            "max_children": 14,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 702,
+            "nb_classes": 39,
             "nb_inconsistent": 0,
-            "nb_individuals": 42,
-            "nb_properties": 244,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "http://www.med.kobe-u.ac.jp/journal/contents/55/E53.pdf",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "V1"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "ONL-MSA",
-        "name": "Mental State Assessment",
+        "acronym": "OCMR",
+        "name": "Ontology of Chinese Medicine for Rheumatism",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 9,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2013-07-09T15:22:06Z",
-            "date_released": "2013-07-09T07:00:00Z",
-            "description": "This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging.\r\nThis MSA ontology module covers the field of mental state assessments, i.e. instruments, instrument variables, assessments, and resulting scores. It includes a generic domain core ontology, that provides a general model of such entities and a general taxonomy of behavioural, neurosychological and neuroclinical instruments, that can be easily extended to model any particular kind of instrument. It also includes such extensions for 8 relatively standard instruments, namely: (1) the Beck-depression-inventory-(BDI-II), (2) the Expanded-Disability-Status-Scale, (3) the Controlled-oral-word-association-test, (4) the Free-and-Cued-Selective-Reminding-Test-with-Immediate-Recall-16-item-version-(The-Gr\u00f6ber-and-Buschke-test), (5) the Mini-Mental-State, (6) the Stroop-color-and-word-test, (7) the Trail-making-test-(TMT), (8) the Wechsler-Adult-Intelligence-Scale-third-edition, (9) the Clinical-Dementia-Rating-scale, (10) the Category-verbal-fluency, (11) the Rey-Osterrieth-Complex-Figure-Test-(CFT).\r\n",
-            "documentation": "http://neurolog.unice.fr/ontoneurolog/v3.0/OntoSpec-documentation.zip",
+            "date_created": "2017-04-21T01:08:03Z",
+            "date_released": "2017-04-21T01:08:03Z",
+            "description": "OCMR is a biomedical ontology that represents anti-rheumatism traditional Chinese medicines and related information.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ONL-MSA/1/onl-msa.owl",
+            "download_url": "media/ontologies/OCMR/3/ocmr.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://neurolog.unice.fr/ontoneurolog/v3.0/Documentation_OntoNeuroLOGv3.pdf",
-            "id": 730,
+            "home_page": "https://github.com/biomedontology/ocmr",
+            "id": 599,
             "indexed": true,
-            "max_children": 119,
-            "max_depth": 10,
+            "max_children": 165,
+            "max_depth": 26,
             "md5sum": null,
-            "nb_classes": 769,
+            "nb_classes": 3359,
             "nb_inconsistent": 0,
-            "nb_individuals": 581,
-            "nb_properties": 249,
+            "nb_individuals": 10,
+            "nb_properties": 112,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "v1"
+            "version": "Vision Release; 1.0.11"
         },
         "topics": null
     },
     {
-        "acronym": "ONL-TASKS",
-        "name": "Cognitive Reserves Assessment Tasks",
+        "acronym": "ONE",
+        "name": "Ontology for Nutritional Epidemiology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2020-10-15T08:18:04Z",
-            "date_released": "2020-10-15T00:00:00Z",
-            "description": "This ontology is an extension of the OntoNeuroLOG ontology, developed in the context of a project supported by Russian Foundation for Basic Research (19-29-01017 grant). The ontology organizes tests that are used for assessing cognitive reserves with the focus on keywords that can be employed in mining of PubMed publications.",
+            "date_created": "2021-03-29T12:45:05.324325Z",
+            "date_released": "2021-03-29T12:45:05.324311Z",
+            "description": "An ontology to standardize research output of nutritional epidemiologic studies.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ONL-TASKS/1/onl-tasks.owl",
+            "domain": "nutritional epidemiology, observational studies, dietary surveys",
+            "download_url": "media/ontologies/ONE/9/one.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4061,
+            "home_page": "https://github.com/cyang0128/Nutritional-epidemiologic-ontologies",
+            "id": 4370,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 47,
+            "max_children": 81,
+            "max_depth": 17,
+            "md5sum": "b7d85b1d9b5edc5cdfb549627e488a34",
+            "nb_classes": 3835,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": "http://ceur-ws.org/Vol-2648/paper3.pdf",
+            "nb_individuals": 303,
+            "nb_properties": 102,
+            "products": [
+                {
+                    "id": "one.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/one.owl"
+                }
+            ],
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 9,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ONS",
-        "name": "Ontology for Nutritional Studies",
+        "acronym": "TXPO",
+        "name": "Toxic Process Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-06-05T13:49:55.557264Z",
-            "date_released": "2023-06-05T13:49:55.557247Z",
-            "description": "An ontology for description of concepts in the nutritional studies domain.",
+            "date_created": "2022-12-12T13:55:32.258325Z",
+            "date_released": "2022-12-12T13:55:32.258303Z",
+            "description": "TOXic Process Ontology (TXPO) systematizes a wide variety of terms involving toxicity courses and processes. The first version of TXPO focuses on liver toxicity.",
             "documentation": null,
-            "domain": "diet, metabolomics, and nutrition",
-            "download_url": "media/ontologies/ONS/25/ons.owl",
+            "domain": "chemistry and biochemistry",
+            "download_url": "media/ontologies/TXPO/28/txpo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/enpadasi/Ontology-for-Nutritional-Studies",
-            "id": 5846,
+            "home_page": "https://toxpilot.nibiohn.go.jp/",
+            "id": 5538,
             "indexed": true,
-            "max_children": 154,
-            "max_depth": 24,
-            "md5sum": "5aa51bb45d4e7317c8d4b46593bd4130",
-            "nb_classes": 6056,
+            "max_children": 723,
+            "max_depth": 17,
+            "md5sum": "fa876b76cc85363e179460a91cf6992a",
+            "nb_classes": 9407,
             "nb_inconsistent": 0,
-            "nb_individuals": 133,
-            "nb_properties": 117,
+            "nb_individuals": 0,
+            "nb_properties": 82,
             "products": [
                 {
-                    "id": "ons.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ons.owl",
-                    "title": "ONS latest release"
+                    "id": "txpo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/txpo.owl"
                 }
             ],
             "publication": null,
             "publications": [
                 {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/29736190",
-                    "title": "ONS: an ontology for a standardized description of interventions and observational studies in nutrition"
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/32883995",
+                    "title": "Ontological approach to the knowledge systematization of a toxic process and toxic course representation framework for early drug risk management"
                 }
             ],
-            "submission_id": 25,
+            "submission_id": 28,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ONTOAD",
-        "name": "Bilingual Ontology of Alzheimer's Disease and Related Diseases",
-        "species": [],
+        "acronym": "OCVDAE",
+        "name": "Ontology of Cardiovascular Drug Adverse Events",
+        "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2013-12-02T18:00:14Z",
-            "date_released": "2013-08-10T00:00:00Z",
-            "description": "OntoAD is a bilingual (English-French) domain ontology for modeling knowledge about Alzheimer's Disease and Related Syndromes.",
+            "date_created": "2016-12-29T23:50:23Z",
+            "date_released": "2016-12-29T00:00:00Z",
+            "description": "OCVDAE is a biomedical ontology of cardiovascular disease drug-associated adverse events",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ONTOAD/2/ontoad.owl",
+            "download_url": "media/ontologies/OCVDAE/2/ocvdae.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 26,
+            "id": 748,
             "indexed": true,
-            "max_children": 173,
-            "max_depth": 34,
+            "max_children": 58,
+            "max_depth": 17,
             "md5sum": null,
-            "nb_classes": 5899,
+            "nb_classes": 2948,
             "nb_inconsistent": 0,
-            "nb_individuals": 2465,
-            "nb_properties": 182,
+            "nb_individuals": 89,
+            "nb_properties": 141,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.0"
+            "version": "Vision Release; 1.0.4"
         },
-        "topics": []
+        "topics": null
     },
     {
-        "acronym": "ONTOAVIDA",
-        "name": "OntoAvida: ontology for Avida digital evolution platform",
+        "acronym": "ODAE",
+        "name": "Ontology of Drug Adverse Events",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 23,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-03-21T12:32:23.245416Z",
-            "date_released": "2022-03-21T12:32:23.245402Z",
-            "description": "OntoAvida develops an integrated vocabulary for the description of the most widely-used computational approach for studying evolution using digital organisms (i.e., self-replicating computer programs that evolve within a user-defined computational environment).",
+            "date_created": "2019-01-12T18:33:03Z",
+            "date_released": "2019-01-12T00:00:00Z",
+            "description": "The Ontology of Drug Adverse Events (ODAE) is a biomedical ontology in the area of drug adverse events, developed by following OBO Foundry principles (e.g., openness, collaboration). ",
             "documentation": null,
-            "domain": "simulation",
-            "download_url": "media/ontologies/ONTOAVIDA/3/ontoavida.owl",
+            "domain": null,
+            "download_url": "media/ontologies/ODAE/1/odae.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://gitlab.com/fortunalab/ontoavida",
-            "id": 4965,
+            "home_page": "https://github.com/ODAE-ontology/ODAE",
+            "id": 2175,
             "indexed": true,
-            "max_children": 512,
-            "max_depth": 4,
-            "md5sum": "8f3cd2b5b6d520fd8f81d33edc237dc6",
-            "nb_classes": 675,
+            "max_children": 57,
+            "max_depth": 15,
+            "md5sum": null,
+            "nb_classes": 3672,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 22,
-            "products": [
-                {
-                    "id": "ontoavida.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ontoavida.owl"
-                },
-                {
-                    "id": "ontoavida.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ontoavida.obo"
-                }
-            ],
+            "nb_individuals": 3,
+            "nb_properties": 130,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Vision Release: 1.0.08"
         },
         "topics": null
     },
     {
-        "acronym": "FB-SP",
-        "name": "Fly Taxonomy",
+        "acronym": "OGI",
+        "name": "Ontology for genetic interval",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2013-06-21T01:02:50Z",
-            "date_released": "2013-06-21T01:02:50Z",
-            "description": "The taxonomy of the family <i>Drosophilidae</i> (largely after Baechli) and of other taxa referred to in FlyBase.",
+            "date_created": "2015-08-21T01:07:21Z",
+            "date_released": "2015-08-21T01:07:21Z",
+            "description": "Using BFO as its framwork, OGI formalized the genomic element by defining an upper class 'genetic interval'.\r\n\r\nThe definition of 'genetic interval' is \"the spatial continuous physical entity which contains ordered genomic sets(DNA, RNA, Allele, Marker,etc.) between and including two points (Nucleic_Acid_Base_Residue) on a chromosome or RNA molecule which must have a liner primary sequence sturcture.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FB-SP/7/fb-sp.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.flybase.org/",
-            "id": 414,
+            "download_url": "media/ontologies/OGI/18/ogi.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://code.google.com/p/ontology-for-genetic-interval/",
+            "id": 438,
             "indexed": true,
-            "max_children": 207,
-            "max_depth": 35,
+            "max_children": 13,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 6600,
+            "nb_classes": 184,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 23,
+            "nb_properties": 59,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 18,
             "taxon": null,
-            "version": "unknown"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "FCC1",
-        "name": "Fernald Community Cohort N",
+        "acronym": "ODHT",
+        "name": "Digital Health Technologies",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2019-10-03T22:17:36Z",
-            "date_released": "2019-10-03T00:00:00Z",
-            "description": "Fernald Community Cohort meta data",
+            "date_created": "2022-09-19T15:54:26Z",
+            "date_released": "2022-08-30T00:00:00Z",
+            "description": "An ontology of digital health technologies (DHTs) with a specific focus on DHT use in clinical trials.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FCC1/1/fcc1.owl",
+            "download_url": "media/ontologies/ODHT/2/odht.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.semanticweb.org/diwaleva/ontologies/2019/9/fcc-ontology",
-            "id": 3296,
+            "home_page": null,
+            "id": 5362,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 2,
+            "max_children": 9,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 9,
+            "nb_classes": 39,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 3,
+            "nb_individuals": 781,
+            "nb_properties": 6,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0"
+            "version": "0.2"
         },
         "topics": null
     },
     {
-        "acronym": "FHHO",
-        "name": "Family Health History Ontology",
+        "acronym": "ONTODM-ALGORITHM",
+        "name": "OntoDM-core/algorithms",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 19,
             "classifiable": true,
-            "date_created": "2009-05-10T21:26:51Z",
-            "date_released": "2008-09-23T07:00:00Z",
-            "description": "The FHHO facilitates representing the family health histories of persons related by biological and/or social family relationships (e.g. step, adoptive) who share genetic, behavioral, and/or environmental risk factors for disease. SWRL rules are included to compute 3 generations of biological relationships based on parentage and family history findings based on personal health findings.",
+            "date_created": "2022-08-30T14:21:25Z",
+            "date_released": "2022-06-06T00:00:00Z",
+            "description": "OntoDM-algorithms is an ontology for data mining and machine learning entities.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FHHO/1/fhho.owl",
+            "download_url": "media/ontologies/ONTODM-ALGORITHM/2/ontodm-algorithm.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://healthsystems.engr.wisc.edu/Alumni/JanePeace.html",
-            "id": 60,
+            "home_page": null,
+            "id": 5332,
             "indexed": true,
-            "max_children": 63,
-            "max_depth": 6,
+            "max_children": 31,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 238,
-            "nb_inconsistent": 0,
-            "nb_individuals": 12,
-            "nb_properties": 433,
+            "nb_classes": 377,
+            "nb_inconsistent": 31,
+            "nb_individuals": 0,
+            "nb_properties": 1,
             "products": null,
-            "publication": "http://healthsystems.engr.wisc.edu/Alumni/JanePeace.html",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1"
+            "version": "0.1"
         },
         "topics": null
     },
     {
-        "acronym": "FOBI",
-        "name": "Food-Biomarker Ontology",
+        "acronym": "ONTODM-KDD",
+        "name": "Ontology of Data Mining Investigations",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 107,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2022-05-09T12:45:36.999726Z",
-            "date_released": "2022-05-09T12:45:36.999712Z",
-            "description": "FOBI (Food-Biomarker Ontology) is an ontology to represent food intake data and associate it with metabolomic data",
-            "documentation": null,
-            "domain": "diet, metabolomics, and nutrition",
-            "download_url": "media/ontologies/FOBI/7/fobi.owl",
+            "date_created": "2012-06-06T12:37:22Z",
+            "date_released": "2012-06-06T07:00:00Z",
+            "description": "OntoDM-KDD is an ontology for representing data mining investigations. Its goal is to allow the representation of knowledge discovery processes and be general enough to represent the data mining investigations. The ontology is based on the CRISP-DM process methodology.",
+            "documentation": "www.ontodm.com",
+            "domain": null,
+            "download_url": "media/ontologies/ONTODM-KDD/1/ontodm-kdd.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/pcastellanoescuder/FoodBiomarkerOntology",
-            "id": 5069,
+            "home_page": "www.ontodm.com",
+            "id": 61,
             "indexed": true,
-            "max_children": 148,
-            "max_depth": 9,
-            "md5sum": "894f2bbe2c41f1ac47e92e24b94d14dc",
-            "nb_classes": 1334,
-            "nb_inconsistent": 148,
+            "max_children": 25,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 264,
+            "nb_inconsistent": 10,
             "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": [
-                {
-                    "format": "owl-rdf/xml",
-                    "id": "fobi.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fobi.owl",
-                    "title": "FOBI is an ontology to represent food intake data and associate it with metabolomic data"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/32556148",
-                    "title": "FOBI: an ontology to represent food intake data and associate it with metabolomic data"
-                },
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34601570",
-                    "title": "The fobitools framework: the first steps towards food enrichment analysis"
-                }
-            ],
-            "submission_id": 7,
+            "nb_properties": 34,
+            "products": null,
+            "publication": "www.ontodm.com",
+            "publications": null,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "0.1"
         },
         "topics": null
     },
     {
-        "acronym": "foaf",
-        "name": "FOAF (people, organizations, groups)",
+        "acronym": "ONTOKBCF",
+        "name": "Ontological Knowledge Base Model for Cystic Fibrosis",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2012-08-31T10:20:55Z",
-            "date_released": "2012-07-12T07:00:00Z",
-            "description": "This file is imported by vivo-core-public-1.5.owl.  It contains terms relating to people, groups and organizations from the http://xmlns.com/foaf/0.1/ namespace that are included in the vivo ontology.",
+            "date_created": "2013-01-16T18:11:04Z",
+            "date_released": "2013-01-14T08:00:00Z",
+            "description": "OntoKBCF is an ontological knowledge base model for cystic fibrosis. There are molecular genetic information (i.e. gene mutations) and health information included in OntoKBCF. The purposes of OntoKBCF include management of molecular genetic information and health information and embedding OntoKBCF into EHR settings.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/foaf/1/foaf.owl",
+            "download_url": "media/ontologies/ONTOKBCF/3/ontokbcf.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
-            "id": 247,
+            "home_page": null,
+            "id": 84,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 1,
+            "max_children": 20,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 15,
+            "nb_classes": 408,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 24,
+            "nb_properties": 35,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.5"
+            "version": "1301"
         },
         "topics": null
     },
     {
-        "acronym": "FOAF",
-        "name": "Friend of a Friend Vocabulary",
+        "acronym": "ORDO_DE",
+        "name": "ORDO GERMAN",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2019-02-01T01:38:31Z",
-            "date_released": "2019-01-31T00:00:00Z",
-            "description": "Friend of a Friend Vocabulary",
-            "documentation": null,
+            "date_created": "2023-07-03T15:03:41Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "This ontology is the german version of ORDO with definition, labels and synonyms in german.",
+            "documentation": "https://www.orphadata.com/ordo/",
             "domain": null,
-            "download_url": "media/ontologies/FOAF/1/foaf.owl",
+            "download_url": "media/ontologies/ORDO_DE/13/ordo_de.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2240,
+            "home_page": "https://www.orphadata.com/ordo/",
+            "id": 5921,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 2,
+            "max_children": 4370,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 22,
+            "nb_classes": 15447,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 40,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 13,
             "taxon": null,
-            "version": "0.99"
+            "version": "4.3"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PCL",
+        "name": "Provisional Cell Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 26,
+            "classifiable": true,
+            "date_created": "2023-03-06T13:32:24.337058Z",
+            "date_released": "2023-03-06T13:32:24.337043Z",
+            "description": "Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties.",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/PCL/22/pcl.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obophenotype/provisional_cell_ontology",
+            "id": 5707,
+            "indexed": true,
+            "max_children": 110886,
+            "max_depth": 32,
+            "md5sum": "e0c8e6791bfae3dd934fa06d4200ceb1",
+            "nb_classes": 121029,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1825,
+            "nb_properties": 299,
+            "products": [
+                {
+                    "id": "pcl.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl.owl"
+                },
+                {
+                    "id": "pcl.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl.obo"
+                },
+                {
+                    "id": "pcl.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl.json"
+                },
+                {
+                    "id": "pcl-base.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-base.owl"
+                },
+                {
+                    "id": "pcl-base.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-base.obo"
+                },
+                {
+                    "id": "pcl-base.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-base.json"
+                },
+                {
+                    "id": "pcl-full.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-full.owl"
+                },
+                {
+                    "id": "pcl-full.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-full.obo"
+                },
+                {
+                    "id": "pcl-full.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-full.json"
+                },
+                {
+                    "id": "pcl-simple.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-simple.owl"
+                },
+                {
+                    "id": "pcl-simple.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-simple.obo"
+                },
+                {
+                    "id": "pcl-simple.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-simple.json"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 22,
+            "taxon": null,
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "FDC-GDMT",
         "name": "Ontology for Generic Dataset Metadata Template",
         "species": null,
@@ -36691,232 +36554,219 @@
             "submission_id": 7,
             "taxon": null,
             "version": "0.8"
         },
         "topics": null
     },
     {
-        "acronym": "FISH-AST",
-        "name": "FISH Archaeological Sciences Thesaurus",
+        "acronym": "FIDEO",
+        "name": "Food Interactions with Drugs Evidence Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 20,
             "classifiable": true,
-            "date_created": "2017-09-11T21:21:13Z",
-            "date_released": "2013-07-17T00:00:00Z",
-            "description": "Terminology used for recording the techniques, recovery methods and materials associated with archaeological sciences. Maintained by Historic England on behalf of the FISH Terminology Working Group.",
+            "date_created": "2023-02-06T13:00:09.675535Z",
+            "date_released": "2023-02-06T13:00:09.675518Z",
+            "description": "Food-Drug interactions automatically extracted from scientific literature",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/FISH-AST/1/fish-ast.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "http://heritagedata.org/live/schemes/560.html",
-            "id": 385,
+            "domain": "diet, metabolomics, and nutrition",
+            "download_url": "media/ontologies/FIDEO/7/fideo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://gitub.u-bordeaux.fr/erias/fideo",
+            "id": 5639,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 2,
+            "max_children": 1574,
+            "max_depth": 9,
+            "md5sum": "1e0f1977b36dc7681221f2c8e0ac7b93",
+            "nb_classes": 3548,
             "nb_inconsistent": 0,
-            "nb_individuals": 125,
-            "nb_properties": 0,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 12,
+            "products": [
+                {
+                    "id": "fideo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fideo.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 7,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "FISHO",
-        "name": "Fish Ontology",
+        "acronym": "FB-SP",
+        "name": "Fly Taxonomy",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-08-28T08:51:50Z",
-            "date_released": "2017-08-28T00:00:00Z",
-            "description": "This Fish Ontology is an ontology created as how the author views the fish structure following the book \"The Diversity of Fishes: Biology, Evolution and Ecology\" as the main reference, which covers one part of view on ichthyology, with an emphasis on diversity and adaptation. This ontology is created with the mindset of categorizing fish automatically based on the attributes and terms mined from the fish specimen.\r\n\r\nMost of the basic organization from this ontology follows Nelson's research due to its synthetic and broad approach, while recognizing that Nelson's conclusions are one of many alternative interpretations of the literature, which is mostly agreed by our main references, and many fish and fisheries researchers.\r\n\r\nMost of the classification used in this ontology is in no means completed and just following the recent classification provided by the books\r\n\r\nFor now our work focused more on two group of fish which is early jawless fish, and advanced jawed fishes. The classification from both of this group are structured following the reference.",
-            "documentation": "https://github.com/mohdnajib1985/FishOntology",
+            "date_created": "2013-06-21T01:02:50Z",
+            "date_released": "2013-06-21T01:02:50Z",
+            "description": "The taxonomy of the family <i>Drosophilidae</i> (largely after Baechli) and of other taxa referred to in FlyBase.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FISHO/1/fisho.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://mohdnajib1985.github.io/FOWebPage/",
-            "id": 649,
+            "download_url": "media/ontologies/FB-SP/7/fb-sp.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.flybase.org/",
+            "id": 414,
             "indexed": true,
-            "max_children": 21,
-            "max_depth": 8,
+            "max_children": 207,
+            "max_depth": 35,
             "md5sum": null,
-            "nb_classes": 386,
+            "nb_classes": 6600,
             "nb_inconsistent": 0,
-            "nb_individuals": 8,
-            "nb_properties": 28,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": "https://peerj.com/preprints/3020/",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 7,
             "taxon": null,
-            "version": "Version 1.0.3 (Possible addition)-Considering to add more fish study related terms in the ontology such as fish immune sytem, and Fish Locomotion and Feeding Behavior.-Also focused on completing other classification of fish, and adding more relationships between the classes.Version 1.0.2-Added a lot of annotations and metadata for fish terms, and added several more subclasses for the ontology. Version 1.0.1-Change  the structure and the hierarchy of a lot of classes to follow the main references. Several classes are deleted and marked as \"deprecated\" or \"In Review\" in case they might be useful later.Version 1.0.0-The first version of the fish ontology is created."
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "FLYGLYCODB",
-        "name": "FlyGlycoDB",
+        "acronym": "FCC1",
+        "name": "Fernald Community Cohort N",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2016-12-22T06:03:35Z",
-            "date_released": "2016-12-22T00:00:00Z",
-            "description": "Ontology to describe glycogenes of Drosophila.  Includes mutants, activity, references and links to FlyBase and UniProt.",
+            "date_created": "2019-10-03T22:17:36Z",
+            "date_released": "2019-10-03T00:00:00Z",
+            "description": "Fernald Community Cohort meta data",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FLYGLYCODB/2/flyglycodb.owl",
+            "download_url": "media/ontologies/FCC1/1/fcc1.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://fly.glycoinfo.org",
-            "id": 447,
+            "home_page": "http://www.semanticweb.org/diwaleva/ontologies/2019/9/fcc-ontology",
+            "id": 3296,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 4,
+            "max_children": 3,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 114,
+            "nb_classes": 9,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 21,
+            "nb_individuals": 1,
+            "nb_properties": 3,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "FBbi",
-        "name": "Biological Imaging Methods Ontology",
+        "acronym": "ONTOLURGENCES",
+        "name": "Emergency care ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-09-14T01:02:48Z",
-            "date_released": "2016-09-14T01:02:47Z",
-            "description": "A structured controlled vocabulary of sample preparation, visualization and imaging methods used in biomedical research.",
+            "date_created": "2015-10-27T17:32:00Z",
+            "date_released": "2015-10-27T00:00:00Z",
+            "description": "Emergency care ontology build during LERUDI project.\r\nhttp://www.ncbi.nlm.nih.gov/pubmed/25160343 (v4.0)",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/FBbi/14/fbbi.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://cellimagelibrary.org/",
-            "id": 227,
+            "download_url": "media/ontologies/ONTOLURGENCES/1/ontolurgences.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 569,
             "indexed": true,
-            "max_children": 19,
-            "max_depth": 9,
+            "max_children": 115,
+            "max_depth": 16,
             "md5sum": null,
-            "nb_classes": 624,
-            "nb_inconsistent": 0,
+            "nb_classes": 10031,
+            "nb_inconsistent": 16,
             "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_properties": 61,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 1,
             "taxon": null,
-            "version": "unknown"
+            "version": "3.0.4"
         },
         "topics": null
     },
     {
-        "acronym": "FOODON",
-        "name": "Food Ontology",
+        "acronym": "FHHO",
+        "name": "Family Health History Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-07-17T12:18:42.946132Z",
-            "date_released": "2023-07-17T12:18:42.946108Z",
-            "description": "A broadly scoped ontology representing entities which bear a \u201cfood role\u201d. It encompasses materials in natural ecosystems and agriculture that are consumed by humans and domesticated animals. This includes any generic (unbranded) raw or processed food material found in processing plants, markets, stores or food distribution points. FoodOn also imports nutritional component and dietary pattern terms from other OBO Foundry ontologies to support interoperability in diet and nutrition research",
+            "date_created": "2009-05-10T21:26:51Z",
+            "date_released": "2008-09-23T07:00:00Z",
+            "description": "The FHHO facilitates representing the family health histories of persons related by biological and/or social family relationships (e.g. step, adoptive) who share genetic, behavioral, and/or environmental risk factors for disease. SWRL rules are included to compute 3 generations of biological relationships based on parentage and family history findings based on personal health findings.",
             "documentation": null,
-            "domain": "diet, metabolomics, and nutrition",
-            "download_url": "media/ontologies/FOODON/77/foodon.owl",
+            "domain": null,
+            "download_url": "media/ontologies/FHHO/1/fhho.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://foodon.org/",
-            "id": 5945,
+            "home_page": "http://healthsystems.engr.wisc.edu/Alumni/JanePeace.html",
+            "id": 60,
             "indexed": true,
-            "max_children": 2117,
-            "max_depth": 30,
-            "md5sum": "8e90cb450feccc92d0c23f2ab42ec233",
-            "nb_classes": 34372,
+            "max_children": 63,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 238,
             "nb_inconsistent": 0,
-            "nb_individuals": 435,
-            "nb_properties": 74,
-            "products": [
-                {
-                    "format": "owl-rdf/xml",
-                    "id": "foodon.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/foodon.owl",
-                    "title": "FoodOn ontology with import file references and over 9,000 food products"
-                },
-                {
-                    "format": "owl-rdf/xml",
-                    "id": "foodon_core.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/foodon_core.owl",
-                    "title": "FoodOn core ontology (currently the same as foodon.owl)"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/31304272",
-                    "title": "FoodOn: a harmonized food ontology to increase global food traceability, quality control and data integration"
-                }
-            ],
-            "submission_id": 77,
+            "nb_individuals": 12,
+            "nb_properties": 433,
+            "products": null,
+            "publication": "http://healthsystems.engr.wisc.edu/Alumni/JanePeace.html",
+            "publications": null,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ONTOLURGENCES",
-        "name": "Emergency care ontology",
+        "acronym": "FLYGLYCODB",
+        "name": "FlyGlycoDB",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2015-10-27T17:32:00Z",
-            "date_released": "2015-10-27T00:00:00Z",
-            "description": "Emergency care ontology build during LERUDI project.\r\nhttp://www.ncbi.nlm.nih.gov/pubmed/25160343 (v4.0)",
+            "date_created": "2016-12-22T06:03:35Z",
+            "date_released": "2016-12-22T00:00:00Z",
+            "description": "Ontology to describe glycogenes of Drosophila.  Includes mutants, activity, references and links to FlyBase and UniProt.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ONTOLURGENCES/1/ontolurgences.owl",
+            "download_url": "media/ontologies/FLYGLYCODB/2/flyglycodb.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 569,
+            "home_page": "http://fly.glycoinfo.org",
+            "id": 447,
             "indexed": true,
-            "max_children": 115,
-            "max_depth": 16,
+            "max_children": 14,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 10031,
-            "nb_inconsistent": 16,
+            "nb_classes": 114,
+            "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 61,
+            "nb_properties": 21,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "3.0.4"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
         "acronym": "ONTOMA",
         "name": "Ontology of Alternative Medicine, French",
         "species": null,
@@ -36947,49 +36797,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": "Version 1.1\nDate: 11-2011\n"
         },
         "topics": null
     },
     {
-        "acronym": "ONTONEO",
-        "name": "Obstetric and Neonatal Ontology",
+        "acronym": "FOAF",
+        "name": "Friend of a Friend Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 14,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-05-03T12:40:10.611929Z",
-            "date_released": "2021-05-03T12:40:10.611907Z",
-            "description": "The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.",
+            "date_created": "2019-02-01T01:38:31Z",
+            "date_released": "2019-01-31T00:00:00Z",
+            "description": "Friend of a Friend Vocabulary",
             "documentation": null,
-            "domain": "biomedical, health",
-            "download_url": "media/ontologies/ONTONEO/12/ontoneo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/FOAF/1/foaf.owl",
             "has_ontology_language": "OWL",
-            "home_page": "ontoneo.com",
-            "id": 4442,
+            "home_page": null,
+            "id": 2240,
             "indexed": true,
-            "max_children": 27,
-            "max_depth": 14,
-            "md5sum": "15a05f1634bb71e3aca21ca53750ea21",
-            "nb_classes": 1797,
-            "nb_inconsistent": 21,
-            "nb_individuals": 17,
-            "nb_properties": 444,
-            "products": [
-                {
-                    "id": "ontoneo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ontoneo.owl"
-                }
-            ],
+            "max_children": 10,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 22,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 40,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 12,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "0.99"
         },
         "topics": null
     },
     {
         "acronym": "ONTOPARON",
         "name": "Ontology of Amyotrophic Lateral Sclerosis, all modules",
         "species": null,
@@ -37122,14 +36967,171 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
+        "acronym": "ONTONEO",
+        "name": "Obstetric and Neonatal Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 14,
+            "classifiable": true,
+            "date_created": "2021-05-03T12:40:10.611929Z",
+            "date_released": "2021-05-03T12:40:10.611907Z",
+            "description": "The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.",
+            "documentation": null,
+            "domain": "biomedical, health",
+            "download_url": "media/ontologies/ONTONEO/12/ontoneo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "ontoneo.com",
+            "id": 4442,
+            "indexed": true,
+            "max_children": 27,
+            "max_depth": 14,
+            "md5sum": "15a05f1634bb71e3aca21ca53750ea21",
+            "nb_classes": 1797,
+            "nb_inconsistent": 21,
+            "nb_individuals": 17,
+            "nb_properties": 444,
+            "products": [
+                {
+                    "id": "ontoneo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ontoneo.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 12,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "FOBI",
+        "name": "Food-Biomarker Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 107,
+            "classifiable": true,
+            "date_created": "2022-05-09T12:45:36.999726Z",
+            "date_released": "2022-05-09T12:45:36.999712Z",
+            "description": "FOBI (Food-Biomarker Ontology) is an ontology to represent food intake data and associate it with metabolomic data",
+            "documentation": null,
+            "domain": "diet, metabolomics, and nutrition",
+            "download_url": "media/ontologies/FOBI/7/fobi.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/pcastellanoescuder/FoodBiomarkerOntology",
+            "id": 5069,
+            "indexed": true,
+            "max_children": 148,
+            "max_depth": 9,
+            "md5sum": "894f2bbe2c41f1ac47e92e24b94d14dc",
+            "nb_classes": 1334,
+            "nb_inconsistent": 148,
+            "nb_individuals": 0,
+            "nb_properties": 4,
+            "products": [
+                {
+                    "format": "owl-rdf/xml",
+                    "id": "fobi.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fobi.owl",
+                    "title": "FOBI is an ontology to represent food intake data and associate it with metabolomic data"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/32556148",
+                    "title": "FOBI: an ontology to represent food intake data and associate it with metabolomic data"
+                },
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/34601570",
+                    "title": "The fobitools framework: the first steps towards food enrichment analysis"
+                }
+            ],
+            "submission_id": 7,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "foaf",
+        "name": "FOAF (people, organizations, groups)",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2012-08-31T10:20:55Z",
+            "date_released": "2012-07-12T07:00:00Z",
+            "description": "This file is imported by vivo-core-public-1.5.owl.  It contains terms relating to people, groups and organizations from the http://xmlns.com/foaf/0.1/ namespace that are included in the vivo ontology.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/foaf/1/foaf.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://sourceforge.net/apps/mediawiki/vivo/index.php?title=Ontology",
+            "id": 247,
+            "indexed": true,
+            "max_children": 11,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 15,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 24,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.5"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "FBbi",
+        "name": "Biological Imaging Methods Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2016-09-14T01:02:48Z",
+            "date_released": "2016-09-14T01:02:47Z",
+            "description": "A structured controlled vocabulary of sample preparation, visualization and imaging methods used in biomedical research.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/FBbi/14/fbbi.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://cellimagelibrary.org/",
+            "id": 227,
+            "indexed": true,
+            "max_children": 19,
+            "max_depth": 9,
+            "md5sum": null,
+            "nb_classes": 624,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 2,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 14,
+            "taxon": null,
+            "version": "unknown"
+        },
+        "topics": null
+    },
+    {
         "acronym": "FBBI",
         "name": "Biological Imaging Methods Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -37229,97 +37231,14 @@
             "submission_id": 2,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "PCL",
-        "name": "Provisional Cell Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 26,
-            "classifiable": true,
-            "date_created": "2023-03-06T13:32:24.337058Z",
-            "date_released": "2023-03-06T13:32:24.337043Z",
-            "description": "Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties.",
-            "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/PCL/22/pcl.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/provisional_cell_ontology",
-            "id": 5707,
-            "indexed": true,
-            "max_children": 110886,
-            "max_depth": 32,
-            "md5sum": "e0c8e6791bfae3dd934fa06d4200ceb1",
-            "nb_classes": 121029,
-            "nb_inconsistent": 0,
-            "nb_individuals": 1825,
-            "nb_properties": 299,
-            "products": [
-                {
-                    "id": "pcl.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl.owl"
-                },
-                {
-                    "id": "pcl.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl.obo"
-                },
-                {
-                    "id": "pcl.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl.json"
-                },
-                {
-                    "id": "pcl-base.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-base.owl"
-                },
-                {
-                    "id": "pcl-base.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-base.obo"
-                },
-                {
-                    "id": "pcl-base.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-base.json"
-                },
-                {
-                    "id": "pcl-full.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-full.owl"
-                },
-                {
-                    "id": "pcl-full.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-full.obo"
-                },
-                {
-                    "id": "pcl-full.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-full.json"
-                },
-                {
-                    "id": "pcl-simple.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-simple.owl"
-                },
-                {
-                    "id": "pcl-simple.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-simple.obo"
-                },
-                {
-                    "id": "pcl-simple.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pcl-simple.json"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 22,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
         "acronym": "FBBT",
         "name": "Drosophila gross anatomy",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
@@ -37503,14 +37422,82 @@
             "submission_id": 21,
             "taxon": null,
             "version": "2014-09-23"
         },
         "topics": null
     },
     {
+        "acronym": "FISH-AST",
+        "name": "FISH Archaeological Sciences Thesaurus",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2017-09-11T21:21:13Z",
+            "date_released": "2013-07-17T00:00:00Z",
+            "description": "Terminology used for recording the techniques, recovery methods and materials associated with archaeological sciences. Maintained by Historic England on behalf of the FISH Terminology Working Group.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/FISH-AST/1/fish-ast.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "http://heritagedata.org/live/schemes/560.html",
+            "id": 385,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 0,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 125,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "FISHO",
+        "name": "Fish Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2017-08-28T08:51:50Z",
+            "date_released": "2017-08-28T00:00:00Z",
+            "description": "This Fish Ontology is an ontology created as how the author views the fish structure following the book \"The Diversity of Fishes: Biology, Evolution and Ecology\" as the main reference, which covers one part of view on ichthyology, with an emphasis on diversity and adaptation. This ontology is created with the mindset of categorizing fish automatically based on the attributes and terms mined from the fish specimen.\r\n\r\nMost of the basic organization from this ontology follows Nelson's research due to its synthetic and broad approach, while recognizing that Nelson's conclusions are one of many alternative interpretations of the literature, which is mostly agreed by our main references, and many fish and fisheries researchers.\r\n\r\nMost of the classification used in this ontology is in no means completed and just following the recent classification provided by the books\r\n\r\nFor now our work focused more on two group of fish which is early jawless fish, and advanced jawed fishes. The classification from both of this group are structured following the reference.",
+            "documentation": "https://github.com/mohdnajib1985/FishOntology",
+            "domain": null,
+            "download_url": "media/ontologies/FISHO/1/fisho.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://mohdnajib1985.github.io/FOWebPage/",
+            "id": 649,
+            "indexed": true,
+            "max_children": 21,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 386,
+            "nb_inconsistent": 0,
+            "nb_individuals": 8,
+            "nb_properties": 28,
+            "products": null,
+            "publication": "https://peerj.com/preprints/3020/",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "Version 1.0.3 (Possible addition)-Considering to add more fish study related terms in the ontology such as fish immune sytem, and Fish Locomotion and Feeding Behavior.-Also focused on completing other classification of fish, and adding more relationships between the classes.Version 1.0.2-Added a lot of annotations and metadata for fish terms, and added several more subclasses for the ontology. Version 1.0.1-Change  the structure and the hierarchy of a lot of classes to follow the main references. Several classes are deleted and marked as \"deprecated\" or \"In Review\" in case they might be useful later.Version 1.0.0-The first version of the fish ontology is created."
+        },
+        "topics": null
+    },
+    {
         "acronym": "FBDV",
         "name": "Drosophila development",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 6,
             "classifiable": true,
@@ -37595,47 +37582,115 @@
             "submission_id": 19,
             "taxon": null,
             "version": "2016-10-14"
         },
         "topics": null
     },
     {
-        "acronym": "FIDEO",
-        "name": "Food Interactions with Drugs Evidence Ontology",
+        "acronym": "FOODON",
+        "name": "Food Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 20,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-02-06T13:00:09.675535Z",
-            "date_released": "2023-02-06T13:00:09.675518Z",
-            "description": "Food-Drug interactions automatically extracted from scientific literature",
+            "date_created": "2023-07-17T12:18:42.946132Z",
+            "date_released": "2023-07-17T12:18:42.946108Z",
+            "description": "A broadly scoped ontology representing entities which bear a \u201cfood role\u201d. It encompasses materials in natural ecosystems and agriculture that are consumed by humans and domesticated animals. This includes any generic (unbranded) raw or processed food material found in processing plants, markets, stores or food distribution points. FoodOn also imports nutritional component and dietary pattern terms from other OBO Foundry ontologies to support interoperability in diet and nutrition research",
             "documentation": null,
             "domain": "diet, metabolomics, and nutrition",
-            "download_url": "media/ontologies/FIDEO/7/fideo.owl",
+            "download_url": "media/ontologies/FOODON/77/foodon.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://gitub.u-bordeaux.fr/erias/fideo",
-            "id": 5639,
+            "home_page": "https://foodon.org/",
+            "id": 5945,
             "indexed": true,
-            "max_children": 1574,
-            "max_depth": 9,
-            "md5sum": "1e0f1977b36dc7681221f2c8e0ac7b93",
-            "nb_classes": 3548,
+            "max_children": 2117,
+            "max_depth": 30,
+            "md5sum": "8e90cb450feccc92d0c23f2ab42ec233",
+            "nb_classes": 34372,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 12,
+            "nb_individuals": 435,
+            "nb_properties": 74,
             "products": [
                 {
-                    "id": "fideo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fideo.owl"
+                    "format": "owl-rdf/xml",
+                    "id": "foodon.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/foodon.owl",
+                    "title": "FoodOn ontology with import file references and over 9,000 food products"
+                },
+                {
+                    "format": "owl-rdf/xml",
+                    "id": "foodon_core.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/foodon_core.owl",
+                    "title": "FoodOn core ontology (currently the same as foodon.owl)"
                 }
             ],
             "publication": null,
-            "publications": null,
-            "submission_id": 7,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/31304272",
+                    "title": "FoodOn: a harmonized food ontology to increase global food traceability, quality control and data integration"
+                }
+            ],
+            "submission_id": 77,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "GNO",
+        "name": "Glycan Naming and Subsumption Ontology (GNOme)",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2023-07-31T12:25:55.349163Z",
+            "date_released": "2023-07-31T12:25:55.349133Z",
+            "description": "GlyTouCan provides stable accessions for glycans described at varyious degrees of characterization, including compositions (no linkage) and topologies (no carbon bond positions or anomeric configurations). GNOme organizes these stable accessions for interative browsing, for text-based searching, and for automated reasoning with well-defined characterization levels.",
+            "documentation": null,
+            "domain": "chemistry and biochemistry",
+            "download_url": "media/ontologies/GNO/20/gno.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://gnome.glyomics.org/",
+            "id": 5970,
+            "indexed": true,
+            "max_children": 14418,
+            "max_depth": 14,
+            "md5sum": "f761cb6aee409bca5507bb1fcd690412",
+            "nb_classes": 161683,
+            "nb_inconsistent": 0,
+            "nb_individuals": 5,
+            "nb_properties": 0,
+            "products": [
+                {
+                    "description": "Glycan Naming and Subsumption Ontology, OWL format (primary)",
+                    "id": "gno.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/gno.owl"
+                },
+                {
+                    "description": "Glycan Naming and Subsumption Ontology, OBO format (automated conversion from OWL)",
+                    "id": "gno.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/gno.obo"
+                },
+                {
+                    "description": "Glycan Naming and Subsumption Ontology, JSON format (automated conversion from OWL)",
+                    "id": "gno.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/gno.json"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://doi.org/10.5281/zenodo.6678278",
+                    "title": "GNOme - Glycan Naming and Subsumption Ontology"
+                }
+            ],
+            "submission_id": 20,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "GO",
@@ -37758,282 +37813,194 @@
                 "label": "All life"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "GEXO",
-        "name": "Gene Expression Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 49,
-            "classifiable": true,
-            "date_created": "2015-12-16T08:29:35Z",
-            "date_released": "2015-12-16T00:00:00Z",
-            "description": "An application ontology for the domain of gene expression. The ontology integrates fragments of GO and MI with data from GOA, IntAct, UniProt, NCBI, and orthology relations using ontological properties from RO and ISO.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/GEXO/2/gexo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.semantic-systems-biology.org/apo",
-            "id": 217,
-            "indexed": true,
-            "max_children": 50996,
-            "max_depth": 21,
-            "md5sum": null,
-            "nb_classes": 166254,
-            "nb_inconsistent": 0,
-            "nb_individuals": 157102,
-            "nb_properties": 12,
-            "products": null,
-            "publication": "http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-014-0386-y",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2015"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "G-O",
-        "name": "Geriatrics Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 9,
-            "classifiable": true,
-            "date_created": "2016-11-27T15:40:12Z",
-            "date_released": "2016-08-01T00:00:00Z",
-            "description": "The Ontology defines the Geriatrics system with classes. It also has object properties and data properties which are used to create a Decision support system for the elderly.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/G-O/1/g-o.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://healthcareelderly.me/GeriatricsWebApp/web/ontoBrowser.html",
-            "id": 70,
-            "indexed": true,
-            "max_children": 149,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 296,
-            "nb_inconsistent": 0,
-            "nb_individuals": 246,
-            "nb_properties": 15,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "GFO",
-        "name": "General Formal Ontology",
+        "acronym": "UPA",
+        "name": "Unipathway",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 13,
             "classifiable": true,
-            "date_created": "2010-03-02T19:23:37Z",
-            "date_released": "2006-01-07T08:00:00Z",
-            "description": "The General Formal Ontology (GFO) is a top-level ontology integrating objects and processes.",
+            "date_created": "2019-07-29T15:03:30.563721Z",
+            "date_released": "2019-07-29T15:03:30.563696Z",
+            "description": "A manually curated resource for the representation and annotation of metabolic pathways",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/GFO/1/gfo.owl",
+            "domain": "pathways",
+            "download_url": "media/ontologies/UPA/1/upa.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.onto-med.de/ontologies/gfo/",
-            "id": 226,
+            "home_page": "https://github.com/geneontology/unipathway",
+            "id": 2784,
             "indexed": true,
-            "max_children": 6,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 45,
+            "max_children": 3612,
+            "max_depth": 9,
+            "md5sum": "22b3019d4798fa587d0b57732c73fe4b",
+            "nb_classes": 4672,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 41,
-            "products": null,
+            "nb_properties": 96,
+            "products": [
+                {
+                    "id": "upa.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/upa.owl"
+                },
+                {
+                    "id": "upa.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/upa.obo"
+                }
+            ],
             "publication": null,
-            "publications": null,
+            "publications": [
+                {
+                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/22102589",
+                    "title": "UniPathway: a resource for the exploration and annotation of metabolic pathways"
+                }
+            ],
             "submission_id": 1,
             "taxon": null,
-            "version": "Version 1.0 ($Revision: 1.13 $)"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "GO-EXT",
-        "name": "Gene Ontology Extension",
+        "acronym": "GLYCORDF",
+        "name": "GlycoRDF",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2011-06-24T15:01:44Z",
-            "date_released": "2011-06-24T07:00:00Z",
-            "description": "Gene Ontology Extension",
-            "documentation": "http://www.geneontology.org/GO.doc.shtml",
-            "domain": null,
-            "download_url": "media/ontologies/GO-EXT/2/go-ext.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.geneontology.org",
-            "id": 706,
-            "indexed": true,
-            "max_children": 5313,
-            "max_depth": 15,
-            "md5sum": null,
-            "nb_classes": 35873,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 6,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "GFVO",
-        "name": "Genomic Feature and Variation Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2016-10-14T23:45:32Z",
-            "date_released": "2015-07-14T00:00:00Z",
-            "description": "The Genomic Feature and Variation Ontology (GFVO) is modeled to represent genomic data using the Resource Description Format (RDF) or Linked Data using JSON (JSON-LD). It is captures the contents of data files that adhere to the Generic Feature Format Version 3 (GFF3, http://www.sequenceontology.org/resources/gff3.html), the General Transfer Format (GTF, http://mblab.wustl.edu/GTF22.html), the Genome Variation Format Version 1 (GVF, http://www.sequenceontology.org/resources/gvf.html), and the Variant Call Format (VCF, http://vcftools.sourceforge.net/specs.html), as well as the FASTA format (http://en.wikipedia.org/wiki/FASTA_format). The creation of the ontology was inspired by previous work of Robert Hoehndorf on RDF2OWL (http://code.google.com/p/rdf2owl).\r\n\r\nNOTE: GFVO in BioPortal is a simplified version without ontology imports and having class/property equivalences removed. The full ontology is available at: http://www.biointerchange.org/gfvo",
-            "documentation": "https://www.codamono.com/biointerchange/documentation/",
+            "date_created": "2015-08-17T10:58:01Z",
+            "date_released": "2014-09-24T00:00:00Z",
+            "description": "Ontology for Glycomcis data.",
+            "documentation": "https://github.com/ReneRanzinger/GlycoRDF/blob/master/ontology/documentation.docx",
             "domain": null,
-            "download_url": "media/ontologies/GFVO/20/gfvo.owl",
+            "download_url": "media/ontologies/GLYCORDF/14/glycordf.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.codamono.com/biointerchange/",
-            "id": 499,
+            "home_page": "https://github.com/ReneRanzinger/GlycoRDF",
+            "id": 338,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 6,
+            "max_children": 32,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 102,
+            "nb_classes": 113,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 33,
+            "nb_individuals": 219,
+            "nb_properties": 188,
             "products": null,
-            "publication": "https://peerj.com/articles/933/",
+            "publication": "http://bioinformatics.oxfordjournals.org/content/31/6/919.long",
             "publications": null,
-            "submission_id": 20,
+            "submission_id": 14,
             "taxon": null,
-            "version": "1.0.6"
+            "version": "1.00"
         },
         "topics": null
     },
     {
-        "acronym": "GLYCOCOO",
-        "name": "GlycoConjugate Ontology",
+        "acronym": "GEXO",
+        "name": "Gene Expression Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 49,
             "classifiable": true,
-            "date_created": "2018-10-27T01:04:00Z",
-            "date_released": "2018-10-27T01:03:59Z",
-            "description": "GlycoConjugate Ontology (GlycoCoO) as a standard semantic framework to describe and represent glycoconjugate data in RDF.",
+            "date_created": "2015-12-16T08:29:35Z",
+            "date_released": "2015-12-16T00:00:00Z",
+            "description": "An application ontology for the domain of gene expression. The ontology integrates fragments of GO and MI with data from GOA, IntAct, UniProt, NCBI, and orthology relations using ontological properties from RO and ISO.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GLYCOCOO/2/glycocoo.owl",
+            "download_url": "media/ontologies/GEXO/2/gexo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/glycoinfo/GlycoCoO",
-            "id": 1965,
+            "home_page": "http://www.semantic-systems-biology.org/apo",
+            "id": 217,
             "indexed": true,
-            "max_children": 29,
-            "max_depth": 3,
+            "max_children": 50996,
+            "max_depth": 21,
             "md5sum": null,
-            "nb_classes": 51,
+            "nb_classes": 166254,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 27,
+            "nb_individuals": 157102,
+            "nb_properties": 12,
             "products": null,
-            "publication": null,
+            "publication": "http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-014-0386-y",
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.1.3"
+            "version": "2015"
         },
         "topics": null
     },
     {
-        "acronym": "GLYCORDF",
-        "name": "GlycoRDF",
+        "acronym": "ONTOREPLICOV",
+        "name": "OntoRepliCov",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2015-08-17T10:58:01Z",
-            "date_released": "2014-09-24T00:00:00Z",
-            "description": "Ontology for Glycomcis data.",
-            "documentation": "https://github.com/ReneRanzinger/GlycoRDF/blob/master/ontology/documentation.docx",
+            "date_created": "2021-04-21T07:36:45Z",
+            "date_released": "2021-04-21T00:00:00Z",
+            "description": "OntoRepliCov is an ontology designed to describe the different stages of the SARS-CoV-2 replication process.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GLYCORDF/14/glycordf.owl",
+            "download_url": "media/ontologies/ONTOREPLICOV/1/ontoreplicov.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ReneRanzinger/GlycoRDF",
-            "id": 338,
+            "home_page": null,
+            "id": 4434,
             "indexed": true,
-            "max_children": 32,
-            "max_depth": 3,
+            "max_children": 22,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 113,
+            "nb_classes": 85,
             "nb_inconsistent": 0,
-            "nb_individuals": 219,
-            "nb_properties": 188,
+            "nb_individuals": 15,
+            "nb_properties": 13,
             "products": null,
-            "publication": "http://bioinformatics.oxfordjournals.org/content/31/6/919.long",
+            "publication": null,
             "publications": null,
-            "submission_id": 14,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.00"
+            "version": "0"
         },
         "topics": null
     },
     {
-        "acronym": "ONTOREPLICOV",
-        "name": "OntoRepliCov",
+        "acronym": "PEAO",
+        "name": "Plant Experimental Assay Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-04-21T07:36:45Z",
-            "date_released": "2021-04-21T00:00:00Z",
-            "description": "OntoRepliCov is an ontology designed to describe the different stages of the SARS-CoV-2 replication process.",
+            "date_created": "2014-12-17T15:10:11Z",
+            "date_released": "2014-12-17T00:00:00Z",
+            "description": "The Plant domain has been the subject of several attempts to structure and formally define terms and corresponding relations, such as their anatomical features, developmental stages, and the application of particular experimental procedures to a biological problem. However, a focus on experimental assays in order to describe the whole experimental procedure, to the best of our knowledge, has only been attempted in the context of a very general description based on classical views of the scientific method. In this study, we focus on the development and proposal of an ontology dedicated to the description of these experimental procedures, regardless of the scientific questions that prompted the assays. This ontology includes entities from three distinct realms (biological, physical and data), which include both experimental products, their relations and the protocols describing their manipulation. The final outcome is a useful and comprehensive ontology in the plant domain, to be used as a log book by experimentalists, providing a formal relation between entities.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ONTOREPLICOV/1/ontoreplicov.owl",
+            "download_url": "media/ontologies/PEAO/1/peao.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4434,
+            "home_page": "https://bitbucket.org/PlantExpAssay/ontology",
+            "id": 290,
             "indexed": true,
-            "max_children": 22,
-            "max_depth": 4,
+            "max_children": 75,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 85,
+            "nb_classes": 2298,
             "nb_inconsistent": 0,
-            "nb_individuals": 15,
-            "nb_properties": 13,
+            "nb_individuals": 1,
+            "nb_properties": 74,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "GML",
         "name": "Ontology for Geography Markup Language (GML3.0)",
         "species": null,
@@ -38180,72 +38147,72 @@
             "submission_id": 1,
             "taxon": null,
             "version": "version 1.0"
         },
         "topics": null
     },
     {
-        "acronym": "PEAO",
-        "name": "Plant Experimental Assay Ontology",
+        "acronym": "ATC",
+        "name": "Anatomical Therapeutic Chemical Classification",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2014-12-17T15:10:11Z",
-            "date_released": "2014-12-17T00:00:00Z",
-            "description": "The Plant domain has been the subject of several attempts to structure and formally define terms and corresponding relations, such as their anatomical features, developmental stages, and the application of particular experimental procedures to a biological problem. However, a focus on experimental assays in order to describe the whole experimental procedure, to the best of our knowledge, has only been attempted in the context of a very general description based on classical views of the scientific method. In this study, we focus on the development and proposal of an ontology dedicated to the description of these experimental procedures, regardless of the scientific questions that prompted the assays. This ontology includes entities from three distinct realms (biological, physical and data), which include both experimental products, their relations and the protocols describing their manipulation. The final outcome is a useful and comprehensive ontology in the plant domain, to be used as a log book by experimentalists, providing a formal relation between entities.",
-            "documentation": null,
+            "date_created": "2017-02-06T21:25:36Z",
+            "date_released": "2017-02-06T21:25:36Z",
+            "description": "The Anatomical Therapeutic Chemical (ATC) Classification System is used for the classification of active ingredients of drugs according to the organ or system on which they act and their therapeutic, pharmacological and chemical properties. It is controlled by the World Health Organization Collaborating Centre for Drug Statistics Methodology (WHOCC), and was first published in 1976. ",
+            "documentation": "http://en.wikipedia.org/wiki/Anatomical_Therapeutic_Chemical_Classification_System",
             "domain": null,
-            "download_url": "media/ontologies/PEAO/1/peao.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://bitbucket.org/PlantExpAssay/ontology",
-            "id": 290,
+            "download_url": "media/ontologies/ATC/6/atc.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": null,
+            "id": 186,
             "indexed": true,
-            "max_children": 75,
-            "max_depth": 9,
+            "max_children": 31,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 2298,
+            "nb_classes": 6135,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 74,
+            "nb_individuals": 0,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
-            "version": null
+            "version": "2016AB"
         },
         "topics": null
     },
     {
         "acronym": "PECO",
         "name": "Plant Experimental Conditions Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-07-24T12:43:07.471959Z",
-            "date_released": "2023-07-24T12:43:07.471947Z",
+            "date_created": "2023-07-31T13:21:47.631147Z",
+            "date_released": "2023-07-31T13:21:47.631135Z",
             "description": "A structured, controlled vocabulary which describes the treatments, growing conditions, and/or study types used in plant biology experiments.",
             "documentation": null,
             "domain": "investigations",
-            "download_url": "media/ontologies/PECO/28/peco.owl",
+            "download_url": "media/ontologies/PECO/29/peco.owl",
             "has_ontology_language": "OWL",
             "home_page": "http://planteome.org/",
-            "id": 5961,
+            "id": 5974,
             "indexed": true,
-            "max_children": 603,
+            "max_children": 604,
             "max_depth": 31,
-            "md5sum": "a21166af756d21038bfd46b15dead4c0",
-            "nb_classes": 3124,
+            "md5sum": "eaec00f4c4338b33a96a377b746fd854",
+            "nb_classes": 3086,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 123,
+            "nb_properties": 119,
             "products": [
                 {
                     "id": "peco.owl",
                     "ontology_purl": "http://purl.obolibrary.org/obo/peco.owl"
                 },
                 {
                     "id": "peco.obo",
@@ -38255,147 +38222,187 @@
             "publication": null,
             "publications": [
                 {
                     "id": "https://www.ncbi.nlm.nih.gov/pubmed/29186578",
                     "title": "The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics."
                 }
             ],
-            "submission_id": 28,
+            "submission_id": 29,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "UO",
-        "name": "Units of measurement ontology",
+        "acronym": "APOLLO-SV",
+        "name": "Apollo Structured Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-05T16:07:40.709791Z",
-            "date_released": "2023-06-05T16:07:40.709776Z",
-            "description": "Metrical units for use in conjunction with PATO",
+            "date_created": "2017-12-01T02:00:14Z",
+            "date_released": "2017-12-01T02:00:14Z",
+            "description": "Apollo-SV, developed as an OWL specification, defines the terms and relations necessary for interoperation between epidemic simulators and public health application software that interface with these simulators.",
             "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/UO/60/uo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/APOLLO-SV/21/apollo-sv.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/bio-ontology-research-group/unit-ontology",
-            "id": 5849,
+            "home_page": "https://github.com/ApolloDev/apollo-sv",
+            "id": 889,
             "indexed": true,
-            "max_children": 38,
-            "max_depth": 6,
-            "md5sum": "7d27f7fd05c6808bac7973dcce13d89a",
-            "nb_classes": 564,
+            "max_children": 42,
+            "max_depth": 37,
+            "md5sum": null,
+            "nb_classes": 1523,
             "nb_inconsistent": 0,
-            "nb_individuals": 275,
-            "nb_properties": 2,
-            "products": [
-                {
-                    "id": "uo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/uo.owl"
-                },
-                {
-                    "id": "uo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/uo.obo"
-                }
-            ],
+            "nb_individuals": 30,
+            "nb_properties": 119,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/23060432",
-                    "title": "The Units Ontology: a tool for integrating units of measurement in science"
-                }
-            ],
-            "submission_id": 60,
+            "publications": null,
+            "submission_id": 21,
             "taxon": null,
-            "version": null
+            "version": "v4.0.1"
         },
         "topics": null
     },
     {
-        "acronym": "GMM",
-        "name": "GoMapMan",
+        "acronym": "G-O",
+        "name": "Geriatrics Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 9,
             "classifiable": true,
-            "date_created": "2016-05-16T23:52:51Z",
-            "date_released": "2015-08-09T00:00:00Z",
-            "description": "GoMapMan is an open web-accessible resource for gene functional annotations in the plant sciences. It was developed to facilitate improvement, consolidation and visualization of gene annotations across several plant species.",
-            "documentation": "http://www.gomapman.org/node/2",
+            "date_created": "2016-11-27T15:40:12Z",
+            "date_released": "2016-08-01T00:00:00Z",
+            "description": "The Ontology defines the Geriatrics system with classes. It also has object properties and data properties which are used to create a Decision support system for the elderly.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GMM/25/gmm.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.gomapman.org/",
-            "id": 145,
+            "download_url": "media/ontologies/G-O/1/g-o.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://healthcareelderly.me/GeriatricsWebApp/web/ontoBrowser.html",
+            "id": 70,
             "indexed": true,
-            "max_children": 79,
+            "max_children": 149,
             "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 1896,
+            "nb_classes": 296,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 246,
+            "nb_properties": 15,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 25,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.2"
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "UPA",
-        "name": "Unipathway",
+        "acronym": "GFO",
+        "name": "General Formal Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 13,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-07-29T15:03:30.563721Z",
-            "date_released": "2019-07-29T15:03:30.563696Z",
-            "description": "A manually curated resource for the representation and annotation of metabolic pathways",
+            "date_created": "2010-03-02T19:23:37Z",
+            "date_released": "2006-01-07T08:00:00Z",
+            "description": "The General Formal Ontology (GFO) is a top-level ontology integrating objects and processes.",
             "documentation": null,
-            "domain": "pathways",
-            "download_url": "media/ontologies/UPA/1/upa.owl",
+            "domain": null,
+            "download_url": "media/ontologies/GFO/1/gfo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/geneontology/unipathway",
-            "id": 2784,
+            "home_page": "http://www.onto-med.de/ontologies/gfo/",
+            "id": 226,
             "indexed": true,
-            "max_children": 3612,
-            "max_depth": 9,
-            "md5sum": "22b3019d4798fa587d0b57732c73fe4b",
-            "nb_classes": 4672,
+            "max_children": 6,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 45,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 96,
-            "products": [
-                {
-                    "id": "upa.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/upa.owl"
-                },
-                {
-                    "id": "upa.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/upa.obo"
-                }
-            ],
+            "nb_properties": 41,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "http://www.ncbi.nlm.nih.gov/pubmed/22102589",
-                    "title": "UniPathway: a resource for the exploration and annotation of metabolic pathways"
-                }
-            ],
+            "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Version 1.0 ($Revision: 1.13 $)"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "GO-EXT",
+        "name": "Gene Ontology Extension",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2011-06-24T15:01:44Z",
+            "date_released": "2011-06-24T07:00:00Z",
+            "description": "Gene Ontology Extension",
+            "documentation": "http://www.geneontology.org/GO.doc.shtml",
+            "domain": null,
+            "download_url": "media/ontologies/GO-EXT/2/go-ext.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.geneontology.org",
+            "id": 706,
+            "indexed": true,
+            "max_children": 5313,
+            "max_depth": 15,
+            "md5sum": null,
+            "nb_classes": 35873,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 6,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ATO",
+        "name": "Amphibian Taxonomy Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 10,
+            "classifiable": true,
+            "date_created": "2009-11-02T21:58:28Z",
+            "date_released": "2009-02-26T08:00:00Z",
+            "description": "A taxonomy of Amphibia",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/ATO/2/ato.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.amphibanat.org",
+            "id": 565,
+            "indexed": true,
+            "max_children": 704,
+            "max_depth": 9,
+            "md5sum": null,
+            "nb_classes": 6135,
+            "nb_inconsistent": 0,
+            "nb_individuals": 73,
+            "nb_properties": 1,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "See Remote Site"
         },
         "topics": null
     },
     {
         "acronym": "arp",
         "name": "Arpenteur",
         "species": [
@@ -38465,44 +38472,92 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ATC",
-        "name": "Anatomical Therapeutic Chemical Classification",
+        "acronym": "GMM",
+        "name": "GoMapMan",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2017-02-06T21:25:36Z",
-            "date_released": "2017-02-06T21:25:36Z",
-            "description": "The Anatomical Therapeutic Chemical (ATC) Classification System is used for the classification of active ingredients of drugs according to the organ or system on which they act and their therapeutic, pharmacological and chemical properties. It is controlled by the World Health Organization Collaborating Centre for Drug Statistics Methodology (WHOCC), and was first published in 1976. ",
-            "documentation": "http://en.wikipedia.org/wiki/Anatomical_Therapeutic_Chemical_Classification_System",
+            "date_created": "2016-05-16T23:52:51Z",
+            "date_released": "2015-08-09T00:00:00Z",
+            "description": "GoMapMan is an open web-accessible resource for gene functional annotations in the plant sciences. It was developed to facilitate improvement, consolidation and visualization of gene annotations across several plant species.",
+            "documentation": "http://www.gomapman.org/node/2",
             "domain": null,
-            "download_url": "media/ontologies/ATC/6/atc.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": null,
-            "id": 186,
+            "download_url": "media/ontologies/GMM/25/gmm.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.gomapman.org/",
+            "id": 145,
             "indexed": true,
-            "max_children": 31,
-            "max_depth": 4,
+            "max_children": 79,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 6135,
+            "nb_classes": 1896,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 7,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 25,
             "taxon": null,
-            "version": "2016AB"
+            "version": "1.2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "UO",
+        "name": "Units of measurement ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2023-06-05T16:07:40.709791Z",
+            "date_released": "2023-06-05T16:07:40.709776Z",
+            "description": "Metrical units for use in conjunction with PATO",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/UO/60/uo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/bio-ontology-research-group/unit-ontology",
+            "id": 5849,
+            "indexed": true,
+            "max_children": 38,
+            "max_depth": 6,
+            "md5sum": "7d27f7fd05c6808bac7973dcce13d89a",
+            "nb_classes": 564,
+            "nb_inconsistent": 0,
+            "nb_individuals": 275,
+            "nb_properties": 2,
+            "products": [
+                {
+                    "id": "uo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/uo.owl"
+                },
+                {
+                    "id": "uo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/uo.obo"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/23060432",
+                    "title": "The Units Ontology: a tool for integrating units of measurement in science"
+                }
+            ],
+            "submission_id": 60,
+            "taxon": null,
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "ATMO",
         "name": "African Traditional Medicine Ontology",
         "species": null,
@@ -38533,14 +38588,58 @@
             "submission_id": 3,
             "taxon": null,
             "version": "1.101"
         },
         "topics": null
     },
     {
+        "acronym": "ARO",
+        "name": "Antibiotic Resistance Ontology",
+        "species": [],
+        "status": "Classified",
+        "submission": {
+            "avg_children": 8,
+            "classifiable": true,
+            "date_created": "2023-05-29T12:00:21.441535Z",
+            "date_released": "2023-05-29T12:00:21.441511Z",
+            "description": "Antibiotic resistance genes and mutations",
+            "documentation": null,
+            "domain": "microbiology",
+            "download_url": "media/ontologies/ARO/26/aro.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/arpcard/aro",
+            "id": 5825,
+            "indexed": true,
+            "max_children": 954,
+            "max_depth": 9,
+            "md5sum": "45ea19df4107afb3139dffa1ee5f03eb",
+            "nb_classes": 6897,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 12,
+            "products": [
+                {
+                    "id": "aro.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/aro.owl"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/31665441",
+                    "title": "CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database."
+                }
+            ],
+            "submission_id": 26,
+            "taxon": null,
+            "version": null
+        },
+        "topics": []
+    },
+    {
         "acronym": "ONTOSIM",
         "name": "The Ontology of Information System on Mortality",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -38567,48 +38666,14 @@
             "submission_id": 2,
             "taxon": null,
             "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "ATO",
-        "name": "Amphibian Taxonomy Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 10,
-            "classifiable": true,
-            "date_created": "2009-11-02T21:58:28Z",
-            "date_released": "2009-02-26T08:00:00Z",
-            "description": "A taxonomy of Amphibia",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ATO/2/ato.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.amphibanat.org",
-            "id": 565,
-            "indexed": true,
-            "max_children": 704,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 6135,
-            "nb_inconsistent": 0,
-            "nb_individuals": 73,
-            "nb_properties": 1,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "See Remote Site"
-        },
-        "topics": null
-    },
-    {
         "acronym": "QUDT",
         "name": "Quantities, Units, Dimensions, and Types Ontology",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
@@ -38635,14 +38700,48 @@
             "submission_id": 4,
             "taxon": null,
             "version": "$Id: OSG_qudt-(v1.1).ttl 4989 2011-06-01 21:56:40Z RalphHodgson $"
         },
         "topics": null
     },
     {
+        "acronym": "GFVO",
+        "name": "Genomic Feature and Variation Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2016-10-14T23:45:32Z",
+            "date_released": "2015-07-14T00:00:00Z",
+            "description": "The Genomic Feature and Variation Ontology (GFVO) is modeled to represent genomic data using the Resource Description Format (RDF) or Linked Data using JSON (JSON-LD). It is captures the contents of data files that adhere to the Generic Feature Format Version 3 (GFF3, http://www.sequenceontology.org/resources/gff3.html), the General Transfer Format (GTF, http://mblab.wustl.edu/GTF22.html), the Genome Variation Format Version 1 (GVF, http://www.sequenceontology.org/resources/gvf.html), and the Variant Call Format (VCF, http://vcftools.sourceforge.net/specs.html), as well as the FASTA format (http://en.wikipedia.org/wiki/FASTA_format). The creation of the ontology was inspired by previous work of Robert Hoehndorf on RDF2OWL (http://code.google.com/p/rdf2owl).\r\n\r\nNOTE: GFVO in BioPortal is a simplified version without ontology imports and having class/property equivalences removed. The full ontology is available at: http://www.biointerchange.org/gfvo",
+            "documentation": "https://www.codamono.com/biointerchange/documentation/",
+            "domain": null,
+            "download_url": "media/ontologies/GFVO/20/gfvo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.codamono.com/biointerchange/",
+            "id": 499,
+            "indexed": true,
+            "max_children": 8,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 102,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 33,
+            "products": null,
+            "publication": "https://peerj.com/articles/933/",
+            "publications": null,
+            "submission_id": 20,
+            "taxon": null,
+            "version": "1.0.6"
+        },
+        "topics": null
+    },
+    {
         "acronym": "QUDT2",
         "name": "Quantities, Units, Dimensions and Types (QUDT) Schema - Version 2.0",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -38669,211 +38768,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "GNO",
-        "name": "Glycan Naming and Subsumption Ontology (GNOme)",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2023-05-15T12:50:33.958072Z",
-            "date_released": "2023-05-15T12:50:33.958056Z",
-            "description": "GlyTouCan provides stable accessions for glycans described at varyious degrees of characterization, including compositions (no linkage) and topologies (no carbon bond positions or anomeric configurations). GNOme organizes these stable accessions for interative browsing, for text-based searching, and for automated reasoning with well-defined characterization levels.",
-            "documentation": null,
-            "domain": "chemistry and biochemistry",
-            "download_url": "media/ontologies/GNO/19/gno.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://gnome.glyomics.org/",
-            "id": 5808,
-            "indexed": true,
-            "max_children": 14351,
-            "max_depth": 13,
-            "md5sum": "d0b2828de04096b8ddc79fd782f35044",
-            "nb_classes": 159269,
-            "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 0,
-            "products": [
-                {
-                    "description": "Glycan Naming and Subsumption Ontology, OWL format (primary)",
-                    "id": "gno.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/gno.owl"
-                },
-                {
-                    "description": "Glycan Naming and Subsumption Ontology, OBO format (automated conversion from OWL)",
-                    "id": "gno.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/gno.obo"
-                },
-                {
-                    "description": "Glycan Naming and Subsumption Ontology, JSON format (automated conversion from OWL)",
-                    "id": "gno.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/gno.json"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://doi.org/10.5281/zenodo.6678278",
-                    "title": "GNOme - Glycan Naming and Subsumption Ontology"
-                }
-            ],
-            "submission_id": 19,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "APOLLO-SV",
-        "name": "Apollo Structured Vocabulary",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2017-12-01T02:00:14Z",
-            "date_released": "2017-12-01T02:00:14Z",
-            "description": "Apollo-SV, developed as an OWL specification, defines the terms and relations necessary for interoperation between epidemic simulators and public health application software that interface with these simulators.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/APOLLO-SV/21/apollo-sv.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ApolloDev/apollo-sv",
-            "id": 889,
-            "indexed": true,
-            "max_children": 42,
-            "max_depth": 37,
-            "md5sum": null,
-            "nb_classes": 1523,
-            "nb_inconsistent": 0,
-            "nb_individuals": 30,
-            "nb_properties": 119,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 21,
-            "taxon": null,
-            "version": "v4.0.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "ARO",
-        "name": "Antibiotic Resistance Ontology",
-        "species": [],
-        "status": "Classified",
-        "submission": {
-            "avg_children": 8,
-            "classifiable": true,
-            "date_created": "2023-05-29T12:00:21.441535Z",
-            "date_released": "2023-05-29T12:00:21.441511Z",
-            "description": "Antibiotic resistance genes and mutations",
-            "documentation": null,
-            "domain": "microbiology",
-            "download_url": "media/ontologies/ARO/26/aro.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/arpcard/aro",
-            "id": 5825,
-            "indexed": true,
-            "max_children": 954,
-            "max_depth": 9,
-            "md5sum": "45ea19df4107afb3139dffa1ee5f03eb",
-            "nb_classes": 6897,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 12,
-            "products": [
-                {
-                    "id": "aro.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/aro.owl"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/31665441",
-                    "title": "CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database."
-                }
-            ],
-            "submission_id": 26,
-            "taxon": null,
-            "version": null
-        },
-        "topics": []
-    },
-    {
-        "acronym": "ABD",
-        "name": "Anthology of Biosurveillance Diseases",
+        "acronym": "GLYCOCOO",
+        "name": "GlycoConjugate Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-09-14T01:00:04Z",
-            "date_released": "2016-09-14T01:00:04Z",
-            "description": "Our disease ontology provides information on infectious diseases, disease synonyms, transmission pathways, disease agents, affected populations and disease properties. Diseases have been grouped into syndromic disease categories, such that programmers can look through relevant categories, as well as at specific diseases. Organisms, linked to both agents and populations, are structured hierarchically, to provide multiple levels of organism resolution. In addition, both disease transmission and relevant disease properties are available to search. Disease properties include tags like 'notifiable diseases' and 'economic importance' to flag particular disease characteristics that may be of interest, but are not captured elsewhere.",
-            "documentation": "http://brd.bsvgateway.org/api",
+            "date_created": "2018-10-27T01:04:00Z",
+            "date_released": "2018-10-27T01:03:59Z",
+            "description": "GlycoConjugate Ontology (GlycoCoO) as a standard semantic framework to describe and represent glycoconjugate data in RDF.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ABD/4/abd.owl",
+            "download_url": "media/ontologies/GLYCOCOO/2/glycocoo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://brd.bsvgateway.org/disease",
-            "id": 505,
+            "home_page": "https://github.com/glycoinfo/GlycoCoO",
+            "id": 1965,
             "indexed": true,
-            "max_children": 758,
+            "max_children": 29,
             "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 1445,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 3,
-            "products": null,
-            "publication": "http://brd.bsvgateway.org/reference",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "AO",
-        "name": "Asthma Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2017-02-23T22:21:58Z",
-            "date_released": "2016-03-02T00:00:00Z",
-            "description": "Asthma ontology",
-            "documentation": "www.clininf.eu",
-            "domain": null,
-            "download_url": "media/ontologies/AO/2/ao.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "www.clininf.eu",
-            "id": 288,
-            "indexed": true,
-            "max_children": 30,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 289,
+            "nb_classes": 51,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 1,
+            "nb_properties": 27,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.3"
+            "version": "1.1.3"
         },
         "topics": null
     },
     {
         "acronym": "GO-PLUS",
         "name": "go-plus",
         "species": null,
@@ -39006,14 +38938,48 @@
             "submission_id": 2,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
+        "acronym": "OPTION",
+        "name": "OPTImization Algorithm Benchmarking ONtology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2021-01-30T16:27:58Z",
+            "date_released": "2021-01-30T00:00:00Z",
+            "description": "OPTION:OPTImization Algorithm Benchmarking ONtology",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/OPTION/1/option.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4235,
+            "indexed": true,
+            "max_children": 36,
+            "max_depth": 11,
+            "md5sum": null,
+            "nb_classes": 286,
+            "nb_inconsistent": 0,
+            "nb_individuals": 23,
+            "nb_properties": 58,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.0.0"
+        },
+        "topics": null
+    },
+    {
         "acronym": "OOEVV",
         "name": "Ontology of Experimental Variables and Values",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -39040,14 +39006,48 @@
             "submission_id": 4,
             "taxon": null,
             "version": "0.0.4"
         },
         "topics": null
     },
     {
+        "acronym": "ooevv-vaccine",
+        "name": "OoEVV vaccine protection experiments ",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2012-07-19T06:12:07Z",
+            "date_released": "2012-07-18T07:00:00Z",
+            "description": "This serves as an instantiation of the OoEVV ontology design pattern for variables derived from vaccine protection studies that have been previously indexed in the Los Alamos National Laboratories (LANL) HIV Vaccine Trials Database. This is a comprehensive index of research outcomes derived from vaccine studies in nonhuman primates and provides an excellent index of papers for examination with our approach. ",
+            "documentation": "http://www.isi.edu/projects/ooevv/overview",
+            "domain": null,
+            "download_url": "media/ontologies/ooevv-vaccine/1/ooevv-vaccine.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://bmkeg.isi.edu",
+            "id": 140,
+            "indexed": true,
+            "max_children": 10,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 48,
+            "nb_inconsistent": 0,
+            "nb_individuals": 168,
+            "nb_properties": 50,
+            "products": null,
+            "publication": "http://www.isi.edu/research_groups/bmkeg/publications",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "0.0.4"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ooevv-tractTrace",
         "name": "OoEVV tract tracing experiments ",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -39074,44 +39074,44 @@
             "submission_id": 1,
             "taxon": null,
             "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "ooevv-vaccine",
-        "name": "OoEVV vaccine protection experiments ",
+        "acronym": "OPDE",
+        "name": "OPD-Ethiopia",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2012-07-19T06:12:07Z",
-            "date_released": "2012-07-18T07:00:00Z",
-            "description": "This serves as an instantiation of the OoEVV ontology design pattern for variables derived from vaccine protection studies that have been previously indexed in the Los Alamos National Laboratories (LANL) HIV Vaccine Trials Database. This is a comprehensive index of research outcomes derived from vaccine studies in nonhuman primates and provides an excellent index of papers for examination with our approach. ",
-            "documentation": "http://www.isi.edu/projects/ooevv/overview",
+            "date_created": "2021-06-02T12:31:18Z",
+            "date_released": "2021-06-02T00:00:00Z",
+            "description": "This is an OPD Registry Form in Ethiopia.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ooevv-vaccine/1/ooevv-vaccine.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://bmkeg.isi.edu",
-            "id": 140,
+            "download_url": "media/ontologies/OPDE/1/opde.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 4579,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 4,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 48,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 168,
-            "nb_properties": 50,
+            "nb_individuals": 59,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://www.isi.edu/research_groups/bmkeg/publications",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "0.0.4"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "OOSTT",
         "name": "Ontology of Organizational Structures of Trauma centers and Trauma systems",
         "species": null,
@@ -39147,48 +39147,14 @@
             "submission_id": 11,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SCIO",
-        "name": "Spinal Cord Injury Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2017-09-20T14:29:51Z",
-            "date_released": "2017-09-19T00:00:00Z",
-            "description": "Representation of pre-clinical studies in the domain of spinal cord injury therapies",
-            "documentation": "http://www.psink.de",
-            "domain": null,
-            "download_url": "media/ontologies/SCIO/2/scio.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://www.psink.de",
-            "id": 672,
-            "indexed": true,
-            "max_children": 41,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 526,
-            "nb_inconsistent": 0,
-            "nb_individuals": 1788,
-            "nb_properties": 210,
-            "products": null,
-            "publication": "https://pub.uni-bielefeld.de/publication/2913603",
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "1"
-        },
-        "topics": null
-    },
-    {
         "acronym": "OPB",
         "name": "Ontology of Physics for Biology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -39215,146 +39181,112 @@
             "submission_id": 13,
             "taxon": null,
             "version": "1.05"
         },
         "topics": null
     },
     {
-        "acronym": "OPDE",
-        "name": "OPD-Ethiopia",
+        "acronym": "OPTION-ONTOLOGY",
+        "name": "Optimization Algorithm Benchmarking Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-06-02T12:31:18Z",
-            "date_released": "2021-06-02T00:00:00Z",
-            "description": "This is an OPD Registry Form in Ethiopia.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OPDE/1/opde.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4579,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 59,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "HINO",
-        "name": "Human Interaction Network Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 30,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2014-06-27T19:32:47Z",
-            "date_released": "2014-06-27T19:32:47Z",
-            "description": "The Human Interaction Network Ontology (HINO) is an INO extension for the domain of human interaction networks. It has currently incoporated Reactome reactions and pathways. Like INO, HINO aligns with BFO. HINO is developed by following the OBO Foundry principles. ",
+            "date_created": "2022-11-30T16:00:18Z",
+            "date_released": "2021-04-12T00:00:00Z",
+            "description": "OPTImization Algorithm Benchmarking ONtology",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/HINO/6/hino.owl",
+            "download_url": "media/ontologies/OPTION-ONTOLOGY/3/option-ontology.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.ino-ontology.org/",
-            "id": 367,
+            "home_page": null,
+            "id": 5531,
             "indexed": true,
-            "max_children": 8962,
-            "max_depth": 27,
+            "max_children": 36,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 37777,
+            "nb_classes": 347,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 71,
+            "nb_individuals": 28,
+            "nb_properties": 59,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 3,
             "taxon": null,
-            "version": "Vision Release: 1.0.69"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "GVO",
-        "name": "Genome Variation Ontology",
+        "acronym": "ABD",
+        "name": "Anthology of Biosurveillance Diseases",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-09-09T00:52:14Z",
-            "date_released": "2022-09-08T00:00:00Z",
-            "description": "The Genome Variation Ontology (GVO) is an ontology for the systematic description of various genomic variations, including complex structural variations in genomes.",
-            "documentation": null,
+            "date_created": "2016-09-14T01:00:04Z",
+            "date_released": "2016-09-14T01:00:04Z",
+            "description": "Our disease ontology provides information on infectious diseases, disease synonyms, transmission pathways, disease agents, affected populations and disease properties. Diseases have been grouped into syndromic disease categories, such that programmers can look through relevant categories, as well as at specific diseases. Organisms, linked to both agents and populations, are structured hierarchically, to provide multiple levels of organism resolution. In addition, both disease transmission and relevant disease properties are available to search. Disease properties include tags like 'notifiable diseases' and 'economic importance' to flag particular disease characteristics that may be of interest, but are not captured elsewhere.",
+            "documentation": "http://brd.bsvgateway.org/api",
             "domain": null,
-            "download_url": "media/ontologies/GVO/3/gvo.owl",
+            "download_url": "media/ontologies/ABD/4/abd.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://genome-variation.org/resource/gvo",
-            "id": 5349,
+            "home_page": "http://brd.bsvgateway.org/disease",
+            "id": 505,
             "indexed": true,
-            "max_children": 19,
-            "max_depth": 4,
+            "max_children": 758,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 77,
+            "nb_classes": 1445,
             "nb_inconsistent": 0,
-            "nb_individuals": 99,
-            "nb_properties": 17,
+            "nb_individuals": 0,
+            "nb_properties": 3,
             "products": null,
-            "publication": null,
+            "publication": "http://brd.bsvgateway.org/reference",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 4,
             "taxon": null,
-            "version": "2022-09-09"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "HPIO",
-        "name": "Host Pathogen Interactions Ontology",
+        "acronym": "AO",
+        "name": "Asthma Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2011-02-08T20:29:10Z",
-            "date_released": "2011-01-25T08:00:00Z",
-            "description": "Ontology for host pathogen interactions in farmed animals",
-            "documentation": null,
+            "date_created": "2017-02-23T22:21:58Z",
+            "date_released": "2016-03-02T00:00:00Z",
+            "description": "Asthma ontology",
+            "documentation": "www.clininf.eu",
             "domain": null,
-            "download_url": "media/ontologies/HPIO/2/hpio.owl",
+            "download_url": "media/ontologies/AO/2/ao.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 770,
+            "home_page": "www.clininf.eu",
+            "id": 288,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 10,
+            "max_children": 30,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 275,
+            "nb_classes": 289,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 6,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1"
+            "version": "1.3"
         },
         "topics": null
     },
     {
         "acronym": "OPDZIMBABWE",
         "name": "OPDZimbabwe",
         "species": null,
@@ -39497,146 +39429,214 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "OPTION",
-        "name": "OPTImization Algorithm Benchmarking ONtology",
+        "acronym": "ORDO_CS",
+        "name": "ORDO Czech",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2021-01-30T16:27:58Z",
-            "date_released": "2021-01-30T00:00:00Z",
-            "description": "OPTION:OPTImization Algorithm Benchmarking ONtology",
-            "documentation": null,
+            "date_created": "2023-07-03T15:02:27Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "This ontology is the czech version of ORDO with definition, labels and synonyms in czech.",
+            "documentation": "https://www.orphadata.com/ordo/",
             "domain": null,
-            "download_url": "media/ontologies/OPTION/1/option.owl",
+            "download_url": "media/ontologies/ORDO_CS/3/ordo_cs.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4235,
+            "home_page": "https://www.orphadata.com/ordo/",
+            "id": 5924,
             "indexed": true,
-            "max_children": 36,
+            "max_children": 4370,
             "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 286,
+            "nb_classes": 15447,
             "nb_inconsistent": 0,
-            "nb_individuals": 23,
-            "nb_properties": 58,
+            "nb_individuals": 0,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "4.3"
         },
         "topics": null
     },
     {
-        "acronym": "OPTION-ONTOLOGY",
-        "name": "Optimization Algorithm Benchmarking Ontology",
+        "acronym": "SCIO",
+        "name": "Spinal Cord Injury Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-11-30T16:00:18Z",
-            "date_released": "2021-04-12T00:00:00Z",
-            "description": "OPTImization Algorithm Benchmarking ONtology",
+            "date_created": "2017-09-20T14:29:51Z",
+            "date_released": "2017-09-19T00:00:00Z",
+            "description": "Representation of pre-clinical studies in the domain of spinal cord injury therapies",
+            "documentation": "http://www.psink.de",
+            "domain": null,
+            "download_url": "media/ontologies/SCIO/2/scio.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.psink.de",
+            "id": 672,
+            "indexed": true,
+            "max_children": 41,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 526,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1788,
+            "nb_properties": 210,
+            "products": null,
+            "publication": "https://pub.uni-bielefeld.de/publication/2913603",
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "HINO",
+        "name": "Human Interaction Network Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 30,
+            "classifiable": true,
+            "date_created": "2014-06-27T19:32:47Z",
+            "date_released": "2014-06-27T19:32:47Z",
+            "description": "The Human Interaction Network Ontology (HINO) is an INO extension for the domain of human interaction networks. It has currently incoporated Reactome reactions and pathways. Like INO, HINO aligns with BFO. HINO is developed by following the OBO Foundry principles. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OPTION-ONTOLOGY/3/option-ontology.owl",
+            "download_url": "media/ontologies/HINO/6/hino.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5531,
+            "home_page": "http://www.ino-ontology.org/",
+            "id": 367,
             "indexed": true,
-            "max_children": 36,
-            "max_depth": 11,
+            "max_children": 8962,
+            "max_depth": 27,
             "md5sum": null,
-            "nb_classes": 347,
+            "nb_classes": 37777,
             "nb_inconsistent": 0,
-            "nb_individuals": 28,
-            "nb_properties": 59,
+            "nb_individuals": 1,
+            "nb_properties": 71,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 6,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "Vision Release: 1.0.69"
         },
         "topics": null
     },
     {
-        "acronym": "ORDO_CS",
-        "name": "ORDO Czech",
+        "acronym": "GVO",
+        "name": "Genome Variation Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-07-03T15:02:27Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "This ontology is the czech version of ORDO with definition, labels and synonyms in czech.",
-            "documentation": "https://www.orphadata.com/ordo/",
+            "date_created": "2022-09-09T00:52:14Z",
+            "date_released": "2022-09-08T00:00:00Z",
+            "description": "The Genome Variation Ontology (GVO) is an ontology for the systematic description of various genomic variations, including complex structural variations in genomes.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ORDO_CS/3/ordo_cs.owl",
+            "download_url": "media/ontologies/GVO/3/gvo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com/ordo/",
-            "id": 5924,
+            "home_page": "http://genome-variation.org/resource/gvo",
+            "id": 5349,
             "indexed": true,
-            "max_children": 4370,
-            "max_depth": 11,
+            "max_children": 19,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 15447,
+            "nb_classes": 77,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_individuals": 99,
+            "nb_properties": 17,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 3,
             "taxon": null,
-            "version": "4.3"
+            "version": "2022-09-09"
         },
         "topics": null
     },
     {
-        "acronym": "iceci-mechanism",
-        "name": "iceci-mechanism",
+        "acronym": "HPIO",
+        "name": "Host Pathogen Interactions Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2010-05-07T21:09:44Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI Mechanism of injury-full version C2",
+            "date_created": "2011-02-08T20:29:10Z",
+            "date_released": "2011-01-25T08:00:00Z",
+            "description": "Ontology for host pathogen interactions in farmed animals",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-mechanism/2/iceci-mechanism.owl",
+            "download_url": "media/ontologies/HPIO/2/hpio.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 213,
+            "id": 770,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 4,
+            "max_children": 20,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 233,
+            "nb_classes": 275,
             "nb_inconsistent": 0,
-            "nb_individuals": 230,
-            "nb_properties": 7,
+            "nb_individuals": 0,
+            "nb_properties": 6,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "1.02"
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "OREX-SAM",
+        "name": "OSIRIS-REx sample type",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2022-09-08T23:31:32Z",
+            "date_released": "2022-09-08T00:00:00Z",
+            "description": "Sample types from JSC-SAMIS Interface Control Document. Edited by S.M. Richard. These sample types are limited to those expected from the OSIRIS-REx sample return, this is not an exhaustive sample type categorization.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/OREX-SAM/1/orex-sam.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5347,
+            "indexed": true,
+            "max_children": 3,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 5,
+            "nb_inconsistent": 0,
+            "nb_individuals": 25,
+            "nb_properties": 17,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "0.2.0"
         },
         "topics": null
     },
     {
         "acronym": "GEOSPARQL",
         "name": "GeoSPARQL",
         "species": null,
@@ -39667,78 +39667,565 @@
             "submission_id": 1,
             "taxon": null,
             "version": "OGC GeoSPARQL 1.0"
         },
         "topics": null
     },
     {
-        "acronym": "iceci-substance_use",
-        "name": "iceci-substance_use",
+        "acronym": "FLOPO",
+        "name": "Flora Phenotype Ontology",
+        "species": [],
+        "status": "Classified",
+        "submission": {
+            "avg_children": 69,
+            "classifiable": true,
+            "date_created": "2019-09-09T12:20:11.778262Z",
+            "date_released": "2019-09-09T12:20:11.778238Z",
+            "description": "Traits and phenotypes of flowering plants occurring in digitized Floras",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/FLOPO/10/flopo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/flora-phenotype-ontology/flopoontology",
+            "id": 2901,
+            "indexed": true,
+            "max_children": 2289,
+            "max_depth": 11,
+            "md5sum": "b93ed5e841a68be02c4f5ef40d923fd2",
+            "nb_classes": 28971,
+            "nb_inconsistent": 107,
+            "nb_individuals": 0,
+            "nb_properties": 135,
+            "products": [
+                {
+                    "id": "flopo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/flopo.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 10,
+            "taxon": {
+                "id": "NCBITaxon:33090",
+                "label": "Viridiplantae"
+            },
+            "version": null
+        },
+        "topics": []
+    },
+    {
+        "acronym": "ORDO_PT",
+        "name": "ORDO Portuguese",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 15,
+            "classifiable": true,
+            "date_created": "2023-07-03T15:10:57Z",
+            "date_released": "2023-07-01T00:00:00Z",
+            "description": "This ontology is the portuguese version of ORDO with definition, labels and synonyms in portuguese.",
+            "documentation": "https://www.orphadata.com/ordo/",
+            "domain": null,
+            "download_url": "media/ontologies/ORDO_PT/4/ordo_pt.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.orphadata.com/ordo/",
+            "id": 5927,
+            "indexed": true,
+            "max_children": 4370,
+            "max_depth": 11,
+            "md5sum": null,
+            "nb_classes": 15447,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 13,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 4,
+            "taxon": null,
+            "version": "4.3"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PAYOR",
+        "name": "Payor",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2022-12-07T20:00:29Z",
+            "date_released": "2022-12-07T00:00:00Z",
+            "description": "This class refers to the entity that made the payment on behalf of the interviewee.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PAYOR/1/payor.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 5542,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 6,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PARTUMDO",
+        "name": "postpartum depression ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2010-05-07T21:17:27Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "ICECI Substance use\t C6 and C7",
+            "date_created": "2021-07-08T15:08:06Z",
+            "date_released": "2021-07-08T00:00:00Z",
+            "description": "This ontology is for use in extracting postpartum depression condition codes, and also comorbidities and diseases often mistaken for postpartum depression. The ontology describes not only the codes but also the relationships between these various diseases/risk factors for use in automatically extracting relevant patient populations from Electronic Health Records",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/iceci-substance_use/2/iceci-substance_use.owl",
+            "download_url": "media/ontologies/PARTUMDO/1/partumdo.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 161,
+            "id": 4572,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
+            "max_children": 23,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 16,
+            "nb_classes": 734,
             "nb_inconsistent": 0,
-            "nb_individuals": 13,
-            "nb_properties": 7,
+            "nb_individuals": 4844,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.02"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "O-REXTECH",
-        "name": "OSIRIS-REx Analytical Technique",
+        "acronym": "PAE",
+        "name": "Plant Anatomy",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2013-08-22T01:04:12Z",
+            "date_released": "2015-08-20T00:00:00Z",
+            "description": "REPLACED BY: Plant Ontology (PO). A controlled vocabulary of plant morphological and anatomical structures representing organs, tissues, cell types, and their biological relationships based on spatial and developmental organization. Note that this has been subsumed into the PO. This file is created by filtering plant_ontology_assert.obo to contain only terms from the plant anatomical entity branch of the PO. For more information, please see: http://palea.cgrb.oregonstate.edu/viewsvn/Poc/tags/live/",
+            "documentation": "http://www.plantontology.org/docs/docs.html",
+            "domain": null,
+            "download_url": "media/ontologies/PAE/13/pae.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://www.plantontology.org",
+            "id": 128,
+            "indexed": true,
+            "max_children": 62,
+            "max_depth": 10,
+            "md5sum": null,
+            "nb_classes": 1548,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 9,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 13,
+            "taxon": null,
+            "version": "Release #20"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "APOLLO_SV",
+        "name": "Apollo Structured Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-09-08T23:27:27Z",
-            "date_released": "2022-09-08T00:00:00Z",
-            "description": "second draft, M4M workshop, testing SKOS-Play and workflows. Add top concepts and more hierarchy.",
+            "date_created": "2023-06-05T12:00:15.394476Z",
+            "date_released": "2023-06-05T12:00:15.394460Z",
+            "description": "An OWL2 ontology of phenomena in infectious disease epidemiology and population biology for use in epidemic simulation.",
+            "documentation": null,
+            "domain": "health",
+            "download_url": "media/ontologies/APOLLO_SV/22/apollo_sv.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/ApolloDev/apollo-sv",
+            "id": 5836,
+            "indexed": true,
+            "max_children": 39,
+            "max_depth": 37,
+            "md5sum": "3c2b6e7d010fffa102974ef3bd4cecee",
+            "nb_classes": 1693,
+            "nb_inconsistent": 0,
+            "nb_individuals": 27,
+            "nb_properties": 220,
+            "products": [
+                {
+                    "id": "apollo_sv.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/apollo_sv.owl"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://doi.org/10.1186/s13326-016-0092-y",
+                    "title": "The Apollo Structured Vocabulary: an OWL2 ontology of phenomena in infectious disease epidemiology and population biology for use in epidemic simulation"
+                }
+            ],
+            "submission_id": 22,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PATO",
+        "name": "Phenotype And Trait Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-06-05T13:51:39.265623Z",
+            "date_released": "2023-06-05T13:51:39.265610Z",
+            "description": "An ontology of phenotypic qualities (properties, attributes or characteristics)",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/PATO/195/pato.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/pato-ontology/pato/",
+            "id": 5847,
+            "indexed": true,
+            "max_children": 1003,
+            "max_depth": 33,
+            "md5sum": "bd472f4971492109493d4ad8a779a8dd",
+            "nb_classes": 10589,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 309,
+            "products": [
+                {
+                    "id": "pato.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pato.owl"
+                },
+                {
+                    "id": "pato.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pato.obo"
+                },
+                {
+                    "id": "pato.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pato.json"
+                },
+                {
+                    "description": "Includes axioms linking to other ontologies, but no imports of those ontologies",
+                    "id": "pato/pato-base.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pato/pato-base.owl"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/28387809",
+                    "title": "The anatomy of phenotype ontologies: principles, properties and applications"
+                },
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/15642100",
+                    "title": "Using ontologies to describe mouse phenotypes"
+                }
+            ],
+            "submission_id": 195,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "GFO-BIO",
+        "name": "General Formal Ontology for Biology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2010-03-02T19:27:14Z",
+            "date_released": "2010-07-12T07:00:00Z",
+            "description": "GFO-Bio is a biological core ontology built on the General Formal Ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/O-REXTECH/2/o-rextech.skos",
+            "download_url": "media/ontologies/GFO-BIO/1/gfo-bio.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.onto-med.de/ontologies/gfo-bio/",
+            "id": 59,
+            "indexed": true,
+            "max_children": 11,
+            "max_depth": 11,
+            "md5sum": null,
+            "nb_classes": 168,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1,
+            "nb_properties": 73,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PAV",
+        "name": "PAV Provenance, Authoring and Versioning",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2017-01-20T02:12:41Z",
+            "date_released": "2017-01-20T02:12:41Z",
+            "description": "PAV is a lightweight ontology for tracking Provenance, Authoring and Versioning. PAV specializes the W3C provenance ontology PROV-O in order to describe authorship, curation and digital creation of online resources.\r\n\r\nThis ontology describes the defined PAV properties and their usage. Note that PAV does not define any explicit classes or domain/ranges, as every property is meant to be used directly on the described online resource.",
+            "documentation": "http://pav-ontology.github.io/pav/",
+            "domain": null,
+            "download_url": "media/ontologies/PAV/3/pav.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/pav-ontology/pav/",
+            "id": 701,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 0,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 9,
+            "nb_properties": 40,
+            "products": null,
+            "publication": "https://github.com/pav-ontology/pav/wiki/Publications",
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "2.3.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "GLYCO",
+        "name": "Glycomics Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": false,
+            "date_created": "2012-10-29T12:31:37Z",
+            "date_released": "2012-10-29T07:00:00Z",
+            "description": "The GlycO ontology provides an up-to-date knowledge base of experimentally verified glycan structures. Glycan (oligosaccharide or polysaccharide) structures are represented as trees of monosaccharide residues. Linkage to proteins and lipids is supported as well. Insertion of a new glycan is controlled by curation process that includes matching the new glycan against a canonical glyco-tree (a highly branched representation for a family of glycans).",
+            "documentation": "http://glycomics.ccrc.uga.edu/core4/informatics-ontologies.html",
+            "domain": null,
+            "download_url": "media/ontologies/GLYCO/1/glyco.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://glycomics.ccrc.uga.edu/core4/informatics-ontologies.html",
+            "id": 122,
+            "indexed": false,
+            "max_children": 11,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 230,
+            "nb_inconsistent": 0,
+            "nb_individuals": 1784,
+            "nb_properties": 65,
+            "products": null,
+            "publication": "http://glycomics.ccrc.uga.edu/core4/informatics-ontologies.html",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "4.6.0.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PATIT",
+        "name": "Placental Investigative Technique",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2020-01-13T20:09:29Z",
+            "date_released": "2020-01-13T00:00:00Z",
+            "description": "Experimental methods and observation types used in placental research. The PAT Investigative Techniques ontology builds on work done for a handful of existing ontologies, including the Ontology for Biomedical Investigations (OBI) and NCI Thesaurus (NCIT), and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PATIT/1/patit.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3304,
+            "indexed": true,
+            "max_children": 18,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 139,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PATMHC",
+        "name": "Placental Maternal Health Conditions",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2020-01-13T19:47:07Z",
+            "date_released": "2020-01-13T00:00:00Z",
+            "description": "Physical and psychological disorders in the mother that occur during pregnancy, childbirth, and the postpartum period. These include both pre-existing and gestational maternal health conditions that may result in adverse pregnancy outcomes. The Maternal Conditions Ontology builds on the work done for a handful of existing ontologies, including the Human Phenotype Ontology (HPO) and NCI Thesaurus (NCIT), and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PATMHC/1/patmhc.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3308,
+            "indexed": true,
+            "max_children": 11,
+            "max_depth": 6,
+            "md5sum": null,
+            "nb_classes": 228,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "ZONMW-GENERIC",
+        "name": "ZonMw Generic Terms",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2022-05-25T20:01:09Z",
+            "date_released": "2022-05-25T00:00:00Z",
+            "description": "General-purpose terms for the ZonMw ",
+            "documentation": "https://raw.githubusercontent.com/fair-data-collective/zonmw-generic-vocabulary/",
+            "domain": null,
+            "download_url": "media/ontologies/ZONMW-GENERIC/21/zonmw-generic.skos",
             "has_ontology_language": "SKOS",
+            "home_page": "https://raw.githubusercontent.com/fair-data-collective/zonmw-generic-vocabulary/",
+            "id": 5110,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 162,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 21,
+            "taxon": null,
+            "version": "1.5.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PATGV",
+        "name": "Placental Genetic Variance",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": true,
+            "date_created": "2020-01-13T20:00:27Z",
+            "date_released": "2020-01-13T00:00:00Z",
+            "description": "Includes variations of DNA sequence, chromosomal structure and copy number, as well as RNA and translational variation. The Genetic Variation ontology expands on work done for Variation Ontology (VariO) and Sequence Types and Features Ontology (SO) which can be found on NCBO. This ontology was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PATGV/1/patgv.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5354,
+            "id": 3299,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 2,
+            "max_children": 10,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 5,
+            "nb_classes": 73,
             "nb_inconsistent": 0,
-            "nb_individuals": 56,
-            "nb_properties": 17,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.0.2"
+            "version": "1.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PATHLEX",
+        "name": "Anatomic Pathology Lexicon",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 28,
+            "classifiable": true,
+            "date_created": "2013-01-22T20:05:19Z",
+            "date_released": "2013-01-22T08:00:00Z",
+            "description": "PathLex, a comprehensive lexicon\u2014a unified language of anatomic pathology terms\u2014for standardized indexing and retrieval of anatomic pathology information resources. ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PATHLEX/5/pathlex.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 210,
+            "indexed": true,
+            "max_children": 1522,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 1786,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 5,
+            "taxon": null,
+            "version": "1.2"
         },
         "topics": null
     },
     {
         "acronym": "OREX-WG",
         "name": "OSIRIS-REx sample analysis workgroups",
         "species": null,
@@ -39915,44 +40402,78 @@
             "submission_id": 11,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "GFO-BIO",
-        "name": "General Formal Ontology for Biology",
+        "acronym": "iceci-mechanism",
+        "name": "iceci-mechanism",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2010-03-02T19:27:14Z",
-            "date_released": "2010-07-12T07:00:00Z",
-            "description": "GFO-Bio is a biological core ontology built on the General Formal Ontology.",
+            "date_created": "2010-05-07T21:09:44Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "ICECI Mechanism of injury-full version C2",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/GFO-BIO/1/gfo-bio.owl",
+            "download_url": "media/ontologies/iceci-mechanism/2/iceci-mechanism.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.onto-med.de/ontologies/gfo-bio/",
-            "id": 59,
+            "home_page": null,
+            "id": 213,
             "indexed": true,
             "max_children": 11,
-            "max_depth": 11,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 168,
+            "nb_classes": 233,
             "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 73,
+            "nb_individuals": 230,
+            "nb_properties": 7,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.02"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PATCT",
+        "name": "Placental Cell Type",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2020-01-13T20:16:28Z",
+            "date_released": "2020-01-13T00:00:00Z",
+            "description": "Cell types associated with placental structure and function. This Cell Type ontology builds on the Cell Ontology (CL) as well as the BRENDA Tissue and Enzyme Source Ontology (BTO) and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PATCT/1/patct.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 3300,
+            "indexed": true,
+            "max_children": 10,
+            "max_depth": 5,
+            "md5sum": null,
+            "nb_classes": 153,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": "1.0"
         },
         "topics": null
     },
     {
         "acronym": "ORDO_PL",
         "name": "ORDO POLISH",
         "species": null,
@@ -39983,48 +40504,14 @@
             "submission_id": 8,
             "taxon": null,
             "version": "4.3"
         },
         "topics": null
     },
     {
-        "acronym": "GLYCO",
-        "name": "Glycomics Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": false,
-            "date_created": "2012-10-29T12:31:37Z",
-            "date_released": "2012-10-29T07:00:00Z",
-            "description": "The GlycO ontology provides an up-to-date knowledge base of experimentally verified glycan structures. Glycan (oligosaccharide or polysaccharide) structures are represented as trees of monosaccharide residues. Linkage to proteins and lipids is supported as well. Insertion of a new glycan is controlled by curation process that includes matching the new glycan against a canonical glyco-tree (a highly branched representation for a family of glycans).",
-            "documentation": "http://glycomics.ccrc.uga.edu/core4/informatics-ontologies.html",
-            "domain": null,
-            "download_url": "media/ontologies/GLYCO/1/glyco.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://glycomics.ccrc.uga.edu/core4/informatics-ontologies.html",
-            "id": 122,
-            "indexed": false,
-            "max_children": 11,
-            "max_depth": 8,
-            "md5sum": null,
-            "nb_classes": 230,
-            "nb_inconsistent": 0,
-            "nb_individuals": 1784,
-            "nb_properties": 65,
-            "products": null,
-            "publication": "http://glycomics.ccrc.uga.edu/core4/informatics-ontologies.html",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "4.6.0.0"
-        },
-        "topics": null
-    },
-    {
         "acronym": "PACO",
         "name": "Physical Activity Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
@@ -40051,710 +40538,712 @@
             "submission_id": 2,
             "taxon": null,
             "version": "0.2"
         },
         "topics": null
     },
     {
-        "acronym": "PAE",
-        "name": "Plant Anatomy",
+        "acronym": "ZP",
+        "name": "Zebrafish Phenotype Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2013-08-22T01:04:12Z",
-            "date_released": "2015-08-20T00:00:00Z",
-            "description": "REPLACED BY: Plant Ontology (PO). A controlled vocabulary of plant morphological and anatomical structures representing organs, tissues, cell types, and their biological relationships based on spatial and developmental organization. Note that this has been subsumed into the PO. This file is created by filtering plant_ontology_assert.obo to contain only terms from the plant anatomical entity branch of the PO. For more information, please see: http://palea.cgrb.oregonstate.edu/viewsvn/Poc/tags/live/",
-            "documentation": "http://www.plantontology.org/docs/docs.html",
-            "domain": null,
-            "download_url": "media/ontologies/PAE/13/pae.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://www.plantontology.org",
-            "id": 128,
+            "date_created": "2022-12-19T13:40:54.436734Z",
+            "date_released": "2022-12-19T13:40:54.436722Z",
+            "description": "The Zebrafish Phenotype Ontology formally defines all phenotypes of the Zebrafish model organism.",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/ZP/8/zp.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/obophenotype/zebrafish-phenotype-ontology",
+            "id": 5566,
             "indexed": true,
-            "max_children": 62,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 1548,
+            "max_children": 3872,
+            "max_depth": 41,
+            "md5sum": "03430b567bf153216c0fa4c3440b3b24",
+            "nb_classes": 63473,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 9,
-            "products": null,
+            "nb_individuals": 18,
+            "nb_properties": 299,
+            "products": [
+                {
+                    "id": "zp.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/zp.owl"
+                },
+                {
+                    "id": "zp.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/zp.obo"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 13,
+            "submission_id": 8,
             "taxon": null,
-            "version": "Release #20"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "ORDO_PT",
-        "name": "ORDO Portuguese",
+        "acronym": "iceci-substance_use",
+        "name": "iceci-substance_use",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-07-03T15:10:57Z",
-            "date_released": "2023-07-01T00:00:00Z",
-            "description": "This ontology is the portuguese version of ORDO with definition, labels and synonyms in portuguese.",
-            "documentation": "https://www.orphadata.com/ordo/",
+            "date_created": "2010-05-07T21:17:27Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "ICECI Substance use\t C6 and C7",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ORDO_PT/4/ordo_pt.owl",
+            "download_url": "media/ontologies/iceci-substance_use/2/iceci-substance_use.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://www.orphadata.com/ordo/",
-            "id": 5927,
+            "home_page": null,
+            "id": 161,
             "indexed": true,
-            "max_children": 4370,
-            "max_depth": 11,
+            "max_children": 5,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 15447,
+            "nb_classes": 16,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_individuals": 13,
+            "nb_properties": 7,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 4,
+            "submission_id": 2,
             "taxon": null,
-            "version": "4.3"
+            "version": "1.02"
         },
         "topics": null
     },
     {
-        "acronym": "OREX-SAM",
-        "name": "OSIRIS-REx sample type",
+        "acronym": "O-REXTECH",
+        "name": "OSIRIS-REx Analytical Technique",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-09-08T23:31:32Z",
+            "date_created": "2022-09-08T23:27:27Z",
             "date_released": "2022-09-08T00:00:00Z",
-            "description": "Sample types from JSC-SAMIS Interface Control Document. Edited by S.M. Richard. These sample types are limited to those expected from the OSIRIS-REx sample return, this is not an exhaustive sample type categorization.",
+            "description": "second draft, M4M workshop, testing SKOS-Play and workflows. Add top concepts and more hierarchy.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OREX-SAM/1/orex-sam.skos",
+            "download_url": "media/ontologies/O-REXTECH/2/o-rextech.skos",
             "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 5347,
+            "id": 5354,
             "indexed": true,
             "max_children": 3,
             "max_depth": 2,
             "md5sum": null,
             "nb_classes": 5,
             "nb_inconsistent": 0,
-            "nb_individuals": 25,
+            "nb_individuals": 56,
             "nb_properties": 17,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.2.0"
+            "version": "0.0.2"
         },
         "topics": null
     },
     {
-        "acronym": "PARTUMDO",
-        "name": "postpartum depression ontology",
+        "acronym": "PATEL",
+        "name": "Archana Patel",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-07-08T15:08:06Z",
-            "date_released": "2021-07-08T00:00:00Z",
-            "description": "This ontology is for use in extracting postpartum depression condition codes, and also comorbidities and diseases often mistaken for postpartum depression. The ontology describes not only the codes but also the relationships between these various diseases/risk factors for use in automatically extracting relevant patient populations from Electronic Health Records",
+            "date_created": "2021-03-18T20:20:34Z",
+            "date_released": "2021-03-18T00:00:00Z",
+            "description": "Covid19 Impact on Banking Ontology (Covid19-IBO) provides semantic information about the impact of the Covid-19 on the banking sector of India",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PARTUMDO/1/partumdo.owl",
+            "download_url": "media/ontologies/PATEL/1/patel.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4572,
+            "id": 4359,
             "indexed": true,
-            "max_children": 23,
-            "max_depth": 10,
+            "max_children": 14,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 734,
+            "nb_classes": 123,
             "nb_inconsistent": 0,
-            "nb_individuals": 4844,
-            "nb_properties": 13,
+            "nb_individuals": 0,
+            "nb_properties": 28,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "PATCT",
-        "name": "Placental Cell Type",
+        "acronym": "AURA",
+        "name": "KB_Bio_101",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 15,
             "classifiable": true,
-            "date_created": "2020-01-13T20:16:28Z",
-            "date_released": "2020-01-13T00:00:00Z",
-            "description": "Cell types associated with placental structure and function. This Cell Type ontology builds on the Cell Ontology (CL) as well as the BRENDA Tissue and Enzyme Source Ontology (BTO) and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
-            "documentation": null,
+            "date_created": "2014-01-21T19:51:39Z",
+            "date_released": "2014-01-21T00:00:00Z",
+            "description": "KB_Bio_101 is the ontology of the AURA + Inquire project @ SRI International, Menlo Park. \r\n\r\nThis work is owned by Vulcan Inc. and is licensed for use under the Creative Commons Attribution-NonCommerical-ShareAlike 3.0 license (http://creativecommons.org/licenses/by-nc-sa/3.0/). Vulcan Inc., June 2013",
+            "documentation": "http://www.ai.sri.com/~halo/public/exported-kb/export-doc.pdf",
             "domain": null,
-            "download_url": "media/ontologies/PATCT/1/patct.owl",
+            "download_url": "media/ontologies/AURA/1/aura.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3300,
+            "home_page": "http://www.ai.sri.com/people/chaudhri",
+            "id": 58,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 5,
+            "max_children": 21058,
+            "max_depth": 23,
             "md5sum": null,
-            "nb_classes": 153,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_classes": 27396,
+            "nb_inconsistent": 1,
+            "nb_individuals": 634,
+            "nb_properties": 358,
             "products": null,
-            "publication": null,
+            "publication": "http://www.ai.sri.com/software/AURA",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "PATEL",
-        "name": "Archana Patel",
+        "acronym": "PDO",
+        "name": "Pathogenic Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2021-03-18T20:20:34Z",
-            "date_released": "2021-03-18T00:00:00Z",
-            "description": "Covid19 Impact on Banking Ontology (Covid19-IBO) provides semantic information about the impact of the Covid-19 on the banking sector of India",
+            "date_created": "2016-03-31T09:16:15Z",
+            "date_released": "2016-03-31T00:00:00Z",
+            "description": "An ontology for describing both human infectious diseases caused by microbes and the diseases that is related to microbial infection. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PATEL/1/patel.owl",
+            "download_url": "media/ontologies/PDO/6/pdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4359,
+            "home_page": "http://mdb.bio.titech.ac.jp/pdo",
+            "id": 592,
             "indexed": true,
-            "max_children": 14,
-            "max_depth": 6,
+            "max_children": 26,
+            "max_depth": 7,
             "md5sum": null,
-            "nb_classes": 123,
+            "nb_classes": 388,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 28,
+            "nb_properties": 13,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 6,
             "taxon": null,
-            "version": "1"
+            "version": "Version 0.7"
         },
         "topics": null
     },
     {
-        "acronym": "PATGV",
-        "name": "Placental Genetic Variance",
+        "acronym": "PGXO",
+        "name": "PGxO",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2020-01-13T20:00:27Z",
-            "date_released": "2020-01-13T00:00:00Z",
-            "description": "Includes variations of DNA sequence, chromosomal structure and copy number, as well as RNA and translational variation. The Genetic Variation ontology expands on work done for Variation Ontology (VariO) and Sequence Types and Features Ontology (SO) which can be found on NCBO. This ontology was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
-            "documentation": null,
+            "date_created": "2017-06-09T21:58:19Z",
+            "date_released": "2017-06-09T00:00:00Z",
+            "description": "PGxO: a lite Pharmacogenomic Ontology.\r\nThis ontology aims at providing a small set of concepts and roles that may type elements of pharmacogenomic\r\n relationships.\r\nIts concepts has been mapped to other existing ontologies. Mappings are not included in this version for lisibility purposes. Mapping are available on demand. ",
+            "documentation": "https://pmonnin.github.io/nettab2017.html",
             "domain": null,
-            "download_url": "media/ontologies/PATGV/1/patgv.owl",
+            "download_url": "media/ontologies/PGXO/2/pgxo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3299,
+            "home_page": "https://members.loria.fr/ACoulet/",
+            "id": 577,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 5,
+            "max_children": 5,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 73,
+            "nb_classes": 11,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 4,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0"
+            "version": "0.2"
         },
         "topics": null
     },
     {
-        "acronym": "PATHLEX",
-        "name": "Anatomic Pathology Lexicon",
+        "acronym": "PGHD-COM",
+        "name": "DATA COMMUNICATION",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 28,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2013-01-22T20:05:19Z",
-            "date_released": "2013-01-22T08:00:00Z",
-            "description": "PathLex, a comprehensive lexicon\u2014a unified language of anatomic pathology terms\u2014for standardized indexing and retrieval of anatomic pathology information resources. ",
+            "date_created": "2022-12-02T16:19:46Z",
+            "date_released": "2022-12-02T00:00:00Z",
+            "description": "Vocabulary describing the method with which a specific metadata instance was sent to CEDAR",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PATHLEX/5/pathlex.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/PGHD-COM/1/pghd-com.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 210,
+            "id": 5530,
             "indexed": true,
-            "max_children": 1522,
-            "max_depth": 4,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 1786,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 2,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 5,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.2"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PATIT",
-        "name": "Placental Investigative Technique",
+        "acronym": "PMA",
+        "name": "Portfolio Management Application",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2084,
             "classifiable": true,
-            "date_created": "2020-01-13T20:09:29Z",
-            "date_released": "2020-01-13T00:00:00Z",
-            "description": "Experimental methods and observation types used in placental research. The PAT Investigative Techniques ontology builds on work done for a handful of existing ontologies, including the Ontology for Biomedical Investigations (OBI) and NCI Thesaurus (NCIT), and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
+            "date_created": "2010-10-19T00:04:33Z",
+            "date_released": "2010-05-11T07:00:00Z",
+            "description": "The Portfolio Management Application (PMA) is a system for tracking grants and producing reports - Users can access grant data through a query interface and a variety of pre-defined forms and reports",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PATIT/1/patit.owl",
+            "download_url": "media/ontologies/PMA/4/pma.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 3304,
+            "id": 511,
             "indexed": true,
-            "max_children": 18,
-            "max_depth": 5,
+            "max_children": 2084,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 139,
+            "nb_classes": 2084,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 9,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 4,
             "taxon": null,
-            "version": "1.0"
+            "version": "0.9.1"
         },
         "topics": null
     },
     {
-        "acronym": "PATMHC",
-        "name": "Placental Maternal Health Conditions",
+        "acronym": "POSITIONALH",
+        "name": "NeuroMorpho.Org bilateral positions",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2020-01-13T19:47:07Z",
-            "date_released": "2020-01-13T00:00:00Z",
-            "description": "Physical and psychological disorders in the mother that occur during pregnancy, childbirth, and the postpartum period. These include both pre-existing and gestational maternal health conditions that may result in adverse pregnancy outcomes. The Maternal Conditions Ontology builds on the work done for a handful of existing ontologies, including the Human Phenotype Ontology (HPO) and NCI Thesaurus (NCIT), and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts.",
+            "date_created": "2016-08-29T19:43:02Z",
+            "date_released": "2016-08-29T00:00:00Z",
+            "description": "A hierarchy of bilateral positions commonly attached to all brain regions in bilateral species.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PATMHC/1/patmhc.owl",
+            "download_url": "media/ontologies/POSITIONALH/2/positionalh.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 3308,
+            "id": 388,
             "indexed": true,
-            "max_children": 11,
-            "max_depth": 6,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 228,
+            "nb_classes": 3,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 88,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "PATO",
-        "name": "Phenotype And Trait Ontology",
+        "acronym": "PP",
+        "name": "Pipeline Patterns Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-05T13:51:39.265623Z",
-            "date_released": "2023-06-05T13:51:39.265610Z",
-            "description": "An ontology of phenotypic qualities (properties, attributes or characteristics)",
+            "date_created": "2014-12-17T14:59:53Z",
+            "date_released": "2014-12-17T00:00:00Z",
+            "description": "To answer the need for a workflow representing a sequence of data transformations, a high-level process ontology was defined, denoted Pipeline Patterns (PP) ontology. \r\n\r\nEvery concept is generically a Data (class from EDAM), which can be further specified as an Entry (P:00002) or an Exit (P:00009). All Entries can be subjected to an Operation (class from EDAM) producing an Intermediate (P:00010). An Intermediate has a TemporalEntity associated to represent its time of creation. Some Entries are specifically of the type Exit, meaning that they will represent concepts that exit the current category and be passed to the next one on the workflow. Additionally, in order to permit the treatment of a set of objects as a single entity, this high-level ontology considers the concept of List (P:00004) and more specifically an Aggregate (P:00005), containing a set of Data or Entry, respectively.\r\nThe concept Intermediate is an Entry, allowing for the successive application of an operation to an Element. This is particularly useful in the case of the application of sequential set of operations.",
             "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/PATO/195/pato.owl",
+            "domain": null,
+            "download_url": "media/ontologies/PP/1/pp.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/pato-ontology/pato/",
-            "id": 5847,
+            "home_page": "https://bitbucket.org/PlantExpAssay/ontology",
+            "id": 667,
             "indexed": true,
-            "max_children": 1003,
-            "max_depth": 33,
-            "md5sum": "bd472f4971492109493d4ad8a779a8dd",
-            "nb_classes": 10589,
+            "max_children": 5,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 15,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 309,
-            "products": [
-                {
-                    "id": "pato.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pato.owl"
-                },
-                {
-                    "id": "pato.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pato.obo"
-                },
-                {
-                    "id": "pato.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pato.json"
-                },
-                {
-                    "description": "Includes axioms linking to other ontologies, but no imports of those ontologies",
-                    "id": "pato/pato-base.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pato/pato-base.owl"
-                }
-            ],
+            "nb_properties": 4,
+            "products": null,
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/28387809",
-                    "title": "The anatomy of phenotype ontologies: principles, properties and applications"
-                },
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/15642100",
-                    "title": "Using ontologies to describe mouse phenotypes"
-                }
-            ],
-            "submission_id": 195,
+            "publications": null,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "PAV",
-        "name": "PAV Provenance, Authoring and Versioning",
+        "acronym": "PDQ",
+        "name": "Physician Data Query",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-01-20T02:12:41Z",
-            "date_released": "2017-01-20T02:12:41Z",
-            "description": "PAV is a lightweight ontology for tracking Provenance, Authoring and Versioning. PAV specializes the W3C provenance ontology PROV-O in order to describe authorship, curation and digital creation of online resources.\r\n\r\nThis ontology describes the defined PAV properties and their usage. Note that PAV does not define any explicit classes or domain/ranges, as every property is meant to be used directly on the described online resource.",
-            "documentation": "http://pav-ontology.github.io/pav/",
+            "date_created": "2017-02-06T21:26:18Z",
+            "date_released": "2017-02-06T21:26:18Z",
+            "description": "Physician Data Query (PDQ) Terminology is part of NCI's comprehensive cancer information database, which contains expert summaries on a wide range of cancer topics, a listing of some 30,000 cancer clinical trials from around the world, a directory of genetics services professionals, the NCI Dictionary of Cancer Terms, and the NCI Drug Dictionary.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PAV/3/pav.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/pav-ontology/pav/",
-            "id": 701,
+            "download_url": "media/ontologies/PDQ/11/pdq.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": null,
+            "id": 717,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 0,
+            "max_children": 89,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 7871,
             "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 40,
+            "nb_individuals": 0,
+            "nb_properties": 17,
             "products": null,
-            "publication": "https://github.com/pav-ontology/pav/wiki/Publications",
+            "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 11,
             "taxon": null,
-            "version": "2.3.1"
+            "version": "2016AB"
         },
         "topics": null
     },
     {
-        "acronym": "ZONMW-GENERIC",
-        "name": "ZonMw Generic Terms",
+        "acronym": "PREGONTO",
+        "name": "Pregnancy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2022-05-25T20:01:09Z",
-            "date_released": "2022-05-25T00:00:00Z",
-            "description": "General-purpose terms for the ZonMw ",
-            "documentation": "https://raw.githubusercontent.com/fair-data-collective/zonmw-generic-vocabulary/",
+            "date_created": "2018-10-31T21:49:36Z",
+            "date_released": "2018-10-31T00:00:00Z",
+            "description": "The pregnancy ontology is a biomedical ontology used for identifying pregnancies in routine health databases. The ontology was primarily developed for conducting surveillance of vaccine uptake in pregnant women in the UK.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ZONMW-GENERIC/21/zonmw-generic.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://raw.githubusercontent.com/fair-data-collective/zonmw-generic-vocabulary/",
-            "id": 5110,
+            "download_url": "media/ontologies/PREGONTO/1/pregonto.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 1983,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 17,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 39,
             "nb_inconsistent": 0,
-            "nb_individuals": 162,
+            "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 21,
+            "submission_id": 1,
             "taxon": null,
-            "version": "1.5.0"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PAYOR",
-        "name": "Payor",
+        "acronym": "PREMEDONTO",
+        "name": "Precision Medicine Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2022-12-07T20:00:29Z",
-            "date_released": "2022-12-07T00:00:00Z",
-            "description": "This class refers to the entity that made the payment on behalf of the interviewee.",
+            "date_created": "2018-11-25T01:43:33Z",
+            "date_released": "2018-11-24T00:00:00Z",
+            "description": "Precision MedicinePreMedOnto is an application ontology built for the precision medicine domain on top of gold standard biomedical ontologies. The ontology learning process involves mining the PubMed repository to extract domain specific texts, abstracts and vocabulary as sources of data. The information gathered is clustered and outlined to determine main modules. It reuses terms and concepts from NCIT, MESH and IOBC to construct the ontology hierarchy.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PAYOR/1/payor.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/PREMEDONTO/1/premedonto.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5542,
+            "id": 2035,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 90,
+            "max_depth": 8,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 543,
             "nb_inconsistent": 0,
-            "nb_individuals": 6,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 10,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "APOLLO_SV",
-        "name": "Apollo Structured Vocabulary",
+        "acronym": "PPO",
+        "name": "Plant Phenology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-05T12:00:15.394476Z",
-            "date_released": "2023-06-05T12:00:15.394460Z",
-            "description": "An OWL2 ontology of phenomena in infectious disease epidemiology and population biology for use in epidemic simulation.",
+            "date_created": "2018-11-05T17:45:33.511392Z",
+            "date_released": "2018-11-05T17:45:33.511365Z",
+            "description": "An ontology for describing the phenology of individual plants and populations of plants, and for integrating plant phenological data across sources and scales.",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/APOLLO_SV/22/apollo_sv.owl",
+            "domain": "plant phenotypes",
+            "download_url": "media/ontologies/PPO/9/ppo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ApolloDev/apollo-sv",
-            "id": 5836,
+            "home_page": "https://github.com/PlantPhenoOntology/PPO",
+            "id": 1980,
             "indexed": true,
-            "max_children": 39,
-            "max_depth": 37,
-            "md5sum": "3c2b6e7d010fffa102974ef3bd4cecee",
-            "nb_classes": 1693,
+            "max_children": 56,
+            "max_depth": 15,
+            "md5sum": "a514f45d53e90667ec62b3753df1beb7",
+            "nb_classes": 443,
             "nb_inconsistent": 0,
-            "nb_individuals": 27,
-            "nb_properties": 220,
+            "nb_individuals": 18,
+            "nb_properties": 33,
             "products": [
                 {
-                    "id": "apollo_sv.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/apollo_sv.owl"
+                    "id": "ppo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ppo.owl"
                 }
             ],
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://doi.org/10.1186/s13326-016-0092-y",
-                    "title": "The Apollo Structured Vocabulary: an OWL2 ontology of phenomena in infectious disease epidemiology and population biology for use in epidemic simulation"
-                }
-            ],
-            "submission_id": 22,
-            "taxon": null,
+            "publications": null,
+            "submission_id": 9,
+            "taxon": {
+                "id": "NCBITaxon:33090",
+                "label": "Viridiplantae"
+            },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "AURA",
-        "name": "KB_Bio_101",
+        "acronym": "KENYAANC",
+        "name": "kenyaancfile",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 15,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2014-01-21T19:51:39Z",
-            "date_released": "2014-01-21T00:00:00Z",
-            "description": "KB_Bio_101 is the ontology of the AURA + Inquire project @ SRI International, Menlo Park. \r\n\r\nThis work is owned by Vulcan Inc. and is licensed for use under the Creative Commons Attribution-NonCommerical-ShareAlike 3.0 license (http://creativecommons.org/licenses/by-nc-sa/3.0/). Vulcan Inc., June 2013",
-            "documentation": "http://www.ai.sri.com/~halo/public/exported-kb/export-doc.pdf",
+            "date_created": "2021-11-13T04:33:31Z",
+            "date_released": "2021-11-12T00:00:00Z",
+            "description": "Ontology on kenyan ANC terms.  ",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/AURA/1/aura.owl",
+            "download_url": "media/ontologies/KENYAANC/1/kenyaanc.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.ai.sri.com/people/chaudhri",
-            "id": 58,
+            "home_page": null,
+            "id": 4698,
             "indexed": true,
-            "max_children": 21058,
-            "max_depth": 23,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 27396,
-            "nb_inconsistent": 1,
-            "nb_individuals": 634,
-            "nb_properties": 358,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 76,
+            "nb_properties": 56,
             "products": null,
-            "publication": "http://www.ai.sri.com/software/AURA",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.0"
+            "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "FLOPO",
-        "name": "Flora Phenotype Ontology",
-        "species": [],
+        "acronym": "KORO",
+        "name": "Knowledge Object Reference Ontology",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 69,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2019-09-09T12:20:11.778262Z",
-            "date_released": "2019-09-09T12:20:11.778238Z",
-            "description": "Traits and phenotypes of flowering plants occurring in digitized Floras",
+            "date_created": "2017-09-17T16:13:07Z",
+            "date_released": "2017-03-09T00:00:00Z",
+            "description": "An ontology to define the parts of a Knowledge Object and their relations built using the Basic Formal Ontology and Information Artifact Ontology",
             "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/FLOPO/10/flopo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/KORO/2/koro.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/flora-phenotype-ontology/flopoontology",
-            "id": 2901,
+            "home_page": null,
+            "id": 91,
             "indexed": true,
-            "max_children": 2289,
-            "max_depth": 11,
-            "md5sum": "b93ed5e841a68be02c4f5ef40d923fd2",
-            "nb_classes": 28971,
-            "nb_inconsistent": 107,
-            "nb_individuals": 0,
-            "nb_properties": 135,
-            "products": [
-                {
-                    "id": "flopo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/flopo.owl"
-                }
-            ],
+            "max_children": 8,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 86,
+            "nb_inconsistent": 0,
+            "nb_individuals": 24,
+            "nb_properties": 54,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 10,
-            "taxon": {
-                "id": "NCBITaxon:33090",
-                "label": "Viridiplantae"
-            },
-            "version": null
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.1"
         },
-        "topics": []
+        "topics": null
     },
     {
-        "acronym": "ZP",
-        "name": "Zebrafish Phenotype Ontology",
+        "acronym": "PCO",
+        "name": "Population and Community Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-12-19T13:40:54.436734Z",
-            "date_released": "2022-12-19T13:40:54.436722Z",
-            "description": "The Zebrafish Phenotype Ontology formally defines all phenotypes of the Zebrafish model organism.",
+            "date_created": "2021-05-03T12:41:20.270330Z",
+            "date_released": "2021-05-03T12:41:20.270317Z",
+            "description": "An ontology about groups of interacting organisms such as populations and communities",
             "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/ZP/8/zp.owl",
+            "domain": "collections of organisms",
+            "download_url": "media/ontologies/PCO/18/pco.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/zebrafish-phenotype-ontology",
-            "id": 5566,
+            "home_page": "https://github.com/PopulationAndCommunityOntology/pco",
+            "id": 4443,
             "indexed": true,
-            "max_children": 3872,
-            "max_depth": 41,
-            "md5sum": "03430b567bf153216c0fa4c3440b3b24",
-            "nb_classes": 63473,
+            "max_children": 16,
+            "max_depth": 31,
+            "md5sum": "8d8ef36ff49c36bd50a9a6b02087e3e2",
+            "nb_classes": 221,
             "nb_inconsistent": 0,
             "nb_individuals": 18,
-            "nb_properties": 299,
+            "nb_properties": 110,
             "products": [
                 {
-                    "id": "zp.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/zp.owl"
-                },
-                {
-                    "id": "zp.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/zp.obo"
+                    "id": "pco.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/pco.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 18,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PGXO",
-        "name": "PGxO",
+        "acronym": "PGHD-COMM",
+        "name": "PGHD COMMUNICATION",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-06-09T21:58:19Z",
-            "date_released": "2017-06-09T00:00:00Z",
-            "description": "PGxO: a lite Pharmacogenomic Ontology.\r\nThis ontology aims at providing a small set of concepts and roles that may type elements of pharmacogenomic\r\n relationships.\r\nIts concepts has been mapped to other existing ontologies. Mappings are not included in this version for lisibility purposes. Mapping are available on demand. ",
-            "documentation": "https://pmonnin.github.io/nettab2017.html",
+            "date_created": "2022-12-12T12:52:29Z",
+            "date_released": "2022-12-07T00:00:00Z",
+            "description": "Ontology containing terms for the collection of Blood Pressure data via Interactive Voice Response in the context of the VODANA project.",
+            "documentation": "https://github.com/abdullahikawu/PGHD/blob/main/vocabularies/",
             "domain": null,
-            "download_url": "media/ontologies/PGXO/2/pgxo.owl",
+            "download_url": "media/ontologies/PGHD-COMM/2/pghd-comm.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/abdullahikawu/PGHD/",
+            "id": 5557,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 8,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PLANTSO",
+        "name": "Plant Stress Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 1,
+            "classifiable": true,
+            "date_created": "2020-11-20T18:40:08Z",
+            "date_released": "2020-05-19T00:00:00Z",
+            "description": "This ontology describes biotic and abiotic stresses that a plant may encounter.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PLANTSO/1/plantso.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://members.loria.fr/ACoulet/",
-            "id": 577,
+            "home_page": "https://github.com/Planteome/plant-stress-ontology",
+            "id": 4078,
             "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
+            "max_children": 105,
+            "max_depth": 36,
             "md5sum": null,
-            "nb_classes": 11,
+            "nb_classes": 3764,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 4,
+            "nb_properties": 115,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.2"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "PHAGE",
         "name": "Phylogenetics Ontology",
         "species": [],
@@ -40896,116 +41385,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "PGHD-COMM",
-        "name": "PGHD COMMUNICATION",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2022-12-12T12:52:29Z",
-            "date_released": "2022-12-07T00:00:00Z",
-            "description": "Ontology containing terms for the collection of Blood Pressure data via Interactive Voice Response in the context of the VODANA project.",
-            "documentation": "https://github.com/abdullahikawu/PGHD/blob/main/vocabularies/",
-            "domain": null,
-            "download_url": "media/ontologies/PGHD-COMM/2/pghd-comm.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/abdullahikawu/PGHD/",
-            "id": 5557,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 8,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "1.1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PLANTSO",
-        "name": "Plant Stress Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2020-11-20T18:40:08Z",
-            "date_released": "2020-05-19T00:00:00Z",
-            "description": "This ontology describes biotic and abiotic stresses that a plant may encounter.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PLANTSO/1/plantso.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/Planteome/plant-stress-ontology",
-            "id": 4078,
-            "indexed": true,
-            "max_children": 105,
-            "max_depth": 36,
-            "md5sum": null,
-            "nb_classes": 3764,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 115,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PMA",
-        "name": "Portfolio Management Application",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2084,
-            "classifiable": true,
-            "date_created": "2010-10-19T00:04:33Z",
-            "date_released": "2010-05-11T07:00:00Z",
-            "description": "The Portfolio Management Application (PMA) is a system for tracking grants and producing reports - Users can access grant data through a query interface and a variety of pre-defined forms and reports",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PMA/4/pma.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 511,
-            "indexed": true,
-            "max_children": 2084,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 2084,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 9,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "0.9.1"
-        },
-        "topics": null
-    },
-    {
         "acronym": "PR",
         "name": "PRotein Ontology (PRO)",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 6,
             "classifiable": true,
@@ -41050,14 +41437,48 @@
             "submission_id": 83,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "PDO_CAS",
+        "name": "Plant Diversity Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2015-04-30T11:28:10Z",
+            "date_released": "2015-04-30T00:00:00Z",
+            "description": "Plant Diversity Ontology (PDO) is the outcome of the construction tasks of domain ontology of science in the project of Scientific & Technological Knowledge Organization Systems (STKOS). The structure of PDO centers on species, including micro level of molecular, cell and tissue, medium level of plant organs, species and population, and macro level of ecosystem. The classes of PDO also capture the habitat, environmental conditions and expand the general characteristics of plants, completing the instances richness for plant species, geographical, classification and morphology. Furthermore, PDO builds the relationships among species, region, morphology, plant organs, developmental stage, revealing the list of species in an area, the morphology and life form of plants, and development process of species. In sum, PDO shows a growth process or development stage of a species and its structure characteristics in particular space and time scale, realizing the description of species diversity, ecosystem diversity and genetic diversity. At present, PDO has been applied to the platform of Go to Selected Open Access Journals (GoOA) developed by National Science Library of Chinese Academy of Sciences, supporting concept retrieval and concept extension for GoOA.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PDO_CAS/1/pdo_cas.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://gooa.las.ac.cn/external/index.jsp",
+            "id": 726,
+            "indexed": true,
+            "max_children": 46,
+            "max_depth": 0,
+            "md5sum": null,
+            "nb_classes": 380,
+            "nb_inconsistent": 0,
+            "nb_individuals": 12168,
+            "nb_properties": 88,
+            "products": null,
+            "publication": "http://124.16.154.130:8080/lis/CN/10.13266/j.issn.0252-3116.2015.03.017",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "PhenomeNETSH",
         "name": "PhenomeNET Ontology - Shrinked version",
         "species": [],
         "status": "Classified",
         "submission": {
             "avg_children": 7,
             "classifiable": true,
@@ -41224,501 +41645,463 @@
                 "label": "Porifera"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "POSITIONALH",
-        "name": "NeuroMorpho.Org bilateral positions",
+        "acronym": "PYRAMIDAL3H",
+        "name": "NeuroMorpho.Org Pyramidal cell dendritic shape ",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 1,
             "classifiable": true,
-            "date_created": "2016-08-29T19:43:02Z",
-            "date_released": "2016-08-29T00:00:00Z",
-            "description": "A hierarchy of bilateral positions commonly attached to all brain regions in bilateral species.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/POSITIONALH/2/positionalh.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 388,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 3,
-            "nb_inconsistent": 0,
-            "nb_individuals": 88,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PP",
-        "name": "Pipeline Patterns Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2014-12-17T14:59:53Z",
-            "date_released": "2014-12-17T00:00:00Z",
-            "description": "To answer the need for a workflow representing a sequence of data transformations, a high-level process ontology was defined, denoted Pipeline Patterns (PP) ontology. \r\n\r\nEvery concept is generically a Data (class from EDAM), which can be further specified as an Entry (P:00002) or an Exit (P:00009). All Entries can be subjected to an Operation (class from EDAM) producing an Intermediate (P:00010). An Intermediate has a TemporalEntity associated to represent its time of creation. Some Entries are specifically of the type Exit, meaning that they will represent concepts that exit the current category and be passed to the next one on the workflow. Additionally, in order to permit the treatment of a set of objects as a single entity, this high-level ontology considers the concept of List (P:00004) and more specifically an Aggregate (P:00005), containing a set of Data or Entry, respectively.\r\nThe concept Intermediate is an Entry, allowing for the successive application of an operation to an Element. This is particularly useful in the case of the application of sequential set of operations.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PP/1/pp.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://bitbucket.org/PlantExpAssay/ontology",
-            "id": 667,
-            "indexed": true,
-            "max_children": 5,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 15,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 4,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PPO",
-        "name": "Plant Phenology Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2018-11-05T17:45:33.511392Z",
-            "date_released": "2018-11-05T17:45:33.511365Z",
-            "description": "An ontology for describing the phenology of individual plants and populations of plants, and for integrating plant phenological data across sources and scales.",
-            "documentation": null,
-            "domain": "plant phenotypes",
-            "download_url": "media/ontologies/PPO/9/ppo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/PlantPhenoOntology/PPO",
-            "id": 1980,
-            "indexed": true,
-            "max_children": 56,
-            "max_depth": 15,
-            "md5sum": "a514f45d53e90667ec62b3753df1beb7",
-            "nb_classes": 443,
-            "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 33,
-            "products": [
-                {
-                    "id": "ppo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ppo.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 9,
-            "taxon": {
-                "id": "NCBITaxon:33090",
-                "label": "Viridiplantae"
-            },
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "KENYAANC",
-        "name": "kenyaancfile",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-11-13T04:33:31Z",
-            "date_released": "2021-11-12T00:00:00Z",
-            "description": "Ontology on kenyan ANC terms.  ",
+            "date_created": "2016-09-01T01:20:52Z",
+            "date_released": "2016-09-01T01:20:52Z",
+            "description": "A hierarchy of Pyramidal cell types based on their dendritic shape.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/KENYAANC/1/kenyaanc.owl",
+            "download_url": "media/ontologies/PYRAMIDAL3H/2/pyramidal3h.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4698,
+            "id": 745,
             "indexed": true,
             "max_children": 2,
             "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 76,
-            "nb_properties": 56,
+            "nb_individuals": 3,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "KORO",
-        "name": "Knowledge Object Reference Ontology",
+        "acronym": "PYRAMIDALH",
+        "name": "NeuroMorpho.Org Pyramidal cell systems connectivity",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-09-17T16:13:07Z",
-            "date_released": "2017-03-09T00:00:00Z",
-            "description": "An ontology to define the parts of a Knowledge Object and their relations built using the Basic Formal Ontology and Information Artifact Ontology",
+            "date_created": "2017-03-31T01:10:06Z",
+            "date_released": "2017-03-31T01:10:06Z",
+            "description": "A hierarchy of Pyramidal cell types projecting to achieve specific systems connectivity such as cortico-thalamic, cortico-spinal, cortico-cortical projections.  ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/KORO/2/koro.owl",
+            "download_url": "media/ontologies/PYRAMIDALH/3/pyramidalh.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 91,
+            "id": 576,
             "indexed": true,
             "max_children": 8,
-            "max_depth": 8,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 86,
+            "nb_classes": 10,
             "nb_inconsistent": 0,
-            "nb_individuals": 24,
-            "nb_properties": 54,
+            "nb_individuals": 15,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.1"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "PCO",
-        "name": "Population and Community Ontology",
+        "acronym": "RBO",
+        "name": "Radiation Biology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2021-05-03T12:41:20.270330Z",
-            "date_released": "2021-05-03T12:41:20.270317Z",
-            "description": "An ontology about groups of interacting organisms such as populations and communities",
+            "date_created": "2023-06-26T13:52:19.740488Z",
+            "date_released": "2023-06-26T13:52:19.740469Z",
+            "description": "RBO is an ontology for the effects of radiation on biota in terrestrial and space environments.",
             "documentation": null,
-            "domain": "collections of organisms",
-            "download_url": "media/ontologies/PCO/18/pco.owl",
+            "domain": "environment",
+            "download_url": "media/ontologies/RBO/23/rbo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/PopulationAndCommunityOntology/pco",
-            "id": 4443,
+            "home_page": "https://github.com/Radiobiology-Informatics-Consortium/RBO",
+            "id": 5893,
             "indexed": true,
-            "max_children": 16,
-            "max_depth": 31,
-            "md5sum": "8d8ef36ff49c36bd50a9a6b02087e3e2",
-            "nb_classes": 221,
+            "max_children": 267,
+            "max_depth": 41,
+            "md5sum": "d8673d25006799a0d5269bc7944ab52c",
+            "nb_classes": 5485,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 110,
+            "nb_individuals": 351,
+            "nb_properties": 275,
             "products": [
                 {
-                    "id": "pco.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/pco.owl"
+                    "id": "rbo.owl",
+                    "name": "Radiation Biology Ontology main release in OWL format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo.owl"
+                },
+                {
+                    "id": "rbo.obo",
+                    "name": "Radiation Biology Ontology additional release in OBO format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo.obo"
+                },
+                {
+                    "id": "rbo.json",
+                    "name": "Radiation Biology Ontology additional release in OBOJSon format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo.json"
+                },
+                {
+                    "id": "rbo/rbo-base.owl",
+                    "name": "Radiation Biology Ontology main release in OWL format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo/rbo-base.owl"
+                },
+                {
+                    "id": "rbo/rbo-base.obo",
+                    "name": "Radiation Biology Ontology additional release in OBO format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo/rbo-base.obo"
+                },
+                {
+                    "id": "rbo/rbo-base.json",
+                    "name": "Radiation Biology Ontology additional release in OBOJSon format",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo/rbo-base.json"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 18,
+            "submission_id": 23,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PDO",
-        "name": "Pathogenic Disease Ontology",
+        "acronym": "REPO",
+        "name": "Reproductive Trait and Phenotype Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2016-03-31T09:16:15Z",
-            "date_released": "2016-03-31T00:00:00Z",
-            "description": "An ontology for describing both human infectious diseases caused by microbes and the diseases that is related to microbial infection. ",
+            "date_created": "2011-10-12T13:06:04Z",
+            "date_released": "2011-10-12T07:00:00Z",
+            "description": "Ontology for livestock reproductive traits and phenotypes",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PDO/6/pdo.owl",
+            "download_url": "media/ontologies/REPO/3/repo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://mdb.bio.titech.ac.jp/pdo",
-            "id": 592,
+            "home_page": null,
+            "id": 531,
             "indexed": true,
-            "max_children": 26,
-            "max_depth": 7,
+            "max_children": 14,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 388,
+            "nb_classes": 96,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 13,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 3,
             "taxon": null,
-            "version": "Version 0.7"
+            "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "PDO_CAS",
-        "name": "Plant Diversity Ontology",
+        "acronym": "PYRAMIDAL1H",
+        "name": "NeuroMorpho.Org Pyramidal cell microconnectivity",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2015-04-30T11:28:10Z",
-            "date_released": "2015-04-30T00:00:00Z",
-            "description": "Plant Diversity Ontology (PDO) is the outcome of the construction tasks of domain ontology of science in the project of Scientific & Technological Knowledge Organization Systems (STKOS). The structure of PDO centers on species, including micro level of molecular, cell and tissue, medium level of plant organs, species and population, and macro level of ecosystem. The classes of PDO also capture the habitat, environmental conditions and expand the general characteristics of plants, completing the instances richness for plant species, geographical, classification and morphology. Furthermore, PDO builds the relationships among species, region, morphology, plant organs, developmental stage, revealing the list of species in an area, the morphology and life form of plants, and development process of species. In sum, PDO shows a growth process or development stage of a species and its structure characteristics in particular space and time scale, realizing the description of species diversity, ecosystem diversity and genetic diversity. At present, PDO has been applied to the platform of Go to Selected Open Access Journals (GoOA) developed by National Science Library of Chinese Academy of Sciences, supporting concept retrieval and concept extension for GoOA.",
+            "date_created": "2016-08-29T19:44:17Z",
+            "date_released": "2016-08-29T00:00:00Z",
+            "description": "A hierarchy of Pyramidal cell microcolumnar connectivity within the cortex.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PDO_CAS/1/pdo_cas.owl",
+            "download_url": "media/ontologies/PYRAMIDAL1H/2/pyramidal1h.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://gooa.las.ac.cn/external/index.jsp",
-            "id": 726,
+            "home_page": null,
+            "id": 40,
             "indexed": true,
-            "max_children": 46,
-            "max_depth": 0,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 380,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 12168,
-            "nb_properties": 88,
+            "nb_individuals": 3,
+            "nb_properties": 0,
             "products": null,
-            "publication": "http://124.16.154.130:8080/lis/CN/10.13266/j.issn.0252-3116.2015.03.017",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": null
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "PDQ",
-        "name": "Physician Data Query",
+        "acronym": "PYRAMIDAL2H",
+        "name": "NeuroMorpho.Org Pyramidal cell gross connectivity",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2017-02-06T21:26:18Z",
-            "date_released": "2017-02-06T21:26:18Z",
-            "description": "Physician Data Query (PDQ) Terminology is part of NCI's comprehensive cancer information database, which contains expert summaries on a wide range of cancer topics, a listing of some 30,000 cancer clinical trials from around the world, a directory of genetics services professionals, the NCI Dictionary of Cancer Terms, and the NCI Drug Dictionary.",
+            "date_created": "2016-08-29T19:50:14Z",
+            "date_released": "2016-08-29T00:00:00Z",
+            "description": "A hierarchy of Pyramidal cell gross connectivity projecting to different cortico-cortical or cortico-subcortical regions.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PDQ/11/pdq.umls",
-            "has_ontology_language": "UMLS",
+            "download_url": "media/ontologies/PYRAMIDAL2H/2/pyramidal2h.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 717,
+            "id": 342,
             "indexed": true,
-            "max_children": 89,
-            "max_depth": 10,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 7871,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 17,
+            "nb_individuals": 3,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 2,
             "taxon": null,
-            "version": "2016AB"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "PGHD-COM",
-        "name": "DATA COMMUNICATION",
-        "species": null,
+        "acronym": "rare_deneme",
+        "name": "Alper K.",
+        "species": [],
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-12-02T16:19:46Z",
-            "date_released": "2022-12-02T00:00:00Z",
-            "description": "Vocabulary describing the method with which a specific metadata instance was sent to CEDAR",
+            "date_created": "2023-01-14T18:49:19.763214Z",
+            "date_released": "2023-01-14T18:49:19.763229Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PGHD-COM/1/pghd-com.skos",
-            "has_ontology_language": "SKOS",
+            "download_url": "media/ontologies/rare_deneme/1/rare_deneme.owl",
+            "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 5530,
+            "id": 5604,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 2495,
+            "max_depth": 34,
             "md5sum": null,
-            "nb_classes": 2,
+            "nb_classes": 17964,
             "nb_inconsistent": 0,
-            "nb_individuals": 2,
-            "nb_properties": 0,
+            "nb_individuals": 0,
+            "nb_properties": 46,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "1.0"
         },
-        "topics": null
+        "topics": [
+            "biomedical"
+        ]
     },
     {
-        "acronym": "PREGONTO",
-        "name": "Pregnancy Ontology",
+        "acronym": "LAND-SURFACE",
+        "name": "Land Surface classifiers",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2018-10-31T21:49:36Z",
-            "date_released": "2018-10-31T00:00:00Z",
-            "description": "The pregnancy ontology is a biomedical ontology used for identifying pregnancies in routine health databases. The ontology was primarily developed for conducting surveillance of vaccine uptake in pregnant women in the UK.",
-            "documentation": null,
+            "date_created": "2021-04-01T23:36:24Z",
+            "date_released": "2021-04-02T00:00:00Z",
+            "description": "Machine-readable representation of the classifiers described in chapter 7 Land Surface, by R.C. McDonald, R.F. Isbell and J.G. Speight, in Australian soil and land survey field handbook (3rd edn).\r\n\r\nLand surface is concerned mainly with surface phenomena affecting land use and soil development that have traditionally been noted at the point of soil observation.",
+            "documentation": "https://www.publish.csiro.au/book/5230",
             "domain": null,
-            "download_url": "media/ontologies/PREGONTO/1/pregonto.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/LAND-SURFACE/1/land-surface.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 1983,
+            "id": 4403,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 3,
+            "max_children": 52,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 39,
+            "nb_classes": 68,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 514,
+            "nb_properties": 58,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
+            "version": "3rd Edition"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "MICRO",
+        "name": "Ontology of Prokaryotic Phenotypic and Metabolic Characters",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2018-06-18T13:34:09.904613Z",
+            "date_released": "2018-06-18T13:34:09.904586Z",
+            "description": "An ontology of prokaryotic phenotypic and metabolic characters",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/MICRO/3/micro.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/carrineblank/MicrO",
+            "id": 1571,
+            "indexed": true,
+            "max_children": 432,
+            "max_depth": 22,
+            "md5sum": "db28fb7fa8eecd4859bb3643b31ecf53",
+            "nb_classes": 17287,
+            "nb_inconsistent": 2,
+            "nb_individuals": 13,
+            "nb_properties": 247,
+            "products": [
+                {
+                    "id": "micro.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/micro.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PREMEDONTO",
-        "name": "Precision Medicine Ontology",
+        "acronym": "RELO",
+        "name": "Relations Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 1,
             "classifiable": true,
-            "date_created": "2018-11-25T01:43:33Z",
-            "date_released": "2018-11-24T00:00:00Z",
-            "description": "Precision MedicinePreMedOnto is an application ontology built for the precision medicine domain on top of gold standard biomedical ontologies. The ontology learning process involves mining the PubMed repository to extract domain specific texts, abstracts and vocabulary as sources of data. The information gathered is clustered and outlined to determine main modules. It reuses terms and concepts from NCIT, MESH and IOBC to construct the ontology hierarchy.",
+            "date_created": "2017-10-03T01:37:59Z",
+            "date_released": "2017-10-03T01:37:59Z",
+            "description": "A collection of relations intended primarily for standardization across ontologies in the OBO Foundry and wider OBO library. It incorporates ROCore upper-level relations such as part of as well as biology-specific relationship types such as develops from.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PREMEDONTO/1/premedonto.owl",
+            "download_url": "media/ontologies/RELO/2/relo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 2035,
+            "home_page": "https://github.com/oborel/obo-relations/",
+            "id": 673,
             "indexed": true,
-            "max_children": 90,
-            "max_depth": 8,
+            "max_children": 5,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 543,
+            "nb_classes": 79,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 10,
+            "nb_individuals": 5,
+            "nb_properties": 528,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "RCTV2",
-        "name": "Read Clinical Terminology Version 2",
+        "acronym": "NMOBR_1",
+        "name": "NeuroMorpho.Org Complete Metadata Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-04-08T09:01:27Z",
-            "date_released": "2016-04-11T00:00:00Z",
-            "description": "NHS UK Read Codes Version 2",
+            "date_created": "2017-11-28T02:12:09Z",
+            "date_released": "2017-11-28T02:12:09Z",
+            "description": "The complete hierarchy of species, brain regions, cell types, and experimental conditions representing the neuronal reconstructions shared on NeuroMorpho.Org.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/RCTV2/1/rctv2.owl",
+            "download_url": "media/ontologies/NMOBR_1/18/nmobr_1.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 24,
+            "id": 864,
             "indexed": true,
-            "max_children": 62,
-            "max_depth": 4,
+            "max_children": 408,
+            "max_depth": 40,
             "md5sum": null,
-            "nb_classes": 88854,
+            "nb_classes": 2538,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 1119,
+            "nb_properties": 2,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 18,
             "taxon": null,
-            "version": "2015"
+            "version": "2.1"
         },
         "topics": null
     },
     {
-        "acronym": "RH-MESH",
-        "name": "Robert Hoehndorf Version of MeSH",
-        "species": null,
+        "acronym": "RBO-LKK",
+        "name": "RBO",
+        "species": [
+            "Multiple"
+        ],
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2014-04-21T23:50:39Z",
-            "date_released": "2014-04-21T23:50:38Z",
-            "description": "Medical Subjects Headings Thesaurus 2014, Modified version.\r\n\r\nThis is an OWL representation of MeSH so that it can be integrated with other ontologies represented in OWL. It contains all terms that are in MeSH and in the MeSH concept tree, including the pharmacological actions (represented as subclass relations).\r\n\r\nConcepts in the MeSH concept tree are merged with MeSH term ids.\r\n\r\n<p><p>\r\n--------------------------\r\n<p>\r\n\r\nNote that, while this ontology is composed of MeSH terms, it does not correspond directly to UMLS MeSH. In particular, a single term in UMLS MeSH often results in multiple classes in this ontology.\r\n\r\n\r\nFor example, in MeSH there is a single term with the label \"Heart\". The internal code (unique identifier) for this term is A07.541. In \r\nthis ontology there are two entirely different terms, both of which have the label \"Heart\". The codes for these terms are A07.541 \r\nand D006321. \r\n",
+            "date_created": "2022-01-20T15:41:48.642884Z",
+            "date_released": "2022-01-20T15:41:48.642905Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/RH-MESH/3/rh-mesh.owl",
+            "download_url": "media/ontologies/RBO-LKK/3/rbo-lkk.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://code.google.com/p/pharmgkb-owl/",
-            "id": 253,
+            "home_page": null,
+            "id": 4840,
             "indexed": true,
-            "max_children": 4006,
-            "max_depth": 12,
+            "max_children": 175,
+            "max_depth": 37,
             "md5sum": null,
-            "nb_classes": 305349,
+            "nb_classes": 4004,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_individuals": 350,
+            "nb_properties": 244,
             "products": null,
-            "publication": "http://leechuck.de/publications.html",
+            "publication": null,
             "publications": null,
             "submission_id": 3,
             "taxon": null,
-            "version": "2014"
+            "version": "0.0.test3"
         },
-        "topics": null
+        "topics": [
+            "radiation",
+            "space to some extent"
+        ]
     },
     {
         "acronym": "RNAO",
         "name": "RNA ontology",
         "species": null,
         "status": "Classified",
         "submission": {
@@ -41815,48 +42198,14 @@
             "submission_id": 2,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "LAND-SURFACE",
-        "name": "Land Surface classifiers",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2021-04-01T23:36:24Z",
-            "date_released": "2021-04-02T00:00:00Z",
-            "description": "Machine-readable representation of the classifiers described in chapter 7 Land Surface, by R.C. McDonald, R.F. Isbell and J.G. Speight, in Australian soil and land survey field handbook (3rd edn).\r\n\r\nLand surface is concerned mainly with surface phenomena affecting land use and soil development that have traditionally been noted at the point of soil observation.",
-            "documentation": "https://www.publish.csiro.au/book/5230",
-            "domain": null,
-            "download_url": "media/ontologies/LAND-SURFACE/1/land-surface.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4403,
-            "indexed": true,
-            "max_children": 52,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 68,
-            "nb_inconsistent": 0,
-            "nb_individuals": 514,
-            "nb_properties": 58,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "3rd Edition"
-        },
-        "topics": null
-    },
-    {
         "acronym": "MIAPA",
         "name": "MIAPA Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -41888,87 +42237,14 @@
             "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "MICRO",
-        "name": "Ontology of Prokaryotic Phenotypic and Metabolic Characters",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2018-06-18T13:34:09.904613Z",
-            "date_released": "2018-06-18T13:34:09.904586Z",
-            "description": "An ontology of prokaryotic phenotypic and metabolic characters",
-            "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/MICRO/3/micro.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/carrineblank/MicrO",
-            "id": 1571,
-            "indexed": true,
-            "max_children": 432,
-            "max_depth": 22,
-            "md5sum": "db28fb7fa8eecd4859bb3643b31ecf53",
-            "nb_classes": 17287,
-            "nb_inconsistent": 2,
-            "nb_individuals": 13,
-            "nb_properties": 247,
-            "products": [
-                {
-                    "id": "micro.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/micro.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "RELO",
-        "name": "Relations Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2017-10-03T01:37:59Z",
-            "date_released": "2017-10-03T01:37:59Z",
-            "description": "A collection of relations intended primarily for standardization across ontologies in the OBO Foundry and wider OBO library. It incorporates ROCore upper-level relations such as part of as well as biology-specific relationship types such as develops from.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/RELO/2/relo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/oborel/obo-relations/",
-            "id": 673,
-            "indexed": true,
-            "max_children": 5,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 79,
-            "nb_inconsistent": 0,
-            "nb_individuals": 5,
-            "nb_properties": 528,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
         "acronym": "PW",
         "name": "Pathway ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -42013,44 +42289,44 @@
             "submission_id": 82,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "NMOBR_1",
-        "name": "NeuroMorpho.Org Complete Metadata Ontology",
+        "acronym": "RAO",
+        "name": "Rheumatoid Arthritis ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2017-11-28T02:12:09Z",
-            "date_released": "2017-11-28T02:12:09Z",
-            "description": "The complete hierarchy of species, brain regions, cell types, and experimental conditions representing the neuronal reconstructions shared on NeuroMorpho.Org.",
+            "date_created": "2015-12-08T14:45:44Z",
+            "date_released": "2015-12-08T00:00:00Z",
+            "description": "A controlled vocabulary of RA-relevant information. Currently, only treatment information is available. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/NMOBR_1/18/nmobr_1.owl",
+            "download_url": "media/ontologies/RAO/1/rao.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 864,
+            "id": 546,
             "indexed": true,
-            "max_children": 408,
-            "max_depth": 40,
+            "max_children": 204,
+            "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 2538,
+            "nb_classes": 280,
             "nb_inconsistent": 0,
-            "nb_individuals": 1119,
-            "nb_properties": 2,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "http://www.sciencedirect.com/science/article/pii/S0933365716300689",
             "publications": null,
-            "submission_id": 18,
+            "submission_id": 1,
             "taxon": null,
-            "version": "2.1"
+            "version": "Version 1.0"
         },
         "topics": null
     },
     {
         "acronym": "RETO",
         "name": "Regulation of Transcription Ontology",
         "species": null,
@@ -42120,87 +42396,80 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "rare_deneme",
-        "name": "Alper K.",
-        "species": [],
+        "acronym": "RH-MESH",
+        "name": "Robert Hoehndorf Version of MeSH",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2023-01-14T18:49:19.763214Z",
-            "date_released": "2023-01-14T18:49:19.763229Z",
-            "description": "",
+            "date_created": "2014-04-21T23:50:39Z",
+            "date_released": "2014-04-21T23:50:38Z",
+            "description": "Medical Subjects Headings Thesaurus 2014, Modified version.\r\n\r\nThis is an OWL representation of MeSH so that it can be integrated with other ontologies represented in OWL. It contains all terms that are in MeSH and in the MeSH concept tree, including the pharmacological actions (represented as subclass relations).\r\n\r\nConcepts in the MeSH concept tree are merged with MeSH term ids.\r\n\r\n<p><p>\r\n--------------------------\r\n<p>\r\n\r\nNote that, while this ontology is composed of MeSH terms, it does not correspond directly to UMLS MeSH. In particular, a single term in UMLS MeSH often results in multiple classes in this ontology.\r\n\r\n\r\nFor example, in MeSH there is a single term with the label \"Heart\". The internal code (unique identifier) for this term is A07.541. In \r\nthis ontology there are two entirely different terms, both of which have the label \"Heart\". The codes for these terms are A07.541 \r\nand D006321. \r\n",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/rare_deneme/1/rare_deneme.owl",
+            "download_url": "media/ontologies/RH-MESH/3/rh-mesh.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5604,
+            "home_page": "https://code.google.com/p/pharmgkb-owl/",
+            "id": 253,
             "indexed": true,
-            "max_children": 2495,
-            "max_depth": 34,
+            "max_children": 4006,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 17964,
+            "nb_classes": 305349,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 46,
+            "nb_properties": 1,
             "products": null,
-            "publication": null,
+            "publication": "http://leechuck.de/publications.html",
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 3,
             "taxon": null,
-            "version": "1.0"
+            "version": "2014"
         },
-        "topics": [
-            "biomedical"
-        ]
+        "topics": null
     },
     {
-        "acronym": "RBO-LKK",
-        "name": "RBO",
-        "species": [
-            "Multiple"
-        ],
+        "acronym": "RCTV2",
+        "name": "Read Clinical Terminology Version 2",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2022-01-20T15:41:48.642884Z",
-            "date_released": "2022-01-20T15:41:48.642905Z",
-            "description": "",
+            "date_created": "2016-04-08T09:01:27Z",
+            "date_released": "2016-04-11T00:00:00Z",
+            "description": "NHS UK Read Codes Version 2",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/RBO-LKK/3/rbo-lkk.owl",
+            "download_url": "media/ontologies/RCTV2/1/rctv2.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 4840,
+            "id": 24,
             "indexed": true,
-            "max_children": 175,
-            "max_depth": 37,
+            "max_children": 62,
+            "max_depth": 4,
             "md5sum": null,
-            "nb_classes": 4004,
+            "nb_classes": 88854,
             "nb_inconsistent": 0,
-            "nb_individuals": 350,
-            "nb_properties": 244,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 1,
             "taxon": null,
-            "version": "0.0.test3"
+            "version": "2015"
         },
-        "topics": [
-            "radiation",
-            "space to some extent"
-        ]
+        "topics": null
     },
     {
         "acronym": "REXO",
         "name": "Regulation of Gene Expression Ontolology",
         "species": null,
         "status": "Classified",
         "submission": {
@@ -42331,347 +42600,155 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "RAO",
-        "name": "Rheumatoid Arthritis ontology",
+        "acronym": "REPRODUCE-ME",
+        "name": "REPRODUCE-ME",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2015-12-08T14:45:44Z",
-            "date_released": "2015-12-08T00:00:00Z",
-            "description": "A controlled vocabulary of RA-relevant information. Currently, only treatment information is available. ",
-            "documentation": null,
+            "date_created": "2019-07-08T14:20:25Z",
+            "date_released": "2019-01-14T00:00:00Z",
+            "description": "The REPRODUCE-ME ontology is an extension of the PROV-O and the P-Plan ontology to describe a complete path of a scientific experiment. It expresses the REPRODUCE-ME Data Model using the OWL2 Web Ontology Language (OWL2). It provides a set of classes and properties to represent a scientific experiment including its computational and non-computational steps to track the provenance of results. It describes an experiment and its data, agents, activities, plans, steps, variables, instruments, materials and settings required for its reproducibility. ",
+            "documentation": "https://w3id.org/reproduceme",
             "domain": null,
-            "download_url": "media/ontologies/RAO/1/rao.owl",
+            "download_url": "media/ontologies/REPRODUCE-ME/7/reproduce-me.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 546,
+            "home_page": "https://w3id.org/reproduceme/research",
+            "id": 2725,
             "indexed": true,
-            "max_children": 204,
+            "max_children": 34,
             "max_depth": 6,
             "md5sum": null,
-            "nb_classes": 280,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
-            "publication": "http://www.sciencedirect.com/science/article/pii/S0933365716300689",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "Version 1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "RBO",
-        "name": "Radiation Biology Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2023-06-26T13:52:19.740488Z",
-            "date_released": "2023-06-26T13:52:19.740469Z",
-            "description": "RBO is an ontology for the effects of radiation on biota in terrestrial and space environments.",
-            "documentation": null,
-            "domain": "environment",
-            "download_url": "media/ontologies/RBO/23/rbo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/Radiobiology-Informatics-Consortium/RBO",
-            "id": 5893,
-            "indexed": true,
-            "max_children": 267,
-            "max_depth": 41,
-            "md5sum": "d8673d25006799a0d5269bc7944ab52c",
-            "nb_classes": 5485,
-            "nb_inconsistent": 0,
-            "nb_individuals": 351,
-            "nb_properties": 275,
-            "products": [
-                {
-                    "id": "rbo.owl",
-                    "name": "Radiation Biology Ontology main release in OWL format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo.owl"
-                },
-                {
-                    "id": "rbo.obo",
-                    "name": "Radiation Biology Ontology additional release in OBO format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo.obo"
-                },
-                {
-                    "id": "rbo.json",
-                    "name": "Radiation Biology Ontology additional release in OBOJSon format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo.json"
-                },
-                {
-                    "id": "rbo/rbo-base.owl",
-                    "name": "Radiation Biology Ontology main release in OWL format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo/rbo-base.owl"
-                },
-                {
-                    "id": "rbo/rbo-base.obo",
-                    "name": "Radiation Biology Ontology additional release in OBO format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo/rbo-base.obo"
-                },
-                {
-                    "id": "rbo/rbo-base.json",
-                    "name": "Radiation Biology Ontology additional release in OBOJSon format",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/rbo/rbo-base.json"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 23,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "REPO",
-        "name": "Reproductive Trait and Phenotype Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2011-10-12T13:06:04Z",
-            "date_released": "2011-10-12T07:00:00Z",
-            "description": "Ontology for livestock reproductive traits and phenotypes",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/REPO/3/repo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 531,
-            "indexed": true,
-            "max_children": 14,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 96,
+            "nb_classes": 347,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_individuals": 255,
+            "nb_properties": 71,
             "products": null,
-            "publication": null,
+            "publication": "https://w3id.org/reproduceme/research",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 7,
             "taxon": null,
             "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "PYRAMIDAL1H",
-        "name": "NeuroMorpho.Org Pyramidal cell microconnectivity",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 1,
-            "classifiable": true,
-            "date_created": "2016-08-29T19:44:17Z",
-            "date_released": "2016-08-29T00:00:00Z",
-            "description": "A hierarchy of Pyramidal cell microcolumnar connectivity within the cortex.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PYRAMIDAL1H/2/pyramidal1h.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 40,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 4,
-            "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "PYRAMIDAL2H",
-        "name": "NeuroMorpho.Org Pyramidal cell gross connectivity",
+        "acronym": "LANDFORM",
+        "name": "Landform classifiers",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2016-08-29T19:50:14Z",
-            "date_released": "2016-08-29T00:00:00Z",
-            "description": "A hierarchy of Pyramidal cell gross connectivity projecting to different cortico-cortical or cortico-subcortical regions.",
-            "documentation": null,
+            "date_created": "2021-04-07T00:40:27Z",
+            "date_released": "2021-04-02T00:00:00Z",
+            "description": "Machine-readable representation of the classifiers described in chapter 5 Landform, by J.G. Speight, in Australian soil and land survey field handbook (3rd edn).\r\n\r\nIn this technique for describing landforms, the whole land surface is viewed as a mosaic of tiles of odd shapes and sizes. The scheme is intended to produce a record of observations rather than inferences.\r\n\r\nThe data was converted from the print representation to this linked-data form by Linda Gregory assisted by Simon J D Cox.",
+            "documentation": "https://www.publish.csiro.au/book/5230",
             "domain": null,
-            "download_url": "media/ontologies/PYRAMIDAL2H/2/pyramidal2h.owl",
-            "has_ontology_language": "OWL",
+            "download_url": "media/ontologies/LANDFORM/2/landform.skos",
+            "has_ontology_language": "SKOS",
             "home_page": null,
-            "id": 342,
+            "id": 4397,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
+            "max_children": 52,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 4,
+            "nb_classes": 68,
             "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 0,
+            "nb_individuals": 406,
+            "nb_properties": 64,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 2,
             "taxon": null,
-            "version": "2.0"
+            "version": "3rd edition"
         },
         "topics": null
     },
     {
-        "acronym": "PYRAMIDAL3H",
-        "name": "NeuroMorpho.Org Pyramidal cell dendritic shape ",
+        "acronym": "SCDO",
+        "name": "Sickle Cell Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2016-09-01T01:20:52Z",
-            "date_released": "2016-09-01T01:20:52Z",
-            "description": "A hierarchy of Pyramidal cell types based on their dendritic shape.",
+            "date_created": "2021-05-10T13:17:51.080405Z",
+            "date_released": "2021-05-10T13:17:51.080389Z",
+            "description": "An ontology for the standardization of terminology and integration of knowledge about Sickle Cell Disease.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PYRAMIDAL3H/2/pyramidal3h.owl",
+            "domain": "disease",
+            "download_url": "media/ontologies/SCDO/18/scdo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 745,
+            "home_page": "https://scdontology.h3abionet.org/",
+            "id": 4468,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 4,
+            "max_children": 91,
+            "max_depth": 11,
+            "md5sum": "90809637cc07c0f7937375fd79812904",
+            "nb_classes": 2072,
             "nb_inconsistent": 0,
-            "nb_individuals": 3,
-            "nb_properties": 0,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 61,
+            "products": [
+                {
+                    "id": "scdo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/scdo.owl"
+                },
+                {
+                    "id": "scdo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/scdo.obo"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "REPRODUCE-ME",
-        "name": "REPRODUCE-ME",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2019-07-08T14:20:25Z",
-            "date_released": "2019-01-14T00:00:00Z",
-            "description": "The REPRODUCE-ME ontology is an extension of the PROV-O and the P-Plan ontology to describe a complete path of a scientific experiment. It expresses the REPRODUCE-ME Data Model using the OWL2 Web Ontology Language (OWL2). It provides a set of classes and properties to represent a scientific experiment including its computational and non-computational steps to track the provenance of results. It describes an experiment and its data, agents, activities, plans, steps, variables, instruments, materials and settings required for its reproducibility. ",
-            "documentation": "https://w3id.org/reproduceme",
-            "domain": null,
-            "download_url": "media/ontologies/REPRODUCE-ME/7/reproduce-me.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://w3id.org/reproduceme/research",
-            "id": 2725,
-            "indexed": true,
-            "max_children": 34,
-            "max_depth": 6,
-            "md5sum": null,
-            "nb_classes": 347,
-            "nb_inconsistent": 0,
-            "nb_individuals": 255,
-            "nb_properties": 71,
-            "products": null,
-            "publication": "https://w3id.org/reproduceme/research",
-            "publications": null,
-            "submission_id": 7,
+            "submission_id": 18,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PYRAMIDALH",
-        "name": "NeuroMorpho.Org Pyramidal cell systems connectivity",
+        "acronym": "SK",
+        "name": "Digital Forensic",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2017-03-31T01:10:06Z",
-            "date_released": "2017-03-31T01:10:06Z",
-            "description": "A hierarchy of Pyramidal cell types projecting to achieve specific systems connectivity such as cortico-thalamic, cortico-spinal, cortico-cortical projections.  ",
+            "date_created": "2019-11-28T15:12:45Z",
+            "date_released": "2019-11-28T00:00:00Z",
+            "description": "An abstraction based approach.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PYRAMIDALH/3/pyramidalh.owl",
+            "download_url": "media/ontologies/SK/1/sk.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 576,
+            "id": 3142,
             "indexed": true,
             "max_children": 8,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 10,
-            "nb_inconsistent": 0,
-            "nb_individuals": 15,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "LANDFORM",
-        "name": "Landform classifiers",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2021-04-07T00:40:27Z",
-            "date_released": "2021-04-02T00:00:00Z",
-            "description": "Machine-readable representation of the classifiers described in chapter 5 Landform, by J.G. Speight, in Australian soil and land survey field handbook (3rd edn).\r\n\r\nIn this technique for describing landforms, the whole land surface is viewed as a mosaic of tiles of odd shapes and sizes. The scheme is intended to produce a record of observations rather than inferences.\r\n\r\nThe data was converted from the print representation to this linked-data form by Linda Gregory assisted by Simon J D Cox.",
-            "documentation": "https://www.publish.csiro.au/book/5230",
-            "domain": null,
-            "download_url": "media/ontologies/LANDFORM/2/landform.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4397,
-            "indexed": true,
-            "max_children": 52,
-            "max_depth": 2,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 68,
+            "nb_classes": 38,
             "nb_inconsistent": 0,
-            "nb_individuals": 406,
-            "nb_properties": 64,
+            "nb_individuals": 4,
+            "nb_properties": 4,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "3rd edition"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "RO",
         "name": "Relation Ontology",
         "species": null,
@@ -43075,14 +43152,48 @@
             "submission_id": 11,
             "taxon": null,
             "version": "2023-04-28"
         },
         "topics": null
     },
     {
+        "acronym": "MO",
+        "name": "Microarray and Gene Expression Data Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2009-05-21T17:24:23Z",
+            "date_released": "2015-08-19T00:00:00Z",
+            "description": "Concepts, definitions, terms, and resources for standardized description of a microarray experiment in support of MAGE v.1.",
+            "documentation": "http://mged.sourceforge.net/software/downloads.php",
+            "domain": null,
+            "download_url": "media/ontologies/MO/1/mo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://mged.sourceforge.net/ontologies/MGEDontology.php",
+            "id": 291,
+            "indexed": true,
+            "max_children": 31,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 236,
+            "nb_inconsistent": 0,
+            "nb_individuals": 698,
+            "nb_properties": 127,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1.3.1.1"
+        },
+        "topics": null
+    },
+    {
         "acronym": "OF",
         "name": "OntoFood",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -43109,44 +43220,112 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "MO",
-        "name": "Microarray and Gene Expression Data Ontology",
+        "acronym": "RNRMU",
+        "name": "RNRMU",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2009-05-21T17:24:23Z",
-            "date_released": "2015-08-19T00:00:00Z",
-            "description": "Concepts, definitions, terms, and resources for standardized description of a microarray experiment in support of MAGE v.1.",
-            "documentation": "http://mged.sourceforge.net/software/downloads.php",
+            "date_created": "2014-07-04T07:19:21Z",
+            "date_released": "2014-09-30T00:00:00Z",
+            "description": "The terms of RNRMU",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/MO/1/mo.owl",
+            "download_url": "media/ontologies/RNRMU/1/rnrmu.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://mged.sourceforge.net/ontologies/MGEDontology.php",
-            "id": 291,
+            "home_page": null,
+            "id": 205,
             "indexed": true,
-            "max_children": 31,
-            "max_depth": 8,
+            "max_children": 67,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 236,
+            "nb_classes": 2310,
             "nb_inconsistent": 0,
-            "nb_individuals": 698,
-            "nb_properties": 127,
+            "nb_individuals": 0,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.3.1.1"
+            "version": "0.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "SARSMUTONTO",
+        "name": "Ontology for SARS-CoV-2  lineages and mutations",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2023-02-02T02:22:39Z",
+            "date_released": "2023-02-02T02:22:36Z",
+            "description": "The SARSMutOnto ontology provides a list of all SARS-CoV-2 Pango lineages while maintaining their hierarchy (lineage/sublineage) describing in detail all lineage mutations",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/SARSMUTONTO/7/sarsmutonto.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/jbakkas/SARSMutOnto",
+            "id": 5646,
+            "indexed": true,
+            "max_children": 456,
+            "max_depth": 18,
+            "md5sum": null,
+            "nb_classes": 2388,
+            "nb_inconsistent": 0,
+            "nb_individuals": 3100,
+            "nb_properties": 2,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 7,
+            "taxon": null,
+            "version": "1.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "SASAP",
+        "name": "The State of Alaska's Salmon and People Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2022-08-06T02:30:37Z",
+            "date_released": "2022-07-22T00:00:00Z",
+            "description": "An ontology which represents information about the State of Alaska's Salmon and People (SASAP) project, as well as knowledge about salmon, features of their habitats, salmon stakeholders, and related entities.",
+            "documentation": "https://github.com/DataONEorg/sem-prov-ontologies",
+            "domain": null,
+            "download_url": "media/ontologies/SASAP/2/sasap.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/DataONEorg/sem-prov-ontologies",
+            "id": 5273,
+            "indexed": true,
+            "max_children": 20,
+            "max_depth": 3,
+            "md5sum": null,
+            "nb_classes": 30,
+            "nb_inconsistent": 0,
+            "nb_individuals": 221,
+            "nb_properties": 12,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "0.1.1"
         },
         "topics": null
     },
     {
         "acronym": "SATO",
         "name": "SATO (IDEAS expAnded wiTh BCIO): workflow for designers of patient-centered mobile health behaviour change intervention applications",
         "species": null,
@@ -43211,14 +43390,53 @@
             "submission_id": 7,
             "taxon": null,
             "version": "Version 0.5 alpha"
         },
         "topics": null
     },
     {
+        "acronym": "SBO",
+        "name": "Systems Biology Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2023-06-05T16:17:43.842448Z",
+            "date_released": "2023-06-05T16:17:43.842437Z",
+            "description": "Terms commonly used in Systems Biology, and in particular in computational modeling.",
+            "documentation": null,
+            "domain": "chemistry and biochemistry",
+            "download_url": "media/ontologies/SBO/816/sbo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://www.ebi.ac.uk/sbo/",
+            "id": 5854,
+            "indexed": true,
+            "max_children": 32,
+            "max_depth": 10,
+            "md5sum": "af8224034238aba1be4ee361ff9ddefa",
+            "nb_classes": 695,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
+            "products": [
+                {
+                    "id": "sbo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/sbo.owl"
+                }
+            ],
+            "publication": null,
+            "publications": null,
+            "submission_id": 816,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "SALMON",
         "name": "Salmon Ontology",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -43245,256 +43463,310 @@
             "submission_id": 2,
             "taxon": null,
             "version": "0.3.1"
         },
         "topics": null
     },
     {
-        "acronym": "RNRMU",
-        "name": "RNRMU",
+        "acronym": "SBOL",
+        "name": "Synthetic Biology Open Language Visual Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2014-07-04T07:19:21Z",
-            "date_released": "2014-09-30T00:00:00Z",
-            "description": "The terms of RNRMU",
+            "date_created": "2013-07-18T19:00:30Z",
+            "date_released": "2013-04-26T07:00:00Z",
+            "description": "Synthetic Biology Open Language Visual (SBOLv) is an ontology to represent standardized graphical notation for synthetic biology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/RNRMU/1/rnrmu.owl",
+            "download_url": "media/ontologies/SBOL/2/sbol.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "www.sbolstandard.org/visual",
+            "id": 514,
+            "indexed": true,
+            "max_children": 18,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 41,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 4,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "unknown"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "PANET",
+        "name": "PaN Experimental technique",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2021-06-19T01:14:32Z",
+            "date_released": "2021-06-19T01:14:30Z",
+            "description": "The PaNET ontology provides a taxonomy and thesaurus of photon and neutron (PaN) experimental techniques, based mainly on accelerator-based light sources and neutron facilities. The primary use of the ontology is to enhance the FAIRness of PaN data catalogues services. The ontology defines specific techniques in terms of more general technique classes and provides synonyms and references.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/PANET/24/panet.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 205,
+            "id": 4629,
             "indexed": true,
-            "max_children": 67,
-            "max_depth": 3,
+            "max_children": 41,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 2310,
+            "nb_classes": 379,
+            "nb_inconsistent": 0,
+            "nb_individuals": 8,
+            "nb_properties": 1,
+            "products": null,
+            "publication": "https://doi.org/10.5281/zenodo.4806026",
+            "publications": null,
+            "submission_id": 24,
+            "taxon": null,
+            "version": "1.0.0"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "SNMD_BDY",
+        "name": "Snomed, Body Structure",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2012-05-04T21:02:51Z",
+            "date_released": "2012-04-23T07:00:00Z",
+            "description": "The Body Structure subtree in SNOMED-CT. It has been created in order to built a proper ontology for our project, ONCO-i2b2, through the NCBO Ontology Tools. ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/SNMD_BDY/2/snmd_bdy.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 710,
+            "indexed": true,
+            "max_children": 93,
+            "max_depth": 30,
+            "md5sum": null,
+            "nb_classes": 20227,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.1"
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "SARSMUTONTO",
-        "name": "Ontology for SARS-CoV-2  lineages and mutations",
+        "acronym": "SMASHPHYSICAL",
+        "name": "SMASH ontology, Physical Activity Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-02-02T02:22:39Z",
-            "date_released": "2023-02-02T02:22:36Z",
-            "description": "The SARSMutOnto ontology provides a list of all SARS-CoV-2 Pango lineages while maintaining their hierarchy (lineage/sublineage) describing in detail all lineage mutations",
+            "date_created": "2015-06-03T01:14:49Z",
+            "date_released": "2015-06-03T01:14:49Z",
+            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.\r\n\r\nPhysical activities for users in SMASH ontology.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SARSMUTONTO/7/sarsmutonto.owl",
+            "download_url": "media/ontologies/SMASHPHYSICAL/4/smashphysical.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/jbakkas/SARSMutOnto",
-            "id": 5646,
+            "home_page": null,
+            "id": 125,
             "indexed": true,
-            "max_children": 456,
-            "max_depth": 18,
+            "max_children": 8,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 2388,
+            "nb_classes": 74,
             "nb_inconsistent": 0,
-            "nb_individuals": 3100,
-            "nb_properties": 2,
+            "nb_individuals": 29,
+            "nb_properties": 51,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 4,
             "taxon": null,
-            "version": "1.1"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SASAP",
-        "name": "The State of Alaska's Salmon and People Ontology",
+        "acronym": "ORTH",
+        "name": "Orthology Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2022-08-06T02:30:37Z",
-            "date_released": "2022-07-22T00:00:00Z",
-            "description": "An ontology which represents information about the State of Alaska's Salmon and People (SASAP) project, as well as knowledge about salmon, features of their habitats, salmon stakeholders, and related entities.",
-            "documentation": "https://github.com/DataONEorg/sem-prov-ontologies",
+            "date_created": "2016-04-04T01:10:14Z",
+            "date_released": "2016-04-04T01:10:13Z",
+            "description": "Orthology Ontology",
+            "documentation": "https://github.com/qfo/OrthologyOntology",
             "domain": null,
-            "download_url": "media/ontologies/SASAP/2/sasap.owl",
+            "download_url": "media/ontologies/ORTH/4/orth.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/DataONEorg/sem-prov-ontologies",
-            "id": 5273,
+            "home_page": "https://github.com/qfo/OrthologyOntology",
+            "id": 608,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 3,
+            "max_children": 118,
+            "max_depth": 37,
             "md5sum": null,
-            "nb_classes": 30,
+            "nb_classes": 4715,
             "nb_inconsistent": 0,
-            "nb_individuals": 221,
-            "nb_properties": 12,
+            "nb_individuals": 610,
+            "nb_properties": 845,
             "products": null,
-            "publication": null,
+            "publication": "http://www.jbiomedsem.com/content/7/1/34",
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 4,
             "taxon": null,
-            "version": "0.1.1"
+            "version": "2015-11-27"
         },
         "topics": null
     },
     {
-        "acronym": "SBO",
-        "name": "Systems Biology Ontology",
+        "acronym": "SMASHSOCIAL",
+        "name": "SMASH ontology, Social Activity Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-06-05T16:17:43.842448Z",
-            "date_released": "2023-06-05T16:17:43.842437Z",
-            "description": "Terms commonly used in Systems Biology, and in particular in computational modeling.",
+            "date_created": "2015-06-03T01:14:50Z",
+            "date_released": "2015-06-03T01:14:50Z",
+            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.\r\n\r\nSocial activities for users in the SMASH ontology.",
             "documentation": null,
-            "domain": "chemistry and biochemistry",
-            "download_url": "media/ontologies/SBO/816/sbo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/SMASHSOCIAL/4/smashsocial.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://www.ebi.ac.uk/sbo/",
-            "id": 5854,
+            "home_page": null,
+            "id": 444,
             "indexed": true,
-            "max_children": 32,
-            "max_depth": 10,
-            "md5sum": "af8224034238aba1be4ee361ff9ddefa",
-            "nb_classes": 695,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
-            "products": [
-                {
-                    "id": "sbo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/sbo.owl"
-                }
-            ],
+            "max_children": 10,
+            "max_depth": 7,
+            "md5sum": null,
+            "nb_classes": 108,
+            "nb_inconsistent": 2,
+            "nb_individuals": 29,
+            "nb_properties": 63,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 816,
+            "submission_id": 4,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SBOL",
-        "name": "Synthetic Biology Open Language Visual Ontology",
+        "acronym": "SMD_NPLSM",
+        "name": "Neoplasm_hamartoma",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 6,
             "classifiable": true,
-            "date_created": "2013-07-18T19:00:30Z",
-            "date_released": "2013-04-26T07:00:00Z",
-            "description": "Synthetic Biology Open Language Visual (SBOLv) is an ontology to represent standardized graphical notation for synthetic biology.",
+            "date_created": "2011-11-07T15:45:01Z",
+            "date_released": "2011-11-07T08:00:00Z",
+            "description": "subset of snomed-ct regarding neoplasm and hamartoma",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SBOL/2/sbol.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "www.sbolstandard.org/visual",
-            "id": 514,
+            "download_url": "media/ontologies/SMD_NPLSM/1/smd_nplsm.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 392,
             "indexed": true,
-            "max_children": 18,
-            "max_depth": 2,
+            "max_children": 397,
+            "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 41,
+            "nb_classes": 1595,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 4,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
             "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "SCDO",
-        "name": "Sickle Cell Disease Ontology",
+        "acronym": "SMD_NPLSM_BRST",
+        "name": "Neoplasm_breast_cancer",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2021-05-10T13:17:51.080405Z",
-            "date_released": "2021-05-10T13:17:51.080389Z",
-            "description": "An ontology for the standardization of terminology and integration of knowledge about Sickle Cell Disease.",
+            "date_created": "2011-11-16T15:29:28Z",
+            "date_released": "2011-11-16T08:00:00Z",
+            "description": "This a subset of Neoplasm_hamartoma view, based on Snomed clinical practice's used terms (Fondazione S.Maugeri anatomic pathology dep.). This subset is not to be meant exhaustive, it has been created in order to built a proper ontology for our project, ONCO-i2b2, through the NCBO Ontology Tools. ",
             "documentation": null,
-            "domain": "disease",
-            "download_url": "media/ontologies/SCDO/18/scdo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/SMD_NPLSM_BRST/1/smd_nplsm_brst.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://scdontology.h3abionet.org/",
-            "id": 4468,
+            "home_page": null,
+            "id": 339,
             "indexed": true,
-            "max_children": 91,
-            "max_depth": 11,
-            "md5sum": "90809637cc07c0f7937375fd79812904",
-            "nb_classes": 2072,
+            "max_children": 126,
+            "max_depth": 8,
+            "md5sum": null,
+            "nb_classes": 980,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 61,
-            "products": [
-                {
-                    "id": "scdo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/scdo.owl"
-                },
-                {
-                    "id": "scdo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/scdo.obo"
-                }
-            ],
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 18,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "SK",
-        "name": "Digital Forensic",
+        "acronym": "SMO",
+        "name": "Syrian Movies Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2019-11-28T15:12:45Z",
-            "date_released": "2019-11-28T00:00:00Z",
-            "description": "An abstraction based approach.",
+            "date_created": "2022-04-13T19:08:50Z",
+            "date_released": "2022-04-13T00:00:00Z",
+            "description": "Syrian Movies Ontology, For Education.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SK/1/sk.owl",
+            "download_url": "media/ontologies/SMO/1/smo.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 3142,
+            "id": 5042,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 5,
+            "max_children": 3,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 38,
+            "nb_classes": 6,
             "nb_inconsistent": 0,
-            "nb_individuals": 4,
-            "nb_properties": 4,
+            "nb_individuals": 65,
+            "nb_properties": 10,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
@@ -43665,48 +43937,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ORTH",
-        "name": "Orthology Ontology",
-        "species": null,
-        "status": "Unloadable",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2016-04-04T01:10:14Z",
-            "date_released": "2016-04-04T01:10:13Z",
-            "description": "Orthology Ontology",
-            "documentation": "https://github.com/qfo/OrthologyOntology",
-            "domain": null,
-            "download_url": "media/ontologies/ORTH/4/orth.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/qfo/OrthologyOntology",
-            "id": 608,
-            "indexed": true,
-            "max_children": 118,
-            "max_depth": 37,
-            "md5sum": null,
-            "nb_classes": 4715,
-            "nb_inconsistent": 0,
-            "nb_individuals": 610,
-            "nb_properties": 845,
-            "products": null,
-            "publication": "http://www.jbiomedsem.com/content/7/1/34",
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": "2015-11-27"
-        },
-        "topics": null
-    },
-    {
         "acronym": "OPDRONT",
         "name": "OPD Register Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
@@ -43767,316 +44005,231 @@
             "submission_id": 1,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "PANET",
-        "name": "PaN Experimental technique",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2021-06-19T01:14:32Z",
-            "date_released": "2021-06-19T01:14:30Z",
-            "description": "The PaNET ontology provides a taxonomy and thesaurus of photon and neutron (PaN) experimental techniques, based mainly on accelerator-based light sources and neutron facilities. The primary use of the ontology is to enhance the FAIRness of PaN data catalogues services. The ontology defines specific techniques in terms of more general technique classes and provides synonyms and references.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/PANET/24/panet.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4629,
-            "indexed": true,
-            "max_children": 41,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 379,
-            "nb_inconsistent": 0,
-            "nb_individuals": 8,
-            "nb_properties": 1,
-            "products": null,
-            "publication": "https://doi.org/10.5281/zenodo.4806026",
-            "publications": null,
-            "submission_id": 24,
-            "taxon": null,
-            "version": "1.0.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SMASHPHYSICAL",
-        "name": "SMASH ontology, Physical Activity Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2015-06-03T01:14:49Z",
-            "date_released": "2015-06-03T01:14:49Z",
-            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.\r\n\r\nPhysical activities for users in SMASH ontology.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/SMASHPHYSICAL/4/smashphysical.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 125,
-            "indexed": true,
-            "max_children": 8,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 74,
-            "nb_inconsistent": 0,
-            "nb_individuals": 29,
-            "nb_properties": 51,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SMASHSOCIAL",
-        "name": "SMASH ontology, Social Activity Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2015-06-03T01:14:50Z",
-            "date_released": "2015-06-03T01:14:50Z",
-            "description": "The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts.\r\n\r\nSocial activities for users in the SMASH ontology.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/SMASHSOCIAL/4/smashsocial.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 444,
-            "indexed": true,
-            "max_children": 10,
-            "max_depth": 7,
-            "md5sum": null,
-            "nb_classes": 108,
-            "nb_inconsistent": 2,
-            "nb_individuals": 29,
-            "nb_properties": 63,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 4,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SMD_NPLSM",
-        "name": "Neoplasm_hamartoma",
+        "acronym": "SNMD_BC",
+        "name": "Snomed neoplasm, breast cancer",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2011-11-07T15:45:01Z",
-            "date_released": "2011-11-07T08:00:00Z",
-            "description": "subset of snomed-ct regarding neoplasm and hamartoma",
+            "date_created": "2011-11-22T09:49:56Z",
+            "date_released": "2011-11-22T08:00:00Z",
+            "description": "This a subset of snomed-ct regarding neoplasm and hamartoma, based on Snomed clinical practice's used terms (Fondazione S.Maugeri anatomic pathology dep.). This subset is not to be meant exhaustive, it has been created in order to built a proper ontology for our project, ONCO-i2b2, through the NCBO Ontology Tools.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SMD_NPLSM/1/smd_nplsm.owl",
+            "download_url": "media/ontologies/SNMD_BC/1/snmd_bc.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 392,
+            "id": 323,
             "indexed": true,
             "max_children": 397,
             "max_depth": 9,
             "md5sum": null,
-            "nb_classes": 1595,
+            "nb_classes": 1493,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
             "products": null,
-            "publication": null,
+            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/22536972",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
             "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "SMD_NPLSM_BRST",
-        "name": "Neoplasm_breast_cancer",
+        "acronym": "ONTOSINASC",
+        "name": "Ontology of Information System on Live Births",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2011-11-16T15:29:28Z",
-            "date_released": "2011-11-16T08:00:00Z",
-            "description": "This a subset of Neoplasm_hamartoma view, based on Snomed clinical practice's used terms (Fondazione S.Maugeri anatomic pathology dep.). This subset is not to be meant exhaustive, it has been created in order to built a proper ontology for our project, ONCO-i2b2, through the NCBO Ontology Tools. ",
-            "documentation": null,
+            "date_created": "2020-08-12T23:20:05Z",
+            "date_released": "2020-08-12T00:00:00Z",
+            "description": "[pt-br] A Ontologia do SINASC foi desenvolvida no contexto do projeto DIASUS. Ela buscou resgatar a estrutura presente na ficha de preenchimento da Declara\u00e7\u00e3o de Nascido Vivo - DN, do Sistema de Informa\u00e7\u00e3o de Nascidos Vivos - SINASC, do Sistema \u00danico de Sa\u00fade, do Brasil. \r\n\r\n[en] The SINASC Ontology was developed in the context of DIASUS project. This ontology intent to rescue the data structure presented in Live Birth Certificate form, from Information System on Live Births - SINASC, from Brazilian Public Health System - SUS. ",
+            "documentation": "https://gitlab.com/beliefchangetools/diasus/-/tree/master/OF_SINASC",
             "domain": null,
-            "download_url": "media/ontologies/SMD_NPLSM_BRST/1/smd_nplsm_brst.owl",
+            "download_url": "media/ontologies/ONTOSINASC/1/ontosinasc.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 339,
+            "id": 3866,
             "indexed": true,
-            "max_children": 126,
-            "max_depth": 8,
+            "max_children": 10,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 980,
+            "nb_classes": 11,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 34,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "unknown"
+            "version": "1.0.0"
         },
         "topics": null
     },
     {
-        "acronym": "SMO",
-        "name": "Syrian Movies Ontology",
+        "acronym": "TO",
+        "name": "Plant Trait Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2022-04-13T19:08:50Z",
-            "date_released": "2022-04-13T00:00:00Z",
-            "description": "Syrian Movies Ontology, For Education.",
+            "date_created": "2023-07-24T14:52:57.927122Z",
+            "date_released": "2023-07-24T14:52:57.927111Z",
+            "description": "A controlled vocabulary to describe phenotypic traits in plants.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/SMO/1/smo.owl",
+            "domain": "phenotype",
+            "download_url": "media/ontologies/TO/11/to.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 5042,
+            "home_page": "http://browser.planteome.org/amigo",
+            "id": 5962,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 6,
+            "max_children": 402,
+            "max_depth": 32,
+            "md5sum": "21402c06e39dca968d2e707f75bc91b0",
+            "nb_classes": 5290,
             "nb_inconsistent": 0,
-            "nb_individuals": 65,
-            "nb_properties": 10,
-            "products": null,
+            "nb_individuals": 9,
+            "nb_properties": 173,
+            "products": [
+                {
+                    "id": "to.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/to.owl"
+                },
+                {
+                    "id": "to.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/to.obo"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/29186578",
+                    "title": "The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics."
+                }
+            ],
+            "submission_id": 11,
+            "taxon": {
+                "id": "NCBITaxon:33090",
+                "label": "Viridiplantae"
+            },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SNMD_BC",
-        "name": "Snomed neoplasm, breast cancer",
+        "acronym": "THREELPH",
+        "name": "NeuroMorpho.Org three layers in subcortical regions",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2011-11-22T09:49:56Z",
-            "date_released": "2011-11-22T08:00:00Z",
-            "description": "This a subset of snomed-ct regarding neoplasm and hamartoma, based on Snomed clinical practice's used terms (Fondazione S.Maugeri anatomic pathology dep.). This subset is not to be meant exhaustive, it has been created in order to built a proper ontology for our project, ONCO-i2b2, through the NCBO Ontology Tools.",
+            "date_created": "2016-08-29T19:53:28Z",
+            "date_released": "2016-08-29T00:00:00Z",
+            "description": "A hierarchy of three layers that are most commonly referred to in subcortical regions.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SNMD_BC/1/snmd_bc.owl",
+            "download_url": "media/ontologies/THREELPH/2/threelph.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 323,
+            "id": 638,
             "indexed": true,
-            "max_children": 397,
-            "max_depth": 9,
+            "max_children": 3,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 1493,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 10,
             "nb_properties": 0,
             "products": null,
-            "publication": "http://www.ncbi.nlm.nih.gov/pubmed/22536972",
+            "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "unknown"
+            "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "SNMD_BDY",
-        "name": "Snomed, Body Structure",
+        "acronym": "TIM",
+        "name": "Terra Interoperability Model",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2012-05-04T21:02:51Z",
-            "date_released": "2012-04-23T07:00:00Z",
-            "description": "The Body Structure subtree in SNOMED-CT. It has been created in order to built a proper ontology for our project, ONCO-i2b2, through the NCBO Ontology Tools. ",
-            "documentation": null,
+            "date_created": "2021-11-04T22:00:58Z",
+            "date_released": "2021-11-04T00:00:00Z",
+            "description": "The Terra Interoperability Model captures a common set of concepts and relationships for biomedical research intended to facilitate and encourage data sharing and reuse. Its purpose is to enable researchers to find highly connected biomedical data in a federated search space and support interoperability among datasets.",
+            "documentation": "https://datamodel.terra.bio/",
             "domain": null,
-            "download_url": "media/ontologies/SNMD_BDY/2/snmd_bdy.owl",
+            "download_url": "media/ontologies/TIM/1/tim.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 710,
+            "home_page": "https://github.com/DataBiosphere/terra-interoperability-model",
+            "id": 4683,
             "indexed": true,
-            "max_children": 93,
-            "max_depth": 30,
+            "max_children": 80,
+            "max_depth": 10,
             "md5sum": null,
-            "nb_classes": 20227,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_classes": 1278,
+            "nb_inconsistent": 2,
+            "nb_individuals": 82,
+            "nb_properties": 159,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 1,
             "taxon": null,
-            "version": "unknown"
+            "version": "Created with TopBraid Composer"
         },
         "topics": null
     },
     {
-        "acronym": "ONTOSINASC",
-        "name": "Ontology of Information System on Live Births",
+        "acronym": "TM-MER",
+        "name": "Traditional Medicine Meridian Value Sets",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 5,
+            "avg_children": 20,
             "classifiable": true,
-            "date_created": "2020-08-12T23:20:05Z",
-            "date_released": "2020-08-12T00:00:00Z",
-            "description": "[pt-br] A Ontologia do SINASC foi desenvolvida no contexto do projeto DIASUS. Ela buscou resgatar a estrutura presente na ficha de preenchimento da Declara\u00e7\u00e3o de Nascido Vivo - DN, do Sistema de Informa\u00e7\u00e3o de Nascidos Vivos - SINASC, do Sistema \u00danico de Sa\u00fade, do Brasil. \r\n\r\n[en] The SINASC Ontology was developed in the context of DIASUS project. This ontology intent to rescue the data structure presented in Live Birth Certificate form, from Information System on Live Births - SINASC, from Brazilian Public Health System - SUS. ",
-            "documentation": "https://gitlab.com/beliefchangetools/diasus/-/tree/master/OF_SINASC",
+            "date_created": "2011-02-02T01:39:56Z",
+            "date_released": "2011-02-01T08:00:00Z",
+            "description": "The meridian value set used in the International Classification of Traditional Medicine.",
+            "documentation": "https://sites.google.com/site/whoictm/home",
             "domain": null,
-            "download_url": "media/ontologies/ONTOSINASC/1/ontosinasc.owl",
+            "download_url": "media/ontologies/TM-MER/1/tm-mer.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 3866,
+            "home_page": "https://sites.google.com/site/whoictm/home",
+            "id": 636,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 2,
+            "max_children": 20,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 11,
+            "nb_classes": 20,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 34,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "1"
         },
         "topics": null
     },
     {
         "acronym": "PLIO",
         "name": "Protein-ligand interaction ontology",
         "species": null,
@@ -44107,48 +44260,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "PLOSTHES",
-        "name": "PLOS Thesaurus",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 4,
-            "classifiable": true,
-            "date_created": "2017-09-21T15:53:00Z",
-            "date_released": "2017-09-14T00:00:00Z",
-            "description": "The PLOS Thesaurus includes over 10,000 Subject Area terms covering the wide range of research topics included the PLOS journals.  It was initially built for PLOS in 2012 by Access Innovations (http://www.accessinn.com/). The entire corpus of PLOS articles was analysed to ensure that the thesaurus covers relevant research domains comprehensively. Updates are released several times a year.",
-            "documentation": "http://journals.plos.org/plosone/s/help-using-this-site#loc-subject-areas",
-            "domain": null,
-            "download_url": "media/ontologies/PLOSTHES/10/plosthes.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": "https://github.com/PLOS/plos-thesaurus",
-            "id": 625,
-            "indexed": true,
-            "max_children": 4,
-            "max_depth": 0,
-            "md5sum": null,
-            "nb_classes": 4,
-            "nb_inconsistent": 0,
-            "nb_individuals": 10712,
-            "nb_properties": 8,
-            "products": null,
-            "publication": "http://www.plos.org/",
-            "publications": null,
-            "submission_id": 10,
-            "taxon": null,
-            "version": "plosthes.2017-1"
-        },
-        "topics": null
-    },
-    {
         "acronym": "TM-CONST",
         "name": "Traditional Medicine Constitution Value Set",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 20,
             "classifiable": true,
@@ -44209,129 +44328,118 @@
             "submission_id": 1,
             "taxon": null,
             "version": "0.1"
         },
         "topics": null
     },
     {
-        "acronym": "PhenomeNET",
-        "name": "PhenomeNET Ontology",
-        "species": [
-            "Human"
-        ],
+        "acronym": "TMA",
+        "name": "Tissue Microarray Ontology",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-11-12T09:59:39.243131Z",
-            "date_released": "2017-11-12T09:59:39.243151Z",
-            "description": "An integrated phenotype ontology, combining phenotypes from human, mouse, fish.",
+            "date_created": "2010-05-07T18:57:44Z",
+            "date_released": "2010-05-07T07:00:00Z",
+            "description": "Tissue microarrays (TMA) are enormously useful tools for translational research, but incompatibilities in database systems between various researchers and institutions prevent the efficient sharing of data that could help realize their full potential.  The Resource Description Framework (RDF) provides a flexible method to represent knowledge in triples, which take the form Subject-Predicate-Object.  All data resources are described using Uniform Resource Identifiers (URI), which are global in scope.  We present an OWL (Web Ontology Language) schema that expands upon the TMA data exchange specification to address this issue and assist in data sharing and integration.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/PhenomeNET/1/phenomenet.owl",
+            "download_url": "media/ontologies/TMA/1/tma.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 817,
+            "id": 27,
             "indexed": true,
-            "max_children": 24788,
-            "max_depth": 18,
+            "max_children": 3,
+            "max_depth": 2,
             "md5sum": null,
-            "nb_classes": 244208,
+            "nb_classes": 7,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 177,
+            "nb_properties": 25,
             "products": null,
-            "publication": "http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005500",
+            "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1.1"
+            "version": "1.0.0"
         },
-        "topics": [
-            "phenotype"
-        ]
+        "topics": null
     },
     {
-        "acronym": "Pheno-e",
-        "name": "Pheno-e Ontology",
-        "species": [
-            "human",
-            "mouse",
-            "fish",
-            "yeast and Fly"
-        ],
+        "acronym": "TOK",
+        "name": "Terminological and Ontological Knowledge Resources Ontology",
+        "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 7,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2021-02-11T12:11:04.223460Z",
-            "date_released": "2021-02-11T12:11:04.223485Z",
-            "description": "",
-            "documentation": null,
+            "date_created": "2010-05-20T01:48:26Z",
+            "date_released": "2010-01-01T08:00:00Z",
+            "description": "An Ontology describing Resources having different formats. This Ontology can be used to annotate and describe Terminological, Ontological Knowledge resources.",
+            "documentation": "http://cui.unige.ch/isi/onto/tok/OWL_Doc/",
             "domain": null,
-            "download_url": "media/ontologies/Pheno-e/7/pheno-e.owl",
+            "download_url": "media/ontologies/TOK/3/tok.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4257,
+            "home_page": "http://cui.unige.ch/~ghoula/",
+            "id": 572,
             "indexed": true,
-            "max_children": 29317,
-            "max_depth": 23,
+            "max_children": 17,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 336704,
+            "nb_classes": 193,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 240,
+            "nb_individuals": 25,
+            "nb_properties": 113,
             "products": null,
-            "publication": null,
+            "publication": "http://cui.unige.ch/~ghoula/",
             "publications": null,
-            "submission_id": 7,
+            "submission_id": 3,
             "taxon": null,
-            "version": "0.0.6"
+            "version": "0.2.1"
         },
-        "topics": [
-            "An integrated phenotype ontology",
-            "combining phenotypes from human",
-            "mouse",
-            "fish",
-            "yeast and Fly"
-        ]
+        "topics": null
     },
     {
-        "acronym": "TM-MER",
-        "name": "Traditional Medicine Meridian Value Sets",
-        "species": null,
+        "acronym": "PhenomeNET",
+        "name": "PhenomeNET Ontology",
+        "species": [
+            "Human"
+        ],
         "status": "Classified",
         "submission": {
-            "avg_children": 20,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2011-02-02T01:39:56Z",
-            "date_released": "2011-02-01T08:00:00Z",
-            "description": "The meridian value set used in the International Classification of Traditional Medicine.",
-            "documentation": "https://sites.google.com/site/whoictm/home",
+            "date_created": "2017-11-12T09:59:39.243131Z",
+            "date_released": "2017-11-12T09:59:39.243151Z",
+            "description": "An integrated phenotype ontology, combining phenotypes from human, mouse, fish.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/TM-MER/1/tm-mer.owl",
+            "download_url": "media/ontologies/PhenomeNET/1/phenomenet.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://sites.google.com/site/whoictm/home",
-            "id": 636,
+            "home_page": null,
+            "id": 817,
             "indexed": true,
-            "max_children": 20,
-            "max_depth": 0,
+            "max_children": 24788,
+            "max_depth": 18,
             "md5sum": null,
-            "nb_classes": 20,
+            "nb_classes": 244208,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 177,
             "products": null,
-            "publication": null,
+            "publication": "http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005500",
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "1"
+            "version": "1.1"
         },
-        "topics": null
+        "topics": [
+            "phenotype"
+        ]
     },
     {
         "acronym": "TIMEBANK",
         "name": "Timebank Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
@@ -44394,78 +44502,78 @@
             "submission_id": 1,
             "taxon": null,
             "version": "20210819"
         },
         "topics": null
     },
     {
-        "acronym": "TMA",
-        "name": "Tissue Microarray Ontology",
+        "acronym": "TIME",
+        "name": "OWL-Time",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 0,
             "classifiable": true,
-            "date_created": "2010-05-07T18:57:44Z",
-            "date_released": "2010-05-07T07:00:00Z",
-            "description": "Tissue microarrays (TMA) are enormously useful tools for translational research, but incompatibilities in database systems between various researchers and institutions prevent the efficient sharing of data that could help realize their full potential.  The Resource Description Framework (RDF) provides a flexible method to represent knowledge in triples, which take the form Subject-Predicate-Object.  All data resources are described using Uniform Resource Identifiers (URI), which are global in scope.  We present an OWL (Web Ontology Language) schema that expands upon the TMA data exchange specification to address this issue and assist in data sharing and integration.",
-            "documentation": null,
+            "date_created": "2017-09-19T01:12:12Z",
+            "date_released": "2017-09-19T01:12:12Z",
+            "description": "OWL-Time is an OWL-2 DL ontology of temporal concepts, for describing the temporal properties of resources in the world or described in Web pages. The ontology provides a vocabulary for expressing facts about topological (ordering) relations among instants and intervals, together with information about durations, and about temporal position including date-time information. Time positions and durations may be expressed using either the conventional (Gregorian) calendar and clock, or using another temporal reference system such as Unix-time, geologic time, or different calendars.",
+            "documentation": "https://www.w3.org/TR/owl-time/",
             "domain": null,
-            "download_url": "media/ontologies/TMA/1/tma.owl",
+            "download_url": "media/ontologies/TIME/2/time.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 27,
+            "home_page": "https://www.w3.org/TR/owl-time/",
+            "id": 188,
             "indexed": true,
-            "max_children": 3,
-            "max_depth": 2,
+            "max_children": 0,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 7,
+            "nb_classes": 0,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 25,
+            "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 2,
             "taxon": null,
-            "version": "1.0.0"
+            "version": "2017"
         },
         "topics": null
     },
     {
-        "acronym": "TOK",
-        "name": "Terminological and Ontological Knowledge Resources Ontology",
+        "acronym": "PLOSTHES",
+        "name": "PLOS Thesaurus",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 4,
             "classifiable": true,
-            "date_created": "2010-05-20T01:48:26Z",
-            "date_released": "2010-01-01T08:00:00Z",
-            "description": "An Ontology describing Resources having different formats. This Ontology can be used to annotate and describe Terminological, Ontological Knowledge resources.",
-            "documentation": "http://cui.unige.ch/isi/onto/tok/OWL_Doc/",
+            "date_created": "2017-09-21T15:53:00Z",
+            "date_released": "2017-09-14T00:00:00Z",
+            "description": "The PLOS Thesaurus includes over 10,000 Subject Area terms covering the wide range of research topics included the PLOS journals.  It was initially built for PLOS in 2012 by Access Innovations (http://www.accessinn.com/). The entire corpus of PLOS articles was analysed to ensure that the thesaurus covers relevant research domains comprehensively. Updates are released several times a year.",
+            "documentation": "http://journals.plos.org/plosone/s/help-using-this-site#loc-subject-areas",
             "domain": null,
-            "download_url": "media/ontologies/TOK/3/tok.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://cui.unige.ch/~ghoula/",
-            "id": 572,
+            "download_url": "media/ontologies/PLOSTHES/10/plosthes.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": "https://github.com/PLOS/plos-thesaurus",
+            "id": 625,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 5,
+            "max_children": 4,
+            "max_depth": 0,
             "md5sum": null,
-            "nb_classes": 193,
+            "nb_classes": 4,
             "nb_inconsistent": 0,
-            "nb_individuals": 25,
-            "nb_properties": 113,
+            "nb_individuals": 10712,
+            "nb_properties": 8,
             "products": null,
-            "publication": "http://cui.unige.ch/~ghoula/",
+            "publication": "http://www.plos.org/",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 10,
             "taxon": null,
-            "version": "0.2.1"
+            "version": "plosthes.2017-1"
         },
         "topics": null
     },
     {
         "acronym": "TM-OTHER-FACTORS",
         "name": "Traditional Medicine Other Factors Value Set",
         "species": null,
@@ -44564,231 +44672,149 @@
             "submission_id": 4,
             "taxon": null,
             "version": "1.1"
         },
         "topics": null
     },
     {
-        "acronym": "TO",
-        "name": "Plant Trait Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2023-07-24T14:52:57.927122Z",
-            "date_released": "2023-07-24T14:52:57.927111Z",
-            "description": "A controlled vocabulary to describe phenotypic traits in plants.",
-            "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/TO/11/to.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://browser.planteome.org/amigo",
-            "id": 5962,
-            "indexed": true,
-            "max_children": 402,
-            "max_depth": 32,
-            "md5sum": "21402c06e39dca968d2e707f75bc91b0",
-            "nb_classes": 5290,
-            "nb_inconsistent": 0,
-            "nb_individuals": 9,
-            "nb_properties": 173,
-            "products": [
-                {
-                    "id": "to.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/to.owl"
-                },
-                {
-                    "id": "to.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/to.obo"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/29186578",
-                    "title": "The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics."
-                }
-            ],
-            "submission_id": 11,
-            "taxon": {
-                "id": "NCBITaxon:33090",
-                "label": "Viridiplantae"
-            },
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "THREELPH",
-        "name": "NeuroMorpho.Org three layers in subcortical regions",
-        "species": null,
+        "acronym": "Pheno-e",
+        "name": "Pheno-e Ontology",
+        "species": [
+            "human",
+            "mouse",
+            "fish",
+            "yeast and Fly"
+        ],
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 7,
             "classifiable": true,
-            "date_created": "2016-08-29T19:53:28Z",
-            "date_released": "2016-08-29T00:00:00Z",
-            "description": "A hierarchy of three layers that are most commonly referred to in subcortical regions.",
+            "date_created": "2021-02-11T12:11:04.223460Z",
+            "date_released": "2021-02-11T12:11:04.223485Z",
+            "description": "",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/THREELPH/2/threelph.owl",
+            "download_url": "media/ontologies/Pheno-e/7/pheno-e.owl",
             "has_ontology_language": "OWL",
             "home_page": null,
-            "id": 638,
-            "indexed": true,
-            "max_children": 3,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 4,
-            "nb_inconsistent": 0,
-            "nb_individuals": 10,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TIM",
-        "name": "Terra Interoperability Model",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-11-04T22:00:58Z",
-            "date_released": "2021-11-04T00:00:00Z",
-            "description": "The Terra Interoperability Model captures a common set of concepts and relationships for biomedical research intended to facilitate and encourage data sharing and reuse. Its purpose is to enable researchers to find highly connected biomedical data in a federated search space and support interoperability among datasets.",
-            "documentation": "https://datamodel.terra.bio/",
-            "domain": null,
-            "download_url": "media/ontologies/TIM/1/tim.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/DataBiosphere/terra-interoperability-model",
-            "id": 4683,
-            "indexed": true,
-            "max_children": 80,
-            "max_depth": 10,
-            "md5sum": null,
-            "nb_classes": 1278,
-            "nb_inconsistent": 2,
-            "nb_individuals": 82,
-            "nb_properties": 159,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "Created with TopBraid Composer"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TIME",
-        "name": "OWL-Time",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 0,
-            "classifiable": true,
-            "date_created": "2017-09-19T01:12:12Z",
-            "date_released": "2017-09-19T01:12:12Z",
-            "description": "OWL-Time is an OWL-2 DL ontology of temporal concepts, for describing the temporal properties of resources in the world or described in Web pages. The ontology provides a vocabulary for expressing facts about topological (ordering) relations among instants and intervals, together with information about durations, and about temporal position including date-time information. Time positions and durations may be expressed using either the conventional (Gregorian) calendar and clock, or using another temporal reference system such as Unix-time, geologic time, or different calendars.",
-            "documentation": "https://www.w3.org/TR/owl-time/",
-            "domain": null,
-            "download_url": "media/ontologies/TIME/2/time.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.w3.org/TR/owl-time/",
-            "id": 188,
+            "id": 4257,
             "indexed": true,
-            "max_children": 0,
-            "max_depth": 0,
+            "max_children": 29317,
+            "max_depth": 23,
             "md5sum": null,
-            "nb_classes": 0,
+            "nb_classes": 336704,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 0,
+            "nb_properties": 240,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 7,
             "taxon": null,
-            "version": "2017"
+            "version": "0.0.6"
         },
-        "topics": null
+        "topics": [
+            "An integrated phenotype ontology",
+            "combining phenotypes from human",
+            "mouse",
+            "fish",
+            "yeast and Fly"
+        ]
     },
     {
-        "acronym": "VODANA-UG-OPD",
-        "name": "VODANA-UGANDA-OPD",
+        "acronym": "WBBT",
+        "name": "C. elegans Gross Anatomy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 21,
             "classifiable": true,
-            "date_created": "2021-06-21T15:31:49Z",
-            "date_released": "2021-06-21T00:00:00Z",
-            "description": "This is an ontology on the data collected using an Out Patient Register within at the hospitals and facilities in Uganda.",
+            "date_created": "2023-07-03T15:43:39.231329Z",
+            "date_released": "2023-07-03T15:43:39.231317Z",
+            "description": "A structured controlled vocabulary of the anatomy of <i>Caenorhabditis elegans</i>.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/VODANA-UG-OPD/1/vodana-ug-opd.owl",
+            "domain": "anatomy and development",
+            "download_url": "media/ontologies/WBBT/24/wbbt.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 4591,
+            "home_page": "https://github.com/obophenotype/c-elegans-gross-anatomy-ontology",
+            "id": 5913,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
+            "max_children": 2768,
+            "max_depth": 9,
+            "md5sum": "190892bd4bc036e887d915b7b3116cbe",
+            "nb_classes": 7573,
             "nb_inconsistent": 0,
-            "nb_individuals": 23,
-            "nb_properties": 26,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 119,
+            "products": [
+                {
+                    "id": "wbbt.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/wbbt.owl"
+                },
+                {
+                    "id": "wbbt.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/wbbt.obo"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/18629098",
+                    "title": "Building a cell and anatomy ontology of Caenorhabditis elegans"
+                }
+            ],
+            "submission_id": 24,
+            "taxon": {
+                "id": "NCBITaxon:6237",
+                "label": "Caenorhabditis"
+            },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "VODANA-MPA",
-        "name": "VODANA-MIGRANTS-PRESS ARTICLES",
+        "acronym": "VTO",
+        "name": "Vertebrate Taxonomy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2022-01-04T16:29:15Z",
-            "date_released": "2022-01-04T00:00:00Z",
-            "description": "migrants Press Articles ontologies",
+            "date_created": "2020-11-16T13:20:18.985505Z",
+            "date_released": "2020-11-16T13:20:18.985481Z",
+            "description": "Comprehensive hierarchy of extinct and extant vertebrate taxa.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/VODANA-MPA/3/vodana-mpa.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4819,
+            "download_url": "media/ontologies/VTO/15/vto.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/phenoscape/vertebrate-taxonomy-ontology",
+            "id": 4054,
             "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
+            "max_children": 451,
+            "max_depth": 39,
+            "md5sum": "45719ac2b0dbf0984a2a6555a20547c7",
+            "nb_classes": 107254,
             "nb_inconsistent": 0,
-            "nb_individuals": 186,
-            "nb_properties": 16,
-            "products": null,
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": [
+                {
+                    "id": "vto.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/vto.owl"
+                },
+                {
+                    "id": "vto.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/vto.obo"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 15,
             "taxon": null,
-            "version": "2"
+            "version": null
         },
         "topics": null
     },
     {
         "acronym": "VODANANIGERIA",
         "name": "VODANANIGERIA",
         "species": null,
@@ -44819,172 +44845,14 @@
             "submission_id": 3,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "VODANA-TERMS",
-        "name": "HEIGHT/LENGTH FOR AGE Z-SCORE",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-03-28T17:47:09Z",
-            "date_released": "2021-03-28T00:00:00Z",
-            "description": "This is the HEIGHT/LENGTH FOR AGE Z-SCORE term.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/VODANA-TERMS/1/vodana-terms.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4376,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 4,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "VSAO",
-        "name": "Vertebrate Skeletal Anatomy Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2013-08-23T01:22:51Z",
-            "date_released": "2013-08-23T01:22:51Z",
-            "description": "VSAO is an anatomy ontology covering the vertebrate skeletal system. VSAO integrates terms for skeletal cells, tissues, biological processes, organs (skeletal elements such as bones and cartilages), and subdivisions of the skeletal system.",
-            "documentation": "https://www.phenoscape.org/wiki/Skeletal_Anatomy_Jamboree",
-            "domain": null,
-            "download_url": "media/ontologies/VSAO/20/vsao.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "http://phenoscape.org/wiki/Main_Page",
-            "id": 677,
-            "indexed": true,
-            "max_children": 18,
-            "max_depth": 9,
-            "md5sum": null,
-            "nb_classes": 322,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 18,
-            "products": null,
-            "publication": "http://dx.plos.org/10.1371/journal.pone.0051070",
-            "publications": null,
-            "submission_id": 20,
-            "taxon": null,
-            "version": "2012-11-06"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "VT",
-        "name": "Vertebrate trait ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2023-07-17T15:02:02.098945Z",
-            "date_released": "2023-07-17T15:02:02.098934Z",
-            "description": "An ontology of traits covering vertebrates",
-            "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/VT/92/vt.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/AnimalGenome/vertebrate-trait-ontology",
-            "id": 5950,
-            "indexed": true,
-            "max_children": 94,
-            "max_depth": 13,
-            "md5sum": "5abfc8a45000a82f1a975f0e9e4c1ac6",
-            "nb_classes": 3820,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 2,
-            "products": [
-                {
-                    "id": "vt.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/vt.owl"
-                }
-            ],
-            "publication": null,
-            "publications": null,
-            "submission_id": 92,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "WBBT",
-        "name": "C. elegans Gross Anatomy Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 21,
-            "classifiable": true,
-            "date_created": "2023-07-03T15:43:39.231329Z",
-            "date_released": "2023-07-03T15:43:39.231317Z",
-            "description": "A structured controlled vocabulary of the anatomy of <i>Caenorhabditis elegans</i>.",
-            "documentation": null,
-            "domain": "anatomy and development",
-            "download_url": "media/ontologies/WBBT/24/wbbt.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/c-elegans-gross-anatomy-ontology",
-            "id": 5913,
-            "indexed": true,
-            "max_children": 2768,
-            "max_depth": 9,
-            "md5sum": "190892bd4bc036e887d915b7b3116cbe",
-            "nb_classes": 7573,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 119,
-            "products": [
-                {
-                    "id": "wbbt.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/wbbt.owl"
-                },
-                {
-                    "id": "wbbt.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/wbbt.obo"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/18629098",
-                    "title": "Building a cell and anatomy ontology of Caenorhabditis elegans"
-                }
-            ],
-            "submission_id": 24,
-            "taxon": {
-                "id": "NCBITaxon:6237",
-                "label": "Caenorhabditis"
-            },
-            "version": null
-        },
-        "topics": null
-    },
-    {
         "acronym": "WB-BT",
         "name": "C. elegans Gross Anatomy Vocabulary",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 12,
             "classifiable": true,
@@ -45096,53 +44964,44 @@
             "submission_id": 49,
             "taxon": null,
             "version": "unknown"
         },
         "topics": null
     },
     {
-        "acronym": "VTO",
-        "name": "Vertebrate Taxonomy Ontology",
+        "acronym": "ROLEO",
+        "name": "Role Ontology",
         "species": null,
-        "status": "Classified",
+        "status": "Unloadable",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 39,
             "classifiable": true,
-            "date_created": "2020-11-16T13:20:18.985505Z",
-            "date_released": "2020-11-16T13:20:18.985481Z",
-            "description": "Comprehensive hierarchy of extinct and extant vertebrate taxa.",
+            "date_created": "2010-10-23T11:13:05Z",
+            "date_released": "2010-10-23T07:00:00Z",
+            "description": "The Role Ontology (RoleO) is a ontology in the domain of role classification. RoleO aims to standardize role classification and support computer-assisted reasoning.  RoleO is a community-based ontology, and its development follows the OBO Foundry principles.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/VTO/15/vto.owl",
+            "download_url": "media/ontologies/ROLEO/1/roleo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/phenoscape/vertebrate-taxonomy-ontology",
-            "id": 4054,
+            "home_page": "http://sourceforge.net/projects/roleo",
+            "id": 460,
             "indexed": true,
-            "max_children": 451,
-            "max_depth": 39,
-            "md5sum": "45719ac2b0dbf0984a2a6555a20547c7",
-            "nb_classes": 107254,
+            "max_children": 563,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 673,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
             "nb_properties": 0,
-            "products": [
-                {
-                    "id": "vto.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/vto.owl"
-                },
-                {
-                    "id": "vto.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/vto.obo"
-                }
-            ],
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 15,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Vision Release; 1.0.11"
         },
         "topics": null
     },
     {
         "acronym": "VODANAKENYA",
         "name": "VODANA KENYA",
         "species": null,
@@ -45173,44 +45032,44 @@
             "submission_id": 4,
             "taxon": null,
             "version": "0.1.1"
         },
         "topics": null
     },
     {
-        "acronym": "ROLEO",
-        "name": "Role Ontology",
+        "acronym": "VODANA-TERMS",
+        "name": "HEIGHT/LENGTH FOR AGE Z-SCORE",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 39,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2010-10-23T11:13:05Z",
-            "date_released": "2010-10-23T07:00:00Z",
-            "description": "The Role Ontology (RoleO) is a ontology in the domain of role classification. RoleO aims to standardize role classification and support computer-assisted reasoning.  RoleO is a community-based ontology, and its development follows the OBO Foundry principles.",
+            "date_created": "2021-03-28T17:47:09Z",
+            "date_released": "2021-03-28T00:00:00Z",
+            "description": "This is the HEIGHT/LENGTH FOR AGE Z-SCORE term.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/ROLEO/1/roleo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://sourceforge.net/projects/roleo",
-            "id": 460,
+            "download_url": "media/ontologies/VODANA-TERMS/1/vodana-terms.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 4376,
             "indexed": true,
-            "max_children": 563,
-            "max_depth": 4,
+            "max_children": 2,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 673,
+            "nb_classes": 2,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
+            "nb_individuals": 4,
             "nb_properties": 0,
             "products": null,
             "publication": null,
             "publications": null,
             "submission_id": 1,
             "taxon": null,
-            "version": "Vision Release; 1.0.11"
+            "version": "1"
         },
         "topics": null
     },
     {
         "acronym": "VODANA-GENERAL",
         "name": "VODANATERMS",
         "species": null,
@@ -45529,60 +45388,150 @@
             "submission_id": 2,
             "taxon": null,
             "version": "2.0"
         },
         "topics": null
     },
     {
-        "acronym": "FYPO",
-        "name": "Fission Yeast Phenotype Ontology",
+        "acronym": "VODANA-UG-OPD",
+        "name": "VODANA-UGANDA-OPD",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2021-06-21T15:31:49Z",
+            "date_released": "2021-06-21T00:00:00Z",
+            "description": "This is an ontology on the data collected using an Out Patient Register within at the hospitals and facilities in Uganda.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/VODANA-UG-OPD/1/vodana-ug-opd.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 4591,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 23,
+            "nb_properties": 26,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "VODANA-MPA",
+        "name": "VODANA-MIGRANTS-PRESS ARTICLES",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2022-01-04T16:29:15Z",
+            "date_released": "2022-01-04T00:00:00Z",
+            "description": "migrants Press Articles ontologies",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/VODANA-MPA/3/vodana-mpa.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 4819,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 186,
+            "nb_properties": 16,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "VSAO",
+        "name": "Vertebrate Skeletal Anatomy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-07-17T12:24:59.619327Z",
-            "date_released": "2023-07-17T12:24:59.619312Z",
-            "description": "FYPO is a formal ontology of phenotypes observed in fission yeast.",
+            "date_created": "2013-08-23T01:22:51Z",
+            "date_released": "2013-08-23T01:22:51Z",
+            "description": "VSAO is an anatomy ontology covering the vertebrate skeletal system. VSAO integrates terms for skeletal cells, tissues, biological processes, organs (skeletal elements such as bones and cartilages), and subdivisions of the skeletal system.",
+            "documentation": "https://www.phenoscape.org/wiki/Skeletal_Anatomy_Jamboree",
+            "domain": null,
+            "download_url": "media/ontologies/VSAO/20/vsao.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "http://phenoscape.org/wiki/Main_Page",
+            "id": 677,
+            "indexed": true,
+            "max_children": 18,
+            "max_depth": 9,
+            "md5sum": null,
+            "nb_classes": 322,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 18,
+            "products": null,
+            "publication": "http://dx.plos.org/10.1371/journal.pone.0051070",
+            "publications": null,
+            "submission_id": 20,
+            "taxon": null,
+            "version": "2012-11-06"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "VT",
+        "name": "Vertebrate trait ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-07-17T15:02:02.098945Z",
+            "date_released": "2023-07-17T15:02:02.098934Z",
+            "description": "An ontology of traits covering vertebrates",
             "documentation": null,
             "domain": "phenotype",
-            "download_url": "media/ontologies/FYPO/442/fypo.owl",
+            "download_url": "media/ontologies/VT/92/vt.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/pombase/fypo",
-            "id": 5946,
+            "home_page": "https://github.com/AnimalGenome/vertebrate-trait-ontology",
+            "id": 5950,
             "indexed": true,
-            "max_children": 392,
-            "max_depth": 21,
-            "md5sum": "bf018340ee68e910b1118b178e5ab87d",
-            "nb_classes": 16480,
+            "max_children": 94,
+            "max_depth": 13,
+            "md5sum": "5abfc8a45000a82f1a975f0e9e4c1ac6",
+            "nb_classes": 3820,
             "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 172,
+            "nb_individuals": 0,
+            "nb_properties": 2,
             "products": [
                 {
-                    "id": "fypo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fypo.owl"
-                },
-                {
-                    "id": "fypo.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/fypo.obo"
+                    "id": "vt.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/vt.owl"
                 }
             ],
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/23658422",
-                    "title": "FYPO: The Fission Yeast Phenotype Ontology."
-                }
-            ],
-            "submission_id": 442,
-            "taxon": {
-                "id": "NCBITaxon:4896",
-                "label": "S. pombe"
-            },
+            "publications": null,
+            "submission_id": 92,
+            "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "ECGF",
         "name": "ECG Findings",
@@ -45614,14 +45563,48 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
+        "acronym": "NEMO",
+        "name": "Neural ElectroMagnetic Ontology",
+        "species": null,
+        "status": "Unloadable",
+        "submission": {
+            "avg_children": 4,
+            "classifiable": false,
+            "date_created": "2013-10-21T14:51:44Z",
+            "date_released": "2013-10-21T00:00:00Z",
+            "description": "Neural ElectroMagnetic Ontology (NEMO) describes classes of event-related brain potentials (ERP) and their properties, including spatial, temporal, and functional (cognitive/behavioral) attributes, and data-level attributes (acquisition and analysis parameters). Working (unstable), Release (stable), and Old (archival) versions of NEMO can be found on sourceforge: http://sourceforge.net/projects/nemoontologies/.",
+            "documentation": "https://portal.nemo.nic.uoregon.edu",
+            "domain": null,
+            "download_url": "media/ontologies/NEMO/24/nemo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "http://nemo.nic.uoregon.edu",
+            "id": 517,
+            "indexed": false,
+            "max_children": 56,
+            "max_depth": 14,
+            "md5sum": null,
+            "nb_classes": 1851,
+            "nb_inconsistent": 0,
+            "nb_individuals": 120,
+            "nb_properties": 98,
+            "products": null,
+            "publication": "http://nemo.nic.uoregon.edu/wiki/Publications",
+            "publications": null,
+            "submission_id": 24,
+            "taxon": null,
+            "version": "v3.10"
+        },
+        "topics": null
+    },
+    {
         "acronym": "ECGT",
         "name": "ECG Terms",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -45949,14 +45932,116 @@
             "submission_id": 12,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
+        "acronym": "ETANC",
+        "name": "ETHIOPIANC",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2021-09-03T06:33:17Z",
+            "date_released": "2021-09-02T00:00:00Z",
+            "description": "Ethiopian Antenatal care (ANC) register vocabulary ",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/ETANC/2/etanc.skos",
+            "has_ontology_language": "SKOS",
+            "home_page": null,
+            "id": 4641,
+            "indexed": true,
+            "max_children": 2,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 2,
+            "nb_inconsistent": 0,
+            "nb_individuals": 52,
+            "nb_properties": 12,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": "0.2"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "FTC",
+        "name": "Functional Therapeutic Chemical Classification System",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 7,
+            "classifiable": true,
+            "date_created": "2016-02-15T16:38:02Z",
+            "date_released": "2016-02-15T00:00:00Z",
+            "description": "The Functional Therapeutic Chemical Classification System (FTC) defines over 20'000 mechanisms and modes of action for approuved drugs. The resource abstracts away from the traditional chemical structure-based approach and focuses solely on the mode of action of drugs. The classification sorts therapeutic agents based on their function in the human body.",
+            "documentation": "https://www.ebi.ac.uk/chembl/ftc",
+            "domain": null,
+            "download_url": "media/ontologies/FTC/1/ftc.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://www.ebi.ac.uk/chembl/ftc",
+            "id": 256,
+            "indexed": true,
+            "max_children": 25098,
+            "max_depth": 16,
+            "md5sum": null,
+            "nb_classes": 61426,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 10,
+            "products": null,
+            "publication": "https://www.ebi.ac.uk/chembl/ftc",
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "FUNCTIONALH",
+        "name": "NeuroMorpho.Org cell types function",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2016-08-30T01:03:36Z",
+            "date_released": "2016-08-30T01:03:36Z",
+            "description": "A hierarchy of neuron types based on functional role.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/FUNCTIONALH/3/functionalh.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 635,
+            "indexed": true,
+            "max_children": 7,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 12,
+            "nb_inconsistent": 0,
+            "nb_individuals": 11,
+            "nb_properties": 0,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "2.0"
+        },
+        "topics": null
+    },
+    {
         "acronym": "FOVT",
         "name": "FuTRES Ontology of Vertebrate Traits",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -46111,116 +46196,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1"
         },
         "topics": null
     },
     {
-        "acronym": "ETANC",
-        "name": "ETHIOPIANC",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2021-09-03T06:33:17Z",
-            "date_released": "2021-09-02T00:00:00Z",
-            "description": "Ethiopian Antenatal care (ANC) register vocabulary ",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/ETANC/2/etanc.skos",
-            "has_ontology_language": "SKOS",
-            "home_page": null,
-            "id": 4641,
-            "indexed": true,
-            "max_children": 2,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 2,
-            "nb_inconsistent": 0,
-            "nb_individuals": 52,
-            "nb_properties": 12,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "0.2"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FTC",
-        "name": "Functional Therapeutic Chemical Classification System",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 7,
-            "classifiable": true,
-            "date_created": "2016-02-15T16:38:02Z",
-            "date_released": "2016-02-15T00:00:00Z",
-            "description": "The Functional Therapeutic Chemical Classification System (FTC) defines over 20'000 mechanisms and modes of action for approuved drugs. The resource abstracts away from the traditional chemical structure-based approach and focuses solely on the mode of action of drugs. The classification sorts therapeutic agents based on their function in the human body.",
-            "documentation": "https://www.ebi.ac.uk/chembl/ftc",
-            "domain": null,
-            "download_url": "media/ontologies/FTC/1/ftc.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://www.ebi.ac.uk/chembl/ftc",
-            "id": 256,
-            "indexed": true,
-            "max_children": 25098,
-            "max_depth": 16,
-            "md5sum": null,
-            "nb_classes": 61426,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 10,
-            "products": null,
-            "publication": "https://www.ebi.ac.uk/chembl/ftc",
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "1"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "FUNCTIONALH",
-        "name": "NeuroMorpho.Org cell types function",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 2,
-            "classifiable": true,
-            "date_created": "2016-08-30T01:03:36Z",
-            "date_released": "2016-08-30T01:03:36Z",
-            "description": "A hierarchy of neuron types based on functional role.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/FUNCTIONALH/3/functionalh.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 635,
-            "indexed": true,
-            "max_children": 7,
-            "max_depth": 2,
-            "md5sum": null,
-            "nb_classes": 12,
-            "nb_inconsistent": 0,
-            "nb_individuals": 11,
-            "nb_properties": 0,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 3,
-            "taxon": null,
-            "version": "2.0"
-        },
-        "topics": null
-    },
-    {
         "acronym": "FMA-SUBSET",
         "name": "Foundational Model of Anatomy (subset)",
         "species": [],
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -46281,44 +46264,95 @@
             "submission_id": 5,
             "taxon": null,
             "version": "0.9.2"
         },
         "topics": null
     },
     {
-        "acronym": "NEMO",
-        "name": "Neural ElectroMagnetic Ontology",
+        "acronym": "FYPO",
+        "name": "Fission Yeast Phenotype Ontology",
         "species": null,
-        "status": "Unloadable",
+        "status": "Classified",
         "submission": {
-            "avg_children": 4,
-            "classifiable": false,
-            "date_created": "2013-10-21T14:51:44Z",
-            "date_released": "2013-10-21T00:00:00Z",
-            "description": "Neural ElectroMagnetic Ontology (NEMO) describes classes of event-related brain potentials (ERP) and their properties, including spatial, temporal, and functional (cognitive/behavioral) attributes, and data-level attributes (acquisition and analysis parameters). Working (unstable), Release (stable), and Old (archival) versions of NEMO can be found on sourceforge: http://sourceforge.net/projects/nemoontologies/.",
-            "documentation": "https://portal.nemo.nic.uoregon.edu",
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2023-07-31T12:22:33.763301Z",
+            "date_released": "2023-07-31T12:22:33.763287Z",
+            "description": "FYPO is a formal ontology of phenotypes observed in fission yeast.",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/FYPO/443/fypo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/pombase/fypo",
+            "id": 5969,
+            "indexed": true,
+            "max_children": 392,
+            "max_depth": 21,
+            "md5sum": "e2cc938b543d890ed3cb70a98edc74b5",
+            "nb_classes": 16484,
+            "nb_inconsistent": 0,
+            "nb_individuals": 18,
+            "nb_properties": 172,
+            "products": [
+                {
+                    "id": "fypo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fypo.owl"
+                },
+                {
+                    "id": "fypo.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/fypo.obo"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/23658422",
+                    "title": "FYPO: The Fission Yeast Phenotype Ontology."
+                }
+            ],
+            "submission_id": 443,
+            "taxon": {
+                "id": "NCBITaxon:4896",
+                "label": "S. pombe"
+            },
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "GENE-CDS",
+        "name": "Genomic Clinical Decision Support Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2012-11-21T15:07:04Z",
+            "date_released": "2012-11-21T08:00:00Z",
+            "description": "The Genomic CDS ontology aims to unify several functionalities in a single resource, being:\r\n* A knowledge base for clinical pharmacogenomics/pharmacogenetics that can be used for question-answering (e.g., which SNPs are associated with this drug?)\r\n* A rule base for clinical decision support (e.g., inferring that a patient with a specific set of SNPs requires a lowered dose of warfarin and generating a CDS message that can be viewed by clinicians)\r\n* A tool for checking data consistency (e.g., highlighting which allele definitions in PharmGKB are overlapping, or which clinical decision support rules are matching the same group of patients)",
+            "documentation": "http://www.genomic-cds.org/",
             "domain": null,
-            "download_url": "media/ontologies/NEMO/24/nemo.owl",
+            "download_url": "media/ontologies/GENE-CDS/1/gene-cds.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://nemo.nic.uoregon.edu",
-            "id": 517,
-            "indexed": false,
-            "max_children": 56,
-            "max_depth": 14,
+            "home_page": "http://samwald.info",
+            "id": 92,
+            "indexed": true,
+            "max_children": 576,
+            "max_depth": 3,
             "md5sum": null,
-            "nb_classes": 1851,
+            "nb_classes": 2265,
             "nb_inconsistent": 0,
-            "nb_individuals": 120,
-            "nb_properties": 98,
+            "nb_individuals": 0,
+            "nb_properties": 2,
             "products": null,
-            "publication": "http://nemo.nic.uoregon.edu/wiki/Publications",
+            "publication": null,
             "publications": null,
-            "submission_id": 24,
+            "submission_id": 1,
             "taxon": null,
-            "version": "v3.10"
+            "version": "0.8.0"
         },
         "topics": null
     },
     {
         "acronym": "GCO",
         "name": "Genome Component Ontology",
         "species": null,
@@ -46383,48 +46417,14 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "GENE-CDS",
-        "name": "Genomic Clinical Decision Support Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2012-11-21T15:07:04Z",
-            "date_released": "2012-11-21T08:00:00Z",
-            "description": "The Genomic CDS ontology aims to unify several functionalities in a single resource, being:\r\n* A knowledge base for clinical pharmacogenomics/pharmacogenetics that can be used for question-answering (e.g., which SNPs are associated with this drug?)\r\n* A rule base for clinical decision support (e.g., inferring that a patient with a specific set of SNPs requires a lowered dose of warfarin and generating a CDS message that can be viewed by clinicians)\r\n* A tool for checking data consistency (e.g., highlighting which allele definitions in PharmGKB are overlapping, or which clinical decision support rules are matching the same group of patients)",
-            "documentation": "http://www.genomic-cds.org/",
-            "domain": null,
-            "download_url": "media/ontologies/GENE-CDS/1/gene-cds.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "http://samwald.info",
-            "id": 92,
-            "indexed": true,
-            "max_children": 576,
-            "max_depth": 3,
-            "md5sum": null,
-            "nb_classes": 2265,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 2,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "0.8.0"
-        },
-        "topics": null
-    },
-    {
         "acronym": "HASCO",
         "name": "Human-Aware Science Ontology",
         "species": null,
         "status": "Unloadable",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -46587,14 +46587,149 @@
             "submission_id": 5,
             "taxon": null,
             "version": "1.1.0"
         },
         "topics": null
     },
     {
+        "acronym": "T4FS",
+        "name": "terms4FAIRskills",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 5,
+            "classifiable": true,
+            "date_created": "2023-03-06T15:12:22.079904Z",
+            "date_released": "2023-03-06T15:12:22.079889Z",
+            "description": "A terminology for the skills necessary to make data FAIR and to keep it FAIR.",
+            "documentation": null,
+            "domain": "information",
+            "download_url": "media/ontologies/T4FS/3/t4fs.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/terms4fairskills/FAIRterminology",
+            "id": 5709,
+            "indexed": true,
+            "max_children": 110,
+            "max_depth": 9,
+            "md5sum": "09ec10664781171973973d0cb97cd28b",
+            "nb_classes": 564,
+            "nb_inconsistent": 0,
+            "nb_individuals": 18,
+            "nb_properties": 27,
+            "products": [
+                {
+                    "id": "t4fs.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs.owl"
+                },
+                {
+                    "id": "t4fs.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs.obo"
+                },
+                {
+                    "id": "t4fs.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs.json"
+                },
+                {
+                    "id": "t4fs-community.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs-community.owl",
+                    "title": "This community view of T4FS makes the ontology available in OWL without upper-level ontology (ULO) terms to give the user community a simpler view of the term hierarchy."
+                },
+                {
+                    "id": "t4fs-community.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs-community.obo",
+                    "title": "This community view of T4FS makes the ontology available in OBO format without upper-level ontology (ULO) terms to give the user community a simpler view of the term hierarchy."
+                },
+                {
+                    "id": "t4fs-community.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs-community.json",
+                    "title": "This community view of T4FS makes the ontology available in JSON format without upper-level ontology (ULO) terms to give the user community a simpler view of the term hierarchy."
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://doi.org/10.5281/zenodo.4705219",
+                    "title": "EOSC Co-creation funded project 074: Delivery of a proof of concept for terms4FAIRskills: Technical report"
+                }
+            ],
+            "submission_id": 3,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TEMPO",
+        "name": "TempO",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 1,
+            "classifiable": true,
+            "date_created": "2023-03-10T02:20:01Z",
+            "date_released": "2023-03-10T02:20:00Z",
+            "description": "TempO is a lightweight ontology for describing temporal validity and efficacy according to a tritemporal data model.",
+            "documentation": "http://purl.org/tempo/html",
+            "domain": null,
+            "download_url": "media/ontologies/TEMPO/3/tempo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/hroptatyr/tempo/",
+            "id": 5724,
+            "indexed": true,
+            "max_children": 1,
+            "max_depth": 1,
+            "md5sum": null,
+            "nb_classes": 1,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 1,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 3,
+            "taxon": null,
+            "version": "0.1"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "TRIAGE",
+        "name": "Nurse triage",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2015-09-24T01:46:41Z",
+            "date_released": "2015-09-24T00:00:00Z",
+            "description": "An ontology showing the process domain of a triage nurse.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/TRIAGE/2/triage.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 387,
+            "indexed": true,
+            "max_children": 8,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 32,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 29,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 2,
+            "taxon": null,
+            "version": null
+        },
+        "topics": null
+    },
+    {
         "acronym": "svuOnt",
         "name": "svuOntology",
         "species": [
             "Academic"
         ],
         "status": "Classified",
         "submission": {
@@ -46671,126 +46806,78 @@
                 "label": "Ixodidae"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "SYN",
-        "name": "Sage Bionetworks Synapse Ontology",
+        "acronym": "TAO",
+        "name": "Teleost Anatomy Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 5,
             "classifiable": true,
-            "date_created": "2013-01-07T20:28:04Z",
-            "date_released": "2012-11-15T08:00:00Z",
-            "description": "The controlled vocabulary used for various entity properties in the Synapse platform.",
+            "date_created": "2013-08-23T01:10:25Z",
+            "date_released": "2015-08-20T00:00:00Z",
+            "description": "TAO is a multispecies anatomy ontology for teleost fishes. Originally seeded from ZFA, but intended to cover terms relevant to other taxa.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SYN/3/syn.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 598,
+            "download_url": "media/ontologies/TAO/135/tao.obo",
+            "has_ontology_language": "OBO",
+            "home_page": "https://www.nescent.org/phenoscape/Main_Page",
+            "id": 675,
             "indexed": true,
-            "max_children": 1041,
-            "max_depth": 30,
+            "max_children": 78,
+            "max_depth": 12,
             "md5sum": null,
-            "nb_classes": 14462,
+            "nb_classes": 3428,
             "nb_inconsistent": 0,
             "nb_individuals": 0,
-            "nb_properties": 2,
+            "nb_properties": 24,
             "products": null,
-            "publication": null,
+            "publication": "http://dx.doi.org/10.1093/sysbio/syq013",
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 135,
             "taxon": null,
-            "version": ".3"
+            "version": "2012-08-10"
         },
         "topics": null
     },
     {
-        "acronym": "SWEET",
-        "name": "Semantic Web for Earth and Environment Technology Ontology",
+        "acronym": "SYN",
+        "name": "Sage Bionetworks Synapse Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 4,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2017-11-29T02:29:11Z",
-            "date_released": "2017-11-29T02:29:11Z",
-            "description": "The Semantic Web for Earth and Environmental Terminology is a mature foundational ontology that contains over 6000 concepts organized in 200 ontologies represented in OWL. Top level concepts include Representation (math, space, science, time, data), Realm (Ocean, Land Surface, Terrestrial Hydroshere, Atmosphere, etc.), Phenomena (macro-scale ecological and physical), Processes (micro-scale physical, biological, chemical, and mathematical), Human Activities (Decision, Commerce, Jurisdiction, Environmental, Research). Originally developed by NASA Jet Propulsion Labs under Rob Raskin, SWEET is now officially under the governance of the ESIP foundation. ",
-            "documentation": "http://sweet.jpl.nasa.gov/",
+            "date_created": "2013-01-07T20:28:04Z",
+            "date_released": "2012-11-15T08:00:00Z",
+            "description": "The controlled vocabulary used for various entity properties in the Synapse platform.",
+            "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/SWEET/2/sweet.owl",
+            "download_url": "media/ontologies/SYN/3/syn.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ESIPFed/sweet/blob/master/README.md",
-            "id": 897,
+            "home_page": null,
+            "id": 598,
             "indexed": true,
-            "max_children": 51,
-            "max_depth": 11,
+            "max_children": 1041,
+            "max_depth": 30,
             "md5sum": null,
-            "nb_classes": 4543,
+            "nb_classes": 14462,
             "nb_inconsistent": 0,
-            "nb_individuals": 2020,
-            "nb_properties": 327,
+            "nb_individuals": 0,
+            "nb_properties": 2,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 2,
-            "taxon": null,
-            "version": "3.1.0"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "SWO",
-        "name": "Software ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 11,
-            "classifiable": true,
-            "date_created": "2023-03-06T15:11:42.465757Z",
-            "date_released": "2023-03-06T15:11:42.465746Z",
-            "description": "The Software Ontology (SWO) is a resource for describing software tools, their types, tasks, versions, provenance and associated data. It contains detailed information on licensing and formats as well as software applications themselves, mainly (but not limited) to the bioinformatics community.",
-            "documentation": null,
-            "domain": "information technology",
-            "download_url": "media/ontologies/SWO/32/swo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/allysonlister/swo",
-            "id": 5708,
-            "indexed": true,
-            "max_children": 677,
-            "max_depth": 16,
-            "md5sum": "ecbf9eabd3ccad51ba26c0667bf1258a",
-            "nb_classes": 1971,
-            "nb_inconsistent": 0,
-            "nb_individuals": 446,
-            "nb_properties": 161,
-            "products": [
-                {
-                    "id": "swo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/swo.owl"
-                },
-                {
-                    "id": "swo.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/swo.json"
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/25068035",
-                    "title": "The Software Ontology (SWO): a resource for reproducibility in biomedical data analysis, curation and digital preservation"
-                }
-            ],
-            "submission_id": 32,
+            "submission_id": 3,
             "taxon": null,
-            "version": null
+            "version": ".3"
         },
         "topics": null
     },
     {
         "acronym": "SYMP",
         "name": "Symptom Ontology",
         "species": null,
@@ -46842,298 +46929,202 @@
                 "label": "Homo sapiens"
             },
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "T4FS",
-        "name": "terms4FAIRskills",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2023-03-06T15:12:22.079904Z",
-            "date_released": "2023-03-06T15:12:22.079889Z",
-            "description": "A terminology for the skills necessary to make data FAIR and to keep it FAIR.",
-            "documentation": null,
-            "domain": "information",
-            "download_url": "media/ontologies/T4FS/3/t4fs.owl",
-            "has_ontology_language": "OWL",
-            "home_page": "https://github.com/terms4fairskills/FAIRterminology",
-            "id": 5709,
-            "indexed": true,
-            "max_children": 110,
-            "max_depth": 9,
-            "md5sum": "09ec10664781171973973d0cb97cd28b",
-            "nb_classes": 564,
-            "nb_inconsistent": 0,
-            "nb_individuals": 18,
-            "nb_properties": 27,
-            "products": [
-                {
-                    "id": "t4fs.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs.owl"
-                },
-                {
-                    "id": "t4fs.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs.obo"
-                },
-                {
-                    "id": "t4fs.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs.json"
-                },
-                {
-                    "id": "t4fs-community.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs-community.owl",
-                    "title": "This community view of T4FS makes the ontology available in OWL without upper-level ontology (ULO) terms to give the user community a simpler view of the term hierarchy."
-                },
-                {
-                    "id": "t4fs-community.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs-community.obo",
-                    "title": "This community view of T4FS makes the ontology available in OBO format without upper-level ontology (ULO) terms to give the user community a simpler view of the term hierarchy."
-                },
-                {
-                    "id": "t4fs-community.json",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/t4fs-community.json",
-                    "title": "This community view of T4FS makes the ontology available in JSON format without upper-level ontology (ULO) terms to give the user community a simpler view of the term hierarchy."
-                }
-            ],
-            "publication": null,
-            "publications": [
-                {
-                    "id": "https://doi.org/10.5281/zenodo.4705219",
-                    "title": "EOSC Co-creation funded project 074: Delivery of a proof of concept for terms4FAIRskills: Technical report"
-                }
-            ],
-            "submission_id": 3,
-            "taxon": null,
-            "version": null
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TAO",
-        "name": "Teleost Anatomy Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 5,
-            "classifiable": true,
-            "date_created": "2013-08-23T01:10:25Z",
-            "date_released": "2015-08-20T00:00:00Z",
-            "description": "TAO is a multispecies anatomy ontology for teleost fishes. Originally seeded from ZFA, but intended to cover terms relevant to other taxa.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/TAO/135/tao.obo",
-            "has_ontology_language": "OBO",
-            "home_page": "https://www.nescent.org/phenoscape/Main_Page",
-            "id": 675,
-            "indexed": true,
-            "max_children": 78,
-            "max_depth": 12,
-            "md5sum": null,
-            "nb_classes": 3428,
-            "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 24,
-            "products": null,
-            "publication": "http://dx.doi.org/10.1093/sysbio/syq013",
-            "publications": null,
-            "submission_id": 135,
-            "taxon": null,
-            "version": "2012-08-10"
-        },
-        "topics": null
-    },
-    {
-        "acronym": "TEMPO",
-        "name": "TempO",
+        "acronym": "SWEET",
+        "name": "Semantic Web for Earth and Environment Technology Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 1,
+            "avg_children": 4,
             "classifiable": true,
-            "date_created": "2023-03-10T02:20:01Z",
-            "date_released": "2023-03-10T02:20:00Z",
-            "description": "TempO is a lightweight ontology for describing temporal validity and efficacy according to a tritemporal data model.",
-            "documentation": "http://purl.org/tempo/html",
+            "date_created": "2017-11-29T02:29:11Z",
+            "date_released": "2017-11-29T02:29:11Z",
+            "description": "The Semantic Web for Earth and Environmental Terminology is a mature foundational ontology that contains over 6000 concepts organized in 200 ontologies represented in OWL. Top level concepts include Representation (math, space, science, time, data), Realm (Ocean, Land Surface, Terrestrial Hydroshere, Atmosphere, etc.), Phenomena (macro-scale ecological and physical), Processes (micro-scale physical, biological, chemical, and mathematical), Human Activities (Decision, Commerce, Jurisdiction, Environmental, Research). Originally developed by NASA Jet Propulsion Labs under Rob Raskin, SWEET is now officially under the governance of the ESIP foundation. ",
+            "documentation": "http://sweet.jpl.nasa.gov/",
             "domain": null,
-            "download_url": "media/ontologies/TEMPO/3/tempo.owl",
+            "download_url": "media/ontologies/SWEET/2/sweet.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/hroptatyr/tempo/",
-            "id": 5724,
+            "home_page": "https://github.com/ESIPFed/sweet/blob/master/README.md",
+            "id": 897,
             "indexed": true,
-            "max_children": 1,
-            "max_depth": 1,
+            "max_children": 51,
+            "max_depth": 11,
             "md5sum": null,
-            "nb_classes": 1,
+            "nb_classes": 4543,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 1,
+            "nb_individuals": 2020,
+            "nb_properties": 327,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 3,
+            "submission_id": 2,
             "taxon": null,
-            "version": "0.1"
+            "version": "3.1.0"
         },
         "topics": null
     },
     {
-        "acronym": "TRIAGE",
-        "name": "Nurse triage",
+        "acronym": "SWO",
+        "name": "Software ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 11,
             "classifiable": true,
-            "date_created": "2015-09-24T01:46:41Z",
-            "date_released": "2015-09-24T00:00:00Z",
-            "description": "An ontology showing the process domain of a triage nurse.",
+            "date_created": "2023-03-06T15:11:42.465757Z",
+            "date_released": "2023-03-06T15:11:42.465746Z",
+            "description": "The Software Ontology (SWO) is a resource for describing software tools, their types, tasks, versions, provenance and associated data. It contains detailed information on licensing and formats as well as software applications themselves, mainly (but not limited) to the bioinformatics community.",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/TRIAGE/2/triage.owl",
+            "domain": "information technology",
+            "download_url": "media/ontologies/SWO/32/swo.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 387,
+            "home_page": "https://github.com/allysonlister/swo",
+            "id": 5708,
             "indexed": true,
-            "max_children": 8,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 32,
+            "max_children": 677,
+            "max_depth": 16,
+            "md5sum": "ecbf9eabd3ccad51ba26c0667bf1258a",
+            "nb_classes": 1971,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 29,
-            "products": null,
+            "nb_individuals": 446,
+            "nb_properties": 161,
+            "products": [
+                {
+                    "id": "swo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/swo.owl"
+                },
+                {
+                    "id": "swo.json",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/swo.json"
+                }
+            ],
             "publication": null,
-            "publications": null,
-            "submission_id": 2,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/25068035",
+                    "title": "The Software Ontology (SWO): a resource for reproducibility in biomedical data analysis, curation and digital preservation"
+                }
+            ],
+            "submission_id": 32,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OMIM",
-        "name": "Online Mendelian Inheritance in Man",
+        "acronym": "OM",
+        "name": "Ontology of units of Measure",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 380,
+            "avg_children": 8,
             "classifiable": true,
-            "date_created": "2017-02-06T21:26:18Z",
-            "date_released": "2017-02-06T21:26:17Z",
-            "description": "Online Mendelian Inheritance in Man, OMIM (TM). McKusick-Nathans Institute for Genetic Medicine, Johns Hopkins University (Baltimore, MD) and National Center for Biotechnology Information, National Library of Medicine",
-            "documentation": "http://www.ncbi.nlm.nih.gov/omim/",
+            "date_created": "2023-03-18T01:15:50Z",
+            "date_released": "2023-03-18T01:15:47Z",
+            "description": "The Ontology of units of Measure (OM) 2.0 models concepts and relations important to scientific research. It has a strong focus on units, quantities, measurements, and dimensions. OM is modelled in OWL 2 - Web Ontology Language. The OM ontology provides classes, instances, and properties that represent the different concepts used for defining and using measures and units. It includes, for instance, common units such as the SI units metre and kilogram, but also units from other systems of units such as the mile or nautical mile. For many application areas it includes more specific units and quantities, such as the unit of the Hubble constant or the quantity vaselife. The following application areas are supported by OM: Geometry; Mechanics; Thermodynamics; Electromagnetism; Fluid mechanics; Chemical physics; Photometry; Radiometry and Radiobiology; Nuclear physics; Astronomy and Astrophysics; Cosmology; Earth science; Meteorology; Material science; Microbiology; Economics; Information technology and Typography.",
+            "documentation": "https://github.com/HajoRijgersberg/OM/blob/master/README.md",
             "domain": null,
-            "download_url": "media/ontologies/OMIM/11/omim.umls",
-            "has_ontology_language": "UMLS",
-            "home_page": "http://www.ncbi.nlm.nih.gov/omim/",
-            "id": 539,
+            "download_url": "media/ontologies/OM/14/om.owl",
+            "has_ontology_language": "OWL",
+            "home_page": "https://github.com/HajoRijgersberg/OM",
+            "id": 5732,
             "indexed": true,
-            "max_children": 6075,
-            "max_depth": 2,
+            "max_children": 205,
+            "max_depth": 5,
             "md5sum": null,
-            "nb_classes": 93042,
+            "nb_classes": 815,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 15,
+            "nb_individuals": 2242,
+            "nb_properties": 23,
             "products": null,
-            "publication": "http://www.ncbi.nlm.nih.gov/omim/",
+            "publication": "https://github.com/HajoRijgersberg/OM/blob/master/README.md#papers-on-om",
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 14,
             "taxon": null,
-            "version": "2016AB"
+            "version": "2.0.39"
         },
         "topics": null
     },
     {
-        "acronym": "OMO",
-        "name": "OBO Metadata Ontology",
+        "acronym": "OGSF",
+        "name": "Ontology of Genetic Susceptibility Factor",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-12T15:00:05.678846Z",
-            "date_released": "2023-06-12T15:00:05.678834Z",
-            "description": "An ontology specifies terms that are used to annotate ontology terms for all OBO ontologies. The ontology was developed as part of Information Artifact Ontology (IAO).",
+            "date_created": "2019-03-18T14:04:24.490878Z",
+            "date_released": "2019-03-18T14:04:24.490843Z",
+            "description": "An application ontology to represent genetic susceptibility to a specific disease, adverse event, or a pathological process.",
             "documentation": null,
-            "domain": "information",
-            "download_url": "media/ontologies/OMO/6/omo.owl",
+            "domain": null,
+            "download_url": "media/ontologies/OGSF/11/ogsf.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/information-artifact-ontology/ontology-metadata",
-            "id": 5864,
+            "home_page": "none",
+            "id": 2366,
             "indexed": true,
-            "max_children": 7,
-            "max_depth": 9,
-            "md5sum": "f6bc193c14328dc6c0f61a9b6e00b0c7",
-            "nb_classes": 28,
+            "max_children": 17,
+            "max_depth": 17,
+            "md5sum": "2f9daddd57d4fdb023da27dd2ae797e7",
+            "nb_classes": 389,
             "nb_inconsistent": 0,
-            "nb_individuals": 19,
-            "nb_properties": 0,
+            "nb_individuals": 57,
+            "nb_properties": 143,
             "products": [
                 {
-                    "id": "omo.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/omo.owl"
+                    "id": "ogsf.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ogsf.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 11,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OMP",
-        "name": "Ontology of Microbial Phenotypes",
+        "acronym": "OMIABIS",
+        "name": "Ontologized MIABIS",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2023-06-12T15:00:17.627764Z",
-            "date_released": "2023-06-12T15:00:17.627747Z",
-            "description": "An ontology of phenotypes covering microbes",
+            "date_created": "2017-12-13T10:41:44.978933Z",
+            "date_released": "2017-12-13T10:41:44.978918Z",
+            "description": "An ontological version of MIABIS (Minimum Information About BIobank data Sharing)",
             "documentation": null,
-            "domain": "phenotype",
-            "download_url": "media/ontologies/OMP/56/omp.owl",
+            "domain": "health",
+            "download_url": "media/ontologies/OMIABIS/2/omiabis.owl",
             "has_ontology_language": "OWL",
-            "home_page": "http://microbialphenotypes.org",
-            "id": 5865,
+            "home_page": "https://github.com/OMIABIS/omiabis-dev",
+            "id": 1044,
             "indexed": true,
-            "max_children": 59,
-            "max_depth": 10,
-            "md5sum": "a8c82c7e392f8386595c72e00c55c7df",
-            "nb_classes": 2309,
+            "max_children": 21,
+            "max_depth": 16,
+            "md5sum": "6458e451e13d1280168f4a0ce627d262",
+            "nb_classes": 427,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 61,
+            "nb_individuals": 15,
+            "nb_properties": 75,
             "products": [
                 {
-                    "id": "omp.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/omp.owl"
-                },
-                {
-                    "id": "omp.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/omp.obo"
+                    "id": "omiabis.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/omiabis.owl"
                 }
             ],
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://doi.org/10.1186/s12866-014-0294-3",
-                    "title": "An ontology for microbial phenotypes"
-                }
-            ],
-            "submission_id": 56,
+            "publications": null,
+            "submission_id": 2,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "OMRSE",
@@ -47252,48 +47243,14 @@
             "submission_id": 10,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "GPML",
-        "name": "Graphical Pathway Markup Language",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 19,
-            "classifiable": true,
-            "date_created": "2017-12-09T02:09:03Z",
-            "date_released": "2017-12-09T02:09:03Z",
-            "description": "GPML is the file format used to store pathway content at WikiPathways. GPML is short for Graphical Pathway Markup Language, and it is a custom XML format compatible with pathway visualization and analysis tools such as Cytoscape and PathVisio.\r\n\r\nGet it as RDFa in HTML: http://vocabularies.wikipathways.org/gpml\r\n\r\nOr get it translated to RDF/XML from either of these URLs:\r\nhttp://getschema.org/rdfaliteextractor?url=http%3A%2F%2Fvocabularies.wikipathways.org%2Fgpml&out=rdf\r\n\r\nor\r\n\r\nhttp://rdf-translator.appspot.com/convert/rdfa/xml/http%3A%2F%2Fvocabularies.wikipathways.org%2Fgpml",
-            "documentation": "http://www.wikipathways.org/index.php/Help:Frequently_Asked_Questions#What_is_GPML.3F",
-            "domain": null,
-            "download_url": "media/ontologies/GPML/331/gpml.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 947,
-            "indexed": true,
-            "max_children": 19,
-            "max_depth": 1,
-            "md5sum": null,
-            "nb_classes": 19,
-            "nb_inconsistent": 0,
-            "nb_individuals": 1,
-            "nb_properties": 25,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 331,
-            "taxon": null,
-            "version": "2013a"
-        },
-        "topics": null
-    },
-    {
         "acronym": "NIF-RTH",
         "name": "NIF - Resource Type Hieracrchy (view)",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 3,
             "classifiable": true,
@@ -47354,358 +47311,401 @@
             "submission_id": 1,
             "taxon": null,
             "version": "1.0"
         },
         "topics": null
     },
     {
-        "acronym": "OGSF",
-        "name": "Ontology of Genetic Susceptibility Factor",
+        "acronym": "OLAM",
+        "name": "Ontology of Laboratory Animal Medicine",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2019-03-18T14:04:24.490878Z",
-            "date_released": "2019-03-18T14:04:24.490843Z",
-            "description": "An application ontology to represent genetic susceptibility to a specific disease, adverse event, or a pathological process.",
+            "date_created": "2021-02-20T02:11:24Z",
+            "date_released": "2021-02-20T02:11:23Z",
+            "description": "The OLAM is community-based biomedical ontology in the field of laboratory animal medicine, with the aim to ontologically represent entities and the relations among the entities related to laboratory animal medicine. ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OGSF/11/ogsf.owl",
+            "download_url": "media/ontologies/OLAM/6/olam.owl",
             "has_ontology_language": "OWL",
-            "home_page": "none",
-            "id": 2366,
+            "home_page": "https://github.com/olam-ontology/OLAM",
+            "id": 4293,
             "indexed": true,
-            "max_children": 17,
-            "max_depth": 17,
-            "md5sum": "2f9daddd57d4fdb023da27dd2ae797e7",
-            "nb_classes": 389,
+            "max_children": 13,
+            "max_depth": 14,
+            "md5sum": null,
+            "nb_classes": 209,
             "nb_inconsistent": 0,
-            "nb_individuals": 57,
-            "nb_properties": 143,
-            "products": [
-                {
-                    "id": "ogsf.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ogsf.owl"
-                }
-            ],
+            "nb_individuals": 25,
+            "nb_properties": 51,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 11,
+            "submission_id": 6,
             "taxon": null,
-            "version": null
+            "version": "Vision Release: 1.0.12"
         },
         "topics": null
     },
     {
-        "acronym": "OHD",
-        "name": "Oral Health and Disease Ontology",
+        "acronym": "OLATDV",
+        "name": "Medaka Developmental Stages",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 23,
             "classifiable": true,
-            "date_created": "2023-07-24T12:41:36.592260Z",
-            "date_released": "2023-07-24T12:41:36.592243Z",
-            "description": "The Oral Health and Disease Ontology is used for representing the diagnosis and treatment of dental maladies.",
+            "date_created": "2020-03-16T17:45:40.732364Z",
+            "date_released": "2020-03-16T17:45:40.732343Z",
+            "description": "Life cycle stages for Medaka",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/OHD/6/ohd.owl",
+            "domain": null,
+            "download_url": "media/ontologies/OLATDV/8/olatdv.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://purl.obolibrary.org/obo/ohd/home",
-            "id": 5960,
+            "home_page": "https://github.com/obophenotype/developmental-stage-ontologies/wiki/OlatDv",
+            "id": 3446,
             "indexed": true,
-            "max_children": 546,
-            "max_depth": 18,
-            "md5sum": "7837481823797878544808866a3029c0",
-            "nb_classes": 2914,
+            "max_children": 46,
+            "max_depth": 2,
+            "md5sum": "7a141a79bb21fea6041bd4ce1a8098db",
+            "nb_classes": 47,
             "nb_inconsistent": 0,
-            "nb_individuals": 87,
-            "nb_properties": 80,
+            "nb_individuals": 0,
+            "nb_properties": 3,
             "products": [
                 {
-                    "id": "ohd.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ohd.owl"
+                    "id": "olatdv.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/olatdv.obo"
                 },
                 {
-                    "id": "ohd/dev/ohd.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ohd/dev/ohd.owl",
-                    "title": "OHD dev"
+                    "id": "olatdv.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/olatdv.owl"
                 }
             ],
             "publication": null,
-            "publications": [
-                {
-                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/32819435",
-                    "title": "Structuring, reuse and analysis of electronic dental data using the Oral Health and Disease Ontology"
-                }
-            ],
-            "submission_id": 6,
+            "publications": null,
+            "submission_id": 8,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OHMI",
-        "name": "Ontology of Host-Microbiome Interactions",
+        "acronym": "OLIVES",
+        "name": "Ontology for Influenza Vaccine Effectiveness Studies",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2019-10-01T03:19:31.020205Z",
-            "date_released": "2019-10-01T03:19:31.020152Z",
-            "description": "The Ontology of Host-Microbiome Interactions aims to ontologically represent and standardize various entities and relations related to microbiomes, microbiome host organisms (e.g., human and mouse), and the interactions between the hosts and microbiomes at different conditions.",
+            "date_created": "2018-10-15T16:23:22Z",
+            "date_released": "2018-10-16T00:00:00Z",
+            "description": "The Ontology for Vaccine Effectiveness Studies (OLIVES) ",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OHMI/16/ohmi.owl",
+            "download_url": "media/ontologies/OLIVES/1/olives.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/ohmi-ontology/ohmi",
-            "id": 2960,
+            "home_page": null,
+            "id": 1940,
             "indexed": true,
-            "max_children": 30,
-            "max_depth": 18,
-            "md5sum": "8a36dcac3d81305a3953e3750f28e97f",
-            "nb_classes": 1025,
+            "max_children": 10,
+            "max_depth": 4,
+            "md5sum": null,
+            "nb_classes": 62,
             "nb_inconsistent": 0,
-            "nb_individuals": 12,
-            "nb_properties": 129,
-            "products": [
-                {
-                    "id": "ohmi.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ohmi.owl"
-                }
-            ],
+            "nb_individuals": 0,
+            "nb_properties": 0,
+            "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 16,
+            "submission_id": 1,
             "taxon": null,
-            "version": null
+            "version": "Inital work-in-progress"
         },
         "topics": null
     },
     {
-        "acronym": "OHPI",
-        "name": "Ontology of Host Pathogen Interactions",
+        "acronym": "OMIM",
+        "name": "Online Mendelian Inheritance in Man",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 6,
+            "avg_children": 380,
             "classifiable": true,
-            "date_created": "2019-10-01T03:21:01.637748Z",
-            "date_released": "2019-10-01T03:21:01.637715Z",
-            "description": "OHPI is a community-driven ontology of host-pathogen interactions (OHPI) and represents the virulence factors (VFs) and how the mutants of VFs in the Victors database become less virulence inside a host organism or host cells. It is developed to represent manually curated HPI knowledge available in the PHIDIAS resource.",
-            "documentation": null,
+            "date_created": "2017-02-06T21:26:18Z",
+            "date_released": "2017-02-06T21:26:17Z",
+            "description": "Online Mendelian Inheritance in Man, OMIM (TM). McKusick-Nathans Institute for Genetic Medicine, Johns Hopkins University (Baltimore, MD) and National Center for Biotechnology Information, National Library of Medicine",
+            "documentation": "http://www.ncbi.nlm.nih.gov/omim/",
             "domain": null,
-            "download_url": "media/ontologies/OHPI/12/ohpi.owl",
+            "download_url": "media/ontologies/OMIM/11/omim.umls",
+            "has_ontology_language": "UMLS",
+            "home_page": "http://www.ncbi.nlm.nih.gov/omim/",
+            "id": 539,
+            "indexed": true,
+            "max_children": 6075,
+            "max_depth": 2,
+            "md5sum": null,
+            "nb_classes": 93042,
+            "nb_inconsistent": 0,
+            "nb_individuals": 0,
+            "nb_properties": 15,
+            "products": null,
+            "publication": "http://www.ncbi.nlm.nih.gov/omim/",
+            "publications": null,
+            "submission_id": 11,
+            "taxon": null,
+            "version": "2016AB"
+        },
+        "topics": null
+    },
+    {
+        "acronym": "OMP",
+        "name": "Ontology of Microbial Phenotypes",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 2,
+            "classifiable": true,
+            "date_created": "2023-06-12T15:00:17.627764Z",
+            "date_released": "2023-06-12T15:00:17.627747Z",
+            "description": "An ontology of phenotypes covering microbes",
+            "documentation": null,
+            "domain": "phenotype",
+            "download_url": "media/ontologies/OMP/56/omp.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/OHPI/ohpi",
-            "id": 2961,
+            "home_page": "http://microbialphenotypes.org",
+            "id": 5865,
             "indexed": true,
-            "max_children": 1603,
-            "max_depth": 19,
-            "md5sum": "f2179922c693fdacfe4c653e731812c5",
-            "nb_classes": 11292,
+            "max_children": 59,
+            "max_depth": 10,
+            "md5sum": "a8c82c7e392f8386595c72e00c55c7df",
+            "nb_classes": 2309,
             "nb_inconsistent": 0,
-            "nb_individuals": 295,
-            "nb_properties": 320,
+            "nb_individuals": 0,
+            "nb_properties": 61,
             "products": [
                 {
-                    "id": "ohpi.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/ohpi.owl"
+                    "id": "omp.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/omp.owl"
+                },
+                {
+                    "id": "omp.obo",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/omp.obo"
                 }
             ],
             "publication": null,
-            "publications": null,
-            "submission_id": 12,
+            "publications": [
+                {
+                    "id": "https://doi.org/10.1186/s12866-014-0294-3",
+                    "title": "An ontology for microbial phenotypes"
+                }
+            ],
+            "submission_id": 56,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OLAM",
-        "name": "Ontology of Laboratory Animal Medicine",
+        "acronym": "GPML",
+        "name": "Graphical Pathway Markup Language",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 2,
+            "avg_children": 19,
             "classifiable": true,
-            "date_created": "2021-02-20T02:11:24Z",
-            "date_released": "2021-02-20T02:11:23Z",
-            "description": "The OLAM is community-based biomedical ontology in the field of laboratory animal medicine, with the aim to ontologically represent entities and the relations among the entities related to laboratory animal medicine. ",
-            "documentation": null,
+            "date_created": "2017-12-09T02:09:03Z",
+            "date_released": "2017-12-09T02:09:03Z",
+            "description": "GPML is the file format used to store pathway content at WikiPathways. GPML is short for Graphical Pathway Markup Language, and it is a custom XML format compatible with pathway visualization and analysis tools such as Cytoscape and PathVisio.\r\n\r\nGet it as RDFa in HTML: http://vocabularies.wikipathways.org/gpml\r\n\r\nOr get it translated to RDF/XML from either of these URLs:\r\nhttp://getschema.org/rdfaliteextractor?url=http%3A%2F%2Fvocabularies.wikipathways.org%2Fgpml&out=rdf\r\n\r\nor\r\n\r\nhttp://rdf-translator.appspot.com/convert/rdfa/xml/http%3A%2F%2Fvocabularies.wikipathways.org%2Fgpml",
+            "documentation": "http://www.wikipathways.org/index.php/Help:Frequently_Asked_Questions#What_is_GPML.3F",
             "domain": null,
-            "download_url": "media/ontologies/OLAM/6/olam.owl",
+            "download_url": "media/ontologies/GPML/331/gpml.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/olam-ontology/OLAM",
-            "id": 4293,
+            "home_page": null,
+            "id": 947,
             "indexed": true,
-            "max_children": 13,
-            "max_depth": 14,
+            "max_children": 19,
+            "max_depth": 1,
             "md5sum": null,
-            "nb_classes": 209,
+            "nb_classes": 19,
             "nb_inconsistent": 0,
-            "nb_individuals": 25,
-            "nb_properties": 51,
+            "nb_individuals": 1,
+            "nb_properties": 25,
             "products": null,
             "publication": null,
             "publications": null,
-            "submission_id": 6,
+            "submission_id": 331,
             "taxon": null,
-            "version": "Vision Release: 1.0.12"
+            "version": "2013a"
         },
         "topics": null
     },
     {
-        "acronym": "OLATDV",
-        "name": "Medaka Developmental Stages",
+        "acronym": "OMO",
+        "name": "OBO Metadata Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 23,
+            "avg_children": 2,
             "classifiable": true,
-            "date_created": "2020-03-16T17:45:40.732364Z",
-            "date_released": "2020-03-16T17:45:40.732343Z",
-            "description": "Life cycle stages for Medaka",
+            "date_created": "2023-07-31T13:21:01.620434Z",
+            "date_released": "2023-07-31T13:21:01.620419Z",
+            "description": "An ontology specifies terms that are used to annotate ontology terms for all OBO ontologies. The ontology was developed as part of Information Artifact Ontology (IAO).",
             "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/OLATDV/8/olatdv.owl",
+            "domain": "information",
+            "download_url": "media/ontologies/OMO/7/omo.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/obophenotype/developmental-stage-ontologies/wiki/OlatDv",
-            "id": 3446,
+            "home_page": "https://github.com/information-artifact-ontology/ontology-metadata",
+            "id": 5973,
             "indexed": true,
-            "max_children": 46,
-            "max_depth": 2,
-            "md5sum": "7a141a79bb21fea6041bd4ce1a8098db",
-            "nb_classes": 47,
+            "max_children": 7,
+            "max_depth": 9,
+            "md5sum": "6f07a38175373564a974e1f0bfd64531",
+            "nb_classes": 28,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 3,
+            "nb_individuals": 19,
+            "nb_properties": 0,
             "products": [
                 {
-                    "id": "olatdv.obo",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/olatdv.obo"
-                },
-                {
-                    "id": "olatdv.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/olatdv.owl"
+                    "id": "omo.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/omo.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 8,
+            "submission_id": 7,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OLIVES",
-        "name": "Ontology for Influenza Vaccine Effectiveness Studies",
+        "acronym": "OHPI",
+        "name": "Ontology of Host Pathogen Interactions",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 3,
+            "avg_children": 6,
             "classifiable": true,
-            "date_created": "2018-10-15T16:23:22Z",
-            "date_released": "2018-10-16T00:00:00Z",
-            "description": "The Ontology for Vaccine Effectiveness Studies (OLIVES) ",
+            "date_created": "2019-10-01T03:21:01.637748Z",
+            "date_released": "2019-10-01T03:21:01.637715Z",
+            "description": "OHPI is a community-driven ontology of host-pathogen interactions (OHPI) and represents the virulence factors (VFs) and how the mutants of VFs in the Victors database become less virulence inside a host organism or host cells. It is developed to represent manually curated HPI knowledge available in the PHIDIAS resource.",
             "documentation": null,
             "domain": null,
-            "download_url": "media/ontologies/OLIVES/1/olives.owl",
+            "download_url": "media/ontologies/OHPI/12/ohpi.owl",
             "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 1940,
+            "home_page": "https://github.com/OHPI/ohpi",
+            "id": 2961,
             "indexed": true,
-            "max_children": 10,
-            "max_depth": 4,
-            "md5sum": null,
-            "nb_classes": 62,
+            "max_children": 1603,
+            "max_depth": 19,
+            "md5sum": "f2179922c693fdacfe4c653e731812c5",
+            "nb_classes": 11292,
             "nb_inconsistent": 0,
-            "nb_individuals": 0,
-            "nb_properties": 0,
-            "products": null,
+            "nb_individuals": 295,
+            "nb_properties": 320,
+            "products": [
+                {
+                    "id": "ohpi.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ohpi.owl"
+                }
+            ],
             "publication": null,
             "publications": null,
-            "submission_id": 1,
+            "submission_id": 12,
             "taxon": null,
-            "version": "Inital work-in-progress"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OM",
-        "name": "Ontology of units of Measure",
+        "acronym": "OHD",
+        "name": "Oral Health and Disease Ontology",
         "species": null,
         "status": "Classified",
         "submission": {
-            "avg_children": 8,
+            "avg_children": 3,
             "classifiable": true,
-            "date_created": "2023-03-18T01:15:50Z",
-            "date_released": "2023-03-18T01:15:47Z",
-            "description": "The Ontology of units of Measure (OM) 2.0 models concepts and relations important to scientific research. It has a strong focus on units, quantities, measurements, and dimensions. OM is modelled in OWL 2 - Web Ontology Language. The OM ontology provides classes, instances, and properties that represent the different concepts used for defining and using measures and units. It includes, for instance, common units such as the SI units metre and kilogram, but also units from other systems of units such as the mile or nautical mile. For many application areas it includes more specific units and quantities, such as the unit of the Hubble constant or the quantity vaselife. The following application areas are supported by OM: Geometry; Mechanics; Thermodynamics; Electromagnetism; Fluid mechanics; Chemical physics; Photometry; Radiometry and Radiobiology; Nuclear physics; Astronomy and Astrophysics; Cosmology; Earth science; Meteorology; Material science; Microbiology; Economics; Information technology and Typography.",
-            "documentation": "https://github.com/HajoRijgersberg/OM/blob/master/README.md",
-            "domain": null,
-            "download_url": "media/ontologies/OM/14/om.owl",
+            "date_created": "2023-07-24T12:41:36.592260Z",
+            "date_released": "2023-07-24T12:41:36.592243Z",
+            "description": "The Oral Health and Disease Ontology is used for representing the diagnosis and treatment of dental maladies.",
+            "documentation": null,
+            "domain": "health",
+            "download_url": "media/ontologies/OHD/6/ohd.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/HajoRijgersberg/OM",
-            "id": 5732,
+            "home_page": "https://purl.obolibrary.org/obo/ohd/home",
+            "id": 5960,
             "indexed": true,
-            "max_children": 205,
-            "max_depth": 5,
-            "md5sum": null,
-            "nb_classes": 815,
+            "max_children": 546,
+            "max_depth": 18,
+            "md5sum": "7837481823797878544808866a3029c0",
+            "nb_classes": 2914,
             "nb_inconsistent": 0,
-            "nb_individuals": 2242,
-            "nb_properties": 23,
-            "products": null,
-            "publication": "https://github.com/HajoRijgersberg/OM/blob/master/README.md#papers-on-om",
-            "publications": null,
-            "submission_id": 14,
+            "nb_individuals": 87,
+            "nb_properties": 80,
+            "products": [
+                {
+                    "id": "ohd.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ohd.owl"
+                },
+                {
+                    "id": "ohd/dev/ohd.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ohd/dev/ohd.owl",
+                    "title": "OHD dev"
+                }
+            ],
+            "publication": null,
+            "publications": [
+                {
+                    "id": "https://www.ncbi.nlm.nih.gov/pubmed/32819435",
+                    "title": "Structuring, reuse and analysis of electronic dental data using the Oral Health and Disease Ontology"
+                }
+            ],
+            "submission_id": 6,
             "taxon": null,
-            "version": "2.0.39"
+            "version": null
         },
         "topics": null
     },
     {
-        "acronym": "OMIABIS",
-        "name": "Ontologized MIABIS",
+        "acronym": "OHMI",
+        "name": "Ontology of Host-Microbiome Interactions",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 2,
             "classifiable": true,
-            "date_created": "2017-12-13T10:41:44.978933Z",
-            "date_released": "2017-12-13T10:41:44.978918Z",
-            "description": "An ontological version of MIABIS (Minimum Information About BIobank data Sharing)",
+            "date_created": "2019-10-01T03:19:31.020205Z",
+            "date_released": "2019-10-01T03:19:31.020152Z",
+            "description": "The Ontology of Host-Microbiome Interactions aims to ontologically represent and standardize various entities and relations related to microbiomes, microbiome host organisms (e.g., human and mouse), and the interactions between the hosts and microbiomes at different conditions.",
             "documentation": null,
-            "domain": "health",
-            "download_url": "media/ontologies/OMIABIS/2/omiabis.owl",
+            "domain": null,
+            "download_url": "media/ontologies/OHMI/16/ohmi.owl",
             "has_ontology_language": "OWL",
-            "home_page": "https://github.com/OMIABIS/omiabis-dev",
-            "id": 1044,
+            "home_page": "https://github.com/ohmi-ontology/ohmi",
+            "id": 2960,
             "indexed": true,
-            "max_children": 21,
-            "max_depth": 16,
-            "md5sum": "6458e451e13d1280168f4a0ce627d262",
-            "nb_classes": 427,
+            "max_children": 30,
+            "max_depth": 18,
+            "md5sum": "8a36dcac3d81305a3953e3750f28e97f",
+            "nb_classes": 1025,
             "nb_inconsistent": 0,
-            "nb_individuals": 15,
-            "nb_properties": 75,
+            "nb_individuals": 12,
+            "nb_properties": 129,
             "products": [
                 {
-                    "id": "omiabis.owl",
-                    "ontology_purl": "http://purl.obolibrary.org/obo/omiabis.owl"
+                    "id": "ohmi.owl",
+                    "ontology_purl": "http://purl.obolibrary.org/obo/ohmi.owl"
                 }
             ],
             "publication": null,
             "publications": null,
-            "submission_id": 2,
+            "submission_id": 16,
             "taxon": null,
             "version": null
         },
         "topics": null
     },
     {
         "acronym": "Spitali",
@@ -47742,48 +47742,14 @@
             "version": "se di"
         },
         "topics": [
             "Spitali"
         ]
     },
     {
-        "acronym": "VEO",
-        "name": "Visualized Emotion Ontology",
-        "species": null,
-        "status": "Classified",
-        "submission": {
-            "avg_children": 3,
-            "classifiable": true,
-            "date_created": "2017-12-27T16:20:48Z",
-            "date_released": "2017-12-27T00:00:00Z",
-            "description": "Visual Emotion Ontology (VEO) is an ontology that links visualizations with specific emotions from the Ortony, Clore, and Collins' description of human emotions.",
-            "documentation": null,
-            "domain": null,
-            "download_url": "media/ontologies/VEO/1/veo.owl",
-            "has_ontology_language": "OWL",
-            "home_page": null,
-            "id": 1130,
-            "indexed": true,
-            "max_children": 35,
-            "max_depth": 15,
-            "md5sum": null,
-            "nb_classes": 1012,
-            "nb_inconsistent": 0,
-            "nb_individuals": 44,
-            "nb_properties": 39,
-            "products": null,
-            "publication": null,
-            "publications": null,
-            "submission_id": 1,
-            "taxon": null,
-            "version": "0.5"
-        },
-        "topics": null
-    },
-    {
         "acronym": "VFB_DRIVERS",
         "name": "vfb_drivers",
         "species": null,
         "status": "Classified",
         "submission": {
             "avg_children": 1420,
             "classifiable": true,
@@ -48094,9 +48060,43 @@
             "publication": null,
             "publications": null,
             "submission_id": 4,
             "taxon": null,
             "version": "3.0"
         },
         "topics": null
+    },
+    {
+        "acronym": "VEO",
+        "name": "Visualized Emotion Ontology",
+        "species": null,
+        "status": "Classified",
+        "submission": {
+            "avg_children": 3,
+            "classifiable": true,
+            "date_created": "2017-12-27T16:20:48Z",
+            "date_released": "2017-12-27T00:00:00Z",
+            "description": "Visual Emotion Ontology (VEO) is an ontology that links visualizations with specific emotions from the Ortony, Clore, and Collins' description of human emotions.",
+            "documentation": null,
+            "domain": null,
+            "download_url": "media/ontologies/VEO/1/veo.owl",
+            "has_ontology_language": "OWL",
+            "home_page": null,
+            "id": 1130,
+            "indexed": true,
+            "max_children": 35,
+            "max_depth": 15,
+            "md5sum": null,
+            "nb_classes": 1012,
+            "nb_inconsistent": 0,
+            "nb_individuals": 44,
+            "nb_properties": 39,
+            "products": null,
+            "publication": null,
+            "publications": null,
+            "submission_id": 1,
+            "taxon": null,
+            "version": "0.5"
+        },
+        "topics": null
     }
 ]
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/agroportal/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/agroportal/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/agroportal/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/agroportal/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/agroportal/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/agroportal/raw.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/bartoc/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/bartoc/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/bartoc/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/bartoc/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/biocontext/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/biocontext/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/biocontext/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/biocontext/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/biocontext/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/biocontext/raw.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/biolink/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/biolink/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/biolink/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/biolink/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/biolink/raw.yaml` & `bioregistry-0.9.91/src/bioregistry/data/external/biolink/raw.yaml`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/bioportal/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/bioportal/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/bioportal/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/bioportal/processed.json`

 * *Files 0% similar despite different names*

#### Pretty-printed

 * *Similarity: 0.9998464263294773%*

 * *Differences: {"'DOID'": "{'version': '2023-07-20'}",*

 * * "'GO-PLUS'": "{'version': '2023-07-27'}",*

 * * "'NPOKB'": "{'version': '2023-07-30'}",*

 * * "'VFB_DRIVERS'": "{'version': '2023-07-31'}"}*

```diff
@@ -2422,15 +2422,15 @@
             "name": "Lynn Schriml"
         },
         "description": "Creating a comprehensive classification of human diseases organized by etiology.",
         "homepage": "https://disease-ontology.org/",
         "name": "Human Disease Ontology",
         "prefix": "DOID",
         "publication": "https://disease-ontology.org/community/publications",
-        "version": "2023-07-24"
+        "version": "2023-07-20"
     },
     "DOREMUS-KEYS": {
         "contact": {
             "email": "nlorimer@stanford.edu",
             "name": "Nancy Lorimer"
         },
         "description": "The sum of relations, melodic and harmonic, existing between the tones of a scale or musical system.",
@@ -3963,15 +3963,15 @@
             "name": "GO Mailing List"
         },
         "description": "This is the fully axiomatised version of the GO. It includes cross ontology relationships (axioms) and imports additional required ontologies including ChEBI, cell ontology and Uberon. It also includes a complete set of relationship types including some not in go.obo/go.owl. This version is only available in OWL.",
         "homepage": "http://geneontology.org",
         "name": "go-plus",
         "prefix": "GO-PLUS",
         "publication": "http://www.geneontology.org/page/publications",
-        "version": "2023-06-11"
+        "version": "2023-07-27"
     },
     "GPML": {
         "contact": {
             "email": "wikipathways-devel@googlegroups.com",
             "name": "WikiPathways Team"
         },
         "description": "GPML is the file format used to store pathway content at WikiPathways. GPML is short for Graphical Pathway Markup Language, and it is a custom XML format compatible with pathway visualization and analysis tools such as Cytoscape and PathVisio. Get it as RDFa in HTML: http://vocabularies.wikipathways.org/gpml Or get it translated to RDF/XML from either of these URLs: http://getschema.org/rdfaliteextractor?url=http%3A%2F%2Fvocabularies.wikipathways.org%2Fgpml&out=rdf or http://rdf-translator.appspot.com/convert/rdfa/xml/http%3A%2F%2Fvocabularies.wikipathways.org%2Fgpml",
@@ -7098,15 +7098,15 @@
             "email": "tgbugs@gmail.com",
             "name": "Tom Gillespie"
         },
         "description": "An ontology of neuron types based on the phenotypic dimensions of cells that can be measure experimentally.",
         "homepage": "https://github.com/SciCrunch/NIF-Ontology",
         "name": "Neuron Phenotype Ontology",
         "prefix": "NPOKB",
-        "version": "2023-07-26"
+        "version": "2023-07-30"
     },
     "NRO": {
         "contact": {
             "email": "chenjiafeng@sr.gxmu.edu.cn",
             "name": "ChenJiafeng"
         },
         "description": "The neural reprogramming ontology encompasses the domain of neural regenerative cell reprogramming knowledge. As research progresses, researchers and specialists in the field of reprogramming could continue to expand the bounds of this foundation, so adding to its depth.",
@@ -10595,15 +10595,15 @@
         "contact": {
             "email": "cp390@cam.ac.uk",
             "name": "Clare Pilgrim"
         },
         "description": "An ontology of Drosophila melanogaster drivers and expression patterns.",
         "name": "vfb_drivers",
         "prefix": "VFB_DRIVERS",
-        "version": "2023-07-24"
+        "version": "2023-07-31"
     },
     "VHOG": {
         "contact": {
             "email": "bgee@isb-sib.ch",
             "name": "Bgee team"
         },
         "description": "vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. The mapping to species-specific anatomical ontologies is provided as a separated file: http://bgee.unil.ch/download/organ_association_vHOG.txt This mapping represents homology-strict relationships, in the sense of HOM:0000007 \"historical homology\". Each mapping has been manually reviewed, and we provide confidence codes and references when available.",
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/bioportal/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/bioportal/raw.json`

 * *Files 0% similar despite different names*

#### Pretty-printed

 * *Similarity: 0.99983164983165%*

 * *Differences: {'319': "{'version': '2023-07-20'}",*

 * * '67': "{'version': '2023-07-31'}",*

 * * '793': "{'version': '2023-07-30'}",*

 * * '940': "{'version': '2023-07-27'}"}*

```diff
@@ -3689,15 +3689,15 @@
             "single_class": "https://data.bioontology.org/ontologies/VFB_DRIVERS/classes/{class_id}",
             "submissions": "https://data.bioontology.org/ontologies/VFB_DRIVERS/submissions",
             "ui": "http://bioportal.bioontology.org/ontologies/VFB_DRIVERS",
             "views": "https://data.bioontology.org/ontologies/VFB_DRIVERS/views"
         },
         "name": "vfb_drivers",
         "ontologyType": "https://data.bioontology.org/ontology_types/ONTOLOGY",
-        "version": "2023-07-24"
+        "version": "2023-07-31"
     },
     {
         "@id": "https://data.bioontology.org/ontologies/AURA",
         "@type": "http://data.bioontology.org/metadata/Ontology",
         "acronym": "AURA",
         "administeredBy": [
             "https://data.bioontology.org/users/miwessel"
@@ -17469,15 +17469,15 @@
             "submissions": "https://data.bioontology.org/ontologies/DOID/submissions",
             "ui": "http://bioportal.bioontology.org/ontologies/DOID",
             "views": "https://data.bioontology.org/ontologies/DOID/views"
         },
         "name": "Human Disease Ontology",
         "ontologyType": "https://data.bioontology.org/ontology_types/ONTOLOGY",
         "publication": "https://disease-ontology.org/community/publications",
-        "version": "2023-07-24"
+        "version": "2023-07-20"
     },
     {
         "@id": "https://data.bioontology.org/ontologies/OM",
         "@type": "http://data.bioontology.org/metadata/Ontology",
         "acronym": "OM",
         "administeredBy": [
             "https://data.bioontology.org/users/WFBR+FI"
@@ -43438,15 +43438,15 @@
             "single_class": "https://data.bioontology.org/ontologies/NPOKB/classes/{class_id}",
             "submissions": "https://data.bioontology.org/ontologies/NPOKB/submissions",
             "ui": "http://bioportal.bioontology.org/ontologies/NPOKB",
             "views": "https://data.bioontology.org/ontologies/NPOKB/views"
         },
         "name": "Neuron Phenotype Ontology",
         "ontologyType": "https://data.bioontology.org/ontology_types/ONTOLOGY",
-        "version": "2023-07-26"
+        "version": "2023-07-30"
     },
     {
         "@id": "https://data.bioontology.org/ontologies/UPA",
         "@type": "http://data.bioontology.org/metadata/Ontology",
         "acronym": "UPA",
         "administeredBy": [
             "https://data.bioontology.org/users/vendetti"
@@ -51468,15 +51468,15 @@
             "submissions": "https://data.bioontology.org/ontologies/GO-PLUS/submissions",
             "ui": "http://bioportal.bioontology.org/ontologies/GO-PLUS",
             "views": "https://data.bioontology.org/ontologies/GO-PLUS/views"
         },
         "name": "go-plus",
         "ontologyType": "https://data.bioontology.org/ontology_types/ONTOLOGY",
         "publication": "http://www.geneontology.org/page/publications",
-        "version": "2023-06-11"
+        "version": "2023-07-27"
     },
     {
         "@id": "https://data.bioontology.org/ontologies/ICPS",
         "@type": "http://data.bioontology.org/metadata/Ontology",
         "acronym": "ICPS",
         "administeredBy": [
             "https://data.bioontology.org/users/ocorcho"
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/cellosaurus/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/cellosaurus/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/cellosaurus/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/cellosaurus/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/cellosaurus/raw.txt` & `bioregistry-0.9.91/src/bioregistry/data/external/cellosaurus/raw.txt`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/cheminf/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/cheminf/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/cropoct/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/cropoct/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/cropoct/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/cropoct/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/cropoct/raw.yaml` & `bioregistry-0.9.91/src/bioregistry/data/external/cropoct/raw.yaml`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ecoportal/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/ecoportal/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ecoportal/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/ecoportal/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ecoportal/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/ecoportal/raw.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/edam/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/edam/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/edam/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/edam/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/fairsharing/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/fairsharing/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/fairsharing/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/fairsharing/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/go/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/go/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/go/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/go/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/go/raw.yml` & `bioregistry-0.9.91/src/bioregistry/data/external/go/raw.yml`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/hl7/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/hl7/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/miriam/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/miriam/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/miriam/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/miriam/raw.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/n2t/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/n2t/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/n2t/raw.yml` & `bioregistry-0.9.91/src/bioregistry/data/external/n2t/raw.yml`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ncbi/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/ncbi/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ncbi/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/ncbi/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ncbi/raw.html` & `bioregistry-0.9.91/src/bioregistry/data/external/ncbi/raw.html`

 * *Files 0% similar despite different names*

#### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ncbi/raw.html` & `bioregistry-0.9.91/src/bioregistry/data/external/ncbi/raw.html`

```diff
@@ -30,15 +30,15 @@
     <meta name="modified" content="2021-10-25T17:46:25Z"/>
     <meta xmlns:ncbi-portal="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="cms-edit-aux-url" content="http://cms.ncbi.nlm.nih.gov/node//edit"/>
     <!-- Page headcontent end -->
     <!-- PageFixtures resources begin -->
     <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4208398/css/4207974/4206132.css" xml:base="http://127.0.0.1/sites/static/header_footer"/>
     <!-- PageFixtures resources end -->
     <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"/>
-    <meta name="ncbi_phid" content="CE8874B64C5911B100000000039102CD.m_6"/>
+    <meta name="ncbi_phid" content="CE8C35DD4C854E6100000000013200F8.m_6"/>
     <meta name="referrer" content="origin-when-cross-origin"/>
     <link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4181609/css/4121862/3974050/3917732/251717/4108189/14534/45193/3534283/4128070/3407145/4005757/4062871.css"/>
     <link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4181609/css/3529741/3529739.css" media="print"/>
   </head>
   <body class=" col2  custom-page">
     <div class="grid">
       <div class="col twelve_col nomargin shadow">
@@ -3248,17 +3248,17 @@
             <div id="NCBIFooter_dynamic">
               <div class="breadcrumbs">
                 You are here:
                 <span id="breadcrumb_text">
                   <a href="/guide/">NCBI</a>
                 </span>
               </div>
-              <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/default.asp?Time=2023-07-29T21:07:58-04:00&amp;Snapshot=%2Fprojects%2Fstaticsites%2Fgenbank%2Fgenbank@2.20&amp;Host=portal101&amp;ncbi_phid=CE8874B64C5911B100000000039102CD&amp;ncbi_session=CE8874B64C5B7ED1_0913SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgenbank%2Fcollab%2Fdb_xref%2F&amp;Ncbi_App=genbank&amp;Page=custom-page&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
+              <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/default.asp?Time=2023-07-31T21:08:57-04:00&amp;Snapshot=%2Fprojects%2Fstaticsites%2Fgenbank%2Fgenbank@2.20&amp;Host=portal105&amp;ncbi_phid=CE8C35DD4C854E6100000000013200F8&amp;ncbi_session=CE8C35DD4C85B291_0306SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgenbank%2Fcollab%2Fdb_xref%2F&amp;Ncbi_App=genbank&amp;Page=custom-page&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
               <noscript>
-                <img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=genbank&amp;ncbi_db=&amp;ncbi_pdid=custom-page&amp;ncbi_phid=CE8874B64C5911B100000000039102CD"/>
+                <img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=genbank&amp;ncbi_db=&amp;ncbi_pdid=custom-page&amp;ncbi_phid=CE8C35DD4C854E6100000000013200F8"/>
               </noscript>
             </div>
             <div xmlns:xi="http://www.w3.org/2001/XInclude">
               <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer">
                 <section class="icon-section">
                   <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
                   <div class="grid-container container">
@@ -3465,12 +3465,12 @@
         <span class="PAFAppResources"/>
       </div>
       <!-- /.twelve_col -->
     </div>
     <!-- /.grid -->
     <!-- usually for JS scripts at page bottom -->
     <span class="pagefixtures"/>
-    <!-- CE8874B64C5B7ED1_0913SID /projects/staticsites/genbank/genbank@2.20 portal101 v4.1.r643667 Fri, Jan 14 2022 01:18:35 -->
-    <span id="portal-csrf-token" style="display:none" data-token="CE8874B64C5B7ED1_0913SID"/>
+    <!-- CE8C35DD4C85B291_0306SID /projects/staticsites/genbank/genbank@2.20 portal105 v4.1.r643667 Fri, Jan 14 2022 01:18:35 -->
+    <span id="portal-csrf-token" style="display:none" data-token="CE8C35DD4C85B291_0306SID"/>
     <script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4181609/js/3879255/4121861/1490097/4087685.js" snapshot="genbank"/>
   </body>
 </html>
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/obofoundry/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/obofoundry/processed.json`

 * *Files 0% similar despite different names*

#### Pretty-printed

 * *Similarity: 0.9998046875%*

 * *Differences: {"'hancestro'": "{'contact': 'dwelter.ontologist@gmail.com', 'homepage': "*

 * *                "'https://ebispot.github.io/hancestro/', 'repository': "*

 * *                "'https://github.com/EBISPOT/hancestro'}"}*

```diff
@@ -2030,34 +2030,34 @@
         "contact.orcid": "0000-0001-6512-3296",
         "deprecated": true,
         "domain": "organisms",
         "homepage": "http://www.mesquiteproject.org/ontology/Habronattus/index.html",
         "name": "Habronattus courtship"
     },
     "hancestro": {
-        "contact": "danielle.welter@uni.lu",
+        "contact": "dwelter.ontologist@gmail.com",
         "contact.github": "daniwelter",
         "contact.label": "Danielle Welter",
         "contact.orcid": "0000-0003-1058-2668",
         "deprecated": false,
         "description": "The Human Ancestry Ontology (HANCESTRO) provides a systematic description of the ancestry concepts used in the NHGRI-EBI Catalog of published genome-wide association studies.",
         "domain": "organisms",
         "download.owl": "http://purl.obolibrary.org/obo/hancestro.owl",
-        "homepage": "https://github.com/EBISPOT/ancestro",
+        "homepage": "https://ebispot.github.io/hancestro/",
         "license": "CC BY 4.0",
         "license.url": "https://creativecommons.org/licenses/by/4.0/",
         "name": "Human Ancestry Ontology",
         "preferredPrefix": "HANCESTRO",
         "publications": [
             {
                 "id": "https://www.ncbi.nlm.nih.gov/pubmed/29448949",
                 "title": "A standardized framework for representation of ancestry data in genomics studies, with application to the NHGRI-EBI GWAS Catalog"
             }
         ],
-        "repository": "https://github.com/EBISPOT/ancestro"
+        "repository": "https://github.com/EBISPOT/hancestro"
     },
     "hao": {
         "contact": "diapriid@gmail.com",
         "contact.github": "mjy",
         "contact.label": "Matt Yoder",
         "contact.orcid": "0000-0002-5640-5491",
         "deprecated": false,
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/obofoundry/raw.yaml` & `bioregistry-0.9.91/src/bioregistry/data/external/obofoundry/raw.yaml`

 * *Files 0% similar despite different names*

```diff
@@ -2891,50 +2891,50 @@
     - description: Glutamine scyllo-inositol transaminase reaction and associated
         GO term
       url: https://www.rhea-db.org/rhea/22920
     type: mapping
     user: https://www.rhea-db.org
 - activity_status: active
   contact:
-    email: danielle.welter@uni.lu
+    email: dwelter.ontologist@gmail.com
     github: daniwelter
     label: Danielle Welter
     orcid: 0000-0003-1058-2668
   description: The Human Ancestry Ontology (HANCESTRO) provides a systematic description
     of the ancestry concepts used in the NHGRI-EBI Catalog of published genome-wide
     association studies.
   domain: organisms
-  homepage: https://github.com/EBISPOT/ancestro
+  homepage: https://ebispot.github.io/hancestro/
   id: hancestro
   layout: ontology_detail
   license:
     label: CC BY 4.0
     logo: http://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png
     url: https://creativecommons.org/licenses/by/4.0/
   ontology_purl: http://purl.obolibrary.org/obo/hancestro.owl
   preferredPrefix: HANCESTRO
   products:
-  - description: The full version of HANCESTRO in OWL format
+  - description: The full version of HANCESTRO in OWL format, with BFO upper hierarchy
+      for easier integration with other ontologies
     id: hancestro.owl
     ontology_purl: http://purl.obolibrary.org/obo/hancestro.owl
     title: HANCESTRO
-  - description: Version of HANCESTRO with BFO upper hierarchy for easier integration
-      with other ontologies
-    id: hancestro_bfo.owl
-    ontology_purl: http://purl.obolibrary.org/obo/hancestro_bfo.owl
-    title: HANCESTRO BFO
+  - description: Base version of HANCESTRO
+    id: hancestro-base.owl
+    ontology_purl: http://purl.obolibrary.org/obo/hancestro-base.owl
+    title: HANCESTRO Base
   publications:
   - id: https://www.ncbi.nlm.nih.gov/pubmed/29448949
     title: A standardized framework for representation of ancestry data in genomics
       studies, with application to the NHGRI-EBI GWAS Catalog
-  repository: https://github.com/EBISPOT/ancestro
+  repository: https://github.com/EBISPOT/hancestro
   tags:
   - ancestry
   title: Human Ancestry Ontology
-  tracker: https://github.com/EBISPOT/ancestro/issues
+  tracker: https://github.com/EBISPOT/hancestro/issues
   usages:
   - description: The Experimental Factor Ontology (EFO) provides a systematic description
       of many experimental variables available in EBI databases, and for external
       projects such as the NHGRI GWAS catalogue. It combines parts of several biological
       ontologies, such as anatomy, disease and chemical compounds.
     examples:
     - description: Population category defined using ancestry informative markers
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ols/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/ols/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ols/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/ols/raw.json`

 * *Files 2% similar despite different names*

#### Pretty-printed

 * *Similarity: 0.9947916666666666%*

 * *Differences: {"'_embedded'": "{'ontologies': {0: {'loaded': '2023-07-28T10:30:58.619468840', 'updated': "*

 * *                "'2023-07-28T10:30:58.619468840'}, 1: {'loaded': '2023-07-28T10:56:09.634487950', "*

 * *                "'updated': '2023-07-28T10:56:09.634487950'}, 2: {'loaded': "*

 * *                "'2023-07-28T10:56:29.854552211', 'updated': '2023-07-28T10:56:29.854552211'}, 3: "*

 * *                "{'loaded': '2023-07-28T11:10:19.364754309', 'updated': "*

 * *                "'2023-07-28T11:10:19.364754309'}, 4: {'loaded': '2 […]*

```diff
@@ -48,22 +48,22 @@
                 "languages": [
                     "de",
                     "en",
                     "en-us",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:04.354013340",
+                "loaded": "2023-07-28T10:30:58.619468840",
                 "message": "",
                 "numberOfIndividuals": 2,
                 "numberOfProperties": 186,
                 "numberOfTerms": 1963,
                 "ontologyId": "ado",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:04.354013340",
+                "updated": "2023-07-28T10:30:58.619468840",
                 "version": "2.0.0"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/aeo/individuals"
                     },
@@ -106,22 +106,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:55.760422864",
+                "loaded": "2023-07-28T10:56:09.634487950",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 26,
                 "numberOfTerms": 250,
                 "ontologyId": "aeo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:55.760422864",
+                "updated": "2023-07-28T10:56:09.634487950",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/aero/individuals"
                     },
@@ -164,22 +164,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:17.874306299",
+                "loaded": "2023-07-28T10:56:29.854552211",
                 "message": "",
                 "numberOfIndividuals": 29,
                 "numberOfProperties": 187,
                 "numberOfTerms": 539,
                 "ontologyId": "aero",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:17.874306299",
+                "updated": "2023-07-28T10:56:29.854552211",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/afo/individuals"
                     },
@@ -231,22 +231,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:30.317607141",
+                "loaded": "2023-07-28T11:10:19.364754309",
                 "message": "",
                 "numberOfIndividuals": 49,
                 "numberOfProperties": 430,
                 "numberOfTerms": 3043,
                 "ontologyId": "afo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:30.317607141",
+                "updated": "2023-07-28T11:10:19.364754309",
                 "version": "REC/2023/06"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/agro/individuals"
                     },
@@ -292,22 +292,22 @@
                 "languages": [
                     "de",
                     "en",
                     "es",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:05.256789614",
+                "loaded": "2023-07-28T10:30:59.613248466",
                 "message": "",
                 "numberOfIndividuals": 552,
                 "numberOfProperties": 293,
                 "numberOfTerms": 4162,
                 "ontologyId": "agro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:05.256789614",
+                "updated": "2023-07-28T10:30:59.613248466",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/aism/individuals"
                     },
@@ -351,22 +351,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "es"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:07.451221239",
+                "loaded": "2023-07-28T10:31:01.880266827",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 547,
                 "numberOfTerms": 7443,
                 "ontologyId": "aism",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:07.451221239",
+                "updated": "2023-07-28T10:31:01.880266827",
                 "version": "2023-04-14"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/amphx/individuals"
                     },
@@ -417,22 +417,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:10.218119660",
+                "loaded": "2023-07-28T10:31:04.593324779",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 10,
                 "numberOfTerms": 403,
                 "ontologyId": "amphx",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:10.218119660",
+                "updated": "2023-07-28T10:31:04.593324779",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/apo/individuals"
                     },
@@ -475,22 +475,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:10.353484444",
+                "loaded": "2023-07-28T10:31:04.733370566",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 27,
                 "numberOfTerms": 619,
                 "ontologyId": "apo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:10.353484444",
+                "updated": "2023-07-28T10:31:04.733370566",
                 "version": "2023-04-27"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/apollo_sv/individuals"
                     },
@@ -535,22 +535,22 @@
                 "lang": "en",
                 "languages": [
                     "de",
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:10.567979133",
+                "loaded": "2023-07-28T10:31:04.939791001",
                 "message": "",
                 "numberOfIndividuals": 27,
                 "numberOfProperties": 366,
                 "numberOfTerms": 1691,
                 "ontologyId": "apollo_sv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:10.567979133",
+                "updated": "2023-07-28T10:31:04.939791001",
                 "version": "2023-01-10"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/aro/individuals"
                     },
@@ -593,22 +593,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:11.480331155",
+                "loaded": "2023-07-28T10:31:05.844787574",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 25,
                 "numberOfTerms": 6897,
                 "ontologyId": "aro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:11.480331155",
+                "updated": "2023-07-28T10:31:05.844787574",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bao/individuals"
                     },
@@ -666,22 +666,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "fr",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:28.266847484",
+                "loaded": "2023-07-28T11:10:17.259896360",
                 "message": "",
                 "numberOfIndividuals": 1,
                 "numberOfProperties": 447,
                 "numberOfTerms": 8711,
                 "ontologyId": "bao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:28.266847484",
+                "updated": "2023-07-28T11:10:17.259896360",
                 "version": "2.8.6"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bcgo/individuals"
                     },
@@ -725,22 +725,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:18.127185093",
+                "loaded": "2023-07-28T10:56:30.109877431",
                 "message": "",
                 "numberOfIndividuals": 46,
                 "numberOfProperties": 171,
                 "numberOfTerms": 2270,
                 "ontologyId": "bcgo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:18.127185093",
+                "updated": "2023-07-28T10:56:30.109877431",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bcio/individuals"
                     },
@@ -797,22 +797,22 @@
                     "de",
                     "en",
                     "es",
                     "fr",
                     "la"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:43.986656308",
+                "loaded": "2023-07-28T11:10:32.647058697",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 74,
                 "numberOfTerms": 1029,
                 "ontologyId": "bcio",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:43.986656308",
+                "updated": "2023-07-28T11:10:32.647058697",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bco/individuals"
                     },
@@ -856,22 +856,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:12.783818561",
+                "loaded": "2023-07-28T10:31:07.112556321",
                 "message": "",
                 "numberOfIndividuals": 64,
                 "numberOfProperties": 472,
                 "numberOfTerms": 253,
                 "ontologyId": "bco",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:12.783818561",
+                "updated": "2023-07-28T10:31:07.112556321",
                 "version": "2021-11-14"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bfo/individuals"
                     },
@@ -914,22 +914,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:12.895497511",
+                "loaded": "2023-07-28T10:31:07.242685227",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 22,
                 "numberOfTerms": 35,
                 "ontologyId": "bfo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:12.895497511",
+                "updated": "2023-07-28T10:31:07.242685227",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bila/individuals"
                     },
@@ -972,22 +972,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:18.593053005",
+                "loaded": "2023-07-28T10:56:30.562394598",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 20,
                 "numberOfTerms": 114,
                 "ontologyId": "bila",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:18.593053005",
+                "updated": "2023-07-28T10:56:30.562394598",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bspo/individuals"
                     },
@@ -1030,22 +1030,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:12.965054624",
+                "loaded": "2023-07-28T10:31:07.320510663",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 236,
                 "numberOfTerms": 169,
                 "ontologyId": "bspo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:12.965054624",
+                "updated": "2023-07-28T10:31:07.320510663",
                 "version": "2023-05-27"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/bto/individuals"
                     },
@@ -1093,22 +1093,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:13.500790255",
+                "loaded": "2023-07-28T10:31:07.874315040",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 37,
                 "numberOfTerms": 6569,
                 "ontologyId": "bto",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:13.500790255",
+                "updated": "2023-07-28T10:31:07.874315040",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cao/individuals"
                     },
@@ -1156,22 +1156,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:45.618184755",
+                "loaded": "2023-07-28T11:05:28.924369832",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 45,
                 "numberOfTerms": 445,
                 "ontologyId": "cao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:45.618184755",
+                "updated": "2023-07-28T11:05:28.924369832",
                 "version": "0.2"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/caro/individuals"
                     },
@@ -1214,22 +1214,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:14.950831723",
+                "loaded": "2023-07-28T10:31:09.325869955",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 279,
                 "numberOfTerms": 720,
                 "ontologyId": "caro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:14.950831723",
+                "updated": "2023-07-28T10:31:09.325869955",
                 "version": "2023-03-15"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ccf/individuals"
                     },
@@ -1284,22 +1284,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:20:20.153971864",
+                "loaded": "2023-07-28T11:12:14.326411721",
                 "message": "",
                 "numberOfIndividuals": 6987,
                 "numberOfProperties": 238,
                 "numberOfTerms": 8836,
                 "ontologyId": "ccf",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:20:20.153971864",
+                "updated": "2023-07-28T11:12:14.326411721",
                 "version": "2.2.1"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cco/individuals"
                     },
@@ -1350,22 +1350,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:15:02.873711594",
+                "loaded": "2023-07-28T11:06:42.543940363",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 12,
                 "numberOfTerms": 277685,
                 "ontologyId": "cco",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:15:02.873711594",
+                "updated": "2023-07-28T11:06:42.543940363",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cdao/individuals"
                     },
@@ -1408,22 +1408,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:15.396962007",
+                "loaded": "2023-07-28T10:31:09.771107507",
                 "message": "",
                 "numberOfIndividuals": 26,
                 "numberOfProperties": 139,
                 "numberOfTerms": 164,
                 "ontologyId": "cdao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:15.396962007",
+                "updated": "2023-07-28T10:31:09.771107507",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cdno/individuals"
                     },
@@ -1466,22 +1466,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:15.991361860",
+                "loaded": "2023-07-28T10:31:10.366134610",
                 "message": "",
                 "numberOfIndividuals": 44,
                 "numberOfProperties": 259,
                 "numberOfTerms": 2060,
                 "ontologyId": "cdno",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:15.991361860",
+                "updated": "2023-07-28T10:31:10.366134610",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ceph/individuals"
                     },
@@ -1524,22 +1524,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:55.996171105",
+                "loaded": "2023-07-28T10:56:09.901720057",
                 "message": "",
                 "numberOfIndividuals": 24,
                 "numberOfProperties": 135,
                 "numberOfTerms": 1574,
                 "ontologyId": "ceph",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:55.996171105",
+                "updated": "2023-07-28T10:56:09.901720057",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/chebi/individuals"
                     },
@@ -1582,22 +1582,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:41:46.951463240",
+                "loaded": "2023-07-28T10:31:41.270536294",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 47,
                 "numberOfTerms": 183247,
                 "ontologyId": "chebi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:41:46.951463240",
+                "updated": "2023-07-28T10:31:41.270536294",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cheminf/individuals"
                     },
@@ -1640,22 +1640,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:42:32.290271695",
+                "loaded": "2023-07-28T10:32:29.964702212",
                 "message": "",
                 "numberOfIndividuals": 20,
                 "numberOfProperties": 515,
                 "numberOfTerms": 2116,
                 "ontologyId": "cheminf",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:42:32.290271695",
+                "updated": "2023-07-28T10:32:29.964702212",
                 "version": "2.1.0"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/chiro/individuals"
                     },
@@ -1698,22 +1698,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:42:33.430703229",
+                "loaded": "2023-07-28T10:32:32.081791010",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 59,
                 "numberOfTerms": 13279,
                 "ontologyId": "chiro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:42:33.430703229",
+                "updated": "2023-07-28T10:32:32.081791010",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/chmo/individuals"
                     },
@@ -1756,22 +1756,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:42:35.699662943",
+                "loaded": "2023-07-28T10:32:34.346284432",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 64,
                 "numberOfTerms": 3101,
                 "ontologyId": "chmo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:42:35.699662943",
+                "updated": "2023-07-28T10:32:34.346284432",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cio/individuals"
                     },
@@ -1814,22 +1814,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:42:42.173892802",
+                "loaded": "2023-07-28T10:32:39.751437957",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 17,
                 "numberOfTerms": 44,
                 "ontologyId": "cio",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:42:42.173892802",
+                "updated": "2023-07-28T10:32:39.751437957",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cl/individuals"
                     },
@@ -1872,22 +1872,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:42:44.978130578",
+                "loaded": "2023-07-28T10:32:42.546899006",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 539,
                 "numberOfTerms": 16046,
                 "ontologyId": "cl",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:42:44.978130578",
+                "updated": "2023-07-28T10:32:42.546899006",
                 "version": "2023-07-20"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/clao/individuals"
                     },
@@ -1933,22 +1933,22 @@
                 "languages": [
                     "de",
                     "en",
                     "es",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:42:50.457060566",
+                "loaded": "2023-07-28T10:32:47.892646154",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 42,
                 "numberOfTerms": 1516,
                 "ontologyId": "clao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:42:50.457060566",
+                "updated": "2023-07-28T10:32:47.892646154",
                 "version": "2020-16-05"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/clo/individuals"
                     },
@@ -1992,22 +1992,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:42:53.178251856",
+                "loaded": "2023-07-28T10:32:50.608645203",
                 "message": "",
                 "numberOfIndividuals": 26,
                 "numberOfProperties": 392,
                 "numberOfTerms": 43325,
                 "ontologyId": "clo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:42:53.178251856",
+                "updated": "2023-07-28T10:32:50.608645203",
                 "version": "2.1.178"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/clyh/individuals"
                     },
@@ -2058,22 +2058,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:00.457415598",
+                "loaded": "2023-07-28T10:32:57.938566470",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 9,
                 "numberOfTerms": 265,
                 "ontologyId": "clyh",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:00.457415598",
+                "updated": "2023-07-28T10:32:57.938566470",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cmo/individuals"
                     },
@@ -2116,22 +2116,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:00.846829734",
+                "loaded": "2023-07-28T10:32:58.319950344",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 22,
                 "numberOfTerms": 3927,
                 "ontologyId": "cmo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:00.846829734",
+                "updated": "2023-07-28T10:32:58.319950344",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cmpo/individuals"
                     },
@@ -2182,22 +2182,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:19.787805276",
+                "loaded": "2023-07-28T11:10:09.453931929",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 76,
                 "numberOfTerms": 1058,
                 "ontologyId": "cmpo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:19.787805276",
+                "updated": "2023-07-28T11:10:09.453931929",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cob/individuals"
                     },
@@ -2240,22 +2240,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:01.464473372",
+                "loaded": "2023-07-28T10:32:58.992303778",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 36,
                 "numberOfTerms": 73,
                 "ontologyId": "cob",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:01.464473372",
+                "updated": "2023-07-28T10:32:58.992303778",
                 "version": "2023-05-12"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/colao/individuals"
                     },
@@ -2300,22 +2300,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "fr",
                     "it"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:01.665905389",
+                "loaded": "2023-07-28T10:32:59.228297224",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 56,
                 "numberOfTerms": 905,
                 "ontologyId": "colao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:01.665905389",
+                "updated": "2023-07-28T10:32:59.228297224",
                 "version": "2023-04-08"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/covoc/individuals"
                     },
@@ -2368,22 +2368,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:44.275727148",
+                "loaded": "2023-07-28T11:10:32.952895634",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 652,
                 "numberOfTerms": 563,
                 "ontologyId": "covoc",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:44.275727148",
+                "updated": "2023-07-28T11:10:32.952895634",
                 "version": "2022-10-26"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cpont/individuals"
                     },
@@ -2437,22 +2437,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:20:23.987443619",
+                "loaded": "2023-07-28T11:12:18.316880189",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 125,
                 "numberOfTerms": 5762,
                 "ontologyId": "cpont",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:20:23.987443619",
+                "updated": "2023-07-28T11:12:18.316880189",
                 "version": "2023-04-18"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cro/individuals"
                     },
@@ -2495,22 +2495,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:02.065306092",
+                "loaded": "2023-07-28T10:32:59.994694288",
                 "message": "",
                 "numberOfIndividuals": 42,
                 "numberOfProperties": 73,
                 "numberOfTerms": 145,
                 "ontologyId": "cro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:02.065306092",
+                "updated": "2023-07-28T10:32:59.994694288",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cryoem/individuals"
                     },
@@ -2563,22 +2563,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:46.719811911",
+                "loaded": "2023-07-28T11:10:35.279860950",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 20,
                 "numberOfTerms": 111,
                 "ontologyId": "cryoem",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:46.719811911",
+                "updated": "2023-07-28T11:10:35.279860950",
                 "version": "2021-03-09"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/cteno/individuals"
                     },
@@ -2621,22 +2621,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:02.231107868",
+                "loaded": "2023-07-28T10:33:00.248388848",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 315,
                 "numberOfTerms": 709,
                 "ontologyId": "cteno",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:02.231107868",
+                "updated": "2023-07-28T10:33:00.248388848",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/dc/individuals"
                     },
@@ -2683,22 +2683,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:46.877767031",
+                "loaded": "2023-07-28T11:05:30.243673071",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 15,
                 "numberOfTerms": 0,
                 "ontologyId": "dc",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:46.877767031",
+                "updated": "2023-07-28T11:05:30.243673071",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/dcterms/individuals"
                     },
@@ -2745,22 +2745,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:46.812264433",
+                "loaded": "2023-07-28T11:05:30.161337804",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 55,
                 "numberOfTerms": 22,
                 "ontologyId": "dcterms",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:46.812264433",
+                "updated": "2023-07-28T11:05:30.161337804",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ddanat/individuals"
                     },
@@ -2803,22 +2803,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:04.163824146",
+                "loaded": "2023-07-28T10:33:01.637422158",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 22,
                 "numberOfTerms": 134,
                 "ontologyId": "ddanat",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:04.163824146",
+                "updated": "2023-07-28T10:33:01.637422158",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ddpheno/individuals"
                     },
@@ -2861,22 +2861,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:04.316695088",
+                "loaded": "2023-07-28T10:33:01.794276386",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 24,
                 "numberOfTerms": 1259,
                 "ontologyId": "ddpheno",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:04.316695088",
+                "updated": "2023-07-28T10:33:01.794276386",
                 "version": "2023-02-19"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/dicom/individuals"
                     },
@@ -2927,22 +2927,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:51.236138322",
+                "loaded": "2023-07-28T11:05:34.625278034",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 56,
                 "numberOfTerms": 4712,
                 "ontologyId": "dicom",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:51.236138322",
+                "updated": "2023-07-28T11:05:34.625278034",
                 "version": "2023c_20230704"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/dideo/individuals"
                     },
@@ -2986,22 +2986,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:04.589896425",
+                "loaded": "2023-07-28T10:33:02.125486660",
                 "message": "",
                 "numberOfIndividuals": 24,
                 "numberOfProperties": 305,
                 "numberOfTerms": 684,
                 "ontologyId": "dideo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:04.589896425",
+                "updated": "2023-07-28T10:33:02.125486660",
                 "version": "release version 2022-06-14"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/disdriv/individuals"
                     },
@@ -3044,22 +3044,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:04.783363589",
+                "loaded": "2023-07-28T10:33:02.314287744",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 11,
                 "numberOfTerms": 13,
                 "ontologyId": "disdriv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:04.783363589",
+                "updated": "2023-07-28T10:33:02.314287744",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/doid/individuals"
                     },
@@ -3103,22 +3103,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:06.170389399",
+                "loaded": "2023-07-28T10:33:03.761520938",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 174,
                 "numberOfTerms": 18368,
                 "ontologyId": "doid",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:06.170389399",
+                "updated": "2023-07-28T10:33:03.761520938",
                 "version": "2023-07-24"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/dpo/individuals"
                     },
@@ -3161,22 +3161,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:09.138027265",
+                "loaded": "2023-07-28T10:33:06.904298385",
                 "message": "",
                 "numberOfIndividuals": 41,
                 "numberOfProperties": 311,
                 "numberOfTerms": 1421,
                 "ontologyId": "dpo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:09.138027265",
+                "updated": "2023-07-28T10:33:06.904298385",
                 "version": "2023-07-05"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/dron/individuals"
                     },
@@ -3220,22 +3220,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "de",
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:43:41.943873603",
+                "loaded": "2023-07-28T10:33:39.493646620",
                 "message": "",
                 "numberOfIndividuals": 20,
                 "numberOfProperties": 269,
                 "numberOfTerms": 672420,
                 "ontologyId": "dron",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:43:41.943873603",
+                "updated": "2023-07-28T10:33:39.493646620",
                 "version": "2023-06-11"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/duo/individuals"
                     },
@@ -3278,22 +3278,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:02.545041526",
+                "loaded": "2023-07-28T10:35:59.329412762",
                 "message": "",
                 "numberOfIndividuals": 20,
                 "numberOfProperties": 134,
                 "numberOfTerms": 308,
                 "ontologyId": "duo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:02.545041526",
+                "updated": "2023-07-28T10:35:59.329412762",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ecao/individuals"
                     },
@@ -3336,22 +3336,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:02.697780638",
+                "loaded": "2023-07-28T10:35:59.544527195",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 169,
                 "numberOfTerms": 496,
                 "ontologyId": "ecao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:02.697780638",
+                "updated": "2023-07-28T10:35:59.544527195",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/eco/individuals"
                     },
@@ -3394,22 +3394,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:03.188530970",
+                "loaded": "2023-07-28T10:36:00.174499782",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 91,
                 "numberOfTerms": 3160,
                 "ontologyId": "eco",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:03.188530970",
+                "updated": "2023-07-28T10:36:00.174499782",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ecocore/individuals"
                     },
@@ -3452,22 +3452,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:04.827505971",
+                "loaded": "2023-07-28T10:36:01.858118031",
                 "message": "",
                 "numberOfIndividuals": 17,
                 "numberOfProperties": 483,
                 "numberOfTerms": 5586,
                 "ontologyId": "ecocore",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:04.827505971",
+                "updated": "2023-07-28T10:36:01.858118031",
                 "version": "2022-03-09"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ecto/individuals"
                     },
@@ -3522,22 +3522,22 @@
                     "gsw",
                     "it",
                     "pt",
                     "swg",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:08.720192257",
+                "loaded": "2023-07-28T10:36:05.731150040",
                 "message": "",
                 "numberOfIndividuals": 60,
                 "numberOfProperties": 480,
                 "numberOfTerms": 11864,
                 "ontologyId": "ecto",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:08.720192257",
+                "updated": "2023-07-28T10:36:05.731150040",
                 "version": "2023-02-14"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/edam/individuals"
                     },
@@ -3595,22 +3595,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:14:21.444953083",
+                "loaded": "2023-07-28T11:06:02.741748507",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 77,
                 "numberOfTerms": 3473,
                 "ontologyId": "edam",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:14:21.444953083",
+                "updated": "2023-07-28T11:06:02.741748507",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/efo/individuals"
                     },
@@ -3668,22 +3668,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:14:03.584039980",
+                "loaded": "2023-07-28T11:05:44.530995044",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 223,
                 "numberOfTerms": 50497,
                 "ontologyId": "efo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:14:03.584039980",
+                "updated": "2023-07-28T11:05:44.530995044",
                 "version": "3.56.0"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ehdaa2/individuals"
                     },
@@ -3726,22 +3726,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:56.554036233",
+                "loaded": "2023-07-28T10:56:10.449628254",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 27,
                 "numberOfTerms": 2734,
                 "ontologyId": "ehdaa2",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:56.554036233",
+                "updated": "2023-07-28T10:56:10.449628254",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/emap/individuals"
                     },
@@ -3784,22 +3784,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:21.122033339",
+                "loaded": "2023-07-28T10:56:33.136332697",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 10,
                 "numberOfTerms": 19444,
                 "ontologyId": "emap",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:21.122033339",
+                "updated": "2023-07-28T10:56:33.136332697",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/emapa/individuals"
                     },
@@ -3842,22 +3842,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:12.963358166",
+                "loaded": "2023-07-28T10:36:10.006159132",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 26,
                 "numberOfTerms": 8774,
                 "ontologyId": "emapa",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:12.963358166",
+                "updated": "2023-07-28T10:36:10.006159132",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/enm/individuals"
                     },
@@ -3923,22 +3923,22 @@
                     "en-br",
                     "en-us",
                     "es",
                     "it",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:21.635321857",
+                "loaded": "2023-07-28T11:10:11.371963850",
                 "message": "",
                 "numberOfIndividuals": 834,
                 "numberOfProperties": 2384,
                 "numberOfTerms": 25917,
                 "ontologyId": "enm",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:21.635321857",
+                "updated": "2023-07-28T11:10:11.371963850",
                 "version": "10.0"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ensemblglossary/individuals"
                     },
@@ -3989,22 +3989,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:53.693708658",
+                "loaded": "2023-07-28T11:05:35.285514895",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 5,
                 "numberOfTerms": 377,
                 "ontologyId": "ensemblglossary",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:53.693708658",
+                "updated": "2023-07-28T11:05:35.285514895",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/envo/individuals"
                     },
@@ -4053,22 +4053,22 @@
                     "en-br",
                     "en-us",
                     "es",
                     "it",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:14.842077072",
+                "loaded": "2023-07-28T10:36:12.097022032",
                 "message": "",
                 "numberOfIndividuals": 44,
                 "numberOfProperties": 314,
                 "numberOfTerms": 6996,
                 "ontologyId": "envo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:14.842077072",
+                "updated": "2023-07-28T10:36:12.097022032",
                 "version": "2023-02-13"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/epio/individuals"
                     },
@@ -4111,22 +4111,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:16.185877869",
+                "loaded": "2023-07-28T10:36:13.671670332",
                 "message": "",
                 "numberOfIndividuals": 1,
                 "numberOfProperties": 133,
                 "numberOfTerms": 1916,
                 "ontologyId": "epio",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:16.185877869",
+                "updated": "2023-07-28T10:36:13.671670332",
                 "version": "2021-05-28"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/eupath/individuals"
                     },
@@ -4171,22 +4171,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "fr",
                     "pt"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:16.744188799",
+                "loaded": "2023-07-28T10:36:14.314352816",
                 "message": "",
                 "numberOfIndividuals": 92,
                 "numberOfProperties": 205,
                 "numberOfTerms": 5109,
                 "ontologyId": "eupath",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:16.744188799",
+                "updated": "2023-07-28T10:36:14.314352816",
                 "version": "2023-05-30"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/exo/individuals"
                     },
@@ -4229,22 +4229,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:17.505362475",
+                "loaded": "2023-07-28T10:36:15.162421744",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 23,
                 "numberOfTerms": 152,
                 "ontologyId": "exo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:17.505362475",
+                "updated": "2023-07-28T10:36:15.162421744",
                 "version": "2022-06-29"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fao/individuals"
                     },
@@ -4287,22 +4287,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:17.583343792",
+                "loaded": "2023-07-28T10:36:15.259221752",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 36,
                 "numberOfTerms": 121,
                 "ontologyId": "fao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:17.583343792",
+                "updated": "2023-07-28T10:36:15.259221752",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fbbi/individuals"
                     },
@@ -4345,22 +4345,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:17.695879595",
+                "loaded": "2023-07-28T10:36:15.383287610",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 140,
                 "numberOfTerms": 647,
                 "ontologyId": "fbbi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:17.695879595",
+                "updated": "2023-07-28T10:36:15.383287610",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fbbt/individuals"
                     },
@@ -4403,22 +4403,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:21.434862860",
+                "loaded": "2023-07-28T10:36:19.204932111",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 353,
                 "numberOfTerms": 21049,
                 "ontologyId": "fbbt",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:21.434862860",
+                "updated": "2023-07-28T10:36:19.204932111",
                 "version": "2023-07-05"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fbcv/individuals"
                     },
@@ -4461,22 +4461,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:30.165669254",
+                "loaded": "2023-07-28T10:36:28.321106928",
                 "message": "",
                 "numberOfIndividuals": 1,
                 "numberOfProperties": 241,
                 "numberOfTerms": 2291,
                 "ontologyId": "fbcv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:30.165669254",
+                "updated": "2023-07-28T10:36:28.321106928",
                 "version": "2023-07-06"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fbdv/individuals"
                     },
@@ -4519,22 +4519,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:30.699330343",
+                "loaded": "2023-07-28T10:36:28.899259091",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 215,
                 "numberOfTerms": 253,
                 "ontologyId": "fbdv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:30.699330343",
+                "updated": "2023-07-28T10:36:28.899259091",
                 "version": "2023-07-05"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fbsp/individuals"
                     },
@@ -4577,22 +4577,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:25.031912070",
+                "loaded": "2023-07-28T10:56:36.893943377",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 20,
                 "numberOfTerms": 6600,
                 "ontologyId": "fbsp",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:25.031912070",
+                "updated": "2023-07-28T10:56:36.893943377",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fideo/individuals"
                     },
@@ -4635,22 +4635,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:30.986008556",
+                "loaded": "2023-07-28T10:36:29.212705289",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 13,
                 "numberOfTerms": 3548,
                 "ontologyId": "fideo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:30.986008556",
+                "updated": "2023-07-28T10:36:29.212705289",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fix/individuals"
                     },
@@ -4693,22 +4693,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:45.808754269",
+                "loaded": "2023-07-28T10:55:59.271015138",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 15,
                 "numberOfTerms": 1163,
                 "ontologyId": "fix",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:45.808754269",
+                "updated": "2023-07-28T10:55:59.271015138",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/flopo/individuals"
                     },
@@ -4751,22 +4751,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:34.690869504",
+                "loaded": "2023-07-28T10:36:33.091483198",
                 "message": "",
                 "numberOfIndividuals": 9,
                 "numberOfProperties": 737,
                 "numberOfTerms": 36818,
                 "ontologyId": "flopo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:34.690869504",
+                "updated": "2023-07-28T10:36:33.091483198",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fma/individuals"
                     },
@@ -4809,22 +4809,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:02.295036513",
+                "loaded": "2023-07-28T10:56:14.374413596",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 15,
                 "numberOfTerms": 78977,
                 "ontologyId": "fma",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:02.295036513",
+                "updated": "2023-07-28T10:56:14.374413596",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fobi/individuals"
                     },
@@ -4868,22 +4868,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "de",
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:41.217470710",
+                "loaded": "2023-07-28T10:36:39.585982898",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 29,
                 "numberOfTerms": 1799,
                 "ontologyId": "fobi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:41.217470710",
+                "updated": "2023-07-28T10:36:39.585982898",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/foodon/individuals"
                     },
@@ -4990,22 +4990,22 @@
                     "zh-hans",
                     "zh-hant",
                     "zh-tans",
                     "zh-tant",
                     "zu"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:43.459549281",
+                "loaded": "2023-07-28T10:36:41.932315968",
                 "message": "",
                 "numberOfIndividuals": 435,
                 "numberOfProperties": 138,
                 "numberOfTerms": 34371,
                 "ontologyId": "foodon",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:43.459549281",
+                "updated": "2023-07-28T10:36:41.932315968",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fovt/individuals"
                     },
@@ -5048,22 +5048,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:50.323296192",
+                "loaded": "2023-07-28T10:36:48.835393308",
                 "message": "",
                 "numberOfIndividuals": 27,
                 "numberOfProperties": 578,
                 "numberOfTerms": 11321,
                 "ontologyId": "fovt",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:50.323296192",
+                "updated": "2023-07-28T10:36:48.835393308",
                 "version": "2023-05-31"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/fypo/individuals"
                     },
@@ -5106,22 +5106,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:46:57.636554238",
+                "loaded": "2023-07-28T10:36:56.266587219",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 296,
                 "numberOfTerms": 16479,
                 "ontologyId": "fypo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:46:57.636554238",
+                "updated": "2023-07-28T10:36:56.266587219",
                 "version": "2023-07-12"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/gecko/individuals"
                     },
@@ -5164,22 +5164,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:04.016877199",
+                "loaded": "2023-07-28T10:37:02.846093048",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 17,
                 "numberOfTerms": 154,
                 "ontologyId": "gecko",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:04.016877199",
+                "updated": "2023-07-28T10:37:02.846093048",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/geno/individuals"
                     },
@@ -5222,22 +5222,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:04.915595110",
+                "loaded": "2023-07-28T10:37:03.808529219",
                 "message": "",
                 "numberOfIndividuals": 21,
                 "numberOfProperties": 208,
                 "numberOfTerms": 423,
                 "ontologyId": "geno",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:04.915595110",
+                "updated": "2023-07-28T10:37:03.808529219",
                 "version": "2023-02-21"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/geo/individuals"
                     },
@@ -5281,22 +5281,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:05.061536929",
+                "loaded": "2023-07-28T10:37:03.939157698",
                 "message": "",
                 "numberOfIndividuals": 4,
                 "numberOfProperties": 111,
                 "numberOfTerms": 107,
                 "ontologyId": "geo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:05.061536929",
+                "updated": "2023-07-28T10:37:03.939157698",
                 "version": "production version 2016-03-26"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/gexo/individuals"
                     },
@@ -5347,22 +5347,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:03.360982721",
+                "loaded": "2023-07-28T11:09:50.954807619",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 12,
                 "numberOfTerms": 166254,
                 "ontologyId": "gexo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:03.360982721",
+                "updated": "2023-07-28T11:09:50.954807619",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/gno/individuals"
                     },
@@ -5405,22 +5405,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:13.149897110",
+                "loaded": "2023-07-28T10:37:11.966359246",
                 "message": "",
                 "numberOfIndividuals": 5,
                 "numberOfProperties": 14,
                 "numberOfTerms": 159269,
                 "ontologyId": "gno",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:13.149897110",
+                "updated": "2023-07-28T10:37:11.966359246",
                 "version": "v1.11.1"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/go/individuals"
                     },
@@ -5463,22 +5463,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:37.412157350",
+                "loaded": "2023-07-28T10:37:36.415983419",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 69,
                 "numberOfTerms": 51204,
                 "ontologyId": "go",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:37.412157350",
+                "updated": "2023-07-28T10:37:36.415983419",
                 "version": "2023-06-11"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/gsso/individuals"
                     },
@@ -5599,22 +5599,22 @@
                     "ur",
                     "xno",
                     "yur",
                     "zh",
                     "zun"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:20.966167152",
+                "loaded": "2023-07-28T10:55:35.719444553",
                 "message": "",
                 "numberOfIndividuals": 3004,
                 "numberOfProperties": 292,
                 "numberOfTerms": 13192,
                 "ontologyId": "gsso",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:20.966167152",
+                "updated": "2023-07-28T10:55:35.719444553",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/hancestro/individuals"
                     },
@@ -5657,22 +5657,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:48.780625087",
+                "loaded": "2023-07-28T10:37:47.933752285",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 24,
                 "numberOfTerms": 584,
                 "ontologyId": "hancestro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:48.780625087",
+                "updated": "2023-07-28T10:37:47.933752285",
                 "version": "2.8"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/hao/individuals"
                     },
@@ -5715,22 +5715,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:49.252105357",
+                "loaded": "2023-07-28T10:37:48.375278130",
                 "message": "",
                 "numberOfIndividuals": 2764,
                 "numberOfProperties": 21,
                 "numberOfTerms": 2597,
                 "ontologyId": "hao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:49.252105357",
+                "updated": "2023-07-28T10:37:48.375278130",
                 "version": "2023-06-01"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/hcao/individuals"
                     },
@@ -5781,22 +5781,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:35.822260209",
+                "loaded": "2023-07-28T11:10:24.745354231",
                 "message": "",
                 "numberOfIndividuals": 31,
                 "numberOfProperties": 711,
                 "numberOfTerms": 22606,
                 "ontologyId": "hcao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:35.822260209",
+                "updated": "2023-07-28T11:10:24.745354231",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/hom/individuals"
                     },
@@ -5839,22 +5839,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:50.384843659",
+                "loaded": "2023-07-28T10:37:49.193621552",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 18,
                 "numberOfTerms": 66,
                 "ontologyId": "hom",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:50.384843659",
+                "updated": "2023-07-28T10:37:49.193621552",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/hp/individuals"
                     },
@@ -5905,22 +5905,22 @@
                     "nl",
                     "nna",
                     "tr",
                     "tw",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:31.928091247",
+                "loaded": "2023-07-28T10:55:45.821927882",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 549,
                 "numberOfTerms": 29810,
                 "ontologyId": "hp",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:31.928091247",
+                "updated": "2023-07-28T10:55:45.821927882",
                 "version": "2023-07-21"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/hsapdv/individuals"
                     },
@@ -5963,22 +5963,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:50.471217440",
+                "loaded": "2023-07-28T10:37:49.275444733",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 37,
                 "numberOfTerms": 245,
                 "ontologyId": "hsapdv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:50.471217440",
+                "updated": "2023-07-28T10:37:49.275444733",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/hso/individuals"
                     },
@@ -6028,22 +6028,22 @@
                     "en",
                     "es",
                     "fr",
                     "it",
                     "ja"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:50.645451352",
+                "loaded": "2023-07-28T10:37:49.457019287",
                 "message": "",
                 "numberOfIndividuals": 192,
                 "numberOfProperties": 303,
                 "numberOfTerms": 448,
                 "ontologyId": "hso",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:50.645451352",
+                "updated": "2023-07-28T10:37:49.457019287",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/htn/individuals"
                     },
@@ -6087,22 +6087,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:50.871321746",
+                "loaded": "2023-07-28T10:37:49.721024370",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 424,
                 "numberOfTerms": 603,
                 "ontologyId": "htn",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:50.871321746",
+                "updated": "2023-07-28T10:37:49.721024370",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/iao/individuals"
                     },
@@ -6145,22 +6145,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:51.038002216",
+                "loaded": "2023-07-28T10:37:49.883762469",
                 "message": "",
                 "numberOfIndividuals": 20,
                 "numberOfProperties": 127,
                 "numberOfTerms": 263,
                 "ontologyId": "iao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:51.038002216",
+                "updated": "2023-07-28T10:37:49.883762469",
                 "version": "2022-11-07"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/iceo/individuals"
                     },
@@ -6205,22 +6205,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:53.057703044",
+                "loaded": "2023-07-28T10:37:51.865783695",
                 "message": "",
                 "numberOfIndividuals": 6,
                 "numberOfProperties": 133,
                 "numberOfTerms": 30914,
                 "ontologyId": "iceo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:53.057703044",
+                "updated": "2023-07-28T10:37:51.865783695",
                 "version": "2.1"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ico/individuals"
                     },
@@ -6264,22 +6264,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:57.979575517",
+                "loaded": "2023-07-28T10:37:57.455932852",
                 "message": "",
                 "numberOfIndividuals": 26,
                 "numberOfProperties": 187,
                 "numberOfTerms": 963,
                 "ontologyId": "ico",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:57.979575517",
+                "updated": "2023-07-28T10:37:57.455932852",
                 "version": "1.0.187"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ido/individuals"
                     },
@@ -6322,22 +6322,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:58.184391095",
+                "loaded": "2023-07-28T10:37:57.674286398",
                 "message": "",
                 "numberOfIndividuals": 20,
                 "numberOfProperties": 104,
                 "numberOfTerms": 518,
                 "ontologyId": "ido",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:58.184391095",
+                "updated": "2023-07-28T10:37:57.674286398",
                 "version": "2017-11-03"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/idocovid19/individuals"
                     },
@@ -6390,22 +6390,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:43.320375283",
+                "loaded": "2023-07-28T11:10:32.267980413",
                 "message": "",
                 "numberOfIndividuals": 23,
                 "numberOfProperties": 110,
                 "numberOfTerms": 485,
                 "ontologyId": "idocovid19",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:43.320375283",
+                "updated": "2023-07-28T11:10:32.267980413",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/idomal/individuals"
                     },
@@ -6448,22 +6448,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:12.468479871",
+                "loaded": "2023-07-28T10:56:24.658413616",
                 "message": "",
                 "numberOfIndividuals": 595,
                 "numberOfProperties": 46,
                 "numberOfTerms": 3158,
                 "ontologyId": "idomal",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:12.468479871",
+                "updated": "2023-07-28T10:56:24.658413616",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ino/individuals"
                     },
@@ -6507,22 +6507,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:58.331408429",
+                "loaded": "2023-07-28T10:37:57.837862589",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 124,
                 "numberOfTerms": 328,
                 "ontologyId": "ino",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:58.331408429",
+                "updated": "2023-07-28T10:37:57.837862589",
                 "version": "1.1.13"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/kisao/individuals"
                     },
@@ -6568,22 +6568,22 @@
                 "languages": [
                     "de",
                     "en",
                     "end",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:44.760140669",
+                "loaded": "2023-07-28T10:55:58.233001625",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 19,
                 "numberOfTerms": 510,
                 "ontologyId": "kisao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:44.760140669",
+                "updated": "2023-07-28T10:55:58.233001625",
                 "version": "2.30"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/labo/individuals"
                     },
@@ -6627,22 +6627,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:58.443035164",
+                "loaded": "2023-07-28T10:37:57.960849659",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 78,
                 "numberOfTerms": 169,
                 "ontologyId": "labo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:58.443035164",
+                "updated": "2023-07-28T10:37:57.960849659",
                 "version": "2021-06-08"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/lepao/individuals"
                     },
@@ -6686,22 +6686,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "de",
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:58.657682036",
+                "loaded": "2023-07-28T10:37:58.201529031",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 84,
                 "numberOfTerms": 822,
                 "ontologyId": "lepao",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:58.657682036",
+                "updated": "2023-07-28T10:37:58.201529031",
                 "version": "2022-12-29"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ma/individuals"
                     },
@@ -6744,22 +6744,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:47:59.141798857",
+                "loaded": "2023-07-28T10:37:58.682280373",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 15,
                 "numberOfTerms": 3257,
                 "ontologyId": "ma",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:47:59.141798857",
+                "updated": "2023-07-28T10:37:58.682280373",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/maxo/individuals"
                     },
@@ -6805,22 +6805,22 @@
                 "languages": [
                     "da",
                     "de",
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:02.006757626",
+                "loaded": "2023-07-28T10:38:01.536486289",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 538,
                 "numberOfTerms": 13850,
                 "ontologyId": "maxo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:02.006757626",
+                "updated": "2023-07-28T10:38:01.536486289",
                 "version": "2023-06-06"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mco/individuals"
                     },
@@ -6864,22 +6864,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:07.411546218",
+                "loaded": "2023-07-28T10:38:07.030582246",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 108,
                 "numberOfTerms": 3383,
                 "ontologyId": "mco",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:07.411546218",
+                "updated": "2023-07-28T10:38:07.030582246",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mcro/individuals"
                     },
@@ -6922,22 +6922,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:08.770735501",
+                "loaded": "2023-07-28T10:38:08.483393202",
                 "message": "",
                 "numberOfIndividuals": 24,
                 "numberOfProperties": 338,
                 "numberOfTerms": 966,
                 "ontologyId": "mcro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:08.770735501",
+                "updated": "2023-07-28T10:38:08.483393202",
                 "version": "2023-03-07"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mf/individuals"
                     },
@@ -6980,22 +6980,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:08.968632311",
+                "loaded": "2023-07-28T10:38:08.683878537",
                 "message": "",
                 "numberOfIndividuals": 19,
                 "numberOfProperties": 90,
                 "numberOfTerms": 400,
                 "ontologyId": "mf",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:08.968632311",
+                "updated": "2023-07-28T10:38:08.683878537",
                 "version": "2022-07-19"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mfmo/individuals"
                     },
@@ -7038,22 +7038,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:09.133538235",
+                "loaded": "2023-07-28T10:38:08.873100588",
                 "message": "",
                 "numberOfIndividuals": 52,
                 "numberOfProperties": 828,
                 "numberOfTerms": 261,
                 "ontologyId": "mfmo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:09.133538235",
+                "updated": "2023-07-28T10:38:08.873100588",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mfoem/individuals"
                     },
@@ -7099,22 +7099,22 @@
                 "languages": [
                     "de",
                     "en",
                     "es",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:09.349945627",
+                "loaded": "2023-07-28T10:38:09.101008494",
                 "message": "",
                 "numberOfIndividuals": 19,
                 "numberOfProperties": 99,
                 "numberOfTerms": 619,
                 "ontologyId": "mfoem",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:09.349945627",
+                "updated": "2023-07-28T10:38:09.101008494",
                 "version": "2022-07-19"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mfomd/individuals"
                     },
@@ -7158,22 +7158,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "es"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:09.593508391",
+                "loaded": "2023-07-28T10:38:09.429351590",
                 "message": "",
                 "numberOfIndividuals": 73,
                 "numberOfProperties": 378,
                 "numberOfTerms": 1172,
                 "ontologyId": "mfomd",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:09.593508391",
+                "updated": "2023-07-28T10:38:09.429351590",
                 "version": "2020-04-26"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mi/individuals"
                     },
@@ -7218,22 +7218,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:09.969610560",
+                "loaded": "2023-07-28T10:38:09.801139483",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 45,
                 "numberOfTerms": 1623,
                 "ontologyId": "mi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:09.969610560",
+                "updated": "2023-07-28T10:38:09.801139483",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/miapa/individuals"
                     },
@@ -7276,22 +7276,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:10.307485530",
+                "loaded": "2023-07-28T10:38:10.155577663",
                 "message": "",
                 "numberOfIndividuals": 38,
                 "numberOfProperties": 229,
                 "numberOfTerms": 238,
                 "ontologyId": "miapa",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:10.307485530",
+                "updated": "2023-07-28T10:38:10.155577663",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/micro/individuals"
                     },
@@ -7337,22 +7337,22 @@
                 "languages": [
                     "25 \u00b0c",
                     "25\u00b0c    * adjusted as required to meet performance standards   directions suspend 46g in 1 litre of distilled water. sterilise by autoclaving at 121\u00b0c for 15 minutes. cool to 50-55\u00b0c and aseptically add 5-10% sterile defibrinated horse blood sr0050.  mix well and pour into sterile petri dishes.",
                     "25\u00b0c  directions suspend 64 grams in 1 litre of distilled water and soak for 15 minutes, with occasional stirring. distribute into 18mm diameter tubes to a depth of 50mm so that the bottom of the tube is filled with liver particles. agitate frequently during distribution to keep the liver tissue in suspension.  sterilise by autoclaving for 20 minutes at 115\u00b0 c. inoculate when cool and then aseptically seal with a layer of sterile 2% oxoid agar no.3 (lp0013) solution.  description a liver infusion medium, containing liver particles, for the examination of foods for saccharolytic or putrefactive mesophilic and thermophilic anaerobes. also recommended for the maintenance of aerobes and anaerobes in pure culture.  technique gillespie in communication with scarr1 recommended liver broth for the examination of canners\u2019 sugar for hydrogen swells caused by thermophilic anaerobes (clostridium thermosaccharolyticum). a 20% w/v solution of the sugar is steamed for 30 minutes to destroy vegetative forms and inoculated into liver broth sealed with agar. the standard proposed was a maximum of 1 positive tube in six - with 20ml inocula incubated for 72 hours at 56\u00b0c.  this medium should be made up only when required for use. storage of the reconstituted medium is not recommended because air may be absorbed and the re-steaming necessary for the restoration of anaerobic conditions darkens the medium. liver broth is not an homogeneous medium; consisting of a layer of liver particles and a cloudy supernatant. growth produces an obvious increase in turbidity and some organisms (e.g. clostridium thermosaccharolyticum) also produce gas which often pushes the agar plug towards the top of the tube. some organisms also digest the solid liver tissue.  storage conditions and shelf life store the dehydrated medium at 10-30\u00b0c and use before the expiry date on the label. store the prepared medium at 2-8\u00b0 c.  appearance dehydrated medium: brown powder / granules prepared medium: dark brown solution / particles",
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:52.298451315",
+                "loaded": "2023-07-28T10:56:05.304180211",
                 "message": "",
                 "numberOfIndividuals": 19,
                 "numberOfProperties": 605,
                 "numberOfTerms": 19246,
                 "ontologyId": "micro",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:52.298451315",
+                "updated": "2023-07-28T10:56:05.304180211",
                 "version": "MicrO (An Ontology of Prokaryotic Phenotypic and Metabolic Characters).  Version 1.5.1 released 6/14/2018.\n\nIncludes terms and term synonyms extracted from > 3000 prokaryotic taxonomic descriptions, collected from a large number of taxonomic descriptions from Archaea, Cyanobacteria, Bacteroidetes, Firmicutes and Mollicutes.\n\nThe ontology and the synonym lists were developed to facilitate the automated extraction of phenotypic data and character states from prokaryotic taxonomic descriptions using a natural language processing algorithm (MicroPIE).  MicroPIE was developed by Hong Cui, Elvis Hsin-Hui Wu, and Jin Mao (University of Arizona) in collaboration with Carrine E. Blank (University of Montana) and Lisa R. Moore (University of Southern Maine).\n\nDescriptions and links to MicroPIE can be found at http://avatol.org/ngp/nlp/overview-2/.\nhttps://github.com/biosemantics/micropie2\n\nThe most current version of MicrO can be downloaded from https://github.com/carrineblank/MicrO."
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/miro/individuals"
                     },
@@ -7395,22 +7395,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:13.367553027",
+                "loaded": "2023-07-28T10:56:25.560014109",
                 "message": "",
                 "numberOfIndividuals": 126,
                 "numberOfProperties": 40,
                 "numberOfTerms": 4409,
                 "ontologyId": "miro",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:13.367553027",
+                "updated": "2023-07-28T10:56:25.560014109",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mmo/individuals"
                     },
@@ -7453,22 +7453,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:10.439444670",
+                "loaded": "2023-07-28T10:38:10.313962549",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 26,
                 "numberOfTerms": 701,
                 "ontologyId": "mmo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:10.439444670",
+                "updated": "2023-07-28T10:38:10.313962549",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mmusdv/individuals"
                     },
@@ -7511,22 +7511,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:10.614312932",
+                "loaded": "2023-07-28T10:38:10.473879991",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 37,
                 "numberOfTerms": 113,
                 "ontologyId": "mmusdv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:10.614312932",
+                "updated": "2023-07-28T10:38:10.473879991",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mod/individuals"
                     },
@@ -7569,22 +7569,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:11.184199744",
+                "loaded": "2023-07-28T10:38:11.074463475",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 47,
                 "numberOfTerms": 2098,
                 "ontologyId": "mod",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:11.184199744",
+                "updated": "2023-07-28T10:38:11.074463475",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mondo/individuals"
                     },
@@ -7628,22 +7628,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:21.732952897",
+                "loaded": "2023-07-28T10:38:21.365540344",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 620,
                 "numberOfTerms": 45656,
                 "ontologyId": "mondo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:21.732952897",
+                "updated": "2023-07-28T10:38:21.365540344",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mop/individuals"
                     },
@@ -7686,22 +7686,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:36.439091523",
+                "loaded": "2023-07-28T10:38:36.909459527",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 38,
                 "numberOfTerms": 3686,
                 "ontologyId": "mop",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:36.439091523",
+                "updated": "2023-07-28T10:38:36.909459527",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mp/individuals"
                     },
@@ -7744,22 +7744,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:42.261982488",
+                "loaded": "2023-07-28T10:38:42.647120577",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 598,
                 "numberOfTerms": 33483,
                 "ontologyId": "mp",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:42.261982488",
+                "updated": "2023-07-28T10:38:42.647120577",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mpath/individuals"
                     },
@@ -7802,22 +7802,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:52.056059409",
+                "loaded": "2023-07-28T10:38:52.392411709",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 22,
                 "numberOfTerms": 891,
                 "ontologyId": "mpath",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:52.056059409",
+                "updated": "2023-07-28T10:38:52.392411709",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mpio/individuals"
                     },
@@ -7861,22 +7861,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:52.213423205",
+                "loaded": "2023-07-28T10:38:52.571317977",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 67,
                 "numberOfTerms": 51,
                 "ontologyId": "mpio",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:52.213423205",
+                "updated": "2023-07-28T10:38:52.571317977",
                 "version": "release version 2019-01-30"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/mro/individuals"
                     },
@@ -7919,22 +7919,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:48:56.222877786",
+                "loaded": "2023-07-28T10:38:56.659008017",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 41,
                 "numberOfTerms": 51390,
                 "ontologyId": "mro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:48:56.222877786",
+                "updated": "2023-07-28T10:38:56.659008017",
                 "version": "2023-07-07"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ms/individuals"
                     },
@@ -7979,22 +7979,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:49:05.414485612",
+                "loaded": "2023-07-28T10:39:05.896942079",
                 "message": "",
                 "numberOfIndividuals": 14,
                 "numberOfProperties": 681,
                 "numberOfTerms": 15031,
                 "ontologyId": "ms",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:49:05.414485612",
+                "updated": "2023-07-28T10:39:05.896942079",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/msio/individuals"
                     },
@@ -8047,22 +8047,22 @@
                 "lang": "en",
                 "languages": [
                     "de",
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:47.671529237",
+                "loaded": "2023-07-28T11:05:31.065700345",
                 "message": "",
                 "numberOfIndividuals": 33,
                 "numberOfProperties": 133,
                 "numberOfTerms": 1294,
                 "ontologyId": "msio",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:47.671529237",
+                "updated": "2023-07-28T11:05:31.065700345",
                 "version": "1.0.1"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/nbo/individuals"
                     },
@@ -8105,22 +8105,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:49:09.128431248",
+                "loaded": "2023-07-28T10:39:09.603848324",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 326,
                 "numberOfTerms": 4546,
                 "ontologyId": "nbo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:49:09.128431248",
+                "updated": "2023-07-28T10:39:09.603848324",
                 "version": "2023-07-04"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ncbitaxon/individuals"
                     },
@@ -8167,22 +8167,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:50:28.754741132",
+                "loaded": "2023-07-28T10:40:28.661105506",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 30,
                 "numberOfTerms": 2511800,
                 "ontologyId": "ncbitaxon",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:50:28.754741132",
+                "updated": "2023-07-28T10:40:28.661105506",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ncit/individuals"
                     },
@@ -8225,22 +8225,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:56:35.867333221",
+                "loaded": "2023-07-28T10:46:36.927583667",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 247,
                 "numberOfTerms": 174278,
                 "ontologyId": "ncit",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:56:35.867333221",
+                "updated": "2023-07-28T10:46:36.927583667",
                 "version": "22.07d"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ncro/individuals"
                     },
@@ -8283,22 +8283,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:29.492214731",
+                "loaded": "2023-07-28T10:47:33.316017344",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 66,
                 "numberOfTerms": 3078,
                 "ontologyId": "ncro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:29.492214731",
+                "updated": "2023-07-28T10:47:33.316017344",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ngbo/individuals"
                     },
@@ -8342,22 +8342,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:29.930755306",
+                "loaded": "2023-07-28T10:47:33.788414595",
                 "message": "",
                 "numberOfIndividuals": 2,
                 "numberOfProperties": 252,
                 "numberOfTerms": 1514,
                 "ontologyId": "ngbo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:29.930755306",
+                "updated": "2023-07-28T10:47:33.788414595",
                 "version": "2022-10-05"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/nmrcv/individuals"
                     },
@@ -8412,22 +8412,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:30.018660478",
+                "loaded": "2023-07-28T11:10:19.052362869",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 46,
                 "numberOfTerms": 792,
                 "ontologyId": "nmrcv",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:30.018660478",
+                "updated": "2023-07-28T11:10:19.052362869",
                 "version": "1.1.0"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/nomen/individuals"
                     },
@@ -8470,22 +8470,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:30.175668529",
+                "loaded": "2023-07-28T10:47:34.051267642",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 122,
                 "numberOfTerms": 263,
                 "ontologyId": "nomen",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:30.175668529",
+                "updated": "2023-07-28T10:47:34.051267642",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/oae/individuals"
                     },
@@ -8529,22 +8529,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:30.862817438",
+                "loaded": "2023-07-28T10:47:34.752887257",
                 "message": "",
                 "numberOfIndividuals": 4,
                 "numberOfProperties": 236,
                 "numberOfTerms": 10589,
                 "ontologyId": "oae",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:30.862817438",
+                "updated": "2023-07-28T10:47:34.752887257",
                 "version": "1.2.47"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/oarcs/individuals"
                     },
@@ -8587,22 +8587,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:32.378179869",
+                "loaded": "2023-07-28T10:47:36.372080391",
                 "message": "",
                 "numberOfIndividuals": 323,
                 "numberOfProperties": 19,
                 "numberOfTerms": 308,
                 "ontologyId": "oarcs",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:32.378179869",
+                "updated": "2023-07-28T10:47:36.372080391",
                 "version": "2019-04-18"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/oba/individuals"
                     },
@@ -8647,22 +8647,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:38.229998423",
+                "loaded": "2023-07-28T10:47:42.585083049",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 458,
                 "numberOfTerms": 61456,
                 "ontologyId": "oba",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:38.229998423",
+                "updated": "2023-07-28T10:47:42.585083049",
                 "version": "2023-06-27"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/obcs/individuals"
                     },
@@ -8706,22 +8706,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:51.458727724",
+                "loaded": "2023-07-28T10:47:56.603180305",
                 "message": "",
                 "numberOfIndividuals": 25,
                 "numberOfProperties": 75,
                 "numberOfTerms": 810,
                 "ontologyId": "obcs",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:51.458727724",
+                "updated": "2023-07-28T10:47:56.603180305",
                 "version": "101"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/obi/individuals"
                     },
@@ -8766,22 +8766,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:52.007930063",
+                "loaded": "2023-07-28T10:47:57.164871025",
                 "message": "",
                 "numberOfIndividuals": 306,
                 "numberOfProperties": 176,
                 "numberOfTerms": 4887,
                 "ontologyId": "obi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:52.007930063",
+                "updated": "2023-07-28T10:47:57.164871025",
                 "version": "2023-07-25"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/obib/individuals"
                     },
@@ -8824,22 +8824,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:57:53.043346321",
+                "loaded": "2023-07-28T10:47:58.423029312",
                 "message": "",
                 "numberOfIndividuals": 226,
                 "numberOfProperties": 172,
                 "numberOfTerms": 1803,
                 "ontologyId": "obib",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:57:53.043346321",
+                "updated": "2023-07-28T10:47:58.423029312",
                 "version": "2023-04-05"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ogg/individuals"
                     },
@@ -8882,22 +8882,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:07.118716592",
+                "loaded": "2023-07-28T10:48:13.264994941",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 125,
                 "numberOfTerms": 69688,
                 "ontologyId": "ogg",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:07.118716592",
+                "updated": "2023-07-28T10:48:13.264994941",
                 "version": "Vision Release: 1.0.59"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ogms/individuals"
                     },
@@ -8940,22 +8940,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:16.153487978",
+                "loaded": "2023-07-28T10:48:22.704390619",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 59,
                 "numberOfTerms": 186,
                 "ontologyId": "ogms",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:16.153487978",
+                "updated": "2023-07-28T10:48:22.704390619",
                 "version": "2021-08-19"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ogsf/individuals"
                     },
@@ -8998,22 +8998,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:16.260466057",
+                "loaded": "2023-07-28T10:48:22.815247361",
                 "message": "",
                 "numberOfIndividuals": 57,
                 "numberOfProperties": 215,
                 "numberOfTerms": 406,
                 "ontologyId": "ogsf",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:16.260466057",
+                "updated": "2023-07-28T10:48:22.815247361",
                 "version": "2.0"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ohd/individuals"
                     },
@@ -9057,22 +9057,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:16.581310887",
+                "loaded": "2023-07-28T10:48:23.192830687",
                 "message": "",
                 "numberOfIndividuals": 87,
                 "numberOfProperties": 196,
                 "numberOfTerms": 2914,
                 "ontologyId": "ohd",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:16.581310887",
+                "updated": "2023-07-28T10:48:23.192830687",
                 "version": "2023-07-22"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ohmi/individuals"
                     },
@@ -9116,22 +9116,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:17.220605697",
+                "loaded": "2023-07-28T10:48:23.953350012",
                 "message": "",
                 "numberOfIndividuals": 12,
                 "numberOfProperties": 207,
                 "numberOfTerms": 1025,
                 "ontologyId": "ohmi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:17.220605697",
+                "updated": "2023-07-28T10:48:23.953350012",
                 "version": "2019-09-17"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ohpi/individuals"
                     },
@@ -9176,22 +9176,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:18.923603559",
+                "loaded": "2023-07-28T10:48:25.639512911",
                 "message": "",
                 "numberOfIndividuals": 295,
                 "numberOfProperties": 516,
                 "numberOfTerms": 11291,
                 "ontologyId": "ohpi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:18.923603559",
+                "updated": "2023-07-28T10:48:25.639512911",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/oio/individuals"
                     },
@@ -9238,22 +9238,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:45.516627678",
+                "loaded": "2023-07-28T11:05:28.844858950",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 44,
                 "numberOfTerms": 12,
                 "ontologyId": "oio",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:45.516627678",
+                "updated": "2023-07-28T11:05:28.844858950",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/olatdv/individuals"
                     },
@@ -9296,22 +9296,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:21.476604217",
+                "loaded": "2023-07-28T10:48:28.355270922",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 15,
                 "numberOfTerms": 47,
                 "ontologyId": "olatdv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:21.476604217",
+                "updated": "2023-07-28T10:48:28.355270922",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/om/individuals"
                     },
@@ -9372,22 +9372,22 @@
                 "languages": [
                     "en",
                     "ja",
                     "nl",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:44.723216403",
+                "loaded": "2023-07-28T11:10:33.387830636",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 70,
                 "numberOfTerms": 2306,
                 "ontologyId": "om",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:44.723216403",
+                "updated": "2023-07-28T11:10:33.387830636",
                 "version": "2.0.40"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/omiabis/individuals"
                     },
@@ -9431,22 +9431,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en(x,y,t)'."
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:30.379281251",
+                "loaded": "2023-07-28T10:56:43.152238672",
                 "message": "",
                 "numberOfIndividuals": 15,
                 "numberOfProperties": 109,
                 "numberOfTerms": 427,
                 "ontologyId": "omiabis",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:30.379281251",
+                "updated": "2023-07-28T10:56:43.152238672",
                 "version": " 2014-05-28"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/omit/individuals"
                     },
@@ -9489,22 +9489,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:24.206609208",
+                "loaded": "2023-07-28T10:48:31.254590180",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 188,
                 "numberOfTerms": 90928,
                 "ontologyId": "omit",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:24.206609208",
+                "updated": "2023-07-28T10:48:31.254590180",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/omo/individuals"
                     },
@@ -9547,22 +9547,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:32.529380643",
+                "loaded": "2023-07-28T10:48:39.731097352",
                 "message": "",
                 "numberOfIndividuals": 19,
                 "numberOfProperties": 69,
                 "numberOfTerms": 28,
                 "ontologyId": "omo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:32.529380643",
+                "updated": "2023-07-28T10:48:39.731097352",
                 "version": "2023-06-09"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/omp/individuals"
                     },
@@ -9605,22 +9605,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:32.788424881",
+                "loaded": "2023-07-28T10:48:39.977510531",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 131,
                 "numberOfTerms": 2309,
                 "ontologyId": "omp",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:32.788424881",
+                "updated": "2023-07-28T10:48:39.977510531",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/omrse/individuals"
                     },
@@ -9663,22 +9663,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:33.286560423",
+                "loaded": "2023-07-28T10:48:40.473097938",
                 "message": "",
                 "numberOfIndividuals": 20,
                 "numberOfProperties": 292,
                 "numberOfTerms": 646,
                 "ontologyId": "omrse",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:33.286560423",
+                "updated": "2023-07-28T10:48:40.473097938",
                 "version": "2023-05-19"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/one/individuals"
                     },
@@ -9722,22 +9722,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:33.814638778",
+                "loaded": "2023-07-28T10:48:40.996761057",
                 "message": "",
                 "numberOfIndividuals": 326,
                 "numberOfProperties": 318,
                 "numberOfTerms": 4100,
                 "ontologyId": "one",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:33.814638778",
+                "updated": "2023-07-28T10:48:40.996761057",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ons/individuals"
                     },
@@ -9792,22 +9792,22 @@
                     "es",
                     "fr",
                     "haw",
                     "ml",
                     "th"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:35.082804147",
+                "loaded": "2023-07-28T10:48:42.241785112",
                 "message": "",
                 "numberOfIndividuals": 153,
                 "numberOfProperties": 395,
                 "numberOfTerms": 8025,
                 "ontologyId": "ons",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:35.082804147",
+                "updated": "2023-07-28T10:48:42.241785112",
                 "version": "Release version 1.2.3"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ontoavida/individuals"
                     },
@@ -9850,22 +9850,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:36.991777143",
+                "loaded": "2023-07-28T10:48:43.414431881",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 58,
                 "numberOfTerms": 674,
                 "ontologyId": "ontoavida",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:36.991777143",
+                "updated": "2023-07-28T10:48:43.414431881",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ontoneo/individuals"
                     },
@@ -9910,22 +9910,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us",
                     "pt"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:37.526770800",
+                "loaded": "2023-07-28T10:48:43.907023692",
                 "message": "",
                 "numberOfIndividuals": 17,
                 "numberOfProperties": 950,
                 "numberOfTerms": 2120,
                 "ontologyId": "ontoneo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:37.526770800",
+                "updated": "2023-07-28T10:48:43.907023692",
                 "version": "2021-04-29"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/oostt/individuals"
                     },
@@ -9968,22 +9968,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:38.002401149",
+                "loaded": "2023-07-28T10:48:44.390253302",
                 "message": "",
                 "numberOfIndividuals": 22,
                 "numberOfProperties": 190,
                 "numberOfTerms": 717,
                 "ontologyId": "oostt",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:38.002401149",
+                "updated": "2023-07-28T10:48:44.390253302",
                 "version": "release version - 2023-03-30"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/opl/individuals"
                     },
@@ -10026,22 +10026,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:38.187841628",
+                "loaded": "2023-07-28T10:48:44.583236395",
                 "message": "",
                 "numberOfIndividuals": 19,
                 "numberOfProperties": 119,
                 "numberOfTerms": 515,
                 "ontologyId": "opl",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:38.187841628",
+                "updated": "2023-07-28T10:48:44.583236395",
                 "version": "2023-06-23"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/opmi/individuals"
                     },
@@ -10087,22 +10087,22 @@
                 "languages": [
                     "de",
                     "en",
                     "en-us",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:38.553879732",
+                "loaded": "2023-07-28T10:48:44.927179129",
                 "message": "",
                 "numberOfIndividuals": 36,
                 "numberOfProperties": 282,
                 "numberOfTerms": 3149,
                 "ontologyId": "opmi",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:38.553879732",
+                "updated": "2023-07-28T10:48:44.927179129",
                 "version": "Vision Release: 1.0.161"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ordo/individuals"
                     },
@@ -10158,22 +10158,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:14:17.627784255",
+                "loaded": "2023-07-28T11:05:58.878104537",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 33,
                 "numberOfTerms": 15447,
                 "ontologyId": "ordo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:14:17.627784255",
+                "updated": "2023-07-28T11:05:58.878104537",
                 "version": "4.3"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ornaseq/individuals"
                     },
@@ -10216,22 +10216,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:39.041106037",
+                "loaded": "2023-07-28T10:48:45.408306948",
                 "message": "",
                 "numberOfIndividuals": 1,
                 "numberOfProperties": 17,
                 "numberOfTerms": 163,
                 "ontologyId": "ornaseq",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:39.041106037",
+                "updated": "2023-07-28T10:48:45.408306948",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/orth/individuals"
                     },
@@ -10285,22 +10285,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:46.645726551",
+                "loaded": "2023-07-28T11:10:35.192582342",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 39,
                 "numberOfTerms": 33,
                 "ontologyId": "orth",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:46.645726551",
+                "updated": "2023-07-28T11:10:35.192582342",
                 "version": "This ontology is the second version of the orthology ontology first published in 2015 and available at http://purl.bioontology.org/ontology/ORTH .\n@authors T.M. de Farias, H. Chiba, J.T. Fernandez-Breis, E. Antezana, D. Kuznetsov, C. Dessimoz and F. Villiers."
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ovae/individuals"
                     },
@@ -10344,22 +10344,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:39.349860024",
+                "loaded": "2023-07-28T10:48:45.700491848",
                 "message": "",
                 "numberOfIndividuals": 22,
                 "numberOfProperties": 247,
                 "numberOfTerms": 1554,
                 "ontologyId": "ovae",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:39.349860024",
+                "updated": "2023-07-28T10:48:45.700491848",
                 "version": "Vision Release: 1.0.34"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/owl/individuals"
                     },
@@ -10406,22 +10406,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:46.669546150",
+                "loaded": "2023-07-28T11:05:29.984605331",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 58,
                 "numberOfTerms": 32,
                 "ontologyId": "owl",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:46.669546150",
+                "updated": "2023-07-28T11:05:29.984605331",
                 "version": "$Date: 2009/11/15 10:54:12 $"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pato/individuals"
                     },
@@ -10464,22 +10464,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:41.460724948",
+                "loaded": "2023-07-28T10:48:47.699649801",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 526,
                 "numberOfTerms": 10588,
                 "ontologyId": "pato",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:41.460724948",
+                "updated": "2023-07-28T10:48:47.699649801",
                 "version": "2023-05-18"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pcl/individuals"
                     },
@@ -10530,22 +10530,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:58:49.121618315",
+                "loaded": "2023-07-28T10:48:55.005878635",
                 "message": "",
                 "numberOfIndividuals": 1825,
                 "numberOfProperties": 554,
                 "numberOfTerms": 121028,
                 "ontologyId": "pcl",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:58:49.121618315",
+                "updated": "2023-07-28T10:48:55.005878635",
                 "version": "2023-02-27"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pco/individuals"
                     },
@@ -10588,22 +10588,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:01.989942959",
+                "loaded": "2023-07-28T10:49:08.030593976",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 197,
                 "numberOfTerms": 220,
                 "ontologyId": "pco",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:01.989942959",
+                "updated": "2023-07-28T10:49:08.030593976",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pdro/individuals"
                     },
@@ -10647,22 +10647,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:02.099497198",
+                "loaded": "2023-07-28T10:49:08.149919542",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 93,
                 "numberOfTerms": 310,
                 "ontologyId": "pdro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:02.099497198",
+                "updated": "2023-07-28T10:49:08.149919542",
                 "version": "2021-06-08"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pdumdv/individuals"
                     },
@@ -10705,22 +10705,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:02.211157706",
+                "loaded": "2023-07-28T10:49:08.269874264",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 17,
                 "numberOfTerms": 21,
                 "ontologyId": "pdumdv",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:02.211157706",
+                "updated": "2023-07-28T10:49:08.269874264",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/peco/individuals"
                     },
@@ -10763,22 +10763,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:02.776769379",
+                "loaded": "2023-07-28T10:49:08.815769656",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 182,
                 "numberOfTerms": 3550,
                 "ontologyId": "peco",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:02.776769379",
+                "updated": "2023-07-28T10:49:08.815769656",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/phipo/individuals"
                     },
@@ -10821,22 +10821,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:04.139621612",
+                "loaded": "2023-07-28T10:49:10.128520001",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 308,
                 "numberOfTerms": 3865,
                 "ontologyId": "phipo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:04.139621612",
+                "updated": "2023-07-28T10:49:10.128520001",
                 "version": "2023-07-06"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/plana/individuals"
                     },
@@ -10881,22 +10881,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:05.507853660",
+                "loaded": "2023-07-28T10:49:11.503265401",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 291,
                 "numberOfTerms": 875,
                 "ontologyId": "plana",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:05.507853660",
+                "updated": "2023-07-28T10:49:11.503265401",
                 "version": "2023-03-13"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/planp/individuals"
                     },
@@ -10939,22 +10939,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:06.317125442",
+                "loaded": "2023-07-28T10:49:12.292753554",
                 "message": "",
                 "numberOfIndividuals": 19,
                 "numberOfProperties": 324,
                 "numberOfTerms": 3560,
                 "ontologyId": "planp",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:06.317125442",
+                "updated": "2023-07-28T10:49:12.292753554",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/po/individuals"
                     },
@@ -10997,22 +10997,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:07.701252123",
+                "loaded": "2023-07-28T10:49:13.627986831",
                 "message": "",
                 "numberOfIndividuals": 9,
                 "numberOfProperties": 207,
                 "numberOfTerms": 2026,
                 "ontologyId": "po",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:07.701252123",
+                "updated": "2023-07-28T10:49:13.627986831",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/poro/individuals"
                     },
@@ -11056,22 +11056,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:08.361923706",
+                "loaded": "2023-07-28T10:49:14.324921958",
                 "message": "",
                 "numberOfIndividuals": 2,
                 "numberOfProperties": 142,
                 "numberOfTerms": 932,
                 "ontologyId": "poro",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:08.361923706",
+                "updated": "2023-07-28T10:49:14.324921958",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ppo/individuals"
                     },
@@ -11115,22 +11115,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T16:59:08.588626010",
+                "loaded": "2023-07-28T10:49:14.542649066",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 101,
                 "numberOfTerms": 443,
                 "ontologyId": "ppo",
                 "status": "LOADED",
-                "updated": "2023-07-26T16:59:08.588626010",
+                "updated": "2023-07-28T10:49:14.542649066",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pr/individuals"
                     },
@@ -11173,22 +11173,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:00:17.672864867",
+                "loaded": "2023-07-28T10:50:18.170785692",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 56,
                 "numberOfTerms": 334852,
                 "ontologyId": "pr",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:00:17.672864867",
+                "updated": "2023-07-28T10:50:18.170785692",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/probonto/individuals"
                     },
@@ -11237,22 +11237,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:26.261148027",
+                "loaded": "2023-07-28T11:10:15.952098521",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 44,
                 "numberOfTerms": 39,
                 "ontologyId": "probonto",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:26.261148027",
+                "updated": "2023-07-28T11:10:15.952098521",
                 "version": "2.5.0"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/proco/individuals"
                     },
@@ -11296,22 +11296,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:02:59.311060093",
+                "loaded": "2023-07-28T10:53:12.067814636",
                 "message": "",
                 "numberOfIndividuals": 17,
                 "numberOfProperties": 149,
                 "numberOfTerms": 922,
                 "ontologyId": "proco",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:02:59.311060093",
+                "updated": "2023-07-28T10:53:12.067814636",
                 "version": "PROCO release 20220414"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/prov/individuals"
                     },
@@ -11366,22 +11366,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:45.076785176",
+                "loaded": "2023-07-28T11:10:33.763294176",
                 "message": "",
                 "numberOfIndividuals": 1,
                 "numberOfProperties": 72,
                 "numberOfTerms": 31,
                 "ontologyId": "prov",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:45.076785176",
+                "updated": "2023-07-28T11:10:33.763294176",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/psdo/individuals"
                     },
@@ -11426,22 +11426,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:02:59.598166741",
+                "loaded": "2023-07-28T10:53:14.543481321",
                 "message": "",
                 "numberOfIndividuals": 37,
                 "numberOfProperties": 268,
                 "numberOfTerms": 1205,
                 "ontologyId": "psdo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:02:59.598166741",
+                "updated": "2023-07-28T10:53:14.543481321",
                 "version": "1.0.1"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pso/individuals"
                     },
@@ -11484,22 +11484,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:00.111557689",
+                "loaded": "2023-07-28T10:53:15.076789214",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 160,
                 "numberOfTerms": 3762,
                 "ontologyId": "pso",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:00.111557689",
+                "updated": "2023-07-28T10:53:15.076789214",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/pw/individuals"
                     },
@@ -11542,22 +11542,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:00.838265636",
+                "loaded": "2023-07-28T10:53:15.932234507",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 20,
                 "numberOfTerms": 2642,
                 "ontologyId": "pw",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:00.838265636",
+                "updated": "2023-07-28T10:53:15.932234507",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/rbo/individuals"
                     },
@@ -11602,22 +11602,22 @@
                 "lang": "en",
                 "languages": [
                     "de",
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:02.164483326",
+                "loaded": "2023-07-28T10:53:17.355802185",
                 "message": "",
                 "numberOfIndividuals": 351,
                 "numberOfProperties": 496,
                 "numberOfTerms": 5484,
                 "ontologyId": "rbo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:02.164483326",
+                "updated": "2023-07-28T10:53:17.355802185",
                 "version": "2023-06-15"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/rdfs/individuals"
                     },
@@ -11664,22 +11664,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:46.085599803",
+                "loaded": "2023-07-28T11:05:29.407392783",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 9,
                 "numberOfTerms": 6,
                 "ontologyId": "rdfs",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:46.085599803",
+                "updated": "2023-07-28T11:05:29.407392783",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/reproduceme/individuals"
                     },
@@ -11736,22 +11736,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:44.951115066",
+                "loaded": "2023-07-28T11:10:33.628443713",
                 "message": "",
                 "numberOfIndividuals": 255,
                 "numberOfProperties": 209,
                 "numberOfTerms": 389,
                 "ontologyId": "reproduceme",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:44.951115066",
+                "updated": "2023-07-28T11:10:33.628443713",
                 "version": "1.1"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/reto/individuals"
                     },
@@ -11802,22 +11802,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:17:36.880605897",
+                "loaded": "2023-07-28T11:09:24.291013558",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 12,
                 "numberOfTerms": 147738,
                 "ontologyId": "reto",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:17:36.880605897",
+                "updated": "2023-07-28T11:09:24.291013558",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/rex/individuals"
                     },
@@ -11860,22 +11860,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:48.605519170",
+                "loaded": "2023-07-28T10:56:01.913646321",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 17,
                 "numberOfTerms": 552,
                 "ontologyId": "rex",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:48.605519170",
+                "updated": "2023-07-28T10:56:01.913646321",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/rexo/individuals"
                     },
@@ -11926,22 +11926,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:17:12.274886560",
+                "loaded": "2023-07-28T11:08:59.990091919",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 12,
                 "numberOfTerms": 158239,
                 "ontologyId": "rexo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:17:12.274886560",
+                "updated": "2023-07-28T11:08:59.990091919",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/ro/individuals"
                     },
@@ -11990,22 +11990,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:05.345497100",
+                "loaded": "2023-07-28T10:53:19.104243146",
                 "message": "",
                 "numberOfIndividuals": 24,
                 "numberOfProperties": 792,
                 "numberOfTerms": 47,
                 "ontologyId": "ro",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:05.345497100",
+                "updated": "2023-07-28T10:53:19.104243146",
                 "version": "2023-06-28"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/rs/individuals"
                     },
@@ -12048,22 +12048,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:05.697329153",
+                "loaded": "2023-07-28T10:53:19.455677909",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 18,
                 "numberOfTerms": 4768,
                 "ontologyId": "rs",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:05.697329153",
+                "updated": "2023-07-28T10:53:19.455677909",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/rxno/individuals"
                     },
@@ -12106,22 +12106,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:06.407182402",
+                "loaded": "2023-07-28T10:53:20.262497896",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 49,
                 "numberOfTerms": 1019,
                 "ontologyId": "rxno",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:06.407182402",
+                "updated": "2023-07-28T10:53:20.262497896",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/sbo/individuals"
                     },
@@ -12166,22 +12166,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:44.981167653",
+                "loaded": "2023-07-28T10:55:58.456966842",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 1,
                 "numberOfTerms": 695,
                 "ontologyId": "sbo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:44.981167653",
+                "updated": "2023-07-28T10:55:58.456966842",
                 "version": "28:08:2021 03:13"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/scdo/individuals"
                     },
@@ -12224,22 +12224,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:45.269679636",
+                "loaded": "2023-07-28T10:55:58.735848260",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 145,
                 "numberOfTerms": 2071,
                 "ontologyId": "scdo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:45.269679636",
+                "updated": "2023-07-28T10:55:58.735848260",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/schemaorg_http/individuals"
                     },
@@ -12286,22 +12286,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:47.013228485",
+                "loaded": "2023-07-28T11:05:30.390015095",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 1465,
                 "numberOfTerms": 903,
                 "ontologyId": "schemaorg_http",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:47.013228485",
+                "updated": "2023-07-28T11:05:30.390015095",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/schemaorg_https/individuals"
                     },
@@ -12348,22 +12348,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:47.374246042",
+                "loaded": "2023-07-28T11:05:30.771636166",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 1465,
                 "numberOfTerms": 903,
                 "ontologyId": "schemaorg_https",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:47.374246042",
+                "updated": "2023-07-28T11:05:30.771636166",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/sdgio/individuals"
                     },
@@ -12414,22 +12414,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:53.484839446",
+                "loaded": "2023-07-28T11:05:35.072377768",
                 "message": "",
                 "numberOfIndividuals": 470,
                 "numberOfProperties": 194,
                 "numberOfTerms": 906,
                 "ontologyId": "sdgio",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:53.484839446",
+                "updated": "2023-07-28T11:05:35.072377768",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/semapv/individuals"
                     },
@@ -12478,22 +12478,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:31.476414210",
+                "loaded": "2023-07-28T10:56:44.245276556",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 20,
                 "numberOfTerms": 47,
                 "ontologyId": "semapv",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:31.476414210",
+                "updated": "2023-07-28T10:56:44.245276556",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/sepio/individuals"
                     },
@@ -12536,22 +12536,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:06.613073330",
+                "loaded": "2023-07-28T10:53:20.470571857",
                 "message": "",
                 "numberOfIndividuals": 21,
                 "numberOfProperties": 313,
                 "numberOfTerms": 125,
                 "ontologyId": "sepio",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:06.613073330",
+                "updated": "2023-07-28T10:53:20.470571857",
                 "version": "2023-06-13"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/shareloc/individuals"
                     },
@@ -12594,22 +12594,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:20:17.936827017",
+                "loaded": "2023-07-28T11:12:11.939124864",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 0,
                 "numberOfTerms": 95,
                 "ontologyId": "shareloc",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:20:17.936827017",
+                "updated": "2023-07-28T11:12:11.939124864",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/sibo/individuals"
                     },
@@ -12652,22 +12652,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:05:48.782609485",
+                "loaded": "2023-07-28T10:56:02.088783732",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 20,
                 "numberOfTerms": 525,
                 "ontologyId": "sibo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:05:48.782609485",
+                "updated": "2023-07-28T10:56:02.088783732",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/sio/individuals"
                     },
@@ -12722,22 +12722,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:29.529738200",
+                "loaded": "2023-07-28T11:10:18.607069702",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 238,
                 "numberOfTerms": 1584,
                 "ontologyId": "sio",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:29.529738200",
+                "updated": "2023-07-28T11:10:18.607069702",
                 "version": "1.59"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/skos/individuals"
                     },
@@ -12784,22 +12784,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:13:46.738363893",
+                "loaded": "2023-07-28T11:05:30.078146208",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 56,
                 "numberOfTerms": 6,
                 "ontologyId": "skos",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:13:46.738363893",
+                "updated": "2023-07-28T11:05:30.078146208",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/slm/individuals"
                     },
@@ -12846,22 +12846,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:07:54.062931304",
+                "loaded": "2023-07-28T10:58:07.707935897",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 50,
                 "numberOfTerms": 962502,
                 "ontologyId": "slm",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:07:54.062931304",
+                "updated": "2023-07-28T10:58:07.707935897",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/snomed/individuals"
                     },
@@ -12914,22 +12914,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-gb",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:19:12.563409429",
+                "loaded": "2023-07-28T11:11:02.443081647",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 124,
                 "numberOfTerms": 352449,
                 "ontologyId": "snomed",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:19:12.563409429",
+                "updated": "2023-07-28T11:11:02.443081647",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/so/individuals"
                     },
@@ -12972,22 +12972,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:06.986866467",
+                "loaded": "2023-07-28T10:53:20.862306833",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 91,
                 "numberOfTerms": 2729,
                 "ontologyId": "so",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:06.986866467",
+                "updated": "2023-07-28T10:53:20.862306833",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/spd/individuals"
                     },
@@ -13030,22 +13030,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:07.516286145",
+                "loaded": "2023-07-28T10:53:21.483877829",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 22,
                 "numberOfTerms": 832,
                 "ontologyId": "spd",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:07.516286145",
+                "updated": "2023-07-28T10:53:21.483877829",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/srao/individuals"
                     },
@@ -13105,22 +13105,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:18:31.187782266",
+                "loaded": "2023-07-28T11:10:20.200325847",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 17,
                 "numberOfTerms": 435,
                 "ontologyId": "srao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:18:31.187782266",
+                "updated": "2023-07-28T11:10:20.200325847",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/stato/individuals"
                     },
@@ -13165,22 +13165,22 @@
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us",
                     "fr"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:07.740173164",
+                "loaded": "2023-07-28T10:53:21.703821347",
                 "message": "",
                 "numberOfIndividuals": 14,
                 "numberOfProperties": 113,
                 "numberOfTerms": 889,
                 "ontologyId": "stato",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:07.740173164",
+                "updated": "2023-07-28T10:53:21.703821347",
                 "version": "RC1.4"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/swo/individuals"
                     },
@@ -13224,22 +13224,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:08.131154600",
+                "loaded": "2023-07-28T10:53:22.117978889",
                 "message": "",
                 "numberOfIndividuals": 446,
                 "numberOfProperties": 240,
                 "numberOfTerms": 1971,
                 "ontologyId": "swo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:08.131154600",
+                "updated": "2023-07-28T10:53:22.117978889",
                 "version": "2023-03-05"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/symp/individuals"
                     },
@@ -13282,22 +13282,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:08.582652571",
+                "loaded": "2023-07-28T10:53:22.563615042",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 23,
                 "numberOfTerms": 1013,
                 "ontologyId": "symp",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:08.582652571",
+                "updated": "2023-07-28T10:53:22.563615042",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/t4fs/individuals"
                     },
@@ -13340,22 +13340,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:08.757078982",
+                "loaded": "2023-07-28T10:53:22.744445986",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 62,
                 "numberOfTerms": 563,
                 "ontologyId": "t4fs",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:08.757078982",
+                "updated": "2023-07-28T10:53:22.744445986",
                 "version": "2023-03-02"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/tads/individuals"
                     },
@@ -13398,22 +13398,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:14.243641171",
+                "loaded": "2023-07-28T10:56:26.446923461",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 17,
                 "numberOfTerms": 628,
                 "ontologyId": "tads",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:14.243641171",
+                "updated": "2023-07-28T10:56:26.446923461",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/tao/individuals"
                     },
@@ -13456,22 +13456,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:30.770213437",
+                "loaded": "2023-07-28T10:56:43.524906252",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 49,
                 "numberOfTerms": 3372,
                 "ontologyId": "tao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:30.770213437",
+                "updated": "2023-07-28T10:56:43.524906252",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/taxrank/individuals"
                     },
@@ -13514,22 +13514,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:08.905820509",
+                "loaded": "2023-07-28T10:53:22.913516741",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 11,
                 "numberOfTerms": 61,
                 "ontologyId": "taxrank",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:08.905820509",
+                "updated": "2023-07-28T10:53:22.913516741",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/tgma/individuals"
                     },
@@ -13572,22 +13572,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:14.758194178",
+                "loaded": "2023-07-28T10:56:26.923926840",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 25,
                 "numberOfTerms": 1864,
                 "ontologyId": "tgma",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:14.758194178",
+                "updated": "2023-07-28T10:56:26.923926840",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/to/individuals"
                     },
@@ -13630,22 +13630,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:09.764211786",
+                "loaded": "2023-07-28T10:53:23.875754075",
                 "message": "",
                 "numberOfIndividuals": 9,
                 "numberOfProperties": 446,
                 "numberOfTerms": 6736,
                 "ontologyId": "to",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:09.764211786",
+                "updated": "2023-07-28T10:53:23.875754075",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/trans/individuals"
                     },
@@ -13688,22 +13688,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:11.035511262",
+                "loaded": "2023-07-28T10:53:25.368057755",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 16,
                 "numberOfTerms": 35,
                 "ontologyId": "trans",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:11.035511262",
+                "updated": "2023-07-28T10:53:25.368057755",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/tto/individuals"
                     },
@@ -13746,22 +13746,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:14.918714870",
+                "loaded": "2023-07-28T10:53:29.020506187",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 27,
                 "numberOfTerms": 38705,
                 "ontologyId": "tto",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:14.918714870",
+                "updated": "2023-07-28T10:53:29.020506187",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/txpo/individuals"
                     },
@@ -13805,22 +13805,22 @@
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en",
                     "en-us"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:22.064236319",
+                "loaded": "2023-07-28T10:53:36.724430233",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 148,
                 "numberOfTerms": 9406,
                 "ontologyId": "txpo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:22.064236319",
+                "updated": "2023-07-28T10:53:36.724430233",
                 "version": "2022/12/07"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/uberon/individuals"
                     },
@@ -13863,22 +13863,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:28.429990135",
+                "loaded": "2023-07-28T10:53:43.159287651",
                 "message": "",
                 "numberOfIndividuals": 9,
                 "numberOfProperties": 627,
                 "numberOfTerms": 26232,
                 "ontologyId": "uberon",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:28.429990135",
+                "updated": "2023-07-28T10:53:43.159287651",
                 "version": "2023-07-25"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/uo/individuals"
                     },
@@ -13921,22 +13921,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:36.865602800",
+                "loaded": "2023-07-28T10:53:52.538793212",
                 "message": "",
                 "numberOfIndividuals": 275,
                 "numberOfProperties": 4,
                 "numberOfTerms": 564,
                 "ontologyId": "uo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:36.865602800",
+                "updated": "2023-07-28T10:53:52.538793212",
                 "version": "2023-05-25"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/upa/individuals"
                     },
@@ -13979,22 +13979,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:16.484021408",
+                "loaded": "2023-07-28T10:56:28.542439509",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 142,
                 "numberOfTerms": 4672,
                 "ontologyId": "upa",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:16.484021408",
+                "updated": "2023-07-28T10:56:28.542439509",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/vbo/individuals"
                     },
@@ -14037,22 +14037,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:45.420621800",
+                "loaded": "2023-07-28T10:54:00.434638451",
                 "message": "",
                 "numberOfIndividuals": 216,
                 "numberOfProperties": 226,
                 "numberOfTerms": 18757,
                 "ontologyId": "vbo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:45.420621800",
+                "updated": "2023-07-28T10:54:00.434638451",
                 "version": "2023-06-02"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/vo/individuals"
                     },
@@ -14098,22 +14098,22 @@
                 "languages": [
                     "en",
                     "en-us",
                     "pt",
                     "zh"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:51.730648463",
+                "loaded": "2023-07-28T10:54:07.150408354",
                 "message": "",
                 "numberOfIndividuals": 167,
                 "numberOfProperties": 342,
                 "numberOfTerms": 6947,
                 "ontologyId": "vo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:51.730648463",
+                "updated": "2023-07-28T10:54:07.150408354",
                 "version": "1.1.218"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/vsao/individuals"
                     },
@@ -14156,22 +14156,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:06:31.392584115",
+                "loaded": "2023-07-28T10:56:44.157311663",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 42,
                 "numberOfTerms": 314,
                 "ontologyId": "vsao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:06:31.392584115",
+                "updated": "2023-07-28T10:56:44.157311663",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/vt/individuals"
                     },
@@ -14214,22 +14214,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:03:53.114087519",
+                "loaded": "2023-07-28T10:54:08.529613043",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 20,
                 "numberOfTerms": 3820,
                 "ontologyId": "vt",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:03:53.114087519",
+                "updated": "2023-07-28T10:54:08.529613043",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/vto/individuals"
                     },
@@ -14272,22 +14272,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:01.738613173",
+                "loaded": "2023-07-28T10:54:16.573232419",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 27,
                 "numberOfTerms": 107254,
                 "ontologyId": "vto",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:01.738613173",
+                "updated": "2023-07-28T10:54:16.573232419",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/wbbt/individuals"
                     },
@@ -14330,22 +14330,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:18.478796792",
+                "loaded": "2023-07-28T10:54:33.044950214",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 179,
                 "numberOfTerms": 7572,
                 "ontologyId": "wbbt",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:18.478796792",
+                "updated": "2023-07-28T10:54:33.044950214",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/wbls/individuals"
                     },
@@ -14388,22 +14388,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:19.824596600",
+                "loaded": "2023-07-28T10:54:34.388372529",
                 "message": "",
                 "numberOfIndividuals": 9,
                 "numberOfProperties": 184,
                 "numberOfTerms": 796,
                 "ontologyId": "wbls",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:19.824596600",
+                "updated": "2023-07-28T10:54:34.388372529",
                 "version": "2023-07-10"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/wbphenotype/individuals"
                     },
@@ -14446,22 +14446,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:22.194658643",
+                "loaded": "2023-07-28T10:54:36.612270327",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 533,
                 "numberOfTerms": 12727,
                 "ontologyId": "wbphenotype",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:22.194658643",
+                "updated": "2023-07-28T10:54:36.612270327",
                 "version": "2023-07-10"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/xao/individuals"
                     },
@@ -14504,22 +14504,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:26.102475332",
+                "loaded": "2023-07-28T10:54:40.591885950",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 42,
                 "numberOfTerms": 1775,
                 "ontologyId": "xao",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:26.102475332",
+                "updated": "2023-07-28T10:54:40.591885950",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/xco/individuals"
                     },
@@ -14562,22 +14562,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:26.618328681",
+                "loaded": "2023-07-28T10:54:41.102344339",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 25,
                 "numberOfTerms": 690,
                 "ontologyId": "xco",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:26.618328681",
+                "updated": "2023-07-28T10:54:41.102344339",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/xlmod/individuals"
                     },
@@ -14620,22 +14620,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:26.872792440",
+                "loaded": "2023-07-28T10:54:41.362484295",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 45,
                 "numberOfTerms": 1102,
                 "ontologyId": "xlmod",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:26.872792440",
+                "updated": "2023-07-28T10:54:41.362484295",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/xpo/individuals"
                     },
@@ -14678,22 +14678,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:32.760498001",
+                "loaded": "2023-07-28T10:54:46.896196384",
                 "message": "",
                 "numberOfIndividuals": 17,
                 "numberOfProperties": 494,
                 "numberOfTerms": 30655,
                 "ontologyId": "xpo",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:32.760498001",
+                "updated": "2023-07-28T10:54:46.896196384",
                 "version": "2022-09-15"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/zeco/individuals"
                     },
@@ -14736,22 +14736,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:43.963927665",
+                "loaded": "2023-07-28T10:54:59.757838611",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 44,
                 "numberOfTerms": 161,
                 "ontologyId": "zeco",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:43.963927665",
+                "updated": "2023-07-28T10:54:59.757838611",
                 "version": "2022-02-14"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/zfa/individuals"
                     },
@@ -14794,22 +14794,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:44.504351911",
+                "loaded": "2023-07-28T10:55:00.253460254",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 40,
                 "numberOfTerms": 3271,
                 "ontologyId": "zfa",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:44.504351911",
+                "updated": "2023-07-28T10:55:00.253460254",
                 "version": "2022-12-09"
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/zfs/individuals"
                     },
@@ -14852,22 +14852,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:46.314591228",
+                "loaded": "2023-07-28T10:55:01.233578415",
                 "message": "",
                 "numberOfIndividuals": 0,
                 "numberOfProperties": 21,
                 "numberOfTerms": 54,
                 "ontologyId": "zfs",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:46.314591228",
+                "updated": "2023-07-28T10:55:01.233578415",
                 "version": null
             },
             {
                 "_links": {
                     "individuals": {
                         "href": "https://www.ebi.ac.uk/ols4/api/ontologies/zp/individuals"
                     },
@@ -14910,22 +14910,22 @@
                 },
                 "fileHash": null,
                 "lang": "en",
                 "languages": [
                     "en"
                 ],
                 "loadAttempts": 0,
-                "loaded": "2023-07-26T17:04:57.885683480",
+                "loaded": "2023-07-28T10:55:12.087168317",
                 "message": "",
                 "numberOfIndividuals": 18,
                 "numberOfProperties": 492,
                 "numberOfTerms": 63471,
                 "ontologyId": "zp",
                 "status": "LOADED",
-                "updated": "2023-07-26T17:04:57.885683480",
+                "updated": "2023-07-28T10:55:12.087168317",
                 "version": "2022-12-17"
             }
         ]
     },
     "_links": {
         "self": {
             "href": "https://www.ebi.ac.uk/ols4/api/ontologies?page=0&size=1000"
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ontobee/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/ontobee/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ontobee/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/ontobee/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/ontobee/raw.html` & `bioregistry-0.9.91/src/bioregistry/data/external/ontobee/raw.html`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/prefixcommons/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/prefixcommons/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/prefixcommons/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/prefixcommons/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/prefixcommons/raw.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/prefixcommons/raw.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/re3data/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/re3data/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/re3data/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/re3data/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/uniprot/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/uniprot/curation.tsv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/uniprot/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/uniprot/processed.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/uniprot/raw.json` & `bioregistry-0.9.91/src/bioregistry/data/external/uniprot/raw.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/wikidata/curation.tsv` & `bioregistry-0.9.91/src/bioregistry/data/external/wikidata/curation.tsv`

 * *Files 0% similar despite different names*

```diff
@@ -121,15 +121,15 @@
 P2007	ZooBank publication ID	http://zoobank.org/	identifier for a publication at ZooBank	https://zoobank.org/References/$1	00A7FB9A-757E-46E2-807F-11F94B6D2020|CC7DC93E-9BB6-4F1B-B96F-FD929E6FE1FD
 P2008	IPNI publication ID	https://www.ipni.org/	identifier for a publication in the International Plant Names Index	https://www.ipni.org/p/$1	9105-2
 P2017	isomeric SMILES		dedicated SMILES for isomer	http://www.simolecule.com/cdkdepict/depict/bow/svg?smi=$1&zoom=2.0&annotate=cip	O=C(O)[C@@H](N)C|[37S]
 P2024	German cattle breed ID	http://www.gesetze-im-internet.de/viehverkv_2007/anlage_6.html	identifier of a cattle breed in German national law		71|89
 P2026	Avibase ID	https://avibase.bsc-eoc.org/avibase.jsp	identifier for a species, subspecies, or genus in the global bird database Avibase	https://avibase.bsc-eoc.org/species.jsp?avibaseid=$1	240E33900CE34D44|68DF97EF92389F4D|D3FDB845E68693CE
 P2027	Colour Index International constitution ID		identifier issued by Colour Index International, for manufactured colour products		73000
 P2036	African Plant Database ID	https://africanplantdatabase.ch/	identifier for a plant taxon, in the Conservatoire et Jardin botaniques de Genève's African Plant Database of scientific names	https://africanplantdatabase.ch/en/nomen/$1	177173|51051
-P2040	CITES Species+ ID	http://www.speciesplus.net/	identifier for a taxon in the Species+ database (CITES)	https://speciesplus.net/#/taxon_concepts/$1/legal	111|5541
+P2040	CITES Species+ ID	http://www.speciesplus.net/	identifier for a taxon in the Species+ database (CITES)	https://speciesplus.net/#/taxon_concepts/$1	111|5541
 P2065	NIAID ChemDB ID		ID in NIAID ChemDB	http://chemdb.niaid.nih.gov/CompoundDetails.aspx?AIDSNO=$1	112081
 P2070	Fellow of the Royal Society ID	https://royalsociety.org/	Fellow ID of the Royal Society	https://royalsociety.org/people/$1/	10960|11074|12010|12229|12897
 P2072	CDB Chemical ID		ID in UCI chemical database	http://cdb.ics.uci.edu/cgibin/ChemicalDetailWeb.py?chemical_id=$1	5320420|6692143
 P2074	internetmedicin.se ID	http://www.internetmedicin.se	ID in the Swedish database about medical topics	http://www.internetmedicin.se/page.aspx?id=$1	3911
 P2083	Leadscope ID	http://www.leadscope.com	identifier for a chemical compound in leadscope.com	http://www.leadscope.com/structure_search_results.php?ss_string=$1	LS-2365
 P2085	Nikkaji ID	http://www.genome.jp/kegg	identifier for a chemical compound in the Japan Chemical Substance Dictionary (Nikkaji)	https://jglobal.jst.go.jp/en/redirect?Nikkaji_No=$1	J2.375B|J4.021E
 P2086	CDD Public ID	http://www.collaborativedrug.com/	identifier for a chemical compound in the Collaborative Drug Discovery database	http://www.collaborativedrug.com/public/structures/$1	950699
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/external/wikidata/processed.json` & `bioregistry-0.9.91/src/bioregistry/data/external/wikidata/processed.json`

 * *Files 0% similar despite different names*

#### Pretty-printed

 * *Similarity: 0.9999579407806191%*

 * *Differences: {"'P2040'": "{'uri_format': 'https://speciesplus.net/#/taxon_concepts/$1'}"}*

```diff
@@ -2148,15 +2148,15 @@
             "111",
             "5541"
         ],
         "homepage": "http://www.speciesplus.net/",
         "name": "CITES Species+ ID",
         "pattern": "^\\d{1,5}$",
         "prefix": "P2040",
-        "uri_format": "https://speciesplus.net/#/taxon_concepts/$1/legal"
+        "uri_format": "https://speciesplus.net/#/taxon_concepts/$1"
     },
     "P2057": {
         "database": "Q5937262",
         "description": "ID in Human Metabolome Database",
         "example": [
             "HMDB0002111",
             "HMDB0034235"
```

### Comparing `bioregistry-0.9.90/src/bioregistry/data/metaregistry.json` & `bioregistry-0.9.91/src/bioregistry/data/metaregistry.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/mismatch.json` & `bioregistry-0.9.91/src/bioregistry/data/mismatch.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/data/processing_ols.json` & `bioregistry-0.9.91/src/bioregistry/data/processing_ols.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/cli.py` & `bioregistry-0.9.91/src/bioregistry/export/cli.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/prefix_maps.py` & `bioregistry-0.9.91/src/bioregistry/export/prefix_maps.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/rdf_export.py` & `bioregistry-0.9.91/src/bioregistry/export/rdf_export.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/sssom_export.py` & `bioregistry-0.9.91/src/bioregistry/export/sssom_export.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/tables_export.py` & `bioregistry-0.9.91/src/bioregistry/export/tables_export.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/tsv_export.py` & `bioregistry-0.9.91/src/bioregistry/export/tsv_export.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/warnings_export.py` & `bioregistry-0.9.91/src/bioregistry/export/warnings_export.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/export/yaml_export.py` & `bioregistry-0.9.91/src/bioregistry/export/yaml_export.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/__init__.py` & `bioregistry-0.9.91/src/bioregistry/external/__init__.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/aberowl.py` & `bioregistry-0.9.91/src/bioregistry/external/aberowl.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/bartoc.py` & `bioregistry-0.9.91/src/bioregistry/external/bartoc.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/biocontext.py` & `bioregistry-0.9.91/src/bioregistry/external/biocontext.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/biolink.py` & `bioregistry-0.9.91/src/bioregistry/external/biolink.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/bioportal.py` & `bioregistry-0.9.91/src/bioregistry/external/bioportal.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/cellosaurus.py` & `bioregistry-0.9.91/src/bioregistry/external/cellosaurus.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/cheminf.py` & `bioregistry-0.9.91/src/bioregistry/external/cheminf.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/cropoct.py` & `bioregistry-0.9.91/src/bioregistry/external/cropoct.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/edam.py` & `bioregistry-0.9.91/src/bioregistry/external/edam.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/fairsharing.py` & `bioregistry-0.9.91/src/bioregistry/external/fairsharing.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/go.py` & `bioregistry-0.9.91/src/bioregistry/external/go.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/hl7/OID_Report.csv` & `bioregistry-0.9.91/src/bioregistry/external/hl7/OID_Report.csv`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/hl7/__init__.py` & `bioregistry-0.9.91/src/bioregistry/external/hl7/__init__.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/miriam.py` & `bioregistry-0.9.91/src/bioregistry/external/miriam.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/n2t.py` & `bioregistry-0.9.91/src/bioregistry/external/n2t.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/ncbi.py` & `bioregistry-0.9.91/src/bioregistry/external/ncbi.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/obofoundry.py` & `bioregistry-0.9.91/src/bioregistry/external/obofoundry.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/ols.py` & `bioregistry-0.9.91/src/bioregistry/external/ols.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/ontobee.py` & `bioregistry-0.9.91/src/bioregistry/external/ontobee.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/prefixcommons.py` & `bioregistry-0.9.91/src/bioregistry/external/prefixcommons.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/re3data.py` & `bioregistry-0.9.91/src/bioregistry/external/re3data.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/uniprot.py` & `bioregistry-0.9.91/src/bioregistry/external/uniprot.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/external/wikidata.py` & `bioregistry-0.9.91/src/bioregistry/external/wikidata.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/gh/github_client.py` & `bioregistry-0.9.91/src/bioregistry/gh/github_client.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/gh/new_prefix.py` & `bioregistry-0.9.91/src/bioregistry/gh/new_prefix.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/health/check_homepages.py` & `bioregistry-0.9.91/src/bioregistry/health/check_homepages.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/health/check_providers.py` & `bioregistry-0.9.91/src/bioregistry/health/check_providers.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/health/cli.py` & `bioregistry-0.9.91/src/bioregistry/health/cli.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/license_standardizer.py` & `bioregistry-0.9.91/src/bioregistry/license_standardizer.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/lint.py` & `bioregistry-0.9.91/src/bioregistry/lint.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/metaresource_api.py` & `bioregistry-0.9.91/src/bioregistry/metaresource_api.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/pandas.py` & `bioregistry-0.9.91/src/bioregistry/pandas.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/parse_iri.py` & `bioregistry-0.9.91/src/bioregistry/parse_iri.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/parse_version_iri.py` & `bioregistry-0.9.91/src/bioregistry/parse_version_iri.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/record_accumulator.py` & `bioregistry-0.9.91/src/bioregistry/record_accumulator.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/resolve.py` & `bioregistry-0.9.91/src/bioregistry/resolve.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/resolve_identifier.py` & `bioregistry-0.9.91/src/bioregistry/resolve_identifier.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/resource_manager.py` & `bioregistry-0.9.91/src/bioregistry/resource_manager.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/schema/constants.py` & `bioregistry-0.9.91/src/bioregistry/schema/constants.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/schema/schema.json` & `bioregistry-0.9.91/src/bioregistry/schema/schema.json`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/schema/struct.py` & `bioregistry-0.9.91/src/bioregistry/schema/struct.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/schema/utils.py` & `bioregistry-0.9.91/src/bioregistry/schema/utils.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/schema_utils.py` & `bioregistry-0.9.91/src/bioregistry/schema_utils.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/summary.py` & `bioregistry-0.9.91/src/bioregistry/summary.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/upload_ndex.py` & `bioregistry-0.9.91/src/bioregistry/upload_ndex.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/uri_format.py` & `bioregistry-0.9.91/src/bioregistry/uri_format.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/utils.py` & `bioregistry-0.9.91/src/bioregistry/utils.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry/version.py` & `bioregistry-0.9.91/src/bioregistry/version.py`

 * *Files 1% similar despite different names*

```diff
@@ -8,15 +8,15 @@
 
 __all__ = [
     "VERSION",
     "get_version",
     "get_git_hash",
 ]
 
-VERSION = "0.9.90"
+VERSION = "0.9.91"
 
 
 def get_git_hash() -> Optional[str]:
     """Get the bioregistry git hash."""
     with open(os.devnull, "w") as devnull:
         try:
             ret = check_output(  # noqa: S603,S607
```

### Comparing `bioregistry-0.9.90/src/bioregistry.egg-info/PKG-INFO` & `bioregistry-0.9.91/src/bioregistry.egg-info/PKG-INFO`

 * *Files 0% similar despite different names*

```diff
@@ -1,10 +1,10 @@
 Metadata-Version: 2.1
 Name: bioregistry
-Version: 0.9.90
+Version: 0.9.91
 Summary: Integrated registry of biological databases and nomenclatures
 Home-page: https://github.com/biopragmatics/bioregistry
 Download-URL: https://github.com/biopragmatics/bioregistry/releases
 Author: Charles Tapley Hoyt
 Author-email: cthoyt@gmail.com
 Maintainer: Charles Tapley Hoyt
 Maintainer-email: cthoyt@gmail.com
```

#### html2text {}

```diff
@@ -1,8 +1,8 @@
-Metadata-Version: 2.1 Name: bioregistry Version: 0.9.90 Summary: Integrated
+Metadata-Version: 2.1 Name: bioregistry Version: 0.9.91 Summary: Integrated
 registry of biological databases and nomenclatures Home-page: https://
 github.com/biopragmatics/bioregistry Download-URL: https://github.com/
 biopragmatics/bioregistry/releases Author: Charles Tapley Hoyt Author-email:
 cthoyt@gmail.com Maintainer: Charles Tapley Hoyt Maintainer-email:
 cthoyt@gmail.com License: MIT Project-URL: Bug Tracker, https://github.com/
 biopragmatics/bioregistry/issues Keywords: databases,biological
 databases,biomedical databases Classifier: Development Status :: 4 - Beta
```

### Comparing `bioregistry-0.9.90/src/bioregistry.egg-info/SOURCES.txt` & `bioregistry-0.9.91/src/bioregistry.egg-info/SOURCES.txt`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/src/bioregistry.egg-info/requires.txt` & `bioregistry-0.9.91/src/bioregistry.egg-info/requires.txt`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_api.py` & `bioregistry-0.9.91/tests/test_api.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_collections.py` & `bioregistry-0.9.91/tests/test_collections.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_contexts.py` & `bioregistry-0.9.91/tests/test_contexts.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_data.py` & `bioregistry-0.9.91/tests/test_data.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_data_slow.py` & `bioregistry-0.9.91/tests/test_data_slow.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_duplicates.py` & `bioregistry-0.9.91/tests/test_duplicates.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_identifiers_org.py` & `bioregistry-0.9.91/tests/test_identifiers_org.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_indra.py` & `bioregistry-0.9.91/tests/test_indra.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_manager.py` & `bioregistry-0.9.91/tests/test_manager.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_metaregistry.py` & `bioregistry-0.9.91/tests/test_metaregistry.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_ols.py` & `bioregistry-0.9.91/tests/test_ols.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_pandas.py` & `bioregistry-0.9.91/tests/test_pandas.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_resolve.py` & `bioregistry-0.9.91/tests/test_resolve.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_sparql.py` & `bioregistry-0.9.91/tests/test_sparql.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_usages.py` & `bioregistry-0.9.91/tests/test_usages.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_utils.py` & `bioregistry-0.9.91/tests/test_utils.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_web/test_api.py` & `bioregistry-0.9.91/tests/test_web/test_api.py`

 * *Files identical despite different names*

### Comparing `bioregistry-0.9.90/tests/test_web/test_ui.py` & `bioregistry-0.9.91/tests/test_web/test_ui.py`

 * *Files identical despite different names*

